<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="a8fbb460-bdf5-446b-a0a2-2ded176a7d14" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;issue:issue:pii\:S0092867421X00207;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867422009229;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Accurate de novo design of membrane-traversing macrocycles">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(22)00922-9">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/ec6deb2d-7868-42f9-ac77-9c9d8c1ed6ef/fx1.jpg">
    


        <meta property="og:description" content="An investigation of the design principles of macrocyclic peptide membrane permeability
and oral bioavailability enables the generation of synthetic macrocycles that fold
into the predicted conformation, can cross membranes, and even adopt different conformations
depending on polar versus nonpolar contexts.">
    











        <meta name="twitter:title" content="Accurate de novo design of membrane-traversing macrocycles">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/ec6deb2d-7868-42f9-ac77-9c9d8c1ed6ef/fx1.jpg">
    


        <meta name="twitter:description" content="An investigation of the design principles of macrocyclic peptide membrane permeability
and oral bioavailability enables the generation of synthetic macrocycles that fold
into the predicted conformation, can cross membranes, and even adopt different conformations
depending on polar versus nonpolar contexts.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(22)00922-9">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Accurate de novo design of membrane-traversing macrocycles: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Accurate de novo design of membrane-traversing macrocycles"> <meta name="citation_date" content="2022/09/15"> <meta name="citation_online_date" content="2022/08/29"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="19"> <meta name="citation_firstpage" content="3520"> <meta name="citation_lastpage" content="3532.e26"> <meta name="citation_pmid" content="36041435"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867422009229"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867422009229"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867422009229"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2022.07.019"> <meta name="citation_pmid" content="36041435"> <meta name="citation_author" content="Gaurav Bhardwaj"><meta name="citation_author_email" content="gauravb@uw.edu"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Biological Physics, Structure and Design program, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Jacob O’Connor"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Biological Physics, Structure and Design program, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Department of Biochemistry, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Stephen Rettie"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Molecular Cell and Biology program, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Yen-Hua Huang"><meta name="citation_author_institution" content="Institute for Molecular Bioscience, Australian Research Council Centre of Excellence for Innovations in Peptide and Protein Science, The University of Queensland, Brisbane, QLD 4072, Australia"><meta name="citation_author" content="Theresa A. Ramelot"><meta name="citation_author_institution" content="Department of Chemistry and Chemical Biology and Center for Biotechnology and Interdisciplinary Sciences, Rensselaer Polytechnic Institute, Troy, NY 12180, USA"><meta name="citation_author" content="Vikram Khipple Mulligan"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Gizem Gokce Alpkilic"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Molecular Engineering and Sciences Program, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Jonathan Palmer"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Asim K. Bera"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Matthew J. Bick"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Maddalena Di Piazza"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Xinting Li"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Parisa Hosseinzadeh"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Timothy W. Craven"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Department of Biochemistry, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Roberto Tejero"><meta name="citation_author_institution" content="Departamento de Quίmica Fίsica, Universidad de Valencia, Avenida Dr. Moliner 50, Burjassot, 46100 Valencia, Spain"><meta name="citation_author" content="Anna Lauko"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Biological Physics, Structure and Design program, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Department of Biochemistry, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Ryan Choi"><meta name="citation_author_institution" content="Department of Medicine, Division of Allergy and Infectious Disease, University of Washington, Seattle, WA, USA"><meta name="citation_author" content="Calina Glynn"><meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California-Los Angeles, Los Angeles, CA, USA"><meta name="citation_author" content="Linlin Dong"><meta name="citation_author_institution" content="Takeda Pharmaceuticals Inc., Cambridge, MA, USA"><meta name="citation_author" content="Robert Griffin"><meta name="citation_author_institution" content="Takeda Pharmaceuticals Inc., Cambridge, MA, USA"><meta name="citation_author" content="Wesley C. van Voorhis"><meta name="citation_author_institution" content="Department of Medicine, Division of Allergy and Infectious Disease, University of Washington, Seattle, WA, USA"><meta name="citation_author" content="Jose Rodriguez"><meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California-Los Angeles, Los Angeles, CA, USA"><meta name="citation_author" content="Lance Stewart"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author" content="Gaetano T. Montelione"><meta name="citation_author_institution" content="Department of Chemistry and Chemical Biology and Center for Biotechnology and Interdisciplinary Sciences, Rensselaer Polytechnic Institute, Troy, NY 12180, USA"><meta name="citation_author" content="David Craik"><meta name="citation_author_institution" content="Institute for Molecular Bioscience, Australian Research Council Centre of Excellence for Innovations in Peptide and Protein Science, The University of Queensland, Brisbane, QLD 4072, Australia"><meta name="citation_author" content="David Baker"><meta name="citation_author_email" content="dabaker@uw.edu"><meta name="citation_author_institution" content="Institute for Protein Design, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Department of Biochemistry, University of Washington, Seattle, WA 98195, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, University of Washington, Seattle, WA 98195, USA"><meta name="citation_keywords" content="computational design; peptide design; membrane permeability; oral bioavailability"><meta name="citation_reference" content="Adolf-Bryfogle, J., Labonte, J.W., Kraft, J.C., Shapovalov, M., Raemisch, S., Lütteke, T., DiMaio, F., Bahl, C.D., Pallesen, J., King, N.P., et al. (2021). Growing Glycans in Rosetta: Accurate de novo glycan modeling, density fitting, and rational sequon design."><meta name="citation_reference" content="citation_title=The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design; citation_author=R.F. Alford; citation_author=A. Leaver-Fay; citation_author=J.R. Jeliazkov; citation_author=M.J. O’Meara; citation_author=F.P. DiMaio; citation_author=H. Park; citation_author=M.V. Shapovalov; citation_author=P.D. Renfrew; citation_author=V.K. Mulligan; citation_author=K. Kappel; citation_author=et al.; citation_journal_title=J. Chem. Theory Comput.; citation_volume=13; citation_firstpage=3031; citation_lastpage=3048; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Temperature dependence of 1H chemical shifts in proteins; citation_author=N.J. Baxter; citation_author=M.P. Williamson; citation_journal_title=J. Biomol. NMR; citation_volume=9; citation_firstpage=359; citation_lastpage=369; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Accurate de novo design of hyperstable constrained peptides; citation_author=G. Bhardwaj; citation_author=V.K. Mulligan; citation_author=C.D. Bahl; citation_author=J.M. Gilmore; citation_author=P.J. Harvey; citation_author=O. Cheneval; citation_author=G.W. Buchko; citation_author=S.V.S.R.K. Pulavarti; citation_author=Q. Kaas; citation_author=A. Eletsky; citation_author=et al.; citation_journal_title=Nature; citation_volume=538; citation_firstpage=329; citation_lastpage=335; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Probing the physicochemical boundaries of cell permeability and oral bioavailability in lipophilic macrocycles inspired by natural products; citation_author=A.T. Bockus; citation_author=K.W. Lexa; citation_author=C.R. Pye; citation_author=A.S. Kalgutkar; citation_author=J.W. Gardner; citation_author=K.C.R. Hund; citation_author=W.M. Hewitt; citation_author=J.A. Schwochert; citation_author=E. Glassey; citation_author=D.A. Price; citation_author=et al.; citation_journal_title=J.&nbsp;Med. Chem.; citation_volume=58; citation_firstpage=4581; citation_lastpage=4589; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Bioactive and membrane-permeable cyclic peptide natural products; citation_title=Practical medicinal chemistry with Macrocycles; citation_author=A.T. Bockus; citation_author=R.S. Lokey; citation_firstpage=101; citation_lastpage=132; citation_publication_date=2017; citation_publisher=John Wiley &amp; Sons, Inc, Hoboken, NJ, USA; "><meta name="citation_reference" content="citation_title=Amide proton temperature coefficients as hydrogen bond indicators in proteins; citation_author=T. Cierpicki; citation_author=J. Otlewski; citation_journal_title=J. Biomol. NMR; citation_volume=21; citation_firstpage=249; citation_lastpage=261; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=A kinematic view of loop closure; citation_author=E.A. Coutsias; citation_author=C. Seok; citation_author=M.P. Jacobson; citation_author=K.A. Dill; citation_journal_title=J. Comput. Chem.; citation_volume=25; citation_firstpage=510; citation_lastpage=528; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=NMRPipe: a multidimensional spectral processing system based on UNIX pipes; citation_author=F. Delaglio; citation_author=S. Grzesiek; citation_author=G.W. Vuister; citation_author=G. Zhu; citation_author=J. Pfeifer; citation_author=A. Bax; citation_journal_title=J.&nbsp;Biomol. NMR; citation_volume=6; citation_firstpage=277; citation_lastpage=293; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=High throughput artificial membrane permeability assay for blood-brain barrier; citation_author=L. Di; citation_author=E.H. Kerns; citation_author=K. Fan; citation_author=O.J. McConnell; citation_author=G.T. Carter; citation_journal_title=Eur. J. Med. Chem.; citation_volume=38; citation_firstpage=223; citation_lastpage=232; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Coot: model-building tools for molecular graphics; citation_author=P. Emsley; citation_author=K. Cowtan; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=60; citation_firstpage=2126; citation_lastpage=2132; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=RosettaScripts: a scripting language interface to the Rosetta macromolecular modeling suite; citation_author=S.J. Fleishman; citation_author=A. Leaver-Fay; citation_author=J.E. Corn; citation_author=E.-M. Strauch; citation_author=S.D. Khare; citation_author=N. Koga; citation_author=J. Ashworth; citation_author=P. Murphy; citation_author=F. Richter; citation_author=G. Lemmon; citation_author=et al.; citation_journal_title=PLoS One; citation_volume=6; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Design and development of a cyclic decapeptide scaffold with suitable properties for bioavailability and oral exposure; citation_author=M. Fouché; citation_author=M. Schäfer; citation_author=J. Berghausen; citation_author=S. Desrayaud; citation_author=M. Blatter; citation_author=P. Piéchon; citation_author=I. Dix; citation_author=A. Martin Garcia; citation_author=H.-J. Roth; citation_journal_title=ChemMedChem; citation_volume=11; citation_firstpage=1048; citation_lastpage=1059; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Direct evidence for CH···π interaction mediated stabilization of Pro-cisPro bond in peptides with Pro-Pro-aromatic motifs; citation_author=H.K. Ganguly; citation_author=B. Majumder; citation_author=S. Chattopadhyay; citation_author=P. Chakrabarti; citation_author=G. Basu; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=134; citation_firstpage=4661; citation_lastpage=4669; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=NMR studies of the rates of proline cis–trans isomerization in oligopeptides; citation_author=C. Grathwohl; citation_author=K. Wüthrich; citation_journal_title=Biopolymers; citation_volume=20; citation_firstpage=2623; citation_lastpage=2633; citation_publication_date=1981; "><meta name="citation_reference" content="citation_title=Torsion angle dynamics for NMR structure calculation with the new program DYANA; citation_author=P. Güntert; citation_author=C. Mumenthaler; citation_author=K. Wüthrich; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=273; citation_firstpage=283; citation_lastpage=298; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Protein NMR structure determination with automated NOE assignment using the new software CANDID and the torsion angle dynamics algorithm DYANA; citation_author=T. Herrmann; citation_author=P. Güntert; citation_author=K. Wüthrich; citation_journal_title=J. Mol. Biol.; citation_volume=319; citation_firstpage=209; citation_lastpage=227; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Cell-permeable cyclic peptides from synthetic libraries inspired by natural products; citation_author=W.M. Hewitt; citation_author=S.S.F. Leung; citation_author=C.R. Pye; citation_author=A.R. Ponkey; citation_author=M. Bednarek; citation_author=M.P. Jacobson; citation_author=R.S. Lokey; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=137; citation_firstpage=715; citation_lastpage=721; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Cyclic penta- and Hexaleucine peptides without N-methylation are orally absorbed; citation_author=T.A. Hill; citation_author=R.-J. Lohman; citation_author=H.N. Hoang; citation_author=D.S. Nielsen; citation_author=C.C.G. Scully; citation_author=W.M. Kok; citation_author=L. Liu; citation_author=A.J. Lucke; citation_author=M.J. Stoermer; citation_author=C.I. Schroeder; citation_author=et al.; citation_journal_title=ACS Med. Chem. Lett.; citation_volume=5; citation_firstpage=1148; citation_lastpage=1151; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Comprehensive computational design of ordered peptide macrocycles; citation_author=P. Hosseinzadeh; citation_author=G. Bhardwaj; citation_author=V.K. Mulligan; citation_author=M.D. Shortridge; citation_author=T.W. Craven; citation_author=F. Pardo-Avila; citation_author=S.A. Rettie; citation_author=D.E. Kim; citation_author=D.A. Silva; citation_author=Y.M. Ibrahim; citation_author=et al.; citation_journal_title=Science; citation_volume=358; citation_firstpage=1461; citation_lastpage=1466; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Anchor extension: a structure-guided approach to design cyclic peptides targeting enzyme active sites; citation_author=P. Hosseinzadeh; citation_author=P.R. Watson; citation_author=T.W. Craven; citation_author=X. Li; citation_author=S. Rettie; citation_author=F. Pardo-Avila; citation_author=A.K. Bera; citation_author=V.K. Mulligan; citation_author=P. Lu; citation_author=A.S. Ford; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=12; citation_firstpage=3384; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=ShelXle: a Qt graphical user interface for SHELXL; citation_author=C.B. Hübschle; citation_author=G.M. Sheldrick; citation_author=B. Dittrich; citation_journal_title=J. Appl. Crystallogr.; citation_volume=44; citation_firstpage=1281; citation_lastpage=1284; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Matplotlib: A 2D Graphics; citation_author=J.D. Hunter; citation_journal_title=Environment; citation_volume=9; citation_firstpage=90; citation_lastpage=95; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=XDS; citation_author=W. Kabsch; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=66; citation_firstpage=125; citation_lastpage=132; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Physicochemical high throughput screening: parallel artificial membrane permeation assay in the description of passive absorption processes; citation_author=M. Kansy; citation_author=F. Senner; citation_author=K. Gubernator; citation_journal_title=J. Med. Chem.; citation_volume=41; citation_firstpage=1007; citation_lastpage=1010; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy; citation_author=W. Lee; citation_author=M. Tonelli; citation_author=J.L. Markley; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=1325; citation_lastpage=1327; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling; citation_author=D.J. Mandell; citation_author=E.A. Coutsias; citation_author=T. Kortemme; citation_journal_title=Nat. Methods; citation_volume=6; citation_firstpage=551; citation_lastpage=552; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=cis-Peptide Bonds: A Key for intestinal Permeability of Peptides?; citation_author=U.K. Marelli; citation_author=O. Ovadia; citation_author=A.O. Frank; citation_author=J. Chatterjee; citation_author=C. Gilon; citation_author=A. Hoffman; citation_author=H. Kessler; citation_journal_title=Chemistry; citation_volume=21; citation_firstpage=15148; citation_lastpage=15152; citation_publication_date=2015; "><meta name="citation_reference" content="McKinney, W. (2010). Data structures for statistical computing in python. In Proceedings of the 9th Python in Science Conference, (Austin, TX), pp. 51–56."><meta name="citation_reference" content="citation_title=The emerging role of computational design in peptide macrocycle drug discovery. Expert Opin; citation_author=V.K. Mulligan; citation_journal_title=Drug Discov; citation_volume=15; citation_firstpage=833; citation_lastpage=852; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Computationally designed peptide macrocycle inhibitors of New Delhi metallo-β-lactamase 1; citation_author=V.K. Mulligan; citation_author=S. Workman; citation_author=T. Sun; citation_author=S. Rettie; citation_author=X. Li; citation_author=L.J. Worrall; citation_author=T.W. Craven; citation_author=D.T. King; citation_author=P. Hosseinzadeh; citation_author=A.M. Watkins; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=118; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Trends in peptide drug discovery; citation_author=M. Muttenthaler; citation_author=G.F. King; citation_author=D.J. Adams; citation_author=P.F. Alewood; citation_journal_title=Nat. Rev. Drug Discov; citation_volume=20; citation_doi=10.1038/s41573-020-00135-8; citation_firstpage=309; citation_lastpage=325; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Orally absorbed cyclic peptides; citation_author=D.S. Nielsen; citation_author=N.E. Shepherd; citation_author=W. Xu; citation_author=A.J. Lucke; citation_author=M.J. Stoermer; citation_author=D.P. Fairlie; citation_journal_title=Chem. Rev.; citation_volume=117; citation_firstpage=8094; citation_lastpage=8128; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The effect of multiple N-methylation on intestinal permeability of cyclic hexapeptides; citation_author=O. Ovadia; citation_author=S. Greenberg; citation_author=J. Chatterjee; citation_author=B. Laufer; citation_author=F. Opperer; citation_author=H. Kessler; citation_author=C. Gilon; citation_author=A. Hoffman; citation_journal_title=Mol. Pharm.; citation_volume=8; citation_firstpage=479; citation_lastpage=487; citation_publication_date=2011; "><meta name="citation_reference" content="Park, H., Bradley, P., Greisen, P., Jr, Liu, Y., Mulligan, V.K., Kim, D.E., Baker, D., and DiMaio, F. (2016). Simultaneous Optimization of Biomolecular Energy Functions on Features from Small Molecules and Macromolecules. J. Chem. Theory Comput. 12, 6201–6212."><meta name="citation_reference" content="citation_title=Getting. In emerging methods and design principles for cell-penetrant peptides; citation_author=L. Peraro; citation_author=J.A. Kritzer; citation_journal_title=Angew. Chem.; citation_volume=130; citation_doi=10.1002/ange.201801361; citation_firstpage=12042; citation_lastpage=12057; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Nonclassical size dependence of permeation defines bounds for passive adsorption of large drug molecules; citation_author=C.R. Pye; citation_author=W.M. Hewitt; citation_author=J. Schwochert; citation_author=T.D. Haddad; citation_author=C.E. Townsend; citation_author=L. Etienne; citation_author=Y. Lao; citation_author=C. Limberakis; citation_author=A. Furukawa; citation_author=A.M. Mathiowetz; citation_author=et al.; citation_journal_title=J.&nbsp;Med. Chem.; citation_volume=60; citation_firstpage=1665; citation_lastpage=1672; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Optimizing PK properties of cyclic peptides: the effect of side chain substitutions on permeability and clearance; citation_author=A.C. Rand; citation_author=S.S.F. Leung; citation_author=H. Eng; citation_author=C.J. Rotter; citation_author=R. Sharma; citation_author=A.S. Kalgutkar; citation_author=Y. Zhang; citation_author=M.V. Varma; citation_author=K.A. Farley; citation_author=B. Khunte; citation_author=et al.; citation_journal_title=Medchemcomm; citation_volume=3; citation_firstpage=1282; citation_lastpage=1289; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Incorporation of noncanonical amino acids into Rosetta and use in computational protein-peptide interface design; citation_author=P.D. Renfrew; citation_author=E.J. Choi; citation_author=R. Bonneau; citation_author=B. Kuhlman; citation_journal_title=PLoS One; citation_volume=7; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Conformational flexibility, internal hydrogen bonding, and passive membrane permeability: successful in&nbsp;silico prediction of the relative permeabilities of cyclic peptides; citation_author=T. Rezai; citation_author=J.E. Bock; citation_author=M.V. Zhou; citation_author=C. Kalyanaraman; citation_author=R.S. Lokey; citation_author=M.P. Jacobson; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=128; citation_firstpage=14073; citation_lastpage=14080; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Accelerated Caco-2 cell permeability model for drug discovery; citation_author=E. Sevin; citation_author=L. Dehouck; citation_author=A. Fabulas-da Costa; citation_author=R. Cecchelli; citation_author=M.P. Dehouck; citation_author=S. Lundquist; citation_author=M. Culot; citation_journal_title=J. Pharmacol. Toxicol. Methods; citation_volume=68; citation_firstpage=334; citation_lastpage=339; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=SHELXT--Integrated space-group and crystal-structure determination; citation_author=G.M. Sheldrick; citation_journal_title=Acta Crystallogr. A Found. Adv.; citation_volume=71; citation_firstpage=3; citation_lastpage=8; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Crystal strcuture refinement with SHELXL; citation_author=G.M. Sheldrick; citation_journal_title=Acta Crystallogr. C Struct. Chem.; citation_volume=71; citation_doi=10.1107/s2053229614024218; citation_firstpage=3; citation_lastpage=8; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=PDBStat: a universal restraint converter and restraint analysis software package for protein NMR; citation_author=R. Tejero; citation_author=D. Snyder; citation_author=B. Mao; citation_author=J.M. Aramini; citation_author=G.T. Montelione; citation_journal_title=J.&nbsp;Biomol. NMR; citation_volume=56; citation_firstpage=337; citation_lastpage=351; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Limits on variations in protein backbone dynamics from precise measurements of scalar couplings; citation_author=B. Vögeli; citation_author=J. Ying; citation_author=A. Grishaev; citation_author=A. Bax; citation_journal_title=J. Am. Chem. Soc.; citation_volume=129; citation_firstpage=9377; citation_lastpage=9385; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients; citation_author=C.K. Wang; citation_author=S.E. Northfield; citation_author=B. Colless; citation_author=S. Chaousis; citation_author=I. Hamernig; citation_author=R.J. Lohman; citation_author=D.S. Nielsen; citation_author=C.I. Schroeder; citation_author=S. Liras; citation_author=D.A. Price; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=111; citation_firstpage=17504; citation_lastpage=17509; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Exploring experimental and computational markers of cyclic peptides: Charting islands of permeability; citation_author=C.K. Wang; citation_author=S.E. Northfield; citation_author=J.E. Swedberg; citation_author=B. Colless; citation_author=S. Chaousis; citation_author=D.A. Price; citation_author=S. Liras; citation_author=D.J. Craik; citation_journal_title=Eur. J. Med. Chem.; citation_volume=97; citation_firstpage=202; citation_lastpage=213; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Conformational flexibility is a determinant of permeability for cyclosporin; citation_author=C.K. Wang; citation_author=J.E. Swedberg; citation_author=P.J. Harvey; citation_author=Q. Kaas; citation_author=D.J. Craik; citation_journal_title=J.&nbsp;Phys. Chem. B; citation_volume=122; citation_firstpage=2261; citation_lastpage=2276; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Seaborn: statistical data visualization; citation_author=M.L. Waskom; citation_journal_title=J. Open Source Softw.; citation_volume=6; citation_firstpage=3021; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds; citation_author=T.R. White; citation_author=C.M. Renzelman; citation_author=A.C. Rand; citation_author=T. Rezai; citation_author=C.M. McEwen; citation_author=V.M. Gelev; citation_author=R.A. Turner; citation_author=R.G. Linington; citation_author=S.S.F. Leung; citation_author=A.S. Kalgutkar; citation_author=et al.; citation_journal_title=Nat. Chem. Biol.; citation_volume=7; citation_firstpage=810; citation_lastpage=817; citation_publication_date=2011; "><meta name="citation_reference" content="Winn, M.D., Refinemen, M.-L.S., Pannu, N.J., Murshudov, G.N., and Others (2001). CCP4 v6. 3.0 Program References. Acta Crystallogr. 57, 122–133."><meta name="citation_reference" content="citation_title=NMR structure calculation for all small molecule ligands and non-standard residues from the PDB Chemical Component Dictionary; citation_author=E.M. Yilmaz; citation_author=P. Güntert; citation_journal_title=J. Biomol. NMR; citation_volume=63; citation_firstpage=21; citation_lastpage=37; citation_publication_date=2015; "><meta name="citation_abstract" content="<h2>Summary</h2><p>We use computational design coupled with experimental characterization to systematically investigate the design principles for macrocycle membrane permeability and oral bioavailability. We designed 184 6–12 residue macrocycles with a wide range of predicted structures containing noncanonical backbone modifications and experimentally determined structures of 35; 29 are very close to the computational models. With such control, we show that membrane permeability can be systematically achieved by ensuring all amide (NH) groups are engaged in internal hydrogen bonding interactions. 84 designs over the 6–12 residue size range cross membranes with an apparent permeability greater than 1&nbsp;×&nbsp;10<sup>−6</sup> cm/s. Designs with exposed NH groups can be&nbsp;made membrane permeable through the design of an alternative isoenergetic fully hydrogen-bonded state favored in the lipid membrane. The ability to robustly design membrane-permeable and orally bioavailable peptides&nbsp;with high structural accuracy should contribute to the next generation of designed macrocycle therapeutics.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7707&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(22)00922-9&amp;be=3297&amp;fe=7418&amp;dc=7080&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749594448361,%22n%22:0,%22f%22:2565,%22dn%22:2566,%22dne%22:2566,%22c%22:2566,%22s%22:2566,%22ce%22:2566,%22rq%22:2566,%22rp%22:3281,%22rpe%22:3951,%22dl%22:3286,%22di%22:4317,%22ds%22:7080,%22de%22:7274,%22dc%22:7418,%22l%22:7418,%22le%22:7513%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c50 {
  padding: 0.683em 0.681em 0 0;
}

mjx-c.mjx-c61 {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c70 {
  padding: 0.442em 0.556em 0.194em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c6C {
  padding: 0.694em 0.278em 0 0;
}

mjx-c.mjx-c6E {
  padding: 0.442em 0.556em 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c43 {
  padding: 0.705em 0.722em 0.022em 0;
}

mjx-c.mjx-c41 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c2F {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c65 {
  padding: 0.448em 0.444em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c2217 {
  padding: 0.462em 0.5em 0 0;
}

mjx-c.mjx-c56 {
  padding: 0.683em 0.75em 0.022em 0;
}

mjx-c.mjx-c44 {
  padding: 0.683em 0.764em 0 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c74 {
  padding: 0.615em 0.389em 0.011em 0;
}

mjx-c.mjx-c3B {
  padding: 0.431em 0.278em 0.193em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900922-9" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00922-9"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(22)00922-9"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900922-9" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-75181" aria-expanded="false" aria-controls="login-links-75181" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-75181" id="login-links--75181" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900922-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900922-9" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900922-9" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-75181" data-opens="#links-dialog-75181" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-75181" aria-modal="true"><button data-closes="#links-dialog-75181" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900922-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900922-9" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900922-9" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2022.07.019">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(22)00922-9" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0020-7"><span>Volume 185</span>, <span>Issue 19</span></a></span><span class="meta-panel__pages"><span>p3520-3532.e26</span></span><span class="meta-panel__onlineDate">September 15, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=19&amp;pii=S0092-8674%2821%29X0020-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Accurate <i>de novo</i> design of membrane-traversing macrocycles</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Gaurav</span> <span property="familyName">Bhardwaj</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gaurav</span> <span property="familyName">Bhardwaj</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:gauravb@uw.edu" property="email" title="Send email to Gaurav Bhardwaj">gauravb@uw.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn4"><div id="ntpara0025" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Biological Physics, Structure and Design program, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bhardwaj/Gaurav" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">17</a></sup> <a id="eadd0010" href="mailto:gauravb@uw.edu" property="email" title="Send email to Gaurav Bhardwaj">gauravb@uw.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Jacob</span> <span property="familyName">O’Connor</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jacob</span> <span property="familyName">O’Connor</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn4"><div id="ntpara0025" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Biological Physics, Structure and Design program, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/O%E2%80%99Connor/Jacob" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">17</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Stephen</span> <span property="familyName">Rettie</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stephen</span> <span property="familyName">Rettie</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Molecular Cell and Biology program, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rettie/Stephen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Yen-Hua</span> <span property="familyName">Huang</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yen-Hua</span> <span property="familyName">Huang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Molecular Bioscience, Australian Research Council Centre of Excellence for Innovations in Peptide and Protein Science, The University of Queensland, Brisbane, QLD 4072, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Huang/Yen-Hua" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Theresa A.</span> <span property="familyName">Ramelot</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Theresa A.</span> <span property="familyName">Ramelot</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Chemical Biology and Center for Biotechnology and Interdisciplinary Sciences, Rensselaer Polytechnic Institute, Troy, NY 12180, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ramelot/Theresa+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Vikram Khipple</span> <span property="familyName">Mulligan</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Vikram Khipple</span> <span property="familyName">Mulligan</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn3"><div id="ntpara0020" role="paragraph">Present address: Center for Computational Biology, Flatiron Institute, New York, NY 10010, USA</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Mulligan/Vikram+Khipple" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">16</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Gizem Gokce</span> <span property="familyName">Alpkilic</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gizem Gokce</span> <span property="familyName">Alpkilic</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Molecular Engineering and Sciences Program, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Alpkilic/Gizem+Gokce" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Jonathan</span> <span property="familyName">Palmer</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jonathan</span> <span property="familyName">Palmer</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Palmer/Jonathan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Asim K.</span> <span property="familyName">Bera</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Asim K.</span> <span property="familyName">Bera</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bera/Asim+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Matthew J.</span> <span property="familyName">Bick</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matthew J.</span> <span property="familyName">Bick</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">Present address: Neoleukin Therapeutics, Seattle, WA 98102, USA</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bick/Matthew+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Maddalena</span> <span property="familyName">Di Piazza</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Maddalena</span> <span property="familyName">Di Piazza</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Di+Piazza/Maddalena" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Xinting</span> <span property="familyName">Li</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xinting</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Xinting" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Parisa</span> <span property="familyName">Hosseinzadeh</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Parisa</span> <span property="familyName">Hosseinzadeh</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn2"><div id="ntpara0015" role="paragraph">Present address: Phil and Penny Knight Campus, University of Oregon, Eugene, OR 97403, USA</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hosseinzadeh/Parisa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Timothy W.</span> <span property="familyName">Craven</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Timothy W.</span> <span property="familyName">Craven</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Craven/Timothy+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Roberto</span> <span property="familyName">Tejero</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Roberto</span> <span property="familyName">Tejero</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Departamento de Quίmica Fίsica, Universidad de Valencia, Avenida Dr. Moliner 50, Burjassot, 46100 Valencia, Spain</span></div></div><div class="core-author-link"><a href="/authored-by/Tejero/Roberto" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Anna</span> <span property="familyName">Lauko</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anna</span> <span property="familyName">Lauko</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Biological Physics, Structure and Design program, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lauko/Anna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Ryan</span> <span property="familyName">Choi</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ryan</span> <span property="familyName">Choi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Division of Allergy and Infectious Disease, University of Washington, Seattle, WA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Choi/Ryan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Calina</span> <span property="familyName">Glynn</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Calina</span> <span property="familyName">Glynn</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Biochemistry, University of California-Los Angeles, Los Angeles, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Glynn/Calina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Linlin</span> <span property="familyName">Dong</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Linlin</span> <span property="familyName">Dong</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Takeda Pharmaceuticals Inc., Cambridge, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Dong/Linlin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Robert</span> <span property="familyName">Griffin</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Robert</span> <span property="familyName">Griffin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Takeda Pharmaceuticals Inc., Cambridge, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Griffin/Robert" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Wesley C.</span> <span property="familyName">van Voorhis</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Wesley C.</span> <span property="familyName">van Voorhis</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Division of Allergy and Infectious Disease, University of Washington, Seattle, WA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/van+Voorhis/Wesley+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Jose</span> <span property="familyName">Rodriguez</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jose</span> <span property="familyName">Rodriguez</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Biochemistry, University of California-Los Angeles, Los Angeles, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rodriguez/Jose" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Lance</span> <span property="familyName">Stewart</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lance</span> <span property="familyName">Stewart</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Stewart/Lance" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Gaetano T.</span> <span property="familyName">Montelione</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gaetano T.</span> <span property="familyName">Montelione</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Chemical Biology and Center for Biotechnology and Interdisciplinary Sciences, Rensselaer Polytechnic Institute, Troy, NY 12180, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Montelione/Gaetano+T" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">David</span> <span property="familyName">Craik</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David</span> <span property="familyName">Craik</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Molecular Bioscience, Australian Research Council Centre of Excellence for Innovations in Peptide and Protein Science, The University of Queensland, Brisbane, QLD 4072, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Craik/David" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">David</span> <span property="familyName">Baker</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David</span> <span property="familyName">Baker</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:dabaker@uw.edu" property="email" title="Send email to David Baker">dabaker@uw.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">18</div><div id="fn5"><div id="ntpara0030" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Baker/David" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a role="doc-noteref">18</a></sup> <a id="eadd0015" href="mailto:dabaker@uw.edu" property="email" title="Send email to David Baker">dabaker@uw.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Institute for Protein Design, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Medicinal Chemistry, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Biological Physics, Structure and Design program, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Biochemistry, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Molecular Cell and Biology program, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Institute for Molecular Bioscience, Australian Research Council Centre of Excellence for Innovations in Peptide and Protein Science, The University of Queensland, Brisbane, QLD 4072, Australia</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Department of Chemistry and Chemical Biology and Center for Biotechnology and Interdisciplinary Sciences, Rensselaer Polytechnic Institute, Troy, NY 12180, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Molecular Engineering and Sciences Program, University of Washington, Seattle, WA 98195, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Departamento de Quίmica Fίsica, Universidad de Valencia, Avenida Dr. Moliner 50, Burjassot, 46100 Valencia, Spain</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Department of Medicine, Division of Allergy and Infectious Disease, University of Washington, Seattle, WA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Department of Chemistry and Biochemistry, University of California-Los Angeles, Los Angeles, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Takeda Pharmaceuticals Inc., Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">Howard Hughes Medical Institute, University of Washington, Seattle, WA 98195, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">Present address: Neoleukin Therapeutics, Seattle, WA 98102, USA</div></div></div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn2"><div id="ntpara0015" role="paragraph">Present address: Phil and Penny Knight Campus, University of Oregon, Eugene, OR 97403, USA</div></div></div><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn3"><div id="ntpara0020" role="paragraph">Present address: Center for Computational Biology, Flatiron Institute, New York, NY 10010, USA</div></div></div><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn4"><div id="ntpara0025" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">18</div><div id="fn5"><div id="ntpara0030" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received December 29, 2021</span>; <span>Revised May 1, 2022</span>; <span>Accepted July 21, 2022</span>; <span>Published online August 29, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2022.07.019" property="sameAs" target="_blank">10.1016/j.cell.2022.07.019 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867422009229" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2022 The Author(s). Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution (CC BY 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2022.07.019" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0020-7"><img src="/cms/asset/fa11a010-9223-4062-83b2-1de023b46670/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 19"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2822%2900922-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (10)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900922-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.07.019">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2822%2900922-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2022.07.019&amp;pii=S0092-8674%2822%2900922-9" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900922-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.07.019">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2022.07.019" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867422009229&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867422009229&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=19&amp;pii=S0092-8674%2821%29X0020-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00992-8" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(22)00995-3" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (10)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Computational design of diverse permeable macrocycles beyond the “rule-of-five” space</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">X-ray and NMR structures of designed macrocycles match their computational models</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Designed macrocycles are permeable <i>in&nbsp;vitro</i> and orally bioavailable <i>in&nbsp;vivo</i></div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Designed chameleonic peptides show solvent-dependent conformational switching</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">We use computational design coupled with experimental characterization to systematically investigate the design principles for macrocycle membrane permeability and oral bioavailability. We designed 184 6–12 residue macrocycles with a wide range of predicted structures containing noncanonical backbone modifications and experimentally determined structures of 35; 29 are very close to the computational models. With such control, we show that membrane permeability can be systematically achieved by ensuring all amide (NH) groups are engaged in internal hydrogen bonding interactions. 84 designs over the 6–12 residue size range cross membranes with an apparent permeability greater than 1&nbsp;×&nbsp;10<sup>−6</sup> cm/s. Designs with exposed NH groups can be&nbsp;made membrane permeable through the design of an alternative isoenergetic fully hydrogen-bonded state favored in the lipid membrane. The ability to robustly design membrane-permeable and orally bioavailable peptides&nbsp;with high structural accuracy should contribute to the next generation of designed macrocycle therapeutics.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/656782f0-335f-4119-b0fc-e304c5710beb/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2022.07.019/asset/ec6deb2d-7868-42f9-ac77-9c9d8c1ed6ef/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/656782f0-335f-4119-b0fc-e304c5710beb/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;computational+design&quot;" alt="Search on this keyword" rel="nofollow">computational design</a></li><li><a href="/action/doSearch?AllField=&quot;peptide+design&quot;" alt="Search on this keyword" rel="nofollow">peptide design</a></li><li><a href="/action/doSearch?AllField=&quot;membrane+permeability&quot;" alt="Search on this keyword" rel="nofollow">membrane permeability</a></li><li><a href="/action/doSearch?AllField=&quot;oral+bioavailability&quot;" alt="Search on this keyword" rel="nofollow">oral bioavailability</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0060" role="paragraph">Macrocyclic peptides have considerable potential as therapeutics with advantages over small molecules in the ability to disrupt protein-protein interactions, and over proteins in metabolic stability and ability to cross biological membranes (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Muttenthaler et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Muttenthaler, M. ∙ King, G.F. ∙ Adams, D.J. ...</div><div><strong>Trends in peptide drug discovery</strong></div><div><em>Nat. Rev. Drug Discov.</em> 2021; <strong>20</strong>:309-325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41573-020-00135-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85100542944" target="_blank" aria-label="Open Scopus in new tab">Scopus (924)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33536635/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41573-020-00135-8&amp;pmid=33536635" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Peptides with intrinsic membrane permeability can access intracellular drug targets, translocate across intestinal epithelial cells enabling oral delivery, and penetrate the blood-brain barrier by traversing brain microvascular endothelial cells. Naturally occurring macrocycles, such as cyclosporine A and griselimycin, suggest potential mechanisms for permeability of large macrocycles that are beyond the traditional “rule of five” (Ro5) (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Bockus and Lokey, 2017</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bockus, A.T. ∙ Lokey, R.S.</div><div><strong>Bioactive and membrane-permeable cyclic peptide natural products</strong></div><div><em>Practical medicinal chemistry with Macrocycles.</em>John Wiley &amp; Sons, Inc, Hoboken, NJ, USA, 2017 101-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/9781119092599.ch5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F9781119092599.ch5" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Bockus et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bockus, A.T. ∙ Lexa, K.W. ∙ Pye, C.R. ...</div><div><strong>Probing the physicochemical boundaries of cell permeability and oral bioavailability in lipophilic macrocycles inspired by natural products</strong></div><div><em>J.&nbsp;Med. Chem.</em> 2015; <strong>58</strong>:4581-4589</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jmedchem.5b00128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84935898517" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25950816/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jmedchem.5b00128&amp;pmid=25950816" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Hewitt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Hewitt, W.M. ∙ Leung, S.S.F. ∙ Pye, C.R. ...</div><div><strong>Cell-permeable cyclic peptides from synthetic libraries inspired by natural products</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2015; <strong>137</strong>:715-721</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja508766b" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84921484823" target="_blank" aria-label="Open Scopus in new tab">Scopus (176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja508766b&amp;pmid=25517352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Nielsen et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Nielsen, D.S. ∙ Shepherd, N.E. ∙ Xu, W. ...</div><div><strong>Orally absorbed cyclic peptides</strong></div><div><em>Chem. Rev.</em> 2017; <strong>117</strong>:8094-8128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.chemrev.6b00838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85021636298" target="_blank" aria-label="Open Scopus in new tab">Scopus (295)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28541045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.chemrev.6b00838&amp;pmid=28541045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, adapting the insights from a&nbsp;few naturally permeable macrocycles to develop new membrane-permeable peptides of diverse shapes and sizes has proven very difficult. Studies of naturally permeable macrocycles have identified some common features, such as low polar surface area and lack of unsatisfied hydrogen bond donors (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Bockus et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bockus, A.T. ∙ Lexa, K.W. ∙ Pye, C.R. ...</div><div><strong>Probing the physicochemical boundaries of cell permeability and oral bioavailability in lipophilic macrocycles inspired by natural products</strong></div><div><em>J.&nbsp;Med. Chem.</em> 2015; <strong>58</strong>:4581-4589</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jmedchem.5b00128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84935898517" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25950816/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jmedchem.5b00128&amp;pmid=25950816" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Peraro and Kritzer, 2018</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Peraro, L. ∙ Kritzer, J.A.</div><div><strong>Getting. In emerging methods and design principles for cell-penetrant peptides</strong></div><div><em>Angew. Chem.</em> 2018; <strong>130</strong>:12042-12057</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ange.201801361" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fange.201801361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Rand et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Rand, A.C. ∙ Leung, S.S.F. ∙ Eng, H. ...</div><div><strong>Optimizing PK properties of cyclic peptides: the effect of side chain substitutions on permeability and clearance</strong></div><div><em>Medchemcomm.</em> 2012; <strong>3</strong>:1282-1289</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c2md20203d" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84867025688" target="_blank" aria-label="Open Scopus in new tab">Scopus (116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23133740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc2md20203d&amp;pmid=23133740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Rezai et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Rezai, T. ∙ Bock, J.E. ∙ Zhou, M.V. ...</div><div><strong>Conformational flexibility, internal hydrogen bonding, and passive membrane permeability: successful in&nbsp;silico prediction of the relative permeabilities of cyclic peptides</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2006; <strong>128</strong>:14073-14080</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja063076p" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-33750482579" target="_blank" aria-label="Open Scopus in new tab">Scopus (321)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17061890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja063076p&amp;pmid=17061890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conformational switching between different structures in aqueous and membrane environments has also been described as a potential approach to achieve passive membrane permeability (<span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Peraro and Kritzer, 2018</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Peraro, L. ∙ Kritzer, J.A.</div><div><strong>Getting. In emerging methods and design principles for cell-penetrant peptides</strong></div><div><em>Angew. Chem.</em> 2018; <strong>130</strong>:12042-12057</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ange.201801361" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fange.201801361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Wang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Wang, C.K. ∙ Swedberg, J.E. ∙ Harvey, P.J. ...</div><div><strong>Conformational flexibility is a determinant of permeability for cyclosporin</strong></div><div><em>J.&nbsp;Phys. Chem. B.</em> 2018; <strong>122</strong>:2261-2276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jpcb.7b12419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85042759162" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29400464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jpcb.7b12419&amp;pmid=29400464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">White&nbsp;et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>White, T.R. ∙ Renzelman, C.M. ∙ Rand, A.C. ...</div><div><strong>On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</strong></div><div><em>Nat. Chem. Biol.</em> 2011; <strong>7</strong>:810-817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nchembio.664" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-80054853098" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21946276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnchembio.664&amp;pmid=21946276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The incorporation of these features into library-based methods has enabled the development of several new membrane-permeable cyclic peptides. However, previous work has largely been limited to cases of smaller peptides, usually 5–7 amino acids (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Hewitt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Hewitt, W.M. ∙ Leung, S.S.F. ∙ Pye, C.R. ...</div><div><strong>Cell-permeable cyclic peptides from synthetic libraries inspired by natural products</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2015; <strong>137</strong>:715-721</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja508766b" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84921484823" target="_blank" aria-label="Open Scopus in new tab">Scopus (176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja508766b&amp;pmid=25517352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Hill et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Hill, T.A. ∙ Lohman, R.-J. ∙ Hoang, H.N. ...</div><div><strong>Cyclic penta- and Hexaleucine peptides without N-methylation are orally absorbed</strong></div><div><em>ACS Med. Chem. Lett.</em> 2014; <strong>5</strong>:1148-1151</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ml5002823" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84907923894" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25313329/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fml5002823&amp;pmid=25313329" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Wang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Wang, C.K. ∙ Northfield, S.E. ∙ Colless, B. ...</div><div><strong>Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:17504-17509</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1417611111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84916618640" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25416591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1417611111&amp;pmid=25416591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">White et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>White, T.R. ∙ Renzelman, C.M. ∙ Rand, A.C. ...</div><div><strong>On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</strong></div><div><em>Nat. Chem. Biol.</em> 2011; <strong>7</strong>:810-817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nchembio.664" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-80054853098" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21946276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnchembio.664&amp;pmid=21946276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, extending these principles to the design of novel membrane-permeable peptides with&nbsp;extensive chemical and structural diversity has remained challenging, as it requires an understanding of the relationships between structure, flexibility, and permeability and the ability to accurately control the sequence and structural features of macrocycles simultaneously (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Mulligan, 2020</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Mulligan, V.K.</div><div><strong>The emerging role of computational design in peptide macrocycle drug discovery. Expert Opin</strong></div><div><em>Drug Discov.</em> 2020; <strong>15</strong>:833-852</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.K.MulliganThe+emerging+role+of+computational+design+in+peptide+macrocycle+drug+discovery.+Expert+OpinDrug+Discov152020833852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0065" role="paragraph">Here, we take advantage of the ability of computational design to specify macrocycle structure to systematically explore the determinants of passive membrane permeability (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We use a design-build-test approach where we design peptides containing different structural features, determine their crystal and solution structures, and evaluate their permeability. We consider three structural features: the satisfaction of all hydrogen bond donors through the formation of intrapeptide hydrogen bonds, the presence of <i>cis</i>-peptide bonds, and the ability to switch conformations between aqueous and lipid environments.</div></section><section id="sec-2"><h2>Results</h2><div id="p0070" role="paragraph">We first investigated whether designed macrocycles with widely diverse lengths and structures sharing only the property of full internal satisfaction of all NH groups could robustly traverse lipid membranes. We extended the Rosetta generalized kinematic closure (genKIC) method to stochastically generate ensembles of ∼10<sup>6</sup> N-to-C cyclic backbone conformations for 6–12 residue polyglycine peptides, sampling cyclic backbone conformations by selecting phi/psi torsions randomly from flat-bottom symmetric Ramachandran tables (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). From these large sets, we selected backbones that make at least two intramolecular hydrogen bonds and carried&nbsp;out Rosetta combinatorial sequence design restricting L- and D-amino acids to negative and positive phi regions of Ramachandran space, respectively, and incorporating conformationally constrained amino acids, such as L-proline, D-proline, and α-aminoisobutyric acid (AIB), at structurally compatible sites. To eliminate exposed and unsatisfied hydrogen bond donors, amino acids with an unsatisfied hydrogen bond donor in the backbone were mutated to their N-methylated variants, and only nonpolar amino acids were allowed during the sequence design step (see <a id="crosref0380" href="#sec-4">STAR Methods</a> and <a id="crosrefs0010" href="#figs1">Figure&nbsp;S1</a>A). We selected low-energy designs with two or more intramolecular hydrogen bonds and five or fewer N-methylated amino acids. The conformational energy landscape for selected designs was characterized by generating 10<sup>5</sup>–10<sup>6</sup> alternative conformations and evaluating the energy&nbsp;and backbone RMSD to the design model. We chose sequences with funnel-like energy landscapes converging on their corresponding design models (see <a id="crosref0385" href="#sec-4">STAR Methods</a> and <a id="crosrefs0015" href="#figs1">Figure&nbsp;S1</a>B). Overall structural diversity was assessed using a backbone torsion angle-based clustering method (<span class="dropBlock reference-citations"><a id="body-ref-sref12-3" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-3" aria-controls="sref12-3">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Each residue was assigned a torsion bin (A&nbsp;[right-handed helical region], B [right-handed strand region], X [mirror of A], Y [mirror of B], O [amino acids with phi&nbsp;&lt;&nbsp;0 and <i>cis</i>-peptide bond between the residue i and i+1], and Z [amino acids with phi&nbsp;&gt;&nbsp;0 and <i>cis</i>-peptide bond between the residues i&nbsp;and i+1]), and the resultant torsion bin strings (for example, XYABOX for a six residue peptide) were clustered. Because the choice of a starting residue is arbitrary in a cycle, and both&nbsp;passive membrane permeability and fold propensity are invariant to mirroring, clusters with bin strings that transform into each&nbsp;other under circular permutation or mirror&nbsp;inversion were combined, and members of the resulting nondegenerate clusters&nbsp;(which we represent by the lowest alphabetical order&nbsp;of the torsion bin string over all permutations and inversions) were selected for chemical synthesis and experimental characterization.</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/802874cb-242e-4d8f-915e-580cbee57048/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2022.07.019/asset/0c4121e7-9a77-441f-8d2b-4976ce5ffd69/main.assets/figs1.jpg" height="600" width="656" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/802874cb-242e-4d8f-915e-580cbee57048/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Design and selection of membrane-permeable peptides, related to <a id="crosref1025" href="#fig1">Figures&nbsp;1</a> and <a id="crosref1030" href="#fig3">3</a> and <a id="crosref1035" href="#sec-4">STAR Methods</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0095" role="paragraph">(A) Overall schematic of the <i>in silico</i> pipeline for the design of membrane-permeable peptides. Design process starts with a linear polyglycine peptide chain that is cyclized using Rosetta generalized kinematic closure (genKIC) protocol. Iterative rounds of amino acid sequence design and N-methylation of non-hydrogen-bonded NH groups are performed to design low-energy macrocycles with no unsatisfied backbone NH groups. The process is repeated to sample 10<sup>5</sup>–10<sup>6</sup> design models that are clustered to identify permeable macrocycles with diverse shapes and sizes.</div><div id="fspara0100" role="paragraph">(B) An example energy versus RMSD to design plot from structure prediction runs using Rosetta simple_cycpep_predict application. Diverse conformations for a given amino acid sequence are generated using generalized kinematic closure (genKIC) protocol and energy-minimized using Rosetta FastRelax protocol. Each orange point represents an independently predicted structure. Blue dots represent the local minimization of the designed macrocycle structure. Landscapes that funnel into the design structure as the lowest energy structure and have a big energy gap (ΔE) between the designed fold and other unfolded states are selected for experimental characterization.</div></div></div></div></figcaption></figure></div><section id="sec-2-1"><h3>Membrane permeability of 6–8 residue designed macrocycles</h3><div id="p0075" role="paragraph">We first tested the ability of our design pipeline to control both&nbsp;macrocycle structure and membrane permeability on 6–8 residue macrocycles spanning a diversity of structures and N-methylation patterns. For 6, 7, and 8 residue macrocycles, we selected 8, 5, and 19 designs (representing 6, 5, and 16 clusters), respectively, with fully satisfied backbone NH groups and funnel-like energy landscapes and spanning a diversity of sequences, structures, N-methylation patterns, and structural motifs (see <a id="crosref0395" href="#mmc1">Data S1</a> and <a id="crosref0400" href="#figs2">S2</a> for sequences and design models). Selected macrocycles were chemically synthesized and purified using reverse-phase high-performance liquid chromatography (RP-HPLC). In some cases, two peaks were observed with the expected MW; in such cases we experimentally tested the two peaks separately and refer to these as p1 and p2 in the design names.</div><div id="p0080" role="paragraph">To evaluate the accuracy of the design method, we determined X-ray crystal structures for two 7 residue designs and fifteen 8 residue designs. The structures for two 7 residue and twelve of the 8 residue macrocycles were very close to the computational design models (backbone atom RMSD&nbsp;&lt;&nbsp;1.2&nbsp;Å) (<a id="crosref0405" href="#fig1">Figure&nbsp;1</a>; <a id="crosref0410" href="#mmc1">Table&nbsp;S1</a>). In seven cases with remarkably low RMSDs below 0.5&nbsp;Å, the design models are within the experimental resolution of the X-ray data. Three of the designs, D8.1, D8.2, and D8.12, are internally symmetric: D8.1 with a backbone RMSD to the design model of 0.21&nbsp;Å has an internal S2 mirror symmetry composed of four beta turns and two gamma turn motifs, D8.2 has C2 symmetry with a combination of 4 beta turns and 2 alpha turns (RMSD 0.24&nbsp;Å), and D8.12 has C2 symmetry with four internal hydrogen bonds and no N-methylated amino acids (RMSD 0.48&nbsp;Å) (<a id="crosref0415" href="#fig1">Figure&nbsp;1</a>). Design D8.5.p2 with no N-methylated amino acids is stabilized by four internal hydrogen bonds forming four beta turns and an alpha turn motif (RMSD 0.24&nbsp;Å). D8.10 with 0.35&nbsp;Å RMSD between model and X-ray structure has three N-methylated amino acids and two proline residues in its sequence; its other three amino acids are involved in three internal hydrogen bonds. D7.6 features three beta turns and single N-methylated amino acid and matches very closely with the design (RMSD 0.35&nbsp;Å). D7.8 has five internal hydrogen bonds and no N-methylated amino acids (RMSD 0.5&nbsp;Å). Overall, the close matches between the experimental structures and the design models validate that our approach can very accurately specify macrocycle structure (<a id="crosref0420" href="#mmc1">Table&nbsp;S1</a>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/4427ce97-40df-4997-87eb-c7d6b25358f5/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2022.07.019/asset/e461f39f-573e-4e68-b764-c71cb7b42049/main.assets/gr1.jpg" height="938" width="722" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/4427ce97-40df-4997-87eb-c7d6b25358f5/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Computational design and structure validation of 6–8 amino acid macrocycles</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">Structural validation of computationally designed macrocycles. Each panel shows the design model and torsion bin string describing the design model (left), hydrogen bonding pattern for the design model (middle), and superposition between the design model (blue) and the X-ray structure (orange) (right). For hydrogen bonding graphs, the orange boxes highlight the designed intramolecular hydrogen bonds. Amino acids without a backbone hydrogen bond donor (proline, D-proline, and N-methylated amino acids) are marked by darker gray columns. Sidechains for non proline residues not shown for clarity in the superposition graphs. RMSD between the design model and X-ray structure was calculated over all backbone heavy atoms (C, CA, N, O, and CN). <sup>∗</sup> in the macrocycle sequence denotes the N-methylated amino acid positions and lower case denotes the D-amino acids.</div><div id="fspara0020" role="paragraph">See also <a id="crosref0980" href="#mmc1">Table&nbsp;S1</a>, <a id="crosrefs0020" href="#figs1">Figures&nbsp;S1–S4</a>, and <a id="crosref0985" href="#mmc1">Data S1</a> and <a id="crosref0990" href="#figs2">S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Having found that the macrocycles fold as designed, we next investigated their membrane permeabilities using transwell permeability assays. The rates of traversal across artificial membranes in parallel artificial membrane permeability assays (PAMPAs) (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Di et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Di, L. ∙ Kerns, E.H. ∙ Fan, K. ...</div><div><strong>High throughput artificial membrane permeability assay for blood-brain barrier</strong></div><div><em>Eur. J. Med. Chem.</em> 2003; <strong>38</strong>:223-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0223-5234(03)00012-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84962359573" target="_blank" aria-label="Open Scopus in new tab">Scopus (978)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12667689/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0223-5234%2803%2900012-6&amp;pmid=12667689" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were determined by mass-spectrometry-based quantification of peptide concentrations in the donor&nbsp;and acceptor wells (see <a id="crosref0430" href="#sec-4">STAR Methods</a>). Eight 6-mers, five 7-mers, and sixteen 8-mers had apparent permeabilities (P<sub>app</sub>) greater than 1&nbsp;×&nbsp;10<sup>−7</sup> cm/s (see <a id="crosref0435" href="#mmc1">Data S3</a>). Of these, eight 6-mers, five 7-mers, and ten 8-mers had P<sub>app</sub> greater than 1&nbsp;×&nbsp;10<sup>−6</sup> cm/s (<a id="crosref0440" href="#fig2">Figure&nbsp;2</a>A). PAMPA is a good measure of passive transport across lipid membranes; however, oral bioavailability <i>in&nbsp;vivo</i> involves cellular barriers with efflux transporters. Therefore, we tested a subset of the designed macrocycles in transwell Caco-2 assays, with colorectal epithelial cells as the barrier between the donor and acceptor wells, and in the opposite direction of the efflux transport. In Caco-2 assays, permeability greater than 1&nbsp;×&nbsp;10<sup>−6</sup> cm/s is considered indicative of adequate cellular permeability for a candidate drug. Because Caco-2 assays are resource intensive, we focused on testing the larger-sized 8 residue macrocycles and did not carry out this assay on the 6 and 7 residue macrocycles. We again observed high permeability (<a id="crosref0445" href="#fig2">Figure&nbsp;2</a>B) in Caco-2 assays: out of the eight 8-mer designs tested in Caco-2 assays, 6 designs had P<sub>app</sub> greater than 1&nbsp;×&nbsp;10<sup>−6</sup> cm/s, and 4 designs higher than 1&nbsp;×&nbsp;10<sup>−5</sup> cm/s. Design D8.1 with no N-methylated amino acids showed a very high P<sub>app</sub> of 23.27&nbsp;×&nbsp;10<sup>−6</sup> cm/s.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/0dbb1d4b-9f61-440d-b8be-cdba784b83bc/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2022.07.019/asset/8c9f7083-17e4-4670-8341-4e6839a94063/main.assets/gr2.jpg" height="648" width="656" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/0dbb1d4b-9f61-440d-b8be-cdba784b83bc/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Permeability measurements of computationally designed macrocycles in PAMPA and Caco-2 assays</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0030" role="paragraph">(A) Apparent permeability (P<sub>app</sub>) of 6–12 amino acid macrocycles in PAMPA assay. Peptides are grouped based on sequence length. Isobaric peaks (denoted p1 and p2) were seen for some peptides during purification and were assayed separately. Bar height: average P<sub>app</sub> from three replicates; error bars: standard deviation calculated from three replicates.</div><div id="fspara0035" role="paragraph">(B) Apparent permeability (P<sub>app</sub>) of designed 8–12 amino acid macrocycles (salmon-colored bars) measured in the apical to the basal direction in the Caco-2 assays. P<sub>app</sub> for quinidine and atenolol used as negative and positive controls (gray-colored bars). Bar height: average P<sub>app</sub> from three replicates; error bars: standard deviation calculated from three replicates.</div><div id="fspara0040" role="paragraph">See also <a id="crosref0995" href="#mmc1">Data S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph">The membrane-traversing macrocycles with P<sub>app</sub> greater than 1&nbsp;×&nbsp;10<sup>−6</sup> cm/s and funnel-like energy landscapes cover a wide range of structures, populating six, five, and nine clusters for 6, 7, and 8 residue macrocycles, respectively. While previous studies with natural scaffolds and library-based methods have identified permeable macrocycles, these usually require multiple N-methylated or N-alkylated amino acids. For example, 7 of 11 amino acids in cyclosporine are N-methylated, which comes at the cost of increased molecular flexibility and synthetic difficulties. In our 6–8 residue permeable designs, the number of N-methylated amino acids ranges between 0 and 3. Five of the structurally validated designs, D7.8, D8.1, D8.2, D8.5, and D8.12, have no N-methyl amino acids highlighting the precise control offered by computational methods to design structures with complete hydrogen bond donor satisfaction through internal hydrogen bonding and prolines (<a id="crosref0450" href="#fig1">Figure&nbsp;1</a>; <a id="crosref0455" href="#mmc1">Data S2</a>). D8.1, D8.2, D8.5.p2, and D8.12 are the largest passively permeable macrocycles we are aware of that lack N-methylated or N-alkylated amino acids. The extensive internal hydrogen bonding enables full NH bond satisfaction in these designs with no N-methylated amino acids; designs D8.1 and D8.2 have six internal hydrogen bonds that stabilize the structure. Some of the 6 and 7 residue design models (D6.3, D6.5, D6.9, and D7.8) contain a geometrically strained arrangement of overlapping beta and gamma turns (<a id="crosref0460" href="#figs2">Figure&nbsp;S2</a>), in which the middle amino acid may have a partially or fully unsatisfied NH group. Accordingly, we also designed and tested variants with the middle&nbsp;amino acid N-methylated, and in all cases, the variant with the additional N-methylation is more permeable than the original. However, across all the permeable designs, permeability does not correlate with the number of N-methylated amino&nbsp;acids; for example, D8.1 is the most permeable 8 residue design in PAMPA and Caco-2 assays and does not have any N-methylated amino acids. This illustrates that maximizing hydrogen bond satisfaction and using N-methylation sparingly in the folded structure is a viable strategy for achieving permeability.</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/0fe1203e-2b5a-42c9-99c5-4f1490038fd9/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2022.07.019/asset/aaf53044-475e-4ddc-823f-30ca63620616/main.assets/figs2.jpg" height="537" width="497" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/0fe1203e-2b5a-42c9-99c5-4f1490038fd9/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">N-methylation of geometrically strained turn types, related to <a id="crosref1040" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0110" role="paragraph">Examples of geometrically strained arrangement of overlapping gamma and beta turns seen in some of the design models. For such designs, variants with N-methylated middle residue were also generated and tested experimentally. N-methyls are shown as orange. Intramolecular hydrogen bonding interactions are shown as green dashes.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Design of membrane-permeable 9–12 residue macrocycles</h3><div id="p0095" role="paragraph">Earlier work on passively permeable peptides has been primarily limited to 5–7 amino acids with a few cases of larger peptide derivatives of natural products (<span class="dropBlock reference-citations"><a id="body-ref-sref10-3" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-3" aria-controls="sref10-3">Hewitt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Hewitt, W.M. ∙ Leung, S.S.F. ∙ Pye, C.R. ...</div><div><strong>Cell-permeable cyclic peptides from synthetic libraries inspired by natural products</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2015; <strong>137</strong>:715-721</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja508766b" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84921484823" target="_blank" aria-label="Open Scopus in new tab">Scopus (176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja508766b&amp;pmid=25517352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Hill et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Hill, T.A. ∙ Lohman, R.-J. ∙ Hoang, H.N. ...</div><div><strong>Cyclic penta- and Hexaleucine peptides without N-methylation are orally absorbed</strong></div><div><em>ACS Med. Chem. Lett.</em> 2014; <strong>5</strong>:1148-1151</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ml5002823" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84907923894" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25313329/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fml5002823&amp;pmid=25313329" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Wang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Wang, C.K. ∙ Northfield, S.E. ∙ Colless, B. ...</div><div><strong>Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:17504-17509</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1417611111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84916618640" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25416591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1417611111&amp;pmid=25416591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26-3" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-3" aria-controls="sref26-3">White et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref26-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>White, T.R. ∙ Renzelman, C.M. ∙ Rand, A.C. ...</div><div><strong>On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</strong></div><div><em>Nat. Chem. Biol.</em> 2011; <strong>7</strong>:810-817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nchembio.664" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-80054853098" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21946276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnchembio.664&amp;pmid=21946276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This is because purely lipophilic peptides show a steep decrease in permeability with size, as observed in a study of 8-mer, 9-mer, and 10-mer peptides with all their residues N-methylated (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Pye et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Pye, C.R. ∙ Hewitt, W.M. ∙ Schwochert, J. ...</div><div><strong>Nonclassical size dependence of permeation defines bounds for passive adsorption of large drug molecules</strong></div><div><em>J.&nbsp;Med. Chem.</em> 2017; <strong>60</strong>:1665-1672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jmedchem.6b01483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85015084736" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28059508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jmedchem.6b01483&amp;pmid=28059508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To determine whether our design principles can circumvent this trend, we used our computational pipeline to design a wide variety of larger macrocycles ranging from 9 to 12 amino acids.</div><div id="p0100" role="paragraph">We selected for synthesis and characterization seventeen 9-mer, forty-one 10-mer, nineteen 11-mer, and eight 12-mer macrocycles with funnel-like energy landscapes spanning 16, 37, 18, and 8 different structural clusters, respectively, with widely ranging structures and between 1 and 6 N-methylated amino acids (see <a id="crosref0490" href="#mmc1">Data files S1</a> and <a id="crosref0495" href="#figs2">S2</a> for sequences and design models). 50 designs spanning 12, 23, 13, and 2 structural clusters for 9, 10, 11, and 12 residue macrocycles, respectively, were permeable in PAMPA assays with apparent permeability greater than 1&nbsp;×&nbsp;10<sup>−7</sup> cm/s (see <a id="crosref0500" href="#mmc1">Data S3</a>). Out of these, ten 9-mers, sixteen 10-mers, nine 11-mers, and one 12-mer showed significant permeability (P<sub>app</sub>&nbsp;&gt;&nbsp;1&nbsp;×&nbsp;10<sup>−6</sup> cm/s) in PAMPA assays (<a id="crosref0505" href="#fig2">Figure&nbsp;2</a>A). There was a size dependence in their permeability, but the drop-off is less steep than what has been observed in previous studies on non-designed macrocycles, resulting in significant permeabilities beyond the typical size range for permeable drug like compounds ("rule of five”, see&nbsp;introduction). In Caco-2 assays, three 9-mer, eight 10-mer, seven 11-mer designs, and one 12-mer had P<sub>app</sub> greater than 1&nbsp;×&nbsp;10<sup>−6</sup> cm/s (<a id="crosref0510" href="#fig2">Figure&nbsp;2</a>B). Despite their large size, multiple 10 and 11 residue designs show considerably high P<sub>app</sub> in Caco-2 assays: D10.19, D10.30, and D10.34 have rates greater than 1&nbsp;×&nbsp;10<sup>−5</sup> cm/s, and three other designs have P<sub>app</sub> between 0.5 and 1.0&nbsp;×&nbsp;10<sup>−5</sup> cm/s. One 11-mer design, D11.3, has a high P<sub>app</sub> of 9.11&nbsp;×&nbsp;10<sup>−6</sup> cm/s in Caco-2 assays (<a id="crosref0515" href="#fig2">Figure&nbsp;2</a>B; <a id="crosref0520" href="#mmc1">Data S3</a>). Most of the 12 residue designs tested were not permeable, but D12.6 has a P<sub>app</sub> value of 1.47&nbsp;×&nbsp;10<sup>−6</sup> cm/s and 2.84&nbsp;×&nbsp;10<sup>−6</sup> cm/s in the PAMPA and Caco-2 assays, respectively (<a id="crosref0525" href="#fig2">Figure&nbsp;2</a>B).</div><div id="p0105" role="paragraph">To evaluate the structural accuracy of our design models and to confirm structure-activity relationships present in the design models and membrane permeability data described above, we sought to determine their experimental structures. We successfully crystallized and solved the high-resolution X-ray crystal structures for five 9-mers, six 10-mers, and four 11-mers. Out of these 15 structures, three 9-mers, five 10-mers, and four 11-mer macrocycles matched closely (backbone RMSD of 1.2&nbsp;Å or less) with their design models (<a id="crosref0530" href="#fig3">Figure&nbsp;3</a>; <a id="crosref0535" href="#mmc1">Table&nbsp;S2</a>). Design D9.8 (RMSD 0.33&nbsp;Å) features three N-methylated amino acids, and the structure is stabilized by 1 alpha turn, 2 beta turns, and 1 gamma turn. D10.31 (RMSD 0.45&nbsp;Å) has two N-methylated amino acids and five internal hydrogen bonds stabilizing the macrocyclic structure; structures solved in isopropanol:water and ethyl acetate:pentane mixtures were identical. Design D10.1 contains five intramolecular hydrogen bonds, and the crystal structure is nearly identical to the design model (backbone RMSD of 0.27&nbsp;Å and all heavy atom RMSD of 0.47&nbsp;Å). Designs&nbsp;D10.21, D10.22, and D10.23 each contain five N-methylated amino acids; D10.21 is stabilized by three internal hydrogen bonds and two prolines, while D10.22 and D10.23 have two internal hydrogen bonds (backbone RMSDs of 0.9, 0.82, and 0.41&nbsp;Å, respectively). D11.3 and D11.4 have five internal hydrogen bonds, and their crystal structures have backbone RMSD less than 0.55&nbsp;Å to the design model. Design D11.1, with a backbone RMSD of 0.43&nbsp;Å between the design model and the X-ray crystal structure, contains five internal hydrogen bonds, 2 N-methylated amino acids, and three prolines.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/b60f7b64-6b0d-400a-a3a4-fb089de8f51a/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2022.07.019/asset/77f62e5e-8f4b-4e5d-9cc0-85e56cdae71f/main.assets/gr3.jpg" height="937" width="720" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/b60f7b64-6b0d-400a-a3a4-fb089de8f51a/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Computational design and structure validation of 9–12 amino acid macrocycles</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0050" role="paragraph">Structural validation of computationally designed macrocycles. Each panel shows the design model and torsion bin string describing the design model (left), hydrogen bonding pattern for the design model (middle), and superposition between the design model (blue) and the X-ray structure (orange) (right). For hydrogen bonding graphs, the orange boxes highlight the designed intramolecular hydrogen bonds. Amino acids without a backbone hydrogen bond donor (proline, D-proline, and N-methylated amino acids) are marked by darker gray columns. Side chains for non proline residues not shown for clarity in the superposition graphs. RMSD between the design model and X-ray structure was calculated over all the backbone heavy atoms (C, CA, N, O, and CN). <sup>∗</sup> in macrocycle sequence denotes the N-methylated amino acid positions and lowercase denotes the D-amino acids.</div><div id="fspara0055" role="paragraph">See also <a id="crosref1000" href="#mmc1">Table&nbsp;S2</a>, <a id="crosref1105" href="#figs1">Figures&nbsp;S1</a>, <a id="crosref1110" href="#figs3">S3</a>, and <a id="crosref1115" href="#figs5">S5</a>, and <a id="crosref1005" href="#mmc1">Data S1</a> and <a id="crosref1010" href="#figs2">S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">The structural accuracy of our designs coupled with a large number of permeability measurements provides insight into the relationship between permeability and NH satisfaction in these larger peptides. Overall, almost all peptides without exposed NH groups were membrane permeable. In contrast to results with nondesigned macrocycles, there was not a strong correlation between permeability and size; indeed some of the 10 and 11 residue peptides were highly permeable. Crystal structures of macrocycles that were not permeable or show low permeability (<a id="crosref0540" href="#figs3">Figure&nbsp;S3</a>) further support the importance of NH satisfaction: these did not match the fully satisfied design models and included exposed polar groups. Although hydrogen bond satisfaction appears necessary for permeability, it is not sufficient: X-ray structures for D10.21, D10.22, and D10.23 matched very closely (RMSD&nbsp;&lt;&nbsp;1&nbsp;Å) with their design models and had no unsatisfied NH groups but were not permeable in transwell assays.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/3e127695-8b39-4033-bbef-6f610dea2dab/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2022.07.019/asset/1d0a9623-f021-4c53-b10c-5af6eae00854/main.assets/figs3.jpg" height="450" width="656" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/3e127695-8b39-4033-bbef-6f610dea2dab/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Effect of exposed and unsatisfied polar groups on macrocycle permeability, related to <a id="crosref1045" href="#fig1">Figures&nbsp;1</a> and <a id="crosref1050" href="#fig3">3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0120" role="paragraph">Designs with exposed polar NH or OH groups in the X-ray crystal structures (orange sticks) are not permeable or show low permeability in PAMPA. Dashed black lines denote the intramolecular hydrogen bonds. The exposed NH or OH groups are denoted by the arrows.</div><div id="fspara0125" role="paragraph">See <a id="crosref1055" href="#mmc1">Data S2</a> and <a id="crosref1060" href="#figs3">S3</a> for design models, structures, and permeability data.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-3"><h3>Permeability of <i>cis</i>-peptide bond-containing macrocycles</h3><div id="p0115" role="paragraph">Our results thus far establish that computational design for satisfying all NHs can robustly generate highly membrane-permeable designs well beyond the Ro5 limits. We next investigated other possible contributions to membrane permeability. It has been proposed that <i>cis</i>-peptide bonds can enhance permeability (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Marelli et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Marelli, U.K. ∙ Ovadia, O. ∙ Frank, A.O. ...</div><div><strong>cis-Peptide Bonds: A Key for intestinal Permeability of Peptides?</strong></div><div><em>Chemistry.</em> 2015; <strong>21</strong>:15148-15152</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/chem.201501600" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84944711138" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26337831/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fchem.201501600&amp;pmid=26337831" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>cis</i>-peptide bonds are present in a number of our designs. X-ray crystal structure of design D8.31 has <i>cis</i>-peptide bonds at the two N-methylated D-leucines at residue positions 3 and 8 that are part of a rare beta-turn formed by a Nme-D-Pro(i+1)–Nme-D-aa(i+2) motif. Design D8.13 contains three prolines with one designed to be in a <i>cis</i>-peptide bond stabilized by an aromatic AA(i+1)–Proline(i+2) motif recapitulated in the crystal structure (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Ganguly et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Ganguly, H.K. ∙ Majumder, B. ∙ Chattopadhyay, S. ...</div><div><strong>Direct evidence for CH···π interaction mediated stabilization of Pro-cisPro bond in peptides with Pro-Pro-aromatic motifs</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2012; <strong>134</strong>:4661-4669</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja209334v" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84858233524" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22335630/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja209334v&amp;pmid=22335630" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). D8.6 contains two N-methylated amino acids in a row and four intramolecular hydrogen bonds; one of the N-methylated D-alanine undergoes a <i>trans</i>-to-<i>cis</i> switch in the X-ray crystal structure, but because the switch happens around an N-methylated amino acid, the overall NH satisfaction in the macrocycle is still maintained. Similarly, in design D8.9, an N-methylated D-leucine at amino acid position 3 undergoes a <i>trans</i>-to-<i>cis</i> switch but maintains the overall satisfaction of the peptide backbone, and the crystal structure of D9.16 also contains a <i>trans</i>-to-<i>cis</i> omega flip relative to the design model.</div><div id="p0120" role="paragraph">Over our macrocycle set, there is little association between the presence of <i>cis</i>-peptide bonds and the extent of membrane permeability. The permeabilities of <i>cis</i>-peptide containing D8.9, D8.13, D8.14, D8.15, D8.6, D9.13, and D10.62 are in the same range as those of all-<i>trans</i> macrocycles with the same number of residues. The secondary contribution of <i>cis</i>-peptide bonds is further illustrated by D8.13, D8.14, and D8.15, which have similar sequences and structures (<a id="crosref0555" href="#figs4">Figure&nbsp;S4</a>) with <i>cis</i>-peptide bonds at the same position; D8.13 is not membrane permeable, and D8.14 and D8.15 have PAMPA P<sub>app</sub> of 9.71&nbsp;×&nbsp;10<sup>−7</sup> and 7.68&nbsp;×&nbsp;10<sup>−7</sup> cm/s. D8.13 has a tyrosine residue in place of a phenylalanine residue, creating an unsatisfied OH group that likely prevents permeability.</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/c058313f-7cf4-4e46-92b8-46e132a5f2c1/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2022.07.019/asset/fcba320f-97d1-4bec-86d3-60ddc0a528eb/main.assets/figs4.jpg" height="274" width="759" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/c058313f-7cf4-4e46-92b8-46e132a5f2c1/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Differences in permeability of macrocycles with <i>cis</i>-peptide bonds, related to <a id="crosref1065" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0135" role="paragraph">Superposition between the designed structure (blue sticks) and X-ray structure (orange sticks) of three closely related <i>cis</i>-peptide bond-containing designs, D8.13 (left panel), D8.14 (middle panel), and D8.15 (right panel). All three design models match closely (RMSD over all backbone atoms [N, CA, C, O, and CN]&nbsp;&lt;&nbsp;1.0&nbsp;Å) with respective X-ray structures. All three design models feature a <i>cis</i>-peptide bond in the validated structures. However, the D8.13 is not permeable in PAMPA, while both D8.14 and D8.15 show significant permeability (P<sub>app</sub>&nbsp;&gt;&nbsp;1&nbsp;×&nbsp;10<sup>−7</sup> cm/s), indicating that <i>cis</i>-peptide bond alone is not enough to drive permeability in these macrocycles.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-4"><h3>Design of membrane-permeable chameleonic macrocycles</h3><div id="p0125" role="paragraph">While the above results suggest that <i>cis</i>-peptide bonds do not inherently increase membrane permeability, we reasoned that <i>cis</i>-<i>trans</i> isomerization of the peptide bond could be a powerful design principle for generating peptides with both an open state, with polar groups exposed to interact with a therapeutic target, and a membrane-permeable closed state, with all NH groups making intrapeptide hydrogen bonds (<span class="dropBlock reference-citations"><a id="body-ref-sref26-4" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-4" aria-controls="sref26-4">White et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref26-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>White, T.R. ∙ Renzelman, C.M. ∙ Rand, A.C. ...</div><div><strong>On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</strong></div><div><em>Nat. Chem. Biol.</em> 2011; <strong>7</strong>:810-817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nchembio.664" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-80054853098" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21946276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnchembio.664&amp;pmid=21946276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The relatively slow (milliseconds timescale) kinetics of <i>cis</i>-<i>trans</i> isomerization allows for a peptide to populate multiple conformations that can interconvert fast enough to be biologically relevant but are distinguishable by NMR (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Grathwohl and Wüthrich, 1981</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Grathwohl, C. ∙ Wüthrich, K.</div><div><strong>NMR studies of the rates of proline cis–trans isomerization in oligopeptides</strong></div><div><em>Biopolymers.</em> 1981; <strong>20</strong>:2623-2633</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bip.1981.360201209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84985715908" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbip.1981.360201209" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). There is some evidence that isomerization around the <i>cis</i>-peptide bond in cyclosporine interconverts distinct binding-competent and membrane-permeable states (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Wang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Wang, C.K. ∙ Swedberg, J.E. ∙ Harvey, P.J. ...</div><div><strong>Conformational flexibility is a determinant of permeability for cyclosporin</strong></div><div><em>J.&nbsp;Phys. Chem. B.</em> 2018; <strong>122</strong>:2261-2276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jpcb.7b12419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85042759162" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29400464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jpcb.7b12419&amp;pmid=29400464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To test this hypothesis, we designed macrocycles that undergo <i>cis</i>-<i>trans</i> isomerization. D11.25 provides a start in this direction: the structure prediction calculations identified two very similar conformations that differ by a <i>trans</i>-to-<i>cis</i>-peptide bond flip&nbsp;around the only N-methylated amino acid in the sequence (<a id="crosref0575" href="#figs5">Figure&nbsp;S5</a>). The all-<i>trans</i> conformation is stabilized by six intramolecular hydrogen bonds that satisfy all the available NH groups, while the <i>cis</i> form, which is closely recapitulated in the crystal structure (backbone RMSD 0.53&nbsp;Å), exposes one NH group from D-leucine at position 10 (<a id="crosref0580" href="#fig3">Figure&nbsp;3</a>; <a id="crosref0585" href="#mmc1">Figure&nbsp;S13</a>).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/ac26d099-81c6-4695-a142-c250f0214b1f/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2022.07.019/asset/2232c03d-b9b8-4c70-915d-1a3f13652659/main.assets/figs5.jpg" height="458" width="759" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/ac26d099-81c6-4695-a142-c250f0214b1f/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text"><i>ccis</i>-<i>trans</i> isomerization of the peptide bonds generates alternative low-energy states, related to <a id="crosref1070" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0145" role="paragraph">(A) Structure prediction calculations for the design D11.25 sequence show two low-energy states. Orange points: conformations with no <i>cis</i>-peptide bonds; gray points: predicted conformations with at least one <i>cis</i>-peptide bonds; and blue points: conformations generated by the local energy minimization of the design model.</div><div id="fspara0150" role="paragraph">(B and C) (B) Lowest energy “<i>trans</i>” state with all peptide bonds in <i>trans</i> conformation, (C)&nbsp;lowest energy “<i>cis</i>” state with the N-methylated amino acid in the <i>cis</i> conformation.</div><div id="fspara0155" role="paragraph">(D) X-ray structure for D11.25 matches the <i>cis</i> state and exposed NH group from a D-leucine. The position of <i>cis</i>-peptide bond is highlighted in the dashed square. <sup>∗</sup> in the <i>cis</i> state denotes the position of the unsatisfied NH groups.</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph">We set out to systematically design chameleonic peptides with two distinct energy minima differing by isomerization around a peptide bond: one that has no exposed amides and can traverse membranes, and one that exposes amides to solution and hence has the potential to bind polar target sites. We used three approaches to identify peptides that can populate multiple distinct but almost isoenergetic states (with differences in Rosetta calculated energies of less than 5&nbsp;kcal/mol). First, we used large-scale structure prediction calculations to generate energy landscapes for many designs (<a id="crosref0590" href="#mmc1">Data file S1</a>), searched for those with two or more minima, and identified 45 with Rosetta calculated energy differences between the two states of less than 5&nbsp;kcal/mol. Second,&nbsp;for 20 additional cases, where the calculated energy difference between the states was &gt;5&nbsp;kcal/mol, we developed a genetic algorithm-based multistate design method (see <a id="crosref0595" href="#sec-4">STAR Methods</a>) to optimize the sequence such&nbsp;that&nbsp;the two alternative states have similar energies. Full&nbsp;energy landscape calculations were then performed for these new sequences to confirm the presence of two isoenergetic minima. Third, starting from crystallographically confirmed designs, we introduced destabilizing mutations that led to a second low-energy minimum in the energy landscape, and generated an additional two peptides predicted to adopt two states.</div><div id="p0135" role="paragraph">We synthesized 67 macrocycles spanning two 6-mers, two&nbsp;7-mers, fourteen 8-mers, twenty-five 9-mers, seventeen 10-mers, five 11-mers, and two 12-mers predicted to have alternative low-energy states. Out of these, 50 macrocycles have PAMPA P<sub>app</sub> greater than 1&nbsp;×&nbsp;10<sup>−7</sup> cm/s, and 25 show significant apparent permeability (P<sub>app</sub>&nbsp;&gt;&nbsp;1&nbsp;×&nbsp;10<sup>−6</sup> cm/s) (<a id="crosref0600" href="#fig2">Figure&nbsp;2</a>A; <a id="crosref0605" href="#mmc1">Data&nbsp;S3</a>). 9 out of the 10 designs tested in Caco-2 permeability assays show significant permeability: four 8-mers, two 9-mers, one 10-mer, and two 11-mers show P<sub>app</sub> greater than 1&nbsp;×&nbsp;10<sup>−6</sup> cm/s. Designs D8.21 and D8.33 show Caco-2 P<sub>app</sub> greater than 1&nbsp;×&nbsp;10<sup>−5</sup> cm/s (<a id="crosref0610" href="#fig2">Figure&nbsp;2</a>B). We selected 19 peptides that showed significant permeability for further studies with NMR. The lower success rate in these designs is consistent with their greater polarity, the challenges in designing multiple states, and possibly much slower membrane transversal rates limited by the kinetics of conformational isomerization. 1D <sup>1</sup>H NMR in d<sub>6</sub>-DMSO and CDCl<sub>3</sub> indicated that the equilibrium between the structured conformations was strongly solvent dependent for seven of these designs, with switching between conformational states, or ratios of conformational states, between polar and nonpolar solvents. Energy landscape calculations also revealed that switch design generally resulted in more than two discrete low-energy states—an expected consequence of introducing interactions that favor more than one state. We clustered the low-energy ensembles from structure prediction calculations for each macrocycle and assigned each distinct minimum a state (cluster) identifier ranked on the lowest energy structure in that cluster. For three of these seven macrocycles (D8.21, D8.31, and D9.16) designed with the first approach, we succeeded in solving crystal structures matching the design model or one of the predicted alternate low-energy structures (<a id="crosref0615" href="#fig4">Figure&nbsp;4</a>). We then carried out more detailed solution NMR studies of these designs using 2D NOESY, ROESY, TOCSY, <sup>13</sup>C-<sup>1</sup>H HSQC, and <sup>13</sup>C-<sup>1</sup>H TOCSY-HSQC experiments in CDCl<sub>3</sub>, d<sub>6</sub>-DMSO, and a 50:50 d<sub>6</sub>-DMSO/H<sub>2</sub>O mixture (see <a id="crosref0620" href="#sec-4">STAR Methods</a>), assigning the prolines and N-methylated amino acids in each conformation as either <i>trans</i> or <i>cis</i> peptide conformations. The heteronuclear NMR studies were conducted using <sup>1</sup>H detection of natural abundance <sup>13</sup>C and <sup>15</sup>N nuclei. These NMR data were then analyzed to determine structures of multiple states, simultaneously, of each peptide, providing a total of 11 solution NMR structures for three peptides in various solvents (<a id="crosref0625" href="#fig4">Figures&nbsp;4</a> and <a id="crosref0630" href="#figs6">S6</a>; <a id="crosref0635" href="#mmc1">Tables&nbsp;S3</a> and <a id="crosref0640" href="#mmc1">S4</a>).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/52a2e762-bc2f-4b00-a112-9f318a04a122/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2022.07.019/asset/82a17271-4456-4327-a13b-a9d8900e8ee6/main.assets/gr4.jpg" height="430" width="759" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/52a2e762-bc2f-4b00-a112-9f318a04a122/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Design and structural characterization of conformation switching macrocycles</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0065" role="paragraph">Design models and experimentally determined structures (X-ray and NMR) for different conformational states of designs D8.31 (left), D8.21 (middle), and D9.16 (right). The design model and predicted low-energy states are shown in the top row. The superposition between the predicted low-energy states and the experimental structures is shown in the gray boxes. The solvent conditions for the NMR structures and low-energy states in superposition plots are indicated by similar colors in the labels and structures. In conditions with multiple conformations, the relative percentage of each conformation in the solution is also indicated. All three designs show solvent-dependent changes in the populations and switching between at least two different conformations. The conformational switch in D8.31 does not change the number of unsatisfied NHs, but for both D8.21 and D9.16, the states with fewer unsatisfied NHs are favored in the nonpolar solvents.</div><div id="fspara0070" role="paragraph">See also <a id="crosref1120" href="#mmc1">Tables&nbsp;S3</a> and <a id="crosref1125" href="#mmc1">S4</a>, and <a id="crosref1140" href="#figs6">Figures S6, S7, S8, S9, and S10</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/3a2f3410-db7f-4913-9f76-7bbbff57493a/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2022.07.019/asset/d81db9c2-5b18-4d77-9df4-c3b6743c3020/main.assets/figs6.jpg" height="937" width="587" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/3a2f3410-db7f-4913-9f76-7bbbff57493a/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">NMR structures in different solvents, related to <a id="crosref1075" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0165" role="paragraph">NMR-derived structures in the indicated solvents d<sub>6</sub>-DMSO, CDCl<sub>3</sub>, or 50:50 d<sub>6</sub>-DMSO/H<sub>2</sub>O (DMSO<sub>50</sub>) are shown for D9.16, D8.31, and D8.21. The overlay structure of the ensemble of 20 lowest energy structures is shown along with the backbone structure of the medoid conformation with NH protons labeled. The amide proton temperature coefficients Δδ(<sup>1</sup>H)/ΔT are given for each of the HN resonances in each conformation. Less negative coefficients indicate increased hydrogen bond propensity and correlate with hydrogen bonds in the structures. In general, upon increasing temperature, amide <sup>1</sup>H chemical shifts move upfield, which is attributed to a lengthening of the hydrogen bond and decreased shielding from the hydrogen bond acceptor (<span class="dropBlock reference-citations"><a id="body-ref-optGbjjMMdkkU" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optGbjjMMdkkU" aria-controls="optGbjjMMdkkU">Baxter and Williamson, 1997)</a><div class="dropBlock__holder" data-db-target-of="optGbjjMMdkkU"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Baxter, N.J. ∙ Williamson, M.P.</div><div><strong>Temperature dependence of 1H chemical shifts in proteins</strong></div><div><em>J. Biomol. NMR.</em> 1997; <strong>9</strong>:359-369</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1018334207887" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0031160103" target="_blank" aria-label="Open Scopus in new tab">Scopus (456)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9255942/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1018334207887&amp;pmid=9255942" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Large changes in chemical shift give large negative temperature coefficients and indicate solvent exposed or weakly hydrogen-bonded NHs. It was empirically found that for proteins in aqueous solution that amide protons with Δδ(<sup>1</sup>H)/ΔT that are more positive than −4.6 ppm/K (less negative and even sometimes positive) are indicative of intramolecular hydrogen bonds (<span class="dropBlock reference-citations"><a id="body-ref-optnjAB8SH7Mv" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optnjAB8SH7Mv" aria-controls="optnjAB8SH7Mv">Cierpicki and Otlewski, 2001)</a><div class="dropBlock__holder" data-db-target-of="optnjAB8SH7Mv"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Cierpicki, T. ∙ Otlewski, J.</div><div><strong>Amide proton temperature coefficients as hydrogen bond indicators in proteins</strong></div><div><em>J. Biomol. NMR.</em> 2001; <strong>21</strong>:249-261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1012911329730" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0035215289" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11775741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1012911329730&amp;pmid=11775741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. For cyclic peptides, temperature coefficients have been used as a measure of hydrogen bonding potential and correlated with MD simulations and predicted structures in aqueous solution as well as in chloroform and DMSO (<span class="dropBlock reference-citations"><a id="body-ref-optogFgrOAoxu" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optogFgrOAoxu" aria-controls="optogFgrOAoxu">Wang et al., 2015)</a><div class="dropBlock__holder" data-db-target-of="optogFgrOAoxu"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Wang, C.K. ∙ Northfield, S.E. ∙ Swedberg, J.E. ...</div><div><strong>Exploring experimental and computational markers of cyclic peptides: Charting islands of permeability</strong></div><div><em>Eur. J. Med. Chem.</em> 2015; <strong>97</strong>:202-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ejmech.2015.04.049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84929084835" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25974856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ejmech.2015.04.049&amp;pmid=25974856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. In most cases, the NOESY or ROESY data with the <sup>3</sup>J<sub>hnha</sub> coupling-derived dihedral restraints was sufficient to give a converged structure for the peptides. However, in the case of the <i>trans</i>-<i>trans</i> variant of D8.21, the symmetry and the more open conformation with fewer NOEs did not give a unique conformational solution. In these cases, the conformation that best correlated the temperature coefficients was selected.</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">Peptide D8.31 is an 8 amino acid macrocycle with a symmetric repeat sequence (ap<sup>∗</sup>lvap<sup>∗</sup>lv, <sup>∗</sup> represents N-methylated amino acids); the lowest energy state (LE_1) is C2 symmetric with both N-methylated amino acids in <i>cis</i> peptide bond conformations (“<i>cis</i>-<i>cis</i>”), and the second-lowest energy alternative state (LE_2) is asymmetric with one N-methylated leucine in the <i>cis</i>&nbsp;conformation (“<i>cis</i>-<i>trans</i>”) (<a id="crosref0645" href="#fig4">Figure&nbsp;4</a>, left panel; <a id="crosref0650" href="#figs6">Figures&nbsp;S6</a> and&nbsp;<a id="crosref0655" href="#figs7">S7</a>). The <i>cis</i>-<i>trans</i> isomerization occurs around an N-methylated amino acid; hence, both states have no unsatisfied NH groups. The crystal structure in the ethyl acetate:pentane solution is similar to the <i>cis</i>-<i>cis</i> LE_1. In d<sub>6</sub>-DMSO solution, two structures are present in slow exchange, a <i>cis</i>-<i>cis</i> conformation (∼50%) similar to LE_1, and <i>cis</i>-<i>trans</i> conformation (∼50%) similar to the LE_2 (<a id="crosref0660" href="#fig4">Figure&nbsp;4</a>, left panels). In nonpolar CDCl<sub>3</sub> solution, two conformations are also observed, with the dominant form (∼70%) matching the LE_1 state (<a id="crosref0665" href="#fig4">Figure&nbsp;4</a>, left panels). The correspondence between these experimental X-ray crystal structures, solution NMR structures, and the predicted low-energy states for D8.31 demonstrates that Rosetta calculations can guide the design of macrocycles adopting multiple states. However, as the two different states have the same number of exposed NHs, these data do not directly address the contribution of conformational switching to membrane permeability. More relevant are the two other macrocycles, D8.21 and D9.16.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/9f132870-6b8c-4144-8850-500730169172/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2022.07.019/asset/4754ca46-194b-4a12-9296-218a6de0c947/main.assets/figs7.jpg" height="501" width="759" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/9f132870-6b8c-4144-8850-500730169172/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Low-energy structural clusters for design D8.31, related to <a id="crosref1080" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">(A) The 250 lowest energy predicted structures for D8.31 were selected and clustered using the Rosetta energy_based_clustering application. The cluster naming (LE_X) is based on the ranking of the lowest energy member from each cluster. The lowest energy structure from each identified cluster is labeled on the energy versus RMSD plot. Orange points: predicted structures with no <i>cis</i>-peptide bond; Gray points: predicted structures with at least one <i>cis</i>-peptide bond.</div><div id="fspara0180" role="paragraph">(B) Lowest energy member from each cluster is shown in the stick representation. Position of <i>cis</i>-peptide bonds indicated in labels. Side chains for non proline (or D-proline) positions are not shown for clarity. The boxes with gray background denote the structures that match the X-ray crystal or the NMR structures.</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">Macrocycle D8.21 also has a symmetric repeat sequence (v<sup>∗</sup>LpLv<sup>∗</sup>LpL) with predicted low-energy “<i>trans</i>-<i>trans</i>” states (2&nbsp;variants, LE_1 and LE_2), “<i>trans</i>-<i>cis</i>” states (LE_3), and “<i>cis</i>-<i>cis</i>” (LE_6) states (<a id="crosref0670" href="#figs8">Figure&nbsp;S8</a>). The <i>trans</i>-<i>trans</i> LE_2 state has exposed NH groups, as well as two NHs forming surface-exposed hydrogen bonds. Both the <i>trans</i>-<i>cis</i> LE_3 and <i>cis</i>-<i>cis</i> LE_6 states have saturated NH groups that form hydrogen bonds in the core of the structure. The X-ray crystal structure of D8.21 under aqueous conditions is a <i>cis</i>-<i>cis</i> confomation similar to that of LE_6 (<a id="crosref0675" href="#fig4">Figure&nbsp;4</a>, middle panel). In d<sub>6</sub>-DMSO solution, the peptide is a single conformationally symmetric <i>trans</i>-<i>trans</i> conformation similar to LE_2, while in 50:50 d<sub>6</sub>-DMSO/<sup>2</sup>H<sub>2</sub>O, it is a 50:50 equilibrium mixture of this same <i>trans</i>-<i>trans</i> conformer (LE_2) and a <i>cis</i>-<i>cis</i> conformation similar to both the X-ray crystal structure and LE_6 (<a id="crosref0680" href="#fig4">Figure&nbsp;4</a>, middle panel). In CDCl<sub>3</sub>, the peptide also adopts two conformational states in equilibrium; the LE_2 symmetric <i>trans</i>-<i>trans</i> conformation with ∼47% population (see <a id="crosref0685" href="#figs9">Figure&nbsp;S9</a>C and legend for possible alternative state), and an asymmetric <i>cis</i>-<i>trans</i> structure with all of its NHs satisfied, matching LE_3 with ∼53% population (<a id="crosref0690" href="#fig4">Figure&nbsp;4</a>, middle panel; <a id="crosref0695" href="#figs7">Figures&nbsp;S7</a> and <a id="crosref0700" href="#figs9">S9</a>). The degree of solvent exposure of NH groups observed in these NMR structures is also generally consistent with temperature dependence of amide chemical shift data (<a id="crosref0705" href="#figs6">Figure&nbsp;S6</a>), and we observe significant stabilization of the <i>trans</i>-<i>cis</i> LE_3 conformer with all NHs satisfied in nonpolar CDCL<sub>3</sub> compared with more polar solvent. Stabilization of LE_3 with NH groups in internal hydrogen bonds in more nonpolar solvent drives conformational switching, increasing the relative population of this state from 0% in a polar solvent to 53% in a nonpolar solvent while reducing the population of the LE_2 conformer with unsatisfied surface NH groups from 100% in polar DMSO (and ∼50% in 50:50 d<sub>6</sub>-DMSO/H<sub>2</sub>O) to less than 50% population in CDCl<sub>3</sub>.</div><div class="figure-wrap"><figure id="figs8" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/9fb13e1b-07dd-489f-8ece-62422bdd0ccb/main.assets/figs8_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs8" src="/cms/10.1016/j.cell.2022.07.019/asset/e0e6caa8-97b5-408c-a0be-8f4299cedadc/main.assets/figs8.jpg" height="510" width="759" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/9fb13e1b-07dd-489f-8ece-62422bdd0ccb/main.assets/figs8_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs8-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S8</span> <span class="figure__title__text">Low-energy structural clusters for design D8.21, related to <a id="crosref1085" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs8-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0190" role="paragraph">(A) The 500 lowest energy predicted structures for D8.21 were selected and clustered using the Rosetta energy_based_clustering application. The cluster naming (LE_X) is based on the ranking of the lowest energy member from each cluster. The lowest energy structure from each identified cluster is labeled on the energy versus RMSD plot. Orange points: predicted structures with no <i>cis</i>-peptide bond; gray points: predicted structures with at least one <i>cis</i>-peptide bond; blue points: structures obtained after local minimization of the design model (LE_0).</div><div id="fspara0195" role="paragraph">(B) Lowest energy member from each cluster is shown in the stick representation. The color of stick representation is based on the presence or absence of any <i>cis</i>-peptide bond in the structure. Orange: structures with no <i>cis</i>-peptide bonds; gray: structures with at least one <i>cis</i>-peptide bond; blue: design model. Position of <i>cis</i>-peptide bonds indicated in labels. Side chains for non proline (or D-proline) positions are not shown for clarity. The boxes with gray background denote the structures that match the X-ray crystal or the NMR structures.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs9" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/b318d836-d425-4cf9-b165-87d46b01b0c9/main.assets/figs9_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs9" src="/cms/10.1016/j.cell.2022.07.019/asset/6d2d1fdf-705b-4773-b6ee-b7091aeb69e2/main.assets/figs9.jpg" height="637" width="656" aria-labelledby="figs9-title figs9-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/b318d836-d425-4cf9-b165-87d46b01b0c9/main.assets/figs9_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs9-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S9</span> <span class="figure__title__text">NMR structures of D8.21 have conformational ambiguity, related to <a id="crosref1090" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs9-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0205" role="paragraph">(A) 1D <sup>1</sup>H NMR spectrum of D8.21 in CDCl<sub>3</sub> collected at 600 MHz and 293 K. Amide <sup>1</sup>H peaks for each conformation are indicated.</div><div id="fspara0210" role="paragraph">(B and C) (B) The <i>trans</i>-<i>trans</i> NMR structure (47% population) in CDCl<sub>3</sub> that best matches the NMR data. It has an “open” conformation with only surface hydrogen bonds and buried but unsatisfied NHs and is similar to the <i>trans</i>-<i>trans</i> conformation observed in DMSO and DMSO-water. A representative member of this ensemble (left panel) shows four surface hydrogen bonds, whereas the ensemble of 20 structures (backbone—middle panel; backbone plus sidechain—right panel) shows the variability in orientations of the NH donor and carbonyl acceptor conformations along with buried, unsatisfied NHs. (B) An alternate “closed” <i>trans</i>-<i>trans</i> conformation. In order to assess whether the NMR data obtained for D8.21 in CHCl<sub>3</sub> could possibly be fit to a more closed conformation with buried hydrogen bonds, we also used Cyana to reassign the ROESY data subject to restraints imposed for the four specific hydrogen bonds observed in the “closed” <i>trans</i>-<i>cis</i> conformation (47% in CHCl<sub>3</sub>), which are also observed in the designed and predicted low-energy <i>trans</i>-<i>trans</i> state (design model and LE_1 in <a id="optyjtPnV2ByX" href="#figs8">Figure S8</a>). Although the open <i>trans</i>-<i>trans</i> conformation (B)&nbsp;fits the NMR ROESY and amide temperature coefficient data better than the closed conformation, the alternate closed <i>trans</i>-<i>trans</i> conformation (C)&nbsp;cannot be ruled out. This ambiguity is due primarily to the chemical shift degeneracy that results from the repetitive 4-residue sequence (v<sup>∗</sup>LpL)<sub>2</sub> and the symmetry of this conformation. The assignment of ROEs between degenerate/symmetric chemical shifts has multiple possibilities. For example, the assignment of ROEs to short distances between D-Pro 3 and D-Pro 7 rings in the open conformation is ambiguous because they are indistinguishable from intraresidue D-proline peaks. For the closed conformation, the D-Val 1 and 5 HNs have a characteristic short distance that is not observable in the ROESY data due to their degenerate chemical shifts. The alternative assignments of the ROEs by Cyana are based on preliminary structures and can be influenced by one or two manual restraints guiding it toward one or the other conformation.</div><div id="fspara0215" role="paragraph">(D) The <i>trans-cis</i> conformation determined from NMR data in CHCl<sub>3</sub> (53% population).</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph">Design D9.16 has two N-methylated amino acids and two prolines (p<sup>∗</sup>AAv<sup>∗</sup>LLLPl). The low-energy design model is a “<i>trans</i>-<i>trans</i>” conformation (LE_0) with no unsatisfied NH groups. The predicted low-energy states include a “<i>trans</i>-<i>cis</i>” state (LE_10) with exposed NH groups (<a id="crosref0710" href="#fig4">Figure&nbsp;4</a>, right panel; <a id="crosref0715" href="#figs6">Figures&nbsp;S6</a> and <a id="crosref0720" href="#figs10">S10</a>). The X-ray crystal structure from aqueous conditions is in a <i>trans</i>-<i>cis</i> conformation that matches LE_10. In polar solvents (i.e., 50:50 d<sub>6</sub>-DMSO/<sup>2</sup>H<sub>2</sub>O and 100% d<sub>6</sub>-DMSO), the peptide has a single <i>trans</i>-<i>cis</i> conformation, with two exposed and one buried unsatisfied NH groups, that matches LE_10 (<a id="crosref0725" href="#fig4">Figure&nbsp;4</a>, right panel). NMR data in nonpolar solvent (CDCl<sub>3</sub>) reveal an equilibrium between two states: a <i>trans</i>-<i>cis</i> state that matches the LE_10 state observed in both d<sub>6</sub>-DMSO and the crystal structure, and a <i>trans</i>-<i>trans</i> state with all backbone NHs satisfied that matches closely with the designed <i>trans</i>-<i>trans</i> state LE_0 (<a id="crosref0730" href="#fig4">Figure&nbsp;4</a>, right panel). The population of the <i>trans</i>-<i>cis</i> LE_10 with exposed NH groups changes from 100% in a polar solvent to 45% in a nonpolar solvent, while the population of <i>trans</i>-<i>trans</i> LE_0 with all NH groups satisfied switches from 0% in a polar solvent to 55% in a nonpolar solvent.</div><div class="figure-wrap"><figure id="figs10" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/ab2f885a-6ed7-4431-b742-1962d33b1d6d/main.assets/figs10_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs10" src="/cms/10.1016/j.cell.2022.07.019/asset/41099327-1b91-4af6-b366-6dde3b31fa43/main.assets/figs10.jpg" height="520" width="762" aria-labelledby="figs10-title figs10-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/ab2f885a-6ed7-4431-b742-1962d33b1d6d/main.assets/figs10_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs10-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S10</span> <span class="figure__title__text">Low-energy structural clusters for design D9.16, related to <a id="crosref1100" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs10-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0225" role="paragraph">(A) The 250 lowest energy predicted structures for D9.16 were selected and clustered using the Rosetta energy_based_clustering application. The cluster naming (LE_X) is based on the ranking of the lowest energy member from each cluster. The lowest energy structure for 10 lowest energy clusters is labeled on the energy versus RMSD plot. Orange points: predicted structures with no <i>cis</i>-peptide bond; gray points: predicted structures with at least one <i>cis</i>-peptide bond; blue points: structures obtained after local minimization of the design model (LE_0).</div><div id="fspara0230" role="paragraph">(B) Lowest energy member from each of the 10 lowest energy clusters is shown in the stick representation. The color of stick representation is based on the presence or absence of any <i>cis</i>-peptide bond in the structure. Orange: structures with no <i>cis</i>-peptide bonds; gray: structures with at least one <i>cis</i>-peptide bond; blue: design model. Position of <i>cis</i>-peptide bonds indicated in labels. Side chains for non proline (or D-proline) positions are not shown for clarity. The boxes with gray background denote the structures that match the X-ray crystal or the NMR structures.</div></div></div></div></figcaption></figure></div><div id="p0155" role="paragraph">Taken together, these data indicate that the D8.21, D8.31, and D9.16 macrocycles indeed populate multiple states, with the low-energy states closely matching the experimental crystal and NMR structures. Consistent with our conformational switch design strategy, the relative populations of the different states observed for D8.21 and D9.16 are solvent dependent, with the state having less exposed or no unsatisfied NH groups favored in the more nonpolar solvent. Overall, the solvent change shifts the equilibrium between the low-energy conformations of these chameleonic macrocycles. The macrocycles are also, as intended by design, membrane permeable, but due to difficulties in characterizing the state of macrocycles during membrane traversal, we cannot attribute this permeability specifically to one of the designed states. Boding well for the future design of membrane-permeable macrocycles targeting polar binding sites, it is notable that both D8.21 and D9.16 expose backbone NHs in one state, yet retain significant permeability.</div></section><section id="sec-2-5"><h3>Oral bioavailability</h3><div id="p0160" role="paragraph">Oral bioavailability is a desirable therapeutic property that requires stability against the low pH and proteases in the gastrointestinal tract and permeation across the epithelial cells in the gut. We selected four macrocycles covering different sizes and rates of <i>in&nbsp;vitro</i> permeability for <i>in&nbsp;vivo</i> oral bioavailability and pharmacokinetic studies in rodent models. Plasma exposure for one 8-mer (D8.3.p1), one 10-mer (D10.1), and two 11-mer peptides (D11.2 and D11.3) was measured after single-dose administration via intravenous (IV), subcutaneous (SQ), and oral (PO) routes. The amount of unmodified drug in plasma was quantified by mass spectrometry and the fraction of unmodified drug (%F) in plasma after oral delivery was determined using IV dosing as reference (see <a id="crosref0735" href="#sec-4">STAR Methods</a>). The peptides were well tolerated without any adverse effects at the doses tested. All four designs had substantial oral exposure and demonstrated comparable or better oral bioavailability than most other natural orally absorbed peptides. Designs D8.3.p1, D11.2, and D10.1 have a good %F between 7.5% and 11% (<a id="crosref0740" href="#fig5">Figure&nbsp;5</a>; <a id="crosref0745" href="#mmc1">Data S4</a>). The 11-mer design, D11.3, was tested for oral bioavailability in male Swiss albino mice and had a very high oral bioavailability (%F) of 40% despite its large size. The designs also demonstrated other favorable drug-like attributes, such as long plasma half-life (T<sub>1/2</sub>). D11.3 has a T<sub>1/2</sub> of 5.58&nbsp;h after IV dosing, and D10.1 has a T<sub>1/2</sub> of 3.75&nbsp;h after SQ administration (<a id="crosref0750" href="#fig5">Figure&nbsp;5</a>; <a id="crosref0755" href="#mmc1">Data S4</a>). Overall, these <i>in&nbsp;vivo</i> data validate that these computationally designed and structurally validated peptides are robust to low pH and protease exposure and get absorbed efficiently across the gut epithelial barrier.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.07.019/asset/a478f72f-3ee6-40f8-b4c1-f956b909b417/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2022.07.019/asset/7d5c1d69-adc7-4a50-94d6-e0455402a00e/main.assets/gr5.jpg" height="445" width="656" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/a478f72f-3ee6-40f8-b4c1-f956b909b417/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Designed macrocycles are orally bioavailable <i>in&nbsp;vivo</i> in rodent models</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0080" role="paragraph">Plasma concentration of unmodified full-length peptides measured after intravenous (IV), subcutaneous (SQ), and oral (PO) administration in mice (D8.3.p1, D10.1, and D11.3) and rats (D11.2) (n&nbsp;= 3 mice per dosing route for D8.3.p1, D10.1, and D11.3 and n&nbsp;= 3 rats per dosing route for D11.2). D8.3.p1 and D10.1 were studied in female BALB/c mice, D11.2 was studied in male Sprague Dawley (SD) rats, and D11.3 was studied in male swiss albino mice.</div><div id="fspara0085" role="paragraph">See also <a id="crosref1020" href="#mmc1">Data S5</a>.</div></div></div></div></figcaption></figure></div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0165" role="paragraph">We have shown that the ability to precisely control the structure enables the robust design of a wide diversity of membrane-permeable macrocycles that goes considerably beyond those discovered previously, primarily N-methylated beta hairpins (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">Bockus et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bockus, A.T. ∙ Lexa, K.W. ∙ Pye, C.R. ...</div><div><strong>Probing the physicochemical boundaries of cell permeability and oral bioavailability in lipophilic macrocycles inspired by natural products</strong></div><div><em>J.&nbsp;Med. Chem.</em> 2015; <strong>58</strong>:4581-4589</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jmedchem.5b00128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84935898517" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25950816/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jmedchem.5b00128&amp;pmid=25950816" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Fouché et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Fouché, M. ∙ Schäfer, M. ∙ Berghausen, J. ...</div><div><strong>Design and development of a cyclic decapeptide scaffold with suitable properties for bioavailability and oral exposure</strong></div><div><em>ChemMedChem.</em> 2016; <strong>11</strong>:1048-1059</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cmdc.201600082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84966389321" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27154275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcmdc.201600082&amp;pmid=27154275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11-3" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-3" aria-controls="sref11-3">Hill et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref11-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Hill, T.A. ∙ Lohman, R.-J. ∙ Hoang, H.N. ...</div><div><strong>Cyclic penta- and Hexaleucine peptides without N-methylation are orally absorbed</strong></div><div><em>ACS Med. Chem. Lett.</em> 2014; <strong>5</strong>:1148-1151</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ml5002823" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84907923894" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25313329/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fml5002823&amp;pmid=25313329" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Ovadia et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Ovadia, O. ∙ Greenberg, S. ∙ Chatterjee, J. ...</div><div><strong>The effect of multiple N-methylation on intestinal permeability of cyclic hexapeptides</strong></div><div><em>Mol. Pharm.</em> 2011; <strong>8</strong>:479-487</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/mp1003306" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-79953811147" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21375270/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fmp1003306&amp;pmid=21375270" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24-3" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-3" aria-controls="sref24-3">Wang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref24-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Wang, C.K. ∙ Northfield, S.E. ∙ Colless, B. ...</div><div><strong>Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:17504-17509</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1417611111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84916618640" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25416591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1417611111&amp;pmid=25416591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Like natural products, our designs achieve this high permeability through conformational shielding of polar groups using a diversity of local structures and internal hydrogen bonds. In total, we designed, synthesized, and validated 84 structurally diverse macrocycles with good to excellent permeability, including 6–8 residue macrocycles with high permeability and no N-methyl groups, and 9–12 residue membrane-permeable macrocycles with a single N-methylated amino acid in the sequence. The passive permeability of the designs in PAMPA translates to good permeability across Caco-2 cells and oral bioavailability in rodent models. The importance of computational-design-based control over structure is highlighted by the strong correlation between the extent of permeability and sub-angstrom match between experimental structure and design model: of the 35 designs for which we succeeded in determining crystal or NMR structures, 21 macrocycles out of the 25 that closely matched (RMSD&nbsp;&lt;&nbsp;1&nbsp;Å) the design states were all membrane permeable (P<sub>app</sub>&nbsp;&gt;&nbsp;1&nbsp;×&nbsp;10<sup>−7</sup> cm/s). While the very close agreement (RMSD&nbsp;&lt;&nbsp;1.2&nbsp;Å) between the models of 29 out of the 35 designs and the corresponding experimental crystal or NMR structures indicates that the design method has very high accuracy, we cannot exclude the possibility that designs for which we lack X-ray or NMR structures fold into alternate conformations important for permeability.</div><div id="p0170" role="paragraph">The design methods and membrane-permeable macrocycles presented in this paper provide the basis for the development of orally bioavailable macrocyclic therapeutics. The ability to precisely control structure should enable the targeting of a wide range of binding site geometries, and the ability to robustly design for membrane permeability should allow access to intracellular targets and oral delivery. Our energy landscape-based approach to designing peptides that exhibit cyclosporine-like chameleonic behavior, switching between a lipophilic state and a second, relatively polar, state as solvent polarity changes, should enable the design of macrocycles that can bind intracellular therapeutic targets with exposed polar groups while retaining membrane permeability.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0175" role="paragraph">While there is a good match between the X-ray crystal structures, NMR structures, and permeability data, it cannot be ruled out that designs have alternate conformations that affect their&nbsp;permeability but did not crystallize. While we focus here on achieving membrane permeability through intrapeptide hydrogen bonding, sidechain-mediated shielding of unsatisfied NH groups could also be effective; the structurally validated macrocycles provide templates to systematically evaluate the effects of NH shielding on permeability. We do not yet have as precise control over the chameleonic peptide alternative structures as we do with the single-state designs. This should improve with increases in energy function precision and improvements in&nbsp;landscape sampling, both methodological and through increases in available computing power.</div><div id="p0180" role="paragraph">The key challenge going forward is incorporating target binding functionality into the designs while maintaining control over structure and permeability. In this paper, the design calculations were focused on optimizing folding precision and permeability. Designing binders may require the incorporation of polar or other functional groups that make energetically favorable interactions with the target but may not be optimal for folding or permeability (or both). Our current efforts are focused on addressing this challenge by investigating the effects of polar side chains on permeability, and more generally, by designing and characterizing binders for a variety of intracellular targets.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0185" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Cl-TCP(Cl) ProTide resin</td><td>CEM</td><td>R005</td></tr><tr><td>Fmoc amino acids</td><td>P3 biosystems</td><td>N/A</td></tr><tr><td>N,N-Dimethylformamide</td><td>Fisher</td><td>D119</td></tr><tr><td>Piperidine</td><td>Millipore Sigma</td><td>104094</td></tr><tr><td>Trifluoroacetic acid</td><td>Millipore Sigma</td><td>T6508</td></tr><tr><td>Dichloromethane</td><td>Fisher</td><td>D151</td></tr><tr><td>PyAOP</td><td>Millipore Sigma</td><td>8512210025</td></tr><tr><td>N,N-Diisopropylethylamine</td><td>Millipore Sigma</td><td>D125806</td></tr><tr><td>2-Propanol</td><td>Fisher</td><td>A451</td></tr><tr><td>d6-DMSO with 0.03% TMS</td><td>Krackeler</td><td>CAS # 2206-17-1</td></tr><tr><td>chloroform-d with 0.03% (v/v) TMS</td><td>Krackeler</td><td>CAS #865-49-6</td></tr><tr><td>Pre-coated PAMPA plates</td><td>Corning</td><td>353015</td></tr><tr><td>DMEM/F12 medium</td><td>Gibco</td><td>11320033</td></tr><tr><td>RPMI medium</td><td>Gibco</td><td>11875119</td></tr><tr><td>alamarBlue cell viability reagent</td><td>Invitrogen</td><td>DAL1025</td></tr><tr><td>HEPES</td><td>Gibco</td><td>15630080</td></tr><tr><td>L-glutamine</td><td>Gibco</td><td>25030081</td></tr><tr><td>Trypsin-EDTA (0.25%), phenol red</td><td>Gibco</td><td>25200056</td></tr><tr><td>Sodium pyruvate</td><td>Gibco</td><td>11360070</td></tr><tr><td>Fetal Bovine Serum</td><td>VWR Seradigm</td><td>97068-085</td></tr><tr><td>Microvette 100 w/ capillary straw, 100&nbsp;μL Lithium heparin</td><td>SAI infusion technologies</td><td>MVC-H-100</td></tr><tr><td>Tween 80</td><td>Sigma-Aldrich</td><td>P8074</td></tr><tr><td>Dimethyl sulfoxide</td><td>Sigma-Aldrich</td><td>D2650</td></tr><tr><td>PEG 400</td><td>Sigma-Aldrich</td><td>PX1286B</td></tr><tr><td>DPBS</td><td>Gibco</td><td>14190136</td></tr><tr><td>Dextrose</td><td>Sigma-Aldrich</td><td>D9434</td></tr><tr><td>Transwell® polyester membrane cell culture inserts</td><td>Corning</td><td>CLS3460</td></tr><tr><td>DMEM - Dulbecco's Modified Eagle Medium</td><td>Gibco</td><td>11965118</td></tr><tr><td>Fetal Bovine Serum</td><td>Gibco</td><td>10099141</td></tr><tr><td>Penicillin-Streptomycin</td><td>Gibco</td><td>15070063</td></tr><tr><td>MEM Non-Essential Amino Acids Solution</td><td>Gibco</td><td>11140050</td></tr><tr><td>Puromycin dihydrochloride</td><td>Sigma-Aldrich</td><td>P9620</td></tr><tr><td>Hank's Balanced Salt Solution (HBSS)</td><td>Gibco</td><td>14175079</td></tr><tr><td>Atenolol</td><td>Sigma-Aldrich</td><td>A7655</td></tr><tr class="rowsep"><td>Quinidine</td><td>Sigma-Aldrich</td><td>Q3625</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>D7.6 X-ray crystal structure</td><td>This paper</td><td>CCDC 131411</td></tr><tr><td>D7.8 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131412</td></tr><tr><td>D8.1 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131449</td></tr><tr><td>D8.2 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131417</td></tr><tr><td>D8.3.p1 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131450</td></tr><tr><td>D8.5.p2 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131423</td></tr><tr><td>D8.6 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131424</td></tr><tr><td>D8.9 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131249</td></tr><tr><td>D8.10 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131251</td></tr><tr><td>D8.12 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131252</td></tr><tr><td>D8.13 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131463</td></tr><tr><td>D8.14 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131425</td></tr><tr><td>D8.15 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131426</td></tr><tr><td>D8.17 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131427</td></tr><tr><td>D8.19 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131428</td></tr><tr><td>D8.25 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131253</td></tr><tr><td>D8.26 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131291</td></tr><tr><td>D9.1 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131429</td></tr><tr><td>D9.8 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131430</td></tr><tr><td>D9.24 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131245</td></tr><tr><td>D9.29 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131431</td></tr><tr><td>D9.30 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131432</td></tr><tr><td>D10.1 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131433</td></tr><tr><td>D10.21 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131434</td></tr><tr><td>D10.22 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131435</td></tr><tr><td>D10.23 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131436</td></tr><tr><td>D10.31 X-ray crystal structure in IPA: water</td><td>This paper</td><td>CCDC 2131438</td></tr><tr><td>D10.31 X-ray crystal structure in Ethyl Acetate: Pentane</td><td>This paper</td><td>CCDC 2131437</td></tr><tr><td>D10.65 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131439</td></tr><tr><td>D11.1 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131440</td></tr><tr><td>D11.3 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131451</td></tr><tr><td>D11.4 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131441</td></tr><tr><td>D11.25 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131442</td></tr><tr><td>D8.31 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131292</td></tr><tr><td>D8.21 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131443</td></tr><tr><td>D9.16 X-ray crystal structure</td><td>This paper</td><td>CCDC 2131293</td></tr><tr><td>D8.3.p2 X-ray crystal structure</td><td>This paper</td><td>PDB <a id="intref0020" href="http://firstglance.jmol.org/fg.htm?mol7UCP" target="_blank">7UCP</a></td></tr><tr><td>D9.16 in d6-dMSO (trans-cis) solution NMR structure</td><td>This paper</td><td>PDB <a id="intref0025" href="http://firstglance.jmol.org/fg.htm?mol7UBC" target="_blank">7UBC</a> &amp; BioMagResDB 30997</td></tr><tr><td>D8.31 in d6-DMSO (cis-cis) NMR structure</td><td>This paper</td><td>PDB <a id="intref0030" href="http://firstglance.jmol.org/fg.htm?mol7UBD" target="_blank">7UBD</a> &amp; BioMagResDB 30998</td></tr><tr><td>D8.31 d6-DMSO (cis-trans) NMR structure</td><td>This paper</td><td>PDB <a id="intref0035" href="http://firstglance.jmol.org/fg.htm?mol8CTO" target="_blank">8CTO</a> &amp; BioMagResDB 31021</td></tr><tr><td>D8.21 in d6-DMSO (trans-trans) NMR structure</td><td>This paper</td><td>PDB <a id="intref0040" href="http://firstglance.jmol.org/fg.htm?mol7UBE" target="_blank">7UBE</a> &amp; BioMagResDB 30999</td></tr><tr><td>D8.21 in 50:50 d6-DMSO/H2O (trans-trans) NMR structure</td><td>This paper</td><td>PDB <a id="intref0045" href="http://firstglance.jmol.org/fg.htm?mol7UBF" target="_blank">7UBF</a> &amp; BioMagResDB 31000</td></tr><tr><td>D8.21 in 50:50 d6-DMSO/H2O (cis-cis) NMR structure</td><td>This paper</td><td>PDB <a id="intref0050" href="http://firstglance.jmol.org/fg.htm?mol8CUN" target="_blank">8CUN</a> &amp; BioMagResDB 31022</td></tr><tr><td>D9.16 in CDCl3 (trans-trans) NMR structure</td><td>This paper</td><td>PDB <a id="intref0055" href="http://firstglance.jmol.org/fg.htm?mol7UBG" target="_blank">7UBG</a> &amp; BioMagResDB 31001</td></tr><tr><td>D9.16 in CDCl3 (trans-cis) NMR structure</td><td>This paper</td><td>PDB <a id="intref0060" href="http://firstglance.jmol.org/fg.htm?mol7UZL" target="_blank">7UZL</a> &amp; BioMagResDB 31019</td></tr><tr><td>D8.31 in CDCl3 (cis-cis) NMR structure</td><td>This paper</td><td>PDB <a id="intref0065" href="http://firstglance.jmol.org/fg.htm?mol7UBH" target="_blank">7UBH</a> &amp; BioMagResDB 31002</td></tr><tr><td>D8.21 in CDCl3 (trans-trans) NMR structure</td><td>This paper</td><td>PDB <a id="intref0070" href="http://firstglance.jmol.org/fg.htm?mol7UBI" target="_blank">7UBI</a> &amp; BioMagResDB 31003</td></tr><tr class="rowsep"><td>D8.21 in CDCl3 (trans-cis) NMR structure</td><td>This paper</td><td>PDB <a id="intref0075" href="http://firstglance.jmol.org/fg.htm?mol8CWA" target="_blank">8CWA</a> &amp; BioMagResDB 31023</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>Hep G2 human hepatocellular carcinoma cell line</td><td>ATCC</td><td>HB-8065</td></tr><tr><td>WIL2-NS human B lymphocyte cell line</td><td>ATCC</td><td>CRL-8155</td></tr><tr class="rowsep"><td>CACO-2 Cell Line human</td><td>CellBank Australia</td><td>86010202</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr><td>10-12&nbsp;week old female BALB/cAnNCrl mice</td><td>Charles River</td><td>strain code 028</td></tr><tr class="rowsep"><td>Male JVC Sprague Dawley rats</td><td>Hilltop Lab animals incorporated</td><td><a id="intref0080" href="http://www.hilltoplabs.com/" target="_blank">http://www.hilltoplabs.com/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Software</b></td></tr><tr><td>Rosetta and PyRosetta Scripts for Macrocycle Design</td><td>This study</td><td><a id="crosref0785" href="#mmc1">Data S5</a></td></tr><tr><td>TopSpin 3.5 and 4.0.9</td><td>Bruker BioSpin, Inc</td><td><a id="intref0085" href="http://www.bruker.com" target="_blank">www.bruker.com</a></td></tr><tr><td>Cyana 3.98.13</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Güntert et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Güntert, P. ∙ Mumenthaler, C. ∙ Wüthrich, K.</div><div><strong>Torsion angle dynamics for NMR structure calculation with the new program DYANA</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1997; <strong>273</strong>:283-298</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jmbi.1997.1284" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0031576336" target="_blank" aria-label="Open Scopus in new tab">Scopus (2598)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9367762/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjmbi.1997.1284&amp;pmid=9367762" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0090" href="https://www.las.jp/english/products/cyana.html" target="_blank">https://www.las.jp/english/products/cyana.html</a></td></tr><tr><td>NMRFAM-SPARKY</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref13-1" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-1" aria-controls="sref13-1">Lee et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref13-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Lee, W. ∙ Tonelli, M. ∙ Markley, J.L.</div><div><strong>NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:1325-1327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84927748573" target="_blank" aria-label="Open Scopus in new tab">Scopus (1330)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25505092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu830&amp;pmid=25505092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0095" href="https://nmrfam.wisc.edu/nmrfam-sparky-distribution/" target="_blank">https://nmrfam.wisc.edu/nmrfam-sparky-distribution/</a></td></tr><tr><td>NMRPipe v10.9</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Delaglio et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Delaglio, F. ∙ Grzesiek, S. ∙ Vuister, G.W. ...</div><div><strong>NMRPipe: a multidimensional spectral processing system based on UNIX pipes</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 1995; <strong>6</strong>:277-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00197809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0029400480" target="_blank" aria-label="Open Scopus in new tab">Scopus (12158)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8520220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00197809&amp;pmid=8520220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0100" href="https://www.ibbr.umd.edu/nmrpipe/" target="_blank">https://www.ibbr.umd.edu/nmrpipe/</a></td></tr><tr><td>PDBStat v5.21.6</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref23-1" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-1" aria-controls="sref23-1">Tejero et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref23-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Tejero, R. ∙ Snyder, D. ∙ Mao, B. ...</div><div><strong>PDBStat: a universal restraint converter and restraint analysis software package for protein NMR</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2013; <strong>56</strong>:337-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-013-9753-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84883463996" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23897031/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-013-9753-7&amp;pmid=23897031" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0105" href="http://rtti7.uv.es/%7Eroberto/Index.php?sec=pdbstat" target="_blank">http://rtti7.uv.es/∼roberto/Index.php?sec=pdbstat</a></td></tr><tr><td>Phoenix WinNonlin®, Version 6.3</td><td>Pharsight Corp</td><td><a id="intref0110" href="https://www.certara.com/software/phoenix-winnonlin/" target="_blank">https://www.certara.com/software/phoenix-winnonlin/</a></td></tr><tr><td>XDS</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Winn et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div>Winn, M.D., Refinemen, M.-L.S., Pannu, N.J., Murshudov, G.N., and Others (2001). CCP4 v6. 3.0 Program References. Acta Crystallogr. <i>57</i>, 122–133.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Winn%2C+M.D.%2C+Refinemen%2C+M.-L.S.%2C+Pannu%2C+N.J.%2C+Murshudov%2C+G.N.%2C+and+Others+%282001%29.+CCP4+v6.+3.0+Program+References.+Acta+Crystallogr.+57%2C+122%E2%80%93133." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0115" href="https://xds.mr.mpg.de/" target="_blank">https://xds.mr.mpg.de/</a></td></tr><tr><td>CCP4</td><td><span class="dropBlock reference-citations"><a id="body-ref-optg8m7ZUq91m-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optg8m7ZUq91m-1" aria-controls="optg8m7ZUq91m-1">Kabsch, 2010</a><div class="dropBlock__holder" data-db-target-of="optg8m7ZUq91m-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Kabsch, W.</div><div><strong>XDS</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:125-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444909047337" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-76449099287" target="_blank" aria-label="Open Scopus in new tab">Scopus (12932)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444909047337&amp;pmid=20124692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0120" href="https://www.ccp4.ac.uk/" target="_blank">https://www.ccp4.ac.uk/</a></td></tr><tr><td>SHELXL</td><td><span class="dropBlock reference-citations"><a id="body-ref-optM9Rq3dRHgc-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optM9Rq3dRHgc-1" aria-controls="optM9Rq3dRHgc-1">Sheldrick, 2015a</a><div class="dropBlock__holder" data-db-target-of="optM9Rq3dRHgc-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Sheldrick, G.M.</div><div><strong>SHELXT--Integrated space-group and crystal-structure determination</strong></div><div><em>Acta Crystallogr. A Found. Adv.</em> 2015; <strong>71</strong>:3-8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2053273314026370" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84920370689" target="_blank" aria-label="Open Scopus in new tab">Scopus (30086)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25537383/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2053273314026370&amp;pmid=25537383" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-optgPLOKl5zI3-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optgPLOKl5zI3-1" aria-controls="optgPLOKl5zI3-1">2015b</a><div class="dropBlock__holder" data-db-target-of="optgPLOKl5zI3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Sheldrick, G.M.</div><div><strong>Crystal strcuture refinement with SHELXL</strong></div><div><em>Acta Crystallogr. C Struct. Chem.</em> 2015; <strong>71</strong>:3-8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/s2053229614024218" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84937398809" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25567568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2Fs2053229614024218&amp;pmid=25567568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0125" href="https://shelx.uni-goettingen.de/" target="_blank">https://shelx.uni-goettingen.de/</a></td></tr><tr class="rowsep"><td>SHELXLe</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Hübschle, 2011</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Hübschle, C.B. ∙ Sheldrick, G.M. ∙ Dittrich, B.</div><div><strong>ShelXle: a Qt graphical user interface for SHELXL</strong></div><div><em>J. Appl. Crystallogr.</em> 2011; <strong>44</strong>:1281-1284</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0021889811043202" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-82055161785" target="_blank" aria-label="Open Scopus in new tab">Scopus (2900)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22477785/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0021889811043202&amp;pmid=22477785" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0130" href="https://www.shelxle.org/shelx/eingabe.php" target="_blank">https://www.shelxle.org/shelx/eingabe.php</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Bruker Avance II 600 MHz NMR System</td><td>Bruker Biospin, Inc.</td><td><a id="intref0135" href="https://www.bruker.com/en.html" target="_blank">https://www.bruker.com/en.html</a></td></tr><tr><td>Bruker Avance III 600 MHz NMR System</td><td>Bruker Biospin, Inc.</td><td><a id="intref0140" href="https://www.bruker.com/en.html" target="_blank">https://www.bruker.com/en.html</a></td></tr><tr><td>Bruker Avance II 800 MHz NMR System</td><td>Bruker Biospin, Inc.</td><td><a id="intref0145" href="https://www.bruker.com/en.html" target="_blank">https://www.bruker.com/en.html</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2822%2900922-9" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0190" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, David Baker (<a id="intref0150" href="mailto:dabaker@uw.edu">dabaker@uw.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0195" role="paragraph">All the codes and data used in the manuscript are available in <a id="crosref0810" href="#supplementary-material">supplemental information</a> files. Other materials are available upon request.</div></section></section><section id="sec-4-3"><h3>Experimental Model and Subject Details</h3><section id="sec-4-3-1"><h4>Caco-2 assays</h4><div id="p0200" role="paragraph">Human colon adenocarcinoma-derived Caco-2 cell line was used as a model system to investigate the intestinal epithelial permeability of our designs. Caco-2 cells were propagated in T125 flasks (Corning, Merck) using Dulbecco’s modified Eagle medium supplemented with 10% (v/v) fetal bovine serum, 1% (v/v) nonessential amino acids, and 1% (v/v) penicillin-streptomycin (Gibco,&nbsp;Thermo Fisher Scientific) at 37°C in an atmosphere of 5% CO2. Cells were subcultured when they reached 80% confluence.</div></section><section id="sec-4-3-2"><h4><i>In&nbsp;vivo</i> bioavailability assays</h4><div id="p0205" role="paragraph">In&nbsp;vivo bioavailability assays were conducted in mice and rats. D8.3.p1 and D10.1 were studied in female Balb/C mice by van Voorhis group at University of Washington. D11.2 was evaluated in Sprague Dawley (SD) rats by Robert Griffin’s group at Takeda Pharmaceuticals. D11.3 was studied in the male swiss albino mice by the contract research organization, GVK Biosciences Pvt. Ltd., Hyderabad, India. All animals used in these studies were handled in strict accordance with practices made to minimize suffering. All animal experiments conducted at the University of Washington, USA, and Takeda Pharmaceuticals, USA were approved by the Institutional Animal Care and Use Committee. Animal studies at GVK Biosciences were reviewed and approved by the Institutional Animal Ethics Committee of GVK Biosciences.</div></section></section><section id="sec-4-4"><h3>Method Details</h3><section id="sec-4-4-1"><h4>Enhancements to the Rosetta software suite</h4><div id="p0210" role="paragraph">In order to allow the design, modeling, and structure prediction of cyclic peptides incorporating N-methylated amino acids in the Rosetta software suite, a number of enhancements were needed. First, Rosetta’s kinematic machinery, which keeps track of updating Cartesian coordinates of atoms as internal degrees of freedom (such as torsion angles) of a polymer change, was refactored. Code for generating ideal coordinates of atoms whose positions depend on peptide bonds (such as the carbonyl oxygen and amide proton) was altered to allow whole chemical groups (such as an N-methyl group) to be placed. Rosetta’s protein cutpoint code was also refactored.&nbsp;This code ensures that a special term in the Rosetta energy function (called chainbreak) penalizes poor peptide bond geometry when a peptide bond is located at a break in the kinematic tree, as inevitably happens at some point in a cyclic peptide; effectively, this keeps a cyclic peptide from opening during energy minimization. The needed changes made this term compatible with <i>cis</i>-peptide bonds and with N-methylation.</div><div id="p0215" role="paragraph">Rosetta’s residue type system is organized into residue types (distinct chemical entities) and patches (small variations on existing types). Chemical modifications such as a protonated N-terminus, a deprotonated C-terminus, or a methylated side-chain are typically handled with patches, which instruct Rosetta to add a variant by altering an existing residue type’s geometry, rather than by adding an entirely new residue type. We supported backbone N-methylation in the same way, adding a patch to convert an amino acid residue type to its N-methylated equivalent. However, the addition of a methyl group on the backbone nitrogen greatly alters both the backbone and side-chain conformational preferences of a given residue type. We added support to Rosetta’s patching system to permit new mainchain potentials and side-chain rotamer libraries to be specified for a patched residue type as part of the patching process. We then computed new mainchain Ramachandran potentials for N-methyl-glycine (sarcosine) and for N-methyl-L-alanine, using the AMBER GAFF force field for N-methyl-glycine and Gaussian quantum mechanics calculations with the B3LYP/6-311+G(d,p) level of theory for N-methyl-L-alanine. Since the steric repulsion of an N-methyl group on the i<sup>th</sup> amino acid residue greatly influences the conformational preferences of the i-1<sup>st</sup> amino acid residue in much the same way that the proline side-chain’s steric repulsion when proline is at the i<sup>th</sup> position affects the conformational preferences of the i-1<sup>st</sup> residue, Rosetta’s rama_prepro energy term (which computes the mainchain potential, and applies different Ramachandran potentials to residues preceding proline) was extended to apply the pre-proline Ramachandran potentials to positions preceding N-methylated positions as well. Pre-proline/pre-N-methyl Ramachandran potentials were similarly computed for N-methyl-glycine and N-methyl-L-alanine. Using the previously-described MakeRotLib protocol (<span class="dropBlock reference-citations"><a id="body-ref-optiUo6YTl30I-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optiUo6YTl30I-1" aria-controls="optiUo6YTl30I-1">Renfrew et al., 2012</a><div class="dropBlock__holder" data-db-target-of="optiUo6YTl30I-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Renfrew, P.D. ∙ Choi, E.J. ∙ Bonneau, R. ...</div><div><strong>Incorporation of noncanonical amino acids into Rosetta and use in computational protein-peptide interface design</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>, e32637</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0032637" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84858266350" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0032637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), rotamer libraries for the following N-methylated L-amino acid residues were generated: arginine, asparagine, aspartate, cysteine, glutamine, glutamate, histidine, isoleucine, leucine, lysine, methionine, phenylalanine, serine, threonine, tryptophan, tyrosine, and valine. Rosetta automatically mirrors mainchain potentials and rotamer libraries that were generated for L-amino acid residues when sampling conformations or computing energies of D-amino acid residues, and this functionality was extended to N-methylated amino acids. Building on past work to generalize Rosetta’s fa_dun sidechain potential to use rotamer probabilities computed with MakeRotLib when computing the energies of side-chain conformations (<span class="dropBlock reference-citations"><a id="body-ref-optM1vu63fEjC-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optM1vu63fEjC-1" aria-controls="optM1vu63fEjC-1">Mulligan et al., 2021</a><div class="dropBlock__holder" data-db-target-of="optM1vu63fEjC-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Mulligan, V.K. ∙ Workman, S. ∙ Sun, T. ...</div><div><strong>Computationally designed peptide macrocycle inhibitors of New Delhi metallo-β-lactamase 1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong>, e2012800118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2012800118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85102719411" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2012800118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we removed special-case code that had previously been included for peptoid rotamer&nbsp;libraries (<span class="dropBlock reference-citations"><a id="body-ref-optiUo6YTl30I-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optiUo6YTl30I-2" aria-controls="optiUo6YTl30I-2">Renfrew et al., 2012</a><div class="dropBlock__holder" data-db-target-of="optiUo6YTl30I-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Renfrew, P.D. ∙ Choi, E.J. ∙ Bonneau, R. ...</div><div><strong>Incorporation of noncanonical amino acids into Rosetta and use in computational protein-peptide interface design</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>, e32637</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0032637" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84858266350" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0032637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), permitting all rotamer sampling and scoring to be handled by the newly-generalized rotamer&nbsp;code.</div><div id="p0220" role="paragraph">During testing, we found that the close interactions between an N-methyl group, the preceding carbonyl oxygen, and the preceding and adjacent side-chains resulted in frequent clashes that were highly sensitive to the rotation of the N-methyl group. In some cases, these clashes would result in Rosetta choosing distinctly sub-optimal side-chain rotamers during rotamer optimization (packing). We, therefore, added support for freely-rotating N-methyl groups, allowing precise positions of methyl hydrogen atoms to be adjusted during packing or gradient-descent energy minimization.</div><div id="p0225" role="paragraph">To facilitate conformational sampling during design and structure prediction, we modified the generalized kinematic closure (GeneralizedKIC) method described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref1-3" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-3" aria-controls="sref1-3">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-optsrMTRQR98l-1" href="#" role="doc-biblioref" data-xml-rid="bib12 bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optsrMTRQR98l-1" aria-controls="optsrMTRQR98l-1">Hosseinzadeh et al., 2017, 2021</a><div class="dropBlock__holder" data-db-target-of="optsrMTRQR98l-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Hosseinzadeh, P. ∙ Watson, P.R. ∙ Craven, T.W. ...</div><div><strong>Anchor extension: a structure-guided approach to design cyclic peptides targeting enzyme active sites</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:3384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23609-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85107586900" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34099674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23609-8&amp;pmid=34099674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-optM1vu63fEjC-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optM1vu63fEjC-2" aria-controls="optM1vu63fEjC-2">Mulligan et al., 2021</a><div class="dropBlock__holder" data-db-target-of="optM1vu63fEjC-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Mulligan, V.K. ∙ Workman, S. ∙ Sun, T. ...</div><div><strong>Computationally designed peptide macrocycle inhibitors of New Delhi metallo-β-lactamase 1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong>, e2012800118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2012800118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85102719411" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2012800118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, GenerlizedKIC-based peptide macrocycle conformational sampling works by building an extended chain, randomizing all but 6 backbone torsion angles, and then solving analytically for the values of the remaining 6 “pivot residue” torsion angles in order to ensure ideal peptide bond geometry at the breakpoint. In order to better bias sampling based on the conformational preferences of each amino acid, given its residue type and the presence or absence of a methyl group on its backbone nitrogen and on the backbone nitrogen of the next amino acid, we added a new GeneralizedKIC perturber, called randomize_backbone_by_rama_prepro, which uses the same Ramachandran potentials that are used for scoring to bias sampling. Since pivot residue torsion angles are determined analytically rather than being sampled from a biased distribution, a given kinematic closure solution may have backbone torsion values in any region of Ramachandran space. We therefore also added a GeneralizedKIC filter, called rama_prepro_check, which uses the precomputed Ramachandran potentials to discard solutions with pivot residues in poor regions of Ramachandran space. Because kinematic closure calculations and filtering are performed on raw vectors of degrees of freedom without manipulating full atomic-resolution models, they are extremely fast, allowing thousands of conformations to be sampled in seconds; downstream design steps in a design protocol occupy the majority of the computing time. Documentation for the GeneralizedKIC perturbers and filters may be found on the Rosetta help wiki at <a id="intref0155" href="https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/composite_protocols/generalized_kic/GeneralizedKICperturber" target="_blank">https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/composite_protocols/generalized_kic/GeneralizedKICperturber</a> and at <a id="intref0160" href="https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/composite_protocols/generalized_kic/GeneralizedKICfilter" target="_blank">https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/composite_protocols/generalized_kic/GeneralizedKICfilter</a>, respectively.</div><div id="p0230" role="paragraph">In addition, a new residue selector, called Unsat, was developed to permit the identification of amino acid residues with unsatisfied backbone hydrogen bond donors or acceptors in order to permit automatic N-methylation in the context of a RosettaScripts or PyRosetta&nbsp;design protocol. This selector is compatible with both asymmetric and symmetric structures. Documentation for the&nbsp;Unsat&nbsp;residue selector may be found in the Rosetta online documentation (<a id="intref0165" href="https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/ResidueSelectors/ResidueSelectors#residueselectors_conformation-dependent-residue-selectors_unsatselector" target="_blank">https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/ResidueSelectors/ResidueSelectors#residueselectors_conformation-dependent-residue-selectors_unsatselector</a>).</div><div id="p0235" role="paragraph">We added support for using N-methylated amino acids in Rosetta’s simple_cycpep_predict application, which is used for the prediction of peptide macrocycle structure from its amino acid sequence and for estimating the folding propensity of each designed peptide in a pool in order to rank and prioritize designs for synthesis. During peptide conformational sampling, simple_cycpep_predict uses the GeneralizedKIC method described above to sample closed conformations of a macrocycle rapidly, with the newly-implemented randomize_backbone_by_rama_prepro GeneralizedKIC perturber to bias sampling and the rama_prepro_check GeneralizedKIC filter to discard solutions with poor backbone conformations at pivot atoms. Rosetta’s energy_based_clustering application was also modified to allow it to cluster N-methylated peptides, and to include the carbon of the backbone methyl group&nbsp;as an atom used when aligning structures and computing RMSD values. These applications have documentation available on the Rosetta help wiki, at <a id="intref0170" href="https://www.rosettacommons.org/docs/latest/structure_prediction/simple_cycpep_predict#full-inputs_additional-flags-for-n-methylated-amino-acids" target="_blank">https://www.rosettacommons.org/docs/latest/structure_prediction/simple_cycpep_predict#full-inputs_additional-flags-for-n-methylated-amino-acids</a> and at <a id="intref0175" href="https://www.rosettacommons.org/docs/latest/application_documentation/analysis/energy_based_clustering_application" target="_blank">https://www.rosettacommons.org/docs/latest/application_documentation/analysis/energy_based_clustering_application</a>, respectively. Testing of these changes revealed a small bug related to an uninitialized variable in the Lazaridus-Karplus term in the Rosetta energy function, which was also corrected.</div><div id="p0240" role="paragraph">As of October 2021 (Rosetta Git revision a9ab4ac590fd0a4e2def5739deaeab02c72c949d), these enhancements have all been incorporated into public releases of Rosetta, with the exception of the N-methylated rotamer libraries. Due to their size, these libraries are not distributed with Rosetta (but for the N-methyl tryptophan rotamer library, used for unit testing), but all are bundled with noncanonical amino acid libraries distributed separately. Extensive unit tests have also been added to ensure that energy calculations involving N-methylated amino acids are invariant to cyclic permutation or to mirroring. Rosetta source code and compiled binaries are made freely available to academic, government, and not-for-profit users, and can be licensed from the University of Washington for corporate and for-profit use. To download and use Rosetta, please visit <a id="intref0180" href="https://www.rosettacommons.org/software/license-and-download" target="_blank">https://www.rosettacommons.org/software/license-and-download</a>.</div></section><section id="sec-4-4-2"><h4>Computational design of structured membrane-permeable peptides</h4><div id="p0245" role="paragraph">We modified the Rosetta generalized kinematic closure (GeneralizedKIC) based macrocycle design protocol described previously&nbsp;(<span class="dropBlock reference-citations"><a id="body-ref-sref1-4" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-4" aria-controls="sref1-4">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-optsrMTRQR98l-2" href="#" role="doc-biblioref" data-xml-rid="bib12 bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optsrMTRQR98l-2" aria-controls="optsrMTRQR98l-2">Hosseinzadeh et al., 2017, 2021</a><div class="dropBlock__holder" data-db-target-of="optsrMTRQR98l-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Hosseinzadeh, P. ∙ Watson, P.R. ∙ Craven, T.W. ...</div><div><strong>Anchor extension: a structure-guided approach to design cyclic peptides targeting enzyme active sites</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:3384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23609-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85107586900" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34099674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23609-8&amp;pmid=34099674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-optM1vu63fEjC-3" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optM1vu63fEjC-3" aria-controls="optM1vu63fEjC-3">Mulligan et al., 2021</a><div class="dropBlock__holder" data-db-target-of="optM1vu63fEjC-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Mulligan, V.K. ∙ Workman, S. ∙ Sun, T. ...</div><div><strong>Computationally designed peptide macrocycle inhibitors of New Delhi metallo-β-lactamase 1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong>, e2012800118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2012800118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85102719411" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2012800118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to enable design of 6 to 12 amino acid macrocycles capable of traversing the lipid membranes and cellular barriers. All steps for macrocycle design were implemented in RosettaScripts (<span class="dropBlock reference-citations"><a id="body-ref-optNogaYkWwmi" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optNogaYkWwmi" aria-controls="optNogaYkWwmi">Fleishman et al., 2011</a><div class="dropBlock__holder" data-db-target-of="optNogaYkWwmi"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Fleishman, S.J. ∙ Leaver-Fay, A. ∙ Corn, J.E. ...</div><div><strong>RosettaScripts: a scripting language interface to the Rosetta macromolecular modeling suite</strong></div><div><em>PLoS One.</em> 2011; <strong>6</strong>, e20161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0020161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-79959615159" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21731610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0020161&amp;pmid=21731610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0250" role="paragraph">Briefly, we chose a macrocycle size and initiated design calculations by constructing a linear polyglycine backbone of the selected amino acid length using the Rosetta PeptideStubMover (<span class="dropBlock reference-citations"><a id="body-ref-sref1-5" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-5" aria-controls="sref1-5">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this work, we performed separate design runs for each macrocycle size between 6 to 12 amino acids. Next, we declared a bond between the ‘C’ atom from the last residue and ‘N’ atom of the first residue in the polyglycine peptide and set up the distance, angle, and dihedral constraints for N-to-C terminal cyclization using the Rosetta PeptideCyclizeMover mover (<span class="dropBlock reference-citations"><a id="body-ref-sref12-6" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-6" aria-controls="sref12-6">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We selected one residue randomly as the ‘anchor residue’ and three additional residues as the ‘pivot residues’. The omega torsions for all residues were set to 180°; ɸ and ψ torsions for the anchor residues were randomly selected from a flat bottom mirror-symmetric Ramachandran table using the SetTorsion mover (<span class="dropBlock reference-citations"><a id="body-ref-sref1-6" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-6" aria-controls="sref1-6">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Following, we used the GeneralizedKIC mover to identify cyclic polyglycine peptides from the linear peptides (<span class="dropBlock reference-citations"><a id="body-ref-sref1-7" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-7" aria-controls="sref1-7">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Within the GenKIC mover, the ɸ and ψ dihedrals of the non-pivot and non-anchor residues were randomly drawn from a flat bottom mirror-symmetric Ramachandran table. The ɸ and ψ dihedrals for the pivot residues were calculated analytically by the kinematic closure algorithm (<span class="dropBlock reference-citations"><a id="body-ref-sref1-8" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-8" aria-controls="sref1-8">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-opt9b2tozWJtl" href="#" role="doc-biblioref" data-xml-rid="bib31 bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opt9b2tozWJtl" aria-controls="opt9b2tozWJtl">Coutsias et al., 2004; Mandell et al., 2009</a><div class="dropBlock__holder" data-db-target-of="opt9b2tozWJtl"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Coutsias, E.A. ∙ Seok, C. ∙ Jacobson, M.P. ...</div><div><strong>A kinematic view of loop closure</strong></div><div><em>J. Comput. Chem.</em> 2004; <strong>25</strong>:510-528</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.10416" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-1442281405" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14735570/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.10416&amp;pmid=14735570" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Mandell, D.J. ∙ Coutsias, E.A. ∙ Kortemme, T.</div><div><strong>Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling</strong></div><div><em>Nat. Methods.</em> 2009; <strong>6</strong>:551-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth0809-551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-68349104348" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19644455/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth0809-551&amp;pmid=19644455" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to find a combination of dihedral angles that give N-to-C cyclic peptide backbones. The criteria for closure were further defined to include a minimum number of internal backbone-to-backbone hydrogen bonds. The number of internal hydrogen bonds required was based on the length of the macrocycle: a minimum of 1 internal hydrogen bond was required for 6-7 amino acids, 2 hydrogen bonds were required for 8-9 amino acids, and 3 internal hydrogen bonds were required for macrocycles with 10 or more amino acids. In cases where GenKIC returned multiple cyclic solutions, we selected the lowest energy solutions based on a custom Rosetta energy function that includes only the fa_rep, fa_atr, hbond_sr_bb, hbond_lr_bb, rama_prepro, and p_aa_pp score terms (<span class="dropBlock reference-citations"><a id="body-ref-opty67Lg6LigC-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opty67Lg6LigC-1" aria-controls="opty67Lg6LigC-1">Alford et al., 2017</a><div class="dropBlock__holder" data-db-target-of="opty67Lg6LigC-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alford, R.F. ∙ Leaver-Fay, A. ∙ Jeliazkov, J.R. ...</div><div><strong>The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design</strong></div><div><em>J. Chem. Theory Comput.</em> 2017; <strong>13</strong>:3031-3048</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jctc.7b00125" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85020732573" target="_blank" aria-label="Open Scopus in new tab">Scopus (844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28430426/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jctc.7b00125&amp;pmid=28430426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For each successfully closed cyclic backbone, we used the Rosetta FastDesign mover to design an amino acid sequence that tried to minimize the overall energy of the macrocycle (<span class="dropBlock reference-citations"><a id="body-ref-sref1-9" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-9" aria-controls="sref1-9">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0255" role="paragraph">Since our goal was to design macrocycles that can traverse the lipid membranes passively, we removed NH groups not involved in hydrogen bonding by mutating the amino acids with such ‘unsatisfied’ backbone NH groups to their N-methyl variants. Residues with the unsatisfied NH groups were selected using the Unsat selector and mutated using the ModifyVariantType mover to their N-methylated versions. However, given the different torsional preferences of the N-methylated amino acids, it is possible that the mutations to N-methyl amino acids could expose NH groups from other amino acids as well. Therefore, we followed an iterative approach with three rounds of amino acid sequence design and N-methylation of exposed backbone NH groups in-between. In the first round of designs we used an energy function with upweighted (5X) backbone hydrogen bonding score terms to favor more internal backbone-to-backbone hydrogen bonds. For the second round of design, we used the standard Rosetta beta_nov16&nbsp;weights with constraints (<span class="dropBlock reference-citations"><a id="body-ref-opty67Lg6LigC-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opty67Lg6LigC-2" aria-controls="opty67Lg6LigC-2">Alford et al., 2017</a><div class="dropBlock__holder" data-db-target-of="opty67Lg6LigC-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alford, R.F. ∙ Leaver-Fay, A. ∙ Jeliazkov, J.R. ...</div><div><strong>The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design</strong></div><div><em>J. Chem. Theory Comput.</em> 2017; <strong>13</strong>:3031-3048</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jctc.7b00125" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85020732573" target="_blank" aria-label="Open Scopus in new tab">Scopus (844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28430426/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jctc.7b00125&amp;pmid=28430426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Park et al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div>Park, H., Bradley, P., Greisen, P., Jr, Liu, Y., Mulligan, V.K., Kim, D.E., Baker, D., and DiMaio, F. (2016). Simultaneous Optimization of Biomolecular Energy Functions on Features from Small Molecules and Macromolecules. J. Chem. Theory Comput. <i>12</i>, 6201–6212.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Park%2C+H.%2C+Bradley%2C+P.%2C+Greisen%2C+P.%2C+Jr%2C+Liu%2C+Y.%2C+Mulligan%2C+V.K.%2C+Kim%2C+D.E.%2C+Baker%2C+D.%2C+and+DiMaio%2C+F.+%282016%29.+Simultaneous+Optimization+of+Biomolecular+Energy+Functions+on+Features+from+Small+Molecules+and+Macromolecules.+J.+Chem.+Theory+Comput.+12%2C+6201%E2%80%936212." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the third round of design, we also allowed for cartesian minimization. We only allowed hydrophobic amino acids during design, and only D-amino acids were allowed at residue positions with positive ɸ, and only L-amino acids were allowed at residues with negative ɸ values. For some runs, we also used the AddCompositionConstraintMover to limit the minimum and maximum number of allowed prolines, D-prolines, and some bulky hydrophobic amino acids in the designed peptides (<span class="dropBlock reference-citations"><a id="body-ref-sref12-7" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-7" aria-controls="sref12-7">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Given the difficulty in synthesizing peptides with multiple N-methylated amino acids, we filtered the design models based on total number N-methylated amino acids in the design models and lack of any exposed NH groups in the final designed state using Rosetta SimpleMetrics (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Adolf-Bryfogle et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div>Adolf-Bryfogle, J., Labonte, J.W., Kraft, J.C., Shapovalov, M., Raemisch, S., Lütteke, T., DiMaio, F., Bahl, C.D., Pallesen, J., King, N.P., et al. (2021). Growing Glycans in Rosetta: Accurate de novo glycan modeling, density fitting, and rational sequon design.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Adolf-Bryfogle%2C+J.%2C+Labonte%2C+J.W.%2C+Kraft%2C+J.C.%2C+Shapovalov%2C+M.%2C+Raemisch%2C+S.%2C+L%C3%BCtteke%2C+T.%2C+DiMaio%2C+F.%2C+Bahl%2C+C.D.%2C+Pallesen%2C+J.%2C+King%2C+N.P.%2C+et+al.+%282021%29.+Growing+Glycans+in+Rosetta%3A+Accurate+de+novo+glycan+modeling%2C+density+fitting%2C+and+rational+sequon+design." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0260" role="paragraph">For each selected size range, approximately 10<sup>5</sup> designs were sampled. Design models were clustered based on the torsion bin strings calculated from the backbone dihedral angles as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref12-8" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-8" aria-controls="sref12-8">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Next, we pursued structure prediction of lowest-energy scoring designs from diverse clusters using Rosetta simple_cycpep_predict application (<span class="dropBlock reference-citations"><a id="body-ref-sref1-10" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-10" aria-controls="sref1-10">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12-9" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-9" aria-controls="sref12-9">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref12-10" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-10" aria-controls="sref12-10">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We evaluated the Energy vs. RMSD-to-Design plots (<a id="crosref0890" href="#figs2">Figure&nbsp;S2</a>) from structure prediction calculations and selected the structured and conformation-switching peptides based on the number of low-energy states. In cases where the structure prediction calculations identified conformations with lower energies (denoted by _LE) than the design model, the lower energy conformation was used as the model for structural comparisons.</div><div id="p0265" role="paragraph">Rosetta Macromolecular Modeling Suite can be downloaded from <a id="intref0185" href="https://rosettacommons.org/software" target="_blank">https://rosettacommons.org/software</a>. Rosetta Documentation describing the details of RosettaScripts and components used in the protocol described above are available at <a id="intref0190" href="https://new.rosettacommons.org/docs/latest/Home" target="_blank">https://new.rosettacommons.org/docs/latest/Home</a>. An example folder with the scripts and example command-line flags can also be found in the <a id="crosref0895" href="#mmc9">Data S5</a>. A sample RosettaScripts for designing a 10-amino acid peptide with N-methylated amino acids is provided below.</div><div id="p0270" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ROSETTASCRIPTS&gt;</span></div><div id="p0275" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Define scorefunctions to be used later in the script</span></div><div id="p0280" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;SCOREFXNS&gt;</span></div><div id="p0285" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ScoreFunction name="SFXN_STD" weights="beta_nov16.wts" &gt;</span></div><div id="p0290" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="atom_pair_constraint" weight="1" /&gt;</span></div><div id="p0295" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="dihedral_constraint" weight="1" /&gt;</span></div><div id="p0300" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="angle_constraint" weight="1" /&gt;</span></div><div id="p0305" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="aa_composition" weight="1.0" /&gt;</span></div><div id="p0310" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/ScoreFunction&gt;</span></div><div id="p0315" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ScoreFunction name="SFXN_STD_high_hbond" weights="beta_nov16.wts" &gt;</span></div><div id="p0320" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="atom_pair_constraint" weight="1" /&gt;</span></div><div id="p0325" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="dihedral_constraint" weight="1" /&gt;</span></div><div id="p0330" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="angle_constraint" weight="1" /&gt;</span></div><div id="p0335" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="aa_composition" weight="1.0" /&gt;</span></div><div id="p0340" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="hbond_sr_bb" weight="5.0" /&gt;</span></div><div id="p0345" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="hbond_lr_bb" weight="5.0" /&gt;</span></div><div id="p0350" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/ScoreFunction&gt;</span></div><div id="p0355" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ScoreFunction name="SFXN_CART" weights="beta_nov16_cart.wts" &gt;</span></div><div id="p0360" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="atom_pair_constraint" weight="1" /&gt;</span></div><div id="p0365" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="dihedral_constraint" weight="1" /&gt;</span></div><div id="p0370" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="angle_constraint" weight="1" /&gt;</span></div><div id="p0375" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="aa_composition" weight="1.0" /&gt;</span></div><div id="p0380" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/ScoreFunction&gt;</span></div><div id="p0385" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ScoreFunction name="SFXN_hbond_bb" weights="empty.wts" symmetric="0"&gt;</span></div><div id="p0390" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="fa_rep" weight="0.1" /&gt;</span></div><div id="p0395" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="fa_atr" weight="0.2" /&gt;</span></div><div id="p0400" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="hbond_sr_bb" weight="2.0" /&gt;</span></div><div id="p0405" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="hbond_lr_bb" weight="2.0" /&gt;</span></div><div id="p0410" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="rama_prepro" weight="0.45" /&gt;</span></div><div id="p0415" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="omega" weight="0.4" /&gt;</span></div><div id="p0420" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Reweight scoretype="p_aa_pp" weight="0.6" /&gt;</span></div><div id="p0425" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/ScoreFunction&gt;</span></div><div id="p0430" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/SCOREFXNS&gt;</span></div><div id="p0435" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;RESIDUE_SELECTORS&gt;</span></div><div id="p0440" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Define different residue selections to be used later in the script</span></div><div id="p0445" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Chain name="chainA" chains="A"/&gt;</span></div><div id="p0450" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Chain name="sel_A" chains="1"/&gt;</span></div><div id="p0455" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#select the regions with phi less than or greater than zero for designing with L- and D-amino acids</span></div><div id="p0460" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Phi name="posPhi" select_positive_phi="true" /&gt;</span></div><div id="p0465" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Phi name="negPhi" select_positive_phi="false" /&gt;</span></div><div id="p0470" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Bin name="ProBin" bin="LPRO" bin_params_file="PRO_DPRO" /&gt;</span></div><div id="p0475" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Bin name="DProBin" bin="DPRO" bin_params_file="PRO_DPRO" /&gt;</span></div><div id="p0480" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#we need to select residues that are outside +/-2 amino acids of each residue to calculate the internal h-bonds</span></div><div id="p0485" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_1" resnums="4,5,6,7,8"/&gt;</span></div><div id="p0490" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_2" resnums="5,6,7,8,9"/&gt;</span></div><div id="p0495" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_3" resnums="6,7,8,9,10"/&gt;</span></div><div id="p0500" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_4" resnums="1,7,8,9,10"/&gt;</span></div><div id="p0505" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_5" resnums="1,2,8,9,10"/&gt;</span></div><div id="p0510" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_6" resnums="1,2,3,9,10"/&gt;</span></div><div id="p0515" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_7" resnums="1,2,3,4,10"/&gt;</span></div><div id="p0520" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_8" resnums="1,2,3,4,5"/&gt;</span></div><div id="p0525" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_9" resnums="2,3,4,5,6"/&gt;</span></div><div id="p0530" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Index name="hbonds_to_10" resnums="3,4,5,6,7"/&gt;</span></div><div id="p0535" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#select the residues with unsatisfied NH grups</span></div><div id="p0540" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Unsat name="unsat_amines" scorefxn="SFXN_STD" check_acceptors="false" consider_mainchain_only="false"/&gt;</span></div><div id="p0545" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;And name="ProBin_unsat" selectors="ProBin,unsat_amines" /&gt;</span></div><div id="p0550" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;And name="DProBin_unsat" selectors="DProBin,unsat_amines" /&gt;</span></div><div id="p0555" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/RESIDUE_SELECTORS&gt;</span></div><div id="p0560" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;SIMPLE_METRICS&gt;</span></div><div id="p0565" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;SelectedResidueCountMetric name="count_unsats" residue_selector="unsat_amines" /&gt;</span></div><div id="p0570" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/SIMPLE_METRICS&gt;</span></div><div id="p0575" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;TASKOPERATIONS&gt;</span></div><div id="p0580" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;InitializeFromCommandline name="init"/&gt;</span></div><div id="p0585" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;IncludeCurrent name="current"/&gt;</span></div><div id="p0590" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;LimitAromaChi2 name="limitchi2" chi2max="110" chi2min="70" include_trp="True" /&gt;</span></div><div id="p0595" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>### expanded chi1 and chi2 rotamer libraries</span></div><div id="p0600" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ExtraRotamersGeneric name="ex1_ex2" ex1="1" ex2="1"/&gt;</span></div><div id="p0605" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ExtraRotamersGeneric name="ex1" ex1="1"/&gt;</span></div><div id="p0610" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Add the allowed residues from resfile that apply to selected regions of the peptide</span></div><div id="p0615" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ReadResfile name="l_res" filename="l_res.txt" selector="negPhi"/&gt;</span></div><div id="p0620" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ReadResfile name="d_res" filename="d_res.txt" selector="posPhi"/&gt;</span></div><div id="p0625" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ReadResfile name="lpro_res" filename="pro.txt" selector="ProBin_unsat"/&gt;</span></div><div id="p0630" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ReadResfile name="dpro_res" filename="dpro.txt" selector="DProBin_unsat"/&gt;</span></div><div id="p0635" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/TASKOPERATIONS&gt;</span></div><div id="p0640" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;FILTERS&gt;</span></div><div id="p0645" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#calculation of internal h-bonds in the peptide</span></div><div id="p0650" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds1" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="1" residue_selector="hbonds_to_1"/&gt;</span></div><div id="p0655" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds2" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="2" residue_selector="hbonds_to_2"/&gt;</span></div><div id="p0660" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds3" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="3" residue_selector="hbonds_to_3"/&gt;</span></div><div id="p0665" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds4" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="4" residue_selector="hbonds_to_4"/&gt;</span></div><div id="p0670" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds5" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="5" residue_selector="hbonds_to_5"/&gt;</span></div><div id="p0675" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds6" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="6" residue_selector="hbonds_to_6"/&gt;</span></div><div id="p0680" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds7" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="7" residue_selector="hbonds_to_7"/&gt;</span></div><div id="p0685" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds8" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="8" residue_selector="hbonds_to_8"/&gt;</span></div><div id="p0690" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds9" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="9" residue_selector="hbonds_to_9"/&gt;</span></div><div id="p0695" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;HbondsToResidue name="hbonds10" partners="0" energy_cutoff="-0.25" backbone="1" bb_bb="1" sidechain="0" residue="10" residue_selector="hbonds_to_10"/&gt;</span></div><div id="p0700" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;CombinedValue name="total_hbonds" threshold="-3.0"&gt; #change it to the -1<sup>∗</sup>number of hydrogens bonds required</span></div><div id="p0705" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds1" factor="-0.5"/&gt;</span></div><div id="p0710" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds2" factor="-0.5"/&gt;</span></div><div id="p0715" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds3" factor="-0.5"/&gt;</span></div><div id="p0720" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds4" factor="-0.5"/&gt;</span></div><div id="p0725" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds5" factor="-0.5"/&gt;</span></div><div id="p0730" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds6" factor="-0.5"/&gt;</span></div><div id="p0735" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds7" factor="-0.5"/&gt;</span></div><div id="p0740" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds8" factor="-0.5"/&gt;</span></div><div id="p0745" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds9" factor="-0.5"/&gt;</span></div><div id="p0750" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter_name="hbonds10" factor="-0.5"/&gt;</span></div><div id="p0755" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/CombinedValue&gt;</span></div><div id="p0760" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Remove designs with any residue with a hydrogen bond acceptor that makes more than allowed number of h-bonds</span></div><div id="p0765" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;OversaturatedHbondAcceptorFilter name="oversat" scorefxn="SFXN_STD" max_allowed_oversaturated="0" consider_mainchain_only="false" confidence="1" /&gt;</span></div><div id="p0770" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Remove designs with more than 5 N-methylated amino acids</span></div><div id="p0775" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ResidueCount name="nme_count" max_residue_count="5" include_property="N_METHYLATED" confidence="1" /&gt;</span></div><div id="p0780" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ResidueCount name="nme_count2" max_residue_count="5" include_property="N_METHYLATED" confidence="1" /&gt;</span></div><div id="p0785" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;SimpleMetricFilter name="C6s" metric="count_unsats" cutoff="0" comparison_type="lt_or_eq"/&gt;</span></div><div id="p0790" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/FILTERS&gt;</span></div><div id="p0795" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;MOVERS&gt;</span></div><div id="p0800" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Construct the intial poly glycine chain</span></div><div id="p0805" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;PeptideStubMover name="intial_stub" reset="true"&gt;</span></div><div id="p0810" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0815" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0820" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0825" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0830" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0835" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0840" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0845" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0850" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0855" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Append resname="GLY" /&gt;</span></div><div id="p0860" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/PeptideStubMover&gt;</span></div><div id="p0865" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Declare the peptide bond and cyclization constraints between the N- and C-terminal</span></div><div id="p0870" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;PeptideCyclizeMover name="cyclize_peptide" /&gt;</span></div><div id="p0875" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Set the intial phi/psi/omega torsions for the poly glycine chain</span></div><div id="p0880" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;SetTorsion name="torsion1"&gt;</span></div><div id="p0885" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Torsion residue="ALL" torsion_name="omega" angle="180.0" /&gt;</span></div><div id="p0890" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Torsion residue="1,2,3,4,5,6,7,8,9,10" torsion_name="rama" angle="rama_biased" custom_rama_table="flat_symm_dl_aa_ramatable"/&gt;</span></div><div id="p0895" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/SetTorsion&gt;</span></div><div id="p0900" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Set some filtering criteria based on internal h-bonds and oversaturation to accept or reject the solution from GenKIC</span></div><div id="p0905" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ParsedProtocol name="preselection_pp"&gt;</span></div><div id="p0910" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter="total_hbonds"/&gt;</span></div><div id="p0915" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter="oversat"/&gt;</span></div><div id="p0920" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="cyclize_peptide" /&gt;</span></div><div id="p0925" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/ParsedProtocol&gt;</span></div><div id="p0930" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#GenKIC method for cyclizing the peptide</span></div><div id="p0935" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;GeneralizedKIC name="genkic1" closure_attempts="1000" selector="lowest_energy_selector" pre_selection_mover="preselection_pp" stop_when_n_solutions_found="1" stop_if_no_solution="500" selector_scorefunction="SFXN_hbond_bb" &gt;</span></div><div id="p0940" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="7" /&gt;</span></div><div id="p0945" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="8" /&gt;</span></div><div id="p0950" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="9" /&gt;</span></div><div id="p0955" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="10" /&gt;</span></div><div id="p0960" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="1" /&gt;</span></div><div id="p0965" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="2" /&gt;</span></div><div id="p0970" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="3" /&gt;</span></div><div id="p0975" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="4" /&gt;</span></div><div id="p0980" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue res_index="5" /&gt; #we are leaving resn 6 out of loop to be closed</span></div><div id="p0985" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;SetPivots atom1="CA" atom2="CA" atom3="CA" res1="7" res2="2" res3="5" /&gt;</span></div><div id="p0990" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;CloseBond prioratom_res="10" prioratom="CA" res1="10" atom1="C" res2="1" atom2="N" followingatom="CA" followingatom_res="1" angle1="121.69997" angle2="116.199993" bondlength="1.328685" torsion="180.0" randomize_flanking_torsions="false" /&gt;</span></div><div id="p0995" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddPerturber effect="set_dihedral"&gt;</span></div><div id="p1000" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddAtoms atom1="C" res1="10" res2="1" atom2="N" /&gt;</span></div><div id="p1005" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddValue value="180.0" /&gt;</span></div><div id="p1010" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/AddPerturber&gt;</span></div><div id="p1015" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddPerturber effect="randomize_alpha_backbone_by_rama" custom_rama_table="flat_symm_dl_aa_ramatable"&gt;</span></div><div id="p1020" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="7" /&gt;</span></div><div id="p1025" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="8" /&gt;</span></div><div id="p1030" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="9"/&gt;</span></div><div id="p1035" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="10"/&gt;</span></div><div id="p1040" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="1"/&gt;</span></div><div id="p1045" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="2"/&gt;</span></div><div id="p1050" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="3"/&gt;</span></div><div id="p1055" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="4"/&gt;</span></div><div id="p1060" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddResidue index="5"/&gt;</span></div><div id="p1065" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/AddPerturber&gt;</span></div><div id="p1070" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddFilter type="loop_bump_check" /&gt;</span></div><div id="p1075" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddFilter type="rama_prepro_check" residue="7" rama_cutoff_energy="2" /&gt;</span></div><div id="p1080" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddFilter type="rama_prepro_check" residue="2" rama_cutoff_energy="2" /&gt;</span></div><div id="p1085" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddFilter type="rama_prepro_check" residue="5" rama_cutoff_energy="2" /&gt;</span></div><div id="p1090" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/GeneralizedKIC&gt;</span></div><div id="p1095" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Add constraints to set min and max number of some amino acids</span></div><div id="p1100" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddCompositionConstraintMover name="global" filename="comp/global_preferences.comp" selector="sel_A" /&gt;</span></div><div id="p1105" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;AddCompositionConstraintMover name="pro_comp" filename="comp/pro.comp" selector="sel_A" /&gt;</span></div><div id="p1110" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#sequence design with upweighted hydrogen bonding score terms to favor internal backbon-to-backbone hydrogen bonds</span></div><div id="p1115" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;FastDesign name="fdesign_hbond_upweighted" scorefxn="SFXN_STD_high_hbond" repeats="1" task_operations="limitchi2,ex1_ex2,ex1,d_res,dpro_res,l_res,lpro_res" ramp_down_constraints="False" &gt;</span></div><div id="p1120" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;MoveMap name="fdesign_step1_mm"&gt;</span></div><div id="p1125" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Chain number="1" chi="1" bb="1"/&gt;</span></div><div id="p1130" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/MoveMap&gt;</span></div><div id="p1135" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/FastDesign&gt;</span></div><div id="p1140" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#Calculate the number of unsats in the peptide</span></div><div id="p1145" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;RunSimpleMetrics name="run_metrics" metrics="count_unsats" prefix="end_unsat" /&gt;</span></div><div id="p1150" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#mutate the selected residues to their N-methyl versions</span></div><div id="p1155" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;ModifyVariantType name="N_methylation" add_type="N_METHYLATION" residue_selector="unsat_amines" /&gt;</span></div><div id="p1160" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#design amino acid sequence with STD sfxn</span></div><div id="p1165" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;FastDesign name="fdesign" scorefxn="SFXN_STD" repeats="2" task_operations="limitchi2,ex1_ex2,ex1,d_res,l_res" ramp_down_constraints="False" &gt;</span></div><div id="p1170" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;MoveMap name="fdesign_step2_mm"&gt;</span></div><div id="p1175" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Chain number="1" chi="1" bb="1"/&gt;</span></div><div id="p1180" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/MoveMap&gt;</span></div><div id="p1185" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/FastDesign&gt;</span></div><div id="p1190" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>#design amino acid sequence with Cartesian sfxn</span></div><div id="p1195" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;FastDesign name="fdesign_cart" scorefxn="SFXN_CART" repeats="1" task_operations="limitchi2,ex1_ex2,ex1,d_res,l_res" ramp_down_constraints="False" cartesian="True" &gt;</span></div><div id="p1200" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;MoveMap name="fdesign_step3_mm"&gt;</span></div><div id="p1205" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Chain number="1" chi="1" bb="1"/&gt;</span></div><div id="p1210" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/MoveMap&gt;</span></div><div id="p1215" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/FastDesign&gt;</span></div><div id="p1220" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/MOVERS&gt;</span></div><div id="p1225" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;PROTOCOLS&gt;</span></div><div id="p1230" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="intial_stub" /&gt;</span></div><div id="p1235" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="torsion1" /&gt;</span></div><div id="p1240" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="cyclize_peptide" /&gt;</span></div><div id="p1245" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="genkic1" /&gt;</span></div><div id="p1250" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="cyclize_peptide" /&gt;</span></div><div id="p1255" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="global" /&gt;</span></div><div id="p1260" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="fdesign_hbond_upweighted" /&gt;</span></div><div id="p1265" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter="total_hbonds" /&gt;</span></div><div id="p1270" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter="oversat" /&gt;</span></div><div id="p1275" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="N_methylation" /&gt;</span></div><div id="p1280" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add filter="nme_count" /&gt;</span></div><div id="p1285" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="pro_comp" /&gt;</span></div><div id="p1290" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="fdesign" /&gt;</span></div><div id="p1295" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="N_methylation" /&gt;</span></div><div id="p1300" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="fdesign_cart" /&gt;</span></div><div id="p1305" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="cyclize_peptide" /&gt;</span></div><div id="p1310" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;Add mover="run_metrics" /&gt;</span></div><div id="p1315" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/PROTOCOLS&gt;</span></div><div id="p1320" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;OUTPUT scorefxn="SFXN_CART"/&gt;</span></div><div id="p1325" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;/ROSETTASCRIPTS&gt;</span></div><div id="p1330" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>We used the following command line options to run the Rosettascripts described earlier:</span></div><div id="p1335" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>&lt;path_to_rosetta_binary_files&gt;/rosetta_scripts.default.linuxgccrelease -in:path:database &lt;path_to_rosetta&gt;/main/database -parser:protocol permeability_design.xml -in:file:s input.pdb -nstruct 100&nbsp;-overwrite -beta_nov16 -packer_palette:extra_base_type_file d_packer_palette_hydrophobic.txt&nbsp;-corrections:beta_nov16 true</span></div><div id="p1340" role="paragraph">The example script filters designs to have at least 3 internal hydrogen bonds and less than 5 N-methylated amino acids. These are pretty stringent constraints, so most of the design attempts will fail to return a solution that satisfies all these constraints. For testing purposes, we recommend changing these thresholds to more permissible values. The script above was tested with Rosetta Git Revision 61f38062d297c1a83107a77cd5b33afceeee53a0.</div><div id="p1345" role="paragraph">A packer palette file defining the D-amino acids to be used during design is also required to run the protocol described above. Here is an example packer_palette (<b>d_packer_palette_hydrophobic.txt</b>). The following packer palette only allows design with D-alanine, D-phenylalanine, D-isoleucine, D-leucine, D-proline, and D-valine. The file can be tuned to include more amino acids (including non-canonical amino acids) or exclude any specific amino acid that must be avoided during the sequence design step.</div><div id="p1350" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>DALA DPHE DILE DLEU DPRO DVAL</span></div><div id="p1355" role="paragraph">Running the Rosetta Scripts protocol described above also requires a residue file that restricts the allowed amino acids at positions with positive ɸ values to only D-amino acids. Following is an example resfile (<b>d_res.txt)</b> that can be used to design with only D-amino acids when combined with a residue selector that selects for positions with positive ɸ values.</div><div id="p1360" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>NOTAA CDEQNSTYRHKWG</span></div><div id="p1365" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>start</span></div><div id="p1370" role="paragraph">Similarly, the design method also requires a resfile file to define amino acids allowed at residue positions with negative phi values. An example resfile (<b>l_res.txt</b>) file is given below:</div><div id="p1375" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>PIKAA AVLFIP</span></div><div id="p1380" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>start</span></div><div id="p1385" role="paragraph">We also use amino acids composition constraints to control the distribution and total number of some amino acids. Details about the AA composition constraints and AA composition energy can be found in the help wiki: <a id="intref0195" href="https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/movers_pages/AddCompositionConstraintMover" target="_blank">https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/movers_pages/AddCompositionConstraintMover</a></div><div id="p1390" role="paragraph">An example composition constraint file (<b>global.comp</b>) is shown below.</div><div id="p1395" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>PENALTY_DEFINITION</span></div><div id="p1400" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>TYPE PHE DPH</span></div><div id="p1405" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>DELTA_START -2</span></div><div id="p1410" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>DELTA_END 2</span></div><div id="p1415" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>PENALTIES 0 0 0 5 25</span></div><div id="p1420" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>ABSOLUTE 2</span></div><div id="p1425" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>BEFORE_FUNCTION CONSTANT</span></div><div id="p1430" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>AFTER_FUNCTION QUADRATIC</span></div><div id="p1435" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>END_PENALTY_DEFINITION</span></div><div id="p1440" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>PENALTY_DEFINITION</span></div><div id="p1445" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>TYPE ALA DAL</span></div><div id="p1450" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>DELTA_START -2</span></div><div id="p1455" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>DELTA_END 2</span></div><div id="p1460" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>PENALTIES 0 0 0 5 25</span></div><div id="p1465" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>ABSOLUTE 2</span></div><div id="p1470" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>BEFORE_FUNCTION CONSTANT</span></div><div id="p1475" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>AFTER_FUNCTION QUADRATIC</span></div><div id="p1480" role="paragraph"><span data-style="monospace"><span style="display: inline-block; width: 1em;"></span>END_PENALTY_DEFINITION</span></div></section><section id="sec-4-4-3"><h4>Multistate Design of conformation-switching peptides</h4><div id="p1485" role="paragraph">We used a multistate design method to generate chameleonic macrocycles. Specifically, we implemented a genetic algorithm in PyRosetta-3 that optimizes mutations to obtain two isoenergetic low energy states for the designed amino acid sequences (<i>see</i> <a id="crosref0900" href="#mmc1">Data S5</a> <i>for scripts and required files</i>). The starting sequence is used to generate a list of 1000 variants that contain mutations to different hydrophobic residues while maintaining the original chirality and N-methylation patterns. Each mutant sequence is then threaded onto the original backbone conformations and scored using the REF2015 Rosetta energy function (<span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Park et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div>Park, H., Bradley, P., Greisen, P., Jr, Liu, Y., Mulligan, V.K., Kim, D.E., Baker, D., and DiMaio, F. (2016). Simultaneous Optimization of Biomolecular Energy Functions on Features from Small Molecules and Macromolecules. J. Chem. Theory Comput. <i>12</i>, 6201–6212.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Park%2C+H.%2C+Bradley%2C+P.%2C+Greisen%2C+P.%2C+Jr%2C+Liu%2C+Y.%2C+Mulligan%2C+V.K.%2C+Kim%2C+D.E.%2C+Baker%2C+D.%2C+and+DiMaio%2C+F.+%282016%29.+Simultaneous+Optimization+of+Biomolecular+Energy+Functions+on+Features+from+Small+Molecules+and+Macromolecules.+J.+Chem.+Theory+Comput.+12%2C+6201%E2%80%936212." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The sequences are filtered to assure that the total energy of the sequence on both backbones is less than 10&nbsp;kcal/mol and that the difference between the two states is less than 6&nbsp;kcal/mol. In order to select for sequences that stabilize both conformations while maintaining low energy, sequences are then given a final score equal to (−5×abs(eA − eB))-(eA-eB) where eA and eB are the Rosetta scores of a given sequence threaded onto the first and second conformations, respectively. The best 500 sequences based on this metric are carried on to the next cycle of evaluation, where a point mutant of each sequence is added to the list, and the process is repeated. This algorithm was run for 1,000 generations and then the best sequence was chosen. Structure prediction is performed to ensure both desired conformational states are featured as low-energy minima in the conformational landscape.</div></section><section id="sec-4-4-4"><h4>Conversion of structured peptides to conformation-switching peptides</h4><div id="p1490" role="paragraph">For some of the crystallographic confirmed designs, we attempted to identify amino acid substitutions that could create secondary isoenergetic minima. We implemented a PyRosetta script that loops through each amino acid position of a given structure and mutates the original residue to other hydrophobic amino acids while maintaining chirality and N-methylation patterns (<i>see</i> <a id="crosref0905" href="#supplementary-material">supplemental information</a> <i>for scripts and required files</i>). Each mutated version of the original structure was then energy minimized using the Rosetta FastRelax protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref1-11" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-11" aria-controls="sref1-11">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to ease any strain induced by the mutation. The structure-energy landscape for the full set of mutated sequence-structure pairs was evaluated using Rosetta cycpep_predict application (<span class="dropBlock reference-citations"><a id="body-ref-sref1-12" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-12" aria-controls="sref1-12">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12-11" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-11" aria-controls="sref12-11">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and designs with isoenergetic alternate states separated by <i>cis</i>/<i>trans</i> isomerization were selected for experimental characterization.</div></section><section id="sec-4-4-5"><h4>Structure prediction of the designed macrocycles</h4><div id="p1495" role="paragraph">We used the Rosetta simple_cycpep_predict application as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref1-13" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-13" aria-controls="sref1-13">Bhardwaj et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12-12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-12" aria-controls="sref12-12">Hosseinzadeh et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to evaluate and conformational landscape for the designed amino acid sequences of macrocycles. We ran the structure prediction calculations using the Rosetta@Home platform. For each macrocycle, we generated&nbsp;&gt;&nbsp;10<sup>4</sup> energy minimized cyclic conformations of the designed amino acid sequence. For each conformation, we calculated the RMSD to the design model and the energy using the Rosetta REF2015 energy function (<span class="dropBlock reference-citations"><a id="body-ref-sref44-3" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-3" aria-controls="sref44-3">Park et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref44-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div>Park, H., Bradley, P., Greisen, P., Jr, Liu, Y., Mulligan, V.K., Kim, D.E., Baker, D., and DiMaio, F. (2016). Simultaneous Optimization of Biomolecular Energy Functions on Features from Small Molecules and Macromolecules. J. Chem. Theory Comput. <i>12</i>, 6201–6212.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Park%2C+H.%2C+Bradley%2C+P.%2C+Greisen%2C+P.%2C+Jr%2C+Liu%2C+Y.%2C+Mulligan%2C+V.K.%2C+Kim%2C+D.E.%2C+Baker%2C+D.%2C+and+DiMaio%2C+F.+%282016%29.+Simultaneous+Optimization+of+Biomolecular+Energy+Functions+on+Features+from+Small+Molecules+and+Macromolecules.+J.+Chem.+Theory+Comput.+12%2C+6201%E2%80%936212." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The parameters used to run the scripts on Rosetta@Home are equivalent to running the following script on a compute cluster:</div><div id="p1500" role="paragraph">/home/gauravb/cycpep_cst_fix_copy/Rosetta/main/source/bin/simple_cycpep_predict.default.linuxgccrelease -in:path:database /home/gauravb/cycpep_cst_fix_copy/Rosetta/main/database @input.flags -nstruct 100</div><div id="p1505" role="paragraph">An example flags file (input.flags) is shown below:</div><div id="p1510" role="paragraph">-cyclic_peptide:sequence_file [SEQ_TEXT_FILE]</div><div id="p1515" role="paragraph">-symmetric_gly_tables true</div><div id="p1520" role="paragraph">-score:weights ref2015.wts</div><div id="p1525" role="paragraph">-in:file:native [DESIGN_PDB_FILE]</div><div id="p1530" role="paragraph">-in:file:fullatom</div><div id="p1535" role="paragraph">-cyclic_peptide:genkic_closure_attempts 1000</div><div id="p1540" role="paragraph">-cyclic_peptide:genkic_min_solution_count 1</div><div id="p1545" role="paragraph">-cyclic_peptide:min_genkic_hbonds 1</div><div id="p1550" role="paragraph">-cyclic_peptide:min_final_hbonds 1</div><div id="p1555" role="paragraph">-cyclic_peptide:fast_relax_rounds 20</div><div id="p1560" role="paragraph">-cyclic_peptide:rama_cutoff 10.0</div><div id="p1565" role="paragraph">-ex1</div><div id="p1570" role="paragraph">-ex2</div><div id="p1575" role="paragraph">-extrachi_cutoff 5</div><div id="p1580" role="paragraph">-mute all</div><div id="p1585" role="paragraph">-out:file:silent [OUTPUT_FILE_NAME]</div><div id="p1590" role="paragraph">-cyclic_peptide:n_methyl_positions [N_METHYLATED_POSITIONS]</div><div id="p1595" role="paragraph">In the flags file above, SEQ_TEXT_FILE needs to be replaced with the path for a text file containing the amino acid sequence of the design model, DESIGN_PDB_FILE needs to be replaced with the path for the PDB file of the design model, OUTPUT_FILE_NAME needs to be replaced with the path and filename for the output silent file name, and N_METHYLATED_POSITIONS needs to be replaced with the comma-separated list of amino acid positions that are N-methylated in the design model.</div><div id="p1600" role="paragraph">After the completion of the structure prediction runs, we plotted the Energy vs. RMSD to the design model and selected macrocycles that converged to the design model as the unique low-energy states or as one of the alternate isoenergetic states.</div></section><section id="sec-4-4-6"><h4>Peptide synthesis and purification</h4><div id="p1605" role="paragraph">Macrocyclic peptides were either purchased from Wuxi AppTec at greater than 99% purity or synthesized in-house using the Fmoc-based solid-phase peptide synthesis methods on an automated CEM Liberty Blue peptide synthesizer at a 0.1&nbsp;mmol scale. Linear peptides were assembled on Cl-TCP(Cl) ProTide resin purchased from CEM, using standard coupling of 2&nbsp;minutes at 90°C and deprotection for 1&nbsp;minute at 90°C, unless the Fmoc-amino acid was being coupled to an N-methylated amino acid in which case coupling was performed for 10&nbsp;minutes at 90°C. We used Oxyma Pure/DIC as the coupling agent and 20% piperidine in DMF for deprotection as per standard protocols by CEM. Linear, protected peptides were released from the resin by repeated washes with 1% TFA in DCM. The wash volumes containing protected peptide were ejected into a round bottom flask containing a 50:50 mixture of acetonitrile and water of greater volume than the volume of the washes. DCM was removed by rotary evaporation and the resulting mixture was lyophilized to dryness. Dry, protected peptides were solubilized in DCM and placed on a stir plate with a stir bar. Two equivalents of PyAOP were added directly to the solution followed by 5 equivalents of DIEA and left to stir overnight at room temperature. DCM was removed by rotary evaporation leaving an oil-like solution in the round bottom flask, which was resuspended in 50:50 acetonitrile:water and purified by reverse-phase high-performance liquid chromatography (RP-HPLC). Peptides were purified on an Agilent Infinity 1260 HPLC using 1% per minute gradient on Agilent ZORBAX SB-C18, 80Å, 5&nbsp;μm, 9.4 x 250&nbsp;mm column with a gradient of solvent A: 0.1% TFA in water, and solvent B: 0.1% TFA in acetonitrile. Mass spectrometry was used to confirm the synthesis of the correct product; purified peptides were direct-injected on a Thermo TSQ Quantum Access mass spectrometer. Isobaric peaks matching the expected molecular weight (labeled as p1 and p2) were seen in some cases and were tested separately for permeability and structure. Shifting the cyclization point during resynthesis removed the isobaric peaks for a subset of such cases. For example, D10.1 was resynthesized by CSBio for rodent studies with a different cyclization point and did not show isobaric peaks during purification. Designs, D8.29, D9.21, D9.28, D9.31, D9.40, D10.40, D10.44, and D11.14 were synthesized and tested with norleucine replacing methionine residues to avoid issues with sidechain oxidation.</div></section><section id="sec-4-4-7"><h4>Parallel Artificial Membrane Permeability Assay (PAMPA)</h4><div id="p1610" role="paragraph">Passive permeability was assayed using standard methods on a Corning® BioCoat™ Pre-coated PAMPA Plate System (<span class="dropBlock reference-citations"><a id="body-ref-optPHWTCfBq6j" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optPHWTCfBq6j" aria-controls="optPHWTCfBq6j">Kansy et al., 1998</a><div class="dropBlock__holder" data-db-target-of="optPHWTCfBq6j"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Kansy, M. ∙ Senner, F. ∙ Gubernator, K.</div><div><strong>Physicochemical high throughput screening: parallel artificial membrane permeation assay in the description of passive absorption processes</strong></div><div><em>J. Med. Chem.</em> 1998; <strong>41</strong>:1007-1010</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jm970530e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0032568397" target="_blank" aria-label="Open Scopus in new tab">Scopus (1147)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9544199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjm970530e&amp;pmid=9544199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Starting stock solutions of peptides were prepared by adding 1-2&nbsp;mg of peptide in 1&nbsp;ml of DMSO solution. Stock solutions were diluted 20X in Phosphate buffered saline (PBS) buffer to create solutions with 5% DMSO. 300 uL of peptide solution was added to the donor well and 250 uL of 5% DMSO 1X PBS was added to the acceptor well. Donor and acceptor plates were incubated together for 16-20 hours and transferred to 96-well plates at the end of incubation for measuring concentrations of peptide in starting solution, donor wells, and acceptor wells using an RP-HPLC and mass spectrometry on Agilent 6230 LC/TOF. Samples were separated on a 20%/min gradient of Solvent A (water, 0.1% formic acid) and Solvent B (acetonitrile, 0.1% formic acid) ran using Acquity UPLC BEH C18 1.7&nbsp;μm column. The area under the curve for the peaks matching the peptide mass was calculated, and peptide concentrations were calculated by fitting sample peak areas to a calibration curve of an 8 point two-fold serial dilution series from the starting donor solution. Propranolol and Cyclosporine were used as positive controls during the PAMPA. Apparent permeability (P<sub>app</sub>) were calculated as follows:<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper P Subscript normal a normal p normal p Baseline equals minus normal l normal n left parenthesis 1 minus normal upper C Subscript normal upper A Baseline divided by normal upper C Subscript normal e Baseline right parenthesis divided by normal upper A asterisk left parenthesis 1 divided by normal upper V Subscript normal upper D Baseline plus 1 divided by normal upper V Subscript normal upper A Baseline right parenthesis asterisk normal t semicolon normal upper C Subscript normal e Baseline equals left parenthesis normal upper C Subscript normal upper D Baseline asterisk normal upper V Subscript normal upper D Baseline plus normal upper C Subscript normal upper A Baseline asterisk normal upper V Subscript normal upper A Baseline right parenthesis divided by left parenthesis normal upper V Subscript normal upper D Baseline plus normal upper V Subscript normal upper A Baseline right parenthesis semicolon" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(115 (113 (7 0 (6 1 4 2 5 3)) 9 (112 (36 (33 11 (32 12 30 13 31 (29 14 (22 15 17 (27 (21 19 20) 23 (26 24 25))) 28))) 34 35) 38 (57 40 (55 (46 41 42 (45 43 44)) 48 (54 49 50 (53 51 52))) 56) 59 61)) 62 (114 (66 64 65) 68 (110 (96 70 (93 (92 (73 71 72) 75 (79 77 78)) 81 (94 (85 83 84) 87 (91 89 90))) 95) 97 (109 98 (107 (101 99 100) 103 (106 104 105)) 108))) 111)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="115" data-semantic-children="113,62,114,111" data-semantic-content="62,111" data-semantic-owns="113 62 114 111" data-semantic-speech="normal upper P Subscript normal a normal p normal p Baseline equals minus normal l normal n left parenthesis 1 minus normal upper C Subscript normal upper A Baseline divided by normal upper C Subscript normal e Baseline right parenthesis divided by normal upper A asterisk left parenthesis 1 divided by normal upper V Subscript normal upper D Baseline plus 1 divided by normal upper V Subscript normal upper A Baseline right parenthesis asterisk normal t semicolon normal upper C Subscript normal e Baseline equals left parenthesis normal upper C Subscript normal upper D Baseline asterisk normal upper V Subscript normal upper D Baseline plus normal upper C Subscript normal upper A Baseline asterisk normal upper V Subscript normal upper A Baseline right parenthesis divided by left parenthesis normal upper V Subscript normal upper D Baseline plus normal upper V Subscript normal upper A Baseline right parenthesis semicolon"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="113" data-semantic-children="7,112" data-semantic-content="9" data-semantic-parent="115" data-semantic-owns="7 9 112"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-parent="113" data-semantic-owns="0 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c50">P</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="7" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c61">a</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c70">p</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c70">p</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="113" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="112" data-semantic-children="36,57,61" data-semantic-content="38,59" data-semantic-parent="113" data-semantic-owns="36 38 57 59 61"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="36" data-semantic-children="33,35" data-semantic-content="34" data-semantic-parent="112" data-semantic-owns="33 34 35"><mjx-mrow data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="33" data-semantic-children="32" data-semantic-content="11" data-semantic-parent="36" data-semantic-owns="11 32"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="11" data-semantic-parent="33" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="32" data-semantic-children="12,13,29" data-semantic-content="30,31" data-semantic-parent="33" data-semantic-owns="12 30 13 31 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="32"><mjx-c class="mjx-c6C">l</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="32"><mjx-c class="mjx-c6E">n</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="29" data-semantic-children="22" data-semantic-content="14,28" data-semantic-parent="32" data-semantic-owns="14 22 28"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="14" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="22" data-semantic-children="15,27" data-semantic-content="17" data-semantic-parent="29" data-semantic-owns="15 17 27"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="22"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="17" data-semantic-parent="22" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="27" data-semantic-children="21,26" data-semantic-content="23" data-semantic-parent="22" data-semantic-owns="21 23 26"><mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="21" data-semantic-children="19,20" data-semantic-parent="27" data-semantic-owns="19 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c43">C</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c41">A</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="23" data-semantic-parent="27" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="24,25" data-semantic-parent="27" data-semantic-owns="24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c43">C</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c65">e</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="28" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="34" data-semantic-parent="36" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="36"><mjx-c class="mjx-c41">A</mjx-c></mjx-mi></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="112" data-semantic-operator="infixop,∗"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="3" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="57" data-semantic-children="55" data-semantic-content="40,56" data-semantic-parent="112" data-semantic-owns="40 55 56"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="40" data-semantic-parent="57" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="55" data-semantic-children="46,54" data-semantic-content="48" data-semantic-parent="57" data-semantic-owns="46 48 54"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="46" data-semantic-children="41,45" data-semantic-content="42" data-semantic-parent="55" data-semantic-owns="41 42 45"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="46"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="42" data-semantic-parent="46" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="45" data-semantic-children="43,44" data-semantic-parent="46" data-semantic-owns="43 44"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="45"><mjx-c class="mjx-c56">V</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="45"><mjx-c class="mjx-c44">D</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="48" data-semantic-parent="55" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="54" data-semantic-children="49,53" data-semantic-content="50" data-semantic-parent="55" data-semantic-owns="49 50 53"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="54"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="50" data-semantic-parent="54" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="53" data-semantic-children="51,52" data-semantic-parent="54" data-semantic-owns="51 52"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="53"><mjx-c class="mjx-c56">V</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="53"><mjx-c class="mjx-c41">A</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="56" data-semantic-parent="57" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="59" data-semantic-parent="112" data-semantic-operator="infixop,∗"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="112"><mjx-c class="mjx-c74">t</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="semicolon" data-semantic-id="62" data-semantic-parent="115" data-semantic-operator="punctuated"><mjx-c class="mjx-c3B">;</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="114" data-semantic-children="66,110" data-semantic-content="68" data-semantic-parent="115" data-semantic-owns="66 68 110"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="66" data-semantic-children="64,65" data-semantic-parent="114" data-semantic-owns="64 65"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="66"><mjx-c class="mjx-c43">C</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="65" data-semantic-parent="66"><mjx-c class="mjx-c65">e</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="68" data-semantic-parent="114" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="4" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="110" data-semantic-children="96,109" data-semantic-content="97" data-semantic-parent="114" data-semantic-owns="96 97 109"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="96" data-semantic-children="93" data-semantic-content="70,95" data-semantic-parent="110" data-semantic-owns="70 93 95"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="70" data-semantic-parent="96" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="93" data-semantic-children="92,94" data-semantic-content="81" data-semantic-parent="96" data-semantic-owns="92 81 94"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="92" data-semantic-children="73,79" data-semantic-content="75" data-semantic-parent="93" data-semantic-owns="73 75 79"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="73" data-semantic-children="71,72" data-semantic-parent="92" data-semantic-owns="71 72"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="73"><mjx-c class="mjx-c43">C</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="73"><mjx-c class="mjx-c44">D</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="75" data-semantic-parent="92" data-semantic-operator="infixop,∗"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="79" data-semantic-children="77,78" data-semantic-parent="92" data-semantic-owns="77 78"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="77" data-semantic-parent="79"><mjx-c class="mjx-c56">V</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="78" data-semantic-parent="79"><mjx-c class="mjx-c44">D</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="81" data-semantic-parent="93" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="94" data-semantic-children="85,91" data-semantic-content="87" data-semantic-parent="93" data-semantic-owns="85 87 91"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="85" data-semantic-children="83,84" data-semantic-parent="94" data-semantic-owns="83 84"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="83" data-semantic-parent="85"><mjx-c class="mjx-c43">C</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="84" data-semantic-parent="85"><mjx-c class="mjx-c41">A</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="87" data-semantic-parent="94" data-semantic-operator="infixop,∗"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="91" data-semantic-children="89,90" data-semantic-parent="94" data-semantic-owns="89 90"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="89" data-semantic-parent="91"><mjx-c class="mjx-c56">V</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="90" data-semantic-parent="91"><mjx-c class="mjx-c41">A</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="95" data-semantic-parent="96" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="97" data-semantic-parent="110" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="109" data-semantic-children="107" data-semantic-content="98,108" data-semantic-parent="110" data-semantic-owns="98 107 108"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="98" data-semantic-parent="109" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="107" data-semantic-children="101,106" data-semantic-content="103" data-semantic-parent="109" data-semantic-owns="101 103 106"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="101" data-semantic-children="99,100" data-semantic-parent="107" data-semantic-owns="99 100"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="99" data-semantic-parent="101"><mjx-c class="mjx-c56">V</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="100" data-semantic-parent="101"><mjx-c class="mjx-c44">D</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="103" data-semantic-parent="107" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="106" data-semantic-children="104,105" data-semantic-parent="107" data-semantic-owns="104 105"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="104" data-semantic-parent="106"><mjx-c class="mjx-c56">V</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="105" data-semantic-parent="106"><mjx-c class="mjx-c41">A</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="108" data-semantic-parent="109" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="semicolon" data-semantic-id="111" data-semantic-parent="115" data-semantic-operator="punctuated"><mjx-c class="mjx-c3B">;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div></div><div id="p1615" role="paragraph">t&nbsp;= incubation time (s),</div><div id="p1620" role="paragraph">C<sub>A</sub>&nbsp;= compound concentration in the acceptor well at t,</div><div id="p1625" role="paragraph">C<sub>D</sub>&nbsp;= compound concentration in the donor well at t,</div><div id="p1630" role="paragraph">C<sub>e</sub>&nbsp;= compound at equilibrium concentration,</div><div id="p1635" role="paragraph">V<sub>A</sub>&nbsp;= acceptor well volume,</div><div id="p1640" role="paragraph">V<sub>D</sub>&nbsp;= donor well volume,</div><div id="p1645" role="paragraph">A&nbsp;= filter area.</div></section><section id="sec-4-4-8"><h4>Caco-2 permeability assay</h4><div id="p1650" role="paragraph">The permeability of designed peptides was evaluated using an accelerated 6-day Caco-2 permeability assay as described previously (<span class="dropBlock reference-citations"><a id="body-ref-optElUPnApqkJ" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optElUPnApqkJ" aria-controls="optElUPnApqkJ">Sevin et al., 2013</a><div class="dropBlock__holder" data-db-target-of="optElUPnApqkJ"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Sevin, E. ∙ Dehouck, L. ∙ Fabulas-da Costa, A. ...</div><div><strong>Accelerated Caco-2 cell permeability model for drug discovery</strong></div><div><em>J. Pharmacol. Toxicol. Methods.</em> 2013; <strong>68</strong>:334-339</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.vascn.2013.07.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84883386431" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23916595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.vascn.2013.07.004&amp;pmid=23916595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, cells were seeded into the 12-well Transwell plates with 0.4&nbsp;μm polyester membrane inserts (Corning, Merck) to give 4.0&nbsp;×&nbsp;10<sup>5</sup> cells/well. The complete assay medium contained Dulbecco’s modified Eagle medium supplemented with 10% (v/v) fetal bovine serum, 1% (v/v) nonessential amino acids, and 0.4&nbsp;μg/mL puromycin. The medium in plates was renewed every second day. The inserts received 0.75&nbsp;mL of assay medium in the upper compartment (apical chamber) and 1.5&nbsp;mL in the bottom compartment (basolateral chamber) individually. After 6-day incubation or until the cells formed monolayers, the medium in the apical and basolateral chamber was replaced with peptide solutions at 50&nbsp;μM or Hanks' Balanced Salt Solution (HBSS), separately. The plates were incubated for 90&nbsp;min at 37°C on an orbital shaker (60&nbsp;rpm). The peptide concentration of solutions in the apical and basolateral compartments was subsequently quantified using LC-MS (Qstar Elite, AB Sciex Australia) to evaluate the transmembrane flux of peptides in the apical-to-basolateral direction. Membrane integrity of the Caco-2 cell monolayer was validated using trans-epithelial electrical resistance (TEER) measurements before and after the assays. Atenolol and quinidine were included as negative and positive controls in all assays.</div></section><section id="sec-4-4-9"><h4>Single-crystal x-ray diffraction of peptides</h4><div id="p1655" role="paragraph">Based on the availability of macrocycles, we tried to crystallize 139 designed macrocycles (44 6-8mers and 95 6-12mers) using slow evaporation in 50:50 water:ACN or 50:50 water:isopropanol mixtures or vapor diffusion in ethyl acetate and pentane. Crystal diffraction data were collected from a single crystal at the synchrotron (on APS 24ID-C) and at 100 K. Unit cell refinement, and data reduction were performed using XDS and CCP4 suites (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib38 bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Kabsch, 2010; Winn et al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Kabsch, W.</div><div><strong>XDS</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:125-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444909047337" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-76449099287" target="_blank" aria-label="Open Scopus in new tab">Scopus (12932)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444909047337&amp;pmid=20124692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div>Winn, M.D., Refinemen, M.-L.S., Pannu, N.J., Murshudov, G.N., and Others (2001). CCP4 v6. 3.0 Program References. Acta Crystallogr. <i>57</i>, 122–133.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Winn%2C+M.D.%2C+Refinemen%2C+M.-L.S.%2C+Pannu%2C+N.J.%2C+Murshudov%2C+G.N.%2C+and+Others+%282001%29.+CCP4+v6.+3.0+Program+References.+Acta+Crystallogr.+57%2C+122%E2%80%93133." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The structure was identified by direct methods and refined by full-matrix least-squares on F<sup>2</sup> with anisotropic displacement parameters for the non-H atoms using SHELXL-2016 (<span class="dropBlock reference-citations"><a id="body-ref-optgPLOKl5zI3-2" href="#" role="doc-biblioref" data-xml-rid="bib48 bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optgPLOKl5zI3-2" aria-controls="optgPLOKl5zI3-2">Sheldrick, 2015a,b</a><div class="dropBlock__holder" data-db-target-of="optgPLOKl5zI3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Sheldrick, G.M.</div><div><strong>SHELXT--Integrated space-group and crystal-structure determination</strong></div><div><em>Acta Crystallogr. A Found. Adv.</em> 2015; <strong>71</strong>:3-8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2053273314026370" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84920370689" target="_blank" aria-label="Open Scopus in new tab">Scopus (30086)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25537383/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2053273314026370&amp;pmid=25537383" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Sheldrick, G.M.</div><div><strong>Crystal strcuture refinement with SHELXL</strong></div><div><em>Acta Crystallogr. C Struct. Chem.</em> 2015; <strong>71</strong>:3-8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/s2053229614024218" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84937398809" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25567568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2Fs2053229614024218&amp;pmid=25567568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Structure analysis was aided by using Coot/Shelxle (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib32 bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Emsley and Cowtan, 2004; Hübschle, 2011</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Emsley, P. ∙ Cowtan, K.</div><div><strong>Coot: model-building tools for molecular graphics</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2004; <strong>60</strong>:2126-2132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444904019158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-13244281317" target="_blank" aria-label="Open Scopus in new tab">Scopus (25409)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15572765/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444904019158&amp;pmid=15572765" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Hübschle, C.B. ∙ Sheldrick, G.M. ∙ Dittrich, B.</div><div><strong>ShelXle: a Qt graphical user interface for SHELXL</strong></div><div><em>J. Appl. Crystallogr.</em> 2011; <strong>44</strong>:1281-1284</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0021889811043202" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-82055161785" target="_blank" aria-label="Open Scopus in new tab">Scopus (2900)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22477785/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0021889811043202&amp;pmid=22477785" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The hydrogen atoms on heavy atoms were calculated in ideal positions with isotropic displacement parameters set to 1.2 × U<sub>eq</sub> of the attached atoms. Since direct methods can return initial models with mirror handedness in 50% of the cases, we refined the structures with the handedness of the initial model from direct methods, and inverted using ROSETTA flip_chirality mover for alignment and RMSD calculation. Structural coordinates of the refined X-ray structures deposited to the Cambridge Crystallography Data Centre (CCDC) with the following identifiers: 2131411 (D7.6), 2131412 (D7.8), 2131449 (D8.1), 2131417 (D8.2), 2131450 (D8.3.p1), 2131423 (D8.5.p2), 2131424 (D8.6), 2131249 (D8.9), 2131251 (D8.10), 2131252 (D8.12), 2131463 (D8.13), 2131425 (D8.14), 2131426 (D8.15), 2131427 (D8.17), 2131428 (D8.19), 2131253 (D8.25), 2131291 (D8.26), 2131429 (D9.1), 2131430 (D9.8), 2131245 (D9.24), 2131431 (D9.29), 2131432 (D9.30), 2131433 (D10.1), 2131434 (D10.21), 2131435 (D10.22), 2131436 (D10.23), 2131438 (D10.31 IPA), 2131437 (D10.31 ETP), 2131439 (D10.65), 2131440 (D11.1), 2131451 (D11.3), 2131441 (D11.4), 2131442 (D11.25), 2131292 (D8.31), 2131443 (D8.21), 2131293 (D9.16). X-ray structure for D8.3.p2 deposited to Protein data bank with ID: 7UCP. PDB files for all the X-ray structures are also available in the <a id="crosref0935" href="#mmc1">Data S6</a> (Experimental Structures).</div></section><section id="sec-4-4-10"><h4>NMR sample preparation and assignments</h4><div id="p1660" role="paragraph">Peptide samples for 2D data collection were prepared by dissolving 5-10&nbsp;mg of peptide in d<sub>6</sub>-DMSO, CDCl<sub>3</sub>, or mixtures of H<sub>2</sub>O and d<sub>6</sub>-DMSO. For D8.21 an additional sample with 50% d<sub>6</sub>-DMSO and 50% D<sub>2</sub>O, rather than H<sub>2</sub>O, was prepared for additional data collection to remove complications from water peak suppression; peptide solubility was greatly decreased over the solubility in pure organic solvents. NMR data were collected on Bruker Avance II 600 MHz, Avance III 600 MHz, or Avance II 800 MHz spectrometers, equipped with TCI cryoprobes. In all cases data was collected at 293 K and referenced to internal TMS.</div><div id="p1665" role="paragraph">Resonance assignments were made using 2D experiments, including heteronuclear <sup>15</sup>N-<sup>1</sup>H HSQC, <sup>3</sup>C-<sup>1</sup>H HSQC, HMBC, HSQC-TOCSY (tau<sub>mix</sub>&nbsp;= 60&nbsp;ms), as well as homonuclear <sup>1</sup>H-<sup>1</sup>H COSY, TOCSY (80&nbsp;ms), ROESY (tau<sub>mix</sub>&nbsp;= 200&nbsp;ms, 7.8 kHz), and NOESY (tau<sub>mix</sub>&nbsp;= 250&nbsp;ms). Spectra were processed using TopSpin (Bruker) or NMRPipe and visualized in NMRFAM-SPARKY (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Delaglio et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Delaglio, F. ∙ Grzesiek, S. ∙ Vuister, G.W. ...</div><div><strong>NMRPipe: a multidimensional spectral processing system based on UNIX pipes</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 1995; <strong>6</strong>:277-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00197809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0029400480" target="_blank" aria-label="Open Scopus in new tab">Scopus (12158)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8520220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00197809&amp;pmid=8520220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref13-2" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-2" aria-controls="sref13-2">Lee et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref13-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Lee, W. ∙ Tonelli, M. ∙ Markley, J.L.</div><div><strong>NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:1325-1327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84927748573" target="_blank" aria-label="Open Scopus in new tab">Scopus (1330)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25505092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu830&amp;pmid=25505092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), allowing for the generation of chemical shift tables in BioMagResBank (BMRB) format from manual assignments.</div></section><section id="sec-4-4-11"><h4>NMR structure determination</h4><div id="p1670" role="paragraph">Cyclic peptide structures were calculated using CYANA to generate an ensemble of 20 lowest energy conformations (<span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Güntert et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Güntert, P. ∙ Mumenthaler, C. ∙ Wüthrich, K.</div><div><strong>Torsion angle dynamics for NMR structure calculation with the new program DYANA</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1997; <strong>273</strong>:283-298</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jmbi.1997.1284" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0031576336" target="_blank" aria-label="Open Scopus in new tab">Scopus (2598)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9367762/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjmbi.1997.1284&amp;pmid=9367762" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-optCpJzMpTriq" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optCpJzMpTriq" aria-controls="optCpJzMpTriq">Herrmann et al., 2002</a><div class="dropBlock__holder" data-db-target-of="optCpJzMpTriq"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Herrmann, T. ∙ Güntert, P. ∙ Wüthrich, K.</div><div><strong>Protein NMR structure determination with automated NOE assignment using the new software CANDID and the torsion angle dynamics algorithm DYANA</strong></div><div><em>J. Mol. Biol.</em> 2002; <strong>319</strong>:209-227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-2836(02)00241-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0036308102" target="_blank" aria-label="Open Scopus in new tab">Scopus (1356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12051947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-2836%2802%2900241-3&amp;pmid=12051947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Input files were chemical shifts in BMRB format, phi angle restraints, and either 2D NOESY or 2D ROESY peaks lists with peak intensities. NOESY peaks lists were used for samples in DMSO, and ROESY data for samples in CDCl<sub>3</sub> since NOESY data was weaker. ROESY data was used to identify peaks due to exchange between the conformations, which have the same sign as the diagonal, and these were not included in the peak lists. Phi angle restraints for HN peaks were determined from measured <sup>3</sup>J<sub>HNHa</sub> coupling constants from high resolution <sup>1</sup>H 1D spectra that were converted to phi angles using the Karplus relationship and parameters with error bars of&nbsp;+/- 30 degrees (<span class="dropBlock reference-citations"><a id="body-ref-optKJLDz0ySQe" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optKJLDz0ySQe" aria-controls="optKJLDz0ySQe">Vögeli et al., 2007</a><div class="dropBlock__holder" data-db-target-of="optKJLDz0ySQe"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Vögeli, B. ∙ Ying, J. ∙ Grishaev, A. ...</div><div><strong>Limits on variations in protein backbone dynamics from precise measurements of scalar couplings</strong></div><div><em>J. Am. Chem. Soc.</em> 2007; <strong>129</strong>:9377-9385</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja070324o" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-34547670259" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17608477/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja070324o&amp;pmid=17608477" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p1675" role="paragraph">Cyana library files were obtained for D-amino acids and N-methylated residues using CYLIB (<span class="dropBlock reference-citations"><a id="body-ref-optqmdefJ4GOW" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optqmdefJ4GOW" aria-controls="optqmdefJ4GOW">Yilmaz and Güntert, 2015</a><div class="dropBlock__holder" data-db-target-of="optqmdefJ4GOW"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Yilmaz, E.M. ∙ Güntert, P.</div><div><strong>NMR structure calculation for all small molecule ligands and non-standard residues from the PDB Chemical Component Dictionary</strong></div><div><em>J. Biomol. NMR.</em> 2015; <strong>63</strong>:21-37</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-015-9959-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84942373727" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26123317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-015-9959-y&amp;pmid=26123317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Dihedral omega angles were restrained to <i>cis</i> or <i>trans</i> based on nuclear Overhauser effect <b>(</b>NOE) assignments and cyclic peptides were linked using covalent restraints between the N- and C-termini to constrain the peptide bond geometries. For samples with two major conformations, structures were calculated by creating an extended sequence file with the sequences separated by a 12-13 residue non-steric (invisible) linker and performing the restraint-based modeling calculations of both conformers together. This was done to account for all NOE/ROE cross peaks in the 2D spectra corresponding to both conformations. This avoids problems that would be associated with trying to manually separate the peak list for each conformation, a process complicated by resonance overlap, especially in the methyl group region. Exchange peaks were identified by their opposite sign (from NOE cross peaks) in the ROESY spectrum and were removed from NOESY and ROESY peak lists prior to structure calculations. Conformations with symmetry as indicated by chemical shift degeneracy were restrained to C2 symmetry. Chemical shift assignments and NOE completeness were evaluated by PDBStat v5.21.5 (<span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">Tejero et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Tejero, R. ∙ Snyder, D. ∙ Mao, B. ...</div><div><strong>PDBStat: a universal restraint converter and restraint analysis software package for protein NMR</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2013; <strong>56</strong>:337-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-013-9753-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84883463996" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23897031/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-013-9753-7&amp;pmid=23897031" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). NOESY/ROESY peaks were evaluated by reading Cyana assigned peak lists back into Sparky and manually assessed. Structures were validated with PDBStat v5.21.5 (<span class="dropBlock reference-citations"><a id="body-ref-sref23-3" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-3" aria-controls="sref23-3">Tejero et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref23-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Tejero, R. ∙ Snyder, D. ∙ Mao, B. ...</div><div><strong>PDBStat: a universal restraint converter and restraint analysis software package for protein NMR</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2013; <strong>56</strong>:337-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-013-9753-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84883463996" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23897031/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-013-9753-7&amp;pmid=23897031" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and NMR structure validation statistics are given in <a id="crosref0965" href="#mmc1">Table&nbsp;S4</a>. Chemical shifts assignments for <sup>1</sup>H, <sup>13</sup>C, and <sup>15</sup>N resonances are given in <a id="crosref0970" href="#mmc1">Table&nbsp;S3</a>. NMR structures for the chameleonic peptides are available in Protein Data Bank with the following PDB ID and BMRB IDs: 7UBC and 30997 (D9.16 trans-cis d<sub>6</sub>-DMSO), 7UBD and 30998 (D8.31 cis-cis d<sub>6</sub>-DMSO), 8CTO and 31021 (D8.31 cis-trans d<sub>6</sub>-DMSO), 7UBE and 30999 (D8.21 trans-trans d<sub>6</sub>-DMSO), 7UBF and 31000 (D8.21 trans-trans 50:50 d<sub>6</sub>-DMSO / H<sub>2</sub>O), 8CUN and 31022 (D8.21 cis-cis 50:50 d<sub>6</sub>-DMSO / H<sub>2</sub>O), 7UBG and 31001 (D9.16 trans-trans CDCl<sub>3</sub>), 7UZL and 31019 (D9.16 trans-cis CDCl<sub>3</sub>), 7UBH and 31002 (D8.31 cis-cis CDCl<sub>3</sub>), 7UBI and 31003 (D8.21 trans-trans CDCl<sub>3</sub>), 8CWA and 31023 (D8.21 trans-cis CDCl<sub>3</sub>). PDB files for all the NMR structures are also available in the <a id="crosref0975" href="#mmc1">Data S6</a> (Experimental Structures).</div></section><section id="sec-4-4-12"><h4>NMR amide temperature coefficient studies</h4><div id="p1680" role="paragraph">Temperature dependence of amide <sup>1</sup>H chemical shifts were determined by recording 1D NMR variable temperature experiments on a Bruker Avance III 600 MHz spectrometer. Spectra were measured on d<sub>6</sub>-DMSO samples from 293 to 323 K and for CDCl<sub>3</sub> samples from 288 to 323 K, recorded in 5 K increments and referenced to TMS at 0 ppm. Previous assignments were used and HN amide proton temperature coefficients Δδ(<sup>1</sup>H)/ΔT (ppb/K) were determined from the slope of the linear fit of chemical shift vs. temperature.</div></section><section id="sec-4-4-13"><h4><i>In&nbsp;vivo</i> oral bioavailability assays</h4><div id="p1685" role="paragraph">We selected four designed macrocycles with diverse structures and <i>in&nbsp;vitro</i> rates of permeability for <i>in&nbsp;vivo</i> oral bioavailability studies in the rodent models. Due to the intensive resource requirements for such studies, these studies were conducted by three different research groups. D8.3.p1 and D10.1 were studied in female Balb/C mice by van Voorhis group at University of Washington. D11.2 was evaluated in Sprague Dawley (SD) rats by Robert Griffin’s group at Takeda Pharmaceuticals. D11.3 was studied in the male swiss albino mice by the contract research organization, GVK Biosciences Pvt. Ltd., Hyderabad, India.</div></section><section id="sec-4-4-14"><h4>Evaluating pharmacokinetic properties of D8.3.p1 and D10.1</h4><div id="p1690" role="paragraph">As a preliminary safety screen, peptides were first tested for cytotoxicity <i>in&nbsp;vitro</i> against two mammalian cell lines. CRL-8155 human lymphocyte and HepG2 human hepatocyte cells are seeded in 96-well plates and incubated for 48 hours in the presence of test peptide (serial-2 dilutions, from 80&nbsp;μM to 1.25&nbsp;μM, in triplicate). At the end of the incubation period, cells were visually assessed before alamarBlue™, a resazurin-based cell viability reagent which measures metabolic activity is added to the plate, and fluorescence was measured on a microplate reader. Fluorescence signals resulting from cell viability changes were compared with control wells to calculate 50% cytotoxic concentration (CC<sub>50</sub>) values.</div><div id="p1695" role="paragraph">Pharmacokinetic properties were evaluated in Balb/C female mice (10-12&nbsp;weeks old, avg. weight 20g, in triplicates) after dosing the peptides through oral gavage (PO), subcutaneous (SQ), and intravenous (IV) routes. Blood samples were collected at multiple time points and plasma was separated from whole blood using centrifugation and stored at -80°C. Drug was extracted from plasma using 80% Acetonitrile (ACN) in water with 0.1% formic acid (FA) and an internal standard. Samples were mixed, centrifuged, and supernatant harvested for LC-MS analysis. For RP-HPLC analysis, peptides were evaluated using Agilent ZORBAX Eclipse Plus C18&nbsp;1.8&nbsp;μm, 2.1&nbsp;mm x 50&nbsp;mm and Waters Xevo TQ-S micro Triple Quadrupole/ACQUITY UPLC H-Class. A two-component system composed of mobile phase A (0.1% FA in water) and mobile phase B (0.1% FA in 100% ACN) was used at a flow rate of 0.25&nbsp;mL/min. Peptides were bound to an Agilent ZORBAX C18 column at 40°C for 1&nbsp;min with 95% mobile phase A, and then were eluted with a linear gradient from 5 to 95% mobile phase B for 4&nbsp;min. MS was operated in multiple reaction monitoring (MRM) mode via the positive electrospray ionization interface using two transitions: <i>m/z</i> 886.6/128 and <i>m/z</i> 886.6/156 for D8.3.p1, and <i>m/z</i> 1139.92/99.97 for D10.1. Pharmacokinetic parameters were modeled using the Phoenix WinNonlin software package.</div></section><section id="sec-4-4-15"><h4>Experimental Parameters for D8.3.p1</h4><div id="p1700" role="paragraph">Dosing vehicle: 5% DMSO/5% Tween 80/90% PBS (PO, SQ); 5% DMSO/5% Tween 80/20% PEG 400/70% D5W (IV)</div><div id="p1705" role="paragraph">Dose volume: 200&nbsp;mL (PO, SQ); 50&nbsp;mL (IV)</div><div id="p1710" role="paragraph">Dose: 5&nbsp;mg/kg, 0.5&nbsp;mg/mL (PO, SQ); 1&nbsp;mg/kg, 0.4&nbsp;mg/mL (IV)</div><div id="p1715" role="paragraph">Sampling times: 15, 30, 60, 120, 240, 360, 1440&nbsp;min (PO, SQ); 5, 15, 30, 60, 120, 240, 360&nbsp;min (IV)</div></section><section id="sec-4-4-16"><h4>Experimental Parameters for D10.1</h4><div id="p1720" role="paragraph">Dosing vehicle: 5% DMSO/5% Tween 80/90% PBS (PO, SQ); 5% DMSO/5% Tween 80/20% PEG 400/70% D5W (IV)</div><div id="p1725" role="paragraph">Dose volume: 200&nbsp;mL (PO, SQ); 50&nbsp;mL (IV)</div><div id="p1730" role="paragraph">Dose: 12.2&nbsp;mg/kg, 1.22&nbsp;mg/mL (PO, SQ); 2.45&nbsp;mg/kg, 0.98&nbsp;mg/mL (IV)</div><div id="p1735" role="paragraph">Sampling times: 15, 30, 60, 120, 240, 360, 1440&nbsp;min (PO, SQ); 5, 15, 30, 60, 120, 240, 360&nbsp;min (IV)</div></section><section id="sec-4-4-17"><h4>Evaluating pharmacokinetic properties of D11.2</h4><div id="p1740" role="paragraph">We evaluated the pharmacokinetic properties for D11.2 in male SD rats (n=3). D11.2 was dissolved in 2% DMSO/0.1% Tween80 and dosed by IV, PO, and SQ routes. Plasma samples were collected at multiple time points up to 24 hours. The quantification limit was established to be 2&nbsp;ng/mL for the assay conditions tested. Pharmacokinetic parameters were modeled using the Phoenix WinNonlin software package.</div></section><section id="sec-4-4-18"><h4>Experimental Parameters for D11.2</h4><div id="p1745" role="paragraph">Dosing vehicle: DMSO (2%)&nbsp;+ 0.1% Tween80(IV, PO, SQ)</div><div id="p1750" role="paragraph">Dose: 1&nbsp;mg/kg (PO, SQ, IV)<div id="ulist0020" role="list"><div id="u0055" role="listitem"><div class="content"><div id="p1755" role="paragraph">Sampling times: 0.083, 0.25, 0.5, 1, 2, 4, 6, 8, 24&nbsp;h (PO, SQ, IV)</div></div></div></div></div></section><section id="sec-4-4-19"><h4>Evaluating pharmacokinetic properties of D11.3</h4><div id="p1760" role="paragraph">Pharmacokinetic properties of D11.3 were evaluated by the contract research organization, GVK Bio in Swiss Albino male mice after dosing the peptides through oral gavage (PO), subcutaneous (SQ), and intravenous (IV) routes. Plasma samples were collected at multiple time points up to 24 hours. Animals were dosed with 27 gauge needle through tail vein for IV, gastric gavage needle for PO, and 27 gauge needle on dorsal side for SQ. Plasma concentrations were determined using LC-MS/MS based quantification. Pharmacokinetic parameters were determined using non-compartmental analysis with Phoenix software version 8.1. All animals were healthy throughout the duration of the study.</div></section><section id="sec-4-4-20"><h4>Experimental Parameters for D11.3</h4><div id="p1765" role="paragraph">Dosing vehicle: DMSO (10%)&nbsp;+ 10% solutol in PBS (90%) (IV, PO, SQ)</div><div id="p1770" role="paragraph">Dose volume: 200&nbsp;mL (PO, SQ); 50&nbsp;mL (IV)</div><div id="p1775" role="paragraph">Dose: 10&nbsp;mg/kg (PO) 1&nbsp;mg/kg (IV), 5mg/kg (SQ)</div><div id="p1780" role="paragraph">Sampling times: 0.08, 0.25, 0.5, 1, 3, 5, 8, 24&nbsp;h (IV, PO, SQ)</div></section><section id="sec-4-4-21"><h4>Images and Figures</h4><div id="p1785" role="paragraph">The hydrogen bonding and structure prediction data was plotted using python scripts using pandas (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">McKinney, 2010</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div>McKinney, W. (2010). Data structures for statistical computing in python. In Proceedings of the 9th Python in Science Conference, (Austin, TX), pp. 51–56.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=McKinney%2C+W.+%282010%29.+Data+structures+for+statistical+computing+in+python.+In+Proceedings+of+the+9th+Python+in+Science+Conference%2C+%28Austin%2C+TX%29%2C+pp.+51%E2%80%9356." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), matplotlib (<span class="dropBlock reference-citations"><a id="body-ref-optQS1ISndA26" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optQS1ISndA26" aria-controls="optQS1ISndA26">Hunter, 2007</a><div class="dropBlock__holder" data-db-target-of="optQS1ISndA26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Hunter, J.D.</div><div><strong>Matplotlib: A 2D Graphics</strong></div><div><em>Environment.</em> 2007; <strong>9</strong>:90-95</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.D.HunterMatplotlib%3A+A+2D+GraphicsEnvironment920079095" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and seaborn (<span class="dropBlock reference-citations"><a id="body-ref-optfKm2A3IJUh" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optfKm2A3IJUh" aria-controls="optfKm2A3IJUh">Waskom, 2021</a><div class="dropBlock__holder" data-db-target-of="optfKm2A3IJUh"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Waskom, M.L.</div><div><strong>Seaborn: statistical data visualization</strong></div><div><em>J. Open Source Softw.</em> 2021; <strong>6</strong>:3021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.21105/joss.03021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.21105%2Fjoss.03021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) libraries. Three-dimensional structures were rendered in PyMol. Figures&nbsp;were assembled using Biorender.</div></section></section><section id="sec-4-5"><h3>Quantification and Statistical Analysis</h3><div id="p1790" role="paragraph">Sample sizes for PAMPA, Caco-2, and <i>in&nbsp;vivo</i> bioavailability studies are noted in the figure legends. Bar graphs for the PAMPA and Caco-2 assays were plotted in GraphPad Prism using the mean values and the standard deviation of replicate measurements as bar height and error bars, respectively. Pharmacokinetic parameters for <i>in&nbsp;vivo</i> studies were modeled using the Phoenix WinNonlin software package.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p1795" role="paragraph">This work was supported with funds provided by the TED Audacious Project (D.B., G.B., J.O., G.G.A., and X.L.); a gift from Gates Ventures (G.B., L.S., and D.B.); Eric and Wendy Schmidt by recommendation of the Schmidt Futures (D.B. and S.R.); the Nordstrom Barrier Institute for Protein Design Directors Fund (M.J.B.); Wu Tsai Translational Fund (G.B.); Bill and Melinda Gates Foundation grant INV-010680 (D.B.); The Open Philanthropy Project Improving Protein Design Fund (D.B., M.D.P., and A.K.B.); Takeda (R.G. and L.D.); Howard Hughes Medical Institute (D.B.); Washington State Supplement Funding to support the Institute for Protein Design (L.S.); Department of Defense CDMRP Prostate Cancer Research Program Postdoctoral Training Grant (G.B.); Simons Foundation (V.K.M.); Defense Threat Reduction Agency grant HDTRA1-19-1-0003 (D.B. and A.L.); National Institutes of Health Grants R01AG063845 (S.R. and D.B.), R01GM120574 and R35-GM141818 (G.T.M.); NIH Ruth Kirschstein F32 award F32GM120791-02 (P.H.); and Washington Research Foundation (P.H. and T.W.C.). We thank the University of Washington’s Hyak supercomputing team, Innovative and Novel Computational Impact on Theory and Experiment program, and the volunteers of Rosetta@Home project for providing compute resources. This research used resources of the Argonne Leadership Computing Facility, which is a DOE Office of Science User Facility supported under Contract DE-AC02-06CH11357. Crystallographic work is based upon research conducted at the Northeastern Collaborative Access Team beamlines, which are funded by the National Institute of General Medical Sciences from the National Institutes of Health (P30 GM124165). This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357. NMR studies were supported by the RPI Center for Biotechnology and Interdisciplinary Studies core facilities. We thank Scott Lokey, Hahnbeom Park, Adam Moyer, Patrick Salveson, Brian Weitzner, and members of Baker and Bhardwaj labs for discussions; Douglas Renfrew for help with the N-methyl rotamer library code; Adam Brown, Nicole Bebrin, Matt Hulverson, and Lynn Barrett for supporting <i>in&nbsp;vivo</i> studies; Luki Goldschmidt and Patrick Vecchiato for computing support; Alex Kang and Hannah Ngyuen for help with X-ray crystallography; Scott McCallum and GVT Swapna for advice with NMR studies; and Peter Güntert for advice on NMR structure calculations of cyclic peptides containing non-natural amino acids.</div><section id="sec-5"><h3>Author contributions</h3><div id="p1800" role="paragraph">G.B., J.O., and D.B. designed the study. G.B., J.O., G.G.A., J.P., and A.L. designed the 6–12-amino-acid peptide macrocycles. V.K.M., P.H., and T.W.C. developed computational tools to support the design of macrocycles with N-methylated amino acids. J.O., G.G., and S.R. evaluated the peptide permeability in parallel artificial membrane permeability assays. Y.-H.H. determined the Caco-2 permeability. Y.-H.H. and D.C. analyzed the Caco-2 permeability data. S.R., M.D.P., X.L., and G.B. synthesized and purified the peptides. A.K.B., G.B., S.R., M.J.B., and C.G. determined the X-ray structures. A.K.B., G.B., S.R., C.G., M.J.B., and J.R. analyzed the X-ray data. R.C. and L.D. collected the <i>in&nbsp;vivo</i> bioavailability data. R.C., W.C.v.V., R.G., and L.S. analyzed the pharmacokinetics data from rodent models. T.A.R. collected NMR data and solved the NMR structures of peptides. R.T. developed tools to analyze NMR data for peptides with non-natural amino acids. T.A.R. and G.T.M. designed NMR studies, analyzed NMR data, and wrote the sections describing NMR studies. G.B., J.O., T.A.R., G.T.M., and D.B. wrote the manuscript with help from all the authors.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p1805" role="paragraph">V.K.M. is a cofounder and shareholder of Menten AI, a biotechnology company. G.T.M. is a cofounder of Nexomics Biosciences, Inc., a structural biology contract research organization. A provisional patent covering the membrane-permeable peptides described in this paper has been filed by the University of Washington, Seattle. G.B., L.S., and D.B. are cofounders and shareholders of an early-stage biotechnology company that has licensed the provisional patent.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (10)</h2><div id="p1815" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/731a58d0-211b-4504-97b4-5f8f3227fe51/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (79.33 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table&nbsp;S1. RMSD between the designed models and X-ray crystal structures of the 7- and 8-amino-acid macrocycles, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/dddca6c1-f5f5-4938-adb4-075f77b44c28/mmc2.pdf" download="mmc2.pdf" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">PDF (69.75 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table&nbsp;S2. RMSD between the designed models and X-ray crystal structures of the 9–11 residue macrocycles, related to Figure&nbsp;3</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/1603bcf8-4901-4e41-8267-d96ffec0ac55/mmc3.pdf" download="mmc3.pdf" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">PDF (91.85 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table&nbsp;S3. NMR chemical shift assignments for peptides D9.16, D8.21, and D8.31, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/9b2096ba-a863-4348-a465-f1ffb0541564/mmc4.pdf" download="mmc4.pdf" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">PDF (103.21 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table&nbsp;S4. NMR validation tables for designs D9.16, D8.31, and D8.21. RMSD calculated for 20 lowest energy structures out of 100, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/e93aa71e-a5f6-4e0f-8faa-532206cb70ef/mmc5.pdf" download="mmc5.pdf" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">PDF (15.77 MB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Data S1. Structure prediction plots for all designs, related to Figures&nbsp;1 and 3</div></div></div><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/f6f69fb2-8822-4897-b19d-f0474e0ee2ce/mmc6.zip" download="mmc6.zip" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Archive (877.83 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Data S2. Amino acid sequences and computational models, related to Figures&nbsp;1 and 3</div></div></div><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/04fbb571-6234-4063-8594-7751a10b17dc/mmc7.zip" download="mmc7.zip" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Archive (159.92 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Data S3. PAMPA and Caco-2 data, related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/bfd0b5c0-952b-4e37-a8f6-cbfcce3135db/mmc8.pdf" download="mmc8.pdf" aria-labelledby="mmc8-heading mmc8-link"><span id="mmc8-link">PDF (131.79 KB)</span></a></div><div class="core-description"><div id="mmc8-heading" class="heading">Data S4. Oral bioavailability data, related to Figure&nbsp;5</div></div></div><div id="mmc9" class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/8290313b-df6d-4ec2-96fe-1a38cc2f2d30/mmc9.zip" download="mmc9.zip" aria-labelledby="mmc9-heading mmc9-link"><span id="mmc9-link">Archive (81.73 KB)</span></a></div><div class="core-description"><div id="mmc9-heading" class="heading">Data S5. Example design scripts, related to STAR Methods</div></div></div><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.07.019/attachment/3bbd8235-dd58-42a1-b501-2c8765bbf5cb/mmc10.zip" download="mmc10.zip" aria-labelledby="mmc10-heading mmc10-link"><span id="mmc10-link">Archive (727.07 KB)</span></a></div><div class="core-description"><div id="mmc10-heading" class="heading">Data S6. PDB files for experimental structures, related to STAR Methods</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">X-ray crystal structures described in the manuscript have been deposited to the Cambridge Crystallography Data Centre (CCDC) and are publicly available as of the date of publication. CCDC IDs are listed in the <a id="crosref0250" href="#sec-4-1">key resources table</a>.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">NMR structures and associated data for the chameleonic peptides have been deposited to Protein Data Bank (PDB) and Biological Magnetic Resonance Bank (BMRB) and are publicly available as of the date of publication. PDB and BMRM IDs are listed in the <a id="crosref0255" href="#sec-4-1">key resources table</a>.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">All the scripts used for the design and validations of peptides were implemented in Rosetta Macromolecular Modeling Suite and PyRosetta. The Rosetta software suite and PyRosetta can be downloaded from <a id="intref0010" href="https://www.rosettacommons.org/" target="_blank">https://www.rosettacommons.org/</a> and <a id="intref0015" href="https://www.pyrosetta.org/" target="_blank">https://www.pyrosetta.org/</a>. The scripts used to design macrocycle and example command-line flags are included in the <a id="crosref0260" href="#supplementary-material">supplemental information</a> files.</div></div></div><div id="u0045" role="listitem"><div class="label">•</div><div class="content"><div id="p0050" role="paragraph">Example script files, macrocycle sequences, design models, and permeability values are available in the <a id="crosref0265" href="#supplementary-material">supplemental information</a> files.</div></div></div><div id="u0050" role="listitem"><div class="label">•</div><div class="content"><div id="p0055" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the <a id="crosref0270" href="#sec-4-2-1">lead contact</a> upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">1.</a></div>Adolf-Bryfogle, J., Labonte, J.W., Kraft, J.C., Shapovalov, M., Raemisch, S., Lütteke, T., DiMaio, F., Bahl, C.D., Pallesen, J., King, N.P., et al. (2021). Growing Glycans in Rosetta: Accurate de novo glycan modeling, density fitting, and rational sequon design.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Adolf-Bryfogle%2C+J.%2C+Labonte%2C+J.W.%2C+Kraft%2C+J.C.%2C+Shapovalov%2C+M.%2C+Raemisch%2C+S.%2C+L%C3%BCtteke%2C+T.%2C+DiMaio%2C+F.%2C+Bahl%2C+C.D.%2C+Pallesen%2C+J.%2C+King%2C+N.P.%2C+et+al.+%282021%29.+Growing+Glycans+in+Rosetta%3A+Accurate+de+novo+glycan+modeling%2C+density+fitting%2C+and+rational+sequon+design." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="opty67Lg6LigC" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opty67Lg6LigC-1" title="View in article">2.</a></div><div>Alford, R.F. ∙ Leaver-Fay, A. ∙ Jeliazkov, J.R. ...</div><div><strong>The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design</strong></div><div><em>J. Chem. Theory Comput.</em> 2017; <strong>13</strong>:3031-3048</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jctc.7b00125" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85020732573" target="_blank" aria-label="Open Scopus in new tab">Scopus (844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28430426/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jctc.7b00125&amp;pmid=28430426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="optGbjjMMdkkU" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">3.</a></div><div>Baxter, N.J. ∙ Williamson, M.P.</div><div><strong>Temperature dependence of 1H chemical shifts in proteins</strong></div><div><em>J. Biomol. NMR.</em> 1997; <strong>9</strong>:359-369</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1018334207887" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0031160103" target="_blank" aria-label="Open Scopus in new tab">Scopus (456)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9255942/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1018334207887&amp;pmid=9255942" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">4.</a></div><div>Bhardwaj, G. ∙ Mulligan, V.K. ∙ Bahl, C.D. ...</div><div><strong>Accurate de novo design of hyperstable constrained peptides</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:329-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84988352784" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19791&amp;pmid=27626386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">5.</a></div><div>Bockus, A.T. ∙ Lexa, K.W. ∙ Pye, C.R. ...</div><div><strong>Probing the physicochemical boundaries of cell permeability and oral bioavailability in lipophilic macrocycles inspired by natural products</strong></div><div><em>J.&nbsp;Med. Chem.</em> 2015; <strong>58</strong>:4581-4589</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jmedchem.5b00128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84935898517" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25950816/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jmedchem.5b00128&amp;pmid=25950816" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">6.</a></div><div>Bockus, A.T. ∙ Lokey, R.S.</div><div><strong>Bioactive and membrane-permeable cyclic peptide natural products</strong></div><div><em>Practical medicinal chemistry with Macrocycles.</em>John Wiley &amp; Sons, Inc, Hoboken, NJ, USA, 2017 101-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/9781119092599.ch5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F9781119092599.ch5" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="optnjAB8SH7Mv" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">7.</a></div><div>Cierpicki, T. ∙ Otlewski, J.</div><div><strong>Amide proton temperature coefficients as hydrogen bond indicators in proteins</strong></div><div><em>J. Biomol. NMR.</em> 2001; <strong>21</strong>:249-261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1012911329730" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0035215289" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11775741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1012911329730&amp;pmid=11775741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="optRN6vLFxNLI" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opt9b2tozWJtl" title="View in article">8.</a></div><div>Coutsias, E.A. ∙ Seok, C. ∙ Jacobson, M.P. ...</div><div><strong>A kinematic view of loop closure</strong></div><div><em>J. Comput. Chem.</em> 2004; <strong>25</strong>:510-528</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.10416" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-1442281405" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14735570/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.10416&amp;pmid=14735570" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">9.</a></div><div>Delaglio, F. ∙ Grzesiek, S. ∙ Vuister, G.W. ...</div><div><strong>NMRPipe: a multidimensional spectral processing system based on UNIX pipes</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 1995; <strong>6</strong>:277-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00197809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0029400480" target="_blank" aria-label="Open Scopus in new tab">Scopus (12158)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8520220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00197809&amp;pmid=8520220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">10.</a></div><div>Di, L. ∙ Kerns, E.H. ∙ Fan, K. ...</div><div><strong>High throughput artificial membrane permeability assay for blood-brain barrier</strong></div><div><em>Eur. J. Med. Chem.</em> 2003; <strong>38</strong>:223-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0223-5234(03)00012-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84962359573" target="_blank" aria-label="Open Scopus in new tab">Scopus (978)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12667689/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0223-5234%2803%2900012-6&amp;pmid=12667689" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="optt7it7pQuea" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-2" title="View in article">11.</a></div><div>Emsley, P. ∙ Cowtan, K.</div><div><strong>Coot: model-building tools for molecular graphics</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2004; <strong>60</strong>:2126-2132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444904019158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-13244281317" target="_blank" aria-label="Open Scopus in new tab">Scopus (25409)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15572765/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444904019158&amp;pmid=15572765" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="optNogaYkWwmi" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optNogaYkWwmi" title="View in article">12.</a></div><div>Fleishman, S.J. ∙ Leaver-Fay, A. ∙ Corn, J.E. ...</div><div><strong>RosettaScripts: a scripting language interface to the Rosetta macromolecular modeling suite</strong></div><div><em>PLoS One.</em> 2011; <strong>6</strong>, e20161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0020161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-79959615159" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21731610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0020161&amp;pmid=21731610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">13.</a></div><div>Fouché, M. ∙ Schäfer, M. ∙ Berghausen, J. ...</div><div><strong>Design and development of a cyclic decapeptide scaffold with suitable properties for bioavailability and oral exposure</strong></div><div><em>ChemMedChem.</em> 2016; <strong>11</strong>:1048-1059</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cmdc.201600082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84966389321" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27154275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcmdc.201600082&amp;pmid=27154275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">14.</a></div><div>Ganguly, H.K. ∙ Majumder, B. ∙ Chattopadhyay, S. ...</div><div><strong>Direct evidence for CH···π interaction mediated stabilization of Pro-cisPro bond in peptides with Pro-Pro-aromatic motifs</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2012; <strong>134</strong>:4661-4669</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja209334v" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84858233524" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22335630/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja209334v&amp;pmid=22335630" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">15.</a></div><div>Grathwohl, C. ∙ Wüthrich, K.</div><div><strong>NMR studies of the rates of proline cis–trans isomerization in oligopeptides</strong></div><div><em>Biopolymers.</em> 1981; <strong>20</strong>:2623-2633</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bip.1981.360201209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84985715908" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbip.1981.360201209" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">16.</a></div><div>Güntert, P. ∙ Mumenthaler, C. ∙ Wüthrich, K.</div><div><strong>Torsion angle dynamics for NMR structure calculation with the new program DYANA</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1997; <strong>273</strong>:283-298</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jmbi.1997.1284" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0031576336" target="_blank" aria-label="Open Scopus in new tab">Scopus (2598)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9367762/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjmbi.1997.1284&amp;pmid=9367762" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="optCpJzMpTriq" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optCpJzMpTriq" title="View in article">17.</a></div><div>Herrmann, T. ∙ Güntert, P. ∙ Wüthrich, K.</div><div><strong>Protein NMR structure determination with automated NOE assignment using the new software CANDID and the torsion angle dynamics algorithm DYANA</strong></div><div><em>J. Mol. Biol.</em> 2002; <strong>319</strong>:209-227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-2836(02)00241-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0036308102" target="_blank" aria-label="Open Scopus in new tab">Scopus (1356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12051947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-2836%2802%2900241-3&amp;pmid=12051947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">18.</a></div><div>Hewitt, W.M. ∙ Leung, S.S.F. ∙ Pye, C.R. ...</div><div><strong>Cell-permeable cyclic peptides from synthetic libraries inspired by natural products</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2015; <strong>137</strong>:715-721</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja508766b" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84921484823" target="_blank" aria-label="Open Scopus in new tab">Scopus (176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja508766b&amp;pmid=25517352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">19.</a></div><div>Hill, T.A. ∙ Lohman, R.-J. ∙ Hoang, H.N. ...</div><div><strong>Cyclic penta- and Hexaleucine peptides without N-methylation are orally absorbed</strong></div><div><em>ACS Med. Chem. Lett.</em> 2014; <strong>5</strong>:1148-1151</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ml5002823" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84907923894" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25313329/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fml5002823&amp;pmid=25313329" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">20.</a></div><div>Hosseinzadeh, P. ∙ Bhardwaj, G. ∙ Mulligan, V.K. ...</div><div><strong>Comprehensive computational design of ordered peptide macrocycles</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1461-1466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap7577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85040187593" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29242347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap7577&amp;pmid=29242347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="optsrMTRQR98l" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optsrMTRQR98l-1" title="View in article">21.</a></div><div>Hosseinzadeh, P. ∙ Watson, P.R. ∙ Craven, T.W. ...</div><div><strong>Anchor extension: a structure-guided approach to design cyclic peptides targeting enzyme active sites</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:3384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23609-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85107586900" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34099674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23609-8&amp;pmid=34099674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">22.</a></div><div>Hübschle, C.B. ∙ Sheldrick, G.M. ∙ Dittrich, B.</div><div><strong>ShelXle: a Qt graphical user interface for SHELXL</strong></div><div><em>J. Appl. Crystallogr.</em> 2011; <strong>44</strong>:1281-1284</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0021889811043202" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-82055161785" target="_blank" aria-label="Open Scopus in new tab">Scopus (2900)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22477785/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0021889811043202&amp;pmid=22477785" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="optQS1ISndA26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optQS1ISndA26" title="View in article">23.</a></div><div>Hunter, J.D.</div><div><strong>Matplotlib: A 2D Graphics</strong></div><div><em>Environment.</em> 2007; <strong>9</strong>:90-95</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.D.HunterMatplotlib%3A+A+2D+GraphicsEnvironment920079095" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="optg8m7ZUq91m" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optg8m7ZUq91m-1" title="View in article">24.</a></div><div>Kabsch, W.</div><div><strong>XDS</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:125-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444909047337" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-76449099287" target="_blank" aria-label="Open Scopus in new tab">Scopus (12932)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444909047337&amp;pmid=20124692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="optPHWTCfBq6j" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optPHWTCfBq6j" title="View in article">25.</a></div><div>Kansy, M. ∙ Senner, F. ∙ Gubernator, K.</div><div><strong>Physicochemical high throughput screening: parallel artificial membrane permeation assay in the description of passive absorption processes</strong></div><div><em>J. Med. Chem.</em> 1998; <strong>41</strong>:1007-1010</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jm970530e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-0032568397" target="_blank" aria-label="Open Scopus in new tab">Scopus (1147)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9544199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjm970530e&amp;pmid=9544199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13-1" title="View in article">26.</a></div><div>Lee, W. ∙ Tonelli, M. ∙ Markley, J.L.</div><div><strong>NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:1325-1327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84927748573" target="_blank" aria-label="Open Scopus in new tab">Scopus (1330)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25505092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu830&amp;pmid=25505092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="opt9b2tozWJtl" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opt9b2tozWJtl" title="View in article">27.</a></div><div>Mandell, D.J. ∙ Coutsias, E.A. ∙ Kortemme, T.</div><div><strong>Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling</strong></div><div><em>Nat. Methods.</em> 2009; <strong>6</strong>:551-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth0809-551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-68349104348" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19644455/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth0809-551&amp;pmid=19644455" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">28.</a></div><div>Marelli, U.K. ∙ Ovadia, O. ∙ Frank, A.O. ...</div><div><strong>cis-Peptide Bonds: A Key for intestinal Permeability of Peptides?</strong></div><div><em>Chemistry.</em> 2015; <strong>21</strong>:15148-15152</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/chem.201501600" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84944711138" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26337831/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fchem.201501600&amp;pmid=26337831" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">29.</a></div>McKinney, W. (2010). Data structures for statistical computing in python. In Proceedings of the 9th Python in Science Conference, (Austin, TX), pp. 51–56.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=McKinney%2C+W.+%282010%29.+Data+structures+for+statistical+computing+in+python.+In+Proceedings+of+the+9th+Python+in+Science+Conference%2C+%28Austin%2C+TX%29%2C+pp.+51%E2%80%9356." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">30.</a></div><div>Mulligan, V.K.</div><div><strong>The emerging role of computational design in peptide macrocycle drug discovery. Expert Opin</strong></div><div><em>Drug Discov.</em> 2020; <strong>15</strong>:833-852</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.K.MulliganThe+emerging+role+of+computational+design+in+peptide+macrocycle+drug+discovery.+Expert+OpinDrug+Discov152020833852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="optM1vu63fEjC" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optM1vu63fEjC-1" title="View in article">31.</a></div><div>Mulligan, V.K. ∙ Workman, S. ∙ Sun, T. ...</div><div><strong>Computationally designed peptide macrocycle inhibitors of New Delhi metallo-β-lactamase 1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong>, e2012800118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2012800118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85102719411" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2012800118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">32.</a></div><div>Muttenthaler, M. ∙ King, G.F. ∙ Adams, D.J. ...</div><div><strong>Trends in peptide drug discovery</strong></div><div><em>Nat. Rev. Drug Discov.</em> 2021; <strong>20</strong>:309-325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41573-020-00135-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85100542944" target="_blank" aria-label="Open Scopus in new tab">Scopus (924)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33536635/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41573-020-00135-8&amp;pmid=33536635" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">33.</a></div><div>Nielsen, D.S. ∙ Shepherd, N.E. ∙ Xu, W. ...</div><div><strong>Orally absorbed cyclic peptides</strong></div><div><em>Chem. Rev.</em> 2017; <strong>117</strong>:8094-8128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.chemrev.6b00838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85021636298" target="_blank" aria-label="Open Scopus in new tab">Scopus (295)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28541045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.chemrev.6b00838&amp;pmid=28541045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">34.</a></div><div>Ovadia, O. ∙ Greenberg, S. ∙ Chatterjee, J. ...</div><div><strong>The effect of multiple N-methylation on intestinal permeability of cyclic hexapeptides</strong></div><div><em>Mol. Pharm.</em> 2011; <strong>8</strong>:479-487</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/mp1003306" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-79953811147" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21375270/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fmp1003306&amp;pmid=21375270" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">35.</a></div>Park, H., Bradley, P., Greisen, P., Jr, Liu, Y., Mulligan, V.K., Kim, D.E., Baker, D., and DiMaio, F. (2016). Simultaneous Optimization of Biomolecular Energy Functions on Features from Small Molecules and Macromolecules. J. Chem. Theory Comput. <i>12</i>, 6201–6212.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Park%2C+H.%2C+Bradley%2C+P.%2C+Greisen%2C+P.%2C+Jr%2C+Liu%2C+Y.%2C+Mulligan%2C+V.K.%2C+Kim%2C+D.E.%2C+Baker%2C+D.%2C+and+DiMaio%2C+F.+%282016%29.+Simultaneous+Optimization+of+Biomolecular+Energy+Functions+on+Features+from+Small+Molecules+and+Macromolecules.+J.+Chem.+Theory+Comput.+12%2C+6201%E2%80%936212." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">36.</a></div><div>Peraro, L. ∙ Kritzer, J.A.</div><div><strong>Getting. In emerging methods and design principles for cell-penetrant peptides</strong></div><div><em>Angew. Chem.</em> 2018; <strong>130</strong>:12042-12057</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ange.201801361" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fange.201801361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">37.</a></div><div>Pye, C.R. ∙ Hewitt, W.M. ∙ Schwochert, J. ...</div><div><strong>Nonclassical size dependence of permeation defines bounds for passive adsorption of large drug molecules</strong></div><div><em>J.&nbsp;Med. Chem.</em> 2017; <strong>60</strong>:1665-1672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jmedchem.6b01483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85015084736" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28059508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jmedchem.6b01483&amp;pmid=28059508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">38.</a></div><div>Rand, A.C. ∙ Leung, S.S.F. ∙ Eng, H. ...</div><div><strong>Optimizing PK properties of cyclic peptides: the effect of side chain substitutions on permeability and clearance</strong></div><div><em>Medchemcomm.</em> 2012; <strong>3</strong>:1282-1289</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c2md20203d" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84867025688" target="_blank" aria-label="Open Scopus in new tab">Scopus (116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23133740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc2md20203d&amp;pmid=23133740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="optiUo6YTl30I" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optiUo6YTl30I-1" title="View in article">39.</a></div><div>Renfrew, P.D. ∙ Choi, E.J. ∙ Bonneau, R. ...</div><div><strong>Incorporation of noncanonical amino acids into Rosetta and use in computational protein-peptide interface design</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>, e32637</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0032637" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84858266350" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0032637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">40.</a></div><div>Rezai, T. ∙ Bock, J.E. ∙ Zhou, M.V. ...</div><div><strong>Conformational flexibility, internal hydrogen bonding, and passive membrane permeability: successful in&nbsp;silico prediction of the relative permeabilities of cyclic peptides</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2006; <strong>128</strong>:14073-14080</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja063076p" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-33750482579" target="_blank" aria-label="Open Scopus in new tab">Scopus (321)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17061890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja063076p&amp;pmid=17061890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="optElUPnApqkJ" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optElUPnApqkJ" title="View in article">41.</a></div><div>Sevin, E. ∙ Dehouck, L. ∙ Fabulas-da Costa, A. ...</div><div><strong>Accelerated Caco-2 cell permeability model for drug discovery</strong></div><div><em>J. Pharmacol. Toxicol. Methods.</em> 2013; <strong>68</strong>:334-339</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.vascn.2013.07.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84883386431" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23916595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.vascn.2013.07.004&amp;pmid=23916595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="optM9Rq3dRHgc" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optM9Rq3dRHgc-1" title="View in article">42.</a></div><div>Sheldrick, G.M.</div><div><strong>SHELXT--Integrated space-group and crystal-structure determination</strong></div><div><em>Acta Crystallogr. A Found. Adv.</em> 2015; <strong>71</strong>:3-8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2053273314026370" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84920370689" target="_blank" aria-label="Open Scopus in new tab">Scopus (30086)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25537383/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2053273314026370&amp;pmid=25537383" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="optgPLOKl5zI3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optgPLOKl5zI3-1" title="View in article">43.</a></div><div>Sheldrick, G.M.</div><div><strong>Crystal strcuture refinement with SHELXL</strong></div><div><em>Acta Crystallogr. C Struct. Chem.</em> 2015; <strong>71</strong>:3-8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/s2053229614024218" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84937398809" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25567568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2Fs2053229614024218&amp;pmid=25567568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23-1" title="View in article">44.</a></div><div>Tejero, R. ∙ Snyder, D. ∙ Mao, B. ...</div><div><strong>PDBStat: a universal restraint converter and restraint analysis software package for protein NMR</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2013; <strong>56</strong>:337-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-013-9753-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84883463996" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23897031/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-013-9753-7&amp;pmid=23897031" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="optKJLDz0ySQe" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optKJLDz0ySQe" title="View in article">45.</a></div><div>Vögeli, B. ∙ Ying, J. ∙ Grishaev, A. ...</div><div><strong>Limits on variations in protein backbone dynamics from precise measurements of scalar couplings</strong></div><div><em>J. Am. Chem. Soc.</em> 2007; <strong>129</strong>:9377-9385</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja070324o" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-34547670259" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17608477/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja070324o&amp;pmid=17608477" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">46.</a></div><div>Wang, C.K. ∙ Northfield, S.E. ∙ Colless, B. ...</div><div><strong>Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:17504-17509</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1417611111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84916618640" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25416591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1417611111&amp;pmid=25416591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="optogFgrOAoxu" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">47.</a></div><div>Wang, C.K. ∙ Northfield, S.E. ∙ Swedberg, J.E. ...</div><div><strong>Exploring experimental and computational markers of cyclic peptides: Charting islands of permeability</strong></div><div><em>Eur. J. Med. Chem.</em> 2015; <strong>97</strong>:202-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ejmech.2015.04.049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84929084835" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25974856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ejmech.2015.04.049&amp;pmid=25974856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">48.</a></div><div>Wang, C.K. ∙ Swedberg, J.E. ∙ Harvey, P.J. ...</div><div><strong>Conformational flexibility is a determinant of permeability for cyclosporin</strong></div><div><em>J.&nbsp;Phys. Chem. B.</em> 2018; <strong>122</strong>:2261-2276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jpcb.7b12419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-85042759162" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29400464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jpcb.7b12419&amp;pmid=29400464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="optfKm2A3IJUh" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optfKm2A3IJUh" title="View in article">49.</a></div><div>Waskom, M.L.</div><div><strong>Seaborn: statistical data visualization</strong></div><div><em>J. Open Source Softw.</em> 2021; <strong>6</strong>:3021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.21105/joss.03021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.21105%2Fjoss.03021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">50.</a></div><div>White, T.R. ∙ Renzelman, C.M. ∙ Rand, A.C. ...</div><div><strong>On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</strong></div><div><em>Nat. Chem. Biol.</em> 2011; <strong>7</strong>:810-817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nchembio.664" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-80054853098" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21946276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnchembio.664&amp;pmid=21946276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">51.</a></div>Winn, M.D., Refinemen, M.-L.S., Pannu, N.J., Murshudov, G.N., and Others (2001). CCP4 v6. 3.0 Program References. Acta Crystallogr. <i>57</i>, 122–133.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Winn%2C+M.D.%2C+Refinemen%2C+M.-L.S.%2C+Pannu%2C+N.J.%2C+Murshudov%2C+G.N.%2C+and+Others+%282001%29.+CCP4+v6.+3.0+Program+References.+Acta+Crystallogr.+57%2C+122%E2%80%93133." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="optqmdefJ4GOW" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optqmdefJ4GOW" title="View in article">52.</a></div><div>Yilmaz, E.M. ∙ Güntert, P.</div><div><strong>NMR structure calculation for all small molecule ligands and non-standard residues from the PDB Chemical Component Dictionary</strong></div><div><em>J. Biomol. NMR.</em> 2015; <strong>63</strong>:21-37</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-015-9959-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.07.019&amp;key=2-s2.0-84942373727" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26123317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-015-9959-y&amp;pmid=26123317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (15)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/0c4121e7-9a77-441f-8d2b-4976ce5ffd69/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/e461f39f-573e-4e68-b764-c71cb7b42049/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/8c9f7083-17e4-4670-8341-4e6839a94063/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/aaf53044-475e-4ddc-823f-30ca63620616/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/77f62e5e-8f4b-4e5d-9cc0-85e56cdae71f/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/1d0a9623-f021-4c53-b10c-5af6eae00854/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/fcba320f-97d1-4bec-86d3-60ddc0a528eb/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/2232c03d-b9b8-4c70-915d-1a3f13652659/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/82a17271-4456-4327-a13b-a9d8900e8ee6/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/d81db9c2-5b18-4d77-9df4-c3b6743c3020/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/4754ca46-194b-4a12-9296-218a6de0c947/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs8" aria-label="Go to Figure&amp;nbsp;S8"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/e0e6caa8-97b5-408c-a0be-8f4299cedadc/main.assets/figs8.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs9" aria-label="Go to Figure&amp;nbsp;S9"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/6d2d1fdf-705b-4773-b6ee-b7091aeb69e2/main.assets/figs9.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs10" aria-label="Go to Figure&amp;nbsp;S10"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/41099327-1b91-4af6-b366-6dde3b31fa43/main.assets/figs10.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/asset/7d5c1d69-adc7-4a50-94d6-e0455402a00e/main.assets/gr5.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00207%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867422009229%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (10)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/731a58d0-211b-4504-97b4-5f8f3227fe51/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (79.33 KB)</span></a><div class="core-description">Table&nbsp;S1. RMSD between the designed models and X-ray crystal structures of the 7- and 8-amino-acid macrocycles, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/dddca6c1-f5f5-4938-adb4-075f77b44c28/mmc2.pdf" download="mmc2.pdf"><span id="mmc2-link">PDF (69.75 KB)</span></a><div class="core-description">Table&nbsp;S2. RMSD between the designed models and X-ray crystal structures of the 9–11 residue macrocycles, related to Figure&nbsp;3</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/1603bcf8-4901-4e41-8267-d96ffec0ac55/mmc3.pdf" download="mmc3.pdf"><span id="mmc3-link">PDF (91.85 KB)</span></a><div class="core-description">Table&nbsp;S3. NMR chemical shift assignments for peptides D9.16, D8.21, and D8.31, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/9b2096ba-a863-4348-a465-f1ffb0541564/mmc4.pdf" download="mmc4.pdf"><span id="mmc4-link">PDF (103.21 KB)</span></a><div class="core-description">Table&nbsp;S4. NMR validation tables for designs D9.16, D8.31, and D8.21. RMSD calculated for 20 lowest energy structures out of 100, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/e93aa71e-a5f6-4e0f-8faa-532206cb70ef/mmc5.pdf" download="mmc5.pdf"><span id="mmc5-link">PDF (15.77 MB)</span></a><div class="core-description">Data S1. Structure prediction plots for all designs, related to Figures&nbsp;1 and 3</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/f6f69fb2-8822-4897-b19d-f0474e0ee2ce/mmc6.zip" download="mmc6.zip"><span id="mmc6-link">Archive (877.83 KB)</span></a><div class="core-description">Data S2. Amino acid sequences and computational models, related to Figures&nbsp;1 and 3</div></div></div><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/04fbb571-6234-4063-8594-7751a10b17dc/mmc7.zip" download="mmc7.zip"><span id="mmc7-link">Archive (159.92 KB)</span></a><div class="core-description">Data S3. PAMPA and Caco-2 data, related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/bfd0b5c0-952b-4e37-a8f6-cbfcce3135db/mmc8.pdf" download="mmc8.pdf"><span id="mmc8-link">PDF (131.79 KB)</span></a><div class="core-description">Data S4. Oral bioavailability data, related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/8290313b-df6d-4ec2-96fe-1a38cc2f2d30/mmc9.zip" download="mmc9.zip"><span id="mmc9-link">Archive (81.73 KB)</span></a><div class="core-description">Data S5. Example design scripts, related to STAR Methods</div></div></div><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.07.019/attachment/3bbd8235-dd58-42a1-b501-2c8765bbf5cb/mmc10.zip" download="mmc10.zip"><span id="mmc10-link">Archive (727.07 KB)</span></a><div class="core-description">Data S6. PDB files for experimental structures, related to STAR Methods</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867422009229" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-physical-science/fulltext/S2666-3864(24)00313-8" target="_blank" title="Heparin reversal through an entire encapsulation strategy by giant macrocycles">Heparin reversal through an entire encapsulation strategy by giant macrocycles</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Physical Science</i></span><span id="item_date">June 12, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(24)00654-4" target="_blank" title="The dynamics of red blood cells traversing slits of mechanical heart valves under high shear">The dynamics of red blood cells traversing slits of mechanical heart valves under high shear</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Meng et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">September 27, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(25)00035-2" target="_blank" title="Identification, structure, and agonist design of an androgen membrane receptor">Identification, structure, and agonist design of an androgen membrane receptor</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">January 29, 2025</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem/fulltext/S2451-9294(24)00471-6" target="_blank" title="Bridging gaps between lab- and fab-oriented anode design for proton exchange membrane water electrolyzers">Bridging gaps between lab- and fab-oriented anode design for proton exchange membrane water electrolyzers</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xu et al.</li></ul></div><div class="linked-article__details"><span><i>Chem</i></span><span id="item_date">October 10, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)14335-6" target="_blank" title="Performance and reinforcement of air-cooled embankments traversing degrading permafrost of the Qinghai-Tibet Plateau">Performance and reinforcement of air-cooled embankments traversing degrading permafrost of the Qinghai-Tibet Plateau</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">September 28, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem/fulltext/S2451-9294(25)00119-6" target="_blank" title="Recent advances in membrane and sorbent design for blood purification in sepsis and septic shock">Recent advances in membrane and sorbent design for blood purification in sepsis and septic shock</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ding et al.</li></ul></div><div class="linked-article__details"><span><i>Chem</i></span><span id="item_date">April 3, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-physical-science/fulltext/S2666-3864(22)00461-1" target="_blank" title="Dynamic covalent macrocycles co-delivering genes and drugs against drug-resistant cancer">Dynamic covalent macrocycles co-delivering genes and drugs against drug-resistant cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lyu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Physical Science</i></span><span id="item_date">November 9, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-physical-science/fulltext/S2666-3864(24)00669-6" target="_blank" title="Quantitative synthesis of dynamic combinatorial macrocycles accelerated by preorganization of AIEgens for live visualization of drug release">Quantitative synthesis of dynamic combinatorial macrocycles accelerated by preorganization of AIEgens for live visualization of drug release</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Physical Science</i></span><span id="item_date">December 27, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-biomaterials/fulltext/S3050-5623(24)00003-5" target="_blank" title="Design of an electrospun wrap for improved durability and infection control in Masquelet’s induced membrane technique">Design of an electrospun wrap for improved durability and infection control in Masquelet’s induced membrane technique</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jones et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Biomaterials</i></span><span id="item_date">December 2, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-physical-science/fulltext/S2666-3864(24)00116-4" target="_blank" title="Rational electrode design for low-cost proton exchange membrane water electrolyzers">Rational electrode design for low-cost proton exchange membrane water electrolyzers</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yuan et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Physical Science</i></span><span id="item_date">March 20, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem/fulltext/S2451-9294(24)00484-4" target="_blank" title="Globally aromatic odd-electron π-magnetic macrocycle">Globally aromatic odd-electron π-magnetic macrocycle</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Villalobos et al.</li></ul></div><div class="linked-article__details"><span><i>Chem</i></span><span id="item_date">October 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/structure/fulltext/S0969-2126(25)00021-8" target="_blank" title="Discovery and characterization of potent macrocycle inhibitors of ubiquitin-specific protease-7">Discovery and characterization of potent macrocycle inhibitors of ubiquitin-specific protease-7</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Miranda et al.</li></ul></div><div class="linked-article__details"><span><i>Structure</i></span><span id="item_date">February 20, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/matter/fulltext/S2590-2385(20)30517-8" target="_blank" title="Versatile Roles of Macrocycles in Organic-Inorganic Hybrid Materials for Biomedical Applications">Versatile Roles of Macrocycles in Organic-Inorganic Hybrid Materials for Biomedical Applications</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yue et al.</li></ul></div><div class="linked-article__details"><span><i>Matter</i></span><span id="item_date">November 04, 2020</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-physical-science/fulltext/S2666-3864(24)00336-9" target="_blank" title="Superior polymeric gas separation membrane designed by explainable graph machine learning">Superior polymeric gas separation membrane designed by explainable graph machine learning</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Physical Science</i></span><span id="item_date">June 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)16904-6" target="_blank" title="A rapid approach based on empirical formulas and graph traversal algorithms to automatic generation of fragrance formulas">A rapid approach based on empirical formulas and graph traversal algorithms to automatic generation of fragrance formulas</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Cai et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">December 3, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(24)01313-4" target="_blank" title="BPS2025 - Design of membrane-lytic peptides for the treatment of individual cancer types">BPS2025 - Design of membrane-lytic peptides for the treatment of individual cancer types</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Nwajiaku et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">February 13, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/chemistry/fulltext/S2589-5974(20)30218-5" target="_blank" title="Macrocycles and Related Hosts as Supramolecular Antidotes">Macrocycles and Related Hosts as Supramolecular Antidotes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yin et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Chemistry</i></span><span id="item_date">September 29, 2020</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/matter/fulltext/S2590-2385(23)00572-6" target="_blank" title="Electrocatalyst and electrode design strategies for durable proton exchange membrane fuel cells">Electrocatalyst and electrode design strategies for durable proton exchange membrane fuel cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xiao et al.</li></ul></div><div class="linked-article__details"><span><i>Matter</i></span><span id="item_date">December 8, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(24)03893-1" target="_blank" title="BPS2025 - Design of a miniaturized fluidic cell for suspended lipid bilayer membrane formation and electrophysiological recordings">BPS2025 - Design of a miniaturized fluidic cell for suspended lipid bilayer membrane formation and electrophysiological recordings</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Porro et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">February 13, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(24)02664-X" target="_blank" title="BPS2025 - Cryo-EM study of the yeast GET pathway: GET4/5-driven remodeling of GET3, an ATPase required for accurate targeting of tail-anchored proteins to the ER membrane">BPS2025 - Cryo-EM study of the yeast GET pathway: GET4/5-driven remodeling of GET3, an ATPase required for accurate targeting of tail-anchored proteins to the ER membrane</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Granados-Villanueva et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">February 13, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(24)02809-1" target="_blank" title="BPS2025 - Cryo-EM study of the yeast GET pathway: GET4/5-driven remodeling of GET3, an ATPase required for accurate targeting of tail-anchored proteins to the ER membrane">BPS2025 - Cryo-EM study of the yeast GET pathway: GET4/5-driven remodeling of GET3, an ATPase required for accurate targeting of tail-anchored proteins to the ER membrane</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Granados-Villanueva et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">February 13, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem-catalysis/fulltext/S2667-1093(23)00129-X" target="_blank" title="Bimetallic macrocycles that efficiently catalyze the addition of CO2 to styrene oxide">Bimetallic macrocycles that efficiently catalyze the addition of CO2 to styrene oxide</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Michael P. Schramm</li></ul></div><div class="linked-article__details"><span><i>Chem Catalysis</i></span><span id="item_date">April 20, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem/fulltext/S2451-9294(20)30310-7" target="_blank" title="Plug-and-Play Optical Materials from Fluorescent Dyes and Macrocycles">Plug-and-Play Optical Materials from Fluorescent Dyes and Macrocycles</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Benson et al.</li></ul></div><div class="linked-article__details"><span><i>Chem</i></span><span id="item_date">August 06, 2020</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem/fulltext/S2451-9294(21)00539-8" target="_blank" title="All are aromatic: A 3D globally aromatic cage containing five types of 2D aromatic macrocycles">All are aromatic: A 3D globally aromatic cage containing five types of 2D aromatic macrocycles</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ren et al.</li></ul></div><div class="linked-article__details"><span><i>Chem</i></span><span id="item_date">November 24, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/matter/fulltext/S2590-2385(25)00181-X" target="_blank" title="Artificial intelligence for catalyst design and synthesis">Artificial intelligence for catalyst design and synthesis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Matter</i></span><span id="item_date">May 07, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)07344-4" target="_blank" title="A single-step extraction and immobilization of soybean lipolytic enzymes by using a purpose-designed copolymer of styrene and maleic acid as a membrane lysis agent">A single-step extraction and immobilization of soybean lipolytic enzymes by using a purpose-designed copolymer of styrene and maleic acid as a membrane lysis agent</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Buachi et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">May 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(24)03491-X" target="_blank" title="BPS2025 - Harnessing minimal cells to understand the design principles of living membranes">BPS2025 - Harnessing minimal cells to understand the design principles of living membranes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Safronova et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">February 13, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-physical-science/fulltext/S2666-3864(24)00109-7" target="_blank" title="Macrocycle-based fluorochromic systems">Macrocycle-based fluorochromic systems</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Physical Science</i></span><span id="item_date">March 13, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem/fulltext/S2451-9294(19)30080-4" target="_blank" title="Tetraurea Macrocycles: Aggregation-Driven Binding of Chloride in Aqueous Solutions">Tetraurea Macrocycles: Aggregation-Driven Binding of Chloride in Aqueous Solutions</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wu et al.</li></ul></div><div class="linked-article__details"><span><i>Chem</i></span><span id="item_date">March 25, 2019</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(23)00056-1" target="_blank" title="Designing artificial synthetic promoters for accurate, smart, and versatile gene expression in plants">Designing artificial synthetic promoters for accurate, smart, and versatile gene expression in plants</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yasmeen et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">February 8, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(20)30867-1" target="_blank" title="Donor-Acceptor Conjugated Macrocycles with Polyradical Character and Global Aromaticity">Donor-Acceptor Conjugated Macrocycles with Polyradical Character and Global Aromaticity</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sabuj et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">October 14, 2020</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/nexus/fulltext/S2950-1601(25)00017-8" target="_blank" title="Active learning for advanced biodegradable film design">Active learning for advanced biodegradable film design</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gao et al.</li></ul></div><div class="linked-article__details"><span><i>Nexus</i></span><span id="item_date">March 23, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/microbiology/fulltext/S0966-842X(24)00113-6" target="_blank" title="Influence of antimicrobial peptides on the bacterial membrane curvature and vice versa">Influence of antimicrobial peptides on the bacterial membrane curvature and vice versa</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Cardoso et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Microbiology</i></span><span id="item_date">May 21, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(22)02263-9" target="_blank" title="Accurate molecular dynamics modeling of Pseudomonas aeruginosa outer membrane interaction with signaling molecules">Accurate molecular dynamics modeling of Pseudomonas aeruginosa outer membrane interaction with signaling molecules</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Pirhadi et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">February 10, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(18)31622-9" target="_blank" title="Toward Accurate Prediction and Design of Kinked Alpha Helices in Membrane Proteins">Toward Accurate Prediction and Design of Kinked Alpha Helices in Membrane Proteins</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lasher et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">February 15, 2019</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-physical-science/fulltext/S2666-3864(23)00446-0" target="_blank" title="Designing off-grid green hydrogen plants using dynamic polymer electrolyte membrane electrolyzers to minimize the hydrogen production cost">Designing off-grid green hydrogen plants using dynamic polymer electrolyte membrane electrolyzers to minimize the hydrogen production cost</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ginsberg et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Physical Science</i></span><span id="item_date">October 9, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem/fulltext/S2451-9294(22)00264-9" target="_blank" title="Cross-reactive binding versus selective phosphate sensing in an imine macrocycle sensor">Cross-reactive binding versus selective phosphate sensing in an imine macrocycle sensor</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Radujević et al.</li></ul></div><div class="linked-article__details"><span><i>Chem</i></span><span id="item_date">June 8, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)11282-0" target="_blank" title="Sustainable design: Circular innovation design method under process reengineering">Sustainable design: Circular innovation design method under process reengineering</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 1, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/joule/fulltext/S2542-4351(25)00065-0" target="_blank" title="Systematic feature design for cycle life prediction of lithium-ion batteries during formation">Systematic feature design for cycle life prediction of lithium-ion batteries during formation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Rhyu et al.</li></ul></div><div class="linked-article__details"><span><i>Joule</i></span><span id="item_date">March 28, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-methods/fulltext/S2667-2375(25)00066-9" target="_blank" title="Accurate prediction of absolute prokaryotic abundance from DNA concentration">Accurate prediction of absolute prokaryotic abundance from DNA concentration</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wirbel et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Methods</i></span><span id="item_date">April 28, 2025</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2022.07.019","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Accurate de novo design of membrane-traversing macrocycles</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.07.019/asset/0c4121e7-9a77-441f-8d2b-4976ce5ffd69/main.assets/figs1.jpg" height="600" width="656" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/802874cb-242e-4d8f-915e-580cbee57048/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.07.019/asset/e461f39f-573e-4e68-b764-c71cb7b42049/main.assets/gr1.jpg" height="938" width="722" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/4427ce97-40df-4997-87eb-c7d6b25358f5/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.07.019/asset/8c9f7083-17e4-4670-8341-4e6839a94063/main.assets/gr2.jpg" height="648" width="656" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/0dbb1d4b-9f61-440d-b8be-cdba784b83bc/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.07.019/asset/aaf53044-475e-4ddc-823f-30ca63620616/main.assets/figs2.jpg" height="537" width="497" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/0fe1203e-2b5a-42c9-99c5-4f1490038fd9/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.07.019/asset/77f62e5e-8f4b-4e5d-9cc0-85e56cdae71f/main.assets/gr3.jpg" height="937" width="720" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/b60f7b64-6b0d-400a-a3a4-fb089de8f51a/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.07.019/asset/1d0a9623-f021-4c53-b10c-5af6eae00854/main.assets/figs3.jpg" height="450" width="656" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/3e127695-8b39-4033-bbef-6f610dea2dab/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.07.019/asset/fcba320f-97d1-4bec-86d3-60ddc0a528eb/main.assets/figs4.jpg" height="274" width="759" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/c058313f-7cf4-4e46-92b8-46e132a5f2c1/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.07.019/asset/2232c03d-b9b8-4c70-915d-1a3f13652659/main.assets/figs5.jpg" height="458" width="759" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/ac26d099-81c6-4695-a142-c250f0214b1f/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.07.019/asset/82a17271-4456-4327-a13b-a9d8900e8ee6/main.assets/gr4.jpg" height="430" width="759" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/52a2e762-bc2f-4b00-a112-9f318a04a122/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.07.019/asset/d81db9c2-5b18-4d77-9df4-c3b6743c3020/main.assets/figs6.jpg" height="937" width="587" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/3a2f3410-db7f-4913-9f76-7bbbff57493a/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.07.019/asset/4754ca46-194b-4a12-9296-218a6de0c947/main.assets/figs7.jpg" height="501" width="759" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/9f132870-6b8c-4144-8850-500730169172/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs8" src="/cms/10.1016/j.cell.2022.07.019/asset/e0e6caa8-97b5-408c-a0be-8f4299cedadc/main.assets/figs8.jpg" height="510" width="759" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/9fb13e1b-07dd-489f-8ece-62422bdd0ccb/main.assets/figs8_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs9" src="/cms/10.1016/j.cell.2022.07.019/asset/6d2d1fdf-705b-4773-b6ee-b7091aeb69e2/main.assets/figs9.jpg" height="637" width="656" aria-labelledby="figs9-title figs9-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/b318d836-d425-4cf9-b165-87d46b01b0c9/main.assets/figs9_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs10" src="/cms/10.1016/j.cell.2022.07.019/asset/41099327-1b91-4af6-b366-6dde3b31fa43/main.assets/figs10.jpg" height="520" width="762" aria-labelledby="figs10-title figs10-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/ab2f885a-6ed7-4431-b742-1962d33b1d6d/main.assets/figs10_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.07.019/asset/7d5c1d69-adc7-4a50-94d6-e0455402a00e/main.assets/gr5.jpg" height="445" width="656" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/a478f72f-3ee6-40f8-b4c1-f956b909b417/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.07.019/asset/0c4121e7-9a77-441f-8d2b-4976ce5ffd69/main.assets/figs1.jpg" height="600" width="656" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/802874cb-242e-4d8f-915e-580cbee57048/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.07.019/asset/e461f39f-573e-4e68-b764-c71cb7b42049/main.assets/gr1.jpg" height="938" width="722" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/4427ce97-40df-4997-87eb-c7d6b25358f5/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.07.019/asset/8c9f7083-17e4-4670-8341-4e6839a94063/main.assets/gr2.jpg" height="648" width="656" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/0dbb1d4b-9f61-440d-b8be-cdba784b83bc/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.07.019/asset/aaf53044-475e-4ddc-823f-30ca63620616/main.assets/figs2.jpg" height="537" width="497" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/0fe1203e-2b5a-42c9-99c5-4f1490038fd9/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.07.019/asset/77f62e5e-8f4b-4e5d-9cc0-85e56cdae71f/main.assets/gr3.jpg" height="937" width="720" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/b60f7b64-6b0d-400a-a3a4-fb089de8f51a/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.07.019/asset/1d0a9623-f021-4c53-b10c-5af6eae00854/main.assets/figs3.jpg" height="450" width="656" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/3e127695-8b39-4033-bbef-6f610dea2dab/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.07.019/asset/fcba320f-97d1-4bec-86d3-60ddc0a528eb/main.assets/figs4.jpg" height="274" width="759" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/c058313f-7cf4-4e46-92b8-46e132a5f2c1/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.07.019/asset/2232c03d-b9b8-4c70-915d-1a3f13652659/main.assets/figs5.jpg" height="458" width="759" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/ac26d099-81c6-4695-a142-c250f0214b1f/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.07.019/asset/82a17271-4456-4327-a13b-a9d8900e8ee6/main.assets/gr4.jpg" height="430" width="759" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/52a2e762-bc2f-4b00-a112-9f318a04a122/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.07.019/asset/d81db9c2-5b18-4d77-9df4-c3b6743c3020/main.assets/figs6.jpg" height="937" width="587" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/3a2f3410-db7f-4913-9f76-7bbbff57493a/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.07.019/asset/4754ca46-194b-4a12-9296-218a6de0c947/main.assets/figs7.jpg" height="501" width="759" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/9f132870-6b8c-4144-8850-500730169172/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S8</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs8" src="/cms/10.1016/j.cell.2022.07.019/asset/e0e6caa8-97b5-408c-a0be-8f4299cedadc/main.assets/figs8.jpg" height="510" width="759" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/9fb13e1b-07dd-489f-8ece-62422bdd0ccb/main.assets/figs8_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S9</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs9" src="/cms/10.1016/j.cell.2022.07.019/asset/6d2d1fdf-705b-4773-b6ee-b7091aeb69e2/main.assets/figs9.jpg" height="637" width="656" aria-labelledby="figs9-title figs9-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/b318d836-d425-4cf9-b165-87d46b01b0c9/main.assets/figs9_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S10</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs10" src="/cms/10.1016/j.cell.2022.07.019/asset/41099327-1b91-4af6-b366-6dde3b31fa43/main.assets/figs10.jpg" height="520" width="762" aria-labelledby="figs10-title figs10-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/ab2f885a-6ed7-4431-b742-1962d33b1d6d/main.assets/figs10_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.07.019/asset/7d5c1d69-adc7-4a50-94d6-e0455402a00e/main.assets/gr5.jpg" height="445" width="656" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.07.019/asset/a478f72f-3ee6-40f8-b4c1-f956b909b417/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00922-9';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00922-9';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749594452253","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"185","issueNumber":"19","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867422009229","coverDate":"202209","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Accurate emde novoem design of membranetraversing macrocycles","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc4be69d464ed1',t:'MTc0OTU5NDQ1MS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc4be69d464ed1&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.6.2&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2022.07.019&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>