<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="7956729f-dad2-4e31-b5e0-aaefd29bbb18" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;issue:issue:pii\:S0092867419X00112;ctype:string:Journal Content;article:article:pii\:S0092867420303895;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="A Structure-Based Model for the Complete Transcription Cycle of Influenza Polymerase">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)30389-5">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/ed2cfa66-ca5f-465c-b9ad-b8b687a995c5/fx1.jpg">
    


        <meta property="og:description" content="Influenza polymerase transcribes the negative sense viral RNA genome into mRNA in
the nucleus of infected cells. This work by Cusack and colleagues reports high-resolution
cryo-EM structures of the polymerase at various stages of transcription providing
a molecular basis for the complete transcription cycle, which should enable improved
inhibitor design.">
    











        <meta name="twitter:title" content="A Structure-Based Model for the Complete Transcription Cycle of Influenza Polymerase">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/ed2cfa66-ca5f-465c-b9ad-b8b687a995c5/fx1.jpg">
    


        <meta name="twitter:description" content="Influenza polymerase transcribes the negative sense viral RNA genome into mRNA in
the nucleus of infected cells. This work by Cusack and colleagues reports high-resolution
cryo-EM structures of the polymerase at various stages of transcription providing
a molecular basis for the complete transcription cycle, which should enable improved
inhibitor design.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>A Structure-Based Model for the Complete Transcription Cycle of Influenza Polymerase: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="A Structure-Based Model for the Complete Transcription Cycle of Influenza Polymerase"> <meta name="citation_date" content="2020/05/14"> <meta name="citation_online_date" content="2020/04/17"> <meta name="citation_volume" content="181"> <meta name="citation_issue" content="4"> <meta name="citation_firstpage" content="877"> <meta name="citation_lastpage" content="893.e21"> <meta name="citation_pmid" content="32304664"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867420303895"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867420303895"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867420303895"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2020.03.061"> <meta name="citation_pmid" content="32304664"> <meta name="citation_author" content="Joanna M. Wandzik"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France"><meta name="citation_author" content="Tomas Kouba"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France"><meta name="citation_author" content="Manikandan Karuppasamy"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France"><meta name="citation_author" content="Alexander Pflug"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France"><meta name="citation_author" content="Petra Drncova"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France"><meta name="citation_author" content="Jan Provaznik"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, GeneCore, Meyerhofstraße 1, 69117 Heidelberg, Germany"><meta name="citation_author" content="Nayara Azevedo"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, GeneCore, Meyerhofstraße 1, 69117 Heidelberg, Germany"><meta name="citation_author" content="Stephen Cusack"><meta name="citation_author_email" content="cusack@embl.fr"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France"><meta name="citation_keywords" content="Influenza virus; RNA-dependent RNA polymerase; transcription; cap-snatching; poly-adenylation; secondary 3’; end binding site; cryo-electron microscopy"><meta name="citation_reference" content="citation_title=Real-space refinement in PHENIX for cryo-EM and crystallography; citation_author=P.V. Afonine; citation_author=B.K. Poon; citation_author=R.J. Read; citation_author=O.V. Sobolev; citation_author=T.C. Terwilliger; citation_author=A. Urzhumtsev; citation_author=P.D. Adams; citation_journal_title=Acta Crystallogr. D Struct. Biol.; citation_volume=74; citation_firstpage=531; citation_lastpage=544; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The structure of native influenza virion ribonucleoproteins; citation_author=R. Arranz; citation_author=R. Coloma; citation_author=F.J. Chichón; citation_author=J.J. Conesa; citation_author=J.L. Carrascosa; citation_author=J.M. Valpuesta; citation_author=J. Ortín; citation_author=J. Martín-Benito; citation_journal_title=Science; citation_volume=338; citation_firstpage=1634; citation_lastpage=1637; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Recent developments in the CCP-EM software suite; citation_author=T. Burnley; citation_author=C.M. Palmer; citation_author=M. Winn; citation_journal_title=Acta Crystallogr. D Struct. Biol.; citation_volume=73; citation_firstpage=469; citation_lastpage=477; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Structural insights into influenza A virus ribonucleoproteins reveal a processive helical track as transcription mechanism; citation_author=R. Coloma; citation_author=R. Arranz; citation_author=J.M. de la Rosa-Trevín; citation_author=C.O.S. Sorzano; citation_author=S. Munier; citation_author=D. Carlero; citation_author=N. Naffakh; citation_author=J. Ortín; citation_author=J. Martín-Benito; citation_journal_title=Nat. Microbiol.; citation_doi=10.1038/s41564-020-0675-3; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=WebLogo: a sequence logo generator; citation_author=G.E. Crooks; citation_author=G. Hon; citation_author=J.M. Chandonia; citation_author=S.E. Brenner; citation_journal_title=Genome Res.; citation_volume=14; citation_firstpage=1188; citation_lastpage=1190; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=At the centre: influenza A virus ribonucleoproteins; citation_author=A.J. Eisfeld; citation_author=G. Neumann; citation_author=Y. Kawaoka; citation_journal_title=Nat. Rev. Microbiol.; citation_volume=13; citation_firstpage=28; citation_lastpage=41; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Features and development of Coot; citation_author=P. Emsley; citation_author=B. Lohkamp; citation_author=W.G. Scott; citation_author=K. Cowtan; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=66; citation_firstpage=486; citation_lastpage=501; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Structures of influenza A virus RNA polymerase offer insight into viral genome replication; citation_author=H. Fan; citation_author=A.P. Walker; citation_author=L. Carrique; citation_author=J.R. Keown; citation_author=I. Serna Martin; citation_author=D. Karia; citation_author=J. Sharps; citation_author=N. Hengrung; citation_author=E. Pardon; citation_author=J. Steyaert; citation_author=et al.; citation_journal_title=Nature; citation_volume=573; citation_firstpage=287; citation_lastpage=290; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Structural Insights into Bunyavirus Replication and Its Regulation by the vRNA Promoter; citation_author=P. Gerlach; citation_author=H. Malet; citation_author=S. Cusack; citation_author=J. Reguera; citation_journal_title=Cell; citation_volume=161; citation_firstpage=1267; citation_lastpage=1279; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Je, a versatile suite to handle multiplexed NGS libraries with unique molecular identifiers; citation_author=C. Girardot; citation_author=J. Scholtalbers; citation_author=S. Sauer; citation_author=S.-Y. Su; citation_author=E.E.M. Furlong; citation_journal_title=BMC Bioinformatics; citation_volume=17; citation_doi=10.1186/s12859-016-1284-2; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Structure and synthesis of influenza virus complementary RNAs; citation_author=A.J. Hay; citation_author=J.J. Skehel; citation_author=J. McCauley; citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_volume=288; citation_firstpage=341; citation_lastpage=348; citation_publication_date=1980; "><meta name="citation_reference" content="citation_title=Conserved and host-specific features of influenza virion architecture; citation_author=E.C. Hutchinson; citation_author=P.D. Charles; citation_author=S.S. Hester; citation_author=B. Thomas; citation_author=D. Trudgian; citation_author=M. Martínez-Alonso; citation_author=E. Fodor; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=4816; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Model-based local density sharpening of cryo-EM maps; citation_author=A.J. Jakobi; citation_author=M. Wilmanns; citation_author=C. Sachse; citation_journal_title=eLife; citation_volume=6; citation_firstpage=e27131; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Genetic trans-complementation establishes a new model for influenza virus RNA transcription and replication; citation_author=N. Jorba; citation_author=R. Coloma; citation_author=J. Ortín; citation_journal_title=PLoS Pathog.; citation_volume=5; citation_firstpage=e1000462; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Improvements to the APBS biomolecular solvation software suite; citation_author=E. Jurrus; citation_author=D. Engel; citation_author=K. Star; citation_author=K. Monson; citation_author=J. Brandi; citation_author=L.E. Felberg; citation_author=D.H. Brookes; citation_author=L. Wilson; citation_author=J. Chen; citation_author=K. Liles; citation_author=et al.; citation_journal_title=Protein Sci.; citation_volume=27; citation_firstpage=112; citation_lastpage=128; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Structural snapshots of actively transcribing influenza polymerase; citation_author=T. Kouba; citation_author=P. Drncová; citation_author=S. Cusack; citation_journal_title=Nat. Struct. Mol. Biol.; citation_volume=26; citation_firstpage=460; citation_lastpage=470; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The splicing factor proline-glutamine rich (SFPQ/PSF) is involved in influenza virus transcription; citation_author=S. Landeras-Bueno; citation_author=N. Jorba; citation_author=M. Pérez-Cidoncha; citation_author=J. Ortín; citation_journal_title=PLoS Pathog.; citation_volume=7; citation_firstpage=e1002397; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=LigPlot+: multiple ligand-protein interaction diagrams for drug discovery; citation_author=R.A. Laskowski; citation_author=M.B. Swindells; citation_journal_title=J.&nbsp;Chem. Inf. Model.; citation_volume=51; citation_firstpage=2778; citation_lastpage=2786; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Non-Uniform and Non-Random Binding of Nucleoprotein to Influenza A and B Viral RNA; citation_author=V. Le Sage; citation_author=A.V. Nanni; citation_author=A.R. Bhagwat; citation_author=D.J. Snyder; citation_author=V.S. Cooper; citation_author=S.S. Lakdawala; citation_author=N. Lee; citation_journal_title=Viruses; citation_volume=10; citation_firstpage=E522; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genome-wide analysis of influenza viral RNA and nucleoprotein association; citation_author=N. Lee; citation_author=V. Le Sage; citation_author=A.V. Nanni; citation_author=D.J. Snyder; citation_author=V.S. Cooper; citation_author=S.S. Lakdawala; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_firstpage=8968; citation_lastpage=8977; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Characterization of the polyadenylation signal of influenza virus RNA; citation_author=X. Li; citation_author=P. Palese; citation_journal_title=J.&nbsp;Virol.; citation_volume=68; citation_firstpage=1245; citation_lastpage=1249; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Structural basis of an essential interaction between influenza polymerase and Pol II CTD; citation_author=M. Lukarska; citation_author=G. Fournier; citation_author=A. Pflug; citation_author=P. Resa-Infante; citation_author=S. Reich; citation_author=N. Naffakh; citation_author=S. Cusack; citation_journal_title=Nature; citation_volume=541; citation_firstpage=117; citation_lastpage=121; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=UNAFold: software for nucleic acid folding and hybridization; citation_author=N.R. Markham; citation_author=M. Zuker; citation_journal_title=Methods Mol. Biol.; citation_volume=453; citation_firstpage=3; citation_lastpage=31; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads; citation_author=M. Martin; citation_journal_title=EMBnetjournal; citation_volume=17; citation_firstpage=10; citation_lastpage=12; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Phaser crystallographic software; citation_author=A.J. McCoy; citation_author=R.W. Grosse-Kunstleve; citation_author=P.D. Adams; citation_author=M.D. Winn; citation_author=L.C. Storoni; citation_author=R.J. Read; citation_journal_title=J.&nbsp;Appl. Cryst.; citation_volume=40; citation_firstpage=658; citation_lastpage=674; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Presenting your structures: the CCP4mg molecular-graphics software; citation_author=S. McNicholas; citation_author=E. Potterton; citation_author=K.S. Wilson; citation_author=M.E. Noble; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=67; citation_firstpage=386; citation_lastpage=394; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Discovery of novel 5-hydroxy-4-pyridone-3-carboxy acids as potent inhibitors of influenza Cap-dependent endonuclease; citation_author=M. Miyagawa; citation_author=T. Akiyama; citation_author=M. Mikamiyama-Iwata; citation_author=K. Hattori; citation_author=N. Kurihara; citation_author=Y. Taoda; citation_author=C. Takahashi-Kageyama; citation_author=N. Kurose; citation_author=H. Mikamiyama; citation_author=N. Suzuki; citation_author=et al.; citation_journal_title=Bioorg. Med. Chem. Lett.; citation_volume=26; citation_firstpage=4739; citation_lastpage=4742; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Refinement of macromolecular structures by the maximum-likelihood method; citation_author=G.N. Murshudov; citation_author=A.A. Vagin; citation_author=E.J. Dodson; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=53; citation_firstpage=240; citation_lastpage=255; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=NR‐grep: a fast and flexible pattern-matching tool; citation_author=G. Navarro; citation_journal_title=Softw. Pract. Exper.; citation_volume=31; citation_firstpage=1265; citation_lastpage=1312; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Multiprotein complex production in insect cells by using polyproteins; citation_author=Y. Nie; citation_author=I. Bellon-Echeverria; citation_author=S. Trowitzsch; citation_author=C. Bieniossek; citation_author=I. Berger; citation_journal_title=Methods Mol. Biol.; citation_volume=1091; citation_firstpage=131; citation_lastpage=141; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Ultrastructural and functional analyses of recombinant influenza virus ribonucleoproteins suggest dimerization of nucleoprotein during virus amplification; citation_author=J. Ortega; citation_author=J. Martín-Benito; citation_author=T. Zürcher; citation_author=J.M. Valpuesta; citation_author=J.L. Carrascosa; citation_author=J. Ortín; citation_journal_title=J.&nbsp;Virol.; citation_volume=74; citation_firstpage=156; citation_lastpage=163; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Structural insight into RNA synthesis by influenza D polymerase; citation_author=Q. Peng; citation_author=Y. Liu; citation_author=R. Peng; citation_author=M. Wang; citation_author=W. Yang; citation_author=H. Song; citation_author=Y. Chen; citation_author=S. Liu; citation_author=M. Han; citation_author=X. Zhang; citation_author=et al.; citation_journal_title=Nat. Microbiol.; citation_volume=4; citation_firstpage=1750; citation_lastpage=1759; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=UCSF Chimera--a visualization system for exploratory research and analysis; citation_author=E.F. Pettersen; citation_author=T.D. Goddard; citation_author=C.C. Huang; citation_author=G.S. Couch; citation_author=D.M. Greenblatt; citation_author=E.C. Meng; citation_author=T.E. Ferrin; citation_journal_title=J.&nbsp;Comput. Chem.; citation_volume=25; citation_firstpage=1605; citation_lastpage=1612; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Structure of influenza A polymerase bound to the viral RNA promoter; citation_author=A. Pflug; citation_author=D. Guilligay; citation_author=S. Reich; citation_author=S. Cusack; citation_journal_title=Nature; citation_volume=516; citation_firstpage=355; citation_lastpage=360; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Structural insights into RNA synthesis by the influenza virus transcription-replication machine; citation_author=A. Pflug; citation_author=M. Lukarska; citation_author=P. Resa-Infante; citation_author=S. Reich; citation_author=S. Cusack; citation_journal_title=Virus Res.; citation_volume=234; citation_firstpage=103; citation_lastpage=117; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Capped RNA primer binding to influenza polymerase and implications for the mechanism of cap-binding inhibitors; citation_author=A. Pflug; citation_author=S. Gaudon; citation_author=P. Resa-Infante; citation_author=M. Lethier; citation_author=S. Reich; citation_author=W.M. Schulze; citation_author=S. Cusack; citation_journal_title=Nucleic. Acids Res.; citation_volume=46; citation_firstpage=956; citation_lastpage=971; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A unique cap(m7GpppXm)-dependent influenza virion endonuclease cleaves capped RNAs to generate the primers that initiate viral RNA transcription; citation_author=S.J. Plotch; citation_author=M. Bouloy; citation_author=I. Ulmanen; citation_author=R.M. Krug; citation_journal_title=Cell; citation_volume=23; citation_firstpage=847; citation_lastpage=858; citation_publication_date=1981; "><meta name="citation_reference" content="citation_title=Direct evidence that the poly(A) tail of influenza A virus mRNA is synthesized by reiterative copying of a U track in the virion RNA template; citation_author=L.L. Poon; citation_author=D.C. Pritlove; citation_author=E. Fodor; citation_author=G.G. Brownlee; citation_journal_title=J.&nbsp;Virol.; citation_volume=73; citation_firstpage=3473; citation_lastpage=3476; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=RNA ligands activate the Machupo virus polymerase and guide promoter usage; citation_author=J.D. Pyle; citation_author=S.P.J. Whelan; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=10518; citation_lastpage=10524; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Structural insight into cap-snatching and RNA synthesis by influenza polymerase; citation_author=S. Reich; citation_author=D. Guilligay; citation_author=A. Pflug; citation_author=H. Malet; citation_author=I. Berger; citation_author=T. Crépin; citation_author=D. Hart; citation_author=T. Lunardi; citation_author=M. Nanao; citation_author=R.W. Ruigrok; citation_author=S. Cusack; citation_journal_title=Nature; citation_volume=516; citation_firstpage=361; citation_lastpage=366; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=An in&nbsp;vitro fluorescence based study of initiation of RNA synthesis by influenza B polymerase; citation_author=S. Reich; citation_author=D. Guilligay; citation_author=S. Cusack; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_firstpage=3353; citation_lastpage=3368; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Extreme heterogeneity of influenza virus infection in single cells; citation_author=A.B. Russell; citation_author=C. Trapnell; citation_author=J.D. Bloom; citation_journal_title=eLife; citation_volume=7; citation_firstpage=e32303; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=RELION: implementation of a Bayesian approach to cryo-EM structure determination; citation_author=S.H. Scheres; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=180; citation_firstpage=519; citation_lastpage=530; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Influenza virus RNA polymerase: insights into the mechanisms of viral RNA synthesis; citation_author=A.J. Te Velthuis; citation_author=E. Fodor; citation_journal_title=Nat. Rev. Microbiol.; citation_volume=14; citation_firstpage=479; citation_lastpage=493; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Real-time cryo-electron microscopy data preprocessing with Warp; citation_author=D. Tegunov; citation_author=P. Cramer; citation_journal_title=Nat. Methods; citation_volume=16; citation_firstpage=1146; citation_lastpage=1152; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=STARANISO. Global Phasing Ltd.; citation_author=I.J. Tickle; citation_author=C. Flensburg; citation_author=P. Keller; citation_author=W. Paciorek; citation_author=A. Sharff; citation_author=C. Vonrhein; citation_author=G. Bricogne; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Biochemical characterization of the Lassa virus L protein; citation_author=D. Vogel; citation_author=M. Rosenthal; citation_author=N. Gogrefe; citation_author=S. Reindl; citation_author=S. Günther; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=294; citation_firstpage=8088; citation_lastpage=8100; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Model suggesting that replication of influenza virus is regulated by stabilization of replicative intermediates; citation_author=F.T. Vreede; citation_author=T.E. Jung; citation_author=G.G. Brownlee; citation_journal_title=J.&nbsp;Virol.; citation_volume=78; citation_firstpage=9568; citation_lastpage=9572; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=The mechanism by which influenza A virus nucleoprotein forms oligomers and binds RNA; citation_author=Q. Ye; citation_author=R.M. Krug; citation_author=Y.J. Tao; citation_journal_title=Nature; citation_volume=444; citation_firstpage=1078; citation_lastpage=1082; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Gctf: Real-time CTF determination and correction; citation_author=K. Zhang; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=193; citation_firstpage=1; citation_lastpage=12; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy; citation_author=S.Q. Zheng; citation_author=E. Palovcak; citation_author=J.P. Armache; citation_author=K.A. Verba; citation_author=Y. Cheng; citation_author=D.A. Agard; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=331; citation_lastpage=332; citation_publication_date=2017; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Influenza polymerase uses unique mechanisms to synthesize capped and polyadenylated mRNAs from the genomic viral RNA (vRNA) template, which is packaged inside ribonucleoprotein particles (vRNPs). Here, we visualize by cryoelectron microscopy the conformational dynamics of the polymerase during the complete transcription cycle from pre-initiation to termination, focusing on the template trajectory. After exiting the active site cavity, the template 3′ extremity rebinds into a specific site on the polymerase surface. Here, it remains sequestered during all subsequent transcription steps, forcing the template to loop out as it further translocates. At termination, the strained connection between the&nbsp;bound template 5′ end and the active site results in polyadenylation by stuttering at uridine 17. Upon product dissociation, further conformational changes release the trapped template, allowing recycling back into the pre-initiation state. Influenza polymerase thus performs transcription while tightly binding to and protecting both template ends, allowing efficient production of multiple mRNAs from a single vRNP.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=4976&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5&amp;be=2797&amp;fe=4859&amp;dc=4038&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749586373843,%22n%22:0,%22f%22:2163,%22dn%22:2163,%22dne%22:2163,%22c%22:2163,%22s%22:2163,%22ce%22:2163,%22rq%22:2164,%22rp%22:2777,%22rpe%22:3537,%22dl%22:2781,%22di%22:3936,%22ds%22:4037,%22de%22:4216,%22dc%22:4858,%22l%22:4859,%22le%22:4957%7D,%22navigation%22:%7B%7D%7D&amp;fp=3248&amp;fcp=3248&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style>stream {
  position: relative;
  display: block;
}
</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-msqrt {
  display: inline-block;
  text-align: left;
}

mjx-root {
  display: inline-block;
  white-space: nowrap;
}

mjx-surd {
  display: inline-block;
  vertical-align: top;
}

mjx-sqrt {
  display: inline-block;
  padding-top: .075em;
}

mjx-sqrt > mjx-box {
  border-top: .075em solid;
  padding-left: .03em;
  margin-left: -.03em;
}

mjx-sqrt.mjx-tall > mjx-box {
  padding-left: .3em;
  margin-left: -.3em;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D453 {
  padding: 0.705em 0.552em 0.205em 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D434 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-cB7 {
  padding: 0.303em 0.278em 0 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c1D458 {
  padding: 0.694em 0.521em 0.011em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D435 {
  padding: 0.683em 0.759em 0 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D436 {
  padding: 0.705em 0.76em 0.022em 0;
}

mjx-c.mjx-c1D44F {
  padding: 0.694em 0.429em 0.011em 0;
}

mjx-c.mjx-c1D439 {
  padding: 0.68em 0.749em 0 0;
}

mjx-c.mjx-c1D443 {
  padding: 0.683em 0.754em 0 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c28.MM-S4 {
  padding: 1.146em 0.597em 0.646em 0;
}

mjx-c.mjx-c1D43C {
  padding: 0.683em 0.497em 0 0;
}

mjx-c.mjx-c1D45C {
  padding: 0.442em 0.485em 0.011em 0;
}

mjx-c.mjx-c1D462 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c29.MM-S4 {
  padding: 1.146em 0.597em 0.646em 0;
}

mjx-c.mjx-c28.MM-S5 {
  padding: 1.296em 0.663em 0.796em 0;
}

mjx-c.mjx-c1D43F {
  padding: 0.683em 0.681em 0 0;
}

mjx-c.mjx-c1D43E {
  padding: 0.683em 0.889em 0 0;
}

mjx-c.mjx-c1D437 {
  padding: 0.683em 0.828em 0 0;
}

mjx-c.mjx-c221A.MM-LO {
  padding: 1.15em 1.02em 0.65em 0;
}

mjx-c.mjx-c34 {
  padding: 0.677em 0.5em 0 0;
}

mjx-c.mjx-c29.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930389-5" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)30389-5"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930389-5" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-34312" aria-expanded="false" aria-controls="login-links-34312" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-34312" id="login-links--34312" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930389-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930389-5" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930389-5" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-34312" data-opens="#links-dialog-34312" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-34312" aria-modal="true"><button data-closes="#links-dialog-34312" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930389-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930389-5" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930389-5" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.03.061">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(20)30389-5" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0011-2"><span>Volume 181</span>, <span>Issue 4</span></a></span><span class="meta-panel__pages"><span>p877-893.e21</span></span><span class="meta-panel__onlineDate">May 14, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=4&amp;pii=S0092-8674%2819%29X0011-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">A Structure-Based Model for the Complete Transcription Cycle of Influenza Polymerase</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Joanna M.</span> <span property="familyName">Wandzik</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joanna M.</span> <span property="familyName">Wandzik</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">3</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France</span></div></div><div class="core-author-link"><a href="/authored-by/Wandzik/Joanna+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">3</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Tomas</span> <span property="familyName">Kouba</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tomas</span> <span property="familyName">Kouba</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">3</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France</span></div></div><div class="core-author-link"><a href="/authored-by/Kouba/Tomas" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">3</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Manikandan</span> <span property="familyName">Karuppasamy</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Manikandan</span> <span property="familyName">Karuppasamy</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France</span></div></div><div class="core-author-link"><a href="/authored-by/Karuppasamy/Manikandan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Alexander</span> <span property="familyName">Pflug</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alexander</span> <span property="familyName">Pflug</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">4</div><div id="fn2"><div id="ntpara0015" role="paragraph">Present address: Astra-Zeneca, Cambridge CB2 0AA, UK</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France</span></div></div><div class="core-author-link"><a href="/authored-by/Pflug/Alexander" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Petra</span> <span property="familyName">Drncova</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Petra</span> <span property="familyName">Drncova</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France</span></div></div><div class="core-author-link"><a href="/authored-by/Drncova/Petra" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Jan</span> <span property="familyName">Provaznik</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jan</span> <span property="familyName">Provaznik</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, GeneCore, Meyerhofstraße 1, 69117 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Provaznik/Jan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Nayara</span> <span property="familyName">Azevedo</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nayara</span> <span property="familyName">Azevedo</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, GeneCore, Meyerhofstraße 1, 69117 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Azevedo/Nayara" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Stephen</span> <span property="familyName">Cusack</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stephen</span> <span property="familyName">Cusack</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:cusack@embl.fr" property="email" title="Send email to Stephen Cusack">cusack@embl.fr</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">5</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France</span></div></div><div class="core-author-link"><a href="/authored-by/Cusack/Stephen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">5</a></sup> <a id="eadd0010" href="mailto:cusack@embl.fr" property="email" title="Send email to Stephen Cusack">cusack@embl.fr</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">European Molecular Biology Laboratory, 71 Avenue des Martyrs, CS 90181, 38042 Grenoble Cedex 9, France</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">European Molecular Biology Laboratory, GeneCore, Meyerhofstraße 1, 69117 Heidelberg, Germany</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">3</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">4</div><div id="fn2"><div id="ntpara0015" role="paragraph">Present address: Astra-Zeneca, Cambridge CB2 0AA, UK</div></div></div><div role="doc-footnote" data-has="label"><div class="label">5</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received November 4, 2019</span>; <span>Revised February 18, 2020</span>; <span>Accepted March 26, 2020</span>; <span>Published online April 17, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2020.03.061" property="sameAs" target="_blank">10.1016/j.cell.2020.03.061 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867420303895" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2020.03.061" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0011-2"><img src="/cms/asset/9a6e91e6-87e4-4b2f-8401-172e1cadaf8e/cov200h.gif" alt="Cover Image - Cell, Volume 181, Issue 4"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2820%2930389-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (2)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930389-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.03.061">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2930389-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.03.061&amp;pii=S0092-8674%2820%2930389-5" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930389-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.03.061">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.03.061" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867420303895&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867420303895&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=4&amp;pii=S0092-8674%2819%29X0011-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)30483-9" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)30338-X" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental Information (2)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Cryo-EM snapshots of the transcription elongation, termination, and recycling states</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">After being copied, the template 3′ end rebinds the polymerase in a secondary site</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Mechanism of viral mRNA poly(A) tail formation by stuttering elucidated</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Efficient reformation of the promoter allows multiple transcripts from one RNP</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Influenza polymerase uses unique mechanisms to synthesize capped and polyadenylated mRNAs from the genomic viral RNA (vRNA) template, which is packaged inside ribonucleoprotein particles (vRNPs). Here, we visualize by cryoelectron microscopy the conformational dynamics of the polymerase during the complete transcription cycle from pre-initiation to termination, focusing on the template trajectory. After exiting the active site cavity, the template 3′ extremity rebinds into a specific site on the polymerase surface. Here, it remains sequestered during all subsequent transcription steps, forcing the template to loop out as it further translocates. At termination, the strained connection between the&nbsp;bound template 5′ end and the active site results in polyadenylation by stuttering at uridine 17. Upon product dissociation, further conformational changes release the trapped template, allowing recycling back into the pre-initiation state. Influenza polymerase thus performs transcription while tightly binding to and protecting both template ends, allowing efficient production of multiple mRNAs from a single vRNP.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/858f7de6-7315-44d2-ad3e-43a88b934cbc/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2020.03.061/asset/ed2cfa66-ca5f-465c-b9ad-b8b687a995c5/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/858f7de6-7315-44d2-ad3e-43a88b934cbc/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;Influenza+virus&quot;" alt="Search on this keyword" rel="nofollow">Influenza virus</a></li><li><a href="/action/doSearch?AllField=&quot;RNA-dependent+RNA+polymerase&quot;" alt="Search on this keyword" rel="nofollow">RNA-dependent RNA polymerase</a></li><li><a href="/action/doSearch?AllField=&quot;transcription&quot;" alt="Search on this keyword" rel="nofollow">transcription</a></li><li><a href="/action/doSearch?AllField=&quot;cap-snatching&quot;" alt="Search on this keyword" rel="nofollow">cap-snatching</a></li><li><a href="/action/doSearch?AllField=&quot;poly-adenylation&quot;" alt="Search on this keyword" rel="nofollow">poly-adenylation</a></li><li><a href="/action/doSearch?AllField=&quot;secondary+3%E2%80%99&quot;" alt="Search on this keyword" rel="nofollow">secondary 3’</a></li><li><a href="/action/doSearch?AllField=&quot;end+binding+site&quot;" alt="Search on this keyword" rel="nofollow">end binding site</a></li><li><a href="/action/doSearch?AllField=&quot;cryo-electron+microscopy&quot;" alt="Search on this keyword" rel="nofollow">cryo-electron microscopy</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">The heterotrimeric influenza polymerase, with subunits PA, PB1,&nbsp;and PB2, binds to the conserved 5′ and 3′ termini, the “promoter,” of each of the eight negative sense, single-stranded viral RNA (vRNA) genome segments, which are packaged by the viral nucleoprotein (NP) into ribonucleoprotein particles (vRNPs). Following viral entry into the infected cell, vRNPs are transported to the nucleus where they associate with transcribing Pol II and the viral polymerase performs “primary” transcription to generate viral mRNA leading to viral protein synthesis (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib5 bib32 bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Eisfeld et&nbsp;al., 2015; Pflug et&nbsp;al., 2017; Te Velthuis and Fodor, 2016</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Eisfeld, A.J. ∙ Neumann, G. ∙ Kawaoka, Y.</div><div><strong>At the centre: influenza A virus ribonucleoproteins</strong></div><div><em>Nat. Rev. Microbiol.</em> 2015; <strong>13</strong>:28-41</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrmicro3367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84924970829" target="_blank" aria-label="Open Scopus in new tab">Scopus (309)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25417656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrmicro3367&amp;pmid=25417656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Pflug, A. ∙ Lukarska, M. ∙ Resa-Infante, P. ...</div><div><strong>Structural insights into RNA synthesis by the influenza virus transcription-replication machine</strong></div><div><em>Virus Res.</em> 2017; <strong>234</strong>:103-117</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virusres.2017.01.013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85011796730" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28115197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virusres.2017.01.013&amp;pmid=28115197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Te Velthuis, A.J. ∙ Fodor, E.</div><div><strong>Influenza virus RNA polymerase: insights into the mechanisms of viral RNA synthesis</strong></div><div><em>Nat. Rev. Microbiol.</em> 2016; <strong>14</strong>:479-493</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrmicro.2016.87" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84978162055" target="_blank" aria-label="Open Scopus in new tab">Scopus (307)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27396566/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrmicro.2016.87&amp;pmid=27396566" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Influenza polymerase employs unique mechanisms to synthesize 5′ capped and 3′ poly-adenylated viral mRNA. Capped oligomers of 10–15 nt long, pirated from nascent Pol II transcripts, are used to prime transcription (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib20 bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Lukarska et&nbsp;al., 2017; Plotch et&nbsp;al., 1981</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Lukarska, M. ∙ Fournier, G. ∙ Pflug, A. ...</div><div><strong>Structural basis of an essential interaction between influenza polymerase and Pol II CTD</strong></div><div><em>Nature.</em> 2017; <strong>541</strong>:117-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature20594" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85016158967" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28002402/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature20594&amp;pmid=28002402" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Plotch, S.J. ∙ Bouloy, M. ∙ Ulmanen, I. ...</div><div><strong>A unique cap(m7GpppXm)-dependent influenza virion endonuclease cleaves capped RNAs to generate the primers that initiate viral RNA transcription</strong></div><div><em>Cell.</em> 1981; <strong>23</strong>:847-858</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2F0092-8674%2881%2990449-9&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2F0092-8674%2881%2990449-9&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0019394947" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6261960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2881%2990449-9&amp;pmid=6261960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Polyadenylation is achieved by stuttering of the polymerase on an oligo(U) stretch proximal to the template 5′ end (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Poon et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Poon, L.L. ∙ Pritlove, D.C. ∙ Fodor, E. ...</div><div><strong>Direct evidence that the poly(A) tail of influenza A virus mRNA is synthesized by reiterative copying of a U track in the virion RNA template</strong></div><div><em>J.&nbsp;Virol.</em> 1999; <strong>73</strong>:3473-3476</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.73.4.3473-3476.1999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10074205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.73.4.3473-3476.1999&amp;pmid=10074205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The same vRNPs perform unprimed replication of vRNA into complementary cRNA, which is only stable when packaged into cRNPs using newly synthesized polymerase and NP (<span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Vreede et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Vreede, F.T. ∙ Jung, T.E. ∙ Brownlee, G.G.</div><div><strong>Model suggesting that replication of influenza virus is regulated by stabilization of replicative intermediates</strong></div><div><em>J.&nbsp;Virol.</em> 2004; <strong>78</strong>:9568-9572</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.78.17.9568-9572.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-4143153932" target="_blank" aria-label="Open Scopus in new tab">Scopus (173)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15308750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.78.17.9568-9572.2004&amp;pmid=15308750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). cRNPs are the template for production of progeny vRNPs that can perform “secondary” transcription and/or be exported for packaging into new virions. Consistent with this, primary mRNA synthesis continues in cells infected in the presence of the translation inhibitor cycloheximide, but the stable synthesis of cRNA is inhibited due to the lack of NP and polymerase (<span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib9 bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Hay et&nbsp;al., 1980; Vreede et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Hay, A.J. ∙ Skehel, J.J. ∙ McCauley, J.</div><div><strong>Structure and synthesis of influenza virus complementary RNAs</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 1980; <strong>288</strong>:341-348</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.1980.0010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0019332259" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6103551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.1980.0010&amp;pmid=6103551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Vreede, F.T. ∙ Jung, T.E. ∙ Brownlee, G.G.</div><div><strong>Model suggesting that replication of influenza virus is regulated by stabilization of replicative intermediates</strong></div><div><em>J.&nbsp;Virol.</em> 2004; <strong>78</strong>:9568-9572</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.78.17.9568-9572.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-4143153932" target="_blank" aria-label="Open Scopus in new tab">Scopus (173)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15308750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.78.17.9568-9572.2004&amp;pmid=15308750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, a single primary vRNP can perform transcription without additional viral components (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Jorba et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Jorba, N. ∙ Coloma, R. ∙ Ortín, J.</div><div><strong>Genetic trans-complementation establishes a new model for influenza virus RNA transcription and replication</strong></div><div><em>PLoS Pathog.</em> 2009; <strong>5</strong>:e1000462</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1000462" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-67249108203" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19478885/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1000462&amp;pmid=19478885" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and continued accumulation of viral mRNA in the presence of cycloheximide (<span class="dropBlock reference-citations"><a id="body-ref-sref45-3" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-3" aria-controls="sref45-3">Vreede et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref45-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Vreede, F.T. ∙ Jung, T.E. ∙ Brownlee, G.G.</div><div><strong>Model suggesting that replication of influenza virus is regulated by stabilization of replicative intermediates</strong></div><div><em>J.&nbsp;Virol.</em> 2004; <strong>78</strong>:9568-9572</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.78.17.9568-9572.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-4143153932" target="_blank" aria-label="Open Scopus in new tab">Scopus (173)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15308750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.78.17.9568-9572.2004&amp;pmid=15308750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) shows that single vRNPs can perform multiple rounds of transcription. The implied efficient recycling mechanism is consistent with the need to rapidly generate large amounts of viral mRNA from the relatively small number of RNPs that initially infect the cell during a true infection (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Russell et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Russell, A.B. ∙ Trapnell, C. ∙ Bloom, J.D.</div><div><strong>Extreme heterogeneity of influenza virus infection in single cells</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e32303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.32303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85043531075" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29451492/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.32303&amp;pmid=29451492" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0035" role="paragraph">We have previously described how influenza polymerase accesses nascent host-capped transcripts by binding directly to the serine 5 phosphorylated CTD of Pol II (<span class="dropBlock reference-citations"><a id="body-ref-sref20-2" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-2" aria-controls="sref20-2">Lukarska et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref20-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Lukarska, M. ∙ Fournier, G. ∙ Pflug, A. ...</div><div><strong>Structural basis of an essential interaction between influenza polymerase and Pol II CTD</strong></div><div><em>Nature.</em> 2017; <strong>541</strong>:117-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature20594" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85016158967" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28002402/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature20594&amp;pmid=28002402" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), how rotation of the PB2 cap-binding domain allows the capped primer to be excised by the PA endonuclease and then directed into the polymerase active site to initiate RNA synthesis (<span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib14 bib33 bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Kouba et&nbsp;al., 2019; Pflug et&nbsp;al., 2018; Reich et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Pflug, A. ∙ Gaudon, S. ∙ Resa-Infante, P. ...</div><div><strong>Capped RNA primer binding to influenza polymerase and implications for the mechanism of cap-binding inhibitors</strong></div><div><em>Nucleic. Acids Res.</em> 2018; <strong>46</strong>:956-971</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85044658250" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29202182/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1210&amp;pmid=29202182" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Reich, S. ∙ Guilligay, D. ∙ Pflug, A. ...</div><div><strong>Structural insight into cap-snatching and RNA synthesis by influenza polymerase</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:361-366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84922245983" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14009&amp;pmid=25409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and revealed the sequence of conformational changes that accompany the transition from the initiation to elongation phase of transcription (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we present a&nbsp;detailed mechanistic model of the entire cycle of mRNA synthesis by influenza polymerase based on high resolution cryoelectron microscopy (cryo-EM) structures of actively transcribing bat influenza A (FluA) polymerase that cover the complete transcription cycle from pre-initiation, through elongation and termination, to product dissociation and template recycling. These structures give new insight into the template trajectory after passing through the active site, the mechanics of poly-adenylation, and how template recycling likely occurs. In particular, we find that after translocation through the active site, the 3′ extremity of the template docks into a partially buried binding site on the polymerase surface and remains sequestered there through termination and product dissociation. By tightly holding on to both extremities of the template throughout mRNA synthesis, the polymerase both protects the template from degradation and can efficiently recycle it, reform the promoter, and initiate the next cycle of transcription. The proposed model is fully compatible with transcription occurring, physiologically, in the RNP context, which necessitates NP uncoating and recoating of the incoming and outgoing template strands, respectively.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Transcription Elongation State of Bat FluA Polymerase in the Pre-catalytic State</h3><div id="p0040" role="paragraph">To obtain snapshots of successive states of actively transcribing polymerase, we developed strategies to stall the polymerase after different amounts of template translocation (<a id="crosref0110" href="#fig1">Figures 1</a>A and 1B). Previously we used an extended 18+3-mer template, capped 15-mer primer, and 14-mer 5′ vRNA activating “hook,” together with ATP and GTP, to stall influenza B (FluB) polymerase, through lack of CTP, after the addition of only five nucleotides to the primer (<span class="dropBlock reference-citations"><a id="body-ref-sref14-3" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-3" aria-controls="sref14-3">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we used bat FluA polymerase with the same RNAs but added first ATP and GTP, and subsequently CTP together with a non-hydrolysable UTP analog, UpNHpp (<a id="crosref0120" href="#fig1">Figure&nbsp;1</a>D). The template then translocates by 12 nt from the initiation state before stalling at the first adenine in the template sequence at position 15 from the native 3′ end of&nbsp;the template. At this point, the principle product is a capped 26-mer, as confirmed by next-generation sequencing of the transcription reaction products (<a id="crosref0125" href="#fig1">Figures 1</a>D and <a id="crosref0130" href="#figs6">S6</a>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/0ac709b2-2981-46f8-8362-cc2b1ee70a2d/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2020.03.061/asset/7b4120f8-32e4-4e50-b2aa-c641778655cf/main.assets/gr1.jpg" height="817" width="749" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0ac709b2-2981-46f8-8362-cc2b1ee70a2d/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Structural Snapshots of the Influenza Polymerase Transcription Cycle</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Sequence and secondary structure of nucleic acid moieties in each indicated complex with the designation of the template shown. The promoter vRNA 5′ (pink) and 3′ (yellow) ends are separate for the pre-initiation and elongation states, but connected for the v44 and v44-6U templates. The capped primer/product is blue with nucleotides numbered from the 5′ end, excluding the cap. The bracketed numbers on the 3′ vRNA indicate distance from the template 3′ end. Nucleotides not visible in the corresponding structure are light gray or indicated by a dotted line for the product dissociation structure.</div><div id="fspara0020" role="paragraph">(B) Atomic structure of the RNA moieties, represented as spheres and colored as in (A). The UpNHpp is red in the elongation complex, pyrophosphate is orange in the termination complexes, and magnesium ions are green.</div><div id="fspara0025" role="paragraph">(C) Overall structures of each bat influenza A polymerase complex in cartoon representation with color-code: PA (green), PB1 (blue), PB2/1-250 (PB2-N, red), PB1/β-ribbon up position (beige), PB1/β-ribbon-three-stranded sheet (light orange), PB2-cap binding domain (orange), PB2-midlink domain (purple), PB2-627 domain (salmon), and PB2-NLS (dark red). The RNA is colored as in (A). See also <a id="crosref0080c" href="#figs1">Figures S1</a>, <a id="crosref0080b" href="#figs2">S2</a>, and <a id="crosref0080a" href="#figs4">S4</a> and <a id="crosref1215" href="#mmc2">Table S1</a>.</div><div id="fspara0030" role="paragraph">(D) Elongation products formed using bat influenza A polymerase with the capped 15-mer primer with different combinations of ATP, GTP, CTP, and non-hydrolysable UTP analog (UpNHpp). Standards (RNAs with cap1) are indicated on the left and expected products on the right. The capped 26-mer product is formed with or without addition of UpNHpp, as expected, but longer products, generated by misincorporation (read-through) or primer realignment, are also present as indicated. The sample for cryo-EM was pre-incubated with ATP and GTP, subsequently CTP and UpNHpp were added to the reaction for 2&nbsp;min (lane 13) before freezing. This minimized unwanted products, which accumulate over longer time (lanes 14–17). The yellow box indicates the band whose equivalents were used for RNA sequencing.</div><div id="fspara0035" role="paragraph">(E) <i>In&nbsp;vitro</i> transcription assay using the 44-mer mini-vRNA template with disrupted (v44, left) or native 6xU polyadenylation signal (v44-6U, right) in the presence of the capped 13-mer primer, endonuclease inhibitor, and NTPs as indicated, shows synthesis of a single main product (v44) and polyadenylated products of heterogeneous length (v44-6U). The products migrate slower than expected due to robust hybridization with the complementary template (see text). The yellow boxes indicate bands whose equivalents were used for RNA sequencing.</div><div id="fspara0040" role="paragraph">See also <a id="crosref1220" href="#fig7">Figures 7</a>, <a id="crosref1225" href="#figs4">S4</a>B, and S4C and <a id="crosref1230" href="#mmc2">Table S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0045" role="paragraph">Cryo-EM analysis of this early elongation state confirms the biochemical characterization with the two major 3D classes clearly showing a template-product duplex with UpNHpp bound in a pre-catalytic configuration base-paired to A15 of the template at the&nbsp;+1 position (<a id="crosref0135" href="#fig1">Figures 1</a>B and <a id="crosref0140" href="#figs1">S1</a>; <a id="crosref0145" href="#mmc2">Table S1</a>; <a id="crosref0150" href="#mmc1">Method S1</a>B). In one 3D class, at 3.02&nbsp;Å resolution (PDB: <a id="intref0010" href="http://firstglance.jmol.org/fg.htm?mol6T0V" target="_blank">6T0V</a>), the cap-binding domain still associates with the polymerase core with the cap-proximal part of the primer bound to it. There is smeared density for the bulged-out part of the product mRNA connecting to the duplex. The second major 3D class is at 2.50&nbsp;Å resolution (PDB: <a id="intref0015" href="http://firstglance.jmol.org/fg.htm?mol6SZV" target="_blank">6SZV</a>) (<a id="crosref0155" href="#fig1">Figure&nbsp;1</a>B; <a id="crosref0160" href="#mmc1">Method S1</a>B), enabling ions, ribose hydroxyls, base substituents, and &gt;210 water molecules to be resolved. In this structure, there is no density for the cap-binding domain, and only the part of the product in the product-template duplex within the active site cavity is visible. Comparison of this elongation state with the structure of the bat FluA pre-initiation state (PDB: <a id="intref0020" href="http://firstglance.jmol.org/fg.htm?mol6T0N" target="_blank">6T0N</a>) (<a id="crosref0165" href="#fig1">Figure&nbsp;1</a>B; <a id="crosref0170" href="#mmc1">Method S1</a>A) shows that the initiation-elongation transition of bat FluA polymerase exhibits the same characteristic conformational changes as previously described for FluB polymerase (<span class="dropBlock reference-citations"><a id="body-ref-sref14-4" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-4" aria-controls="sref14-4">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), confirming that the mechanism is likely general for all influenza polymerases (<a id="crosref0180" href="#figs2">Figure&nbsp;S2</a>). These changes include (1) complete extrusion of the priming loop and opening of the template exit channel, (2) promoter collapse involving ∼28° rotation of the PB1 β-ribbon, (3) refolding of PB1/667-681 and formation of a three-strand sheet with the β-ribbon (<a id="crosref0185" href="#figs2">Figure&nbsp;S2</a>A), (4) outward rotation of the PB1 thumb domain and associated PB2-N domains by ∼5°, (5) displacement of PB2/37-44 to allow duplex growth, and (6) duplex strand-separation by stacking of PB2 lid domain Tyr205 and PB1/Arg706 on the last duplex base pair (<a id="crosref0190" href="#figs2">Figure&nbsp;S2</a>B). The only difference with FluB polymerase is that in FluA, promoter collapse is correlated with a refolding and shift of the PA arch residues 372–385, bringing a cluster of acidic residues PA/D375, E377, and D378 into contact with PB1/K188 and R203 on the PB1 β-ribbon (all cited residues conserved in FluA) (<a id="crosref0195" href="#figs2">Figure&nbsp;S2</a>C). The FluA elongation structure allows visualization at high resolution of the pre-catalytic state with incoming non-hydrolysable UpNHpp with both magnesium ions, Mg(A) and Mg(B), configured for catalysis (<a id="crosref0200" href="#figs3">Figure&nbsp;S3</a>A). Compared to the inactive configuration (<span class="dropBlock reference-citations"><a id="body-ref-sref14-5" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-5" aria-controls="sref14-5">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Mg(A) shifts by ∼5&nbsp;Å from the Mg(A′) position to coordinate the 3′ hydroxyl of the priming nucleotide and the α-phosphate of the incoming NTP, with concomitant rotamer changes of the motif A and C aspartates, but otherwise no major structural changes occur (<a id="crosref0210" href="#figs3">Figure&nbsp;S3</a>A, right). This structure, together with the high resolution post-catalytic state (see below, <a id="crosref0215" href="#figs3">Figure&nbsp;S3</a>B), provides details that will be useful for drug development targeting the RNA synthesis active site but will not be described further here.</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/99538f76-533c-4b80-8b42-c9fbb4deee0a/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2020.03.061/asset/2c14656f-15d9-4391-9ad7-4f6c0993041f/main.assets/figs1.jpg" height="867" width="678" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/99538f76-533c-4b80-8b42-c9fbb4deee0a/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Cryo-Electron Microscopy of Influenza Polymerase Conformations from Pre-initiation to Recycling States, Related to <a id="crosref1290" href="#fig1">Figure&nbsp;1</a> and <a id="crosref1295" href="#mmc1">Methods S1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0255" role="paragraph">A. Representative micrograph at 165,000 nominal magnification with defocus of ∼2.0&nbsp;μm (left) and two-dimensional (2D) classes of influenza polymerase complex (right). Data are from the pre-termination state structure determination.</div><div id="fspara0260" role="paragraph">B. Distribution of local resolution of the maps for pre-initiation (left), elongation (middle) and pre-termination complexes (right) and a slice through the polymerase core for each map. Local resolution maps were calculated with RELION and resolution range is indicated according to the color bar.</div><div id="fspara0265" role="paragraph">C. Fourier shell correlation (FSC) curves for all complexes described in this study. The plot of the FSC between two independently refined half-maps shows the overall resolution of the two maps as indicated by the gold standard FSC 0.143 cut-off criteria.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/79d6c09c-8f76-4d6b-8775-93115dfd8337/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2020.03.061/asset/2cce2733-2cce-4978-882c-10181f95129c/main.assets/figs2.jpg" height="439" width="500" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/79d6c09c-8f76-4d6b-8775-93115dfd8337/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Conformational Changes Underlying the Transition from the Pre-initiation to the Elongation States in Bat FluA Polymerase, Related to <a id="crosref1300" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0275" role="paragraph">A. The transition from pre-initiation (gray) to elongation complex (color) is marked by the collapse of the PB1/β-ribbon with formation of a three-stranded β sheet together with refolded peptide PB1/667-681; extrusion of the priming loop (ruby) and shift of PB2/37-44 peptide (red) as previously observed in the FluB elongation complex (<span class="dropBlock reference-citations"><a id="body-ref-sref14-6" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-6" aria-controls="sref14-6">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0280" role="paragraph">B. The collapse of the PB1/β-ribbon is accompanied by shift of PA/arch that forms new interactions between PB1/K188, R203 with PA/D378 and D375, respectively.</div><div id="fspara0285" role="paragraph">C. Residues involved in the RNA duplex strand separation are PB2/helical lid Tyr205 and PB1/Arg706, which stack on template (yellow) and product (blue) strands, respectively.</div><div id="fspara0290" role="paragraph">(D). SDS-PAGE of purification of trimeric bat FluA polymerase.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/6d0c38c2-940a-4144-baa2-521151b8700c/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2020.03.061/asset/ac064d5d-3fb7-4eff-b583-d37630525113/main.assets/figs3.jpg" height="457" width="499" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/6d0c38c2-940a-4144-baa2-521151b8700c/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Active Site in the Pre- and Post-catalytic States, Related to <a id="crosref1310" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0300" role="paragraph">A. CryoEM density for UpNHpp moiety (red sticks) stalled in the pre-catalytic state in the elongation complex (left) and RNA-protein contacts in the catalytic site (right) with conserved RNA-dependent RNA polymerase motifs colored accordingly: motif A (green), B (dark gray), C (pink), D (cyan), E (orange) and F (yellow) and RNA duplex between vRNA (yellow) and product strand (blue) with magnesium ions (green spheres) and water molecules (red spheres). Compared to the inactive configuration previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref14-7" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-7" aria-controls="sref14-7">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Mg(A’) (gray sphere coordinated by gray residues) shifts by ∼5&nbsp;Å (arrow) with repositioning of D446.</div><div id="fspara0305" role="paragraph">B. CryoEM density for A37 incorporated into the product strand (blue) and leaving pyrophosphate (orange) in the post-catalytic state in the pre-termination complex (left) and RNA-protein contacts color coded as described in A (right).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Template Exit Pathway</h3><div id="p0050" role="paragraph">Compared to the previously described FluB early elongation state, in the FluA UpNHpp structure, the template has translocated an additional 7 nt beyond the end of the product-template duplex. Four of these are reasonably well-resolved in the cryo-EM map, thus revealing the template exit path (<a id="crosref0220" href="#fig2">Figures 2</a>A and 2B). The exiting template runs in a narrow channel between residues 680-697 of PB1 helix α20 on one side, and the base of the extruded priming loop (PB1/632-634) and the PB2-N1 domain (PB2/54-110) on the other side. The floor of the groove is formed by extended peptide PB1/671-674 with basic residues K669, R670, and R672 contributing to its electro-positivity. PB1/His633 stacks on U4, PB2/Arg88 is close to the phosphates of U4 and G3, and PB1/Arg687 is in proximity to the phosphates of C2 and U1. Prior to the transition to elongation, the exit groove does not exist (<a id="crosref0225" href="#fig2">Figures 2</a>A–2C). It is blocked first by the unextruded priming loop and second by the PB2/80-90 loop of the PB2-N1 domain, which is sandwiched between the PB1/Arg670-Glu686 salt-bridge and PB1/Arg687 (<a id="crosref0230" href="#fig2">Figures 2</a>B and 2C). Its opening requires the concerted rigid-body movement of the thumb and PB2-N1 domains that accompanies the transition to elongation as well as an additional outward movement of the tip of the PB2/80-90 loop. This correlates with a reconfiguration of the PB1/671-674 peptide with change of salt-bridge partner of Glu686 from Arg670 to Arg672 (<a id="crosref0235" href="#fig2">Figure&nbsp;2</a>B).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/c5b735b3-464f-463b-bd9d-8f3540c4cf67/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2020.03.061/asset/435f6df9-9c91-4081-ad89-8be7c17652c0/main.assets/gr2.jpg" height="873" width="678" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/c5b735b3-464f-463b-bd9d-8f3540c4cf67/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">The Elongation and Pre-termination Complexes Reveal the Trajectory of the Exiting Template</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0050" role="paragraph">(A) In the pre-initiation state (pale colors, top left), the fully ordered priming loop is in the active site cavity and together with the loop PB2-N1/80-90, blocks the template exit channel. In the elongation state (bright colors, bottom left and its superposition with the pre-initiation, right), the extrusion of the priming loop (1) and displacement of the PB2-N1 domain (2) open the template exit groove between PB2-N1 and PB1/α20 through which the template 3′ end navigates (yellow). The transition from pre-initiation to elongation also involved collapse of PB1/β-ribbon (3) and refolding of the PB1/667-681 peptide that forms a three-stranded sheet together with the β-ribbon (4). See also <a id="crosref1235" href="#figs2">Figure&nbsp;S2</a>.</div><div id="fspara0055" role="paragraph">(B) Comparison of the template exit channel in the pre-initiation (left), elongation (middle), and pre-termination states (right). The floor of the exit channel is lined with positively charged residues (PB1/R670, R672, and R687), which together with PB2/K82 and R88 accommodate the emerging RNA template. In the pre-initiation to elongation transition, PB1/E686 exchanges its partner from PB1/R670 to R672. PB1/H633 changes orientation to stack on a base of the emerging template. In the pre-termination state, the RNA bases are disposed differently in the exit groove and the template curves around the PB2-N1 domain to bind in the secondary 3′ end binding site.</div><div id="fspara0060" role="paragraph">(C) Surface representation showing changes in the exit channel occurring from the pre-initiation to the product dissociation states. In the pre-initiation state, the template exit channel is blocked by the stem of the priming loop and close interaction of PB2-N1 and PB1/C-ter domains (black box). During the transition to elongation, the exit channel opens allowing the 3′ end of the template to emerge on the polymerase surface. As elongation progresses the 3′ end navigates into the partially buried secondary binding pocket (black box). In the product dissociation state, the template (dotted) remains in place in the widened exit channel (white arrow) but is disordered except in the secondary 3′ end binding pocket.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-3"><h3>Late Elongation and Termination States</h3><div id="p0055" role="paragraph">To determine the subsequent path of the template and to give insight into the mechanism of polyadenylation, we determined cryo-EM structures after initiating mRNA synthesis with a single-chain 44-mer mini-vRNA template in which the native 3′ and 5′ ends are connected by a loop. The template either had a 6U poly-adenylation (poly(A)) signal starting at the natural position 17 nt from the 5′ end (v44-6U template), which is expected to promote poly(A) tail formation by stuttering, or the 6U sequence was replaced by UGUGUA (v44 template) to produce a defined, non-polyadenylated product (<a id="crosref0240" href="#fig1">Figure&nbsp;1</a>A). The template sequences were chosen so that the first adenine occurred at position 16 (v44-6U) or 17 (v44) from the 5′ end. For the v44-6U template, addition of all NTPs produced, as expected, a smear of polyadenylated products ranging in apparent size from around 50–200 nt (<a id="crosref0245" href="#fig1">Figure&nbsp;1</a>E). In the case of the v44 template, ATP, GTP, CTP, and non-hydrolysable UpNHpp were added with the expectation that transcription would stall with UpNHpp base pairing with A17 at the&nbsp;+1 position, resulting in a capped 37-mer as main product (<a id="crosref0250" href="#fig1">Figure&nbsp;1</a>A). Addition of all NTPs should increase the product size to a 38-mer, resulting from incorporation of U opposite template position A17. In fact, the major products appear to migrate at a size of around 55–60 nt (<a id="crosref0255" href="#fig1">Figure&nbsp;1</a>E). Loading and running of control RNAs or transcription products on denaturing PAGE at different temperatures (<a id="crosref0260" href="#figs4">Figure&nbsp;S4</a>B) clearly shows that the aberrant mobility is due to formation of a high-melting point product-template duplex (<a id="crosref0265" href="#figs4">Figure&nbsp;S4</a>C). To further confirm the identity of the&nbsp;capped transcription products, we performed, after decapping, next-generation sequencing of the main product bands of the 18+3-mer and two v44 template transcription reactions (<a id="crosref0270" href="#fig1">Figures 1</a>D, 1E, boxed, and <a id="crosref0275" href="#figs6">S6</a>; <a id="crosref0280" href="#mmc2">Table S3</a>). We find that of the reads that start with the primer sequence, around 60% end at&nbsp;the expected position with a minority of these exhibiting primer&nbsp;realignment resulting in one or two repeats of the AGC at the primer 3′ end (<a id="crosref0285" href="#figs6">Figure&nbsp;S6</a>D). The isolated band from the v44 reactions also contains approximately 1:1 ratios of the template RNA (<a id="crosref0290" href="#mmc2">Table S3</a>) supporting that it is hybridized to the product in the gel. The combined evidence of the PAGE migration analysis, next-generation sequencing and the sequence register deduced from the cryo-EM density (<a id="crosref0295" href="#figs6">Figures S6</a>A–S6C), thus unambiguously identify the observed transcription products. Interestingly, an apparent larger size product was previously observed for related viral transcription/replication systems (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib36 bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Pyle and Whelan, 2019; Vogel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Pyle, J.D. ∙ Whelan, S.P.J.</div><div><strong>RNA ligands activate the Machupo virus polymerase and guide promoter usage</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:10518-10524</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1900790116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066099874" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31072925/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1900790116&amp;pmid=31072925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Vogel, D. ∙ Rosenthal, M. ∙ Gogrefe, N. ...</div><div><strong>Biochemical characterization of the Lassa virus L protein</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2019; <strong>294</strong>:8088-8100</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.RA118.006973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066070162" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30926610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.RA118.006973&amp;pmid=30926610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and our observations now provide a plausible explanation of this phenomenon.</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/9faf2324-3b04-4727-aa7c-705eedade1ad/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2020.03.061/asset/dcc49d79-b328-4d1c-9b87-2ee3b9aaf700/main.assets/figs4.jpg" height="518" width="762" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/9faf2324-3b04-4727-aa7c-705eedade1ad/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text"><i>In&nbsp;Vitro</i> Transcription by Bat FluA Polymerase Mini-vRNA Templates, Related to <a id="crosref1320" href="#fig1">Figures 1</a>D, 1E, and <a id="crosref1325" href="#figs6">S6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0315" role="paragraph">A. Analysis of the transcription products formed using the v44 template, with disrupted polyadenylation signal, in the presence of endonuclease inhibitor ENDO483 and with addition of selected nucleotides as marked. Misincorporation and run-off transcription products are indicated.</div><div id="fspara0320" role="paragraph">B. Urea-PAGE migration pattern analysis. Three identical gels were loaded and run at different temperatures: on ice (∼4°C, left), at room temperature (middle) and with hot running buffer (∼95°C, right). Synthetic 38-mer (RNA 38, lane 1) and its reverse complement 38-mer (RNA 38-RC, lane 2) individually migrate at the expected size. Their mixture makes a stable RNA duplex (dsRNA-38) with high melting temperature (Tm&nbsp;= 92.8°C, see C). At temperatures low enough that dsRNA-38 is a stable duplex, an apparent double sized product (∼80-mer) is observed (left panel, lane 3, red box) even in conditions of 7&nbsp;M urea PAGE. In contrast, at medium and high temperatures, dsRNA-38 is denatured and migrates as the individual ssRNA components (middle/right panels, lane 3, black box). The 44-mer template RNA (v44) alone migrates at the expected size (all panels, lane 4). The expected 37-mer (without UTP) or 38-mer (with UTP) capped transcription products can form fully complementary RNA duplexes of 27 or 28 base-pairs with the v44 template, with 10 nts primer and 17 or 16 nts template 5′ overhangs, respectively (see C). As for the mixture of the complementary 38-mer control RNA, the product-template hybrids have high melting temperatures (97.8 and 98.3°C, respectively) and are retarded at apparent double the expected size at low and medium temperatures (left/middle panels, compare lane 3, red box and 5 and 7, red star). There is a shift in migration of the product-template hybrid when the product is de-capped (all panels, compare lane 5 with 6 and 7 with 8). At high temperature, the product-template hybrid is melted (right panel, lanes 5-8) and the 37- and 38-mer transcription products migrate in accordance with their sizes. A minor run-off transcription product (54-mer) is present only for the sample where all NTPs were included (right panel, lane 7-8, green star). The capped and de-capped transcription products of the v3′ 18+3 RNA template were also visualized (middle panel, lanes A-B), the main product is the expected 26-mer (red circle) with minor products due to misincorporation/runoff (30-mer, black circle) and probable re-alignment products (green circle).</div><div id="fspara0325" role="paragraph">C. Melting temperatures for dsRNAs calculated using the UNAFold server (<span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Markham and Zuker, 2008</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Markham, N.R. ∙ Zuker, M.</div><div><strong>UNAFold: software for nucleic acid folding and hybridization</strong></div><div><em>Methods Mol. Biol.</em> 2008; <strong>453</strong>:3-31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-60327-429-6_1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84934438960" target="_blank" aria-label="Open Scopus in new tab">Scopus (847)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18712296/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-60327-429-6_1&amp;pmid=18712296" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) show strong hybridization for RNA-38 and its reverse complement and for v44 template RNA and its respective transcription product.</div><div id="fspara0330" role="paragraph">D. Comparison of the polyadenylation signal sequence and position in the three mini-vRNA templates used in this study.</div><div id="fspara0335" role="paragraph">E., F. Comparison of <i>in&nbsp;vitro</i> transcription products using the v44-6U (native polyadenylation signal) or v45-6U (polyadenylation signal shifted one nucleotide away from the 5′ end by insertion of C17) templates shows that U17 is essential for polyadenylation.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-4"><h3>Transcription of the v44 Template Reveals a Secondary 3′ End Binding Site</h3><div id="p0060" role="paragraph">The sample with the v44 template reacted with ATP, GTP, CTP, and non-hydrolysable UpNHpp produced a major 3D class at&nbsp;2.41&nbsp;Å resolution, denoted “pre-termination” state (PDB: <a id="intref0025" href="http://firstglance.jmol.org/fg.htm?mol6SZU" target="_blank">6SZU</a>) (<a id="crosref0300" href="#figs1">Figure&nbsp;S1</a>; <a id="crosref0305" href="#mmc2">Table S1</a>; <a id="crosref0310" href="#mmc1">Methods S1</a>C). This is a pre-translocation product complex with U18 at the&nbsp;+1 position base-pairing with A37 incorporated into the capped 37-mer product, together with pyrophosphate (<a id="crosref0315" href="#fig1">Figures 1</a>A, 1B, <a id="crosref0320" href="#figs3">S3</a>B, <a id="crosref0325" href="#figs5">S5</a>, and <a id="crosref0330" href="#figs6">S6</a>A; <a id="crosref0335" href="#mmc3">Video S1</a>, part 1). However, we expected to observe UpNHpp opposite template A17 at the&nbsp;+1 position. This suggests that placing A17 in the&nbsp;+1 position is unfavorable, probably due to strain in the connection to the firmly bound 5′ hook, but it can occur transiently, because when UTP is used instead of UpNHpp, an extra base is incorporated to form the capped 38-mer product (see above, <a id="crosref0340" href="#fig1">Figures 1</a>E, <a id="crosref0345" href="#figs4">S4</a>A, and S4B). Indeed, cryo-EM analysis of the v44 template incubated with all NTPs, visualized at 2.70&nbsp;Å resolution, unambiguously shows U38 of the product at the&nbsp;+1 position opposite A17 in a complex we denote “termination” state (PDB: <a id="intref0030" href="http://firstglance.jmol.org/fg.htm?mol6TW1" target="_blank">6TW1</a>) (<a id="crosref0350" href="#figs6">Figure&nbsp;S6</a>B; <a id="crosref0355" href="#mmc1">Methods S1</a>D). These observations are discussed below in the context of the mechanism of polyadenylation.</div><div id="p0065" role="paragraph"><div id="mmc3" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc3" data-type="mp4"><span id="video_stream_mmc3_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2ZjBhNGJmMzI1MTgxNDYyNzIyZGJiOTllZjAxZWY3MCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3OTc3fQ.o-VbfVe55Ku30E8dPSVeaU_cIBQ6UKpgnIxIMUz0yM76jKs5_4vtkMLU2d1cVl7KXvJ5e2Wd4VGmDSa97UqxEyZ-VLw8fBTpAqTXuDHlKCgwVuGXlC5xMdoBC78qfacBpn0hkdkhYYlJl71MBYF_XyTX1_IDoMWsXxIWiNyMJseG6QW1PAqMzFmpubGep1B1hRlZhjvuXxXrmTN0Obcp_ccET0pJH4FyEz4xnEaNfxfJExHFYMkHAaiz-D_UVbIlwUy-OB1DAI2i1-RKv0Ytk4Nnz7jbQNT5hDkUpgGRhsp7UtJYpBWPlpRkQ8KpDW-efiKtPOEsDPNOJ8IGJRrlXA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2ZjBhNGJmMzI1MTgxNDYyNzIyZGJiOTllZjAxZWY3MCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3OTc3fQ.o-VbfVe55Ku30E8dPSVeaU_cIBQ6UKpgnIxIMUz0yM76jKs5_4vtkMLU2d1cVl7KXvJ5e2Wd4VGmDSa97UqxEyZ-VLw8fBTpAqTXuDHlKCgwVuGXlC5xMdoBC78qfacBpn0hkdkhYYlJl71MBYF_XyTX1_IDoMWsXxIWiNyMJseG6QW1PAqMzFmpubGep1B1hRlZhjvuXxXrmTN0Obcp_ccET0pJH4FyEz4xnEaNfxfJExHFYMkHAaiz-D_UVbIlwUy-OB1DAI2i1-RKv0Ytk4Nnz7jbQNT5hDkUpgGRhsp7UtJYpBWPlpRkQ8KpDW-efiKtPOEsDPNOJ8IGJRrlXA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc3" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2ZjBhNGJmMzI1MTgxNDYyNzIyZGJiOTllZjAxZWY3MCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3OTc3fQ.o-VbfVe55Ku30E8dPSVeaU_cIBQ6UKpgnIxIMUz0yM76jKs5_4vtkMLU2d1cVl7KXvJ5e2Wd4VGmDSa97UqxEyZ-VLw8fBTpAqTXuDHlKCgwVuGXlC5xMdoBC78qfacBpn0hkdkhYYlJl71MBYF_XyTX1_IDoMWsXxIWiNyMJseG6QW1PAqMzFmpubGep1B1hRlZhjvuXxXrmTN0Obcp_ccET0pJH4FyEz4xnEaNfxfJExHFYMkHAaiz-D_UVbIlwUy-OB1DAI2i1-RKv0Ytk4Nnz7jbQNT5hDkUpgGRhsp7UtJYpBWPlpRkQ8KpDW-efiKtPOEsDPNOJ8IGJRrlXA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.03.061/attachment/116179af-2c5e-4326-88d3-c0ad91f07200/mmc3.mp4" download="mmc3.mp4" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Video (151.07 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Video S1. Conformational Dynamics of Influenza during Transcription, Related to Figures 1–7</div><div id="espara0010" role="paragraph">The pre-termination structure (part 1). Conformational transitions between the pre-initiation, elongation, pre-termination and product-dissociation states (part 2).</div></div></div></div><div id="p0070" role="paragraph">In the pre-termination state structure, there is excellent density&nbsp;for almost the entire 44-mer template RNA, allowing unambiguous assignment of the complete sequence (<a id="crosref0360" href="#figs5">Figures S5</a> and <a id="crosref0365" href="#figs6">S6</a>A). Furthermore, the catalytic metal ions Mg(A) and Mg(B) are clearly visible (<a id="crosref0035a" href="#figs3">Figures S3</a>B and <a id="crosref0370" href="#figs5">S5</a>C) as well as ∼275 ordered water molecules, mainly hydrating the protein-RNA interface (<a id="crosref0375" href="#figs5">Figure&nbsp;S5</a>D). The pre-termination state corresponds to translocation of the template by 24 nt (i.e., capped 13-mer primer extended to a 37-mer), with the template 3′ end 27 nt from the&nbsp;+1 active site position. The important new features of this structure are, first, the well-ordered direct connection of the template from the 5′ hook to the active site, and second, the outgoing template is observed to progress along the exit channel and bend around the PB2-N1 domain, with the five 3′ terminal nucleotides binding into a partially buried groove between the PB1 thumb and PA-C domains, which we call the “secondary 3′ end binding site.”</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/216fa9b9-2075-49fd-bef9-2a9932aadffa/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2020.03.061/asset/cda010d9-4a06-4678-92ed-fd90714706a4/main.assets/figs5.jpg" height="740" width="504" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/216fa9b9-2075-49fd-bef9-2a9932aadffa/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Cryo-EM Density for the Pre-termination State, Related to <a id="crosref1335" href="#fig3">Figures 3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0345" role="paragraph">A. Fit of the entire 44-mer template (v44) into the amplitude scaled (LocScale) cryoEM map of the pre-termination state.</div><div id="fspara0350" role="paragraph">B. Fit of the nts A11-A17 connecting the 5′-hook with the active site&nbsp;+1 position into the cryoEM density in the pre-termination state filtered at 2.41&nbsp;Å resolution.</div><div id="fspara0355" role="paragraph">C. Fit of the RNA duplex between template (yellow) and product (blue) strands into the cryoEM density in the pre-termination state filtered at 2.41&nbsp;Å resolution.</div><div id="fspara0360" role="paragraph">D. Comparison of water molecules (red spheres) contacting vRNA nucleotide U6 in the pre-initiation (left), elongation (middle) and pre-termination (right) state in cryoEM maps (filtered to 2.54&nbsp;Å, 2.50&nbsp;Å and 2.41&nbsp;Å, respectively) show their conserved positions in independent structure determinations.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/62429aee-c0d9-4c56-b3aa-0233b6a00b07/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2020.03.061/asset/f0d4d02f-81c3-45f6-b916-9b8e38c8bcd5/main.assets/figs6.jpg" height="949" width="645" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/62429aee-c0d9-4c56-b3aa-0233b6a00b07/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Verification of the Sequences in the <i>In&nbsp;Vitro</i> Transcription Products by Cryo-EM and RNA Sequencing, Related to <a id="crosref1340" href="#fig5">Figure&nbsp;5</a> and <a id="crosref1345" href="#mmc2">Table S3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0370" role="paragraph">A–C. CryoEM density for the template-product duplex in the pre-termination (A), the termination (B) and the stuttering (C) structures shows the expected sequences.</div><div id="fspara0375" role="paragraph">D. RNA sequencing results for the products of the <i>in&nbsp;vitro</i> transcription reaction with the v44 and v3′ RNA 18+3 templates on gel extracted material as indicated on <a id="crosref1350" href="#fig1">Figures 1</a>D and 1E (yellow box). Bioinformatics analysis was performed on all reads starting with the primer sequence and dividing all products into mutually exclusive categories by applying the appropriate search patterns (<a id="crosref1355" href="#mmc2">Table S3</a>). The table (top) summarizes the number of reads in each of the six indicated categories, which include full-length expected product (with and without re-alignment), aberrant products (with and without re-alignment), primer sequence and full-length run-off product. Of note is that ‘full-length run-off product’ was not observed in the v44&nbsp;+ ATP/GTP/CTP sample and as little as 38 reads were found in v44&nbsp;+ ATP/GTP/CTP/UTP sample indicating that 5′ hook release and run-off transcription is a negligible event under these conditions. A plot (bottom) shows a majority of products with the expected sequence including those that underwent re-alignment during initiation. Remaining products correspond to almost exclusively to aberrant transcription products.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3>Template Entrance Conformation in the Pre-termination and Termination States</h3><div id="p0075" role="paragraph">In the v44 pre-termination state, the template makes an almost direct connection between the 5′ hook and the active site rather than the sinuous route taken by the 3′ end of the promoter at the start of transcription (<a id="crosref0380" href="#fig3">Figure&nbsp;3</a>A). The connector RNA has an irregular but highly ordered conformation that is stabilized by several highly conserved arginines, which both stack on bases and interact with phosphates (<a id="crosref0385" href="#fig3">Figures 3</a>B and <a id="crosref0390" href="#figs5">S5</a>B). The bases of A13 and C15 stack together and are sandwiched between PB1/Q186 and PB1/R203 of the collapsed β-ribbon on the A13 side and PB1/R353 on the C15 side. PB1/R353 also interacts with the phosphate of C15. The base of A16 stacks on the other side of PB1/R353 and also partially on the backbone of PB1/G33 as well as hydrogen bonding with PA/N508. The base of G14 is&nbsp;bulged out into a distinct pocket, stacking between the aliphatic part of PA/R507 and PB2/R38 and with the N7 hydrogen bonding to the main-chain of PB1/L675. PA/R507 also makes multivalent interactions with the phosphate of G14 and base and ribose of A11. PB1/R249 and R126 interact with the phosphates of A17 and U18, respectively, although the base of A17 is less well-ordered having no stacking interactions. The position of nt 1–14 from the 5′ end are the same as in the elongation structure, and similarly for nt 16–18. However, in the&nbsp;elongation structure, the 5′ end hook and the incoming template are not connected (mimicking the indirect linkage by a long loop of RNA for a physiological vRNA template), and in this case, the two segments of RNA are kept apart by a different position of&nbsp;the side chain of PB1/R353 (<a id="crosref0395" href="#fig3">Figure&nbsp;3</a>C). In the termination state, base A17 moves into the&nbsp;+1 position, maximally straightening the connection to the 5′ hook (<a id="crosref0400" href="#fig3">Figure&nbsp;3</a>D). C15 pushes PB1/R353 to a position similar to the elongation state and A16 relocates to pre-termination position of A17.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/f1832c02-460c-47f6-bb23-01724944011d/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2020.03.061/asset/5363c576-6ea3-4f60-ac91-26956886ac6b/main.assets/gr3.jpg" height="910" width="499" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/f1832c02-460c-47f6-bb23-01724944011d/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Structure of the Incoming Template in the Pre-termination and Termination States</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0070" role="paragraph">(A) Superposition of incoming RNA moieties in the pre-initiation complex (5′ vRNA in pink, 3′ vRNA in orange) with the pre-termination state (vRNA in yellow, mRNA product in blue, magnesium ions in green) shows the direct connection of the 5′ hook to the active site at pre-termination compared to the indirect path taken by 3′ vRNA in the pre-initiation state.</div><div id="fspara0075" role="paragraph">(B) Protein-RNA contacts stabilizing vRNA residues A13-G19 in the pre-termination state by interacting with the phosphate backbone (e.g., PA/R507, PB1/R126, R203, R249, R353, and R365), bases (e.g., PA/H505, N508 and PB1/Q186, and K229) or ribose moieties (e.g., PA/R507 and PB1/R353).</div><div id="fspara0080" role="paragraph">(C) Superposition of incoming RNA moieties in the pre-termination state (vRNA in yellow except for 5′ hook in pink, product mRNA in blue) with the elongation state (gray). Whereas in pre-termination the RNA is continuous, throughout elongation, the 5′ and 3′ ends are not directly connected (gray ends in inset). A switch in the orientation of PB1/R353 either stabilizes the direct connection in pre-termination (see C) or allows the RNA to loop out between the stably bound 5′ hook and template translocating into the active site.</div><div id="fspara0085" role="paragraph">(D) Comparison of the incoming template path in the pre-termination state (vRNA in yellow except for 5′ hook in pink, product mRNA in blue), with U18 at the&nbsp;+1 position, and the superposed termination state (gray), with A17 in the&nbsp;+1 position. The RNA conformation between G14 and A16 is straightened to allow A17 to reach the active site with a concomitant adjustment of PB1/R353 to allow C15 to reposition.</div><div id="fspara0090" role="paragraph">See also <a id="crosref1240" href="#figs5">Figures S5</a> and <a id="crosref1245" href="#figs6">S6</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-6"><h3>The Secondary 3′ End Binding Site</h3><div id="p0080" role="paragraph">In the pre-termination and termination states, the outgoing template is guided along a positively charged groove around the PB2-N1 domain and into a partially buried extended pocket between the PA-C and PB1 thumb domains where the 3′ extremity binds in a sequence-specific manner (<a id="crosref0405" href="#fig4">Figure&nbsp;4</a>). For longer templates, this implies that as transcription proceeds, the emerging template would have to bulge away from the protein while the 3′ extremity remains tightly bound in the secondary binding site. Indeed, in the termination state, there is no clear density for the nucleotides along the exit channel, probably due to disorder resulting from the extra base translocation of the template.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/c12315ac-294d-4bc6-b2ad-e2cb0d5e3943/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2020.03.061/asset/f1aafa48-73a4-43c9-a848-59368db99670/main.assets/gr4.jpg" height="891" width="679" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/c12315ac-294d-4bc6-b2ad-e2cb0d5e3943/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">The Secondary 3′ End Binding Site</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0100" role="paragraph">(A) Two views of the electrostatic surface of the pre-termination structure showing the vRNA (yellow) following the positively-charged template exit groove around the polymerase until the 3′ extremity docks in the secondary binding pocket (white box, right panel).</div><div id="fspara0105" role="paragraph">(B) Protein-RNA interactions contributing to the specific binding of the five 3′ terminal residues (yellow, numbering from 3′ end) in the secondary binding pocket formed at the interface of PA (green), PB1 (blue), and PB2 subunits (red) in bat influenza A polymerase. Multiple residues coordinate RNA backbone (e.g., PA/K483, PB1/K553, R560, R571, and PB2/K54) or interact specifically with RNA bases (e.g., PA/K351, T462, Q476, R485, R577, and PB2/M53).</div><div id="fspara0110" role="paragraph">(C) To accommodate the 3′ end RNA, three aromatic residues (PA/F352, Y459, and PB1/Y557) undergo induced fit rotations (as visualized by the transition from gray to colors) to stack with RNA bases (PA/F352 and PB1/Y557 with U4 and C2, respectively) and hydrogen bond with the RNA backbone (PA/Y459 with the ribose of C2 and PB1/Y557 with the phosphate of G2).</div><div id="fspara0115" role="paragraph">(D) Sequence alignment of selected regions of the PA, PB1, and PB2 subunits that contribute to the RNA-protein interactions in the secondary 3′ end binding site. Interacting residues that are strictly conserved in influenza A–D strains are highlighted in red, and those with similar chemical properties, in blue. Note that PA/F352 (green), stacking on vRNA base U4, is specific for bat polymerase. Residue numbering is according to bat FluA/H17N10 strain.</div><div id="fspara0120" role="paragraph">(E) The 3′ vRNA terminal nucleotide positions one to five bind sequence specifically to the 3′ secondary binding site with high affinity. A pyrimidine (Py)-purine (Pu) swap at positions one to four of the conserved 3′-U<sub>1</sub>C<sub>2</sub>Py<sub>3</sub>U<sub>4</sub>Pu<sub>5</sub> sequence consensus for vRNA/cRNA (see <a id="crosref1250" href="#figs7">Figure&nbsp;S7</a>D), or oligo(A) completely abolishes the binding. The graph shows binding curves of FAM-labeled WT (3′-UCGUCU, gray), mutant (3′-AGCACU, blue) and control (3′-AAAAAA, red) 6-mer RNAs measured by fluorescence polarization. Data shown is for n&nbsp;= 4 ± SD.</div><div id="fspara0125" role="paragraph">(F) Cryo-EM density for the 3′ end of the vRNA template in the pre-termination structure at 2.41&nbsp;Å resolution showing clear definition of each base.</div><div id="fspara0130" role="paragraph">See also <a id="crosref1255" href="#figs7">Figure&nbsp;S7</a>.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Comparison of the UpNHpp elongation and pre-termination structures shows that in the first section of the exit groove, bounded by the extruded priming loop, PB2-N1 domain, and PB1 helix α20, different sequences and conformations of RNA can be accommodated as expected for a non-specific translocation channel (<a id="crosref0410" href="#fig2">Figure&nbsp;2</a>B). The trajectory of the template follows the positively charged electrostatic surface of the groove that naturally guides the 3′ extremity into the secondary binding site (<a id="crosref0415" href="#fig4">Figure&nbsp;4</a>A). In this site, identical for both pre- and termination states, there are numerous interactions of PB1/553-571, PA/302, 351-352, 459-491, 577, and PB2/53-54 with the backbone and bases of the 5 terminal nucleotides (3′-UCGUC) (<a id="crosref0420" href="#fig4">Figure&nbsp;4</a>B). For example, the phosphates of nt 1–4 are contacted, respectively, by PB1/K553, PA/K483 and Y459, PB1/Y557 and R560, and PB1/Arg571 (<a id="crosref0425" href="#fig4">Figure&nbsp;4</a>B). Aromatics PB1/Y557, PA/F352, and PA/Y459 each undergo induced fit rotamer flips upon RNA binding with PB1/Y557 and PA/F352 stacking on bases 2 and 3 (<a id="crosref0430" href="#fig4">Figure&nbsp;4</a>C). Base-specific interactions are made, for example, by PA/R577 and T462 to C2 and PA/Q476 to U4. Most of the 3′ end interacting residues are absolutely conserved among all FluA strains (except bat FluA-specific PA/F352), and many residues are functionally conserved in other influenza types (<a id="crosref0435" href="#fig4">Figure&nbsp;4</a>D). Using fluorescence polarization to measure binding of labeled RNA 6-mers to bat FluA polymerase, we showed first that the wild-type 3′-UCGUCU sequence has a K<sub>d</sub> of ∼20&nbsp;nM whereas swapping purines for pyrimidines in the first four positions (3′-AGCACU) or an A<sub>6</sub> sequence had ∼100-fold less affinity (<a id="crosref0440" href="#fig4">Figures 4</a>E and <a id="crosref0445" href="#figs7">S7</a>A). To confirm the site of binding and the importance of various interacting residues, we showed that single mutations of key residues PA/K483, PB1/K553 and R560, and PB2/K54 reduced binding by 15- to 100-fold and double and triple mutants by 200-fold (<a id="crosref0450" href="#figs7">Figure&nbsp;S7</a>B), with no effect of the mutations on protein integrity as judged by the intrinsic RNA synthesis activity of the polymerase (<a id="crosref0455" href="#figs7">Figure&nbsp;S7</a>C) (<span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Reich et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Reich, S. ∙ Guilligay, D. ∙ Cusack, S.</div><div><strong>An in&nbsp;vitro fluorescence based study of initiation of RNA synthesis by influenza B polymerase</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:3353-3368</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28126917/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=28126917" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). By cryo-EM, we confirmed that the site and mode of binding is conserved in FluB polymerase (PDB: <a id="intref0035" href="http://firstglance.jmol.org/fg.htm?mol6T0W" target="_blank">6T0W</a>) (<a id="crosref0465" href="#figs7">Figure&nbsp;S7</a>E; <a id="crosref0470" href="#mmc2">Table S1</a>; <a id="crosref0475" href="#mmc1">Methods S1</a>G). Importantly, our high resolution maps show unambiguous density for vRNA 3′ end nt 1–5 bound specifically in the secondary binding site (<a id="crosref0480" href="#fig4">Figure&nbsp;4</a>F). In contrast, in a lower resolution structure of influenza D polymerase (<span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), vRNA nucleotides positions 2–6 were assigned to the binding site, even though in the corresponding map there is no density for the implied overhanging first nucleotide and the last visible ribose is orientated such that no prolongation is possible (as in the Bat FluA structure). Interestingly, other recent cryo-EM structures for influenza D (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), FluB, and A/H3N2 (<span class="dropBlock reference-citations"><a id="body-ref-sref7-1" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-1" aria-controls="sref7-1">Fan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref7-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Fan, H. ∙ Walker, A.P. ∙ Carrique, L. ...</div><div><strong>Structures of influenza A virus RNA polymerase offer insight into viral genome replication</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:287-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1530-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85072133233" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31485076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1530-7&amp;pmid=31485076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) polymerases, show that the cRNA 3′ end can also bind in this site. In all these cRNA bound structures, the secondary binding site nucleotides are assigned to positions 4–8 of the cRNA. However, we have high resolution cryo-EM maps showing unambiguous binding of cRNA nt 1–5, 3′-UCAUC, in the secondary binding site of bat A polymerase (unpublished data). In summary, our structural and biochemical results are consistent with the secondary binding site being specific for the conserved 3′ extremity of vRNA and&nbsp;cRNA for all influenza strains with consensus sequence 3′-U<sub>1</sub>C<sub>2</sub>A/G<sub>3</sub>U<sub>4</sub>C<sub>5</sub> (exceptionally FluD has an adenine at position 5 from the vRNA 3′ end) (<a id="crosref0500" href="#figs7">Figure&nbsp;S7</a>D). Finally, we note that the influenza polymerase secondary 3′ end binding site corresponds precisely to that observed for 3′ end vRNA binding to La Crosse bunyavirus (LACV) polymerase (<span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Gerlach et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Gerlach, P. ∙ Malet, H. ∙ Cusack, S. ...</div><div><strong>Structural Insights into Bunyavirus Replication and Its Regulation by the vRNA Promoter</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1267-1279</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84930486338" target="_blank" aria-label="Open Scopus in new tab">Scopus (139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26004069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.05.006&amp;pmid=26004069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0510" href="#figs7">Figure&nbsp;S7</a>F, left). For LACV, the binding is also both sequence-specific and&nbsp;with the extreme 3′ end ribose bound to preclude continuation. As discussed below, the sequestration of the 3′ end during active transcription now provides a rationale for the existence of&nbsp;this site.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/3b7aebf9-daa3-4220-af13-f913cd1dcc44/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2020.03.061/asset/2d2faf51-1b64-49ba-9a4a-f0fa8b2fb859/main.assets/figs7.jpg" height="948" width="729" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/3b7aebf9-daa3-4220-af13-f913cd1dcc44/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Validation of Specific Interactions in the vRNA 3′ End Secondary Binding Site and Conservation of Interacting Residues in Other Influenza Strains, Related to <a id="crosref1360" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0385" role="paragraph">A. Fluorescence polarization competition assay. The WT 3′ end 6-mer vRNA, but not a control oligo(A) 6-mer can compete FAM-labeled 6-mer vRNA from the 3′ secondary binding site, as shown by the decrease of fluorescent polarization (FP) signal. Data shown is for n&nbsp;= 2 ± SD.</div><div id="fspara0390" role="paragraph">B. Mutations in key residues interacting with the 3′ end in the secondary binding site (PA/K483E, PB1/K553E, Y557A, R560E and PB2/K54E) and their combinations reduce the binding of the WT FAM-labeled 3′ RNA (3′-UCGUCU-5′) to the apo FluA polymerase, as shown by the decrease of fluorescent polarization (FP) signal and measured K<sub>D</sub>’s. Data shown is for n&nbsp;= 4 ± SD.</div><div id="fspara0395" role="paragraph">C. Transcription activity test (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Reich et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Reich, S. ∙ Guilligay, D. ∙ Cusack, S.</div><div><strong>An in&nbsp;vitro fluorescence based study of initiation of RNA synthesis by influenza B polymerase</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:3353-3368</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28126917/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=28126917" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) on the purified secondary binding site mutants using an 18-mer vRNA template shows that the mutations do not affect the RNA synthesis activity on a short templates that does not reach the secondary site. Data shown is for n&nbsp;= 4 ± SD.</div><div id="fspara0400" role="paragraph">D. Conservation logos representing 3′ ends of vRNA (left) and cRNA (right) comparing all influenza genera. Data were generated with WebLogo online server (<span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Crooks et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Crooks, G.E. ∙ Hon, G. ∙ Chandonia, J.M. ...</div><div><strong>WebLogo: a sequence logo generator</strong></div><div><em>Genome Res.</em> 2004; <strong>14</strong>:1188-1190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.849004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-2142738304" target="_blank" aria-label="Open Scopus in new tab">Scopus (9365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15173120/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.849004&amp;pmid=15173120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and available sequences for the following strains: A/WSN/1933, A/little yellow-shouldered bat/Guatemala/060/2010, B/Lee/1940, C/Johannesburg/1/66 and D/swine/Oklahoma/1334/2011.</div><div id="fspara0405" role="paragraph">E. RNA-protein contacts in the secondary binding pocket in human FluB polymerase recycling complex (left) and overall view on the binding pocket (right). Note U1 is differently orientated than in the bat FluA complex (<a id="crosref1375" href="#fig4">Figure&nbsp;4</a>B). Color code: PA (green), PB1 (cyan) and PB2 (red).</div><div id="fspara0410" role="paragraph">F. Overall view of the 3′ end sequence specific secondary binding pocket in La Crosse virus L protein (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Gerlach et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Gerlach, P. ∙ Malet, H. ∙ Cusack, S. ...</div><div><strong>Structural Insights into Bunyavirus Replication and Its Regulation by the vRNA Promoter</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1267-1279</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84930486338" target="_blank" aria-label="Open Scopus in new tab">Scopus (139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26004069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.05.006&amp;pmid=26004069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the recycling bat FluA polymerase complex (this work) and mode B promoter binding of FluD polymerase (<span class="dropBlock reference-citations"><a id="body-ref-sref30-3" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-3" aria-controls="sref30-3">Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The LACV L protein is monomeric but colored as for influenza polymerase subunits to show the structural similarity. LACV has an additional structural element, the clamp (slate), which partially buries the 3′ end.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-7"><h3>The “Stuttering” State with the v44-6U Template Gives Insight into Polyadenylation</h3><div id="p0090" role="paragraph">A sample of the v44-6U template incubated with all NTPs was&nbsp;analyzed by cryo-EM. In this poly-adenylating sample, one of the 3D classes, at 3.04&nbsp;Å resolution (PDB: <a id="intref0040" href="http://firstglance.jmol.org/fg.htm?mol6T0S" target="_blank">6T0S</a>) (<a id="crosref0515" href="#mmc2">Table S1</a>; <a id="crosref0520" href="#mmc1">Methods S1</a>E) resembles the v44 template termination state, but, according to the base assignment in the duplex (<a id="crosref0525" href="#figs6">Figure&nbsp;S6</a>C), the polymerase has stalled after transcription of the U17-U22 polyadenylation signal, but prior to poly(A)-tail elongation (<a id="crosref0530" href="#fig1">Figures 1</a>A, 1B, <a id="crosref0535" href="#fig5">5A, and 5</a>B). The register of the product strand is the same as in the termination state but the template strand is distorted and the connection between 5′ end nt 13 and&nbsp;17 is not visible. We refer to this as the “stuttering” state. There is weak density for A38 incorporated into the product at the&nbsp;+1 position. However, the U17-A38 base pair is not formed. Instead, template base U17 is at the&nbsp;+2 position and U18 is at the&nbsp;+1 site forming a tilted base pair with A37 of the product (<a id="crosref0540" href="#fig5">Figure&nbsp;5</a>B). Indeed, the proximal part of the template-product duplex is stretched and tilted compared to that in the v44 termination state, before regaining the same geometry at the top of the duplex (<a id="crosref0545" href="#fig5">Figures 5</a>A and 5B). As in the termination state, the template in the exit channel is poorly ordered, whereas its extreme 3′ end is clearly bound in the secondary 3′ end binding site. We interpret the v44-6U stuttering structure as partially translocated. The distortion in the duplex backbone structure during translocation is accompanied by a concerted longitudinal shift in PB1 helix α8 276–292 and the preceding turn (PB1/271-275) as well as PB1/124-128, both of which abut the template backbone (<a id="crosref0550" href="#fig5">Figure&nbsp;5</a>C). Moreover, Mg(A) is observed in the shifted position, Mg(A′), coordinated by PB1/D446 and&nbsp;E491 (<a id="crosref0555" href="#fig5">Figure&nbsp;5</a>B). Based on these structures, a model for the mechanism of polyadenylation is proposed in the discussion (<a id="crosref0560" href="#fig5">Figure&nbsp;5</a>D).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/113f488f-30d3-4928-a7be-6daf11c4261c/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2020.03.061/asset/f0b6564a-b53d-4008-b17c-e8670d5831a9/main.assets/gr5.jpg" height="941" width="639" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/113f488f-30d3-4928-a7be-6daf11c4261c/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">The Polymerase Stutters on U17 to Produce Poly(A) Tails</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0140" role="paragraph">(A) Sequences of the disrupted (v44) and native polyadenylation signal (v44-6U) at position U17-U22 of the two 44-mer mini-vRNA templates used in the pre-termination/termination and stuttering structures, respectively.</div><div id="fspara0145" role="paragraph">(B) Structure of the template-product duplex in the pre-termination (left), termination (middle), and stuttering (right) complexes showing non-standard geometry at the bottom of the helix in the latter. In the pre-termination complex, regular base pairs are formed between template (yellow) and product strands (blue) with A37 at the&nbsp;+1 position base pairing with template U18. In the termination structure, the register shifts by one position forward placing the U38-A17 base pair at the&nbsp;+1 position. In the stuttering complex, the product strand is partially translocated and A38 occupies the&nbsp;+1 site without U17 opposite to it. At the same time U18 is pulled back to the&nbsp;+1 active site position while maintaining a base pair with A37, resulting in a tilted RNA duplex (compare green lines in left/middle and right panels).</div><div id="fspara0150" role="paragraph">(C) Superposition of the RNA duplexes in the stuttering (color) and termination states (gray) showing a concerted longitudinal shift in helix PB1/α8 and the loop PB1/124-128 on the opposite side of the duplex.</div><div id="fspara0155" role="paragraph">(D) The mechanism of polyadenylation with states visualized by structures as indicated. (1) U18 is the last template position that can be transcribed (product in blue) without tension on the template linker (A13-U17) connecting to the 5′ hook. The specific linker conformation is stabilized by multiple protein contacts such as the interaction with PB1/R353 (<a id="crosref1260" href="#fig3">Figure&nbsp;3</a>C). (2) The next translocation (visualized by movement of the penultimate brown base pair in the duplex) pulls (green arrow) U17 (pink) into the&nbsp;+1 position, forcing the A13-U17 linker to partially unfold from its highly structured configuration (dashed lines and pale colors). (3) The maximally stretched A13-U17 linker (as seen in termination structure, <a id="crosref1265" href="#fig3">Figure&nbsp;3</a>D) allows incorporation of ATP (black), which prolongs the oligo(A) stretch to 6 nt, complementary with the template. (4) Motif B PB1/Met410 holds the product in the&nbsp;+1 position (green arrow) but the strain in the A13-U17 template linker pulls U17 backward (red arrow), from the&nbsp;+1 to the&nbsp;+2 position causing the lower half of the RNA duplex to tilt. The stuttering structure visualizes this state which is still stabilized by nine regular base pairs, nevertheless the A13-U17 linker is disordered (dashed lines and pale colors). (5) The opposing forces on each strand of the duplex rupture the duplex with the product slipping by one base, extruding the base in position −9 out of the active site chamber (brown base moves from&nbsp;−9&nbsp;to&nbsp;−10). Slippage is facilitated by the maintenance of five A-U base pairs but three mismatches are created in the upper half of the duplex (red crosses). The product&nbsp;+1 position is then vacant for another round of ATP incorporation. (6) Base-pairing with incoming ATP (gray) pulls U17 transiently back into to the active site&nbsp;+1 position, and upon incorporation of another adenine (7) the system then recycles to state (4).</div><div id="fspara0160" role="paragraph">See also <a id="crosref1270" href="#figs4">Figures S4</a>D–S4F.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-8"><h3>Product Dissociation State</h3><div id="p0095" role="paragraph">The v44 pre-termination and v44-6U samples each have a&nbsp;second 3D class of interest at, respectively, 3.12 (PDB:&nbsp;<a id="intref0045" href="http://firstglance.jmol.org/fg.htm?mol6T0U" target="_blank">6T0U</a>) and 2.82&nbsp;Å (PDB: <a id="intref0050" href="http://firstglance.jmol.org/fg.htm?mol6T0R" target="_blank">6T0R</a>) resolution (<a id="crosref0565" href="#mmc2">Table S1</a>; <a id="crosref0570" href="#mmc1">Methods S1</a>C and&nbsp;S1E). A very similar class is also observed for the v44&nbsp;termination state (data not shown). All these structures clearly show the 5′ proximal part of the template entering the active site (e.g., up to U17 in the v44-6U case) and the&nbsp;extreme 3′ end of the template bound in the secondary binding site as in the termination and stuttering structures, but lack density for the product-template duplex in the active site cavity (<a id="crosref0575" href="#fig6">Figures 6</a>A and 6B). We&nbsp;interpret this as the “product-dissociation” state in which the three-stranded beta-sheet together with the clamped down β-ribbon remains&nbsp;formed, the priming loop is fully extruded, and the template stays threaded through the active site cavity, but the product mRNA has dissociated (<a id="crosref0580" href="#fig1">Figures 1</a>A, 1B, and <a id="crosref0585" href="#fig6">6</a>A). Due to the lack of a well-defined duplex imposing an ordered&nbsp;structure, there is only low resolution density for&nbsp;the template passing through the active site cavity and exit channel, but that in the secondary binding site is unambiguous (<a id="crosref0590" href="#fig6">Figure&nbsp;6</a>B).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/3ea1248a-2e02-4bf6-bb19-8d275be7f042/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2020.03.061/asset/82f00349-cfde-4abf-9516-02ad7270dd0d/main.assets/gr6.jpg" height="944" width="678" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/3ea1248a-2e02-4bf6-bb19-8d275be7f042/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Product Dissociation and Recycling States</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0170" role="paragraph">(A) Superposition of RNA moieties in the product dissociation (template in yellow with the 5′ hook in pink) and the pre-termination structures (template in gray and product in light blue). The position of the 5′ hook is unaltered, but the template path toward the active site cavity changes. The 3′ end of the template remains bound in the secondary binding site (bracketed residue numbering indicates the count from the 3′ end).</div><div id="fspara0175" role="paragraph">(B) In the product dissociation state, densities for 3′ (yellow) and 5′ (pink) termini of vRNA template are well defined indicating that the template extremities are still bound by the polymerase, but residues U18 to U38 (indicated by dashed line) are flexibly threaded through the active site cavity and disordered due to the lack of formation of a stable duplex with the product strand.</div><div id="fspara0180" role="paragraph">(C) The transition from the termination (gray and light blue for product strand) to the product dissociation/dislocated state (bright colors) entails a rotation of the PA-C domain (e.g., α14 and α17), the PB1 thumb domain (e.g., α16, PB1/520-536) and refolding of the PB1 peptide 500-513 adjacent to polymerase motif E (inset). The PB1/504-508 peptide is repositioned and becomes incompatible with the presence of RNA duplex (e.g., Met507 would clash with the product strand) and residues PB1/501-504 form a characteristic third strand with the motif E beta-hairpin (PB1/489-498).</div><div id="fspara0185" role="paragraph">(D) Cryo-EM structure of the recycling state for bat FluA (PDB: <a id="intref0335" href="http://firstglance.jmol.org/fg.htm?mol6T2C" target="_blank">6T2C</a>) corresponding to mode B vRNA promoter binding (<span class="dropBlock reference-citations"><a id="body-ref-sref30-4" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-4" aria-controls="sref30-4">Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this state, the 3′ end remains bound in the secondary binding pocket, but the viral promoter duplex is reformed.</div><div id="fspara0190" role="paragraph">(E) As (D) but for human FluB polymerase (PDB: <a id="interref360" href="http://firstglance.jmol.org/fg.htm?mol6T0W" target="_blank">6T0W</a>). See also <a id="crosref1280" href="#figs8">Figure&nbsp;S8</a>.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">A major difference between the product-dissociation state and the preceding elongation and termination states (which are globally similar) is a further rotation by ∼6°, but about a different axis, of the entire thumb domain (PB1/520-669) and rigidly associated regions of PA-C and PB2-N (PB2/43-250), while preserving template binding through the exit channel into the secondary 3′ end site (<a id="crosref0595" href="#fig6">Figure&nbsp;6</a>C). This we&nbsp;call the “dislocated” state of the polymerase, which is characterized by refolding of the PB1/500-513 peptide immediately after&nbsp;conserved polymerase motif E&nbsp;at&nbsp;the start of the&nbsp;thumb domain (<a id="crosref0600" href="#fig6">Figures 6</a>C and <a id="crosref0605" href="#figs8">S8</a>A–S8C). This peptide (500-GFVANFSMELPSFG-513) is highly conserved in all influenza polymerases (<a id="crosref0610" href="#figs8">Figure&nbsp;S8</a>B). In the&nbsp;dislocated state, a characteristic third beta-strand is formed by PB1/501-503 with the preceding motif E hairpin loop (<a id="crosref0615" href="#fig6">Figures 6</a>C and <a id="crosref0620" href="#figs8">S8</a>C). Furthermore, the new position of&nbsp;PB1/S506-M507-E508 would clash with the product strand&nbsp;in the duplex (<a id="crosref0625" href="#fig6">Figure&nbsp;6</a>C) and PB1/R126 and the PB1/521-536 helix α16 with the template strand, emphasizing that this state is unable to bind a duplex. The two major transitions that occur during transcription, from (pre-) initiation to elongation and from termination to product dissociation (i.e., dislocation), are complex, having distinct local and more global components, as shown in <a id="crosref0630" href="#figs8">Figure&nbsp;S8</a>D and <a id="crosref0635" href="#mmc3">Video S1</a> (part 2).</div><div class="figure-wrap"><figure id="figs8" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/0876b8a4-40e5-4619-9b61-452b8175ae27/main.assets/figs8_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs8" src="/cms/10.1016/j.cell.2020.03.061/asset/8abc2016-a3bc-498a-9dcd-75a66f95f673/main.assets/figs8.jpg" height="945" width="665" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0876b8a4-40e5-4619-9b61-452b8175ae27/main.assets/figs8_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs8-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S8</span> <span class="figure__title__text">Refolding of the Peptide PB1/500-513 and Conformational Transitions in the Polymerase Core as Transcription Proceeds, Related to <a id="crosref1390" href="#fig6">Figures 6</a> and <a id="crosref1395" href="#fig7">7</a>, <a id="crosref1400" href="#mmc2">Table S2</a>, and <a id="crosref1405" href="#mmc3">Video S1</a> (part 2)</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs8-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0420" role="paragraph">A. Fits of peptide PB1/500-513 in the cryoEM density in the pre-termination state (left) and product dissociation/dislocated state (right) shows different folding of this region.</div><div id="fspara0425" role="paragraph">B. Sequence alignment of PB1/500-513 region showing high conservation of this peptide in all influenza strains. Numbering according to the strain A/little yellow-shouldered bat/Guatemala/060/2010(H17N10).</div><div id="fspara0430" role="paragraph">C. Superposition of the RNA-dependent RNA polymerase motif E hairpin loop and peptide PB1/500-513 in the pre-termination (gray) and product dissociation (green) states with human H3N2 (active in blue and dislocated in beige) and FluD polymerase (red). Arrows indicate folds which correspond to closed-active and dislocated-inactive (with extra beta-strand) conformations of the polymerase. The FluD conformation is intermediate.</div><div id="fspara0435" role="paragraph">D. (Left) Superposition of the pre-initiation (light gray) and pre-termination (colors, PA: green, PB1: cyan, PB2: red) structures showing global changes in the protein conformation including the collapse of PB1/β-ribbon accompanied by rotation of PA/arch, extrusion of the priming loop and rotation of PB1/thumb and PB2-N1 domains. PB1/1-551 were superposed, excluding the β-ribbon, i.e., 476 residues aligned with root-mean-square deviation (RMSD) on Ca positons of 0.45&nbsp;Å. The β-ribbon (PB1/182-210) rotated by ∼28°, PB2-N1 (44-101, excluding the loop 80-87) by ∼3.8° and the thumb (PB1/560-670, excluding the priming loop 634-660), by ∼5.7°. (Middle) Superposition of the pre-termination (colors) and product dissociation/dislocated (dark gray) structures showing further conformational changes including correlated rotation of PA-C, PB1/thumb, PB1-PB2 interface and endonuclease domains. PB1/1-498 were superposed (excluding 274-300) i.e., 451 residues aligned with RMSD on Ca positons of 0.37&nbsp;Å. The thumb rotates by ∼6.8° and PB2/93-250 by ∼5.5°. (Right) Superposition of the pre-initiation (color) and product dissociation structures (dark gray) with modeled template (yellow) showing overall changes during the transcription cycle. PB1/1-496 were superposed, excluding the β-ribbon, i.e., 388 matched residues with RMSD on Ca positons of 0.41&nbsp;Å. The thumb rotates by ∼10.8°. For each structure, the complete pre-termination state template is superposed (yellow).</div><div id="fspara0440" role="paragraph">E. Surface representation of the bat FluA product-dissociation state showing template path modeled from the pre-termination state structure. The template is trapped inside a tunnel formed by the PB2-Nter/PB1-Cter helical bundle packing on the 170-loop of the PB2 helical lid domain.</div><div id="fspara0445" role="paragraph">F. Model based on the crystal structure of dimeric apo-H7N9 core showing one observed position of the PB2-Nter/PB1-Cter helical bundle consistent with release of template. The helical bundle swings out with hinges at PB1/670 and PB2/40 (consistent with the limits of visible density in the recycling structures in <a id="optlDTXR9lBZJ" href="#fig6">Figures 6</a>D and 6E) and would need to thread back through the template loop in ‘skipping rope’ fashion to fully free the template. The observed position of the PB2-Nter/PB1-Cter helical bundle is likely a crystallization artifact (for the second monomer in the dimer, it is in a different orientation) but serves to show that substantial relocation of the PB2-Nter/PB1-Cter helical bundle is possible. In several recently published cryoEM structures of the apo-dimer or mode B (recycling) state of A/H3N2 polymerase (PDB: <a id="intref0345" href="http://firstglance.jmol.org/fg.htm?mol6QWL" target="_blank">6QWL</a>, <a id="intref0350" href="http://firstglance.jmol.org/fg.htm?mol6QX3" target="_blank">6QX3</a> and <a id="intref0355" href="http://firstglance.jmol.org/fg.htm?mol6QX8" target="_blank">6QX8</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref7-2" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-2" aria-controls="sref7-2">Fan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref7-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Fan, H. ∙ Walker, A.P. ∙ Carrique, L. ...</div><div><strong>Structures of influenza A virus RNA polymerase offer insight into viral genome replication</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:287-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1530-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85072133233" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31485076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1530-7&amp;pmid=31485076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), PB1 beyond 670 and PB2 before 40 are not present in the cryoEM density, also indicating flexibility at these hinge points.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-9"><h3>Recycling State</h3><div id="p0105" role="paragraph">Interestingly, we and others have observed the dislocated state&nbsp;of the polymerase together with the vRNA template 3′ end bound in the secondary binding site in&nbsp;situations where the polymerase has never been actively transcribing. When either bat FluA or FluB resting polymerase-promoter complexes (i.e., in the absence of NTPs), with or without capped primer, are examined by cryo-EM, in addition to majority 3D classes corresponding to the un-dislocated, promoter-bound, pre-initiation state, there is always a small fraction of particles in the dislocated state, with the 3′ end of the template bound in the secondary binding site (PDB: <a id="intref0055" href="http://firstglance.jmol.org/fg.htm?mol6T2C" target="_blank">6T2C</a>) (<a id="crosref0640" href="#fig6">Figures 6</a>D and 6E; <a id="crosref0645" href="#mmc2">Table S1</a>; <a id="crosref0650" href="#mmc1">Methods S1</a>F). In these open and flexible structures, there is disorder in the C-terminal region of PB1 beyond 670 and the N-terminal region of PB2 up to residue 40 with the PB1-PB2 interface and endonuclease not being visible (this is also the case in recent A/H3N2 polymerase cryo-EM structures, e.g., PDB: <a id="intref0060" href="http://firstglance.jmol.org/fg.htm?mol6QWL" target="_blank">6QWL</a>, <a id="intref0065" href="http://firstglance.jmol.org/fg.htm?mol6QX3" target="_blank">6QX3</a>, and <a id="intref0070" href="http://firstglance.jmol.org/fg.htm?mol6QX8" target="_blank">6QX8</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref7-3" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-3" aria-controls="sref7-3">Fan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref7-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Fan, H. ∙ Walker, A.P. ∙ Carrique, L. ...</div><div><strong>Structures of influenza A virus RNA polymerase offer insight into viral genome replication</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:287-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1530-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85072133233" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31485076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1530-7&amp;pmid=31485076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, whereas the 5′ end side of the promoter duplex is reasonably well-defined, the 3′ end side is less so, and the connection to the secondary site is also poorly ordered. An exception is the recent FluD polymerase structure (PDB: <a id="intref0075" href="http://firstglance.jmol.org/fg.htm?mol6KUR" target="_blank">6KUR</a>), which is also dislocated (<a id="crosref0660" href="#figs8">Figure&nbsp;S8</a>C), but where the connection between the template extremity bound in the secondary site to the promoter duplex region is clearly seen (<a id="crosref0665" href="#figs7">Figure&nbsp;S7</a>F, right). The authors refer to this promoter configuration as mode B, distinct from mode A, when the 3′ end is in the active site as in the pre-initiation state (<span class="dropBlock reference-citations"><a id="body-ref-sref30-5" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-5" aria-controls="sref30-5">Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The RNA bound LACV structure (<span class="dropBlock reference-citations"><a id="body-ref-sref8-3" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-3" aria-controls="sref8-3">Gerlach et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Gerlach, P. ∙ Malet, H. ∙ Cusack, S. ...</div><div><strong>Structural Insights into Bunyavirus Replication and Its Regulation by the vRNA Promoter</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1267-1279</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84930486338" target="_blank" aria-label="Open Scopus in new tab">Scopus (139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26004069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.05.006&amp;pmid=26004069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) also exhibits mode B promoter binding (<a id="crosref0680" href="#figs7">Figure&nbsp;S7</a>F, left). Recent cryo-EM structures of dimeric (PDB: <a id="intref0080" href="http://firstglance.jmol.org/fg.htm?mol6QX8" target="_blank">6QX8</a>) and monomeric (PDB: <a id="intref0085" href="http://firstglance.jmol.org/fg.htm?mol6QX3" target="_blank">6QX3</a>) A/H3N2 polymerase, with or without bound cRNA promoter, are also in the dislocated state (<a id="crosref0685" href="#figs8">Figure&nbsp;S8</a>C). When polymerase is bound to the cRNA promoter, mode B exclusively predominates over mode A (<span class="dropBlock reference-citations"><a id="body-ref-sref30-6" href="#" role="doc-biblioref" data-xml-rid="bib7 bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-6" aria-controls="sref30-6">Fan et&nbsp;al., 2019; Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Fan, H. ∙ Walker, A.P. ∙ Carrique, L. ...</div><div><strong>Structures of influenza A virus RNA polymerase offer insight into viral genome replication</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:287-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1530-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85072133233" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31485076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1530-7&amp;pmid=31485076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), probably because of the two nucleotide longer single-stranded region of the cRNA 3′ end, whereas for vRNA binding, the majority are in mode A (<span class="dropBlock reference-citations"><a id="body-ref-sref30-7" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-7" aria-controls="sref30-7">Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as also observed for bat influenza polymerase.</div><div id="p0110" role="paragraph">We interpret the dislocated, mode B promoter bound, vRNA structure as a critical recycling intermediate between the product disassociation state (also dislocated and highly flexible) and the mode A pre-initiation state in which the promoter is stably reformed (see <a id="crosref0695" href="#sec-3">Discussion</a>).</div></section></section><section id="sec-3"><h2>Discussion</h2><section id="sec-3-1"><h3>Mechanism of Polyadenylation</h3><div id="p0115" role="paragraph">Polyadenylation of influenza mRNAs occurs by “stuttering” at the 5′ proximal oligo(U) motif (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib19 bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Li and Palese, 1994; Poon et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Li, X. ∙ Palese, P.</div><div><strong>Characterization of the polyadenylation signal of influenza virus RNA</strong></div><div><em>J.&nbsp;Virol.</em> 1994; <strong>68</strong>:1245-1249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.68.2.1245-1249.1994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7507179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.68.2.1245-1249.1994&amp;pmid=7507179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Poon, L.L. ∙ Pritlove, D.C. ∙ Fodor, E. ...</div><div><strong>Direct evidence that the poly(A) tail of influenza A virus mRNA is synthesized by reiterative copying of a U track in the virion RNA template</strong></div><div><em>J.&nbsp;Virol.</em> 1999; <strong>73</strong>:3473-3476</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.73.4.3473-3476.1999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10074205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.73.4.3473-3476.1999&amp;pmid=10074205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the pre-termination, termination, and stuttering structures described above allow us to propose a mechanism for exactly how this takes place, consistent with the known biochemistry (<a id="crosref0700" href="#fig5">Figure&nbsp;5</a>D).</div><div id="p0120" role="paragraph">The polyadenylation signal comprises a track of five or six, exceptionally seven, uridines always starting at position 17 from the 5′ end of the vRNA template. The exact position from the 5′ end is critical, because displacement of the oligo(U) tract forward by only one position dramatically decreases polyadenylation levels (<span class="dropBlock reference-citations"><a id="body-ref-sref35-3" href="#" role="doc-biblioref" data-xml-rid="bib19 bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-3" aria-controls="sref35-3">Li and Palese, 1994; Poon et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref35-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Li, X. ∙ Palese, P.</div><div><strong>Characterization of the polyadenylation signal of influenza virus RNA</strong></div><div><em>J.&nbsp;Virol.</em> 1994; <strong>68</strong>:1245-1249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.68.2.1245-1249.1994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7507179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.68.2.1245-1249.1994&amp;pmid=7507179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Poon, L.L. ∙ Pritlove, D.C. ∙ Fodor, E. ...</div><div><strong>Direct evidence that the poly(A) tail of influenza A virus mRNA is synthesized by reiterative copying of a U track in the virion RNA template</strong></div><div><em>J.&nbsp;Virol.</em> 1999; <strong>73</strong>:3473-3476</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.73.4.3473-3476.1999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10074205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.73.4.3473-3476.1999&amp;pmid=10074205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This we confirmed by using a v45-6U template, in which C17 was inserted so that the oligo(U) motif starts at position 18, in which case polyadenylation is eliminated (<a id="crosref0705" href="#figs4">Figures S4</a>D–S4F). The v44 pre-termination state structure reveals the path of the template RNA from the tightly bound 5′ hook structure (A1–A10), via the sandwiched dinucleotide A11-G12 to the irregular but highly ordered 13–17 linker, allowing U18 to enter the&nbsp;+1 position in the active site (<a id="crosref0710" href="#fig3">Figure&nbsp;3</a>C). This structure also shows that even in the presence of a non-hydrolysable NTP complementary to nucleotide 17, the corresponding pre-catalytic base pair is not observed at the&nbsp;+1 position (<a id="crosref0715" href="#fig5">Figure&nbsp;5</a>D, 1). This suggests that it is unfavorable for the template nucleotide at position 17 to enter the&nbsp;+1 position, because this requires disruption of the network of interactions that maintain the conformation of nt 13–16, in particular, PB1/R353 intercalating between A16 and C15 (<a id="crosref0720" href="#fig3">Figures 3</a>B and 3C). However, consistent with the observation that incorporation does occur at position 17 of the v44 template (<a id="crosref0725" href="#fig1">Figure&nbsp;1</a>E), and a uridine at position 17 is essential for polyadenylation (<a id="crosref0730" href="#figs4">Figures S4</a>D and S4F), the termination structure shows that the position 17 base is able to dock into the&nbsp;+1 position despite the necessary stretching of nt 13–16 (<a id="crosref0735" href="#fig3">Figure&nbsp;3</a>D). However, only if an incoming NTP can base pair with it, immediately proceeding to catalysis with incorporation into the product, is the energy cost compensated (<a id="crosref0740" href="#fig5">Figure&nbsp;5</a>D, 2 and 3). Importantly, we could only trap the termination structure because the v44 template sequence results in a strong, G-C rich template-product duplex, which compensates for the maximal stretching of the linker to the 5′ hook. In contrast, in the true polyadenylation situation, with the v44-6U template, there is a considerably weaker, A-U rich duplex, and after incorporation of the adenine opposite U17, the tension in the stretched template causes template backtracking. U17 is pulled out of the active site and U18 moves back into the&nbsp;+1 position, tilting its base pair with the adenine in the −1 position as well as the U:A base pairs involving U19-21 higher up the duplex (<a id="crosref0745" href="#fig5">Figure&nbsp;5</a>D, 4). At this stage, the template-product duplex is shortened to nine base pairs and contains five, tilted A-U base pairs. The unpaired&nbsp;+1A product nucleotide stacks on PB1 motif B M410 but has yet to be translocated. This state corresponds to the partially translocated v44-6U “stuttering” structure that we observe (<a id="crosref0750" href="#fig5">Figure&nbsp;5</a>B). The&nbsp;overall weakening of the duplex, together with the backward&nbsp;pull on the template and forward force on the product, promotes inter-strand slippage and product translocation by one base, leaving the&nbsp;+1 position on the product site vacant (<a id="crosref0755" href="#fig5">Figure&nbsp;5</a>D, 5). The previously described conformational change in the motif B loop (<span class="dropBlock reference-citations"><a id="body-ref-sref14-8" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-8" aria-controls="sref14-8">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) may assist this step. The next incoming ATP can then engage with U17, which is flipped transiently back into the active site for another round of incorporation into the growing poly(A) tail (<a id="crosref0765" href="#fig5">Figure&nbsp;5</a>C, 6 and 7). Initial product slippage, however, introduces three mismatches in the upper half of the duplex (<a id="crosref0770" href="#fig5">Figure&nbsp;5</a>C, 5–7), reducing significantly the number of regular base pairs. This facilitates template-product slippage in the next rounds of ATP incorporation but also increases the probability of product dissociation. Furthermore, as observed, if the oligo(U) tract is six nucleotides long, polyadenylation is more efficient, because there will be fewer mismatches in the duplex, favoring slippage and disfavoring product release.</div><div id="p0125" role="paragraph">In summary, we propose a mechanism of stuttering and slippage in which U17 flips in and out of the active site&nbsp;+1 position, with the template in the active site cavity alternating between 9 and 10 nt in length between every ATP incorporation round without any net translocation. Thus, in contrast to regular translocation during elongation, at polyadenylation, only the product translocates out of the active site chamber.</div></section><section id="sec-3-2"><h3>Overall Model of Transcription</h3><div id="p0130" role="paragraph">Based on our extensive, high resolution structural data showing successive snapshots of actively transcribing influenza polymerase, we propose a comprehensive and coherent model for the full transcription cycle of influenza polymerase (<a id="crosref0775" href="#fig7">Figure&nbsp;7</a>; <a id="crosref0780" href="#mmc4">Video S2</a>). The model provides a strong rationale for the existence of&nbsp;the secondary 3′ end binding site, first observed in LACV polymerase (<span class="dropBlock reference-citations"><a id="body-ref-sref8-4" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-4" aria-controls="sref8-4">Gerlach et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Gerlach, P. ∙ Malet, H. ∙ Cusack, S. ...</div><div><strong>Structural Insights into Bunyavirus Replication and Its Regulation by the vRNA Promoter</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1267-1279</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84930486338" target="_blank" aria-label="Open Scopus in new tab">Scopus (139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26004069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.05.006&amp;pmid=26004069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and subsequently for influenza polymerase (<span class="dropBlock reference-citations"><a id="body-ref-sref30-8" href="#" role="doc-biblioref" data-xml-rid="bib7 bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-8" aria-controls="sref30-8">Fan et&nbsp;al., 2019; Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Fan, H. ∙ Walker, A.P. ∙ Carrique, L. ...</div><div><strong>Structures of influenza A virus RNA polymerase offer insight into viral genome replication</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:287-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1530-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85072133233" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31485076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1530-7&amp;pmid=31485076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.061/asset/0f18deb4-24c4-4435-819b-1c4189cbc9b4/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2020.03.061/asset/d21414e6-d235-4c77-8b4d-3cc3b0ff59f3/main.assets/gr7.jpg" height="888" width="678" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0f18deb4-24c4-4435-819b-1c4189cbc9b4/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Schematic Diagram of the Complete Transcription Cycle</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0200" role="paragraph">(A) In the promoter-bound pre-initiation state the template 3′ end takes an indirect route into the active site (white A in magenta sphere).</div><div id="fspara0205" role="paragraph">(B) A stochastic displacement of the priming loop tip allows primer-template hybridization and incorporation of the first few nucleotides. The growing product/template duplex gradually displaces the priming loop out of the exit channel.</div><div id="fspara0210" role="paragraph">(C) Template translocation initially occurs by straightening of the template entrance pathway, but eventually the promoter melts and the β-ribbon collapses. Thumb rotation widens the active site cavity allowing growth of the product/template duplex to the steady-state nine base pairs, before abutting against the PB2 helical lid, forcing strand separation.</div><div id="fspara0215" role="paragraph">(D) The exiting template is guided along the positively charged exit groove and after translocating ∼24 nt, its extremity docks into the secondary 3′ end site and remains bound there until the end of the transcription cycle.</div><div id="fspara0220" role="paragraph">(E) Further translocation forces the exiting template RNA to loop out.</div><div id="fspara0225" role="paragraph">(F) The polymerase processively transcribes the template until the incoming vRNA loop fully tightens up, whereupon the polymerase flips template base U17 in and out of the active site, producing the poly(A) tail (see <a id="crosref1285" href="#fig5">Figure&nbsp;5</a>D).</div><div id="fspara0230" role="paragraph">(G) Due to mismatches in the A-U rich product-template duplex, the polyadenylated viral mRNA is eventually released, concomitant with a switch of the polymerase to the open, dislocated state. However, the template RNA is still threaded through the polymerase, trapped inside a tunnel.</div><div id="fspara0235" role="paragraph">(H) The template can be released, for instance, by an outward flip of the PB1-C/PB2-N helical bundle that allows it to swish out of the active site cavity like a skipping rope held at each end.</div><div id="fspara0240" role="paragraph">(I) Once the vRNA is completely outside, the promoter can reform with the template 3′ end still bound in the secondary site (recycling state). However, for robust promoter formation, the polymerase must transition back to the non-dislocated, pre-initiation configuration, which forces release of the 3′ end from the secondary site and preferential binding in the active site. Another round of transcription can then start.</div><div id="fspara0245" role="paragraph">(J) In the vRNP context, NP molecules must be successively stripped off the incoming template (dotted) as it translocates into the polymerase.</div><div id="fspara0245a" role="paragraph">(K) Because the transcribing polymerase sequesters roughly the same length of RNA as binds to a single NP (∼25 nts), at least one NP must remain free of RNA (light brown NP(n)) before re-docking onto the growing loop of the exiting template.</div></div></div></div></figcaption></figure></div><div id="p0135" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhMTAxNWZjMTc2NTFhMzIyY2UyODBjMjVjZTMxN2Y0ZiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3OTc3fQ.bJ24WwYTLHqPPGcg-_VxN9GIGN2Ti2VgY-3fGne8DIuUm-Nywl5ykWqrz9Z490yE9qYcm3FxzkvimOTFiVxiNA-srWqa8qLKCjO-WRhGJQZIxO30OiZhOhMLb1FXyKhvA7bddElcDkTyTbbDlOEq51j8j_UsGVVB5VWYCM8Ub8uK8ckMBV35QNkVRk8X9TJNYCH7vOub-1Y2mau0JAeMy06DmXLEX3Zo4Dfi99jRkf-J91eenIY9CfvAoI9IjV6HNSRTjrtV23eQVzPCMUw4YYl7v2620RaXNz5enjGxPsixTu4najgq3gFH0tkDjPc0-zvJ_lOT6_LNfg2bqrUKBA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhMTAxNWZjMTc2NTFhMzIyY2UyODBjMjVjZTMxN2Y0ZiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3OTc3fQ.bJ24WwYTLHqPPGcg-_VxN9GIGN2Ti2VgY-3fGne8DIuUm-Nywl5ykWqrz9Z490yE9qYcm3FxzkvimOTFiVxiNA-srWqa8qLKCjO-WRhGJQZIxO30OiZhOhMLb1FXyKhvA7bddElcDkTyTbbDlOEq51j8j_UsGVVB5VWYCM8Ub8uK8ckMBV35QNkVRk8X9TJNYCH7vOub-1Y2mau0JAeMy06DmXLEX3Zo4Dfi99jRkf-J91eenIY9CfvAoI9IjV6HNSRTjrtV23eQVzPCMUw4YYl7v2620RaXNz5enjGxPsixTu4najgq3gFH0tkDjPc0-zvJ_lOT6_LNfg2bqrUKBA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhMTAxNWZjMTc2NTFhMzIyY2UyODBjMjVjZTMxN2Y0ZiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3OTc3fQ.bJ24WwYTLHqPPGcg-_VxN9GIGN2Ti2VgY-3fGne8DIuUm-Nywl5ykWqrz9Z490yE9qYcm3FxzkvimOTFiVxiNA-srWqa8qLKCjO-WRhGJQZIxO30OiZhOhMLb1FXyKhvA7bddElcDkTyTbbDlOEq51j8j_UsGVVB5VWYCM8Ub8uK8ckMBV35QNkVRk8X9TJNYCH7vOub-1Y2mau0JAeMy06DmXLEX3Zo4Dfi99jRkf-J91eenIY9CfvAoI9IjV6HNSRTjrtV23eQVzPCMUw4YYl7v2620RaXNz5enjGxPsixTu4najgq3gFH0tkDjPc0-zvJ_lOT6_LNfg2bqrUKBA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.03.061/attachment/83ebc8c0-7021-4c55-a304-5559fb6f7065/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (3.22 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S2. Simplified Animation of the Complete Transcription Cycle of Influenza Polymerase, Related to Figure 7</div></div></div></div><div id="p0140" role="paragraph">The initiation and transition to elongation steps have been described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref14-9" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-9" aria-controls="sref14-9">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for FluB and confirmed here for FluA (<a id="crosref0795" href="#figs2">Figure&nbsp;S2</a>). The key elements are the growth of the&nbsp;template-product duplex, which correlates with disruption of the promoter, clamping down of the β-ribbon, opening of the PB1 thumb domain, unblocking of the template exit channel via extrusion of the priming loop and strand separation by the PB2 lid domain (<a id="crosref0800" href="#fig7">Figures 7</a>A–7C). Further elongation drives the template along a positively charged groove and after translocation of ∼24 nt, the leading extremity of the template docks into the 3′ end secondary binding site in a sequence-dependent manner (<a id="crosref0805" href="#fig7">Figure&nbsp;7</a>D). The template 3′ end is sequestered by the polymerase during all subsequent steps of transcription, protecting it from degradation by cellular exonucleases. Thereafter, the exiting template is forced to bulge out in a loop that increases in size as transcription proceeds to termination (<a id="crosref0810" href="#fig7">Figure&nbsp;7</a>E). Correspondingly, the incoming template loop between the bound 5′ hook and the active site decreases in size until nucleotide 18 is at the&nbsp;+1 position and further translocation cannot occur without introducing strain in the linkage to the 5′ hook (<a id="crosref0815" href="#fig7">Figure&nbsp;7</a>F). Poly(A) tail synthesis then occurs as described above&nbsp;(<a id="crosref0820" href="#fig5">Figure&nbsp;5</a>D). The exact determinants of poly(A) length and the potential involvement of host-factors in polyadenylation remain unclear (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Landeras-Bueno et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Landeras-Bueno, S. ∙ Jorba, N. ∙ Pérez-Cidoncha, M. ...</div><div><strong>The splicing factor proline-glutamine rich (SFPQ/PSF) is involved in influenza virus transcription</strong></div><div><em>PLoS Pathog.</em> 2011; <strong>7</strong>:e1002397</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1002397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-81755166636" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22114566/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1002397&amp;pmid=22114566" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At some point, the relatively weak, U-A rich, and mismatched product-template duplex dissociates, and the capped and polyadenylated viral mRNA is released. Duplex destabilization and product-dissociation is likely facilitated by coupling to a conformational switch of the polymerase into the dislocated state. This state is incompatible with duplex binding due to the reconfiguration of the post-motif E peptide and repositioning of PB1/521-536 helix such that they would clash with the product and template strands, respectively (<a id="crosref0830" href="#fig7">Figure&nbsp;7</a>G).</div><div id="p0145" role="paragraph">In the product-dissociation state, the 5′ proximal part of the template forms a loop that is trapped inside a tunnel formed by the active site cavity and exit channel. The tunnel walls comprise the priming loop and PB2-N1 domain on one side and PB1/680-700 and PB2/26-37 on the other side, both of which are connected to the PB2-Nter/PB1-Cter helical bundle that packs against the 170-loop of the PB2 helical lid domain, burying the template (<a id="crosref0835" href="#fig7">Figures 7</a>G and <a id="crosref0840" href="#figs8">S8</a>E). The template cannot be extracted without a dramatic conformational change of the polymerase. One way this may occur is for the PB2-Nter/PB1-Cter helical bundle to peel away, opening the tunnel and allowing the template loop to be released (<a id="crosref0845" href="#fig7">Figure&nbsp;7</a>H). Indication that this is feasible comes from a crystal structure of dimeric A/H7N9 polymerase core (<a id="crosref0850" href="#mmc2">Table S2</a>) (PDB: <a id="intref0090" href="http://firstglance.jmol.org/fg.htm?mol6TU5" target="_blank">6TU5</a>) in which the PB2-Nter/PB1-Cter helical bundle has rotated down as envisaged (<a id="crosref0855" href="#figs8">Figure&nbsp;S8</a>F). This would need to be followed by the PB2-Nter/PB1-Cter helical bundle threading back through the template loop in a “skipping rope” fashion. The polymerase could then reconfigure into the still dislocated “recycling” state, with the 3′ and 5′ ends remaining bound in their respective sites, corresponding to mode B promoter binding, but with the observed weak promoter duplex (<a id="crosref0860" href="#fig7">Figure&nbsp;7</a>I). Finally, the polymerase can switch back into the preferred pre-initiation state with mode A promoter binding. This is probably driven by polymerase closure (i.e., reversing the dislocation) and robust promoter duplex formation resulting in release of the 3′ end from its secondary binding site. In the case of cRNA, which binds preferentially in mode B (<span class="dropBlock reference-citations"><a id="body-ref-sref30-9" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-9" aria-controls="sref30-9">Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), dimer formation may be required to release the 3′ end from the secondary binding site to facilitate cRNA to vRNA replication (<span class="dropBlock reference-citations"><a id="body-ref-sref30-10" href="#" role="doc-biblioref" data-xml-rid="bib7 bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-10" aria-controls="sref30-10">Fan et&nbsp;al., 2019; Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Fan, H. ∙ Walker, A.P. ∙ Carrique, L. ...</div><div><strong>Structures of influenza A virus RNA polymerase offer insight into viral genome replication</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:287-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1530-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85072133233" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31485076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1530-7&amp;pmid=31485076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-3-3"><h3>Transcription in the vRNP Context</h3><div id="p0150" role="paragraph">In the viral RNP, the vRNA genome is pseudo-circularized by promoter binding to the viral polymerase with the rest of the RNA forming a supercoiled helical loop coated by NP (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Arranz et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arranz, R. ∙ Coloma, R. ∙ Chichón, F.J. ...</div><div><strong>The structure of native influenza virion ribonucleoproteins</strong></div><div><em>Science.</em> 2012; <strong>338</strong>:1634-1637</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1228172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84871460710" target="_blank" aria-label="Open Scopus in new tab">Scopus (245)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23180776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1228172&amp;pmid=23180776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Influenza NP is believed to bind to 24–26 nt RNA (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib10 bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Hutchinson et&nbsp;al., 2014; Ortega et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Hutchinson, E.C. ∙ Charles, P.D. ∙ Hester, S.S. ...</div><div><strong>Conserved and host-specific features of influenza virion architecture</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:4816</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms5816" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84918544086" target="_blank" aria-label="Open Scopus in new tab">Scopus (200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25226414/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms5816&amp;pmid=25226414" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Ortega, J. ∙ Martín-Benito, J. ∙ Zürcher, T. ...</div><div><strong>Ultrastructural and functional analyses of recombinant influenza virus ribonucleoproteins suggest dimerization of nucleoprotein during virus amplification</strong></div><div><em>J.&nbsp;Virol.</em> 2000; <strong>74</strong>:156-163</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.74.1.156-163.2000" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-17644443084" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10590102/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.74.1.156-163.2000&amp;pmid=10590102" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), although recent studies show that the coating is not uniform and RNA stem-loops may extrude between the NPs (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib17 bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Le Sage et&nbsp;al., 2018; Lee et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Le Sage, V. ∙ Nanni, A.V. ∙ Bhagwat, A.R. ...</div><div><strong>Non-Uniform and Non-Random Binding of Nucleoprotein to Influenza A and B Viral RNA</strong></div><div><em>Viruses.</em> 2018; <strong>10</strong>:E522</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/v10100522" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85054480616" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30257455/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fv10100522&amp;pmid=30257455" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Lee, N. ∙ Le Sage, V. ∙ Nanni, A.V. ...</div><div><strong>Genome-wide analysis of influenza viral RNA and nucleoprotein association</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:8968-8977</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx584" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85031022619" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28911100/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx584&amp;pmid=28911100" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). During transcription, as the template is progressively translocated through the polymerase active site, the NPs must successively peel off as the pre-transcription template RNA loop shrinks. We have now established that the outgoing, transcribed template gradually bulges out on the other side of the polymerase, between the exit channel and the secondary 3′ end binding site, forming a correspondingly growing loop as transcription proceeds. It is therefore plausible that each successive RNA-stripped NP is transferred the short distance from the incoming to outgoing template strands where it can rebind to the growing loop of post-transcribed RNA (<a id="crosref0875" href="#fig7">Figures 7</a>J and 7K). This ensures that the transcribed RNA remains protected by bound NP, and no naked RNA is exposed to the nuclear environment. Interestingly, the amount of template RNA that is&nbsp;sequestered within the polymerase active site cavity, exit channel, and 3′ end binding site, before the outgoing template begins to bulge out and is able to be rebound by NP, is around 25 nt, the&nbsp;same amount of RNA that can bind to one NP. This suggests that there is a docking site for at least one RNA-free NP before it can rebind the outgoing template strand (<a id="crosref0880" href="#fig7">Figures 7</a>J and 7K). This NP would be held in place by transient interactions with the polymerase as well as daisy-chain like NP-NP interactions with its neighbors (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Ye et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Ye, Q. ∙ Krug, R.M. ∙ Tao, Y.J.</div><div><strong>The mechanism by which influenza A virus nucleoprotein forms oligomers and binds RNA</strong></div><div><em>Nature.</em> 2006; <strong>444</strong>:1078-1082</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05379" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-33845890539" target="_blank" aria-label="Open Scopus in new tab">Scopus (369)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17151603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05379&amp;pmid=17151603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The growing loop of NP-coated, post-transcribed vRNA will presumably acquire a similar structural topology as the original super-helical genome segment. Therefore, images of a transcribing vRNP will likely show only a perturbation at the position of the polymerase as it appears to move from one end to the other, as very recently reported (<span class="dropBlock reference-citations"><a id="body-ref-optsLAhWI6rZA" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optsLAhWI6rZA" aria-controls="optsLAhWI6rZA">Coloma et al., 2020</a><div class="dropBlock__holder" data-db-target-of="optsLAhWI6rZA"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Coloma, R. ∙ Arranz, R. ∙ de la Rosa-Trevín, J.M. ...</div><div><strong>Structural insights into influenza A virus ribonucleoproteins reveal a processive helical track as transcription mechanism</strong></div><div><em>Nat. Microbiol.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0675-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85081665986" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32152587/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0675-3&amp;pmid=32152587" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0155" role="paragraph">Finally, the conservation of the secondary 3′ end binding site in both influenza and La Crosse virus suggests that its role to sequester the template extremity during transcription is maintained in all segmented negative strand polymerases. Moreover, given that the cRNA and vRNA can both bind in the secondary 3′ site and the template exit pathway is likely independent of RNA synthesis mode, it is plausible that the model proposed here for the trajectory of the template is the same for replication and transcription.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key Resources Table</h3><div id="p0160" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Bacterial and Virus Strains</b></td></tr><tr class="rowsep"><td>MultiBac baculovirus expression system</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Nie et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Nie, Y. ∙ Bellon-Echeverria, I. ∙ Trowitzsch, S. ...</div><div><strong>Multiprotein complex production in insect cells by using polyproteins</strong></div><div><em>Methods Mol. Biol.</em> 2014; <strong>1091</strong>:131-141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-62703-691-7_8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84934441860" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24203328/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-62703-691-7_8&amp;pmid=24203328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>GIBCO Express Five SFM</td><td>GIBCO</td><td>Cat.# 10486025</td></tr><tr><td>GIBCO L-Glutamine (200mM)</td><td>GIBCO</td><td>Cat.# 25030149</td></tr><tr><td>His60 NiNTA Superflow resin</td><td>Clontech / Takara Bio</td><td>Cat.# 635660</td></tr><tr><td>Strep-Tactin Superflow high capacity resin</td><td>IBA Lifesciences</td><td>Cat.# 2-1208-025</td></tr><tr><td>cOmplete, Mini, EDTA-free Protease Inhibitor Cocktail</td><td>Roche</td><td>Cat.# 11836170001</td></tr><tr><td>UpNHpp</td><td>Jena Bioscience</td><td>Cat.# NU-930S</td></tr><tr><td>NTP set</td><td>Thermo Fisher</td><td>Cat.# R0481</td></tr><tr><td>Decade Markers System</td><td>Thermo Fisher</td><td>Cat.# AM7778</td></tr><tr><td>EasyTides [gamma-32P]-ATP</td><td>Perkin Elmer</td><td>Cat.# NEG502H-250UC</td></tr><tr><td>EasyTides [alpha-32P]-ATP</td><td>Perkin Elmer</td><td>Cat.# NEG503H-250UC</td></tr><tr><td>EasyTides [alpha-32P]-GTP</td><td>Perkin Elmer</td><td>Cat.# NEG506H-250UC</td></tr><tr><td>RNA 5′ Pyrophosphohydrolase (RppH)</td><td>NEB</td><td>Cat.# M0356S</td></tr><tr><td>Monarch® RNA Cleanup Kit (10&nbsp;μg)</td><td>NEB</td><td>Cat.# T2030L</td></tr><tr><td>Low Range ssRNA Ladder</td><td>NEB</td><td>Cat.# N0364S</td></tr><tr><td>SYBR Gold Nucleic Acid Gel Stain</td><td>Thermo Fisher</td><td>Cat.# S11494</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (wild-type)</td><td><span class="dropBlock reference-citations"><a id="body-ref-optqxjmgDaVgc-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optqxjmgDaVgc-1" aria-controls="optqxjmgDaVgc-1">Pflug et al., 2014</a><div class="dropBlock__holder" data-db-target-of="optqxjmgDaVgc-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Pflug, A. ∙ Guilligay, D. ∙ Reich, S. ...</div><div><strong>Structure of influenza A polymerase bound to the viral RNA promoter</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:355-360</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84922257981" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14008&amp;pmid=25409142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (PA/K134A)</td><td>This study</td><td>N/A</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (PA/K483E)</td><td>This study</td><td>N/A</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (PB1/K553E)</td><td>This study</td><td>N/A</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (PB1/Y557A)</td><td>This study</td><td>N/A</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (PB1/R560E)</td><td>This study</td><td>N/A</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (PB2/K54E)</td><td>This study</td><td>N/A</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (PA/K483E, PB1/K553E)</td><td>This study</td><td>N/A</td></tr><tr><td>Influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (PA/K483E, PB1/K553E, PB1/Y557A)</td><td>This study</td><td>N/A</td></tr><tr><td>Influenza B/Memphis/13/03 polymerase (wild-type)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Reich et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Reich, S. ∙ Guilligay, D. ∙ Pflug, A. ...</div><div><strong>Structural insight into cap-snatching and RNA synthesis by influenza polymerase</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:361-366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84922245983" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14009&amp;pmid=25409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td>Endonuclease inhibitor 5-Hydroxy-4-oxo-1-{[1-(1H-pyrrolo[2,3-b]pyridin-1-yl)cyclopentyl]methyl}-1,4-dihydro-3-pyridinecarboxylic acid. SMILES: N1(C2(CN3C&nbsp;= C(C(&nbsp;= O)C(&nbsp;= C3)O)C(&nbsp;= O)O)CCCC2)c2c(cccn2)C&nbsp;= C1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Miyagawa et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Miyagawa, M. ∙ Akiyama, T. ∙ Mikamiyama-Iwata, M. ...</div><div><strong>Discovery of novel 5-hydroxy-4-pyridone-3-carboxy acids as potent inhibitors of influenza Cap-dependent endonuclease</strong></div><div><em>Bioorg. Med. Chem. Lett.</em> 2016; <strong>26</strong>:4739-4742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bmcl.2016.08.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84985995054" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27568084/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bmcl.2016.08.038&amp;pmid=27568084" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; Gift from Savira pharmaceuticals.</td><td>Cat.# ENDO483</td></tr><tr class="rowsep"><td colspan="3"><b>Critical Commercial Assays</b></td></tr><tr><td>SMARTer® smRNA-Seq kit</td><td>Takara Bio USA</td><td>Cat.# 635029</td></tr><tr><td>Agilent DNA High Sensitivity Bioanalyzer Kit</td><td>Agilent Technologies</td><td>Cat.# 5067-4626</td></tr><tr><td>Agilent RNA Pico 6000 Bioanalyzer Kit</td><td>Agilent Technologies</td><td>Cat.# 5067-1513</td></tr><tr><td>Qubit® RNA HS Assay Kit</td><td>Life Technologies</td><td>Cat.# Q32852</td></tr><tr><td>Qubit® dsDNA HS Kit</td><td>Life Technologies</td><td>Cat.# Q32854</td></tr><tr><td>SPRIselect Reagent Kit</td><td>Beckman Coulter</td><td>Cat.# B23319</td></tr><tr class="rowsep"><td>MiSeq Reagent Kit v3 (150-cycle)</td><td>Illumina</td><td>Cat.# MS-102-3001</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase pre-initiation complex</td><td>This study</td><td>6T0N (PDB); 10356 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase elongation complex with incoming UTP analog (core&nbsp;+ endonuclease only)</td><td>This study</td><td>6SZV (PDB); 10355 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase elongation complex with incoming UTP analog (complete polymerase)</td><td>This study</td><td>6T0V (PDB); 10360 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase pre-termination complex with pyrophosphate using 44-mer vRNA template with mutated oligo(U) sequence</td><td>This study</td><td>6SZU (PDB); 10354 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase product dissociation complex using 44-mer vRNA template with mutated oligo(U) sequence</td><td>This study</td><td>6T0R (PDB); 10357 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase termination complex with pyrophosphate using 44-mer vRNA template with mutated oligo(U) sequence</td><td>This study</td><td>6TW1 (PDB); 10603 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase stuttering complex using 44-mer vRNA template with intact oligo(U) sequence</td><td>This study</td><td>6T0S (PDB); 10358 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase product dissociation complex using 44-mer vRNA template with intact oligo(U) sequence</td><td>This study</td><td>6T0U (PDB); 10359 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Bat Influenza A polymerase recycling complex</td><td>This study</td><td>6T2C (PDB); 10368 (EMDB)</td></tr><tr><td>Cryo-EM map and model of Human Influenza B polymerase recycling complex</td><td>This study</td><td>6T0W (PDB); 10361 (EMDB)</td></tr><tr><td>Crystal structure and model of Influenza A/H7N9 polymerase core (apo)</td><td>This study</td><td>6TU5 (PDB)</td></tr><tr><td>RNA sequencing data, sample v44 (+AGC)</td><td>This study</td><td>SAMEA6563093 (ENA)</td></tr><tr><td>RNA sequencing data, sample v44 (+AGCU)</td><td>This study</td><td>SAMEA6563094 (ENA)</td></tr><tr class="rowsep"><td>RNA sequencing data, sample v3′ RNA 18+3 (+AGC)</td><td>This study</td><td>SAMEA6563095 (ENA)</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Cell Lines</b></td></tr><tr class="rowsep"><td><i>Trichoplusia ni</i> High Five Cells</td><td>Invitrogen</td><td>Cat.# B85502</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr class="rowsep"><td>(see the <a id="crosref0910" href="#sec-4">STAR Methods</a> for all oligonucleotides used)</td><td>N/A</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (wild-type)</td><td><span class="dropBlock reference-citations"><a id="body-ref-optqxjmgDaVgc-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optqxjmgDaVgc-2" aria-controls="optqxjmgDaVgc-2">Pflug et al., 2014</a><div class="dropBlock__holder" data-db-target-of="optqxjmgDaVgc-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Pflug, A. ∙ Guilligay, D. ∙ Reich, S. ...</div><div><strong>Structure of influenza A polymerase bound to the viral RNA promoter</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:355-360</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84922257981" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14008&amp;pmid=25409142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>pKL-PBac-BatPol</td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (PA/K134A)</td><td>This study</td><td>pKL-PBac-BatPol-PA-K134A</td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (PA/K483E)</td><td>This study</td><td>pKL-PBac-BatPol-PA-K483E</td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (PB1/K553E)</td><td>This study</td><td>pKL-PBac-BatPol-PB1-K557E</td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (PB1/Y557A)</td><td>This study</td><td>pKL-PBac-BatPol-PB1-Y557A</td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (PB1/R560E)</td><td>This study</td><td>pKL-PBac-BatPol-PB1-R560E</td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (PB2/K54E)</td><td>This study</td><td>pKL-PBac-BatPol-PB2-K54E</td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (PA/K483E, PB1/K553E)</td><td>This study</td><td>pKL-PBac-BatPol-2xmut</td></tr><tr><td>Expression plasmid for bat Influenza A polymerase (PA/K483E, PB1/K553E, PB1/Y557A)</td><td>This study</td><td>pKL-PBac-BatPol-3xmut</td></tr><tr><td>Expression plasmid for human Influenza B polymerase (wild-type)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref37-3" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-3" aria-controls="sref37-3">Reich et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref37-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Reich, S. ∙ Guilligay, D. ∙ Pflug, A. ...</div><div><strong>Structural insight into cap-snatching and RNA synthesis by influenza polymerase</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:361-366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84922245983" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14009&amp;pmid=25409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>pKL-PBac-FluBPol</td></tr><tr class="rowsep"><td>Expression plasmid for Influenza A/H7N9 polymerase core (PA 201-716, PB1 full-length, PB2 1-127)</td><td>This study</td><td>pFastBacDual-H7N9-core</td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>RELION 3.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Scheres, 2012</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Scheres, S.H.</div><div><strong>RELION: implementation of a Bayesian approach to cryo-EM structure determination</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2012; <strong>180</strong>:519-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2012.09.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84868444740" target="_blank" aria-label="Open Scopus in new tab">Scopus (3642)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23000701/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2012.09.006&amp;pmid=23000701" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>MotionCorr</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85014129582" target="_blank" aria-label="Open Scopus in new tab">Scopus (4513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28250466/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4193&amp;pmid=28250466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Gctf</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref47-1" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-1" aria-controls="sref47-1">Zhang, 2016</a><div class="dropBlock__holder" data-db-target-of="sref47-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2015.11.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84955216953" target="_blank" aria-label="Open Scopus in new tab">Scopus (2386)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26592709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2015.11.003&amp;pmid=26592709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0095" href="https://www.mrc-lmb.cam.ac.uk/kzhang" target="_blank">https://www.mrc-lmb.cam.ac.uk/kzhang</a></td></tr><tr><td>Gautomatch</td><td>N/A</td><td><a id="intref0100" href="https://www.mrc-lmb.cam.ac.uk/kzhang" target="_blank">https://www.mrc-lmb.cam.ac.uk/kzhang</a></td></tr><tr><td>WARP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref42-1" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-1" aria-controls="sref42-1">Tegunov and Cramer, 2019</a><div class="dropBlock__holder" data-db-target-of="sref42-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Tegunov, D. ∙ Cramer, P.</div><div><strong>Real-time cryo-electron microscopy data preprocessing with Warp</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1146-1152</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0580-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85074279272" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31591575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0580-y&amp;pmid=31591575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0105" href="http://www.warpem.com/warp/#" target="_blank">http://www.warpem.com/warp/#</a></td></tr><tr><td>Coot</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444910007493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-77949535720" target="_blank" aria-label="Open Scopus in new tab">Scopus (20200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444910007493&amp;pmid=20383002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0110" href="https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot" target="_blank">https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot</a></td></tr><tr><td>PHENIX</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Afonine et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Afonine, P.V. ∙ Poon, B.K. ∙ Read, R.J. ...</div><div><strong>Real-space refinement in PHENIX for cryo-EM and crystallography</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:531-544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798318006551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85048343902" target="_blank" aria-label="Open Scopus in new tab">Scopus (1516)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798318006551&amp;pmid=29872004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0115" href="https://www.phenix-online.org" target="_blank">https://www.phenix-online.org</a></td></tr><tr><td>CCP-EM</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Burnley et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Burnley, T. ∙ Palmer, C.M. ∙ Winn, M.</div><div><strong>Recent developments in the CCP-EM software suite</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2017; <strong>73</strong>:469-477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798317007859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85020201461" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28580908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798317007859&amp;pmid=28580908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0120" href="https://www.ccpem.ac.uk" target="_blank">https://www.ccpem.ac.uk</a></td></tr><tr><td>LocScale</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Jakobi et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Jakobi, A.J. ∙ Wilmanns, M. ∙ Sachse, C.</div><div><strong>Model-based local density sharpening of cryo-EM maps</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e27131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.27131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85036517329" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29058676/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.27131&amp;pmid=29058676" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>AUTOPROC/STARANISO</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Tickle et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Tickle, I.J. ∙ Flensburg, C. ∙ Keller, P. ...</div><div><strong>STARANISO. Global Phasing Ltd.</strong></div><div><a id="intref0330" href="http://staraniso.globalphasing.org/cgi-bin/staraniso.cgi" target="_blank">http://staraniso.globalphasing.org/cgi-bin/staraniso.cgi</a><div>Date: 2018</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.J.TickleC.FlensburgP.KellerW.PaciorekA.SharffC.VonrheinG.BricogneSTARANISO.+Global+Phasing+Ltd.http%3A%2F%2Fstaraniso.globalphasing.org%2Fcgi-bin%2Fstaraniso.cgi2018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>PHASER</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref23-1" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-1" aria-controls="sref23-1">McCoy et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref23-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>McCoy, A.J. ∙ Grosse-Kunstleve, R.W. ∙ Adams, P.D. ...</div><div><strong>Phaser crystallographic software</strong></div><div><em>J.&nbsp;Appl. Cryst.</em> 2007; <strong>40</strong>:658-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0021889807021206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-34447508216" target="_blank" aria-label="Open Scopus in new tab">Scopus (16083)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19461840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0021889807021206&amp;pmid=19461840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Refmac5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">Murshudov et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Murshudov, G.N. ∙ Vagin, A.A. ∙ Dodson, E.J.</div><div><strong>Refinement of macromolecular structures by the maximum-likelihood method</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 1997; <strong>53</strong>:240-255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444996012255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0030924992" target="_blank" aria-label="Open Scopus in new tab">Scopus (14211)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15299926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444996012255&amp;pmid=15299926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>UCSF Chimera</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Pettersen et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF Chimera--a visualization system for exploratory research and analysis</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2004; <strong>25</strong>:1605-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.20084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-4444221565" target="_blank" aria-label="Open Scopus in new tab">Scopus (33444)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.20084&amp;pmid=15264254" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0125" href="https://www.cgl.ucsf.edu/chimera" target="_blank">https://www.cgl.ucsf.edu/chimera</a></td></tr><tr><td>PyMOL</td><td><a id="intref0130" href="https://www.schrodinger.com/" target="_blank">Schrödinger</a></td><td><a id="intref0135" href="https://pymol.org/2/" target="_blank">https://pymol.org/2/</a></td></tr><tr><td>CCP4MG</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">McNicholas et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>McNicholas, S. ∙ Potterton, E. ∙ Wilson, K.S. ...</div><div><strong>Presenting your structures: the CCP4mg molecular-graphics software</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2011; <strong>67</strong>:386-394</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444911007281" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-79953765276" target="_blank" aria-label="Open Scopus in new tab">Scopus (1069)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21460457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444911007281&amp;pmid=21460457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0140" href="https://www.ccp4.ac.uk/MG" target="_blank">https://www.ccp4.ac.uk/MG</a></td></tr><tr><td>APBS</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref13-1" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-1" aria-controls="sref13-1">Jurrus et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref13-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Jurrus, E. ∙ Engel, D. ∙ Star, K. ...</div><div><strong>Improvements to the APBS biomolecular solvation software suite</strong></div><div><em>Protein Sci.</em> 2018; <strong>27</strong>:112-128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3280" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85039036425" target="_blank" aria-label="Open Scopus in new tab">Scopus (1239)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28836357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3280&amp;pmid=28836357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>LigPlot+</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Laskowski and Swindells, 2011</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Laskowski, R.A. ∙ Swindells, M.B.</div><div><strong>LigPlot+: multiple ligand-protein interaction diagrams for drug discovery</strong></div><div><em>J.&nbsp;Chem. Inf. Model.</em> 2011; <strong>51</strong>:2778-2786</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ci200227u" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-80054911951" target="_blank" aria-label="Open Scopus in new tab">Scopus (4092)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21919503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fci200227u&amp;pmid=21919503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>SnapGene</td><td>SnapGene</td><td><a id="intref0145" href="https://www.snapgene.com:443/" target="_blank">https://www.snapgene.com:443/</a></td></tr><tr><td>Cutadapt v 2.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Martin, 2011</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnetjournal.</em> 2011; <strong>17</strong>:10-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.14806/ej.17.1.200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.14806%2Fej.17.1.200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>NR-grep v 1.1.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Navarro, 2001</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Navarro, G.</div><div><strong>NR‐grep: a fast and flexible pattern-matching tool</strong></div><div><em>Softw. Pract. Exper.</em> 2001; <strong>31</strong>:1265-1312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/spe.411" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0035841458" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fspe.411" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0150" href="https://users.dcc.uchile.cl/%7Egnavarro/software/" target="_blank">https://users.dcc.uchile.cl/~gnavarro/software/</a></td></tr><tr><td>GNU grep</td><td>GNU</td><td><a id="intref0155" href="https://www.gnu.org/software/grep/manual/grep.html" target="_blank">https://www.gnu.org/software/grep/manual/grep.html</a></td></tr><tr><td>Je demultiplexer</td><td><span class="dropBlock reference-citations"><a id="body-ref-optlavlWAsZJq" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optlavlWAsZJq" aria-controls="optlavlWAsZJq">Girardot et al., 2016</a><div class="dropBlock__holder" data-db-target-of="optlavlWAsZJq"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Girardot, C. ∙ Scholtalbers, J. ∙ Sauer, S. ...</div><div><strong>Je, a versatile suite to handle multiplexed NGS libraries with unique molecular identifiers</strong></div><div><em>BMC Bioinformatics.</em> 2016; <strong>17</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-016-1284-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84990841832" target="_blank" aria-label="Open Scopus in new tab">Scopus (105)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27717304/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-016-1284-2&amp;pmid=27717304" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0160" href="https://gbcs.embl.de/portal/tiki-index.php?page=Je" target="_blank">https://gbcs.embl.de/portal/tiki-index.php?page=Je</a></td></tr><tr><td>R version 3.5.1</td><td>R Foundation for Statistical Computing</td><td><a id="intref0165" href="https://www.R-project.org" target="_blank">https://www.R-project.org</a></td></tr><tr><td>cowplot 0.9.4</td><td>R package</td><td><a id="intref0170" href="https://cran.r-project.org/web/packages/cowplot/index.html" target="_blank">https://cran.r-project.org/web/packages/cowplot/index.html</a></td></tr><tr><td>ggplot2 3.2.0</td><td>R package</td><td><a id="intref0175" href="https://cran.r-project.org/web/packages/ggplot2/index.html" target="_blank">https://cran.r-project.org/web/packages/ggplot2/index.html</a></td></tr><tr><td>readr 1.3.1</td><td>R package</td><td><a id="intref0180" href="https://cran.r-project.org/web/packages/readr/index.html" target="_blank">https://cran.r-project.org/web/packages/readr/index.html</a></td></tr><tr><td>GraphPad Prism</td><td>GraphPad</td><td><a id="intref0185" href="https://www.graphpad.com" target="_blank">https://www.graphpad.com</a></td></tr><tr><td>UNAFold</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Markham and Zuker, 2008</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Markham, N.R. ∙ Zuker, M.</div><div><strong>UNAFold: software for nucleic acid folding and hybridization</strong></div><div><em>Methods Mol. Biol.</em> 2008; <strong>453</strong>:3-31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-60327-429-6_1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84934438960" target="_blank" aria-label="Open Scopus in new tab">Scopus (847)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18712296/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-60327-429-6_1&amp;pmid=18712296" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="interref0365" href="http://unafold.rna.albany.edu" target="_blank">http://unafold.rna.albany.edu</a></td></tr><tr class="rowsep"><td>WebLogo</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Crooks et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Crooks, G.E. ∙ Hon, G. ∙ Chandonia, J.M. ...</div><div><strong>WebLogo: a sequence logo generator</strong></div><div><em>Genome Res.</em> 2004; <strong>14</strong>:1188-1190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.849004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-2142738304" target="_blank" aria-label="Open Scopus in new tab">Scopus (9365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15173120/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.849004&amp;pmid=15173120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="interref0370" href="https://weblogo.berkeley.edu/" target="_blank">https://weblogo.berkeley.edu/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Quantifoil R 1.2/1.3 Au 300</td><td>Quantifoil</td><td>N/A</td></tr><tr><td>HiTrap Heparin HP 5&nbsp;mL column</td><td>GE Healthcare</td><td>Cat.# 17040703</td></tr><tr><td>Superdex 200 Increase 10/300 GL</td><td>GE Healthcare</td><td>Cat.# 28990944</td></tr><tr><td>Amicon® Ultra-15 (50&nbsp;kDa cut-off)</td><td>Merck Millipore</td><td>Cat.# UFC905024</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2930389-5" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Lead Contact and Materials Availability</h3><div id="p0165" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Stephen Cusack (<a id="intref0200" href="mailto:cusack@embl.fr">cusack@embl.fr</a>).</div><div id="p0170" role="paragraph">All unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.</div></section><section id="sec-4-3"><h3>Experimental Model and Subject Details</h3><div id="p0175" role="paragraph"><i>Trichoplusia ni</i> HighFive cells (Invitrogen) cultured in GIBCO Express Five TM SFM media supplemented with GIBCO L-glutamine were used for recombinant protein expression.</div></section><section id="sec-4-4"><h3>Method Details</h3><section id="sec-4-4-1"><h4>Site-directed mutagenesis</h4><div id="p0180" role="paragraph">Mutations were introduced in influenza A/little yellow-shouldered bat/2010/H17N10 polymerase (bat FluA), including single mutants (PA/K134A, PA/K483E, PB1/K553E, PB1/Y557A, PB1/R560E and PB2/K54E), a double (PA/K483E&nbsp;+ PB1/Y553E) and a triple mutant (PA/K483E&nbsp;+ PB1/K553E&nbsp;+ PB1/Y557A), by site-directed mutagenesis. Introduction of desired mutations was confirmed by sequencing of the entire polymerase gene.</div></section><section id="sec-4-4-2"><h4>Bat FluA polymerase expression and purification</h4><div id="p0185" role="paragraph">The bat FluA polymerase self-cleaving polyprotein construct was expressed and purified as previously described (<span class="dropBlock reference-citations"><a id="body-ref-optqxjmgDaVgc-3" href="#" role="doc-biblioref" data-xml-rid="bib28 bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optqxjmgDaVgc-3" aria-controls="optqxjmgDaVgc-3">Nie et&nbsp;al., 2014; Pflug et al., 2014</a><div class="dropBlock__holder" data-db-target-of="optqxjmgDaVgc-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Nie, Y. ∙ Bellon-Echeverria, I. ∙ Trowitzsch, S. ...</div><div><strong>Multiprotein complex production in insect cells by using polyproteins</strong></div><div><em>Methods Mol. Biol.</em> 2014; <strong>1091</strong>:131-141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-62703-691-7_8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84934441860" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24203328/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-62703-691-7_8&amp;pmid=24203328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Pflug, A. ∙ Guilligay, D. ∙ Reich, S. ...</div><div><strong>Structure of influenza A polymerase bound to the viral RNA promoter</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:355-360</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84922257981" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14008&amp;pmid=25409142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) unless stated otherwise. Briefly, wild-type and mutant polymerase were expressed in HighFive insect cells using the baculovirus expression system. Cells were collected by centrifugation, re-suspended in lysis buffer (50&nbsp;mM Tris-HCl pH 8, 500&nbsp;mM NaCl, 10% glycerol, 2&nbsp;mM TCEP) supplemented with protease inhibitor (Roche, complete mini, EDTA-free) and disrupted by sonication followed by centrifugation (35 000&nbsp;rpm, 35&nbsp;min, 4°C) to yield cleared supernatant. The latter was subjected to ammonium sulfate precipitation (final concentration 0.5 g/mL) and the pellet was recovered by centrifugation (30 000&nbsp;rpm, 30&nbsp;min, 4°C, rotor 45Ti, Beckman Coulter). The pellet was dissolved in lysis buffer and subjected to a final centrifugation step (35 000&nbsp;rpm, 35&nbsp;min, 4°C). Cleared supernatant was incubated with nickel resin (His60 NiNTA, Clontech) and protein was eluted with lysis buffer supplemented with 400&nbsp;mM imidazole. The sample was diluted to decrease imidazole concentration to 250&nbsp;mM and subjected to affinity chromatography on Strep-Tactin resin (Superflow, IBA). Protein was eluted with lysis buffer supplemented with 2.5&nbsp;mM d-desthiobiotin, diluted to 250&nbsp;mM salt concentration and loaded on heparin column (HiTrap Heparin HP, 5&nbsp;mL, GE Healthcare). Elution was performed with a 25% to 100% gradient of buffers A (50&nbsp;mM HEPES pH 7.5, 5% glycerol, 2&nbsp;mM TCEP) and B (buffer A supplemented with 1&nbsp;M NaCl). At this stage, monomeric and RNA-free wild-type polymerase fractions were pooled, concentrated (50&nbsp;kDa cut-off, Amicon), flash-frozen and stored at −80°C (<a id="crosref1040" href="#figs2">Figure&nbsp;S2</a>D). In the case of mutant polymerases an additional step was introduced. Pooled fractions were concentrated and injected onto size exclusion column (Superdex 200 Increase 10/300, GE Healthcare) equilibrated in buffer C (50&nbsp;mM Tris-HCl pH 7.5, 500&nbsp;mM NaCl, 5% glycerol, 2&nbsp;mM TCEP). Monomeric and RNA-free polymerase fractions were pooled, concentrated (50&nbsp;kDa cut-off, Amicon), flash-frozen and stored at −80°C.</div></section><section id="sec-4-4-3"><h4>A/H7N9 polymerase core expression, purification, crystallization and structure determination</h4><div id="p0190" role="paragraph">PA, PB1 and PB2 of influenza A/Zhejiang/DTID-ZJU01/2013(H7N9) were expressed from synthetic codon-optimized reading frames (DNA2.0) corresponding to GenBank (Uniprot) entries AGJ51952.1 (M9TI86), AGJ51960.1 (M9TLW3) and KJ633805.1 (X5F427), respectively. A co-expression construct for the A/H7N9 core, was generated using the baculovirus expression vector pFastBacDual (Thermo Fisher) as follows. The complete PB1 protein coding sequence was inserted into the PolH-MCS using the restriction sites BamHI and RsrII. The sequence encoding residues 1-127 of PB2, supplemented with a C-terminal TEV-cleavable poly-histidine tag (GSGSENLYFQGSHHHHHHHH) was inserted into the P10-MCS using the restriction sites BbsI and XhoI. The sequence encoding for&nbsp;residues 201-716 of PA was cloned first into the vector pACEBac via the restriction sites BamHI and EcoRI, then amplified from this construct including the PolH promoter and SV40 polyA signal and subcloned into the pFastBacDual_PB1_PB2 construct using a unique AvrII restriction site and SLIC cloning technology.</div><div id="p0195" role="paragraph">The core of A/H7N9 polymerase (PA 201-716, PB1 full-length, PB2 1-127) was produced in HighFive insect cells using the baculovirus expression system. Cells were lysed by sonication in buffer A (50&nbsp;mM Tris-HCl pH 8.0, 500&nbsp;mM NaCl, 10% (v/v) glycerol), cell debris was spun off (30&nbsp;min, 4°C, 35000 g) and ammonium sulfate added to the supernatant (final concentration 0.5 g/ml) to precipitate the protein. The precipitated protein was collected by centrifugation (30&nbsp;min, 4°C, 35000 g), re-suspended in buffer A and a cleared by centrifugation (30&nbsp;min, 4°C, 35000 g) before subjecting it to immobilized metal ion affinity chromatography (IMAC). Elution fractions containing polymerase protein were pooled und subjected to a digestion with TEV protease (in buffer A supplemented with 5&nbsp;mM β-mercaptoethanol). After dialyzing the digested protein sample back into buffer A it was passed through an IMAC column a second time. The sample was diluted to a salt concentration of 250&nbsp;mM NaCl prior to loading it on a heparin column (HiPrep Heparin HP, GE Healthcare) pre-washed with buffer B (50&nbsp;mM HEPES pH 7.5, 5% (v/v) glycerol, 2&nbsp;mM TCEP, 150&nbsp;mM NaCl). The protein was eluted via a salt gradient plateauing at 1&nbsp;M NaCl. Monodisperse and RNA-free polymerase was concentrated to about 9&nbsp;mg/ml, flash-frozen and stored at −80°C.</div><div id="p0200" role="paragraph">A/H7N9 polymerase core crystallized within 4-5&nbsp;days at 4°C in sitting drops in conditions of 0.1&nbsp;M Tris pH 7.0, 8%–13% PEG 8K, 0.2&nbsp;M MgCl<sub>2</sub>, 0.1&nbsp;M guanidine hydrochloride with drop mixing ratios of 1:2 - 1:3 (protein:well) (<a id="crosref1045" href="#mmc2">Table S2</a>). Diffraction data were collected at 100° K on beamline ID30A3 (MASSIF 3) at the European Synchrotron Radiation Facility (ESRF) and integrated and scaled with AUTOPROC with STARANISO (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Tickle et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Tickle, I.J. ∙ Flensburg, C. ∙ Keller, P. ...</div><div><strong>STARANISO. Global Phasing Ltd.</strong></div><div><a id="intref0330" href="http://staraniso.globalphasing.org/cgi-bin/staraniso.cgi" target="_blank">http://staraniso.globalphasing.org/cgi-bin/staraniso.cgi</a><div>Date: 2018</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.J.TickleC.FlensburgP.KellerW.PaciorekA.SharffC.VonrheinG.BricogneSTARANISO.+Global+Phasing+Ltd.http%3A%2F%2Fstaraniso.globalphasing.org%2Fcgi-bin%2Fstaraniso.cgi2018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Initial phases were obtained by molecular replacement using PHASER (<span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">McCoy et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>McCoy, A.J. ∙ Grosse-Kunstleve, R.W. ∙ Adams, P.D. ...</div><div><strong>Phaser crystallographic software</strong></div><div><em>J.&nbsp;Appl. Cryst.</em> 2007; <strong>40</strong>:658-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0021889807021206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-34447508216" target="_blank" aria-label="Open Scopus in new tab">Scopus (16083)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19461840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0021889807021206&amp;pmid=19461840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with the bat FluA polymerase structure (PDB: <a id="intref0205" href="http://firstglance.jmol.org/fg.htm?mol4WSB" target="_blank">4WSB</a>). Structural adjustments were performed with COOT (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444910007493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-77949535720" target="_blank" aria-label="Open Scopus in new tab">Scopus (20200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444910007493&amp;pmid=20383002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and refinement done with Refmac5 (<span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">Murshudov et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Murshudov, G.N. ∙ Vagin, A.A. ∙ Dodson, E.J.</div><div><strong>Refinement of macromolecular structures by the maximum-likelihood method</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 1997; <strong>53</strong>:240-255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444996012255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0030924992" target="_blank" aria-label="Open Scopus in new tab">Scopus (14211)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15299926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444996012255&amp;pmid=15299926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-4"><h4>Endonuclease inhibitor</h4><div id="p0205" role="paragraph">A potent endonuclease inhibitor, denoted ENDO483, was used as indicated below. ENDO483 is 5-Hydroxy-4-oxo-1-{[1-(1H-pyrrolo[2,3-b]pyridin-1-yl)cyclopentyl]methyl}-1,4-dihydro-3-pyridinecarboxylic acid (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Miyagawa et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Miyagawa, M. ∙ Akiyama, T. ∙ Mikamiyama-Iwata, M. ...</div><div><strong>Discovery of novel 5-hydroxy-4-pyridone-3-carboxy acids as potent inhibitors of influenza Cap-dependent endonuclease</strong></div><div><em>Bioorg. Med. Chem. Lett.</em> 2016; <strong>26</strong>:4739-4742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bmcl.2016.08.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84985995054" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27568084/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bmcl.2016.08.038&amp;pmid=27568084" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The sample used was a gift from Savira pharmaceuticals.</div></section><section id="sec-4-4-5"><h4>Cap-dependent polymerase transcription assays</h4><div id="p0210" role="paragraph">To test the transcriptional activity of polymerase toward synthetic vRNA templates used in this study, an <i>in&nbsp;vitro</i> transcription assay&nbsp;based on incorporation of radiolabeled nucleotides (α<sup>32</sup>P-ATP, Perkin Elmer) was performed. 0.8&nbsp;μM bat FluA polymerase pre-bound to 35&nbsp;μM endonuclease inhibitor ENDO483, 0.8&nbsp;μM template RNA v44, v44-6U or v45-6U, 2&nbsp;μM capped RNA primer, 100&nbsp;μM NTPs and 12&nbsp;nM α<sup>32</sup>P-ATP were incubated in cryoEM buffer (see below) at 28°C for 10, 30&nbsp;min and 2 h. Samples were separated by 7&nbsp;M urea, 20% polyacrylamide gel electrophoresis (urea-PAGE) in Tris-borate-EDTA (TBE) buffer, exposed to a phosphor storage screen and imaged with a Typhoon scanner. Decade marker system (Ambion) was used as RNA ladder.</div><div id="p0215" role="paragraph">To test elongation stalling reaction conditions, 0.9&nbsp;μM bat FluA polymerase pre-bound to 35&nbsp;μM endonuclease inhibitor ENDO483, 0.81&nbsp;μM template 18+3-mer 3′ vRNA, 0.81&nbsp;μM 14-mer 5′ vRNA, 1.8&nbsp;μM capped RNA primer and 100&nbsp;μM ATP, GTP (and CTP or CTP and UpNHpp) and 12&nbsp;nM α-32P-ATP were incubated in cryoEM buffer at 28°C for 5 h. Additionally, CTP or CTP and UpNHpp were added for 2, 4 or 10&nbsp;minutes. Samples were separated and visualized by the later procedure. Decade marker system (Ambion) and 20-mer, 23-mer and 28-mer RNAs, 2′-O-methylated at first ribose adjacent to the cap and radio-labeled at the 5′ end were used as markers.</div></section><section id="sec-4-4-6"><h4>Transcription product sequencing</h4><div id="p0220" role="paragraph">To confirm the identity of transcription products and address the discrepancy between expected size and urea-PAGE migration, sequencing of the main product was performed. First <i>in&nbsp;vitro</i> transcription assay was performed by incubating 0.8&nbsp;μM bat FluA polymerase pre-bound to 35&nbsp;μM endonuclease inhibitor ENDO483, 2&nbsp;μM 13-mer capped primer, 0.8&nbsp;μM template RNA v44 and 100&nbsp;μM ATP, GTP, CTP and without or with UTP in cryoEM buffer for 3h at 28°C. Analogous reaction was set up with 0.8&nbsp;μM bat FluA polymerase pre-bound to 35&nbsp;μM endonuclease inhibitor ENDO483, 2&nbsp;μM 15-mer capped primer, 0.8&nbsp;μM template 18+3-mer 3′ vRNA and 14-mer 5′ vRNA and 100&nbsp;μM ATP, GTP and CTP for 3h at 28°C. After completion of the reaction, proteins were removed by Monarch® RNA Cleanup Kit (NEB) and RNAs were then decapped using RNA 5′ pyrophosphohydrolase (RppH, NEB) and subjected again to the Monarch® RNA Cleanup Kit (NEB). Purified RNA were subjected to the urea-PAGE, stained with SYBR Gold (Thermo Fisher) and RNA was extracted from the bands of interest (shown on <a id="crosref1075" href="#fig1">Figures 1</a>D and 1E).</div><div id="p0225" role="paragraph">RNA integrity was checked using the RNA Pico 6000 Assay Kit of the Bioanalyzer 2100 system (Agilent Technologies, Santa Clara, CA), and concentration was measured with Qubit® RNA Assay Kit in Qubit® 2.0 Fluorimeter (Life Technologies). Small RNA-Seq libraries were prepared manually from 2&nbsp;ng of RNA as input using the SMARTer® smRNA-Seq kit for Illumina® Platforms (Takara Bio). Library preparation was done following the manufacturer’s instructions. The size distribution of the libraries was assessed on Bioanalyzer with a DNA High Sensitivity kit (Agilent Technologies), and concentration was measured with Qubit® DNA High Sensitivity kit in Qubit® 2.0 Flurometer (Life Technologies). Libraries that passed the QC step were pooled in equimolar amounts and final pool&nbsp;was purified with SPRI select beads (Beckman Coulter) in a 1.3x ratio. 8 pM solution of the pool of libraries was loaded on the Illumina sequencer MiSeq and sequenced uni-directionally, generating ∼18 million reads, each 150 bases long.</div><div id="p0230" role="paragraph">Base-calling files were processed using Illumina bcl2fastq version 2.20.422 (Illumina, San Diego, USA). Resulting fastq reads were trimmed using Cutadapt v 2.3 (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Martin, 2011</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnetjournal.</em> 2011; <strong>17</strong>:10-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.14806/ej.17.1.200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.14806%2Fej.17.1.200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with settings recommended by the library preparation kit (-u 3 -a ‘AAAAAAAAAAAAAA’) omitting the filter for only reads longer than 25 bases. Trimmed reads were subsequently used as an input for pattern lookup performed by combination of NR-grep (<span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Navarro, 2001</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Navarro, G.</div><div><strong>NR‐grep: a fast and flexible pattern-matching tool</strong></div><div><em>Softw. Pract. Exper.</em> 2001; <strong>31</strong>:1265-1312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/spe.411" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0035841458" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fspe.411" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and GNU grep (<a id="intref0210" href="https://www.gnu.org/software/grep/manual/grep.html" target="_blank">https://www.gnu.org/software/grep/manual/grep.html</a>). NR-grep allows for fuzzy search including substitutions, deletions and insertions. The list of search regular expression patterns and allowed substitutions are in <a id="crosref1090" href="#mmc2">Table S3</a>. After the filtering, reads were sorted and unique occurrences were counted.</div></section><section id="sec-4-4-7"><h4>Urea-PAGE migration analysis</h4><div id="p0235" role="paragraph">To investigate migration and level of denaturation of dsRNA, 2&nbsp;μM 38-mer RNA (RNA38) and equimolar amount of its reverse complementary sequence (RNA38-RC) were incubated separately and together in cryoEM buffer for 15&nbsp;min at room temperature. 2xRNA loading buffer was added to all samples (95% formamide, 0.025% SDS, 0.025% bromophenol blue, 0.025% xylene cyanol FF, 0.5&nbsp;mM EDTA) and samples were boiled at 95°C and analyzed by 7&nbsp;M urea 20% polyacrylamide electrophoresis at three different temperatures. Three gels were loaded and run in parallel either on ice (∼4°C), at room temperature (∼25°C) or with hot buffer (∼95°C) to compare migration of RNA samples. Gels were stained with SYBR Gold stain (Thermo Fisher) and imaged after 15&nbsp;minutes. Decade marker system (Thermo Fisher) and low range ssRNA ladder (NEB) were used as markers.</div></section><section id="sec-4-4-8"><h4>Fluorescence polarization-based RNA synthesis assay</h4><div id="p0240" role="paragraph">In order to test activity of newly produced polymerase mutants, an <i>in&nbsp;vitro</i> transcription assay based on fluorescence polarization (FP) and using short fluorescently labeled template vRNA was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Reich et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Reich, S. ∙ Guilligay, D. ∙ Cusack, S.</div><div><strong>An in&nbsp;vitro fluorescence based study of initiation of RNA synthesis by influenza B polymerase</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:3353-3368</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28126917/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=28126917" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, 0.25&nbsp;μM wild-type or mutant polymerase was prebound to 1.2x excess of 5′-vRNA (5′-pAGUAGUAACAAGAG-3′) and mixed with 1.5&nbsp;μM capped RNA primer (5′-m<sup>7</sup>GpppAAUCUAUAAUAGC-3′), 0.3&nbsp;μM FAM-labeled vRNA template (18-mer, 5′-FAM-Ex-5-UAUACCUCUGCUUCUGCU-3′) and 150&nbsp;μM NTPs in assay buffer (50&nbsp;mM HEPES pH 7.5, 200&nbsp;mM NaCl, 10% (v/v) glycerol, 2&nbsp;mM TCEP, 0.05% Tween20) supplemented with 5&nbsp;mM MgCl<sub>2</sub>. Samples were incubated at 25°C and reaction was quenched by 4.5&nbsp;M NaCl after 1, 1.5, 4, 7, 16, 20, 31, 45&nbsp;min, 1h 30, 2h 15, 3h 30, 5h 30, 19h 30, 21h 30. After reaching equilibrium, fluorescence polarization was measured with a CLARIOstar microplate reader (BMG Labtech) at 25°C using excitation and emission filters of 485 and 520&nbsp;nm, respectively. Progress curves were fitted into double exponential equation with bi-phasic reaction kinetics:<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f left parenthesis t right parenthesis equals negative upper A dot e Superscript minus k 1 t Baseline minus upper B dot e Superscript minus k 2 t Baseline plus upper C" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(41 (35 0 34 (4 1 2 3)) 5 (40 (38 (37 (36 6 7) 8 (18 9 (17 10 (16 (13 11 12) 15 14)))) 19 (39 20 21 (31 22 (30 23 (29 (26 24 25) 28 27))))) 32 33))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="41" data-semantic-children="35,40" data-semantic-content="5" data-semantic-owns="35 5 40" data-semantic-speech="f left parenthesis t right parenthesis equals negative upper A dot e Superscript minus k 1 t Baseline minus upper B dot e Superscript minus k 2 t Baseline plus upper C"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-children="0,4" data-semantic-content="34,0" data-semantic-parent="41" data-semantic-owns="0 34 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="35" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="35" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="41" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="40" data-semantic-children="38,33" data-semantic-content="32" data-semantic-parent="41" data-semantic-owns="38 32 33"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="38" data-semantic-children="37,39" data-semantic-content="19" data-semantic-parent="40" data-semantic-owns="37 19 39"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="37" data-semantic-children="36,18" data-semantic-content="8" data-semantic-parent="38" data-semantic-owns="36 8 18"><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-children="7" data-semantic-content="6" data-semantic-parent="37" data-semantic-owns="6 7"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="36" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="36"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="37" data-semantic-operator="infixop,·"><mjx-c class="mjx-cB7">·</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="9,17" data-semantic-parent="37" data-semantic-owns="9 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="18"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="17" data-semantic-children="16" data-semantic-content="10" data-semantic-parent="18" data-semantic-owns="10 16"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="10" data-semantic-parent="17" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-children="13,14" data-semantic-content="15" data-semantic-parent="17" data-semantic-owns="13 15 14"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="16" data-semantic-owns="11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="19" data-semantic-parent="38" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="39" data-semantic-children="20,31" data-semantic-content="21" data-semantic-parent="38" data-semantic-owns="20 21 31"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="39"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="39" data-semantic-operator="infixop,·"><mjx-c class="mjx-cB7">·</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="31" data-semantic-children="22,30" data-semantic-parent="39" data-semantic-owns="22 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="31"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="30" data-semantic-children="29" data-semantic-content="23" data-semantic-parent="31" data-semantic-owns="23 29"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="23" data-semantic-parent="30" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="29" data-semantic-children="26,27" data-semantic-content="28" data-semantic-parent="30" data-semantic-owns="26 28 27"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="24,25" data-semantic-parent="29" data-semantic-owns="24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="29"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="32" data-semantic-parent="40" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="40"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>where <i>t</i> is reaction time, <i>A</i> and <i>B</i> are observed signal amplitudes, k<sub>1</sub> and k<sub>2</sub> are observed rate constants for corresponding phases and <i>C</i> is final FP signal. Data were analyzed and fitted using GraphPad Prism.</div></section><section id="sec-4-4-9"><h4>Fluorescence polarization-based RNA binding assay</h4><div id="p0245" role="paragraph">Binding assays were performed with short fluorescently labeled wild-type (5′-UCUGCU-3′), mutant (5′-UCACGA-3′) and negative control (5′-AAAAAA-3′) RNA (IBA Lifesciences). Binding reactions were carried with 0.02&nbsp;μM FAM-labeled RNA and increasing concentration of wild-type or mutant polymerase in assay buffer (50&nbsp;mM HEPES pH 7.5, 200&nbsp;mM NaCl, 10% (v/v) glycerol, 2&nbsp;mM TCEP, 0.05% Tween20). The binding reactions were set-up in 20&nbsp;μL volume and incubated at room temperature for about 30&nbsp;min. After reaching equilibrium, fluorescence polarization was measured with a CLARIOstar microplate reader (BMG Labtech) at 25°C using excitation and emission filters of 485 and 520&nbsp;nm, respectively. Recorded fluorescence polarization signal was corrected for changes in fluorescence intensity according to the equation:<div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript b Baseline equals StartStartFraction left parenthesis upper F upper P minus upper F upper P Subscript f r e e Baseline right parenthesis OverOver left parenthesis StartFraction upper I Subscript b o u n d Baseline Over upper I Subscript f r e e Baseline EndFraction right parenthesis dot left parenthesis upper F upper P Subscript b o u n d Baseline minus upper F upper P right parenthesis plus upper F upper P minus upper F upper P Subscript f r e e Baseline EndEndFraction" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(100 (2 0 1) 3 (99 (25 4 (23 (20 5 19 6) 7 (22 8 21 (18 9 (17 10 14 11 15 12 16 13)))) 24) (98 (95 (92 (51 26 (49 (38 27 (37 28 33 29 34 30 35 31 36 32)) (48 39 (47 40 44 41 45 42 46 43))) 50) 52 (76 53 (74 (71 54 70 (66 55 (65 56 61 57 62 58 63 59 64 60))) 67 (73 68 72 69)) 75)) 77 (94 78 93 79)) 80 (97 81 96 (91 82 (90 83 87 84 88 85 89 86))))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="100" data-semantic-children="2,99" data-semantic-content="3" data-semantic-owns="2 3 99" data-semantic-speech="f Subscript b Baseline equals StartStartFraction left parenthesis upper F upper P minus upper F upper P Subscript f r e e Baseline right parenthesis OverOver left parenthesis StartFraction upper I Subscript b o u n d Baseline Over upper I Subscript f r e e Baseline EndFraction right parenthesis dot left parenthesis upper F upper P Subscript b o u n d Baseline minus upper F upper P right parenthesis plus upper F upper P minus upper F upper P Subscript f r e e Baseline EndEndFraction"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="100" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="100" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="99" data-semantic-children="25,98" data-semantic-parent="100" data-semantic-owns="25 98"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="25" data-semantic-children="23" data-semantic-content="4,24" data-semantic-parent="99" data-semantic-owns="4 23 24"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="25" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="23" data-semantic-children="20,22" data-semantic-content="7" data-semantic-parent="25" data-semantic-owns="20 7 22"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="20" data-semantic-children="5,6" data-semantic-content="19" data-semantic-parent="23" data-semantic-owns="5 19 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="20"><mjx-c class="mjx-c1D439">𝐹</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="20"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="7" data-semantic-parent="23" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="8,18" data-semantic-content="21" data-semantic-parent="23" data-semantic-owns="8 21 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="22"><mjx-c class="mjx-c1D439">𝐹</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="9,17" data-semantic-parent="22" data-semantic-owns="9 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="18"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.112em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="17" data-semantic-children="10,11,12,13" data-semantic-content="14,15,16" data-semantic-parent="18" data-semantic-owns="10 14 11 15 12 16 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="17"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="17"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="17"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="17"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="24" data-semantic-parent="25" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="98" data-semantic-children="95,97" data-semantic-content="80" data-semantic-parent="99" data-semantic-owns="95 80 97"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="95" data-semantic-children="92,94" data-semantic-content="77" data-semantic-parent="98" data-semantic-owns="92 77 94"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="92" data-semantic-children="51,76" data-semantic-content="52" data-semantic-parent="95" data-semantic-owns="51 52 76"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="51" data-semantic-children="49" data-semantic-content="26,50" data-semantic-parent="92" data-semantic-owns="26 49 50"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="26" data-semantic-parent="51" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S4">(</mjx-c></mjx-mo><mjx-mrow><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="49" data-semantic-children="38,48" data-semantic-parent="51" data-semantic-owns="38 48"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="38" data-semantic-children="27,37" data-semantic-parent="49" data-semantic-owns="27 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="38"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.34em; margin-left: -0.057em;"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="37" data-semantic-children="28,29,30,31,32" data-semantic-content="33,34,35,36" data-semantic-parent="38" data-semantic-owns="28 33 29 34 30 35 31 36 32"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="37"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="37"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="37"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="37"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="37"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="48" data-semantic-children="39,47" data-semantic-parent="49" data-semantic-owns="39 47"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="48"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.351em; margin-left: -0.057em;"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="47" data-semantic-children="40,41,42,43" data-semantic-content="44,45,46" data-semantic-parent="48" data-semantic-owns="40 44 41 45 42 46 43"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="47"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="44" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="47"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="47"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="46" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="47"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="50" data-semantic-parent="51" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S4">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="52" data-semantic-parent="92" data-semantic-operator="infixop,·"><mjx-c class="mjx-cB7">·</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="76" data-semantic-children="74" data-semantic-content="53,75" data-semantic-parent="92" data-semantic-owns="53 74 75"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="53" data-semantic-parent="76" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="74" data-semantic-children="71,73" data-semantic-content="67" data-semantic-parent="76" data-semantic-owns="71 67 73"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="71" data-semantic-children="54,66" data-semantic-content="70" data-semantic-parent="74" data-semantic-owns="54 70 66"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="54" data-semantic-parent="71"><mjx-c class="mjx-c1D439">𝐹</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="70" data-semantic-parent="71" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="66" data-semantic-children="55,65" data-semantic-parent="71" data-semantic-owns="55 65"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="55" data-semantic-parent="66"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.112em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="65" data-semantic-children="56,57,58,59,60" data-semantic-content="61,62,63,64" data-semantic-parent="66" data-semantic-owns="56 61 57 62 58 63 59 64 60"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="56" data-semantic-parent="65"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="61" data-semantic-parent="65" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="57" data-semantic-parent="65"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="62" data-semantic-parent="65" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="58" data-semantic-parent="65"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="63" data-semantic-parent="65" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="59" data-semantic-parent="65"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="64" data-semantic-parent="65" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="65"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="67" data-semantic-parent="74" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="73" data-semantic-children="68,69" data-semantic-content="72" data-semantic-parent="74" data-semantic-owns="68 72 69"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="68" data-semantic-parent="73"><mjx-c class="mjx-c1D439">𝐹</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="72" data-semantic-parent="73" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="69" data-semantic-parent="73"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="75" data-semantic-parent="76" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="77" data-semantic-parent="95" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="94" data-semantic-children="78,79" data-semantic-content="93" data-semantic-parent="95" data-semantic-owns="78 93 79"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="78" data-semantic-parent="94"><mjx-c class="mjx-c1D439">𝐹</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="93" data-semantic-parent="94" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="79" data-semantic-parent="94"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="80" data-semantic-parent="98" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="97" data-semantic-children="81,91" data-semantic-content="96" data-semantic-parent="98" data-semantic-owns="81 96 91"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="81" data-semantic-parent="97"><mjx-c class="mjx-c1D439">𝐹</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="96" data-semantic-parent="97" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="91" data-semantic-children="82,90" data-semantic-parent="97" data-semantic-owns="82 90"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="82" data-semantic-parent="91"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.112em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="90" data-semantic-children="83,84,85,86" data-semantic-content="87,88,89" data-semantic-parent="91" data-semantic-owns="83 87 84 88 85 89 86"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="83" data-semantic-parent="90"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="87" data-semantic-parent="90" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="84" data-semantic-parent="90"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="88" data-semantic-parent="90" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="85" data-semantic-parent="90"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="89" data-semantic-parent="90" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="86" data-semantic-parent="90"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-math></mjx-container></div></div></div>where f<sub>b</sub> represents the fractional concentration of bound RNA, FP the observed fluorescence polarization, FP<sub>free</sub> and FP<sub>bound</sub> the&nbsp;fluorescence polarization of free and bound RNA, and I<sub>free</sub> and I<sub>bound</sub> the fluorescence intensity of free and bound RNA, respectively. Dissociation constants (K<sub>D</sub>) were obtained using GraphPad Prism software by fitting fluorescence polarization data as a function of total concentration of protein (P) using the following equation:<div id="ufd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript b Baseline equals StartFraction left parenthesis upper L plus upper P plus upper K Subscript upper D Baseline minus StartRoot left parenthesis upper L plus upper P plus upper K Subscript upper D Baseline right parenthesis squared minus 4 upper L upper P right parenthesis EndRoot Over 2 upper L EndFraction" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(45 (2 0 1) 3 (44 (39 4 (38 (37 5 6 7 8 (11 9 10)) 12 (36 (35 (33 (25 (23 13 (21 14 15 16 17 (20 18 19)) 22) 24) 26 (32 27 30 28 31 29)) 34)))) (43 40 42 41)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="45" data-semantic-children="2,44" data-semantic-content="3" data-semantic-owns="2 3 44" data-semantic-speech="f Subscript b Baseline equals StartFraction left parenthesis upper L plus upper P plus upper K Subscript upper D Baseline minus StartRoot left parenthesis upper L plus upper P plus upper K Subscript upper D Baseline right parenthesis squared minus 4 upper L upper P right parenthesis EndRoot Over 2 upper L EndFraction"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="45" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="45" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="44" data-semantic-children="39,43" data-semantic-parent="45" data-semantic-owns="39 43"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="39" data-semantic-children="4,38" data-semantic-content="4" data-semantic-parent="44" data-semantic-owns="4 38"><mjx-mo data-semantic-type="punctuation" data-semantic-role="openfence" data-semantic-id="4" data-semantic-parent="39" data-semantic-operator="punctuated"><mjx-c class="mjx-c28 MM-S5">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="38" data-semantic-children="37,36" data-semantic-content="12" data-semantic-parent="39" data-semantic-owns="37 12 36"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="37" data-semantic-children="5,7,11" data-semantic-content="6,8" data-semantic-parent="38" data-semantic-owns="5 6 7 8 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="37"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="6" data-semantic-parent="37" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="37"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="8" data-semantic-parent="37" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="37" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="12" data-semantic-parent="38" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="36" data-semantic-children="35" data-semantic-parent="38" data-semantic-owns="35"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A MM-LO">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.185em;"><mjx-mrow><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="35" data-semantic-children="33,34" data-semantic-content="34" data-semantic-parent="36" data-semantic-owns="33 34"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="33" data-semantic-children="25,32" data-semantic-content="26" data-semantic-parent="35" data-semantic-owns="25 26 32"><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="25" data-semantic-children="23,24" data-semantic-parent="33" data-semantic-owns="23 24"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="23" data-semantic-children="21" data-semantic-content="13,22" data-semantic-parent="25" data-semantic-owns="13 21 22"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="13" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="21" data-semantic-children="14,16,20" data-semantic-content="15,17" data-semantic-parent="23" data-semantic-owns="14 15 16 17 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="21"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="15" data-semantic-parent="21" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="21"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="17" data-semantic-parent="21" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="21" data-semantic-owns="18 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="22" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="25"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="26" data-semantic-parent="33" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="32" data-semantic-children="27,28,29" data-semantic-content="30,31" data-semantic-parent="33" data-semantic-owns="27 30 28 31 29"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="32"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="32"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="32"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="closefence" data-semantic-id="34" data-semantic-parent="35" data-semantic-operator="punctuated"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-box></mjx-sqrt></mjx-msqrt></mjx-mrow></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="43" data-semantic-children="40,41" data-semantic-content="42" data-semantic-parent="44" data-semantic-owns="40 42 41"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="43"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="42" data-semantic-parent="43" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="43"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-math></mjx-container></div></div></div>where f<sub>b</sub> represents the fractional concentration of bound RNA, L the total concentration of labeled RNA, P the concentration of protein and K<sub>D</sub> the dissociation constant. For the competition assay, 0.04&nbsp;μM wild-type polymerase was pre-bound with an equimolar quantity of FAM-labeled short RNA (5′-UCUGCU-3′) and increasing amounts of unlabeled RNA competitor were added. Data were recorded and fitted in the same manner as for direct binding experiments.</div></section><section id="sec-4-4-10"><h4>CryoEM sample preparation and data collections</h4><div id="p0250" role="paragraph">To capture distinct states of transcription cycle, state specific vRNA templates and sample preparation protocols (see below) were applied. Either wild-type polymerase with endonuclease inhibitor or an endonuclease-inactive polymerase (PA/K134A) were used to prevent possible endonuclease mediated vRNA template or product degradation. All complexes were assembled in cryoEM buffer (50&nbsp;mM HEPES pH 7.5, 150&nbsp;mM NaCl, 5&nbsp;mM MgCl<sub>2</sub>, 2&nbsp;mM TCEP).<div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="content"><div id="p0255" role="paragraph">Sample 1: For the pre-initiation complex, equimolar amounts of bat FluA polymerase at 0.9&nbsp;μM (prebound to 35&nbsp;μM endonuclease inhibitor (ENDO-483)), 3′ vRNA (16-mer) and 5′ vRNA (14-mer) in cryoEM buffer were incubated on ice for 15&nbsp;min.</div></div></div><div id="u0035" role="listitem"><div class="content"><div id="p0260" role="paragraph">Sample 2: For the elongation complex, 0.9&nbsp;μM bat FluA polymerase (pre-bound to 35&nbsp;μM endonuclease inhibitor ENDO483), 0.81&nbsp;μM 3′ vRNA (18+3-mer) and 5′ vRNA (14-mer) and 1.8&nbsp;μM 15-mer capped were mixed in cryoEM buffer and incubated for 5 hours at 28°C with 100&nbsp;μM ATP and GTP. Additionally, 100&nbsp;μM CTP and 300&nbsp;μM UpNHpp were added for 2&nbsp;minutes at 28°C and immediately cooled to 4°C.</div></div></div><div id="u0040" role="listitem"><div class="content"><div id="p0265" role="paragraph">Sample 3 and 4: For the pre-termination and termination complexes, equimolar amounts of bat FluA polymerase at 0.85&nbsp;μM (pre-bound to 35&nbsp;μM endonuclease inhibitor ENDO483) and the v44 template were mixed with 2.5 molar excess of 13-mer capped RNA primer, 100&nbsp;μM ATP, CTP, GTP and 300&nbsp;μM UpNHpp in cryoEM buffer and incubated at 28°C for 4 h. For the termination complex, 100&nbsp;μM UTP instead of UpNHpp was used.</div></div></div><div id="u0045" role="listitem"><div class="content"><div id="p0270" role="paragraph">Sample 5: For the stuttering complex, equimolar amounts of bat FluA PA/K134A polymerase at 0.7&nbsp;μM and the template v44-6U (44-mer template RNA with 6xU polyadenylation signal) were mixed with 2.5 molar excess of 13-mer capped RNA primer, 170&nbsp;μM ATP and 17&nbsp;μM CTP, GTP and UTP in cryoEM buffer and incubated at 30°C for 1.5 h.</div></div></div><div id="u0050" role="listitem"><div class="content"><div id="p0275" role="paragraph">Sample 6: For the recycling complex, equimolar amounts of bat FluA polymerase at 0.9&nbsp;μM (pre-bound to 35&nbsp;μM endonuclease inhibitor ENDO483) and template vRNA7L (34-mer) in cryoEM buffer were incubated on ice for 20&nbsp;min.</div></div></div><div id="u0055" role="listitem"><div class="content"><div id="p0280" role="paragraph">Sample 7: For the FluB recycling complex, wild-type human FluB polymerase at 1.2&nbsp;μM was incubated with 1.25 molar excess of template vRNA7L (34-mer) and 12-mer capped RNA primer in cryoEM buffer.</div></div></div></div></div><div id="p0285" role="paragraph">Aliquots of 3&nbsp;μl were applied to glow discharged grids (R1.2/1.3, Au 300, Quantifoil), blotted for 2&nbsp;s and immediately plunge-frozen&nbsp;in liquid ethane using an FEI Vitrobot IV at 4°C and 100 % humidity. Grids were loaded onto 300 kV FEI Titan Krios or 200 kV FEI Talos Glacios electron microscopes and data were acquired in electron counting mode. Further details regarding each data collection are presented in <a id="crosref1100" href="#mmc2">Table S1</a>.</div></section><section id="sec-4-4-11"><h4>CryoEM data processing and structure determination</h4><div id="p0295" role="paragraph">CryoEM data processing was performed according to previously published strategy (<span class="dropBlock reference-citations"><a id="body-ref-sref14-10" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-10" aria-controls="sref14-10">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and additional details regarding each dataset can be found in <a id="crosref1125" href="#mmc2">Table S1</a> and <a id="crosref1130" href="#mmc1">Methods S1</a>. Briefly, in order to limit radiation damage, only selected initial frames from the collected movies were aligned using MotionCorr2 (<span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85014129582" target="_blank" aria-label="Open Scopus in new tab">Scopus (4513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28250466/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4193&amp;pmid=28250466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and then used for contrast transfer function parameter calculation by Gctf (<span class="dropBlock reference-citations"><a id="body-ref-sref47-2" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-2" aria-controls="sref47-2">Zhang, 2016</a><div class="dropBlock__holder" data-db-target-of="sref47-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2015.11.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84955216953" target="_blank" aria-label="Open Scopus in new tab">Scopus (2386)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26592709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2015.11.003&amp;pmid=26592709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Particle were picked using either WARP (<span class="dropBlock reference-citations"><a id="body-ref-sref42-2" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-2" aria-controls="sref42-2">Tegunov and Cramer, 2019</a><div class="dropBlock__holder" data-db-target-of="sref42-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Tegunov, D. ∙ Cramer, P.</div><div><strong>Real-time cryo-electron microscopy data preprocessing with Warp</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1146-1152</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0580-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85074279272" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31591575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0580-y&amp;pmid=31591575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or Gautomatch (by K. Zhang) and were next subjected to reference-free two-dimensional (2D) classification using RELION 3.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Scheres, 2012</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Scheres, S.H.</div><div><strong>RELION: implementation of a Bayesian approach to cryo-EM structure determination</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2012; <strong>180</strong>:519-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2012.09.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84868444740" target="_blank" aria-label="Open Scopus in new tab">Scopus (3642)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23000701/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2012.09.006&amp;pmid=23000701" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After several rounds of 2D classifications on binned images, particles in classes with poor structural features were discarded. Classes containing good quality particles were subjected to 3D classification with alignment. Next, the classes representing similar conformational states were pooled together for global 3D auto-refinement and were subjected to another round of 3D classification with fine angular searches. This procedure was repeated until conformationally homogeneous classes were obtained. The resulting maps were improved by CTF refinement and Bayesian polishing procedures in RELION and further 3D auto-refined. The final cryoEM density maps were sharpened using post-processing procedure in RELION and the resolution was estimated using 0.143 gold standard Fourier Schell Correlation (FSC) cutoff. Local resolution maps were generated using RELION’s own implementation. Final maps were sharpened and blurred into MTZ format using CCP-EM (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Burnley et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Burnley, T. ∙ Palmer, C.M. ∙ Winn, M.</div><div><strong>Recent developments in the CCP-EM software suite</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2017; <strong>73</strong>:469-477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798317007859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85020201461" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28580908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798317007859&amp;pmid=28580908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and local amplitude scaling was performed using LocScale (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Jakobi et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Jakobi, A.J. ∙ Wilmanns, M. ∙ Sachse, C.</div><div><strong>Model-based local density sharpening of cryo-EM maps</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e27131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.27131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85036517329" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29058676/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.27131&amp;pmid=29058676" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The previously solved structures of bat influenza A polymerases (4WSB, but also models generated in this study) were rigid-body fitted into the EM densities using Molrep in CCP-EM and further model building of RNA and flexible domains was performed manually in COOT (<span class="dropBlock reference-citations"><a id="body-ref-sref6-3" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-3" aria-controls="sref6-3">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref6-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444910007493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-77949535720" target="_blank" aria-label="Open Scopus in new tab">Scopus (20200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444910007493&amp;pmid=20383002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Real-space refinement was done in PHENIX (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Afonine et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Afonine, P.V. ∙ Poon, B.K. ∙ Read, R.J. ...</div><div><strong>Real-space refinement in PHENIX for cryo-EM and crystallography</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:531-544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798318006551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85048343902" target="_blank" aria-label="Open Scopus in new tab">Scopus (1516)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798318006551&amp;pmid=29872004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the highest resolution structures, Ramachandran and side-chain restraints were not used. Structures were analyzed and figures were prepared using following software packages: PyMOL (<a id="intref0215" href="https://www.schrodinger.com/" target="_blank">Schrödinger</a>, Inc.) with APBS plugin (<span class="dropBlock reference-citations"><a id="body-ref-sref13-2" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-2" aria-controls="sref13-2">Jurrus et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref13-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Jurrus, E. ∙ Engel, D. ∙ Star, K. ...</div><div><strong>Improvements to the APBS biomolecular solvation software suite</strong></div><div><em>Protein Sci.</em> 2018; <strong>27</strong>:112-128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3280" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85039036425" target="_blank" aria-label="Open Scopus in new tab">Scopus (1239)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28836357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3280&amp;pmid=28836357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), USCF Chimera (<span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Pettersen et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF Chimera--a visualization system for exploratory research and analysis</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2004; <strong>25</strong>:1605-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.20084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-4444221565" target="_blank" aria-label="Open Scopus in new tab">Scopus (33444)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.20084&amp;pmid=15264254" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), CCP4MG (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">McNicholas et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>McNicholas, S. ∙ Potterton, E. ∙ Wilson, K.S. ...</div><div><strong>Presenting your structures: the CCP4mg molecular-graphics software</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2011; <strong>67</strong>:386-394</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444911007281" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-79953765276" target="_blank" aria-label="Open Scopus in new tab">Scopus (1069)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21460457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444911007281&amp;pmid=21460457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and LigPlot+ (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Laskowski and Swindells, 2011</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Laskowski, R.A. ∙ Swindells, M.B.</div><div><strong>LigPlot+: multiple ligand-protein interaction diagrams for drug discovery</strong></div><div><em>J.&nbsp;Chem. Inf. Model.</em> 2011; <strong>51</strong>:2778-2786</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ci200227u" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-80054911951" target="_blank" aria-label="Open Scopus in new tab">Scopus (4092)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21919503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fci200227u&amp;pmid=21919503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-12"><h4>Oligonucleotides</h4><div id="p0290" role="paragraph"><div class="figure-wrap"><figure id="undtbl2" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>Sequence</th><th>Source</th><th>Name</th></tr></thead><tbody><tr><td>(m<sup>7</sup>Gppp)AAUCUAUAAUAG</td><td>TriLink Biotechnologies</td><td>Capped primer (12-mer)</td></tr><tr><td>(m<sup>7</sup>Gppp)AAUCUAUAAUAGC</td><td>TriLink Biotechnologies</td><td>Capped primer (13-mer)</td></tr><tr><td>(m<sup>7</sup>Gppp)GAAUGCUAUAAUAGC</td><td>TriLink Biotechnologies</td><td>Capped primer (15-mer)</td></tr><tr><td>pAGUAGUAACAAGAG</td><td>IBA Lifesciences</td><td>5′ vRNA (14-mer)</td></tr><tr><td>UACCUCUGCUUCUGCU</td><td>IBA Lifesciences</td><td>3′ vRNA (16-mer)</td></tr><tr><td>UAUACCUCUGCUUCUGCU</td><td>IBA Lifesciences</td><td>3′ vRNA (18-mer)</td></tr><tr><td>UAUACCUCUGCUUCUGCUAUU</td><td>IBA Lifesciences</td><td>3′ vRNA (18+3-mer)</td></tr><tr><td>pAGUAGUAACAAGAGCAAUGUGUCCGUCUCG<br>CCUCUGCUUCUGCU</td><td>IBA Lifesciences</td><td>v44 (44-mer)</td></tr><tr><td>pAGUAGUAACAAGAGCAUUUUUUCCGUCUCG<br>CCUCUGCUUCUGCU</td><td>IBA Lifesciences</td><td>v44-6U (44-mer)</td></tr><tr><td>pAGUAGUAACAAGAGCACUUUUUUCCGUCU<br>CGCCUCUGCUUCUGCU</td><td>IBA Lifesciences</td><td>v45-6U (45-mer)</td></tr><tr><td>pAGUAGUAACAAGAGGUAUUACCUCUGC<br>UUCUGCU</td><td>IBA Lifesciences</td><td>vRNA7L (34-mer)</td></tr><tr><td>UCUGCU</td><td>IBA Lifesciences</td><td>3′ vRNA (6-mer)</td></tr><tr><td>UCACGA</td><td>IBA Lifesciences</td><td>3′ vRNA (mut)</td></tr><tr><td>AAAAAA</td><td>IBA Lifesciences</td><td>6A</td></tr><tr><td>(FAM-Ex-5)-UCUGCU</td><td>IBA Lifesciences</td><td>3′ vRNA (6-mer),<br>FAM-labeled</td></tr><tr><td>(FAM-Ex-5)-AAAAAA</td><td>IBA Lifesciences</td><td>6A, FAM-labeled</td></tr><tr><td>(FAM-Ex-5)-UAUACCUCUGCUUCUGCU</td><td>IBA Lifesciences</td><td>3′ vRNA (18-mer), FAM-labeled</td></tr><tr><td>(m<sup>7</sup>Gppp)-mO<sup>2</sup>GAAUCACAUAAAGACCAGGC</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref14-11" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-11" aria-controls="sref14-11">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>20-mer marker</td></tr><tr><td>(m<sup>7</sup>Gppp)-mO<sup>2</sup>GAAUCUAUACAUAAAGACCAGGC</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref14-12" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-12" aria-controls="sref14-12">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>23-mer marker</td></tr><tr><td>(m<sup>7</sup>Gppp)-mO<sup>2</sup>GAAUCUAUAAUAAACAUAAA<br>GACCAGGC</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref14-13" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-13" aria-controls="sref14-13">Kouba et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>28-mer marker</td></tr><tr><td>pAGUAGUAACAAGAGCAUUUUUCA<br>GACUCUGCUUCUGCU</td><td>IBA Lifesciences</td><td>RNA38 (38-mer)</td></tr><tr><td>AGCAGAAGCAGAGUCUGAAAAAUGC<br>UCUUGUUACUACU</td><td>IBA Lifesciences</td><td>RNA38-RC (38-mer)</td></tr><tr><td>GACTATTACTATGAAGCGGCTTCTAA<br>GCTCAAAGGAGAGAACG</td><td>Thermo Fisher</td><td>PA_K134A_fwd</td></tr><tr><td>CTTAGAAGCCGCTTCATAGTAATAGTCC<br>TCAACCTTCCGTCTGG</td><td>Thermo Fisher</td><td>PA_K134A_rev</td></tr><tr><td>AATCATAACCGAATGTCGGGACACCTCAGG</td><td>Thermo Fisher</td><td>PA_K483E_fwd</td></tr><tr><td>GTCCCGACATTCGGTTATGATTGGGATTAC<br>TTGGTAC</td><td>Thermo Fisher</td><td>PA_K483E_rev</td></tr><tr><td>CAGCTGTTCATAGAGGACTACAGATATAC<br>CTATAGGTGC</td><td>Thermo Fisher</td><td>PB1_K553E_fwd</td></tr><tr><td>TCTGTAGTCCTCTATGAACAGCTGTATGGCCATC</td><td>Thermo Fisher</td><td>PB1_K553E_rev</td></tr><tr><td>TATACCTATGAGTGCCATCGCGGCGATACCAAC</td><td>Thermo Fisher</td><td>PB1_R560E_fwd</td></tr><tr><td>GCGATGGCACTCATAGGTATATCTGTAGTCCTTT<br>ATGAACAGC</td><td>Thermo Fisher</td><td>PB1_R560E_rev</td></tr><tr><td>GACTACAGAGCTACCTATAGGTGCCATCGCGGCG</td><td>Thermo Fisher</td><td>PB1_Y557A_fwd</td></tr><tr><td>CCTATAGGTAGCTCTGTAGTCCTTTATG<br>AACAGCTGTATGGC</td><td>Thermo Fisher</td><td>PB1_Y557A_rev</td></tr><tr><td>CCTATAGGTAGCTCTGTAGTCCTCTATGAACAG<br>CTGTATGGC</td><td>Thermo Fisher</td><td>PB1_Y557A_rev2</td></tr><tr><td>ATGGCCATGGAGTATCCAATAAGCGCC</td><td>Thermo Fisher</td><td>PB2_K54E_fwd</td></tr><tr><td>ATTGGATACTCCATGGCCATCATCCATTTCATCC</td><td>Thermo Fisher</td><td>PB2_K54E_rev</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl2&amp;pii=S0092-8674%2820%2930389-5" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section></section><section id="sec-4-5"><h3>Quantification and Statistical Analysis</h3><div id="p0300" role="paragraph">Enzymatic and binding assays represent a mean of n independent experiments with standard deviation (SD) and the number of experiments (n) is indicated in legends to <a id="crosref1195" href="#fig4">Figures 4</a>E and <a id="crosref1200" href="#figs7">S7</a>A–S7C. Structural data were validated by Phenix validation tools and are presented in <a id="crosref1205" href="#mmc2">Tables S1</a> and <a id="crosref1210" href="#mmc2">S2</a>.</div></section><section id="sec-4-6"><h3>Data and Code Availability</h3><div id="p0305" role="paragraph">Co-ordinates and cryoEM maps generated during this study are available at Protein Data Bank (PDB) and Electron Microscopy Data Bank (EMDB) under following accession numbers:<div id="ulist0020" role="list"><div id="u0060" role="listitem"><div class="label">•</div><div class="content"><div id="p0310" role="paragraph">Bat Influenza A polymerase pre-initiation complex, PDB ID: <a id="intref0220" href="http://firstglance.jmol.org/fg.htm?mol6T0N" target="_blank">6T0N</a>, <a id="intref0225" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10356</div></div></div><div id="u0065" role="listitem"><div class="label">•</div><div class="content"><div id="p0315" role="paragraph">Bat Influenza A polymerase elongation complex with incoming UTP analog (core&nbsp;+ endonuclease only), PDB ID: <a id="intref0230" href="http://firstglance.jmol.org/fg.htm?mol6SZV" target="_blank">6SZV</a>, <a id="intref0235" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10355</div></div></div><div id="u0070" role="listitem"><div class="label">•</div><div class="content"><div id="p0320" role="paragraph">Bat Influenza A polymerase elongation complex with incoming UTP analog (complete polymerase), PDB ID: <a id="intref0240" href="http://firstglance.jmol.org/fg.htm?mol6T0V" target="_blank">6T0V</a>; <a id="intref0245" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10360</div></div></div><div id="u0075" role="listitem"><div class="label">•</div><div class="content"><div id="p0325" role="paragraph">Bat Influenza A polymerase pre-termination complex with pyrophosphate using 44-mer vRNA template with mutated oligo(U) sequence, PDB ID: <a id="intref0250" href="http://firstglance.jmol.org/fg.htm?mol6SZU" target="_blank">6SZU</a>, <a id="intref0255" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10354</div></div></div><div id="u0080" role="listitem"><div class="label">•</div><div class="content"><div id="p0330" role="paragraph">Bat Influenza A polymerase product dissociation complex using 44-mer vRNA template with mutated oligo(U) sequence, PDB ID: <a id="intref0260" href="http://firstglance.jmol.org/fg.htm?mol6T0R" target="_blank">6T0R</a>, <a id="intref0265" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10357</div></div></div><div id="u0085" role="listitem"><div class="label">•</div><div class="content"><div id="p0335" role="paragraph">Bat Influenza A polymerase termination complex with pyrophosphate using 44-mer vRNA template with mutated oligo(U) sequence, PDB ID: <a id="intref0270" href="http://firstglance.jmol.org/fg.htm?mol6TW1" target="_blank">6TW1</a>, <a id="intref0275" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10603</div></div></div><div id="u0090" role="listitem"><div class="label">•</div><div class="content"><div id="p0340" role="paragraph">Bat Influenza A polymerase stuttering complex using 44-mer vRNA template with intact oligo(U) sequence, PDB ID: <a id="intref0280" href="http://firstglance.jmol.org/fg.htm?mol6T0S" target="_blank">6T0S</a>, <a id="intref0285" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10358</div></div></div><div id="u0095" role="listitem"><div class="label">•</div><div class="content"><div id="p0345" role="paragraph">Bat Influenza A polymerase product dissociation complex using 44-mer vRNA template with intact oligo(U) sequence, PDB ID: <a id="intref0290" href="http://firstglance.jmol.org/fg.htm?mol6T0U" target="_blank">6T0U</a>, <a id="intref0295" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10359</div></div></div><div id="u0100" role="listitem"><div class="label">•</div><div class="content"><div id="p0350" role="paragraph">Bat Influenza A polymerase recycling complex, PDB ID: <a id="intref0300" href="http://firstglance.jmol.org/fg.htm?mol6T2C" target="_blank">6T2C</a>, <a id="intref0305" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10368</div></div></div><div id="u0105" role="listitem"><div class="label">•</div><div class="content"><div id="p0355" role="paragraph">Human Influenza B polymerase recycling complex, PDB ID: <a id="intref0310" href="http://firstglance.jmol.org/fg.htm?mol6T0W" target="_blank">6T0W</a>, <a id="intref0315" href="http://firstglance.jmol.org/fg.htm?molEMDB" target="_blank">EMDB</a> ID: 10361</div></div></div></div></div><div id="p0360" role="paragraph">Co-ordinates and structure factors generated during this study are available at Protein Data Bank (PDB) under following accession number:<div id="ulist0025a" role="list"><div id="u0125" role="listitem"><div class="label">•</div><div class="content"><div id="p0410" role="paragraph">Influenza A/H7N9 polymerase core (apo), PDB ID: <a id="interref375" href="http://firstglance.jmol.org/fg.htm?mol6TU5" target="_blank">6TU5</a></div></div></div></div></div><div id="p0365" role="paragraph">RNA sequencing data generated during this study are deposited at the European Nucleotide Archive (ENA) under study accession number PRJEB36795 with following accession numbers for each sample:<div id="ulist0025" role="list"><div id="u0110" role="listitem"><div class="label">•</div><div class="content"><div id="p0370" role="paragraph">Sample v44 (+AGC), ENA ID: SAMEA6563093</div></div></div><div id="u0115" role="listitem"><div class="label">•</div><div class="content"><div id="p0375" role="paragraph">Sample v44 (+AGCU), ENA ID: SAMEA6563094</div></div></div><div id="u0120" role="listitem"><div class="label">•</div><div class="content"><div id="p0380" role="paragraph">Sample v3′18+3 (+AGC), ENA ID: SAMEA6563095</div></div></div></div></div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0385" role="paragraph">We thank Felix Weis and Wim Hagen for access to the Titan Krios at EMBL Heidelberg; Michael Hons, Wojtek Galej, and Erika Pellegrini for access to the Glacios at EMBL Grenoble; and Aymeric Peuch for help using the EMBL-IBS computer cluster. We thank the EMBL-ESRF Joint Structural Biology group for access to beamline ID30-A3. We thank the EMBL Heidelberg GeneCore (Vladimir Benes), the EMBL Grenoble eukaryotic expression facility (Alice Aubert), and Stefan Reich for help with fluorescence data analysis. This work was supported by ANR (ANR-18-CE11-0028 to S.C.). T.K. holds an EMBL Interdisciplinary Postdoc (EI<sub>3</sub>POD) fellowship, co-hosted by Martin Beck (EMBL Heidelberg) and co-funded by the Marie Skłodowska-Curie Actions (664726, European Commission). This work used the platforms of the Grenoble Instruct-ERIC center (ISBG; UMS 3518 CNRS-CEA-UGA-EMBL) within the Grenoble Partnership for Structural Biology (PSB), supported by FRISBI (ANR-10-INBS-05-02) and GRAL, financed by the University Grenoble Alpes Ecoles Universitaires de Recherche CBH-EUR-GS (ANR-17-EURE-0003).</div><section id="sec-5"><h3>Author Contributions</h3><div id="p0390" role="paragraph">J.M.W. and P.D. expressed and purified bat FluA and FluB polymerase. J.M.W. performed and analyzed the FP-based assays. J.M.W. and T.K. performed transcription assays. M.K. pioneered cryo-EM structure determination of influenza polymerase and collected and analyzed the data on FluB polymerase. A.P. cloned, produced, crystallized, and solved the structure of the A/H7N9 polymerase core. N.A. performed RNA sequencing. J.P. analyzed the sequencing results. J.M.W. and T.K. prepared cryo-EM grids, collected cryo-EM data, performed image processing and 3D reconstruction, and built initial models. S.C. conceived and supervised the project and refined atomic models. J.M.W., T.K., and S.C. prepared the manuscript and figures. P.D. made the schematic animation.</div></section><section id="sec-6"><h3>Declaration of Interests</h3><div id="p0395" role="paragraph">S.C., T.K., and P.D. have a patent application related to this work.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (2)</h2><div id="p0405" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.03.061/attachment/4cfe0507-2e1c-4c83-a5b8-57cc06a3a843/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (13.47 MB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Methods S1. Flowchart of Cryo-EM Data Processing Strategy for Transcribing Influenza Polymerase Structures, Related to Figures 1 and S1</div><div id="espara0020" role="paragraph">Pre-initiation (A), elongation (B), pre-termination (C), termination (D), stuttering (E) and recycling complexes (F) and for human FluB polymerase recycling complex (G).</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.03.061/attachment/c7e666b4-c61c-42b6-af0c-b8fd71d63fee/mmc2.pdf" download="mmc2.pdf" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">PDF (597.61 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Document S1. Tables S1–S3</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Afonine, P.V. ∙ Poon, B.K. ∙ Read, R.J. ...</div><div><strong>Real-space refinement in PHENIX for cryo-EM and crystallography</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:531-544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798318006551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85048343902" target="_blank" aria-label="Open Scopus in new tab">Scopus (1516)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798318006551&amp;pmid=29872004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Arranz, R. ∙ Coloma, R. ∙ Chichón, F.J. ...</div><div><strong>The structure of native influenza virion ribonucleoproteins</strong></div><div><em>Science.</em> 2012; <strong>338</strong>:1634-1637</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1228172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84871460710" target="_blank" aria-label="Open Scopus in new tab">Scopus (245)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23180776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1228172&amp;pmid=23180776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Burnley, T. ∙ Palmer, C.M. ∙ Winn, M.</div><div><strong>Recent developments in the CCP-EM software suite</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2017; <strong>73</strong>:469-477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798317007859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85020201461" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28580908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798317007859&amp;pmid=28580908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="optsLAhWI6rZA" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optsLAhWI6rZA" title="View in article">4.</a></div><div>Coloma, R. ∙ Arranz, R. ∙ de la Rosa-Trevín, J.M. ...</div><div><strong>Structural insights into influenza A virus ribonucleoproteins reveal a processive helical track as transcription mechanism</strong></div><div><em>Nat. Microbiol.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0675-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85081665986" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32152587/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0675-3&amp;pmid=32152587" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#figs7" title="View in article">5.</a></div><div>Crooks, G.E. ∙ Hon, G. ∙ Chandonia, J.M. ...</div><div><strong>WebLogo: a sequence logo generator</strong></div><div><em>Genome Res.</em> 2004; <strong>14</strong>:1188-1190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.849004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-2142738304" target="_blank" aria-label="Open Scopus in new tab">Scopus (9365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15173120/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.849004&amp;pmid=15173120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">6.</a></div><div>Eisfeld, A.J. ∙ Neumann, G. ∙ Kawaoka, Y.</div><div><strong>At the centre: influenza A virus ribonucleoproteins</strong></div><div><em>Nat. Rev. Microbiol.</em> 2015; <strong>13</strong>:28-41</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrmicro3367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84924970829" target="_blank" aria-label="Open Scopus in new tab">Scopus (309)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25417656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrmicro3367&amp;pmid=25417656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">7.</a></div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444910007493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-77949535720" target="_blank" aria-label="Open Scopus in new tab">Scopus (20200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444910007493&amp;pmid=20383002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7-1" title="View in article">8.</a></div><div>Fan, H. ∙ Walker, A.P. ∙ Carrique, L. ...</div><div><strong>Structures of influenza A virus RNA polymerase offer insight into viral genome replication</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:287-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1530-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85072133233" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31485076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1530-7&amp;pmid=31485076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">9.</a></div><div>Gerlach, P. ∙ Malet, H. ∙ Cusack, S. ...</div><div><strong>Structural Insights into Bunyavirus Replication and Its Regulation by the vRNA Promoter</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1267-1279</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2Fj.cell.2015.05.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84930486338" target="_blank" aria-label="Open Scopus in new tab">Scopus (139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26004069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.05.006&amp;pmid=26004069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="optlavlWAsZJq" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optlavlWAsZJq" title="View in article">10.</a></div><div>Girardot, C. ∙ Scholtalbers, J. ∙ Sauer, S. ...</div><div><strong>Je, a versatile suite to handle multiplexed NGS libraries with unique molecular identifiers</strong></div><div><em>BMC Bioinformatics.</em> 2016; <strong>17</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-016-1284-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84990841832" target="_blank" aria-label="Open Scopus in new tab">Scopus (105)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27717304/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-016-1284-2&amp;pmid=27717304" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-2" title="View in article">11.</a></div><div>Hay, A.J. ∙ Skehel, J.J. ∙ McCauley, J.</div><div><strong>Structure and synthesis of influenza virus complementary RNAs</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 1980; <strong>288</strong>:341-348</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.1980.0010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0019332259" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6103551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.1980.0010&amp;pmid=6103551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">12.</a></div><div>Hutchinson, E.C. ∙ Charles, P.D. ∙ Hester, S.S. ...</div><div><strong>Conserved and host-specific features of influenza virion architecture</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:4816</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms5816" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84918544086" target="_blank" aria-label="Open Scopus in new tab">Scopus (200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25226414/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms5816&amp;pmid=25226414" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">13.</a></div><div>Jakobi, A.J. ∙ Wilmanns, M. ∙ Sachse, C.</div><div><strong>Model-based local density sharpening of cryo-EM maps</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e27131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.27131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85036517329" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29058676/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.27131&amp;pmid=29058676" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">14.</a></div><div>Jorba, N. ∙ Coloma, R. ∙ Ortín, J.</div><div><strong>Genetic trans-complementation establishes a new model for influenza virus RNA transcription and replication</strong></div><div><em>PLoS Pathog.</em> 2009; <strong>5</strong>:e1000462</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1000462" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-67249108203" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19478885/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1000462&amp;pmid=19478885" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13-1" title="View in article">15.</a></div><div>Jurrus, E. ∙ Engel, D. ∙ Star, K. ...</div><div><strong>Improvements to the APBS biomolecular solvation software suite</strong></div><div><em>Protein Sci.</em> 2018; <strong>27</strong>:112-128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3280" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85039036425" target="_blank" aria-label="Open Scopus in new tab">Scopus (1239)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28836357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3280&amp;pmid=28836357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">16.</a></div><div>Kouba, T. ∙ Drncová, P. ∙ Cusack, S.</div><div><strong>Structural snapshots of actively transcribing influenza polymerase</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:460-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41594-019-0232-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066951902" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41594-019-0232-z&amp;pmid=31160782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">17.</a></div><div>Landeras-Bueno, S. ∙ Jorba, N. ∙ Pérez-Cidoncha, M. ...</div><div><strong>The splicing factor proline-glutamine rich (SFPQ/PSF) is involved in influenza virus transcription</strong></div><div><em>PLoS Pathog.</em> 2011; <strong>7</strong>:e1002397</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1002397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-81755166636" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22114566/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1002397&amp;pmid=22114566" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">18.</a></div><div>Laskowski, R.A. ∙ Swindells, M.B.</div><div><strong>LigPlot+: multiple ligand-protein interaction diagrams for drug discovery</strong></div><div><em>J.&nbsp;Chem. Inf. Model.</em> 2011; <strong>51</strong>:2778-2786</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ci200227u" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-80054911951" target="_blank" aria-label="Open Scopus in new tab">Scopus (4092)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21919503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fci200227u&amp;pmid=21919503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">19.</a></div><div>Le Sage, V. ∙ Nanni, A.V. ∙ Bhagwat, A.R. ...</div><div><strong>Non-Uniform and Non-Random Binding of Nucleoprotein to Influenza A and B Viral RNA</strong></div><div><em>Viruses.</em> 2018; <strong>10</strong>:E522</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/v10100522" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85054480616" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30257455/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fv10100522&amp;pmid=30257455" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">20.</a></div><div>Lee, N. ∙ Le Sage, V. ∙ Nanni, A.V. ...</div><div><strong>Genome-wide analysis of influenza viral RNA and nucleoprotein association</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:8968-8977</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx584" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85031022619" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28911100/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx584&amp;pmid=28911100" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-2" title="View in article">21.</a></div><div>Li, X. ∙ Palese, P.</div><div><strong>Characterization of the polyadenylation signal of influenza virus RNA</strong></div><div><em>J.&nbsp;Virol.</em> 1994; <strong>68</strong>:1245-1249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.68.2.1245-1249.1994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7507179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.68.2.1245-1249.1994&amp;pmid=7507179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">22.</a></div><div>Lukarska, M. ∙ Fournier, G. ∙ Pflug, A. ...</div><div><strong>Structural basis of an essential interaction between influenza polymerase and Pol II CTD</strong></div><div><em>Nature.</em> 2017; <strong>541</strong>:117-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature20594" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85016158967" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28002402/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature20594&amp;pmid=28002402" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#figs4" title="View in article">23.</a></div><div>Markham, N.R. ∙ Zuker, M.</div><div><strong>UNAFold: software for nucleic acid folding and hybridization</strong></div><div><em>Methods Mol. Biol.</em> 2008; <strong>453</strong>:3-31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-60327-429-6_1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84934438960" target="_blank" aria-label="Open Scopus in new tab">Scopus (847)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18712296/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-60327-429-6_1&amp;pmid=18712296" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">24.</a></div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnetjournal.</em> 2011; <strong>17</strong>:10-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.14806/ej.17.1.200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.14806%2Fej.17.1.200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23-1" title="View in article">25.</a></div><div>McCoy, A.J. ∙ Grosse-Kunstleve, R.W. ∙ Adams, P.D. ...</div><div><strong>Phaser crystallographic software</strong></div><div><em>J.&nbsp;Appl. Cryst.</em> 2007; <strong>40</strong>:658-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0021889807021206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-34447508216" target="_blank" aria-label="Open Scopus in new tab">Scopus (16083)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19461840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0021889807021206&amp;pmid=19461840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">26.</a></div><div>McNicholas, S. ∙ Potterton, E. ∙ Wilson, K.S. ...</div><div><strong>Presenting your structures: the CCP4mg molecular-graphics software</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2011; <strong>67</strong>:386-394</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444911007281" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-79953765276" target="_blank" aria-label="Open Scopus in new tab">Scopus (1069)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21460457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444911007281&amp;pmid=21460457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">27.</a></div><div>Miyagawa, M. ∙ Akiyama, T. ∙ Mikamiyama-Iwata, M. ...</div><div><strong>Discovery of novel 5-hydroxy-4-pyridone-3-carboxy acids as potent inhibitors of influenza Cap-dependent endonuclease</strong></div><div><em>Bioorg. Med. Chem. Lett.</em> 2016; <strong>26</strong>:4739-4742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bmcl.2016.08.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84985995054" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27568084/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bmcl.2016.08.038&amp;pmid=27568084" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">28.</a></div><div>Murshudov, G.N. ∙ Vagin, A.A. ∙ Dodson, E.J.</div><div><strong>Refinement of macromolecular structures by the maximum-likelihood method</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 1997; <strong>53</strong>:240-255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444996012255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0030924992" target="_blank" aria-label="Open Scopus in new tab">Scopus (14211)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15299926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444996012255&amp;pmid=15299926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">29.</a></div><div>Navarro, G.</div><div><strong>NR‐grep: a fast and flexible pattern-matching tool</strong></div><div><em>Softw. Pract. Exper.</em> 2001; <strong>31</strong>:1265-1312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/spe.411" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0035841458" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fspe.411" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">30.</a></div><div>Nie, Y. ∙ Bellon-Echeverria, I. ∙ Trowitzsch, S. ...</div><div><strong>Multiprotein complex production in insect cells by using polyproteins</strong></div><div><em>Methods Mol. Biol.</em> 2014; <strong>1091</strong>:131-141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-62703-691-7_8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84934441860" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24203328/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-62703-691-7_8&amp;pmid=24203328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">31.</a></div><div>Ortega, J. ∙ Martín-Benito, J. ∙ Zürcher, T. ...</div><div><strong>Ultrastructural and functional analyses of recombinant influenza virus ribonucleoproteins suggest dimerization of nucleoprotein during virus amplification</strong></div><div><em>J.&nbsp;Virol.</em> 2000; <strong>74</strong>:156-163</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.74.1.156-163.2000" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-17644443084" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10590102/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.74.1.156-163.2000&amp;pmid=10590102" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">32.</a></div><div>Peng, Q. ∙ Liu, Y. ∙ Peng, R. ...</div><div><strong>Structural insight into RNA synthesis by influenza D polymerase</strong></div><div><em>Nat. Microbiol.</em> 2019; <strong>4</strong>:1750-1759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-019-0487-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85068108611" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-019-0487-5&amp;pmid=31209309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">33.</a></div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF Chimera--a visualization system for exploratory research and analysis</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2004; <strong>25</strong>:1605-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.20084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-4444221565" target="_blank" aria-label="Open Scopus in new tab">Scopus (33444)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.20084&amp;pmid=15264254" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="optqxjmgDaVgc" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optqxjmgDaVgc-1" title="View in article">34.</a></div><div>Pflug, A. ∙ Guilligay, D. ∙ Reich, S. ...</div><div><strong>Structure of influenza A polymerase bound to the viral RNA promoter</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:355-360</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84922257981" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14008&amp;pmid=25409142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">35.</a></div><div>Pflug, A. ∙ Lukarska, M. ∙ Resa-Infante, P. ...</div><div><strong>Structural insights into RNA synthesis by the influenza virus transcription-replication machine</strong></div><div><em>Virus Res.</em> 2017; <strong>234</strong>:103-117</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virusres.2017.01.013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85011796730" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28115197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virusres.2017.01.013&amp;pmid=28115197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">36.</a></div><div>Pflug, A. ∙ Gaudon, S. ∙ Resa-Infante, P. ...</div><div><strong>Capped RNA primer binding to influenza polymerase and implications for the mechanism of cap-binding inhibitors</strong></div><div><em>Nucleic. Acids Res.</em> 2018; <strong>46</strong>:956-971</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85044658250" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29202182/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1210&amp;pmid=29202182" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">37.</a></div><div>Plotch, S.J. ∙ Bouloy, M. ∙ Ulmanen, I. ...</div><div><strong>A unique cap(m7GpppXm)-dependent influenza virion endonuclease cleaves capped RNAs to generate the primers that initiate viral RNA transcription</strong></div><div><em>Cell.</em> 1981; <strong>23</strong>:847-858</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2F0092-8674%2881%2990449-9&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=10.1016%2F0092-8674%2881%2990449-9&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-0019394947" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6261960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2881%2990449-9&amp;pmid=6261960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">38.</a></div><div>Poon, L.L. ∙ Pritlove, D.C. ∙ Fodor, E. ...</div><div><strong>Direct evidence that the poly(A) tail of influenza A virus mRNA is synthesized by reiterative copying of a U track in the virion RNA template</strong></div><div><em>J.&nbsp;Virol.</em> 1999; <strong>73</strong>:3473-3476</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.73.4.3473-3476.1999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10074205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.73.4.3473-3476.1999&amp;pmid=10074205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">39.</a></div><div>Pyle, J.D. ∙ Whelan, S.P.J.</div><div><strong>RNA ligands activate the Machupo virus polymerase and guide promoter usage</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:10518-10524</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1900790116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066099874" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31072925/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1900790116&amp;pmid=31072925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">40.</a></div><div>Reich, S. ∙ Guilligay, D. ∙ Pflug, A. ...</div><div><strong>Structural insight into cap-snatching and RNA synthesis by influenza polymerase</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:361-366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84922245983" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14009&amp;pmid=25409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">41.</a></div><div>Reich, S. ∙ Guilligay, D. ∙ Cusack, S.</div><div><strong>An in&nbsp;vitro fluorescence based study of initiation of RNA synthesis by influenza B polymerase</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:3353-3368</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28126917/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=28126917" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">42.</a></div><div>Russell, A.B. ∙ Trapnell, C. ∙ Bloom, J.D.</div><div><strong>Extreme heterogeneity of influenza virus infection in single cells</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e32303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.32303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85043531075" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29451492/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.32303&amp;pmid=29451492" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">43.</a></div><div>Scheres, S.H.</div><div><strong>RELION: implementation of a Bayesian approach to cryo-EM structure determination</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2012; <strong>180</strong>:519-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2012.09.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84868444740" target="_blank" aria-label="Open Scopus in new tab">Scopus (3642)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23000701/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2012.09.006&amp;pmid=23000701" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">44.</a></div><div>Te Velthuis, A.J. ∙ Fodor, E.</div><div><strong>Influenza virus RNA polymerase: insights into the mechanisms of viral RNA synthesis</strong></div><div><em>Nat. Rev. Microbiol.</em> 2016; <strong>14</strong>:479-493</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrmicro.2016.87" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84978162055" target="_blank" aria-label="Open Scopus in new tab">Scopus (307)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27396566/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrmicro.2016.87&amp;pmid=27396566" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42-1" title="View in article">45.</a></div><div>Tegunov, D. ∙ Cramer, P.</div><div><strong>Real-time cryo-electron microscopy data preprocessing with Warp</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1146-1152</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0580-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85074279272" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31591575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0580-y&amp;pmid=31591575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">46.</a></div><div>Tickle, I.J. ∙ Flensburg, C. ∙ Keller, P. ...</div><div><strong>STARANISO. Global Phasing Ltd.</strong></div><div><a id="intref0330" href="http://staraniso.globalphasing.org/cgi-bin/staraniso.cgi" target="_blank">http://staraniso.globalphasing.org/cgi-bin/staraniso.cgi</a><div>Date: 2018</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.J.TickleC.FlensburgP.KellerW.PaciorekA.SharffC.VonrheinG.BricogneSTARANISO.+Global+Phasing+Ltd.http%3A%2F%2Fstaraniso.globalphasing.org%2Fcgi-bin%2Fstaraniso.cgi2018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">47.</a></div><div>Vogel, D. ∙ Rosenthal, M. ∙ Gogrefe, N. ...</div><div><strong>Biochemical characterization of the Lassa virus L protein</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2019; <strong>294</strong>:8088-8100</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.RA118.006973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85066070162" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30926610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.RA118.006973&amp;pmid=30926610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">48.</a></div><div>Vreede, F.T. ∙ Jung, T.E. ∙ Brownlee, G.G.</div><div><strong>Model suggesting that replication of influenza virus is regulated by stabilization of replicative intermediates</strong></div><div><em>J.&nbsp;Virol.</em> 2004; <strong>78</strong>:9568-9572</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.78.17.9568-9572.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-4143153932" target="_blank" aria-label="Open Scopus in new tab">Scopus (173)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15308750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.78.17.9568-9572.2004&amp;pmid=15308750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">49.</a></div><div>Ye, Q. ∙ Krug, R.M. ∙ Tao, Y.J.</div><div><strong>The mechanism by which influenza A virus nucleoprotein forms oligomers and binds RNA</strong></div><div><em>Nature.</em> 2006; <strong>444</strong>:1078-1082</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05379" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-33845890539" target="_blank" aria-label="Open Scopus in new tab">Scopus (369)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17151603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05379&amp;pmid=17151603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47-1" title="View in article">50.</a></div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2015.11.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-84955216953" target="_blank" aria-label="Open Scopus in new tab">Scopus (2386)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26592709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2015.11.003&amp;pmid=26592709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">51.</a></div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.061&amp;key=2-s2.0-85014129582" target="_blank" aria-label="Open Scopus in new tab">Scopus (4513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28250466/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4193&amp;pmid=28250466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (15)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/7b4120f8-32e4-4e50-b2aa-c641778655cf/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/2c14656f-15d9-4391-9ad7-4f6c0993041f/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/2cce2733-2cce-4978-882c-10181f95129c/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/ac064d5d-3fb7-4eff-b583-d37630525113/main.assets/figs3.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/435f6df9-9c91-4081-ad89-8be7c17652c0/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/dcc49d79-b328-4d1c-9b87-2ee3b9aaf700/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/cda010d9-4a06-4678-92ed-fd90714706a4/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/f0d4d02f-81c3-45f6-b916-9b8e38c8bcd5/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/5363c576-6ea3-4f60-ac91-26956886ac6b/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/f1aafa48-73a4-43c9-a848-59368db99670/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/2d2faf51-1b64-49ba-9a4a-f0fa8b2fb859/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/f0b6564a-b53d-4008-b17c-e8670d5831a9/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/82f00349-cfde-4abf-9516-02ad7270dd0d/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs8" aria-label="Go to Figure&amp;nbsp;S8"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/8abc2016-a3bc-498a-9dcd-75a66f95f673/main.assets/figs8.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/asset/d21414e6-d235-4c77-8b4d-3cc3b0ff59f3/main.assets/gr7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00112%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867420303895%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">93</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">144</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">1</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">33</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2020.03.061&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (2)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/attachment/4cfe0507-2e1c-4c83-a5b8-57cc06a3a843/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (13.47 MB)</span></a><div class="core-description">Methods S1. Flowchart of Cryo-EM Data Processing Strategy for Transcribing Influenza Polymerase Structures, Related to Figures 1 and S1</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.03.061/attachment/c7e666b4-c61c-42b6-af0c-b8fd71d63fee/mmc2.pdf" download="mmc2.pdf"><span id="mmc2-link">PDF (597.61 KB)</span></a><div class="core-description">Document S1. Tables S1–S3</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867420303895" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2020.03.061","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">A Structure-Based Model for the Complete Transcription Cycle of Influenza Polymerase</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.03.061/asset/7b4120f8-32e4-4e50-b2aa-c641778655cf/main.assets/gr1.jpg" height="817" width="749" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0ac709b2-2981-46f8-8362-cc2b1ee70a2d/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.03.061/asset/2c14656f-15d9-4391-9ad7-4f6c0993041f/main.assets/figs1.jpg" height="867" width="678" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/99538f76-533c-4b80-8b42-c9fbb4deee0a/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.03.061/asset/2cce2733-2cce-4978-882c-10181f95129c/main.assets/figs2.jpg" height="439" width="500" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/79d6c09c-8f76-4d6b-8775-93115dfd8337/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.03.061/asset/ac064d5d-3fb7-4eff-b583-d37630525113/main.assets/figs3.jpg" height="457" width="499" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/6d0c38c2-940a-4144-baa2-521151b8700c/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.03.061/asset/435f6df9-9c91-4081-ad89-8be7c17652c0/main.assets/gr2.jpg" height="873" width="678" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/c5b735b3-464f-463b-bd9d-8f3540c4cf67/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.03.061/asset/dcc49d79-b328-4d1c-9b87-2ee3b9aaf700/main.assets/figs4.jpg" height="518" width="762" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/9faf2324-3b04-4727-aa7c-705eedade1ad/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.03.061/asset/cda010d9-4a06-4678-92ed-fd90714706a4/main.assets/figs5.jpg" height="740" width="504" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/216fa9b9-2075-49fd-bef9-2a9932aadffa/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.03.061/asset/f0d4d02f-81c3-45f6-b916-9b8e38c8bcd5/main.assets/figs6.jpg" height="949" width="645" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/62429aee-c0d9-4c56-b3aa-0233b6a00b07/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.03.061/asset/5363c576-6ea3-4f60-ac91-26956886ac6b/main.assets/gr3.jpg" height="910" width="499" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/f1832c02-460c-47f6-bb23-01724944011d/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.03.061/asset/f1aafa48-73a4-43c9-a848-59368db99670/main.assets/gr4.jpg" height="891" width="679" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/c12315ac-294d-4bc6-b2ad-e2cb0d5e3943/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2020.03.061/asset/2d2faf51-1b64-49ba-9a4a-f0fa8b2fb859/main.assets/figs7.jpg" height="948" width="729" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/3b7aebf9-daa3-4220-af13-f913cd1dcc44/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.03.061/asset/f0b6564a-b53d-4008-b17c-e8670d5831a9/main.assets/gr5.jpg" height="941" width="639" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/113f488f-30d3-4928-a7be-6daf11c4261c/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.03.061/asset/82f00349-cfde-4abf-9516-02ad7270dd0d/main.assets/gr6.jpg" height="944" width="678" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/3ea1248a-2e02-4bf6-bb19-8d275be7f042/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs8" src="/cms/10.1016/j.cell.2020.03.061/asset/8abc2016-a3bc-498a-9dcd-75a66f95f673/main.assets/figs8.jpg" height="945" width="665" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0876b8a4-40e5-4619-9b61-452b8175ae27/main.assets/figs8_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2020.03.061/asset/d21414e6-d235-4c77-8b4d-3cc3b0ff59f3/main.assets/gr7.jpg" height="888" width="678" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0f18deb4-24c4-4435-819b-1c4189cbc9b4/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.03.061/asset/7b4120f8-32e4-4e50-b2aa-c641778655cf/main.assets/gr1.jpg" height="817" width="749" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0ac709b2-2981-46f8-8362-cc2b1ee70a2d/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.03.061/asset/2c14656f-15d9-4391-9ad7-4f6c0993041f/main.assets/figs1.jpg" height="867" width="678" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/99538f76-533c-4b80-8b42-c9fbb4deee0a/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.03.061/asset/2cce2733-2cce-4978-882c-10181f95129c/main.assets/figs2.jpg" height="439" width="500" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/79d6c09c-8f76-4d6b-8775-93115dfd8337/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.03.061/asset/ac064d5d-3fb7-4eff-b583-d37630525113/main.assets/figs3.jpg" height="457" width="499" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/6d0c38c2-940a-4144-baa2-521151b8700c/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.03.061/asset/435f6df9-9c91-4081-ad89-8be7c17652c0/main.assets/gr2.jpg" height="873" width="678" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/c5b735b3-464f-463b-bd9d-8f3540c4cf67/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.03.061/asset/dcc49d79-b328-4d1c-9b87-2ee3b9aaf700/main.assets/figs4.jpg" height="518" width="762" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/9faf2324-3b04-4727-aa7c-705eedade1ad/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.03.061/asset/cda010d9-4a06-4678-92ed-fd90714706a4/main.assets/figs5.jpg" height="740" width="504" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/216fa9b9-2075-49fd-bef9-2a9932aadffa/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.03.061/asset/f0d4d02f-81c3-45f6-b916-9b8e38c8bcd5/main.assets/figs6.jpg" height="949" width="645" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/62429aee-c0d9-4c56-b3aa-0233b6a00b07/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.03.061/asset/5363c576-6ea3-4f60-ac91-26956886ac6b/main.assets/gr3.jpg" height="910" width="499" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/f1832c02-460c-47f6-bb23-01724944011d/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.03.061/asset/f1aafa48-73a4-43c9-a848-59368db99670/main.assets/gr4.jpg" height="891" width="679" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/c12315ac-294d-4bc6-b2ad-e2cb0d5e3943/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2020.03.061/asset/2d2faf51-1b64-49ba-9a4a-f0fa8b2fb859/main.assets/figs7.jpg" height="948" width="729" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/3b7aebf9-daa3-4220-af13-f913cd1dcc44/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.03.061/asset/f0b6564a-b53d-4008-b17c-e8670d5831a9/main.assets/gr5.jpg" height="941" width="639" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/113f488f-30d3-4928-a7be-6daf11c4261c/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.03.061/asset/82f00349-cfde-4abf-9516-02ad7270dd0d/main.assets/gr6.jpg" height="944" width="678" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/3ea1248a-2e02-4bf6-bb19-8d275be7f042/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S8</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs8" src="/cms/10.1016/j.cell.2020.03.061/asset/8abc2016-a3bc-498a-9dcd-75a66f95f673/main.assets/figs8.jpg" height="945" width="665" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0876b8a4-40e5-4619-9b61-452b8175ae27/main.assets/figs8_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2020.03.061/asset/d21414e6-d235-4c77-8b4d-3cc3b0ff59f3/main.assets/gr7.jpg" height="888" width="678" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.061/asset/0f18deb4-24c4-4435-819b-1c4189cbc9b4/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30389-5';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749586377272","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"181","issueNumber":"4","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867420303895","coverDate":"202005","type":"XOCS-JOURNAL:SCOPE-FULL","title":"A StructureBased Model for the Complete Transcription Cycle of Influenza Polymerase","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94db86c22b1827fa',t:'MTc0OTU4NjM3Ni4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94db86c22b1827fa&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2020.03.061&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>