<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="4ec39c96-2ef1-4ea2-8385-7a782699f5aa" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;issue:issue:pii\:S0092867419X00203;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text;article:article:pii\:S0092867420309387">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Large-Scale Topological Changes Restrain Malignant Progression in Colorectal Cancer">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)30938-7">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/76be94dc-f954-4050-9cf9-83953f2cf9ca/fx1.jpg">
    


        <meta property="og:description" content="Integrated analyses of genome topological, epigenetic, and transcriptional features
of colorectal tumors highlight substantial genome compartmental reorganization associated
with tumor-suppressive rather than oncogenic transcriptional outcomes.">
    











        <meta name="twitter:title" content="Large-Scale Topological Changes Restrain Malignant Progression in Colorectal Cancer">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/76be94dc-f954-4050-9cf9-83953f2cf9ca/fx1.jpg">
    


        <meta name="twitter:description" content="Integrated analyses of genome topological, epigenetic, and transcriptional features
of colorectal tumors highlight substantial genome compartmental reorganization associated
with tumor-suppressive rather than oncogenic transcriptional outcomes.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)30938-7">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Large-Scale Topological Changes Restrain Malignant Progression in Colorectal Cancer: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Large-Scale Topological Changes Restrain Malignant Progression in Colorectal Cancer"> <meta name="citation_date" content="2020/09/17"> <meta name="citation_online_date" content="2020/08/24"> <meta name="citation_volume" content="182"> <meta name="citation_issue" content="6"> <meta name="citation_firstpage" content="1474"> <meta name="citation_lastpage" content="1489.e23"> <meta name="citation_pmid" content="32841603"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867420309387"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867420309387"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867420309387"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2020.07.030"> <meta name="citation_pmid" content="32841603"> <meta name="citation_author" content="Sarah E. Johnstone"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA"><meta name="citation_author" content="Alejandro Reyes"><meta name="citation_author_orcid" content="0000-0001-8717-6612"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Data Sciences, Dana Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA"><meta name="citation_author" content="Yifeng Qi"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Carmen Adriaens"><meta name="citation_author_orcid" content="0000-0001-5186-2741"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA"><meta name="citation_author" content="Esmat Hegazi"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA"><meta name="citation_author" content="Karin Pelka"><meta name="citation_author_orcid" content="0000-0003-2856-8602"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA"><meta name="citation_author" content="Jonathan H. Chen"><meta name="citation_author_orcid" content="0000-0001-6074-9302"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA"><meta name="citation_author" content="Luli S. Zou"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Data Sciences, Dana Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA"><meta name="citation_author" content="Yotam Drier"><meta name="citation_author_orcid" content="0000-0003-1725-2995"><meta name="citation_author_institution" content="The Lautenberg Center for Immunology and Cancer Research, The Hebrew University, Jerusalem, Israel"><meta name="citation_author" content="Vivian Hecht"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author" content="Noam Shoresh"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author" content="Martin K. Selig"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author" content="Caleb A. Lareau"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Program in Biological and Biomedical Sciences, Harvard Medical School, Boston, MA 02215, USA"><meta name="citation_author" content="Sowmya Iyer"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author" content="Son C. Nguyen"><meta name="citation_author_orcid" content="0000-0003-2187-1027"><meta name="citation_author_institution" content="Department of Genetics, Penn Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Eric F. Joyce"><meta name="citation_author_orcid" content="0000-0002-0418-2804"><meta name="citation_author_institution" content="Department of Genetics, Penn Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Nir Hacohen"><meta name="citation_author_orcid" content="0000-0002-2349-2656"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA"><meta name="citation_author" content="Rafael A. Irizarry"><meta name="citation_author_orcid" content="0000-0002-3944-4309"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Data Sciences, Dana Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA"><meta name="citation_author" content="Bin Zhang"><meta name="citation_author_orcid" content="0000-0002-3685-7503"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Martin J. Aryee"><meta name="citation_author_email" content="aryee.martin@mgh.harvard.edu"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA"><meta name="citation_author_institution" content="Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA"><meta name="citation_author" content="Bradley E. Bernstein"><meta name="citation_author_email" content="bernstein.bradley@mgh.harvard.edu"><meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA"><meta name="citation_keywords" content="genome topology; chromatin; DNA methylation; colon cancer; epigenetics; nuclear architecture; compartment"><meta name="citation_reference" content="citation_title=Epigenetic Determinants of Cancer; citation_author=S.B. Baylin; citation_author=P.A. Jones; citation_journal_title=Cold Spring Harb. Perspect. Biol.; citation_volume=8; citation_firstpage=a019505; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Versatile design and synthesis platform for visualizing genomes with Oligopaint FISH probes; citation_author=B.J. Beliveau; citation_author=E.F. Joyce; citation_author=N. Apostolopoulos; citation_author=F. Yilmaz; citation_author=C.Y. Fonseka; citation_author=R.B. McCole; citation_author=Y. Chang; citation_author=J.B. Li; citation_author=T.N. Senaratne; citation_author=B.R. Williams; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=21301; citation_lastpage=21306; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains; citation_author=B.P. Berman; citation_author=D.J. Weisenberger; citation_author=J.F. Aman; citation_author=T. Hinoue; citation_author=Z. Ramjan; citation_author=Y. Liu; citation_author=H. Noushmehr; citation_author=C.P.E. Lange; citation_author=C.M. van Dijk; citation_author=R.A.E.M. Tollenaar; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=44; citation_firstpage=40; citation_lastpage=46; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Genome architecture: domain organization of interphase chromosomes; citation_author=W.A. Bickmore; citation_author=B. van Steensel; citation_journal_title=Cell; citation_volume=152; citation_firstpage=1270; citation_lastpage=1284; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Super-resolution imaging reveals distinct chromatin folding for different epigenetic states; citation_author=A.N. Boettiger; citation_author=B. Bintu; citation_author=J.R. Moffitt; citation_author=S. Wang; citation_author=B.J. Beliveau; citation_author=G. Fudenberg; citation_author=M. Imakaev; citation_author=L.A. Mirny; citation_author=C.-T. Wu; citation_author=X. Zhuang; citation_journal_title=Nature; citation_volume=529; citation_firstpage=418; citation_lastpage=422; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Three-dimensional maps of all chromosomes in human male fibroblast nuclei and prometaphase rosettes; citation_author=A. Bolzer; citation_author=G. Kreth; citation_author=I. Solovei; citation_author=D. Koehler; citation_author=K. Saracoglu; citation_author=C. Fauth; citation_author=S. Müller; citation_author=R. Eils; citation_author=C. Cremer; citation_author=M.R. Speicher; citation_author=T. Cremer; citation_journal_title=PLoS Biol.; citation_volume=3; citation_firstpage=e157; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Stromal gene expression defines poor-prognosis subtypes in colorectal cancer; citation_author=A. Calon; citation_author=E. Lonardo; citation_author=A. Berenguer-Llergo; citation_author=E. Espinet; citation_author=X. Hernando-Momblona; citation_author=M. Iglesias; citation_author=M. Sevillano; citation_author=S. Palomo-Ponce; citation_author=D.V.F. Tauriello; citation_author=D. Byrom; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=47; citation_firstpage=320; citation_lastpage=329; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Comprehensive molecular characterization of human colon and rectal cancer; citation_author=Cancer Genome Atlas Network; citation_journal_title=Nature; citation_volume=487; citation_firstpage=330; citation_lastpage=337; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Accurate loop calling for 3D genomic data with cLoops; citation_author=Y. Cao; citation_author=Z. Chen; citation_author=X. Chen; citation_author=D. Ai; citation_author=G. Chen; citation_author=J. McDermott; citation_author=Y. Huang; citation_author=X. Guo; citation_author=J.-D.J. Han; citation_journal_title=Bioinformatics; citation_volume=36; citation_firstpage=666; citation_lastpage=675; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=High-order chromatin structure and the epigenome in SAHFs; citation_author=T. Chandra; citation_author=M. Narita; citation_journal_title=Nucleus; citation_volume=4; citation_firstpage=23; citation_lastpage=28; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data; citation_author=A. Colaprico; citation_author=T.C. Silva; citation_author=C. Olsen; citation_author=L. Garofano; citation_author=C. Cava; citation_author=D. Garolini; citation_author=T.S. Sabedot; citation_author=T.M. Malta; citation_author=S.M. Pagnotta; citation_author=I. Castiglioni; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e71; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Reproducible RNA-seq analysis using recount2; citation_author=L. Collado-Torres; citation_author=A. Nellore; citation_author=K. Kammers; citation_author=S.E. Ellis; citation_author=M.A. Taub; citation_author=K.D. Hansen; citation_author=A.E. Jaffe; citation_author=B. Langmead; citation_author=J.T. Leek; citation_journal_title=Nat. Biotechnol.; citation_volume=35; citation_firstpage=319; citation_lastpage=321; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The three-dimensional cancer genome; citation_author=M.R. Corces; citation_author=V.G. Corces; citation_journal_title=Curr. Opin. Genet. Dev.; citation_volume=36; citation_firstpage=1; citation_lastpage=7; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Reorganization of chromosome architecture in replicative cellular senescence; citation_author=S.W. Criscione; citation_author=M. De Cecco; citation_author=B. Siranosian; citation_author=Y. Zhang; citation_author=J.A. Kreiling; citation_author=J.M. Sedivy; citation_author=N. Neretti; citation_journal_title=Sci. Adv.; citation_volume=2; citation_firstpage=e1500882; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Senescent cells harbour features of the cancer epigenome; citation_author=H.A. Cruickshanks; citation_author=T. McBryan; citation_author=D.M. Nelson; citation_author=N.D. Vanderkraats; citation_author=P.P. Shah; citation_author=J. van Tuyn; citation_author=T. Singh Rai; citation_author=C. Brock; citation_author=G. Donahue; citation_author=D.S. Dunican; citation_author=et al.; citation_journal_title=Nat. Cell Biol.; citation_volume=15; citation_firstpage=1495; citation_lastpage=1506; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Long-Range Chromatin Interactions; citation_author=J. Dekker; citation_author=T. Misteli; citation_journal_title=Cold Spring Harb. Perspect. Biol.; citation_volume=7; citation_firstpage=a019356; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Transferable model for chromosome architecture; citation_author=M. Di Pierro; citation_author=B. Zhang; citation_author=E.L. Aiden; citation_author=P.G. Wolynes; citation_author=J.N. Onuchic; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=113; citation_firstpage=12168; citation_lastpage=12173; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Rapid targeted mutational analysis of human tumours: a clinical platform to guide personalized cancer medicine; citation_author=D. Dias-Santagata; citation_author=S. Akhavanfard; citation_author=S.S. David; citation_author=K. Vernovsky; citation_author=G. Kuhlmann; citation_author=S.L. Boisvert; citation_author=H. Stubbs; citation_author=U. McDermott; citation_author=J. Settleman; citation_author=E.L. Kwak; citation_author=et al.; citation_journal_title=EMBO Mol. Med.; citation_volume=2; citation_firstpage=146; citation_lastpage=158; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Topological domains in mammalian genomes identified by analysis of chromatin interactions; citation_author=J.R. Dixon; citation_author=S. Selvaraj; citation_author=F. Yue; citation_author=A. Kim; citation_author=Y. Li; citation_author=Y. Shen; citation_author=M. Hu; citation_author=J.S. Liu; citation_author=B. Ren; citation_journal_title=Nature; citation_volume=485; citation_firstpage=376; citation_lastpage=380; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Chromatin architecture reorganization during stem cell differentiation; citation_author=J.R. Dixon; citation_author=I. Jung; citation_author=S. Selvaraj; citation_author=Y. Shen; citation_author=J.E. Antosiewicz-Bourget; citation_author=A.Y. Lee; citation_author=Z. Ye; citation_author=A. Kim; citation_author=N. Rajagopal; citation_author=W. Xie; citation_author=et al.; citation_journal_title=Nature; citation_volume=518; citation_firstpage=331; citation_lastpage=336; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=EphA2 Expression Is a Key Driver of Migration and Invasion and a Poor Prognostic Marker in Colorectal Cancer; citation_author=P.D. Dunne; citation_author=S. Dasgupta; citation_author=J.K. Blayney; citation_author=D.G. McArt; citation_author=K.L. Redmond; citation_author=J.A. Weir; citation_author=C.A. Bradley; citation_author=T. Sasazuki; citation_author=S. Shirasawa; citation_author=T. Wang; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=22; citation_firstpage=230; citation_lastpage=242; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Juicebox Provides a Visualization System for Hi-C Contact Maps with Unlimited Zoom; citation_author=N.C. Durand; citation_author=J.T. Robinson; citation_author=M.S. Shamim; citation_author=I. Machol; citation_author=J.P. Mesirov; citation_author=E.S. Lander; citation_author=E.L. Aiden; citation_journal_title=Cell Syst.; citation_volume=3; citation_firstpage=99; citation_lastpage=101; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Heterochromatin drives compartmentalization of inverted and conventional nuclei; citation_author=M. Falk; citation_author=Y. Feodorova; citation_author=N. Naumova; citation_author=M. Imakaev; citation_author=B.R. Lajoie; citation_author=H. Leonhardt; citation_author=B. Joffe; citation_author=J. Dekker; citation_author=G. Fudenberg; citation_author=I. Solovei; citation_author=et al.; citation_journal_title=Nature; citation_volume=570; citation_firstpage=395; citation_lastpage=399; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genome-wide DNA methylation profiles of low- and high-grade adenoma reveals potential biomarkers for early detection of colorectal carcinoma; citation_author=J. Fan; citation_author=J. Li; citation_author=S. Guo; citation_author=C. Tao; citation_author=H. Zhang; citation_author=W. Wang; citation_author=Y. Zhang; citation_author=D. Zhang; citation_author=S. Ding; citation_author=C. Zeng; citation_journal_title=Clin. Epigenetics; citation_volume=12; citation_firstpage=56; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Insulator dysfunction and oncogene activation in IDH mutant gliomas; citation_author=W.A. Flavahan; citation_author=Y. Drier; citation_author=B.B. Liau; citation_author=S.M. Gillespie; citation_author=A.S. Venteicher; citation_author=A.O. Stemmer-Rachamimov; citation_author=M.L. Suvà; citation_author=B.E. Bernstein; citation_journal_title=Nature; citation_volume=529; citation_firstpage=110; citation_lastpage=114; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Altered chromosomal topology drives oncogenic programs in SDH-deficient GISTs; citation_author=W.A. Flavahan; citation_author=Y. Drier; citation_author=S.E. Johnstone; citation_author=M.L. Hemming; citation_author=D.R. Tarjan; citation_author=E. Hegazi; citation_author=S.J. Shareef; citation_author=N.M. Javed; citation_author=C.P. Raut; citation_author=B.K. Eschle; citation_author=et al.; citation_journal_title=Nature; citation_volume=575; citation_firstpage=229; citation_lastpage=233; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Reconstructing A/B compartments as revealed by Hi-C using long-range correlations in epigenetic data; citation_author=J.-P. Fortin; citation_author=K.D. Hansen; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=180; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Challenges and solutions in patient treatment strategies for stage II colon cancer; citation_author=S. Fotheringham; citation_author=G.A. Mozolowski; citation_author=E.M.A. Murray; citation_author=D.J. Kerr; citation_journal_title=Gastroenterol. Rep. (Oxf.); citation_volume=7; citation_firstpage=151; citation_lastpage=161; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Emerging Contributions of Cancer/Testis Antigens to Neoplastic Behaviors; citation_author=Z.A. Gibbs; citation_author=A.W. Whitehurst; citation_journal_title=Trends Cancer; citation_volume=4; citation_firstpage=701; citation_lastpage=712; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Correlating colorectal cancer risk with field carcinogenesis progression using partial wave spectroscopic microscopy; citation_author=S. Gladstein; citation_author=D. Damania; citation_author=L.M. Almassalha; citation_author=L.T. Smith; citation_author=V. Gupta; citation_author=H. Subramanian; citation_author=D.K. Rex; citation_author=H.K. Roy; citation_author=V. Backman; citation_journal_title=Cancer Med.; citation_volume=7; citation_firstpage=2109; citation_lastpage=2120; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Correcting for cancer genome size and tumour cell content enables better estimation of copy number alterations from next-generation sequence data; citation_author=A. Gusnanto; citation_author=H.M. Wood; citation_author=Y. Pawitan; citation_author=P. Rabbitts; citation_author=S. Berri; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=40; citation_lastpage=47; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Increased methylation variation in epigenetic domains across cancer types; citation_author=K.D. Hansen; citation_author=W. Timp; citation_author=H.C. Bravo; citation_author=S. Sabunciyan; citation_author=B. Langmead; citation_author=O.G. McDonald; citation_author=B. Wen; citation_author=H. Wu; citation_author=Y. Liu; citation_author=D. Diep; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=43; citation_firstpage=768; citation_lastpage=775; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=BSmooth: from whole genome bisulfite sequencing reads to differentially methylated regions; citation_author=K.D. Hansen; citation_author=B. Langmead; citation_author=R.A. Irizarry; citation_journal_title=Genome Biol.; citation_volume=13; citation_firstpage=R83; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Genome-scale analysis of aberrant DNA methylation in colorectal cancer; citation_author=T. Hinoue; citation_author=D.J. Weisenberger; citation_author=C.P.E. Lange; citation_author=H. Shen; citation_author=H.M. Byun; citation_author=D. Van Den Berg; citation_author=S. Malik; citation_author=F. Pan; citation_author=H. Noushmehr; citation_author=C.M. van Dijk; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=271; citation_lastpage=282; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Activation of proto-oncogenes by disruption of chromosome neighborhoods; citation_author=D. Hnisz; citation_author=A.S. Weintraub; citation_author=D.S. Day; citation_author=A.-L. Valton; citation_author=R.O. Bak; citation_author=C.H. Li; citation_author=J. Goldmann; citation_author=B.R. Lajoie; citation_author=Z.P. Fan; citation_author=A.A. Sigova; citation_author=et al.; citation_journal_title=Science; citation_volume=351; citation_firstpage=1454; citation_lastpage=1458; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Global DNA hypomethylation coupled to repressive chromatin domain formation and gene silencing in breast cancer; citation_author=G.C. Hon; citation_author=R.D. Hawkins; citation_author=O.L. Caballero; citation_author=C. Lo; citation_author=R. Lister; citation_author=M. Pelizzola; citation_author=A. Valsesia; citation_author=Z. Ye; citation_author=S. Kuan; citation_author=L.E. Edsall; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=246; citation_lastpage=258; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Robust smooth segmentation approach for array CGH data analysis; citation_author=J. Huang; citation_author=A. Gusnanto; citation_author=K. O’Sullivan; citation_author=J. Staaf; citation_author=A. Borg; citation_author=Y. Pawitan; citation_journal_title=Bioinformatics; citation_volume=23; citation_firstpage=2463; citation_lastpage=2469; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Orchestrating high-throughput genomic analysis with Bioconductor; citation_author=W. Huber; citation_author=V.J. Carey; citation_author=R. Gentleman; citation_author=S. Anders; citation_author=M. Carlson; citation_author=B.S. Carvalho; citation_author=H.C. Bravo; citation_author=S. Davis; citation_author=L. Gatto; citation_author=T. Girke; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=115; citation_lastpage=121; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Pan-cancer analysis of whole genomes; citation_author=ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium; citation_journal_title=Nature; citation_volume=578; citation_firstpage=82; citation_lastpage=93; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Iterative correction of Hi-C data reveals hallmarks of chromosome organization; citation_author=M. Imakaev; citation_author=G. Fudenberg; citation_author=R.P. McCord; citation_author=N. Naumova; citation_author=A. Goloborodko; citation_author=B.R. Lajoie; citation_author=J. Dekker; citation_author=L.A. Mirny; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=999; citation_lastpage=1003; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Three-dimensional chromatin landscapes in T&nbsp;cell acute lymphoblastic leukemia; citation_author=A. Kloetgen; citation_author=P. Thandapani; citation_author=P. Ntziachristos; citation_author=Y. Ghebrechristos; citation_author=S. Nomikou; citation_author=C. Lazaris; citation_author=X. Chen; citation_author=H. Hu; citation_author=S. Bakogianni; citation_author=J. Wang; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=52; citation_firstpage=388; citation_lastpage=400; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Signaling pathways involved in colorectal cancer progression; citation_author=Z. Koveitypour; citation_author=F. Panahi; citation_author=M. Vakilian; citation_author=M. Peymani; citation_author=F. Seyed Forootan; citation_author=M.H. Nasr Esfahani; citation_author=K. Ghaedi; citation_journal_title=Cell Biosci.; citation_volume=9; citation_firstpage=97; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Evolutionary stability of topologically associating domains is associated with conserved gene regulation; citation_author=J. Krefting; citation_author=M.A. Andrade-Navarro; citation_author=J. Ibn-Salem; citation_journal_title=BMC Biol.; citation_volume=16; citation_firstpage=87; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=diffloop: a computational framework for identifying and analyzing differential DNA loops from sequencing data; citation_author=C.A. Lareau; citation_author=M.J. Aryee; citation_journal_title=Bioinformatics; citation_volume=34; citation_firstpage=672; citation_lastpage=674; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=hichipper: a preprocessing pipeline for calling DNA loops from HiChIP data; citation_author=C.A. Lareau; citation_author=M.J. Aryee; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=155; citation_lastpage=156; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Liquid droplet formation by HP1α suggests a role for phase separation in heterochromatin; citation_author=A.G. Larson; citation_author=D. Elnatan; citation_author=M.M. Keenen; citation_author=M.J. Trnka; citation_author=J.B. Johnston; citation_author=A.L. Burlingame; citation_author=D.A. Agard; citation_author=S. Redding; citation_author=G.J. Narlikar; citation_journal_title=Nature; citation_volume=547; citation_firstpage=236; citation_lastpage=240; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=voom: Precision weights unlock linear model analysis tools for RNA-seq read counts; citation_author=C.W. Law; citation_author=Y. Chen; citation_author=W. Shi; citation_author=G.K. Smyth; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=R29; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Software for computing and annotating genomic ranges; citation_author=M. Lawrence; citation_author=W. Huber; citation_author=H. Pagès; citation_author=P. Aboyoun; citation_author=M. Carlson; citation_author=R. Gentleman; citation_author=M.T. Morgan; citation_author=V.J. Carey; citation_journal_title=PLoS Comput. Biol.; citation_volume=9; citation_firstpage=e1003118; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=1754; citation_lastpage=1760; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Adaptive Chromatin Remodeling Drives Glioblastoma Stem Cell Plasticity and Drug Tolerance; citation_author=B.B. Liau; citation_author=C. Sievers; citation_author=L.K. Donohue; citation_author=S.M. Gillespie; citation_author=W.A. Flavahan; citation_author=T.E. Miller; citation_author=A.S. Venteicher; citation_author=C.H. Hebert; citation_author=C.D. Carey; citation_author=S.J. Rodig; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=20; citation_firstpage=233; citation_lastpage=246.e7; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Comprehensive mapping of long-range interactions reveals folding principles of the human genome; citation_author=E. Lieberman-Aiden; citation_author=N.L. van Berkum; citation_author=L. Williams; citation_author=M. Imakaev; citation_author=T. Ragoczy; citation_author=A. Telling; citation_author=I. Amit; citation_author=B.R. Lajoie; citation_author=P.J. Sabo; citation_author=M.O. Dorschner; citation_author=et al.; citation_journal_title=Science; citation_volume=326; citation_firstpage=289; citation_lastpage=293; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Targeting bivalency de-represses Indian Hedgehog and inhibits self-renewal of colorectal cancer-initiating cells; citation_author=E. Lima-Fernandes; citation_author=A. Murison; citation_author=T. da Silva Medina; citation_author=Y. Wang; citation_author=A. Ma; citation_author=C. Leung; citation_author=G.M. Luciani; citation_author=J. Haynes; citation_author=A. Pollett; citation_author=C. Zeller; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=1436; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=ONGene: A literature-based database for human oncogenes; citation_author=Y. Liu; citation_author=J. Sun; citation_author=M. Zhao; citation_journal_title=J.&nbsp;Genet. Genomics; citation_volume=44; citation_firstpage=119; citation_lastpage=121; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2; citation_author=M.I. Love; citation_author=W. Huber; citation_author=S. Anders; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=550; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=RNA-Seq workflow: gene-level exploratory analysis and differential expression; citation_author=M.I. Love; citation_author=S. Anders; citation_author=V. Kim; citation_author=W. Huber; citation_journal_title=F1000Res.; citation_volume=4; citation_firstpage=1070; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=csaw: a Bioconductor package for differential binding analysis of ChIP-seq data using sliding windows; citation_author=A.T.L. Lun; citation_author=G.K. Smyth; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e45; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog); citation_author=J. MacArthur; citation_author=E. Bowler; citation_author=M. Cerezo; citation_author=L. Gil; citation_author=P. Hall; citation_author=E. Hastings; citation_author=H. Junkins; citation_author=A. McMahon; citation_author=A. Milano; citation_author=J. Morales; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=D1; citation_firstpage=D896; citation_lastpage=D901; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value; citation_author=L. Marisa; citation_author=A. de Reyniès; citation_author=A. Duval; citation_author=J. Selves; citation_author=M.P. Gaub; citation_author=L. Vescovo; citation_author=M.-C. Etienne-Grimaldi; citation_author=R. Schiappa; citation_author=D. Guenot; citation_author=M. Ayadi; citation_author=et al.; citation_journal_title=PLoS Med.; citation_volume=10; citation_firstpage=e1001453; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Genome-scale epigenetic reprogramming during epithelial-to-mesenchymal transition; citation_author=O.G. McDonald; citation_author=H. Wu; citation_author=W. Timp; citation_author=A. Doi; citation_author=A.P. Feinberg; citation_journal_title=Nat. Struct. Mol. Biol.; citation_volume=18; citation_firstpage=867; citation_lastpage=874; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=CellProfiler 3.0: Next-generation image processing for biology; citation_author=C. McQuin; citation_author=A. Goodman; citation_author=V. Chernyshev; citation_author=L. Kamentsky; citation_author=B.A. Cimini; citation_author=K.W. Karhohs; citation_author=M. Doan; citation_author=L. Ding; citation_author=S.M. Rafelski; citation_author=D. Thirstrup; citation_author=et al.; citation_journal_title=PLoS Biol.; citation_volume=16; citation_firstpage=e2005970; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Low-Grade Astrocytoma Mutations in IDH1, P53, and ATRX Cooperate to Block Differentiation of Human Neural Stem Cells via Repression of SOX2; citation_author=A.S. Modrek; citation_author=D. Golub; citation_author=T. Khan; citation_author=D. Bready; citation_author=J. Prado; citation_author=C. Bowman; citation_author=J. Deng; citation_author=G. Zhang; citation_author=P.P. Rocha; citation_author=R. Raviram; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=21; citation_firstpage=1267; citation_lastpage=1280; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=HiChIP: efficient and sensitive analysis of protein-directed genome architecture; citation_author=M.R. Mumbach; citation_author=A.J. Rubin; citation_author=R.A. Flynn; citation_author=C. Dai; citation_author=P.A. Khavari; citation_author=W.J. Greenleaf; citation_author=H.Y. Chang; citation_journal_title=Nat. Methods; citation_volume=13; citation_firstpage=919; citation_lastpage=922; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Pathways of Colorectal Carcinogenesis; citation_author=L.H. Nguyen; citation_author=A. Goel; citation_author=D.C. Chung; citation_journal_title=Gastroenterology; citation_volume=158; citation_firstpage=291; citation_lastpage=302; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Spatial partitioning of the regulatory landscape of the X-inactivation centre; citation_author=E.P. Nora; citation_author=B.R. Lajoie; citation_author=E.G. Schulz; citation_author=L. Giorgetti; citation_author=I. Okamoto; citation_author=N. Servant; citation_author=T. Piolot; citation_author=N.L. van Berkum; citation_author=J. Meisig; citation_author=J. Sedat; citation_author=et al.; citation_journal_title=Nature; citation_volume=485; citation_firstpage=381; citation_lastpage=385; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Targeted Degradation of CTCF Decouples Local Insulation of Chromosome Domains from Genomic Compartmentalization; citation_author=E.P. Nora; citation_author=A. Goloborodko; citation_author=A.-L. Valton; citation_author=J.H. Gibcus; citation_author=A. Uebersohn; citation_author=N. Abdennur; citation_author=J. Dekker; citation_author=L.A. Mirny; citation_author=B.G. Bruneau; citation_journal_title=Cell; citation_volume=169; citation_firstpage=930; citation_lastpage=944.e22; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The early pregnancy placenta foreshadows DNA methylation alterations of solid tumors; citation_author=A.V. Nordor; citation_author=D. Nehar-Belaid; citation_author=S. Richon; citation_author=D. Klatzmann; citation_author=D. Bellet; citation_author=V. Dangles-Marie; citation_author=T. Fournier; citation_author=M.J. Aryee; citation_journal_title=Epigenetics; citation_volume=12; citation_firstpage=793; citation_lastpage=803; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Circular binary segmentation for the analysis of array-based DNA copy number data; citation_author=A.B. Olshen; citation_author=E.S. Venkatraman; citation_author=R. Lucito; citation_author=M. Wigler; citation_journal_title=Biostatistics; citation_volume=5; citation_firstpage=557; citation_lastpage=572; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Salmon provides fast and bias-aware quantification of transcript expression; citation_author=R. Patro; citation_author=G. Duggal; citation_author=M.I. Love; citation_author=R.A. Irizarry; citation_author=C. Kingsford; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=417; citation_lastpage=419; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Identification of genes expressed by immune cells of the colon that are regulated by colorectal cancer-associated variants; citation_author=V.D. Peltekova; citation_author=M. Lemire; citation_author=A.M. Qazi; citation_author=S.H.E. Zaidi; citation_author=Q.M. Trinh; citation_author=R. Bielecki; citation_author=M. Rogers; citation_author=L. Hodgson; citation_author=M. Wang; citation_author=D.J.A. D’Souza; citation_author=et al.; citation_journal_title=Int. J. Cancer; citation_volume=134; citation_firstpage=2330; citation_lastpage=2341; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Fast parallel algorithms for short-range molecular dynamics; citation_author=S. Plimpton; citation_journal_title=J.&nbsp;Comput. Phys.; citation_volume=117; citation_firstpage=1; citation_lastpage=19; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Data-driven polymer model for mechanistic exploration of diploid genome organization; citation_author=Y. Qi; citation_author=A. Reyes; citation_author=S.E. Johnstone; citation_author=M.J. Aryee; citation_author=B.E. Bernstein; citation_author=B. Zhang; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.02.27.968735; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Predicting three-dimensional genome organization with chromatin states; citation_author=Y. Qi; citation_author=B. Zhang; citation_journal_title=PLoS Comput. Biol.; citation_volume=15; citation_firstpage=e1007024; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping; citation_author=S.S.P. Rao; citation_author=M.H. Huntley; citation_author=N.C. Durand; citation_author=E.K. Stamenova; citation_author=I.D. Bochkov; citation_author=J.T. Robinson; citation_author=A.L. Sanborn; citation_author=I. Machol; citation_author=A.D. Omer; citation_author=E.S. Lander; citation_author=E.L. Aiden; citation_journal_title=Cell; citation_volume=159; citation_firstpage=1665; citation_lastpage=1680; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=M.D. Robinson; citation_author=D.J. McCarthy; citation_author=G.K. Smyth; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=139; citation_lastpage=140; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Molecular and genetic properties of tumors associated with local immune cytolytic activity; citation_author=M.S. Rooney; citation_author=S.A. Shukla; citation_author=C.J. Wu; citation_author=G. Getz; citation_author=N. Hacohen; citation_journal_title=Cell; citation_volume=160; citation_firstpage=48; citation_lastpage=61; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Condensin II drives large-scale folding and spatial partitioning of interphase chromosomes in Drosophila nuclei; citation_author=L.F. Rosin; citation_author=S.C. Nguyen; citation_author=E.F. Joyce; citation_journal_title=PLoS Genet.; citation_volume=14; citation_firstpage=e1007393; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=DNA-Demethylating Agents Target Colorectal Cancer Cells by Inducing Viral Mimicry by Endogenous Transcripts; citation_author=D. Roulois; citation_author=H. Loo Yau; citation_author=R. Singhania; citation_author=Y. Wang; citation_author=A. Danesh; citation_author=S.Y. Shen; citation_author=H. Han; citation_author=G. Liang; citation_author=P.A. Jones; citation_author=T.J. Pugh; citation_author=et al.; citation_journal_title=Cell; citation_volume=162; citation_firstpage=961; citation_lastpage=973; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Organizational principles of 3D genome architecture; citation_author=M.J. Rowley; citation_author=V.G. Corces; citation_journal_title=Nat. Rev. Genet.; citation_volume=19; citation_firstpage=789; citation_lastpage=800; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A novel role of lamins from genetic disease to cancer biomarkers; citation_author=K.M. Sakthivel; citation_author=P. Sehgal; citation_journal_title=Oncol. Rev.; citation_volume=10; citation_firstpage=309; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=4D Genome Rewiring during Oncogene-Induced and Replicative Senescence; citation_author=S. Sati; citation_author=B. Bonev; citation_author=Q. Szabo; citation_author=D. Jost; citation_author=P. Bensadoun; citation_author=F. Serra; citation_author=V. Loubiere; citation_author=G.L. Papadopoulos; citation_author=J.-C. Rivera-Mulia; citation_author=L. Fritsch; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=78; citation_firstpage=522; citation_lastpage=538.e9; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=E. Frise; citation_author=V. Kaynig; citation_author=M. Longair; citation_author=T. Pietzsch; citation_author=S. Preibisch; citation_author=C. Rueden; citation_author=S. Saalfeld; citation_author=B. Schmid; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=676; citation_lastpage=682; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome; citation_author=A.D. Schmitt; citation_author=M. Hu; citation_author=I. Jung; citation_author=Z. Xu; citation_author=Y. Qiu; citation_author=C.L. Tan; citation_author=Y. Li; citation_author=S. Lin; citation_author=Y. Lin; citation_author=C.L. Barr; citation_author=B. Ren; citation_journal_title=Cell Rep.; citation_volume=17; citation_firstpage=2042; citation_lastpage=2059; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=When lamins go bad: nuclear structure and disease; citation_author=K.H. Schreiber; citation_author=B.K. Kennedy; citation_journal_title=Cell; citation_volume=152; citation_firstpage=1365; citation_lastpage=1375; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Automatic analysis and 3D-modelling of Hi-C data using TADbit reveals structural features of the fly chromatin colors; citation_author=F. Serra; citation_author=D. Baù; citation_author=M. Goodstadt; citation_author=D. Castillo; citation_author=G.J. Filion; citation_author=M.A. Marti-Renom; citation_journal_title=PLoS Comput. Biol.; citation_volume=13; citation_firstpage=e1005665; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=HiC-Pro: an optimized and flexible pipeline for Hi-C data processing; citation_author=N. Servant; citation_author=N. Varoquaux; citation_author=B.R. Lajoie; citation_author=E. Viara; citation_author=C.-J. Chen; citation_author=J.-P. Vert; citation_author=E. Heard; citation_author=J. Dekker; citation_author=E. Barillot; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=259; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Imaging Gene Expression; citation_author=Y. Shav-Tal; citation_publication_date=2013; citation_publisher=Springer; "><meta name="citation_reference" content="citation_title=RepeatMasker Open-4.0. 2013–2015; citation_author=A. Smit; citation_author=R. Hubley; citation_author=P. Green; citation_publication_date=2015; citation_publisher=RepeatMasker; "><meta name="citation_reference" content="citation_title=Transcription factors and 3D genome conformation in cell-fate decisions; citation_author=R. Stadhouders; citation_author=G.J. Filion; citation_author=T. Graf; citation_journal_title=Nature; citation_volume=569; citation_firstpage=345; citation_lastpage=354; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Phase separation drives heterochromatin domain formation; citation_author=A.R. Strom; citation_author=A.V. Emelyanov; citation_author=M. Mir; citation_author=D.V. Fyodorov; citation_author=X. Darzacq; citation_author=G.H. Karpen; citation_journal_title=Nature; citation_volume=547; citation_firstpage=241; citation_lastpage=245; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Accounting for GC-content bias reduces systematic errors and batch effects in ChIP-seq data; citation_author=M. Teng; citation_author=R.A. Irizarry; citation_journal_title=Genome Res.; citation_volume=27; citation_firstpage=1930; citation_lastpage=1938; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The Immune Landscape of Cancer; citation_author=V. Thorsson; citation_author=D.L. Gibbs; citation_author=S.D. Brown; citation_author=D. Wolf; citation_author=D.S. Bortone; citation_author=T.H. Ou Yang; citation_author=E. Porta-Pardo; citation_author=G.F. Gao; citation_author=C.L. Plaisier; citation_author=J.A. Eddy; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Immunity; citation_volume=51; citation_firstpage=411; citation_lastpage=412; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Large hypomethylated blocks as a universal defining epigenetic alteration in human solid tumors; citation_author=W. Timp; citation_author=H.C. Bravo; citation_author=O.G. McDonald; citation_author=M. Goggins; citation_author=C. Umbricht; citation_author=M. Zeiger; citation_author=A.P. Feinberg; citation_author=R.A. Irizarry; citation_journal_title=Genome Med.; citation_volume=6; citation_firstpage=61; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=CpG island methylator phenotype in colorectal cancer; citation_author=M. Toyota; citation_author=N. Ahuja; citation_author=M. Ohe-Toyota; citation_author=J.G. Herman; citation_author=S.B. Baylin; citation_author=J.P. Issa; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=96; citation_firstpage=8681; citation_lastpage=8686; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Lamina-Associated Domains: Links with Chromosome Architecture, Heterochromatin, and Gene Repression; citation_author=B. van Steensel; citation_author=A.S. Belmont; citation_journal_title=Cell; citation_volume=169; citation_firstpage=780; citation_lastpage=791; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Spatial organization of chromatin domains and compartments in single chromosomes; citation_author=S. Wang; citation_author=J.H. Su; citation_author=B.J. Beliveau; citation_author=B. Bintu; citation_author=J.R. Moffitt; citation_author=C.T. Wu; citation_author=X. Zhuang; citation_journal_title=Science; citation_volume=353; citation_firstpage=598; citation_lastpage=602; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Tumor suppressor Pdcd4 attenuates Sin1 translation to inhibit invasion in colon carcinoma; citation_author=Q. Wang; citation_author=J. Zhu; citation_author=Y.-W. Wang; citation_author=Y. Dai; citation_author=Y.-L. Wang; citation_author=C. Wang; citation_author=J. Liu; citation_author=A. Baker; citation_author=N.H. Colburn; citation_author=H.-S. Yang; citation_journal_title=Oncogene; citation_volume=36; citation_firstpage=6225; citation_lastpage=6234; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Dysfunctional epigenetic aging of the normal colon and colorectal cancer risk; citation_author=T. Wang; citation_author=S.K. Maden; citation_author=G.E. Luebeck; citation_author=C.I. Li; citation_author=P.A. Newcomb; citation_author=C.M. Ulrich; citation_author=J.E. Joo; citation_author=D.D. Buchanan; citation_author=R.L. Milne; citation_author=M.C. Southey; citation_author=et al.; citation_journal_title=Clin. Epigenetics; citation_volume=12; citation_firstpage=5; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=BSMAP: whole genome bisulfite sequence MAPping program; citation_author=Y. Xi; citation_author=W. Li; citation_journal_title=BMC Bioinformatics; citation_volume=10; citation_firstpage=232; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Revealing Hi-C subcompartments by imputing inter-chromosomal chromatin interactions; citation_author=K. Xiong; citation_author=J. Ma; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=5069; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Unique DNA methylome profiles in CpG island methylator phenotype colon cancers; citation_author=Y. Xu; citation_author=B. Hu; citation_author=A.J. Choi; citation_author=B. Gopalan; citation_author=B.H. Lee; citation_author=M.F. Kalady; citation_author=J.M. Church; citation_author=A.H. Ting; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=283; citation_lastpage=291; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Super-Resolution Imaging of Higher-Order Chromatin Structures at Different Epigenomic States in Single Mammalian Cells; citation_author=J. Xu; citation_author=H. Ma; citation_author=J. Jin; citation_author=S. Uttam; citation_author=R. Fu; citation_author=Y. Huang; citation_author=Y. Liu; citation_journal_title=Cell Rep.; citation_volume=24; citation_firstpage=873; citation_lastpage=882; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=H3K36 methylation antagonizes PRC2-mediated H3K27 methylation; citation_author=W. Yuan; citation_author=M. Xu; citation_author=C. Huang; citation_author=N. Liu; citation_author=S. Chen; citation_author=B. Zhu; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=286; citation_firstpage=7983; citation_lastpage=7989; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Topology, structures, and energy landscapes of human chromosomes; citation_author=B. Zhang; citation_author=P.G. Wolynes; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=6062; citation_lastpage=6067; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Shape Transitions and Chiral Symmetry Breaking in the Energy Landscape of the Mitotic Chromosome; citation_author=B. Zhang; citation_author=P.G. Wolynes; citation_journal_title=Phys. Rev. Lett.; citation_volume=116; citation_firstpage=248101; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Model-based analysis of ChIP-Seq (MACS); citation_author=Y. Zhang; citation_author=T. Liu; citation_author=C.A. Meyer; citation_author=J. Eeckhoute; citation_author=D.S. Johnson; citation_author=B.E. Bernstein; citation_author=C. Nusbaum; citation_author=R.M. Myers; citation_author=M. Brown; citation_author=W. Li; citation_author=X.S. Liu; citation_journal_title=Genome Biol.; citation_volume=9; citation_firstpage=R137; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=DNA methylation loss in late-replicating domains is linked to mitotic cell division; citation_author=W. Zhou; citation_author=H.Q. Dinh; citation_author=Z. Ramjan; citation_author=D.J. Weisenberger; citation_author=C.M. Nicolet; citation_author=H. Shen; citation_author=P.W. Laird; citation_author=B.P. Berman; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=591; citation_lastpage=602; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Nuclear structure in cancer cells; citation_author=D. Zink; citation_author=A.H. Fischer; citation_author=J.A. Nickerson; citation_journal_title=Nat. Rev. Cancer; citation_volume=4; citation_firstpage=677; citation_lastpage=687; citation_publication_date=2004; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Widespread changes to DNA methylation and chromatin are well documented in cancer, but the fate of higher-order chromosomal structure remains obscure. Here we integrated topological maps for colon tumors and normal colons with epigenetic, transcriptional, and imaging data to characterize alterations to chromatin loops, topologically associated domains, and large-scale compartments. We found that spatial partitioning of the open and closed genome compartments is profoundly compromised in tumors. This reorganization is accompanied by compartment-specific hypomethylation and chromatin changes. Additionally, we identify a compartment at the interface between the canonical A and B compartments that is reorganized in tumors. Remarkably, similar shifts were evident in non-malignant cells that have accumulated excess divisions. Our analyses suggest that these topological changes repress stemness and invasion programs while inducing anti-tumor immunity genes and may therefore restrain malignant progression. Our findings call into question the conventional view that tumor-associated epigenomic alterations are primarily oncogenic.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=6798&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(20)30938-7&amp;be=3338&amp;fe=5410&amp;dc=4773&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749587493786,%22n%22:0,%22f%22:2701,%22dn%22:2702,%22dne%22:2702,%22c%22:2702,%22s%22:2702,%22ce%22:2702,%22rq%22:2702,%22rp%22:3315,%22rpe%22:4193,%22dl%22:3320,%22di%22:4637,%22ds%22:4773,%22de%22:5019,%22dc%22:5410,%22l%22:5410,%22le%22:5523%7D,%22navigation%22:%7B%7D%7D&amp;fp=3780&amp;fcp=3780&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-munder {
  display: inline-block;
  text-align: left;
}

mjx-over {
  text-align: left;
}

mjx-munder:not([limits="false"]) {
  display: inline-table;
}

mjx-munder > mjx-row {
  text-align: left;
}

mjx-under {
  padding-bottom: .1em;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-mtable {
  display: inline-block;
  text-align: center;
  vertical-align: .25em;
  position: relative;
  box-sizing: border-box;
  border-spacing: 0;
  border-collapse: collapse;
}

mjx-mstyle[size="s"] mjx-mtable {
  vertical-align: .354em;
}

mjx-labels {
  position: absolute;
  left: 0;
  top: 0;
}

mjx-table {
  display: inline-block;
  vertical-align: -.5ex;
  box-sizing: border-box;
}

mjx-table > mjx-itable {
  vertical-align: middle;
  text-align: left;
  box-sizing: border-box;
}

mjx-labels > mjx-itable {
  position: absolute;
  top: 0;
}

mjx-mtable[justify="left"] {
  text-align: left;
}

mjx-mtable[justify="right"] {
  text-align: right;
}

mjx-mtable[justify="left"][side="left"] {
  padding-right: 0 ! important;
}

mjx-mtable[justify="left"][side="right"] {
  padding-left: 0 ! important;
}

mjx-mtable[justify="right"][side="left"] {
  padding-right: 0 ! important;
}

mjx-mtable[justify="right"][side="right"] {
  padding-left: 0 ! important;
}

mjx-mtable[align] {
  vertical-align: baseline;
}

mjx-mtable[align="top"] > mjx-table {
  vertical-align: top;
}

mjx-mtable[align="bottom"] > mjx-table {
  vertical-align: bottom;
}

mjx-mtable[side="right"] mjx-labels {
  min-width: 100%;
}

mjx-mtr {
  display: table-row;
  text-align: left;
}

mjx-mtr[rowalign="top"] > mjx-mtd {
  vertical-align: top;
}

mjx-mtr[rowalign="center"] > mjx-mtd {
  vertical-align: middle;
}

mjx-mtr[rowalign="bottom"] > mjx-mtd {
  vertical-align: bottom;
}

mjx-mtr[rowalign="baseline"] > mjx-mtd {
  vertical-align: baseline;
}

mjx-mtr[rowalign="axis"] > mjx-mtd {
  vertical-align: .25em;
}

mjx-mtd {
  display: table-cell;
  text-align: center;
  padding: .215em .4em;
}

mjx-mtd:first-child {
  padding-left: 0;
}

mjx-mtd:last-child {
  padding-right: 0;
}

mjx-mtable > * > mjx-itable > *:first-child > mjx-mtd {
  padding-top: 0;
}

mjx-mtable > * > mjx-itable > *:last-child > mjx-mtd {
  padding-bottom: 0;
}

mjx-tstrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-labels[align="left"] > mjx-mtr > mjx-mtd {
  text-align: left;
}

mjx-labels[align="right"] > mjx-mtr > mjx-mtd {
  text-align: right;
}

mjx-mtd[extra] {
  padding: 0;
}

mjx-mtd[rowalign="top"] {
  vertical-align: top;
}

mjx-mtd[rowalign="center"] {
  vertical-align: middle;
}

mjx-mtd[rowalign="bottom"] {
  vertical-align: bottom;
}

mjx-mtd[rowalign="baseline"] {
  vertical-align: baseline;
}

mjx-mtd[rowalign="axis"] {
  vertical-align: .25em;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-msqrt {
  display: inline-block;
  text-align: left;
}

mjx-root {
  display: inline-block;
  white-space: nowrap;
}

mjx-surd {
  display: inline-block;
  vertical-align: top;
}

mjx-sqrt {
  display: inline-block;
  padding-top: .075em;
}

mjx-sqrt > mjx-box {
  border-top: .075em solid;
  padding-left: .03em;
  margin-left: -.03em;
}

mjx-sqrt.mjx-tall > mjx-box {
  padding-left: .3em;
  margin-left: -.3em;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D44B {
  padding: 0.683em 0.851em 0 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c1D465 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D457 {
  padding: 0.661em 0.412em 0.205em 0;
}

mjx-c.mjx-c1D458 {
  padding: 0.694em 0.521em 0.011em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D459 {
  padding: 0.694em 0.298em 0.011em 0;
}

mjx-c.mjx-c1D467 {
  padding: 0.442em 0.467em 0.011em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D43E {
  padding: 0.683em 0.889em 0 0;
}

mjx-c.mjx-c1D44F {
  padding: 0.694em 0.429em 0.011em 0;
}

mjx-c.mjx-c34 {
  padding: 0.677em 0.5em 0 0;
}

mjx-c.mjx-c38 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D44D {
  padding: 0.683em 0.723em 0 0;
}

mjx-c.mjx-c2211 {
  padding: 0.75em 1.056em 0.25em 0;
}

mjx-c.mjx-c223C {
  padding: 0.366em 0.773em 0 0;
}

mjx-c.mjx-c1D441 {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c1D435 {
  padding: 0.683em 0.759em 0 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D707 {
  padding: 0.442em 0.603em 0.216em 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-c1D6FC {
  padding: 0.442em 0.64em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c1D464 {
  padding: 0.442em 0.716em 0.011em 0;
}

mjx-c.mjx-c1D466 {
  padding: 0.442em 0.49em 0.205em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c2E {
  padding: 0.106em 0.278em 0 0;
}

mjx-c.mjx-c1D45C {
  padding: 0.442em 0.485em 0.011em 0;
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c1D448 {
  padding: 0.683em 0.76em 0.022em 0;
}

mjx-c.mjx-c1D440 {
  padding: 0.683em 1.044em 0 0;
}

mjx-c.mjx-c1D438 {
  padding: 0.68em 0.764em 0 0;
}

mjx-c.mjx-c1D493 {
  padding: 0.452em 0.529em 0.008em 0;
}

mjx-c.mjx-c1D43C {
  padding: 0.683em 0.497em 0 0;
}

mjx-c.mjx-c5B {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c5D {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c1D45D {
  padding: 0.442em 0.503em 0.194em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c7B {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c22EF {
  padding: 0.303em 0.776em 0 0;
}

mjx-c.mjx-c33 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c7D {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c1D454 {
  padding: 0.442em 0.477em 0.205em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c2208 {
  padding: 0.543em 0.667em 0.043em 0;
}

mjx-c.mjx-c7C {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D453 {
  padding: 0.705em 0.552em 0.205em 0;
}

mjx-c.mjx-c7B.MM-S7 {
  padding: 1.75em 0.902em 1.25em 0;
}

mjx-c.mjx-c210E {
  padding: 0.694em 0.576em 0.011em 0;
}

mjx-c.mjx-c1D70E {
  padding: 0.431em 0.571em 0.011em 0;
}

mjx-c.mjx-c2264 {
  padding: 0.631em 0.778em 0.119em 0;
}

mjx-c.mjx-c28.MM-SO {
  padding: 0.797em 0.422em 0.297em 0;
}

mjx-c.mjx-c29.MM-SO {
  padding: 0.797em 0.422em 0.297em 0;
}

mjx-c.mjx-c3E {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c28.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c1D436 {
  padding: 0.705em 0.76em 0.022em 0;
}

mjx-c.mjx-c29.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c1D43D {
  padding: 0.683em 0.633em 0.022em 0;
}

mjx-c.mjx-c1D434 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c36 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c35 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c5B.MM-S3 {
  padding: 0.975em 0.444em 0.475em 0;
}

mjx-c.mjx-c1D447 {
  padding: 0.677em 0.704em 0 0;
}

mjx-c.mjx-c1D437 {
  padding: 0.683em 0.828em 0 0;
}

mjx-c.mjx-c1D6FF {
  padding: 0.712em 0.452em 0.011em 0;
}

mjx-c.mjx-c5D.MM-S3 {
  padding: 0.975em 0.444em 0.475em 0;
}

mjx-c.mjx-c39 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c3C {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c2026 {
  padding: 0.106em 0.837em 0 0;
}

mjx-c.mjx-c1D6FE {
  padding: 0.442em 0.543em 0.215em 0;
}

mjx-c.mjx-c1D445 {
  padding: 0.683em 0.759em 0.022em 0;
}

mjx-c.mjx-c2F {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c221A {
  padding: 0.04em 0.853em 0.96em 0;
}

mjx-c.mjx-c1D700 {
  padding: 0.453em 0.466em 0.022em 0;
}

mjx-c.mjx-c2223.MM-SO {
  padding: 0.851em 0.278em 0.351em 0;
}

mjx-c.mjx-c1D70C {
  padding: 0.442em 0.517em 0.216em 0;
}

mjx-c.mjx-c1D70B {
  padding: 0.431em 0.57em 0.011em 0;
}

mjx-c.mjx-c394 {
  padding: 0.716em 0.833em 0 0;
}

mjx-c.mjx-c22C5 {
  padding: 0.303em 0.278em 0 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930938-7" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30938-7"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)30938-7"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930938-7" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-832444" aria-expanded="false" aria-controls="login-links-832444" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-832444" id="login-links--832444" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930938-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930938-7" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930938-7" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-832444" data-opens="#links-dialog-832444" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-832444" aria-modal="true"><button data-closes="#links-dialog-832444" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930938-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930938-7" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930938-7" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.07.030">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(20)30938-7" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0020-3"><span>Volume 182</span>, <span>Issue 6</span></a></span><span class="meta-panel__pages"><span>p1474-1489.e23</span></span><span class="meta-panel__onlineDate">September 17, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=182&amp;issue=6&amp;pii=S0092-8674%2819%29X0020-3" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Large-Scale Topological Changes Restrain Malignant Progression in Colorectal Cancer</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Sarah E.</span> <span property="familyName">Johnstone</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sarah E.</span> <span property="familyName">Johnstone</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Johnstone/Sarah+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">10</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Alejandro</span> <span property="familyName">Reyes</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alejandro</span> <span property="familyName">Reyes</span> <a class="orcid-id" href="https://orcid.org/0000-0001-8717-6612" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-8717-6612</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Data Sciences, Dana Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Reyes/Alejandro" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a role="doc-noteref">10</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Yifeng</span> <span property="familyName">Qi</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yifeng</span> <span property="familyName">Qi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Qi/Yifeng" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Carmen</span> <span property="familyName">Adriaens</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Carmen</span> <span property="familyName">Adriaens</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5186-2741" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5186-2741</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Adriaens/Carmen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Esmat</span> <span property="familyName">Hegazi</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Esmat</span> <span property="familyName">Hegazi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hegazi/Esmat" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Karin</span> <span property="familyName">Pelka</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Karin</span> <span property="familyName">Pelka</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2856-8602" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2856-8602</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Pelka/Karin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Jonathan H.</span> <span property="familyName">Chen</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jonathan H.</span> <span property="familyName">Chen</span> <a class="orcid-id" href="https://orcid.org/0000-0001-6074-9302" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-6074-9302</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Jonathan+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Luli S.</span> <span property="familyName">Zou</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Luli S.</span> <span property="familyName">Zou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Data Sciences, Dana Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zou/Luli+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Yotam</span> <span property="familyName">Drier</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yotam</span> <span property="familyName">Drier</span> <a class="orcid-id" href="https://orcid.org/0000-0003-1725-2995" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-1725-2995</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Lautenberg Center for Immunology and Cancer Research, The Hebrew University, Jerusalem, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Drier/Yotam" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Vivian</span> <span property="familyName">Hecht</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Vivian</span> <span property="familyName">Hecht</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hecht/Vivian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Noam</span> <span property="familyName">Shoresh</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Noam</span> <span property="familyName">Shoresh</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Shoresh/Noam" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Martin K.</span> <span property="familyName">Selig</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Martin K.</span> <span property="familyName">Selig</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Selig/Martin+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Caleb A.</span> <span property="familyName">Lareau</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Caleb A.</span> <span property="familyName">Lareau</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Program in Biological and Biomedical Sciences, Harvard Medical School, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lareau/Caleb+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Sowmya</span> <span property="familyName">Iyer</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sowmya</span> <span property="familyName">Iyer</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Iyer/Sowmya" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Son C.</span> <span property="familyName">Nguyen</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Son C.</span> <span property="familyName">Nguyen</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2187-1027" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2187-1027</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics, Penn Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nguyen/Son+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Eric F.</span> <span property="familyName">Joyce</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Eric F.</span> <span property="familyName">Joyce</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0418-2804" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0418-2804</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics, Penn Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Joyce/Eric+F" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Nir</span> <span property="familyName">Hacohen</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nir</span> <span property="familyName">Hacohen</span> <a class="orcid-id" href="https://orcid.org/0000-0002-2349-2656" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-2349-2656</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hacohen/Nir" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Rafael A.</span> <span property="familyName">Irizarry</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Rafael A.</span> <span property="familyName">Irizarry</span> <a class="orcid-id" href="https://orcid.org/0000-0002-3944-4309" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-3944-4309</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Data Sciences, Dana Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Irizarry/Rafael+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Bin</span> <span property="familyName">Zhang</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bin</span> <span property="familyName">Zhang</span> <a class="orcid-id" href="https://orcid.org/0000-0002-3685-7503" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-3685-7503</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Bin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Martin J.</span> <span property="familyName">Aryee</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Martin J.</span> <span property="familyName">Aryee</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:aryee.martin@mgh.harvard.edu" property="email" title="Send email to Martin J. Aryee">aryee.martin@mgh.harvard.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Aryee/Martin+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup> <a id="eadd0010" href="mailto:aryee.martin@mgh.harvard.edu" property="email" title="Send email to Martin J. Aryee">aryee.martin@mgh.harvard.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Bradley E.</span> <span property="familyName">Bernstein</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bradley E.</span> <span property="familyName">Bernstein</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:bernstein.bradley@mgh.harvard.edu" property="email" title="Send email to Bradley E. Bernstein">bernstein.bradley@mgh.harvard.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bernstein/Bradley+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">11</a></sup> <a id="eadd0015" href="mailto:bernstein.bradley@mgh.harvard.edu" property="email" title="Send email to Bradley E. Bernstein">bernstein.bradley@mgh.harvard.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Center for Cancer Research, Massachusetts General Hospital, Boston, MA 02129, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Data Sciences, Dana Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Biostatistics, Harvard School of Public Health, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">The Lautenberg Center for Immunology and Cancer Research, The Hebrew University, Jerusalem, Israel</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Program in Biological and Biomedical Sciences, Harvard Medical School, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Department of Genetics, Penn Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received July 8, 2019</span>; <span>Revised May 4, 2020</span>; <span>Accepted July 20, 2020</span>; <span>Published online August 24, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2020.07.030" property="sameAs" target="_blank">10.1016/j.cell.2020.07.030 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867420309387" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2020.07.030" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0020-3"><img src="/cms/asset/8b612a3d-aab1-48b5-91b2-a988671c5946/cov200h.gif" alt="Cover Image - Cell, Volume 182, Issue 6"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2820%2930938-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (7)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930938-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.07.030">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2930938-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.07.030&amp;pii=S0092-8674%2820%2930938-7" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930938-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.07.030">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.07.030" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867420309387&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867420309387&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=182&amp;issue=6&amp;pii=S0092-8674%2819%29X0020-3" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)30998-3" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)30947-8" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental Information (7)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Hierarchical layers of nuclear architecture are altered in colorectal tumors</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">An intermediate genome compartment is defined in primary tissues</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Compartmental reorganization and hypomethylation occur in tumors and aging cells</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Reorganization is associated with tumor-suppressive transcriptional programs</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Widespread changes to DNA methylation and chromatin are well documented in cancer, but the fate of higher-order chromosomal structure remains obscure. Here we integrated topological maps for colon tumors and normal colons with epigenetic, transcriptional, and imaging data to characterize alterations to chromatin loops, topologically associated domains, and large-scale compartments. We found that spatial partitioning of the open and closed genome compartments is profoundly compromised in tumors. This reorganization is accompanied by compartment-specific hypomethylation and chromatin changes. Additionally, we identify a compartment at the interface between the canonical A and B compartments that is reorganized in tumors. Remarkably, similar shifts were evident in non-malignant cells that have accumulated excess divisions. Our analyses suggest that these topological changes repress stemness and invasion programs while inducing anti-tumor immunity genes and may therefore restrain malignant progression. Our findings call into question the conventional view that tumor-associated epigenomic alterations are primarily oncogenic.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/a018e577-15ce-456f-8124-773cface0a70/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2020.07.030/asset/76be94dc-f954-4050-9cf9-83953f2cf9ca/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/a018e577-15ce-456f-8124-773cface0a70/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;genome+topology&quot;" alt="Search on this keyword" rel="nofollow">genome topology</a></li><li><a href="/action/doSearch?AllField=&quot;chromatin&quot;" alt="Search on this keyword" rel="nofollow">chromatin</a></li><li><a href="/action/doSearch?AllField=&quot;DNA+methylation&quot;" alt="Search on this keyword" rel="nofollow">DNA methylation</a></li><li><a href="/action/doSearch?AllField=&quot;colon+cancer&quot;" alt="Search on this keyword" rel="nofollow">colon cancer</a></li><li><a href="/action/doSearch?AllField=&quot;epigenetics&quot;" alt="Search on this keyword" rel="nofollow">epigenetics</a></li><li><a href="/action/doSearch?AllField=&quot;nuclear+architecture&quot;" alt="Search on this keyword" rel="nofollow">nuclear architecture</a></li><li><a href="/action/doSearch?AllField=&quot;compartment&quot;" alt="Search on this keyword" rel="nofollow">compartment</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">For over a century, pathologists have observed changes in the shape, size, and chromatin texture of cancer cell nuclei (<span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Zink et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Zink, D. ∙ Fischer, A.H. ∙ Nickerson, J.A.</div><div><strong>Nuclear structure in cancer cells</strong></div><div><em>Nat. Rev. Cancer.</em> 2004; <strong>4</strong>:677-687</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrc1430" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-4544237763" target="_blank" aria-label="Open Scopus in new tab">Scopus (636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15343274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrc1430&amp;pmid=15343274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Nuclear features help determine cancer subtype and grade, with significant effects on prognosis and therapy. However, despite their diagnostic and clinical importance, the molecular underpinnings of these cancer-associated morphological changes remain poorly understood.</div><div id="p0035" role="paragraph">Molecular and genetic studies have documented widespread epigenetic defects in human tumors, including chromatin regulator mutations, DNA methylation changes, and altered enhancer landscapes (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Baylin and Jones, 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Baylin, S.B. ∙ Jones, P.A.</div><div><strong>Epigenetic Determinants of Cancer</strong></div><div><em>Cold Spring Harb. Perspect. Biol.</em> 2016; <strong>8</strong>:a019505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a019505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84986211886" target="_blank" aria-label="Open Scopus in new tab">Scopus (303)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27194046/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a019505&amp;pmid=27194046" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Certain cancers also harbor mutations of proteins that regulate higher-order chromosomal structure (“genome topology”), including CTCF and cohesin subunits (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Corces and Corces, 2016</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Corces, M.R. ∙ Corces, V.G.</div><div><strong>The three-dimensional cancer genome</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2016; <strong>36</strong>:1-7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gde.2016.01.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84956693585" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26855137/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gde.2016.01.002&amp;pmid=26855137" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Focal topological alterations drive oncogenic transcriptional programs in specific contexts (<span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Flavahan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Flavahan, W.A. ∙ Drier, Y. ∙ Liau, B.B. ...</div><div><strong>Insulator dysfunction and oncogene activation in IDH mutant gliomas</strong></div><div><em>Nature.</em> 2016; <strong>529</strong>:110-114</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16490" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84953924597" target="_blank" aria-label="Open Scopus in new tab">Scopus (541)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26700815/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16490&amp;pmid=26700815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" aria-label="Reference 2019" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">2019</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Flavahan, W.A. ∙ Drier, Y. ∙ Johnstone, S.E. ...</div><div><strong>Altered chromosomal topology drives oncogenic programs in SDH-deficient GISTs</strong></div><div><em>Nature.</em> 2019; <strong>575</strong>:229-233</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1668-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85074865462" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31666694/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1668-3&amp;pmid=31666694" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Hnisz et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hnisz, D. ∙ Weintraub, A.S. ∙ Day, D.S. ...</div><div><strong>Activation of proto-oncogenes by disruption of chromosome neighborhoods</strong></div><div><em>Science.</em> 2016; <strong>351</strong>:1454-1458</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad9024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84960171833" target="_blank" aria-label="Open Scopus in new tab">Scopus (425)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26940867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad9024&amp;pmid=26940867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Kloetgen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Kloetgen, A. ∙ Thandapani, P. ∙ Ntziachristos, P. ...</div><div><strong>Three-dimensional chromatin landscapes in T&nbsp;cell acute lymphoblastic leukemia</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:388-400</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0602-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85082197331" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32203470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0602-9&amp;pmid=32203470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Despite this, genome topology and nuclear organization in human tumors are largely uncharted.</div><div id="p0040" role="paragraph">Technological innovations, including Hi-C (<span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Rao, S.S.P. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), have revealed hierarchical layers of spatial organization (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Bickmore and van Steensel, 2013</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bickmore, W.A. ∙ van Steensel, B.</div><div><strong>Genome architecture: domain organization of interphase chromosomes</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:1270-1284</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2013.02.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2013.02.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84875190221" target="_blank" aria-label="Open Scopus in new tab">Scopus (448)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23498936/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.02.001&amp;pmid=23498936" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Dekker and Misteli, 2015</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Dekker, J. ∙ Misteli, T.</div><div><strong>Long-Range Chromatin Interactions</strong></div><div><em>Cold Spring Harb. Perspect. Biol.</em> 2015; <strong>7</strong>:a019356</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a019356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84943250456" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26430217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a019356&amp;pmid=26430217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Rowley and Corces, 2018</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Rowley, M.J. ∙ Corces, V.G.</div><div><strong>Organizational principles of 3D genome architecture</strong></div><div><em>Nat. Rev. Genet.</em> 2018; <strong>19</strong>:789-800</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-018-0060-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85055726389" target="_blank" aria-label="Open Scopus in new tab">Scopus (237)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30367165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-018-0060-8&amp;pmid=30367165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Chromatin loops occur when distant loci on the linear chromosome are in frequent contact and typically involve interactions between CTCF-bound sites (CTCF loops) and/or between enhancers and promoters (E-P loops). Topologically associating domains (TADs) are sub-megabase regions partitioned by boundary elements often bound by CTCF and cohesin. Finally, the genome is grossly partitioned into two large-scale compartments: an open, transcriptionally active A compartment and a compact, relatively silent B compartment.</div><div id="p0045" role="paragraph">Colorectal adenocarcinoma, the fourth most common epithelial tumor, is a well-characterized model for cancer epigenetics. These tumors exhibit profoundly altered DNA methylation landscapes, including loss of methylation from large genomic ‘‘blocks’’ that cover over half of the genome (<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Berman et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Berman, B.P. ∙ Weisenberger, D.J. ∙ Aman, J.F. ...</div><div><strong>Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains</strong></div><div><em>Nat. Genet.</em> 2011; <strong>44</strong>:40-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.969" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84655162785" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22120008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.969&amp;pmid=22120008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Hansen et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hansen, K.D. ∙ Timp, W. ∙ Bravo, H.C. ...</div><div><strong>Increased methylation variation in epigenetic domains across cancer types</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:768-775</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.865" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79960927422" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21706001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.865&amp;pmid=21706001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The functional implications of hypomethylation remain obscure, but it may impact genome stability and/or transcriptional activity (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Baylin and Jones, 2016</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Baylin, S.B. ∙ Jones, P.A.</div><div><strong>Epigenetic Determinants of Cancer</strong></div><div><em>Cold Spring Harb. Perspect. Biol.</em> 2016; <strong>8</strong>:a019505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a019505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84986211886" target="_blank" aria-label="Open Scopus in new tab">Scopus (303)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27194046/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a019505&amp;pmid=27194046" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conversely, CpG island hypermethylation occurs in a subset of colon tumors, termed the CpG island methylator phenotype (CIMP), and is associated with promoter silencing (<span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Hinoue et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hinoue, T. ∙ Weisenberger, D.J. ∙ Lange, C.P.E. ...</div><div><strong>Genome-scale analysis of aberrant DNA methylation in colorectal cancer</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:271-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.117523.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84856549042" target="_blank" aria-label="Open Scopus in new tab">Scopus (443)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21659424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.117523.110&amp;pmid=21659424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Toyota et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Toyota, M. ∙ Ahuja, N. ∙ Ohe-Toyota, M. ...</div><div><strong>CpG island methylator phenotype in colorectal cancer</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1999; <strong>96</strong>:8681-8686</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.96.15.8681" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-0033587747" target="_blank" aria-label="Open Scopus in new tab">Scopus (1985)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10411935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.96.15.8681&amp;pmid=10411935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, our understanding of these epigenomic changes and their relationship to genome topology has been hindered by a lack of data for primary tumors.</div><div id="p0050" role="paragraph">Here we mapped genome topology across a cohort of colon tumors, normal colons, and colon cancer cell lines and compared successive layers of topology. We focus on large-scale reorganization of the conventional genome compartments A and B and characterize an intermediate compartment at their interface. Remarkably, compartmental reorganization is associated with repression of stem cell, invasion, and metastasis genes and induction of genes associated with anti-tumor immunity. Our results suggest that the most profound topological alterations in tumors are actually a consequence of accumulated cell divisions and that they may have a tumor-suppressive role.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Maps of DNA Methylation, Chromatin State, and Topology in Human Tumors</h3><div id="p0055" role="paragraph">To understand how nuclear architecture is altered in cancer, we profiled genome topology along with DNA methylation, chromatin modifications, and CTCF in primary colon tumors, normal colons, and colon cancer cell lines (<a id="crosref0340" href="#fig1">Figure&nbsp;1</a>A). Our clinical cohort included 26 tumors and 7 normal colon tissue samples (<a id="crosref0345" href="#mmc1">Table S1</a>). Our <i>in&nbsp;vitro</i> models included colon cancer cell lines (HCT116, SW480, RKO, and LS-174T), a line derived from normal colon (FHC), and primary fibroblasts (WI-38). Our full dataset comprises 175 libraries and 28 billion sequencing reads for Hi-C, HiChIP, bisulfite sequencing, chromatin immunoprecipitation sequencing (ChIP-seq) and RNA sequencing (RNA-seq).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/bac858f8-6805-4a50-b10f-f115877c54c5/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2020.07.030/asset/c0931a50-3b49-4d0a-b3e3-b0050d98c02f/main.assets/gr1.jpg" height="707" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/bac858f8-6805-4a50-b10f-f115877c54c5/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Integrated Topological Maps Reveal Tumor-Specific Chromatin Loops and Stable TAD Structure</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Schematic of hierarchical genome organization with indication of genomic scale (left) and summary of genome-wide assays (center) and models (right).</div><div id="fspara0020" role="paragraph">(B) Volcano plot presenting a differential analysis of loops between tumors and normal samples. Loops, represented as dots, with significantly stronger or weaker interactions in tumors compared with normal colon are highlighted in red and green, respectively.</div><div id="fspara0025" role="paragraph">(C) Boxplots depicting expression fold change (log<sub>2</sub>) between tumors and normal samples (y axis) for genes engaged in enhancer-promoter (E-P) loops. Genes are stratified by change in E-P loop strength between tumors and normal colon (x axis).</div><div id="fspara0030" role="paragraph">(D) Genomic view of the <i>EPHA2</i> locus (∼130 kb), showing SMC1 HiChIP loops (arcs) and H3K27ac enrichment for normal colon (green) and colon tumor (purple). The width of the arcs corresponds to the average loop strength summarized for the set of 2 normal and 7 colon samples. An asterisk indicates the differential loop (<a id="crosref1850" href="#sec-4">STAR Methods</a>).</div><div id="fspara0035" role="paragraph">(E) Genomic view of the <i>PDCD4</i> locus (∼100 kb) as in (D).</div><div id="fspara0040" role="paragraph">(F) Hi-C contact map showing pairwise contact frequencies (red heat) between genomic positions across chromosome 7 (rows, columns) in normal colon. Top: Hi-C eigenvector (PC1) based on long-range interactions demarcates compartments A (positive values, blue) and B (negative values, yellow). Right: inset with a magnified view of a representative region reveals TAD structures (highlighted by black triangles). Rotation of this inset by 45° yields a horizontal display of TAD structures (see G).</div><div id="fspara0045" role="paragraph">(G) Horizonal heatmaps showing local Hi-C contact patterns (red heat) across chromosome 14 for normal colon (green), colon tumors (purple), and cell lines (black). Exemplar TAD boundaries are indicated by black arrows.</div></div></div></div></figcaption></figure></div><div id="p0060" role="paragraph">We performed hybrid-capture bisulfite sequencing on 26 tumors, 3 normal colons, and 5 cell lines. Although only two tumors had CpG island hypermethylation (CIMP) (<a id="crosref0350" href="#figs1">Figure&nbsp;S1</a>A), all tumors exhibited degrees of hypomethylation across expansive genomic regions, termed “hypomethylated blocks” (<a id="crosref0355" href="#figs1">Figure&nbsp;S1</a>B; <span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Berman et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Berman, B.P. ∙ Weisenberger, D.J. ∙ Aman, J.F. ...</div><div><strong>Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains</strong></div><div><em>Nat. Genet.</em> 2011; <strong>44</strong>:40-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.969" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84655162785" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22120008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.969&amp;pmid=22120008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Hansen et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hansen, K.D. ∙ Timp, W. ∙ Bravo, H.C. ...</div><div><strong>Increased methylation variation in epigenetic domains across cancer types</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:768-775</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.865" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79960927422" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21706001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.865&amp;pmid=21706001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We also inferred copy number variants (CNVs) for each tumor (<a id="crosref0370" href="#mmc2">Table S2</a>) and controlled for CNV-related variability in further analyses by incorporating terms for copy number estimates into our linear models and verifying results in CNV-stable regions and tumors (<a id="crosref0375" href="#sec-4">STAR Methods</a>).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/6a4ce885-1da1-4c54-a7cb-42356b9c50a1/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2020.07.030/asset/d7af530c-4545-4d2f-aa8a-4368752e0136/main.assets/figs1.jpg" height="132" width="498" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6a4ce885-1da1-4c54-a7cb-42356b9c50a1/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">CpG Island and Block methylation, Related to <a id="crosref1725" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0285" role="paragraph">(A) Boxplot representation of the distribution of DNA methylation values across CpG islands for each tumor sample (x axis). Tumor samples are shown in purple and normal samples are shown in green.</div><div id="fspara0290" role="paragraph">(B) For each sample (x axis), boxplot representation of the distribution of DNA methylation values across open sea CpGs in hypomethylated blocks. Tumor samples are shown in purple and normal samples are shown in green.</div></div></div></div></figcaption></figure></div><div id="p0065" role="paragraph">We integrated high-resolution topological maps and epigenomics data to investigate successive layers of genome organization, from chromatin loops to TADs to large-scale genome compartments, in tissues, tumors, and cell lines.</div></section><section id="sec-2-2"><h3>E-P Loops Are Associated with Oncogenic Transcriptional Programs</h3><div id="p0070" role="paragraph">We began by identifying loops that could influence transcriptional states in tumors. HiChIP assays targeting the cohesin subunit SMC1 (<span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Mumbach et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Mumbach, M.R. ∙ Rubin, A.J. ∙ Flynn, R.A. ...</div><div><strong>HiChIP: efficient and sensitive analysis of protein-directed genome architecture</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:919-922</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84988336216" target="_blank" aria-label="Open Scopus in new tab">Scopus (469)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27643841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3999&amp;pmid=27643841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) reveal CTCF-CTCF loops, which contribute to TAD boundaries, as well as E-P loops, which are hypothesized to mediate enhancer gene activation (<span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Stadhouders et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Stadhouders, R. ∙ Filion, G.J. ∙ Graf, T.</div><div><strong>Transcription factors and 3D genome conformation in cell-fate decisions</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:345-354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1182-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85065766396" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31092938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1182-7&amp;pmid=31092938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We identified 25,125 loops in normal colon or tumors and annotated the subset that connects an histone H3 lysine 27 acetylation (H3K27ac) peak (enhancer-like) to an annotated promoter as E-P loops (n&nbsp;= 14,121). Differential analysis revealed 571 E-P loops that are stronger in tumors and 248 that are weaker in tumors (<a id="crosref0390" href="#fig1">Figure&nbsp;1</a>B). To relate these differential loops to transcription, we evaluated the expression of the corresponding genes in our cohort and across 521 samples from The Cancer Genome Atlas (TCGA) (<span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Cancer Genome Atlas Network, 2012</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cancer Genome Atlas Network</div><div><strong>Comprehensive molecular characterization of human colon and rectal cancer</strong></div><div><em>Nature.</em> 2012; <strong>487</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84863922124" target="_blank" aria-label="Open Scopus in new tab">Scopus (4694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22810696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11252&amp;pmid=22810696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genes connected to E-P loops that were stronger in tumors were upregulated in tumors, whereas genes connected to E-P loops that were weaker in tumors were downregulated (<a id="crosref0400" href="#fig1">Figures 1</a>C and <a id="crosref0405" href="#figs2">S2</a>A; <a id="crosref0410" href="#mmc3">Table S3</a>). This association was evident even when excluding loci subject to CNVs (<a id="crosref0415" href="#figs2">Figure&nbsp;S2</a>B).</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/6bd9f6f0-ffd1-4410-8ba1-09a6c836ceda/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2020.07.030/asset/150c1482-2aa6-4529-aefb-6e18a9d108b7/main.assets/figs2.jpg" height="923" width="762" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6bd9f6f0-ffd1-4410-8ba1-09a6c836ceda/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Chromatin Loops, Related to <a id="crosref1730" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0300" role="paragraph">(A) Boxplots depict expression fold-change (log<sub>2</sub>) between tumors and normal samples (y axis) for genes engaged in enhancer-promoter (E-P) loops. Genes are stratified by change in E-P loop strength between tumors and normal (x axis). This plot is equivalent to <a id="crosref1735" href="#fig1">Figure&nbsp;1</a>C, but uses RNA-seq data from our cohort instead of TCGA data.</div><div id="fspara0305" role="paragraph">(B) Boxplot representation of the distribution of expression log (base 2) fold-changes between tumors and normal stratified by the strength of loop strength fold changes (x axis). This plot is equivalent to <a id="crosref1740" href="#fig1">Figure&nbsp;1</a>C, but includes only copy number stable loci.</div><div id="fspara0310" role="paragraph">(C) Dot plot shows normalized (<i>EPHA2</i> E-P) loop counts for each normal (green) and tumor (purple) sample. Counts are shown for the differential loop highlighted in <a id="crosref1745" href="#fig2">Figure&nbsp;2</a>D.</div><div id="fspara0315" role="paragraph">(D) Dot plot shows <i>EPHA2</i> expression levels for normal colon (green) and tumor (purple). Each point represents an RNA-seq sample from our cohort.</div><div id="fspara0320" role="paragraph">(E) Boxplots depict <i>EPHA2</i> expression in 41 normal colon samples and 480 colon tumors from TCGA.</div><div id="fspara0325" role="paragraph">(F) Normalized (<i>PDCD4</i> E-P) loop counts are shown for each normal (green) and tumor (purple) sample. Counts are shown for the differential loop highlighted in <a id="crosref1750" href="#fig2">Figure&nbsp;2</a>E.</div><div id="fspara0330" role="paragraph">(G) Dot plot shows <i>PDCD4</i> expression levels for normal colons (green) and tumors (purple). Each point represents an RNA-seq sample from our cohort.</div><div id="fspara0335" role="paragraph">(H) Boxplots depict <i>PDCD4</i> expression in 41 normal colon samples and 480 colon tumors from TCGA.</div><div id="fspara0340" role="paragraph">(I-K) Genomic views of the TERC locus (F), COLCA1/2 locus (G) and CXCR4 locus (H). Upper panels show SMC1 HiChIP loops as gray arcs; middle panels show H3K27ac signal in colon tumors in purple. Colon cancer risk SNP positions are indicated by blue lines. H3K27ac peaks with coincident loop anchors are indicated in orange.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">Several topological alterations involved known oncogenes or tumor suppressors. For example, although the locus encoding the receptor tyrosine kinase oncogene <i>EPHA2</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Dunne et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Dunne, P.D. ∙ Dasgupta, S. ∙ Blayney, J.K. ...</div><div><strong>EphA2 Expression Is a Key Driver of Migration and Invasion and a Poor Prognostic Marker in Colorectal Cancer</strong></div><div><em>Clin. Cancer Res.</em> 2016; <strong>22</strong>:230-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-15-0603" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84954526582" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26283684/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-15-0603&amp;pmid=26283684" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) contains multiple enhancers in normal colon and tumors, the gene is connected to a strong enhancer by a tumor-associated E-P loop (<a id="crosref0425" href="#fig1">Figures 1</a>D and <a id="crosref0430" href="#figs2">S2</a>C). Accordingly, <i>EPHA2</i> is upregulated in colon tumors (<a id="crosref0435" href="#figs2">Figures S2</a>D and S2E). Conversely, the <i>PDCD4</i> tumor suppressor (<span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Wang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Wang, Q. ∙ Zhu, J. ∙ Wang, Y.-W. ...</div><div><strong>Tumor suppressor Pdcd4 attenuates Sin1 translation to inhibit invasion in colon carcinoma</strong></div><div><em>Oncogene.</em> 2017; <strong>36</strong>:6225-6234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/onc.2017.228" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85033585599" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28692058/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fonc.2017.228&amp;pmid=28692058" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) loses an E-P loop to a distal enhancer and is downregulated in tumors (<a id="crosref0445" href="#fig1">Figures 1</a>E and <a id="crosref0450" href="#figs2">S2</a>F–S2H). The annotated E-P loops can also facilitate interpretation of single-nucleotide polymorphisms (SNPs) associated with colon cancer risk (<a id="crosref0455" href="#figs2">Figures S2</a>I–S2K; <a id="crosref0460" href="#mmc4">Table S4</a>; <a id="crosref0465" href="#sec-4">STAR Methods</a>).</div></section><section id="sec-2-3"><h3>Topological Boundaries Are Largely Retained in Tumors</h3><div id="p0080" role="paragraph">A next layer of topological organization involves TADs and their boundaries. TADs are evident in Hi-C maps as sub-megabase-scale regions with increased intra-domain interactions (<a id="crosref0470" href="#fig1">Figure&nbsp;1</a>F; <span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Dixon et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Dixon, J.R. ∙ Jung, I. ∙ Selvaraj, S. ...</div><div><strong>Chromatin architecture reorganization during stem cell differentiation</strong></div><div><em>Nature.</em> 2015; <strong>518</strong>:331-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14222" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84923366733" target="_blank" aria-label="Open Scopus in new tab">Scopus (698)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25693564/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14222&amp;pmid=25693564" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CTCF occupies and contributes to the stability of many TAD boundaries (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Rao, S.S.P. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Nora et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Nora, E.P. ∙ Goloborodko, A. ∙ Valton, A.-L. ...</div><div><strong>Targeted Degradation of CTCF Decouples Local Insulation of Chromosome Domains from Genomic Compartmentalization</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:930-944.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2017.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2017.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85019568830" target="_blank" aria-label="Open Scopus in new tab">Scopus (517)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28525758/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.004&amp;pmid=28525758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We used Hi-C data to assess the locations and integrity of TAD boundaries genome wide (<a id="crosref0490" href="#sec-4">STAR Methods</a>). Boundary location and strength were largely concordant between tumors, normal controls, and cell lines (<a id="crosref0495" href="#fig1">Figures 1</a>G and <a id="crosref0500" href="#figs3">S3</a>A–S3D), consistent with previous studies of cell lines and non-malignant tissues (<span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Dixon et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Dixon, J.R. ∙ Selvaraj, S. ∙ Yue, F. ...</div><div><strong>Topological domains in mammalian genomes identified by analysis of chromatin interactions</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:376-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84861095603" target="_blank" aria-label="Open Scopus in new tab">Scopus (3066)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11082&amp;pmid=22495300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Krefting et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Krefting, J. ∙ Andrade-Navarro, M.A. ∙ Ibn-Salem, J.</div><div><strong>Evolutionary stability of topologically associating domains is associated with conserved gene regulation</strong></div><div><em>BMC Biol.</em> 2018; <strong>16</strong>:87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12915-018-0556-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85058139752" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30086749/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12915-018-0556-x&amp;pmid=30086749" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Nora et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Nora, E.P. ∙ Lajoie, B.R. ∙ Schulz, E.G. ...</div><div><strong>Spatial partitioning of the regulatory landscape of the X-inactivation centre</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:381-385</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84861100147" target="_blank" aria-label="Open Scopus in new tab">Scopus (1429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495304/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11049&amp;pmid=22495304" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-1" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-1" aria-controls="sref81-1">Schmitt et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref81-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Schmitt, A.D. ∙ Hu, M. ∙ Jung, I. ...</div><div><strong>A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome</strong></div><div><em>Cell Rep.</em> 2016; <strong>17</strong>:2042-2059</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85003815206" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27851967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2016.10.061&amp;pmid=27851967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Tumors, on average, shared 92% of TAD boundaries with normal colon and 89% with the cell lines. Visual inspection revealed that discordant boundary calls were most often caused by subtle differences in strength rather than complete boundary loss.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/5c980ed7-ddc4-4d79-a858-6e7360599ee4/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2020.07.030/asset/361f8a5c-6595-4fb2-8825-b0848822272d/main.assets/figs3.jpg" height="743" width="762" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/5c980ed7-ddc4-4d79-a858-6e7360599ee4/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Topologically Associated Domains, Related to <a id="crosref1755" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0350" role="paragraph">(A) Horizonal heatmaps show local Hi-C contact patterns (red heat) across chromosome 14 for normal colon (green), colon tumors (purple) and cell lines (black). Validation cohort is highlighted (vertical bars, left).</div><div id="fspara0355" role="paragraph">(B) Heatmap shows pairwise correlations between genome wide TAD boundary scores (blue heat) in normal colons (green), colon tumors (purple) and cell lines (gray). These samples (rows, columns) are ordered according to a complete linkage hierarchical clustering (top). Original cohort is indicated by white squares and validation cohort is in black squares.</div><div id="fspara0360" role="paragraph">(C) Metaplot of the 40Kb-resolution boundary scores are shown for each Hi-C sample. Each panel represents one HiC sample, each row of the heatmap represent one TAD boundary, and the columns represent the genomic position relative to the TAD boundary. Original cohort indicated by gray outlines and validation cohort is indicated by black outlines.</div><div id="fspara0365" role="paragraph">(D) Bar plot of TAD boundary conservation analysis using the approach by Schmitt et&nbsp;al., 2016. Data are summarized over both the original and validation cohorts. Plot shows the number of tumors where TAD boundaries are called at the same location as TADs in colon normal tissues (x axis). The y axis shows the fraction of normal colon TAD boundaries. The overall conservation of TAD boundaries is similar to what has been described across different tissue types (<span class="dropBlock reference-citations"><a id="body-ref-sref81-2" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-2" aria-controls="sref81-2">Schmitt et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref81-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Schmitt, A.D. ∙ Hu, M. ∙ Jung, I. ...</div><div><strong>A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome</strong></div><div><em>Cell Rep.</em> 2016; <strong>17</strong>:2042-2059</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85003815206" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27851967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2016.10.061&amp;pmid=27851967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0370" role="paragraph">(E) Boxplots depict DNA methylation (black) and CTCF binding (gray) for CTCF binding sites that are differential between CIMP and non-CIMP tumors. Data shown for normal colon, non-CIMP tumors and CIMP tumors.</div><div id="fspara0375" role="paragraph">(F) Volcano plot shows differential analysis of CTCF binding sites (points) between CIMP and non-CIMP tumors (points to the upper left represent CTCF binding sites lost in CIMP tumors). Sites that are hypermethylated in CIMP tumors relative to non-CIMP samples are highlighted (red; methylation difference &gt; 15%).</div><div id="fspara0380" role="paragraph">(G) Left: Cartoon schematic of Hi-C heatmap shows a strong loop peak corresponding to an interaction between two CTCF bound loop anchors flanking a TAD (top panel). This theoretical CTCF-CTCF loop interaction is weakened in a sample with reduced CTCF binding at one or both anchors (bottom). Right: Heatmaps show actual Hi-C signals aggregated over CTCF-CTCF loops, revealing interaction peaks (i.e., averaged signal for the pixels corresponding to the tops of the TAD triangles illustrated at left). Top: Heatmaps aggregate signals for loops whose CTCF anchors are stable in normal colon, non-CIMP tumors and CIMP tumors (top). Bottom: Heatmaps aggregate signals for loops whose CTCF anchors are lost in CIMP tumors. These loop anchor interactions are weakened in CIMP tumors.</div><div id="fspara0385" role="paragraph">(H) Boxplots depict fold-change (log<sub>2</sub>) in E-P loop strength between tumors and normal. Loops crossing TAD boundaries are shown. Loops are stratified according to whether the TAD boundary that they span loses CTCF binding and gains methylation in CIMP tumors (lost) or whether it retains CTCF (stable).</div><div id="fspara0390" role="paragraph">(I) Boxplots depict expression fold-change (log<sub>2</sub>) between CIMP and non-CIMP tumors stratified by whether the genes are located in a disrupted TAD or not.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Prior studies have shown that CTCF boundaries may be disrupted by genetic deletion or hypermethylation (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Flavahan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Flavahan, W.A. ∙ Drier, Y. ∙ Liau, B.B. ...</div><div><strong>Insulator dysfunction and oncogene activation in IDH mutant gliomas</strong></div><div><em>Nature.</em> 2016; <strong>529</strong>:110-114</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16490" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84953924597" target="_blank" aria-label="Open Scopus in new tab">Scopus (541)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26700815/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16490&amp;pmid=26700815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" aria-label="Reference 2019" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">2019</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Flavahan, W.A. ∙ Drier, Y. ∙ Johnstone, S.E. ...</div><div><strong>Altered chromosomal topology drives oncogenic programs in SDH-deficient GISTs</strong></div><div><em>Nature.</em> 2019; <strong>575</strong>:229-233</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1668-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85074865462" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31666694/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1668-3&amp;pmid=31666694" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Hnisz et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hnisz, D. ∙ Weintraub, A.S. ∙ Day, D.S. ...</div><div><strong>Activation of proto-oncogenes by disruption of chromosome neighborhoods</strong></div><div><em>Science.</em> 2016; <strong>351</strong>:1454-1458</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad9024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84960171833" target="_blank" aria-label="Open Scopus in new tab">Scopus (425)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26940867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad9024&amp;pmid=26940867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Modrek et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Modrek, A.S. ∙ Golub, D. ∙ Khan, T. ...</div><div><strong>Low-Grade Astrocytoma Mutations in IDH1, P53, and ATRX Cooperate to Block Differentiation of Human Neural Stem Cells via Repression of SOX2</strong></div><div><em>Cell Rep.</em> 2017; <strong>21</strong>:1267-1280</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2017.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2017.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85034835338" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29091765/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2017.10.009&amp;pmid=29091765" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistently, we identified more than 100 TAD boundaries that gain DNA methylation and lose CTCF binding in our hypermethylated tumors (<a id="crosref0545" href="#figs3">Figures S3</a>E and S3F). The integrity of these boundaries was compromised, as evidenced by weaker “peaks” on the Hi-C contact maps (<a id="crosref0550" href="#figs3">Figure&nbsp;S3</a>G) and more frequent cross-boundary E-P interactions in HiChIP data (<a id="crosref0555" href="#figs3">Figure&nbsp;S3</a>H). However, the transcriptional consequences of these boundary losses appeared to be relatively limited (<a id="crosref0560" href="#figs3">Figure&nbsp;S3</a>I).</div><div id="p0090" role="paragraph">In summary, topological boundaries were largely conserved across colon tumors, normal colons, and cell lines, with the exception of a relatively small set of boundaries compromised in hypermethylated tumors.</div></section><section id="sec-2-4"><h3>Megabase-Scale Compartment Structure Is Reorganized in Tumors</h3><div id="p0095" role="paragraph">The genome is partitioned into open A and closed B spatial compartments. TADs in the same compartment have a greater tendency to self-interact, whereas inter-compartmental interactions are disfavored (<span class="dropBlock reference-citations"><a id="body-ref-sref72-3" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-3" aria-controls="sref72-3">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref72-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Rao, S.S.P. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), resulting in the characteristic checkerboard pattern of Hi-C maps (<a id="crosref0570" href="#fig1">Figure&nbsp;1</a>F). We used a standard eigenvector-based method to assign compartments for each of our Hi-C datasets. In contrast to the striking conservation of TAD boundaries, compartment assignments varied between samples. For example, most cell lines were distinct from normal tissues and tumors (<a id="crosref0575" href="#fig2">Figures 2</a>A, 2B, <a id="crosref0580" href="#figs4">S4</a>A, and S4B).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/6e07ece1-e78d-47b5-a483-d2041381eff9/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2020.07.030/asset/045cca42-393b-45e3-a01c-62f21ce16cd5/main.assets/gr2.jpg" height="546" width="765" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6e07ece1-e78d-47b5-a483-d2041381eff9/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Compromised Partitioning and Positioning of Genome Compartments in Tumors</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0055" role="paragraph">(A) Hi-C eigenvectors (PC1) based on long-range interactions demarcate compartments A (positive values, blue) and B (negative values, yellow) across a 45-Mb region of chromosome 6. Data show eigenvectors for normal colon (green), colon tumors (purple), and cell lines (black).</div><div id="fspara0060" role="paragraph">(B) Heatmap showing pairwise correlations between the first Hi-C eigenvector (blue heat) in normal colon (green), colon tumors (purple), and cell lines (gray). Samples (rows, columns) are ordered according to complete linkage hierarchical clustering (top).</div><div id="fspara0065" role="paragraph">(C) Heatmap showing fold change (log<sub>2</sub>) in Hi-C contact frequencies between colon tumors and normal colon across chromosome 1. Data are based on an average of normal colons (n&nbsp;= 4) and tumors (n&nbsp;= 7). Interactions that increase in tumors (red) or decrease in tumors (green) are evident. Top left: the Hi-C eigenvector indicates compartment assignments in normal colon (A, blue; B, yellow).</div><div id="fspara0070" role="paragraph">(D) Schematic of the maximum entropy modeling approach, in which structural models of genome organization are generated and iteratively refined to improve the correlation between Hi-C maps derived from these models <i>in silico</i> and the actual experimental Hi-C data.</div><div id="fspara0075" role="paragraph">(E) Whole-nucleus maximum entropy models (1-Mb resolution) for a representative normal colon sample (N1), showing compartment A in blue and compartment B in yellow.</div><div id="fspara0080" role="paragraph">(F) Whole-nucleus maximum entropy models (1-Mb resolution) for a representative colon tumor sample (T1) as in (E).</div><div id="fspara0085" role="paragraph">(G) Representative transmission electron microscopy (TEM) image of nuclei from normal colon epithelium, showing electron dense heterochromatin (HC) along the nuclear membrane and internally distributed euchromatin (EC). Scale bar, 1 um.</div><div id="fspara0090" role="paragraph">(H) Representative EM image of colon tumor nucleus labeled as in panel (G). Scale bar, 1 um.</div><div id="fspara0095" role="paragraph">(I) Top: Schematic of strategy for quantifying the internal HC in nuclei imaged by EM. Bottom: Histogram shows fraction of nuclei (y axis) that have indicated percentage of internal HC. Normal (green) reflects 102 epithelial cell nuclei from 3 normal colon specimens. Tumor (purple) reflects 184 malignant cell nuclei from 3 colon tumors. Two-sided nested t test, p&nbsp;= 0.006.</div><div id="fspara0100" role="paragraph">(J and K) Representative image of chromosome 12 DNA FISH in nuclei from normal colon epithelial cells (J) and malignant tumor cells (K), labeling compartment A and B regions in blue and yellow, respectively.</div><div id="fspara0105" role="paragraph">(L) Histogram showing a fraction of the signal from compartment B DNA FISH probes in successive radial bins from the center to the periphery of the nuclei. Normal (green) reflects 47 epithelial cell nuclei from 2 normal colon specimens. Tumor (purple) reflects 82 malignant cell nuclei from two tumors for which chromosome 12 was copy number stable (T1 and T4). Innermost bins 1–10 were aggregated.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/21e4163f-9059-40fb-90cc-c05c162bd57f/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2020.07.030/asset/d50bc35f-7fea-4aef-b993-dd0a26d3e53b/main.assets/figs4.jpg" height="951" width="772" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/21e4163f-9059-40fb-90cc-c05c162bd57f/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Compartment Reorganization, Related to <a id="crosref1765" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0400" role="paragraph">(A) Hi-C eigenvectors (PC1) based on long-range interactions demarcate compartments A (positive values, blue) and B (negative values, yellow) across a 45 Mb region of chromosome 6. Data show Hi-C eigenvectors for normal colon (green), colon tumors (purple) and cell lines (black). Validation cohort is indicated (vertical bars, left).</div><div id="fspara0405" role="paragraph">(B) Heatmap shows pairwise correlations between the Hi-C eigenvector (blue heat) in normal colon (green), colon tumors (purple) and cell lines (gray). Samples (rows, columns) are ordered according to a complete linkage hierarchical clustering (top). Original cohort is indicated by white squares and validation cohort by black squares.</div><div id="fspara0410" role="paragraph">(C) Heatmap shows fold-change (log<sub>2</sub>) in Hi-C contact frequencies between colon tumors and normal colon across chromosome 1. Data are based on an average of normal colons (n&nbsp;= 4) and tumors (n&nbsp;= 7). Interactions that increase in tumors (red) or decrease in tumors (green) are evident. Top, left: Hi-C eigenvector indicates compartment assignments in colon tumor (A&nbsp;= blue, B&nbsp;= yellow).</div><div id="fspara0415" role="paragraph">(D) Plot shows average ratio of interactions with the A versus B compartments (y axis), summarized for compartment A and B loci for original (left) and validation (right) cohorts. Each point represents the average for 100 kb windows per sample for normal colons (green) or tumors (purple). Statistical significance was computed by a two-sided Wilcoxon rank sum test comparing tumor versus normal absolute A/B ratio values (original cohort: p&nbsp;= 0.004; validation cohort: p&nbsp;= 0.005).</div><div id="fspara0420" role="paragraph">(E) Whole nucleus maximum entropy models (1 Mb resolution) for two normal colon samples showing compartment A in blue and compartment B in yellow.</div><div id="fspara0425" role="paragraph">(F) Density plots depict radial distributions of compartment A and compartment B regions in the maximum entropy models derived for normal colons (green, as in panel (E) or colon tumors (purple, panel (G)(0&nbsp;= interior of the nucleus, 1&nbsp;= periphery of the nucleus). Each line corresponds to a maximum entropy model derived for one experimental Hi-C dataset (2 colon tumors and 2 normal colons).</div><div id="fspara0430" role="paragraph">(G) Whole nucleus maximum entropy models (1 Mb resolution) for two colon tumors showing compartment A in blue and compartment B in yellow.</div><div id="fspara0435" role="paragraph">(H) Maximum entropy models for the whole genome for normal colons and colon tumors, highlighting copy number stable chromosomes 3 (upper) and 4 (lower). Compartment A is colored in blue and compartment B is colored in yellow.</div><div id="fspara0440" role="paragraph">(I) Maximum entropy model for whole genome for a copy-number stable tumor (T6) sample showing compartment A colored in blue and compartment B colored in yellow.</div><div id="fspara0445" role="paragraph">(J) Genomic view of chromosome 12 shows compartment assignment from normal colon (top) and DNA-FISH probe distribution (bottom; black bars).</div><div id="fspara0450" role="paragraph">(K) Representative transmission electron microscopy (EM) images of nuclei from normal colon epithelium (top row) and colon tumors (bottom row).</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">Comparisons also revealed widespread differences between colon tumors and normal colon (<a id="crosref0585" href="#fig2">Figure&nbsp;2</a>B). To understand these differences, we directly compared Hi-C interaction matrices. Although compartment assignments were mostly concordant between normal colon and tumors (<a id="crosref0590" href="#fig2">Figure&nbsp;2</a>A), long-range interactions between compartments A and B were more frequent in tumors (<a id="crosref0595" href="#fig2">Figure&nbsp;2</a>C). These differential interaction patterns were evident regardless of whether the underlying compartment assignments were derived from normal colon or tumor Hi-C data (<a id="crosref0600" href="#figs4">Figure&nbsp;S4</a>C) and translated to a genome-wide increase in inter-compartment interaction in tumors (p&nbsp;&lt; 0.005) (<a id="crosref0605" href="#figs4">Figure&nbsp;S4</a>D).</div><div id="p0105" role="paragraph">Prior studies have related compartments with nuclear positioning (<span class="dropBlock reference-citations"><a id="body-ref-sref23-1" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-1" aria-controls="sref23-1">Falk et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref23-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Falk, M. ∙ Feodorova, Y. ∙ Naumova, N. ...</div><div><strong>Heterochromatin drives compartmentalization of inverted and conventional nuclei</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:395-399</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1275-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85067277322" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1275-3&amp;pmid=31168090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Wang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Wang, S. ∙ Su, J.H. ∙ Beliveau, B.J. ...</div><div><strong>Spatial organization of chromatin domains and compartments in single chromosomes</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:598-602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf8084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84980417141" target="_blank" aria-label="Open Scopus in new tab">Scopus (285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27445307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf8084&amp;pmid=27445307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Compartment B correlates with lamina-associated domains (LADs), which are located at the nuclear periphery and may be damaged in aging-related disease, senescence, and cancer (<span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Sakthivel and Sehgal, 2016</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Sakthivel, K.M. ∙ Sehgal, P.</div><div><strong>A novel role of lamins from genetic disease to cancer biomarkers</strong></div><div><em>Oncol. Rev.</em> 2016; <strong>10</strong>:309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4081/oncol.2016.309" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85002763692" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27994771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4081%2Foncol.2016.309&amp;pmid=27994771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Schreiber and Kennedy, 2013</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Schreiber, K.H. ∙ Kennedy, B.K.</div><div><strong>When lamins go bad: nuclear structure and disease</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:1365-1375</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2013.02.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2013.02.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84875175486" target="_blank" aria-label="Open Scopus in new tab">Scopus (232)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23498943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.02.015&amp;pmid=23498943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">van Steensel and Belmont, 2017</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>van Steensel, B. ∙ Belmont, A.S.</div><div><strong>Lamina-Associated Domains: Links with Chromosome Architecture, Heterochromatin, and Gene Repression</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:780-791</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2017.04.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2017.04.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85019578993" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28525751/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.04.022&amp;pmid=28525751" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conversely, active loci tend to localize to the nuclear interior. In principle, relative nuclear positioning of genomic loci can be inferred from Hi-C data. We therefore used a maximum entropy approach to derive topological models for normal colon and tumor nuclei (<a id="crosref0635" href="#sec-4">STAR Methods</a>). Our method models the genome at 1-Mb resolution as a 3-dimensional polymer, taking into account linear constraints inherent to the DNA. We then compute an <i>in silico</i> Hi-C map for each polymer model and repeat the process iteratively until the computed map converges on the experimental Hi-C data (<a id="crosref0640" href="#fig2">Figure&nbsp;2</a>D; <a id="crosref0645" href="#sec-4">STAR Methods</a>).</div><div id="p0110" role="paragraph">Polymer models optimized to the normal colon Hi-C data separated compartments A and B and positioned compartment B peripherally (<a id="crosref0650" href="#fig2">Figures 2</a>E, <a id="crosref0655" href="#figs4">S4</a>E, and S4F), consistent with expectation. However, applying the same modeling approach to the colon tumor Hi-C data yielded a strikingly different result (<a id="crosref0660" href="#fig2">Figures 2</a>F and <a id="crosref0665" href="#figs4">S4</a>G). In these models, both compartments were distributed heterogeneously throughout the nucleus. We verified that these changes were not solely driven by genetic alterations by restricting analysis to stable chromosomes and a genomically stable tumor (<a id="crosref0670" href="#figs4">Figures S4</a>H and S4I). These results suggest that the asymmetric radial positioning of compartments A and B is profoundly altered in tumors.</div><div id="p0115" role="paragraph">To further investigate, we directly visualized chromatin and specific genomic loci. First, we imaged specimens by transmission electron microscopy, which revealed characteristic epithelial structures and organization. In epithelial nuclei from normal colon, electron-dense heterochromatin was juxtaposed to the nuclear membrane, consistent with peripheral lamina association, whereas characteristic light-staining euchromatin was visible throughout the nuclear interior (<a id="crosref0675" href="#fig2">Figures 2</a>G and <a id="crosref0680" href="#figs4">S4</a>K). In marked contrast, epithelial tumor nuclei had large dark-staining heterochromatin foci dispersed throughout their interiors (<a id="crosref0685" href="#fig2">Figures 2</a>H, 2I, and <a id="crosref0690" href="#figs4">S4</a>K; <a id="crosref0695" href="#sec-4">STAR Methods</a>).</div><div id="p0120" role="paragraph">We next evaluated the positioning of specific genomic regions using DNA fluorescence <i>in situ</i> hybridization (FISH). We designed ∼26,000 Oligopaint probes targeting loci on chromosome 12 that were assigned to compartment A or B according to Hi-C (<a id="crosref0700" href="#figs4">Figure&nbsp;S4</a>J; <a id="crosref0705" href="#sec-4">STAR Methods</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Beliveau et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Beliveau, B.J. ∙ Joyce, E.F. ∙ Apostolopoulos, N. ...</div><div><strong>Versatile design and synthesis platform for visualizing genomes with Oligopaint FISH probes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:21301-21306</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1213818110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84871822966" target="_blank" aria-label="Open Scopus in new tab">Scopus (172)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23236188/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1213818110&amp;pmid=23236188" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We then labeled the respective compartments with secondary fluorescent probes and visualized them by Airyscan confocal imaging. Chromosome 12 territories were generally positioned peripherally, consistent with prior studies (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Bolzer et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bolzer, A. ∙ Kreth, G. ∙ Solovei, I. ...</div><div><strong>Three-dimensional maps of all chromosomes in human male fibroblast nuclei and prometaphase rosettes</strong></div><div><em>PLoS Biol.</em> 2005; <strong>3</strong>:e157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0030157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-21844444077" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15839726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0030157&amp;pmid=15839726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To quantify radial distributions of the A and B compartments, we scored probe signals according to their intensity in 20 radial bins starting from the nuclear center in 47 nuclei from 2 normal colon samples and 82 nuclei from 2 tumors. Although the localization of fluorescence signals varied, compartment B signals were strongly skewed toward the periphery of normal colon nuclei, whereas compartment A signals were more evenly distributed (<a id="crosref0720" href="#fig2">Figures 2</a>J and 2L). In contrast, in tumor nuclei, compartment B signals lost their peripheral skew and assumed a distribution similar to compartment A (<a id="crosref0725" href="#fig2">Figures 2</a>K and 2L).</div><div id="p0125" role="paragraph">Hence, concordant analyses based on Hi-C polymer models, electron microscopy, and multi-color FISH imaging indicate profound compartmental reorganization in tumor nuclei. Spatial partitioning between compartments is compromised. Compartment B relocates from its physiologic peripheral position toward the nuclear interior.</div></section><section id="sec-2-5"><h3>A Genome Compartment with Intermediate Properties</h3><div id="p0130" role="paragraph">Despite widespread differences in compartmental interactions, A/B assignments were relatively consistent between tumor and normal colon. This prompted us to more closely examine the Hi-C eigenvector, which is a continuous rather than a dichotomous measure. We observed hundreds of large genomic intervals, hundreds of kilobases in size, with eigenvector values that were lower in tumors than in normal colon (<a id="crosref0730" href="#fig3">Figures 3</a>A and 3B). The majority of these intervals were assigned to compartment A in tumor and normal colon because they had positive eigenvectors.&nbsp;However, quantitative analysis indicated that these regions shifted their interactions toward compartment B in tumors.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/51d2a13b-5735-4b38-8afd-6b48d073461a/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2020.07.030/asset/f80ca9a0-13a3-45fb-94dc-4e3fb5605aff/main.assets/gr3.jpg" height="757" width="744" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/51d2a13b-5735-4b38-8afd-6b48d073461a/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">An Intermediate Compartment I Is Also Reorganized in Tumors</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0115" role="paragraph">(A) Heatmap depicting the Hi-C eigenvector used to define compartments, as in <a id="crosref1700" href="#fig2">Figure&nbsp;2</a>A, except with the eigenvector values shown as heatmaps (blue, positive; yellow, negative). Data are shown for a ∼2.3-Mb region (x axis) for normal colon samples (rows with green labels) and colon tumors (rows with purple labels). Hypomethylated blocks are indicated (black bars).</div><div id="fspara0120" role="paragraph">(B) Heatmap depicting the first eigenvector for a 5-Mb region on chromosome 3 as in (A).</div><div id="fspara0125" role="paragraph">(C) Hi-C contact map (top) and first eigenvector (PC1; center) for a representative tumor sample. DNA methylation levels of low-density (open-sea) CpGs (bottom) are shown for three representative normal colon samples (green) and three representative tumors (purple). The represented region on chromosome 17 contains a 3.6-Mb TAD assigned to compartment B (black square) with a hypomethylated block (gray highlight).</div><div id="fspara0130" role="paragraph">(D) Hi-C contact map, first eigenvector (PC1) and DNA methylation are shown for a 200 kb TAD assigned to compartment A with a hypomethylated block (data presented as in panel (C).</div><div id="fspara0135" role="paragraph">(E) Hi-C contact map for chromosome 1 for a representative normal colon sample. Top left: colored bars indicate genomic regions assigned to compartment A (blue), compartment I (light blue), or compartment B (yellow). Magnified panels for representative regions highlight the intermediate long-range inter-compartmental interaction pattern typical of compartment I.</div><div id="fspara0140" role="paragraph">(F) First and second eigenvectors of the chromosome 1 Hi-C matrix for normal colon. Each point represents one 100-kb bin, colored by compartment.</div><div id="fspara0145" role="paragraph">(G) Whole-nucleus maximum entropy model (100-kb resolution) for normal colon, showing compartments A, I, and B.</div><div id="fspara0150" role="paragraph">(H) Representative DNA FISH image (left) and high-magnification image (right) for HCT116 cell nuclei. Signal intensities are shown for compartment A (blue), I (light blue), and B (yellow) regions on chromosome 12.</div><div id="fspara0155" role="paragraph">(I) Barplot indicating the percentage of cells for which the maximum DNA FISH signal intensity for compartment A, B, or I is located at the indicated radial position for 305 HCT116 cell nuclei.</div><div id="fspara0160" role="paragraph">(J) Representative image of nuclei from normal colon epithelial cells. The image shows DNA FISH signal intensities of probes for compartments A, B, and I of chromosome 12. Two chromosome territories are magnified in the insets.</div></div></div></div></figcaption></figure></div><div id="p0135" role="paragraph">When we examined the transitioning regions, we observed that they exhibited a striking loss of DNA methylation in tumors and, in fact, largely coincided with hypomethylated blocks (<a id="crosref0735" href="#fig3">Figures 3</a>A, 3B, and <a id="crosref0740" href="#figs5">S5</a>A). This was unexpected because hypomethylated blocks have been primarily associated only with compartment B (<span class="dropBlock reference-citations"><a id="body-ref-sref3-3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-3" aria-controls="sref3-3">Berman et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref3-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Berman, B.P. ∙ Weisenberger, D.J. ∙ Aman, J.F. ...</div><div><strong>Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains</strong></div><div><em>Nat. Genet.</em> 2011; <strong>44</strong>:40-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.969" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84655162785" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22120008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.969&amp;pmid=22120008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Fortin and Hansen, 2015</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Fortin, J.-P. ∙ Hansen, K.D.</div><div><strong>Reconstructing A/B compartments as revealed by Hi-C using long-range correlations in epigenetic data</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:180</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0741-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84940090291" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26316348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0741-y&amp;pmid=26316348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-3" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-3" aria-controls="sref32-3">Hansen et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref32-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hansen, K.D. ∙ Timp, W. ∙ Bravo, H.C. ...</div><div><strong>Increased methylation variation in epigenetic domains across cancer types</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:768-775</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.865" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79960927422" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21706001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.865&amp;pmid=21706001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that hypomethylated blocks, which tend to span single or consecutive TADs, covered a full 19% of compartment A (<a id="crosref0760" href="#fig3">Figures 3</a>C, 3D, <a id="crosref0765" href="#figs5">S5</a>B, and S5C; mean methylation difference, &gt;10%; n&nbsp;= 1,032; mean size. 217 kb). Examination of Hi-C contact maps revealed that these noncanonical regions do not fit the typical checkerboard pattern that arises from long-range compartmental interactions (<a id="crosref0770" href="#fig3">Figure&nbsp;3</a>E). Rather, they have a distinct contact pattern characterized by intermediate interactions with both conventional compartments (<a id="crosref0775" href="#fig3">Figures 3</a>E) and preferential self-interactions (<a id="crosref0780" href="#figs5">Figure&nbsp;S5</a>D). This distinct contact&nbsp;pattern was evident in normal (<a id="crosref0785" href="#figs5">Figures S5</a>E and S5F) and tumor samples (<a id="crosref0790" href="#figs5">Figures S5</a>G and S5H).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/3509c33f-f6ba-4c7c-a3da-772579207225/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2020.07.030/asset/2a0a27db-75b7-447b-92ad-5c4202c5449d/main.assets/figs5.jpg" height="892" width="759" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/3509c33f-f6ba-4c7c-a3da-772579207225/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Intermediate Compartment I, Related to <a id="crosref1770" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0460" role="paragraph">(A) Density plot shows Hi-C eigenvector difference between tumor and normal for 100 kb windows with PC1&nbsp;&gt; 0 in normal colon tissue. Regions are stratified based on degree of tumor hypomethylation.</div><div id="fspara0465" role="paragraph">(B) Barplot showing the fraction of the genome (y axis) assigned to each compartment (x axis). Black bar represents the fraction of the genome that is block hypomethylated in tumors with respect to normal.</div><div id="fspara0470" role="paragraph">(C) Aggregated contact map shows Hi-C signal averaged over all hypomethylated blocks across normal and tumor samples. The x axis shows genomic positions relative to hypomethylated blocks. The edges of hypomethylated blocks correspond to TAD boundaries.</div><div id="fspara0475" role="paragraph">(D) Plots show average frequency of Hi-C contacts for pairwise interactions that occur within the same genomic compartment (left), and between different compartments (right). Data are shown for four normal colon samples (dots). Compartment I regions have inter-compartment interactions with both A and B regions.</div><div id="fspara0480" role="paragraph">(E-F) Hi-C contact map of observed versus expected interactions in normal colon for two representative regions across chromosomes 6 (E) and 14 (F). Compartment designations are shown for both rows and columns.</div><div id="fspara0485" role="paragraph">(G-H) Hi-C contact map of observed versus expected interactions in colon tumors for two representative regions across chromosomes 6 (G) and 14 (H). Compartment designations are shown for both rows and columns.</div><div id="fspara0490" role="paragraph">(I) Plot shows average ratio of interactions with the A versus B compartments (y axis), summarized for compartment I. Each point represents the average of 100 kb windows for normal colons (green) and tumors (purple). Shown for original (left) and validation (right) cohorts.</div><div id="fspara0495" role="paragraph">(J-L) Scatterplots of first and second (J and L) or first and third (K) eigenvectors for chromosomes 12 (I), 13 (J) and 20 (K) resulting from the eigenvector decomposition method to define compartments. Data are shown for the aggregated normal colon Hi-C matrices. Each point represents one 100 kb bin and is colored by compartment (A: dark blue; I: light blue; B: yellow).</div><div id="fspara0500" role="paragraph">(M) Boxplot shows the distribution of PC1 values resulting from the eigenvector decomposition of the HCT116 Hi-C matrix. Data are shown for the 100Kb-bins that overlap with our DNA-FISH probes. Probes for the respective compartments have the expected distributions of PC1 values.</div><div id="fspara0505" role="paragraph">(N) Barplot indicates the percent of cells for which the maximum DNA-FISH signal intensity for compartments A, B or I is located at the indicated radial position for 102 normal colon nuclei.</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">We considered that these hypomethylated A blocks might reflect an intermediate compartment “I” that interacts with both canonical compartments at baseline and shifts toward B in tumors (<a id="crosref0795" href="#figs5">Figure&nbsp;S5</a>I). In support, we found that compartment I regions could be distinguished on multiple chromosomes by examining additional eigenvectors of the Hi-C matrix (<a id="crosref0800" href="#fig3">Figures 3</a>F and <a id="crosref0805" href="#figs5">S5</a>J–S5L; <a id="crosref0810" href="#sec-4">STAR Methods</a>). Although the order of the declarative eigenvector varied between chromosomes, this suggested that compartment I can be distinguished from structural data alone. Furthermore, our polymer models for normal colon placed compartment I in an intermediate nuclear position between compartments A and B (<a id="crosref0815" href="#fig3">Figure&nbsp;3</a>G).</div><div id="p0145" role="paragraph">To investigate further, we visualized compartment I regions by multi-color FISH. We designed a third set of ∼14,500 oligonucleotide probes complementary to compartment I regions on chromosome 12 and a corresponding secondary probe with a distinct fluorophore (<a id="crosref0820" href="#figs4">Figure&nbsp;S4</a>J). We then used three-color FISH imaging to simultaneously localize compartment A, B, and I regions in HCT116 colon cancer cells. We selected HCT116 cells because chromosome 12 is copy number stable, and the loci targeted by our probes had similar compartment assignments as our primary tissues (<a id="crosref0825" href="#mmc2">Table S2</a>; <a id="crosref0830" href="#figs5">Figure&nbsp;S5</a>M). We visualized and quantified the radial positioning of each compartment in 305 HCT116 nuclei. We found that compartment I is spatially intermediate between the more peripheral compartment B and the more internal compartment A (<a id="crosref0835" href="#fig3">Figures 3</a>H and 3I). We confirmed this observation in primary tissues by quantifying fluorescence FISH signals in normal colon epithelial cell nuclei (<a id="crosref0840" href="#fig3">Figures 3</a>J and <a id="crosref0845" href="#figs5">S5</a>N).</div><div id="p0150" role="paragraph">Thus, a convergence of Hi-C, methylation, polymer modeling, and imaging data support the existence of a third genomic compartment I that interacts with both conventional compartments and adopts an intermediate spatial position in the nucleus. In tumors, compartment I becomes broadly hypomethylated and shifts its interactions toward the B compartment.</div></section><section id="sec-2-6"><h3>Distinct Chromatin and Transcriptional States Support the Three-Compartment Model</h3><div id="p0155" role="paragraph">To investigate whether compartment I is associated with distinct histone modifications, we mapped markers of active regulatory elements (H3K27ac), elongating transcripts (histone H3 lysine 36 trimethylation/H3K36me3), constitutive heterochromatin (histone H3 lysine 9 trimethylation/H3K9me3), and facultative (histone H3 lysine 27 trimethylation/H3K27me3) heterochromatin. As expected, H3K27ac and H3K36me3 were enriched in compartment A (<a id="crosref0850" href="#fig4">Figures 4</a>A, 4C, and 4D), whereas H3K9me3 was enriched in compartment B and relatively increased in tumors (<a id="crosref0855" href="#fig4">Figures 4</a>B–4E).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/29a6890b-13bb-4e0a-a192-c4e0bf52e315/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2020.07.030/asset/4a95a970-e2d6-48f4-bbf2-07326d39dd87/main.assets/gr4.jpg" height="464" width="672" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/29a6890b-13bb-4e0a-a192-c4e0bf52e315/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Distinct Chromatin States Support a Three-Compartment Model</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0170" role="paragraph">(A–C) Plots show 3-compartment model assignments for representative regions on chromosome 20 (A), chromosome 6 (B), and chromosome 4 (C). Dark blue, compartment A; light blue, compartment I; yellow, compartment B. DNA methylation levels of low-density (open-sea) CpGs for three normal samples (green) and three tumor samples (purple) are shown below, along with ChIP-seq profiles for H3K27ac, H3K27me3, and H3K9me3 for a representative normal colon sample (green) and tumor (purple). Hypomethylated blocks are shaded in gray.</div><div id="fspara0175" role="paragraph">(D) Heatmap showing relative levels of H3K27ac, H3K36me3, H3K27me3, and H3K9me3 in normal colon (rows) for compartments A, I, and B (columns). DNA methylation differences between normal colon and tumors (for open-sea CpGs) and relative gene expression in normal colon (TCGA) are also shown.</div><div id="fspara0180" role="paragraph">(E) Plots showing mean and standard deviation of fold change (log<sub>2</sub>) in enrichment of the indicated modification between tumors and normal samples. The respective plots show data for different modifications and are stratified by compartment (x axes).</div></div></div></div></figcaption></figure></div><div id="p0160" role="paragraph">Compartment I was clearly distinguished from both conventional compartments by broad H3K27me3 enrichment (<a id="crosref0860" href="#fig4">Figures 4</a>A, 4C–4E, and <a id="crosref0865" href="#figs6">S6</a>A). H3K27me3 signal intensity was particularly pronounced in tumors and correlated with the degree of DNA hypomethylation. H3K36me3, which antagonizes H3K27me3 and has been proposed to protect against DNA hypomethylation, is depleted in compartment I (<a id="crosref0870" href="#fig4">Figure&nbsp;4</a>D; <span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Yuan et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Yuan, W. ∙ Xu, M. ∙ Huang, C. ...</div><div><strong>H3K36 methylation antagonizes PRC2-mediated H3K27 methylation</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2011; <strong>286</strong>:7983-7989</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1074%2Fjbc.M110.194027&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1074%2Fjbc.M110.194027&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79953143753" target="_blank" aria-label="Open Scopus in new tab">Scopus (293)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21239496/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M110.194027&amp;pmid=21239496" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105-1" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-1" aria-controls="sref105-1">Zhou et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref105-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Zhou, W. ∙ Dinh, H.Q. ∙ Ramjan, Z. ...</div><div><strong>DNA methylation loss in late-replicating domains is linked to mitotic cell division</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:591-602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0073-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85044740324" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29610480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0073-4&amp;pmid=29610480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Compartment I was also notable for relatively low transcriptional levels in normal colon and moderate gene density, both of which were in between compartments A and B (<a id="crosref0885" href="#fig4">Figures 4</a>D and <a id="crosref0890" href="#figs6">S6</a>B).</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/bd83a27c-4a57-4384-94c7-1d6e1d604b76/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2020.07.030/asset/44999e54-071b-4aa6-898b-08467cee6fad/main.assets/figs6.jpg" height="704" width="762" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/bd83a27c-4a57-4384-94c7-1d6e1d604b76/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Related to <a id="crosref1775" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0515" role="paragraph">(A) Boxplots show H3K27me3 fold-change (y axis) between tumors and normal colon as a function of DNA hypomethylation in tumors (x axis).</div><div id="fspara0520" role="paragraph">(B) Histogram shows the distribution of gene density, measured as number of promoters per 100Kb, for each genomic compartment.</div><div id="fspara0525" role="paragraph">(C) For each 100Kb genomic bin, change in DNA methylation upon 5-aza treatment (y axis) is plotted against baseline (DMSO) methylation level. Regions showing high levels of methylation in the control (DMSO) sample showed the greatest loss of methylation upon treatment with 5-aza.</div><div id="fspara0530" role="paragraph">(D) Plots show average DNA methylation for a cohort of normal (N) colon samples and low- (L-A) and high-grade adenomas (H-A) (<span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Fan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Fan, J. ∙ Li, J. ∙ Guo, S. ...</div><div><strong>Genome-wide DNA methylation profiles of low- and high-grade adenoma reveals potential biomarkers for early detection of colorectal carcinoma</strong></div><div><em>Clin. Epigenetics.</em> 2020; <strong>12</strong>:56</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13148-020-00851-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85083871857" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32317010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13148-020-00851-3&amp;pmid=32317010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Points represent individual samples. Data are stratified by compartment.</div><div id="fspara0535" role="paragraph">(E) Boxplot depicts fold-change (log<sub>2</sub>) in expression for genes in compartments A, I or B between tumor and normal colon using TCGA gene expression data. Genes in compartments I and B are downregulated in tumors.</div><div id="fspara0540" role="paragraph">(F) Boxplot depicts fold-change (log<sub>2</sub>) in expression for genes in compartments A, I or B between tumor and normal colon. This panel is equivalent to (E), but the values reflect RNA-seq data from our cohort rather than TCGA.</div><div id="fspara0545" role="paragraph">(G) Boxplot representation of gene expression fold-changes in cancer initiating cells treated with EZH2 inhibitor, relative to control (<span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Lima-Fernandes et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Lima-Fernandes, E. ∙ Murison, A. ∙ da Silva Medina, T. ...</div><div><strong>Targeting bivalency de-represses Indian Hedgehog and inhibits self-renewal of colorectal cancer-initiating cells</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:1436</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09309-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85063726627" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30926792/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09309-4&amp;pmid=30926792" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Data are shown for genes in compartment I, known EZH2 targets in compartment A and expressed genes in compartment A that are not EZH2 targets.</div><div id="fspara0550" role="paragraph">(H) Density plot shows the distribution of methylation differences in CpG islands in tumors, relative to normal colons (x axis). Compartment I is depicted in light blue and compartment B is depicted in yellow. Although both compartments are globally hypomethylated in open sea regions, a relatively larger subset of CpG islands gains methylation in compartment B.</div></div></div></div></figcaption></figure></div><div id="p0165" role="paragraph">Thus, in addition to its topological features, compartment I is distinguished by its facultative heterochromatin state, modest transcriptional output, and methylation changes in tumors.</div></section><section id="sec-2-7"><h3>Compartmental Changes Linked to DNA Hypomethylation and Accumulated Cell Divisions</h3><div id="p0170" role="paragraph">Compartments B and I largely correspond to hypomethylated blocks in tumors. Moreover, we observed a striking correlation between the extent of hypomethylation of a given region and its eigenvector: genomic loci with more extreme hypomethylation became relatively more B-like or compact (<a id="crosref0895" href="#fig5">Figure&nbsp;5</a>A).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/4185ab61-a197-4590-abb9-1231cb901558/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2020.07.030/asset/21fb751d-a142-4952-b31f-25c27859e716/main.assets/gr5.jpg" height="465" width="500" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/4185ab61-a197-4590-abb9-1231cb901558/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Compartmental Reorganization Is Closely Associated with DNA Hypomethylation and Accumulated Divisions</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0190" role="paragraph">(A) Association between Hi-C eigenvector (PC1) and hypomethylation of 100-kb windows in tumor samples. Data are stratified by compartment and extent of hypomethylation. Points and horizontal bars represent point estimates and 95% confidence intervals from a linear regression model.</div><div id="fspara0195" role="paragraph">(B) Data visualized as in (A), showing the association between Hi-C eigenvector and hypomethylation for HCT116 cells treated with 5′-aza versus DMSO.</div><div id="fspara0200" role="paragraph">(C) Plot showing DNA methylation for WI-38 cells at passages 16 (black), 30 (dark gray), and 40 (light gray). Data are stratified by compartment. Bars represent the average of two replicates (dots).</div><div id="fspara0205" role="paragraph">(D) Plot showing DNA methylation (for open-sea CpGs) for a representative region on chromosome 10. Traces represent methylation values for WI-38 fibroblasts at passages 16 (black, n&nbsp;= 2), 30 (dark gray, n&nbsp;= 2), and 40 (light gray, n&nbsp;= 2). Compartment assignments for WI-38 are shown at the top (A, blue; I, light blue; B, yellow).</div><div id="fspara0210" role="paragraph">(E) Data visualized as in (A), showing the association between Hi-C eigenvector and hypomethylation for late (passage 40 [P40]) versus early (P16) WI-38 cells.</div><div id="fspara0215" role="paragraph">(F) Plot showing change over time in Hi-C eigenvector for 100-kb windows that show more than 20% hypomethylation in late-passage (P40) versus early-passage (P16) WI-38 fibroblasts.</div></div></div></div></figcaption></figure></div><div id="p0175" role="paragraph">To further assess this relationship between compartmental changes and DNA hypomethylation, we treated HCT116 cells with the demethylating agent 5′-azacytidine (5-aza) for 24&nbsp;h and measured methylation and topology changes by Hi-C. The treatment reduced methylation of large genomic intervals, with 56% of 100-kb windows losing more than 20% methylation (<a id="crosref0900" href="#figs6">Figure&nbsp;S6</a>C). Notably, we found that genomic regions with the most significant methylation loss shifted their interactions toward compartment B in a topological reorganization reminiscent of tumors (<a id="crosref0905" href="#fig5">Figure&nbsp;5</a>B). This suggested that block hypomethylation may underlie the altered compartment structure in colon tumors.</div><div id="p0180" role="paragraph">Block hypomethylation was originally described in cancer cells (<span class="dropBlock reference-citations"><a id="body-ref-sref3-4" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-4" aria-controls="sref3-4">Berman et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref3-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Berman, B.P. ∙ Weisenberger, D.J. ∙ Aman, J.F. ...</div><div><strong>Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains</strong></div><div><em>Nat. Genet.</em> 2011; <strong>44</strong>:40-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.969" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84655162785" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22120008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.969&amp;pmid=22120008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-4" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-4" aria-controls="sref32-4">Hansen et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref32-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hansen, K.D. ∙ Timp, W. ∙ Bravo, H.C. ...</div><div><strong>Increased methylation variation in epigenetic domains across cancer types</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:768-775</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.865" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79960927422" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21706001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.865&amp;pmid=21706001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Nordor et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Nordor, A.V. ∙ Nehar-Belaid, D. ∙ Richon, S. ...</div><div><strong>The early pregnancy placenta foreshadows DNA methylation alterations of solid tumors</strong></div><div><em>Epigenetics.</em> 2017; <strong>12</strong>:793-803</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/15592294.2017.1342912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85034848501" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F15592294.2017.1342912&amp;pmid=28678605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) but has since been recognized to be a feature of cells that have accumulated many divisions, including aging and senescing cells (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Cruickshanks et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cruickshanks, H.A. ∙ McBryan, T. ∙ Nelson, D.M. ...</div><div><strong>Senescent cells harbour features of the cancer epigenome</strong></div><div><em>Nat. Cell Biol.</em> 2013; <strong>15</strong>:1495-1506</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb2879" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84887945557" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24270890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb2879&amp;pmid=24270890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105-2" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-2" aria-controls="sref105-2">Zhou et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref105-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Zhou, W. ∙ Dinh, H.Q. ∙ Ramjan, Z. ...</div><div><strong>DNA methylation loss in late-replicating domains is linked to mitotic cell division</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:591-602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0073-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85044740324" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29610480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0073-4&amp;pmid=29610480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genome topology and nuclear structure are also altered in fibroblasts passaged to replicative senescence (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Criscione et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Criscione, S.W. ∙ De Cecco, M. ∙ Siranosian, B. ...</div><div><strong>Reorganization of chromosome architecture in replicative cellular senescence</strong></div><div><em>Sci. Adv.</em> 2016; <strong>2</strong>:e1500882</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1500882" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84964957467" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26989773/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1500882&amp;pmid=26989773" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79-1" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-1" aria-controls="sref79-1">Sati et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref79-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Sati, S. ∙ Bonev, B. ∙ Szabo, Q. ...</div><div><strong>4D Genome Rewiring during Oncogene-Induced and Replicative Senescence</strong></div><div><em>Mol. Cell.</em> 2020; <strong>78</strong>:522-538.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.molcel.2020.03.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.molcel.2020.03.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85083005663" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32220303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2020.03.007&amp;pmid=32220303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We hypothesized that the compartmental shifts in tumor cells might relate to those in passaged fibroblasts, in both cases reflecting excessive replications. To test this, we passaged WI-38 fibroblasts over a 14-week course and generated DNA methylation profiles and Hi-C data for cells harvested at passages 16, 30, and 40 (<a id="crosref0945" href="#sec-4">STAR Methods</a>). Late-passage cells continued to replicate and had not yet progressed to replicative senescence. Comparison of early and late passage data confirmed progressive hypomethylation of compartments B and I with passage (<a id="crosref0950" href="#fig5">Figures 5</a>C and 5D). Moreover, late-passage hypomethylation was accompanied by topological changes analogous to tumors: the most hypomethylated regions had reduced eigenvector values, suggestive of compaction (<a id="crosref0955" href="#fig5">Figure&nbsp;5</a>E). Importantly, intermediate-passage fibroblasts exhibited intermediate degrees of hypomethylation and structural reorganization (<a id="crosref0960" href="#fig5">Figures 5</a>C and 5F).</div><div id="p0185" role="paragraph">The indication that compartmental hypomethylation and topological shifts arise gradually as cells accumulate divisions prompted us to examine methylation in colonic adenomas. These pre-malignant lesions are entirely submitted for diagnostic evaluation, precluding assessment of topology, but their DNA methylation can be profiled from paraffin sections. Assessment of a published cohort of adenomas confirmed that compartmental hypomethylation was evident in these pre-malignant lesions and more severe in higher-grade cases (<a id="crosref0965" href="#figs6">Figure&nbsp;S6</a>D; <span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Fan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Fan, J. ∙ Li, J. ∙ Guo, S. ...</div><div><strong>Genome-wide DNA methylation profiles of low- and high-grade adenoma reveals potential biomarkers for early detection of colorectal carcinoma</strong></div><div><em>Clin. Epigenetics.</em> 2020; <strong>12</strong>:56</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13148-020-00851-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85083871857" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32317010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13148-020-00851-3&amp;pmid=32317010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Taken together, these results suggest that compartmental shifts in colorectal tumors closely relate to DNA hypomethylation and may arise gradually over the course of proliferation. Thus, the most profound topological changes in tumors likely reflect their accumulated cell divisions rather than specific oncogenic programs.</div></section><section id="sec-2-8"><h3>Transcriptional Consequences of Compartmental Reorganization</h3><div id="p0190" role="paragraph">We next considered the transcriptional consequences of compartment B/I reorganization. We were struck that the overall transcriptional activity in these compartments was actually reduced in tumors, despite loss of DNA methylation and relocation of compartment B from the nuclear periphery. Considering genes in compartments B and I with detectable expression (transcripts per million/TPM &gt; 0.1), we found that 3-fold more were downregulated than upregulated (<a id="crosref0975" href="#figs6">Figures S6</a>E and S6F). In contrast, compartment A genes did not exhibit such an imbalance.</div><div id="p0195" role="paragraph">We reasoned that gene silencing in the reorganized compartments might be sustained (or enhanced) by their repressive chromatin states. Indeed, compartment I gained widespread H3K27me3 in tumors (<a id="crosref0980" href="#fig4">Figure&nbsp;4</a>E), and analysis of published data showed that many compartment I genes were upregulated in colon cancer-initiating cells treated with the Ezh2 inhibitor (<a id="crosref0985" href="#figs6">Figure&nbsp;S6</a>G; <span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Lima-Fernandes et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Lima-Fernandes, E. ∙ Murison, A. ∙ da Silva Medina, T. ...</div><div><strong>Targeting bivalency de-represses Indian Hedgehog and inhibits self-renewal of colorectal cancer-initiating cells</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:1436</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09309-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85063726627" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30926792/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09309-4&amp;pmid=30926792" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Compartment B is broadly covered by the repressive H3K9me3 mark, and a subset of its promoters is silenced by focal hypermethylation within the hypomethylated blocks (<a id="crosref0995" href="#fig4">Figures 4</a>D and <a id="crosref1000" href="#figs6">S6</a>H). Thus, alternate epigenetic mechanisms may actually further repress compartments B and I upon reorganization.</div><div id="p0200" role="paragraph">We therefore sought to identify specific genes that were deregulated by this process. Because reorganization correlated quantitatively with hypomethylation (<a id="crosref1005" href="#fig5">Figure&nbsp;5</a>A), we reasoned that methylation could be a surrogate for compartmental changes. We collated methylation and RNA-seq data for 239 colorectal tumors (TCGA) and used a correlation metric to identify genes in compartments B and I whose expression was consistently altered in association with block hypomethylation (<a id="crosref1010" href="#fig6">Figures 6</a>A, 6B, and <a id="crosref1015" href="#figs7">S7</a>A; <a id="crosref1020" href="#sec-4">STAR Methods</a>). Notably, genes in compartment A showed no consistent transcriptional change in association with hypomethylation (<a id="crosref1025" href="#fig6">Figure&nbsp;6</a>C).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/aa59690b-14d3-4666-b2d7-6b1bdcbb3ccc/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2020.07.030/asset/090d1720-ea18-450b-b6ee-1f1fea459a10/main.assets/gr6.jpg" height="742" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/aa59690b-14d3-4666-b2d7-6b1bdcbb3ccc/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Compartmental Reorganization Linked to Tumor-Suppressive Transcriptional Programs</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0225" role="paragraph">(A) Volcano plot depicting the association (x axis) between expression and block hypomethylation for genes in compartment B, computed across tumors in the TCGA colorectal cohort with a purity of more than 40%. The y axis indicates the significance of the association. Genes (points) plotted toward the top right are upregulated in association with block hypomethylation. They are highly enriched for cancer germline antigens (CGAs; red) and ERV elements (green). Genes plotted toward the top left are downregulated in association with hypomethylation. A magnified panel (below) highlights high-confidence downregulated genes after excluding genes expressed in non-epithelial cell types (black; <a id="crosref1705" href="#sec-4">STAR Methods</a>). Genes related to EMT, Wnt signaling, invasion, and metastasis are labeled.</div><div id="fspara0230" role="paragraph">(B) Data presented as in (A) for compartment I genes.</div><div id="fspara0235" role="paragraph">(C) Boxplot showing association between expression change and DNA block hypomethylation for genes in compartments A, B, and I. Negative values indicate repression in association with DNA hypomethylation, and positive values indicate upregulation in association with hypomethylation.</div><div id="fspara0240" role="paragraph">(D) Functional gene set annotations enriched (false discovery rate [FDR]&nbsp;&lt; 20%) among 146 high-confidence downregulated B/I genes (from the insets in A and B). <sup>∗</sup>, enriched annotations included multiple overlapping sets related to embryonic development (<a id="crosref1710" href="#mmc6">Table S6</a>).</div><div id="fspara0245" role="paragraph">(E) Plot showing average block methylation levels for normal colon biopsies (y axis) as a function of donor age (x axis). Each point represents one sample from a low-risk (green) or high-risk (magenta) donor (<span class="dropBlock reference-citations"><a id="body-ref-sref96-1" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-1" aria-controls="sref96-1">Wang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref96-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Wang, T. ∙ Maden, S.K. ∙ Luebeck, G.E. ...</div><div><strong>Dysfunctional epigenetic aging of the normal colon and colorectal cancer risk</strong></div><div><em>Clin. Epigenetics.</em> 2020; <strong>12</strong>:5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13148-019-0801-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85077479949" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31900199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13148-019-0801-3&amp;pmid=31900199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Linear regression fit (lines) and 95% confidence intervals (shades) are indicated.</div><div id="fspara0250" role="paragraph">(F) Plots showing average expression of the high-confidence downregulated B/I genes (from the insets in A and B) in clinical specimens. Each point corresponds to a different sample from a cohort of colorectal tumors and normal colons (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Cancer Genome Atlas Network, 2012</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cancer Genome Atlas Network</div><div><strong>Comprehensive molecular characterization of human colon and rectal cancer</strong></div><div><em>Nature.</em> 2012; <strong>487</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84863922124" target="_blank" aria-label="Open Scopus in new tab">Scopus (4694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22810696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11252&amp;pmid=22810696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The y axis represents log<sub>2</sub>-normalized counts.</div><div id="fspara0255" role="paragraph">(G) Kaplan-Meier curve depicting survival outcomes of patients stratified by their average tumor expression of the high-confidence downregulated compartment B/I genes.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/1cdcb24a-5a93-4721-9f89-dc706d743bf5/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2020.07.030/asset/3cb379c7-1441-4046-beee-05c4bf0168fc/main.assets/figs7.jpg" height="951" width="712" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/1cdcb24a-5a93-4721-9f89-dc706d743bf5/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Related to <a id="crosref1790" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0560" role="paragraph">(A) Volcano plot depicts the association between expression and block hypomethylation for all genes in compartments A, I and B. A positive association on the x axis indicates upregulation with hypomethylation and a negative association indicates downregulation with hypomethylation. This panel is equivalent to <a id="crosref1795" href="#fig6">Figures 6</a>A and 6B, but the values were calculated using RNA-seq data from our cohort rather than TCGA data.</div><div id="fspara0565" role="paragraph">(B) Heatmap shows H3K27me3 and H3K9me3 ChIP-seq signal in tumors (first two columns), and the extent of DNA hypomethylation in tumors relative to normal colon (third column). Each row of the heatmap represents a gene. Three groups of genes are shown: Genes in compartment B that are downregulated with hypomethylation, genes in compartment B that are upregulated in with hypomethylation (including CGA genes and ERV repeats), and genes in compartment I that are downregulated with hypomethylation.</div><div id="fspara0570" role="paragraph">(C) Boxplot shows open sea methylation levels for compartments B and I (mean per individual) for a cohort of normal colon samples stratified by age (x axis) (<span class="dropBlock reference-citations"><a id="body-ref-sref96-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-2" aria-controls="sref96-2">Wang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref96-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Wang, T. ∙ Maden, S.K. ∙ Luebeck, G.E. ...</div><div><strong>Dysfunctional epigenetic aging of the normal colon and colorectal cancer risk</strong></div><div><em>Clin. Epigenetics.</em> 2020; <strong>12</strong>:5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13148-019-0801-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85077479949" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31900199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13148-019-0801-3&amp;pmid=31900199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0575" role="paragraph">(D) Volcano plot shows log<sub>2</sub> hazard ratio from a Cox proportional hazards model of 7,694 variable genes with expression sd &gt; 0.5 from previously published cohort (<span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Marisa et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The set of genes in compartments B and I that are downregulated with hypomethylation are shown in red. A positive log hazard ratio indicates higher expression is associated with increased risk of recurrence or death.</div><div id="fspara0580" role="paragraph">(E) Coefficient estimates and 95% confidence intervals for association with recurrence free survival from a Cox Proportional Hazards model. The model includes the 146 gene high risk indicator and the other indicated variables (Data from <span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Marisa et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0585" role="paragraph">(F) Coefficient estimates and 95% confidence intervals for association with recurrence free survival from a Cox Proportional Hazards model fit only to samples from patients with Stage II disease (<span class="dropBlock reference-citations"><a id="body-ref-sref58-3" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-3" aria-controls="sref58-3">Marisa et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref58-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0590" role="paragraph">(G) Boxplots show associations between gene expression and block hypomethylation for 10 epithelial tumor types. Positive values indicate upregulation with hypomethylation. Separate boxes show data for genes outside (‘out’) or inside (‘in’) a hypomethylated block. Data shown for 10 TCGA cohorts: bladder urothelial carcinoma (BLCA), breast invasive carcinoma (BRCA), colon adenocarcinoma (COAD), esophageal squamous cell carcinoma (ESCA), head and neck squamous cell carcinoma (HNSC), lung adenocarcinoma (LUAD), lung squamous cell carcinoma (LUSC), rectal adenocarcinoma (READ), stomach adenocarcinoma (STAD), uterine corpus endometrial cancer (UCEC) (The <span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium, 2020</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium</div><div><strong>Pan-cancer analysis of whole genomes</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:82-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-1969-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85079038817" target="_blank" aria-label="Open Scopus in new tab">Scopus (313)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32025007/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-1969-6&amp;pmid=32025007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div></div></figcaption></figure></div><div id="p0205" role="paragraph">Correlation analysis highlighted two gene sets (<a id="crosref1030" href="#fig6">Figures 6</a>A and 6B). A small set of genes was upregulated with block hypomethylation and included cancer germline antigens (CGAs) and endogenous retroviruses (ERVs). De-repression of CGAs and ERVs has been described in colon tumors and linked to viral mimicry and immunogenicity (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Gibbs and Whitehurst, 2018</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gibbs, Z.A. ∙ Whitehurst, A.W.</div><div><strong>Emerging Contributions of Cancer/Testis Antigens to Neoplastic Behaviors</strong></div><div><em>Trends Cancer.</em> 2018; <strong>4</strong>:701-712</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.trecan.2018.08.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.trecan.2018.08.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85053719310" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30292353/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.trecan.2018.08.005&amp;pmid=30292353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Rooney et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Rooney, M.S. ∙ Shukla, S.A. ∙ Wu, C.J. ...</div><div><strong>Molecular and genetic properties of tumors associated with local immune cytolytic activity</strong></div><div><em>Cell.</em> 2015; <strong>160</strong>:48-61</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.12.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.12.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84920962202" target="_blank" aria-label="Open Scopus in new tab">Scopus (1231)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25594174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.12.033&amp;pmid=25594174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Roulois et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Roulois, D. ∙ Loo Yau, H. ∙ Singhania, R. ...</div><div><strong>DNA-Demethylating Agents Target Colorectal Cancer Cells by Inducing Viral Mimicry by Endogenous Transcripts</strong></div><div><em>Cell.</em> 2015; <strong>162</strong>:961-973</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2015.07.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2015.07.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84940403834" target="_blank" aria-label="Open Scopus in new tab">Scopus (514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26317465/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.07.056&amp;pmid=26317465" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0210" role="paragraph">A much larger set of genes in compartments B and I were downregulated with block hypomethylation (<a id="crosref1050" href="#fig6">Figures 6</a>A and 6B). Downregulated genes in compartment B were marked by H3K9me3 and/or promoter methylation in tumors, whereas those in compartment I were enriched for H3K27me3 (<a id="crosref1055" href="#figs7">Figure&nbsp;S7</a>B). We curated a list of robustly downregulated genes in these compartments (<a id="crosref1060" href="#mmc5">Table S5</a>). To focus on malignant cell-intrinsic expression, we controlled for stromal content and excluded genes with high expression in immune cells or other non-epithelial cell types (<a id="crosref1065" href="#sec-4">STAR Methods</a>). Remarkably, the resulting list of 146 genes was highly enriched for functions related to mesenchymal development, stem cell proliferation, and Wnt signaling (<a id="crosref1070" href="#fig6">Figure&nbsp;6</a>D). Further analysis highlighted specific genes with established roles in colorectal cancer progression, Wnt signaling, epithelial-mesenchymal transition (EMT), invasion, and metastasis (e.g., <i>CCBE1</i>, <i>EPHA4</i>, <i>FGFR1</i>, <i>FGFR2</i>, <i>FZD2</i>, <i>GPR137B</i>, <i>MEIS2</i>, <i>NFIB</i>, <i>PRRX1</i>, <i>PYGO1</i>, <i>SPP1</i>, and <i>TIAM1</i>; <a id="crosref1075" href="#mmc6">Table S6</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Koveitypour et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Koveitypour, Z. ∙ Panahi, F. ∙ Vakilian, M. ...</div><div><strong>Signaling pathways involved in colorectal cancer progression</strong></div><div><em>Cell Biosci.</em> 2019; <strong>9</strong>:97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13578-019-0361-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85076212909" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31827763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13578-019-0361-4&amp;pmid=31827763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Nguyen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Nguyen, L.H. ∙ Goel, A. ∙ Chung, D.C.</div><div><strong>Pathways of Colorectal Carcinogenesis</strong></div><div><em>Gastroenterology.</em> 2020; <strong>158</strong>:291-302</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1053%2Fj.gastro.2019.08.059&amp;cf=fulltext&amp;site=ygast-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1053%2Fj.gastro.2019.08.059&amp;cf=pdf&amp;site=ygast-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31622622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1053%2Fj.gastro.2019.08.059&amp;pmid=31622622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In contrast, only a few of the downregulated genes were nominally associated with tumor-suppressive functions.</div><div id="p0215" role="paragraph">Our analyses suggest that compartmental reorganization drives induction of CGAs and ERVs, which are associated with anti-tumor immunity, and repression of genes with functions in Wnt signaling, EMT, invasion, and metastasis. Hence, the most profound topological alterations evident in tumors are actually associated with tumor-suppressive transcriptional programs.</div></section><section id="sec-2-9"><h3>Compartment-Specific Epigenetic Changes Restrain Tumor Progression</h3><div id="p0220" role="paragraph">Our collective findings suggested that accumulation of excess cell divisions leads to compartmental shifts that enact tumor-suppressive transcriptional programs. They led us to hypothesize that compartmental reorganization hinders malignant progression. To test this, we examined whether the compartmental shifts were predictive of disease risk and outcome.</div><div id="p0225" role="paragraph">First, we considered a recent survey of DNA methylation in 206 normal colon biopsies, stratified into low- and high-risk groups according to whether the donor had a concurrent colorectal tumor elsewhere in the colon (<span class="dropBlock reference-citations"><a id="body-ref-sref96-3" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-3" aria-controls="sref96-3">Wang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref96-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Wang, T. ∙ Maden, S.K. ∙ Luebeck, G.E. ...</div><div><strong>Dysfunctional epigenetic aging of the normal colon and colorectal cancer risk</strong></div><div><em>Clin. Epigenetics.</em> 2020; <strong>12</strong>:5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13148-019-0801-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85077479949" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31900199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13148-019-0801-3&amp;pmid=31900199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Examination of these data confirmed that compartments B and I became progressively hypomethylated with increasing donor age (<a id="crosref1095" href="#figs7">Figure&nbsp;S7</a>C). To test whether hypomethylation was protective, we compared low- and high-risk groups. We found that compartments B and I were significantly less hypomethylated in normal colon biopsies from the high-risk group, consistent with our hypothesis that the compartment shift is tumor suppressive (<a id="crosref1100" href="#fig6">Figure&nbsp;6</a>E).</div><div id="p0230" role="paragraph">Second, we examined two clinical cohorts of colorectal tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref58-4" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-4" aria-controls="sref58-4">Marisa et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref58-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; The <span class="dropBlock reference-citations"><a id="body-ref-sref8-3" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-3" aria-controls="sref8-3">Cancer Genome Atlas Network, 2012</a><div class="dropBlock__holder" data-db-target-of="sref8-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cancer Genome Atlas Network</div><div><strong>Comprehensive molecular characterization of human colon and rectal cancer</strong></div><div><em>Nature.</em> 2012; <strong>487</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84863922124" target="_blank" aria-label="Open Scopus in new tab">Scopus (4694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22810696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11252&amp;pmid=22810696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although these tumors have presumably overcome impediments posed by compartment shifts, we reasoned that the associated transcriptional changes should nonetheless hinder their progression and correlate with favorable patient outcomes. Here we focused on the 146 genes in compartments B and I that were robustly downregulated with block hypomethylation (<a id="crosref1115" href="#mmc5">Table S5</a>). As expected, these genes were expressed at substantially lower levels in tumors (<a id="crosref1120" href="#fig6">Figure&nbsp;6</a>F). However, there was considerable tumor-to-tumor variability. Remarkably, we found that this set of genes was highly enriched for poor prognosis markers in a cohort of 566 colon tumors (<a id="crosref1125" href="#figs7">Figure&nbsp;S7</a>D; p&nbsp;= 2.4&nbsp;× 10<sup>−10</sup>; <span class="dropBlock reference-citations"><a id="body-ref-sref58-5" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-5" aria-controls="sref58-5">Marisa et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref58-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A risk score constructed from their average expression was a strong predictor of shorter recurrence-free survival (RFS) (<a id="crosref1135" href="#fig6">Figure&nbsp;6</a>G; p&nbsp;= 0.0007). This prognostic association was also validated in a second cohort of 443 tumors from TCGA (p&nbsp;= 0.03).</div><div id="p0235" role="paragraph">Survival difference was evident even after controlling for microsatellite instability (MSI), BRAF mutations, and clinical stage (p&nbsp;= 0.025) (<a id="crosref1140" href="#figs7">Figure&nbsp;S7</a>E). It was evident even in node-negative stage II tumors (p&nbsp;= 0.039), which is significant, given the clinical challenge associated with the uncertain course of these intermediate-stage tumors (<a id="crosref1145" href="#figs7">Figure&nbsp;S7</a>F) (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Fotheringham et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Fotheringham, S. ∙ Mozolowski, G.A. ∙ Murray, E.M.A. ...</div><div><strong>Challenges and solutions in patient treatment strategies for stage II colon cancer</strong></div><div><em>Gastroenterol. Rep. (Oxf.).</em> 2019; <strong>7</strong>:151-161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/gastro/goz006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85071096116" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31217978/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgastro%2Fgoz006&amp;pmid=31217978" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The gene set was also associated with metastases (p&nbsp;= 0.018), consistent with its functional annotations and supportive of its clinical significance.</div><div id="p0240" role="paragraph">Finally, we considered whether compartmental reorganization could be a general tumor-suppressive mechanism. We examined methylation and expression data for cohorts spanning 10 epithelial tumor types (<span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium, 2020</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium</div><div><strong>Pan-cancer analysis of whole genomes</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:82-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-1969-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85079038817" target="_blank" aria-label="Open Scopus in new tab">Scopus (313)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32025007/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-1969-6&amp;pmid=32025007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We confirmed that block hypomethylation correlated with reduced gene expression in all 10 cancers (<a id="crosref1160" href="#figs7">Figure&nbsp;S7</a>G). We next collated genes in blocks that were significantly downregulated in association with hypomethylation in each cohort. The resulting sets were highly overlapping and include a shared set of 367 genes that were commonly downregulated with hypomethylation in at least 7 of the 10 tumor types (<a id="crosref1165" href="#mmc7">Table S7</a>). Remarkably, these shared genes were enriched for annotated oncogenes (p&nbsp;= 0.002), suggesting that compartmental reorganization may also hinder the development and progression of other epithelial tumor types.</div><div id="p0245" role="paragraph">In conclusion, we document profound compartmental shifts in tumors and other cells that have accumulated many divisions (<a id="crosref1170" href="#fig7">Figures 7</a>A and 7B). This reorganization is associated with widespread transcriptional changes, including repression of EMT, invasion, metastasis, and stemness programs. Further analysis of methylation and expression data for normal colon biopsies and tumor cohorts supports the hypothesis that the compartmental shifts and associated transcriptional programs restrain tumor progression.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.07.030/asset/a1260b7a-a782-4304-83d2-85044e04b112/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2020.07.030/asset/7966fd35-72ed-4b4d-9f80-ac4755bb004b/main.assets/gr7.jpg" height="263" width="607" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/a1260b7a-a782-4304-83d2-85044e04b112/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Compartment Shifts in Excessively Replicated Cells Restrain Malignant Progression</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0265" role="paragraph">The schematic depicts compartment shifts and proposed functional consequences.</div><div id="fspara0270" role="paragraph">(A) In normal nuclei, compartments A, B, and I are robustly partitioned and spatially segregated. In tumor or aging cells that have accumulated excess divisions, compartmental organization is compromised, and compartment-specific epigenetic states are altered.</div><div id="fspara0275" role="paragraph">(B) Exemplar loci are shown for each compartment in normal (top) or pathologic (bottom) states. Hypomethylation of compartment B induces ERVs and CGAs, which promote anti-tumor immunity. Repressive chromatin in compartments B and I downregulates genes associated with EMT, invasion, and stemness.</div></div></div></div></figcaption></figure></div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0250" role="paragraph">We presented a systematic integration of genome topology, methylation, and chromatin state in colorectal cancer. Our data and analyses parse multiple organizational layers, from E-P loops to TADs to compartment structures. In particular, they revealed three principles of large-scale compartmental organization. First, topology data for primary tissues uncovered a structurally distinct intermediate compartment I. Second, comparison of tumors and normal colon revealed widespread changes in spatial partitioning, nuclear positioning, and epigenetic states of compartments B and I that appear to be shared by tumor, aging, and other excessively replicated cells. Third, these compartmental changes correlate with and may promote tumor-suppressive expression programs associated with reduced cancer risk and better prognosis. Although tumor-associated epigenetic changes are typically construed to be oncogenic, our findings suggest that these most profound topological alterations actually restrain malignant progression.</div><div id="p0255" role="paragraph">Evidence of compartment I emerged from our analysis of primary epithelial tissue. Compartment I resides at the interface between the A and B compartments and engages in promiscuous long-range interactions with both conventional compartments. Polymer models and FISH imaging data indicate that compartment I regions occupy intermediate radial positions in nuclei. Compartment I is also distinguished epigenetically by broad H3K27me3 and robust block hypomethylation in tumors. Compartment I is distinct from previously described sub-compartments, showing the highest overlap (47%) with B1 (<span class="dropBlock reference-citations"><a id="body-ref-sref72-4" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-4" aria-controls="sref72-4">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref72-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Rao, S.S.P. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It may relate to nuclear foci documented in high-resolution imaging studies (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Boettiger et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Boettiger, A.N. ∙ Bintu, B. ∙ Moffitt, J.R. ...</div><div><strong>Super-resolution imaging reveals distinct chromatin folding for different epigenetic states</strong></div><div><em>Nature.</em> 2016; <strong>529</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16496" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84955290042" target="_blank" aria-label="Open Scopus in new tab">Scopus (455)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26760202/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16496&amp;pmid=26760202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Rowley and Corces, 2018</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Rowley, M.J. ∙ Corces, V.G.</div><div><strong>Organizational principles of 3D genome architecture</strong></div><div><em>Nat. Rev. Genet.</em> 2018; <strong>19</strong>:789-800</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-018-0060-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85055726389" target="_blank" aria-label="Open Scopus in new tab">Scopus (237)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30367165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-018-0060-8&amp;pmid=30367165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Xu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Xu, J. ∙ Ma, H. ∙ Jin, J. ...</div><div><strong>Super-Resolution Imaging of Higher-Order Chromatin Structures at Different Epigenomic States in Single Mammalian Cells</strong></div><div><em>Cell Rep.</em> 2018; <strong>24</strong>:873-882</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2018.06.085&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2018.06.085&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85049905839" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30044984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.06.085&amp;pmid=30044984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, <span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Boettiger et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Boettiger, A.N. ∙ Bintu, B. ∙ Moffitt, J.R. ...</div><div><strong>Super-resolution imaging reveals distinct chromatin folding for different epigenetic states</strong></div><div><em>Nature.</em> 2016; <strong>529</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16496" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84955290042" target="_blank" aria-label="Open Scopus in new tab">Scopus (455)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26760202/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16496&amp;pmid=26760202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> visualized a compartment enriched for Polycomb-associated marks and developmental genes in <i>Drosophila</i>.</div><div id="p0260" role="paragraph">The coherent organization of compartments A, B, and I in normal colon was profoundly distorted in tumors. We observed a breakdown of partitioning between compartments A and B, whereas compartment I shifted its interactions toward the closed B compartment. The aberrations appear to be closely related to nuclear architecture. Concordant polymer models, electron microscopy, and FISH imaging data indicate that compartment B loses its tight association with the periphery and shifts toward the nuclear interior. <span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">Falk et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Falk, M. ∙ Feodorova, Y. ∙ Naumova, N. ...</div><div><strong>Heterochromatin drives compartmentalization of inverted and conventional nuclei</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:395-399</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1275-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85067277322" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1275-3&amp;pmid=31168090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> found previously that the radial asymmetry of compartments A and B is inverted in rod photoreceptors. However, compartmental partitioning was largely maintained in photoreceptors, in contrast to the overall disorganization of compartmental structure in tumors.</div><div id="p0265" role="paragraph">The breakdown of compartment structure was closely tied to pervasive changes to methylation and chromatin state. Compartments B and I acquired near-uniform hypomethylation in tumors and became further enriched for their characteristic chromatin states, H3K9me3 and H3K27me3, a pattern that has also been described for hypomethylated loci in breast cancer cell lines (<span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Hon et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Hon, G.C. ∙ Hawkins, R.D. ∙ Caballero, O.L. ...</div><div><strong>Global DNA hypomethylation coupled to repressive chromatin domain formation and gene silencing in breast cancer</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:246-258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.125872.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84856528270" target="_blank" aria-label="Open Scopus in new tab">Scopus (337)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22156296/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.125872.111&amp;pmid=22156296" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The reorganized compartments might relate to phase-separated condensates and nuclear foci thought to play wide-ranging roles in gene and genome regulation (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Larson et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Larson, A.G. ∙ Elnatan, D. ∙ Keenen, M.M. ...</div><div><strong>Liquid droplet formation by HP1α suggests a role for phase separation in heterochromatin</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:236-240</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85024372369" target="_blank" aria-label="Open Scopus in new tab">Scopus (520)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28636604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22822&amp;pmid=28636604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Strom et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Strom, A.R. ∙ Emelyanov, A.V. ∙ Mir, M. ...</div><div><strong>Phase separation drives heterochromatin domain formation</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:241-245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22989" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85024405877" target="_blank" aria-label="Open Scopus in new tab">Scopus (559)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28636597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22989&amp;pmid=28636597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Notably, high-resolution analyses have identified senescence-associated heterochromatin foci with a central density of H3K9me3-marked heterochromatin surrounded by a ring of H3K27me3 (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Chandra and Narita, 2013</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Chandra, T. ∙ Narita, M.</div><div><strong>High-order chromatin structure and the epigenome in SAHFs</strong></div><div><em>Nucleus.</em> 2013; <strong>4</strong>:23-28</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/nucl.23189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84873555534" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23232545/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Fnucl.23189&amp;pmid=23232545" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79-2" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-2" aria-controls="sref79-2">Sati et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref79-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Sati, S. ∙ Bonev, B. ∙ Szabo, Q. ...</div><div><strong>4D Genome Rewiring during Oncogene-Induced and Replicative Senescence</strong></div><div><em>Mol. Cell.</em> 2020; <strong>78</strong>:522-538.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.molcel.2020.03.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.molcel.2020.03.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85083005663" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32220303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2020.03.007&amp;pmid=32220303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), features consistent with our three-compartment model in tumors.</div><div id="p0270" role="paragraph">Compartmental reorganization appears to be tightly linked to DNA hypomethylation and proliferative history. Hypomethylated blocks in tumors correspond to compartments B and I, with the most severely hypomethylated loci undergoing more negative eigenvector shifts, indicative of compaction. Methylation loss may be causal because demethylating agents directly induce topological changes. Although block hypomethylation was initially described in cancer, it is increasingly recognized to be a common feature of cells that have accumulated excess divisions, including aging and senescing cells (<span class="dropBlock reference-citations"><a id="body-ref-sref3-5" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-5" aria-controls="sref3-5">Berman et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref3-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Berman, B.P. ∙ Weisenberger, D.J. ∙ Aman, J.F. ...</div><div><strong>Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains</strong></div><div><em>Nat. Genet.</em> 2011; <strong>44</strong>:40-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.969" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84655162785" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22120008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.969&amp;pmid=22120008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Cruickshanks et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cruickshanks, H.A. ∙ McBryan, T. ∙ Nelson, D.M. ...</div><div><strong>Senescent cells harbour features of the cancer epigenome</strong></div><div><em>Nat. Cell Biol.</em> 2013; <strong>15</strong>:1495-1506</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb2879" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84887945557" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24270890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb2879&amp;pmid=24270890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Nordor et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Nordor, A.V. ∙ Nehar-Belaid, D. ∙ Richon, S. ...</div><div><strong>The early pregnancy placenta foreshadows DNA methylation alterations of solid tumors</strong></div><div><em>Epigenetics.</em> 2017; <strong>12</strong>:793-803</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/15592294.2017.1342912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85034848501" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F15592294.2017.1342912&amp;pmid=28678605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Timp et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Timp, W. ∙ Bravo, H.C. ∙ McDonald, O.G. ...</div><div><strong>Large hypomethylated blocks as a universal defining epigenetic alteration in human solid tumors</strong></div><div><em>Genome Med.</em> 2014; <strong>6</strong>:61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-014-0061-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84906837231" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25191524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-014-0061-y&amp;pmid=25191524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Indeed, compartments B and I become progressively hypomethylated and structurally reorganized in passaged fibroblasts. Moreover, colon adenomas exhibit intermediate compartmental hypomethylation, consistent with a replicative history between normal colon and tumors. Although future studies are needed to examine the topology of these and other pre-malignant lesions, our findings suggest that compartmental shifts are not a consequence of malignancy but, rather, arise progressively as cells accumulate divisions.</div><div id="p0275" role="paragraph">We therefore propose that the compartmental reorganization reflects a fundamental epigenetic process primed by excess cell divisions, a perspective that enabled us to interpret attendant transcriptional changes. Compartmental hypomethylation was associated with repression of compartment B and I genes, likely as a consequence of repressive epigenetic states that arise in the hypomethylated compartments. Repressed genes were enriched for oncogenic functions related to EMT, invasion, and Wnt signaling, leading us to speculate that the topological shifts present a barrier to tumorigenesis. Indeed, prior studies have shown that cultured cells undergoing EMT remodel large heterochromatin domains (<span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">McDonald et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>McDonald, O.G. ∙ Wu, H. ∙ Timp, W. ...</div><div><strong>Genome-scale epigenetic reprogramming during epithelial-to-mesenchymal transition</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2011; <strong>18</strong>:867-874</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.2084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79961023271" target="_blank" aria-label="Open Scopus in new tab">Scopus (249)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21725293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.2084&amp;pmid=21725293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although most compartment B and I genes were downregulated, CGAs and ERVs with pro-immunity functions were induced and could complement restriction of stemness and invasion programs to restrain malignant progression in aging colonic epithelium or pre-malignant lesions.</div><div id="p0280" role="paragraph">Final support for a proposed tumor-suppressive role emerged from our analysis of clinical cohorts. First, methylation profiles for normal colon biopsies revealed that age-associated compartment B and I hypomethylation was associated with reduced colorectal cancer risk (<span class="dropBlock reference-citations"><a id="body-ref-sref96-4" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-4" aria-controls="sref96-4">Wang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref96-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Wang, T. ∙ Maden, S.K. ∙ Luebeck, G.E. ...</div><div><strong>Dysfunctional epigenetic aging of the normal colon and colorectal cancer risk</strong></div><div><em>Clin. Epigenetics.</em> 2020; <strong>12</strong>:5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13148-019-0801-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85077479949" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31900199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13148-019-0801-3&amp;pmid=31900199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), consistent with our model and with a recent report relating morphological changes in uninvolved colonic nuclei to tumor risk (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Gladstein et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Gladstein, S. ∙ Damania, D. ∙ Almassalha, L.M. ...</div><div><strong>Correlating colorectal cancer risk with field carcinogenesis progression using partial wave spectroscopic microscopy</strong></div><div><em>Cancer Med.</em> 2018; <strong>7</strong>:2109-2120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cam4.1357" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85044297841" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29573208/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcam4.1357&amp;pmid=29573208" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Second, examination of colorectal tumor cohorts revealed that a transcriptional signature of compartmental shift was predictive of patient outcome and likelihood of metastasis. Finally, a pan-cancer analysis suggested that the compartmental shifts and tumor-suppressive effects may be generalizable to other epithelial cancers. Future studies of these pervasive architectural changes and their functional significance in cancer and aging could inform new strategies for early detection, patient stratification, and therapeutic intervention.</div></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key Resources Table</h3><div id="p0285" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>CTCF</td><td>Cell signaling</td><td>Cat# 3418; RRID: <a id="interref80" href="https://antibodyregistry.org/AB_2086791" target="_blank">AB_2086791</a></td></tr><tr><td>SMC1</td><td>Bethyl</td><td>Cat# A300-055A; RRID <a id="interref85" href="https://antibodyregistry.org/AB_2192467" target="_blank">AB_2192467</a></td></tr><tr><td>H3K27ac</td><td>Active Motif</td><td>Cat# 39133; RRID <a id="interref90" href="https://antibodyregistry.org/AB_2561016" target="_blank">AB_2561016</a></td></tr><tr><td>H3K9me3</td><td>Abcam</td><td>Cat# 8898; RRID <a id="interref95" href="https://antibodyregistry.org/AB_306848" target="_blank">AB_306848</a></td></tr><tr><td>H3K27me3</td><td>Cell Signaling</td><td>Cat# 9733; RRID <a id="interref100" href="https://antibodyregistry.org/AB_2616029" target="_blank">AB_2616029</a></td></tr><tr class="rowsep"><td>H3K36me3</td><td>Abcam</td><td>Cat# 9050; RRID <a id="interref105" href="https://antibodyregistry.org/AB_306966" target="_blank">AB_306966</a></td></tr><tr class="rowsep"><td colspan="3"><b>Biological Samples</b></td></tr><tr class="rowsep"><td>See <a id="crosref1265" href="#mmc1">Table S1</a> for a list of patient samples included in the study.</td><td>&nbsp;</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>Azacitidine (5-azacitidine)</td><td>Selleckchem</td><td>S1782</td></tr><tr><td>Triethanolamine</td><td>Sigma</td><td>90279</td></tr><tr><td>Acetic Anhydride</td><td>Sigma</td><td>320102</td></tr><tr><td>Formamide</td><td>Sigma</td><td>47671</td></tr><tr><td>CX1723 Supelco Citric Acid, Anhydrous</td><td>Sigma</td><td>CX1723</td></tr><tr><td>Thermo ScientificRNase A, DNase and protease-free (10&nbsp;mg/mL)</td><td>ThermoScientific</td><td>EN0531</td></tr><tr><td>Dextran sulfate sodium salt from Leuconostoc spp.</td><td>Sigma</td><td>D8906</td></tr><tr><td>Invitrogen DAPI (4’,6-Diamidino-2-Phenylindole, Dihydrochloride)</td><td>ThermoFisher</td><td>D1306</td></tr><tr><td>SlowFade Gold Antifade Mountant</td><td>Invitrogen</td><td>S36936</td></tr><tr><td>Tissue-Tek<sup>∗</sup> O.C.T. Compound</td><td>VWR/Sakura</td><td>25608-930</td></tr><tr><td>Oligo Pool, 160mers</td><td>Twist Biosciences</td><td>&nbsp;</td></tr><tr><td>Taq polymerase</td><td>Thermo Fisher</td><td>18038042</td></tr><tr class="rowsep"><td>Maxima H Minus Reverse Transcriptase (200&nbsp;U/μL)</td><td>Thermo Fisher</td><td>EP0751</td></tr><tr class="rowsep"><td colspan="3"><b>Critical Commercial Assays</b></td></tr><tr><td>Nextera DNA Library Prep Kit (for HiChIP)</td><td>Illumina</td><td>20018704</td></tr><tr><td>NEXTFLEX bisulfite library prep kit</td><td>Perkin Elmer</td><td>NOVA-5119-01</td></tr><tr><td>HiScribe T7 High Yield RNA Synthesis Kit</td><td>NEB</td><td>E2040S</td></tr><tr class="rowsep"><td>Kapa HiFi Hotstart PCR Kit</td><td>Roche</td><td>#KK2502</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr><td>Imaging data</td><td>This paper</td><td>Mendeley data: <a id="interref0110" href="https://dx.doi.org/10.17632/6k4hjfw76.1" target="_blank">https://dx.doi.org/10.17632/6k4hjfw76.1</a></td></tr><tr class="rowsep"><td>Raw and processed sequencing data</td><td>This paper</td><td>GEO: <a id="intref0015" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE133928" target="_blank">GSE133928</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Cell Lines</b></td></tr><tr><td>HCT116</td><td>ATCC</td><td>CCL-247</td></tr><tr><td>SW480</td><td>ATCC</td><td>CCL-228</td></tr><tr><td>LS 174T</td><td>ATCC</td><td>CL-188</td></tr><tr><td>RKO</td><td>ATCC</td><td>CRL-2577</td></tr><tr><td>FHC</td><td>ATCC</td><td>CRL-1831</td></tr><tr class="rowsep"><td>WI38</td><td>Coriell</td><td>AG06814-N</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>Chr 12 Probe F- for DNA FISH probe preparation</td><td>IDT</td><td>CGGTCCCGTCCGAGGTATAC</td></tr><tr><td>Chr 12 Probe R- for DNA FISH probe preparation</td><td>IDT</td><td>TCCAATACGCACCGATCGAG</td></tr><tr><td>chr12_A_all (Secondary 1 Binding Site) F</td><td>IDT</td><td>CACCGACGTCGCATAGAACGGAAGAGCGTGTG<br>GACAGCCGGTTCGGTCGTTC</td></tr><tr><td>chr12_A_all (Secondary 1 Binding Site) R</td><td>IDT</td><td>TAATACGACTCACTATAGGGCGGTCCCGTCCG<br>AGGTATAC</td></tr><tr><td>chr12_B_all (Secondary 5 Binding Site) F</td><td>IDT</td><td>TAGCGCAGGAGGTCCACGACGTGCAAGGGTG<br>TTCGTTCACCGCGCGTTGAAG</td></tr><tr><td>chr12_B_all (Secondary 5 Binding Site) R</td><td>IDT</td><td>TAATACGACTCACTATAGGGCGGTCCCGTCCG<br>AGGTATAC</td></tr><tr><td>chr12_I_all (Secondary 6 Binding Site) F</td><td>IDT</td><td>CACACGCTCTCCGTCTTGGCCGTGGTCGATCA<br>GCGATCTGCGCATGGTAATC</td></tr><tr><td>chr12_I_all (Secondary 6 Binding Site) R</td><td>IDT</td><td>TAATACGACTCACTATAGGGCGGTCCCGTCCG<br>AGGTATAC</td></tr><tr><td>Secondary 1- Alexa488</td><td>IDT</td><td>ACACACGCTCTTCCGTTCTATGCGACGTCGGTGA</td></tr><tr><td>Secondary 1- Alexa647</td><td>IDT</td><td>ACACACGCTCTTCCGTTCTATGCGACGTCGGTGA</td></tr><tr><td>Secondary 5- Atto565</td><td>IDT</td><td>ACACCCTTGCACGTCGTGGACCTCCTGCGCTA</td></tr><tr class="rowsep"><td>Secondary 6- Alexa647</td><td>IDT</td><td>TGATCGACCACGGCCAAGACGGAGAGCGTGTG</td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>Juicebox</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Durand et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Durand, N.C. ∙ Robinson, J.T. ∙ Shamim, M.S. ...</div><div><strong>Juicebox Provides a Visualization System for Hi-C Contact Maps with Unlimited Zoom</strong></div><div><em>Cell Syst.</em> 2016; <strong>3</strong>:99-101</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cels.2015.07.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cels.2015.07.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27467250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.07.012&amp;pmid=27467250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0020" href="http://aidenlab.org/juicebox/" target="_blank">http://aidenlab.org/juicebox/</a></td></tr><tr><td>Cell Profiler</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">McQuin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>McQuin, C. ∙ Goodman, A. ∙ Chernyshev, V. ...</div><div><strong>CellProfiler 3.0: Next-generation image processing for biology</strong></div><div><em>PLoS Biol.</em> 2018; <strong>16</strong>:e2005970</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.2005970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85051293036" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29969450/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.2005970&amp;pmid=29969450" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>version 3.1.9</td></tr><tr><td>FIJI</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (19328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>version 2.0.0-rc-69/1.52p</td></tr><tr><td>HiC-Pro</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref84-1" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-1" aria-controls="sref84-1">Servant et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref84-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Servant, N. ∙ Varoquaux, N. ∙ Lajoie, B.R. ...</div><div><strong>HiC-Pro: an optimized and flexible pipeline for Hi-C data processing</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0831-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84952861819" target="_blank" aria-label="Open Scopus in new tab">Scopus (450)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0831-x&amp;pmid=26619908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>version 2.10.0</td></tr><tr><td>Bioconductor</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Huber et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Huber, W. ∙ Carey, V.J. ∙ Gentleman, R. ...</div><div><strong>Orchestrating high-throughput genomic analysis with Bioconductor</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:115-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84961289551" target="_blank" aria-label="Open Scopus in new tab">Scopus (1290)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25633503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3252&amp;pmid=25633503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>release 3.11</td></tr><tr><td>Code supporting this study</td><td>This paper</td><td><a id="intref0025" href="https://github.com/aryeelab/colon-dna-topology" target="_blank">https://github.com/aryeelab/colon-dna-topology</a></td></tr><tr><td>OligoMiner scripts</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Beliveau et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Beliveau, B.J. ∙ Joyce, E.F. ∙ Apostolopoulos, N. ...</div><div><strong>Versatile design and synthesis platform for visualizing genomes with Oligopaint FISH probes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:21301-21306</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1213818110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84871822966" target="_blank" aria-label="Open Scopus in new tab">Scopus (172)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23236188/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1213818110&amp;pmid=23236188" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0030" href="https://github.com/beliveau-lab/OligoMiner" target="_blank">https://github.com/beliveau-lab/OligoMiner</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2930938-7" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource Availability</h3><section id="sec-4-2-1"><h4>Lead Contact</h4><div id="p0290" role="paragraph">Further information and requests for resources and reagents should be directed to the Lead Contact, Bradley Bernstein (<a id="intref0035" href="mailto:Bernstein.Bradley@mgh.harvard.edu">Bernstein.Bradley@mgh.harvard.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials Availability</h4><div id="p0295" role="paragraph">No unique reagents were generated for this study.</div></section><section id="sec-4-2-3"><h4>Data and Code Availability</h4><div id="p0300" role="paragraph">All next generation sequencing data generated in the study were deposited at&nbsp;dbGaP&nbsp;and&nbsp;the Gene Expression Omnibus (GEO): <a id="interref75" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE133928" target="_blank">GSE133928</a>. Original data including all raw microscopy images were deposited at Mendeley Data: <a id="PC_linkZJ77fpAisa" href="https://dx.doi.org/10.17632/6k4hjkfw76.1" target="_blank">https://dx.doi.org/10.17632/6k4hjkfw76.1</a>. Code supporting the study is deposited at Github: <a id="PC_link4ESderB3Cq" href="https://github.com/aryeelab/colon-dna-topology/" target="_blank">https://github.com/aryeelab/colon-dna-topology/</a>.</div></section></section><section id="sec-4-3"><h3>Experimental Model and Subject Details</h3><section id="sec-4-3-1"><h4>Human tumor specimens</h4><div id="p0310" role="paragraph">Tumors included in this cohort are either collected as part of a Massachusetts General Hospital (MGH) Pathology discarded tissue tumor banking protocol or on a consented protocol. All samples were acquired with Institutional Review Board approval (2012P001475/PHS). For the discarded tissue cohort, the tissue was collected from colonic adenocarcinomas at the time of surgical resection at MGH prior to 2015 and saved as part of a de-identified tissue bank. As a de-identified cohort, no clinical data is available. A second set of tumors came from patients at the MGH who were consented preoperatively to take part in the study. For a subset of these patients, normal colon was taken from adjacent normal areas in resection specimens from consented patients. Both normal and tumor tissue from the consented cohort was snap frozen. A summary of the tissue (normal and tumor) samples is provided in <a id="crosref1300" href="#mmc1">Table S1</a>. Tumor genotyping was based on the SNaPshot assay, performed at Massachusetts General Hospital (<span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Dias-Santagata et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Dias-Santagata, D. ∙ Akhavanfard, S. ∙ David, S.S. ...</div><div><strong>Rapid targeted mutational analysis of human tumours: a clinical platform to guide personalized cancer medicine</strong></div><div><em>EMBO Mol. Med.</em> 2010; <strong>2</strong>:146-158</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/emmm.201000070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-77954566882" target="_blank" aria-label="Open Scopus in new tab">Scopus (325)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20432502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Femmm.201000070&amp;pmid=20432502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our clinical cohort included 26 tumors, and 7 normal colon tissue samples (<a id="crosref1310" href="#mmc1">Table S1</a>).</div></section><section id="sec-4-3-2"><h4>Cell lines</h4><div id="p0315" role="paragraph">Our <i>in&nbsp;vitro</i> models included colon cancer cell lines (HCT116, SW480, RKO, LS-174T), a line derived from normal fetal colonic epithelium (FHC) and a primary fibroblast line (WI-38). Colon cell lines were purchased from ATCC: HCT116 (CCL-247), SW480 (CCL-228), LS174-T(CL-188), RKO (CRL-2577) and FHC (CRL-1831). The primary fibroblast line WI38 was obtained from Coriell (AG06814-N). HCT116 and SW480 were grown in McCoy’s 5A medium (GIBCO 16600082), 10% FBS and 0.5% pen-strep (GIBCO 10378016). LS174T and RKO were grown in EMEM (ATCC 30-2003), 10% FBS and pen-strep (GIBCO 10378016). FHC was cultured as per ATCC in DMEM/F12 media (ATCC 30-2006), 25mM HEPES, 10&nbsp;ng/ml cholera toxin, 0.005&nbsp;mg/ml insulin, 0.005&nbsp;mg/ml transferrin, 100&nbsp;ng/ml hydrocortisone, 20&nbsp;ng/ml recombinant EGF (Thermo Fisher PHG0311) and 10% FBS. WI38 was cultivated in EMEM with 15% FBS and passaged serially (approximately twice weekly) for 14&nbsp;weeks.</div></section></section><section id="sec-4-4"><h3>Method Details</h3><section id="sec-4-4-1"><h4>Tissue dissociation and crosslinking</h4><div id="p0320" role="paragraph">Tumor and normal colon samples were diced on dry ice into small (&lt; 1mm) pieces and resuspended in ice cold PBS (GIBCO 10010-023). When crosslinking, formaldehyde was added to 1% and tissue was rotated at room temperature (RT) for 15&nbsp;minutes. Glycine (2.5 M) was added to quench the formaldehyde and samples were rotated for an additional 5&nbsp;minutes at RT. For cell lines, cells were pelleted, crosslinked in 1% formaldehyde for 10&nbsp;minutes at 37 degrees and quenched with 2.5M glycine. All crosslinked samples were washed in ice cold PBS with protease inhibitors, pelleted and flash frozen in liquid nitrogen.</div></section><section id="sec-4-4-2"><h4>Hybrid selection bisulfite sequencing</h4><div id="p0325" role="paragraph">Total genomic DNA was isolated using the DNAeasy Blood &amp; Tissue Kit (QIAGEN) and sheared using the Covaris LE220. Ampure XP beads (Agencourt) were then used to size select gDNA fragments within 150-320bp, and sheared distribution was verified via BioAnalyzer (Agilient) prior to continuation. Sheared genomic DNA (1 ug) was then generated into a sequencing library via end repair, 3′ A base tailing (KAPA Hyper Prep Kit #KK8502) and sequencing adaptor ligation (Roche SeqCap Epi Enrichment System). Post-ligation clean-up was performed using Ampure XP beads. Following bead clean-up, DNA library was then bisulfite converted using the EZ DNA Methylation-Lightning Kit (Zymo Research) and amplified via PCR using KAPA HiFi U+ HotStart ReadyMix (KAPA #KK2800). Equal concentration of each bisulfite-converted library was then combined in sets of either three or four libraries/per pool along with SeqCap Epi universal &amp; indexing oligos and bisulfite capture enhancer (SeqCap Epi Accessory Kit). Each pool was subsequently lyophilized using TOMY Micro-Vac (MV100) and resuspended in hybridization buffer (SeqCap Epi Hybridization and Wash Kit) prior to being hybridized to SeqCap Epi Probe Pool (Roche) for 72 hours at 47C in a thermocycler with a heated lid at 57C. Following the 72-hour incubation, captured bisulfite-converted libraries were recovered (SeqCap Pure Capture Bead Kit) at 47C in a thermocycler for 45&nbsp;minutes, with intermediate vortexing every 15&nbsp;minutes. Capture beads were then washed (SeqCap Hybridization and Wash Kit) in a 47C water bath at room temperature, respectively. Captured bisulfite-converted libraries were amplified via PCR (SeqCap Epi Accessory Kit). Libraries were sequenced with 10% PhiX spike-in as 100-base paired end reads on the HiSeq2500 in rapid run mode.</div></section><section id="sec-4-4-3"><h4>Whole genome bisulfite library preparation</h4><div id="p0330" role="paragraph">Total genomic DNA was isolated using the DNAeasy Blood &amp; Tissue Kit (QIAGEN) and for each sample, one microgram of DNA was sheared using the Covaris LE220. Libraries were prepared according to manufacturer’s instructions using the NEXTFLEX bisulfite library prep kit (NOVA-5119-01). Libraries were sequenced on the Illumina NextSeq500 instrument.</div></section><section id="sec-4-4-4"><h4>ChIP-seq</h4><div id="p0335" role="paragraph">We generated chromatin state maps (H3K27ac, H3K36me3, H3K9me3 and H3K27me3) and binding profiles for the CTCF insulator protein by ChIP-seq. ChIP-seq was performed as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Liau et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Liau, B.B. ∙ Sievers, C. ∙ Donohue, L.K. ...</div><div><strong>Adaptive Chromatin Remodeling Drives Glioblastoma Stem Cell Plasticity and Drug Tolerance</strong></div><div><em>Cell Stem Cell.</em> 2017; <strong>20</strong>:233-246.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.stem.2016.11.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.stem.2016.11.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85008406243" target="_blank" aria-label="Open Scopus in new tab">Scopus (155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27989769/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2016.11.003&amp;pmid=27989769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, crosslinked cells were lysed and DNA was sheared to between 400 and 2,000 base pair fragments. Antibodies were as follows: CTCF (Cell signaling #3418), H3K27ac (Active Motif #39133), H3K9me3 (Abcam #8898), H3K27me3 (Cell Signaling #97335) and H3K36me3 (Abcam #9050). ChIP DNA was used to generate sequencing libraries by end repair (End-It DNA repair kit, Epicenter), 3′ A base overhang addition via Klenow fragment (NEB), and ligation of barcoded sequencing adapters. Barcoded fragments were amplified via PCR. Libraries were sequenced as 38-base paired-end reads on an Illumina NextSeq500 instrument.</div></section><section id="sec-4-4-5"><h4>Hi-C</h4><div id="p0340" role="paragraph">Hi-C maps of chromosome topology were initially generated for a cohort of 7 primary tumors, 4 normal colon tissue samples and 5 cell lines (cohort 1; <a id="crosref1320" href="#mmc1">Table S1</a>). We then confirmed our results by acquiring Hi-C data for a validation cohort of 5 tumors and 3 normal colon samples (cohort 2; <a id="crosref1325" href="#mmc1">Table S1</a>).</div><div id="p0345" role="paragraph"><i>In situ</i> Hi-C was performed as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref72-5" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-5" aria-controls="sref72-5">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref72-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Rao, S.S.P. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, crosslinked cells or tumor were thawed on ice in Hi-C lysis buffer. Tissue samples were mechanically disrupted with the Biomasher tissue grinder (Kimble Chase). Tissue and cell line samples were permeabilized in 0.5% SDS at 37 degrees, quenched with Triton X-100 and chromatin was digested with 100-200U MboI at 37 degrees overnight. Nuclei were then pelleted, ends were marked with biotin-14-dATP (ThermoFisher 19524016) and chromatin was ligated for 5 hours by T4 DNA ligase (M0202). Samples were treated with proteinase K at 55 degrees for 30&nbsp;minutes and cross-links were reversed at 68 degrees overnight. DNA was ethanol precipitated and sheared on a Covaris LE220. DNA was cleaned up via AMPure XP beads (Beckman Coulter, A63881) and quantified by Qubit dsDNA High Sensitivity Assay (Life Technologies, Q32854). Samples were bound to Dynabeads MyOne Streptavidin T1 beads (Life technologies, 65602) and washed. End repair, dATP attachment and adaptor ligation was performed. Final PCR amplification was performed using barcoded sequencing primers and PCR. Libraries were purified using AMPure XP beads and sequenced on either a NextSeq500 (150 cycle kit), HiSeq2500 (high output; 200 cycle kit) or NovaSeq S4 (200 cycles).</div></section><section id="sec-4-4-6"><h4>HiChIP</h4><div id="p0350" role="paragraph">We acquired SMC1 HiChIP data for cohort 1 (<a id="crosref1335" href="#mmc1">Table S1</a>). HiChIP was performed as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Mumbach et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Mumbach, M.R. ∙ Rubin, A.J. ∙ Flynn, R.A. ...</div><div><strong>HiChIP: efficient and sensitive analysis of protein-directed genome architecture</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:919-922</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84988336216" target="_blank" aria-label="Open Scopus in new tab">Scopus (469)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27643841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3999&amp;pmid=27643841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, crosslinked samples were lysed in Hi-C lysis buffer and chromatin was permeabilized in 0.5% SDS at 63C for 10&nbsp;minutes. Chromatin was digested with MboI for 2 hours at 37C. Overhangs were filled in and marked with Biotin-dATP (ThermoFisher 19524016), and ends were ligated with T4 DNA Ligase (NEB M0202) for 4 hours at room temperature. Nuclei were pelleted and lysed and chromatin was sheared on the Covaris E220 with the following conditions: Fill level 5, Duty Cycle 5, PIP 140, Cycles/burst 200, Time 4&nbsp;minutes. Samples were clarified, diluted in ChIP dilution buffer and precleared with Protein G beads (Invitrogen 11205D) for 1 hour at 4C. Samples were cleared on magnet and supernatant was added to antibody. Chromatin was ChIP’d overnight at 4C with rotation. Protein G beads were added and incubated for 2 hours at 4C with rotation. Beads were washed with low salt, high salt and LiCl buffer. Sample was eluted in ChIP elution buffer, treated with Proteinase K and crosslinks were reversed. DNA was purified using the Zymo clean &amp; concentrate kit (DCC-100). Streptavidin M280 beads (Invitrogen 11205D) were washed and resuspended in 2x biotin binding buffer and DNA was bound for 15&nbsp;minutes at room temperature. Beads were washe and Tn5 fragmentation was carried out as per Mumbach et&nbsp;al.,2016, with dilutions of Tn5 to process low input samples. Libraries were amplified using the Nextera DNA Library Prep kit (Illumina). Material was cleaned up using Ampure XP beads. Libraries were sequenced on NextSeq500 (150 cycle kit) or the HiSeq2500 (high output; 200 cycle kit).</div></section><section id="sec-4-4-7"><h4>RNA-seq</h4><div id="p0355" role="paragraph">Whole RNA was extracted using the QIAGEN RNeasy kit according to the manufacturer’s protocol. For RNA-seq library preparation, Poly(A)<sup>+</sup> RNA was enriched using magnetic oligo(dT)-beads (Life Technologies) and then ligated to RNA adaptors for sequencing. RNA-seq was performed with two biological replicates per colon cancer line and in singlicate for tumor samples. Libraries were sequenced as 38-base paired-end reads on an Illumina NextSeq500 instrument.</div></section><section id="sec-4-4-8"><h4>Electron microscopy</h4><div id="p0360" role="paragraph">Fresh tissue biopsies were placed directly into EM fixative (2.5% glutaraldehyde, 2.0% paraformaldehyde, 0.025 calcium chloride in a 0.1M sodium cacodylate buffer, pH 7.4) and allowed to fix for 3 hours at room temperature or ON at 4°C. Further processing was done in an EMS (Electron Microscopy Sciences) Lynx ll automatic tissue processor. Briefly, tissues were post-fixed with osmium tetroxide, dehydrated in a series of ethanol solutions, en block stained in the 70% ethanol step with uranyl acetate, further dehydrated in 100% ethanol and propylene oxide. Tissues were infiltrated in a series of propylene oxide, Epon mixtures and embedded in pure Epon. The Epon blocks were polymerized overnight in a 60°C oven. One micron sections were cut using glass knives and stained with toluidine blue. Representative areas were chosen by light microscopy. Thin sections were cut using an LKB ultramicrotome and diamond knife. The sections were stained with Sato’s lead stain and examined with a FEI Mogagni transmission electron microscope. Images were captured with an AMT (Advanced Microscopy Techniques) 2K digital CCD camera.</div></section><section id="sec-4-4-9"><h4>DNA-FISH</h4><section id="sec-4-4-9-1"><h5>Generation of DNA-FISH probes</h5><div id="p0365" role="paragraph">Based on evidence of CNV stability in tumors with available biological material, chromosome 12 was chosen for validation experiments. Only the p-arm of chromosome 12 was considered to avoid measuring arm-related differences in nuclear positioning. For compartments A and B, consecutive regions of at least 300Kb with absolute PC1 values larger than 0.5 in normal colon were identified (see methods section of eigenvector decomposition of Hi-C matrices). For these consecutive segments, the 100Kb regions at the middle of each segment were selected. For compartment I, a set of 100Kb regions was selected so that their linear distances to the B candidate regions was equivalent to the distance between the A candidate regions and the B candidate regions. Furthermore, the candidate regions were screened to have the same compartment-specific characteristics in HCT116 cells. Then, oligopaint libraries were designed using the Oligominer pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">Beliveau et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Beliveau, B.J. ∙ Joyce, E.F. ∙ Apostolopoulos, N. ...</div><div><strong>Versatile design and synthesis platform for visualizing genomes with Oligopaint FISH probes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:21301-21306</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1213818110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84871822966" target="_blank" aria-label="Open Scopus in new tab">Scopus (172)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23236188/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1213818110&amp;pmid=23236188" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Specifically, candidate regions were mined for probes using the length requirement of 80 nucleotides of homology, melting temperature range of 47-80°C, and default settings for the remaining Oligominer parameters. Resulting oligos yielded an average probe density of 4.6 probes/kb. An oligo pool (Twist Bioscience) was synthesized such that all probes targeting A, B, or I regions could be created in aggregate. Single-stranded probes were produced using PCR, T7 RNA synthesis, and reverse transcription as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Rosin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Rosin, L.F. ∙ Nguyen, S.C. ∙ Joyce, E.F.</div><div><strong>Condensin II drives large-scale folding and spatial partitioning of interphase chromosomes in Drosophila nuclei</strong></div><div><em>PLoS Genet.</em> 2018; <strong>14</strong>:e1007393</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1007393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85050986036" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30001329/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1007393&amp;pmid=30001329" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Shav-Tal, 2013</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Shav-Tal, Y.</div><div><strong>Imaging Gene Expression</strong></div><div>Springer, 2013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-62703-526-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-62703-526-2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-9-2"><h5>DNA-FISH on tissue</h5><div id="p0370" role="paragraph">For DNA-FISH, tissue samples were fixed in 4% paraformaldehyde (FisherScientific, 15710) for 2 hours, washed in 1X PBS and soaked for 2x 10&nbsp;minutes in 0.5M NH4Cl (26.6g/L in PBS, Sigma-Aldrich, 213330). The samples were cryoprotected by overnight incubation in 30% w/v sucrose (Sigma-Aldrich, S0389) in PBS at 4°C, nutating. The next morning, the sucrose was replaced with 30% sucrose, 50% OCT (Tissue-Tek<sup>∗</sup> O.C.T. Compound, Sakura, 25608-930) for two hours at 4°C, nutating. They were embedded in plastic peel-away molds in OCT, frozen on a cold block in liquid nitrogen, and stored in the freezer until sectioning. 5&nbsp;μm sections were sectioned using a cryostat, collected on Superfrost Plus microscope slides (FisherSci 22-037-246) and stored at −20°C.</div><div id="p0375" role="paragraph">Slides were thawed at room temperature for 30’ and rehydrated in 10&nbsp;mM sodium citrate for 5′. Next, the slides were incubated for 10’ in 10&nbsp;mM sodium citrate at 80°C and allowed to cool down at room temperature for ∼30’. They were washed twice at 2X SSC, 5′ each, and transferred to 50% formamide (Sigma Aldrich, F9037) in 2X SSCT (SSC&nbsp;+ 0.1% Tween-20) for at least 1h. Following denaturation, the tissue was dehydrated in an ethanol row (70%, 95%, 100%, three minutes each) and air-dried for at least 90&nbsp;minutes. They were then acetylated with acetic anhydride as follows: dried slides were equilibrated in 0.1M fresh ethanolamine (Sigma Aldrich, 90279) in dH<sub>2</sub>O, pH 8.0 for 10&nbsp;minutes, then transferred to 0.25% v/v acetic anhydride (Sigma Aldrich, 320102) in 0.1M ethanolamine for 5′ followed by washing for 10’ in 2X SSCT. During the previous steps, probes against compartments A, B and I were prepared by adding 1 uL of 100 uM dNTPs (Life Technologies, R1121) and 50 pmol probes each per slide, speed vacuuming on high for 20’ (or until the liquid had evaporated), and resuspending the probes in 5.25 uL per sample. This probe-dNTP mix was then added to the hybridization mixture containing a final concentration of 50% formamide, 10 ug RnaseA (Thermo Scientific EN0531) and 1x Dextran Sulfate Mix (10% Dextran Sulfate D8906, Sigma Aldrich, 2xSSC, 0.1% Tween-20). Next, the slides were mounted with probe mix, covered with a glass coverslip and rubber cement (Staples, EPI231) and incubated on a hot block at 42°C for 1-3 hours to allow infiltration with the probe. Then, the slides were heat-shocked to denature the section and probe at 85°C for 7&nbsp;minutes and incubated overnight (16h+) at 37°C in a temperature-controlled oven. The next day, the coverslips were removed from the sections and washed at 60°C for 15&nbsp;minutes in pre-warmed 2X SSCT, at room temperature for 10’ in 2X SSCT, at room temperature for 10’ in 0.2X SSCT, and transferred to 2X SSC while secondary mix was prepared consisting of 10% formamide, 1x Dextran Sulfate Mix and 10 pmol of secondary probes labeled with Cy3 (B compartment), Cy5 (A or I compartment) and A488 (A compartment) in dH2O. The sections were incubated with secondary mix in a humid chamber at room temperature in the dark for at least two hours, followed by the same washes as before. In the last step with 2X SSC, the nuclei were counterstained with DAPI (Sigma-Aldrich, D1305, 5&nbsp;μg/mL final concentration), briefly washed in 2X SSC and mounted using SlowFade Gold Antifade Mountant (Invitrogen, S36936).</div></section><section id="sec-4-4-9-3"><h5>DNA-FISH on HCT116 cells</h5><div id="p0380" role="paragraph">Cells were grown on glass coverslips, fixed in 4% PFA for 15&nbsp;minutes, and processed as described above with the following adjustments: sodium citrate and acetic anhydride treatments were omitted; instead, the cells were permeabilized after fixation using 0.5% Triton X-100 for 15’ followed by a 5′ wash in PBS and alcohol denaturation. Prior to probe mix addition, the samples were re-equilibrated using 2X SSCT&nbsp;+ formamide (50% v/v 4x SSCT&nbsp;+ 50% v/v formamide) for 1h at 37°C. The probes were then immediately denatured at 80°C for 5′ and incubated overnight at 37°C. The DNA-FISH stainings were imaged on a Zeiss LSM800 confocal microscope with Airyscan settings and a 63x oil objective.</div></section></section><section id="sec-4-4-10"><h4>Treatment with 5-azacytidine</h4><div id="p0385" role="paragraph">5-azacytidine was obtained from SelleckChem (S1782) and HCT116 cells were cultured in the presence of 5 uM or 1:2000 DMSO control. Cells were plated in media containing 5-aza or DMSO and harvested at 24 h.</div></section></section><section id="sec-4-5"><h3>Quantification and Statistical Analysis</h3><section id="sec-4-5-1"><h4>DNA methylation data preprocessing and quantification</h4><div id="p0390" role="paragraph">BSMAP version 2.74 was used both to map the sequenced reads to the reference genome (hg19) and to calculate the methylated fraction for each CpG across the genome (<span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Xi and Li, 2009</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Xi, Y. ∙ Li, W.</div><div><strong>BSMAP: whole genome bisulfite sequence MAPping program</strong></div><div><em>BMC Bioinformatics.</em> 2009; <strong>10</strong>:232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-10-232" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-68949141599" target="_blank" aria-label="Open Scopus in new tab">Scopus (510)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19635165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-10-232&amp;pmid=19635165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Data was further analyzed within the framework of the bsseq Bioconductor package (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Hansen et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hansen, K.D. ∙ Langmead, B. ∙ Irizarry, R.A.</div><div><strong>BSmooth: from whole genome bisulfite sequencing reads to differentially methylated regions</strong></div><div><em>Genome Biol.</em> 2012; <strong>13</strong>:R83</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2012-13-10-r83" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23034175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2012-13-10-r83&amp;pmid=23034175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0395" role="paragraph">In order to avoid potential biases introduced by the DNA capture enrichment step, genomic coordinates of hypomethylated blocks were defined based on differentially methylated regions (DMRs) using previously published whole-genome bisulfite sequencing data (<span class="dropBlock reference-citations"><a id="body-ref-sref32-5" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-5" aria-controls="sref32-5">Hansen et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref32-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hansen, K.D. ∙ Timp, W. ∙ Bravo, H.C. ...</div><div><strong>Increased methylation variation in epigenetic domains across cancer types</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:768-775</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.865" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79960927422" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21706001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.865&amp;pmid=21706001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). DMRs with methylation differences larger than 10% between tumors and normal samples were considered. Consecutive DMRs smaller than 500Kb bins were merged if the genomic distance between the DMRs was smaller than 10% of the width of the individual DMRs. Only merged DMRs larger than 100Kb were considered for further analysis in order to match the resolution of the compartment calls of the Hi-C data. To quantify block-level methylation in our samples, we used only methylation at open-sea CpGs, i.e., those located &gt; 2kb from CpG islands. We confirmed that hypomethylated blocks were recapitulated in our capture bisulfite sequencing data, displaying methylation differences both between tumor samples and normal samples, and between regions inside a block and the flanking genomic regions.</div><div id="p0400" role="paragraph">For each sample, the degree of CIMP hypermethylation was assessed by measuring average methylation at CIMP-specific methylated islands (<span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Xu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Xu, Y. ∙ Hu, B. ∙ Choi, A.J. ...</div><div><strong>Unique DNA methylome profiles in CpG island methylator phenotype colon cancers</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:283-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.122788.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84863011458" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21990380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.122788.111&amp;pmid=21990380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Samples with the highest methylation levels at these sites were labeled as CIMP. A clinical genotyping assay (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Dias-Santagata et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Dias-Santagata, D. ∙ Akhavanfard, S. ∙ David, S.S. ...</div><div><strong>Rapid targeted mutational analysis of human tumours: a clinical platform to guide personalized cancer medicine</strong></div><div><em>EMBO Mol. Med.</em> 2010; <strong>2</strong>:146-158</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/emmm.201000070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-77954566882" target="_blank" aria-label="Open Scopus in new tab">Scopus (325)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20432502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Femmm.201000070&amp;pmid=20432502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) revealed <i>BRAF</i> mutations in the two CIMP tumors, consistent with reported associations (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Hinoue et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hinoue, T. ∙ Weisenberger, D.J. ∙ Lange, C.P.E. ...</div><div><strong>Genome-scale analysis of aberrant DNA methylation in colorectal cancer</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:271-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.117523.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84856549042" target="_blank" aria-label="Open Scopus in new tab">Scopus (443)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21659424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.117523.110&amp;pmid=21659424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). TCGA methylation data was used to verify that the methylation levels of our CIMP samples were comparable to the methylation levels of CIMP tumors in TCGA. TCGA DNA methylation data was downloaded using TCGAbiolinks (<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Colaprico et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Colaprico, A. ∙ Silva, T.C. ∙ Olsen, C. ...</div><div><strong>TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e71</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84966269471" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26704973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1507&amp;pmid=26704973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Comparison of CpG island hypermethylation and block hypomethylation across our samples and a larger cohort of colon tumors from the Cancer Genome Atlas (TCGA) (<span class="dropBlock reference-citations"><a id="body-ref-sref8-4" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-4" aria-controls="sref8-4">Cancer Genome Atlas Network, 2012</a><div class="dropBlock__holder" data-db-target-of="sref8-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cancer Genome Atlas Network</div><div><strong>Comprehensive molecular characterization of human colon and rectal cancer</strong></div><div><em>Nature.</em> 2012; <strong>487</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84863922124" target="_blank" aria-label="Open Scopus in new tab">Scopus (4694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22810696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11252&amp;pmid=22810696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) confirmed that CIMP and block hypomethylation are indeed independent features.</div></section><section id="sec-4-5-2"><h4>Cancer-associated DNA methylation alterations in normal colon tissue</h4><div id="p0405" role="paragraph">We obtained pre-processed Illumina EPIC and 450k methylation array beta values data from the colon tissue samples described in Wang et&nbsp;al., 2020 (NCBI GEO <a id="intref0055" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132804" target="_blank">GSE132804</a> Series Matrix files). Analysis was restricted to probes common to the two platforms. We computed a block hypomethylation score for each sample as the average DNA methylation of open-sea CpGs (i.e., those &gt; 2kb from a CpG island) within hypomethylated blocks, using the same block region definitions used elsewhere in the study for tumor versus normal comparisons. We verified that the low and high risk group samples were spread across the two platforms (EPIC: 52 high risk, 57 low risk; 450k: 49 high risk, 48 low risk), and that results were consistent when analysis was performed for each platform separately.</div></section><section id="sec-4-5-3"><h4>Hi-C analysis</h4><div id="p0410" role="paragraph">We confirmed data quality by assessing the fraction of cis-long range contacts for each library. Each map contained an average of 320 million contacts, for an average resolution of 10 Kb. We used these data to derive TAD and compartment structures.</div><div id="p0415" role="paragraph">Data were controlled for quality, mapped to the reference genome (hg19) and converted into interaction matrices using HiC-Pro v2.10.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref84-2" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-2" aria-controls="sref84-2">Servant et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref84-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Servant, N. ∙ Varoquaux, N. ∙ Lajoie, B.R. ...</div><div><strong>HiC-Pro: an optimized and flexible pipeline for Hi-C data processing</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0831-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84952861819" target="_blank" aria-label="Open Scopus in new tab">Scopus (450)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0831-x&amp;pmid=26619908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using pipeline code available at <a id="intref0060" href="https://github.com/aryeelab/topology_tools" target="_blank">https://github.com/aryeelab/topology_tools</a>. Within sample normalization was performed using the Iterative Correction and Eigenvector decomposition (ICE) method (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Imakaev et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Imakaev, M. ∙ Fudenberg, G. ∙ McCord, R.P. ...</div><div><strong>Iterative correction of Hi-C data reveals hallmarks of chromosome organization</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:999-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2148" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84866997011" target="_blank" aria-label="Open Scopus in new tab">Scopus (536)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22941365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2148&amp;pmid=22941365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For each chromosome in each sample, compartments were called using the standard PCA method (<span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Lieberman-Aiden et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Lieberman-Aiden, E. ∙ van Berkum, N.L. ∙ Williams, L. ...</div><div><strong>Comprehensive mapping of long-range interactions reveals folding principles of the human genome</strong></div><div><em>Science.</em> 2009; <strong>326</strong>:289-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1181369" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-70349873824" target="_blank" aria-label="Open Scopus in new tab">Scopus (3713)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19815776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1181369&amp;pmid=19815776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, the interaction matrix <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X equals x Subscript i j" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (8 2 (7 3 6 4)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="10" data-semantic-children="0,8" data-semantic-content="1" data-semantic-owns="0 1 8" data-semantic-speech="upper X equals x Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="2,7" data-semantic-parent="10" data-semantic-owns="2 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="3,4" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="3 6 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-math></mjx-container>was transformed into an observed over expected (O/E) matrix by dividing each element of the matrix by the expected interaction frequency for a given distance from the diagonal <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="k equals i minus j" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 1 (5 2 3 4))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="6" data-semantic-children="0,5" data-semantic-content="1" data-semantic-owns="0 1 5" data-semantic-speech="k equals i minus j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-children="2,4" data-semantic-content="3" data-semantic-parent="6" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, defined as the mean of the values <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="x Subscript i j" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 (5 1 4 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="0,5" data-semantic-owns="0 5" data-semantic-speech="x Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="1,2" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="1 4 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>with the same value of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="k" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="k"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-math></mjx-container>. A correlation matrix was generated by estimating the pairwise correlation coefficients of all the rows of the O/E matrix. Then, an eigendecomposition was performed on the correlation matrix, and the sign of the first eigenvector was used to assign compartment labels. We used expression data and GC content to flip the sign of the eigenvector such that values larger than 0 correspond to open (A) regions and values smaller than 0 correspond to closed (B) regions.</div><div id="p0420" role="paragraph">In addition to using the eigenvector (PC1) metric, we also directly quantified the tendency of each region to interact with other regions in either the A or B compartments. We calculated the “A/B interaction ratio,” defined for each 100kb genomic window as the ratio of interaction frequency with the A versus B compartments using the O/E matrix. Specifically, we calculate log<sub>2</sub>(mean O/E interaction frequency with A regions) – log<sub>2</sub>(mean O/E interaction frequency with B regions). For each comparison, A and B were defined in the ‘baseline’ condition: normal colon tissue, HCT116 cells treated with DMSO, or early passage fibroblasts (passage 16). The same A and B definitions were used for all samples within a comparison. We confirmed that assessing the relationship between compartmental change and hypomethylation gave consistent results when using A/B interaction ratio instead of eigenvector as the outcome (See “<a id="crosref1415" href="#sec-4-5-4">Association of compartmental organization with DNA hypomethylation</a>”).</div><div id="p0425" role="paragraph">In primary tissues, compartment I was defined as those genomic regions with a positive value of the first eigenvector that were within a block of DNA hypomethylation (defined by comparing tumors versus normal). We verified that compartment I could be identified based only on Hi-C data. To do this, we plotted the first 5 eigenvectors of the matrix decomposition used to define compartments A and B. The first eigenvector separates compartment A and B. Compartment I could be identified using the second eigenvector on several chromosomes. In other chromosomes, compartment I was often separated by lower eigenvectors, and became more evident when we applied the matrix decomposition method to individual chromosome arms. For IMR90 cells, compartment I was assigned as in primary tissues using DNA methylation differences between proliferating and senescent cells. For HCT116, 100Kb genomic bins were labeled as I if two consecutive genomic bins had a positive value on the eigenvector decomposition described (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Lieberman-Aiden et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Lieberman-Aiden, E. ∙ van Berkum, N.L. ∙ Williams, L. ...</div><div><strong>Comprehensive mapping of long-range interactions reveals folding principles of the human genome</strong></div><div><em>Science.</em> 2009; <strong>326</strong>:289-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1181369" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-70349873824" target="_blank" aria-label="Open Scopus in new tab">Scopus (3713)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19815776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1181369&amp;pmid=19815776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and if their open sea CpG DNA methylation values were equal or less than 80%. We confirmed that this approach enriched for regions that were consistent with our definition of compartment I, i.e., that had intermediate A/B contact Hi-C patterns and that were enriched for the H3K27me3 chromatin mark compared to compartments A and B.</div><div id="p0430" role="paragraph">Sub-compartment structures defined by Rao et&nbsp;al., 2014 were called using SNIPER (<span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Xiong and Ma, 2019</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Xiong, K. ∙ Ma, J.</div><div><strong>Revealing Hi-C subcompartments by imputing inter-chromosomal chromatin interactions</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:5069</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12954-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85074716041" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31699985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12954-4&amp;pmid=31699985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SNIPER uses Hi-C data and sub-compartment calls to train a model that is able to learn the interaction patterns of each sub-compartment. These trained models can be used to define sub-compartments from Hi-C data that are distinct from the training Hi-C data. To call sub-compartments in colon tissue, we used the pre-trained models provided by SNIPER that were trained using 5% of GM12878 Hi-C data (<span class="dropBlock reference-citations"><a id="body-ref-sref72-6" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-6" aria-controls="sref72-6">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref72-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Rao, S.S.P. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as these data were of a comparable coverage to our individual HiC samples.</div><div id="p0435" role="paragraph">Insulation scores were calculated by defining two windows of length <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="l"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-math></mjx-container>, one upstream and one downstream of a given genomic position. For each chromosome, the log (base 2) ratio of the sum of interaction counts within each window and the interaction counts between the two windows was calculated at each position. These log ratios were further transformed into <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="z" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="z"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi></mjx-math></mjx-container>-scores by subtracting the median and dividing by the median absolute deviation. In order to capture boundaries of TADs of varying sizes, including nested TAD structures, the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="z" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="z"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi></mjx-math></mjx-container>-scores were calculated genome-wide for different values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="l"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-math></mjx-container>, specifically <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l equals 200 upper K b" tabindex="0" ctxtmenu_counter="8" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 1 (7 2 5 3 6 4))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="8" data-semantic-children="0,7" data-semantic-content="1" data-semantic-owns="0 1 7" data-semantic-speech="l equals 200 upper K b"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="8" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="2,3,4" data-semantic-content="5,6" data-semantic-parent="8" data-semantic-owns="2 5 3 6 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l equals 400 upper K b" tabindex="0" ctxtmenu_counter="9" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 1 (7 2 5 3 6 4))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="8" data-semantic-children="0,7" data-semantic-content="1" data-semantic-owns="0 1 7" data-semantic-speech="l equals 400 upper K b"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="8" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="2,3,4" data-semantic-content="5,6" data-semantic-parent="8" data-semantic-owns="2 5 3 6 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c34" noic="true" style="padding-top: 0.677em;">4</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.677em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.677em;">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l equals 800 upper K b" tabindex="0" ctxtmenu_counter="10" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 1 (7 2 5 3 6 4))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="8" data-semantic-children="0,7" data-semantic-content="1" data-semantic-owns="0 1 7" data-semantic-speech="l equals 800 upper K b"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="8" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="2,3,4" data-semantic-content="5,6" data-semantic-parent="8" data-semantic-owns="2 5 3 6 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c38" noic="true" style="padding-top: 0.666em;">8</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Boundaries were called for genomic positions where the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="z" tabindex="0" ctxtmenu_counter="11" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="z"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi></mjx-math></mjx-container>-scores were larger than the 90% quantile of a standard normal distribution in at least one resolution in one sample. Three strategies were used to assess the stability of TADs across samples: (1) we calculated the correlation coefficients of the insulation scores, (2) for each sample, we defined the position of TAD boundaries in that sample and calculated the percentage of TAD boundaries where the insulation scores of other samples were also in a local minima and (3) we plotted metaplots of the insulation scores to visually inspect conservation of TAD boundaries.</div><div id="p0440" role="paragraph">Juicebox (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Durand et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Durand, N.C. ∙ Robinson, J.T. ∙ Shamim, M.S. ...</div><div><strong>Juicebox Provides a Visualization System for Hi-C Contact Maps with Unlimited Zoom</strong></div><div><em>Cell Syst.</em> 2016; <strong>3</strong>:99-101</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cels.2015.07.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cels.2015.07.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27467250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.07.012&amp;pmid=27467250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used for exploratory visualization of the Hi-C data.</div></section><section id="sec-4-5-4"><h4>Association of compartmental organization with DNA hypomethylation</h4><div id="p0445" role="paragraph">We used linear mixed effects models to assess the association between open-sea CpG hypomethylation and Hi-C eigenvector in three settings: 1) tumor versus normal colon, 2) HCT116 cells treated with DMSO (control) or 5-azacytidine, and 3) aging WI38 fibroblasts sampled at passage 16, 30 and 40. For each 100kb window in each sample we computed DNA hypomethylation relative to a baseline condition (normal colon, HCT116+DMSO, or WI-38 passage 16). In each case, we fit linear mixed effects model to I and B separately with eigenvector as the outcome, hypomethylation bin as a fixed effect and random effect terms for sample and genomic window. The intercept corresponds to regions with minimal (&lt; 15%) hypomethylation, and the coefficient estimates shown in <a id="crosref1440" href="#fig5">Figure&nbsp;5</a> represent the mean eigenvector change relative to these methylation-stable regions. An additional model was used to assess PC1 change over time in WI-38 fibroblasts for 100kb windows that show &gt; 20% hypomethylation in late passage (P40) versus early passage (P16) WI-38 fibroblasts. This model uses passage number as the fixed effect instead of hypomethylation.</div></section><section id="sec-4-5-5"><h4>HiChIP analysis</h4><div id="p0450" role="paragraph">Data were controlled for quality, mapped to the reference genome (hg19) and converted into interaction matrices using the HiC-Pro pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref84-3" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-3" aria-controls="sref84-3">Servant et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref84-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Servant, N. ∙ Varoquaux, N. ∙ Lajoie, B.R. ...</div><div><strong>HiC-Pro: an optimized and flexible pipeline for Hi-C data processing</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0831-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84952861819" target="_blank" aria-label="Open Scopus in new tab">Scopus (450)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0831-x&amp;pmid=26619908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Chromatin loops were called using the hichipper 0.7.3 with the parameter to use user-defined peaks (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Lareau and Aryee, 2018b</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Lareau, C.A. ∙ Aryee, M.J.</div><div><strong>hichipper: a preprocessing pipeline for calling DNA loops from HiChIP data</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:155-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4583" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85042766402" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29489746/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4583&amp;pmid=29489746" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). hichipper was run for each HiChIP sample using the union of CTCF and H3K27ac peaks as the predefined peak set. The hichipper pipeline defines potential loop anchors by extending peaks by a fixed window (i.e., 500bp) to account for uncertainty in the peak calling and merging peaks whose genomic distance is below 500bp. These extended peaks are overlapped with restriction fragments and are further extended to the edges of the restriction fragments they overlap. For each pair of potential loop anchors, hichipper counts the number of valid contact pairs (defined by HiC-Pro) that support their 3D interaction. Running this step for each sample results in a matrix <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Z Subscript i j Baseline equals z Subscript i j" tabindex="0" ctxtmenu_counter="12" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (5 0 (4 1 3 2)) 6 (13 8 (12 9 11 10)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="14" data-semantic-children="5,13" data-semantic-content="6" data-semantic-owns="5 6 13" data-semantic-speech="upper Z Subscript i j Baseline equals z Subscript i j"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="14" data-semantic-owns="0 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="6" data-semantic-parent="14" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="8,12" data-semantic-parent="14" data-semantic-owns="8 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="13"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.002em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="12" data-semantic-children="9,10" data-semantic-content="11" data-semantic-parent="13" data-semantic-owns="9 11 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="12"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> where each column is a sample <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="j" tabindex="0" ctxtmenu_counter="13" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="j"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-math></mjx-container> and each row <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="14" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-math></mjx-container> represents a pair of loop anchors (i.e., a loop), and<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="z Subscript i j" tabindex="0" ctxtmenu_counter="15" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 1 (6 2 5 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="1,6" data-semantic-owns="1 6" data-semantic-speech="z Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="7"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.002em;"><mjx-mrow size="s"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="2,3" data-semantic-content="5" data-semantic-parent="7" data-semantic-owns="2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the number of valid read pairs that supports a loop <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="16" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-math></mjx-container> in sample <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="j" tabindex="0" ctxtmenu_counter="17" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="j"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-math></mjx-container>. To distinguish between random background contacts from contacts due to DNA looping, hichipper runs the mango background correction model on the sum of loop counts across samples (i.e., <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="sigma summation Underscript j Endscripts z Subscript i j" tabindex="0" ctxtmenu_counter="18" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (2 0 1) (8 3 (7 4 6 5)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="9" data-semantic-children="2,8" data-semantic-content="0" data-semantic-owns="2 8" data-semantic-speech="sigma summation Underscript j Endscripts z Subscript i j"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="9" data-semantic-owns="3 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.002em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-math></mjx-container>). The mango correction consists of modeling the counts using a binomial distribution to estimate the probability of observing the counts between two genomic loci given its genomic distance. The resulting p values are corrected for multiple testing. Loops with a q-value smaller than 0.1, with at least 4 valid contacts in two or more samples and with at least 20 counts across all samples were considered high-confidence loops and were considered for further analysis. Significant loops were annotated as enhancer-promoter loops if one of the anchors overlapped an H3K27ac peak (enhancer-like) and the other anchor overlapped the promoter of a gene.</div><div id="p0455" role="paragraph">To assess the robustness of our results to the loop calling algorithm, we repeated our analyses using a different loop calling algorithm, cLoops (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Cao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Cao, Y. ∙ Chen, Z. ∙ Chen, X. ...</div><div><strong>Accurate loop calling for 3D genomic data with cLoops</strong></div><div><em>Bioinformatics.</em> 2019; <strong>36</strong>:666-675</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.CaoZ.ChenX.ChenD.AiG.ChenJ.McDermottY.HuangX.GuoJ.-D.J.HanAccurate+loop+calling+for+3D+genomic+data+with+cLoopsBioinformatics362019666675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We ran cLoops version 0.92 using the parameters “-hic -eps 2500,5000,7500,10000 -minPts 3,5,10 -j -s -w” for each sample and used the union of the per-sample significant loops as our final set of loops. The global trends described in the main text were robust to the loop calling algorithm.</div><div id="p0460" role="paragraph">The software tool diffloop was used to test for differential looping (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Lareau and Aryee, 2018a</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Lareau, C.A. ∙ Aryee, M.J.</div><div><strong>diffloop: a computational framework for identifying and analyzing differential DNA loops from sequencing data</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:672-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btx623" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85042523227" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29028898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtx623&amp;pmid=29028898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). diffloop uses the statistical engine of the edgeR package (<span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (13800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), where the matrix of loop counts <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Z Subscript i j" tabindex="0" ctxtmenu_counter="19" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="upper Z Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-math></mjx-container> is modeled using generalized linear models (GLM) of the negative binomial distribution:<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Z Subscript i j Baseline tilde upper N upper B left parenthesis mu Subscript i j Baseline comma alpha Subscript i Baseline right parenthesis" tabindex="0" ctxtmenu_counter="20" style="font-size: 130%;"><mjx-math data-semantic-structure="(28 (5 0 (4 1 3 2)) 6 (27 8 26 (25 9 24 (23 10 (21 (16 11 (15 12 14 13)) 17 (20 18 19)) 22))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="28" data-semantic-children="5,27" data-semantic-content="6" data-semantic-owns="5 6 27" data-semantic-speech="upper Z Subscript i j Baseline tilde upper N upper B left parenthesis mu Subscript i j Baseline comma alpha Subscript i Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="28" data-semantic-owns="0 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="6" data-semantic-parent="28" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="27" data-semantic-children="8,25" data-semantic-content="26" data-semantic-parent="28" data-semantic-owns="8 26 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="27"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="25" data-semantic-children="9,23" data-semantic-content="24,9" data-semantic-parent="27" data-semantic-owns="9 24 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="25" data-semantic-operator="appl"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="23" data-semantic-children="21" data-semantic-content="10,22" data-semantic-parent="25" data-semantic-owns="10 21 22"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="10" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="21" data-semantic-children="16,17,20" data-semantic-content="17" data-semantic-parent="23" data-semantic-owns="16 17 20"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="16" data-semantic-children="11,15" data-semantic-parent="21" data-semantic-owns="11 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="16"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="15" data-semantic-children="12,13" data-semantic-content="14" data-semantic-parent="16" data-semantic-owns="12 14 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="17" data-semantic-parent="21" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="21" data-semantic-owns="18 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="22" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div><div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu Subscript i j Baseline equals n Subscript i j Baseline z Subscript i j" tabindex="0" ctxtmenu_counter="21" style="font-size: 130%;"><mjx-math data-semantic-structure="(21 (5 0 (4 1 3 2)) 6 (20 (12 7 (11 8 10 9)) 19 (18 13 (17 14 16 15))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="21" data-semantic-children="5,20" data-semantic-content="6" data-semantic-owns="5 6 20" data-semantic-speech="mu Subscript i j Baseline equals n Subscript i j Baseline z Subscript i j"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="21" data-semantic-owns="0 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="6" data-semantic-parent="21" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-children="12,18" data-semantic-content="19" data-semantic-parent="21" data-semantic-owns="12 19 18"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="12" data-semantic-children="7,11" data-semantic-parent="20" data-semantic-owns="7 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="11" data-semantic-children="8,9" data-semantic-content="10" data-semantic-parent="12" data-semantic-owns="8 10 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="13,17" data-semantic-parent="20" data-semantic-owns="13 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="18"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.002em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="17" data-semantic-children="14,15" data-semantic-content="16" data-semantic-parent="18" data-semantic-owns="14 16 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="17"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu Subscript i j" tabindex="0" ctxtmenu_counter="22" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="mu Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the fitted mean and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript i" tabindex="0" ctxtmenu_counter="23" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="alpha Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the dispersion estimate, which is estimated using the common dispersion method from edgeR. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n Subscript i j" tabindex="0" ctxtmenu_counter="24" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="n Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are normalization factors that account both for library size and for copy number differences between samples. Specifically, we overlapped the genomic coordinates of the loop anchors with the copy number estimates of each sample (see Section “<a id="crosref1470" href="#sec-4-5-6">Copy number variant analysis</a>” for details) to generate a matrix of copy number estimates (loops <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="25" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-math></mjx-container> times sample <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="j" tabindex="0" ctxtmenu_counter="26" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="j"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-math></mjx-container>). This matrix of copy numbers was row-centered. The resulting matrix was multiplied by the library size factors estimated by edgeR and the resulting values defined <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n Subscript i j" tabindex="0" ctxtmenu_counter="27" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="n Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, which were introduced as offsets when fitting the GLMs. To test for differential looping between conditions, a GLM was fitted for each gene and a likelihood ratio test was used to calculate p values for each loop and the Benjamini-Hochberg was used to correct for multiple testing.</div></section><section id="sec-4-5-6"><h4>Copy number variant analysis</h4><div id="p0465" role="paragraph">The coverage of our DNA methylation sequencing assays was used to assess copy number alterations in tumors. For each sample, the number of sequencing reads was counted for non-overlapping fixed-width genomic regions of 40Kb and CNAnorm was used to infer copy number alterations. CNAnorm inputs two vectors of read counts, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="w Subscript k l" tabindex="0" ctxtmenu_counter="28" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="w Subscript k l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-math></mjx-container>and<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="y Subscript k l" tabindex="0" ctxtmenu_counter="29" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 1 (5 2 4 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="1,5" data-semantic-owns="1 5" data-semantic-speech="y Subscript k l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="2,3" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="2 4 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, which represent read counts for genomic region <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l" tabindex="0" ctxtmenu_counter="30" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="l"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-math></mjx-container> of chromosome <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="k" tabindex="0" ctxtmenu_counter="31" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="k"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-math></mjx-container> for a tumor sample and a matching normal sample, respectively. CNAnorm performs the following steps to determine copy number alterations. First, the ratio of the read counts is calculated, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript k l Baseline equals StartFraction w Subscript k l Baseline Over y Subscript k l Baseline EndFraction period" tabindex="0" ctxtmenu_counter="32" style="font-size: 130%;"><mjx-math data-semantic-structure="(24 (23 (5 0 (4 1 3 2)) 6 (20 (13 8 (12 9 11 10)) (19 14 (18 15 17 16)))) 21)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="24" data-semantic-children="23,21" data-semantic-content="21" data-semantic-owns="23 21" data-semantic-speech="r Subscript k l Baseline equals StartFraction w Subscript k l Baseline Over y Subscript k l Baseline EndFraction period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="23" data-semantic-children="5,20" data-semantic-content="6" data-semantic-parent="24" data-semantic-owns="5 6 20"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="23" data-semantic-owns="0 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="6" data-semantic-parent="23" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="20" data-semantic-children="13,19" data-semantic-parent="23" data-semantic-owns="13 19"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="8,12" data-semantic-parent="20" data-semantic-owns="8 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="13"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="12" data-semantic-children="9,10" data-semantic-content="11" data-semantic-parent="13" data-semantic-owns="9 11 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="12"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="19" data-semantic-children="14,18" data-semantic-parent="20" data-semantic-owns="14 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="19"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="18" data-semantic-children="15,16" data-semantic-content="17" data-semantic-parent="19" data-semantic-owns="15 17 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="18"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="18"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="21" data-semantic-parent="24" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-math></mjx-container>This ratio is normalized using a loess-based method that defines <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript k l Superscript n o r m" tabindex="0" ctxtmenu_counter="33" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (13 0 (4 1 3 2)) (12 5 9 6 10 7 11 8))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="0,4,12" data-semantic-collapsed="(14 (13 0 4) 12)" data-semantic-owns="0 4 12" data-semantic-speech="r Subscript k l Superscript n o r m"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="5,6,7,8" data-semantic-content="9,10,11" data-semantic-parent="14" data-semantic-owns="5 9 6 10 7 11 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="12"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="12"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="14" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, which removes technical dependencies between <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript k l" tabindex="0" ctxtmenu_counter="34" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="r Subscript k l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and GC content. Second, to remove random error variability, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript k l Superscript n o r m" tabindex="0" ctxtmenu_counter="35" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (13 0 (4 1 3 2)) (12 5 9 6 10 7 11 8))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="0,4,12" data-semantic-collapsed="(14 (13 0 4) 12)" data-semantic-owns="0 4 12" data-semantic-speech="r Subscript k l Superscript n o r m"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="5,6,7,8" data-semantic-content="9,10,11" data-semantic-parent="14" data-semantic-owns="5 9 6 10 7 11 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="12"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="12"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="14" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> is smoothed throughout the genome (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Huang et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Huang, J. ∙ Gusnanto, A. ∙ O’Sullivan, K. ...</div><div><strong>Robust smooth segmentation approach for array CGH data analysis</strong></div><div><em>Bioinformatics.</em> 2007; <strong>23</strong>:2463-2469</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btm359" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-34548725570" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17660206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtm359&amp;pmid=17660206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Third, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript k l Superscript n o r m" tabindex="0" ctxtmenu_counter="36" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (13 0 (4 1 3 2)) (12 5 9 6 10 7 11 8))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="0,4,12" data-semantic-collapsed="(14 (13 0 4) 12)" data-semantic-owns="0 4 12" data-semantic-speech="r Subscript k l Superscript n o r m"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="5,6,7,8" data-semantic-content="9,10,11" data-semantic-parent="14" data-semantic-owns="5 9 6 10 7 11 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="12"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="12"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="14" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> values are transformed so that the most common genomic regions are centered to ratio one. Then, CNAnorm normalizes for tumor purity by shrinking <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript k l Superscript n o r m" tabindex="0" ctxtmenu_counter="37" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (13 0 (4 1 3 2)) (12 5 9 6 10 7 11 8))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="0,4,12" data-semantic-collapsed="(14 (13 0 4) 12)" data-semantic-owns="0 4 12" data-semantic-speech="r Subscript k l Superscript n o r m"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="5,6,7,8" data-semantic-content="9,10,11" data-semantic-parent="14" data-semantic-owns="5 9 6 10 7 11 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="12"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="12"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="14" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> so that the modes of the distribution fit values resulting from copy number alteration processes (deletion&nbsp;= 0, deletion of one chromosome copy&nbsp;= 0.5, no CNV&nbsp;= 1, amplification of one copy&nbsp;= 1.5, etc). Finally, a circular binary segmentation algorithm is used to define regions with copy number alterations (<span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Olshen et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Olshen, A.B. ∙ Venkatraman, E.S. ∙ Lucito, R. ...</div><div><strong>Circular binary segmentation for the analysis of array-based DNA copy number data</strong></div><div><em>Biostatistics.</em> 2004; <strong>5</strong>:557-572</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/biostatistics/kxh008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-3543105225" target="_blank" aria-label="Open Scopus in new tab">Scopus (1458)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15475419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbiostatistics%2Fkxh008&amp;pmid=15475419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0470" role="paragraph">Since not all our tumor samples contained a matching normal sample, we used the normal sample with the highest coverage as the reference sample for all our tumors. We used copy number calls to verify that the epigenetic differences between tumors and normal were not only driven by copy number alterations by introducing the estimated CNV as offsets in the statistical models when doing inferences and making sure that the epigenetic differences were present after we masked genomic regions with CNVs.</div></section><section id="sec-4-5-7"><h4>SNP Analysis</h4><div id="p0475" role="paragraph">We identified 28 colon cancer risk SNPs (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">MacArthur et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>MacArthur, J. ∙ Bowler, E. ∙ Cerezo, M. ...</div><div><strong>The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog)</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:D896-D901</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw1133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85016161935" target="_blank" aria-label="Open Scopus in new tab">Scopus (937)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899670/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw1133&amp;pmid=27899670" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) that coincide with E-P loop enhancer anchors and assigned target genes based on the corresponding promoter contact (<a id="crosref1490" href="#mmc4">Table S4</a>). These looping data confirmed predicted targets, including risk SNPs previously associated with <i>COLCA1/2</i> or <i>TERC</i> expression (<a id="crosref1495" href="#figs2">Figures S2</a>I–S2K; <span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Peltekova et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Peltekova, V.D. ∙ Lemire, M. ∙ Qazi, A.M. ...</div><div><strong>Identification of genes expressed by immune cells of the colon that are regulated by colorectal cancer-associated variants</strong></div><div><em>Int. J. Cancer.</em> 2014; <strong>134</strong>:2330-2341</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ijc.28557" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84895930259" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24154973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fijc.28557&amp;pmid=24154973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). 5 of the 28 risk SNPs were associated with distal genes rather than the nearest promoter (<a id="crosref1505" href="#mmc4">Table S4</a>).</div></section><section id="sec-4-5-8"><h4>Gene expression analysis</h4><section id="sec-4-5-8-1"><h5>Download and processing of transcriptome data</h5><div id="p0480" role="paragraph">RNA-seq processed data from TCGA was downloaded programmatically using TCGAbiolinks (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Colaprico et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Colaprico, A. ∙ Silva, T.C. ∙ Olsen, C. ...</div><div><strong>TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e71</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84966269471" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26704973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1507&amp;pmid=26704973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and recount2 (<span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Collado-Torres et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Collado-Torres, L. ∙ Nellore, A. ∙ Kammers, K. ...</div><div><strong>Reproducible RNA-seq analysis using recount2</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:319-321</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85017363752" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28398307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3838&amp;pmid=28398307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Gene count normalization was done using DESeq2 (<span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (16659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and statistical inference was done using limma-voom (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Law et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Law, C.W. ∙ Chen, Y. ∙ Shi, W. ...</div><div><strong>voom: Precision weights unlock linear model analysis tools for RNA-seq read counts</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:R29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2014-15-2-r29" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84896735766" target="_blank" aria-label="Open Scopus in new tab">Scopus (1883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24485249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2014-15-2-r29&amp;pmid=24485249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genomic ranges operations were done using GenomicRanges infrastructure (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Lawrence et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Lawrence, M. ∙ Huber, W. ∙ Pagès, H. ...</div><div><strong>Software for computing and annotating genomic ranges</strong></div><div><em>PLoS Comput. Biol.</em> 2013; <strong>9</strong>:e1003118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84883368195" target="_blank" aria-label="Open Scopus in new tab">Scopus (1187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23950696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003118&amp;pmid=23950696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). RepeatMasker was used to extract the genomic coordinates of repeat elements (<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Smit et al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Smit, A. ∙ Hubley, R. ∙ Green, P.</div><div><strong>RepeatMasker Open-4.0. 2013–2015</strong></div><div>RepeatMasker, 2015</div><div><a id="interref0115" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.SmitR.HubleyP.GreenRepeatMasker+Open-4.0.+2013%E2%80%9320152015RepeatMaskerhttp%3A%2F%2Fwww.repeatmasker.org" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For each repeat element in the human genome, we calculated the total coverage for each TCGA sample and followed the default recount2 pipeline to estimate scaled read counts. Our in-house RNA-seq data was quantified using salmon (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Patro et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Patro, R. ∙ Duggal, G. ∙ Love, M.I. ...</div><div><strong>Salmon provides fast and bias-aware quantification of transcript expression</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:417-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4197" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85014549629" target="_blank" aria-label="Open Scopus in new tab">Scopus (1602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28263959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4197&amp;pmid=28263959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and differential expression analyses were done as described in Bioconductor’s RNA-seq workflow (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Love et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Love, M.I. ∙ Anders, S. ∙ Kim, V. ...</div><div><strong>RNA-Seq workflow: gene-level exploratory analysis and differential expression</strong></div><div><em>F1000Res.</em> 2015; <strong>4</strong>:1070</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.12688/f1000research.7035.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26674615/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.12688%2Ff1000research.7035.1&amp;pmid=26674615" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section><section id="sec-4-5-9"><h4>Pan-cancer analysis of gene expression associated to hypomethylation</h4><div id="p0485" role="paragraph">To test for association between block level (open sea) DNA methylation and gene expression, we defined a block score as the mean open sea CpG DNA methylation across hypomethylation blocks for each TCGA sample. To minimize confounding correlations driven by stromal cell fractions, only tumors with stromal cell fractions smaller than 60% were considered for further analysis (<span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Thorsson et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Thorsson, V. ∙ Gibbs, D.L. ∙ Brown, S.D. ..., Cancer Genome Atlas Research Network</div><div><strong>The Immune Landscape of Cancer</strong></div><div><em>Immunity.</em> 2019; <strong>51</strong>:411-412</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85070213901" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31433971/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2019.08.004&amp;pmid=31433971" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, the remaining TCGA tumors were grouped into 5 equally-sized bins according to their stromal cell fraction content. A limma-voom model was fitted using as predictors the block scores and the stromal cell fraction bins (i.e., categorical labels) as a blocking factor. This strategy enabled the identification of genes correlated with hypomethylation, while adjusting for potential confounding by stromal cell content. Using this pipeline, we defined genes associated to block hypomethylation in each tumor type. We tested for oncogene enrichment using Fisher’s exact test using published oncogene annotations (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Liu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Liu, Y. ∙ Sun, J. ∙ Zhao, M.</div><div><strong>ONGene: A literature-based database for human oncogenes</strong></div><div><em>J.&nbsp;Genet. Genomics.</em> 2017; <strong>44</strong>:119-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jgg.2016.12.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85011271324" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28162959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jgg.2016.12.004&amp;pmid=28162959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-10"><h4>Exclusion of genes of likely non-tumor cell-origin</h4><div id="p0490" role="paragraph">For downstream analyses, we sought to exclude from analyses those genes whose expression patterns could be driven by differences&nbsp;in stromal or immune cell composition by two filters: 1) We downloaded gene expression data from cell populations purified using fluorescence activated cell sorting (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Calon et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Calon, A. ∙ Lonardo, E. ∙ Berenguer-Llergo, A. ...</div><div><strong>Stromal gene expression defines poor-prognosis subtypes in colorectal cancer</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:320-329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3225" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84925818452" target="_blank" aria-label="Open Scopus in new tab">Scopus (507)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25706628/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3225&amp;pmid=25706628" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and excluded genes where the average expression level was higher (&gt; 2 fold change) in fibroblast cells compared to epithelial cells. 2) We downloaded GTEx blood RNA-Seq data from the recount2 project (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Collado-Torres et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Collado-Torres, L. ∙ Nellore, A. ∙ Kammers, K. ...</div><div><strong>Reproducible RNA-seq analysis using recount2</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:319-321</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85017363752" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28398307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3838&amp;pmid=28398307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and scaled toward a target read count of 1,000,000 reads using the <i>scale_counts</i> function of the <i>recount</i> R/Bioconductor package. We excluded genes where the median scaled count in blood samples was greater than 10.</div></section><section id="sec-4-5-11"><h4>Survival analysis</h4><div id="p0495" role="paragraph">Preprocessed Affymetrix U133Plus2 microarray gene expression data (NCBI GEO <a id="intref0065" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582" target="_blank">GSE39582</a>) was downloaded using the curatedCRCData R/Bioconductor package (<span class="dropBlock reference-citations"><a id="body-ref-sref58-6" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-6" aria-controls="sref58-6">Marisa et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref58-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Eight of 566 samples with low average pairwise correlation (&lt; 0.9) with other samples were excluded in a QC filtering step. Gene expression values were Z-score transformed.</div><div id="p0500" role="paragraph">We computed a gene expression risk score as the average normalized expression level of the 146 genes that are downregulated with hypomethylation in the I and B compartments for samples from <span class="dropBlock reference-citations"><a id="body-ref-sref58-7" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-7" aria-controls="sref58-7">Marisa et&nbsp;al. (2013)</a><div class="dropBlock__holder" data-db-target-of="sref58-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and the TCGA COAD cohort (<span class="dropBlock reference-citations"><a id="body-ref-sref58-8" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-8" aria-controls="sref58-8">Marisa et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref58-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For <span class="dropBlock reference-citations"><a id="body-ref-sref58-9" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-9" aria-controls="sref58-9">Marisa et&nbsp;al. (2013)</a><div class="dropBlock__holder" data-db-target-of="sref58-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, we restricted to the 145 genes where the corresponding gene symbol was present. We defined a “high” score as those values that were 2sd (robustly estimated with the R ‘mad’ function) above the median for the <span class="dropBlock reference-citations"><a id="body-ref-sref58-10" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-10" aria-controls="sref58-10">Marisa et&nbsp;al. (2013)</a><div class="dropBlock__holder" data-db-target-of="sref58-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> study, and 1 sd above the median for the TCGA cohort. We constructed Kaplan-Meier survival curves using available data&nbsp;for recurrence-free survival (<span class="dropBlock reference-citations"><a id="body-ref-sref58-11" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-11" aria-controls="sref58-11">Marisa et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref58-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and overall survival (<span class="dropBlock reference-citations"><a id="body-ref-sref8-5" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-5" aria-controls="sref8-5">Cancer Genome Atlas Network, 2012</a><div class="dropBlock__holder" data-db-target-of="sref8-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cancer Genome Atlas Network</div><div><strong>Comprehensive molecular characterization of human colon and rectal cancer</strong></div><div><em>Nature.</em> 2012; <strong>487</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84863922124" target="_blank" aria-label="Open Scopus in new tab">Scopus (4694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22810696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11252&amp;pmid=22810696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To assess the association of expression risk score with survival outcomes while adjusting for known risk factors we fit a Cox Proportional Hazards model to data from the Marisa et&nbsp;al. study, using gene expression risk score, clinical stage, BRAF mutation status and MSI status as predictors.</div></section><section id="sec-4-5-12"><h4>ChIP-seq analysis</h4><div id="p0505" role="paragraph">Reads were mapped to the reference genome (hg19) using bwa version 0.7.12 (<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (19639)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CTCF peaks were called using GCAPC with default parameters (<span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Teng and Irizarry, 2017</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Teng, M. ∙ Irizarry, R.A.</div><div><strong>Accounting for GC-content bias reduces systematic errors and batch effects in ChIP-seq data</strong></div><div><em>Genome Res.</em> 2017; <strong>27</strong>:1930-1938</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.220673.117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85046699965" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29025895/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.220673.117&amp;pmid=29025895" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and H3K27ac peaks were called using MACS (<span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (6187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The data revealed the expected punctate peaks of the enhancer-associated mark, H3K27ac, as well as broader regions of the repressive modifications, H3K9me3 and H3K27me3. CTCF binding sites were highly enriched for the CTCF binding motif (OR&nbsp;= 19.04; p&nbsp;&lt; 10<sup>−15</sup>). For differential CTCF peak analysis, the union of peaks that were detected in at least two samples were considered and reads were counted for each peak in each sample. Differential CTCF binding was inferred using DESeq2, introducing offsets to the generalized linear model to normalize for library size (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (16659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), copy number differences, and non-linear trends (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Lun and Smyth, 2016</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Lun, A.T.L. ∙ Smyth, G.K.</div><div><strong>csaw: a Bioconductor package for differential binding analysis of ChIP-seq data using sliding windows</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1191" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84963820955" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26578583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1191&amp;pmid=26578583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Copy number estimates for each genomic region were obtained using CNAnorm (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Gusnanto et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Gusnanto, A. ∙ Wood, H.M. ∙ Pawitan, Y. ...</div><div><strong>Correcting for cancer genome size and tumour cell content enables better estimation of copy number alterations from next-generation sequence data</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:40-47</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr593" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84855185866" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22039209/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr593&amp;pmid=22039209" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To account for technical experimental variation, the loadings of the first principal component were introduced as a covariate on the generalized linear model. CTCF sites were considered lost if they had a q-value smaller than 0.1 and a methylation difference larger than 20%. To evaluate the effects of methylation of CTCF sites in chromatin looping, we performed aggregate peak analyses based on a comprehensive list of loops annotated on the human genome (<span class="dropBlock reference-citations"><a id="body-ref-sref72-7" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-7" aria-controls="sref72-7">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref72-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Rao, S.S.P. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Lost CTCF peaks were assigned to loops if they overlapped with loop’s bidirectional CTCF motifs. To assign genes upregulated upon TAD boundary disruption, only peaks within 50Kb of a TAD boundary were considered. For each gene near a disrupted TAD boundary, a linear model was fitted on TCGA data using gene expression as a response variable and the average methylation at the lost CTCF site as predictors. Genes were selected if they had a significant positive (at a false discovery rate of 15%) association between their expression and the methylation levels at the corresponding TAD boundary CT<i>CF.</i></div></section><section id="sec-4-5-13"><h4>DNA polymer modeling</h4><div id="p0510" role="paragraph">To visualize genome organization in normal tissues and primary tumors, we applied a computational approach to derive polymer models and 3D structures for each sample. As illustrated in <a id="crosref1640" href="#fig2">Figure&nbsp;2</a>D, these models were fine-tuned to ensure that simulated in-silico contact maps reproduce the corresponding experimental results. We followed the same protocol to parameterize the models for different samples independently, and any difference between simulated structures, if observed, should reflect the alterations in genome organization detected by Hi-C.</div><div id="p0515" role="paragraph">The polymer models explicitly consider each individual chromosome as a string of beads for structural representation. Only one copy of the two homologs was included since the Hi-C data do not provide allele-specific contacts. We studied the genome organization at two resolutions that represent each bead as either 1Mb or 100kb long genomic segments. The 1Mb-resolution model is computationally efficient and allows us to explore the genome organization in multiple samples. On the other hand, the 100kb-resolution model provides a more detailed representation of the genome and will enable us to characterize the spatial localization of compartment I, introduced in the main text.</div><div id="p0520" role="paragraph">A key innovation of the polymer modeling approach is its use of an ensemble of structures, instead of a single, unique conformation, to reproduce Hi-C data. The ensemble of structures is assumed to follow a Boltzmann distribution with a potential energy function U<sub>ME</sub>(r), the expression of which can be derived following the maximum entropy principle (<span class="dropBlock reference-citations"><a id="body-ref-optmEKNnOQn6J" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optmEKNnOQn6J" aria-controls="optmEKNnOQn6J">Qi et al., 2020</a><div class="dropBlock__holder" data-db-target-of="optmEKNnOQn6J"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Qi, Y. ∙ Reyes, A. ∙ Johnstone, S.E. ...</div><div><strong>Data-driven polymer model for mechanistic exploration of diploid genome organization</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.02.27.968735" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85098899239" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32511383/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.02.27.968735&amp;pmid=32511383" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71-1" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-1" aria-controls="sref71-1">Qi and Zhang, 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Qi, Y. ∙ Zhang, B.</div><div><strong>Predicting three-dimensional genome organization with chromatin states</strong></div><div><em>PLoS Comput. Biol.</em> 2019; <strong>15</strong>:e1007024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85068537423" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31181064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1007024&amp;pmid=31181064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib102 bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Zhang and Wolynes, 2015, 2016</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Zhang, B. ∙ Wolynes, P.G.</div><div><strong>Topology, structures, and energy landscapes of human chromosomes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:6062-6067</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1506257112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84929190426" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25918364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1506257112&amp;pmid=25918364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Zhang, B. ∙ Wolynes, P.G.</div><div><strong>Shape Transitions and Chiral Symmetry Breaking in the Energy Landscape of the Mitotic Chromosome</strong></div><div><em>Phys. Rev. Lett.</em> 2016; <strong>116</strong>:248101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.116.248101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27367409/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.116.248101&amp;pmid=27367409" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and is provided below. Parameters of the polymer model are solely encoded in the energy function, and their values were determined iteratively such that the simulated structures reproduce Hi-C contact maps. Molecular dynamics simulations were carried out to collect structures consistent with the energy function and the Boltzmann distribution.</div></section><section id="sec-4-5-14"><h4>Energy function</h4><div id="p0525" role="paragraph">The potential energy function for the genome adopts the following form:<div id="fd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic upper M upper E Baseline left parenthesis bold italic r right parenthesis equals sigma summation Underscript upper I Endscripts left bracket upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis plus upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis right bracket plus upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis comma" tabindex="0" ctxtmenu_counter="38" style="font-size: 130%;"><mjx-math data-semantic-structure="(52 (51 (49 (2 0 1) 48 (6 3 4 5)) 7 (50 (47 (10 8 9) (35 11 (33 (32 12 31 (18 13 (16 14 15) 17)) 19 (30 (22 20 21) 29 (28 23 (26 24 25) 27))) 34)) 36 (46 (39 37 38) 45 (43 40 41 42)))) 44)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="52" data-semantic-children="51,44" data-semantic-content="44" data-semantic-owns="51 44" data-semantic-speech="upper U Subscript italic upper M upper E Baseline left parenthesis bold italic r right parenthesis equals sigma summation Underscript upper I Endscripts left bracket upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis plus upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis right bracket plus upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="51" data-semantic-children="49,50" data-semantic-content="7" data-semantic-parent="52" data-semantic-owns="49 7 50"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-children="2,6" data-semantic-content="48,0" data-semantic-parent="51" data-semantic-owns="2 48 6"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="49" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D440" noic="true" style="padding-top: 0.683em;">𝑀</mjx-c><mjx-c class="mjx-c1D438" style="padding-top: 0.683em;">𝐸</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="48" data-semantic-parent="49" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="3,5" data-semantic-parent="49" data-semantic-owns="3 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="51" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="50" data-semantic-children="47,46" data-semantic-content="36" data-semantic-parent="51" data-semantic-owns="47 36 46"><mjx-mrow data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="47" data-semantic-children="10,35" data-semantic-content="8" data-semantic-parent="50" data-semantic-owns="10 35"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="47" data-semantic-owns="8 9" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="8" data-semantic-parent="10" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="35" data-semantic-children="33" data-semantic-content="11,34" data-semantic-parent="47" data-semantic-owns="11 33 34"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="11" data-semantic-parent="35" data-semantic-operator="fenced"><mjx-c class="mjx-c5B">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="33" data-semantic-children="32,30" data-semantic-content="19" data-semantic-parent="35" data-semantic-owns="32 19 30"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="32" data-semantic-children="12,18" data-semantic-content="31,12" data-semantic-parent="33" data-semantic-owns="12 31 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="32" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="18" data-semantic-children="16" data-semantic-content="13,17" data-semantic-parent="32" data-semantic-owns="13 16 17"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="13" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="18" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="19" data-semantic-parent="33" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="30" data-semantic-children="22,28" data-semantic-content="29,20" data-semantic-parent="33" data-semantic-owns="22 29 28"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="22" data-semantic-children="20,21" data-semantic-parent="30" data-semantic-owns="20 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="22" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="21" data-semantic-parent="22"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.694em;">𝑒</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D459" style="padding-top: 0.694em;">𝑙</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="29" data-semantic-parent="30" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="28" data-semantic-children="26" data-semantic-content="23,27" data-semantic-parent="30" data-semantic-owns="23 26 27"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="23" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="24,25" data-semantic-parent="28" data-semantic-owns="24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="27" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="34" data-semantic-parent="35" data-semantic-operator="fenced"><mjx-c class="mjx-c5D">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="36" data-semantic-parent="50" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-children="39,43" data-semantic-content="45,37" data-semantic-parent="50" data-semantic-owns="39 45 43"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="39" data-semantic-children="37,38" data-semantic-parent="46" data-semantic-owns="37 38"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="39" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="38" data-semantic-parent="39"><mjx-c class="mjx-c1D450" noic="true" style="padding-top: 0.626em;">𝑐</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.626em;">𝑜</mjx-c><mjx-c class="mjx-c1D45A" noic="true" style="padding-top: 0.626em;">𝑚</mjx-c><mjx-c class="mjx-c1D45D" noic="true" style="padding-top: 0.626em;">𝑝</mjx-c><mjx-c class="mjx-c1D461" style="padding-top: 0.626em;">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="43" data-semantic-children="41" data-semantic-content="40,42" data-semantic-parent="46" data-semantic-owns="40 41 42"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="40" data-semantic-parent="43" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="43"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="42" data-semantic-parent="43" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="44" data-semantic-parent="52" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">[1]</div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="bold italic r" tabindex="0" ctxtmenu_counter="39" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="bold italic r"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi></mjx-math></mjx-container> represents the 3D conformation of the entire genome. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I" tabindex="0" ctxtmenu_counter="40" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper I"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-math></mjx-container> indexes over different chromosomes and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="bold italic r Subscript upper I" tabindex="0" ctxtmenu_counter="41" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="bold italic r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> corresponds to the conformation of chromosome <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I" tabindex="0" ctxtmenu_counter="42" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper I"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-math></mjx-container>. By definition, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="bold italic r equals StartSet bold italic r 1 comma bold italic r 2 comma midline horizontal ellipsis comma bold italic r 23 EndSet period" tabindex="0" ctxtmenu_counter="43" style="font-size: 130%;"><mjx-math data-semantic-structure="(22 (21 0 1 (19 2 (17 (5 3 4) 6 (9 7 8) 10 11 12 (16 14 15)) 18)) 20)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="22" data-semantic-children="21,20" data-semantic-content="20" data-semantic-owns="21 20" data-semantic-speech="bold italic r equals StartSet bold italic r 1 comma bold italic r 2 comma midline horizontal ellipsis comma bold italic r 23 EndSet period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="21" data-semantic-children="0,19" data-semantic-content="1" data-semantic-parent="22" data-semantic-owns="0 1 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="21"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="21" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="19" data-semantic-children="17" data-semantic-content="2,18" data-semantic-parent="21" data-semantic-owns="2 17 18"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="19" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="17" data-semantic-children="5,6,9,10,11,12,16" data-semantic-content="6,10,12" data-semantic-parent="19" data-semantic-owns="5 6 9 10 11 12 16"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="17" data-semantic-owns="3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="17" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="17" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="10" data-semantic-parent="17" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="11" data-semantic-parent="17"><mjx-c class="mjx-c22EF">⋯</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="17" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="17" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.666em;">3</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="18" data-semantic-parent="19" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="20" data-semantic-parent="22" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="44" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 7 (6 1 (4 2 3) 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="8" data-semantic-children="0,6" data-semantic-content="7,0" data-semantic-owns="0 7 6" data-semantic-speech="upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="1,5" data-semantic-parent="8" data-semantic-owns="1 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="6" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="45" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (2 0 1) 9 (8 3 (6 4 5) 7))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="10" data-semantic-children="2,8" data-semantic-content="9,0" data-semantic-owns="2 9 8" data-semantic-speech="upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="10" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.694em;">𝑒</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D459" style="padding-top: 0.694em;">𝑙</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="3,7" data-semantic-parent="10" data-semantic-owns="3 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="8" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> are generic potentials shared by all chromosomes, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis" tabindex="0" ctxtmenu_counter="46" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (2 0 1) 7 (6 3 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-children="2,6" data-semantic-content="7,0" data-semantic-owns="2 7 6" data-semantic-speech="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="8" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450" noic="true" style="padding-top: 0.626em;">𝑐</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.626em;">𝑜</mjx-c><mjx-c class="mjx-c1D45A" noic="true" style="padding-top: 0.626em;">𝑚</mjx-c><mjx-c class="mjx-c1D45D" noic="true" style="padding-top: 0.626em;">𝑝</mjx-c><mjx-c class="mjx-c1D461" style="padding-top: 0.626em;">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="3,5" data-semantic-parent="8" data-semantic-owns="3 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> describes compartment-type-specific interactions within the same chromosome and between different chromosomes.</div><div id="p0530" role="paragraph">Specifically, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="47" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 7 (6 1 (4 2 3) 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="8" data-semantic-children="0,6" data-semantic-content="7,0" data-semantic-owns="0 7 6" data-semantic-speech="upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="1,5" data-semantic-parent="8" data-semantic-owns="1 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="6" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> is the energy function for a confined homopolymer and consists of four terms, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic b o n d" tabindex="0" ctxtmenu_counter="48" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript italic b o n d"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D44F" noic="true" style="padding-top: 0.694em;">𝑏</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.694em;">𝑜</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.694em;">𝑛</mjx-c><mjx-c class="mjx-c1D451" style="padding-top: 0.694em;">𝑑</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic a n g l e" tabindex="0" ctxtmenu_counter="49" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript italic a n g l e"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.694em;">𝑛</mjx-c><mjx-c class="mjx-c1D454" noic="true" style="padding-top: 0.694em;">𝑔</mjx-c><mjx-c class="mjx-c1D459" noic="true" style="padding-top: 0.694em;">𝑙</mjx-c><mjx-c class="mjx-c1D452" style="padding-top: 0.694em;">𝑒</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic s c" tabindex="0" ctxtmenu_counter="50" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript italic s c"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D460" noic="true" style="padding-top: 0.442em;">𝑠</mjx-c><mjx-c class="mjx-c1D450" style="padding-top: 0.442em;">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript c" tabindex="0" ctxtmenu_counter="51" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript c"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic b o n d" tabindex="0" ctxtmenu_counter="52" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript italic b o n d"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D44F" noic="true" style="padding-top: 0.694em;">𝑏</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.694em;">𝑜</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.694em;">𝑛</mjx-c><mjx-c class="mjx-c1D451" style="padding-top: 0.694em;">𝑑</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the bonding potential between neighboring beads. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic a n g l e" tabindex="0" ctxtmenu_counter="53" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript italic a n g l e"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.694em;">𝑛</mjx-c><mjx-c class="mjx-c1D454" noic="true" style="padding-top: 0.694em;">𝑔</mjx-c><mjx-c class="mjx-c1D459" noic="true" style="padding-top: 0.694em;">𝑙</mjx-c><mjx-c class="mjx-c1D452" style="padding-top: 0.694em;">𝑒</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the angular potential applied among every three neighboring beads to define the persistence length of the polymer. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic s c" tabindex="0" ctxtmenu_counter="54" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript italic s c"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D460" noic="true" style="padding-top: 0.442em;">𝑠</mjx-c><mjx-c class="mjx-c1D450" style="padding-top: 0.442em;">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is a soft-core potential applied to all the non-bonded pairs to enforce the excluded volume effect among genomic loci. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript c" tabindex="0" ctxtmenu_counter="55" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript c"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> models a spherical boundary and is introduced to mimic the confinement effect applied by the nuclear envelop onto the chromosomes. The radii of the spherical confinement is chosen to ensure a volume fraction of 0.1. Explicit expressions for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="56" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 7 (6 1 (4 2 3) 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="8" data-semantic-children="0,6" data-semantic-content="7,0" data-semantic-owns="0 7 6" data-semantic-speech="upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="1,5" data-semantic-parent="8" data-semantic-owns="1 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="6" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> can be found in <span class="dropBlock reference-citations"><a id="body-ref-sref102-2" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-2" aria-controls="sref102-2">Zhang and Wolynes, 2015</a><div class="dropBlock__holder" data-db-target-of="sref102-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Zhang, B. ∙ Wolynes, P.G.</div><div><strong>Topology, structures, and energy landscapes of human chromosomes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:6062-6067</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1506257112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84929190426" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25918364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1506257112&amp;pmid=25918364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and <span class="dropBlock reference-citations"><a id="body-ref-sref71-2" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-2" aria-controls="sref71-2">Qi and Zhang, 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Qi, Y. ∙ Zhang, B.</div><div><strong>Predicting three-dimensional genome organization with chromatin states</strong></div><div><em>PLoS Comput. Biol.</em> 2019; <strong>15</strong>:e1007024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85068537423" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31181064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1007024&amp;pmid=31181064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div><div id="p0535" role="paragraph"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="57" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (2 0 1) 9 (8 3 (6 4 5) 7))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="10" data-semantic-children="2,8" data-semantic-content="9,0" data-semantic-owns="2 9 8" data-semantic-speech="upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="10" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.694em;">𝑒</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D459" style="padding-top: 0.694em;">𝑙</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="3,7" data-semantic-parent="10" data-semantic-owns="3 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="8" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> is introduced to reproduce the power law decay of the contact probability as a function of genomic separation for each chromosome (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Di Pierro et al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Di Pierro, M. ∙ Zhang, B. ∙ Aiden, E.L. ...</div><div><strong>Transferable model for chromosome architecture</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:12168-12173</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1613607113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84992395910" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27688758/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1613607113&amp;pmid=27688758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71-3" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-3" aria-controls="sref71-3">Qi and Zhang, 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Qi, Y. ∙ Zhang, B.</div><div><strong>Predicting three-dimensional genome organization with chromatin states</strong></div><div><em>PLoS Comput. Biol.</em> 2019; <strong>15</strong>:e1007024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85068537423" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31181064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1007024&amp;pmid=31181064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It describes the tendency for chromosomes to collapse and form territories in addition to what has been enforced by the confinement potential <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript c" tabindex="0" ctxtmenu_counter="58" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper U Subscript c"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="59" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (2 0 1) 9 (8 3 (6 4 5) 7))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="10" data-semantic-children="2,8" data-semantic-content="9,0" data-semantic-owns="2 9 8" data-semantic-speech="upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="10" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.694em;">𝑒</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D459" style="padding-top: 0.694em;">𝑙</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="3,7" data-semantic-parent="10" data-semantic-owns="3 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="8" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> is defined as<div id="fd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis equals sigma summation Underscript i comma j element of upper I Endscripts alpha Subscript italic i d e a l Baseline left parenthesis StartAbsoluteValue j minus i EndAbsoluteValue right parenthesis f left parenthesis r Subscript i j Baseline right parenthesis comma" tabindex="0" ctxtmenu_counter="60" style="font-size: 130%;"><mjx-math data-semantic-structure="(53 (52 (51 (2 0 1) 50 (8 3 (6 4 5) 7)) 9 (49 (18 10 (17 11 12 (16 13 14 15))) (48 (46 (21 19 20) 45 (31 22 (29 23 (27 24 25 26) 28) 30)) 47 (44 32 43 (41 33 (39 34 (38 35 37 36)) 40))))) 42)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="53" data-semantic-children="52,42" data-semantic-content="42" data-semantic-owns="52 42" data-semantic-speech="upper U Subscript italic i d e a l Baseline left parenthesis bold italic r Subscript upper I Baseline right parenthesis equals sigma summation Underscript i comma j element of upper I Endscripts alpha Subscript italic i d e a l Baseline left parenthesis StartAbsoluteValue j minus i EndAbsoluteValue right parenthesis f left parenthesis r Subscript i j Baseline right parenthesis comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="52" data-semantic-children="51,49" data-semantic-content="9" data-semantic-parent="53" data-semantic-owns="51 9 49"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="51" data-semantic-children="2,8" data-semantic-content="50,0" data-semantic-parent="52" data-semantic-owns="2 50 8"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="51" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.694em;">𝑒</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D459" style="padding-top: 0.694em;">𝑙</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="50" data-semantic-parent="51" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="3,7" data-semantic-parent="51" data-semantic-owns="3 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="8" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="52" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="49" data-semantic-children="18,48" data-semantic-content="10" data-semantic-parent="52" data-semantic-owns="18 48"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="18" data-semantic-children="10,17" data-semantic-parent="49" data-semantic-owns="10 17" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="10" data-semantic-parent="18" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="17" data-semantic-children="11,12,16" data-semantic-content="12" data-semantic-parent="18" data-semantic-owns="11 12 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="17"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="17" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="16" data-semantic-children="13,15" data-semantic-content="14" data-semantic-parent="17" data-semantic-owns="13 14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="16"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="14" data-semantic-parent="16" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="48" data-semantic-children="46,44" data-semantic-content="47" data-semantic-parent="49" data-semantic-owns="46 47 44"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="46" data-semantic-children="21,31" data-semantic-content="45,19" data-semantic-parent="48" data-semantic-owns="21 45 31"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="21" data-semantic-children="19,20" data-semantic-parent="46" data-semantic-owns="19 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.694em;">𝑒</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D459" style="padding-top: 0.694em;">𝑙</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="31" data-semantic-children="29" data-semantic-content="22,30" data-semantic-parent="46" data-semantic-owns="22 29 30"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="22" data-semantic-parent="31" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="29" data-semantic-children="27" data-semantic-content="23,28" data-semantic-parent="31" data-semantic-owns="23 27 28"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="23" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="27" data-semantic-children="24,26" data-semantic-content="25" data-semantic-parent="29" data-semantic-owns="24 25 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="27"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="25" data-semantic-parent="27" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="27"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="28" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="30" data-semantic-parent="31" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="47" data-semantic-parent="48" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="44" data-semantic-children="32,41" data-semantic-content="43,32" data-semantic-parent="48" data-semantic-owns="32 43 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="44" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="41" data-semantic-children="39" data-semantic-content="33,40" data-semantic-parent="44" data-semantic-owns="33 39 40"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="33" data-semantic-parent="41" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="39" data-semantic-children="34,38" data-semantic-parent="41" data-semantic-owns="34 38"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="39"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="38" data-semantic-children="35,36" data-semantic-content="37" data-semantic-parent="39" data-semantic-owns="35 37 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="38"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="37" data-semantic-parent="38" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="38"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="40" data-semantic-parent="41" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="42" data-semantic-parent="53" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">[2]</div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f left parenthesis r Subscript i j Baseline right parenthesis" tabindex="0" ctxtmenu_counter="61" style="font-size: 130%;"><mjx-math data-semantic-structure="(11 0 10 (9 1 (7 2 (6 3 5 4)) 8))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="11" data-semantic-children="0,9" data-semantic-content="10,0" data-semantic-owns="0 10 9" data-semantic-speech="f left parenthesis r Subscript i j Baseline right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="11" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="7" data-semantic-content="1,8" data-semantic-parent="11" data-semantic-owns="1 7 8"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="2,6" data-semantic-parent="9" data-semantic-owns="2 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="7" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> determines the contact probability of a genomic pair with a spatial distance of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript i j" tabindex="0" ctxtmenu_counter="62" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="r Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i comma j" tabindex="0" ctxtmenu_counter="63" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="3" data-semantic-children="0,1,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="i comma j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> index over all pairs of non-bonded chromatin beads from chromosome <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I" tabindex="0" ctxtmenu_counter="64" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper I"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-math></mjx-container>. Following <span class="dropBlock reference-citations"><a id="body-ref-sref71-4" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-4" aria-controls="sref71-4">Qi and Zhang (2019)</a><div class="dropBlock__holder" data-db-target-of="sref71-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Qi, Y. ∙ Zhang, B.</div><div><strong>Predicting three-dimensional genome organization with chromatin states</strong></div><div><em>PLoS Comput. Biol.</em> 2019; <strong>15</strong>:e1007024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85068537423" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31181064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1007024&amp;pmid=31181064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, we define <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f left parenthesis r right parenthesis" tabindex="0" ctxtmenu_counter="65" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 5 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-children="0,4" data-semantic-content="5,0" data-semantic-owns="0 5 4" data-semantic-speech="f left parenthesis r right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> as<div id="fd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f left parenthesis r right parenthesis equals StartLayout Enlarged left brace 1st Row  one half left bracket 1 plus italic hyperbolic tangent left parenthesis sigma left parenthesis r Subscript c Baseline minus r right parenthesis right parenthesis right bracket comma i f r less than or equals r Subscript c Baseline 2nd Row  one half left parenthesis StartFraction r Subscript c Baseline Over r EndFraction right parenthesis Superscript 4 Baseline comma i f r greater than r Subscript c Baseline EndLayout" tabindex="0" ctxtmenu_counter="66" style="font-size: 130%;"><mjx-math data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="93" data-semantic-children="92,90" data-semantic-content="5" data-semantic-owns="92 5 90" data-semantic-structure="(93 (92 0 91 (4 2 1 3)) 5 (90 (54 (52 (46 (9 7 8) 45 (33 31 (30 10 11 (29 12 28 (27 25 (24 13 23 (22 20 (19 (16 14 15) 17 18) 21)) 26))) 32)) 34 (51 (50 36 49 (48 37 47 39)) 40 (43 41 42)))) (89 (87 (81 (57 55 56) 80 (67 (65 63 (62 (60 58 59) 61) 64) 66)) 68 (86 (85 72 84 (83 73 82 75)) 76 (79 77 78))))))" data-semantic-speech="f left parenthesis r right parenthesis equals StartLayout Enlarged left brace 1st Row  one half left bracket 1 plus italic hyperbolic tangent left parenthesis sigma left parenthesis r Subscript c Baseline minus r right parenthesis right parenthesis right bracket comma i f r less than or equals r Subscript c Baseline 2nd Row  one half left parenthesis StartFraction r Subscript c Baseline Over r EndFraction right parenthesis Superscript 4 Baseline comma i f r greater than r Subscript c Baseline EndLayout" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="92" data-semantic-children="0,4" data-semantic-content="91,0" data-semantic-parent="93" data-semantic-owns="0 91 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="92" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="91" data-semantic-parent="92" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="1" data-semantic-content="2,3" data-semantic-parent="92" data-semantic-owns="2 1 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="93" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="cases" data-semantic-role="binomial" data-semantic-id="90" data-semantic-children="54,89" data-semantic-content="6" data-semantic-parent="93" data-semantic-owns="54 89"><mjx-mo data-semantic-type="punctuation" data-semantic-role="openfence" data-semantic-id="6" data-semantic-parent="90"><mjx-c class="mjx-c7B MM-S7">{</mjx-c></mjx-mo><mjx-mtable justify="left" style="min-width: 14.995em;"><mjx-table><mjx-itable><mjx-mtr data-semantic-type="line" data-semantic-role="binomial" data-semantic-id="54" data-semantic-children="52" data-semantic-parent="90" data-semantic-owns="52"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="52" data-semantic-children="46,34,51" data-semantic-content="34" data-semantic-parent="54" data-semantic-owns="46 34 51"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="46" data-semantic-children="9,33" data-semantic-content="45" data-semantic-parent="52" data-semantic-owns="9 45 33"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="vulgar" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="46" data-semantic-owns="7 8"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="33" data-semantic-children="30" data-semantic-content="31,32" data-semantic-parent="46" data-semantic-owns="31 30 32"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="31" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c5B">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="30" data-semantic-children="10,29" data-semantic-content="11" data-semantic-parent="33" data-semantic-owns="10 11 29"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="30"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="11" data-semantic-parent="30" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="29" data-semantic-children="12,27" data-semantic-content="28,12" data-semantic-parent="30" data-semantic-owns="12 28 27"><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="italic" data-semantic-id="12" data-semantic-parent="29" data-semantic-operator="appl"><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.694em;">𝑡</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.694em;">𝑛</mjx-c><mjx-c class="mjx-c210E" style="padding-top: 0.694em;">ℎ</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="27" data-semantic-children="24" data-semantic-content="25,26" data-semantic-parent="29" data-semantic-owns="25 24 26"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="25" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-children="13,22" data-semantic-content="23" data-semantic-parent="27" data-semantic-owns="13 23 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="24"><mjx-c class="mjx-c1D70E">𝜎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="22" data-semantic-children="19" data-semantic-content="20,21" data-semantic-parent="24" data-semantic-owns="20 19 21"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="20" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="19" data-semantic-children="16,18" data-semantic-content="17" data-semantic-parent="22" data-semantic-owns="16 17 18"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="19" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="17" data-semantic-parent="19" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="32" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c5D">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="34" data-semantic-parent="52" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="51" data-semantic-children="50,43" data-semantic-content="40" data-semantic-parent="52" data-semantic-owns="50 40 43"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="50" data-semantic-children="36,48" data-semantic-content="49" data-semantic-parent="51" data-semantic-owns="36 49 48"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="50"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="49" data-semantic-parent="50" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-children="37,39" data-semantic-content="47,37" data-semantic-parent="50" data-semantic-owns="37 47 39"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="48" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="47" data-semantic-parent="48" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="48"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="40" data-semantic-parent="51" data-semantic-operator="relseq,≤"><mjx-c class="mjx-c2264">≤</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="43" data-semantic-children="41,42" data-semantic-parent="51" data-semantic-owns="41 42"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="43"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="43"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="binomial" data-semantic-id="89" data-semantic-children="87" data-semantic-parent="90" data-semantic-owns="87"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="87" data-semantic-children="81,68,86" data-semantic-content="68" data-semantic-parent="89" data-semantic-owns="81 68 86"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="81" data-semantic-children="57,67" data-semantic-content="80" data-semantic-parent="87" data-semantic-owns="57 80 67"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="vulgar" data-semantic-annotation="clearspeak:simple" data-semantic-id="57" data-semantic-children="55,56" data-semantic-parent="81" data-semantic-owns="55 56"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="55" data-semantic-parent="57"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="56" data-semantic-parent="57"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="80" data-semantic-parent="81" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="67" data-semantic-children="65,66" data-semantic-parent="81" data-semantic-owns="65 66"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="65" data-semantic-children="62" data-semantic-content="63,64" data-semantic-parent="67" data-semantic-owns="63 62 64"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="63" data-semantic-parent="65" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="62" data-semantic-children="60,61" data-semantic-parent="65" data-semantic-owns="60 61"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="60" data-semantic-children="58,59" data-semantic-parent="62" data-semantic-owns="58 59"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="58" data-semantic-parent="60"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="59" data-semantic-parent="60"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="62"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="64" data-semantic-parent="65" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.524em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="66" data-semantic-parent="67"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="68" data-semantic-parent="87" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="86" data-semantic-children="85,79" data-semantic-content="76" data-semantic-parent="87" data-semantic-owns="85 76 79"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="85" data-semantic-children="72,83" data-semantic-content="84" data-semantic-parent="86" data-semantic-owns="72 84 83"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="85"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="84" data-semantic-parent="85" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="83" data-semantic-children="73,75" data-semantic-content="82,73" data-semantic-parent="85" data-semantic-owns="73 82 75"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="73" data-semantic-parent="83" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="82" data-semantic-parent="83" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="75" data-semantic-parent="83"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="76" data-semantic-parent="86" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="79" data-semantic-children="77,78" data-semantic-parent="86" data-semantic-owns="77 78"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="77" data-semantic-parent="79"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="78" data-semantic-parent="79"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr></mjx-itable></mjx-table></mjx-mtable></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">[3]</div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript c Baseline equals 2 period 0" tabindex="0" ctxtmenu_counter="67" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (7 (2 0 1) 3 4) 5 6)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="8" data-semantic-children="7,5,6" data-semantic-content="5" data-semantic-owns="7 5 6" data-semantic-speech="r Subscript c Baseline equals 2 period 0"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="7" data-semantic-children="2,4" data-semantic-content="3" data-semantic-parent="8" data-semantic-owns="2 3 4"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="7" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="7" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="5" data-semantic-parent="8" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="sigma equals 2 period 0" tabindex="0" ctxtmenu_counter="68" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="sigma equals 2 period 0"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D70E">𝜎</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript italic i d e a l Baseline left parenthesis StartAbsoluteValue i minus j EndAbsoluteValue right parenthesis" tabindex="0" ctxtmenu_counter="69" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (2 0 1) 13 (12 3 (10 4 (8 5 6 7) 9) 11))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="14" data-semantic-children="2,12" data-semantic-content="13,0" data-semantic-owns="2 13 12" data-semantic-speech="alpha Subscript italic i d e a l Baseline left parenthesis StartAbsoluteValue i minus j EndAbsoluteValue right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="14" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.694em;">𝑒</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D459" style="padding-top: 0.694em;">𝑙</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="3,11" data-semantic-parent="14" data-semantic-owns="3 10 11"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="10" data-semantic-children="8" data-semantic-content="4,9" data-semantic-parent="12" data-semantic-owns="4 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="4" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-parent="10" data-semantic-owns="5 6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> measures of the strength of the contact at a given genomic separation <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartAbsoluteValue i minus j EndAbsoluteValue" tabindex="0" ctxtmenu_counter="70" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 (4 1 2 3) 5)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="6" data-semantic-children="4" data-semantic-content="0,5" data-semantic-owns="0 4 5" data-semantic-speech="StartAbsoluteValue i minus j EndAbsoluteValue"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and its value can be determined from Hi-C data as detailed below. It contributes to a total of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N minus 1" tabindex="0" ctxtmenu_counter="71" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper N minus 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> parameters, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N" tabindex="0" ctxtmenu_counter="72" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper N"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-math></mjx-container> is the number of beads for the longest chromosome (chromosome 1, 249Mb).</div><div id="p0540" role="paragraph">For the 1Mb-resolution model, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis" tabindex="0" ctxtmenu_counter="73" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (2 0 1) 7 (6 3 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-children="2,6" data-semantic-content="7,0" data-semantic-owns="2 7 6" data-semantic-speech="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="8" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450" noic="true" style="padding-top: 0.626em;">𝑐</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.626em;">𝑜</mjx-c><mjx-c class="mjx-c1D45A" noic="true" style="padding-top: 0.626em;">𝑚</mjx-c><mjx-c class="mjx-c1D45D" noic="true" style="padding-top: 0.626em;">𝑝</mjx-c><mjx-c class="mjx-c1D461" style="padding-top: 0.626em;">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="3,5" data-semantic-parent="8" data-semantic-owns="3 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> is defined as<div id="fd4" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis equals sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts alpha Subscript italic i n t r a Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper I Baseline right parenthesis f left parenthesis r Subscript i j Baseline right parenthesis plus sigma summation Underscript upper I comma upper J Endscripts sigma summation Underscript i element of upper I comma j element of upper J Endscripts alpha Subscript italic i n t e r Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper J Baseline right parenthesis f left parenthesis r Subscript i j Baseline right parenthesis comma" tabindex="0" ctxtmenu_counter="74" style="font-size: 130%;"><mjx-math data-semantic-structure="(113 (112 (110 (2 0 1) 109 (6 3 4 5)) 7 (111 (108 (10 8 9) (107 (16 11 (15 12 13 14)) (106 (104 (19 17 18) 103 (34 20 (33 (25 (24 21 22) 23) 26 (31 (30 27 28) 29)) 32)) 105 (102 35 101 (44 36 (42 37 (41 38 40 39)) 43))))) 45 (100 (51 46 (50 47 48 49)) (99 (63 52 (62 (60 53 54 55) 56 (61 57 58 59))) (98 (96 (66 64 65) 95 (81 67 (80 (72 (71 68 69) 70) 73 (78 (77 74 75) 76)) 79)) 97 (94 82 93 (91 83 (89 84 (88 85 87 86)) 90))))))) 92)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="113" data-semantic-children="112,92" data-semantic-content="92" data-semantic-owns="112 92" data-semantic-speech="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis equals sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts alpha Subscript italic i n t r a Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper I Baseline right parenthesis f left parenthesis r Subscript i j Baseline right parenthesis plus sigma summation Underscript upper I comma upper J Endscripts sigma summation Underscript i element of upper I comma j element of upper J Endscripts alpha Subscript italic i n t e r Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper J Baseline right parenthesis f left parenthesis r Subscript i j Baseline right parenthesis comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="112" data-semantic-children="110,111" data-semantic-content="7" data-semantic-parent="113" data-semantic-owns="110 7 111"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="110" data-semantic-children="2,6" data-semantic-content="109,0" data-semantic-parent="112" data-semantic-owns="2 109 6"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="110" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450" noic="true" style="padding-top: 0.626em;">𝑐</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.626em;">𝑜</mjx-c><mjx-c class="mjx-c1D45A" noic="true" style="padding-top: 0.626em;">𝑚</mjx-c><mjx-c class="mjx-c1D45D" noic="true" style="padding-top: 0.626em;">𝑝</mjx-c><mjx-c class="mjx-c1D461" style="padding-top: 0.626em;">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="109" data-semantic-parent="110" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="3,5" data-semantic-parent="110" data-semantic-owns="3 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="112" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="111" data-semantic-children="108,100" data-semantic-content="45" data-semantic-parent="112" data-semantic-owns="108 45 100"><mjx-mrow data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="108" data-semantic-children="10,107" data-semantic-content="8" data-semantic-parent="111" data-semantic-owns="10 107"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="108" data-semantic-owns="8 9" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="8" data-semantic-parent="10" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="107" data-semantic-children="16,106" data-semantic-content="11" data-semantic-parent="108" data-semantic-owns="16 106"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="16" data-semantic-children="11,15" data-semantic-parent="107" data-semantic-owns="11 15" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="11" data-semantic-parent="16" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="15" data-semantic-children="12,13,14" data-semantic-content="13" data-semantic-parent="16" data-semantic-owns="12 13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="13" data-semantic-parent="15" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="106" data-semantic-children="104,102" data-semantic-content="105" data-semantic-parent="107" data-semantic-owns="104 105 102"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="104" data-semantic-children="19,34" data-semantic-content="103,17" data-semantic-parent="106" data-semantic-owns="19 103 34"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="19" data-semantic-children="17,18" data-semantic-parent="104" data-semantic-owns="17 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D45F" noic="true" style="padding-top: 0.661em;">𝑟</mjx-c><mjx-c class="mjx-c1D44E" style="padding-top: 0.661em;">𝑎</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="103" data-semantic-parent="104" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="34" data-semantic-children="33" data-semantic-content="20,32" data-semantic-parent="104" data-semantic-owns="20 33 32"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="20" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="33" data-semantic-children="25,26,31" data-semantic-content="26" data-semantic-parent="34" data-semantic-owns="25 26 31"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="25" data-semantic-children="21,22,23" data-semantic-parent="33" data-semantic-collapsed="(25 (24 21 22) 23)" data-semantic-owns="21 22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="25"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="25"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="26" data-semantic-parent="33" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="31" data-semantic-children="27,28,29" data-semantic-parent="33" data-semantic-collapsed="(31 (30 27 28) 29)" data-semantic-owns="27 28 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="31"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="31"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="32" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="105" data-semantic-parent="106" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="102" data-semantic-children="35,44" data-semantic-content="101,35" data-semantic-parent="106" data-semantic-owns="35 101 44"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="102" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="101" data-semantic-parent="102" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="44" data-semantic-children="42" data-semantic-content="36,43" data-semantic-parent="102" data-semantic-owns="36 42 43"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="36" data-semantic-parent="44" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="42" data-semantic-children="37,41" data-semantic-parent="44" data-semantic-owns="37 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="42"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="41" data-semantic-children="38,39" data-semantic-content="40" data-semantic-parent="42" data-semantic-owns="38 40 39"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="41"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="40" data-semantic-parent="41" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="41"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="43" data-semantic-parent="44" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="45" data-semantic-parent="111" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="100" data-semantic-children="51,99" data-semantic-content="46" data-semantic-parent="111" data-semantic-owns="51 99"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="51" data-semantic-children="46,50" data-semantic-parent="100" data-semantic-owns="46 50" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="46" data-semantic-parent="51" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="50" data-semantic-children="47,48,49" data-semantic-content="48" data-semantic-parent="51" data-semantic-owns="47 48 49"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="50"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="48" data-semantic-parent="50" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="50"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="99" data-semantic-children="63,98" data-semantic-content="52" data-semantic-parent="100" data-semantic-owns="63 98"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="63" data-semantic-children="52,62" data-semantic-parent="99" data-semantic-owns="52 62" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="52" data-semantic-parent="63" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="62" data-semantic-children="60,56,61" data-semantic-content="56" data-semantic-parent="63" data-semantic-owns="60 56 61"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="60" data-semantic-children="53,55" data-semantic-content="54" data-semantic-parent="62" data-semantic-owns="53 54 55"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="60"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="54" data-semantic-parent="60" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="55" data-semantic-parent="60"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="56" data-semantic-parent="62" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="61" data-semantic-children="57,59" data-semantic-content="58" data-semantic-parent="62" data-semantic-owns="57 58 59"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="57" data-semantic-parent="61"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="58" data-semantic-parent="61" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="59" data-semantic-parent="61"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="98" data-semantic-children="96,94" data-semantic-content="97" data-semantic-parent="99" data-semantic-owns="96 97 94"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="96" data-semantic-children="66,81" data-semantic-content="95,64" data-semantic-parent="98" data-semantic-owns="66 95 81"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="66" data-semantic-children="64,65" data-semantic-parent="96" data-semantic-owns="64 65"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="66" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="65" data-semantic-parent="66"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.661em;">𝑒</mjx-c><mjx-c class="mjx-c1D45F" style="padding-top: 0.661em;">𝑟</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="95" data-semantic-parent="96" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="81" data-semantic-children="80" data-semantic-content="67,79" data-semantic-parent="96" data-semantic-owns="67 80 79"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="67" data-semantic-parent="81" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="80" data-semantic-children="72,73,78" data-semantic-content="73" data-semantic-parent="81" data-semantic-owns="72 73 78"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="72" data-semantic-children="68,69,70" data-semantic-parent="80" data-semantic-collapsed="(72 (71 68 69) 70)" data-semantic-owns="68 69 70"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="68" data-semantic-parent="72"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="70" data-semantic-parent="72" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="69" data-semantic-parent="72"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="73" data-semantic-parent="80" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="78" data-semantic-children="74,75,76" data-semantic-parent="80" data-semantic-collapsed="(78 (77 74 75) 76)" data-semantic-owns="74 75 76"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="74" data-semantic-parent="78"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="76" data-semantic-parent="78" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="75" data-semantic-parent="78"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="79" data-semantic-parent="81" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="97" data-semantic-parent="98" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="94" data-semantic-children="82,91" data-semantic-content="93,82" data-semantic-parent="98" data-semantic-owns="82 93 91"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="82" data-semantic-parent="94" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="93" data-semantic-parent="94" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="91" data-semantic-children="89" data-semantic-content="83,90" data-semantic-parent="94" data-semantic-owns="83 89 90"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="83" data-semantic-parent="91" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="89" data-semantic-children="84,88" data-semantic-parent="91" data-semantic-owns="84 88"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="84" data-semantic-parent="89"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="88" data-semantic-children="85,86" data-semantic-content="87" data-semantic-parent="89" data-semantic-owns="85 87 86"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="85" data-semantic-parent="88"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="87" data-semantic-parent="88" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="86" data-semantic-parent="88"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="90" data-semantic-parent="91" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="92" data-semantic-parent="113" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">[4]</div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I" tabindex="0" ctxtmenu_counter="75" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper I"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper J" tabindex="0" ctxtmenu_counter="76" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper J"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-math></mjx-container> index over different chromosomes and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="77" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="j" tabindex="0" ctxtmenu_counter="78" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="j"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-math></mjx-container> index over non-bonded pairs of chromatin beads. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper C Subscript i Superscript upper I" tabindex="0" ctxtmenu_counter="79" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="upper C Subscript i Superscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> denotes the compartment type for bead <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="80" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-math></mjx-container> from chromosome <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I" tabindex="0" ctxtmenu_counter="81" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper I"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-math></mjx-container> and can be either <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A" tabindex="0" ctxtmenu_counter="82" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper A"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-math></mjx-container> or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper B" tabindex="0" ctxtmenu_counter="83" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper B"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-math></mjx-container>. We used different parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript italic i n t r a" tabindex="0" ctxtmenu_counter="84" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="alpha Subscript italic i n t r a"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D45F" noic="true" style="padding-top: 0.661em;">𝑟</mjx-c><mjx-c class="mjx-c1D44E" style="padding-top: 0.661em;">𝑎</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript italic i n t e r" tabindex="0" ctxtmenu_counter="85" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="alpha Subscript italic i n t e r"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.661em;">𝑒</mjx-c><mjx-c class="mjx-c1D45F" style="padding-top: 0.661em;">𝑟</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for intra- and inter-chromosome interactions to account for the presence of different molecular players that organize the genome at various lengthscales (<span class="dropBlock reference-citations"><a id="body-ref-sref71-5" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-5" aria-controls="sref71-5">Qi and Zhang, 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Qi, Y. ∙ Zhang, B.</div><div><strong>Predicting three-dimensional genome organization with chromatin states</strong></div><div><em>PLoS Comput. Biol.</em> 2019; <strong>15</strong>:e1007024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85068537423" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31181064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1007024&amp;pmid=31181064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This potential contributes a total of 6 parameters to the model.</div><div id="p0545" role="paragraph">Therefore, for the 1Mb-resolution model, the total number of parameters is <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N minus 1 plus 6 equals 254" tabindex="0" ctxtmenu_counter="86" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (7 0 1 2) 3 4) 5 6)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="8,6" data-semantic-content="5" data-semantic-owns="8 5 6" data-semantic-speech="upper N minus 1 plus 6 equals 254"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="8" data-semantic-children="7,4" data-semantic-content="3" data-semantic-parent="9" data-semantic-owns="7 3 4"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="7" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="8" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c36">6</mjx-c></mjx-mn></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.677em;">2</mjx-c><mjx-c class="mjx-c35" noic="true" style="padding-top: 0.677em;">5</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0550" role="paragraph">For the 100kb-resolution model, we further separated the intra-chromosome potential into intra- and inter-TAD interactions depending on whether the pair of beads are within the same topologically associating domain (TAD) or not. Specifically,<div id="fd5" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis equals sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts left bracket alpha Subscript italic i n t r a Superscript italic upper T upper A upper D Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper I Baseline right parenthesis delta Subscript upper T Sub Subscript i Sub Superscript upper I Subscript comma upper T Sub Subscript j Sub Superscript upper I Subscript Baseline plus alpha Subscript italic i n t e r Superscript italic upper T upper A upper D Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper I Baseline right parenthesis left parenthesis 1 minus delta Subscript upper T Sub Subscript i Sub Superscript upper I Subscript comma upper T Sub Subscript j Sub Superscript upper I Subscript Baseline right parenthesis right bracket f left parenthesis r Subscript i j Baseline right parenthesis plus sigma summation Underscript upper I comma upper J Endscripts sigma summation Underscript i element of upper I comma j element of upper J Endscripts alpha Subscript italic i n t e r Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper J Baseline right parenthesis f left parenthesis r Subscript i j Baseline right parenthesis comma" tabindex="0" ctxtmenu_counter="87" style="font-size: 130%;"><mjx-math data-semantic-structure="(181 (180 (178 (2 0 1) 177 (6 3 4 5)) 7 (179 (176 (10 8 9) (175 (16 11 (15 12 13 14)) (174 (103 17 (101 (98 (96 (22 (21 18 19) 20) 95 (37 23 (35 (28 (27 24 25) 26) 29 (34 (33 30 31) 32)) 36)) 97 (51 38 (50 (43 (42 39 40) 41) 44 (49 (48 45 46) 47)))) 52 (100 (94 (57 (56 53 54) 55) 93 (72 58 (70 (63 (62 59 60) 61) 64 (69 (68 65 66) 67)) 71)) 99 (92 73 (91 74 75 (89 76 (88 (81 (80 77 78) 79) 82 (87 (86 83 84) 85)))) 90))) 102) 173 (172 105 171 (114 106 (112 107 (111 108 110 109)) 113))))) 115 (170 (121 116 (120 117 118 119)) (169 (133 122 (132 (130 123 124 125) 126 (131 127 128 129))) (168 (166 (136 134 135) 165 (151 137 (150 (142 (141 138 139) 140) 143 (148 (147 144 145) 146)) 149)) 167 (164 152 163 (161 153 (159 154 (158 155 157 156)) 160))))))) 162)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="181" data-semantic-children="180,162" data-semantic-content="162" data-semantic-owns="180 162" data-semantic-speech="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis equals sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts left bracket alpha Subscript italic i n t r a Superscript italic upper T upper A upper D Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper I Baseline right parenthesis delta Subscript upper T Sub Subscript i Sub Superscript upper I Subscript comma upper T Sub Subscript j Sub Superscript upper I Subscript Baseline plus alpha Subscript italic i n t e r Superscript italic upper T upper A upper D Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper I Baseline right parenthesis left parenthesis 1 minus delta Subscript upper T Sub Subscript i Sub Superscript upper I Subscript comma upper T Sub Subscript j Sub Superscript upper I Subscript Baseline right parenthesis right bracket f left parenthesis r Subscript i j Baseline right parenthesis plus sigma summation Underscript upper I comma upper J Endscripts sigma summation Underscript i element of upper I comma j element of upper J Endscripts alpha Subscript italic i n t e r Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper J Baseline right parenthesis f left parenthesis r Subscript i j Baseline right parenthesis comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="180" data-semantic-children="178,179" data-semantic-content="7" data-semantic-parent="181" data-semantic-owns="178 7 179"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="178" data-semantic-children="2,6" data-semantic-content="177,0" data-semantic-parent="180" data-semantic-owns="2 177 6"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="178" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450" noic="true" style="padding-top: 0.626em;">𝑐</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.626em;">𝑜</mjx-c><mjx-c class="mjx-c1D45A" noic="true" style="padding-top: 0.626em;">𝑚</mjx-c><mjx-c class="mjx-c1D45D" noic="true" style="padding-top: 0.626em;">𝑝</mjx-c><mjx-c class="mjx-c1D461" style="padding-top: 0.626em;">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="177" data-semantic-parent="178" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="3,5" data-semantic-parent="178" data-semantic-owns="3 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="180" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="179" data-semantic-children="176,170" data-semantic-content="115" data-semantic-parent="180" data-semantic-owns="176 115 170"><mjx-mrow data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="176" data-semantic-children="10,175" data-semantic-content="8" data-semantic-parent="179" data-semantic-owns="10 175"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="176" data-semantic-owns="8 9" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="8" data-semantic-parent="10" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="175" data-semantic-children="16,174" data-semantic-content="11" data-semantic-parent="176" data-semantic-owns="16 174"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="16" data-semantic-children="11,15" data-semantic-parent="175" data-semantic-owns="11 15" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="11" data-semantic-parent="16" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="15" data-semantic-children="12,13,14" data-semantic-content="13" data-semantic-parent="16" data-semantic-owns="12 13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="13" data-semantic-parent="15" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="174" data-semantic-children="103,172" data-semantic-content="173" data-semantic-parent="175" data-semantic-owns="103 173 172"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="103" data-semantic-children="101" data-semantic-content="17,102" data-semantic-parent="174" data-semantic-owns="17 101 102"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="17" data-semantic-parent="103" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="101" data-semantic-children="98,100" data-semantic-content="52" data-semantic-parent="103" data-semantic-owns="98 52 100"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="98" data-semantic-children="96,51" data-semantic-content="97" data-semantic-parent="101" data-semantic-owns="96 97 51"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="96" data-semantic-children="22,37" data-semantic-content="95,18" data-semantic-parent="98" data-semantic-owns="22 95 37"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="simple function" data-semantic-id="22" data-semantic-children="18,19,20" data-semantic-parent="96" data-semantic-collapsed="(22 (21 18 19) 20)" data-semantic-owns="18 19 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="22" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="20" data-semantic-parent="22"><mjx-c class="mjx-c1D447" noic="true" style="padding-top: 0.716em;">𝑇</mjx-c><mjx-c class="mjx-c1D434" noic="true" style="padding-top: 0.716em;">𝐴</mjx-c><mjx-c class="mjx-c1D437" style="padding-top: 0.716em;">𝐷</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="19" data-semantic-parent="22"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D45F" noic="true" style="padding-top: 0.661em;">𝑟</mjx-c><mjx-c class="mjx-c1D44E" style="padding-top: 0.661em;">𝑎</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="95" data-semantic-parent="96" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="37" data-semantic-children="35" data-semantic-content="23,36" data-semantic-parent="96" data-semantic-owns="23 35 36"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="23" data-semantic-parent="37" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="35" data-semantic-children="28,29,34" data-semantic-content="29" data-semantic-parent="37" data-semantic-owns="28 29 34"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="24,25,26" data-semantic-parent="35" data-semantic-collapsed="(28 (27 24 25) 26)" data-semantic-owns="24 25 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="28"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="28"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="29" data-semantic-parent="35" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="34" data-semantic-children="30,31,32" data-semantic-parent="35" data-semantic-collapsed="(34 (33 30 31) 32)" data-semantic-owns="30 31 32"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="34"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="34" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="34"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="36" data-semantic-parent="37" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="97" data-semantic-parent="98" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="51" data-semantic-children="38,50" data-semantic-parent="98" data-semantic-owns="38 50"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="51"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="50" data-semantic-children="43,44,49" data-semantic-content="44" data-semantic-parent="51" data-semantic-owns="43 44 49"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="43" data-semantic-children="39,40,41" data-semantic-parent="50" data-semantic-collapsed="(43 (42 39 40) 41)" data-semantic-owns="39 40 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="43"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="43" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="43"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="44" data-semantic-parent="50" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="49" data-semantic-children="45,46,47" data-semantic-parent="50" data-semantic-collapsed="(49 (48 45 46) 47)" data-semantic-owns="45 46 47"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="49"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="49" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-parent="49"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="52" data-semantic-parent="101" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="100" data-semantic-children="94,92" data-semantic-content="99" data-semantic-parent="101" data-semantic-owns="94 99 92"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="94" data-semantic-children="57,72" data-semantic-content="93,53" data-semantic-parent="100" data-semantic-owns="57 93 72"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="simple function" data-semantic-id="57" data-semantic-children="53,54,55" data-semantic-parent="94" data-semantic-collapsed="(57 (56 53 54) 55)" data-semantic-owns="53 54 55"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="57" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="55" data-semantic-parent="57"><mjx-c class="mjx-c1D447" noic="true" style="padding-top: 0.716em;">𝑇</mjx-c><mjx-c class="mjx-c1D434" noic="true" style="padding-top: 0.716em;">𝐴</mjx-c><mjx-c class="mjx-c1D437" style="padding-top: 0.716em;">𝐷</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="54" data-semantic-parent="57"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.661em;">𝑒</mjx-c><mjx-c class="mjx-c1D45F" style="padding-top: 0.661em;">𝑟</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="93" data-semantic-parent="94" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="72" data-semantic-children="70" data-semantic-content="58,71" data-semantic-parent="94" data-semantic-owns="58 70 71"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="58" data-semantic-parent="72" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="70" data-semantic-children="63,64,69" data-semantic-content="64" data-semantic-parent="72" data-semantic-owns="63 64 69"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="63" data-semantic-children="59,60,61" data-semantic-parent="70" data-semantic-collapsed="(63 (62 59 60) 61)" data-semantic-owns="59 60 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="59" data-semantic-parent="63"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="63" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="63"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="64" data-semantic-parent="70" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="69" data-semantic-children="65,66,67" data-semantic-parent="70" data-semantic-collapsed="(69 (68 65 66) 67)" data-semantic-owns="65 66 67"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="65" data-semantic-parent="69"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="67" data-semantic-parent="69" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="66" data-semantic-parent="69"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="71" data-semantic-parent="72" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="99" data-semantic-parent="100" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="92" data-semantic-children="91" data-semantic-content="73,90" data-semantic-parent="100" data-semantic-owns="73 91 90"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="73" data-semantic-parent="92" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="91" data-semantic-children="74,89" data-semantic-content="75" data-semantic-parent="92" data-semantic-owns="74 75 89"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="74" data-semantic-parent="91"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="75" data-semantic-parent="91" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="89" data-semantic-children="76,88" data-semantic-parent="91" data-semantic-owns="76 88"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="76" data-semantic-parent="89"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="88" data-semantic-children="81,82,87" data-semantic-content="82" data-semantic-parent="89" data-semantic-owns="81 82 87"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="81" data-semantic-children="77,78,79" data-semantic-parent="88" data-semantic-collapsed="(81 (80 77 78) 79)" data-semantic-owns="77 78 79"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="77" data-semantic-parent="81"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="79" data-semantic-parent="81" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="78" data-semantic-parent="81"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="82" data-semantic-parent="88" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="87" data-semantic-children="83,84,85" data-semantic-parent="88" data-semantic-collapsed="(87 (86 83 84) 85)" data-semantic-owns="83 84 85"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="83" data-semantic-parent="87"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="85" data-semantic-parent="87" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="84" data-semantic-parent="87"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="90" data-semantic-parent="92" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="102" data-semantic-parent="103" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="173" data-semantic-parent="174" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="172" data-semantic-children="105,114" data-semantic-content="171,105" data-semantic-parent="174" data-semantic-owns="105 171 114"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="105" data-semantic-parent="172" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="171" data-semantic-parent="172" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="114" data-semantic-children="112" data-semantic-content="106,113" data-semantic-parent="172" data-semantic-owns="106 112 113"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="106" data-semantic-parent="114" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="112" data-semantic-children="107,111" data-semantic-parent="114" data-semantic-owns="107 111"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="107" data-semantic-parent="112"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="111" data-semantic-children="108,109" data-semantic-content="110" data-semantic-parent="112" data-semantic-owns="108 110 109"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="108" data-semantic-parent="111"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="110" data-semantic-parent="111" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="109" data-semantic-parent="111"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="113" data-semantic-parent="114" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="115" data-semantic-parent="179" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="170" data-semantic-children="121,169" data-semantic-content="116" data-semantic-parent="179" data-semantic-owns="121 169"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="121" data-semantic-children="116,120" data-semantic-parent="170" data-semantic-owns="116 120" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="116" data-semantic-parent="121" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="120" data-semantic-children="117,118,119" data-semantic-content="118" data-semantic-parent="121" data-semantic-owns="117 118 119"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="117" data-semantic-parent="120"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="118" data-semantic-parent="120" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="119" data-semantic-parent="120"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="169" data-semantic-children="133,168" data-semantic-content="122" data-semantic-parent="170" data-semantic-owns="133 168"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="133" data-semantic-children="122,132" data-semantic-parent="169" data-semantic-owns="122 132" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="122" data-semantic-parent="133" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="132" data-semantic-children="130,126,131" data-semantic-content="126" data-semantic-parent="133" data-semantic-owns="130 126 131"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="130" data-semantic-children="123,125" data-semantic-content="124" data-semantic-parent="132" data-semantic-owns="123 124 125"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="123" data-semantic-parent="130"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="124" data-semantic-parent="130" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="125" data-semantic-parent="130"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="126" data-semantic-parent="132" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="131" data-semantic-children="127,129" data-semantic-content="128" data-semantic-parent="132" data-semantic-owns="127 128 129"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="127" data-semantic-parent="131"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="128" data-semantic-parent="131" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="129" data-semantic-parent="131"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="168" data-semantic-children="166,164" data-semantic-content="167" data-semantic-parent="169" data-semantic-owns="166 167 164"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="166" data-semantic-children="136,151" data-semantic-content="165,134" data-semantic-parent="168" data-semantic-owns="136 165 151"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="136" data-semantic-children="134,135" data-semantic-parent="166" data-semantic-owns="134 135"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="134" data-semantic-parent="136" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="135" data-semantic-parent="136"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.661em;">𝑒</mjx-c><mjx-c class="mjx-c1D45F" style="padding-top: 0.661em;">𝑟</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="165" data-semantic-parent="166" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="151" data-semantic-children="150" data-semantic-content="137,149" data-semantic-parent="166" data-semantic-owns="137 150 149"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="137" data-semantic-parent="151" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="150" data-semantic-children="142,143,148" data-semantic-content="143" data-semantic-parent="151" data-semantic-owns="142 143 148"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="142" data-semantic-children="138,139,140" data-semantic-parent="150" data-semantic-collapsed="(142 (141 138 139) 140)" data-semantic-owns="138 139 140"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="138" data-semantic-parent="142"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="140" data-semantic-parent="142" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="139" data-semantic-parent="142"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="143" data-semantic-parent="150" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="148" data-semantic-children="144,145,146" data-semantic-parent="150" data-semantic-collapsed="(148 (147 144 145) 146)" data-semantic-owns="144 145 146"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="144" data-semantic-parent="148"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="146" data-semantic-parent="148" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="145" data-semantic-parent="148"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="149" data-semantic-parent="151" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="167" data-semantic-parent="168" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="164" data-semantic-children="152,161" data-semantic-content="163,152" data-semantic-parent="168" data-semantic-owns="152 163 161"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="152" data-semantic-parent="164" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="163" data-semantic-parent="164" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="161" data-semantic-children="159" data-semantic-content="153,160" data-semantic-parent="164" data-semantic-owns="153 159 160"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="153" data-semantic-parent="161" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="159" data-semantic-children="154,158" data-semantic-parent="161" data-semantic-owns="154 158"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="154" data-semantic-parent="159"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="158" data-semantic-children="155,156" data-semantic-content="157" data-semantic-parent="159" data-semantic-owns="155 157 156"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="155" data-semantic-parent="158"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="157" data-semantic-parent="158" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="156" data-semantic-parent="158"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="160" data-semantic-parent="161" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="162" data-semantic-parent="181" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">[5]</div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper T Subscript i Superscript upper I" tabindex="0" ctxtmenu_counter="88" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="upper T Subscript i Superscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> denotes the TAD index for bead <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="89" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-math></mjx-container> from chromosome <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I" tabindex="0" ctxtmenu_counter="90" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper I"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-math></mjx-container>. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="delta Subscript upper T Sub Subscript i Sub Superscript upper I Subscript comma upper T Sub Subscript j Sub Superscript upper I" tabindex="0" ctxtmenu_counter="91" style="font-size: 130%;"><mjx-math data-semantic-structure="(13 0 (12 (5 (4 1 2) 3) 6 (11 (10 7 8) 9)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="13" data-semantic-children="0,12" data-semantic-owns="0 12" data-semantic-speech="delta Subscript upper T Sub Subscript i Sub Superscript upper I Subscript comma upper T Sub Subscript j Sub Superscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="13"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="5,6,11" data-semantic-content="6" data-semantic-parent="13" data-semantic-owns="5 6 11"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="12" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="7,8,9" data-semantic-parent="12" data-semantic-collapsed="(11 (10 7 8) 9)" data-semantic-owns="7 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="11"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the Kronecker delta function and equals to 1 if <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper T Subscript i Superscript upper I Baseline equals upper T Subscript j Superscript upper I" tabindex="0" ctxtmenu_counter="92" style="font-size: 130%;"><mjx-math data-semantic-structure="(11 (4 (3 0 1) 2) 5 (10 (9 6 7) 8))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="4,10" data-semantic-content="5" data-semantic-owns="4 5 10" data-semantic-speech="upper T Subscript i Superscript upper I Baseline equals upper T Subscript j Superscript upper I"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="11" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="6,7,8" data-semantic-parent="11" data-semantic-collapsed="(10 (9 6 7) 8)" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="10"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.12em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10" style="margin-left: 0.249em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="10"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and 0 otherwise. The positions of TAD boundaries were determined from experimental Hi-C data using the software TADbit (<span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Serra et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Serra, F. ∙ Baù, D. ∙ Goodstadt, M. ...</div><div><strong>Automatic analysis and 3D-modelling of Hi-C data using TADbit reveals structural features of the fly chromatin colors</strong></div><div><em>PLoS Comput. Biol.</em> 2017; <strong>13</strong>:e1005665</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1005665" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85026659339" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28723903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1005665&amp;pmid=28723903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper C Subscript i Superscript upper I" tabindex="0" ctxtmenu_counter="93" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="upper C Subscript i Superscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> can adopt three values: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A comma upper B" tabindex="0" ctxtmenu_counter="94" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 1 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="4" data-semantic-children="0,1,3" data-semantic-content="1" data-semantic-owns="0 1 3" data-semantic-speech="upper A comma upper B"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I" tabindex="0" ctxtmenu_counter="95" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper I"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-math></mjx-container>. Therefore, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis" tabindex="0" ctxtmenu_counter="96" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (2 0 1) 7 (6 3 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-children="2,6" data-semantic-content="7,0" data-semantic-owns="2 7 6" data-semantic-speech="upper U Subscript italic c o m p t Baseline left parenthesis bold italic r right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="8" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D450" noic="true" style="padding-top: 0.626em;">𝑐</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.626em;">𝑜</mjx-c><mjx-c class="mjx-c1D45A" noic="true" style="padding-top: 0.626em;">𝑚</mjx-c><mjx-c class="mjx-c1D45D" noic="true" style="padding-top: 0.626em;">𝑝</mjx-c><mjx-c class="mjx-c1D461" style="padding-top: 0.626em;">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="3,5" data-semantic-parent="8" data-semantic-owns="3 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> contributes 18 parameters to the model. The total number of parameters for the 100kb-resolution model is thus <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2492 plus 18 equals 2510" tabindex="0" ctxtmenu_counter="97" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="6" data-semantic-children="5,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="2492 plus 18 equals 2510"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.677em;">2</mjx-c><mjx-c class="mjx-c34" noic="true" style="padding-top: 0.677em;">4</mjx-c><mjx-c class="mjx-c39" noic="true" style="padding-top: 0.677em;">9</mjx-c><mjx-c class="mjx-c32" style="padding-top: 0.677em;">2</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c35" noic="true" style="padding-top: 0.666em;">5</mjx-c><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>.</div></section><section id="sec-4-5-15"><h4>Parameter optimization</h4><div id="p0555" role="paragraph">Parameters in the above energy function can be derived using the iterative algorithm introduced in our previous works (<span class="dropBlock reference-citations"><a id="body-ref-sref71-6" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-6" aria-controls="sref71-6">Qi and Zhang, 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Qi, Y. ∙ Zhang, B.</div><div><strong>Predicting three-dimensional genome organization with chromatin states</strong></div><div><em>PLoS Comput. Biol.</em> 2019; <strong>15</strong>:e1007024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85068537423" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31181064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1007024&amp;pmid=31181064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In particular, parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript italic i d e a l Baseline left parenthesis StartAbsoluteValue j minus i EndAbsoluteValue right parenthesis" tabindex="0" ctxtmenu_counter="98" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (2 0 1) 13 (12 3 (10 4 (8 5 6 7) 9) 11))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="14" data-semantic-children="2,12" data-semantic-content="13,0" data-semantic-owns="2 13 12" data-semantic-speech="alpha Subscript italic i d e a l Baseline left parenthesis StartAbsoluteValue j minus i EndAbsoluteValue right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="14" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.694em;">𝑒</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D459" style="padding-top: 0.694em;">𝑙</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="3,11" data-semantic-parent="14" data-semantic-owns="3 10 11"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="10" data-semantic-children="8" data-semantic-content="4,9" data-semantic-parent="12" data-semantic-owns="4 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="4" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-parent="10" data-semantic-owns="5 6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript italic i n t r a Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="99" style="font-size: 130%;"><mjx-math data-semantic-structure="(19 (2 0 1) 18 (17 15 (14 (7 (6 3 4) 5) 8 (13 (12 9 10) 11)) 16))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="19" data-semantic-children="2,17" data-semantic-content="18,0" data-semantic-owns="2 18 17" data-semantic-speech="alpha Subscript italic i n t r a Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper I Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="19" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D45F" noic="true" style="padding-top: 0.661em;">𝑟</mjx-c><mjx-c class="mjx-c1D44E" style="padding-top: 0.661em;">𝑎</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="17" data-semantic-children="14" data-semantic-content="15,16" data-semantic-parent="19" data-semantic-owns="15 14 16"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="15" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="14" data-semantic-children="7,8,13" data-semantic-content="8" data-semantic-parent="17" data-semantic-owns="7 8 13"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="3,4,5" data-semantic-parent="14" data-semantic-collapsed="(7 (6 3 4) 5)" data-semantic-owns="3 4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="14" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="9,10,11" data-semantic-parent="14" data-semantic-collapsed="(13 (12 9 10) 11)" data-semantic-owns="9 10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="13"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript italic i n t e r Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper J Baseline right parenthesis" tabindex="0" ctxtmenu_counter="100" style="font-size: 130%;"><mjx-math data-semantic-structure="(19 (2 0 1) 18 (17 15 (14 (7 (6 3 4) 5) 8 (13 (12 9 10) 11)) 16))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="19" data-semantic-children="2,17" data-semantic-content="18,0" data-semantic-owns="2 18 17" data-semantic-speech="alpha Subscript italic i n t e r Baseline left parenthesis upper C Subscript i Superscript upper I Baseline comma upper C Subscript j Superscript upper J Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="19" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.661em;">𝑛</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.661em;">𝑡</mjx-c><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.661em;">𝑒</mjx-c><mjx-c class="mjx-c1D45F" style="padding-top: 0.661em;">𝑟</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="17" data-semantic-children="14" data-semantic-content="15,16" data-semantic-parent="19" data-semantic-owns="15 14 16"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="15" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="14" data-semantic-children="7,8,13" data-semantic-content="8" data-semantic-parent="17" data-semantic-owns="7 8 13"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="3,4,5" data-semantic-parent="14" data-semantic-collapsed="(7 (6 3 4) 5)" data-semantic-owns="3 4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="14" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="9,10,11" data-semantic-parent="14" data-semantic-collapsed="(13 (12 9 10) 11)" data-semantic-owns="9 10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="13"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> are tuned to ensure that the following ensemble averages determined with simulated genome conformation matches corresponding experimental constraints calculated using Hi-C data.<div id="fd6" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts f left parenthesis r Subscript i j Baseline right parenthesis delta Subscript StartAbsoluteValue j minus i EndAbsoluteValue comma s Baseline greater than equals sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts f Subscript i j Superscript italic exp Baseline delta Subscript StartAbsoluteValue j minus i EndAbsoluteValue comma s Baseline comma italic f o r s equals 1 comma ellipsis comma upper N minus 1" tabindex="0" ctxtmenu_counter="101" style="font-size: 130%;"><mjx-math data-semantic-structure="(93 (88 (39 37 (36 (2 0 1) (35 (8 3 (7 4 5 6)) (34 (32 9 31 (18 16 (15 10 (14 11 13 12)) 17)) 33 (30 19 (29 (26 20 (25 21 22 23) 24) 27 28))))) 38) 40 (87 (43 41 42) (86 (49 44 (48 45 46 47)) (85 (57 (56 50 (54 51 53 52)) 55) 84 (69 58 (68 (65 59 (64 60 61 62) 63) 66 67)))))) 70 (91 (90 71 89 73) 74 75) 76 77 79 (92 81 82 83))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="93" data-semantic-children="88,70,91,76,77,79,92" data-semantic-content="70,76,79" data-semantic-owns="88 70 91 76 77 79 92" data-semantic-speech="less than sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts f left parenthesis r Subscript i j Baseline right parenthesis delta Subscript StartAbsoluteValue j minus i EndAbsoluteValue comma s Baseline greater than equals sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts f Subscript i j Superscript italic exp Baseline delta Subscript StartAbsoluteValue j minus i EndAbsoluteValue comma s Baseline comma italic f o r s equals 1 comma ellipsis comma upper N minus 1"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="88" data-semantic-children="39,87" data-semantic-content="40" data-semantic-parent="93" data-semantic-owns="39 40 87"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="inequality" data-semantic-id="39" data-semantic-children="36" data-semantic-content="37,38" data-semantic-parent="88" data-semantic-owns="37 36 38"><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="37" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="36" data-semantic-children="2,35" data-semantic-content="0" data-semantic-parent="39" data-semantic-owns="2 35"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="36" data-semantic-owns="0 1" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="35" data-semantic-children="8,34" data-semantic-content="3" data-semantic-parent="36" data-semantic-owns="8 34"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="35" data-semantic-owns="3 7" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="4,5,6" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="34" data-semantic-children="32,30" data-semantic-content="33" data-semantic-parent="35" data-semantic-owns="32 33 30"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="32" data-semantic-children="9,18" data-semantic-content="31,9" data-semantic-parent="34" data-semantic-owns="9 31 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="32" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="18" data-semantic-children="15" data-semantic-content="16,17" data-semantic-parent="32" data-semantic-owns="16 15 17"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="16" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="10,14" data-semantic-parent="18" data-semantic-owns="10 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="15"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="14" data-semantic-children="11,12" data-semantic-content="13" data-semantic-parent="15" data-semantic-owns="11 13 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="30" data-semantic-children="19,29" data-semantic-parent="34" data-semantic-owns="19 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="30"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.177em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="29" data-semantic-children="26,27,28" data-semantic-content="27" data-semantic-parent="30" data-semantic-owns="26 27 28"><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="26" data-semantic-children="25" data-semantic-content="20,24" data-semantic-parent="29" data-semantic-owns="20 25 24"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="20" data-semantic-parent="26" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="25" data-semantic-children="21,23" data-semantic-content="22" data-semantic-parent="26" data-semantic-owns="21 22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="25"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="22" data-semantic-parent="25" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="24" data-semantic-parent="26" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="27" data-semantic-parent="29" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="29"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="40" data-semantic-parent="88" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="87" data-semantic-children="43,86" data-semantic-content="41" data-semantic-parent="88" data-semantic-owns="43 86"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="43" data-semantic-children="41,42" data-semantic-parent="87" data-semantic-owns="41 42" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="41" data-semantic-parent="43" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="43"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="86" data-semantic-children="49,85" data-semantic-content="44" data-semantic-parent="87" data-semantic-owns="49 85"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="49" data-semantic-children="44,48" data-semantic-parent="86" data-semantic-owns="44 48" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="44" data-semantic-parent="49" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="48" data-semantic-children="45,46,47" data-semantic-content="46" data-semantic-parent="49" data-semantic-owns="45 46 47"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="48"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="46" data-semantic-parent="48" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="48"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="85" data-semantic-children="57,69" data-semantic-content="84,50" data-semantic-parent="86" data-semantic-owns="57 84 69"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="simple function" data-semantic-id="57" data-semantic-children="50,54,55" data-semantic-parent="85" data-semantic-collapsed="(57 (56 50 54) 55)" data-semantic-owns="50 54 55"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="57" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="italic" data-semantic-id="55" data-semantic-parent="57" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.442em;">𝑒</mjx-c><mjx-c class="mjx-c1D465" noic="true" style="padding-top: 0.442em;">𝑥</mjx-c><mjx-c class="mjx-c1D45D" style="padding-top: 0.442em;">𝑝</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="54" data-semantic-children="51,52" data-semantic-content="53" data-semantic-parent="57" data-semantic-owns="51 53 52"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="54"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="53" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="54"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="84" data-semantic-parent="85" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="69" data-semantic-children="58,68" data-semantic-parent="85" data-semantic-owns="58 68"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="58" data-semantic-parent="69"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.177em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="68" data-semantic-children="65,66,67" data-semantic-content="66" data-semantic-parent="69" data-semantic-owns="65 66 67"><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="65" data-semantic-children="64" data-semantic-content="59,63" data-semantic-parent="68" data-semantic-owns="59 64 63"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="59" data-semantic-parent="65" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="64" data-semantic-children="60,62" data-semantic-content="61" data-semantic-parent="65" data-semantic-owns="60 61 62"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="64"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="61" data-semantic-parent="64" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="62" data-semantic-parent="64"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="63" data-semantic-parent="65" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="66" data-semantic-parent="68" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="67" data-semantic-parent="68"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="70" data-semantic-parent="93" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="91" data-semantic-children="90,75" data-semantic-content="74" data-semantic-parent="93" data-semantic-owns="90 74 75"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="90" data-semantic-children="71,73" data-semantic-content="89" data-semantic-parent="91" data-semantic-owns="71 89 73"><mjx-mi data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="71" data-semantic-parent="90"><mjx-c class="mjx-c1D453" noic="true" style="padding-top: 0.705em;">𝑓</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.705em;">𝑜</mjx-c><mjx-c class="mjx-c1D45F" style="padding-top: 0.705em;">𝑟</mjx-c></mjx-mi><mjx-mspace data-semantic-type="operator" data-semantic-role="space" data-semantic-id="89" data-semantic-parent="90" data-semantic-operator="infixop,⁢" style="width: 0.25em;"></mjx-mspace><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="73" data-semantic-parent="90"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="74" data-semantic-parent="91" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="75" data-semantic-parent="91"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="76" data-semantic-parent="93" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="77" data-semantic-parent="93"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="79" data-semantic-parent="93" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="92" data-semantic-children="81,83" data-semantic-content="82" data-semantic-parent="93" data-semantic-owns="81 82 83"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="81" data-semantic-parent="92"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="82" data-semantic-parent="92" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="83" data-semantic-parent="92"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">[6]</div></div><div id="ufd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts f left parenthesis r Subscript i j Baseline right parenthesis delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1 Baseline delta Subscript upper C Sub Subscript j Sub Superscript upper I Subscript comma c 2 Baseline greater than equals sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts f Subscript i j Superscript italic exp Baseline delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1 Baseline delta Subscript upper C Sub Subscript j Sub Superscript upper I Subscript comma c 2 Baseline comma italic f o r left parenthesis c 1 comma c 2 right parenthesis element of StartSet left parenthesis upper A comma upper A right parenthesis comma left parenthesis upper A comma upper B right parenthesis comma left parenthesis upper B comma upper B right parenthesis EndSet" tabindex="0" ctxtmenu_counter="102" style="font-size: 130%;"><mjx-math data-semantic-structure="(147 (143 (52 50 (49 (2 0 1) (48 (8 3 (7 4 5 6)) (47 (44 9 43 (18 16 (15 10 (14 11 13 12)) 17)) 45 (30 19 (29 (24 (23 20 21) 22) 25 (28 26 27))) 46 (42 31 (41 (36 (35 32 33) 34) 37 (40 38 39)))))) 51) 53 (142 (56 54 55) (141 (62 57 (61 58 59 60)) (140 (70 (69 63 (67 64 66 65)) 68) 139 (138 (82 71 (81 (76 (75 72 73) 74) 77 (80 78 79))) 137 (94 83 (93 (88 (87 84 85) 86) 89 (92 90 91)))))))) 95 (146 (145 96 144 (108 106 (105 (100 98 99) 101 (104 102 103)) 107)) 109 (136 134 (133 (116 114 (113 110 111 112) 115) 117 (124 122 (121 118 119 120) 123) 125 (132 130 (129 126 127 128) 131)) 135)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="147" data-semantic-children="143,95,146" data-semantic-content="95" data-semantic-owns="143 95 146" data-semantic-speech="less than sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts f left parenthesis r Subscript i j Baseline right parenthesis delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1 Baseline delta Subscript upper C Sub Subscript j Sub Superscript upper I Subscript comma c 2 Baseline greater than equals sigma summation Underscript upper I Endscripts sigma summation Underscript i comma j Endscripts f Subscript i j Superscript italic exp Baseline delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1 Baseline delta Subscript upper C Sub Subscript j Sub Superscript upper I Subscript comma c 2 Baseline comma italic f o r left parenthesis c 1 comma c 2 right parenthesis element of StartSet left parenthesis upper A comma upper A right parenthesis comma left parenthesis upper A comma upper B right parenthesis comma left parenthesis upper B comma upper B right parenthesis EndSet"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="143" data-semantic-children="52,142" data-semantic-content="53" data-semantic-parent="147" data-semantic-owns="52 53 142"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="inequality" data-semantic-id="52" data-semantic-children="49" data-semantic-content="50,51" data-semantic-parent="143" data-semantic-owns="50 49 51"><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="50" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="49" data-semantic-children="2,48" data-semantic-content="0" data-semantic-parent="52" data-semantic-owns="2 48"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="49" data-semantic-owns="0 1" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="48" data-semantic-children="8,47" data-semantic-content="3" data-semantic-parent="49" data-semantic-owns="8 47"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="48" data-semantic-owns="3 7" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="4,5,6" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="47" data-semantic-children="44,30,42" data-semantic-content="45,46" data-semantic-parent="48" data-semantic-owns="44 45 30 46 42"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="44" data-semantic-children="9,18" data-semantic-content="43,9" data-semantic-parent="47" data-semantic-owns="9 43 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="44" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="18" data-semantic-children="15" data-semantic-content="16,17" data-semantic-parent="44" data-semantic-owns="16 15 17"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="16" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="10,14" data-semantic-parent="18" data-semantic-owns="10 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="15"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="14" data-semantic-children="11,12" data-semantic-content="13" data-semantic-parent="15" data-semantic-owns="11 13 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="30" data-semantic-children="19,29" data-semantic-parent="47" data-semantic-owns="19 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="30"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="29" data-semantic-children="24,25,28" data-semantic-content="25" data-semantic-parent="30" data-semantic-owns="24 25 28"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="20,21,22" data-semantic-parent="29" data-semantic-collapsed="(24 (23 20 21) 22)" data-semantic-owns="20 21 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="24"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="24" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="24"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="25" data-semantic-parent="29" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="29" data-semantic-owns="26 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="46" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="42" data-semantic-children="31,41" data-semantic-parent="47" data-semantic-owns="31 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="42"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="41" data-semantic-children="36,37,40" data-semantic-content="37" data-semantic-parent="42" data-semantic-owns="36 37 40"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="36" data-semantic-children="32,33,34" data-semantic-parent="41" data-semantic-collapsed="(36 (35 32 33) 34)" data-semantic-owns="32 33 34"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="36"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="36" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="36"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="37" data-semantic-parent="41" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="40" data-semantic-children="38,39" data-semantic-parent="41" data-semantic-owns="38 39"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="40"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="40"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="51" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="53" data-semantic-parent="143" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="142" data-semantic-children="56,141" data-semantic-content="54" data-semantic-parent="143" data-semantic-owns="56 141"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="56" data-semantic-children="54,55" data-semantic-parent="142" data-semantic-owns="54 55" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="54" data-semantic-parent="56" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="55" data-semantic-parent="56"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="141" data-semantic-children="62,140" data-semantic-content="57" data-semantic-parent="142" data-semantic-owns="62 140"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="62" data-semantic-children="57,61" data-semantic-parent="141" data-semantic-owns="57 61" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="57" data-semantic-parent="62" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="61" data-semantic-children="58,59,60" data-semantic-content="59" data-semantic-parent="62" data-semantic-owns="58 59 60"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="58" data-semantic-parent="61"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="59" data-semantic-parent="61" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="61"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="140" data-semantic-children="70,138" data-semantic-content="139,63" data-semantic-parent="141" data-semantic-owns="70 139 138"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="simple function" data-semantic-id="70" data-semantic-children="63,67,68" data-semantic-parent="140" data-semantic-collapsed="(70 (69 63 67) 68)" data-semantic-owns="63 67 68"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="63" data-semantic-parent="70" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="italic" data-semantic-id="68" data-semantic-parent="70" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.442em;">𝑒</mjx-c><mjx-c class="mjx-c1D465" noic="true" style="padding-top: 0.442em;">𝑥</mjx-c><mjx-c class="mjx-c1D45D" style="padding-top: 0.442em;">𝑝</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="67" data-semantic-children="64,65" data-semantic-content="66" data-semantic-parent="70" data-semantic-owns="64 66 65"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="67"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="66" data-semantic-parent="67" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="65" data-semantic-parent="67"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="139" data-semantic-parent="140" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="138" data-semantic-children="82,94" data-semantic-content="137" data-semantic-parent="140" data-semantic-owns="82 137 94"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="82" data-semantic-children="71,81" data-semantic-parent="138" data-semantic-owns="71 81"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="82"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="81" data-semantic-children="76,77,80" data-semantic-content="77" data-semantic-parent="82" data-semantic-owns="76 77 80"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="76" data-semantic-children="72,73,74" data-semantic-parent="81" data-semantic-collapsed="(76 (75 72 73) 74)" data-semantic-owns="72 73 74"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="76"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="74" data-semantic-parent="76" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="73" data-semantic-parent="76"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="77" data-semantic-parent="81" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="80" data-semantic-children="78,79" data-semantic-parent="81" data-semantic-owns="78 79"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="78" data-semantic-parent="80"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="79" data-semantic-parent="80"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="137" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="94" data-semantic-children="83,93" data-semantic-parent="138" data-semantic-owns="83 93"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="83" data-semantic-parent="94"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="93" data-semantic-children="88,89,92" data-semantic-content="89" data-semantic-parent="94" data-semantic-owns="88 89 92"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="88" data-semantic-children="84,85,86" data-semantic-parent="93" data-semantic-collapsed="(88 (87 84 85) 86)" data-semantic-owns="84 85 86"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="84" data-semantic-parent="88"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="86" data-semantic-parent="88" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="85" data-semantic-parent="88"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="89" data-semantic-parent="93" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="92" data-semantic-children="90,91" data-semantic-parent="93" data-semantic-owns="90 91"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="90" data-semantic-parent="92"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="91" data-semantic-parent="92"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="95" data-semantic-parent="147" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="146" data-semantic-children="145,136" data-semantic-content="109" data-semantic-parent="147" data-semantic-owns="145 109 136"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="145" data-semantic-children="96,108" data-semantic-content="144,96" data-semantic-parent="146" data-semantic-owns="96 144 108"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="96" data-semantic-parent="145" data-semantic-operator="appl"><mjx-c class="mjx-c1D453" noic="true" style="padding-top: 0.705em;">𝑓</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.705em;">𝑜</mjx-c><mjx-c class="mjx-c1D45F" style="padding-top: 0.705em;">𝑟</mjx-c></mjx-mi><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="144" data-semantic-parent="145" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="108" data-semantic-children="105" data-semantic-content="106,107" data-semantic-parent="145" data-semantic-owns="106 105 107"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="106" data-semantic-parent="108" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="105" data-semantic-children="100,101,104" data-semantic-content="101" data-semantic-parent="108" data-semantic-owns="100 101 104"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="100" data-semantic-children="98,99" data-semantic-parent="105" data-semantic-owns="98 99"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="98" data-semantic-parent="100"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="99" data-semantic-parent="100"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="101" data-semantic-parent="105" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="104" data-semantic-children="102,103" data-semantic-parent="105" data-semantic-owns="102 103"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="102" data-semantic-parent="104"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="103" data-semantic-parent="104"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="107" data-semantic-parent="108" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="109" data-semantic-parent="146" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="136" data-semantic-children="133" data-semantic-content="134,135" data-semantic-parent="146" data-semantic-owns="134 133 135"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="134" data-semantic-parent="136" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="133" data-semantic-children="116,117,124,125,132" data-semantic-content="117,125" data-semantic-parent="136" data-semantic-owns="116 117 124 125 132"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="116" data-semantic-children="113" data-semantic-content="114,115" data-semantic-parent="133" data-semantic-owns="114 113 115"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="114" data-semantic-parent="116" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="113" data-semantic-children="110,111,112" data-semantic-content="111" data-semantic-parent="116" data-semantic-owns="110 111 112"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="110" data-semantic-parent="113"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="111" data-semantic-parent="113" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="112" data-semantic-parent="113"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="115" data-semantic-parent="116" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="117" data-semantic-parent="133" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="124" data-semantic-children="121" data-semantic-content="122,123" data-semantic-parent="133" data-semantic-owns="122 121 123"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="122" data-semantic-parent="124" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="121" data-semantic-children="118,119,120" data-semantic-content="119" data-semantic-parent="124" data-semantic-owns="118 119 120"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="118" data-semantic-parent="121"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="119" data-semantic-parent="121" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="120" data-semantic-parent="121"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="123" data-semantic-parent="124" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="125" data-semantic-parent="133" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="132" data-semantic-children="129" data-semantic-content="130,131" data-semantic-parent="133" data-semantic-owns="130 129 131"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="130" data-semantic-parent="132" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="129" data-semantic-children="126,127,128" data-semantic-content="127" data-semantic-parent="132" data-semantic-owns="126 127 128"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="126" data-semantic-parent="129"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="127" data-semantic-parent="129" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="128" data-semantic-parent="129"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="131" data-semantic-parent="132" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="135" data-semantic-parent="136" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div><div id="ufd4" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than sigma summation Underscript upper I comma upper J Endscripts sigma summation Underscript i element of upper I comma j element of upper J Endscripts f left parenthesis r Subscript i j Baseline right parenthesis delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1 Baseline delta Subscript upper C Sub Subscript j Sub Superscript upper J Subscript comma c 2 Baseline greater than equals sigma summation Underscript upper I comma upper J Endscripts sigma summation Underscript i element of upper I comma j element of upper J Endscripts f Subscript i j Superscript italic exp Baseline delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1 Baseline delta Subscript upper C Sub Subscript j Sub Superscript upper J Subscript comma c 2 Baseline comma italic f o r left parenthesis c 1 comma c 2 right parenthesis element of StartSet left parenthesis upper A comma upper A right parenthesis comma left parenthesis upper A comma upper B right parenthesis comma left parenthesis upper B comma upper B right parenthesis EndSet" tabindex="0" ctxtmenu_counter="103" style="font-size: 130%;"><mjx-math data-semantic-structure="(165 (161 (61 59 (58 (5 0 (4 1 2 3)) (57 (17 6 (16 (14 7 8 9) 10 (15 11 12 13))) (56 (53 18 52 (27 25 (24 19 (23 20 22 21)) 26)) 54 (39 28 (38 (33 (32 29 30) 31) 34 (37 35 36))) 55 (51 40 (50 (45 (44 41 42) 43) 46 (49 47 48)))))) 60) 62 (160 (68 63 (67 64 65 66)) (159 (80 69 (79 (77 70 71 72) 73 (78 74 75 76))) (158 (88 (87 81 (85 82 84 83)) 86) 157 (156 (100 89 (99 (94 (93 90 91) 92) 95 (98 96 97))) 155 (112 101 (111 (106 (105 102 103) 104) 107 (110 108 109)))))))) 113 (164 (163 114 162 (126 124 (123 (118 116 117) 119 (122 120 121)) 125)) 127 (154 152 (151 (134 132 (131 128 129 130) 133) 135 (142 140 (139 136 137 138) 141) 143 (150 148 (147 144 145 146) 149)) 153)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="165" data-semantic-children="161,113,164" data-semantic-content="113" data-semantic-owns="161 113 164" data-semantic-speech="less than sigma summation Underscript upper I comma upper J Endscripts sigma summation Underscript i element of upper I comma j element of upper J Endscripts f left parenthesis r Subscript i j Baseline right parenthesis delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1 Baseline delta Subscript upper C Sub Subscript j Sub Superscript upper J Subscript comma c 2 Baseline greater than equals sigma summation Underscript upper I comma upper J Endscripts sigma summation Underscript i element of upper I comma j element of upper J Endscripts f Subscript i j Superscript italic exp Baseline delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1 Baseline delta Subscript upper C Sub Subscript j Sub Superscript upper J Subscript comma c 2 Baseline comma italic f o r left parenthesis c 1 comma c 2 right parenthesis element of StartSet left parenthesis upper A comma upper A right parenthesis comma left parenthesis upper A comma upper B right parenthesis comma left parenthesis upper B comma upper B right parenthesis EndSet"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="161" data-semantic-children="61,160" data-semantic-content="62" data-semantic-parent="165" data-semantic-owns="61 62 160"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="inequality" data-semantic-id="61" data-semantic-children="58" data-semantic-content="59,60" data-semantic-parent="161" data-semantic-owns="59 58 60"><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="59" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="58" data-semantic-children="5,57" data-semantic-content="0" data-semantic-parent="61" data-semantic-owns="5 57"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="58" data-semantic-owns="0 4" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="4" data-semantic-children="1,2,3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="57" data-semantic-children="17,56" data-semantic-content="6" data-semantic-parent="58" data-semantic-owns="17 56"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="17" data-semantic-children="6,16" data-semantic-parent="57" data-semantic-owns="6 16" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="6" data-semantic-parent="17" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="14,10,15" data-semantic-content="10" data-semantic-parent="17" data-semantic-owns="14 10 15"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="14" data-semantic-children="7,9" data-semantic-content="8" data-semantic-parent="16" data-semantic-owns="7 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="8" data-semantic-parent="14" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="14"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="10" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="15" data-semantic-children="11,13" data-semantic-content="12" data-semantic-parent="16" data-semantic-owns="11 12 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="15"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="12" data-semantic-parent="15" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="56" data-semantic-children="53,39,51" data-semantic-content="54,55" data-semantic-parent="57" data-semantic-owns="53 54 39 55 51"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="53" data-semantic-children="18,27" data-semantic-content="52,18" data-semantic-parent="56" data-semantic-owns="18 52 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="53" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="52" data-semantic-parent="53" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="27" data-semantic-children="24" data-semantic-content="25,26" data-semantic-parent="53" data-semantic-owns="25 24 26"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="25" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="19,23" data-semantic-parent="27" data-semantic-owns="19 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="24"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="23" data-semantic-children="20,21" data-semantic-content="22" data-semantic-parent="24" data-semantic-owns="20 22 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="23"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="23"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="54" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="39" data-semantic-children="28,38" data-semantic-parent="56" data-semantic-owns="28 38"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="39"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="38" data-semantic-children="33,34,37" data-semantic-content="34" data-semantic-parent="39" data-semantic-owns="33 34 37"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="33" data-semantic-children="29,30,31" data-semantic-parent="38" data-semantic-collapsed="(33 (32 29 30) 31)" data-semantic-owns="29 30 31"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="33"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="33" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="33"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="34" data-semantic-parent="38" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="37" data-semantic-children="35,36" data-semantic-parent="38" data-semantic-owns="35 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="37"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="37"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="51" data-semantic-children="40,50" data-semantic-parent="56" data-semantic-owns="40 50"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="51"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.249em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="50" data-semantic-children="45,46,49" data-semantic-content="46" data-semantic-parent="51" data-semantic-owns="45 46 49"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="45" data-semantic-children="41,42,43" data-semantic-parent="50" data-semantic-collapsed="(45 (44 41 42) 43)" data-semantic-owns="41 42 43"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="45"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="45" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="45"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="46" data-semantic-parent="50" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="49" data-semantic-children="47,48" data-semantic-parent="50" data-semantic-owns="47 48"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="49"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="49"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="60" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="62" data-semantic-parent="161" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="160" data-semantic-children="68,159" data-semantic-content="63" data-semantic-parent="161" data-semantic-owns="68 159"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="68" data-semantic-children="63,67" data-semantic-parent="160" data-semantic-owns="63 67" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="63" data-semantic-parent="68" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="67" data-semantic-children="64,65,66" data-semantic-content="65" data-semantic-parent="68" data-semantic-owns="64 65 66"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="67"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="65" data-semantic-parent="67" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="66" data-semantic-parent="67"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="159" data-semantic-children="80,158" data-semantic-content="69" data-semantic-parent="160" data-semantic-owns="80 158"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="80" data-semantic-children="69,79" data-semantic-parent="159" data-semantic-owns="69 79" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="69" data-semantic-parent="80" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="79" data-semantic-children="77,73,78" data-semantic-content="73" data-semantic-parent="80" data-semantic-owns="77 73 78"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="77" data-semantic-children="70,72" data-semantic-content="71" data-semantic-parent="79" data-semantic-owns="70 71 72"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="70" data-semantic-parent="77"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="71" data-semantic-parent="77" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="77"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="73" data-semantic-parent="79" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="78" data-semantic-children="74,76" data-semantic-content="75" data-semantic-parent="79" data-semantic-owns="74 75 76"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="74" data-semantic-parent="78"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="75" data-semantic-parent="78" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="76" data-semantic-parent="78"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="158" data-semantic-children="88,156" data-semantic-content="157,81" data-semantic-parent="159" data-semantic-owns="88 157 156"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="simple function" data-semantic-id="88" data-semantic-children="81,85,86" data-semantic-parent="158" data-semantic-collapsed="(88 (87 81 85) 86)" data-semantic-owns="81 85 86"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="81" data-semantic-parent="88" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="italic" data-semantic-id="86" data-semantic-parent="88" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.442em;">𝑒</mjx-c><mjx-c class="mjx-c1D465" noic="true" style="padding-top: 0.442em;">𝑥</mjx-c><mjx-c class="mjx-c1D45D" style="padding-top: 0.442em;">𝑝</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="85" data-semantic-children="82,83" data-semantic-content="84" data-semantic-parent="88" data-semantic-owns="82 84 83"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="82" data-semantic-parent="85"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="84" data-semantic-parent="85" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="83" data-semantic-parent="85"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="157" data-semantic-parent="158" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="156" data-semantic-children="100,112" data-semantic-content="155" data-semantic-parent="158" data-semantic-owns="100 155 112"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="100" data-semantic-children="89,99" data-semantic-parent="156" data-semantic-owns="89 99"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="89" data-semantic-parent="100"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="99" data-semantic-children="94,95,98" data-semantic-content="95" data-semantic-parent="100" data-semantic-owns="94 95 98"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="94" data-semantic-children="90,91,92" data-semantic-parent="99" data-semantic-collapsed="(94 (93 90 91) 92)" data-semantic-owns="90 91 92"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="90" data-semantic-parent="94"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="92" data-semantic-parent="94" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="91" data-semantic-parent="94"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="95" data-semantic-parent="99" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="98" data-semantic-children="96,97" data-semantic-parent="99" data-semantic-owns="96 97"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="96" data-semantic-parent="98"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="97" data-semantic-parent="98"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="155" data-semantic-parent="156" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="112" data-semantic-children="101,111" data-semantic-parent="156" data-semantic-owns="101 111"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="101" data-semantic-parent="112"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.249em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="111" data-semantic-children="106,107,110" data-semantic-content="107" data-semantic-parent="112" data-semantic-owns="106 107 110"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="106" data-semantic-children="102,103,104" data-semantic-parent="111" data-semantic-collapsed="(106 (105 102 103) 104)" data-semantic-owns="102 103 104"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="102" data-semantic-parent="106"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="104" data-semantic-parent="106" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="103" data-semantic-parent="106"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="107" data-semantic-parent="111" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="110" data-semantic-children="108,109" data-semantic-parent="111" data-semantic-owns="108 109"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="108" data-semantic-parent="110"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="109" data-semantic-parent="110"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="113" data-semantic-parent="165" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="164" data-semantic-children="163,154" data-semantic-content="127" data-semantic-parent="165" data-semantic-owns="163 127 154"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="163" data-semantic-children="114,126" data-semantic-content="162,114" data-semantic-parent="164" data-semantic-owns="114 162 126"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="114" data-semantic-parent="163" data-semantic-operator="appl"><mjx-c class="mjx-c1D453" noic="true" style="padding-top: 0.705em;">𝑓</mjx-c><mjx-c class="mjx-c1D45C" noic="true" style="padding-top: 0.705em;">𝑜</mjx-c><mjx-c class="mjx-c1D45F" style="padding-top: 0.705em;">𝑟</mjx-c></mjx-mi><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="162" data-semantic-parent="163" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="126" data-semantic-children="123" data-semantic-content="124,125" data-semantic-parent="163" data-semantic-owns="124 123 125"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="124" data-semantic-parent="126" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="123" data-semantic-children="118,119,122" data-semantic-content="119" data-semantic-parent="126" data-semantic-owns="118 119 122"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="118" data-semantic-children="116,117" data-semantic-parent="123" data-semantic-owns="116 117"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="116" data-semantic-parent="118"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="117" data-semantic-parent="118"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="119" data-semantic-parent="123" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="122" data-semantic-children="120,121" data-semantic-parent="123" data-semantic-owns="120 121"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="120" data-semantic-parent="122"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="121" data-semantic-parent="122"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="125" data-semantic-parent="126" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="127" data-semantic-parent="164" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="154" data-semantic-children="151" data-semantic-content="152,153" data-semantic-parent="164" data-semantic-owns="152 151 153"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="152" data-semantic-parent="154" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="151" data-semantic-children="134,135,142,143,150" data-semantic-content="135,143" data-semantic-parent="154" data-semantic-owns="134 135 142 143 150"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="134" data-semantic-children="131" data-semantic-content="132,133" data-semantic-parent="151" data-semantic-owns="132 131 133"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="132" data-semantic-parent="134" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="131" data-semantic-children="128,129,130" data-semantic-content="129" data-semantic-parent="134" data-semantic-owns="128 129 130"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="128" data-semantic-parent="131"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="129" data-semantic-parent="131" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="130" data-semantic-parent="131"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="133" data-semantic-parent="134" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="135" data-semantic-parent="151" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="142" data-semantic-children="139" data-semantic-content="140,141" data-semantic-parent="151" data-semantic-owns="140 139 141"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="140" data-semantic-parent="142" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="139" data-semantic-children="136,137,138" data-semantic-content="137" data-semantic-parent="142" data-semantic-owns="136 137 138"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="136" data-semantic-parent="139"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="137" data-semantic-parent="139" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="138" data-semantic-parent="139"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="141" data-semantic-parent="142" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="143" data-semantic-parent="151" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="150" data-semantic-children="147" data-semantic-content="148,149" data-semantic-parent="151" data-semantic-owns="148 147 149"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="148" data-semantic-parent="150" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="147" data-semantic-children="144,145,146" data-semantic-content="145" data-semantic-parent="150" data-semantic-owns="144 145 146"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="144" data-semantic-parent="147"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="145" data-semantic-parent="147" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="146" data-semantic-parent="147"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="149" data-semantic-parent="150" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="153" data-semantic-parent="154" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>In the above equations, the Kronecker delta function <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1" tabindex="0" ctxtmenu_counter="104" style="font-size: 130%;"><mjx-math data-semantic-structure="(11 0 (10 (5 (4 1 2) 3) 6 (9 7 8)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="11" data-semantic-children="0,10" data-semantic-owns="0 10" data-semantic-speech="delta Subscript upper C Sub Subscript i Sub Superscript upper I Subscript comma c 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="11"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.238em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="5,6,9" data-semantic-content="6" data-semantic-parent="11" data-semantic-owns="5 6 9"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="10" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="10" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>equals to 1 if <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper C Subscript i Superscript upper I Baseline equals c 1" tabindex="0" ctxtmenu_counter="105" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (4 (3 0 1) 2) 5 (8 6 7))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="4,8" data-semantic-content="5" data-semantic-owns="4 5 8" data-semantic-speech="upper C Subscript i Superscript upper I Baseline equals c 1"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="9" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="9" data-semantic-owns="6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and 0 otherwise. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="delta Subscript upper C Sub Subscript j Sub Superscript upper J Subscript comma c 2" tabindex="0" ctxtmenu_counter="106" style="font-size: 130%;"><mjx-math data-semantic-structure="(11 0 (10 (5 (4 1 2) 3) 6 (9 7 8)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="11" data-semantic-children="0,10" data-semantic-owns="0 10" data-semantic-speech="delta Subscript upper C Sub Subscript j Sub Superscript upper J Subscript comma c 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="11"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.249em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="5,6,9" data-semantic-content="6" data-semantic-parent="11" data-semantic-owns="5 6 9"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="10" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.045em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5" style="margin-left: 0.138em;"><mjx-c class="mjx-c1D43D">𝐽</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="10" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is similarly defined. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript i j Superscript italic exp" tabindex="0" ctxtmenu_counter="107" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (6 0 (4 1 3 2)) 5)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,4,5" data-semantic-collapsed="(7 (6 0 4) 5)" data-semantic-owns="0 4 5" data-semantic-speech="f Subscript i j Superscript italic exp"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="italic" data-semantic-id="5" data-semantic-parent="7" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.442em;">𝑒</mjx-c><mjx-c class="mjx-c1D465" noic="true" style="padding-top: 0.442em;">𝑥</mjx-c><mjx-c class="mjx-c1D45D" style="padding-top: 0.442em;">𝑝</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="7" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> is the contact probability between the pair of genomic segments <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="108" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="j" tabindex="0" ctxtmenu_counter="109" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="j"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-math></mjx-container> determined from Hi-C. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U Subscript italic upper M upper E Baseline left parenthesis bold italic r right parenthesis" tabindex="0" ctxtmenu_counter="110" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (2 0 1) 7 (6 3 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-children="2,6" data-semantic-content="7,0" data-semantic-owns="2 7 6" data-semantic-speech="upper U Subscript italic upper M upper E Baseline left parenthesis bold italic r right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="8" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D440" noic="true" style="padding-top: 0.683em;">𝑀</mjx-c><mjx-c class="mjx-c1D438" style="padding-top: 0.683em;">𝐸</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="3,5" data-semantic-parent="8" data-semantic-owns="3 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> can be shown as the least biased potential to reproduce these experimental constraints following the maximum entropy principle.</div><div id="p0560" role="paragraph">The constraints used to parameterize the 100kb-resolution model can be similarly defined.</div></section><section id="sec-4-5-16"><h4>Molecular dynamics simulation details</h4><div id="p0565" role="paragraph">The software package LAMMPS (<span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Plimpton, 1995</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Plimpton, S.</div><div><strong>Fast parallel algorithms for short-range molecular dynamics</strong></div><div><em>J.&nbsp;Comput. Phys.</em> 1995; <strong>117</strong>:1-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jcph.1995.1039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-0002467378" target="_blank" aria-label="Open Scopus in new tab">Scopus (23779)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjcph.1995.1039" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to carry out molecular dynamics simulations with reduced units and collect ensembles of genome organization. Simulations were maintained at a constant temperature <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper T equals 1 period 0" tabindex="0" ctxtmenu_counter="111" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="upper T equals 1 period 0"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> via the Langevin dynamics with a damping coefficient <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma equals 10 period 0" tabindex="0" ctxtmenu_counter="112" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="gamma equals 10 period 0"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and a time step of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="d t equals 0 period 01" tabindex="0" ctxtmenu_counter="113" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (7 0 6 1) 2 3) 4 5)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="9" data-semantic-children="8,4,5" data-semantic-content="4" data-semantic-owns="8 4 5" data-semantic-speech="d t equals 0 period 01"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="8" data-semantic-children="7,3" data-semantic-content="2" data-semantic-parent="9" data-semantic-owns="7 2 3"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="0,1" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="0 6 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="8" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="4" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="5" data-semantic-parent="9"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0570" role="paragraph">To generate an initial configuration for these simulations, we first placed all the chromosomes consecutively on a cubic lattice with an edge length of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="0 period 9 upper R divided by StartRoot 3 EndRoot" tabindex="0" ctxtmenu_counter="114" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 (8 2 7 3) 4 (6 5)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="0,1,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="0 period 9 upper R divided by StartRoot 3 EndRoot"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="9" data-semantic-children="8,6" data-semantic-content="4" data-semantic-parent="10" data-semantic-owns="8 4 6"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="8" data-semantic-children="2,3" data-semantic-content="7" data-semantic-parent="9" data-semantic-owns="2 7 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c39">9</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="4" data-semantic-parent="9" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="6" data-semantic-children="5" data-semantic-parent="9" data-semantic-owns="5"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R" tabindex="0" ctxtmenu_counter="115" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper R"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-math></mjx-container> is the radii of the spherical confinement introduced to ensure a volume fraction of 0.1. This configuration was subsequently equilibrated along a 100,000-step-long simulation under the potential <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="sigma summation Underscript upper I Endscripts upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="116" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 (2 0 1) (11 3 10 (9 4 (7 5 6) 8)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="12" data-semantic-children="2,11" data-semantic-content="0" data-semantic-owns="2 11" data-semantic-speech="sigma summation Underscript upper I Endscripts upper U left parenthesis bold italic r Subscript upper I Baseline right parenthesis"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="12" data-semantic-owns="0 1" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="11" data-semantic-children="3,9" data-semantic-content="10,3" data-semantic-parent="12" data-semantic-owns="3 10 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="11" data-semantic-operator="appl"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="7" data-semantic-content="4,8" data-semantic-parent="11" data-semantic-owns="4 7 8"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="9" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold-italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D493">𝒓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> to relax both the topology and energy of the polymer structures. The last configuration from this equilibration trajectory was then used to initialize our whole genome simulations. We note that the long sampling time used in our simulations ensures their convergence. Therefore, all the results presented in the manuscript are independent of this initial configuration.</div><div id="p0575" role="paragraph">Parameters of the whole genome models were determined iteratively. We initialized the first iteration of these simulations using the equilibrated configuration mentioned above. All subsequent simulations were initialized using the end configurations from the previous iteration. During each iteration, we carried out six independent ten-million-time-step-long simulations for the 1Mb-resolution model and ten independent two-million-time-step-long simulations for the 100kb-resolution model. Genome conformations were saved at every 2000 timesteps to calculate the ensemble averages. A total of 10 iterations were performed for the 1Mb-resolution model to reach an error of less than 5%. We define the error as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="epsilon equals sigma summation StartAbsoluteValue f Subscript i Superscript italic s i m Baseline minus f Subscript i Superscript italic exp Baseline EndAbsoluteValue divided by sigma summation f Subscript i Superscript italic exp" tabindex="0" ctxtmenu_counter="117" style="font-size: 130%;"><mjx-math data-semantic-structure="(28 0 1 (27 (26 2 (17 3 (15 (8 (7 4 5) 6) 9 (14 (13 10 11) 12)) 16)) 18 (25 19 (24 (23 20 21) 22))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="28" data-semantic-children="0,27" data-semantic-content="1" data-semantic-owns="0 1 27" data-semantic-speech="epsilon equals sigma summation StartAbsoluteValue f Subscript i Superscript italic s i m Baseline minus f Subscript i Superscript italic exp Baseline EndAbsoluteValue divided by sigma summation f Subscript i Superscript italic exp"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="28"><mjx-c class="mjx-c1D700">𝜀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="28" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="27" data-semantic-children="26,25" data-semantic-content="18" data-semantic-parent="28" data-semantic-owns="26 18 25"><mjx-mrow data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="26" data-semantic-children="2,17" data-semantic-content="2" data-semantic-parent="27" data-semantic-owns="2 17"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="2" data-semantic-parent="26" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="17" data-semantic-children="15" data-semantic-content="3,16" data-semantic-parent="26" data-semantic-owns="3 15 16"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="3" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c2223 MM-SO">∣</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="15" data-semantic-children="8,14" data-semantic-content="9" data-semantic-parent="17" data-semantic-owns="8 9 14"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="4,5,6" data-semantic-parent="15" data-semantic-collapsed="(8 (7 4 5) 6)" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.292em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="6" data-semantic-parent="8" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D460" noic="true" style="padding-top: 0.661em;">𝑠</mjx-c><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45A" style="padding-top: 0.661em;">𝑚</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="9" data-semantic-parent="15" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msubsup space="3" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="10,11,12" data-semantic-parent="15" data-semantic-collapsed="(14 (13 10 11) 12)" data-semantic-owns="10 11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="14"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="italic" data-semantic-id="12" data-semantic-parent="14" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.442em;">𝑒</mjx-c><mjx-c class="mjx-c1D465" noic="true" style="padding-top: 0.442em;">𝑥</mjx-c><mjx-c class="mjx-c1D45D" style="padding-top: 0.442em;">𝑝</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="16" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c2223 MM-SO">∣</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="18" data-semantic-parent="27" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="25" data-semantic-children="19,24" data-semantic-content="19" data-semantic-parent="27" data-semantic-owns="19 24"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="19" data-semantic-parent="25" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="20,21,22" data-semantic-parent="25" data-semantic-collapsed="(24 (23 20 21) 22)" data-semantic-owns="20 21 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="24"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="italic" data-semantic-id="22" data-semantic-parent="24" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.442em;">𝑒</mjx-c><mjx-c class="mjx-c1D465" noic="true" style="padding-top: 0.442em;">𝑥</mjx-c><mjx-c class="mjx-c1D45D" style="padding-top: 0.442em;">𝑝</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="24"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript i Superscript italic exp" tabindex="0" ctxtmenu_counter="118" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="f Subscript i Superscript italic exp"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.294em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D452" noic="true" style="padding-top: 0.442em;">𝑒</mjx-c><mjx-c class="mjx-c1D465" noic="true" style="padding-top: 0.442em;">𝑥</mjx-c><mjx-c class="mjx-c1D45D" style="padding-top: 0.442em;">𝑝</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> are the experimental constraints defined in <a id="crosref1690" href="#fd6">Equation 6</a> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript i Superscript italic s i m" tabindex="0" ctxtmenu_counter="119" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="f Subscript i Superscript italic s i m"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.292em; margin-left: -0.062em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.163em;"><mjx-c class="mjx-c1D460" noic="true" style="padding-top: 0.661em;">𝑠</mjx-c><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.661em;">𝑖</mjx-c><mjx-c class="mjx-c1D45A" style="padding-top: 0.661em;">𝑚</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> are the corresponding ensemble averages determined from computer simulation. We used 35 iterations for the 100kb-resolution model to reach an error of less than 15%.</div><div id="p0580" role="paragraph">With the converged parameters, we performed additional six independent twenty-million-time-step-long simulations for the 1Mb-resolution model and ten independent four-million-time-step-long simulations for the 100kb-resolution model. A total of 60,000 and 20,000 structures were collected for the 1Mb- and 100kb-resolution model respectively to perform all the analysis presented in the main text.</div></section><section id="sec-4-5-17"><h4>Radial density profile</h4><div id="p0585" role="paragraph">The compartment-specific radial density functions were calculated with the following expression<div id="ufd5" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho left parenthesis r right parenthesis equals StartFraction n left parenthesis r right parenthesis Over 4 pi r squared upper Delta r dot upper N EndFraction" tabindex="0" ctxtmenu_counter="120" style="font-size: 130%;"><mjx-math data-semantic-structure="(31 (30 0 29 (4 1 2 3)) 5 (28 (12 6 11 (10 7 8 9)) (27 (26 13 22 14 23 (17 15 16) 24 18 25 19) 20 21)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="31" data-semantic-children="30,28" data-semantic-content="5" data-semantic-owns="30 5 28" data-semantic-speech="rho left parenthesis r right parenthesis equals StartFraction n left parenthesis r right parenthesis Over 4 pi r squared upper Delta r dot upper N EndFraction"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-children="0,4" data-semantic-content="29,0" data-semantic-parent="31" data-semantic-owns="0 29 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="30" data-semantic-operator="appl"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="29" data-semantic-parent="30" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="30" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="31" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="28" data-semantic-children="12,27" data-semantic-parent="31" data-semantic-owns="12 27"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-children="6,10" data-semantic-content="11,6" data-semantic-parent="28" data-semantic-owns="6 11 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="12" data-semantic-operator="appl"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="7,9" data-semantic-parent="12" data-semantic-owns="7 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="27" data-semantic-children="26,21" data-semantic-content="20" data-semantic-parent="28" data-semantic-owns="26 20 21"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="26" data-semantic-children="13,14,17,18,19" data-semantic-content="22,23,24,25" data-semantic-parent="27" data-semantic-owns="13 22 14 23 17 24 18 25 19"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="26"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="26"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="26" data-semantic-owns="15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.289em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="18" data-semantic-parent="26"><mjx-c class="mjx-c394">Δ</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="26"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="27" data-semantic-operator="infixop,⋅"><mjx-c class="mjx-c22C5">⋅</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="27"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-math></mjx-container></div></div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r" tabindex="0" ctxtmenu_counter="121" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="r"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-math></mjx-container> is the spatial distance from the nuclear center. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n left parenthesis r right parenthesis" tabindex="0" ctxtmenu_counter="122" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 5 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-children="0,4" data-semantic-content="5,0" data-semantic-owns="0 5 4" data-semantic-speech="n left parenthesis r right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="appl"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> is the number of genomic loci of a given compartment type found in the spherical shell from <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r" tabindex="0" ctxtmenu_counter="123" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="r"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-math></mjx-container> to <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r plus upper Delta r" tabindex="0" ctxtmenu_counter="124" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 1 (5 2 4 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="6" data-semantic-children="0,5" data-semantic-content="1" data-semantic-owns="0 1 5" data-semantic-speech="r plus upper Delta r"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="2,3" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="2 4 3"><mjx-mtext data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c394">Δ</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, and the angular brackets indicates an ensemble average over all the simulated genome structures. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N" tabindex="0" ctxtmenu_counter="125" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper N"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-math></mjx-container> is the total number of genomic loci of that given compartment type.</div></section><section id="sec-4-5-18"><h4>Electron Microscopy Analysis</h4><div id="p0590" role="paragraph">Quantifications were performed using Fiji version 2.0.0-rc-69/1.52p by thresholding the 8-bit EM images. The outer border of each nucleus was delineated using the freehand tool and its area and % positive for the heterochromatin threshold was measured by set measurements &gt; limit to Threshold. Next, the same nucleus was measured again with a freehand selection on the inner border of the nucleus, excluding the peripheral heterochromatin (the chromatin touching the nuclear membrane). The percentage internal heterochromatin was calculated by dividing the internal heterochromatin by the area of the internal measurement and multiplied by the total nuclear area and plotted as the percentage of internal over the percentage total heterochromatin. Independent data from three samples with N&nbsp;= 19, 37, 46 nuclei for normal tissues and N&nbsp;= 19, 95, 71 nuclei for tumor tissues were obtained. Visualization of the data and statistical testing were performed using Prism 8 version 8.4.2. The data was imported as a nested dataset and statistical significance was tested using a nested, unpaired, two-sided t test with alpha&nbsp;= 0.05. Data were represented using a frequency distribution plot with percentage internal heterochromatin using bin size 6% on the x axis and the percentage of cells in each of those bins on the y axis.</div></section><section id="sec-4-5-19"><h4>DNA-FISH analysis</h4><div id="p0595" role="paragraph">To calculate redistribution of the A and B compartment in primary tissues, the nuclei for 2D images were manually curated to delineate intact single tumor and colon epithelial nuclei in FIJI version 2.0.0-rc-69/1.52p (N&nbsp;= 2 tumors and 2 normal samples). This step was necessary to avoid generating data from poorly oriented and non-tumor and -colon epithelial nuclei such as immune cells and fibroblasts. As the latter cells have strikingly different nuclear morphologies, we could easily exclude them by visual inspection. Next, the pictures were loaded into Cellprofiler version 3.1.9, the nuclei and compartment spots segmented, and the original channel images masked on the identified DNA-FISH spots. The radial intensity distribution of the masked images was calculated in the nuclei using 20 scaled bins per cell. To determine redistribution of the B compartment in copy number stable tumors toward the nuclear interior, the Fraction at Distance of each masked image bin was plotted in Prism Version 8.4.2. Because the chromosome territory of chromosome 12 is in general peripherally located, bin 1-10 were summed together for their visualization.</div><div id="p0600" role="paragraph">Radial distribution of DNA-FISH in cells was quantified using Cellprofiler version 3.1.9 2 (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">McQuin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>McQuin, C. ∙ Goodman, A. ∙ Chernyshev, V. ...</div><div><strong>CellProfiler 3.0: Next-generation image processing for biology</strong></div><div><em>PLoS Biol.</em> 2018; <strong>16</strong>:e2005970</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.2005970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85051293036" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29969450/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.2005970&amp;pmid=29969450" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), segmenting the nuclei, followed by fill holes and exclusion of cells touching the edge of the image. Next, the radial distribution of each channel (A compartment (A488); B compartment (Cy3) and I compartment (Cy5) was measured using the module MeasureObjectIntensityDistribution using each nucleus as the center of the points. To obtain distributions for each cell’s radial bin in which the maximum of the signal was located, we multiplied the Fraction at Distance for each bin and channel with each Mean Fraction value, and gave a value of 1 to the bin containing the highest value. The counts for each cell and channel were then plotted using Prism Version 8.4.2. Representative images and insets in all panels were generated using FIJI version 2.0.0-rc-69/1.52p.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0605" role="paragraph">We thank Ryanne Boursiquot for assistance with sequencing; Mohammed Miri for assistance with clinical samples; Elizabeth Gaskell, Volker Hovestadt, Christine Eyler and Ryan Corcoran, Angela Shih, and Omer Yilmaz for thoughtful discussions; and Leslie Gaffney for graphic support. S.E.J. is supported by NIH T32CA009216. A.R. and R.A.I. are supported by NIH R01GM083084 and R01HG005220. J.H.C. is supported by NIH 1T32CA207021-01. M.J.A. is supported by a Broad Institute Merkin Fellowship. B.E.B. is the Bernard and Mildred Kayden Endowed MGH Research Institute Chair and an American Cancer Society Research Professor. This research was supported by the National Cancer Institute (DP1CA216873) and the Starr Cancer Consortium. This paper is dedicated to the memory of Yaw Adu Kuffour.</div><section id="sec-5"><h3>Author Contributions</h3><div id="p0610" role="paragraph">Conception and Experimental Design, S.E.J., A.R., M.J.A., and B.E.B.; Methodology and Data Acquisition, S.E.J., A.R., C.A., E.H., K.P., J.H.C., and B.E.B.; Analysis and Interpretation of Data, S.E.J., A.R., Y.Q., C.A., L.S.Z., Y.D., V.H., N.S., M.K.S., C.L., S.I., S.C.N., E.F.J., N.H., R.A.I., B.Z., M.J.A., and B.E.B. Manuscript Writing and Revision, S.E.J., A.R., M.J.A., and B.E.B.</div></section><section id="sec-6"><h3>Declaration of Interests</h3><div id="p0615" role="paragraph">N.H. is an equity holder of BioNTech and a consultant for Related Sciences. M.J.A. declares outside interest in Excelsior Genomics. B.E.B. declares outside interests in Fulcrum Therapeutics, 1CellBio, HiFiBio, Arsenal Biosciences, Cell Signaling Technologies, BioMillenia, and Nohla Therapeutics.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (7)</h2><div id="p0625" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.07.030/attachment/6b2be7e5-4a19-405d-b2a1-a315aa3d3eea/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (12.83 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Summary of Cohort Characteristics and Assays Performed, Related to Figure&nbsp;1</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.07.030/attachment/ff87b797-ab8d-4f4d-a2fa-1f1c1d801363/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (72.81 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. CNVs for the Tumor Cohort, Related to Figures 1 and 2</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.07.030/attachment/c503b538-3ac2-4b1e-9d09-0ffd97849226/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (26.05 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Differential Tumor versus Normal E-P Loops Associated with Changes in Gene Expression (SMC1 HiChIP), Related to Figure&nbsp;1</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.07.030/attachment/4a103af3-f4a5-424f-aedf-79849afc2772/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (16.34 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Colon Cancer Genome-wide Association Study (GWAS) Risk SNPs and Putative Target Genes Based on E-P Loops (SMC1 HiChIP), Related to Figure&nbsp;1</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.07.030/attachment/0ad56b60-e497-4277-85ba-2f6691a2df16/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (12.41 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Compartment I/B Genes Downregulated with Block Hypomethylation, Related to Figure&nbsp;6</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.07.030/attachment/23d05a28-5de0-40c8-adb2-90559edc9ef4/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (32.59 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table S6. Gene Set Enrichment Analysis for Compartment I/B Genes Downregulated with Block Hypomethylation, Related to Figure&nbsp;6</div></div></div><div id="mmc7" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.07.030/attachment/662c5557-e0ec-44c8-8789-8469a155bce9/mmc7.xlsx" download="mmc7.xlsx" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Spreadsheet (20.02 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Table S7. Gene Set Downregulated in Pan-cancer Analysis, Related to Figure&nbsp;6</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Baylin, S.B. ∙ Jones, P.A.</div><div><strong>Epigenetic Determinants of Cancer</strong></div><div><em>Cold Spring Harb. Perspect. Biol.</em> 2016; <strong>8</strong>:a019505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a019505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84986211886" target="_blank" aria-label="Open Scopus in new tab">Scopus (303)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27194046/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a019505&amp;pmid=27194046" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Beliveau, B.J. ∙ Joyce, E.F. ∙ Apostolopoulos, N. ...</div><div><strong>Versatile design and synthesis platform for visualizing genomes with Oligopaint FISH probes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:21301-21306</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1213818110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84871822966" target="_blank" aria-label="Open Scopus in new tab">Scopus (172)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23236188/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1213818110&amp;pmid=23236188" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Berman, B.P. ∙ Weisenberger, D.J. ∙ Aman, J.F. ...</div><div><strong>Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains</strong></div><div><em>Nat. Genet.</em> 2011; <strong>44</strong>:40-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.969" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84655162785" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22120008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.969&amp;pmid=22120008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Bickmore, W.A. ∙ van Steensel, B.</div><div><strong>Genome architecture: domain organization of interphase chromosomes</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:1270-1284</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2013.02.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2013.02.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84875190221" target="_blank" aria-label="Open Scopus in new tab">Scopus (448)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23498936/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.02.001&amp;pmid=23498936" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Boettiger, A.N. ∙ Bintu, B. ∙ Moffitt, J.R. ...</div><div><strong>Super-resolution imaging reveals distinct chromatin folding for different epigenetic states</strong></div><div><em>Nature.</em> 2016; <strong>529</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16496" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84955290042" target="_blank" aria-label="Open Scopus in new tab">Scopus (455)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26760202/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16496&amp;pmid=26760202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Bolzer, A. ∙ Kreth, G. ∙ Solovei, I. ...</div><div><strong>Three-dimensional maps of all chromosomes in human male fibroblast nuclei and prometaphase rosettes</strong></div><div><em>PLoS Biol.</em> 2005; <strong>3</strong>:e157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0030157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-21844444077" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15839726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0030157&amp;pmid=15839726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Calon, A. ∙ Lonardo, E. ∙ Berenguer-Llergo, A. ...</div><div><strong>Stromal gene expression defines poor-prognosis subtypes in colorectal cancer</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:320-329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3225" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84925818452" target="_blank" aria-label="Open Scopus in new tab">Scopus (507)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25706628/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3225&amp;pmid=25706628" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Cancer Genome Atlas Network</div><div><strong>Comprehensive molecular characterization of human colon and rectal cancer</strong></div><div><em>Nature.</em> 2012; <strong>487</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84863922124" target="_blank" aria-label="Open Scopus in new tab">Scopus (4694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22810696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11252&amp;pmid=22810696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Cao, Y. ∙ Chen, Z. ∙ Chen, X. ...</div><div><strong>Accurate loop calling for 3D genomic data with cLoops</strong></div><div><em>Bioinformatics.</em> 2019; <strong>36</strong>:666-675</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.CaoZ.ChenX.ChenD.AiG.ChenJ.McDermottY.HuangX.GuoJ.-D.J.HanAccurate+loop+calling+for+3D+genomic+data+with+cLoopsBioinformatics362019666675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Chandra, T. ∙ Narita, M.</div><div><strong>High-order chromatin structure and the epigenome in SAHFs</strong></div><div><em>Nucleus.</em> 2013; <strong>4</strong>:23-28</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/nucl.23189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84873555534" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23232545/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Fnucl.23189&amp;pmid=23232545" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Colaprico, A. ∙ Silva, T.C. ∙ Olsen, C. ...</div><div><strong>TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e71</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84966269471" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26704973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1507&amp;pmid=26704973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Collado-Torres, L. ∙ Nellore, A. ∙ Kammers, K. ...</div><div><strong>Reproducible RNA-seq analysis using recount2</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:319-321</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85017363752" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28398307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3838&amp;pmid=28398307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Corces, M.R. ∙ Corces, V.G.</div><div><strong>The three-dimensional cancer genome</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2016; <strong>36</strong>:1-7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gde.2016.01.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84956693585" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26855137/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gde.2016.01.002&amp;pmid=26855137" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Criscione, S.W. ∙ De Cecco, M. ∙ Siranosian, B. ...</div><div><strong>Reorganization of chromosome architecture in replicative cellular senescence</strong></div><div><em>Sci. Adv.</em> 2016; <strong>2</strong>:e1500882</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1500882" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84964957467" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26989773/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1500882&amp;pmid=26989773" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Cruickshanks, H.A. ∙ McBryan, T. ∙ Nelson, D.M. ...</div><div><strong>Senescent cells harbour features of the cancer epigenome</strong></div><div><em>Nat. Cell Biol.</em> 2013; <strong>15</strong>:1495-1506</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb2879" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84887945557" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24270890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb2879&amp;pmid=24270890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Dekker, J. ∙ Misteli, T.</div><div><strong>Long-Range Chromatin Interactions</strong></div><div><em>Cold Spring Harb. Perspect. Biol.</em> 2015; <strong>7</strong>:a019356</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a019356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84943250456" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26430217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a019356&amp;pmid=26430217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Di Pierro, M. ∙ Zhang, B. ∙ Aiden, E.L. ...</div><div><strong>Transferable model for chromosome architecture</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:12168-12173</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1613607113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84992395910" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27688758/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1613607113&amp;pmid=27688758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Dias-Santagata, D. ∙ Akhavanfard, S. ∙ David, S.S. ...</div><div><strong>Rapid targeted mutational analysis of human tumours: a clinical platform to guide personalized cancer medicine</strong></div><div><em>EMBO Mol. Med.</em> 2010; <strong>2</strong>:146-158</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/emmm.201000070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-77954566882" target="_blank" aria-label="Open Scopus in new tab">Scopus (325)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20432502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Femmm.201000070&amp;pmid=20432502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Dixon, J.R. ∙ Selvaraj, S. ∙ Yue, F. ...</div><div><strong>Topological domains in mammalian genomes identified by analysis of chromatin interactions</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:376-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84861095603" target="_blank" aria-label="Open Scopus in new tab">Scopus (3066)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11082&amp;pmid=22495300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Dixon, J.R. ∙ Jung, I. ∙ Selvaraj, S. ...</div><div><strong>Chromatin architecture reorganization during stem cell differentiation</strong></div><div><em>Nature.</em> 2015; <strong>518</strong>:331-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14222" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84923366733" target="_blank" aria-label="Open Scopus in new tab">Scopus (698)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25693564/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14222&amp;pmid=25693564" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Dunne, P.D. ∙ Dasgupta, S. ∙ Blayney, J.K. ...</div><div><strong>EphA2 Expression Is a Key Driver of Migration and Invasion and a Poor Prognostic Marker in Colorectal Cancer</strong></div><div><em>Clin. Cancer Res.</em> 2016; <strong>22</strong>:230-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-15-0603" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84954526582" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26283684/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-15-0603&amp;pmid=26283684" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Durand, N.C. ∙ Robinson, J.T. ∙ Shamim, M.S. ...</div><div><strong>Juicebox Provides a Visualization System for Hi-C Contact Maps with Unlimited Zoom</strong></div><div><em>Cell Syst.</em> 2016; <strong>3</strong>:99-101</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cels.2015.07.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cels.2015.07.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27467250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.07.012&amp;pmid=27467250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23-1" title="View in article">23.</a></div><div>Falk, M. ∙ Feodorova, Y. ∙ Naumova, N. ...</div><div><strong>Heterochromatin drives compartmentalization of inverted and conventional nuclei</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:395-399</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1275-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85067277322" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1275-3&amp;pmid=31168090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">24.</a></div><div>Fan, J. ∙ Li, J. ∙ Guo, S. ...</div><div><strong>Genome-wide DNA methylation profiles of low- and high-grade adenoma reveals potential biomarkers for early detection of colorectal carcinoma</strong></div><div><em>Clin. Epigenetics.</em> 2020; <strong>12</strong>:56</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13148-020-00851-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85083871857" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32317010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13148-020-00851-3&amp;pmid=32317010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">25.</a></div><div>Flavahan, W.A. ∙ Drier, Y. ∙ Liau, B.B. ...</div><div><strong>Insulator dysfunction and oncogene activation in IDH mutant gliomas</strong></div><div><em>Nature.</em> 2016; <strong>529</strong>:110-114</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16490" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84953924597" target="_blank" aria-label="Open Scopus in new tab">Scopus (541)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26700815/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16490&amp;pmid=26700815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">26.</a></div><div>Flavahan, W.A. ∙ Drier, Y. ∙ Johnstone, S.E. ...</div><div><strong>Altered chromosomal topology drives oncogenic programs in SDH-deficient GISTs</strong></div><div><em>Nature.</em> 2019; <strong>575</strong>:229-233</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1668-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85074865462" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31666694/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1668-3&amp;pmid=31666694" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Fortin, J.-P. ∙ Hansen, K.D.</div><div><strong>Reconstructing A/B compartments as revealed by Hi-C using long-range correlations in epigenetic data</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:180</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0741-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84940090291" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26316348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0741-y&amp;pmid=26316348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>Fotheringham, S. ∙ Mozolowski, G.A. ∙ Murray, E.M.A. ...</div><div><strong>Challenges and solutions in patient treatment strategies for stage II colon cancer</strong></div><div><em>Gastroenterol. Rep. (Oxf.).</em> 2019; <strong>7</strong>:151-161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/gastro/goz006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85071096116" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31217978/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgastro%2Fgoz006&amp;pmid=31217978" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Gibbs, Z.A. ∙ Whitehurst, A.W.</div><div><strong>Emerging Contributions of Cancer/Testis Antigens to Neoplastic Behaviors</strong></div><div><em>Trends Cancer.</em> 2018; <strong>4</strong>:701-712</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.trecan.2018.08.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.trecan.2018.08.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85053719310" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30292353/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.trecan.2018.08.005&amp;pmid=30292353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Gladstein, S. ∙ Damania, D. ∙ Almassalha, L.M. ...</div><div><strong>Correlating colorectal cancer risk with field carcinogenesis progression using partial wave spectroscopic microscopy</strong></div><div><em>Cancer Med.</em> 2018; <strong>7</strong>:2109-2120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cam4.1357" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85044297841" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29573208/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcam4.1357&amp;pmid=29573208" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Gusnanto, A. ∙ Wood, H.M. ∙ Pawitan, Y. ...</div><div><strong>Correcting for cancer genome size and tumour cell content enables better estimation of copy number alterations from next-generation sequence data</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:40-47</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr593" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84855185866" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22039209/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr593&amp;pmid=22039209" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>Hansen, K.D. ∙ Timp, W. ∙ Bravo, H.C. ...</div><div><strong>Increased methylation variation in epigenetic domains across cancer types</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:768-775</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.865" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79960927422" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21706001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.865&amp;pmid=21706001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Hansen, K.D. ∙ Langmead, B. ∙ Irizarry, R.A.</div><div><strong>BSmooth: from whole genome bisulfite sequencing reads to differentially methylated regions</strong></div><div><em>Genome Biol.</em> 2012; <strong>13</strong>:R83</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2012-13-10-r83" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23034175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2012-13-10-r83&amp;pmid=23034175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Hinoue, T. ∙ Weisenberger, D.J. ∙ Lange, C.P.E. ...</div><div><strong>Genome-scale analysis of aberrant DNA methylation in colorectal cancer</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:271-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.117523.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84856549042" target="_blank" aria-label="Open Scopus in new tab">Scopus (443)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21659424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.117523.110&amp;pmid=21659424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">35.</a></div><div>Hnisz, D. ∙ Weintraub, A.S. ∙ Day, D.S. ...</div><div><strong>Activation of proto-oncogenes by disruption of chromosome neighborhoods</strong></div><div><em>Science.</em> 2016; <strong>351</strong>:1454-1458</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad9024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84960171833" target="_blank" aria-label="Open Scopus in new tab">Scopus (425)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26940867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad9024&amp;pmid=26940867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Hon, G.C. ∙ Hawkins, R.D. ∙ Caballero, O.L. ...</div><div><strong>Global DNA hypomethylation coupled to repressive chromatin domain formation and gene silencing in breast cancer</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:246-258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.125872.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84856528270" target="_blank" aria-label="Open Scopus in new tab">Scopus (337)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22156296/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.125872.111&amp;pmid=22156296" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Huang, J. ∙ Gusnanto, A. ∙ O’Sullivan, K. ...</div><div><strong>Robust smooth segmentation approach for array CGH data analysis</strong></div><div><em>Bioinformatics.</em> 2007; <strong>23</strong>:2463-2469</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btm359" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-34548725570" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17660206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtm359&amp;pmid=17660206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Huber, W. ∙ Carey, V.J. ∙ Gentleman, R. ...</div><div><strong>Orchestrating high-throughput genomic analysis with Bioconductor</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:115-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84961289551" target="_blank" aria-label="Open Scopus in new tab">Scopus (1290)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25633503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3252&amp;pmid=25633503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#figs7" title="View in article">39.</a></div><div>ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium</div><div><strong>Pan-cancer analysis of whole genomes</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:82-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-1969-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85079038817" target="_blank" aria-label="Open Scopus in new tab">Scopus (313)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32025007/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-1969-6&amp;pmid=32025007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Imakaev, M. ∙ Fudenberg, G. ∙ McCord, R.P. ...</div><div><strong>Iterative correction of Hi-C data reveals hallmarks of chromosome organization</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:999-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2148" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84866997011" target="_blank" aria-label="Open Scopus in new tab">Scopus (536)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22941365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2148&amp;pmid=22941365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">41.</a></div><div>Kloetgen, A. ∙ Thandapani, P. ∙ Ntziachristos, P. ...</div><div><strong>Three-dimensional chromatin landscapes in T&nbsp;cell acute lymphoblastic leukemia</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:388-400</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0602-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85082197331" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32203470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0602-9&amp;pmid=32203470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Koveitypour, Z. ∙ Panahi, F. ∙ Vakilian, M. ...</div><div><strong>Signaling pathways involved in colorectal cancer progression</strong></div><div><em>Cell Biosci.</em> 2019; <strong>9</strong>:97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13578-019-0361-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85076212909" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31827763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13578-019-0361-4&amp;pmid=31827763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Krefting, J. ∙ Andrade-Navarro, M.A. ∙ Ibn-Salem, J.</div><div><strong>Evolutionary stability of topologically associating domains is associated with conserved gene regulation</strong></div><div><em>BMC Biol.</em> 2018; <strong>16</strong>:87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12915-018-0556-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85058139752" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30086749/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12915-018-0556-x&amp;pmid=30086749" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Lareau, C.A. ∙ Aryee, M.J.</div><div><strong>diffloop: a computational framework for identifying and analyzing differential DNA loops from sequencing data</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:672-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btx623" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85042523227" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29028898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtx623&amp;pmid=29028898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Lareau, C.A. ∙ Aryee, M.J.</div><div><strong>hichipper: a preprocessing pipeline for calling DNA loops from HiChIP data</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:155-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4583" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85042766402" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29489746/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4583&amp;pmid=29489746" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Larson, A.G. ∙ Elnatan, D. ∙ Keenen, M.M. ...</div><div><strong>Liquid droplet formation by HP1α suggests a role for phase separation in heterochromatin</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:236-240</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85024372369" target="_blank" aria-label="Open Scopus in new tab">Scopus (520)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28636604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22822&amp;pmid=28636604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Law, C.W. ∙ Chen, Y. ∙ Shi, W. ...</div><div><strong>voom: Precision weights unlock linear model analysis tools for RNA-seq read counts</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:R29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2014-15-2-r29" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84896735766" target="_blank" aria-label="Open Scopus in new tab">Scopus (1883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24485249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2014-15-2-r29&amp;pmid=24485249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Lawrence, M. ∙ Huber, W. ∙ Pagès, H. ...</div><div><strong>Software for computing and annotating genomic ranges</strong></div><div><em>PLoS Comput. Biol.</em> 2013; <strong>9</strong>:e1003118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84883368195" target="_blank" aria-label="Open Scopus in new tab">Scopus (1187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23950696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003118&amp;pmid=23950696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (19639)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">50.</a></div><div>Liau, B.B. ∙ Sievers, C. ∙ Donohue, L.K. ...</div><div><strong>Adaptive Chromatin Remodeling Drives Glioblastoma Stem Cell Plasticity and Drug Tolerance</strong></div><div><em>Cell Stem Cell.</em> 2017; <strong>20</strong>:233-246.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.stem.2016.11.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.stem.2016.11.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85008406243" target="_blank" aria-label="Open Scopus in new tab">Scopus (155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27989769/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2016.11.003&amp;pmid=27989769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">51.</a></div><div>Lieberman-Aiden, E. ∙ van Berkum, N.L. ∙ Williams, L. ...</div><div><strong>Comprehensive mapping of long-range interactions reveals folding principles of the human genome</strong></div><div><em>Science.</em> 2009; <strong>326</strong>:289-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1181369" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-70349873824" target="_blank" aria-label="Open Scopus in new tab">Scopus (3713)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19815776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1181369&amp;pmid=19815776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">52.</a></div><div>Lima-Fernandes, E. ∙ Murison, A. ∙ da Silva Medina, T. ...</div><div><strong>Targeting bivalency de-represses Indian Hedgehog and inhibits self-renewal of colorectal cancer-initiating cells</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:1436</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09309-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85063726627" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30926792/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09309-4&amp;pmid=30926792" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">53.</a></div><div>Liu, Y. ∙ Sun, J. ∙ Zhao, M.</div><div><strong>ONGene: A literature-based database for human oncogenes</strong></div><div><em>J.&nbsp;Genet. Genomics.</em> 2017; <strong>44</strong>:119-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jgg.2016.12.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85011271324" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28162959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jgg.2016.12.004&amp;pmid=28162959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">54.</a></div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (16659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Love, M.I. ∙ Anders, S. ∙ Kim, V. ...</div><div><strong>RNA-Seq workflow: gene-level exploratory analysis and differential expression</strong></div><div><em>F1000Res.</em> 2015; <strong>4</strong>:1070</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.12688/f1000research.7035.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26674615/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.12688%2Ff1000research.7035.1&amp;pmid=26674615" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Lun, A.T.L. ∙ Smyth, G.K.</div><div><strong>csaw: a Bioconductor package for differential binding analysis of ChIP-seq data using sliding windows</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1191" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84963820955" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26578583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1191&amp;pmid=26578583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>MacArthur, J. ∙ Bowler, E. ∙ Cerezo, M. ...</div><div><strong>The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog)</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:D896-D901</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw1133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85016161935" target="_blank" aria-label="Open Scopus in new tab">Scopus (937)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899670/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw1133&amp;pmid=27899670" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#figs7" title="View in article">58.</a></div><div>Marisa, L. ∙ de Reyniès, A. ∙ Duval, A. ...</div><div><strong>Gene expression classification of colon cancer into molecular subtypes: characterization, validation, and prognostic value</strong></div><div><em>PLoS Med.</em> 2013; <strong>10</strong>:e1001453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pmed.1001453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84878302099" target="_blank" aria-label="Open Scopus in new tab">Scopus (643)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23700391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pmed.1001453&amp;pmid=23700391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">59.</a></div><div>McDonald, O.G. ∙ Wu, H. ∙ Timp, W. ...</div><div><strong>Genome-scale epigenetic reprogramming during epithelial-to-mesenchymal transition</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2011; <strong>18</strong>:867-874</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.2084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79961023271" target="_blank" aria-label="Open Scopus in new tab">Scopus (249)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21725293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.2084&amp;pmid=21725293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">60.</a></div><div>McQuin, C. ∙ Goodman, A. ∙ Chernyshev, V. ...</div><div><strong>CellProfiler 3.0: Next-generation image processing for biology</strong></div><div><em>PLoS Biol.</em> 2018; <strong>16</strong>:e2005970</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.2005970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85051293036" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29969450/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.2005970&amp;pmid=29969450" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">61.</a></div><div>Modrek, A.S. ∙ Golub, D. ∙ Khan, T. ...</div><div><strong>Low-Grade Astrocytoma Mutations in IDH1, P53, and ATRX Cooperate to Block Differentiation of Human Neural Stem Cells via Repression of SOX2</strong></div><div><em>Cell Rep.</em> 2017; <strong>21</strong>:1267-1280</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2017.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2017.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85034835338" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29091765/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2017.10.009&amp;pmid=29091765" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">62.</a></div><div>Mumbach, M.R. ∙ Rubin, A.J. ∙ Flynn, R.A. ...</div><div><strong>HiChIP: efficient and sensitive analysis of protein-directed genome architecture</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:919-922</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84988336216" target="_blank" aria-label="Open Scopus in new tab">Scopus (469)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27643841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3999&amp;pmid=27643841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Nguyen, L.H. ∙ Goel, A. ∙ Chung, D.C.</div><div><strong>Pathways of Colorectal Carcinogenesis</strong></div><div><em>Gastroenterology.</em> 2020; <strong>158</strong>:291-302</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1053%2Fj.gastro.2019.08.059&amp;cf=fulltext&amp;site=ygast-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1053%2Fj.gastro.2019.08.059&amp;cf=pdf&amp;site=ygast-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31622622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1053%2Fj.gastro.2019.08.059&amp;pmid=31622622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">64.</a></div><div>Nora, E.P. ∙ Lajoie, B.R. ∙ Schulz, E.G. ...</div><div><strong>Spatial partitioning of the regulatory landscape of the X-inactivation centre</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:381-385</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84861100147" target="_blank" aria-label="Open Scopus in new tab">Scopus (1429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495304/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11049&amp;pmid=22495304" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>Nora, E.P. ∙ Goloborodko, A. ∙ Valton, A.-L. ...</div><div><strong>Targeted Degradation of CTCF Decouples Local Insulation of Chromosome Domains from Genomic Compartmentalization</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:930-944.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2017.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2017.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85019568830" target="_blank" aria-label="Open Scopus in new tab">Scopus (517)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28525758/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.004&amp;pmid=28525758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">66.</a></div><div>Nordor, A.V. ∙ Nehar-Belaid, D. ∙ Richon, S. ...</div><div><strong>The early pregnancy placenta foreshadows DNA methylation alterations of solid tumors</strong></div><div><em>Epigenetics.</em> 2017; <strong>12</strong>:793-803</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/15592294.2017.1342912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85034848501" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F15592294.2017.1342912&amp;pmid=28678605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">67.</a></div><div>Olshen, A.B. ∙ Venkatraman, E.S. ∙ Lucito, R. ...</div><div><strong>Circular binary segmentation for the analysis of array-based DNA copy number data</strong></div><div><em>Biostatistics.</em> 2004; <strong>5</strong>:557-572</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/biostatistics/kxh008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-3543105225" target="_blank" aria-label="Open Scopus in new tab">Scopus (1458)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15475419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbiostatistics%2Fkxh008&amp;pmid=15475419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">68.</a></div><div>Patro, R. ∙ Duggal, G. ∙ Love, M.I. ...</div><div><strong>Salmon provides fast and bias-aware quantification of transcript expression</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:417-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4197" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85014549629" target="_blank" aria-label="Open Scopus in new tab">Scopus (1602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28263959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4197&amp;pmid=28263959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">69.</a></div><div>Peltekova, V.D. ∙ Lemire, M. ∙ Qazi, A.M. ...</div><div><strong>Identification of genes expressed by immune cells of the colon that are regulated by colorectal cancer-associated variants</strong></div><div><em>Int. J. Cancer.</em> 2014; <strong>134</strong>:2330-2341</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ijc.28557" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84895930259" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24154973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fijc.28557&amp;pmid=24154973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">70.</a></div><div>Plimpton, S.</div><div><strong>Fast parallel algorithms for short-range molecular dynamics</strong></div><div><em>J.&nbsp;Comput. Phys.</em> 1995; <strong>117</strong>:1-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jcph.1995.1039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-0002467378" target="_blank" aria-label="Open Scopus in new tab">Scopus (23779)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjcph.1995.1039" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="optmEKNnOQn6J" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optmEKNnOQn6J" title="View in article">71.</a></div><div>Qi, Y. ∙ Reyes, A. ∙ Johnstone, S.E. ...</div><div><strong>Data-driven polymer model for mechanistic exploration of diploid genome organization</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.02.27.968735" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85098899239" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32511383/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.02.27.968735&amp;pmid=32511383" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71-1" title="View in article">72.</a></div><div>Qi, Y. ∙ Zhang, B.</div><div><strong>Predicting three-dimensional genome organization with chromatin states</strong></div><div><em>PLoS Comput. Biol.</em> 2019; <strong>15</strong>:e1007024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85068537423" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31181064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1007024&amp;pmid=31181064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">73.</a></div><div>Rao, S.S.P. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">74.</a></div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (13800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">75.</a></div><div>Rooney, M.S. ∙ Shukla, S.A. ∙ Wu, C.J. ...</div><div><strong>Molecular and genetic properties of tumors associated with local immune cytolytic activity</strong></div><div><em>Cell.</em> 2015; <strong>160</strong>:48-61</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.12.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2014.12.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84920962202" target="_blank" aria-label="Open Scopus in new tab">Scopus (1231)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25594174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.12.033&amp;pmid=25594174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">76.</a></div><div>Rosin, L.F. ∙ Nguyen, S.C. ∙ Joyce, E.F.</div><div><strong>Condensin II drives large-scale folding and spatial partitioning of interphase chromosomes in Drosophila nuclei</strong></div><div><em>PLoS Genet.</em> 2018; <strong>14</strong>:e1007393</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1007393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85050986036" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30001329/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1007393&amp;pmid=30001329" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">77.</a></div><div>Roulois, D. ∙ Loo Yau, H. ∙ Singhania, R. ...</div><div><strong>DNA-Demethylating Agents Target Colorectal Cancer Cells by Inducing Viral Mimicry by Endogenous Transcripts</strong></div><div><em>Cell.</em> 2015; <strong>162</strong>:961-973</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2015.07.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2015.07.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84940403834" target="_blank" aria-label="Open Scopus in new tab">Scopus (514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26317465/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.07.056&amp;pmid=26317465" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">78.</a></div><div>Rowley, M.J. ∙ Corces, V.G.</div><div><strong>Organizational principles of 3D genome architecture</strong></div><div><em>Nat. Rev. Genet.</em> 2018; <strong>19</strong>:789-800</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-018-0060-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85055726389" target="_blank" aria-label="Open Scopus in new tab">Scopus (237)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30367165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-018-0060-8&amp;pmid=30367165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">79.</a></div><div>Sakthivel, K.M. ∙ Sehgal, P.</div><div><strong>A novel role of lamins from genetic disease to cancer biomarkers</strong></div><div><em>Oncol. Rev.</em> 2016; <strong>10</strong>:309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4081/oncol.2016.309" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85002763692" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27994771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4081%2Foncol.2016.309&amp;pmid=27994771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79-1" title="View in article">80.</a></div><div>Sati, S. ∙ Bonev, B. ∙ Szabo, Q. ...</div><div><strong>4D Genome Rewiring during Oncogene-Induced and Replicative Senescence</strong></div><div><em>Mol. Cell.</em> 2020; <strong>78</strong>:522-538.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.molcel.2020.03.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.molcel.2020.03.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85083005663" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32220303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2020.03.007&amp;pmid=32220303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">81.</a></div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (19328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81-1" title="View in article">82.</a></div><div>Schmitt, A.D. ∙ Hu, M. ∙ Jung, I. ...</div><div><strong>A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome</strong></div><div><em>Cell Rep.</em> 2016; <strong>17</strong>:2042-2059</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85003815206" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27851967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2016.10.061&amp;pmid=27851967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">83.</a></div><div>Schreiber, K.H. ∙ Kennedy, B.K.</div><div><strong>When lamins go bad: nuclear structure and disease</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:1365-1375</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2013.02.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2013.02.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84875175486" target="_blank" aria-label="Open Scopus in new tab">Scopus (232)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23498943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.02.015&amp;pmid=23498943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">84.</a></div><div>Serra, F. ∙ Baù, D. ∙ Goodstadt, M. ...</div><div><strong>Automatic analysis and 3D-modelling of Hi-C data using TADbit reveals structural features of the fly chromatin colors</strong></div><div><em>PLoS Comput. Biol.</em> 2017; <strong>13</strong>:e1005665</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1005665" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85026659339" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28723903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1005665&amp;pmid=28723903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84-1" title="View in article">85.</a></div><div>Servant, N. ∙ Varoquaux, N. ∙ Lajoie, B.R. ...</div><div><strong>HiC-Pro: an optimized and flexible pipeline for Hi-C data processing</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0831-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84952861819" target="_blank" aria-label="Open Scopus in new tab">Scopus (450)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0831-x&amp;pmid=26619908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">86.</a></div><div>Shav-Tal, Y.</div><div><strong>Imaging Gene Expression</strong></div><div>Springer, 2013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-62703-526-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-62703-526-2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">87.</a></div><div>Smit, A. ∙ Hubley, R. ∙ Green, P.</div><div><strong>RepeatMasker Open-4.0. 2013–2015</strong></div><div>RepeatMasker, 2015</div><div><a id="interref0115" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.SmitR.HubleyP.GreenRepeatMasker+Open-4.0.+2013%E2%80%9320152015RepeatMaskerhttp%3A%2F%2Fwww.repeatmasker.org" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">88.</a></div><div>Stadhouders, R. ∙ Filion, G.J. ∙ Graf, T.</div><div><strong>Transcription factors and 3D genome conformation in cell-fate decisions</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:345-354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1182-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85065766396" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31092938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1182-7&amp;pmid=31092938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">89.</a></div><div>Strom, A.R. ∙ Emelyanov, A.V. ∙ Mir, M. ...</div><div><strong>Phase separation drives heterochromatin domain formation</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:241-245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22989" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85024405877" target="_blank" aria-label="Open Scopus in new tab">Scopus (559)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28636597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22989&amp;pmid=28636597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">90.</a></div><div>Teng, M. ∙ Irizarry, R.A.</div><div><strong>Accounting for GC-content bias reduces systematic errors and batch effects in ChIP-seq data</strong></div><div><em>Genome Res.</em> 2017; <strong>27</strong>:1930-1938</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.220673.117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85046699965" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29025895/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.220673.117&amp;pmid=29025895" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">91.</a></div><div>Thorsson, V. ∙ Gibbs, D.L. ∙ Brown, S.D. ..., Cancer Genome Atlas Research Network</div><div><strong>The Immune Landscape of Cancer</strong></div><div><em>Immunity.</em> 2019; <strong>51</strong>:411-412</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.immuni.2019.08.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85070213901" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31433971/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2019.08.004&amp;pmid=31433971" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">92.</a></div><div>Timp, W. ∙ Bravo, H.C. ∙ McDonald, O.G. ...</div><div><strong>Large hypomethylated blocks as a universal defining epigenetic alteration in human solid tumors</strong></div><div><em>Genome Med.</em> 2014; <strong>6</strong>:61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-014-0061-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84906837231" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25191524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-014-0061-y&amp;pmid=25191524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">93.</a></div><div>Toyota, M. ∙ Ahuja, N. ∙ Ohe-Toyota, M. ...</div><div><strong>CpG island methylator phenotype in colorectal cancer</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1999; <strong>96</strong>:8681-8686</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.96.15.8681" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-0033587747" target="_blank" aria-label="Open Scopus in new tab">Scopus (1985)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10411935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.96.15.8681&amp;pmid=10411935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">94.</a></div><div>van Steensel, B. ∙ Belmont, A.S.</div><div><strong>Lamina-Associated Domains: Links with Chromosome Architecture, Heterochromatin, and Gene Repression</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:780-791</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2017.04.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.cell.2017.04.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85019578993" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28525751/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.04.022&amp;pmid=28525751" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">95.</a></div><div>Wang, S. ∙ Su, J.H. ∙ Beliveau, B.J. ...</div><div><strong>Spatial organization of chromatin domains and compartments in single chromosomes</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:598-602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf8084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84980417141" target="_blank" aria-label="Open Scopus in new tab">Scopus (285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27445307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf8084&amp;pmid=27445307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">96.</a></div><div>Wang, Q. ∙ Zhu, J. ∙ Wang, Y.-W. ...</div><div><strong>Tumor suppressor Pdcd4 attenuates Sin1 translation to inhibit invasion in colon carcinoma</strong></div><div><em>Oncogene.</em> 2017; <strong>36</strong>:6225-6234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/onc.2017.228" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85033585599" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28692058/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fonc.2017.228&amp;pmid=28692058" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#fig6" title="View in article">97.</a></div><div>Wang, T. ∙ Maden, S.K. ∙ Luebeck, G.E. ...</div><div><strong>Dysfunctional epigenetic aging of the normal colon and colorectal cancer risk</strong></div><div><em>Clin. Epigenetics.</em> 2020; <strong>12</strong>:5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13148-019-0801-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85077479949" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31900199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13148-019-0801-3&amp;pmid=31900199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">98.</a></div><div>Xi, Y. ∙ Li, W.</div><div><strong>BSMAP: whole genome bisulfite sequence MAPping program</strong></div><div><em>BMC Bioinformatics.</em> 2009; <strong>10</strong>:232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-10-232" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-68949141599" target="_blank" aria-label="Open Scopus in new tab">Scopus (510)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19635165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-10-232&amp;pmid=19635165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">99.</a></div><div>Xiong, K. ∙ Ma, J.</div><div><strong>Revealing Hi-C subcompartments by imputing inter-chromosomal chromatin interactions</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:5069</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12954-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85074716041" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31699985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12954-4&amp;pmid=31699985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">100.</a></div><div>Xu, Y. ∙ Hu, B. ∙ Choi, A.J. ...</div><div><strong>Unique DNA methylome profiles in CpG island methylator phenotype colon cancers</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:283-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.122788.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84863011458" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21990380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.122788.111&amp;pmid=21990380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">101.</a></div><div>Xu, J. ∙ Ma, H. ∙ Jin, J. ...</div><div><strong>Super-Resolution Imaging of Higher-Order Chromatin Structures at Different Epigenomic States in Single Mammalian Cells</strong></div><div><em>Cell Rep.</em> 2018; <strong>24</strong>:873-882</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2018.06.085&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1016%2Fj.celrep.2018.06.085&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85049905839" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30044984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.06.085&amp;pmid=30044984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">102.</a></div><div>Yuan, W. ∙ Xu, M. ∙ Huang, C. ...</div><div><strong>H3K36 methylation antagonizes PRC2-mediated H3K27 methylation</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2011; <strong>286</strong>:7983-7989</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1074%2Fjbc.M110.194027&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=10.1074%2Fjbc.M110.194027&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-79953143753" target="_blank" aria-label="Open Scopus in new tab">Scopus (293)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21239496/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M110.194027&amp;pmid=21239496" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">103.</a></div><div>Zhang, B. ∙ Wolynes, P.G.</div><div><strong>Topology, structures, and energy landscapes of human chromosomes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:6062-6067</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1506257112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-84929190426" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25918364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1506257112&amp;pmid=25918364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">104.</a></div><div>Zhang, B. ∙ Wolynes, P.G.</div><div><strong>Shape Transitions and Chiral Symmetry Breaking in the Energy Landscape of the Mitotic Chromosome</strong></div><div><em>Phys. Rev. Lett.</em> 2016; <strong>116</strong>:248101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.116.248101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27367409/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.116.248101&amp;pmid=27367409" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">105.</a></div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (6187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105-1" title="View in article">106.</a></div><div>Zhou, W. ∙ Dinh, H.Q. ∙ Ramjan, Z. ...</div><div><strong>DNA methylation loss in late-replicating domains is linked to mitotic cell division</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:591-602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0073-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-85044740324" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29610480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0073-4&amp;pmid=29610480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">107.</a></div><div>Zink, D. ∙ Fischer, A.H. ∙ Nickerson, J.A.</div><div><strong>Nuclear structure in cancer cells</strong></div><div><em>Nat. Rev. Cancer.</em> 2004; <strong>4</strong>:677-687</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrc1430" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.07.030&amp;key=2-s2.0-4544237763" target="_blank" aria-label="Open Scopus in new tab">Scopus (636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15343274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrc1430&amp;pmid=15343274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/c0931a50-3b49-4d0a-b3e3-b0050d98c02f/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/d7af530c-4545-4d2f-aa8a-4368752e0136/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/150c1482-2aa6-4529-aefb-6e18a9d108b7/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/361f8a5c-6595-4fb2-8825-b0848822272d/main.assets/figs3.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/045cca42-393b-45e3-a01c-62f21ce16cd5/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/d50bc35f-7fea-4aef-b993-dd0a26d3e53b/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/f80ca9a0-13a3-45fb-94dc-4e3fb5605aff/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/2a0a27db-75b7-447b-92ad-5c4202c5449d/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/4a95a970-e2d6-48f4-bbf2-07326d39dd87/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/44999e54-071b-4aa6-898b-08467cee6fad/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/21fb751d-a142-4952-b31f-25c27859e716/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/090d1720-ea18-450b-b6ee-1f1fea459a10/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/3cb379c7-1441-4046-beee-05c4bf0168fc/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/asset/7966fd35-72ed-4b4d-9f80-ac4755bb004b/main.assets/gr7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00203%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3Apii%5C%3AS0092867420309387" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">130</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">294</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">1</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">250</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2020.07.030&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (7)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/attachment/6b2be7e5-4a19-405d-b2a1-a315aa3d3eea/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (12.83 KB)</span></a><div class="core-description">Table S1. Summary of Cohort Characteristics and Assays Performed, Related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/attachment/ff87b797-ab8d-4f4d-a2fa-1f1c1d801363/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (72.81 KB)</span></a><div class="core-description">Table S2. CNVs for the Tumor Cohort, Related to Figures 1 and 2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/attachment/c503b538-3ac2-4b1e-9d09-0ffd97849226/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (26.05 KB)</span></a><div class="core-description">Table S3. Differential Tumor versus Normal E-P Loops Associated with Changes in Gene Expression (SMC1 HiChIP), Related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/attachment/4a103af3-f4a5-424f-aedf-79849afc2772/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (16.34 KB)</span></a><div class="core-description">Table S4. Colon Cancer Genome-wide Association Study (GWAS) Risk SNPs and Putative Target Genes Based on E-P Loops (SMC1 HiChIP), Related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/attachment/0ad56b60-e497-4277-85ba-2f6691a2df16/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (12.41 KB)</span></a><div class="core-description">Table S5. Compartment I/B Genes Downregulated with Block Hypomethylation, Related to Figure&nbsp;6</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/attachment/23d05a28-5de0-40c8-adb2-90559edc9ef4/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (32.59 KB)</span></a><div class="core-description">Table S6. Gene Set Enrichment Analysis for Compartment I/B Genes Downregulated with Block Hypomethylation, Related to Figure&nbsp;6</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.07.030/attachment/662c5557-e0ec-44c8-8789-8469a155bce9/mmc7.xlsx" download="mmc7.xlsx"><span id="mmc7-link">Spreadsheet (20.02 KB)</span></a><div class="core-description">Table S7. Gene Set Downregulated in Pan-cancer Analysis, Related to Figure&nbsp;6</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867420309387" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cancer-cell/fulltext/S1535-6108(23)00278-7" target="_blank" title="Transformer-based biomarker prediction from colorectal cancer histology: A large-scale multicentric study">Transformer-based biomarker prediction from colorectal cancer histology: A large-scale multicentric study</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wagner et al.</li></ul></div><div class="linked-article__details"><span><i>Cancer Cell</i></span><span id="item_date">August 30, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-host-microbe/fulltext/S1931-3128(25)00136-2" target="_blank" title="An interplay between human genetics and intratumoral microbiota in the progression of colorectal cancer">An interplay between human genetics and intratumoral microbiota in the progression of colorectal cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Host &amp; Microbe</i></span><span id="item_date">April 29, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(23)01008-2" target="_blank" title="Paired microbiome and metabolome analyses associate bile acid changes with colorectal cancer progression">Paired microbiome and metabolome analyses associate bile acid changes with colorectal cancer progression</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Fu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">August 22, 2023</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(25)00326-2" target="_blank" title="THBS2-producing matrix CAFs promote colorectal cancer progression and link to poor prognosis via the CD47-MAPK axis">THBS2-producing matrix CAFs promote colorectal cancer progression and link to poor prognosis via the CD47-MAPK axis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">April 12, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/matter/fulltext/S2590-2385(25)00178-X" target="_blank" title="Large-scale manufacturing sulfide superionic conductor for advancing all-solid-state batteries">Large-scale manufacturing sulfide superionic conductor for advancing all-solid-state batteries</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Matter</i></span><span id="item_date">April 24, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)15580-6" target="_blank" title="Hsa_circ_0004194 suppresses colorectal cancer progression via hsa-miR-27a-3p">Hsa_circ_0004194 suppresses colorectal cancer progression via hsa-miR-27a-3p</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chen et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">October 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(25)00141-X" target="_blank" title="Intestinal microbiota affects the progression of colorectal cancer by participating in the host intestinal arginine catabolism">Intestinal microbiota affects the progression of colorectal cancer by participating in the host intestinal arginine catabolism</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">February 28, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)02444-6" target="_blank" title="Large-scale analysis of cell-cell communication reveals angiogenin-dependent tumor progression in clear cell renal cell carcinoma">Large-scale analysis of cell-cell communication reveals angiogenin-dependent tumor progression in clear cell renal cell carcinoma</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Massenet-Regad et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">October 31, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/chem/fulltext/S2451-9294(24)00606-5" target="_blank" title="The progress and challenges of tin-lead alloyed perovskites: Toward the development of large-scale all-perovskite tandem solar cells">The progress and challenges of tin-lead alloyed perovskites: Toward the development of large-scale all-perovskite tandem solar cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jia et al.</li></ul></div><div class="linked-article__details"><span><i>Chem</i></span><span id="item_date">January 09, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)04196-3" target="_blank" title="Enhancing m6A modification of lncRNA through METTL3 and RBM15 to promote malignant progression in bladder cancer">Enhancing m6A modification of lncRNA through METTL3 and RBM15 to promote malignant progression in bladder cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Huang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">March 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cancer-cell/fulltext/S1535-6108(24)00270-8" target="_blank" title="Integrative plasma and fecal metabolomics identify functional metabolites in adenoma-colorectal cancer progression and as early diagnostic biomarkers">Integrative plasma and fecal metabolomics identify functional metabolites in adenoma-colorectal cancer progression and as early diagnostic biomarkers</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sun et al.</li></ul></div><div class="linked-article__details"><span><i>Cancer Cell</i></span><span id="item_date">August 12, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)02596-3" target="_blank" title="Topologically convergent and divergent large scale complex networks among Alzheimer's disease spectrum patients: A systematic review">Topologically convergent and divergent large scale complex networks among Alzheimer's disease spectrum patients: A systematic review</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 8, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(23)00434-5" target="_blank" title="UTP18-mediated p21 mRNA instability drives adenoma-carcinoma progression in colorectal cancer">UTP18-mediated p21 mRNA instability drives adenoma-carcinoma progression in colorectal cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Pan et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">April 27, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/newton/fulltext/S2950-6360(25)00100-8" target="_blank" title="Large-scale time-multiplexed nanophotonic parametric oscillators">Large-scale time-multiplexed nanophotonic parametric oscillators</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gray et al.</li></ul></div><div class="linked-article__details"><span><i>Newton</i></span><span id="item_date">May 14, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)05323-4" target="_blank" title="CD133-containing microvesicles promote colorectal cancer progression by inducing tumor angiogenesis">CD133-containing microvesicles promote colorectal cancer progression by inducing tumor angiogenesis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Kim et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 4, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00138-4" target="_blank" title="Proteogenomic characterization of primary colorectal cancer and metastatic progression identifies proteome-based subtypes and signatures">Proteogenomic characterization of primary colorectal cancer and metastatic progression identifies proteome-based subtypes and signatures</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Tanaka et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">February 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(25)00714-5" target="_blank" title="A review on topology and control strategies of high-power inverters in large- scale photovoltaic power plants">A review on topology and control strategies of high-power inverters in large- scale photovoltaic power plants</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Azizi et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">January 28, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)01955-2" target="_blank" title="FNIP1 suppresses colorectal cancer progression through inhibiting STAT3 phosphorylation and nuclear translocation">FNIP1 suppresses colorectal cancer progression through inhibiting STAT3 phosphorylation and nuclear translocation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">August 14, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)11037-1" target="_blank" title="ANP32B promotes colorectal cancer cell progression and reduces cell sensitivity to PRAP1 inhibitor through up-regulating HPF1">ANP32B promotes colorectal cancer cell progression and reduces cell sensitivity to PRAP1 inhibitor through up-regulating HPF1</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">December 16, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(25)00147-6" target="_blank" title="Fast3VmrMLM: A fast algorithm that integrates genome-wide scanning with machine learning to accelerate gene mining and breeding by design for polygenic traits in large-scale GWAS datasets">Fast3VmrMLM: A fast algorithm that integrates genome-wide scanning with machine learning to accelerate gene mining and breeding by design for polygenic traits in large-scale GWAS datasets</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">May 22, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)14662-2" target="_blank" title="Resveratrol inhibited colorectal cancer progression by reducing oxidative DNA damage by targeting the JNK signaling pathway">Resveratrol inhibited colorectal cancer progression by reducing oxidative DNA damage by targeting the JNK signaling pathway</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Maleki et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">October 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)00144-0" target="_blank" title="A large-scale sORF screen identifies putative microproteins involved in cancer cell fitness">A large-scale sORF screen identifies putative microproteins involved in cancer cell fitness</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Schlesinger et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">January 23, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)14492-1" target="_blank" title="Awareness of everyday effects of climate change: The climate change perceptual awareness scale (CCPAS)">Awareness of everyday effects of climate change: The climate change perceptual awareness scale (CCPAS)</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Cipriani et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">September 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)01378-0" target="_blank" title="Bifidobacteriaceae diversity in the human microbiome from a large-scale genome-wide analysis">Bifidobacteriaceae diversity in the human microbiome from a large-scale genome-wide analysis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Pasolli et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">November 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/nucleic-acids/fulltext/S2162-2531(23)00177-4" target="_blank" title="Two antisense RNAs—AFAP1-AS1 and MLK7-AS1—promote colorectal cancer progression by sponging miR-149-5p and miR-485-5p">Two antisense RNAs—AFAP1-AS1 and MLK7-AS1—promote colorectal cancer progression by sponging miR-149-5p and miR-485-5p</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Kim et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy - Nucleic Acids</i></span><span id="item_date">July 14, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/matter/fulltext/S2590-2385(25)00171-7" target="_blank" title="Realizing one-dimensional single-crystalline topological nanomaterials through thermomechanical epitaxy">Realizing one-dimensional single-crystalline topological nanomaterials through thermomechanical epitaxy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Matter</i></span><span id="item_date">April 24, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)05526-9" target="_blank" title="Gut microbiota and their derivatives in the progression of colorectal cancer: Mechanisms of action, genome and epigenome contributions">Gut microbiota and their derivatives in the progression of colorectal cancer: Mechanisms of action, genome and epigenome contributions</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ahmad et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)02086-8" target="_blank" title="CircZNF609/miR-324-5p/voltage-dependent anion channel 1 axis promotes malignant progression of ovarian cancer cells">CircZNF609/miR-324-5p/voltage-dependent anion channel 1 axis promotes malignant progression of ovarian cancer cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ren et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">August 30, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)03015-9" target="_blank" title="Toll-like receptor 6 inhibits colorectal cancer progression by suppressing NF-κB signaling">Toll-like receptor 6 inhibits colorectal cancer progression by suppressing NF-κB signaling</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ma et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">February 29, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-plant/fulltext/S1674-2052(25)00090-5" target="_blank" title="Large-scale metabolomic landscape of edible maize reveals convergent changes in metabolite differentiation and facilitates its breeding improvement">Large-scale metabolomic landscape of edible maize reveals convergent changes in metabolite differentiation and facilitates its breeding improvement</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Plant</i></span><span id="item_date">February 28, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)01646-2" target="_blank" title="Genome-scale modeling predicts metabolic differences between macrophage subtypes in colorectal cancer">Genome-scale modeling predicts metabolic differences between macrophage subtypes in colorectal cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gelbach et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">August 9, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)00177-9" target="_blank" title="N6-methyladenosine modification of B7-H3 mRNA promotes the development and progression of colorectal cancer">N6-methyladenosine modification of B7-H3 mRNA promotes the development and progression of colorectal cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chen et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">January 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(23)00854-0" target="_blank" title="Changes in illegal fishing dynamics in a large-scale MPA during COVID-19">Changes in illegal fishing dynamics in a large-scale MPA during COVID-19</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Collins et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">August 21, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/methods/fulltext/S2329-0501(24)00020-2" target="_blank" title="Large-scale generation of IL-12 secreting macrophages from human pluripotent stem cells for cancer therapy">Large-scale generation of IL-12 secreting macrophages from human pluripotent stem cells for cancer therapy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Kang et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Methods &amp; Clinical Development</i></span><span id="item_date">February 1, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/med/fulltext/S2666-6340(25)00100-X" target="_blank" title="Early immune system changes in amyotrophic lateral sclerosis correlate with later disease progression">Early immune system changes in amyotrophic lateral sclerosis correlate with later disease progression</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Murdock et al.</li></ul></div><div class="linked-article__details"><span><i>Med</i></span><span id="item_date">April 25, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)00389-4" target="_blank" title="CircTBC1D22A inhibits the progression of colorectal cancer through autophagy regulated via miR-1825/ATG14 axis">CircTBC1D22A inhibits the progression of colorectal cancer through autophagy regulated via miR-1825/ATG14 axis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sun et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">February 11, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/med/fulltext/S2666-6340(24)00307-6" target="_blank" title="Large-scale pretrained frame generative model enables real-time low-dose DSA imaging: An AI system development and multi-center validation study">Large-scale pretrained frame generative model enables real-time low-dose DSA imaging: An AI system development and multi-center validation study</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhao et al.</li></ul></div><div class="linked-article__details"><span><i>Med</i></span><span id="item_date">August 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/joule/fulltext/S2542-4351(23)00167-8" target="_blank" title="Fully inkjet-printed large-scale photoelectrodes">Fully inkjet-printed large-scale photoelectrodes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hansora et al.</li></ul></div><div class="linked-article__details"><span><i>Joule</i></span><span id="item_date">May 10, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)10495-1" target="_blank" title="Determining expression changes of ANO7 and SLC38A4 membrane transporters in colorectal cancer">Determining expression changes of ANO7 and SLC38A4 membrane transporters in colorectal cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Khosroshahi et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 10, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)02874-8" target="_blank" title="Predicting phage-host interaction via hyperbolic Poincaré graph embedding and large-scale protein language technique">Predicting phage-host interaction via hyperbolic Poincaré graph embedding and large-scale protein language technique</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Pan et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">December 19, 2024</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2020.07.030","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Large-Scale Topological Changes Restrain Malignant Progression in Colorectal Cancer</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.07.030/asset/c0931a50-3b49-4d0a-b3e3-b0050d98c02f/main.assets/gr1.jpg" height="707" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/bac858f8-6805-4a50-b10f-f115877c54c5/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.07.030/asset/d7af530c-4545-4d2f-aa8a-4368752e0136/main.assets/figs1.jpg" height="132" width="498" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6a4ce885-1da1-4c54-a7cb-42356b9c50a1/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.07.030/asset/150c1482-2aa6-4529-aefb-6e18a9d108b7/main.assets/figs2.jpg" height="923" width="762" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6bd9f6f0-ffd1-4410-8ba1-09a6c836ceda/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.07.030/asset/361f8a5c-6595-4fb2-8825-b0848822272d/main.assets/figs3.jpg" height="743" width="762" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/5c980ed7-ddc4-4d79-a858-6e7360599ee4/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.07.030/asset/045cca42-393b-45e3-a01c-62f21ce16cd5/main.assets/gr2.jpg" height="546" width="765" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6e07ece1-e78d-47b5-a483-d2041381eff9/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.07.030/asset/d50bc35f-7fea-4aef-b993-dd0a26d3e53b/main.assets/figs4.jpg" height="951" width="772" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/21e4163f-9059-40fb-90cc-c05c162bd57f/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.07.030/asset/f80ca9a0-13a3-45fb-94dc-4e3fb5605aff/main.assets/gr3.jpg" height="757" width="744" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/51d2a13b-5735-4b38-8afd-6b48d073461a/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.07.030/asset/2a0a27db-75b7-447b-92ad-5c4202c5449d/main.assets/figs5.jpg" height="892" width="759" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/3509c33f-f6ba-4c7c-a3da-772579207225/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.07.030/asset/4a95a970-e2d6-48f4-bbf2-07326d39dd87/main.assets/gr4.jpg" height="464" width="672" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/29a6890b-13bb-4e0a-a192-c4e0bf52e315/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.07.030/asset/44999e54-071b-4aa6-898b-08467cee6fad/main.assets/figs6.jpg" height="704" width="762" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/bd83a27c-4a57-4384-94c7-1d6e1d604b76/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.07.030/asset/21fb751d-a142-4952-b31f-25c27859e716/main.assets/gr5.jpg" height="465" width="500" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/4185ab61-a197-4590-abb9-1231cb901558/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.07.030/asset/090d1720-ea18-450b-b6ee-1f1fea459a10/main.assets/gr6.jpg" height="742" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/aa59690b-14d3-4666-b2d7-6b1bdcbb3ccc/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2020.07.030/asset/3cb379c7-1441-4046-beee-05c4bf0168fc/main.assets/figs7.jpg" height="951" width="712" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/1cdcb24a-5a93-4721-9f89-dc706d743bf5/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2020.07.030/asset/7966fd35-72ed-4b4d-9f80-ac4755bb004b/main.assets/gr7.jpg" height="263" width="607" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/a1260b7a-a782-4304-83d2-85044e04b112/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.07.030/asset/c0931a50-3b49-4d0a-b3e3-b0050d98c02f/main.assets/gr1.jpg" height="707" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/bac858f8-6805-4a50-b10f-f115877c54c5/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.07.030/asset/d7af530c-4545-4d2f-aa8a-4368752e0136/main.assets/figs1.jpg" height="132" width="498" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6a4ce885-1da1-4c54-a7cb-42356b9c50a1/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.07.030/asset/150c1482-2aa6-4529-aefb-6e18a9d108b7/main.assets/figs2.jpg" height="923" width="762" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6bd9f6f0-ffd1-4410-8ba1-09a6c836ceda/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.07.030/asset/361f8a5c-6595-4fb2-8825-b0848822272d/main.assets/figs3.jpg" height="743" width="762" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/5c980ed7-ddc4-4d79-a858-6e7360599ee4/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.07.030/asset/045cca42-393b-45e3-a01c-62f21ce16cd5/main.assets/gr2.jpg" height="546" width="765" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/6e07ece1-e78d-47b5-a483-d2041381eff9/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.07.030/asset/d50bc35f-7fea-4aef-b993-dd0a26d3e53b/main.assets/figs4.jpg" height="951" width="772" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/21e4163f-9059-40fb-90cc-c05c162bd57f/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.07.030/asset/f80ca9a0-13a3-45fb-94dc-4e3fb5605aff/main.assets/gr3.jpg" height="757" width="744" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/51d2a13b-5735-4b38-8afd-6b48d073461a/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.07.030/asset/2a0a27db-75b7-447b-92ad-5c4202c5449d/main.assets/figs5.jpg" height="892" width="759" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/3509c33f-f6ba-4c7c-a3da-772579207225/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.07.030/asset/4a95a970-e2d6-48f4-bbf2-07326d39dd87/main.assets/gr4.jpg" height="464" width="672" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/29a6890b-13bb-4e0a-a192-c4e0bf52e315/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.07.030/asset/44999e54-071b-4aa6-898b-08467cee6fad/main.assets/figs6.jpg" height="704" width="762" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/bd83a27c-4a57-4384-94c7-1d6e1d604b76/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.07.030/asset/21fb751d-a142-4952-b31f-25c27859e716/main.assets/gr5.jpg" height="465" width="500" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/4185ab61-a197-4590-abb9-1231cb901558/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.07.030/asset/090d1720-ea18-450b-b6ee-1f1fea459a10/main.assets/gr6.jpg" height="742" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/aa59690b-14d3-4666-b2d7-6b1bdcbb3ccc/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2020.07.030/asset/3cb379c7-1441-4046-beee-05c4bf0168fc/main.assets/figs7.jpg" height="951" width="712" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/1cdcb24a-5a93-4721-9f89-dc706d743bf5/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2020.07.030/asset/7966fd35-72ed-4b4d-9f80-ac4755bb004b/main.assets/gr7.jpg" height="263" width="607" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.07.030/asset/a1260b7a-a782-4304-83d2-85044e04b112/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30938-7';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30938-7';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749587497770","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"182","issueNumber":"6","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867420309387","coverDate":"202009","type":"XOCS-JOURNAL:SCOPE-FULL","title":"LargeScale Topological Changes Restrain Malignant Progression in Colorectal Cancer","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dba21d0d3f4ee9',t:'MTc0OTU4NzQ5Ni4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dba21d0d3f4ee9&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2020.07.030&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>