<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="21b6d0f0-96e0-4801-8697-80397943751c" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;article:article:pii\:S0092867422007231;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;issue:issue:pii\:S0092867421X00177;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Tissue-resident memory and circulating T cells are early responders to pre-surgical cancer immunotherapy">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(22)00723-1">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/e48ed072-b4be-4fb6-a36c-faf84d0c7847/fx1.jpg">
    


        <meta property="og:description" content="Analysis of immune cell kinetics in oral cancer patients responding to neoadjuvant
immune checkpoint blockade identifies diverse features in circulating and tumor-infiltrating
T&nbsp;cell subpopulations that underlie rapid response to therapy and correlate with pathological
outcome.">
    











        <meta name="twitter:title" content="Tissue-resident memory and circulating T cells are early responders to pre-surgical cancer immunotherapy">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/e48ed072-b4be-4fb6-a36c-faf84d0c7847/fx1.jpg">
    


        <meta name="twitter:description" content="Analysis of immune cell kinetics in oral cancer patients responding to neoadjuvant
immune checkpoint blockade identifies diverse features in circulating and tumor-infiltrating
T&nbsp;cell subpopulations that underlie rapid response to therapy and correlate with pathological
outcome.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(22)00723-1">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Tissue-resident memory and circulating T cells are early responders to pre-surgical cancer immunotherapy: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Tissue-resident memory and circulating T cells are early responders to pre-surgical cancer immunotherapy"> <meta name="citation_date" content="2022/08/04"> <meta name="citation_online_date" content="2022/07/07"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="16"> <meta name="citation_firstpage" content="2918"> <meta name="citation_lastpage" content="2935.e29"> <meta name="citation_pmid" content="35803260"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867422007231"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867422007231"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867422007231"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2022.06.018"> <meta name="citation_pmid" content="35803260"> <meta name="citation_author" content="Adrienne M. Luoma"><meta name="citation_author_orcid" content="0000-0001-9504-2403"><meta name="citation_author_institution" content="Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Immunology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Shengbao Suo"><meta name="citation_author_email" content="suo_shengbao@gzlab.ac.cn"><meta name="citation_author_institution" content="Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Immunology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Guangzhou Laboratory, Guangzhou, Guangdong 510005, China"><meta name="citation_author" content="Yifan Wang"><meta name="citation_author_institution" content="TScan Therapeutics, Waltham, MA 02451, USA"><meta name="citation_author" content="Lauren Gunasti"><meta name="citation_author_institution" content="Department of Radiation Oncology, Brigham and Women's Hospital, Dana-Farber Cancer Institute, Boston, MA 02115, USA"><meta name="citation_author" content="Caroline B.M. Porter"><meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Nancy Nabilsi"><meta name="citation_author_institution" content="TScan Therapeutics, Waltham, MA 02451, USA"><meta name="citation_author" content="Jenny Tadros"><meta name="citation_author_institution" content="TScan Therapeutics, Waltham, MA 02451, USA"><meta name="citation_author" content="Andrew P. Ferretti"><meta name="citation_author_institution" content="TScan Therapeutics, Waltham, MA 02451, USA"><meta name="citation_author" content="Sida Liao"><meta name="citation_author_institution" content="TScan Therapeutics, Waltham, MA 02451, USA"><meta name="citation_author" content="Cagan Gurer"><meta name="citation_author_institution" content="TScan Therapeutics, Waltham, MA 02451, USA"><meta name="citation_author" content="Yu-Hui Chen"><meta name="citation_author_institution" content="Department of Data Science, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Shana Criscitiello"><meta name="citation_author_institution" content="Department of Radiation Oncology, Brigham and Women's Hospital, Dana-Farber Cancer Institute, Boston, MA 02115, USA"><meta name="citation_author" content="Cora A. Ricker"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Danielle Dionne"><meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Orit Rozenblatt-Rosen"><meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Ravindra Uppaluri"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Surgery, Brigham and Women's Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Robert I. Haddad"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Orr Ashenberg"><meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Aviv Regev"><meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute and Koch Institute of Integrative Cancer Research, Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02142, USA"><meta name="citation_author" content="Eliezer M. Van Allen"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Gavin MacBeath"><meta name="citation_author_institution" content="TScan Therapeutics, Waltham, MA 02451, USA"><meta name="citation_author" content="Jonathan D. Schoenfeld"><meta name="citation_author_email" content="jonathan_schoenfeld@dfci.harvard.edu"><meta name="citation_author_institution" content="Department of Radiation Oncology, Brigham and Women's Hospital, Dana-Farber Cancer Institute, Boston, MA 02115, USA"><meta name="citation_author" content="Kai W. Wucherpfennig"><meta name="citation_author_email" content="kai_wucherpfennig@dfci.harvard.edu"><meta name="citation_author_institution" content="Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Immunology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_keywords" content="T cells; cancer; immunotherapy; neoadjuvant therapy; tissue-resident memory T&nbsp;cells; single-cell RNA sequencing"><meta name="citation_reference" content="citation_title=SCENIC: single-cell regulatory network inference and clustering; citation_author=S. Aibar; citation_author=C.B. González-Blas; citation_author=T. Moerman; citation_author=V.A. Huynh-Thu; citation_author=H. Imrichova; citation_author=G. Hulselmans; citation_author=F. Rambow; citation_author=J.C. Marine; citation_author=P. Geurts; citation_author=J. Aerts; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=1083; citation_lastpage=1086; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Neoadjuvant immune checkpoint blockade in high-risk resectable melanoma; citation_author=R.N. Amaria; citation_author=S.M. Reddy; citation_author=H.A. Tawbi; citation_author=M.A. Davies; citation_author=M.I. Ross; citation_author=I.C. Glitza; citation_author=J.N. Cormier; citation_author=C. Lewis; citation_author=W.J. Hwu; citation_author=E. Hanna; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=1649; citation_lastpage=1654; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases; citation_author=M. Andreatta; citation_author=J. Corria-Osorio; citation_author=S. Müller; citation_author=R. Cubas; citation_author=G. Coukos; citation_author=S.J. Carmona; citation_journal_title=Nat. Commun.; citation_volume=12; citation_firstpage=2965; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage; citation_author=D. Aran; citation_author=A.P. Looney; citation_author=L. Liu; citation_author=E. Wu; citation_author=V. Fong; citation_author=A. Hsu; citation_author=S. Chak; citation_author=R.P. Naikawadi; citation_author=P.J. Wolters; citation_author=A.R. Abate; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=20; citation_firstpage=163; citation_lastpage=172; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Intratumoral CD103+ CD8+ T&nbsp;cells predict response to PD-L1 blockade; citation_author=R. Banchereau; citation_author=A.S. Chitre; citation_author=A. Scherl; citation_author=T.D. Wu; citation_author=N.S. Patil; citation_author=P. de Almeida; citation_author=E.E. Kadel; citation_author=S. Madireddi; citation_author=A. Au-Yeung; citation_author=C. Takahashi; citation_author=et al.; citation_journal_title=J.&nbsp;Immunother. Cancer; citation_volume=9; citation_firstpage=e002231; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A single-cell map of intratumoral changes during anti-PD1 treatment of patients with breast cancer; citation_author=A. Bassez; citation_author=H. Vos; citation_author=L. Van Dyck; citation_author=G. Floris; citation_author=I. Arijs; citation_author=C. Desmedt; citation_author=B. Boeckx; citation_author=M. Vanden Bempt; citation_author=I. Nevelsteen; citation_author=K. Lambein; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=27; citation_firstpage=820; citation_lastpage=832; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Coinhibitory pathways in immunotherapy for cancer; citation_author=S.H. Baumeister; citation_author=G.J. Freeman; citation_author=G. Dranoff; citation_author=A.H. Sharpe; citation_journal_title=Annu. Rev. Immunol.; citation_volume=34; citation_firstpage=539; citation_lastpage=573; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Dimensionality reduction for visualizing single-cell data using UMAP; citation_author=E. Becht; citation_author=L. McInnes; citation_author=J. Healy; citation_author=C.-A. Dutertre; citation_author=I.W.H. Kwok; citation_author=L.G. Ng; citation_author=F. Ginhoux; citation_author=E.W. Newell; citation_journal_title=Nat. Biotechnol.; citation_volume=37; citation_firstpage=38; citation_lastpage=44; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Defining 'T cell exhaustion'; citation_author=C.U. Blank; citation_author=W.N. Haining; citation_author=W. Held; citation_author=P.G. Hogan; citation_author=A. Kallies; citation_author=E. Lugli; citation_author=R.C. Lynn; citation_author=M. Philip; citation_author=A. Rao; citation_author=N.P. Restifo; citation_author=et al.; citation_journal_title=Nat. Rev. Immunol.; citation_volume=19; citation_firstpage=665; citation_lastpage=674; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Neoadjuvant versus adjuvant ipilimumab plus nivolumab in macroscopic stage III melanoma; citation_author=C.U. Blank; citation_author=E.A. Rozeman; citation_author=L.F. Fanchi; citation_author=K. Sikorska; citation_author=B. van de Wiel; citation_author=P. Kvistborg; citation_author=O. Krijgsman; citation_author=M. van den Braber; citation_author=D. Philips; citation_author=A. Broeks; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=1655; citation_lastpage=1661; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Absolute quantification of somatic DNA alterations in human cancer; citation_author=S.L. Carter; citation_author=K. Cibulskis; citation_author=E. Helman; citation_author=A. McKenna; citation_author=H. Shen; citation_author=T. Zack; citation_author=P.W. Laird; citation_author=R.C. Onofrio; citation_author=W. Winckler; citation_author=B.A. Weir; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=30; citation_firstpage=413; citation_lastpage=421; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Transcriptional programs of neoantigen-specific TIL in anti-PD-1-treated lung cancers; citation_author=J.X. Caushi; citation_author=J. Zhang; citation_author=Z. Ji; citation_author=A. Vaghasia; citation_author=B. Zhang; citation_author=E.H. Hsiue; citation_author=B.J. Mog; citation_author=W. Hou; citation_author=S. Justesen; citation_author=R. Blosser; citation_author=et al.; citation_journal_title=Nature; citation_volume=596; citation_firstpage=126; citation_lastpage=132; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Neoadjuvant immunotherapy leads to pathological responses in MMR-proficient and MMR-deficient early-stage colon cancers; citation_author=M. Chalabi; citation_author=L.F. Fanchi; citation_author=K.K. Dijkstra; citation_author=J.G. Van den Berg; citation_author=A.G. Aalbers; citation_author=K. Sikorska; citation_author=M. Lopez-Yurda; citation_author=C. Grootscholten; citation_author=G.L. Beets; citation_author=P. Snaebjornsson; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=26; citation_firstpage=566; citation_lastpage=576; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=ContEst: estimating cross-contamination of human samples in next-generation sequencing data; citation_author=K. Cibulskis; citation_author=A. McKenna; citation_author=T. Fennell; citation_author=E. Banks; citation_author=M. DePristo; citation_author=G. Getz; citation_journal_title=Bioinformatics; citation_volume=27; citation_firstpage=2601; citation_lastpage=2602; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Single-cell transcriptomic analysis of tissue-resident memory T&nbsp;cells in human lung cancer; citation_author=J. Clarke; citation_author=B. Panwar; citation_author=A. Madrigal; citation_author=D. Singh; citation_author=R. Gujar; citation_author=O. Wood; citation_author=S.J. Chee; citation_author=S. Eschweiler; citation_author=E.V. King; citation_author=A.S. Awad; citation_author=et al.; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=216; citation_firstpage=2128; citation_lastpage=2149; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=LymphoSeq: analyze high-throughput sequencing of T and B&nbsp;cell receptors. R package version 3.13; citation_author=D. Coffey; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Discovery and characterization of artifactual mutations in deep coverage targeted capture sequencing data due to oxidative DNA damage during sample preparation; citation_author=M. Costello; citation_author=T.J. Pugh; citation_author=T.J. Fennell; citation_author=C. Stewart; citation_author=L. Lichtenstein; citation_author=J.C. Meldrim; citation_author=J.L. Fostel; citation_author=D.C. Friedrich; citation_author=D. Perrin; citation_author=D. Dionne; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=41; citation_firstpage=e67; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Co-expression of CD39 and CD103 identifies tumor-reactive CD8 T&nbsp;cells in human solid tumors; citation_author=T. Duhen; citation_author=R. Duhen; citation_author=R. Montler; citation_author=J. Moses; citation_author=T. Moudgil; citation_author=N.F. de Miranda; citation_author=C.P. Goodall; citation_author=T.C. Blair; citation_author=B.A. Fox; citation_author=J.E. McDermott; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=2724; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The role of myeloid cells in cancer therapies; citation_author=C. Engblom; citation_author=C. Pfirschke; citation_author=M.J. Pittet; citation_journal_title=Nat. Rev. Cancer; citation_volume=16; citation_firstpage=447; citation_lastpage=462; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Peripheral CD8(+) T&nbsp;cell characteristics associated with durable responses to immune checkpoint blockade in patients with metastatic melanoma; citation_author=B.P. Fairfax; citation_author=C.A. Taylor; citation_author=R.A. Watson; citation_author=I. Nassiri; citation_author=S. Danielli; citation_author=H. Fang; citation_author=E.A. Mahé; citation_author=R. Cooper; citation_author=V. Woodcock; citation_author=Z. Traill; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=26; citation_firstpage=193; citation_lastpage=199; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Impact of neoadjuvant durvalumab with or without tremelimumab on CD8(+) tumor lymphocyte density, safety, and efficacy in patients with oropharynx cancer: CIAO trial results; citation_author=R. Ferrarotto; citation_author=D. Bell; citation_author=M.L. Rubin; citation_author=K.A. Hutcheson; citation_author=J.M. Johnson; citation_author=R.P. Goepfert; citation_author=J. Phan; citation_author=Y.Y. Elamin; citation_author=D.K. Torman; citation_author=C.L. Warneke; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=26; citation_firstpage=3211; citation_lastpage=3219; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Unbiased screens show CD8(+) T&nbsp;cells of COVID-19 patients recognize shared epitopes in SARS-CoV-2 that largely reside outside the spike protein; citation_author=A.P. Ferretti; citation_author=T. Kula; citation_author=Y. Wang; citation_author=D.M.V. Nguyen; citation_author=A. Weinheimer; citation_author=G.S. Dunlap; citation_author=Q. Xu; citation_author=N. Nabilsi; citation_author=C.R. Perullo; citation_author=A.W. Cristofaro; citation_author=et al.; citation_journal_title=Immunity; citation_volume=53; citation_firstpage=1095; citation_lastpage=1107.e3; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Neoadjuvant nivolumab for patients with resectable HPV-positive and HPV-negative squamous cell carcinomas of the head and neck in the CheckMate 358 trial; citation_author=R.L. Ferris; citation_author=W.C. Spanos; citation_author=R. Leidner; citation_author=A. Gonçalves; citation_author=U.M. Martens; citation_author=C. Kyi; citation_author=W. Sharfman; citation_author=C.H. Chung; citation_author=L.A. Devriese; citation_author=H. Gauthier; citation_author=et al.; citation_journal_title=J.&nbsp;Immunother. Cancer; citation_volume=9; citation_firstpage=e002568; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Expression of cancer testis antigens in head and neck squamous cell carcinomas; citation_author=D.L. Figueiredo; citation_author=R.C. Mamede; citation_author=R. Proto-Siqueira; citation_author=L. Neder; citation_author=W.A. Silva; citation_author=M.A. Zago; citation_journal_title=Head Neck; citation_volume=28; citation_firstpage=614; citation_lastpage=619; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Neoadjuvant PD-1 blockade in resectable lung cancer; citation_author=P.M. Forde; citation_author=J.E. Chaft; citation_author=K.N. Smith; citation_author=V. Anagnostou; citation_author=T.R. Cottrell; citation_author=M.D. Hellmann; citation_author=M. Zahurak; citation_author=S.C. Yang; citation_author=D.R. Jones; citation_author=S. Broderick; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=378; citation_firstpage=1976; citation_lastpage=1986; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Neoadjuvant nivolumab plus chemotherapy in resectable lung cancer; citation_author=P.M. Forde; citation_author=J. Spicer; citation_author=S. Lu; citation_author=M. Provencio; citation_author=T. Mitsudomi; citation_author=M.M. Awad; citation_author=E. Felip; citation_author=S.R. Broderick; citation_author=J.R. Brahmer; citation_author=S.J. Swanson; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=386; citation_firstpage=1973; citation_lastpage=1985; citation_publication_date=2022; "><meta name="citation_reference" content="citation_title=Tumor-draining lymph nodes are pivotal in PD-1/PD-L1 checkpoint therapy; citation_author=M.F. Fransen; citation_author=M. Schoonderwoerd; citation_author=P. Knopf; citation_author=M.G. Camps; citation_author=L.J. Hawinkels; citation_author=M. Kneilling; citation_author=T. van Hall; citation_author=F. Ossendorp; citation_journal_title=JCI Insight; citation_volume=3; citation_firstpage=e124507; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Neoadjuvant PD-1 immune checkpoint blockade reverses functional immunodominance among tumor antigen-specific T&nbsp;cells; citation_author=J. Friedman; citation_author=E.C. Moore; citation_author=P. Zolkind; citation_author=Y. Robbins; citation_author=P.E. Clavijo; citation_author=L. Sun; citation_author=S. Greene; citation_author=M.V. Morisada; citation_author=W.K. Mydlarz; citation_author=N. Schmitt; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=26; citation_firstpage=679; citation_lastpage=689; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Type, density, and location of immune cells within human colorectal tumors predict clinical outcome; citation_author=J. Galon; citation_author=A. Costes; citation_author=F. Sanchez-Cabo; citation_author=A. Kirilovsky; citation_author=B. Mlecnik; citation_author=C. Lagorce-Pagès; citation_author=M. Tosolini; citation_author=M. Camus; citation_author=A. Berger; citation_author=P. Wind; citation_author=et al.; citation_journal_title=Science; citation_volume=313; citation_firstpage=1960; citation_lastpage=1964; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Tissue-resident memory features are linked to the magnitude of cytotoxic T&nbsp;cell responses in human lung cancer; citation_author=A.P. Ganesan; citation_author=J. Clarke; citation_author=O. Wood; citation_author=E.M. Garrido-Martin; citation_author=S.J. Chee; citation_author=T. Mellows; citation_author=D. Samaniego-Castruita; citation_author=D. Singh; citation_author=G. Seumois; citation_author=A. Alzetani; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=18; citation_firstpage=940; citation_lastpage=950; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Visualizing and interpreting cancer genomics data via the Xena platform; citation_author=M.J. Goldman; citation_author=B. Craft; citation_author=M. Hastie; citation_author=K. Repečka; citation_author=F. McDade; citation_author=A. Kamath; citation_author=A. Banerjee; citation_author=Y. Luo; citation_author=D. Rogers; citation_author=A.N. Brooks; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=38; citation_firstpage=675; citation_lastpage=678; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Contribution of resident and circulating precursors to tumor-infiltrating CD8(+) T&nbsp;cell populations in lung cancer; citation_author=P. Gueguen; citation_author=C. Metoikidou; citation_author=T. Dupic; citation_author=M. Lawand; citation_author=C. Goudot; citation_author=S. Baulande; citation_author=S. Lameiras; citation_author=O. Lantz; citation_author=N. Girard; citation_author=A. Seguin-Givelet; citation_author=et al.; citation_journal_title=Sci. Immunol.; citation_volume=6; citation_firstpage=eabd5778; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A comparative review of methods for comparing means using partially paired data; citation_author=B. Guo; citation_author=Y. Yuan; citation_journal_title=Stat. Methods Med. Res.; citation_volume=26; citation_firstpage=1323; citation_lastpage=1340; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Integrated analysis of multimodal single-cell data; citation_author=Y. Hao; citation_author=S. Hao; citation_author=E. Andersen-Nissen; citation_author=W.M. Mauck; citation_author=S. Zheng; citation_author=A. Butler; citation_author=M.J. Lee; citation_author=A.J. Wilk; citation_author=C. Darby; citation_author=M. Zager; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=3573; citation_lastpage=3587.e29; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Single-cell dissection of cellular components and interactions shaping the tumor immune phenotypes in ovarian cancer; citation_author=M. Hornburg; citation_author=M. Desbois; citation_author=S. Lu; citation_author=Y. Guan; citation_author=A.A. Lo; citation_author=S. Kaufman; citation_author=A. Elrod; citation_author=A. Lotstein; citation_author=T.M. DesRochers; citation_author=J.L. Munoz-Rodriguez; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=39; citation_firstpage=928; citation_lastpage=944.e6; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A single dose of neoadjuvant PD-1 blockade predicts clinical outcomes in resectable melanoma; citation_author=A.C. Huang; citation_author=R.J. Orlowski; citation_author=X. Xu; citation_author=R. Mick; citation_author=S.M. George; citation_author=P.K. Yan; citation_author=S. Manne; citation_author=A.A. Kraya; citation_author=B. Wubbenhorst; citation_author=L. Dorfman; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=25; citation_firstpage=454; citation_lastpage=461; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Signatures of T&nbsp;cell dysfunction and exclusion predict cancer immunotherapy response; citation_author=P. Jiang; citation_author=S. Gu; citation_author=D. Pan; citation_author=J. Fu; citation_author=A. Sahu; citation_author=X. Hu; citation_author=Z. Li; citation_author=N. Traugh; citation_author=X. Bu; citation_author=B. Li; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=1550; citation_lastpage=1558; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Inference and analysis of cell-cell communication using CellChat; citation_author=S. Jin; citation_author=C.F. Guerrero-Juarez; citation_author=L. Zhang; citation_author=I. Chang; citation_author=R. Ramos; citation_author=C.H. Kuan; citation_author=P. Myung; citation_author=M.V. Plikus; citation_author=Q. Nie; citation_journal_title=Nat. Commun.; citation_volume=12; citation_firstpage=1088; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Proliferation of PD-1+ CD8 T&nbsp;cells in peripheral blood after PD-1-targeted therapy in lung cancer patients; citation_author=A.O. Kamphorst; citation_author=R.N. Pillai; citation_author=S. Yang; citation_author=T.H. Nasti; citation_author=R.S. Akondy; citation_author=A. Wieland; citation_author=G.L. Sica; citation_author=K. Yu; citation_author=L. Koenig; citation_author=N.T. Patel; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=114; citation_firstpage=4993; citation_lastpage=4998; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The impact of CTLA-4 blockade and interferon-alpha on clonality of T-cell repertoire in the tumor microenvironment and peripheral blood of metastatic melanoma patients; citation_author=A. Khunger; citation_author=J.A. Rytlewski; citation_author=P. Fields; citation_author=E.C. Yusko; citation_author=A.A. Tarhini; citation_journal_title=Oncoimmunology; citation_volume=8; citation_firstpage=e1652538; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Fast gene set enrichment analysis; citation_author=G. Korotkevich; citation_author=V. Sukhov; citation_author=N. Budin; citation_author=B. Shpak; citation_author=M.N. Artyomov; citation_author=A. Sergushichev; citation_journal_title=Preprint at bioRxiv; citation_doi=10.1101/060012; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Single-cell sequencing links multiregional immune landscapes and tissue-resident T&nbsp;cells in ccRCC to tumor topology and therapy efficacy; citation_author=C. Krishna; citation_author=R.G. DiNatale; citation_author=F. Kuo; citation_author=R.M. Srivastava; citation_author=L. Vuong; citation_author=D. Chowell; citation_author=S. Gupta; citation_author=C. Vanderbilt; citation_author=T.A. Purohit; citation_author=M. Liu; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=39; citation_firstpage=662; citation_lastpage=677.e6; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=T-scan: A genome-wide method for the systematic discovery of T&nbsp;cell epitopes; citation_author=T. Kula; citation_author=M.H. Dezfulian; citation_author=C.I. Wang; citation_author=N.S. Abdelfattah; citation_author=Z.C. Hartman; citation_author=K.W. Wucherpfennig; citation_author=H.K. Lyerly; citation_author=S.J. Elledge; citation_journal_title=Cell; citation_volume=178; citation_firstpage=1016; citation_lastpage=1028.e13; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Human tissue-resident memory T&nbsp;cells are defined by core transcriptional and functional signatures in lymphoid and mucosal sites; citation_author=B.V. Kumar; citation_author=W. Ma; citation_author=M. Miron; citation_author=T. Granot; citation_author=R.S. Guyer; citation_author=D.J. Carpenter; citation_author=T. Senda; citation_author=X. Sun; citation_author=S.H. Ho; citation_author=H. Lerner; citation_author=et al.; citation_journal_title=Cell Rep; citation_volume=20; citation_firstpage=2921; citation_lastpage=2934; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Anti-CTLA-4 therapy broadens the melanoma-reactive CD8+ T&nbsp;cell response; citation_author=P. Kvistborg; citation_author=D. Philips; citation_author=S. Kelderman; citation_author=L. Hageman; citation_author=C. Ottensmeier; citation_author=D. Joseph-Pietras; citation_author=M.J. Welters; citation_author=S. van der Burg; citation_author=E. Kapiteijn; citation_author=O. Michielin; citation_author=et al.; citation_journal_title=Sci. Transl. Med.; citation_volume=6; citation_firstpage=254ra128; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Discovery and saturation analysis of cancer genes across 21 tumour types; citation_author=M.S. Lawrence; citation_author=P. Stojanov; citation_author=C.H. Mermel; citation_author=J.T. Robinson; citation_author=L.A. Garraway; citation_author=T.R. Golub; citation_author=M. Meyerson; citation_author=S.B. Gabriel; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nature; citation_volume=505; citation_firstpage=495; citation_lastpage=501; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Alpha E beta 7 integrin interaction with E-cadherin promotes antitumor CTL activity by triggering lytic granule polarization and exocytosis; citation_author=A. Le Floc'h; citation_author=A. Jalil; citation_author=I. Vergnon; citation_author=B. Le Maux Chansac; citation_author=V. Lazar; citation_author=G. Bismuth; citation_author=S. Chouaib; citation_author=F. Mami-Chouaib; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=204; citation_firstpage=559; citation_lastpage=570; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Neoadjuvant immunotherapy for melanoma; citation_author=A.Y. Lee; citation_author=M.S. Brady; citation_journal_title=J.&nbsp;Surg. Oncol.; citation_volume=123; citation_firstpage=782; citation_lastpage=788; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq; citation_author=B. Li; citation_author=J. Gould; citation_author=Y. Yang; citation_author=S. Sarkizova; citation_author=M. Tabaka; citation_author=O. Ashenberg; citation_author=Y. Rosen; citation_author=M. Slyper; citation_author=M.S. Kowalczyk; citation_author=A.C. Villani; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=17; citation_firstpage=793; citation_lastpage=798; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The Molecular Signatures Database (MSigDB) hallmark gene set collection; citation_author=A. Liberzon; citation_author=C. Birger; citation_author=H. Thorvaldsdóttir; citation_author=M. Ghandi; citation_author=J.P. Mesirov; citation_author=P. Tamayo; citation_journal_title=Cell Syst; citation_volume=1; citation_firstpage=417; citation_lastpage=425; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Primary chemotherapy in resectable oral cavity squamous cell cancer: a randomized controlled trial; citation_author=L. Licitra; citation_author=C. Grandi; citation_author=M. Guzzo; citation_author=L. Mariani; citation_author=S. Lo Vullo; citation_author=F. Valvo; citation_author=P. Quattrone; citation_author=P. Valagussa; citation_author=G. Bonadonna; citation_author=R. Molinari; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Oncol.; citation_volume=21; citation_firstpage=327; citation_lastpage=333; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Improved efficacy of neoadjuvant compared to adjuvant immunotherapy to eradicate metastatic disease; citation_author=J. Liu; citation_author=S.J. Blake; citation_author=M.C. Yong; citation_author=H. Harjunpää; citation_author=S.F. Ngiow; citation_author=K. Takeda; citation_author=A. Young; citation_author=J.S. O'Donnell; citation_author=S. Allen; citation_author=M.J. Smyth; citation_author=et al.; citation_journal_title=Cancer Discov; citation_volume=6; citation_firstpage=1382; citation_lastpage=1399; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data; citation_author=A.T.L. Lun; citation_author=S. Riesenfeld; citation_author=T. Andrews; citation_author=T.P. Dao; citation_author=T. Gomes; citation_author=J.C. Marioni; citation_journal_title=Genome Biol; citation_volume=20; citation_firstpage=63; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Molecular pathways of colon inflammation induced by cancer immunotherapy; citation_author=A.M. Luoma; citation_author=S. Suo; citation_author=H.L. Williams; citation_author=T. Sharova; citation_author=K. Sullivan; citation_author=M. Manos; citation_author=P. Bowling; citation_author=F.S. Hodi; citation_author=O. Rahma; citation_author=R.J. Sullivan; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=655; citation_lastpage=671.e22; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The developmental pathway for CD103(+)CD8+ tissue-resident memory T&nbsp;cells of skin; citation_author=L.K. Mackay; citation_author=A. Rahimpour; citation_author=J.Z. Ma; citation_author=N. Collins; citation_author=A.T. Stock; citation_author=M.L. Hafon; citation_author=J. Vega-Ramos; citation_author=P. Lauzurica; citation_author=S.N. Mueller; citation_author=T. Stefanovic; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=14; citation_firstpage=1294; citation_lastpage=1301; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=A conserved dendritic-cell regulatory program limits antitumour immunity; citation_author=B. Maier; citation_author=A.M. Leader; citation_author=S.T. Chen; citation_author=N. Tung; citation_author=C. Chang; citation_author=J. LeBerichel; citation_author=A. Chudnovskiy; citation_author=S. Maskey; citation_author=L. Walker; citation_author=J.P. Finnigan; citation_author=et al.; citation_journal_title=Nature; citation_volume=580; citation_firstpage=257; citation_lastpage=262; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Resident memory T&nbsp;cells, critical components in tumor immunology; citation_author=F. Mami-Chouaib; citation_author=C. Blanc; citation_author=S. Corgnac; citation_author=S. Hans; citation_author=I. Malenica; citation_author=C. Granier; citation_author=I. Tihy; citation_author=E. Tartour; citation_journal_title=J.&nbsp;Immunother. Cancer; citation_volume=6; citation_firstpage=87; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=TGFbeta attenuates tumour response to PD-L1 blockade by contributing to exclusion of T&nbsp;cells; citation_author=S. Mariathasan; citation_author=S.J. Turley; citation_author=D. Nickles; citation_author=A. Castiglioni; citation_author=K. Yuen; citation_author=Y. Wang; citation_author=E.E. Kadel; citation_author=H. Koeppen; citation_author=J.L. Astarita; citation_author=R. Cubas; citation_author=et al.; citation_journal_title=Nature; citation_volume=554; citation_firstpage=544; citation_lastpage=548; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Inhibitory CD161 receptor identified in glioma-infiltrating T&nbsp;cells by single-cell analysis; citation_author=N.D. Mathewson; citation_author=O. Ashenberg; citation_author=I. Tirosh; citation_author=S. Gritsch; citation_author=E.M. Perez; citation_author=S. Marx; citation_author=L. Jerby-Arnon; citation_author=R. Chanoch-Myers; citation_author=T. Hara; citation_author=A.R. Richman; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=1281; citation_lastpage=1298.e26; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors; citation_author=C.S. McGinnis; citation_author=L.M. Murrow; citation_author=Z.J. Gartner; citation_journal_title=Cell Syst; citation_volume=8; citation_firstpage=329; citation_lastpage=337.e4; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; citation_author=A. McKenna; citation_author=M. Hanna; citation_author=E. Banks; citation_author=A. Sivachenko; citation_author=K. Cibulskis; citation_author=A. Kernytsky; citation_author=K. Garimella; citation_author=D. Altshuler; citation_author=S. Gabriel; citation_author=M. Daly; citation_author=et al.; citation_journal_title=Genome Res; citation_volume=20; citation_firstpage=1297; citation_lastpage=1303; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Pathological response and survival with neoadjuvant therapy in melanoma: a pooled analysis from the International Neoadjuvant Melanoma Consortium (INMC); citation_author=A.M. Menzies; citation_author=R.N. Amaria; citation_author=E.A. Rozeman; citation_author=A.C. Huang; citation_author=M.T. Tetzlaff; citation_author=B.A. van de Wiel; citation_author=S. Lo; citation_author=A.A. Tarhini; citation_author=E.M. Burton; citation_author=T.E. Pennington; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=27; citation_firstpage=301; citation_lastpage=309; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors; citation_author=D. Miao; citation_author=C.A. Margolis; citation_author=N.I. Vokes; citation_author=D. Liu; citation_author=A. Taylor-Weiner; citation_author=S.M. Wankowicz; citation_author=D. Adeegbe; citation_author=D. Keliher; citation_author=B. Schilling; citation_author=A. Tracy; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=1271; citation_lastpage=1281; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=immunarch: an R package for Painless Bioinformatics Analysis of T-cell and B-cell Immune Repertoires; citation_author=V.i.b.R. Nazarov; citation_author=E.; citation_journal_title=Zenodo; citation_doi=10.5281/zenodo.3367200; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Phenotype, specificity and avidity of antitumour CD8(+) T&nbsp;cells in melanoma; citation_author=G. Oliveira; citation_author=K. Stromhaug; citation_author=S. Klaeger; citation_author=T. Kula; citation_author=D.T. Frederick; citation_author=P.M. Le; citation_author=J. Forman; citation_author=T. Huang; citation_author=S. Li; citation_author=W. Zhang; citation_author=et al.; citation_journal_title=Nature; citation_volume=596; citation_firstpage=119; citation_lastpage=125; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Tissue-resident memory T&nbsp;cells in cancer immunosurveillance; citation_author=S.L. Park; citation_author=T. Gebhardt; citation_author=L.K. Mackay; citation_journal_title=Trends Immunol; citation_volume=40; citation_firstpage=735; citation_lastpage=747; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=CD8(+) T&nbsp;cell differentiation and dysfunction in cancer; citation_author=M. Philip; citation_author=A. Schietinger; citation_journal_title=Nat. Rev. Immunol.; citation_volume=22; citation_firstpage=209; citation_lastpage=223; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A single-cell molecular map of mouse gastrulation and early organogenesis; citation_author=B. Pijuan-Sala; citation_author=J.A. Griffiths; citation_author=C. Guibentif; citation_author=T.W. Hiscock; citation_author=W. Jawaid; citation_author=F.J. Calero-Nieto; citation_author=C. Mulas; citation_author=X. Ibarra-Soria; citation_author=R.C.V. Tyser; citation_author=D.L.L. Ho; citation_author=et al.; citation_journal_title=Nature; citation_volume=566; citation_firstpage=490; citation_lastpage=495; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Model to improve specificity for identification of clinically-relevant expanded T&nbsp;cells in peripheral blood; citation_author=J. Rytlewski; citation_author=S. Deng; citation_author=T. Xie; citation_author=C. Davis; citation_author=H. Robins; citation_author=E. Yusko; citation_author=J. Bienkowska; citation_journal_title=PLoS One; citation_volume=14; citation_firstpage=e0213684; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs; citation_author=C.T. Saunders; citation_author=W.S. Wong; citation_author=S. Swamy; citation_author=J. Becq; citation_author=L.J. Murray; citation_author=R.K. Cheetham; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=1811; citation_lastpage=1817; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Single-cell profiling of breast cancer T&nbsp;cells reveals a tissue-resident memory subset associated with improved prognosis; citation_author=P. Savas; citation_author=B. Virassamy; citation_author=C. Ye; citation_author=A. Salim; citation_author=C.P. Mintoff; citation_author=F. Caramia; citation_author=R. Salgado; citation_author=D.J. Byrne; citation_author=Z.L. Teo; citation_author=S. Dushyanthen; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=986; citation_lastpage=993; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Neoadjuvant nivolumab or nivolumab plus ipilimumab in untreated oral cavity squamous cell carcinoma: A Phase 2 open-label randomized clinical trial; citation_author=J.D. Schoenfeld; citation_author=G.J. Hanna; citation_author=V.Y. Jo; citation_author=B. Rawal; citation_author=Y.H. Chen; citation_author=P.S. Catalano; citation_author=A. Lako; citation_author=Z. Ciantra; citation_author=J.L. Weirather; citation_author=S. Criscitiello; citation_author=et al.; citation_journal_title=JAMA Oncol.; citation_volume=6; citation_firstpage=1563; citation_lastpage=1570; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=TCLP: an online cancer cell line catalogue integrating HLA type, predicted neo-epitopes, virus and gene expression; citation_author=J. Scholtalbers; citation_author=S. Boegel; citation_author=T. Bukur; citation_author=M. Byl; citation_author=S. Goerges; citation_author=P. Sorn; citation_author=M. Loewer; citation_author=U. Sahin; citation_author=J.C. Castle; citation_journal_title=Genome Med.; citation_volume=7; citation_firstpage=118; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The future of immune checkpoint therapy; citation_author=P. Sharma; citation_author=J.P. Allison; citation_journal_title=Science; citation_volume=348; citation_firstpage=56; citation_lastpage=61; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing; citation_author=R. Shen; citation_author=V.E. Seshan; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e131; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A distinct gene module for dysfunction uncoupled from activation in tumor-infiltrating T&nbsp;cells; citation_author=M. Singer; citation_author=C. Wang; citation_author=L. Cong; citation_author=N.D. Marjanovic; citation_author=M.S. Kowalczyk; citation_author=H. Zhang; citation_author=J. Nyman; citation_author=K. Sakuishi; citation_author=S. Kurtulus; citation_author=D. Gennert; citation_author=et al.; citation_journal_title=Cell; citation_volume=166; citation_firstpage=1500; citation_lastpage=1511.e9; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics; citation_author=K. Street; citation_author=D. Risso; citation_author=R.B. Fletcher; citation_author=D. Das; citation_author=J. Ngai; citation_author=N. Yosef; citation_author=E. Purdom; citation_author=S. Dudoit; citation_journal_title=BMC Genomics; citation_volume=19; citation_firstpage=477; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Single-cell transcriptomics of human T&nbsp;cells reveals tissue and activation signatures in health and disease; citation_author=P.A. Szabo; citation_author=H.M. Levitin; citation_author=M. Miron; citation_author=M.E. Snyder; citation_author=T. Senda; citation_author=J. Yuan; citation_author=Y.L. Cheng; citation_author=E.C. Bush; citation_author=P. Dogra; citation_author=P. Thapa; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=4706; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Location, location, location: tissue resident memory T&nbsp;cells in mice and humans; citation_author=P.A. Szabo; citation_author=M. Miron; citation_author=D.L. Farber; citation_journal_title=Sci. Immunol.; citation_volume=4; citation_firstpage=eaas9673; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=DeTiN: overcoming tumor-in-normal contamination; citation_author=A. Taylor-Weiner; citation_author=C. Stewart; citation_author=T. Giordano; citation_author=M. Miller; citation_author=M. Rosenberg; citation_author=A. Macbeth; citation_author=N. Lennon; citation_author=E. Rheinbay; citation_author=D.A. Landau; citation_author=C.J. Wu; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=531; citation_lastpage=534; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq; citation_author=I. Tirosh; citation_author=B. Izar; citation_author=S.M. Prakadan; citation_author=M.H. Wadsworth; citation_author=D. Treacy; citation_author=J.J. Trombetta; citation_author=A. Rotem; citation_author=C. Rodman; citation_author=C. Lian; citation_author=G. Murphy; citation_author=et al.; citation_journal_title=Science; citation_volume=352; citation_firstpage=189; citation_lastpage=196; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Neoadjuvant checkpoint blockade for cancer immunotherapy; citation_author=S.L. Topalian; citation_author=J.M. Taube; citation_author=D.M. Pardoll; citation_journal_title=Science; citation_volume=367; citation_firstpage=eaax0182; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=PD-1 blockade induces responses by inhibiting adaptive immune resistance; citation_author=P.C. Tumeh; citation_author=C.L. Harview; citation_author=J.H. Yearley; citation_author=I.P. Shintaku; citation_author=E.J. Taylor; citation_author=L. Robert; citation_author=B. Chmielowski; citation_author=M. Spasic; citation_author=G. Henry; citation_author=V. Ciobanu; citation_author=et al.; citation_journal_title=Nature; citation_volume=515; citation_firstpage=568; citation_lastpage=571; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Neoadjuvant and adjuvant pembrolizumab in resectable locally advanced, human papillomavirus-unrelated head and neck cancer: A multicenter, Phase II trial; citation_author=R. Uppaluri; citation_author=K.M. Campbell; citation_author=A.M. Egloff; citation_author=P. Zolkind; citation_author=Z.L. Skidmore; citation_author=B. Nussenbaum; citation_author=R.C. Paniello; citation_author=J.T. Rich; citation_author=R. Jackson; citation_author=P. Pipkorn; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=26; citation_firstpage=5140; citation_lastpage=5152; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Trajectory-based differential expression analysis for single-cell sequencing data; citation_author=K. Van den Berge; citation_author=H. Roux de Bézieux; citation_author=K. Street; citation_author=W. Saelens; citation_author=R. Cannoodt; citation_author=Y. Saeys; citation_author=S. Dudoit; citation_author=L. Clement; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=1201; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=CD8(+) T&nbsp;cell states in human cancer: insights from single-cell analysis; citation_author=A.M. van der Leun; citation_author=D.S. Thommen; citation_author=T.N. Schumacher; citation_journal_title=Nat. Rev. Cancer; citation_volume=20; citation_firstpage=218; citation_lastpage=232; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Learning from clinical trials of neoadjuvant checkpoint blockade; citation_author=J.M. Versluis; citation_author=G.V. Long; citation_author=C.U. Blank; citation_journal_title=Nat. Med.; citation_volume=26; citation_firstpage=475; citation_lastpage=484; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response; citation_author=T.D. Wu; citation_author=S. Madireddi; citation_author=P.E. de Almeida; citation_author=R. Banchereau; citation_author=Y.J. Chen; citation_author=A.S. Chitre; citation_author=E.Y. Chiang; citation_author=H. Iftikhar; citation_author=W.E. O'Gorman; citation_author=A. Au-Yeung; citation_author=et al.; citation_journal_title=Nature; citation_volume=579; citation_firstpage=274; citation_lastpage=278; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=GMM-Demux: sample demultiplexing, multiplet detection, experiment planning, and novel cell-type verification in single cell sequencing; citation_author=H. Xin; citation_author=Q. Lian; citation_author=Y. Jiang; citation_author=J. Luo; citation_author=X. Wang; citation_author=C. Erb; citation_author=Z. Xu; citation_author=X. Zhang; citation_author=E. Heidrich-O'Hare; citation_author=Q. Yan; citation_author=et al.; citation_journal_title=Genome Biol; citation_volume=21; citation_firstpage=188; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Clonal replacement of tumor-specific T&nbsp;cells following PD-1 blockade; citation_author=K.E. Yost; citation_author=A.T. Satpathy; citation_author=D.K. Wells; citation_author=Y. Qi; citation_author=C. Wang; citation_author=R. Kageyama; citation_author=K.L. McNamara; citation_author=J.M. Granja; citation_author=K.Y. Sarin; citation_author=R.A. Brown; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=25; citation_firstpage=1251; citation_lastpage=1259; citation_publication_date=2019; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Neoadjuvant immune checkpoint blockade has shown promising clinical activity. Here, we characterized early kinetics in tumor-infiltrating and circulating immune cells in oral cancer patients treated with neoadjuvant anti-PD-1 or anti-PD-1/CTLA-4 in a clinical trial (NCT02919683). Tumor-infiltrating CD8 T&nbsp;cells that clonally expanded during immunotherapy expressed elevated tissue-resident memory and cytotoxicity programs, which were already active prior to therapy, supporting the capacity for rapid response. Systematic target discovery revealed that treatment-expanded tumor T&nbsp;cell clones in responding patients recognized several self-antigens, including the cancer-specific antigen MAGEA1. Treatment also induced a systemic immune response characterized by expansion of activated T&nbsp;cells enriched for tumor-infiltrating T&nbsp;cell clonotypes, including both pre-existing and emergent clonotypes undetectable prior to therapy. The frequency of activated blood CD8 T&nbsp;cells, notably pre-treatment PD-1-positive KLRG1-negative T&nbsp;cells, was strongly associated with intra-tumoral pathological response. These results demonstrate how neoadjuvant checkpoint blockade induces local and systemic tumor immunity.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=8857&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(22)00723-1&amp;be=5212&amp;fe=8559&amp;dc=8161&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749594046883,%22n%22:0,%22f%22:3633,%22dn%22:3634,%22dne%22:3634,%22c%22:3634,%22s%22:3634,%22ce%22:3634,%22rq%22:3634,%22rp%22:4759,%22rpe%22:6255,%22dl%22:4764,%22di%22:6711,%22ds%22:8160,%22de%22:8391,%22dc%22:8557,%22l%22:8558,%22le%22:8665%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-munderover {
  display: inline-block;
  text-align: left;
}

mjx-munderover:not([limits="false"]) {
  padding-top: .1em;
}

mjx-munderover:not([limits="false"]) > * {
  display: block;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c2264 {
  padding: 0.631em 0.778em 0.119em 0;
}

mjx-c.mjx-c2265 {
  padding: 0.631em 0.778em 0.119em 0;
}

mjx-c.mjx-c1D453 {
  padding: 0.705em 0.552em 0.205em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D462 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c1D45C {
  padding: 0.442em 0.485em 0.011em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c1D45D {
  padding: 0.442em 0.503em 0.194em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D447 {
  padding: 0.677em 0.704em 0 0;
}

mjx-c.mjx-c1D465 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D44F {
  padding: 0.694em 0.429em 0.011em 0;
}

mjx-c.mjx-c1D459 {
  padding: 0.694em 0.298em 0.011em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c1D436 {
  padding: 0.705em 0.76em 0.022em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c1D466 {
  padding: 0.442em 0.49em 0.205em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c2211 {
  padding: 0.75em 1.056em 0.25em 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-cD7 {
  padding: 0.491em 0.778em 0 0;
}

mjx-c.mjx-c6C {
  padding: 0.694em 0.278em 0 0;
}

mjx-c.mjx-c6E {
  padding: 0.442em 0.556em 0 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c2F {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c1D440 {
  padding: 0.683em 1.044em 0 0;
}

mjx-c.mjx-c1D43C {
  padding: 0.683em 0.497em 0 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c28.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c2F.MM-SO {
  padding: 0.905em 0.617em 0.405em 0;
}

mjx-c.mjx-c1D434 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c1D435 {
  padding: 0.683em 0.759em 0 0;
}

mjx-c.mjx-c29.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900723-1" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00723-1"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(22)00723-1"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900723-1" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-713473" aria-expanded="false" aria-controls="login-links-713473" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-713473" id="login-links--713473" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900723-1" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900723-1" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900723-1" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-713473" data-opens="#links-dialog-713473" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-713473" aria-modal="true"><button data-closes="#links-dialog-713473" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900723-1" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900723-1" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900723-1" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2022.06.018">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail long-title" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(22)00723-1" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0017-7"><span>Volume 185</span>, <span>Issue 16</span></a></span><span class="meta-panel__pages"><span>p2918-2935.e29</span></span><span class="meta-panel__onlineDate">August 04, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=16&amp;pii=S0092-8674%2821%29X0017-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Tissue-resident memory and circulating T cells are early responders to pre-surgical cancer immunotherapy</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Adrienne M.</span> <span property="familyName">Luoma</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Adrienne M.</span> <span property="familyName">Luoma</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9504-2403" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9504-2403</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Immunology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Luoma/Adrienne+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">13</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Shengbao</span> <span property="familyName">Suo</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shengbao</span> <span property="familyName">Suo</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:suo_shengbao@gzlab.ac.cn" property="email" title="Send email to Shengbao Suo">suo_shengbao@gzlab.ac.cn</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Immunology, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Guangzhou Laboratory, Guangzhou, Guangdong 510005, China</span></div></div><div class="core-author-link"><a href="/authored-by/Suo/Shengbao" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">13</a></sup> <a id="eadd0010" href="mailto:suo_shengbao@gzlab.ac.cn" property="email" title="Send email to Shengbao Suo">suo_shengbao@gzlab.ac.cn</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Yifan</span> <span property="familyName">Wang</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yifan</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">TScan Therapeutics, Waltham, MA 02451, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Yifan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Lauren</span> <span property="familyName">Gunasti</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lauren</span> <span property="familyName">Gunasti</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Radiation Oncology, Brigham and Women's Hospital, Dana-Farber Cancer Institute, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gunasti/Lauren" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Caroline B.M.</span> <span property="familyName">Porter</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Caroline B.M.</span> <span property="familyName">Porter</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Porter/Caroline+B+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Nancy</span> <span property="familyName">Nabilsi</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nancy</span> <span property="familyName">Nabilsi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">TScan Therapeutics, Waltham, MA 02451, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nabilsi/Nancy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Jenny</span> <span property="familyName">Tadros</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jenny</span> <span property="familyName">Tadros</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">TScan Therapeutics, Waltham, MA 02451, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Tadros/Jenny" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Andrew P.</span> <span property="familyName">Ferretti</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew P.</span> <span property="familyName">Ferretti</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">TScan Therapeutics, Waltham, MA 02451, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ferretti/Andrew+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Sida</span> <span property="familyName">Liao</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sida</span> <span property="familyName">Liao</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">TScan Therapeutics, Waltham, MA 02451, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Liao/Sida" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Cagan</span> <span property="familyName">Gurer</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Cagan</span> <span property="familyName">Gurer</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">TScan Therapeutics, Waltham, MA 02451, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gurer/Cagan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Yu-Hui</span> <span property="familyName">Chen</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yu-Hui</span> <span property="familyName">Chen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Data Science, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Yu-Hui" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Shana</span> <span property="familyName">Criscitiello</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shana</span> <span property="familyName">Criscitiello</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Radiation Oncology, Brigham and Women's Hospital, Dana-Farber Cancer Institute, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Criscitiello/Shana" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Cora A.</span> <span property="familyName">Ricker</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Cora A.</span> <span property="familyName">Ricker</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ricker/Cora+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Danielle</span> <span property="familyName">Dionne</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Danielle</span> <span property="familyName">Dionne</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Dionne/Danielle" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Orit</span> <span property="familyName">Rozenblatt-Rosen</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Orit</span> <span property="familyName">Rozenblatt-Rosen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rozenblatt-Rosen/Orit" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Ravindra</span> <span property="familyName">Uppaluri</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ravindra</span> <span property="familyName">Uppaluri</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Surgery, Brigham and Women's Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Uppaluri/Ravindra" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Robert I.</span> <span property="familyName">Haddad</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Robert I.</span> <span property="familyName">Haddad</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Haddad/Robert+I" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Orr</span> <span property="familyName">Ashenberg</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Orr</span> <span property="familyName">Ashenberg</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ashenberg/Orr" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Aviv</span> <span property="familyName">Regev</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Aviv</span> <span property="familyName">Regev</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn1"><div id="ntpara0010" role="paragraph">Present address: Genentech, 1 DNA Way, South San Francisco, CA 94080, USA</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute and Koch Institute of Integrative Cancer Research, Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Regev/Aviv" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Eliezer M.</span> <span property="familyName">Van Allen</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Eliezer M.</span> <span property="familyName">Van Allen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Van+Allen/Eliezer+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Gavin</span> <span property="familyName">MacBeath</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gavin</span> <span property="familyName">MacBeath</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">TScan Therapeutics, Waltham, MA 02451, USA</span></div></div><div class="core-author-link"><a href="/authored-by/MacBeath/Gavin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Jonathan D.</span> <span property="familyName">Schoenfeld</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jonathan D.</span> <span property="familyName">Schoenfeld</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:jonathan_schoenfeld@dfci.harvard.edu" property="email" title="Send email to Jonathan D. Schoenfeld">jonathan_schoenfeld@dfci.harvard.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn3"><div id="ntpara0020" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Radiation Oncology, Brigham and Women's Hospital, Dana-Farber Cancer Institute, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Schoenfeld/Jonathan+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a role="doc-noteref">14</a></sup> <a id="eadd0015" href="mailto:jonathan_schoenfeld@dfci.harvard.edu" property="email" title="Send email to Jonathan D. Schoenfeld">jonathan_schoenfeld@dfci.harvard.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Kai W.</span> <span property="familyName">Wucherpfennig</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kai W.</span> <span property="familyName">Wucherpfennig</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:kai_wucherpfennig@dfci.harvard.edu" property="email" title="Send email to Kai W. Wucherpfennig">kai_wucherpfennig@dfci.harvard.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn3"><div id="ntpara0020" role="paragraph">Senior author</div></div></div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn4"><div id="ntpara0025" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Immunology, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wucherpfennig/Kai+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a>,<a role="doc-noteref">14</a>,<a role="doc-noteref">15</a></sup> <a id="eadd0020" href="mailto:kai_wucherpfennig@dfci.harvard.edu" property="email" title="Send email to Kai W. Wucherpfennig">kai_wucherpfennig@dfci.harvard.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Immunology, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Guangzhou Laboratory, Guangzhou, Guangdong 510005, China</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">TScan Therapeutics, Waltham, MA 02451, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Radiation Oncology, Brigham and Women's Hospital, Dana-Farber Cancer Institute, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Department of Data Science, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Department of Surgery, Brigham and Women's Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Howard Hughes Medical Institute and Koch Institute of Integrative Cancer Research, Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn1"><div id="ntpara0010" role="paragraph">Present address: Genentech, 1 DNA Way, South San Francisco, CA 94080, USA</div></div></div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn3"><div id="ntpara0020" role="paragraph">Senior author</div></div></div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn4"><div id="ntpara0025" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received November 27, 2021</span>; <span>Revised April 26, 2022</span>; <span>Accepted June 9, 2022</span>; <span>Published online July 7, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2022.06.018" property="sameAs" target="_blank">10.1016/j.cell.2022.06.018 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867422007231" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2022 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2022.06.018" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0017-7"><img src="/cms/asset/ef1b48a8-ec76-4113-ac5b-c6ee9bc3a720/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 16"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2822%2900723-1" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900723-1" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.06.018">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2822%2900723-1" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2022.06.018&amp;pii=S0092-8674%2822%2900723-1" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900723-1" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.06.018">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2022.06.018" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867422007231&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867422007231&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=16&amp;pii=S0092-8674%2821%29X0017-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00847-9" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(22)00849-2" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Neoadjuvant ICB induces local and systemic T&nbsp;cell responses in oral cancer patients</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Treatment-responsive T&nbsp;cell clones recognize self-antigens including tumor antigens</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Responding T&nbsp;cells in tumors display a tissue-resident memory program</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Circulating PD-1+KLRG1-CD8 T&nbsp;cell frequency correlates with pathological response</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Neoadjuvant immune checkpoint blockade has shown promising clinical activity. Here, we characterized early kinetics in tumor-infiltrating and circulating immune cells in oral cancer patients treated with neoadjuvant anti-PD-1 or anti-PD-1/CTLA-4 in a clinical trial (NCT02919683). Tumor-infiltrating CD8 T&nbsp;cells that clonally expanded during immunotherapy expressed elevated tissue-resident memory and cytotoxicity programs, which were already active prior to therapy, supporting the capacity for rapid response. Systematic target discovery revealed that treatment-expanded tumor T&nbsp;cell clones in responding patients recognized several self-antigens, including the cancer-specific antigen MAGEA1. Treatment also induced a systemic immune response characterized by expansion of activated T&nbsp;cells enriched for tumor-infiltrating T&nbsp;cell clonotypes, including both pre-existing and emergent clonotypes undetectable prior to therapy. The frequency of activated blood CD8 T&nbsp;cells, notably pre-treatment PD-1-positive KLRG1-negative T&nbsp;cells, was strongly associated with intra-tumoral pathological response. These results demonstrate how neoadjuvant checkpoint blockade induces local and systemic tumor immunity.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/5033a4b5-2720-4f01-bdeb-f72c4f327cc0/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2022.06.018/asset/e48ed072-b4be-4fb6-a36c-faf84d0c7847/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/5033a4b5-2720-4f01-bdeb-f72c4f327cc0/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;T+cells&quot;" alt="Search on this keyword" rel="nofollow">T cells</a></li><li><a href="/action/doSearch?AllField=&quot;cancer&quot;" alt="Search on this keyword" rel="nofollow">cancer</a></li><li><a href="/action/doSearch?AllField=&quot;immunotherapy&quot;" alt="Search on this keyword" rel="nofollow">immunotherapy</a></li><li><a href="/action/doSearch?AllField=&quot;neoadjuvant+therapy&quot;" alt="Search on this keyword" rel="nofollow">neoadjuvant therapy</a></li><li><a href="/action/doSearch?AllField=&quot;tissue-resident+memory+T%C2%A0cells&quot;" alt="Search on this keyword" rel="nofollow">tissue-resident memory T&nbsp;cells</a></li><li><a href="/action/doSearch?AllField=&quot;single-cell+RNA+sequencing&quot;" alt="Search on this keyword" rel="nofollow">single-cell RNA sequencing</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">Immune checkpoint blockade (ICB) therapy induces durable clinical responses in a diverse range of human cancers by enhancing T&nbsp;cell-mediated immunity (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Baumeister et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Baumeister, S.H. ∙ Freeman, G.J. ∙ Dranoff, G. ...</div><div><strong>Coinhibitory pathways in immunotherapy for cancer</strong></div><div><em>Annu. Rev. Immunol.</em> 2016; <strong>34</strong>:539-573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-032414-112049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84969850678" target="_blank" aria-label="Open Scopus in new tab">Scopus (700)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26927206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-032414-112049&amp;pmid=26927206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Sharma and Allison, 2015</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Sharma, P. ∙ Allison, J.P.</div><div><strong>The future of immune checkpoint therapy</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:56-61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa8172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84928774156" target="_blank" aria-label="Open Scopus in new tab">Scopus (3636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25838373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa8172&amp;pmid=25838373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Recent clinical trials have highlighted the potential of neoadjuvant ICB for resectable tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Topalian et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Topalian, S.L. ∙ Taube, J.M. ∙ Pardoll, D.M.</div><div><strong>Neoadjuvant checkpoint blockade for cancer immunotherapy</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:eaax0182</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aax0182" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85078710390" target="_blank" aria-label="Open Scopus in new tab">Scopus (580)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32001626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aax0182&amp;pmid=32001626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Versluis et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Versluis, J.M. ∙ Long, G.V. ∙ Blank, C.U.</div><div><strong>Learning from clinical trials of neoadjuvant checkpoint blockade</strong></div><div><em>Nat. Med.</em> 2020; <strong>26</strong>:475-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-020-0829-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85083385305" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32273608/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-020-0829-0&amp;pmid=32273608" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), demonstrating several conceptual advantages. First, the integrity of lymphatics connecting the tumor to draining lymph nodes is maintained, enabling priming of new T&nbsp;cell responses as demonstrated in murine models (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Fransen et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Fransen, M.F. ∙ Schoonderwoerd, M. ∙ Knopf, P. ...</div><div><strong>Tumor-draining lymph nodes are pivotal in PD-1/PD-L1 checkpoint therapy</strong></div><div><em>JCI Insight.</em> 2018; <strong>3</strong>:e124507</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/jci.insight.124507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85063246976" target="_blank" aria-label="Open Scopus in new tab">Scopus (231)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30518694/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2Fjci.insight.124507&amp;pmid=30518694" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Second, ICB-induced expansion of T&nbsp;cells within the tumor and its infiltrative margin may enhance local control. Third, immune function is not compromised by prior chemotherapy and radiation therapy (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Friedman et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Friedman, J. ∙ Moore, E.C. ∙ Zolkind, P. ...</div><div><strong>Neoadjuvant PD-1 immune checkpoint blockade reverses functional immunodominance among tumor antigen-specific T&nbsp;cells</strong></div><div><em>Clin. Cancer Res.</em> 2020; <strong>26</strong>:679-689</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-19-2209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85079020271" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31645352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-19-2209&amp;pmid=31645352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Liu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Liu, J. ∙ Blake, S.J. ∙ Yong, M.C. ...</div><div><strong>Improved efficacy of neoadjuvant compared to adjuvant immunotherapy to eradicate metastatic disease</strong></div><div><em>Cancer Discov.</em> 2016; <strong>6</strong>:1382-1399</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-16-0577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85007295572" target="_blank" aria-label="Open Scopus in new tab">Scopus (614)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27663893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-16-0577&amp;pmid=27663893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0055" role="paragraph">Early-stage neoadjuvant ICB trials in resectable melanoma, lung, colon, and head and neck cancer have shown significant clinical activity, although results need to be confirmed in phase 3 trials (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Amaria et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Amaria, R.N. ∙ Reddy, S.M. ∙ Tawbi, H.A. ...</div><div><strong>Neoadjuvant immune checkpoint blockade in high-risk resectable melanoma</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:1649-1654</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0197-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85054583492" target="_blank" aria-label="Open Scopus in new tab">Scopus (589)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30297909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0197-1&amp;pmid=30297909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Blank et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Blank, C.U. ∙ Rozeman, E.A. ∙ Fanchi, L.F. ...</div><div><strong>Neoadjuvant versus adjuvant ipilimumab plus nivolumab in macroscopic stage III melanoma</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:1655-1661</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0198-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85054564616" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30297911/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0198-0&amp;pmid=30297911" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref13-1" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-1" aria-controls="sref13-1">Chalabi et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref13-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Chalabi, M. ∙ Fanchi, L.F. ∙ Dijkstra, K.K. ...</div><div><strong>Neoadjuvant immunotherapy leads to pathological responses in MMR-proficient and MMR-deficient early-stage colon cancers</strong></div><div><em>Nat. Med.</em> 2020; <strong>26</strong>:566-576</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-020-0805-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85083283363" target="_blank" aria-label="Open Scopus in new tab">Scopus (812)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32251400/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-020-0805-8&amp;pmid=32251400" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Forde et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Forde, P.M. ∙ Chaft, J.E. ∙ Smith, K.N. ...</div><div><strong>Neoadjuvant PD-1 blockade in resectable lung cancer</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1976-1986</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1716078" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85047781450" target="_blank" aria-label="Open Scopus in new tab">Scopus (1396)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29658848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1716078&amp;pmid=29658848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Huang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Huang, A.C. ∙ Orlowski, R.J. ∙ Xu, X. ...</div><div><strong>A single dose of neoadjuvant PD-1 blockade predicts clinical outcomes in resectable melanoma</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:454-461</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0357-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85062071459" target="_blank" aria-label="Open Scopus in new tab">Scopus (460)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0357-y&amp;pmid=30804515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Lee and Brady, 2021</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Lee, A.Y. ∙ Brady, M.S.</div><div><strong>Neoadjuvant immunotherapy for melanoma</strong></div><div><em>J.&nbsp;Surg. Oncol.</em> 2021; <strong>123</strong>:782-788</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jso.26229" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85091801696" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33002195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjso.26229&amp;pmid=33002195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Menzies et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Menzies, A.M. ∙ Amaria, R.N. ∙ Rozeman, E.A. ...</div><div><strong>Pathological response and survival with neoadjuvant therapy in melanoma: a pooled analysis from the International Neoadjuvant Melanoma Consortium (INMC)</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:301-309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-020-01188-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85100662461" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33558722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-020-01188-3&amp;pmid=33558722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A recent meta-analysis of four neoadjuvant melanoma trials demonstrated a 2-year relapse-free survival of 100% and 94%–96% for patients with complete or major/partial pathological responses, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Menzies et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Menzies, A.M. ∙ Amaria, R.N. ∙ Rozeman, E.A. ...</div><div><strong>Pathological response and survival with neoadjuvant therapy in melanoma: a pooled analysis from the International Neoadjuvant Melanoma Consortium (INMC)</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:301-309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-020-01188-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85100662461" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33558722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-020-01188-3&amp;pmid=33558722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In patients with resectable lung cancer treated with neoadjuvant PD-1 blockade, a major pathological response was observed in nine of 20 patients (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Forde et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Forde, P.M. ∙ Chaft, J.E. ∙ Smith, K.N. ...</div><div><strong>Neoadjuvant PD-1 blockade in resectable lung cancer</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1976-1986</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1716078" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85047781450" target="_blank" aria-label="Open Scopus in new tab">Scopus (1396)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29658848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1716078&amp;pmid=29658848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A high response rate was also observed in mismatch repair-deficient colon cancer with neoadjuvant anti-PD-1 plus CTLA-4 therapy (<span class="dropBlock reference-citations"><a id="body-ref-sref13-2" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-2" aria-controls="sref13-2">Chalabi et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref13-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Chalabi, M. ∙ Fanchi, L.F. ∙ Dijkstra, K.K. ...</div><div><strong>Neoadjuvant immunotherapy leads to pathological responses in MMR-proficient and MMR-deficient early-stage colon cancers</strong></div><div><em>Nat. Med.</em> 2020; <strong>26</strong>:566-576</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-020-0805-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85083283363" target="_blank" aria-label="Open Scopus in new tab">Scopus (812)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32251400/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-020-0805-8&amp;pmid=32251400" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A recent phase 3 clinical trial in patients with resectable lung cancer demonstrated that neoadjuvant PD-1 monoclonal antibody (mAb) plus chemotherapy resulted in longer event-free survival than chemotherapy and a higher frequency of complete pathological responses (24% for combination therapy, 2.2% for monotherapy) (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Forde et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Forde, P.M. ∙ Spicer, J. ∙ Lu, S. ...</div><div><strong>Neoadjuvant nivolumab plus chemotherapy in resectable lung cancer</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2022; <strong>386</strong>:1973-1985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2202170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85130221027" target="_blank" aria-label="Open Scopus in new tab">Scopus (1089)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35403841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2202170&amp;pmid=35403841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0060" role="paragraph">Recent single-cell RNA sequencing (scRNA-seq) studies have revealed the diversity of T&nbsp;cell states within the tumor microenvironment (TME), including a prominent population of CD8 T&nbsp;cells that express key markers of tissue-resident memory (Trm) T&nbsp;cells (<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">van der Leun et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>van der Leun, A.M. ∙ Thommen, D.S. ∙ Schumacher, T.N.</div><div><strong>CD8(+) T&nbsp;cell states in human cancer: insights from single-cell analysis</strong></div><div><em>Nat. Rev. Cancer.</em> 2020; <strong>20</strong>:218-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41568-019-0235-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85079438043" target="_blank" aria-label="Open Scopus in new tab">Scopus (769)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32024970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41568-019-0235-4&amp;pmid=32024970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Trm play an essential role in immunity to pathogens, persist long-term, and act as early responders to pathogen re-entry (<span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Gueguen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Gueguen, P. ∙ Metoikidou, C. ∙ Dupic, T. ...</div><div><strong>Contribution of resident and circulating precursors to tumor-infiltrating CD8(+) T&nbsp;cell populations in lung cancer</strong></div><div><em>Sci. Immunol.</em> 2021; <strong>6</strong>:eabd5778</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.abd5778" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33514641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.abd5778&amp;pmid=33514641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Szabo et&nbsp;al., 2019b</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Szabo, P.A. ∙ Miron, M. ∙ Farber, D.L.</div><div><strong>Location, location, location: tissue resident memory T&nbsp;cells in mice and humans</strong></div><div><em>Sci. Immunol.</em> 2019; <strong>4</strong>:eaas9673</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aas9673" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85064505042" target="_blank" aria-label="Open Scopus in new tab">Scopus (355)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30952804/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aas9673&amp;pmid=30952804" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Trm express key molecules that enforce a tissue-resident state, including integrin αEβ7 (αE chain also designated CD103), CD69, and HOBIT (encoded by <i>ZNF683</i>) (<span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Kumar et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kumar, B.V. ∙ Ma, W. ∙ Miron, M. ...</div><div><strong>Human tissue-resident memory T&nbsp;cells are defined by core transcriptional and functional signatures in lymphoid and mucosal sites</strong></div><div><em>Cell Rep.</em> 2017; <strong>20</strong>:2921-2934</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.celrep.2017.08.078&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.celrep.2017.08.078&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85029578875" target="_blank" aria-label="Open Scopus in new tab">Scopus (674)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28930685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2017.08.078&amp;pmid=28930685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Integrin αEβ7 binds to E-cadherin on epithelial cells, including epithelial cancer cells, and promotes the polarized release of T&nbsp;cell cytotoxic granules to target cells (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Le Floc'h et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Le Floc'h, A. ∙ Jalil, A. ∙ Vergnon, I. ...</div><div><strong>Alpha E beta 7 integrin interaction with E-cadherin promotes antitumor CTL activity by triggering lytic granule polarization and exocytosis</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:559-570</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20061524" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-33947415318" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17325197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20061524&amp;pmid=17325197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CD69 and HOBIT promote tissue residency by inhibiting expression of molecules required for T&nbsp;cell egress. Trm are poised for rapid response but are highly regulated to prevent immunopathology. Intra-tumoral Trm express several inhibitory receptors, including PD-1 and CTLA-4, and may therefore be responsive to ICB therapy (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Clarke et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Clarke, J. ∙ Panwar, B. ∙ Madrigal, A. ...</div><div><strong>Single-cell transcriptomic analysis of tissue-resident memory T&nbsp;cells in human lung cancer</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2019; <strong>216</strong>:2128-2149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20190249" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85071787869" target="_blank" aria-label="Open Scopus in new tab">Scopus (152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31227543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20190249&amp;pmid=31227543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Park et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Park, S.L. ∙ Gebhardt, T. ∙ Mackay, L.K.</div><div><strong>Tissue-resident memory T&nbsp;cells in cancer immunosurveillance</strong></div><div><em>Trends Immunol.</em> 2019; <strong>40</strong>:735-747</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.it.2019.06.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.it.2019.06.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85067962417" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31255505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2019.06.002&amp;pmid=31255505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Several clinical studies have correlated Trm infiltration with clinical outcomes (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Banchereau et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Banchereau, R. ∙ Chitre, A.S. ∙ Scherl, A. ...</div><div><strong>Intratumoral CD103+ CD8+ T&nbsp;cells predict response to PD-L1 blockade</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2021; <strong>9</strong>:e002231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jitc-2020-002231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85103885386" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33827905/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjitc-2020-002231&amp;pmid=33827905" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Duhen et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Duhen, T. ∙ Duhen, R. ∙ Montler, R. ...</div><div><strong>Co-expression of CD39 and CD103 identifies tumor-reactive CD8 T&nbsp;cells in human solid tumors</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:2724</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-05072-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85049972292" target="_blank" aria-label="Open Scopus in new tab">Scopus (551)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30006565/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-05072-0&amp;pmid=30006565" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Ganesan et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Ganesan, A.P. ∙ Clarke, J. ∙ Wood, O. ...</div><div><strong>Tissue-resident memory features are linked to the magnitude of cytotoxic T&nbsp;cell responses in human lung cancer</strong></div><div><em>Nat. Immunol.</em> 2017; <strong>18</strong>:940-950</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3775" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85025815252" target="_blank" aria-label="Open Scopus in new tab">Scopus (372)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28628092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3775&amp;pmid=28628092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-1" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-1" aria-controls="sref42-1">Krishna et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref42-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Krishna, C. ∙ DiNatale, R.G. ∙ Kuo, F. ...</div><div><strong>Single-cell sequencing links multiregional immune landscapes and tissue-resident T&nbsp;cells in ccRCC to tumor topology and therapy efficacy</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:662-677.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.03.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.03.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85104966318" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33861994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2021.03.007&amp;pmid=33861994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, a Trm gene expression signature correlated with longer survival in patients with early-stage triple-negative breast cancer (<span class="dropBlock reference-citations"><a id="body-ref-sref71-1" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-1" aria-controls="sref71-1">Savas et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref71-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Savas, P. ∙ Virassamy, B. ∙ Ye, C. ...</div><div><strong>Single-cell profiling of breast cancer T&nbsp;cells reveals a tissue-resident memory subset associated with improved prognosis</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:986-993</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0078-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85049055113" target="_blank" aria-label="Open Scopus in new tab">Scopus (656)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0078-7&amp;pmid=29942092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, increased CD8 T&nbsp;cell infiltration is a strong positive prognostic indicator even in the absence of immunotherapy, and many associated CD8 T&nbsp;cell markers in pre-treatment biopsies correlate with longer survival following ICB (<span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Galon et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Galon, J. ∙ Costes, A. ∙ Sanchez-Cabo, F. ...</div><div><strong>Type, density, and location of immune cells within human colorectal tumors predict clinical outcome</strong></div><div><em>Science.</em> 2006; <strong>313</strong>:1960-1964</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1129139" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-33749319703" target="_blank" aria-label="Open Scopus in new tab">Scopus (5115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17008531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1129139&amp;pmid=17008531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Tumeh et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Tumeh, P.C. ∙ Harview, C.L. ∙ Yearley, J.H. ...</div><div><strong>PD-1 blockade induces responses by inhibiting adaptive immune resistance</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:568-571</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84920956732" target="_blank" aria-label="Open Scopus in new tab">Scopus (5183)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13954&amp;pmid=25428505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), complicating the relationship of individual Trm genes to response correlates.</div><div id="p0065" role="paragraph">In head and neck squamous cell carcinomas (HNSCC), early studies of neoadjuvant ICB have demonstrated promising pathological response rates and the potential for significant tumor downstaging (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Ferrarotto et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Ferrarotto, R. ∙ Bell, D. ∙ Rubin, M.L. ...</div><div><strong>Impact of neoadjuvant durvalumab with or without tremelimumab on CD8(+) tumor lymphocyte density, safety, and efficacy in patients with oropharynx cancer: CIAO trial results</strong></div><div><em>Clin. Cancer Res.</em> 2020; <strong>26</strong>:3211-3219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-19-3977" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85087469863" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32269052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-19-3977&amp;pmid=32269052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Ferris et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Ferris, R.L. ∙ Spanos, W.C. ∙ Leidner, R. ...</div><div><strong>Neoadjuvant nivolumab for patients with resectable HPV-positive and HPV-negative squamous cell carcinomas of the head and neck in the CheckMate 358 trial</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2021; <strong>9</strong>:e002568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jitc-2021-002568" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85107491601" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34083421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjitc-2021-002568&amp;pmid=34083421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Uppaluri et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Uppaluri, R. ∙ Campbell, K.M. ∙ Egloff, A.M. ...</div><div><strong>Neoadjuvant and adjuvant pembrolizumab in resectable locally advanced, human papillomavirus-unrelated head and neck cancer: A multicenter, Phase II trial</strong></div><div><em>Clin. Cancer Res.</em> 2020; <strong>26</strong>:5140-5152</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-20-1695" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85097627415" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32665297/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-20-1695&amp;pmid=32665297" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we leveraged the unique opportunity offered by this clinical trial design to investigate early therapy-induced changes in tumor-infiltrating and circulating immune cells. Comprehensive scRNA-seq and T&nbsp;cell receptor (TCR) repertoire analysis showed that CD8 T&nbsp;cells with a tissue residency program were early responders to ICB. Kinetic analysis of tumor-infiltrating and circulating immune cells revealed striking treatment-induced expansion of emergent T&nbsp;cell clones in tumors and the systemic circulation that were undetectable prior to therapy. Neoadjuvant ICB thus enhanced local and systemic tumor immunity in oral SCC.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>HNSCC neoadjuvant ICB clinical trial and study design</h3><div id="p0070" role="paragraph">We recently reported the clinical findings from a phase 2 trial of neoadjuvant ICB in untreated patients with HNSCC of the oral cavity (<span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Schoenfeld et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Schoenfeld, J.D. ∙ Hanna, G.J. ∙ Jo, V.Y. ...</div><div><strong>Neoadjuvant nivolumab or nivolumab plus ipilimumab in untreated oral cavity squamous cell carcinoma: A Phase 2 open-label randomized clinical trial</strong></div><div><em>JAMA Oncol.</em> 2020; <strong>6</strong>:1563-1570</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jamaoncol.2020.2955" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85090783150" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32852531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Fjamaoncol.2020.2955&amp;pmid=32852531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Patients (mean [SD] age, 62 [12] years; 18 men [62%] and 11 women [38%]) were randomized to receive monotherapy with a PD-1 mAb (nivolumab, 3&nbsp;mg/kg) weeks 1 and 3 (n&nbsp;= 14 patients) or combination therapy with two doses of PD-1 mAb and a single infusion of a CTLA-4 mAb (ipilimumab, 1&nbsp;mg/kg in week 1) (n&nbsp;= 15 patients); surgery was performed 3–7&nbsp;days following the second dose (<a id="crosref0385" href="#fig1">Figure&nbsp;1</a>A). Several indicators provided evidence of response despite the short time frame of treatment, with greater than 50% of patients across cohorts achieving a reduction in tumor volume prior to surgery (<a id="crosref0390" href="#fig1">Figure&nbsp;1</a>B). Four patients had a major/complete pathological response of &gt;90% based on histological analysis of surgical specimens (one monotherapy and three combination therapy patients) (<a id="crosref0395" href="#figs1">Figure&nbsp;S1</a>A). Updated analysis with the 3-year follow-up demonstrated overall survival in &gt;80% of enrolled patients, despite detection of lymph node metastases in 58.6% of patients at initial presentation (<a id="crosref0400" href="#fig1">Figure&nbsp;1</a>C). These survival data compare favorably with historical datasets, including a randomized trial of neoadjuvant chemotherapy versus no chemotherapy in resectable HNSCC with 3-year overall survival rates of 61.9% and 60.2% in the experimental and control arms, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Licitra et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Licitra, L. ∙ Grandi, C. ∙ Guzzo, M. ...</div><div><strong>Primary chemotherapy in resectable oral cavity squamous cell cancer: a randomized controlled trial</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2003; <strong>21</strong>:327-333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2003.06.146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-0037440210" target="_blank" aria-label="Open Scopus in new tab">Scopus (236)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12525526/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2003.06.146&amp;pmid=12525526" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Oral cavity tumors are not commonly associated with human papillomavirus (HPV); all six tested patients were found to be negative for p16/HPV. Pre-treatment tumor mutational burden (TMB) was not correlated with response in the subset of analyzed patients (<a id="crosref0410" href="#figs1">Figure&nbsp;S1</a>B).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/5fe8ca01-b188-47ee-a1d4-bf7feda23660/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.018/asset/0a4ac243-40ec-450a-856a-b3aff04903f6/main.assets/gr1.jpg" height="693" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/5fe8ca01-b188-47ee-a1d4-bf7feda23660/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Study design and clinical response data</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Tumor and blood sample collection from HNSCC patients for single-cell RNA sequencing and TCR repertoire analysis.</div><div id="fspara0020" role="paragraph">(B) Clinical metadata and analyses performed for each patient, including treatment cohort and clinical response metrics (volumetric response based on bidirectional tumor measurements, clinical and pathological downstaging at time of surgery, and pathological response based on histological assessment).</div><div id="fspara0025" role="paragraph">(C) Overall and progression-free survival of patients for a median follow-up period of 36&nbsp;months.</div><div id="fspara0030" role="paragraph">See also <a id="crosref1595" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref1600" href="#mmc1">Table&nbsp;S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/17e5e84c-ac2a-4708-8877-0072850b688b/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.018/asset/4f3e8c7d-02ba-4b5a-8d9e-b39c39a736de/main.assets/figs1.jpg" height="941" width="744" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/17e5e84c-ac2a-4708-8877-0072850b688b/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Global analysis of scRNA-seq data on tumor-infiltrating immune cells, related to <a id="crosref1670" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0365" role="paragraph">(A) Pathological response for each patient shown in descending order and colored by response bins and cohort.</div><div id="fspara0370" role="paragraph">(B) Tumor mutational burden (TMB) of pre-treatment samples stratified by volumetric response (n&nbsp;= 10, p&nbsp;= n.s., two-sided Mann-Whitney U test).</div><div id="fspara0375" role="paragraph">(C) Gating strategy used for sorting of CD45+ mononuclear cells (MNCs) from HNSCC tumor cell suspensions for scRNA-seq. CD66b+ neutrophils were excluded from the analysis due to poor compatibility with the 10X Genomics platform.</div><div id="fspara0380" role="paragraph">(D) Quality metrics for CD45+ MNC dataset from tumor (left) and blood (right) specimens. Average for patients in each group with standard error of the mean are shown.</div><div id="fspara0385" role="paragraph">(E and F) UMAP embedding of major immune cell lineages (E)&nbsp;and patient ID (F) (n&nbsp;= 19 post-Tx samples and six matched pre-Tx samples).</div><div id="fspara0390" role="paragraph">(G) UMAP embedding showing normalized expression of immune lineage marker genes.</div><div id="fspara0395" role="paragraph">(H) Representation of major immune cell populations in scRNA-seq data from pre-tx (n&nbsp;= 6) and post-tx (n&nbsp;= 19) tumor samples.</div><div id="fspara0400" role="paragraph">(I) Cell cluster frequency shown as a fraction of total cells for each patient grouped by treatment stage (six paired pre-Tx and post-Tx patients, partially paired two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05). Average and SEM are shown.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">The neoadjuvant ICB trial provided an opportunity to investigate early immune responses during ICB without interference by therapies that compromise immune function (<a id="crosref0415" href="#fig1">Figures&nbsp;1</a>A and 1B; <a id="crosref0420" href="#mmc1">Table&nbsp;S1</a>). We performed sc(TCR+RNA)-seq of tumor-infiltrating CD45+ immune cells collected post-treatment during surgery (n&nbsp;= 19 patients); six of these patients (three per treatment arm) also had a pre-treatment research biopsy (<a id="crosref0425" href="#fig1">Figures&nbsp;1</a>B and <a id="crosref0430" href="#figs1">S1</a>C). The combined tumor scRNA-seq dataset contained 74,557 high-quality cells from well-defined immune lineages (<a id="crosref0435" href="#figs1">Figures&nbsp;S1</a>D–S1I). scRNA-seq data were also generated from circulating immune cells at three time points (pre-treatment, 2-week on-treatment, 10- to 12-week follow-up). Additionally, we performed deep sequencing of the TCR repertoire (TCR β chain) from pre- and post-treatment tumor and blood specimens. These TCR sequences were used to annotate sc(TCR+RNA)-seq data for sensitive and quantitative detection of treatment-responsive T&nbsp;cell clonotypes (<a id="crosref0440" href="#fig1">Figures&nbsp;1</a>A and 1B).</div></section><section id="sec-2-2"><h3>Identification of treatment-expanded CD8 and CD4 T&nbsp;cell populations</h3><div id="p0080" role="paragraph">Tumor-infiltrating CD8 T&nbsp;cells (CD8 TILs) partitioned to seven sub-clusters (<a id="crosref0445" href="#fig2">Figure&nbsp;2</a>A), with two major populations distinguished by expression of <i>GZMK</i> (clusters 1 &amp; 2) versus <i>GZMB</i> (clusters 3 &amp; 4) (<a id="crosref0450" href="#fig2">Figure&nbsp;2</a>B), similar to scRNA-seq studies of other epithelial tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Hornburg et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hornburg, M. ∙ Desbois, M. ∙ Lu, S. ...</div><div><strong>Single-cell dissection of cellular components and interactions shaping the tumor immune phenotypes in ovarian cancer</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:928-944.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85107327427" target="_blank" aria-label="Open Scopus in new tab">Scopus (158)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33961783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2021.04.004&amp;pmid=33961783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells in clusters 3 &amp; 4 expressed Trm hallmark genes <i>ITGAE</i> (CD103) and <i>ZNF683</i> (HOBIT) (<a id="crosref0460" href="#fig2">Figures&nbsp;2</a>B and <a id="crosref0465" href="#figs2">S2</a>A; <a id="crosref0470" href="#mmc2">Table&nbsp;S2</a>) implicated in protective tumor immunity (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Park et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Park, S.L. ∙ Gebhardt, T. ∙ Mackay, L.K.</div><div><strong>Tissue-resident memory T&nbsp;cells in cancer immunosurveillance</strong></div><div><em>Trends Immunol.</em> 2019; <strong>40</strong>:735-747</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.it.2019.06.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.it.2019.06.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85067962417" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31255505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2019.06.002&amp;pmid=31255505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71-2" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-2" aria-controls="sref71-2">Savas et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref71-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Savas, P. ∙ Virassamy, B. ∙ Ye, C. ...</div><div><strong>Single-cell profiling of breast cancer T&nbsp;cells reveals a tissue-resident memory subset associated with improved prognosis</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:986-993</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0078-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85049055113" target="_blank" aria-label="Open Scopus in new tab">Scopus (656)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0078-7&amp;pmid=29942092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>ITGAE</i>+ clusters 3 &amp; 4 cells expressed similar cytotoxicity and inhibitory receptor gene programs, although cells in cluster 4 were distinguished by a cell cycle signature (<a id="crosref0485" href="#fig2">Figures&nbsp;2</a>B and <a id="crosref0490" href="#figs2">S2</a>A). We validated the presence of CD8 Trm cells within tumors from a subset of patients by flow cytometry using the CD103 and CD69 surface markers. CD103<sup>+</sup> CD69<sup>+</sup> CD8 T&nbsp;cells expressed higher levels of granzyme B and PD-1 compared with CD103<sup>−</sup> cells (<a id="crosref0495" href="#fig2">Figures&nbsp;2</a>C, <a id="crosref0500" href="#figs2">S2</a>B, and S2C). Comparison of matched pre- and post-treatment specimens demonstrated a post-treatment increase in the percentage of cells in clusters 4 and 1. Cluster 1 was increased in both treatment cohorts, whereas a significant increase in cluster 4 (cycling cells) was observed only with combination therapy (<a id="crosref0505" href="#fig2">Figure&nbsp;2</a>D).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/0ddbd122-5c77-43a0-89a6-180ff350bd31/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.018/asset/87f646f0-b5ed-4c89-b48a-ea836d87fdfd/main.assets/gr2.jpg" height="951" width="758" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/0ddbd122-5c77-43a0-89a6-180ff350bd31/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Identification of treatment-responsive CD8 and CD4 T&nbsp;cell populations</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0040" role="paragraph">(A) UMAP embedding of tumor CD8 T&nbsp;cells. Inset shows the normalized expression level of <i>CD3D</i> gene (n&nbsp;= 19 post-Tx patients, n&nbsp;= 6 pre-Tx patients).</div><div id="fspara0045" role="paragraph">(B) Normalized expression of selected markers defining tumor CD8 T&nbsp;cell clusters.</div><div id="fspara0050" role="paragraph">(C) Identification of CD103+C69+ Trm cells by flow cytometry, pre-gated on live/CD45+/CD3+/CD8+ T&nbsp;cells; quantification of PD-1 and granzyme B protein level for indicated populations (patient P32).</div><div id="fspara0055" role="paragraph">(D) Tumor CD8 T&nbsp;cell cluster frequency comparing paired pre-Tx and post-Tx samples (n&nbsp;= 6 patients). Right, fraction of tumor CD8 T&nbsp;cells in clusters 1 and 4 for pairs of pre-Tx and post-Tx samples split by cohort (two-sided paired t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05).</div><div id="fspara0060" role="paragraph">(E) Single-cell TCR clonotype size projected onto a UMAP embedding for pre- and post-Tx samples. Bar plot displays the average clonotype size in each pre-Tx and post-Tx CD8 T&nbsp;cell cluster (C1 &amp; C2 versus C3 &amp; C4 paired two-sided Wilcoxon rank-sum test p&nbsp;= 0.016).</div><div id="fspara0065" role="paragraph">(F) Sharing of expanded TCR clonotypes across tumor CD8 T&nbsp;cell clusters in post-Tx tumors. Top, clonotypes from tumor CD8 clusters 1 &amp; 2 or clusters 3 &amp; 4 and sharing across clusters. Bottom, heatmap indicating the number of shared expanded TCR clonotypes for each cluster pair in aggregated patients. Bold boxes indicate statistically significant sharing of expanded clonotypes between clusters, accounting for cluster size (FDR&nbsp;&lt;&nbsp;1e−5, one-sided Fisher’s exact test followed by Benjamini-Hochberg correction).</div><div id="fspara0070" role="paragraph">(G) Fraction of clonotypes from tumor CD8 T&nbsp;cell clusters 1–4 shared with blood bulk TCR repertoire; TCRs aggregated from individual patients (n&nbsp;= 10, paired two-sided Wilcoxon rank-sum test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01).</div><div id="fspara0075" role="paragraph">(H) UMAP embedding showing sub-clusters of total tumor CD4 T&nbsp;cells (n&nbsp;= 19 post-Tx samples and n&nbsp;= 6 paired pre-Tx samples).</div><div id="fspara0080" role="paragraph">(I) Normalized expression of selected marker genes in tumor CD4 T&nbsp;cells.</div><div id="fspara0085" role="paragraph">(J) Left, average tumor CD4 T&nbsp;cell cluster frequency in paired pre-Tx and post-Tx samples. Right, tumor CD4 cluster 5 frequency for paired pre-Tx and post-Tx tumor samples, split by cohort (paired two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05).</div><div id="fspara0090" role="paragraph">(K) Comparison of tumor CD4 T&nbsp;cell cluster frequency between treatment cohorts in tumor post-Tx samples. Average and SEM are shown for each patient group (n&nbsp;= 10 mono patients, n&nbsp;= 9 combo patients, two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05).</div><div id="fspara0095" role="paragraph">(L) Single-cell TCR clonotype size projected onto tumor CD4&nbsp;UMAP visualization. Bar plot displays the average clonotype size for each pre-Tx and post-Tx CD4 T&nbsp;cell cluster.</div><div id="fspara0100" role="paragraph">(M) Sharing of expanded TCR clonotypes across tumor post-Tx CD4 T&nbsp;cells as in (F). Bold boxes indicate statistically significant clonotype sharing between clusters (FDR&nbsp;&lt;&nbsp;1e−5, one-sided Fisher’s exact test followed by Benjamini-Hochberg correction).</div><div id="fspara0105" role="paragraph">See also <a id="crosref1605" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref1610" href="#mmc2">Table&nbsp;S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/8b7c7a3e-c189-4ea2-8a12-b4cf44e95d84/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.018/asset/f25c859e-9ad8-4041-8500-dedf4627bfbe/main.assets/figs2.jpg" height="928" width="769" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/8b7c7a3e-c189-4ea2-8a12-b4cf44e95d84/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Characterization of tumor CD8 and CD4 T&nbsp;cell populations, related to <a id="crosref1675" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0410" role="paragraph">(A) Heatmap showing differentially expressed genes across tumor CD8 T&nbsp;cell sub-clusters. The top 10 cluster-associated genes, among highly variable genes, are shown and ranked by averaged ln (fold change) compared with all other clusters. Each row was <i>Z</i> score normalized.</div><div id="fspara0415" role="paragraph">(B) Gating strategy for flow cytometric analysis of tumor CD8 T&nbsp;cells.</div><div id="fspara0420" role="paragraph">(C) Granzyme B+ CD8 T&nbsp;cells shown as a percentage of indicated populations determined by flow cytometry. Shown are seven patients for whom this analysis was performed (paired t test).</div><div id="fspara0425" role="paragraph">(D) Pseudotime trajectories of pre-treatment tumor CD8 T&nbsp;cells showing two potential differentiation trajectories (left); cells are colored based on tumor CD8+ T&nbsp;cell clusters as in <a id="crosref1680" href="#fig2">Figure&nbsp;2</a>A. Heatmaps display gene expression dynamics of key marker genes (top) and inhibitor receptor genes (bottom) for trajectories 1&nbsp;and&nbsp;2. The colored bar plot on the top of each heatmap indicates cluster identities along the trajectory.</div><div id="fspara0430" role="paragraph">(E) Relationship between CD8 clusters 3 and 4. Left, clusters 3 &amp; 4 from main tumor CD8 T&nbsp;cell UMAP embedding were selected. Right, re-clustering and UMAP embedding of selected cells based on highly variable genes. Exclusion of cell cycle genes reveals mixing of cells originating from clusters 3 &amp; 4, indicative of shared gene programs.</div><div id="fspara0435" role="paragraph">(F) Fraction of cells among CD8 T&nbsp;cell clusters that share a TCR with indicated clusters C2 (left) and C3 (right) at the post-Tx time point shown as a fraction of total cells in each cluster. Statistical comparisons relevant to C1–C4 TCR sharing are shown (two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.0001).</div><div id="fspara0440" role="paragraph">(G) Sharing of expanded TCR clonotypes across tumor pre- and post-treatment CD8 T&nbsp;cell clusters from aggregated patients (n&nbsp;= 5 with pre/post pairs). Values indicate the −log<sub>10</sub>(FDR) of shared TCR clonotypes for each cluster pair across time, values with FDR&nbsp;&lt;&nbsp;0.05 are labeled as 0.</div><div id="fspara0445" role="paragraph">(H) Violin plots showing the normalized top two differentially expressed genes for tumor CD4 T&nbsp;cell clusters. The y axis of violin plots represents the normalized gene expression value.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Pseudotime analysis identified two major trajectories, composed primarily of cells from <i>GMZK</i>+ clusters 1 &amp; 2 (trajectory 1) and <i>ITGAE</i>+ clusters 3 &amp; 4 (trajectory 2) (<a id="crosref0510" href="#figs2">Figure&nbsp;S2</a>D). Further analysis showed that cells from clusters 3 &amp; 4 formed a single cluster following removal of cell cycle genes, further supporting the dynamic relationship of T&nbsp;cells in both clusters (<a id="crosref0515" href="#figs2">Figure&nbsp;S2</a>E). We then examined the frequency and cluster distribution of TCR clonotypes, defined as T&nbsp;cells with identical TCR sequences. Pre-treatment samples showed similar average clonotype sizes across clusters 1–4, whereas in post-treatment samples, <i>ITGAE</i><sup><i>+</i></sup> clusters 3–4 were enriched for larger clonotype sizes (<a id="crosref0520" href="#fig2">Figure&nbsp;2</a>E). We observed significant clonotype sharing between cells in <i>ITGAE</i>+ clusters 3 &amp; 4 and separately between <i>GZMK</i>+ clusters 1 &amp; 2 (<a id="crosref0525" href="#fig2">Figures&nbsp;2</a>F, <a id="crosref0530" href="#figs2">S2</a>F, and S2G), consistent with the two major trajectories defined by pseudotime analysis (<a id="crosref0535" href="#figs2">Figure&nbsp;S2</a>D).</div><div id="p0090" role="paragraph">Among tumor-infiltrating CD4 T&nbsp;cells, the percentage of cycling CD4 T&nbsp;cells was increased following combination therapy (<a id="crosref0540" href="#fig2">Figures&nbsp;2</a>H–2K and <a id="crosref0545" href="#figs2">S2</a>H; <a id="crosref0550" href="#mmc2">Table&nbsp;S2</a>). Substantial TCR clonotype sharing between effector and cycling CD4 T&nbsp;cell clusters (clusters 4 &amp; 5, respectively) was observed in post-treatment samples (<a id="crosref0555" href="#fig2">Figures&nbsp;2</a>L and 2M). Effector CD4 T&nbsp;cell cluster 4 showed higher expression of <i>CXCL13</i>, <i>IFNG</i>, and inhibitory receptor genes as in corresponding CD8 clusters (<a id="crosref0560" href="#fig2">Figure&nbsp;2</a>I).</div></section><section id="sec-2-3"><h3>Treatment-induced clonal expansion of tissue-resident memory T&nbsp;cells</h3><div id="p0095" role="paragraph">T cell activation following recognition of MHC-bound peptide antigens results in substantial proliferative expansion, and TCR-sequencing enables sensitive detection of clonal expansion based on the unique TCR gene rearrangements of each T&nbsp;cell. We identified TCR clonotypes that expanded in response to therapy (Tx-E TCRs) and investigated the corresponding gene expression programs (<a id="crosref0565" href="#fig3">Figure&nbsp;3</a>A). The following considerations were important for our strategy. First, we integrated scRNA-seq data with bulk TCR β chain repertoire-seq for sensitive and quantitative detection of treatment-expanded (Tx-E) TCRs. From tumor tissue, we obtained an average of &gt;19,000 and 31,000 productive bulk TCR β sequences in pre- and post-treatment samples, respectively (<a id="crosref0570" href="#figs3">Figures&nbsp;S3</a>A and S3B). Second, we excluded TCRs that were more abundant in blood to focus the analysis on tumor-associated TCRs (<a id="crosref0575" href="#figs3">Figures&nbsp;S3</a>C and S3D). Third, we identified TCRs enriched in tumors relative to blood that did not clonally expand during ICB (Tx-NE TCRs). TCR sequences detected only once and potentially marking naïve blood T&nbsp;cells were excluded. We compared the expression profiles of these Tx-E and treatment non-expanded (Tx-NE) T&nbsp;cells (<a id="crosref0580" href="#fig3">Figure&nbsp;3</a>A), thereby isolating gene programs specific to treatment-related expansion. To identify significantly expanded Tx-E clonotypes, we used a model accounting for differences in sampling depth and temporal variability (<a id="crosref0585" href="#figs3">Figure&nbsp;S3</a>E). Overall, Tx-E clonotypes comprised less than 1% of total clonotypes, explaining the absence of overall clonality changes at these early treatment time points (<a id="crosref0590" href="#figs3">Figures&nbsp;S3</a>F and S3G; <span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Amaria et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Amaria, R.N. ∙ Reddy, S.M. ∙ Tawbi, H.A. ...</div><div><strong>Neoadjuvant immune checkpoint blockade in high-risk resectable melanoma</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:1649-1654</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0197-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85054583492" target="_blank" aria-label="Open Scopus in new tab">Scopus (589)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30297909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0197-1&amp;pmid=30297909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/9323abcc-0d32-47ff-8565-6d6690eb2d09/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.018/asset/0cd1b84b-8fc0-46fa-8b48-3bd78bc47c7d/main.assets/gr3.jpg" height="608" width="760" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/9323abcc-0d32-47ff-8565-6d6690eb2d09/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Treatment-induced clonal expansion of tissue-resident memory T&nbsp;cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0115" role="paragraph">(A) Strategy for discovery of Tx-E and Tx-NE tumor TCRs.</div><div id="fspara0120" role="paragraph">(B) Classification of Tx-E TCR origin. Fraction of each clonotype category is shown for individual patients (pre-existing versus emergent, paired two-sided t test).</div><div id="fspara0125" role="paragraph">(C) Projection of TCRs with significant Tx-E or Tx-NE onto UMAP embeddings of tumor CD8 TIL. Bar plots show fraction of Tx-E and Tx-NE cells in CD8 T&nbsp;cell clusters (paired two-sided t test).</div><div id="fspara0130" role="paragraph">(D) Projection of Tx-E TCRs onto post-treatment scRNA-seq data, grouped by pre-existing or emergent clonotypes.</div><div id="fspara0135" role="paragraph">(E) Quantification of Tx-E cell fraction per cluster in pre-Tx versus post-Tx samples (average and SEM for each patient group, n&nbsp;= 5 pre-Tx patients, n&nbsp;= 13 post-Tx patients, two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05).</div><div id="fspara0140" role="paragraph">(F) Quantification of Tx-E cell fraction per cluster in post-Tx samples comparing treatment cohorts (average and SEM for each patient group, n&nbsp;= 7 mono patients, n&nbsp;= 6 combo therapy patients, two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05).</div><div id="fspara0145" role="paragraph">(G) UMAP embedding of post-Tx CD4 clusters highlighting tumor Tx-NE/Tx-E TCRs; bar plots show fraction of Tx-NE and Tx-E cells in tumor CD4 T&nbsp;cell clusters (paired two-sided t test).</div><div id="fspara0150" role="paragraph">See also <a id="crosref1615" href="#figs3">Figure&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/0678bc58-a8d5-4bb7-a1cc-0bccf75c4591/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.018/asset/cbf96089-ce7a-441a-9f59-6398c258ee82/main.assets/figs3.jpg" height="894" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/0678bc58-a8d5-4bb7-a1cc-0bccf75c4591/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Identification of Tx-E TCR clonotypes within tumors, related to <a id="crosref1685" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0455" role="paragraph">(A) Number of productive bulk TCR β chain sequences for blood and tumor samples from each patient.</div><div id="fspara0460" role="paragraph">(B) Distribution of tumor TCR clonotype size determined by bulk TCR β chain sequencing of pre-Tx and post-Tx samples. Values on the top of bar plot represent p values between pre-Tx and post-Tx samples in each category.</div><div id="fspara0465" role="paragraph">(C) Cartoon showing different categories of tumor TCR clonotypes and examples for each of these categories with TCR frequencies (%) in tumor and blood.</div><div id="fspara0470" role="paragraph">(D) Fraction of tumor TCR clonotypes classified as “blood-contaminated TCRs” (% of TCRs higher in blood than tumor sequences) and removed from subsequent tumor Tx-E/Tx-NE analysis.</div><div id="fspara0475" role="paragraph">(E) Scatter plots of TCR clonotype frequencies for selected patients comparing pre-Tx and post-Tx samples. TCR clonotypes with count &gt;5 (red dotted line) were tested for differential abundance using a beta-binomial model. Orange clonotypes increased post-Tx; cyan clonotypes decreased post-Tx; changes in dark gray clones were not statistically significant.</div><div id="fspara0480" role="paragraph">(F) Frequency of total tumor Tx-E (left) and emergent (right) TCR clonotypes shown as percentage of total clonotypes, compared between treatment cohorts (n&nbsp;= 7 mono patients, n&nbsp;= 8 combo patients, p values are calculated by two-sided t test).</div><div id="fspara0485" role="paragraph">(G) Overall assessment of clonality using tumor bulk TCR β repertoire sequencing comparing pre-Tx and post-Tx samples (left, two-sided t test) or treatment cohorts (middle, post-Tx patient samples, two-sided t test). Morisita similarity between pre- and post-treatment samples shown for treatment cohorts (right, two-sided t test).</div><div id="fspara0490" role="paragraph">(H) Down-sampling of bulk TCR β datasets for indicated patients and analysis of pre-existing versus emergent TCR clonotype status. For each subsampling interval, the fraction of pre-existing clonotypes that retain their status (not re-classified as emergent due to loss of sequencing depth in pre-Tx dataset) is shown (average and SEM are shown for each subsampling interval).</div><div id="fspara0495" role="paragraph">(I) Integration of Tx-E TCRs identified by deep TCR repertoire sequencing with scRNA-seq data. Heatmap displaying the percentage of Tx-E TCRs among tumor CD8 T&nbsp;cell clusters (<a id="crosref1690" href="#fig2">Figure&nbsp;2</a>A) from pre-Tx and post-Tx tumor samples for each patient. Gray color indicates absence of Tx-E TCRs in each cluster; white box indicates that no sample was available.</div><div id="fspara0500" role="paragraph">(J) Heatmap displaying the percentage of Tx-E TCRs in pre-Tx and post-Tx samples among CD4 T&nbsp;cell clusters (<a id="crosref1695" href="#fig2">Figure&nbsp;2</a>H) for the indicated patients. Gray color indicates absence of Tx-E TCRs in each cluster; white boxes indicate no sample collected.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">The majority of Tx-E TCRs (mean of 59.1%) were detected within tumors prior to therapy (“pre-existing TCRs”); however, a substantial fraction of “emergent TCRs” (mean of 40.9%) were identified only in post- but not in pre-treatment tumors (<a id="crosref0600" href="#fig3">Figure&nbsp;3</a>B). Classification of pre-existing versus emergent clonotype status was consistent even with substantial dataset down-sampling (<a id="crosref0605" href="#figs3">Figure&nbsp;S3</a>H). We mapped the bulk TCR β chain sequences to the scRNA-seq data from CD8 TILs. Cells mapped to Tx-E clonotypes were significantly enriched in CD8 T&nbsp;cell cluster 3 (<i>ITGAE</i>+) and cluster 4 (cycling) (<a id="crosref0610" href="#fig3">Figures&nbsp;3</a>C–3F and <a id="crosref0615" href="#figs3">S3</a>I), whereas Tx-NE cells were enriched in <i>GZMK</i>+ clusters 1 &amp; 2. This differential distribution of Tx-E and Tx-NE CD8 T&nbsp;cells was particularly notable at the pre-treatment time point, suggesting that CD8 T&nbsp;cells in <i>ITGAE</i><sup>+</sup> clusters 3 &amp; 4 were poised to respond to ICB therapy (<a id="crosref0620" href="#fig3">Figure&nbsp;3</a>C). Both pre-existing and emergent TCRs were enriched in <i>ITGAE</i><sup>+</sup> clusters 3 &amp; 4. (<a id="crosref0625" href="#fig3">Figure&nbsp;3</a>D). Notably, the fraction of Tx-E cells in cluster 3 was significantly higher post-treatment in patients on combination therapy, likely explained by high expression of both PD-1 and CTLA-4 receptors among cells in this cluster (<a id="crosref0630" href="#fig3">Figures&nbsp;3</a>E and 3F). Although treatment-related clonal expansion was less marked in CD4 T&nbsp;cells, CD4 Tx-E could also be readily identified among effector (<i>CXCL13</i>+) and cycling clusters (clusters 4 &amp; 5) (<a id="crosref0635" href="#fig3">Figures&nbsp;3</a>G and <a id="crosref0640" href="#figs3">S3</a>J). These data identified T&nbsp;cells with tissue residency and effector programs as early responders to neoadjuvant ICB therapy.</div></section><section id="sec-2-4"><h3>Shared gene programs of treatment-expanded CD8 and CD4 T&nbsp;cells</h3><div id="p0105" role="paragraph">Next, we identified differentially expressed (DE) genes between Tx-E and Tx-NE tumor-infiltrating T&nbsp;cells. Effector, tissue residency, and inhibitory receptor genes were upregulated in Tx-E, whereas <i>GZMK</i> was the top gene upregulated in Tx-NE cells (<a id="crosref0645" href="#fig4">Figures&nbsp;4</a>A and <a id="crosref0650" href="#figs4">S4</a>A; <a id="crosref0655" href="#mmc3">Table&nbsp;S3</a>). Upregulated genes in Tx-E cells included genes previously identified in human Trm isolated from epithelia, including <i>ITGAE</i>, <i>ZNF683</i>, <i>GPR25</i>, and <i>CXCR6</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Kumar et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kumar, B.V. ∙ Ma, W. ∙ Miron, M. ...</div><div><strong>Human tissue-resident memory T&nbsp;cells are defined by core transcriptional and functional signatures in lymphoid and mucosal sites</strong></div><div><em>Cell Rep.</em> 2017; <strong>20</strong>:2921-2934</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.celrep.2017.08.078&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.celrep.2017.08.078&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85029578875" target="_blank" aria-label="Open Scopus in new tab">Scopus (674)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28930685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2017.08.078&amp;pmid=28930685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Szabo et&nbsp;al., 2019a</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Szabo, P.A. ∙ Levitin, H.M. ∙ Miron, M. ...</div><div><strong>Single-cell transcriptomics of human T&nbsp;cells reveals tissue and activation signatures in health and disease</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4706</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12464-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85073621555" target="_blank" aria-label="Open Scopus in new tab">Scopus (345)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31624246/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12464-3&amp;pmid=31624246" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0670" href="#figs4">Figure&nbsp;S4</a>A). Tx-E cells had higher activity scores for Trm, cytotoxicity, NK cell receptor, and inhibitory receptor gene signatures, consistent with their enrichment in <i>ITGAE</i><sup>+</sup> clusters 3 &amp; 4 (<a id="crosref0675" href="#fig4">Figures&nbsp;4</a>B, 4C, and <a id="crosref0680" href="#figs4">S4</a>B; <a id="crosref0685" href="#mmc3">Tables&nbsp;S3</a> and <a id="crosref0690" href="#mmc4">S4</a>). Pre- and post-treatment T&nbsp;cells with Tx-E TCRs shared many DE genes, indicating that T&nbsp;cells with this gene expression program were poised to respond to ICB treatment (<a id="crosref0695" href="#figs4">Figures&nbsp;S4</a>C and S4D). A similar analysis of Tx-E versus Tx-NE genes for CD4 T&nbsp;cells revealed differential expression of genes encoding effector molecules and inhibitory receptors in Tx-E cells, whereas T<sub>reg</sub> signature genes (<i>FOXP3</i> and <i>IL2RA</i>) were notably enriched in Tx-NE cells (<a id="crosref0700" href="#figs4">Figures&nbsp;S4</a>E and S4F; <a id="crosref0705" href="#mmc3">Table&nbsp;S3</a>). Comparison of CD8 and CD4 T&nbsp;cells identified significant overlap in the set of top Tx-E genes, including <i>CXCL13</i>, <i>IFNG</i>, <i>LAG3</i>, and <i>GZMA</i> (<a id="crosref0710" href="#fig4">Figure&nbsp;4</a>D).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/b36b2ba8-9871-4151-929d-146d951628b3/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.018/asset/5f45d401-756c-4315-a6dd-e1068a1ed481/main.assets/gr4.jpg" height="653" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/b36b2ba8-9871-4151-929d-146d951628b3/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Gene programs of Tx-E T&nbsp;cells and relation to clinical response</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0160" role="paragraph">(A) Top 20 differentially expressed (DE) genes between tumor CD8 Tx-NE and Tx-E cells in pre-Tx and post-Tx samples; averaged scaled gene expression of all cells in each group.</div><div id="fspara0165" role="paragraph">(B and C) Comparison of averaged activity scores for indicated gene signatures between Tx-NE and Tx-E T&nbsp;cells in pre-Tx and post-Tx patient samples (n&nbsp;= 4 pre-Tx patients, n&nbsp;= 13 post-Tx patients, two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01). For two pre-Tx patients, we were unable to calculate scores due to missing bulk TCR time point/lack of pre-Tx cells.</div><div id="fspara0170" role="paragraph">(D) Top Tx-E versus Tx-NE DE genes shared by tumor CD4 and CD8 T&nbsp;cells in post-Tx samples.</div><div id="fspara0175" role="paragraph">(E and F) Signature gene sets from (E)&nbsp;used to analyze bulk RNA-seq data from an independent ICB-treated patient cohort (metastatic urothelial cancer patients treated with anti-PD-L1). (F) Left, activity scores for patients with complete or partial response (CR/PR) versus stable disease or progressive disease (SD/PD), two-sided t test. Right, Kaplan-Meier curves showing overall survival (likelihood ratio test).</div><div id="fspara0180" role="paragraph">See also <a id="crosref1620" href="#figs4">Figure&nbsp;S4</a> and <a id="crosref1625" href="#mmc3">Tables&nbsp;S3</a> and <a id="crosref1630" href="#mmc4">S4</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/7b67d036-dbb8-467e-a4c6-a8df016373c1/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.018/asset/dab596af-3fe7-41ad-9736-96829298483d/main.assets/figs4.jpg" height="942" width="734" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/7b67d036-dbb8-467e-a4c6-a8df016373c1/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Gene programs associated with T&nbsp;cell clonal expansion within tumors, related to <a id="crosref1700" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0510" role="paragraph">(A) Ranking of differentially expressed genes between tumor CD8 Tx-E and Tx-NE cells in post-Tx samples, colored by p value (Wilcoxon rank-sum test). Selected genes are highlighted by indicated categories.</div><div id="fspara0515" role="paragraph">(B) Normalized expression (log-transformed) of key genes comparing tumor CD8 Tx-NE and Tx-E CD8 T&nbsp;cells aggregated by patient in a pseudo-bulk approach (n&nbsp;= 13 patients, p values calculated by two-sided Wilcoxon rank-sum test).</div><div id="fspara0520" role="paragraph">(C) Differentially expressed genes between Tx-NE or Tx-E cells shared by CD8 T&nbsp;cells in pre-Tx and post-Tx samples. Color bar representing averaged scaled gene expression level is shown for each gene across all cells in each group.</div><div id="fspara0525" role="paragraph">(D) Functional enrichment analysis of hallmark gene sets comparing Tx-E cells to Tx-NE tumor CD8 T&nbsp;cells. NES, normalized enrichment score.</div><div id="fspara0530" role="paragraph">(E) Heatmap showing top 20 differentially expressed genes between tumor Tx-NE and Tx-E CD4 T&nbsp;cells in post-Tx samples, shown as averaged scaled gene expression of all cells in each group. Marker genes belonging to indicated gene programs are colored as in (A).</div><div id="fspara0535" role="paragraph">(F) Normalized expression (log-transformed) of key genes comparing tumor CD4 Tx-NE and Tx-E CD8 T&nbsp;cells aggregated by patient in a pseudo-bulk approach (n&nbsp;= 13 patients, p values calculated by two-sided Wilcoxon rank-sum test).</div><div id="fspara0540" role="paragraph">(G) Association of CD8 T&nbsp;cell gene signatures with clinical response in an independent ICB-treated patient cohort (metastatic urothelial cancer patients treated with anti-PD-L1). General T&nbsp;cell, Tx-E (post-Tx), and Tx-NE (post-tx) CD8 T&nbsp;cell gene signatures are shown. Left: signature scores for patients with complete or partial response (CR/PR) versus stable disease or progressive disease (SD/PD), two-sided t test. Right: overall survival (likelihood ratio test).</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">We next asked whether gene signatures of Tx-E T&nbsp;cells were associated with outcomes in a large independent ICB treatment cohort, as the tumor scRNA-seq cohort size in the present trial was not powered for such an assessment. A large bulk RNA-seq dataset was not available from ICB-treated HNSCC patients. However, urothelial cancer has a mutational burden and ICB-response rate comparable with HNSCC (<span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">Miao et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Miao, D. ∙ Margolis, C.A. ∙ Vokes, N.I. ...</div><div><strong>Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1271-1281</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0200-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85052712350" target="_blank" aria-label="Open Scopus in new tab">Scopus (386)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30150660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0200-2&amp;pmid=30150660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with an available RNA-seq dataset from a phase 3 urothelial cancer clinical trial evaluating a PD-L1 blocking mAb (<span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Mariathasan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Mariathasan, S. ∙ Turley, S.J. ∙ Nickles, D. ...</div><div><strong>TGFbeta attenuates tumour response to PD-L1 blockade by contributing to exclusion of T&nbsp;cells</strong></div><div><em>Nature.</em> 2018; <strong>554</strong>:544-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85042495633" target="_blank" aria-label="Open Scopus in new tab">Scopus (3242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29443960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25501&amp;pmid=29443960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We analyzed specific gene signatures from both Tx-E and Tx-NE T&nbsp;cells, the latter serving as a control signature (<a id="crosref0725" href="#mmc4">Table&nbsp;S4</a>). These signatures did not include general T&nbsp;cell lineage genes such as <i>CD8A, CD8B</i>, and <i>CD3D</i>, which considered alone were not predictive of response or survival in this cohort (<a id="crosref0730" href="#figs4">Figure&nbsp;S4</a>G). The Tx-E CD8 T&nbsp;cell gene signature from pre-treatment samples correlated with response and overall survival, in contrast to the Tx-NE signature (<a id="crosref0735" href="#fig4">Figures&nbsp;4</a>E and 4F). Tx-E signatures from post-treatment samples again identified a response correlation for Tx-E but not Tx-NE gene sets (<a id="crosref0740" href="#figs4">Figure&nbsp;S4</a>G). These findings demonstrate that CD8 T&nbsp;cells with gene programs of tissue residency, cytotoxicity, and inhibitory receptors were early responders to neoadjuvant ICB.</div></section><section id="sec-2-5"><h3>Expanded T&nbsp;cell clones recognize diverse shared target antigens</h3><div id="p0115" role="paragraph">Investigation of the antigen specificity of Tx-E T&nbsp;cells could uncover novel targets for adoptive T&nbsp;cell therapy. We used T-Scan, a comprehensive, genome-wide screening method (<span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Kula et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kula, T. ∙ Dezfulian, M.H. ∙ Wang, C.I. ...</div><div><strong>T-scan: A genome-wide method for the systematic discovery of T&nbsp;cell epitopes</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1016-1028.e13</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85070183509" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31398327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.009&amp;pmid=31398327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to investigate the specificity of 80 CD8 T&nbsp;cell-derived TCRs from five patients (P13, P19, P20, P21, and P32), four of whom had a substantial pathological response of &gt;50%. Genes encoding each TCR were synthesized, and primary CD8 T&nbsp;cells expressing these exogenous TCRs were co-cultured with an epitope library in HEK293 target cells expressing the relevant MHC class I alleles (<a id="crosref0750" href="#mmc5">Table&nbsp;S5</a>). The library included 559,359 protein fragments spanning all wild-type human proteins, common SNPs (&gt;1% frequency), 2,903 cancer mutations, and 293 human endogenous retroviral sequences. Target cells also expressed a granzyme B-activated fluorescence reporter, enabling enrichment of library cells targeted by CD8 T&nbsp;cells (<a id="crosref0755" href="#fig5">Figure&nbsp;5</a>A).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/e21f2e34-9905-43b3-9274-0d7a7ea096d2/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.018/asset/d3a5eed8-1cd1-44f2-a45c-3e9c86dd8e42/main.assets/gr5.jpg" height="828" width="759" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/e21f2e34-9905-43b3-9274-0d7a7ea096d2/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Identification of tumor-associated targets of expanded TCRs</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0190" role="paragraph">(A) Representation of results from a genome-wide T-scan screen. Each circle represents a single 90-aa protein fragment; y axis represents the fold change of each peptide in relation to the unsorted input library (eight sorted replicates). Protein fragments with fold enrichment &gt;2.5 that share overlapping peptide sequences from same target are highlighted.</div><div id="fspara0195" role="paragraph">(B) List of targets identified along with associated HLA, TCR expansion, and epitope information.</div><div id="fspara0200" role="paragraph">(C) Treatment-related expansion of clonotypes with validated target antigens. Frequency calculated from bulk TCR β chain dataset.</div><div id="fspara0205" role="paragraph">(D) Validation of the T-scan screen hits. Target cells (HEK293T) that expressed the relevant HLAs (indicated in B) were pulsed with the indicated peptides; non-pulsed cells were used as controls. IFNγ release by CD8 T&nbsp;cells expressing the patient TCRs following incubation with target cells (average and SD of n&nbsp;= 3, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01; <sup>∗∗∗</sup>p&nbsp;&lt;&nbsp;0.001 Student’s t test).</div><div id="fspara0210" role="paragraph">(E) Projection of TCRs with validated target antigens onto tumor CD8&nbsp;UMAP embedding. Three low-frequency TCRs were not identified in filtered CD8 T&nbsp;cell sub-clusters, although single-cell TCR pairs could be identified in un-filtered data.</div><div id="fspara0215" role="paragraph">(F) Cancer cell reactivity of primary T&nbsp;cells expressing MAGEA1-specific TCRs. CD8 T&nbsp;cells expressing P32-39 or P32-41 TCRs were co-cultured for 18&nbsp;h with a panel of C<sup>∗</sup>07:02 positive cancer cell lines that expressed MAGEA1 at variable levels; HEK293T cells engineered to express C<sup>∗</sup>07:02 ± MAGEA1 protein fragment were included as controls. IFNγ concentration in the supernatant was measured by ELISA assay. Each condition compared with C<sup>∗</sup>07:02 HEK293 control (average and SD of n&nbsp;= 3, <sup>∗∗∗∗</sup>p&nbsp;&lt;&nbsp;0.0001 one-way ANOVA and Bartlett’s post-test).</div><div id="fspara0220" role="paragraph">(G) Cytotoxic activity of T&nbsp;cells expressing MAGEA1-specific TCRs. A101D and A2058 tumor cells were co-cultured with P32-39 TCR, P32-41, or with non-transduced control donor T&nbsp;cells at indicated E:T ratios. Red fluorescence was measured over time and normalized to time point 0. Circles represent an average of&nbsp;n&nbsp;= 2.</div><div id="fspara0225" role="paragraph">See also <a id="crosref1635" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref1640" href="#mmc5">Table&nbsp;S5</a>.</div></div></div></div></figcaption></figure></div><div id="p0120" role="paragraph">This screen identified six target antigens for seven TCRs (<a id="crosref0760" href="#fig5">Figure&nbsp;5</a>B) that represented pre-existing or emergent clonotypes that expanded substantially during treatment (<a id="crosref0765" href="#fig5">Figures&nbsp;5</a>B and 5C). Patterns of antigen specificity included either broad expression in cancer and healthy tissues (DST, AHNAK2) or overexpression in a subset of cancers with limited expression in healthy tissues (endogenous retroviral antigen ERVMER34-1 and MAGEA1) (<a id="crosref0770" href="#fig5">Figures&nbsp;5</a>B, <a id="crosref0775" href="#figs5">S5</a>A, and S5B). HEK293 cell clones expressing single HLA alleles and 90-aa protein fragments were used to deconvolve TCRs, define HLA restriction, and map peptide epitopes (<a id="crosref0780" href="#fig5">Figures&nbsp;5</a>D, <a id="crosref0785" href="#figs5">S5</a>C, and S5D). TCRs with identified cancer-associated antigens nearly exclusively mapped to cells from clusters 3 &amp; 4, despite selection of TCRs based on treatment-expansion rather than cluster assignment (<a id="crosref0790" href="#fig5">Figures&nbsp;5</a>E and <a id="crosref0795" href="#figs5">S5</a>E).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/e9a1e181-4788-42b8-842a-1d0824a44ade/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.018/asset/4b33d044-7a9f-449f-86ab-580cf9d5a9c6/main.assets/figs5.jpg" height="941" width="754" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/e9a1e181-4788-42b8-842a-1d0824a44ade/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Characterization of TCRs with identified target antigens and analysis of tumor-infiltrating myeloid cells, related to <a id="crosref1705" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0550" role="paragraph">(A) Heatmap representing the mRNA expression profile for targets recognized by Tx-E TCRs. Values depicts the mean of the top quartile of normalized TPM for each indicated tissue type from TCGA (left, indicated cancer types) or GTEx (right, normal tissues).</div><div id="fspara0555" role="paragraph">(B) Expression profile of MAGEA1. Expression of MAGEA1 is plotted for HNSCC (TCGA) or normal tissues (GTEx).</div><div id="fspara0560" role="paragraph">(C) Heatmaps illustrating TCR and HLA deconvolution. For TCR deconvolution (left), each row represents a single TCR co-cultured with non-transduced reporter cells or reporter cells transduced with MAGEA1 protein fragment; reporter activity (% IFP) was measured in triplicates (columns). For HLA deconvolution (right), each row represents a reporter cell line engineered to express a single HLA, as indicated, ± MAGEA1 protein fragment.</div><div id="fspara0565" role="paragraph">(D) Dose-response of P32-39 and P32-41 TCRs against minimal MAGEA1 peptide epitope (VRFFFPSL) using IFNγ secretion as a readout. Average&nbsp;±&nbsp;SD are shown (n = 3).</div><div id="fspara0570" role="paragraph">(E) Distribution of TCRs associated with validated targets in <a id="optRJ0BubgjI1" href="#fig5">Figure 5B</a> among tumor CD8 T&nbsp;cell clusters.</div><div id="fspara0575" role="paragraph">(F) UMAP embedding showing sub-clustering and annotation of tumor myeloid cells (13,060 cells, n&nbsp;= 19 post-Tx samples, including 6 paired pre-Tx and post-Tx samples).</div><div id="fspara0580" role="paragraph">(G) Normalized expression profiles of selected key marker genes for different myeloid clusters.</div><div id="fspara0585" role="paragraph">(H) Comparison of the averaged cell frequency of each myeloid cell cluster between paired pre-Tx and post-Tx samples. Left, clusters labeled with red (blue) asterisk indicates significant increase (decrease) in post-Tx samples (n&nbsp;= 6, paired two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01). Right, line plots show paired pre-Tx and post-Tx samples from individual patients (n&nbsp;= 6).</div><div id="fspara0590" role="paragraph">(I) Comparison of tumor myeloid cell cluster frequency between treatment cohorts in tumor post-Tx samples. Average and SEM are shown for each patient group (n&nbsp;= 10 mono patients, n&nbsp;= 9 combo patients, two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05).</div><div id="fspara0595" role="paragraph">(J) UMAP projection of normalized expression profiles of indicated genes.</div><div id="fspara0600" role="paragraph">(K) Expression of receptor-ligand pairs that can mediate interaction between tumor CD8 T&nbsp;cell clusters C3/C4 and myeloid cell clusters (top: ligand from T&nbsp;cells and receptor on myeloid cells; bottom: ligand from myeloid cells and receptor on T&nbsp;cells). Dot color reflects communication probabilities and dot size represents computed p values (one-sided permutation test) for interactions that increased in post-treatment samples; empty space indicates no increase post-treatment.</div><div id="fspara0605" role="paragraph">(L) Violin plots showing the normalized scRNA-seq expression distribution of selected ligand-receptor pairs for pre-Tx and post-Tx samples, compared across tumor CD8 T&nbsp;cell and myeloid cell clusters.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">Of particular interest was a previously unidentified C<sup>∗</sup>07:02-restricted epitope of the cancer-testis antigen MAGEA1. The <i>MAGEA1</i> gene was previously shown to be overexpressed in HNSCC but not in healthy adult tissues except testis (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Figueiredo et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Figueiredo, D.L. ∙ Mamede, R.C. ∙ Proto-Siqueira, R. ...</div><div><strong>Expression of cancer testis antigens in head and neck squamous cell carcinomas</strong></div><div><em>Head Neck.</em> 2006; <strong>28</strong>:614-619</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/hed.20380" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-33745798609" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16475205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fhed.20380&amp;pmid=16475205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Two distinct TCR clonotypes, P32-39 and P32-41, mapped to the same epitope, indicative of a polyclonal response to MAGEA1 (<a id="crosref0805" href="#figs5">Figure&nbsp;S5</a>D). Both TCRs recognized cancer cell lines with endogenous expression of both C<sup>∗</sup>07:02 and MAGEA1, and the degree of T&nbsp;cell activation (IFNγ secretion) correlated with MAGEA1 expression levels (<a id="crosref0810" href="#fig5">Figure&nbsp;5</a>F). T&nbsp;cells with these TCRs efficiently killed cancer cell lines with endogenous expression of HLA-C<sup>∗</sup>07:02 and MAGEA1 (<a id="crosref0815" href="#fig5">Figure&nbsp;5</a>G). These results indicated that novel targets for TCR-engineered T&nbsp;cell therapy could be identified by investigating T&nbsp;cell clonotypes that expanded early during ICB.</div></section><section id="sec-2-6"><h3>Neoadjuvant ICB modulates T&nbsp;cell-myeloid interactions</h3><div id="p0130" role="paragraph">Diverse populations of myeloid cells shape T&nbsp;cell-mediated tumor immunity through expression of PD-1 and CTLA-4 ligands and additional cell surface and secreted molecules (<span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Engblom et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Engblom, C. ∙ Pfirschke, C. ∙ Pittet, M.J.</div><div><strong>The role of myeloid cells in cancer therapies</strong></div><div><em>Nat. Rev. Cancer.</em> 2016; <strong>16</strong>:447-462</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrc.2016.54" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84976272206" target="_blank" aria-label="Open Scopus in new tab">Scopus (539)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27339708/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrc.2016.54&amp;pmid=27339708" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Investigation of tumor myeloid populations revealed 13,060 cells that partitioned into nine sub-clusters (<a id="crosref0011" href="#figs5">Figures&nbsp;S5</a>F and <a id="crosref0825" href="#figs3">S3</a>G; <a id="crosref0830" href="#mmc2">Table&nbsp;S2</a>). The CD14+ monocyte cluster significantly increased post-treatment (cluster 1), whereas dendritic cells decreased post-treatment (<a id="crosref0835" href="#figs5">Figures&nbsp;S5</a>H and S5I). We then examined putative interactions between myeloid cell populations and T&nbsp;cells, based on expression of cognate receptor-ligand pairs. The <i>CD80</i> and <i>CD86</i> ligands for the inhibitory CTLA-4 receptor were preferentially expressed by <i>LAMP3</i><sup>+</sup> DCs (cluster 8, mregDCs) (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Maier et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Maier, B. ∙ Leader, A.M. ∙ Chen, S.T. ...</div><div><strong>A conserved dendritic-cell regulatory program limits antitumour immunity</strong></div><div><em>Nature.</em> 2020; <strong>580</strong>:257-262</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2134-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85082198759" target="_blank" aria-label="Open Scopus in new tab">Scopus (440)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32269339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2134-y&amp;pmid=32269339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), although <i>CD86</i> exhibited broader expression than <i>CD80</i>. <i>PDL1</i> (<i>CD274</i>) expression was also enriched among <i>LAMP3</i>+ DCs as well as <i>CXCL8</i>+ TAMs (<a id="crosref0845" href="#figs5">Figure&nbsp;S5</a>J). Systematic analysis of receptor-ligand interactions between CD8 T&nbsp;cell clusters 3 &amp; 4 and myeloid clusters showed many post-treatment changes, including the ligand-receptor pair of <i>CXCL16</i> – <i>CXCR6</i> and <i>IFNG</i>-inducible chemokines <i>CXCL9</i> and <i>CXCL10</i> (<a id="crosref0850" href="#figs5">Figures&nbsp;S5</a>K and S5L).</div></section><section id="sec-2-7"><h3>Neoadjuvant ICB enhances systemic tumor immunity</h3><div id="p0135" role="paragraph">We investigated whether neoadjuvant ICB could also enhance systemic tumor immunity, which may be of particular importance for protecting against distant metastases (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Fairfax et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Fairfax, B.P. ∙ Taylor, C.A. ∙ Watson, R.A. ...</div><div><strong>Peripheral CD8(+) T&nbsp;cell characteristics associated with durable responses to immune checkpoint blockade in patients with metastatic melanoma</strong></div><div><em>Nat. Med.</em> 2020; <strong>26</strong>:193-199</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0734-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85079320198" target="_blank" aria-label="Open Scopus in new tab">Scopus (196)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32042196/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0734-6&amp;pmid=32042196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">Kamphorst et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Kamphorst, A.O. ∙ Pillai, R.N. ∙ Yang, S. ...</div><div><strong>Proliferation of PD-1+ CD8 T&nbsp;cells in peripheral blood after PD-1-targeted therapy in lung cancer patients</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:4993-4998</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1705327114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85019093717" target="_blank" aria-label="Open Scopus in new tab">Scopus (588)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28446615/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1705327114&amp;pmid=28446615" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Wu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We performed sc(TCR+RNA)-seq on peripheral blood mononuclear cells (PBMCs) collected pre- and on-treatment from 27 patients (<a id="crosref0870" href="#fig1">Figure&nbsp;1</a>B; <a id="crosref0875" href="#figs6">Figure&nbsp;S6</a>A) and integrated these data with bulk TCR-seq for sensitive detection of treatment-induced clonal expansion (<a id="crosref0880" href="#fig1">Figure&nbsp;1</a>B).</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/7d37f76a-927f-4eef-ab1e-ff58822f38e2/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.018/asset/24fc7f3d-7af3-4a91-aa99-242197c9154c/main.assets/figs6.jpg" height="941" width="741" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/7d37f76a-927f-4eef-ab1e-ff58822f38e2/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Global analysis of blood immune cell populations at single-cell resolution, related to <a id="crosref1710" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0615" role="paragraph">ScRNA-seq was performed for blood CD45+ immune cells at three time points, pre-treatment (B1, n&nbsp;= 27 samples), 2-week on-treatment (B2, n&nbsp;= 24 samples) and post-treatment (B3, n&nbsp;= 18 samples).</div><div id="fspara0620" role="paragraph">(A) UMAP embedding showing major CD45+ immune cell populations (left) and distribution of cells from individual patients (right) (145,584 cells, 27 patients).</div><div id="fspara0625" role="paragraph">(B) UMAP embedding colored by patient showing the distribution of blood CD8+ T&nbsp;cells across clusters (left). Normalized top two differentially expressed genes in blood CD8 T&nbsp;cell clusters (right). The y axis of violin plots represents the normalized gene expression value.</div><div id="fspara0630" role="paragraph">(C) UMAP embedding showing sub-clusters and annotations of blood CD4 T&nbsp;cells from all samples (33,446 cells, 27 patients).</div><div id="fspara0635" role="paragraph">(D) Violin plots showing the normalized top two differentially expressed genes and other key marker genes for blood CD4 T&nbsp;cell clusters. The y axis of violin plots represents the normalized gene expression value.</div><div id="fspara0640" role="paragraph">(E) Comparison of blood CD4 T&nbsp;cell cluster frequency across time (top) and between treatment cohorts (bottom). Average and SEM are shown for treatment time points (top, partially paired two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01, <sup>∗∗∗</sup>p&nbsp;&lt;&nbsp;0.001) and treatment cohorts (bottom, n&nbsp;= 14 mono patients, n&nbsp;= 10 combo patients, two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01).</div><div id="fspara0645" role="paragraph">(F) UMAP embedding showing sub-clusters and annotations of blood myeloid cells (top) and distribution of cells from each individual patient (bottom).</div><div id="fspara0650" role="paragraph">(G) UMAP embedding colored by normalized expression profiles of selected marker genes in blood myeloid cells.</div><div id="fspara0655" role="paragraph">(H) Comparison of myeloid blood cell cluster frequency across time (left) and between treatment cohorts (right). Average and SEM are shown for all treatment time points (partially paired two-sided t test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01, <sup>∗∗∗</sup>p&nbsp;&lt;&nbsp;0.001) and treatment cohorts (right, n&nbsp;= 14 mono patients, n&nbsp;= 10 combo patients, two-sided t&nbsp;test, n.s. p&nbsp;&gt;&nbsp;0.05).</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">Among blood CD8 T&nbsp;cells, only cluster 6 changed in frequency over the course of treatment. These cells exhibited an activated, proliferative state based on <i>HLA-DRA</i>, <i>CD38</i>, and <i>MKI67</i> expression (<a id="crosref0885" href="#fig6">Figures&nbsp;6</a>A–6C and <a id="crosref0890" href="#figs6">S6</a>B; <a id="crosref0895" href="#mmc6">Table&nbsp;S6</a>). Two of nine CD4 T&nbsp;cell sub-clusters increased on treatment, including effector and T<sub>reg</sub> populations (<a id="crosref0900" href="#figs6">Figures&nbsp;S6</a>C–S6E; <a id="crosref0905" href="#mmc6">Table&nbsp;S6</a>). Multiple DC populations significantly decreased in frequency during therapy (<a id="crosref0910" href="#figs6">Figures&nbsp;S6</a>F–S6H; <a id="crosref0915" href="#mmc6">Table&nbsp;S6</a>), potentially due to migration to tumor-draining lymph nodes.</div><div id="p0145" role="paragraph">We next analyzed dynamic changes in circulating TCR clonotypes and their relationship to intra-tumoral T&nbsp;cell populations. Blood TCRs that expanded during treatment based on bulk TCR-seq were enriched in CD38+ CD8 T&nbsp;cell cluster 6 (<a id="crosref0920" href="#fig6">Figure&nbsp;6</a>D). TCRs that expanded in tumors during therapy (Tx-E) were also enriched in the CD38+ CD8 T&nbsp;cell cluster 6, whereas Tx-NE TCRs mapped to multiple clusters (<a id="crosref0925" href="#fig6">Figure&nbsp;6</a>E). Identification of circulating T&nbsp;cells with tumor-associated and treatment-responsive TCRs suggested an important connection to intra-tumoral immune responses, which we further investigated by sorting of CD38+ and HLA-DR+ cells reflecting markers of PBMC CD8 T&nbsp;cell cluster 6 (<a id="crosref0930" href="#fig6">Figure&nbsp;6</a>F). Sorted CD38+ HLA-DR+ T&nbsp;cells showed increased levels of activation markers CXCR3 and PD-1 compared with total CD8 T&nbsp;cells, similar to previous findings in lung cancer (<a id="crosref0935" href="#fig6">Figures&nbsp;6</a>G and 6H; <span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">Kamphorst et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Kamphorst, A.O. ∙ Pillai, R.N. ∙ Yang, S. ...</div><div><strong>Proliferation of PD-1+ CD8 T&nbsp;cells in peripheral blood after PD-1-targeted therapy in lung cancer patients</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:4993-4998</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1705327114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85019093717" target="_blank" aria-label="Open Scopus in new tab">Scopus (588)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28446615/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1705327114&amp;pmid=28446615" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We then performed scRNA-seq of sorted blood CD38+ HLA-DR+ CD8 cells from pre- and on-treatment (B1 and B2) blood samples from four patients. The expression profiles of these sorted cells strongly overlapped with CD8 cluster&nbsp;6, confirming successful enrichment of this population (<a id="crosref0945" href="#fig6">Figures&nbsp;6</a>I–6K and <a id="crosref0950" href="#figs7">S7</a>A; <a id="crosref0955" href="#mmc6">Table&nbsp;S6</a>).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/b8f1b822-416f-4d1f-8285-9aecc75e74d1/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.018/asset/03b126ea-0d51-4094-b93d-e81508d42aa1/main.assets/gr6.jpg" height="814" width="769" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/b8f1b822-416f-4d1f-8285-9aecc75e74d1/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Neoadjuvant ICB enhances systemic tumor immunity</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0235" role="paragraph">(A) UMAP embedding showing sub-clusters of total blood CD8 T&nbsp;cells across three time points (B1 pre-treatment, B2 on-treatment, B3 post-treatment/post-surgery, n&nbsp;= 27 B1, n&nbsp;= 24 B2, n&nbsp;= 18 B3).</div><div id="fspara0240" role="paragraph">(B) Violin plots showing normalized scRNA-seq expression of selected genes for cluster 6 of blood CD8 T&nbsp;cells.</div><div id="fspara0245" role="paragraph">(C) Longitudinal comparison of blood CD8 T&nbsp;cell cluster frequency (partially paired two-sided t test, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01; average and SEM for each patient group).</div><div id="fspara0250" role="paragraph">(D) Mapping of blood Tx-E TCRs (based on bulk TCR β repertoire sequencing) to blood CD8 T&nbsp;cell scRNA-seq data.</div><div id="fspara0255" role="paragraph">(E) Mapping of total tumor TCR clonotypes (left) and tumor Tx-E clonotypes (right), identified by bulk TCR β repertoire sequencing, to blood on-treatment (B2) CD8 T&nbsp;cell scRNA-seq data. Bar plots show distribution of tumor Tx-NE and Tx-E TCRs among blood CD8 T&nbsp;cell clusters (color code for clusters as in A).</div><div id="fspara0260" role="paragraph">(F) Strategy for sorting of Tx-E T&nbsp;cells from B1 and B2 blood samples based on surface markers inferred from scRNA-seq data.</div><div id="fspara0265" role="paragraph">(G) Representative flow cytometry plots for sorting of activated blood CD8 T&nbsp;cells.</div><div id="fspara0270" role="paragraph">(H) Comparison of sorted T&nbsp;cells as determined by flow cytometry (% live CD38+ HLA-DR+ cells) versus scRNA-seq (% CD8 cluster 6 T&nbsp;cells); fit with a linear regression model.</div><div id="fspara0275" role="paragraph">(I) Projection of sorted blood CD8 T&nbsp;cells to reference total blood CD8 T&nbsp;cells; 63.5% of sorted T&nbsp;cells projected to CD8 C6 cluster. Contour lines reflect the density of projected cells on reference UMAP embedding.</div><div id="fspara0280" role="paragraph">(J) Sub-clusters of sorted blood CD8 T&nbsp;cells (n&nbsp;= 4 B1 and B2).</div><div id="fspara0285" role="paragraph">(K) Top DE genes among sub-clusters of sorted blood CD8 T&nbsp;cells, ranked by averaged ln (fold change) compared with all other clusters and <i>Z</i> score normalized.</div><div id="fspara0290" role="paragraph">(L) Fraction of total and sorted blood CD8 TCRs that could be mapped to respective tumor bulk TCR dataset.</div><div id="fspara0295" role="paragraph">(M) Mapping of tumor Tx-E and Tx-NE TCRs to sorted blood CD8 T&nbsp;cells. Left, total sorted CD8 T&nbsp;cells; right, sorted CD8 T&nbsp;cells at B1 and B2 time points (n&nbsp;= 2 patients, P13, P32 for whom we could calculate tumor Tx-E/NE TCRs; n&nbsp;= 2 patients did not have pre-/post-treatment bulk TCR data).</div><div id="fspara0300" role="paragraph">See also <a id="crosref1645" href="#figs6">Figures&nbsp;S6</a> and <a id="crosref1650" href="#figs7">S7</a> and <a id="crosref1655" href="#mmc6">Table&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/8903ae21-c7ac-4ff1-93ea-b495d98218f9/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2022.06.018/asset/d921bbc9-6a31-421b-a8f7-db745b7a99a3/main.assets/figs7.jpg" height="941" width="754" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/8903ae21-c7ac-4ff1-93ea-b495d98218f9/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Connection between blood and tumor T&nbsp;cells and correlation with pathological response, related to <a id="crosref1715" href="#fig6">Figures&nbsp;6</a> and <a id="crosref1720" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0665" role="paragraph">(A) Validation of strategy for sorting blood CD8 cells reflective of cluster 6. Comparison of normalized average expression for selected markers between blood reference CD8 T&nbsp;cell cluster 6 (CD8-C6) and sorted blood CD8 T&nbsp;cells.</div><div id="fspara0670" role="paragraph">(B) Mapping of sorted blood single-cell TCRs to tumor CD8 single-cell data (n&nbsp;= 4 patients). Bar plots showing the fraction of cells mapped to each tumor CD8 T&nbsp;cell cluster.</div><div id="fspara0675" role="paragraph">(C) Frequency of total or emergent tumor Tx-E clonotypes that could also be found in the blood, plotted as a percentage of total tumor clonotypes for both monotherapy and combination therapy cohorts. Each point represents an individual patient for whom these frequencies could be calculated (n&nbsp;= 7 mono, n&nbsp;= 4 combo). Not significant by two-sided t test (p values shown).</div><div id="fspara0680" role="paragraph">(D) Left, representative flow cytometry plots of indicated blood CD4 T&nbsp;cell populations, pre-gated on live CD4+ T&nbsp;cells. Right, frequency (%) of CD4+ Treg cells across time points. Statistical comparison between B1 and B2 is shown (n&nbsp;= 23 B1, n&nbsp;= 24 B2, n&nbsp;= 19 B3, unpaired two-sided Wilcoxon rank-sum test).</div><div id="fspara0685" role="paragraph">(E) Summary of blood T&nbsp;cell populations that statistically increased on-treatment (B1 versus B2, Wilcoxon rank-sum test).</div><div id="fspara0690" role="paragraph">(F) Comparison of indicated CD8 T&nbsp;cell population frequencies (%) across time and within pathological response groups. Paired samples are shown where available.</div><div id="fspara0695" role="paragraph">(G) Flow cytometry analysis comparing markers of activated versus non-activated blood CD8 T&nbsp;cells. Left, representative flow cytometry plots at B1 time point showing PD-1 and KLRG1 expression gated on CD38− HLA-DR− and CD38+ HLA-DR+ populations. Right, summary of frequencies for each patient and statistical comparison for samples with ≥100 cells in each population (n&nbsp;= 14 B1, n&nbsp;= 17 B2, paired Wilcoxon rank-sum test, <sup>∗∗∗∗</sup>p&nbsp;&lt;&nbsp;0.0001).</div><div id="fspara0700" role="paragraph">(H) Flow cytometry analysis comparing markers of KLRG1− versus KLRG1+ cells pre-gated on PD-1+ CD8+ T&nbsp;cells. Top, representative flow cytometry plots at B1 time point showing CD38 and Ki-67 expression gated as shown. Bottom, summary of frequencies for each patient and statistical comparison for samples with ≥100 cells of each population (n&nbsp;= 11 B1, n&nbsp;= 15 B2, paired Wilcoxon rank-sum test, p values indicated).</div><div id="fspara0705" role="paragraph">(I) Summary of blood T&nbsp;cell populations that were statistically related to degree of pathological response (low versus high response, Wilcoxon rank-sum test p&nbsp;&lt;&nbsp;0.05).</div><div id="fspara0710" role="paragraph">(J) Violin plots of tumor CD8 T&nbsp;cell clusters, showing markers discriminating blood T&nbsp;cell populations associated with pathological response. The y axis of violin plots represents the normalized gene expression value.</div><div id="fspara0715" role="paragraph">(K) Correlation between blood T&nbsp;cell populations and TCR expansion in tumors. Frequency (%) of tumor Tx-E clonotypes among total tumor TCR clonotypes is plotted versus the indicated blood T&nbsp;cell populations measured by flow cytometry. Statistical association determined by Pearson correlation (n&nbsp;= 11).</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph">A majority of TCR sequences from sorted blood CD38+ HLA-DR+ CD8 T&nbsp;cells could be matched to tumor-associated TCRs, representing a significant enrichment compared with the total CD8 blood dataset (<a id="crosref0960" href="#fig6">Figures&nbsp;6</a>L and <a id="crosref0965" href="#figs7">S7</a>B). Strikingly, tumor Tx-E TCRs were almost exclusively present in sub-clusters 3 &amp; 4 of sorted blood CD38+ HLA-DR+ CD8 T&nbsp;cells from on-treatment samples (<a id="crosref0970" href="#fig6">Figure&nbsp;6</a>M), which overexpressed chemokine receptor (<i>CXCR6</i>), cytotoxicity (<i>GNLY</i>), and inhibitory receptor (<i>LAG3</i>, <i>HAVCR2</i>) genes (cluster 3) as well as cell cycle genes (cluster 4) (<a id="crosref0975" href="#fig6">Figure&nbsp;6</a>K). These data showed that tumor-associated CD8 T&nbsp;cells could be identified in increased numbers at an early 2-week on-treatment time point in the blood based on key activation markers.</div></section><section id="sec-2-8"><h3>Novel TCRs emerge in tumor and blood during treatment</h3><div id="p0155" role="paragraph">We further investigated the relationship between blood and tumor TCR clonotype expansion over the course of ICB treatment using the bulk TCR-seq datasets. In addition to pre-existing clones (&gt;1 sequence in pre-treatment samples) that expanded during treatment, we identified a substantial number of emergent clonotypes that were undetectable in pre-treatment tumors, both of which were also detected in blood samples (<a id="crosref0980" href="#fig7">Figures&nbsp;7</a>A and 7B). For example, patient P32 showed a strong cellular response to combination therapy and high pathological response. A majority of tumor Tx-E clonotypes were also detected in on-treatment blood samples, including a large number of emergent clonotypes, despite the short 2-week time frame of neoadjuvant ICB. Treatment-responsive clonotypes tended to peak in blood samples at the 2-week on-treatment time point (B2) but contracted to low/undetectable levels at the follow-up time point (B3, ∼10–12&nbsp;weeks) (<a id="crosref0985" href="#fig7">Figures&nbsp;7</a>A and 7B), potentially due to homing to tissues where these T&nbsp;cells could provide protective immunity. Quantification of blood Tx-E clonotypes as percentage of total clonotypes showed a significant increase in patients on combination compared with monotherapy. This increase was only statistically significant for emergent, but not pre-existing, Tx-E TCRs (<a id="crosref0990" href="#fig7">Figure&nbsp;7</a>C). Patients on combination therapy also had a higher percentage of blood-expanded clonotypes shared with the tumor (<a id="crosref0995" href="#fig7">Figures&nbsp;7</a>D and <a id="crosref1000" href="#figs7">S7</a>C), consistent with the finding that CTLA-4 blockade boosts T&nbsp;cell priming (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Kvistborg et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Kvistborg, P. ∙ Philips, D. ∙ Kelderman, S. ...</div><div><strong>Anti-CTLA-4 therapy broadens the melanoma-reactive CD8+ T&nbsp;cell response</strong></div><div><em>Sci. Transl. Med.</em> 2014; <strong>6</strong>:254ra128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/scitranslmed.3008918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84907284200" target="_blank" aria-label="Open Scopus in new tab">Scopus (313)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25232180/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscitranslmed.3008918&amp;pmid=25232180" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Sharma and Allison, 2015</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Sharma, P. ∙ Allison, J.P.</div><div><strong>The future of immune checkpoint therapy</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:56-61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa8172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84928774156" target="_blank" aria-label="Open Scopus in new tab">Scopus (3636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25838373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa8172&amp;pmid=25838373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These data demonstrated that neoadjuvant ICB enhanced systemic tumor immunity.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.018/asset/890d25fb-73e3-4f74-bfe7-ab5aef1e4f6e/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2022.06.018/asset/ed87b54b-0a2d-4b66-81aa-de630d0bb0da/main.assets/gr7.jpg" height="941" width="745" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/890d25fb-73e3-4f74-bfe7-ab5aef1e4f6e/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Activation state of circulating T&nbsp;cells as biomarkers of tumor pathological response</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0310" role="paragraph">(A) Frequencies of tumor Tx-E TCR clonotypes and their associated frequencies in the blood for a patient with an exceptional cellular response (P32).</div><div id="fspara0315" role="paragraph">(B) Frequencies of tumor Tx-E TCR clonotypes and their associated frequencies in the blood. Data for four patients with notable Tx-E TCR abundance are shown. Each row represents one tumor Tx-E clonotype grouped as indicated; gray cell means TCR was not detected for given sample.</div><div id="fspara0320" role="paragraph">(C) Frequency of blood Tx-E TCR clonotypes plotted as percentage of total blood clonotypes within treatment cohorts. Shown are all Tx-E TCRs (left) or split into emergent (middle) and pre-existing (right) TCRs. Each dot represents an individual patient (n&nbsp;= 15 mono patients, n&nbsp;= 12 combo patients; two-sided Wilcoxon rank-sum test).</div><div id="fspara0325" role="paragraph">(D) Frequency of total or emergent blood Tx-E clonotypes shared with tumor, plotted as percentage of total blood clonotypes. Each point represents an individual patient for whom frequencies could be calculated (n&nbsp;= 12 mono, n&nbsp;= 10 combo; two-sided Wilcox test).</div><div id="fspara0330" role="paragraph">(E) Correlation of blood CD8 T&nbsp;cell cluster fraction with tumor pathological response at B1. Difference between low and high pathological response groups was determined (n&nbsp;= 7 for high response, n&nbsp;= 9 for low response, two-sided Wilcoxon rank-sum test, <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01).</div><div id="fspara0335" role="paragraph">(F) Kinetic changes in blood CD4 T&nbsp;cells. Left, frequency of CD38+ HLA-DR+ cells, pre-gated on live/CD3+/CD4+ cells shown across time points (n&nbsp;= 23 B1, n&nbsp;= 24 B2, n&nbsp;= 19 B3). Right, frequency of CD4+ CD38+ HLA-DR+ cells at B2 comparing cohorts (n&nbsp;= 14 B2 mono, n&nbsp;= 10 B2 combo; two-sided unpaired Wilcoxon rank-sum test).</div><div id="fspara0340" role="paragraph">(G) Kinetic changes of blood CD8 T&nbsp;cells. Left, frequency of CD38+ HLA-DR+ cells, pre-gated on live/CD3+/CD8+ cells, shown for each patient across time points and colored by treatment (n&nbsp;= 23 B1, n&nbsp;= 24 B2, n&nbsp;= 19 B3, two-sided Wilcoxon rank-sum test). Right, quantification of Ki-67+ frequency (%) among CD38+ HLA-DR+ and CD38− HLA-DR− cells for samples with ≥100 cells per population (n&nbsp;= 14 B1, n&nbsp;= 17 B2, paired two-sided Wilcoxon rank-sum test, <sup>∗∗∗∗</sup>p&nbsp;&lt;&nbsp;0.0001).</div><div id="fspara0345" role="paragraph">(H) Correlation of pathological response with frequency of CD8 CD38+ HLA-DR+ T&nbsp;cells in pre-Tx and on-Tx blood samples (B1: n&nbsp;= 8 low, n&nbsp;= 9 medium, n&nbsp;= 5 high; B2: n&nbsp;= 8 low, n&nbsp;= 8 medium, n&nbsp;= 6 high; two-sided Wilcoxon rank-sum test).</div><div id="fspara0350" role="paragraph">(I and J) Correlation of pathological response with KLRG1 status of PD-1+ CD8 T&nbsp;cells. (I) Representative flow cytometry plots identifying PD-1+ CD8 T&nbsp;cells (top) and KLRG1 expression by PD-1+ cells (bottom) from low and high pathological response patients at B1 time point. (J) Frequency of PD-1+ cells (left) and PD-1+ KLRG1− cells (right) at B1 and B2 time points; at B2 on-treatment time point, PD-1 was detected with anti-IgG4 (bound nivolumab) (B1: n= 8 low, n&nbsp;= 9 medium, n&nbsp;= 5 high; B2: n&nbsp;= 8 low, n&nbsp;= 8 medium, n&nbsp;= 6 high; two-sided Wilcoxon rank-sum test).</div><div id="fspara0355" role="paragraph">See also <a id="crosref1660" href="#figs7">Figure&nbsp;S7</a> and <a id="crosref1665" href="#mmc7">Table&nbsp;S7</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-9"><h3>Biomarkers of intra-tumoral pathological response</h3><div id="p0160" role="paragraph">The identification of TCR clonotypes that expanded in both blood and tumor during treatment suggested that blood T&nbsp;cell populations could serve as early biomarkers of response to ICB, supported by the correlation of blood CD8 cluster 6 frequency with pathological response (<a id="crosref1015" href="#fig7">Figure&nbsp;7</a>E). We further investigated the relationship between blood T&nbsp;cell populations and clinical response by analyzing serial blood samples from 28 trial patients by flow cytometry (<a id="crosref1020" href="#fig1">Figure&nbsp;1</a>B). Activated CD4 and CD8 T&nbsp;cell populations (CD38+ HLA-DR+) substantially expanded from pre-treatment (B1) to the 2-week on-treatment time point (B2) (<a id="crosref1025" href="#fig7">Figures&nbsp;7</a>F, 7G, <a id="crosref1030" href="#figs7">S7</a>D, and S7E; <a id="crosref1035" href="#mmc7">Table&nbsp;S7</a>). A large proportion of CD38+ HLA-DR+ cells were in a proliferative state based on Ki-67 expression, consistent with the scRNA-seq data (<a id="crosref1040" href="#fig7">Figure&nbsp;7</a>G). Importantly, the frequency of activated CD8 T&nbsp;cells (CD38+ HLA-DR+) in pre-treatment (B1) blood samples was associated with the degree of pathological response in the tumor (<a id="crosref1045" href="#fig7">Figures&nbsp;7</a>H and <a id="crosref1050" href="#figs7">S7</a>I) but was not associated with response at the on-treatment (B2) blood samples due to a broad increase of CD38+HLA-DR+ T&nbsp;cells across response groups (<a id="crosref1055" href="#fig7">Figures&nbsp;7</a>H and <a id="crosref1060" href="#figs7">S7</a>F). Expression of CD38 and HLA-DR activation markers was strongly associated with expression of PD-1, which is also upregulated following T&nbsp;cell activation (<a id="crosref1065" href="#figs7">Figure&nbsp;S7</a>G). However, the frequency of PD-1+ T&nbsp;cells in pre- or on-treatment blood samples did not correlate with pathological response (<a id="crosref1070" href="#fig7">Figures&nbsp;7</a>I and 7J). Further analysis showed that PD-1+ CD8 T&nbsp;cells comprised two populations defined by expression of KLRG1, a marker of terminal differentiation (<a id="crosref1075" href="#fig7">Figures&nbsp;7</a>I and <a id="crosref1080" href="#figs7">S7</a>H). Importantly, in both pre- and on-treatment blood samples (B1 and B2) the percentage of PD-1+ CD8 T&nbsp;cells that were KLRG1 negative was strongly associated with pathological response, highlighting the importance of the differentiation state of circulating PD-1+ T&nbsp;cells in their responsiveness to ICB (<a id="crosref1085" href="#fig7">Figures&nbsp;7</a>J, <a id="crosref1090" href="#figs7">S7</a>H, and S7I; <a id="crosref1095" href="#mmc7">Table&nbsp;S7</a>).</div><div id="p0165" role="paragraph">Finally, we examined expression of these markers in tumor CD8 T&nbsp;cell clusters 3 &amp; 4 that responded to pre-surgical ICB by clonal expansion. These clusters shared high expression of <i>CD38</i>, <i>HLA-DR</i>, and <i>PDCD1</i> (PD-1), although expressing minimal <i>KLRG1</i> (<a id="crosref1100" href="#figs7">Figure&nbsp;S7</a>J). Therefore, blood Tx-E and response-correlated T&nbsp;cell populations shared phenotypic markers with clonally expanded T&nbsp;cells within the tumor. The frequency of activated blood CD4 T&nbsp;cells (CD38+ HLA-DR+) positively correlated with the percentage of expanded TCR clonotypes in tumors (<a id="crosref1105" href="#figs7">Figure&nbsp;S7</a>K). These data identified activated blood T&nbsp;cell populations as important biomarkers of pathological response in pre-treatment and 2-week on-treatment blood samples.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0170" role="paragraph">Here, we identified CD8 TILs with a tissue-residency program as early responders to neoadjuvant ICB. We found that Trm responsive to neoadjuvant ICB were characterized by gene programs related to activation, cytotoxicity, and effector function, as well as inhibitory receptors including PD-1. The Trm program may be important in neoadjuvant ICB by promoting adhesion of CD8 T&nbsp;cells to tumor cells (<i>ITGAE</i>), rapid T&nbsp;cell-mediated effector function (<i>GZMB</i>, <i>PRF1</i>, and <i>IFNG</i>), and recruitment of additional immune cell populations (<i>CXCL13</i>). Our findings can be distinguished from previous studies that correlated Trm infiltration or gene sets in pre-treatment biopsies to cancer prognosis or subsequent ICB clinical response (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Banchereau et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Banchereau, R. ∙ Chitre, A.S. ∙ Scherl, A. ...</div><div><strong>Intratumoral CD103+ CD8+ T&nbsp;cells predict response to PD-L1 blockade</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2021; <strong>9</strong>:e002231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jitc-2020-002231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85103885386" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33827905/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjitc-2020-002231&amp;pmid=33827905" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Clarke et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Clarke, J. ∙ Panwar, B. ∙ Madrigal, A. ...</div><div><strong>Single-cell transcriptomic analysis of tissue-resident memory T&nbsp;cells in human lung cancer</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2019; <strong>216</strong>:2128-2149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20190249" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85071787869" target="_blank" aria-label="Open Scopus in new tab">Scopus (152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31227543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20190249&amp;pmid=31227543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-2" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-2" aria-controls="sref42-2">Krishna et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref42-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Krishna, C. ∙ DiNatale, R.G. ∙ Kuo, F. ...</div><div><strong>Single-cell sequencing links multiregional immune landscapes and tissue-resident T&nbsp;cells in ccRCC to tumor topology and therapy efficacy</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:662-677.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.03.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.03.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85104966318" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33861994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2021.03.007&amp;pmid=33861994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71-3" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-3" aria-controls="sref71-3">Savas et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref71-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Savas, P. ∙ Virassamy, B. ∙ Ye, C. ...</div><div><strong>Single-cell profiling of breast cancer T&nbsp;cells reveals a tissue-resident memory subset associated with improved prognosis</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:986-993</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0078-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85049055113" target="_blank" aria-label="Open Scopus in new tab">Scopus (656)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0078-7&amp;pmid=29942092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although such correlations are informative, many CD8 T&nbsp;cell markers also correlate with longer survival, even in the absence of immunotherapy (<span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Galon et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Galon, J. ∙ Costes, A. ∙ Sanchez-Cabo, F. ...</div><div><strong>Type, density, and location of immune cells within human colorectal tumors predict clinical outcome</strong></div><div><em>Science.</em> 2006; <strong>313</strong>:1960-1964</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1129139" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-33749319703" target="_blank" aria-label="Open Scopus in new tab">Scopus (5115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17008531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1129139&amp;pmid=17008531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Jiang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Jiang, P. ∙ Gu, S. ∙ Pan, D. ...</div><div><strong>Signatures of T&nbsp;cell dysfunction and exclusion predict cancer immunotherapy response</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:1550-1558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0136-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85052583273" target="_blank" aria-label="Open Scopus in new tab">Scopus (2802)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30127393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0136-1&amp;pmid=30127393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Tumeh et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Tumeh, P.C. ∙ Harview, C.L. ∙ Yearley, J.H. ...</div><div><strong>PD-1 blockade induces responses by inhibiting adaptive immune resistance</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:568-571</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84920956732" target="_blank" aria-label="Open Scopus in new tab">Scopus (5183)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13954&amp;pmid=25428505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Through utilization of pre- and post-treatment datasets and integrated TCR-seq approaches, we explicitly identified Trm as the predominant T&nbsp;cell population responding to neoadjuvant ICB.</div><div id="p0175" role="paragraph">It is important to consider whether these treatment-responsive populations exhibited programs of exhaustion. The paradigm of T&nbsp;cell exhaustion was first established in murine models and defined by sequential loss of proliferative capacity, cytokine production, and cytotoxicity (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Blank et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Blank, C.U. ∙ Haining, W.N. ∙ Held, W. ...</div><div><strong>Defining 'T cell exhaustion'</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:665-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0221-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85074300816" target="_blank" aria-label="Open Scopus in new tab">Scopus (855)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0221-9&amp;pmid=31570879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Philip and Schietinger, 2021</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Philip, M. ∙ Schietinger, A.</div><div><strong>CD8(+) T&nbsp;cell differentiation and dysfunction in cancer</strong></div><div><em>Nat. Rev. Immunol.</em> 2021; <strong>22</strong>:209-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-021-00574-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85109907412" target="_blank" aria-label="Open Scopus in new tab">Scopus (396)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34253904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-021-00574-3&amp;pmid=34253904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although exhaustion is defined by functional criteria, T&nbsp;cells are frequently considered to be exhausted based on expression of inhibitory receptors, including PD-1 and CTLA-4. However, inhibitory receptors are also upregulated during T&nbsp;cell activation and are constitutively expressed by tissue-resident T&nbsp;cells, including the treatment-responsive cells identified here (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Mami-Chouaib et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Mami-Chouaib, F. ∙ Blanc, C. ∙ Corgnac, S. ...</div><div><strong>Resident memory T&nbsp;cells, critical components in tumor immunology</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2018; <strong>6</strong>:87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s40425-018-0399-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85052961283" target="_blank" aria-label="Open Scopus in new tab">Scopus (198)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30180905/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs40425-018-0399-6&amp;pmid=30180905" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Singer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Singer, M. ∙ Wang, C. ∙ Cong, L. ...</div><div><strong>A distinct gene module for dysfunction uncoupled from activation in tumor-infiltrating T&nbsp;cells</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:1500-1511.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84986285632" target="_blank" aria-label="Open Scopus in new tab">Scopus (269)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27610572/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.052&amp;pmid=27610572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We hypothesize that CD8 T&nbsp;cells which clonally expanded in response to neoadjuvant ICB were not exhausted, for the following reasons: (1) Exhaustion was originally defined by an early loss of proliferative capacity; however, these cells underwent substantial clonal expansion within a short time frame in response to neoadjuvant ICB, and one of the two clusters enriched for treatment-responsive cells represented actively cycling cells; (2) Exhaustion is also defined by reduced effector responses; however, treatment-responsive cells identified here were characterized by high expression of cytotoxicity, cytokine, and chemokine genes. Rather, we propose that the activity of such tumor-infiltrating Trm (and Trm populations more generally) is highly regulated by multiple inhibitory receptors. This conclusion is consistent with findings from multiple human cancer types, including breast, lung, and ovarian cancers, where Trm are marked by proliferation signatures and clonal expansion, despite PD-1 expression (<span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Bassez et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bassez, A. ∙ Vos, H. ∙ Van Dyck, L. ...</div><div><strong>A single-cell map of intratumoral changes during anti-PD1 treatment of patients with breast cancer</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:820-832</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-021-01323-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85105408508" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33958794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-021-01323-8&amp;pmid=33958794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-3" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-3" aria-controls="sref15-3">Clarke et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref15-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Clarke, J. ∙ Panwar, B. ∙ Madrigal, A. ...</div><div><strong>Single-cell transcriptomic analysis of tissue-resident memory T&nbsp;cells in human lung cancer</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2019; <strong>216</strong>:2128-2149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20190249" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85071787869" target="_blank" aria-label="Open Scopus in new tab">Scopus (152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31227543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20190249&amp;pmid=31227543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Gueguen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Gueguen, P. ∙ Metoikidou, C. ∙ Dupic, T. ...</div><div><strong>Contribution of resident and circulating precursors to tumor-infiltrating CD8(+) T&nbsp;cell populations in lung cancer</strong></div><div><em>Sci. Immunol.</em> 2021; <strong>6</strong>:eabd5778</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.abd5778" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33514641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.abd5778&amp;pmid=33514641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Hornburg et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hornburg, M. ∙ Desbois, M. ∙ Lu, S. ...</div><div><strong>Single-cell dissection of cellular components and interactions shaping the tumor immune phenotypes in ovarian cancer</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:928-944.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85107327427" target="_blank" aria-label="Open Scopus in new tab">Scopus (158)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33961783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2021.04.004&amp;pmid=33961783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We do, however, also note that a substantial fraction of tumor-infiltrating T&nbsp;cells failed to respond to neoadjuvant ICB by clonal expansion, including a fraction of T&nbsp;cells in Trm clusters, and such T&nbsp;cells may indeed be in an exhausted state. The failure of a subset of Trm to respond to ICB may also be related to TCR specificity and/or their location within the tumor.</div><div id="p0180" role="paragraph">We identified two divergent CD8 T&nbsp;cell trajectories in tumors based on TCR clonotype sharing and pseudotime modeling (trajectory 1: <i>GZMK</i>; trajectory 2: <i>ITGAE</i>, <i>GZMB</i>). These distinct trajectories may be related to patterns of T&nbsp;cell antigen specificity in human tumors. Analysis of tumor-infiltrating T&nbsp;cells in human lung cancer following neoadjuvant PD-1 blockade showed that transcriptional programs were related to patterns of antigen specificity: neoantigen-specific T&nbsp;cells expressed elevated Trm markers (<i>ITGAE</i>, <i>HOBIT</i>) and inhibitory receptors (<i>PDCD1</i>, <i>CD39</i>), whereas virus-specific cells expressed <i>GZMK</i> and <i>NKG7</i> (EBV-specific TCRs) or <i>TCF7</i> and <i>IL7R</i> (influenza-specific TCRs) (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Caushi et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Caushi, J.X. ∙ Zhang, J. ∙ Ji, Z. ...</div><div><strong>Transcriptional programs of neoantigen-specific TIL in anti-PD-1-treated lung cancers</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:126-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03752-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85110991277" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34290408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03752-4&amp;pmid=34290408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A recent study of melanoma-infiltrating CD8 T&nbsp;cells supports this conclusion (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Oliveira et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Oliveira, G. ∙ Stromhaug, K. ∙ Klaeger, S. ...</div><div><strong>Phenotype, specificity and avidity of antitumour CD8(+) T&nbsp;cells in melanoma</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:119-125</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03704-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85110973451" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34290406/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03704-y&amp;pmid=34290406" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The transcriptional programs of neoantigen-specific CD8 T&nbsp;cells described in these two studies have similarities to the Trm program identified here (trajectory 2). Clonally expanded TCRs for which we identified tumor antigens using the T-Scan platform also mapped to clusters 3 &amp; 4, consistent with this hypothesis.</div><div id="p0185" role="paragraph">We found that the majority of Tx-E clonotypes (range of 37.5%–85.7% per patient) pre-existed in the tumor prior to treatment, consistent with a recent study of neoadjuvant PD-1 blockade for breast cancer (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Bassez et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bassez, A. ∙ Vos, H. ∙ Van Dyck, L. ...</div><div><strong>A single-cell map of intratumoral changes during anti-PD1 treatment of patients with breast cancer</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:820-832</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-021-01323-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85105408508" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33958794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-021-01323-8&amp;pmid=33958794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conversely, a substantial fraction of such clonotypes were only detectable in tumors following therapy (emergent clonotypes); emergent clones were also previously identified in basal carcinomas treated with a PD-1 mAb (<span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Yost et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Yost, K.E. ∙ Satpathy, A.T. ∙ Wells, D.K. ...</div><div><strong>Clonal replacement of tumor-specific T&nbsp;cells following PD-1 blockade</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:1251-1259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0522-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85069859998" target="_blank" aria-label="Open Scopus in new tab">Scopus (880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31359002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0522-3&amp;pmid=31359002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Integration of tumor and blood TCR repertoire datasets revealed that a large fraction of such emergent clones was present in both on-treatment blood samples (B2 time point) and surgical tumor specimens. These data suggest that early intra-tumoral ICB responses are mediated primarily by pre-existing T&nbsp;cells with a Trm gene program; in parallel, new T&nbsp;cell populations may be primed in tumor-draining lymph nodes that then traffic through the blood into the tumor. Thus, neoadjuvant ICB enhances both local and systemic tumor immunity that could contribute to control of residual local and metastatic disease.</div><div id="p0190" role="paragraph">Systematic analysis of blood T&nbsp;cells populations by scRNA-seq demonstrated enrichment of activated CD8 T&nbsp;cells (cluster&nbsp;6) at the 2-week on-treatment time point. Activated CD4 and CD8 T&nbsp;cells (CD38+ HLA-DR+) expanded significantly on-treatment, and a large fraction of these cells were in a proliferative state based on the Ki-67 marker (<span class="dropBlock reference-citations"><a id="body-ref-sref39-3" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-3" aria-controls="sref39-3">Kamphorst et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref39-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Kamphorst, A.O. ∙ Pillai, R.N. ∙ Yang, S. ...</div><div><strong>Proliferation of PD-1+ CD8 T&nbsp;cells in peripheral blood after PD-1-targeted therapy in lung cancer patients</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:4993-4998</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1705327114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85019093717" target="_blank" aria-label="Open Scopus in new tab">Scopus (588)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28446615/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1705327114&amp;pmid=28446615" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). PD-1 positive CD8 T&nbsp;cells were enriched in this activated population (CD38+ HLA-DR+). Although PD-1+ CD8 T&nbsp;cell frequency did not correlate with response, the percentage of PD-1+ CD8 T&nbsp;cells that were KLRG1-negative was strongly associated with response in both pre-treatment and 2-week on-treatment blood samples. KLRG1 is a marker of terminal differentiation, highlighting the importance of the differentiation state of circulating PD-1+ CD8 T&nbsp;cells. We hypothesize that PD-1+ KLRG1-negative CD8 T&nbsp;cells are less terminally differentiated than KLRG1-positive cells and may therefore have greater proliferative capacity and potential to acquire protective cellular states (such as Trm) in the tumor. Indeed, in a murine model of viral infection, skin Trm cells exclusively originated from KLRG1-negative CD8 T&nbsp;cells (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Mackay et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Mackay, L.K. ∙ Rahimpour, A. ∙ Ma, J.Z. ...</div><div><strong>The developmental pathway for CD103(+)CD8+ tissue-resident memory T&nbsp;cells of skin</strong></div><div><em>Nat. Immunol.</em> 2013; <strong>14</strong>:1294-1301</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2744" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84888008030" target="_blank" aria-label="Open Scopus in new tab">Scopus (948)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24162776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2744&amp;pmid=24162776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Pre-treatment blood biomarkers could therefore be valuable to identify patients who may benefit from pre-surgical ICB.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0195" role="paragraph">A limitation of this study is that paired pre- and on-treatment tumor specimens were only available from six patients because pre-treatment research biopsies were voluntary per the clinical protocol. However, we were able to perform scRNA-seq and flow cytometry analyses on serial blood samples from most patients, which provided important biomarkers of response that were consistent with major findings for tumor-infiltrating T&nbsp;cells. The favorable survival outcomes in both treatment cohorts also precluded assessment of clinical benefit of combination therapy versus monotherapy. Future trials will be required to determine whether this combination therapy improves progression-free and overall survival within the neoadjuvant setting and to further study the incidence of treatment-related adverse events with a low, single dose of anti-CTLA-4.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0200" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Human TruStain FcX™ (Fc Receptor Blocking Solution)</td><td>Biolegend</td><td>Cat# 422302; RRID:<a id="intref0015" href="https://antibodyregistry.org/AB_2818986" target="_blank">AB_2818986</a></td></tr><tr><td>Anti-human CD3 PE-Dazzle 594</td><td>Biolegend</td><td>Cat#300450;<br>RRID:<a id="intref0020" href="https://antibodyregistry.org/AB_2563618" target="_blank">AB_2563618</a></td></tr><tr><td>Anti-human CD45 Pacific Blue</td><td>Biolegend</td><td>Cat#304022<br>RRID:<a id="intref0025" href="https://antibodyregistry.org/AB_493655" target="_blank">AB_493655</a></td></tr><tr><td>Anti-human CD66b PE-Cy7</td><td>Biolegend</td><td>Cat#305116;<br>RRID:<a id="intref0030" href="https://antibodyregistry.org/AB_2566605" target="_blank">AB_2566605</a></td></tr><tr><td>Anti-human CD15 APC</td><td>Biolegend</td><td>Cat#301908;<br>RRID:<a id="intref0035" href="https://antibodyregistry.org/AB_314200" target="_blank">AB_314200</a></td></tr><tr><td>TotalSeq™-C0251 anti-human Hashtag 1 Antibody</td><td>Biolegend</td><td>Cat# 394661; RRID:<a id="intref0040" href="https://antibodyregistry.org/AB_2801031" target="_blank">AB_2801031</a></td></tr><tr><td>TotalSeq™-C0252 anti-human Hashtag 2 Antibody</td><td>Biolegend</td><td>Cat#394663; RRID:<a id="intref0045" href="https://antibodyregistry.org/AB_2801032" target="_blank">AB_2801032</a></td></tr><tr><td>TotalSeq™-C0253 anti-human Hashtag 3 Antibody</td><td>Biolegend</td><td>Cat#394665; RRID:<a id="intref0050" href="https://antibodyregistry.org/AB_2801033" target="_blank">AB_2801033</a></td></tr><tr><td>TotalSeq™-C0254 anti-human Hashtag 4 Antibody</td><td>Biolegend</td><td>Cat#394667; RRID:<a id="intref0055" href="https://antibodyregistry.org/AB_2801034" target="_blank">AB_2801034</a></td></tr><tr><td>TotalSeq™-C0255 anti-human Hashtag 5 Antibody</td><td>Biolegend</td><td>Cat#394669; RRID:<a id="intref0060" href="https://antibodyregistry.org/AB_2801035" target="_blank">AB_2801035</a></td></tr><tr><td>TotalSeq™-C0256 anti-human Hashtag 6 Antibody</td><td>Biolegend</td><td>Cat#394671; RRID:<a id="intref0065" href="https://antibodyregistry.org/AB_2820042" target="_blank">AB_2820042</a></td></tr><tr><td>TotalSeq™-C0257 anti-human Hashtag 7 Antibody</td><td>Biolegend</td><td>Cat#394673; RRID:<a id="intref0070" href="https://antibodyregistry.org/AB_2820043" target="_blank">AB_2820043</a></td></tr><tr><td>Anti-human CXCR3 BV421</td><td>Biolegend</td><td>Cat#353716; RRID:<a id="intref0075" href="https://antibodyregistry.org/AB_2561448" target="_blank">AB_2561448</a></td></tr><tr><td>Anti-human CD8a BV570</td><td>Biolegend</td><td>Cat#301038; RRID:<a id="intref0080" href="https://antibodyregistry.org/AB_2563213" target="_blank">AB_2563213</a></td></tr><tr><td>Anti-human CD4 BV650</td><td>Biolegend</td><td>Cat#317436; RRID:<a id="intref0085" href="https://antibodyregistry.org/AB_2563050" target="_blank">AB_2563050</a></td></tr><tr><td>Anti-human CD3e BV785</td><td>Biolegend</td><td>Cat#300472; RRID:<a id="intref0090" href="https://antibodyregistry.org/AB_2687178" target="_blank">AB_2687178</a></td></tr><tr><td>Anti-human CD38 Alexa Fluor 488</td><td>Biolegend</td><td>Cat#356634; RRID:<a id="intref0095" href="https://antibodyregistry.org/AB_2734381" target="_blank">AB_2734381</a></td></tr><tr><td>Anti-human CD27 PerCP-Cy5.5</td><td>Biolegend</td><td>Cat#356407; RRID:<a id="intref0100" href="https://antibodyregistry.org/AB_2561906" target="_blank">AB_2561906</a></td></tr><tr><td>Anti-human HLA-DR PE/Dazzle594</td><td>Biolegend</td><td>Cat#307654; RRID:<a id="intref0105" href="https://antibodyregistry.org/AB_2563646" target="_blank">AB_2563646</a></td></tr><tr><td>Anti-human KLRG1 PE/Cy7</td><td>Invitrogen</td><td>Cat#25-9488-42; RRID:<a id="intref0110" href="https://antibodyregistry.org/AB_2573546" target="_blank">AB_2573546</a></td></tr><tr><td>Anti-human PD-1 APC</td><td>Biolegend</td><td>Cat#329908; RRID:<a id="intref0115" href="https://antibodyregistry.org/AB_940475" target="_blank">AB_940475</a></td></tr><tr><td>Anti-human γδ TCR BV421</td><td>Biolegend</td><td>Cat#331218; RRID:<a id="intref0120" href="https://antibodyregistry.org/AB_2562317" target="_blank">AB_2562317</a></td></tr><tr><td>Anti-human CD59 V450</td><td>BD</td><td>Cat#560740; RRID:<a id="intref0125" href="https://antibodyregistry.org/AB_1727512" target="_blank">AB_1727512</a></td></tr><tr><td>Anti-human HLA-DR BV570</td><td>Biolegend</td><td>Cat#307638; RRID:<a id="intref0130" href="https://antibodyregistry.org/AB_2650882" target="_blank">AB_2650882</a></td></tr><tr><td>Anti-human CD25 BV605</td><td>Biolegend</td><td>Cat#302632; RRID:<a id="intref0135" href="https://antibodyregistry.org/AB_11218989" target="_blank">AB_11218989</a></td></tr><tr><td>Anti-human CD103 BV785</td><td>Biolegend</td><td>Cat#350230; RRID:<a id="intref0140" href="https://antibodyregistry.org/AB_2734364" target="_blank">AB_2734364</a></td></tr><tr><td>Anti-human CD8a Alexa Fluor 488</td><td>Biolegend</td><td>Cat#300916; RRID:<a id="intref0145" href="https://antibodyregistry.org/AB_756152" target="_blank">AB_756152</a></td></tr><tr><td>Anti-human CD3 Alexa Fluor 532</td><td>eBioscience</td><td>Cat#58-0038-42; RRID:<a id="intref0150" href="https://antibodyregistry.org/AB_11218675" target="_blank">AB_11218675</a></td></tr><tr><td>Anti-human PD-1 PE</td><td>eBioscience</td><td>Cat#12-9969-42; RRID:<a id="intref0155" href="https://antibodyregistry.org/AB_10736473" target="_blank">AB_10736473</a></td></tr><tr><td>Anti-human CD45 PE/Cy7</td><td>Biolegend</td><td>Cat#304010; RRID:<a id="intref0160" href="https://antibodyregistry.org/AB_314398" target="_blank">AB_314398</a></td></tr><tr><td>Anti-human CD45 Per-CpVio700</td><td>Miltenyi</td><td>Cat#130-097-527; RRID:<a id="intref0165" href="https://antibodyregistry.org/AB_2660425" target="_blank">AB_2660425</a></td></tr><tr><td>Anti-human CTLA-4 PE/Cy7</td><td>Biolegend</td><td>Cat#349914; RRID:<a id="intref0170" href="https://antibodyregistry.org/AB_2563098" target="_blank">AB_2563098</a></td></tr><tr><td>Anti-human CD4 Alexa Fluor 700</td><td>eBioscience</td><td>Cat#56-0049-42; RRID:<a id="intref0175" href="https://antibodyregistry.org/AB_11219085" target="_blank">AB_11219085</a></td></tr><tr><td>Anti-human GranzymeB PE/CF594</td><td>BD</td><td>Cat#562462; RRID:<a id="intref0180" href="https://antibodyregistry.org/AB_2737618" target="_blank">AB_2737618</a></td></tr><tr><td>Anti-human FOXP3 APC</td><td>eBioscience</td><td>Cat#17-4777-42</td></tr><tr><td>Anti-human IL-7R BV421</td><td>Biolegend</td><td>Cat#351310; RRID:<a id="intref0185" href="https://antibodyregistry.org/AB_10960140" target="_blank">AB_10960140</a></td></tr><tr><td>Anti-human CD45RO Pacific Blue</td><td>Biolegend</td><td>Cat#304216; RRID:<a id="intref0190" href="https://antibodyregistry.org/AB_493659" target="_blank">AB_493659</a></td></tr><tr><td>Anti-human HLA-DR BV510</td><td>Biolegend</td><td>Cat#307646; RRID:<a id="intref0195" href="https://antibodyregistry.org/AB_2561948" target="_blank">AB_2561948</a></td></tr><tr><td>Anti-human CXCR3 BV605</td><td>Biolegend</td><td>Cat#353728; RRID:<a id="intref0200" href="https://antibodyregistry.org/AB_2563157" target="_blank">AB_2563157</a></td></tr><tr><td>Anti-human CD4 BV711</td><td>Biolegend</td><td>Cat#317440; RRID:<a id="intref0205" href="https://antibodyregistry.org/AB_2562912" target="_blank">AB_2562912</a></td></tr><tr><td>Anti-human PD-1 PE</td><td>Biolegend</td><td>Cat#329906; RRID:<a id="intref0210" href="https://antibodyregistry.org/AB_940483" target="_blank">AB_940483</a></td></tr><tr><td>Anti-human IgG4 Fc PE</td><td>Southern Biotech</td><td>Cat#9200-09; RRID:<a id="intref0215" href="https://antibodyregistry.org/AB_2796693" target="_blank">AB_2796693</a></td></tr><tr><td>Anti-human KLRG1 Alexa Fluor 647</td><td>Biolegend</td><td>Cat#367704; RRID:<a id="intref0220" href="https://antibodyregistry.org/AB_2565981" target="_blank">AB_2565981</a></td></tr><tr><td>Anti-human FOXP3 PE/CF594</td><td>BD</td><td>Cat#563955; RRID:<a id="intref0225" href="https://antibodyregistry.org/AB_2738507" target="_blank">AB_2738507</a></td></tr><tr><td>Anti-human Ki-67 PerCp/Cy5.5</td><td>Biolegend</td><td>Cat#350520; RRID:<a id="intref0230" href="https://antibodyregistry.org/AB_2562295" target="_blank">AB_2562295</a></td></tr><tr><td>Anti-human/mouse GranzymeB PE/Cy7</td><td>Biolegend</td><td>Cat#396410; RRID:<a id="intref0235" href="https://antibodyregistry.org/AB_2801079" target="_blank">AB_2801079</a></td></tr><tr><td>Anti-mouse TCRβ biotin</td><td>Biolegend</td><td>Cat#109204; RRID:<a id="intref0240" href="https://antibodyregistry.org/AB_313427" target="_blank">AB_313427</a></td></tr><tr class="rowsep"><td>Anti-human CD3e</td><td>eBioscience</td><td>Cat# 16-0037-85; RRID:<a id="intref0245" href="https://antibodyregistry.org/AB_468855" target="_blank">AB_468855</a></td></tr><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr><td>Fresh tumor tissue biopsies or surgical resections</td><td>Dana-Farber Cancer Institute/Brigham and Women’s Hospital</td><td>IRB# DF/HCC 16-284</td></tr><tr><td>Paraffin-embedded formalin fixed tumor slides, prepared from tumor biopsies or tumor resections</td><td>Dana-Farber Cancer Institute/Brigham and Women’s Hospital</td><td>IRB# DF/HCC 16-284</td></tr><tr class="rowsep"><td>Peripheral blood mononuclear cells, prepared from whole blood draws</td><td>Dana-Farber Cancer Institute</td><td>IRB# DF/HCC 16-284</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>ACK red blood cell lysis buffer</td><td>Thermo Fisher Scientific</td><td>Cat#A1049201</td></tr><tr><td>Ultrapure Bovine Serum Albumin, RNase-free</td><td>Thermo Fisher Scientific</td><td>Cat#AM2616</td></tr><tr><td>Recombinant Human IL-2</td><td>Peprotech</td><td>Cat#200-02</td></tr><tr><td>Recombinant Human IL-7</td><td>R&amp;D Systems</td><td>Cat#207-IL-025</td></tr><tr><td>Recombinant Human IL-15</td><td>R&amp;D Systems</td><td>Cat#247-ILB-025</td></tr><tr><td>DMEM</td><td>ThermoFisher Scientific</td><td>Cat#11995073</td></tr><tr><td>RPMI 1640</td><td>ThermoFisher Scientific</td><td>Cat#21875034</td></tr><tr class="rowsep"><td>ACK Lysing Buffer</td><td>ThermoFisher Scientific</td><td>Cat#A1049201</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>10X Chromium Single Cell 5′ Library &amp; Gel Bead Kit</td><td>10X Genomics</td><td>Cat#1000006</td></tr><tr><td>10X Chromium Single Cell A Chip Kit</td><td>10X Genomics</td><td>Cat#1000152</td></tr><tr><td>10X Chromium Single Cell V(D)J Enrichment Kit, Human T Cell</td><td>10X Genomics</td><td>Cat#1000005</td></tr><tr><td>10X Chromium Single Cell 5′ Library Construction Kit</td><td>10X Genomics</td><td>Cat#1000020</td></tr><tr><td>10X Chromium Single Cell 5' Feature Barcode Library Kit</td><td>10X Genomics</td><td>Cat#1000080</td></tr><tr><td>10X Chromium i7 Multiplex Kit</td><td>10X Genomics</td><td>Cat#120262</td></tr><tr><td>Chromium i7 Multiplex Kit N, Set A</td><td>10X Genomics</td><td>Cat#1000084</td></tr><tr><td>CD45 MicroBeads, human</td><td>Miltenyi Biotec</td><td>Cat#130-045-801</td></tr><tr><td>Zombie NIR Fixable Viability Kit</td><td>Biolegend</td><td>Cat# 423106</td></tr><tr><td>Zombie Aqua™ Fixable Viability Kit</td><td>Biolegend</td><td>Cat#423102</td></tr><tr><td>StraightFrom® Leukopak® CD8 Microbead Kit</td><td>Miltenyi Biotec</td><td>Cat#130-117-019</td></tr><tr><td>Incucyte® Nuclight Red Lentivirus</td><td>Satorius</td><td>Cat#4476</td></tr><tr><td>CryoStor® CS10</td><td>Stem Cell Technologies</td><td>Cat#07959</td></tr><tr><td>Anti-Biotin MicroBeads</td><td>Miltenyi Biotec</td><td>Cat#130-090-485</td></tr><tr><td>Annexin V MicroBead Kit</td><td>Miltenyi Biotec</td><td>Cat#130-090-201</td></tr><tr class="rowsep"><td>Human IFN-gamma (3rd Gen)</td><td>Protein Simple</td><td>Cat# SPCKA-PS-002574</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Single cell RNA-seq data and bulk TCR sequencing data</td><td>This paper</td><td>GEO: GSE200996</td></tr><tr class="rowsep"><td>Bulk TCR-sequencing data from urothelial cancer dataset</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Mariathasan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Mariathasan, S. ∙ Turley, S.J. ∙ Nickles, D. ...</div><div><strong>TGFbeta attenuates tumour response to PD-L1 blockade by contributing to exclusion of T&nbsp;cells</strong></div><div><em>Nature.</em> 2018; <strong>554</strong>:544-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85042495633" target="_blank" aria-label="Open Scopus in new tab">Scopus (3242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29443960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25501&amp;pmid=29443960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0250" href="http://research-pub.gene.com/IMvigor210CoreBiologies/" target="_blank">http://research-pub.gene.com/IMvigor210CoreBiologies/</a>; EGA: EGAS00001002556</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>MHC-null HEK293T engineered to express a fluorescent reporter activated by T&nbsp;cell killing</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Ferretti et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Ferretti, A.P. ∙ Kula, T. ∙ Wang, Y. ...</div><div><strong>Unbiased screens show CD8(+) T&nbsp;cells of COVID-19 patients recognize shared epitopes in SARS-CoV-2 that largely reside outside the spike protein</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:1095-1107.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.immuni.2020.10.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.immuni.2020.10.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85094974474" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33128877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.10.006&amp;pmid=33128877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Kula et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kula, T. ∙ Dezfulian, M.H. ∙ Wang, C.I. ...</div><div><strong>T-scan: A genome-wide method for the systematic discovery of T&nbsp;cell epitopes</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1016-1028.e13</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85070183509" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31398327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.009&amp;pmid=31398327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>U266B1 Plasma cell myeloma</td><td>ATCC</td><td>Cat#TIB-196</td></tr><tr><td>A101D Melanoma</td><td>ATCC</td><td>Cat#CRL-7898</td></tr><tr><td>A2058 Melanoma</td><td>ATCC</td><td>Cat#CRL-11147</td></tr><tr><td>SKMEL5 Melanoma</td><td>ATCC</td><td>Cat#HTB-70</td></tr><tr><td>786-O Renal cell carcinoma</td><td>ATCC</td><td>Cat#CRL-1932</td></tr><tr class="rowsep"><td>Lenti-X™ 293T Cell Line</td><td>Takara</td><td>Cat#632180</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>FlowJo V10</td><td>TreeStar</td><td>RRID:<a id="intref0255" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_008520" target="_blank">SCR_008520</a></td></tr><tr><td>Prism V9</td><td>Graph Pad</td><td>RRID:<a id="intref0260" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_002798" target="_blank">SCR_002798</a></td></tr><tr><td>Cellranger mkfastq v3.0.0</td><td>10X Genomics</td><td><a id="intref0265" href="http://software.10xgenomics.com" target="_blank">http://software.10xgenomics.com</a></td></tr><tr><td>Cellranger v4.0.0</td><td>10X Genomics</td><td><a id="intref0270" href="http://software.10xgenomics.com" target="_blank">http://software.10xgenomics.com</a></td></tr><tr><td>Cumulus</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Li et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Li, B. ∙ Gould, J. ∙ Yang, Y. ...</div><div><strong>Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:793-798</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-020-0905-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85088650202" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32719530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-020-0905-x&amp;pmid=32719530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0275" href="https://cumulus.readthedocs.io/en/latest/index.html" target="_blank">https://cumulus.readthedocs.io/en/latest/index.html</a></td></tr><tr><td>R (v4.0.0 and v4.1.2)</td><td>The R Foundation</td><td><a id="intref0280" href="https://r-project.org" target="_blank">https://www.r-project.org</a></td></tr><tr><td>DropletUtils (v1.8.0, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Lun et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Lun, A.T.L. ∙ Riesenfeld, S. ∙ Andrews, T. ...</div><div><strong>EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:63</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1662-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85063349999" target="_blank" aria-label="Open Scopus in new tab">Scopus (445)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30902100/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1662-y&amp;pmid=30902100" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Bioconductor</td></tr><tr><td>DoubletFinder (v2.0.2)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">McGinnis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>McGinnis, C.S. ∙ Murrow, L.M. ∙ Gartner, Z.J.</div><div><strong>DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors</strong></div><div><em>Cell Syst.</em> 2019; <strong>8</strong>:329-337.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85064396876" target="_blank" aria-label="Open Scopus in new tab">Scopus (1321)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30954475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2019.03.003&amp;pmid=30954475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0285" href="https://github.com/chris-mcginnis-ucsf/DoubletFinder" target="_blank">https://github.com/chris-mcginnis-ucsf/DoubletFinder</a></td></tr><tr><td>Seurat (v3.2.0, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Hao et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (4942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0290" href="https://satijalab.org/seurat/index.html" target="_blank">https://satijalab.org/seurat/index.html</a></td></tr><tr><td>SingleR (v1.2.4, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Aran et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aran, D. ∙ Looney, A.P. ∙ Liu, L. ...</div><div><strong>Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:163-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-018-0276-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85060111238" target="_blank" aria-label="Open Scopus in new tab">Scopus (1970)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30643263/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-018-0276-y&amp;pmid=30643263" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Bioconductor</td></tr><tr><td>Slingshot (v1.6.1, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Street et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Street, K. ∙ Risso, D. ∙ Fletcher, R.B. ...</div><div><strong>Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics</strong></div><div><em>BMC Genomics.</em> 2018; <strong>19</strong>:477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12864-018-4772-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85048725973" target="_blank" aria-label="Open Scopus in new tab">Scopus (1200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29914354/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12864-018-4772-0&amp;pmid=29914354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Bioconductor</td></tr><tr><td>AUCell (v1.13.1, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Aibar et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aibar, S. ∙ González-Blas, C.B. ∙ Moerman, T. ...</div><div><strong>SCENIC: single-cell regulatory network inference and clustering</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:1083-1086</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4463" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85032583384" target="_blank" aria-label="Open Scopus in new tab">Scopus (2530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4463&amp;pmid=28991892" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Bioconductor</td></tr><tr><td>fgsea (v1.14.0, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Korotkevich et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Korotkevich, G. ∙ Sukhov, V. ∙ Budin, N. ...</div><div><strong>Fast gene set enrichment analysis</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/060012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F060012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Bioconductor</td></tr><tr><td>Molecular Signatures Database</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Liberzon et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Liberzon, A. ∙ Birger, C. ∙ Thorvaldsdóttir, H. ...</div><div><strong>The Molecular Signatures Database (MSigDB) hallmark gene set collection</strong></div><div><em>Cell Syst.</em> 2015; <strong>1</strong>:417-425</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84955328286" target="_blank" aria-label="Open Scopus in new tab">Scopus (6393)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.12.004&amp;pmid=26771021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0295" href="http://software.broadinstitute.org/gsea/msigdb/index.jsp" target="_blank">http://software.broadinstitute.org/gsea/msigdb/index.jsp</a></td></tr><tr><td>CellChat (v1.1.0, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Jin et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Jin, S. ∙ Guerrero-Juarez, C.F. ∙ Zhang, L. ...</div><div><strong>Inference and analysis of cell-cell communication using CellChat</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:1088</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-21246-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85101173884" target="_blank" aria-label="Open Scopus in new tab">Scopus (2221)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-21246-9&amp;pmid=33597522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0300" href="https://github.com/sqjin/CellChat" target="_blank">https://github.com/sqjin/CellChat</a></td></tr><tr><td>LymphoSeq (v1.16.0, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Coffey, 2021</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Coffey, D.</div><div><strong>LymphoSeq: analyze high-throughput sequencing of T and B&nbsp;cell receptors. R package version 3.13</strong></div><div>2021</div><div><a id="interref0010" href="https://bioconductor.org/packages/release/bioc/html/LymphoSeq.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/LymphoSeq.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.CoffeyLymphoSeq%3A+analyze+high-throughput+sequencing+of+T+and+B%C2%A0cell+receptors.+R+package+version+3.132021https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FLymphoSeq.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Bioconductor</td></tr><tr><td>Immunarch (v0.6.5, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Nazarov, 2020</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Nazarov, V.i.b.R. ∙ E.</div><div><strong>immunarch: an R package for Painless Bioinformatics Analysis of T-cell and B-cell Immune Repertoires</strong></div><div><em>Zenodo.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.5281/zenodo.3367200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.5281%2Fzenodo.3367200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0305" href="https://immunarch.com/" target="_blank">https://immunarch.com/</a></td></tr><tr><td>Genome Analysis Toolkit</td><td>Broad Institute</td><td><a id="intref0310" href="https://gatk.broadinstitute.org/hc/en-us" target="_blank">https://gatk.broadinstitute.org/hc/en-us</a></td></tr><tr><td>FACETS</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Shen and Seshan, 2016</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Shen, R. ∙ Seshan, V.E.</div><div><strong>FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84988924813" target="_blank" aria-label="Open Scopus in new tab">Scopus (712)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27270079/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw520&amp;pmid=27270079" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0315" href="https://github.com/vanallenlab/facets" target="_blank">https://github.com/vanallenlab/facets</a></td></tr><tr><td>tradeSeq (v1.2.01, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref85-1" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-1" aria-controls="sref85-1">Van den Berge et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref85-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Van den Berge, K. ∙ Roux de Bézieux, H. ∙ Street, K. ...</div><div><strong>Trajectory-based differential expression analysis for single-cell sequencing data</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:1201</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-14766-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85081332065" target="_blank" aria-label="Open Scopus in new tab">Scopus (285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32139671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-14766-3&amp;pmid=32139671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Bioconductor</td></tr><tr><td>beta-binomial model for bulk TCR expansion analysis</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Rytlewski et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Rytlewski, J. ∙ Deng, S. ∙ Xie, T. ...</div><div><strong>Model to improve specificity for identification of clinically-relevant expanded T&nbsp;cells in peripheral blood</strong></div><div><em>PLoS One.</em> 2019; <strong>14</strong>:e0213684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0213684" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85062994573" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30870493/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0213684&amp;pmid=30870493" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0320" href="https://github.com/jrytlewski/beta_binomial_paper" target="_blank">https://github.com/jrytlewski/beta_binomial_paper</a></td></tr><tr><td>IMvigor210CoreBiologies (v1.0.0, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref58-3" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-3" aria-controls="sref58-3">Mariathasan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref58-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Mariathasan, S. ∙ Turley, S.J. ∙ Nickles, D. ...</div><div><strong>TGFbeta attenuates tumour response to PD-L1 blockade by contributing to exclusion of T&nbsp;cells</strong></div><div><em>Nature.</em> 2018; <strong>554</strong>:544-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85042495633" target="_blank" aria-label="Open Scopus in new tab">Scopus (3242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29443960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25501&amp;pmid=29443960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0325" href="http://research-pub.gene.com/IMvigor210CoreBiologies/" target="_blank">http://research-pub.gene.com/IMvigor210CoreBiologies/</a></td></tr><tr><td>ProjecTILs (v1.0.0, R package)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Andreatta et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0330" href="https://github.com/carmonalab/ProjecTILs" target="_blank">https://github.com/carmonalab/ProjecTILs</a></td></tr><tr><td>GMM-demux</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref89-1" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-1" aria-controls="sref89-1">Xin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref89-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Xin, H. ∙ Lian, Q. ∙ Jiang, Y. ...</div><div><strong>GMM-Demux: sample demultiplexing, multiplet detection, experiment planning, and novel cell-type verification in single cell sequencing</strong></div><div><em>Genome Biol.</em> 2020; <strong>21</strong>:188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-020-02084-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85088907977" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32731885/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-020-02084-2&amp;pmid=32731885" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0335" href="https://github.com/CHPGenetics/GMM-Demux" target="_blank">https://github.com/CHPGenetics/GMM-Demux</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2822%2900723-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0205" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kai&nbsp;W.&nbsp;Wucherpfennig (<a id="intref0340" href="mailto:kai_wucherpfennig@dfci.harvard.edu">kai_wucherpfennig@dfci.harvard.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0210" role="paragraph">This study did not generate new unique reagents.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Patients and sample collection</h4><div id="p0215" role="paragraph">This study investigated specimens from patients with oral cavity squamous cell carcinoma enrolled in a phase 2 clinical trial that investigated neoadjuvant immune checkpoint blockade, as previously described in detail (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Schoenfeld et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Schoenfeld, J.D. ∙ Hanna, G.J. ∙ Jo, V.Y. ...</div><div><strong>Neoadjuvant nivolumab or nivolumab plus ipilimumab in untreated oral cavity squamous cell carcinoma: A Phase 2 open-label randomized clinical trial</strong></div><div><em>JAMA Oncol.</em> 2020; <strong>6</strong>:1563-1570</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jamaoncol.2020.2955" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85090783150" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32852531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Fjamaoncol.2020.2955&amp;pmid=32852531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, newly diagnosed, previously untreated patients with oral cavity squamous cell carcinoma were eligible for enrollment if they presented with locally advanced disease (T2b4) or had local lymph node metastases, without evidence of distant metastases. This protocol (16-284) was approved by the Dana-Farber Harvard Cancer Center (DF- HCC) institutional review board and written informed consent was obtained per institutional guidelines. Patients were an average age of 62 years (+/-12 years, SD) and comprised 62% men (n=18) and 38% women (n=11) (<span class="dropBlock reference-citations"><a id="body-ref-sref72-3" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-3" aria-controls="sref72-3">Schoenfeld et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref72-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Schoenfeld, J.D. ∙ Hanna, G.J. ∙ Jo, V.Y. ...</div><div><strong>Neoadjuvant nivolumab or nivolumab plus ipilimumab in untreated oral cavity squamous cell carcinoma: A Phase 2 open-label randomized clinical trial</strong></div><div><em>JAMA Oncol.</em> 2020; <strong>6</strong>:1563-1570</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jamaoncol.2020.2955" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85090783150" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32852531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Fjamaoncol.2020.2955&amp;pmid=32852531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We analyzed specimens from all patients who were initially treated in this trial (n=15 mono, n=15 combo), although one patient (P02, monotherapy group) was later removed from the formal clinical trial and excluded from response analyses due to subsequent determination of pre-treatment metastatic disease.</div><div id="p0220" role="paragraph">Tumor tissue from pre-treatment biopsies or post-treatment surgical resections was collected in DMEM media (ThermoFisher Scientific # 11995073) for rapid analysis of live cells by scRNA-seq, while another part of the tissue was fixed in formalin. For patients with a complete or near-complete response to treatment, surgical specimens from the necrotic tumor bed were collected for immune cell profiling. FFPE-embedded diagnostic pre-treatment biopsies were also investigated.</div><div id="p0225" role="paragraph">Blood was collected in BD Vacutainer® CPT™ tubes (Becton Dickinson) with sodium heparin and mixed to prevent clotting. Tubes were centrifuged at 1500 <i>g</i> for 15&nbsp;minutes, the PBMC layer was collected and washed with PBS. PBMCs were frozen in a solution of 80% RPMI/10% FBS/10% DMSO (1-15x10<sup>6</sup> cells/mL) and placed in a CoolCell™ (Corning) cell-freezing container at -80° C for up to one week, before transfer to liquid nitrogen for long-term storage.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Tumor cell suspension preparation</h4><div id="p0230" role="paragraph">Fresh tumor tissue was transferred to a 6&nbsp;cm tissue culture dish in a small volume of DMEM media and cut into &lt;2&nbsp;mm pieces with a razor blade. Tissue pieces were centrifuged (400 <i>g</i> for 5&nbsp;minutes at 4°C for all centrifuge steps) and resuspended in 2-5&nbsp;mL of a collagenase-containing enzymatic digestion solution (Miltenyi Human Tumor Dissociation Kit #30-095-929), based on the weight of collected tissue, according to manufacturer’s instructions. Biopsies were incubated at 37°C for 20-30&nbsp;minutes with mixing using a 5-10&nbsp;mL serological pipet and intermittent gentle vortexing at 2-5-minute intervals. The extent of tumor tissue dissociation was variable and likely dependent on extracellular matrix abundance. The resulting cell suspension was passed through a 70&nbsp;μm nylon mesh filter and washed with an additional 5-10&nbsp;mL of RPMI media.</div></section><section id="sec-4-4-2"><h4>Isolation of tumor-infiltrating CD45+ cells for scRNA-seq</h4><div id="p0235" role="paragraph">For patients P14-P32, tumor CD45+ cells were isolated by fluorescence-activated cell sorting (FACS). Filtered tumor cell suspensions were centrifuged and resuspended in 3&nbsp;mL ACK red blood cell lysis buffer (Gibco #A1049201) for 1&nbsp;minute at RT, then immediately diluted with 10&nbsp;mL of PBS (Life Technologies #10437028) to halt osmotic stress. Cells were pelleted and resuspended in Zombie NIR (Biolegend) viability dye solution (1:500 in PBS), followed by a 15&nbsp;minute incubation at RT in the dark. Cells were counted with a hemocytometer, yielding on average 1.3x10<sup>4</sup> live cells/mg of tumor tissue. Up to 500,000 live cells were then resuspended in 100&nbsp;μL of FACS wash buffer (FWB; PBS, 2% FBS, 1&nbsp;mM EDTA) with the addition of the following antibodies: TruStain FcX (Biolegend 422302), CD3-PE/Dazzle (Biolegend 300450), CD45-Pacific Blue (Biolegend 304022), CD66b-PE/Cy7 (Biolegend 305116) and CD15-APC (Biolegend 301908). Cells were incubated on ice for 20&nbsp;minutes, washed with 1&nbsp;mL FWB, filtered through 40&nbsp;μm flow-cap strainers, and then resuspended in 500&nbsp;μL FWB. Cells were sorted using a FACS Aria II or III instrument (BD Biosciences). For most samples, 12-15 x 10<sup>3</sup> live CD45+CD66b- mononuclear cells (MNC) cells were sorted into T&nbsp;cell media (RPMI&nbsp;+ 10% FBS) and immediately transferred to ice. CD66b+ granulocytes were excluded because these cells tend to yield low-quality scRNA-seq data using the 10x Genomics protocol.</div><div id="p0240" role="paragraph">For patient P13, tumor-infiltrating immune cells were isolated with anti-CD45 magnetic beads. Following filtering of dissociated tumor suspension as above, cells were centrifuged and then resuspended in 80&nbsp;μL of MACS buffer (0.5% FBS, 2&nbsp;mM EDTA in PBS). Human CD45 MicroBeads (20&nbsp;μL, Miltenyi) were mixed with cells, and then incubated at 4°C for 15&nbsp;minutes per manufacturer’s protocol. After washing with MACS buffer, beads were applied to a MS magnetic column (Miltenyi) placed in a magnetic MACS separator and washed with 3x500&nbsp;μL of MACS buffer. To elute cells, the column was removed from the magnet and flushed with 1&nbsp;mL of MACS buffer.</div><div id="p0245" role="paragraph">All scRNA-seq was performed using the 10x Genomics 5’ V1 assay and protocols (10x Genomics). CD45+ cells isolated as above were centrifuged and washed with PBS containing 0.05% RNase-free BSA (ThermoFisher Scientific #AM2616), leaving a volume of &lt;32&nbsp;μL after the final wash. Up to 15,000 cells based on cell sorter count were loaded into a Single Cell Chip A channel, along with reverse transcriptase reagent mixture and 5’ gel beads according to the manufacturer’s protocol. Chips were loaded into the 10x Genomics Chromium Controller for single-cell partitioning. Emulsions were immediately recovered from the chip and incubated in a deep-well block Thermocycler (Bio-Rad Laboratories) for the reverse transcription and template-switch reaction. Emulsions containing synthesized cDNAs were stored at 4°C for a maximum of 48 hours.</div></section><section id="sec-4-4-3"><h4>Preparation of total PBMCs for 5’ barcoded scRNA-seq</h4><div id="p0250" role="paragraph">PBMCs isolated as described above were thawed in groups in 5-6 vials at a time, with each vial representing an individual blood draw. Thawed PBMCs were immediately diluted into 10&nbsp;mL PBS, centrifuged (400 <i>g</i> for 5&nbsp;minutes at 4°C for all centrifugation steps) and resuspended in PBS. Cells were counted with a hemacytometer; viability of thawed cells averaged 55%. Live cells were sorted to ensure the quality of scRNA-seq libraries. Between 0.5-1x10<sup>6</sup> cells were stained with Zombie NIR (Biolegend) diluted 1:500 in PBS for 15&nbsp;minutes at RT. Cells were centrifuged and then resuspended in 90&nbsp;μL of FWB, to which the following were added: 5&nbsp;μL of TruStain Fx FcBlock (Biolegend), 2&nbsp;μL of αCD45-Pacific Blue (Biolegend), 2&nbsp;μL of αCD3-PE/Dazzle594 (Biolegend) and 2&nbsp;μL (1&nbsp;μg) of an individual TotalSeqC hashing antibody (Hashtags 1-7, Biolegend). For each group of up to six PBMC vials, unique hashtags were used for subsequent identification of cells from individual samples. Cells were incubated for 20&nbsp;minutes at 4°C, washed twice with 500&nbsp;μL FWB, and filtered through 40&nbsp;μM flow-cap strainers. Live CD45+ cells (50,000 to 100,000 cells) were then separately sorted using a FACS Aria II instrument for each hashtag-labeled sample. Cells were sorted into 500&nbsp;μL 0.1% BSA in 1.5&nbsp;mL DNA LoBind microcentrifuge tubes (Eppendorf). Following sorting, cells were centrifuged, washed with 0.25% BSA, and resuspended in a final volume of 30-50&nbsp;μL 0.25% BSA. Live cells were counted with a hemacytometer.</div><div id="p0255" role="paragraph">For generation of single-cell emulsions, 6000 live cells from each sample (up to six samples) were pooled and loaded into a single Chip A channel, along with the reverse transcriptase reagent mixture and 5’ gel beads, according to the 10x Genomics 5’ V1 manufacturer’s protocol. Chips were loaded into the 10x Genomics Chromium Controller for single-cell partitioning. Emulsions were immediately recovered from the chip and incubated in a deep-well block Thermocycler (Bio-Rad Laboratories) for the reverse transcription and template-switch reaction. Emulsions containing synthesized cDNAs were stored at 4°C for a maximum of 48 hours.</div></section><section id="sec-4-4-4"><h4>Sorting of activated blood T&nbsp;cells for scRNA-seq</h4><div id="p0260" role="paragraph">Frozen PBMC samples from four patients (P13, P19, P26, and P32) at two timepoints (B1 and B2) were sorted using T&nbsp;cell activation markers identified in scRNA-seq data. Cells were thawed in two groups (B1 or B2), diluted in FWB, and counted with a hemacytometer. Up to 5x10<sup>6</sup> cells were then centrifuged and resuspended in 1:500 Zombie Aqua (Biolegend) in PBS and incubated at RT for 15&nbsp;minutes. Cells were resuspended in TruStainFx FcBlock (1:20 dilution) in FWB, incubated at 4°C for 15&nbsp;minutes, after which additional antibodies for sorting and analysis were added as indicated in a total volume of 500&nbsp;μL FWB:<div class="figure-wrap"><figure id="undtbl2" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>Reagent</th><th>Dilution</th><th>Supplier</th><th>Catalog Number</th></tr></thead><tbody><tr><td>BV421 anti-CXCR3</td><td>1:20</td><td>Biolegend</td><td>353716</td></tr><tr><td>BV570 anti-CD8a</td><td>1:50</td><td>Biolegend</td><td>301038</td></tr><tr><td>BV650 anti-CD4</td><td>1:40</td><td>Biolegend</td><td>317436</td></tr><tr><td>BV785 anti-CD3e</td><td>1:50</td><td>Biolegend</td><td>300472</td></tr><tr><td>Alexa Fluor 488 anti-CD38</td><td>1:40</td><td>Biolegend</td><td>356634</td></tr><tr><td>PerCP-Cy5.5 anti-CD27</td><td>1:40</td><td>Biolegend</td><td>356407</td></tr><tr><td>PE/Dazzle594 anti-HLA-DR</td><td>1:20</td><td>Biolegend</td><td>307654</td></tr><tr><td>PE/Cy7 anti-KLRG1</td><td>1:20</td><td>Invitrogen</td><td>25-9488-42</td></tr><tr><td>APC anti-PD-1</td><td>1:20</td><td>Biolegend</td><td>329908</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl2&amp;pii=S0092-8674%2822%2900723-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0265" role="paragraph">Unique sample-specific hashtag TotalSeqC antibodies (Biolegend, #1-4) were added to each sample (10&nbsp;μL per sample) for identification of cells from individual specimens. Cells were stained for 20&nbsp;minutes at 4°C, washed twice with FWB and filtered through a 40&nbsp;μm flow-cap strainer. Using a BD FACS Aria II instrument (BD Biosciences), live CD3+CD8+ cells were sorted based on CD38+HLA-DR+ expression. Approximately 5,000 to 125,000 cells could be sorted for different samples. Sorted cells were washed with 0.25% BSA and counted if numbers permitted (&gt;30,000 cells).</div><div id="p0270" role="paragraph">Before loading into 10x Genomics Single Cell chips, sorted cells from each patient were pooled, using all sorted cells or up to 10,000 cells. Each pool of cells was loaded into single Chip A channel, along with the reverse transcriptase reagent mixture and 5’ gel beads according to the 10x Genomics 5’ V1 assay protocol. Emulsions were immediately recovered from the chip and incubated in a deep-well block Thermocycler (Bio-Rad Laboratories) for the reverse transcription and template-switch reactions.</div></section><section id="sec-4-4-5"><h4>Library preparation and sequencing for 5’ barcoded scRNA-seq</h4><div id="p0275" role="paragraph">For blood (PBMC) libraries, cDNA was amplified (12 cycles) using SC5’ Feature cDNA Primers from the Chromium Single Cell 5’ Feature Barcode Library Kit (10x Genomics). Gene expression, TCR, and hashtag libraries were prepared from cDNA according to the manufacturer’s protocols. Quality of cDNAs and resulting libraries was evaluated using the D5000 and D1000 high-sensitivity kits on a 2200 TapeStation system (Agilent).</div><div id="p0280" role="paragraph">Tumor-infiltrating CD45+ scRNA-seq gene expression libraries were sequenced on the Illumina HiSeq X platform with a Read2 only configuration. Tumor TCR libraries were sequenced on either the Illumina Miseq (V2 flowcell), Hiseq 2500, or HiSeq 4000 platform, all utilizing a 150&nbsp;bp paired-end read configuration. All PBMC libraries (including gene expression, TCR, and hashtag barcodes) were sequenced on the Illumina HiSeq 4000 platform using a 150&nbsp;bp paired-end read configuration. Data were demultiplexed by the appropriate i7 SI-GA sample index (Cellranger mkfastq, 10x Genomics) and fastq files were transferred for downstream processing.</div></section><section id="sec-4-4-6"><h4>Bulk-TCR β chain repertoire sequencing</h4><div id="p0285" role="paragraph">All bulk TCR sequencing was performed using the Adaptive Biotechnologies ImmunoSeq deep TCR β assay. Formalin-fixed paraffin embedded (FFPE) sections were used for tumor TCR sequencing. For PBMC TCR sequencing, between 0.5-2.0 x10<sup>6</sup> PBMCs were pelleted and snap-frozen. DNA extraction, library preparation, and sequencing was performed by Adaptive Biotechnologies according to standard protocols. TCR V-Jβ segment annotation, CDR3β loop sequences, and counts were delivered and analyzed as described below.</div></section><section id="sec-4-4-7"><h4>Flow cytometry analysis of tumor-infiltrating T&nbsp;cell populations</h4><div id="p0290" role="paragraph">Up to 1x10<sup>6</sup> dissociated cells from tumors were resuspended in Zombie NIR (Biolegend) viability dye solution (1:500 in PBS), followed by 15&nbsp;minutes incubation at RT in the dark. Cells were then centrifuged and stained in FWB (50-100&nbsp;μL total volume) for 30&nbsp;minutes at 4° C in the dark using antibodies to the following cell surface markers:</div><div id="p0295" role="paragraph">All steps from cDNA isolation to library preparation were completed as per manufacturer’s protocol for the 10x Genomics 5’ V1 assay (10x Genomics). For libraries of tumor-infiltrating CD45+ cells, isolated cDNA was amplified (14 cycles) using standard cDNA Primer Mix, after which cDNA was divided for TCR enrichment/library preparation with the Chromium Single Cell V(D)J TCR kit, or for preparation of a gene expression library.<div class="figure-wrap"><figure id="undtbl3" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>Reagent</th><th>Dilution</th><th>Supplier</th><th>Catalog Number</th></tr></thead><tbody><tr><td>TruStain Fx FcBlock</td><td>1:20</td><td>Biolegend</td><td>422302</td></tr><tr><td>BV421 anti-γδ TCR</td><td>1:20</td><td>Biolegend</td><td>331218</td></tr><tr><td>V450 anti-CD69</td><td>1:40</td><td>BD</td><td>560740</td></tr><tr><td>BV510 anti-CD14, CD15, CD64, CD163 (Dump mixture)</td><td>1:80 each</td><td>Biolegend, BD</td><td>301842, 323028, 563459, 333628</td></tr><tr><td>BV570 anti-HLA-DR</td><td>1:20</td><td>Biolegend</td><td>307638</td></tr><tr><td>BV605 anti-CD25</td><td>1:20</td><td>Biolegend</td><td>302632</td></tr><tr><td>BV785 anti-CD103</td><td>1:40</td><td>Biolegend</td><td>350230</td></tr><tr><td>Alexa Fluor 488 anti-CD8a</td><td>1:40</td><td>Biolegend</td><td>300916</td></tr><tr><td>Alexa Fluor 532 anti-CD3</td><td>1:40</td><td>eBioscience</td><td>58-0038-42</td></tr><tr><td>PE anti-PD-1</td><td>1:40</td><td>eBioscience</td><td>12-9969-42</td></tr><tr><td>PE-Cy5 anti-CD45<sup>∗</sup></td><td>1:40</td><td>Biolegend</td><td>304010</td></tr><tr><td>Per-CpVio700 anti-CD45<sup>∗</sup></td><td>1:20</td><td>Miltenyi</td><td>130-097-527</td></tr><tr><td>PE/Cy-7 anti-CTLA-4</td><td>1:20</td><td>Biolegend</td><td>349914</td></tr><tr><td>Alexa Fluor 700 anti-CD4</td><td>1:20</td><td>eBioscience</td><td>56-0049-42</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl3&amp;pii=S0092-8674%2822%2900723-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0300" role="paragraph">Following incubation, cells were washed twice with 150-200&nbsp;μL FWB before proceeding to intracellular staining. Cells were then fixed and permeabilized with the FoxP3/transcription factor buffer staining set (eBioscience) for 20&nbsp;min at RT. Cells were washed with 1x permeabilization buffer before proceeding to intracellular staining with the following antibodies diluted in 1X permeabilization buffer:<div class="figure-wrap"><figure id="undtbl4" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>Reagent</th><th>Dilution</th><th>Supplier</th><th>Catalog Number</th></tr></thead><tbody><tr><td>PE/CF594 anti-Granzyme B</td><td>1:50</td><td>BD</td><td>562462</td></tr><tr><td>APC anti-FoxP3</td><td>1:20</td><td>eBioscience</td><td>17-4777-42</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl4&amp;pii=S0092-8674%2822%2900723-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0305" role="paragraph">Cells were stained for 20&nbsp;minutes at RT in the dark, and then washed twice with 1X permeabilization buffer. Stained cells were resuspended in 300&nbsp;μL FWB and stored at 4°C protected from light until analysis within 48 hours.</div></section><section id="sec-4-4-8"><h4>Flow cytometry analysis of blood T&nbsp;cell populations</h4><div id="p0310" role="paragraph">Up to 2x10<sup>6</sup> dissociated PBMCs were resuspended in Zombie NIR (Biolegend) viability dye solution (1:500 in PBS), followed by 15&nbsp;minutes incubation at RT in the dark. Cells were then centrifuged, resuspended in FWB and stained at 0.5-1x106 cells/mL concentration with the following antibodies:<div class="figure-wrap"><figure id="undtbl5" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>Reagent</th><th>Dilution</th><th>Supplier</th><th>Catalog Number</th></tr></thead><tbody><tr><td>TruStain Fx FcBlock</td><td>1:20</td><td>Biolegend</td><td>422302</td></tr><tr><td>BV421 anti-IL-7R</td><td>1:25</td><td>Biolegend</td><td>351310</td></tr><tr><td>Pacific Blue anti-CD45RO</td><td>1:80</td><td>Biolegend</td><td>304216</td></tr><tr><td>BV510 anti-HLA-DR</td><td>1:35</td><td>Biolegend</td><td>307646</td></tr><tr><td>BV570 anti-CD8α</td><td>1:50</td><td>Biolegend</td><td>301038</td></tr><tr><td>BV605 anti-CXCR3</td><td>1:25</td><td>Biolegend</td><td>353728</td></tr><tr><td>BV711 anti-CD4</td><td>1:25</td><td>Biolegend</td><td>317440</td></tr><tr><td>BV785 anti-CD103</td><td>1:50</td><td>Biolegend</td><td>350229</td></tr><tr><td>Alexa Fluor 488 anti-CD38</td><td>1:25</td><td>Biolegend</td><td>356634</td></tr><tr><td>Alexa Fluor 532 anti-CD3</td><td>1:50</td><td>Invitrogen</td><td>58-0038-42</td></tr><tr><td>PE anti-PD-1<sup>∗</sup></td><td>1:40</td><td>Biolegend</td><td>329906</td></tr><tr><td>PE anti-hIgG4<sup>∗</sup></td><td>1:20</td><td>Southern Biotech</td><td>9200-09</td></tr><tr><td>Alexa Fluor 647 anti-KLRG1</td><td>1:20</td><td>Biolegend</td><td>367704</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl5&amp;pii=S0092-8674%2822%2900723-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0315" role="paragraph">Following incubation, cells were washed twice with 150-200&nbsp;μL FWB before proceeding to intracellular staining. Cells were then fixed and permeabilized with the FoxP3/transcription factor buffer staining set (eBioscience) for 20&nbsp;min at RT. Cells were washed with 1x permeabilization buffer before proceeding to intracellular staining with the following antibodies diluted in 1X permeabilization buffer:<div class="figure-wrap"><figure id="undtbl6" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>Reagent</th><th>Dilution</th><th>Supplier</th><th>Catalog Number</th></tr></thead><tbody><tr><td>Alexa Fluor 532 anti-CD3</td><td>1:50</td><td>Invitrogen</td><td>58-0038-42</td></tr><tr><td>PE-CF594 anti-FoxP3</td><td>1:25</td><td>BD</td><td>563955</td></tr><tr><td>PerCP-Cy5.5 anti-Ki-67</td><td>1:25</td><td>Biolegend</td><td>350520</td></tr><tr><td>PE-Cy7 anti-Granzyme B</td><td>1:50</td><td>Biolegend</td><td>396410</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl6&amp;pii=S0092-8674%2822%2900723-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0320" role="paragraph">Cells were stained for 20&nbsp;minutes at RT in the dark, and then washed twice with 1X permeabilization buffer. Stained cells were resuspended in 300&nbsp;μL FWB and analyzed as below.</div></section><section id="sec-4-4-9"><h4>Spectral flow cytometry analysis of tumor and blood T&nbsp;cells</h4><div id="p0325" role="paragraph">Stained cells were passed through a 40&nbsp;μm filter and analyzed using a Sony SP6800 Spectral Analyzer. Single-color controls for spectral unmixing, using the identical antibodies as for analysis, were acquired using OneComp eBeads (ThermoFisher 01-1111-41) and fixed when applicable to match experimental conditions. The single-color control for Zombie NIR viability dye was acquired with PBMCs. Data were analyzed using FlowJo (Tree Star) version 10.4. Dead cells and aggregates were excluded from analyses. Gating was confirmed with fluorescence-minus-one (FMO) controls.</div></section><section id="sec-4-4-10"><h4>T cell engineering for T-scan antigen discovery screen</h4><div id="p0330" role="paragraph">Primary CD8+ T&nbsp;cells from healthy donors were isolated using the StraightFrom® Leukopak® CD8 Microbead Kit (Miltenyi Biotec) according to manufacturer’s protocol. Isolated cells were frozen in CryoStor® CS10 (Stem Cell Technologies) and stored in liquid nitrogen until use. Individual paired TCR alpha and TCR beta sequences were cloned into a single construct expressing mouse TRBC and TRAC regions to create human-mouse TCRs separated by a P2A skip sequence. The TCR constructs were packaged with Lenti-X™ cells (Takara Bio USA, Mountain View, CA).</div><div id="p0335" role="paragraph">CD8+ T&nbsp;cells were thawed, activated, plated in 24-well plates and transduced with lentiviral particles to express a unique orphan HNSCC TCR (per well) or a control TCR with known target recognition. Four days after transduction, cells were harvested and either pooled prior to isolation (for T-Scan screens) or isolated individually (for validation and functional assays). Cells were isolated using anti-mTCR biotin antibody (BioLegend), anti-biotin microbeads (Miltenyi) in MACS running buffer for manual magnetic separation using a QuadroMACS Separator and LS columns (Miltenyi). Isolated cells were washed and resuspended in fresh complete T&nbsp;cell medium and expanded in G-Rex®10 flasks (Wilson Wolf). Cells were then frozen in CryoStor® CS10 and stored in liquid nitrogen until used for experiments.</div></section><section id="sec-4-4-11"><h4>Reporter cell line engineering for T-Scan antigen discovery screen</h4><div id="p0340" role="paragraph">MHC-null HEK293T cells were engineered to express a fluorescent reporter activated by T&nbsp;cell killing as described (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Ferretti et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Ferretti, A.P. ∙ Kula, T. ∙ Wang, Y. ...</div><div><strong>Unbiased screens show CD8(+) T&nbsp;cells of COVID-19 patients recognize shared epitopes in SARS-CoV-2 that largely reside outside the spike protein</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:1095-1107.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.immuni.2020.10.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.immuni.2020.10.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85094974474" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33128877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.10.006&amp;pmid=33128877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43-3" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-3" aria-controls="sref43-3">Kula et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kula, T. ∙ Dezfulian, M.H. ∙ Wang, C.I. ...</div><div><strong>T-scan: A genome-wide method for the systematic discovery of T&nbsp;cell epitopes</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1016-1028.e13</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85070183509" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31398327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.009&amp;pmid=31398327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). HLA alleles relevant to each patient were introduced into reporter cells (<a id="crosref1360" href="#mmc5">Table&nbsp;S5</a>) which were then transduced with a genome-wide library of 90-aa protein fragments, as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref43-4" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-4" aria-controls="sref43-4">Kula et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kula, T. ∙ Dezfulian, M.H. ∙ Wang, C.I. ...</div><div><strong>T-scan: A genome-wide method for the systematic discovery of T&nbsp;cell epitopes</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1016-1028.e13</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85070183509" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31398327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.009&amp;pmid=31398327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-12"><h4>T-Scan screen</h4><div id="p0345" role="paragraph">To stimulate T&nbsp;cells for antigen screens, CD8+ TCR-T cell pools were thawed, and re-stimulated by co-culturing with irradiated allogenic PBMCs in the presence of 0.1&nbsp;μg/mL anti-CD3 (OKT3, eBioscience) and 50&nbsp;U/mL recombinant IL-2 (Peprotech). After expansion for 7&nbsp;days, the T&nbsp;cells were added to library-transduced reporter cells and incubated at 37°C for 4h. After incubation, cells were harvested by trypsinization and labeled with Annexin V magnetic microbeads (Miltenyi) according to the manufacturer’s instructions. Annexin-labeled cells were isolated using an AutoMACS Pro (Miltenyi), and reporter-positive cells were sorted using a MoFlo Astrios EQ cell sorter (Beckman Coulter).</div><div id="p0350" role="paragraph">Genomic DNA (gDNA) was extracted from sorted cells and processed as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref22-3" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-3" aria-controls="sref22-3">Ferretti et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref22-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Ferretti, A.P. ∙ Kula, T. ∙ Wang, Y. ...</div><div><strong>Unbiased screens show CD8(+) T&nbsp;cells of COVID-19 patients recognize shared epitopes in SARS-CoV-2 that largely reside outside the spike protein</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:1095-1107.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.immuni.2020.10.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.immuni.2020.10.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85094974474" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33128877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.10.006&amp;pmid=33128877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Nucleotide sequences were mapped to the reference nucleotide library. The proportion of read counts for each tile was calculated for each screen replicate (n&nbsp;= 8) and for the input for each pool of transduced reporter cells. Enrichment of each tile was calculated by dividing the proportion of the tile in the screen replicate by the proportion of the tile in the input library. A modified geometric mean of the enrichment of an identical tile across the 8 screen replicates (calculated by adding 0.1 to all enrichment values and taking the geometric mean) was used to identify reproducible screen hits. Specific MHC-binding epitopes for each tile above the threshold of 2-fold enrichment were predicted using NetMHC4.0. Candidate epitopes for each tile were selected by identifying predicted strong binding epitopes shared across overlapping adjacent and redundant tiles that were enriched in the screen.</div></section><section id="sec-4-4-13"><h4>Peptide validation assay</h4><div id="p0355" role="paragraph">HEK293T cells expressing the target-relevant HLA alleles were seeded in a 96 well plate. The HEK293T cells were pulsed with the target peptide (Genscript) at 1ug/mL for 1 hour. HEK293T cells for all conditions were then co-cultured with the TCR-transduced T&nbsp;cells at an E:T ratio of 1:1 for 18 hours. Cells were then harvested by pipetting, transferred to V-bottom 96-well plates, and centrifuged at 2,000&nbsp;rpm for 2&nbsp;minutes. The supernatant was collected for IFNγ measurement using an ELISA-based human IFNγ 3rd generation single-plex assay (Protein Simple) according to manufacturer instructions.</div></section><section id="sec-4-4-14"><h4>Cancer Cell Line Reactivity</h4><div id="p0360" role="paragraph">Public databases were evaluated to identify cancer cell lines that expressed C<sup>∗</sup>07:02 (TCLP- Tron Cell Line Portal; <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Scholtalbers et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Scholtalbers, J. ∙ Boegel, S. ∙ Bukur, T. ...</div><div><strong>TCLP: an online cancer cell line catalogue integrating HLA type, predicted neo-epitopes, virus and gene expression</strong></div><div><em>Genome Med.</em> 2015; <strong>7</strong>:118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-015-0240-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84947767711" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26589293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-015-0240-5&amp;pmid=26589293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and variable levels of MAGEA1 transcript (CCLE-Cancer Cell Line Encyclopedia, Broad Institute). The selected cell lines, their MAGEA1 expression levels in transcripts per million (TPM), and cell culture conditions are listed:<div class="figure-wrap"><figure id="undtbl7" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>Cell line</th><th>Origin</th><th>MAGEA1 TPM</th><th>Media</th></tr></thead><tbody><tr><td>U266B1</td><td>Plasma cell myeloma</td><td>657</td><td>RPMI 15% FBS</td></tr><tr><td>A101D</td><td>Melanoma</td><td>359</td><td>DMEM 10% FBS</td></tr><tr><td>A2058</td><td>Melanoma</td><td>128</td><td>DMEM 10% FBS</td></tr><tr><td>SKMEL5</td><td>Melanoma</td><td>8</td><td>EMEM 10% FBS</td></tr><tr><td>786-O</td><td>Renal cell carcinoma</td><td>0.4</td><td>RPMI 10% FBS</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl7&amp;pii=S0092-8674%2822%2900723-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0365" role="paragraph">Cancer cell lines were purchased from ATCC (Manassas, VA) and HEK293T cells were purchased from Takara Bio (Shiga, Japan). All media were supplemented with 1% antibiotics and incubated at 37°C in 5% CO2.</div><div id="p0370" role="paragraph">Cells were seeded in a 96-well plate overnight in appropriate media. On the following day, CD8+ T&nbsp;cells were thawed, washed in complete RPMI-1640 medium, and added to the wells for an effector to target (E:T) ratio of 1:1 for IFNγ measurement and incubated overnight. Cells were then harvested by pipetting, transferred to V-bottom 96-well plates, and centrifuged at 2,000&nbsp;rpm for 2&nbsp;minutes. The supernatant was collected for IFNγ measurement using a human IFNγ 3rd generation single-plex assay (Protein Simple) according to the manufacturer’s instruction.</div></section><section id="sec-4-4-15"><h4>Incucyte® NucLight™ Red transduction and cytotoxicity assay</h4><div id="p0375" role="paragraph">Cancer cell lines were transduced with Incucyte® NucLight™ Red lentivirus reagent (Sartorius, Göttingen, Germany). Labeled cells were selected using 1.5&nbsp;μg/mL of puromycin over 72 hours. Cancer cells expressing Incucyte® NucLight™ were seeded into 96-well plates (Corning, Flintshire, UK). On the same day, T&nbsp;cells were thawed, washed in complete RPMI-1640, and allowed to rest overnight in medium with 50&nbsp;U/mL IL-2 (PeproTech), 5&nbsp;ng/mL IL-7, and 5&nbsp;ng/mL IL-15 (R&amp;D Systems). All cells were rested overnight at 37°C in 5% CO2. Serial dilutions of the T&nbsp;cells were prepared for differing effector to target ratios and co-cultured with the plated cancer cells. One image per well from two technical replicates was taken every 2 hours over a 72 hour time course. Imaging used a 4x objective lens and data were analyzed using the Incucyte® Basic Software. Red channel acquisition time was 400&nbsp;ms, and its background was subtracted by enabling Top-Hat segmentation. Edge split was applied so that the software could recognize closely spaced cells as multiple objects, rather than one object.</div></section><section id="sec-4-4-16"><h4>Exome sequencing for determination of tumor mutational burden</h4><div id="p0380" role="paragraph">DNA was extracted from FFPE tumor sections and from corresponding PBMC frozen pellets (0.5x10<sup>6</sup> cells) by the Broad Institute Genomics platform. Tumor and germline DNA were sequenced using Illumina NovaSeq 6000 and Twist bait and target capture sets. Paired-end reads were aligned to hg19 human reference genome.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><section id="sec-4-5-1"><h4>Flow cytometry data analysis</h4><div id="p0385" role="paragraph">Population frequencies exported from FlowJo were analyzed for statistical significance using Prism 8.4.1 (GraphPad) or R (v4.1.2). Summary plots were prepared using Prism 8.4.1 (GraphPad) or the ggplot2&nbsp;R package (v3.3.5). The statistical method, statistics, and sample numbers for each analysis are shown in the figure legends.</div></section><section id="sec-4-5-2"><h4>Processing of blood single cell RNA-seq data</h4><div id="p0390" role="paragraph">Raw scRNA-sequencing reads were demultiplexed and aligned to the human reference genome (GRCh38-2020-A from 10x Genomics) for each pooled library using Cell Ranger (v4.0.0). Each pooled library contained multiple patient samples and timepoints; cells from each sample were labelled with distinct Hashtag oligonucleotides (HTO, Cell Hashing). Sequencing files for Gene Expression (measure mRNA level of cells) and Antibody Capture (measure HTO levels of cells) were fed to the “cellranger count” function with the –libraries flag, and the sequences of all HTO used in the library were fed to this function with –feature-ref flag. This generated a digital gene expression matrix, which measured the number of unique molecular identifier (UMI) for each gene and each cell barcode. The raw gene-by-barcode count matrix was then processed to filter potential empty drops using emptyDrops function from DropletUtils (v1.8.0, R package) (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Lun et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Lun, A.T.L. ∙ Riesenfeld, S. ∙ Andrews, T. ...</div><div><strong>EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:63</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1662-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85063349999" target="_blank" aria-label="Open Scopus in new tab">Scopus (445)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30902100/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1662-y&amp;pmid=30902100" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with the parameters of “lower&nbsp;= 500, ignore&nbsp;= 5”. We initially selected droplets as containing a cell if total UMI counts of this droplet exceed the 99<sup>th</sup> percentile of the top 10,000 barcodes divided by 10. Finally, only cells with FDR<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than or equals" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="less than or equals"><mjx-c class="mjx-c2264">≤</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>0.01 were retained as real cells for the following analysis. The raw HTO-by-barcode matrix was subsequently filtered based on identified real cells.</div></section><section id="sec-4-5-3"><h4>Demultiplexing samples in each blood library based on hashtags</h4><div id="p0395" role="paragraph">The filtered HTO-by-barcode matrix for each pooled library was processed using GMM-demux, which is a Gaussian mixture model-based multiplet identification method with high performance, distinguishing singlets from doublets or multiplets based on detection of HTOs with default parameters (<span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Xin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Xin, H. ∙ Lian, Q. ∙ Jiang, Y. ...</div><div><strong>GMM-Demux: sample demultiplexing, multiplet detection, experiment planning, and novel cell-type verification in single cell sequencing</strong></div><div><em>Genome Biol.</em> 2020; <strong>21</strong>:188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-020-02084-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85088907977" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32731885/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-020-02084-2&amp;pmid=32731885" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Predicted singlets with a confidence score <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than or equals" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than or equals"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>0.8 were defined as true singleT cells. Cell barcodes were then classified and allocated to respective patient samples based on hashtag identities. After processing, the gene-by-barcode count matrix was refined based on these identified singlets and used for the following analyses. On average, 83.46% of cells from filtered matrix were predicted as singlets, 15.53% were predicted as doublets or multiplets, and 0.28% were negative.</div></section><section id="sec-4-5-4"><h4>Quality control of filtered blood scRNA-seq data</h4><div id="p0400" role="paragraph">We applied the following steps to filter out low-quality cells: (1) built a median-centered median absolute deviation (MAD)-variance normal distribution based on the fraction of mitochondrial genes of all single cells in each dataset and removed the cells with a mitochondrial fraction larger than the upper end of this distribution (determined by Benjamini-Hochberg corrected P&nbsp;&lt;&nbsp;0.01, for blood dataset: 10.03%) (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Pijuan-Sala et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Pijuan-Sala, B. ∙ Griffiths, J.A. ∙ Guibentif, C. ...</div><div><strong>A single-cell molecular map of mouse gastrulation and early organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:490-495</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0933-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85062300354" target="_blank" aria-label="Open Scopus in new tab">Scopus (483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787436/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0933-9&amp;pmid=30787436" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); (2) removed cells with less than 300 identified genes.</div><div id="p0405" role="paragraph">The cell hashing method could efficiently remove sample to sample doublets, although potential doublets within each patient sample sharing the same hashtag could have remained. Utilizing TCR sequences as barcodes of individual T&nbsp;cells, we sequentially applied the following steps to exclude potential doublets: (1) clustering analysis was performed based on all CD45+ cells to identify the major immune cell populations, such as T&nbsp;cells, B cells and myeloid cells; (2) among T&nbsp;cells, cells containing two TCR β chain sequences were removed (unless detected multiple times as a clonally expanded population); (3) DoubletFinder (v2.0.2) was applied to calculate the doublet scores for cells within each patient sample (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">McGinnis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>McGinnis, C.S. ∙ Murrow, L.M. ∙ Gartner, Z.J.</div><div><strong>DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors</strong></div><div><em>Cell Syst.</em> 2019; <strong>8</strong>:329-337.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85064396876" target="_blank" aria-label="Open Scopus in new tab">Scopus (1321)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30954475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2019.03.003&amp;pmid=30954475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We compared doublet scores of all cells and confirmed that TCR-identified doublets showed significantly higher doublets scores compared to the average of remaining cells. Doublet score of 80% quantile of all TCR-derived doublets was then set as cutoff value to further filter out any remaining doublets. After these processing steps, we obtained 145,584 single cells for the blood dataset.</div></section><section id="sec-4-5-5"><h4>Processing of tumor single cell RNA-seq data</h4><div id="p0410" role="paragraph">Raw scRNA-sequencing reads were demultiplexed to FASTQ files using Cell Ranger mkfastq (v3.0.0, 10x Genomics), and the Cumulus cellranger-workflow implementation of Cell Ranger count (v4.0.0) on Terra (<a id="intref0345" href="https://terra.bio/" target="_blank">https://terra.bio/</a>) was used to align reads to the human reference genome (GRCh38-2020-A) and generate a raw counts matrix (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Li et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Li, B. ∙ Gould, J. ∙ Yang, Y. ...</div><div><strong>Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:793-798</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-020-0905-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85088650202" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32719530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-020-0905-x&amp;pmid=32719530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cumulus was used to filter cells from the Cell Ranger raw counts matrix with fewer than 400&nbsp;UMIs, 200 genes, or greater than 20% of UMIs mapped to mitochondrial genes. Count matrices from each sample were pooled and initial clusters were identified based on the Seurat clustering analysis pipeline (described below). Two clusters corresponding to endothelial and stromal cells were excluded. Then, the potential doublets included in tumor were also filtered out by using the same TCR-related strategies as used in the blood data. Finally, the remaining 74,557 tumor-infiltrating immune cells were used for the following analysis.</div></section><section id="sec-4-5-6"><h4>Clustering analysis of blood and tumor-infiltrating immune cells</h4><div id="p0415" role="paragraph">The obtained UMI count matrix from the above process was fed into Seurat (v3.2.0, R package) for further processing (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Hao et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (4942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ‘NormalizeData’ function with default parameters was applied to normalize the expression level of genes in each single cell for both blood and tumor datasets. Then, 2,000 highly variable genes were identified using the ‘FindVariableFeatures’ function with ‘vst’ method. To avoid the dominant effect of T&nbsp;cell receptor variable genes in T&nbsp;cell clustering analysis, all TRAV, TRBV, TRDV and TRGV genes were removed from the high variable gene list. Immunoglobulin variable (IG[X]V genes) genes were also removed from the highly variable gene list for clustering of the whole CD45+ dataset. The ‘ScaleData’ function was used to scale and center gene expression matrices after regressing out heterogeneity associated with mitochondrial contamination. To perform clustering, the first 20 principal components were selected to construct the shared nearest neighbor (SNN) graph with ‘FindNeighbors’ function, and then clusters were determined using the Louvain algorithm. Cluster resolution was based on the principle that each cell cluster should express a unique group of genes.</div><div id="p0420" role="paragraph">To evaluate whether the transcriptome profile of cells from cluster 3 (ITGAE<sup>+</sup> CD8) are associated with cells from cluster 4 (Cycling CD8) in tumor CD8 T&nbsp;cells after mitigating the cycling effect, we subsetted aggregated cells from these two clusters and then re-performed clustering analysis based on the above procedure, with or without regressing out the effects of cell cycle heterogeneity in ‘ScaleData’ function. If the cell cycle effect was not mitigated, cells from these two clusters were again clearly separated even with newly calculated highly variable genes, but mitigating the cell cycle effect through removal of cycle-associated variable genes showed cell mixing. Therefore, cells from cluster 4 are the cycling counterpart of cluster 3, and these two clusters show a very similar transcription profile.</div></section><section id="sec-4-5-7"><h4>Annotation of cell clusters, data visualization and differential expression analysis</h4><div id="p0425" role="paragraph">The following two complementary approaches were used to annotate single cell clusters and sub-clusters: (1) we applied SingleR (v1.2.4, R package), which leverages reference transcriptomic datasets of known cell types to predict the identities of cell clusters in an unbiased way (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Aran et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aran, D. ∙ Looney, A.P. ∙ Liu, L. ...</div><div><strong>Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:163-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-018-0276-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85060111238" target="_blank" aria-label="Open Scopus in new tab">Scopus (1970)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30643263/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-018-0276-y&amp;pmid=30643263" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); (2) we checked whether well-studied marker genes of immune cell types were ranked among the top differentially expressed genes of query clusters, and then assigned the most likely identity for each cell cluster (<span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Luoma et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Luoma, A.M. ∙ Suo, S. ∙ Williams, H.L. ...</div><div><strong>Molecular pathways of colon inflammation induced by cancer immunotherapy</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:655-671.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2020.06.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2020.06.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85087986491" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32603654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.001&amp;pmid=32603654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We first applied SingleR to determine if the predicted annotations for a query cluster based on different reference datasets were consistent (<span class="dropBlock reference-citations"><a id="body-ref-sref34-3" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-3" aria-controls="sref34-3">Hao et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref34-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (4942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and then assigned the predicted cell type annotation to this cluster. If the prediction was not reliable (primarily for sub-cluster annotation), we checked top-ranked differentially expressed genes of this cluster. The ‘FindAllMarkers’ Seurat function with default parameters but with set min.pct= 0.3 was applied to identify differentially expressed genes of different clusters and sub-clusters. The uniform manifold approximation and projection (UMAP) was finally applied based on the above described SNN graph to visualize the single cell transcriptional profile in 2D space (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Becht et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (2577)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0430" role="paragraph">Statistical tests for partially paired samples from pre-Tx and post-Tx group was performed by optimal pooled t-test (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Guo and Yuan, 2017</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Guo, B. ∙ Yuan, Y.</div><div><strong>A comparative review of methods for comparing means using partially paired data</strong></div><div><em>Stat. Methods Med. Res.</em> 2017; <strong>26</strong>:1323-1340</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0962280215577111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85020729450" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25834090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0962280215577111&amp;pmid=25834090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Other bar plots, box plots, violin plots and heatmaps were generated using customized R code through ggplot2 (v3.3.2, R package).</div></section><section id="sec-4-5-8"><h4>Prediction of CD4 and CD8 T&nbsp;cells based on a published reference dataset</h4><div id="p0435" role="paragraph">As some T&nbsp;cell and NK cell populations can share similar transcriptomic profiles, it is difficult to confidently distinguish T&nbsp;cells from NK cells solely based on the clustering analysis described above. To improve T&nbsp;cell prediction, we identified potential T&nbsp;cells based on the expression of T&nbsp;cell marker genes (<i>CD3D</i>, <i>CD3E</i> and <i>CD3G</i>) and expression of rearranged αβ TCR genes. We then performed unsupervised analysis by mapping these identified T&nbsp;cells to previously annotated CITE-seq reference datasets in Seurat, which unambiguously identified CD4 and CD8 T&nbsp;cell subpopulations (<span class="dropBlock reference-citations"><a id="body-ref-sref34-4" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-4" aria-controls="sref34-4">Hao et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref34-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (4942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This annotated reference dataset contained 162,000 PBMC labeled with 228 CITE-seq antibodies and provided different layers of cell annotation. We first normalized the expression matrix of our T&nbsp;cells with ‘SCTransform’ function to match the format of the downloaded reference dataset. Only ‘CD4 T’, ‘CD8 T’ and Other T’ cell populations in the reference dataset were selected for the prediction. Then, ‘FindTransferAnchors’ function was applied to identify anchors between reference and query datasets; the precomputed supervised PCA (spca) transformation and other default parameters were used for the processing. Finally, the ’MapQuery’ function was used to transfer different T&nbsp;cell identities and protein data from reference to the query T&nbsp;cells. CD4&nbsp;T and CD8 T&nbsp;cells were selected based on a prediction score &gt;0.6. A small number of <i>CD8A</i> and <i>CD8B</i> positive cells in the CD4 T&nbsp;cell group and <i>CD4</i> positive cells in CD8 T&nbsp;cells were also removed. For the blood dataset, 33,446 CD4 T&nbsp;cells and 14,207 CD8 T&nbsp;cells were annotated; for the tumor dataset, 24,032 CD4 T&nbsp;cells and 18,126 CD8 T&nbsp;cells were identified. This clustering and annotation analysis was subsequently applied to CD4 and CD8 T&nbsp;cell subsets.</div></section><section id="sec-4-5-9"><h4>Single cell pseudotime analysis</h4><div id="p0440" role="paragraph">To define imputed pseudotime trajectories in tumor pre-Tx CD8 T&nbsp;cells, we applied the slingshot (v1.6.1, R package) (<span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Street et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Street, K. ∙ Risso, D. ∙ Fletcher, R.B. ...</div><div><strong>Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics</strong></div><div><em>BMC Genomics.</em> 2018; <strong>19</strong>:477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12864-018-4772-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85048725973" target="_blank" aria-label="Open Scopus in new tab">Scopus (1200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29914354/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12864-018-4772-0&amp;pmid=29914354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The UMAP matrix obtained from above clustering analysis was fed into ‘slingshot’ funtion, and the IL7R+ CD8 T&nbsp;cell cluster (cluster 5) was set as the root state for evaluating the cell trajectories. Then, tradeSeq (v1.2.01, R package) was used to analyze the genes along the trajectories (<span class="dropBlock reference-citations"><a id="body-ref-sref85-2" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-2" aria-controls="sref85-2">Van den Berge et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref85-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Van den Berge, K. ∙ Roux de Bézieux, H. ∙ Street, K. ...</div><div><strong>Trajectory-based differential expression analysis for single-cell sequencing data</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:1201</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-14766-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85081332065" target="_blank" aria-label="Open Scopus in new tab">Scopus (285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32139671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-14766-3&amp;pmid=32139671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The negative binomial generalized additive model (NB-GAM) was fitted by ‘fitGAM’ function to smooth each gene’s expression; nknots parameter set as 6 was evaluated by ‘evaluateK’ function. Then, ‘predictCells’ function was used to extract fitted expression values for each cell. Finally, the expression profile of interested gene set was visualized by a heatmap, in which cells from each individual trajotory were ordered based on predicted pseudotime.</div></section><section id="sec-4-5-10"><h4>Calculation of gene set activity score</h4><div id="p0445" role="paragraph">We applied AUCell (v1.13.1, R package), incorporated in SCENIC pipeline, to evaluate the activity score of a gene set in each single cell based on transcriptome profiles (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Aibar et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aibar, S. ∙ González-Blas, C.B. ∙ Moerman, T. ...</div><div><strong>SCENIC: single-cell regulatory network inference and clustering</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:1083-1086</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4463" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85032583384" target="_blank" aria-label="Open Scopus in new tab">Scopus (2530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4463&amp;pmid=28991892" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For query gene set, AUCell first builds the area under the recovery curve based on the ranking of top 5% of all genes in each single cell; the x axis represents ranking of these highly expressed genes based on their expression values, and y axis represents the number of genes recovered from the query gene set. The gene set activity score is measured as the proportion of expressed genes in the query gene set and their relative expression levels compared to the other genes in each cell. The tissue-resident memory T (Trm) cell signature genes were derived from a previous publication (<span class="dropBlock reference-citations"><a id="body-ref-sref44-3" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-3" aria-controls="sref44-3">Kumar et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref44-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kumar, B.V. ∙ Ma, W. ∙ Miron, M. ...</div><div><strong>Human tissue-resident memory T&nbsp;cells are defined by core transcriptional and functional signatures in lymphoid and mucosal sites</strong></div><div><em>Cell Rep.</em> 2017; <strong>20</strong>:2921-2934</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.celrep.2017.08.078&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.celrep.2017.08.078&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85029578875" target="_blank" aria-label="Open Scopus in new tab">Scopus (674)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28930685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2017.08.078&amp;pmid=28930685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The cytotoxicity signature contained PRF1, <i>GZMB</i>, <i>GZMA</i>, <i>GZMH</i>, <i>NKG7</i>, and <i>GNLY</i> as derived from a previous publication (<span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Mathewson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Mathewson, N.D. ∙ Ashenberg, O. ∙ Tirosh, I. ...</div><div><strong>Inhibitory CD161 receptor identified in glioma-infiltrating T&nbsp;cells by single-cell analysis</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:1281-1298.e26</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85101668184" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.01.022&amp;pmid=33592174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The inhibitory receptor signature contained: <i>PDCD1</i>, <i>CTLA4</i>, <i>HAVCR2</i>, <i>LAG3, TIGIT</i>. The NK cell receptor signature contained: <i>KLRD1</i>, <i>FGFBP2</i>, <i>FCGR3A</i>, <i>S1PR5</i>, <i>KLRC1</i>, <i>KLRC3</i>, <i>KLRB1</i>, <i>KLRC2</i>.</div></section><section id="sec-4-5-11"><h4>Functional enrichment analysis</h4><div id="p0450" role="paragraph">The fgsea (v1.14.0, R package) was used to perform fast gene set enrichment analysis on clonal expansion associated genes (Tx-E vs Tx-NE) (<span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Korotkevich et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Korotkevich, G. ∙ Sukhov, V. ∙ Budin, N. ...</div><div><strong>Fast gene set enrichment analysis</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/060012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F060012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Pathway gene sets were downloaded from hallmark gene sets from the Molecular Signatures Database (MsigDB, <a id="intref0350" href="http://software.broadinstitute.org/gsea/msigdb/index.jsp" target="_blank">http://software.broadinstitute.org/gsea/msigdb/index.jsp</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Liberzon et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Liberzon, A. ∙ Birger, C. ∙ Thorvaldsdóttir, H. ...</div><div><strong>The Molecular Signatures Database (MSigDB) hallmark gene set collection</strong></div><div><em>Cell Syst.</em> 2015; <strong>1</strong>:417-425</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84955328286" target="_blank" aria-label="Open Scopus in new tab">Scopus (6393)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.12.004&amp;pmid=26771021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These hallmark pathways, as well as genes with averaged log-transformed fold-change (Tx-E vs. Tx-NE cells), were then fed into the ‘fgsea’ function (minSize=2, maxSize=1000, eps=0). Finally, only hallmark gene sets with an adjusted P value &lt;0.05 were selected as the functionally enriched biological states or processes.</div></section><section id="sec-4-5-12"><h4>T cell – myeloid cell interaction analysis</h4><div id="p0455" role="paragraph">To evaluate cell – cell interactions between T&nbsp;cell and myeloid cell clusters in tumors, we applied CellChat (v1.1.0, R package) to infer and visualize intercellular communication based on the expression of known ligand-receptor pairs (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Jin et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Jin, S. ∙ Guerrero-Juarez, C.F. ∙ Zhang, L. ...</div><div><strong>Inference and analysis of cell-cell communication using CellChat</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:1088</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-21246-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85101173884" target="_blank" aria-label="Open Scopus in new tab">Scopus (2221)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-21246-9&amp;pmid=33597522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here we mainly focused on interactions between tumor CD8 T&nbsp;cells of cluster 3 &amp; 4 (C3/C4), given the TCR overlap and lineage relationship for these two groups, as well as all myeloid clusters (M1-M9), with a focus on changes in predicted cell-cell communication between pre-Tx and post-Tx samples. We first inferred T&nbsp;cell - myeloid cell interactions in pre-Tx and post-Tx groups by following CellChat’s recommended analysis of individual datasets with default parameters (except that thresh.p&nbsp;= 0.05 in ‘identifyOverExpressedGenes’ function, raw.use&nbsp;= TRUE and population.size&nbsp;= TRUE in ‘computeCommunProb’ function, min.cells&nbsp;= 10 in ‘filterCommunication’ function). All categories of ligand-receptor interactions in the database were used in the analysis. Subsequently, pre-Tx and post-Tx objects were merged by ‘mergeCellChat’ function for comparison. Finally, the ‘netVisual_bubble’ function was applied to visualize differences of communication probabilities by ligand-receptor pairs in both directions, T&nbsp;cells to myeloid cells as well as myeloid cells to T&nbsp;cells.</div></section><section id="sec-4-5-13"><h4>Processing of T&nbsp;cell receptor (TCR) scRNA-seq data from blood and tumor specimens</h4><div id="p0460" role="paragraph">To generate the single-cell V(D)J sequences and annotate each library, we aligned TCR FASTQ reads to human GRCh38 V(D)J reference (v4.0.0 from 10X Genomics, modified to remove TRDD segments) using “cellranger vdj” function from Cell Ranger (v4.0.0). The filtered contig annotations, which contained high-level annotations of each high-confidenT cellular contig, were further filtered by removing records with ‘raw_consensus_id’ as ‘none’. For blood single-cell V(D)J analysis, the identities of different patient samples in each pooled library were labelled based on demultiplexing of the gene expression matrix. For all clonotypes detected in patient samples, there were 79.37% (blood) and 75.73% (tumor) of clonotypes containing one TRA and one TRB sequences as well as 8.20% (blood) and 8.18% (tumor) of clonotypes containing two TRA and one TRB sequences. Only 1.44% (blood) and 2.49% (tumor) of clonotypes contained one TRA and two TRB sequences, and only 0.53% (blood) and 2.34% (tumor) of clonotypes contained two TRA and two TRB sequences. In total, the TCR sequences for 98.20% of CD4+ T&nbsp;cells and 93.38% of CD8+ T&nbsp;cells could be detected in the blood dataset as well as for 87.05 % of CD4+ T&nbsp;cells and 86.84 % of CD8+ T&nbsp;cells in the tumor dataset.</div></section><section id="sec-4-5-14"><h4>Processing of bulk TCR repertoire sequencing data from tumor and blood samples</h4><div id="p0465" role="paragraph">Raw TCR sequences and counts were pre-processed with ‘productiveSeq’ function (set aggregate as "aminoAcid") from LymphoSeq (v1.16.0, R package) to filter out unproductive TRB rearrangements (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Coffey, 2021</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Coffey, D.</div><div><strong>LymphoSeq: analyze high-throughput sequencing of T and B&nbsp;cell receptors. R package version 3.13</strong></div><div>2021</div><div><a id="interref0010" href="https://bioconductor.org/packages/release/bioc/html/LymphoSeq.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/LymphoSeq.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.CoffeyLymphoSeq%3A+analyze+high-throughput+sequencing+of+T+and+B%C2%A0cell+receptors.+R+package+version+3.132021https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FLymphoSeq.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Matrices of blood and tumor clonotype frequencies were then generated for each patient.</div><div id="p0470" role="paragraph">To account for the possibility of blood contamination of tumor samples, we first identified potential blood clonotypes from tumor data based on the frequency (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="f"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>) of each individual clonotype in both blood and tumor, with the assumption that highly expanded clonotypes in blood would show a higher probability of being detected in tumor. For a patients with pre-Tx and post-Tx bulk TCR datasets in tumor and blood, a clonotype satisfying the criteria <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript p r e minus upper T x Superscript t u m o r" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="(24 (23 0 (12 (9 1 7 2 8 3) 4 (11 5 10 6))) (22 13 18 14 19 15 20 16 21 17))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="0,12,22" data-semantic-collapsed="(24 (23 0 12) 22)" data-semantic-owns="0 12 22" data-semantic-speech="f Subscript p r e minus upper T x Superscript t u m o r"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="24"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.303em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="13,14,15,16,17" data-semantic-content="18,19,20,21" data-semantic-parent="24" data-semantic-owns="13 18 14 19 15 20 16 21 17" style="margin-left: 0.163em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="22"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="22"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="22"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="22"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="22"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="12" data-semantic-children="9,11" data-semantic-content="4" data-semantic-parent="23" data-semantic-owns="9 4 11"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-children="1,2,3" data-semantic-content="7,8" data-semantic-parent="12" data-semantic-owns="1 7 2 8 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="9"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="9"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-parent="24" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="11" data-semantic-children="5,6" data-semantic-content="10" data-semantic-parent="12" data-semantic-owns="5 10 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="11"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>&lt;<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript p r e minus upper T x Superscript b l o o d" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="(24 (23 0 (12 (9 1 7 2 8 3) 4 (11 5 10 6))) (22 13 18 14 19 15 20 16 21 17))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="0,12,22" data-semantic-collapsed="(24 (23 0 12) 22)" data-semantic-owns="0 12 22" data-semantic-speech="f Subscript p r e minus upper T x Superscript b l o o d"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="24"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.303em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="13,14,15,16,17" data-semantic-content="18,19,20,21" data-semantic-parent="24" data-semantic-owns="13 18 14 19 15 20 16 21 17" style="margin-left: 0.163em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="22"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="22"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="22"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="22"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="22"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="12" data-semantic-children="9,11" data-semantic-content="4" data-semantic-parent="23" data-semantic-owns="9 4 11"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-children="1,2,3" data-semantic-content="7,8" data-semantic-parent="12" data-semantic-owns="1 7 2 8 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="9"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="9"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-parent="24" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="11" data-semantic-children="5,6" data-semantic-content="10" data-semantic-parent="12" data-semantic-owns="5 10 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="11"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript p o s t minus upper T x Superscript t u m o r" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 (25 0 (14 (11 1 8 2 9 3 10 4) 5 (13 6 12 7))) (24 15 20 16 21 17 22 18 23 19))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="0,14,24" data-semantic-collapsed="(26 (25 0 14) 24)" data-semantic-owns="0 14 24" data-semantic-speech="f Subscript p o s t minus upper T x Superscript t u m o r"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="26"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.303em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-children="15,16,17,18,19" data-semantic-content="20,21,22,23" data-semantic-parent="26" data-semantic-owns="15 20 16 21 17 22 18 23 19" style="margin-left: 0.163em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="24"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="24"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="24"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="24"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="24"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-children="11,13" data-semantic-content="5" data-semantic-parent="25" data-semantic-owns="11 5 13"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="1,2,3,4" data-semantic-content="8,9,10" data-semantic-parent="14" data-semantic-owns="1 8 2 9 3 10 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="11"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="11"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="26" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="13" data-semantic-children="6,7" data-semantic-content="12" data-semantic-parent="14" data-semantic-owns="6 12 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="13"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="13"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>&lt;<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript p o s t minus upper T x Superscript b l o o d" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 (25 0 (14 (11 1 8 2 9 3 10 4) 5 (13 6 12 7))) (24 15 20 16 21 17 22 18 23 19))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="0,14,24" data-semantic-collapsed="(26 (25 0 14) 24)" data-semantic-owns="0 14 24" data-semantic-speech="f Subscript p o s t minus upper T x Superscript b l o o d"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="26"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.303em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-children="15,16,17,18,19" data-semantic-content="20,21,22,23" data-semantic-parent="26" data-semantic-owns="15 20 16 21 17 22 18 23 19" style="margin-left: 0.163em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="24"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="24"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="24"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="24"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="24"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-children="11,13" data-semantic-content="5" data-semantic-parent="25" data-semantic-owns="11 5 13"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="1,2,3,4" data-semantic-content="8,9,10" data-semantic-parent="14" data-semantic-owns="1 8 2 9 3 10 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="11"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="11"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="26" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="13" data-semantic-children="6,7" data-semantic-content="12" data-semantic-parent="14" data-semantic-owns="6 12 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="13"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="13"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> was considered a blood-contaminated clonotype in tumor. For patients with pre-Tx and post-Tx bulk TCRs data in tumor but only a post-Tx TCR dataset in blood, we considered clonotypes with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript p o s t minus upper T x Superscript t u m o r" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 (25 0 (14 (11 1 8 2 9 3 10 4) 5 (13 6 12 7))) (24 15 20 16 21 17 22 18 23 19))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="0,14,24" data-semantic-collapsed="(26 (25 0 14) 24)" data-semantic-owns="0 14 24" data-semantic-speech="f Subscript p o s t minus upper T x Superscript t u m o r"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="26"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.303em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-children="15,16,17,18,19" data-semantic-content="20,21,22,23" data-semantic-parent="26" data-semantic-owns="15 20 16 21 17 22 18 23 19" style="margin-left: 0.163em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="24"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="24"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="24"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="24"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="24"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-children="11,13" data-semantic-content="5" data-semantic-parent="25" data-semantic-owns="11 5 13"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="1,2,3,4" data-semantic-content="8,9,10" data-semantic-parent="14" data-semantic-owns="1 8 2 9 3 10 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="11"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="11"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="26" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="13" data-semantic-children="6,7" data-semantic-content="12" data-semantic-parent="14" data-semantic-owns="6 12 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="13"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="13"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>&lt;<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript p o s t minus upper T x Superscript b l o o d" tabindex="0" ctxtmenu_counter="8" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 (25 0 (14 (11 1 8 2 9 3 10 4) 5 (13 6 12 7))) (24 15 20 16 21 17 22 18 23 19))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="0,14,24" data-semantic-collapsed="(26 (25 0 14) 24)" data-semantic-owns="0 14 24" data-semantic-speech="f Subscript p o s t minus upper T x Superscript b l o o d"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="26"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.303em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-children="15,16,17,18,19" data-semantic-content="20,21,22,23" data-semantic-parent="26" data-semantic-owns="15 20 16 21 17 22 18 23 19" style="margin-left: 0.163em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="24"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="24"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="24"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="24"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="24"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-children="11,13" data-semantic-content="5" data-semantic-parent="25" data-semantic-owns="11 5 13"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="1,2,3,4" data-semantic-content="8,9,10" data-semantic-parent="14" data-semantic-owns="1 8 2 9 3 10 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="11"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="11"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="26" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="13" data-semantic-children="6,7" data-semantic-content="12" data-semantic-parent="14" data-semantic-owns="6 12 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="13"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="13"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> as potential blood-contaminated clonotypes. On average, approximately 1.87% of TCRs were identified as blood contaminated clonotypes in tumor bulk TCR data and removed for downstream analyses.</div><div id="p0475" role="paragraph">The clonality metric of the TCR repertoire was calculated based on the frequencies of all productive sequences as follows (<span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Khunger et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Khunger, A. ∙ Rytlewski, J.A. ∙ Fields, P. ...</div><div><strong>The impact of CTLA-4 blockade and interferon-alpha on clonality of T-cell repertoire in the tumor microenvironment and peripheral blood of metastatic melanoma patients</strong></div><div><em>Oncoimmunology.</em> 2019; <strong>8</strong>:e1652538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/2162402X.2019.1652538" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85071041224" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31646098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F2162402X.2019.1652538&amp;pmid=31646098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>):<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper C l o n a l i t y equals 1 plus sigma summation Underscript i equals 1 Overscript n Endscripts left parenthesis p Subscript i Baseline times ln left parenthesis p Subscript i Baseline right parenthesis right parenthesis divided by ln left parenthesis n right parenthesis" tabindex="0" ctxtmenu_counter="9" style="font-size: 130%;"><mjx-math data-semantic-structure="(56 (54 0 46 1 47 2 48 3 49 4 50 5 51 6 52 7 53 8) 9 (55 10 11 (45 (18 12 (16 13 14 15) 17) (44 (35 19 (33 (22 20 21) 23 (32 24 31 (30 25 (28 26 27) 29))) 34) 36 (43 37 42 (41 38 39 40))))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="56" data-semantic-children="54,55" data-semantic-content="9" data-semantic-owns="54 9 55" data-semantic-speech="upper C l o n a l i t y equals 1 plus sigma summation Underscript i equals 1 Overscript n Endscripts left parenthesis p Subscript i Baseline times ln left parenthesis p Subscript i Baseline right parenthesis right parenthesis divided by ln left parenthesis n right parenthesis"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="54" data-semantic-children="0,1,2,3,4,5,6,7,8" data-semantic-content="46,47,48,49,50,51,52,53" data-semantic-parent="56" data-semantic-owns="0 46 1 47 2 48 3 49 4 50 5 51 6 52 7 53 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="54"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="46" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="54"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="47" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="54"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="48" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="54"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="49" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="54"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="50" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="54"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="51" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="54"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="52" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="54"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="53" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="54"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="56" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="55" data-semantic-children="10,45" data-semantic-content="11" data-semantic-parent="56" data-semantic-owns="10 11 45"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="55"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="11" data-semantic-parent="55" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="45" data-semantic-children="18,44" data-semantic-content="12" data-semantic-parent="55" data-semantic-owns="18 44"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="18" data-semantic-children="12,16,17" data-semantic-parent="45" data-semantic-owns="12 16 17" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="12" data-semantic-parent="18" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="18"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.284em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="13,15" data-semantic-content="14" data-semantic-parent="18" data-semantic-owns="13 14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="14" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="44" data-semantic-children="35,43" data-semantic-content="36" data-semantic-parent="45" data-semantic-owns="35 36 43"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="35" data-semantic-children="33" data-semantic-content="19,34" data-semantic-parent="44" data-semantic-owns="19 33 34"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="19" data-semantic-parent="35" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-children="22,32" data-semantic-content="23" data-semantic-parent="35" data-semantic-owns="22 23 32"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="22" data-semantic-children="20,21" data-semantic-parent="33" data-semantic-owns="20 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="22"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="22"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="33" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="32" data-semantic-children="24,30" data-semantic-content="31,24" data-semantic-parent="33" data-semantic-owns="24 31 30"><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="24" data-semantic-parent="32" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.694em;">n</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="30" data-semantic-children="28" data-semantic-content="25,29" data-semantic-parent="32" data-semantic-owns="25 28 29"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="25" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="30" data-semantic-owns="26 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="29" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="34" data-semantic-parent="35" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="36" data-semantic-parent="44" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-children="37,41" data-semantic-content="42,37" data-semantic-parent="44" data-semantic-owns="37 42 41"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="37" data-semantic-parent="43" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.694em;">n</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="42" data-semantic-parent="43" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="41" data-semantic-children="39" data-semantic-content="38,40" data-semantic-parent="43" data-semantic-owns="38 39 40"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="38" data-semantic-parent="41" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="41"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="40" data-semantic-parent="41" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>where <i>p</i><sub><i>i</i></sub> is the frequency of clone <i>i</i> from a patient sample with <i>n</i> unique clones. This metric is normalized to the number of unique clones and ranges from 0 to 1. A higher clonality score indicates a more uneven repertoire structure characterized by more highly expanded clones.</div><div id="p0480" role="paragraph">The Morisita index (MI), measuring the repertoire similarity between pre-Tx and post-Tx samples in each patient, was defined by the following equation (<span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Khunger et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Khunger, A. ∙ Rytlewski, J.A. ∙ Fields, P. ...</div><div><strong>The impact of CTLA-4 blockade and interferon-alpha on clonality of T-cell repertoire in the tumor microenvironment and peripheral blood of metastatic melanoma patients</strong></div><div><em>Oncoimmunology.</em> 2019; <strong>8</strong>:e1652538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/2162402X.2019.1652538" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85071041224" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31646098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F2162402X.2019.1652538&amp;pmid=31646098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>):<div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M upper I equals 2 times sigma summation Underscript i equals 1 Overscript n Endscripts a Subscript i Baseline b Subscript i Baseline divided by left parenthesis left parenthesis StartFraction sigma summation Underscript i Overscript n Endscripts a Subscript i Superscript 2 Baseline Over upper A Subscript n Superscript 2 Baseline EndFraction plus StartFraction sigma summation Underscript i Overscript n Endscripts b Subscript i Superscript 2 Baseline Over upper B Subscript n Superscript 2 Baseline EndFraction right parenthesis times upper A Subscript n Baseline upper B Subscript n Baseline right parenthesis" tabindex="0" ctxtmenu_counter="10" style="font-size: 130%;"><mjx-math data-semantic-structure="(76 (74 0 73 1) 2 (75 3 4 (72 (11 5 (9 6 7 8) 10) (71 (19 (14 12 13) 18 (17 15 16)) 20 (70 21 (68 (58 22 (56 (38 (32 (26 23 24 25) (31 (30 27 28) 29)) (37 (36 33 34) 35)) 39 (55 (49 (43 40 41 42) (48 (47 44 45) 46)) (54 (53 50 51) 52))) 57) 59 (67 (62 60 61) 66 (65 63 64))) 69)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="76" data-semantic-children="74,75" data-semantic-content="2" data-semantic-owns="74 2 75" data-semantic-speech="upper M upper I equals 2 times sigma summation Underscript i equals 1 Overscript n Endscripts a Subscript i Baseline b Subscript i Baseline divided by left parenthesis left parenthesis StartFraction sigma summation Underscript i Overscript n Endscripts a Subscript i Superscript 2 Baseline Over upper A Subscript n Superscript 2 Baseline EndFraction plus StartFraction sigma summation Underscript i Overscript n Endscripts b Subscript i Superscript 2 Baseline Over upper B Subscript n Superscript 2 Baseline EndFraction right parenthesis times upper A Subscript n Baseline upper B Subscript n Baseline right parenthesis"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="74" data-semantic-children="0,1" data-semantic-content="73" data-semantic-parent="76" data-semantic-owns="0 73 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="74"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="73" data-semantic-parent="74" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="74"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="76" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="75" data-semantic-children="3,72" data-semantic-content="4" data-semantic-parent="76" data-semantic-owns="3 4 72"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="75"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="75" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="72" data-semantic-children="11,71" data-semantic-content="5" data-semantic-parent="75" data-semantic-owns="11 71"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="11" data-semantic-children="5,9,10" data-semantic-parent="72" data-semantic-owns="5 9 10" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.284em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="11" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="71" data-semantic-children="19,70" data-semantic-content="20" data-semantic-parent="72" data-semantic-owns="19 20 70"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="19" data-semantic-children="14,17" data-semantic-content="18" data-semantic-parent="71" data-semantic-owns="14 18 17"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="12,13" data-semantic-parent="19" data-semantic-owns="12 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="19" data-semantic-owns="15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="20" data-semantic-parent="71" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="70" data-semantic-children="68" data-semantic-content="21,69" data-semantic-parent="71" data-semantic-owns="21 68 69"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="21" data-semantic-parent="70" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="68" data-semantic-children="58,67" data-semantic-content="59" data-semantic-parent="70" data-semantic-owns="58 59 67"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="58" data-semantic-children="56" data-semantic-content="22,57" data-semantic-parent="68" data-semantic-owns="22 56 57"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="22" data-semantic-parent="58" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="56" data-semantic-children="38,55" data-semantic-content="39" data-semantic-parent="58" data-semantic-owns="38 39 55"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-annotation="general:bevelled" data-semantic-id="38" data-semantic-children="32,37" data-semantic-parent="56" data-semantic-owns="32 37" bevelled="ture"><mjx-num style="vertical-align: 0.295em;"><mjx-mrow size="s" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="32" data-semantic-children="26,31" data-semantic-content="23" data-semantic-parent="38" data-semantic-owns="26 31"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="26" data-semantic-children="23,24,25" data-semantic-parent="32" data-semantic-owns="23 24 25"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="23" data-semantic-parent="26" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.287em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="31" data-semantic-children="27,28,29" data-semantic-parent="32" data-semantic-collapsed="(31 (30 27 28) 29)" data-semantic-owns="27 28 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="31"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="31"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-num><mjx-mo style="margin-left: -0.075em; margin-right: -0.075em;"><mjx-c class="mjx-c2F MM-SO">/</mjx-c></mjx-mo><mjx-den style="vertical-align: -0.111em;"><mjx-msubsup size="s" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="37" data-semantic-children="33,34,35" data-semantic-parent="38" data-semantic-collapsed="(37 (36 33 34) 35)" data-semantic-owns="33 34 35"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="37"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="37"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.223em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="37"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-den></mjx-mfrac><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="39" data-semantic-parent="56" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mfrac space="3" data-semantic-type="fraction" data-semantic-role="division" data-semantic-annotation="general:bevelled" data-semantic-id="55" data-semantic-children="49,54" data-semantic-parent="56" data-semantic-owns="49 54" bevelled="ture"><mjx-num style="vertical-align: 0.295em;"><mjx-mrow size="s" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="49" data-semantic-children="43,48" data-semantic-content="40" data-semantic-parent="55" data-semantic-owns="43 48"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="43" data-semantic-children="40,41,42" data-semantic-parent="49" data-semantic-owns="40 41 42"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="40" data-semantic-parent="43" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="43"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.287em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="43"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="48" data-semantic-children="44,45,46" data-semantic-parent="49" data-semantic-collapsed="(48 (47 44 45) 46)" data-semantic-owns="44 45 46"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="48"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.284em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-parent="48"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="48"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-num><mjx-mo style="margin-left: -0.075em; margin-right: -0.075em;"><mjx-c class="mjx-c2F MM-SO">/</mjx-c></mjx-mo><mjx-den style="vertical-align: -0.111em;"><mjx-msubsup size="s" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="54" data-semantic-children="50,51,52" data-semantic-parent="55" data-semantic-collapsed="(54 (53 50 51) 52)" data-semantic-owns="50 51 52"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="54"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="54"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.223em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="54"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-den></mjx-mfrac></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="57" data-semantic-parent="58" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="59" data-semantic-parent="68" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="67" data-semantic-children="62,65" data-semantic-content="66" data-semantic-parent="68" data-semantic-owns="62 66 65"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="62" data-semantic-children="60,61" data-semantic-parent="67" data-semantic-owns="60 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="62"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="62"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="66" data-semantic-parent="67" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="65" data-semantic-children="63,64" data-semantic-parent="67" data-semantic-owns="63 64"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="63" data-semantic-parent="65"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="65"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="69" data-semantic-parent="70" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>where <i>a</i><sub><i>i</i></sub> and <i>b</i><sub><i>i</i></sub> are the counts of clone <i>i</i> in pre-Tx sample A and post-Tx sample B from the same patient, <i>A</i><sub><i>n</i></sub> and <i>B</i><sub><i>n</i></sub> are the total number of clones in samples A and B, and <i>n</i> is the total number of unique clones in the union of A and B samples. The Morisita Index ranges from 0 to 1, with values closer to 1 indicating higher similarity and less repertoire turnover.</div><div id="p0485" role="paragraph">Differences in clonality and Morisita indices between different stages and cohorts were evaluated by two-sided t-test. The Immunarch (v0.6.5, R package) was used to calculate the size distribution of tumor bulk TCR clonotypes by using ‘repClonality’ function (<span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Nazarov, 2020</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Nazarov, V.i.b.R. ∙ E.</div><div><strong>immunarch: an R package for Painless Bioinformatics Analysis of T-cell and B-cell Immune Repertoires</strong></div><div><em>Zenodo.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.5281/zenodo.3367200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.5281%2Fzenodo.3367200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-15"><h4>Identification of differentially expanded clonotypes in blood and tumor samples</h4><div id="p0490" role="paragraph">To identify significantly expanded clonotypes between pre-Tx and post-Tx samples for a given patient, we applied a beta-binomial model, accounting for variance due to random sampling and incorporating time-dependent variance from a highly diverse repertoire (<span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Rytlewski et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Rytlewski, J. ∙ Deng, S. ∙ Xie, T. ...</div><div><strong>Model to improve specificity for identification of clinically-relevant expanded T&nbsp;cells in peripheral blood</strong></div><div><em>PLoS One.</em> 2019; <strong>14</strong>:e0213684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0213684" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85062994573" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30870493/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0213684&amp;pmid=30870493" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Differentially expanded clonotypes after treatment (Tx-E) were further divided into emergent and pre-existing clones based on their fractions in pre-Tx and post-Tx samples. If a Tx-E clonotype could only be detected in the post-Tx sample it was defined as emergent clonotype, but if it was found in both pre- and post-Tx samples it was defined as a pre-existing clonotype. The percentage of Tx-E among total clonotypes between patients from treatment cohorts was compared to evaluate the association between treatment type and clonal expansion.</div><div id="p0495" role="paragraph">Tx-E and Tx-NE clonotypes identified from bulk TCR data were mapped to T&nbsp;cells in scRNA-seq datasets based on sharing of identical <i>TRB</i> sequences. Only clonotype with &gt;1 sequence within a tumor were included in the Tx-NE set for mapping to avoid inclusion of naïve T&nbsp;cells. The fraction of mapped Tx-E cells and non-expanded (Tx-NE) within different CD4 or CD8 T&nbsp;cell sub-clusters was compared to identify which T&nbsp;cell clusters were associated with T&nbsp;cell expansion following ICB. Then, we used ‘FindAllMarkers’ function and Wilcoxon Rank Sum test from Seurat to identify differentially expressed genes between Tx-E and Tx-NE cells in tumor-infiltrating CD4+ and CD8+ T&nbsp;cells (<span class="dropBlock reference-citations"><a id="body-ref-sref34-5" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-5" aria-controls="sref34-5">Hao et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref34-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (4942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To cross-validate individual genes differentially expressed between Tx- E and Tx-NE cells, we applied a pseudobulk strategy aggregating gene expression among single cells in each patient, and then used a two-sided Wilcoxon rank-sum test to evaluate statistical significance.</div></section><section id="sec-4-5-16"><h4>Evaluation of bulk TCR repertoire saturation for identification of emergent clonotypes</h4><div id="p0500" role="paragraph">To evaluate whether the sequencing of TCR repertoire was sufficient to reliably distinguish emergent versus pre-existing expanded clonotypes, we performed down-sampling analysis of the tumor bulk TCR repertoire, under the assumption that some pre-existing expanded clonotypes identified from the whole TCR dataset would switch to emergent clonotypes in down-sampled datasets if the sequencing of bulk TCR dataset is not saturated. Specifically, we subsampled the 10% to 100% (with interval equal to 10%) of TCRs from the whole tumor TCR dataset for each patient and re-identified expanded pre-existing and emergent clonotypes from this down-sampled TCR dataset (<span class="dropBlock reference-citations"><a id="body-ref-sref16-3" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-3" aria-controls="sref16-3">Coffey, 2021</a><div class="dropBlock__holder" data-db-target-of="sref16-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Coffey, D.</div><div><strong>LymphoSeq: analyze high-throughput sequencing of T and B&nbsp;cell receptors. R package version 3.13</strong></div><div>2021</div><div><a id="interref0010" href="https://bioconductor.org/packages/release/bioc/html/LymphoSeq.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/LymphoSeq.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.CoffeyLymphoSeq%3A+analyze+high-throughput+sequencing+of+T+and+B%C2%A0cell+receptors.+R+package+version+3.132021https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FLymphoSeq.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Each down-sampling step was repeated for 10 times to evaluate the variation. Then, we calculated the fraction of pre-existing clonotypes from the original pre-existing expanded clonotypes that switched to emergent clonotypes upon down-sampling. Finally, we generated a saturation curve for each patient, with the x-axis as the down sampled datasets from 10% to 100% of the whole TCR dataset, and y-axis as the fraction of remaining pre-existing clonotypes. On average, only ∼4% of pre-existing expanded clonotypes switched to the emergent state in the 0.80 subsampled dataset.</div></section><section id="sec-4-5-17"><h4>Validation of Tx-E gene signatures in a published dataset</h4><div id="p0505" role="paragraph">Because of the limited number of patients enrolled in this clinical trial, assessing potential statistical correlations between Tx-E associated genes and response metrics was not feasible. Furthermore, it is best practice to probe an independent validation dataset with the discovery gene signature. We utilized a published bulk RNA-seq dataset of pre-treatment biopsies from 298 metastatic urothelial cancers (mUC) patients treated with a PD-L1 mAb (<span class="dropBlock reference-citations"><a id="body-ref-sref58-4" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-4" aria-controls="sref58-4">Mariathasan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref58-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Mariathasan, S. ∙ Turley, S.J. ∙ Nickles, D. ...</div><div><strong>TGFbeta attenuates tumour response to PD-L1 blockade by contributing to exclusion of T&nbsp;cells</strong></div><div><em>Nature.</em> 2018; <strong>554</strong>:544-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85042495633" target="_blank" aria-label="Open Scopus in new tab">Scopus (3242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29443960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25501&amp;pmid=29443960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and obtained the public data from IMvigor210CoreBiologies (v1.0.0, R package).</div><div id="p0510" role="paragraph">To select genes from our CD8Tx-E/NE gene sets to examine in the external dataset, we first filtered Tx-NE and Tx-E lists genes present in CD8 clusters 1-2 and 3, respectively, to ensure genes are well-represented in the main clusters associated with Tx-NE/Tx-E states, and not in cluster 4, which is dominated by a cell cycle signature. We then removed housekeeping genes, including ribosomal genes, mitochondrial genes, and EEF family translation machinery, to enrich for genes specific to T&nbsp;cells. We then performed additional filtering and removal of cell cycle genes as obtained from published signatures (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Tirosh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad0501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84963614956" target="_blank" aria-label="Open Scopus in new tab">Scopus (2751)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27124452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad0501&amp;pmid=27124452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); this was important as the bulk RNA-seq dataset contains cycling tumor cells and associated cell cycle genes that could be anti-correlated with T&nbsp;cell infiltration and proliferation. Finally, we determined whether the resulting gene lists were specific to T and NK cells, rather than represented in other immune lineages or in tumor itself. This step ensured that both Tx-NE and Tx-E lists equally represented T&nbsp;cell-specific genes, correcting for general T&nbsp;cell infiltrate that is known to correlate with response and outcome in bulk RNA-seq datasets. To evaluate T/NK specificity, we utilized lung SSC scRNA-seq datasets containing immune, tumor, and stromal populations (10X Genomics). We evaluated the average expression of Tx-NE/Tx-E genes among different lineages, and selected genes specifically expressed in T/NK cells. The final gene lists are as follows:<div class="figure-wrap"><figure id="undtbl8" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>Signature</th><th>Genes</th></tr></thead><tbody><tr><td>Tx-E, pre-Tx</td><td><i>KLRD1, CXCL13, GNLY, KLRC2, CD7, ITGAE, GZMB, CTLA4, ZNF683, TNFRSF18, CD226, SIRPG, KLRB1, PRF1, TIGIT, LAG3</i></td></tr><tr><td>Tx-NE, pre-Tx</td><td><i>GZMK, CST7, EOMES, GZMM, SAMD3, LIME1, CXCR3, SH2D1A, TRAT1, CD5</i></td></tr><tr><td>TX-E, post-Tx</td><td><i>GZMB, CTLA4, CXCL13, CD7, ITGAE, GZMA, GNLY, SIRPG, KLRD1, IFNG, ZNF683</i></td></tr><tr><td>Tx-NE, post-Tx</td><td><i>GZMK, IL7R, CST7, LEPROTL1, SAMD3, PDCD4</i></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl8&amp;pii=S0092-8674%2822%2900723-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0515" role="paragraph">We then calculated the activity score (as described above) based on the expression profiles of these genes in this mUC dataset. We then applied a two-sided t-test to compare the statistical difference between response and non-response groups based on the activity score of these gene signatures. Finally, we divided patients into two groups based on the activity score of each gene signature (top and bottom 50%) and used the likelihood ratio test to statistically compare survival probabilities.</div></section><section id="sec-4-5-18"><h4>Sharing of TCR sequences across T&nbsp;cell clusters</h4><div id="p0520" role="paragraph">We applied a similar strategy as previously developed (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Luoma et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Luoma, A.M. ∙ Suo, S. ∙ Williams, H.L. ...</div><div><strong>Molecular pathways of colon inflammation induced by cancer immunotherapy</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:655-671.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2020.06.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2020.06.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85087986491" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32603654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.001&amp;pmid=32603654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to statistically evaluate the relationship between tumor T&nbsp;cell clusters based on sharing of TCR clonotypes across clusters. Among CD4+ cells, we identified 145 and 442 shared clonotypes (detected in at least one cell in two or more different clusters) in pre-Tx and post-Tx patient groups, respectively. In CD8+ T&nbsp;cells, 363 and 985 shared clonotypes were detected in pre-Tx and post-Tx patient groups, respectively. We then counted the number of shared clonotypes among different clusters and applied Fisher’s exact test on 2<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="times" tabindex="0" ctxtmenu_counter="11" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="0" data-semantic-speech="times"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>2 cluster-by-cluster contingency tables to evaluate the significance of TCR clonotype sharing. Clonotypes shared at a frequency higher than occurring in a randomized matrix were identified as statistically significant, based on a one-sided Fisher’s exact test followed by Benjamini-Hochberg correction.</div><div id="p0525" role="paragraph">To statistically evaluate the sharing of TCR clonotypes across different clusters between pre-Tx and post-Tx datasets, we also built a 2<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="times" tabindex="0" ctxtmenu_counter="12" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="0" data-semantic-speech="times"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>2 contingency table for each cluster pair across time in pre- and post-Tx samples. Briefly, we counted the number of clonotypes&nbsp;from each pre-Tx cluster that are shared with cells from the same or other post-Tx clusters, and then applied one-sided Fisher’s exact test followed by Benjamini-Hochberg correction to evaluate significance. Only cluster pairs with FDR&lt;0.05 were highlighted.</div></section><section id="sec-4-5-19"><h4>Projection of sorted, activated blood T&nbsp;cells to total blood T&nbsp;cell scRNA-seq data</h4><div id="p0530" role="paragraph">scRNA-seq data from sorted activated blood T&nbsp;cells underwent pre-processing, demultiplexing, quality control, and clustering as performed based on the above-described steps. We subsequently used similar strategies as in ProjecTILs (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Andreatta et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to project sorted, activated blood CD4 and CD8 T&nbsp;cells to total blood CD4 and CD8 T&nbsp;cells, respectively. In brief, we first recomputed PCA and UMAP embeddings by using ‘prcomp’ function from basic R package of stats and ‘umap’ function from umap (v0.2.7.0, R package), respectively. Then, pre-calculated PCA coefficients from the reference blood dataset were applied to the normalized sorted blood scRNA-seq data. Finally, the ‘predict’ function from the umap package was used to transform PCA embeddings into UMAP coordinates. After processing, the sorted blood data were embedded into the original UMAP space of the reference blood dataset, enabling joint visualization as well as classification of the sorted cells into reference cells. To confidentially calculate the percentage of projected T&nbsp;cells in each reference cluster, we first applied the ‘cellstate.predict’ function in ProjecTILs (v1.0.0, R package) to assign a confidence score for each cell using only selected cells with score&gt;0.8 for evaluation. Lineage connections between sorted T&nbsp;cells, reference blood T&nbsp;cells, and bulk TCRs were established based on identical TRB sequences for different datasets.</div></section><section id="sec-4-5-20"><h4>Calculation of tumor mutation burden</h4><div id="p0535" role="paragraph">The quality of the sequencing data was further assessed using several methods included in the Getz Lab CGA WES Characterization Pipeline (<a id="intref0355" href="https://github.com/broadinstitute/CGA_Production_Analysis_Pipeline" target="_blank">https://github.com/broadinstitute/CGA_Production_Analysis_Pipeline</a>) developed at the Broad Institute. Samples were determined to be poor quality if the mean target coverage was &lt; 25X (tumor) or&nbsp;&lt;&nbsp;15X (matched-normal) determined by PicardMultipleMetrics (<a id="intref0360" href="https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.0.0/picard_analysis_CollectMultipleMetrics.php" target="_blank">https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.0.0/picard_analysis_CollectMultipleMetrics.php</a>), the cross-sample contamination of samples estimation ≥ <span data-style="underline">5</span>% determined by ContEst (<span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Cibulskis et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Cibulskis, K. ∙ McKenna, A. ∙ Fennell, T. ...</div><div><strong>ContEst: estimating cross-contamination of human samples in next-generation sequencing data</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2601-2602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr446" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-80052707308" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21803805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr446&amp;pmid=21803805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), tumor&nbsp;purity ≥ 20% determined by FACETS (<span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Shen and Seshan, 2016</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Shen, R. ∙ Seshan, V.E.</div><div><strong>FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84988924813" target="_blank" aria-label="Open Scopus in new tab">Scopus (712)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27270079/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw520&amp;pmid=27270079" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and ABSOLUTE (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Carter et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Carter, S.L. ∙ Cibulskis, K. ∙ Helman, E. ...</div><div><strong>Absolute quantification of somatic DNA alterations in human cancer</strong></div><div><em>Nat. Biotechnol.</em> 2012; <strong>30</strong>:413-421</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2203" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84860782006" target="_blank" aria-label="Open Scopus in new tab">Scopus (1429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22544022/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2203&amp;pmid=22544022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and tumor-in-normal contamination ≥ 20% determined by deTiN (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Taylor-Weiner et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Taylor-Weiner, A. ∙ Stewart, C. ∙ Giordano, T. ...</div><div><strong>DeTiN: overcoming tumor-in-normal contamination</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:531-534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0036-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85048968644" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29941871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-018-0036-9&amp;pmid=29941871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All pre-treatment tumor and normal samples were found to have sufficient quality.</div><div id="p0540" role="paragraph">Somatic variants were called, filtered, and annotated using a modified version of the Getz Lab CGA WES Characterization Pipeline. Somatic single-nucleotide variants were identified by MuTect1 (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Cibulskis et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Cibulskis, K. ∙ McKenna, A. ∙ Fennell, T. ...</div><div><strong>ContEst: estimating cross-contamination of human samples in next-generation sequencing data</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2601-2602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr446" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-80052707308" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21803805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr446&amp;pmid=21803805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and insertions and deletions (indels) were identified by both Strelka (<span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Saunders et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Saunders, C.T. ∙ Wong, W.S. ∙ Swamy, S. ...</div><div><strong>Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:1811-1817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84864153492" target="_blank" aria-label="Open Scopus in new tab">Scopus (1149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22581179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts271&amp;pmid=22581179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and MuTecT2 (<span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">McKenna et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>McKenna, A. ∙ Hanna, M. ∙ Banks, E. ...</div><div><strong>The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.107524.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-77956295988" target="_blank" aria-label="Open Scopus in new tab">Scopus (18075)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20644199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.107524.110&amp;pmid=20644199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To filter false positive germline and common artifacts from somatic variant calls, variants with less than 2% allele frequency were excluded and a Panel of Normal (PoN) filter was employed (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Lawrence et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Lawrence, M.S. ∙ Stojanov, P. ∙ Mermel, C.H. ...</div><div><strong>Discovery and saturation analysis of cancer genes across 21 tumour types</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84892833777" target="_blank" aria-label="Open Scopus in new tab">Scopus (2347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24390350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12912&amp;pmid=24390350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Filters were applied to exclude artifactual variants arising from oxidation of guanine to 8-Oxoguanine during sequencing, formaldehyde deamination of cytosines during DNA fixation, and DNA sequence misalignment to the reference genome (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Costello et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Costello, M. ∙ Pugh, T.J. ∙ Fennell, T.J. ...</div><div><strong>Discovery and characterization of artifactual mutations in deep coverage targeted capture sequencing data due to oxidative DNA damage during sample preparation</strong></div><div><em>Nucleic Acids Res.</em> 2013; <strong>41</strong>:e67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gks1443" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84876020288" target="_blank" aria-label="Open Scopus in new tab">Scopus (332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23303777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgks1443&amp;pmid=23303777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). FFPE artifacts were filtered with an Artifact-Q cut-off value of 35 and poorly mapped reads were filtered by BLAST-like Alignment Tool (BLAT).</div><div id="p0545" role="paragraph">Tumor mutational burden was calculated as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">Miao et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Miao, D. ∙ Margolis, C.A. ∙ Vokes, N.I. ...</div><div><strong>Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1271-1281</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0200-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85052712350" target="_blank" aria-label="Open Scopus in new tab">Scopus (386)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30150660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0200-2&amp;pmid=30150660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, the total number of mutations was divided by the number of megabases covered at a sufficient depth of coverage to detect variants with 80% power given an estimated tumor purity. Tumor purity estimates and cancer cell fractions (CCFs) were inferred from FACETS (<span class="dropBlock reference-citations"><a id="body-ref-sref75-3" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-3" aria-controls="sref75-3">Shen and Seshan, 2016</a><div class="dropBlock__holder" data-db-target-of="sref75-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Shen, R. ∙ Seshan, V.E.</div><div><strong>FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84988924813" target="_blank" aria-label="Open Scopus in new tab">Scopus (712)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27270079/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw520&amp;pmid=27270079" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Clonal mutations were defined as mutations with a CCF equal to or greater than 0.8. Comparisons between clinical outcomes and mutational burden were performed using the Mann-Whitney test in R version 3.6.1.</div></section><section id="sec-4-5-21"><h4>Gene Expression Analysis of T-scan targets</h4><div id="p0550" role="paragraph">Data from TCGA and Gtex were downloaded from the UCSC Xena browser (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Goldman et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Goldman, M.J. ∙ Craft, B. ∙ Hastie, M. ...</div><div><strong>Visualizing and interpreting cancer genomics data via the Xena platform</strong></div><div><em>Nat. Biotechnol.</em> 2020; <strong>38</strong>:675-678</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-020-0546-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85085288347" target="_blank" aria-label="Open Scopus in new tab">Scopus (2054)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32444850/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-020-0546-8&amp;pmid=32444850" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To represent the expression profile of all the genes of interest in one plot, the TPM value for each gene was normalized to the maximum and minimum for each dataset where 1 represented the maximum expression for each gene dataset and 0 represented the minimum for each gene dataset. The mean of the top quartile for each tissue type was calculated to represent the highest expressing samples as the most therapeutically relevant for tumor tissue, or the tissue that pose the highest off tumor risk for normal tissues.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0555" role="paragraph">Funding: this work was supported by the Claudia Adams Barr Program in Cancer Research (DFCI), NIH grants R01 CA238039 (to K.W.W.), T32CA207021 (to A.M.L. and K.W.W.), and U24CA224316 (to C.A.R. and E.M.V.A.). K.W.W. is a member of the Parker Institute for Cancer Immunotherapy (PICI). The clinical trial from which specimens were analyzed had been supported by Bristol-Myers Squibb.</div><div id="p0560" role="paragraph">We would like to thank all patients and providers who participated in this study. We thank Nathan Mathewson, Francesca Alvarez-Calderon, Rong En Tay, and Andrew Wight for help with cell sorting; clinical research coordinators Sarah Faso, Brendan Ross, and Giulia Digiovanni for help with clinical trial execution and sample collection; Mariano Severgnini and Alyssa Masciarelli for collecting PBMC samples; Michal Slyper and Katie Geiger-Schuller for advice on cell hashing; Alexandria Kluge and Erin Shannon for help with exome sequencing; Livnat Jerby-Arnon for input on computational analysis; Asaf Rotem, Sidharth Puram, and Benjamin Izar for advice on tumor dissociation protocols.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0565" role="paragraph">A.M.L., J.D.S., S.S., and K.W.W. designed the study. A.M.L. processed tissue biopsies, generated scRNA-seq gene expression libraries, and performed flow cytometry studies. S.S. and C.B.M.P. performed data processing, and S.S. performed published analyses. C.B.M.P., O.A., and A.R. provided guidance on computational analysis. D.D. performed QC analysis of scRNA-seq libraries and arranged sequencing. Y.W., J.T., N.N., A.P.F., S.L., C.G., and G.M. determined the antigen specificity of TCRs. C.A.R. and E.M.V.A. performed mutational burden analysis. J.D.S. led the clinical trial and oversaw sample allocation. L.G., Y.-H.C., S.C., R.U., and R.I.H. contributed to clinical trial oversight and execution. R.U. additionally helped obtain surgical specimens. O.R.-R. helped to coordinate aspects of this study. A.R. provided advice on study design, A.M.L., S.S., and K.W.W. wrote the paper with input from all authors.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0570" role="paragraph">K.W.W. serves on the SAB of SQZ Biotech, Nextechinvest, Bisou Bioscience Company, and T-Scan Therapeutics and receives sponsored research funding from Novartis. He is a scientific co-founder of Immunitas Therapeutics. J.D.S. reports research support paid to the institution from Merck, BMS, Regeneron, Debiopharm; Consulting/Scientific Advisory Board/Travel fees: Genentech, Immunitas, Debiopharm, BMS, Nanobiotix, Tilos, Castle Biosciences, Astra Zeneca, LEK, Catenion, ACI Clinical, Astellas, Stimit, and Merck KGA; Expert witness fees. Stock options: Immunitas; Equity: Doximity. E.M.V.A. reports Advisory/Consulting: Tango Therapeutics, Genome Medical, Invitae, Enara Bio, Janssen, Manifold Bio, and Monte Rosa; Research support: Novartis, BMS; Equity: Tango Therapeutics, Genome Medical, Syapse, Enara Bio, Manifold Bio, Microsoft, and Monte Rosa; Travel reimbursement: Roche/Genentech; Patents: Institutional patents filed on chromatin mutations and immunotherapy response, and methods for clinical interpretation; intermittent legal consulting on patents for Foaley &amp; Hoag. A.R. is a founder and equity holder of Celsius Therapeutics, an equity holder in Immunitas Therapeutics, and until August 31, 2020, was an SAB member of Syros Pharmaceuticals, Neogene Therapeutics, Asimov, and Thermo Fisher Scientific. From August 1, 2020, A.R. is an employee of Genentech and has equity in Roche.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (7)</h2><div id="p0580" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.018/attachment/d0c71e9c-69b7-4ded-9c1f-88054b583c88/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (15.59 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table&nbsp;S1. Detailed summary of patient response metrics, related to Figure&nbsp;1</div><div id="espara0010" role="paragraph">Three major clinical readouts were used: clinical to pathological downstaging at time of surgery (compared to pretreatment evaluation), volumetric response based on tumor measurements along two axes as well as pathological response evaluated by histological analysis (indicated as % viable tumor post-treatment, also summarized as a tumor regression score and near complete/complete response). Radiologic assessment was attempted (RECIST response) but imaging artifacts (bone, teeth) precluded assessment in the majority of patients.</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.018/attachment/aeab8993-eae4-47e1-96d2-6559807880d0/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (1012.72 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table&nbsp;S2. Marker genes for tumor-infiltrating immune cell subsets, related to Figures&nbsp;2, S2, and S5</div><div id="espara0015" role="paragraph">Differentially expressed genes are shown for tumor-infiltrating immune cell populations (combined samples) for the indicated datasets: CD8+ T&nbsp;cell clusters, CD4+ T&nbsp;cell clusters and myeloid cell clusters.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.018/attachment/37095adf-7956-4c3a-8cfc-763edf3abc45/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (106.94 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table&nbsp;S3. Differentially expressed genes in Tx-E T&nbsp;cells in tumors, related to Figures&nbsp;4 and S4</div><div id="espara0020" role="paragraph">Genes differentially expressed between tumor Tx-E and Tx-NE T&nbsp;cells are shown for the indicated datasets: CD8+ T&nbsp;cell subset (Figure&nbsp;4A, pre-Tx and post-Tx), and CD4+ T&nbsp;cell subset (Figure&nbsp;4E). The top 20 DE genes, and additional key genes, identified by single-cell methods were additionally validated by pseudo-bulk method as indicated.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.018/attachment/3d8d5311-eef0-4558-b0d7-1ce66f907b60/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (10.52 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table&nbsp;S4. Gene signatures used for analyzing Tx-E T&nbsp;cells and independent ICB treatment cohort, related to Figures&nbsp;4 and S4</div><div id="espara0025" role="paragraph">Gene signatures used to calculate activity scores in Tx-E/NE T&nbsp;cells (Figures&nbsp;4C and 4D) and for assessing response and survival in an independent cohort of patients with urothelial cancer treated with a PD-L1 mAb (Figures&nbsp;4F and S4F).</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.018/attachment/4f327d9c-e1c5-44e8-a8b9-102ec6f9cb39/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (11.33 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table&nbsp;S5. T-scan screen summary, related to Figures&nbsp;5 and S5</div><div id="espara0030" role="paragraph">Total numbers of TCRs screened and HLA alleles screened for each patient.</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.018/attachment/16d0df13-1962-443d-a609-b39ba32212bb/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (868.11 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table&nbsp;S6. Marker genes for blood immune cell subsets, related to Figures&nbsp;6 and S6</div><div id="espara0035" role="paragraph">Differentially expressed genes are shown for blood cell immune cell clusters (combined samples) for the indicated datasets: CD8+ T&nbsp;cell clusters, CD4+ T&nbsp;cell clusters, myeloid cell clusters, and sorted CD38+HLA-DR+ blood CD8+ T&nbsp;cells.</div></div></div><div id="mmc7" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.018/attachment/b54231fe-ae45-4895-888a-45958999921c/mmc7.xlsx" download="mmc7.xlsx" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Spreadsheet (11.93 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Table&nbsp;S7. Summary of blood cytometry findings, related to Figures&nbsp;7 and S7</div><div id="espara0040" role="paragraph">Median frequencies and p values of blood T&nbsp;cell populations that were significantly different between timepoints or response groups.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">Processed, de-identified scRNA-seq gene expression, TCR, and bulk TCR datasets generated during this study are publicly available at the Gene Expression Omnibus (GEO) as of the date of publication. The accession number for these datasets is listed in the <a id="crosref1725" href="#sec-4-1">key resources table</a>. The raw scRNA-seq gene expression sequencing data cannot be deposited in a public repository because of restrictions in informed consent. Raw data may be requested from <a id="intref0010" href="mailto:kai_wucherpfennig@dfci.harvard.edu">kai_wucherpfennig@dfci.harvard.edu</a> with appropriate institutional approvals. This paper analyzes existing, publicly available data. The accession number for this dataset is listed in the <a id="crosref1730" href="#sec-4-1">key resources table</a>.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">This paper does not report original code.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the <a id="crosref1735" href="#sec-4-2-1">lead contact</a> upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Aibar, S. ∙ González-Blas, C.B. ∙ Moerman, T. ...</div><div><strong>SCENIC: single-cell regulatory network inference and clustering</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:1083-1086</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4463" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85032583384" target="_blank" aria-label="Open Scopus in new tab">Scopus (2530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4463&amp;pmid=28991892" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Amaria, R.N. ∙ Reddy, S.M. ∙ Tawbi, H.A. ...</div><div><strong>Neoadjuvant immune checkpoint blockade in high-risk resectable melanoma</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:1649-1654</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0197-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85054583492" target="_blank" aria-label="Open Scopus in new tab">Scopus (589)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30297909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0197-1&amp;pmid=30297909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Aran, D. ∙ Looney, A.P. ∙ Liu, L. ...</div><div><strong>Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:163-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-018-0276-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85060111238" target="_blank" aria-label="Open Scopus in new tab">Scopus (1970)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30643263/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-018-0276-y&amp;pmid=30643263" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Banchereau, R. ∙ Chitre, A.S. ∙ Scherl, A. ...</div><div><strong>Intratumoral CD103+ CD8+ T&nbsp;cells predict response to PD-L1 blockade</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2021; <strong>9</strong>:e002231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jitc-2020-002231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85103885386" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33827905/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjitc-2020-002231&amp;pmid=33827905" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Bassez, A. ∙ Vos, H. ∙ Van Dyck, L. ...</div><div><strong>A single-cell map of intratumoral changes during anti-PD1 treatment of patients with breast cancer</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:820-832</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-021-01323-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85105408508" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33958794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-021-01323-8&amp;pmid=33958794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Baumeister, S.H. ∙ Freeman, G.J. ∙ Dranoff, G. ...</div><div><strong>Coinhibitory pathways in immunotherapy for cancer</strong></div><div><em>Annu. Rev. Immunol.</em> 2016; <strong>34</strong>:539-573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-032414-112049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84969850678" target="_blank" aria-label="Open Scopus in new tab">Scopus (700)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26927206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-032414-112049&amp;pmid=26927206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (2577)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Blank, C.U. ∙ Haining, W.N. ∙ Held, W. ...</div><div><strong>Defining 'T cell exhaustion'</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:665-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0221-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85074300816" target="_blank" aria-label="Open Scopus in new tab">Scopus (855)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0221-9&amp;pmid=31570879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Blank, C.U. ∙ Rozeman, E.A. ∙ Fanchi, L.F. ...</div><div><strong>Neoadjuvant versus adjuvant ipilimumab plus nivolumab in macroscopic stage III melanoma</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:1655-1661</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0198-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85054564616" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30297911/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0198-0&amp;pmid=30297911" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Carter, S.L. ∙ Cibulskis, K. ∙ Helman, E. ...</div><div><strong>Absolute quantification of somatic DNA alterations in human cancer</strong></div><div><em>Nat. Biotechnol.</em> 2012; <strong>30</strong>:413-421</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2203" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84860782006" target="_blank" aria-label="Open Scopus in new tab">Scopus (1429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22544022/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2203&amp;pmid=22544022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Caushi, J.X. ∙ Zhang, J. ∙ Ji, Z. ...</div><div><strong>Transcriptional programs of neoantigen-specific TIL in anti-PD-1-treated lung cancers</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:126-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03752-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85110991277" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34290408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03752-4&amp;pmid=34290408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13-1" title="View in article">13.</a></div><div>Chalabi, M. ∙ Fanchi, L.F. ∙ Dijkstra, K.K. ...</div><div><strong>Neoadjuvant immunotherapy leads to pathological responses in MMR-proficient and MMR-deficient early-stage colon cancers</strong></div><div><em>Nat. Med.</em> 2020; <strong>26</strong>:566-576</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-020-0805-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85083283363" target="_blank" aria-label="Open Scopus in new tab">Scopus (812)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32251400/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-020-0805-8&amp;pmid=32251400" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Cibulskis, K. ∙ McKenna, A. ∙ Fennell, T. ...</div><div><strong>ContEst: estimating cross-contamination of human samples in next-generation sequencing data</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2601-2602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr446" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-80052707308" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21803805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr446&amp;pmid=21803805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Clarke, J. ∙ Panwar, B. ∙ Madrigal, A. ...</div><div><strong>Single-cell transcriptomic analysis of tissue-resident memory T&nbsp;cells in human lung cancer</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2019; <strong>216</strong>:2128-2149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20190249" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85071787869" target="_blank" aria-label="Open Scopus in new tab">Scopus (152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31227543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20190249&amp;pmid=31227543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Coffey, D.</div><div><strong>LymphoSeq: analyze high-throughput sequencing of T and B&nbsp;cell receptors. R package version 3.13</strong></div><div>2021</div><div><a id="interref0010" href="https://bioconductor.org/packages/release/bioc/html/LymphoSeq.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/LymphoSeq.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.CoffeyLymphoSeq%3A+analyze+high-throughput+sequencing+of+T+and+B%C2%A0cell+receptors.+R+package+version+3.132021https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FLymphoSeq.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Costello, M. ∙ Pugh, T.J. ∙ Fennell, T.J. ...</div><div><strong>Discovery and characterization of artifactual mutations in deep coverage targeted capture sequencing data due to oxidative DNA damage during sample preparation</strong></div><div><em>Nucleic Acids Res.</em> 2013; <strong>41</strong>:e67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gks1443" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84876020288" target="_blank" aria-label="Open Scopus in new tab">Scopus (332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23303777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgks1443&amp;pmid=23303777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Duhen, T. ∙ Duhen, R. ∙ Montler, R. ...</div><div><strong>Co-expression of CD39 and CD103 identifies tumor-reactive CD8 T&nbsp;cells in human solid tumors</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:2724</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-05072-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85049972292" target="_blank" aria-label="Open Scopus in new tab">Scopus (551)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30006565/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-05072-0&amp;pmid=30006565" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Engblom, C. ∙ Pfirschke, C. ∙ Pittet, M.J.</div><div><strong>The role of myeloid cells in cancer therapies</strong></div><div><em>Nat. Rev. Cancer.</em> 2016; <strong>16</strong>:447-462</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrc.2016.54" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84976272206" target="_blank" aria-label="Open Scopus in new tab">Scopus (539)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27339708/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrc.2016.54&amp;pmid=27339708" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Fairfax, B.P. ∙ Taylor, C.A. ∙ Watson, R.A. ...</div><div><strong>Peripheral CD8(+) T&nbsp;cell characteristics associated with durable responses to immune checkpoint blockade in patients with metastatic melanoma</strong></div><div><em>Nat. Med.</em> 2020; <strong>26</strong>:193-199</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0734-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85079320198" target="_blank" aria-label="Open Scopus in new tab">Scopus (196)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32042196/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0734-6&amp;pmid=32042196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Ferrarotto, R. ∙ Bell, D. ∙ Rubin, M.L. ...</div><div><strong>Impact of neoadjuvant durvalumab with or without tremelimumab on CD8(+) tumor lymphocyte density, safety, and efficacy in patients with oropharynx cancer: CIAO trial results</strong></div><div><em>Clin. Cancer Res.</em> 2020; <strong>26</strong>:3211-3219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-19-3977" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85087469863" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32269052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-19-3977&amp;pmid=32269052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Ferretti, A.P. ∙ Kula, T. ∙ Wang, Y. ...</div><div><strong>Unbiased screens show CD8(+) T&nbsp;cells of COVID-19 patients recognize shared epitopes in SARS-CoV-2 that largely reside outside the spike protein</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:1095-1107.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.immuni.2020.10.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.immuni.2020.10.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85094974474" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33128877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.10.006&amp;pmid=33128877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Ferris, R.L. ∙ Spanos, W.C. ∙ Leidner, R. ...</div><div><strong>Neoadjuvant nivolumab for patients with resectable HPV-positive and HPV-negative squamous cell carcinomas of the head and neck in the CheckMate 358 trial</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2021; <strong>9</strong>:e002568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jitc-2021-002568" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85107491601" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34083421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjitc-2021-002568&amp;pmid=34083421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Figueiredo, D.L. ∙ Mamede, R.C. ∙ Proto-Siqueira, R. ...</div><div><strong>Expression of cancer testis antigens in head and neck squamous cell carcinomas</strong></div><div><em>Head Neck.</em> 2006; <strong>28</strong>:614-619</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/hed.20380" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-33745798609" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16475205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fhed.20380&amp;pmid=16475205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">25.</a></div><div>Forde, P.M. ∙ Chaft, J.E. ∙ Smith, K.N. ...</div><div><strong>Neoadjuvant PD-1 blockade in resectable lung cancer</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1976-1986</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1716078" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85047781450" target="_blank" aria-label="Open Scopus in new tab">Scopus (1396)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29658848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1716078&amp;pmid=29658848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Forde, P.M. ∙ Spicer, J. ∙ Lu, S. ...</div><div><strong>Neoadjuvant nivolumab plus chemotherapy in resectable lung cancer</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2022; <strong>386</strong>:1973-1985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2202170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85130221027" target="_blank" aria-label="Open Scopus in new tab">Scopus (1089)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35403841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2202170&amp;pmid=35403841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Fransen, M.F. ∙ Schoonderwoerd, M. ∙ Knopf, P. ...</div><div><strong>Tumor-draining lymph nodes are pivotal in PD-1/PD-L1 checkpoint therapy</strong></div><div><em>JCI Insight.</em> 2018; <strong>3</strong>:e124507</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/jci.insight.124507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85063246976" target="_blank" aria-label="Open Scopus in new tab">Scopus (231)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30518694/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2Fjci.insight.124507&amp;pmid=30518694" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>Friedman, J. ∙ Moore, E.C. ∙ Zolkind, P. ...</div><div><strong>Neoadjuvant PD-1 immune checkpoint blockade reverses functional immunodominance among tumor antigen-specific T&nbsp;cells</strong></div><div><em>Clin. Cancer Res.</em> 2020; <strong>26</strong>:679-689</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-19-2209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85079020271" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31645352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-19-2209&amp;pmid=31645352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">29.</a></div><div>Galon, J. ∙ Costes, A. ∙ Sanchez-Cabo, F. ...</div><div><strong>Type, density, and location of immune cells within human colorectal tumors predict clinical outcome</strong></div><div><em>Science.</em> 2006; <strong>313</strong>:1960-1964</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1129139" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-33749319703" target="_blank" aria-label="Open Scopus in new tab">Scopus (5115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17008531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1129139&amp;pmid=17008531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Ganesan, A.P. ∙ Clarke, J. ∙ Wood, O. ...</div><div><strong>Tissue-resident memory features are linked to the magnitude of cytotoxic T&nbsp;cell responses in human lung cancer</strong></div><div><em>Nat. Immunol.</em> 2017; <strong>18</strong>:940-950</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3775" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85025815252" target="_blank" aria-label="Open Scopus in new tab">Scopus (372)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28628092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3775&amp;pmid=28628092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Goldman, M.J. ∙ Craft, B. ∙ Hastie, M. ...</div><div><strong>Visualizing and interpreting cancer genomics data via the Xena platform</strong></div><div><em>Nat. Biotechnol.</em> 2020; <strong>38</strong>:675-678</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-020-0546-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85085288347" target="_blank" aria-label="Open Scopus in new tab">Scopus (2054)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32444850/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-020-0546-8&amp;pmid=32444850" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>Gueguen, P. ∙ Metoikidou, C. ∙ Dupic, T. ...</div><div><strong>Contribution of resident and circulating precursors to tumor-infiltrating CD8(+) T&nbsp;cell populations in lung cancer</strong></div><div><em>Sci. Immunol.</em> 2021; <strong>6</strong>:eabd5778</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.abd5778" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33514641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.abd5778&amp;pmid=33514641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Guo, B. ∙ Yuan, Y.</div><div><strong>A comparative review of methods for comparing means using partially paired data</strong></div><div><em>Stat. Methods Med. Res.</em> 2017; <strong>26</strong>:1323-1340</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0962280215577111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85020729450" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25834090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0962280215577111&amp;pmid=25834090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (4942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">35.</a></div><div>Hornburg, M. ∙ Desbois, M. ∙ Lu, S. ...</div><div><strong>Single-cell dissection of cellular components and interactions shaping the tumor immune phenotypes in ovarian cancer</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:928-944.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85107327427" target="_blank" aria-label="Open Scopus in new tab">Scopus (158)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33961783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2021.04.004&amp;pmid=33961783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Huang, A.C. ∙ Orlowski, R.J. ∙ Xu, X. ...</div><div><strong>A single dose of neoadjuvant PD-1 blockade predicts clinical outcomes in resectable melanoma</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:454-461</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0357-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85062071459" target="_blank" aria-label="Open Scopus in new tab">Scopus (460)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0357-y&amp;pmid=30804515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Jiang, P. ∙ Gu, S. ∙ Pan, D. ...</div><div><strong>Signatures of T&nbsp;cell dysfunction and exclusion predict cancer immunotherapy response</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:1550-1558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0136-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85052583273" target="_blank" aria-label="Open Scopus in new tab">Scopus (2802)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30127393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0136-1&amp;pmid=30127393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">38.</a></div><div>Jin, S. ∙ Guerrero-Juarez, C.F. ∙ Zhang, L. ...</div><div><strong>Inference and analysis of cell-cell communication using CellChat</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:1088</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-21246-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85101173884" target="_blank" aria-label="Open Scopus in new tab">Scopus (2221)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-21246-9&amp;pmid=33597522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39-1" title="View in article">39.</a></div><div>Kamphorst, A.O. ∙ Pillai, R.N. ∙ Yang, S. ...</div><div><strong>Proliferation of PD-1+ CD8 T&nbsp;cells in peripheral blood after PD-1-targeted therapy in lung cancer patients</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:4993-4998</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1705327114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85019093717" target="_blank" aria-label="Open Scopus in new tab">Scopus (588)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28446615/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1705327114&amp;pmid=28446615" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">40.</a></div><div>Khunger, A. ∙ Rytlewski, J.A. ∙ Fields, P. ...</div><div><strong>The impact of CTLA-4 blockade and interferon-alpha on clonality of T-cell repertoire in the tumor microenvironment and peripheral blood of metastatic melanoma patients</strong></div><div><em>Oncoimmunology.</em> 2019; <strong>8</strong>:e1652538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/2162402X.2019.1652538" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85071041224" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31646098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F2162402X.2019.1652538&amp;pmid=31646098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Korotkevich, G. ∙ Sukhov, V. ∙ Budin, N. ...</div><div><strong>Fast gene set enrichment analysis</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/060012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F060012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42-1" title="View in article">42.</a></div><div>Krishna, C. ∙ DiNatale, R.G. ∙ Kuo, F. ...</div><div><strong>Single-cell sequencing links multiregional immune landscapes and tissue-resident T&nbsp;cells in ccRCC to tumor topology and therapy efficacy</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:662-677.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.03.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.ccell.2021.03.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85104966318" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33861994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2021.03.007&amp;pmid=33861994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">43.</a></div><div>Kula, T. ∙ Dezfulian, M.H. ∙ Wang, C.I. ...</div><div><strong>T-scan: A genome-wide method for the systematic discovery of T&nbsp;cell epitopes</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1016-1028.e13</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2019.07.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85070183509" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31398327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.009&amp;pmid=31398327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">44.</a></div><div>Kumar, B.V. ∙ Ma, W. ∙ Miron, M. ...</div><div><strong>Human tissue-resident memory T&nbsp;cells are defined by core transcriptional and functional signatures in lymphoid and mucosal sites</strong></div><div><em>Cell Rep.</em> 2017; <strong>20</strong>:2921-2934</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.celrep.2017.08.078&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.celrep.2017.08.078&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85029578875" target="_blank" aria-label="Open Scopus in new tab">Scopus (674)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28930685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2017.08.078&amp;pmid=28930685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Kvistborg, P. ∙ Philips, D. ∙ Kelderman, S. ...</div><div><strong>Anti-CTLA-4 therapy broadens the melanoma-reactive CD8+ T&nbsp;cell response</strong></div><div><em>Sci. Transl. Med.</em> 2014; <strong>6</strong>:254ra128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/scitranslmed.3008918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84907284200" target="_blank" aria-label="Open Scopus in new tab">Scopus (313)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25232180/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscitranslmed.3008918&amp;pmid=25232180" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Lawrence, M.S. ∙ Stojanov, P. ∙ Mermel, C.H. ...</div><div><strong>Discovery and saturation analysis of cancer genes across 21 tumour types</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84892833777" target="_blank" aria-label="Open Scopus in new tab">Scopus (2347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24390350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12912&amp;pmid=24390350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Le Floc'h, A. ∙ Jalil, A. ∙ Vergnon, I. ...</div><div><strong>Alpha E beta 7 integrin interaction with E-cadherin promotes antitumor CTL activity by triggering lytic granule polarization and exocytosis</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:559-570</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20061524" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-33947415318" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17325197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20061524&amp;pmid=17325197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Lee, A.Y. ∙ Brady, M.S.</div><div><strong>Neoadjuvant immunotherapy for melanoma</strong></div><div><em>J.&nbsp;Surg. Oncol.</em> 2021; <strong>123</strong>:782-788</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jso.26229" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85091801696" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33002195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjso.26229&amp;pmid=33002195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">49.</a></div><div>Li, B. ∙ Gould, J. ∙ Yang, Y. ...</div><div><strong>Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:793-798</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-020-0905-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85088650202" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32719530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-020-0905-x&amp;pmid=32719530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Liberzon, A. ∙ Birger, C. ∙ Thorvaldsdóttir, H. ...</div><div><strong>The Molecular Signatures Database (MSigDB) hallmark gene set collection</strong></div><div><em>Cell Syst.</em> 2015; <strong>1</strong>:417-425</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84955328286" target="_blank" aria-label="Open Scopus in new tab">Scopus (6393)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.12.004&amp;pmid=26771021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">51.</a></div><div>Licitra, L. ∙ Grandi, C. ∙ Guzzo, M. ...</div><div><strong>Primary chemotherapy in resectable oral cavity squamous cell cancer: a randomized controlled trial</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2003; <strong>21</strong>:327-333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2003.06.146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-0037440210" target="_blank" aria-label="Open Scopus in new tab">Scopus (236)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12525526/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2003.06.146&amp;pmid=12525526" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Liu, J. ∙ Blake, S.J. ∙ Yong, M.C. ...</div><div><strong>Improved efficacy of neoadjuvant compared to adjuvant immunotherapy to eradicate metastatic disease</strong></div><div><em>Cancer Discov.</em> 2016; <strong>6</strong>:1382-1399</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-16-0577" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85007295572" target="_blank" aria-label="Open Scopus in new tab">Scopus (614)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27663893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-16-0577&amp;pmid=27663893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">53.</a></div><div>Lun, A.T.L. ∙ Riesenfeld, S. ∙ Andrews, T. ...</div><div><strong>EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:63</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1662-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85063349999" target="_blank" aria-label="Open Scopus in new tab">Scopus (445)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30902100/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1662-y&amp;pmid=30902100" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">54.</a></div><div>Luoma, A.M. ∙ Suo, S. ∙ Williams, H.L. ...</div><div><strong>Molecular pathways of colon inflammation induced by cancer immunotherapy</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:655-671.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2020.06.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2020.06.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85087986491" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32603654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.001&amp;pmid=32603654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Mackay, L.K. ∙ Rahimpour, A. ∙ Ma, J.Z. ...</div><div><strong>The developmental pathway for CD103(+)CD8+ tissue-resident memory T&nbsp;cells of skin</strong></div><div><em>Nat. Immunol.</em> 2013; <strong>14</strong>:1294-1301</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2744" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84888008030" target="_blank" aria-label="Open Scopus in new tab">Scopus (948)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24162776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2744&amp;pmid=24162776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Maier, B. ∙ Leader, A.M. ∙ Chen, S.T. ...</div><div><strong>A conserved dendritic-cell regulatory program limits antitumour immunity</strong></div><div><em>Nature.</em> 2020; <strong>580</strong>:257-262</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2134-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85082198759" target="_blank" aria-label="Open Scopus in new tab">Scopus (440)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32269339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2134-y&amp;pmid=32269339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>Mami-Chouaib, F. ∙ Blanc, C. ∙ Corgnac, S. ...</div><div><strong>Resident memory T&nbsp;cells, critical components in tumor immunology</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2018; <strong>6</strong>:87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s40425-018-0399-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85052961283" target="_blank" aria-label="Open Scopus in new tab">Scopus (198)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30180905/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs40425-018-0399-6&amp;pmid=30180905" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">58.</a></div><div>Mariathasan, S. ∙ Turley, S.J. ∙ Nickles, D. ...</div><div><strong>TGFbeta attenuates tumour response to PD-L1 blockade by contributing to exclusion of T&nbsp;cells</strong></div><div><em>Nature.</em> 2018; <strong>554</strong>:544-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85042495633" target="_blank" aria-label="Open Scopus in new tab">Scopus (3242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29443960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25501&amp;pmid=29443960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">59.</a></div><div>Mathewson, N.D. ∙ Ashenberg, O. ∙ Tirosh, I. ...</div><div><strong>Inhibitory CD161 receptor identified in glioma-infiltrating T&nbsp;cells by single-cell analysis</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:1281-1298.e26</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85101668184" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.01.022&amp;pmid=33592174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">60.</a></div><div>McGinnis, C.S. ∙ Murrow, L.M. ∙ Gartner, Z.J.</div><div><strong>DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors</strong></div><div><em>Cell Syst.</em> 2019; <strong>8</strong>:329-337.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85064396876" target="_blank" aria-label="Open Scopus in new tab">Scopus (1321)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30954475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2019.03.003&amp;pmid=30954475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">61.</a></div><div>McKenna, A. ∙ Hanna, M. ∙ Banks, E. ...</div><div><strong>The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.107524.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-77956295988" target="_blank" aria-label="Open Scopus in new tab">Scopus (18075)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20644199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.107524.110&amp;pmid=20644199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">62.</a></div><div>Menzies, A.M. ∙ Amaria, R.N. ∙ Rozeman, E.A. ...</div><div><strong>Pathological response and survival with neoadjuvant therapy in melanoma: a pooled analysis from the International Neoadjuvant Melanoma Consortium (INMC)</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:301-309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-020-01188-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85100662461" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33558722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-020-01188-3&amp;pmid=33558722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63-1" title="View in article">63.</a></div><div>Miao, D. ∙ Margolis, C.A. ∙ Vokes, N.I. ...</div><div><strong>Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1271-1281</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0200-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85052712350" target="_blank" aria-label="Open Scopus in new tab">Scopus (386)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30150660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0200-2&amp;pmid=30150660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">64.</a></div><div>Nazarov, V.i.b.R. ∙ E.</div><div><strong>immunarch: an R package for Painless Bioinformatics Analysis of T-cell and B-cell Immune Repertoires</strong></div><div><em>Zenodo.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.5281/zenodo.3367200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.5281%2Fzenodo.3367200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>Oliveira, G. ∙ Stromhaug, K. ∙ Klaeger, S. ...</div><div><strong>Phenotype, specificity and avidity of antitumour CD8(+) T&nbsp;cells in melanoma</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:119-125</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03704-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85110973451" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34290406/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03704-y&amp;pmid=34290406" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">66.</a></div><div>Park, S.L. ∙ Gebhardt, T. ∙ Mackay, L.K.</div><div><strong>Tissue-resident memory T&nbsp;cells in cancer immunosurveillance</strong></div><div><em>Trends Immunol.</em> 2019; <strong>40</strong>:735-747</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.it.2019.06.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.it.2019.06.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85067962417" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31255505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2019.06.002&amp;pmid=31255505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">67.</a></div><div>Philip, M. ∙ Schietinger, A.</div><div><strong>CD8(+) T&nbsp;cell differentiation and dysfunction in cancer</strong></div><div><em>Nat. Rev. Immunol.</em> 2021; <strong>22</strong>:209-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-021-00574-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85109907412" target="_blank" aria-label="Open Scopus in new tab">Scopus (396)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34253904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-021-00574-3&amp;pmid=34253904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">68.</a></div><div>Pijuan-Sala, B. ∙ Griffiths, J.A. ∙ Guibentif, C. ...</div><div><strong>A single-cell molecular map of mouse gastrulation and early organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:490-495</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0933-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85062300354" target="_blank" aria-label="Open Scopus in new tab">Scopus (483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787436/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0933-9&amp;pmid=30787436" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">69.</a></div><div>Rytlewski, J. ∙ Deng, S. ∙ Xie, T. ...</div><div><strong>Model to improve specificity for identification of clinically-relevant expanded T&nbsp;cells in peripheral blood</strong></div><div><em>PLoS One.</em> 2019; <strong>14</strong>:e0213684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0213684" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85062994573" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30870493/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0213684&amp;pmid=30870493" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">70.</a></div><div>Saunders, C.T. ∙ Wong, W.S. ∙ Swamy, S. ...</div><div><strong>Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:1811-1817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84864153492" target="_blank" aria-label="Open Scopus in new tab">Scopus (1149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22581179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts271&amp;pmid=22581179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71-1" title="View in article">71.</a></div><div>Savas, P. ∙ Virassamy, B. ∙ Ye, C. ...</div><div><strong>Single-cell profiling of breast cancer T&nbsp;cells reveals a tissue-resident memory subset associated with improved prognosis</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:986-993</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0078-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85049055113" target="_blank" aria-label="Open Scopus in new tab">Scopus (656)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0078-7&amp;pmid=29942092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">72.</a></div><div>Schoenfeld, J.D. ∙ Hanna, G.J. ∙ Jo, V.Y. ...</div><div><strong>Neoadjuvant nivolumab or nivolumab plus ipilimumab in untreated oral cavity squamous cell carcinoma: A Phase 2 open-label randomized clinical trial</strong></div><div><em>JAMA Oncol.</em> 2020; <strong>6</strong>:1563-1570</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jamaoncol.2020.2955" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85090783150" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32852531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Fjamaoncol.2020.2955&amp;pmid=32852531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">73.</a></div><div>Scholtalbers, J. ∙ Boegel, S. ∙ Bukur, T. ...</div><div><strong>TCLP: an online cancer cell line catalogue integrating HLA type, predicted neo-epitopes, virus and gene expression</strong></div><div><em>Genome Med.</em> 2015; <strong>7</strong>:118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-015-0240-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84947767711" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26589293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-015-0240-5&amp;pmid=26589293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">74.</a></div><div>Sharma, P. ∙ Allison, J.P.</div><div><strong>The future of immune checkpoint therapy</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:56-61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa8172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84928774156" target="_blank" aria-label="Open Scopus in new tab">Scopus (3636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25838373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa8172&amp;pmid=25838373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">75.</a></div><div>Shen, R. ∙ Seshan, V.E.</div><div><strong>FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84988924813" target="_blank" aria-label="Open Scopus in new tab">Scopus (712)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27270079/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw520&amp;pmid=27270079" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">76.</a></div><div>Singer, M. ∙ Wang, C. ∙ Cong, L. ...</div><div><strong>A distinct gene module for dysfunction uncoupled from activation in tumor-infiltrating T&nbsp;cells</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:1500-1511.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84986285632" target="_blank" aria-label="Open Scopus in new tab">Scopus (269)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27610572/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.052&amp;pmid=27610572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">77.</a></div><div>Street, K. ∙ Risso, D. ∙ Fletcher, R.B. ...</div><div><strong>Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics</strong></div><div><em>BMC Genomics.</em> 2018; <strong>19</strong>:477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12864-018-4772-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85048725973" target="_blank" aria-label="Open Scopus in new tab">Scopus (1200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29914354/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12864-018-4772-0&amp;pmid=29914354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">78.</a></div><div>Szabo, P.A. ∙ Levitin, H.M. ∙ Miron, M. ...</div><div><strong>Single-cell transcriptomics of human T&nbsp;cells reveals tissue and activation signatures in health and disease</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4706</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12464-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85073621555" target="_blank" aria-label="Open Scopus in new tab">Scopus (345)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31624246/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12464-3&amp;pmid=31624246" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">79.</a></div><div>Szabo, P.A. ∙ Miron, M. ∙ Farber, D.L.</div><div><strong>Location, location, location: tissue resident memory T&nbsp;cells in mice and humans</strong></div><div><em>Sci. Immunol.</em> 2019; <strong>4</strong>:eaas9673</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aas9673" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85064505042" target="_blank" aria-label="Open Scopus in new tab">Scopus (355)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30952804/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aas9673&amp;pmid=30952804" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">80.</a></div><div>Taylor-Weiner, A. ∙ Stewart, C. ∙ Giordano, T. ...</div><div><strong>DeTiN: overcoming tumor-in-normal contamination</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:531-534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0036-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85048968644" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29941871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-018-0036-9&amp;pmid=29941871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">81.</a></div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad0501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84963614956" target="_blank" aria-label="Open Scopus in new tab">Scopus (2751)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27124452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad0501&amp;pmid=27124452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">82.</a></div><div>Topalian, S.L. ∙ Taube, J.M. ∙ Pardoll, D.M.</div><div><strong>Neoadjuvant checkpoint blockade for cancer immunotherapy</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:eaax0182</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aax0182" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85078710390" target="_blank" aria-label="Open Scopus in new tab">Scopus (580)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32001626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aax0182&amp;pmid=32001626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">83.</a></div><div>Tumeh, P.C. ∙ Harview, C.L. ∙ Yearley, J.H. ...</div><div><strong>PD-1 blockade induces responses by inhibiting adaptive immune resistance</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:568-571</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-84920956732" target="_blank" aria-label="Open Scopus in new tab">Scopus (5183)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13954&amp;pmid=25428505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">84.</a></div><div>Uppaluri, R. ∙ Campbell, K.M. ∙ Egloff, A.M. ...</div><div><strong>Neoadjuvant and adjuvant pembrolizumab in resectable locally advanced, human papillomavirus-unrelated head and neck cancer: A multicenter, Phase II trial</strong></div><div><em>Clin. Cancer Res.</em> 2020; <strong>26</strong>:5140-5152</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-20-1695" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85097627415" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32665297/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-20-1695&amp;pmid=32665297" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85-1" title="View in article">85.</a></div><div>Van den Berge, K. ∙ Roux de Bézieux, H. ∙ Street, K. ...</div><div><strong>Trajectory-based differential expression analysis for single-cell sequencing data</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:1201</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-14766-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85081332065" target="_blank" aria-label="Open Scopus in new tab">Scopus (285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32139671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-14766-3&amp;pmid=32139671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">86.</a></div><div>van der Leun, A.M. ∙ Thommen, D.S. ∙ Schumacher, T.N.</div><div><strong>CD8(+) T&nbsp;cell states in human cancer: insights from single-cell analysis</strong></div><div><em>Nat. Rev. Cancer.</em> 2020; <strong>20</strong>:218-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41568-019-0235-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85079438043" target="_blank" aria-label="Open Scopus in new tab">Scopus (769)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32024970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41568-019-0235-4&amp;pmid=32024970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">87.</a></div><div>Versluis, J.M. ∙ Long, G.V. ∙ Blank, C.U.</div><div><strong>Learning from clinical trials of neoadjuvant checkpoint blockade</strong></div><div><em>Nat. Med.</em> 2020; <strong>26</strong>:475-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-020-0829-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85083385305" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32273608/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-020-0829-0&amp;pmid=32273608" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">88.</a></div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89-1" title="View in article">89.</a></div><div>Xin, H. ∙ Lian, Q. ∙ Jiang, Y. ...</div><div><strong>GMM-Demux: sample demultiplexing, multiplet detection, experiment planning, and novel cell-type verification in single cell sequencing</strong></div><div><em>Genome Biol.</em> 2020; <strong>21</strong>:188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-020-02084-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85088907977" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32731885/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-020-02084-2&amp;pmid=32731885" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">90.</a></div><div>Yost, K.E. ∙ Satpathy, A.T. ∙ Wells, D.K. ...</div><div><strong>Clonal replacement of tumor-specific T&nbsp;cells following PD-1 blockade</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:1251-1259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0522-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.018&amp;key=2-s2.0-85069859998" target="_blank" aria-label="Open Scopus in new tab">Scopus (880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31359002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0522-3&amp;pmid=31359002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/0a4ac243-40ec-450a-856a-b3aff04903f6/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/4f3e8c7d-02ba-4b5a-8d9e-b39c39a736de/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/87f646f0-b5ed-4c89-b48a-ea836d87fdfd/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/f25c859e-9ad8-4041-8500-dedf4627bfbe/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/0cd1b84b-8fc0-46fa-8b48-3bd78bc47c7d/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/cbf96089-ce7a-441a-9f59-6398c258ee82/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/5f45d401-756c-4315-a6dd-e1068a1ed481/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/dab596af-3fe7-41ad-9736-96829298483d/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/d3a5eed8-1cd1-44f2-a45c-3e9c86dd8e42/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/4b33d044-7a9f-449f-86ab-580cf9d5a9c6/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/24fc7f3d-7af3-4a91-aa99-242197c9154c/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/03b126ea-0d51-4094-b93d-e81508d42aa1/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/d921bbc9-6a31-421b-a8f7-db745b7a99a3/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/asset/ed87b54b-0a2d-4b66-81aa-de630d0bb0da/main.assets/gr7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Barticle%3Aarticle%3Apii%5C%3AS0092867422007231%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00177%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (7)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/attachment/d0c71e9c-69b7-4ded-9c1f-88054b583c88/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (15.59 KB)</span></a><div class="core-description">Table&nbsp;S1. Detailed summary of patient response metrics, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/attachment/aeab8993-eae4-47e1-96d2-6559807880d0/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (1012.72 KB)</span></a><div class="core-description">Table&nbsp;S2. Marker genes for tumor-infiltrating immune cell subsets, related to Figures&nbsp;2, S2, and S5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/attachment/37095adf-7956-4c3a-8cfc-763edf3abc45/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (106.94 KB)</span></a><div class="core-description">Table&nbsp;S3. Differentially expressed genes in Tx-E T&nbsp;cells in tumors, related to Figures&nbsp;4 and S4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/attachment/3d8d5311-eef0-4558-b0d7-1ce66f907b60/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (10.52 KB)</span></a><div class="core-description">Table&nbsp;S4. Gene signatures used for analyzing Tx-E T&nbsp;cells and independent ICB treatment cohort, related to Figures&nbsp;4 and S4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/attachment/4f327d9c-e1c5-44e8-a8b9-102ec6f9cb39/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (11.33 KB)</span></a><div class="core-description">Table&nbsp;S5. T-scan screen summary, related to Figures&nbsp;5 and S5</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/attachment/16d0df13-1962-443d-a609-b39ba32212bb/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (868.11 KB)</span></a><div class="core-description">Table&nbsp;S6. Marker genes for blood immune cell subsets, related to Figures&nbsp;6 and S6</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.018/attachment/b54231fe-ae45-4895-888a-45958999921c/mmc7.xlsx" download="mmc7.xlsx"><span id="mmc7-link">Spreadsheet (11.93 KB)</span></a><div class="core-description">Table&nbsp;S7. Summary of blood cytometry findings, related to Figures&nbsp;7 and S7</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867422007231" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(25)00076-7" target="_blank" title="Collaboration between interleukin-7 and -15 enables adaptation of tissue-resident and circulating memory CD8+ T&nbsp;cells to cytokine deficiency">Collaboration between interleukin-7 and -15 enables adaptation of tissue-resident and circulating memory CD8+ T&nbsp;cells to cytokine deficiency</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jarjour et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">February 28, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(23)00262-5" target="_blank" title="Single-cell protein expression profiling resolves circulating and resident memory T&nbsp;cell diversity across tissues and infection contexts">Single-cell protein expression profiling resolves circulating and resident memory T&nbsp;cell diversity across tissues and infection contexts</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Evrard et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">June 30, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/developmental-cell/fulltext/S1534-5807(25)00249-7" target="_blank" title="Targeting VSIG4+ tissue-resident macrophages enhances T&nbsp;cell cytotoxicity and immunotherapy efficacy in cancer">Targeting VSIG4+ tissue-resident macrophages enhances T&nbsp;cell cytotoxicity and immunotherapy efficacy in cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ma et al.</li></ul></div><div class="linked-article__details"><span><i>Developmental Cell</i></span><span id="item_date">May 7, 2025</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(25)00066-X" target="_blank" title="Mechanistic insight into the induction of liver tissue-resident memory CD8+ T&nbsp;cells by glycolipid-peptide vaccination">Mechanistic insight into the induction of liver tissue-resident memory CD8+ T&nbsp;cells by glycolipid-peptide vaccination</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chua et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">February 12, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cancer-cell/fulltext/S1535-6108(23)00092-2" target="_blank" title="Early immune pressure initiated by tissue-resident memory T&nbsp;cells sculpts tumor evolution in non-small cell lung cancer">Early immune pressure initiated by tissue-resident memory T&nbsp;cells sculpts tumor evolution in non-small cell lung cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Weeden et al.</li></ul></div><div class="linked-article__details"><span><i>Cancer Cell</i></span><span id="item_date">April 21, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(23)00500-9" target="_blank" title="Circulating NK cells establish tissue residency upon acute infection of skin and mediate accelerated effector responses to secondary infection">Circulating NK cells establish tissue residency upon acute infection of skin and mediate accelerated effector responses to secondary infection</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Torcellan et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">December 28, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(24)00522-3" target="_blank" title="Deep profiling deconstructs features associated with memory CD8+ T&nbsp;cell tissue residence">Deep profiling deconstructs features associated with memory CD8+ T&nbsp;cell tissue residence</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Scott et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">December 20, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(23)00544-X" target="_blank" title="iPSC-derived hypoimmunogenic tissue resident memory T&nbsp;cells mediate robust anti-tumor activity against cervical cancer">iPSC-derived hypoimmunogenic tissue resident memory T&nbsp;cells mediate robust anti-tumor activity against cervical cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Furukawa et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">December 12, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(24)00457-6" target="_blank" title="CRISPR screens unveil nutrient-dependent lysosomal and mitochondrial nodes impacting intestinal tissue-resident memory CD8+ T&nbsp;cell formation">CRISPR screens unveil nutrient-dependent lysosomal and mitochondrial nodes impacting intestinal tissue-resident memory CD8+ T&nbsp;cell formation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Raynor et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">October 14, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00015-9" target="_blank" title="Circulating tumor cells reveal early predictors of disease progression in patients with stage III NSCLC undergoing chemoradiation and immunotherapy">Circulating tumor cells reveal early predictors of disease progression in patients with stage III NSCLC undergoing chemoradiation and immunotherapy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Purcell et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">January 22, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/med/fulltext/S2666-6340(23)00290-8" target="_blank" title="Cytotoxic CD4+ tissue-resident memory T&nbsp;cells are associated with asthma severity">Cytotoxic CD4+ tissue-resident memory T&nbsp;cells are associated with asthma severity</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Herrera-De La Mata et al.</li></ul></div><div class="linked-article__details"><span><i>Med</i></span><span id="item_date">October 20, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(24)00323-6" target="_blank" title="Resident memory T&nbsp;cells and cancer">Resident memory T&nbsp;cells and cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gavil et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">August 13, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(25)00095-0" target="_blank" title="Lung-resident memory B cells maintain allergic IgE responses in the respiratory tract">Lung-resident memory B cells maintain allergic IgE responses in the respiratory tract</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Nelson et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">March 25, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(23)00026-2" target="_blank" title="Infection induces tissue-resident memory NK cells that safeguard tissue health">Infection induces tissue-resident memory NK cells that safeguard tissue health</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Schuster et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">February 10, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)09856-0" target="_blank" title="Development and validation of a circulating tumor cells-related signature focusing on biochemical recurrence and immunotherapy response in prostate cancer">Development and validation of a circulating tumor cells-related signature focusing on biochemical recurrence and immunotherapy response in prostate cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">November 20, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(23)00167-2" target="_blank" title="Tissue-resident memory CAR T&nbsp;cells with stem-like characteristics display enhanced efficacy against solid and liquid tumors">Tissue-resident memory CAR T&nbsp;cells with stem-like characteristics display enhanced efficacy against solid and liquid tumors</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jung et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">May 23, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cancer-cell/fulltext/S1535-6108(23)00004-1" target="_blank" title="Intratumoral CD8+ T&nbsp;cells with a tissue-resident memory phenotype mediate local immunity and immune checkpoint responses in breast cancer">Intratumoral CD8+ T&nbsp;cells with a tissue-resident memory phenotype mediate local immunity and immune checkpoint responses in breast cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Virassamy et al.</li></ul></div><div class="linked-article__details"><span><i>Cancer Cell</i></span><span id="item_date">February 23, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(25)00065-8" target="_blank" title="Lung CD4+ resident memory T&nbsp;cells use airway secretory cells to stimulate and regulate onset of allergic airway neutrophilic disease">Lung CD4+ resident memory T&nbsp;cells use airway secretory cells to stimulate and regulate onset of allergic airway neutrophilic disease</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ravi et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">February 17, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00026-3" target="_blank" title="Decidual-tissue-resident memory T&nbsp;cells protect against nonprimary human cytomegalovirus infection at the maternal-fetal interface">Decidual-tissue-resident memory T&nbsp;cells protect against nonprimary human cytomegalovirus infection at the maternal-fetal interface</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Alfi et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">January 23, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(23)00265-0" target="_blank" title="Site-specific development and progressive maturation of human tissue-resident memory T&nbsp;cells over infancy and childhood">Site-specific development and progressive maturation of human tissue-resident memory T&nbsp;cells over infancy and childhood</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Connors et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">July 7, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00971-9" target="_blank" title="Epithelial organoid supports resident memory CD8 T&nbsp;cell differentiation">Epithelial organoid supports resident memory CD8 T&nbsp;cell differentiation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ulibarri et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">August 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/cancer/fulltext/S2405-8033(23)00170-X" target="_blank" title="Circulating cell-free DNA-based multi-cancer early detection">Circulating cell-free DNA-based multi-cancer early detection</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Cancer</i></span><span id="item_date">September 13, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(23)00371-0" target="_blank" title="Interferon-γ production by Tfh cells is required for CXCR3+ pre-memory B cell differentiation and subsequent lung-resident memory B cell responses">Interferon-γ production by Tfh cells is required for CXCR3+ pre-memory B cell differentiation and subsequent lung-resident memory B cell responses</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Arroyo-Díaz et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">September 11, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cancer-cell/fulltext/S1535-6108(22)00123-4" target="_blank" title="Ovarian cancer immunogenicity is governed by a narrow subset of progenitor tissue-resident memory T&nbsp;cells">Ovarian cancer immunogenicity is governed by a narrow subset of progenitor tissue-resident memory T&nbsp;cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Anadon et al.</li></ul></div><div class="linked-article__details"><span><i>Cancer Cell</i></span><span id="item_date">April 14, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/cancer/fulltext/S2405-8033(22)00044-9" target="_blank" title="Elevated circulating memory T cells precede immunotherapy toxicities in melanoma">Elevated circulating memory T cells precede immunotherapy toxicities in melanoma</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Bello et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Cancer</i></span><span id="item_date">March 8, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(25)00791-1" target="_blank" title="Appendiceal cancer in pregnancy and circulating tumor cells detection in the umbilical cord">Appendiceal cancer in pregnancy and circulating tumor cells detection in the umbilical cord</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Kudo et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">January 30, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(25)00072-5" target="_blank" title="A microRNA-regulated transcriptional state defines intratumoral CD8+ T&nbsp;cells that respond to immunotherapy">A microRNA-regulated transcriptional state defines intratumoral CD8+ T&nbsp;cells that respond to immunotherapy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Tang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">February 12, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(24)00322-4" target="_blank" title="Dynamic landscapes and protective immunity coordinated by influenza-specific lung-resident memory CD8+ T&nbsp;cells revealed by intravital imaging">Dynamic landscapes and protective immunity coordinated by influenza-specific lung-resident memory CD8+ T&nbsp;cells revealed by intravital imaging</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>van de Wall et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">July 22, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(24)00222-3" target="_blank" title="Identification of memory mechanism in tissue-resident stem cells via ANGPTL4 beyond immune cells upon viral antigen exposure">Identification of memory mechanism in tissue-resident stem cells via ANGPTL4 beyond immune cells upon viral antigen exposure</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Min et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">April 5, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(24)00320-0" target="_blank" title="Distinct epigenomic landscapes underlie tissue-specific memory T&nbsp;cell differentiation">Distinct epigenomic landscapes underlie tissue-specific memory T&nbsp;cell differentiation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Buquicchio et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">July 22, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(25)00018-X" target="_blank" title="Repetitive antigen stimulation in the periphery dictates the composition and recall responses of brain-resident memory CD8+ T&nbsp;cells">Repetitive antigen stimulation in the periphery dictates the composition and recall responses of brain-resident memory CD8+ T&nbsp;cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Mix et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">February 3, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)00498-X" target="_blank" title="Tissue-resident memory T&nbsp;cell signatures from single-cell analysis associated with better melanoma prognosis">Tissue-resident memory T&nbsp;cell signatures from single-cell analysis associated with better melanoma prognosis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jiang et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">February 20, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(22)00643-4" target="_blank" title="Small intestine and colon tissue-resident memory CD8+ T&nbsp;cells exhibit molecular heterogeneity and differential dependence on Eomes">Small intestine and colon tissue-resident memory CD8+ T&nbsp;cells exhibit molecular heterogeneity and differential dependence on Eomes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lin et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">December 28, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(24)00167-8" target="_blank" title="Circulating microbiome DNA as biomarkers for early diagnosis and recurrence of lung cancer">Circulating microbiome DNA as biomarkers for early diagnosis and recurrence of lung cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chen et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">April 5, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(22)00625-8" target="_blank" title="Corneal tissue-resident memory T cells form a unique immune compartment at the ocular surface">Corneal tissue-resident memory T cells form a unique immune compartment at the ocular surface</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Loi et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">May 24, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)01006-5" target="_blank" title="Distinct Transcriptomic Signatures in Pulmonary Tissue Resident and Vascular Resident-Like T Cells in Nonhuman Primates">Distinct Transcriptomic Signatures in Pulmonary Tissue Resident and Vascular Resident-Like T Cells in Nonhuman Primates</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Melton et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">May 26, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(22)01867-8" target="_blank" title="The aryl hydrocarbon receptor cell intrinsically promotes resident memory CD8+ T&nbsp;cell differentiation and function">The aryl hydrocarbon receptor cell intrinsically promotes resident memory CD8+ T&nbsp;cell differentiation and function</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Dean et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">January 4, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00373-5" target="_blank" title="Generation of circulating autoreactive pre-plasma cells fueled by naive B cells in celiac disease">Generation of circulating autoreactive pre-plasma cells fueled by naive B cells in celiac disease</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lindeman et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">April 4, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)10044-8" target="_blank" title="Highly consistency of PIK3CA mutation spectrum between circulating tumor DNA and paired tissue in lung cancer patients">Highly consistency of PIK3CA mutation spectrum between circulating tumor DNA and paired tissue in lung cancer patients</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 4, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/patterns/fulltext/S2666-3899(25)00133-3" target="_blank" title="Cluster-based human-in-the-loop strategy for improving machine learning-based circulating tumor cell detection in liquid biopsy">Cluster-based human-in-the-loop strategy for improving machine learning-based circulating tumor cell detection in liquid biopsy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Husseini-Wüsthoff et al.</li></ul></div><div class="linked-article__details"><span><i>Patterns</i></span><span id="item_date">May 30, 2025</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2022.06.018","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Tissue-resident memory and circulating T cells are early responders to pre-surgical cancer immunotherapy</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.018/asset/0a4ac243-40ec-450a-856a-b3aff04903f6/main.assets/gr1.jpg" height="693" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/5fe8ca01-b188-47ee-a1d4-bf7feda23660/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.018/asset/4f3e8c7d-02ba-4b5a-8d9e-b39c39a736de/main.assets/figs1.jpg" height="941" width="744" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/17e5e84c-ac2a-4708-8877-0072850b688b/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.018/asset/87f646f0-b5ed-4c89-b48a-ea836d87fdfd/main.assets/gr2.jpg" height="951" width="758" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/0ddbd122-5c77-43a0-89a6-180ff350bd31/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.018/asset/f25c859e-9ad8-4041-8500-dedf4627bfbe/main.assets/figs2.jpg" height="928" width="769" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/8b7c7a3e-c189-4ea2-8a12-b4cf44e95d84/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.018/asset/0cd1b84b-8fc0-46fa-8b48-3bd78bc47c7d/main.assets/gr3.jpg" height="608" width="760" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/9323abcc-0d32-47ff-8565-6d6690eb2d09/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.018/asset/cbf96089-ce7a-441a-9f59-6398c258ee82/main.assets/figs3.jpg" height="894" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/0678bc58-a8d5-4bb7-a1cc-0bccf75c4591/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.018/asset/5f45d401-756c-4315-a6dd-e1068a1ed481/main.assets/gr4.jpg" height="653" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/b36b2ba8-9871-4151-929d-146d951628b3/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.018/asset/dab596af-3fe7-41ad-9736-96829298483d/main.assets/figs4.jpg" height="942" width="734" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/7b67d036-dbb8-467e-a4c6-a8df016373c1/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.018/asset/d3a5eed8-1cd1-44f2-a45c-3e9c86dd8e42/main.assets/gr5.jpg" height="828" width="759" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/e21f2e34-9905-43b3-9274-0d7a7ea096d2/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.018/asset/4b33d044-7a9f-449f-86ab-580cf9d5a9c6/main.assets/figs5.jpg" height="941" width="754" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/e9a1e181-4788-42b8-842a-1d0824a44ade/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.018/asset/24fc7f3d-7af3-4a91-aa99-242197c9154c/main.assets/figs6.jpg" height="941" width="741" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/7d37f76a-927f-4eef-ab1e-ff58822f38e2/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.018/asset/03b126ea-0d51-4094-b93d-e81508d42aa1/main.assets/gr6.jpg" height="814" width="769" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/b8f1b822-416f-4d1f-8285-9aecc75e74d1/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.06.018/asset/d921bbc9-6a31-421b-a8f7-db745b7a99a3/main.assets/figs7.jpg" height="941" width="754" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/8903ae21-c7ac-4ff1-93ea-b495d98218f9/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.06.018/asset/ed87b54b-0a2d-4b66-81aa-de630d0bb0da/main.assets/gr7.jpg" height="941" width="745" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/890d25fb-73e3-4f74-bfe7-ab5aef1e4f6e/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.018/asset/0a4ac243-40ec-450a-856a-b3aff04903f6/main.assets/gr1.jpg" height="693" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/5fe8ca01-b188-47ee-a1d4-bf7feda23660/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.018/asset/4f3e8c7d-02ba-4b5a-8d9e-b39c39a736de/main.assets/figs1.jpg" height="941" width="744" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/17e5e84c-ac2a-4708-8877-0072850b688b/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.018/asset/87f646f0-b5ed-4c89-b48a-ea836d87fdfd/main.assets/gr2.jpg" height="951" width="758" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/0ddbd122-5c77-43a0-89a6-180ff350bd31/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.018/asset/f25c859e-9ad8-4041-8500-dedf4627bfbe/main.assets/figs2.jpg" height="928" width="769" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/8b7c7a3e-c189-4ea2-8a12-b4cf44e95d84/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.018/asset/0cd1b84b-8fc0-46fa-8b48-3bd78bc47c7d/main.assets/gr3.jpg" height="608" width="760" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/9323abcc-0d32-47ff-8565-6d6690eb2d09/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.018/asset/cbf96089-ce7a-441a-9f59-6398c258ee82/main.assets/figs3.jpg" height="894" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/0678bc58-a8d5-4bb7-a1cc-0bccf75c4591/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.018/asset/5f45d401-756c-4315-a6dd-e1068a1ed481/main.assets/gr4.jpg" height="653" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/b36b2ba8-9871-4151-929d-146d951628b3/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.018/asset/dab596af-3fe7-41ad-9736-96829298483d/main.assets/figs4.jpg" height="942" width="734" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/7b67d036-dbb8-467e-a4c6-a8df016373c1/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.018/asset/d3a5eed8-1cd1-44f2-a45c-3e9c86dd8e42/main.assets/gr5.jpg" height="828" width="759" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/e21f2e34-9905-43b3-9274-0d7a7ea096d2/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.018/asset/4b33d044-7a9f-449f-86ab-580cf9d5a9c6/main.assets/figs5.jpg" height="941" width="754" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/e9a1e181-4788-42b8-842a-1d0824a44ade/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.018/asset/24fc7f3d-7af3-4a91-aa99-242197c9154c/main.assets/figs6.jpg" height="941" width="741" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/7d37f76a-927f-4eef-ab1e-ff58822f38e2/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.018/asset/03b126ea-0d51-4094-b93d-e81508d42aa1/main.assets/gr6.jpg" height="814" width="769" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/b8f1b822-416f-4d1f-8285-9aecc75e74d1/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.06.018/asset/d921bbc9-6a31-421b-a8f7-db745b7a99a3/main.assets/figs7.jpg" height="941" width="754" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/8903ae21-c7ac-4ff1-93ea-b495d98218f9/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.06.018/asset/ed87b54b-0a2d-4b66-81aa-de630d0bb0da/main.assets/gr7.jpg" height="941" width="745" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.018/asset/890d25fb-73e3-4f74-bfe7-ab5aef1e4f6e/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00723-1';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00723-1';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749594052478","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"185","issueNumber":"16","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867422007231","coverDate":"202208","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Tissueresident memory and circulating T cells are early responders to presurgical cancer immunotherapy","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc42203c3627ff',t:'MTc0OTU5NDA1MS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc42203c3627ff&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.6.2&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2022.06.018&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>