<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="cefc7bf2-fe34-4e10-ae5e-9ac7a2bde95a" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867422002598;journal:journal:cell;issue:issue:pii\:S0092867421X00098;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(22)00259-8">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/d72e4e8c-c3d7-47e9-b554-6a10ed08462b/fx1.jpg">
    


        <meta property="og:description" content="Cryo-EM and mass spectrometry-based proteomics of insoluble amyloid fibrils derived
from postmortem human brains afflicted with diverse neurodegenerative diseases reveals
widespread fibrillization of an endolysosomal membrane protein, TMEM106B, pointing
toward a potentially pathogenic commonality between distinct proteinopathies.">
    











        <meta name="twitter:title" content="Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/d72e4e8c-c3d7-47e9-b554-6a10ed08462b/fx1.jpg">
    


        <meta name="twitter:description" content="Cryo-EM and mass spectrometry-based proteomics of insoluble amyloid fibrils derived
from postmortem human brains afflicted with diverse neurodegenerative diseases reveals
widespread fibrillization of an endolysosomal membrane protein, TMEM106B, pointing
toward a potentially pathogenic commonality between distinct proteinopathies.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(22)00259-8">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases"> <meta name="citation_date" content="2022/04/14"> <meta name="citation_online_date" content="2022/03/04"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="8"> <meta name="citation_firstpage" content="1346"> <meta name="citation_lastpage" content="1355.e15"> <meta name="citation_pmid" content="35247328"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867422002598"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867422002598"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867422002598"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2022.02.026"> <meta name="citation_pmid" content="35247328"> <meta name="citation_author" content="Andrew Chang"><meta name="citation_author_institution" content="Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA"><meta name="citation_author_institution" content="Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA"><meta name="citation_author" content="Xinyu Xiang"><meta name="citation_author_institution" content="Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA"><meta name="citation_author_institution" content="Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA"><meta name="citation_author_institution" content="Department of Structural Biology, Stanford University School of Medicine, Stanford, CA 94305, USA"><meta name="citation_author" content="Jing Wang"><meta name="citation_author_institution" content="Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA"><meta name="citation_author_institution" content="Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA"><meta name="citation_author" content="Carolyn Lee"><meta name="citation_author_institution" content="Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA"><meta name="citation_author_institution" content="Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA"><meta name="citation_author_institution" content="Department of Microbiology &amp; Immunology, Columbia University, New York, NY 10032, USA"><meta name="citation_author" content="Tamta Arakhamia"><meta name="citation_author_institution" content="Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA"><meta name="citation_author_institution" content="Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA"><meta name="citation_author" content="Marija Simjanoska"><meta name="citation_author_institution" content="Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA"><meta name="citation_author_institution" content="Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA"><meta name="citation_author" content="Chi Wang"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA"><meta name="citation_author" content="Yari Carlomagno"><meta name="citation_author_institution" content="Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA"><meta name="citation_author" content="Guoan Zhang"><meta name="citation_author_institution" content="Proteomics and Metabolomics Core Facility, Weill Cornell Medicine, New York, NY 10021, USA"><meta name="citation_author" content="Shikhar Dhingra"><meta name="citation_author_institution" content="Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA"><meta name="citation_author" content="Manon Thierry"><meta name="citation_author_institution" content="Centre for Cognitive Neurology, Department of Neurology, New York University School of Medicine, New York, NY 10016, USA"><meta name="citation_author" content="Jolien Perneel"><meta name="citation_author_institution" content="Applied and Translational Neurogenomics, VIB Center for Molecular Neurology, VIB, Antwerp, Belgium"><meta name="citation_author_institution" content="Department of Biomedical Sciences, University of Antwerp, Antwerp, Belgium"><meta name="citation_author" content="Bavo Heeman"><meta name="citation_author_institution" content="Applied and Translational Neurogenomics, VIB Center for Molecular Neurology, VIB, Antwerp, Belgium"><meta name="citation_author_institution" content="Department of Biomedical Sciences, University of Antwerp, Antwerp, Belgium"><meta name="citation_author" content="Lauren M. Forgrave"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, BC V6T 2B5, Canada"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, St. Paul’s Hospital, Providence Health Care, Vancouver, Canada"><meta name="citation_author" content="Michael DeTure"><meta name="citation_author_institution" content="Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA"><meta name="citation_author" content="Mari L. DeMarco"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, BC V6T 2B5, Canada"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, St. Paul’s Hospital, Providence Health Care, Vancouver, Canada"><meta name="citation_author" content="Casey N. Cook"><meta name="citation_author_institution" content="Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA"><meta name="citation_author" content="Rosa Rademakers"><meta name="citation_author_institution" content="Applied and Translational Neurogenomics, VIB Center for Molecular Neurology, VIB, Antwerp, Belgium"><meta name="citation_author_institution" content="Department of Biomedical Sciences, University of Antwerp, Antwerp, Belgium"><meta name="citation_author" content="Dennis W. Dickson"><meta name="citation_author_institution" content="Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA"><meta name="citation_author" content="Leonard Petrucelli"><meta name="citation_author_institution" content="Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA"><meta name="citation_author" content="Michael H.B. Stowell"><meta name="citation_author_email" content="stowellm@colorado.edu"><meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of Colorado Boulder, Boulder, CO 80309, USA"><meta name="citation_author" content="Ian R.A. Mackenzie"><meta name="citation_author_email" content="ian.mackenzie@vch.ca"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, BC V6T 2B5, Canada"><meta name="citation_author" content="Anthony W.P. Fitzpatrick"><meta name="citation_author_email" content="anthony.fitzpatrick@columbia.edu"><meta name="citation_author_institution" content="Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA"><meta name="citation_author_institution" content="Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA"><meta name="citation_keywords" content="cryo-EM; neurodegeneration; TMEM106B; FTLD-TDP; DLB; PSP; amyloid fibrils; proteolysis; lysosome; endosome"><meta name="citation_reference" content="citation_title=Real-space refinement in PHENIX for cryo-EM and crystallography; citation_author=P.V. Afonine; citation_author=B.K. Poon; citation_author=R.J. Read; citation_author=O.V. Sobolev; citation_author=T.C. Terwilliger; citation_author=A. Urzhumtsev; citation_author=P.D. Adams; citation_journal_title=Acta Crystallogr. D Struct. Biol.; citation_volume=74; citation_firstpage=531; citation_lastpage=544; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains; citation_author=T. Arakhamia; citation_author=C.E. Lee; citation_author=Y. Carlomagno; citation_author=D.M. Duong; citation_author=S.R. Kundinger; citation_author=K. Wang; citation_author=D. Williams; citation_author=M. DeTure; citation_author=D.W. Dickson; citation_author=C.N. Cook; citation_author=et al.; citation_journal_title=Cell; citation_volume=180; citation_firstpage=633; citation_lastpage=644.e12; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Structure of pathological TDP-43 filaments from ALS with FTLD; citation_author=D. Arseni; citation_author=M. Hasegawa; citation_author=A.G. Murzin; citation_author=F. Kametani; citation_author=M. Arai; citation_author=M. Yoshida; citation_author=B. Ryskeldi-Falcon; citation_journal_title=Nature; citation_volume=601; citation_firstpage=139; citation_lastpage=143; citation_publication_date=2022; "><meta name="citation_reference" content="citation_title=Mutations in progranulin cause tau-negative frontotemporal dementia linked to chromosome 17; citation_author=M. Baker; citation_author=I.R. Mackenzie; citation_author=S.M. Pickering-Brown; citation_author=J. Gass; citation_author=R. Rademakers; citation_author=C. Lindholm; citation_author=J. Snowden; citation_author=J. Adamson; citation_author=A.D. Sadovnick; citation_author=S. Rollinson; citation_author=et al.; citation_journal_title=Nature; citation_volume=442; citation_firstpage=916; citation_lastpage=919; citation_publication_date=2006; "><meta name="citation_reference" content="biochem-fan atom2svg.py. https://gist.github.com/biochem-fan/026ec2f191fee9285424d12fc2b84ce7."><meta name="citation_reference" content="citation_title=Regulated intramembrane proteolysis of the frontotemporal lobar degeneration risk factor, TMEM106B, by signal peptide peptidase-like 2a (SPPL2a); citation_author=O.A. Brady; citation_author=X. Zhou; citation_author=F. Hu; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=289; citation_firstpage=19670; citation_lastpage=19680; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Tau and Aβ imaging, CSF measures, and cognition in Alzheimer’s disease; citation_author=M.R. Brier; citation_author=B. Gordon; citation_author=K. Friedrichsen; citation_author=J. McCarthy; citation_author=A. Stern; citation_author=J. Christensen; citation_author=C. Owen; citation_author=P. Aldea; citation_author=Y. Su; citation_author=J. Hassenstab; citation_author=et al.; citation_journal_title=Sci. Transl. Med.; citation_volume=8; citation_firstpage=338ra66; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Expression of TMEM106B, the frontotemporal lobar degeneration-associated protein, in normal and diseased human brain; citation_author=J.I. Busch; citation_author=M. Martinez-Lage; citation_author=E. Ashbridge; citation_author=M. Grossman; citation_author=V.M. Van Deerlin; citation_author=F. Hu; citation_author=V.M. Lee; citation_author=J.Q. Trojanowski; citation_author=A.S. Chen-Plotkin; citation_journal_title=Acta Neuropathol. Commun.; citation_volume=1; citation_firstpage=36; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Neuropathologic diagnostic and nosologic criteria for frontotemporal lobar degeneration: consensus of the Consortium for Frontotemporal Lobar Degeneration; citation_author=N.J. Cairns; citation_author=E.H. Bigio; citation_author=I.R. Mackenzie; citation_author=M. Neumann; citation_author=V.M. Lee; citation_author=K.J. Hatanpaa; citation_author=C.L. White; citation_author=J.A. Schneider; citation_author=L.T. Grinberg; citation_author=G. Halliday; citation_author=Consortium for Frontotemporal Lobar Degeneration; citation_author=et al.; citation_journal_title=Acta Neuropathol.; citation_volume=114; citation_firstpage=5; citation_lastpage=22; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The AD tau core spontaneously self-assembles and recruits full-length tau to filaments; citation_author=Y. Carlomagno; citation_author=S. Manne; citation_author=M. DeTure; citation_author=M. Prudencio; citation_author=Y.J. Zhang; citation_author=R. Hanna Al-Shaikh; citation_author=J.A. Dunmore; citation_author=L.M. Daughrity; citation_author=Y. Song; citation_author=M. Castanedes-Casey; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=34; citation_firstpage=108843; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Plasma glial fibrillary acidic protein is elevated in cognitively normal older adults at risk of Alzheimer’s disease; citation_author=P. Chatterjee; citation_author=S. Pedrini; citation_author=E. Stoops; citation_author=K. Goozee; citation_author=V.L. Villemagne; citation_author=P.R. Asih; citation_author=I.M.W. Verberk; citation_author=P. Dave; citation_author=K. Taddei; citation_author=H.R. Sohrabi; citation_author=et al.; citation_journal_title=Transl. Psychiatry; citation_volume=11; citation_firstpage=27; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=MolProbity: all-atom structure validation for macromolecular crystallography; citation_author=V.B. Chen; citation_author=W.B. Arendall; citation_author=J.J. Headd; citation_author=D.A. Keedy; citation_author=R.M. Immormino; citation_author=G.J. Kapral; citation_author=L.W. Murray; citation_author=J.S. Richardson; citation_author=D.C. Richardson; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=66; citation_firstpage=12; citation_lastpage=21; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=findMySequence: a neural-network-based approach for identification of unknown proteins in X-ray crystallography and cryo-EM; citation_author=G. Chojnowski; citation_author=A.J. Simpkin; citation_author=D.A. Leonardo; citation_author=W. Seifert-Davila; citation_author=D.E. Vivas-Ruiz; citation_author=R.M. Keegan; citation_author=D.J. Rigden; citation_journal_title=IUCrJ; citation_volume=9; citation_firstpage=86; citation_lastpage=97; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=COSMIC2: A Science Gateway for Cryo-Electron Microscopy Structure Determination; citation_inbook_title=Proceedings of the Practice and Experience in Advanced Research Computing 2017 on Sustainability, Success and Impact; citation_author=M.A. Cianfrocco; citation_author=M. Wong-Barnum; citation_author=C. Youn; citation_author=R. Wagner; citation_author=A.E. Leschziner; citation_doi=10.1145/3093338.3093390; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification; citation_author=J. Cox; citation_author=M. Mann; citation_journal_title=Nat. Biotechnol.; citation_volume=26; citation_firstpage=1367; citation_lastpage=1372; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS; citation_author=M. DeJesus-Hernandez; citation_author=I.R. Mackenzie; citation_author=B.F. Boeve; citation_author=A.L. Boxer; citation_author=M. Baker; citation_author=N.J. Rutherford; citation_author=A.M. Nicholson; citation_author=N.A. Finch; citation_author=H. Flynn; citation_author=J. Adamson; citation_author=et al.; citation_journal_title=Neuron; citation_volume=72; citation_firstpage=245; citation_lastpage=256; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Features and development of Coot; citation_author=P. Emsley; citation_author=B. Lohkamp; citation_author=W.G. Scott; citation_author=K. Cowtan; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=66; citation_firstpage=486; citation_lastpage=501; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Structures of filaments from Pick’s disease reveal a novel tau protein fold; citation_author=B. Falcon; citation_author=W. Zhang; citation_author=A.G. Murzin; citation_author=G. Murshudov; citation_author=H.J. Garringer; citation_author=R. Vidal; citation_author=R.A. Crowther; citation_author=B. Ghetti; citation_author=S.H.W. Scheres; citation_author=M. Goedert; citation_journal_title=Nature; citation_volume=561; citation_firstpage=137; citation_lastpage=140; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Novel tau filament fold in chronic traumatic encephalopathy encloses hydrophobic molecules; citation_author=B. Falcon; citation_author=J. Zivanov; citation_author=W. Zhang; citation_author=A.G. Murzin; citation_author=H.J. Garringer; citation_author=R. Vidal; citation_author=R.A. Crowther; citation_author=K.L. Newell; citation_author=B. Ghetti; citation_author=M. Goedert; citation_author=S.H.W. Scheres; citation_journal_title=Nature; citation_volume=568; citation_firstpage=420; citation_lastpage=423; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A role of the frontotemporal lobar degeneration risk factor TMEM106B in myelination; citation_author=T. Feng; citation_author=R.R. Sheng; citation_author=S. Solé-Domènech; citation_author=M. Ullah; citation_author=X. Zhou; citation_author=C.S. Mendoza; citation_author=L.C.M. Enriquez; citation_author=I.I. Katz; citation_author=D.H. Paushter; citation_author=P.M. Sullivan; citation_author=et al.; citation_journal_title=Brain; citation_volume=143; citation_firstpage=2255; citation_lastpage=2271; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Physiological and pathological functions of TMEM106B: a gene associated with brain aging and multiple brain disorders; citation_author=T. Feng; citation_author=A. Lacrampe; citation_author=F. Hu; citation_journal_title=Acta Neuropathol.; citation_volume=141; citation_firstpage=327; citation_lastpage=339; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=TMEM106B regulates progranulin levels and the penetrance of FTLD in GRN mutation carriers; citation_author=N. Finch; citation_author=M.M. Carrasquillo; citation_author=M. Baker; citation_author=N.J. Rutherford; citation_author=G. Coppola; citation_author=M. Dejesus-Hernandez; citation_author=R. Crook; citation_author=T. Hunter; citation_author=R. Ghidoni; citation_author=L. Benussi; citation_author=et al.; citation_journal_title=Neurology; citation_volume=76; citation_firstpage=467; citation_lastpage=474; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Cryo-EM of amyloid fibrils and cellular aggregates; citation_author=A.W. Fitzpatrick; citation_author=H.R. Saibil; citation_journal_title=Curr. Opin. Struct. Biol.; citation_volume=58; citation_firstpage=34; citation_lastpage=42; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Atomic structure and hierarchical assembly of a cross-β amyloid fibril; citation_author=A.W.P. Fitzpatrick; citation_author=G.T. Debelouchina; citation_author=M.J. Bayro; citation_author=D.K. Clare; citation_author=M.A. Caporini; citation_author=V.S. Bajaj; citation_author=C.P. Jaroniec; citation_author=L. Wang; citation_author=V. Ladizhansky; citation_author=S.A. Müller; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=5468; citation_lastpage=5473; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Cryo-EM structures of tau filaments from Alzheimer’s disease; citation_author=A.W.P. Fitzpatrick; citation_author=B. Falcon; citation_author=S. He; citation_author=A.G. Murzin; citation_author=G. Murshudov; citation_author=H.J. Garringer; citation_author=R.A. Crowther; citation_author=B. Ghetti; citation_author=M. Goedert; citation_author=S.H.W. Scheres; citation_journal_title=Nature; citation_volume=547; citation_firstpage=185; citation_lastpage=190; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Prion-like α-synuclein pathology in the brain of infants with Krabbe disease; citation_author=C. Hatton; citation_author=S.S. Ghanem; citation_author=D.J. Koss; citation_author=I.Y. Abdi; citation_author=E. Gibbons; citation_author=R. Guerreiro; citation_author=J. Bras; citation_author=L. Walker; citation_author=E. Gelpi; citation_author=W. Heywood; citation_author=International DLB Genetics Consortium; citation_author=et al.; citation_journal_title=Brain; citation_doi=10.1093/brain/awac002; citation_publication_date=2022; "><meta name="citation_reference" content="citation_title=Helical reconstruction in RELION; citation_author=S. He; citation_author=S.H.W. Scheres; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=198; citation_firstpage=163; citation_lastpage=176; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Bottom-up structural proteomics: cryoEM of protein complexes enriched from the cellular milieu; citation_author=C.M. Ho; citation_author=X. Li; citation_author=M. Lai; citation_author=T.C. Terwilliger; citation_author=J.R. Beck; citation_author=J. Wohlschlegel; citation_author=D.E. Goldberg; citation_author=A.W.P. Fitzpatrick; citation_author=Z.H. Zhou; citation_journal_title=Nat. Methods; citation_volume=17; citation_firstpage=79; citation_lastpage=85; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Thermodynamic analysis of the aggregation propensity of oxidized Alzheimer’s beta-amyloid variants; citation_author=P. Hortschansky; citation_author=T. Christopeit; citation_author=V. Schroeckh; citation_author=M. Fändrich; citation_journal_title=Protein Sci.; citation_volume=14; citation_firstpage=2915; citation_lastpage=2918; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Degradation and Transmission of Tau by Autophagic-Endolysosomal Networks and Potential Therapeutic Targets for Tauopathy; citation_author=S. Jiang; citation_author=K. Bhaskar; citation_journal_title=Front. Mol. Neurosci.; citation_volume=13; citation_firstpage=586731; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Amyloid fibrils in frontotemporal lobar degeneration with TDP-43 inclusions are composed of TMEM106B, rather than TDP-43; citation_author=Y.X. Jiang; citation_author=Q. Cao; citation_author=M.R. Sawaya; citation_author=R. Abskharon; citation_author=P. Ge; citation_author=M. DeTure; citation_author=D.W. Dickson; citation_author=J.Y. Fu; citation_author=R.R. Ogorzalek Loo; citation_author=J.A. Loo; citation_author=et al.; citation_journal_title=Preprint at bioRxiv; citation_doi=10.1101/2022.01.31.478523; citation_publication_date=2022; "><meta name="citation_reference" content="citation_title=Self-propagation of pathogenic protein aggregates in neurodegenerative diseases; citation_author=M. Jucker; citation_author=L.C. Walker; citation_journal_title=Nature; citation_volume=501; citation_firstpage=45; citation_lastpage=51; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Highly accurate protein structure prediction with AlphaFold; citation_author=J. Jumper; citation_author=R. Evans; citation_author=A. Pritzel; citation_author=T. Green; citation_author=M. Figurnov; citation_author=O. Ronneberger; citation_author=K. Tunyasuvunakool; citation_author=R. Bates; citation_author=A. Žídek; citation_author=A. Potapenko; citation_author=et al.; citation_journal_title=Nature; citation_volume=596; citation_firstpage=583; citation_lastpage=589; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Comparison of Common and Disease-Specific Post-translational Modifications of Pathological Tau Associated With a Wide Range of Tauopathies; citation_author=F. Kametani; citation_author=M. Yoshida; citation_author=T. Matsubara; citation_author=S. Murayama; citation_author=Y. Saito; citation_author=I. Kawakami; citation_author=M. Onaya; citation_author=H. Tanaka; citation_author=A. Kakita; citation_author=A.C. Robinson; citation_author=et al.; citation_journal_title=Front. Neurosci.; citation_volume=14; citation_firstpage=581936; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Protein structure prediction and analysis using the Robetta server; citation_author=D.E. Kim; citation_author=D. Chivian; citation_author=D. Baker; citation_journal_title=Nucleic Acids Res.; citation_volume=32; citation_firstpage=W526-W531; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Membrane orientation and subcellular localization of transmembrane protein 106B (TMEM106B), a major risk factor for frontotemporal lobar degeneration; citation_author=C.M. Lang; citation_author=K. Fellerer; citation_author=B.M. Schwenk; citation_author=P.H. Kuhn; citation_author=E. Kremmer; citation_author=D. Edbauer; citation_author=A. Capell; citation_author=C. Haass; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=287; citation_firstpage=19355; citation_lastpage=19365; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Neurodegenerative tauopathies; citation_author=V.M. Lee; citation_author=M. Goedert; citation_author=J.Q. Trojanowski; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=24; citation_firstpage=1121; citation_lastpage=1159; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Procleave: Predicting Protease-specific Substrate Cleavage Sites by Combining Sequence and Structural Information; citation_author=F. Li; citation_author=A. Leier; citation_author=Q. Liu; citation_author=Y. Wang; citation_author=D. Xiang; citation_author=T. Akutsu; citation_author=G.I. Webb; citation_author=A.I. Smith; citation_author=T. Marquez-Lago; citation_author=J. Li; citation_author=J. Song; citation_journal_title=Genomics Proteomics Bioinformatics; citation_volume=18; citation_firstpage=52; citation_lastpage=64; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The FTLD Risk Factor TMEM106B Regulates the Transport of Lysosomes at the Axon Initial Segment of Motoneurons; citation_author=P. Lüningschrör; citation_author=G. Werner; citation_author=S. Stroobants; citation_author=S. Kakuta; citation_author=B. Dombert; citation_author=D. Sinske; citation_author=R. Wanner; citation_author=R. Lüllmann-Rauch; citation_author=B. Wefers; citation_author=W. Wurst; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=30; citation_firstpage=3506; citation_lastpage=3519.e6; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A harmonized classification system for FTLD-TDP pathology; citation_author=I.R. Mackenzie; citation_author=M. Neumann; citation_author=A. Baborie; citation_author=D.M. Sampathu; citation_author=D. Du Plessis; citation_author=E. Jaros; citation_author=R.H. Perry; citation_author=J.Q. Trojanowski; citation_author=D.M. Mann; citation_author=V.M. Lee; citation_journal_title=Acta Neuropathol.; citation_volume=122; citation_firstpage=111; citation_lastpage=113; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Protein Aggregation and Dysfunction of Autophagy-Lysosomal Pathway: A Vicious Cycle in Lysosomal Storage Diseases; citation_author=A. Monaco; citation_author=A. Fraldi; citation_journal_title=Front. Mol. Neurosci.; citation_volume=13; citation_firstpage=37; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=National Institute on Aging-Alzheimer’s Association guidelines for the neuropathologic assessment of Alzheimer’s disease: a practical approach; citation_author=T.J. Montine; citation_author=C.H. Phelps; citation_author=T.G. Beach; citation_author=E.H. Bigio; citation_author=N.J. Cairns; citation_author=D.W. Dickson; citation_author=C. Duyckaerts; citation_author=M.P. Frosch; citation_author=E. Masliah; citation_author=S.S. Mirra; citation_author=National Institute on Aging; citation_author=Alzheimer’s Association; citation_author=et al.; citation_journal_title=Acta Neuropathol.; citation_volume=123; citation_firstpage=1; citation_lastpage=11; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=TMEM106B p.T185S regulates TMEM106B protein levels: implications for frontotemporal dementia; citation_author=A.M. Nicholson; citation_author=N.A. Finch; citation_author=A. Wojtas; citation_author=M.C. Baker; citation_author=R.B. Perkerson; citation_author=M. Castanedes-Casey; citation_author=L. Rousseau; citation_author=L. Benussi; citation_author=G. Binetti; citation_author=R. Ghidoni; citation_author=et al.; citation_journal_title=J.&nbsp;Neurochem.; citation_volume=126; citation_firstpage=781; citation_lastpage=791; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Self-propagating, molecular-level polymorphism in Alzheimer’s beta-amyloid fibrils; citation_author=A.T. Petkova; citation_author=R.D. Leapman; citation_author=Z. Guo; citation_author=W.M. Yau; citation_author=M.P. Mattson; citation_author=R. Tycko; citation_journal_title=Science; citation_volume=307; citation_firstpage=262; citation_lastpage=265; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=UCSF Chimera--a visualization system for exploratory research and analysis; citation_author=E.F. Pettersen; citation_author=T.D. Goddard; citation_author=C.C. Huang; citation_author=G.S. Couch; citation_author=D.M. Greenblatt; citation_author=E.C. Meng; citation_author=T.E. Ferrin; citation_journal_title=J.&nbsp;Comput. Chem.; citation_volume=25; citation_firstpage=1605; citation_lastpage=1612; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=DeepTracer for fast de novo cryo-EM protein structure modeling and special studies on CoV-related complexes; citation_author=J. Pfab; citation_author=N.M. Phan; citation_author=D. Si; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=118; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Loss of Tmem106b leads to cerebellum Purkinje cell death and motor deficits; citation_author=R. Rademakers; citation_author=A.M. Nicholson; citation_author=Y. Ren; citation_author=S. Koga; citation_author=H.P. Nguyen; citation_author=M. Brooks; citation_author=W. Qiao; citation_author=Z.S. Quicksall; citation_author=B. Matchett; citation_author=R.B. Perkerson; citation_author=et al.; citation_journal_title=Brain Pathol.; citation_volume=31; citation_firstpage=e12945; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The MEROPS database of proteolytic enzymes, their substrates and inhibitors in 2017 and a comparison with peptidases in the PANTHER database; citation_author=N.D. Rawlings; citation_author=A.J. Barrett; citation_author=P.D. Thomas; citation_author=X. Huang; citation_author=A. Bateman; citation_author=R.D. Finn; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D624; citation_lastpage=D632; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=DeepEMhancer: a deep learning solution for cryo-EM volume post-processing; citation_author=R. Sanchez-Garcia; citation_author=J. Gomez-Blanco; citation_author=A. Cuervo; citation_author=J.M. Carazo; citation_author=C.O.S. Sorzano; citation_author=J. Vargas; citation_journal_title=Commun Biol.; citation_volume=4; citation_firstpage=874; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Structures of α-synuclein filaments from multiple system atrophy; citation_author=M. Schweighauser; citation_author=Y. Shi; citation_author=A. Tarutani; citation_author=F. Kametani; citation_author=A.G. Murzin; citation_author=B. Ghetti; citation_author=T. Matsubara; citation_author=T. Tomita; citation_author=T. Ando; citation_author=K. Hasegawa; citation_author=et al.; citation_journal_title=Nature; citation_volume=585; citation_firstpage=464; citation_lastpage=469; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain; citation_author=M. Schweighauser; citation_author=D. Arseni; citation_author=M. Huang; citation_author=S. Lövestam; citation_author=Y. Shi; citation_author=Y. Yang; citation_author=W. Zhang; citation_author=A. Kotecha; citation_author=H.J. Garringer; citation_author=R. Vidal; citation_author=et al.; citation_journal_title=Preprint at bioRxiv; citation_doi=10.1101/2021.11.09.467923; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The FTLD risk factor TMEM106B and MAP6 control dendritic trafficking of lysosomes; citation_author=B.M. Schwenk; citation_author=C.M. Lang; citation_author=S. Hogl; citation_author=S. Tahirovic; citation_author=D. Orozco; citation_author=K. Rentzsch; citation_author=S.F. Lichtenthaler; citation_author=C.C. Hoogenraad; citation_author=A. Capell; citation_author=C. Haass; citation_author=D. Edbauer; citation_journal_title=EMBO J.; citation_volume=33; citation_firstpage=450; citation_lastpage=467; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Structure-based classification of tauopathies; citation_author=Y. Shi; citation_author=W. Zhang; citation_author=Y. Yang; citation_author=A.G. Murzin; citation_author=B. Falcon; citation_author=A. Kotecha; citation_author=M. van Beers; citation_author=A. Tarutani; citation_author=F. Kametani; citation_author=H.J. Garringer; citation_author=et al.; citation_journal_title=Nature; citation_volume=598; citation_firstpage=359; citation_lastpage=363; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Thermodynamic properties underlying the α-helix-to-β-sheet transition, aggregation, and amyloidogenesis of polylysine as probed by calorimetry, densimetry, and ultrasound velocimetry; citation_author=V. Smirnovas; citation_author=R. Winter; citation_author=T. Funck; citation_author=W. Dzwolak; citation_journal_title=J.&nbsp;Phys. Chem. B; citation_volume=109; citation_firstpage=19043; citation_lastpage=19045; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=The CamSol method of rational design of protein mutants with enhanced solubility; citation_author=P. Sormanni; citation_author=F.A. Aprile; citation_author=M. Vendruscolo; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=427; citation_firstpage=478; citation_lastpage=490; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Lysosome size, motility and stress response regulated by fronto-temporal dementia modifier TMEM106B; citation_author=M. Stagi; citation_author=Z.A. Klein; citation_author=T.J. Gould; citation_author=J. Bewersdorf; citation_author=S.M. Strittmatter; citation_journal_title=Mol. Cell. Neurosci.; citation_volume=61; citation_firstpage=226; citation_lastpage=240; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Aged Tmem106b knockout mice display gait deficits in coincidence with Purkinje cell loss and only limited signs of non-motor dysfunction; citation_author=S. Stroobants; citation_author=R. D’Hooge; citation_author=M. Damme; citation_journal_title=Brain Pathol.; citation_volume=31; citation_firstpage=223; citation_lastpage=238; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Modulation of Inflammation by Extracellular Granzyme A; citation_author=K.R. van Daalen; citation_author=J.F. Reijneveld; citation_author=N. Bovenschen; citation_journal_title=Front. Immunol.; citation_volume=11; citation_firstpage=931; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Common variants at 7p21 are associated with frontotemporal lobar degeneration with TDP-43 inclusions; citation_author=V.M. Van Deerlin; citation_author=P.M. Sleiman; citation_author=M. Martinez-Lage; citation_author=A. Chen-Plotkin; citation_author=L.S. Wang; citation_author=N.R. Graff-Radford; citation_author=D.W. Dickson; citation_author=R. Rademakers; citation_author=B.F. Boeve; citation_author=M. Grossman; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=42; citation_firstpage=234; citation_lastpage=239; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=TMEM106B is associated with frontotemporal lobar degeneration in a clinically diagnosed patient cohort; citation_author=J. van der Zee; citation_author=T. Van Langenhove; citation_author=G. Kleinberger; citation_author=K. Sleegers; citation_author=S. Engelborghs; citation_author=R. Vandenberghe; citation_author=P. Santens; citation_author=M. Van den Broeck; citation_author=G. Joris; citation_author=J. Brys; citation_author=et al.; citation_journal_title=Brain; citation_volume=134; citation_firstpage=808; citation_lastpage=815; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Improved protein structure prediction using predicted interresidue orientations; citation_author=J. Yang; citation_author=I. Anishchenko; citation_author=H. Park; citation_author=Z. Peng; citation_author=S. Ovchinnikov; citation_author=D. Baker; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=117; citation_firstpage=1496; citation_lastpage=1503; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Gctf: Real-time CTF determination and correction; citation_author=K. Zhang; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=193; citation_firstpage=1; citation_lastpage=12; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Heparin-induced tau filaments are polymorphic and differ from those in Alzheimer’s and Pick’s diseases; citation_author=W. Zhang; citation_author=B. Falcon; citation_author=A.G. Murzin; citation_author=J. Fan; citation_author=R.A. Crowther; citation_author=M. Goedert; citation_author=S.H. Scheres; citation_journal_title=eLife; citation_volume=8; citation_firstpage=e43584; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Novel tau filament fold in corticobasal degeneration; citation_author=W. Zhang; citation_author=A. Tarutani; citation_author=K.L. Newell; citation_author=A.G. Murzin; citation_author=T. Matsubara; citation_author=B. Falcon; citation_author=R. Vidal; citation_author=H.J. Garringer; citation_author=Y. Shi; citation_author=T. Ikeuchi; citation_author=et al.; citation_journal_title=Nature; citation_volume=580; citation_firstpage=283; citation_lastpage=287; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy; citation_author=S.Q. Zheng; citation_author=E. Palovcak; citation_author=J.-P. Armache; citation_author=K.A. Verba; citation_author=Y. Cheng; citation_author=D.A. Agard; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=331; citation_lastpage=332; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Loss of Tmem106b exacerbates FTLD pathologies and causes motor deficits in progranulin-deficient mice; citation_author=X. Zhou; citation_author=M. Brooks; citation_author=P. Jiang; citation_author=S. Koga; citation_author=A.R. Zuberi; citation_author=M.C. Baker; citation_author=T.M. Parsons; citation_author=M. Castanedes-Casey; citation_author=V. Phillips; citation_author=A.L. Librero; citation_author=et al.; citation_journal_title=EMBO Rep.; citation_volume=21; citation_firstpage=e50197; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=New tools for automated high-resolution cryo-EM structure determination in RELION-3; citation_author=J. Zivanov; citation_author=T. Nakane; citation_author=B.O. Forsberg; citation_author=D. Kimanius; citation_author=W.J. Hagen; citation_author=E. Lindahl; citation_author=S.H. Scheres; citation_journal_title=eLife; citation_volume=7; citation_firstpage=e42166; citation_publication_date=2018; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Misfolding and aggregation of disease-specific proteins, resulting in the formation of filamentous cellular inclusions, is a hallmark of neurodegenerative disease with characteristic filament structures, or conformers, defining each proteinopathy. Here we show that a previously unsolved amyloid fibril composed of a 135 amino acid C-terminal fragment of TMEM106B is a common finding in distinct human neurodegenerative diseases, including cases characterized by abnormal aggregation of TDP-43, tau, or α-synuclein protein. A combination of cryoelectron microscopy and mass spectrometry was used to solve the structures of TMEM106B fibrils at a resolution of 2.7&nbsp;Å from postmortem human brain tissue afflicted with frontotemporal lobar degeneration with TDP-43 pathology (FTLD-TDP, n&nbsp;= 8), progressive supranuclear palsy (PSP, n&nbsp;= 2), or dementia with Lewy bodies (DLB, n&nbsp;= 1). The commonality of abundant amyloid fibrils composed of TMEM106B, a lysosomal/endosomal protein, to a broad range of debilitating human disorders indicates a shared fibrillization pathway that may initiate or accelerate neurodegeneration.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=5927&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(22)00259-8&amp;be=3357&amp;fe=5746&amp;dc=5674&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749593096782,%22n%22:0,%22f%22:2843,%22dn%22:2844,%22dne%22:2844,%22c%22:2844,%22s%22:2844,%22ce%22:2844,%22rq%22:2844,%22rp%22:3341,%22rpe%22:3932,%22dl%22:3347,%22di%22:4138,%22ds%22:5674,%22de%22:5684,%22dc%22:5745,%22l%22:5745,%22le%22:5777%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900259-8" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00259-8"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(22)00259-8"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900259-8" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-271183" aria-expanded="false" aria-controls="login-links-271183" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-271183" id="login-links--271183" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900259-8" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900259-8" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900259-8" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-271183" data-opens="#links-dialog-271183" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-271183" aria-modal="true"><button data-closes="#links-dialog-271183" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900259-8" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900259-8" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900259-8" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2022.02.026">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(22)00259-8" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0009-8"><span>Volume 185</span>, <span>Issue 8</span></a></span><span class="meta-panel__pages"><span>p1346-1355.e15</span></span><span class="meta-panel__onlineDate">April 14, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=8&amp;pii=S0092-8674%2821%29X0009-8" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Andrew</span> <span property="familyName">Chang</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew</span> <span property="familyName">Chang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chang/Andrew" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">14</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Xinyu</span> <span property="familyName">Xiang</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xinyu</span> <span property="familyName">Xiang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Structural Biology, Stanford University School of Medicine, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Xiang/Xinyu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">14</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Jing</span> <span property="familyName">Wang</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jing</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Jing" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">14</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Carolyn</span> <span property="familyName">Lee</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Carolyn</span> <span property="familyName">Lee</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology &amp; Immunology, Columbia University, New York, NY 10032, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lee/Carolyn" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Tamta</span> <span property="familyName">Arakhamia</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tamta</span> <span property="familyName">Arakhamia</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Arakhamia/Tamta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Marija</span> <span property="familyName">Simjanoska</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marija</span> <span property="familyName">Simjanoska</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Simjanoska/Marija" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Chi</span> <span property="familyName">Wang</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Chi</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Chi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Yari</span> <span property="familyName">Carlomagno</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yari</span> <span property="familyName">Carlomagno</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Carlomagno/Yari" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Guoan</span> <span property="familyName">Zhang</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Guoan</span> <span property="familyName">Zhang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Proteomics and Metabolomics Core Facility, Weill Cornell Medicine, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Guoan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Shikhar</span> <span property="familyName">Dhingra</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shikhar</span> <span property="familyName">Dhingra</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Dhingra/Shikhar" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Manon</span> <span property="familyName">Thierry</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Manon</span> <span property="familyName">Thierry</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Centre for Cognitive Neurology, Department of Neurology, New York University School of Medicine, New York, NY 10016, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Thierry/Manon" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Jolien</span> <span property="familyName">Perneel</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jolien</span> <span property="familyName">Perneel</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Applied and Translational Neurogenomics, VIB Center for Molecular Neurology, VIB, Antwerp, Belgium</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biomedical Sciences, University of Antwerp, Antwerp, Belgium</span></div></div><div class="core-author-link"><a href="/authored-by/Perneel/Jolien" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Bavo</span> <span property="familyName">Heeman</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bavo</span> <span property="familyName">Heeman</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Applied and Translational Neurogenomics, VIB Center for Molecular Neurology, VIB, Antwerp, Belgium</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biomedical Sciences, University of Antwerp, Antwerp, Belgium</span></div></div><div class="core-author-link"><a href="/authored-by/Heeman/Bavo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Lauren M.</span> <span property="familyName">Forgrave</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lauren M.</span> <span property="familyName">Forgrave</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, BC V6T 2B5, Canada</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, St. Paul’s Hospital, Providence Health Care, Vancouver, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Forgrave/Lauren+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Michael</span> <span property="familyName">DeTure</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael</span> <span property="familyName">DeTure</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA</span></div></div><div class="core-author-link"><a href="/authored-by/DeTure/Michael" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Mari L.</span> <span property="familyName">DeMarco</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mari L.</span> <span property="familyName">DeMarco</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, BC V6T 2B5, Canada</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, St. Paul’s Hospital, Providence Health Care, Vancouver, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/DeMarco/Mari+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Casey N.</span> <span property="familyName">Cook</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Casey N.</span> <span property="familyName">Cook</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Cook/Casey+N" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Rosa</span> <span property="familyName">Rademakers</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Rosa</span> <span property="familyName">Rademakers</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Applied and Translational Neurogenomics, VIB Center for Molecular Neurology, VIB, Antwerp, Belgium</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biomedical Sciences, University of Antwerp, Antwerp, Belgium</span></div></div><div class="core-author-link"><a href="/authored-by/Rademakers/Rosa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Dennis W.</span> <span property="familyName">Dickson</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Dennis W.</span> <span property="familyName">Dickson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Dickson/Dennis+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Leonard</span> <span property="familyName">Petrucelli</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Leonard</span> <span property="familyName">Petrucelli</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Petrucelli/Leonard" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Michael H.B.</span> <span property="familyName">Stowell</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael H.B.</span> <span property="familyName">Stowell</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:stowellm@colorado.edu" property="email" title="Send email to Michael H.B. Stowell">stowellm@colorado.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular, Cellular and Developmental Biology, University of Colorado Boulder, Boulder, CO 80309, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Stowell/Michael+H+B" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup> <a id="eadd0010" href="mailto:stowellm@colorado.edu" property="email" title="Send email to Michael H.B. Stowell">stowellm@colorado.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Ian R.A.</span> <span property="familyName">Mackenzie</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ian R.A.</span> <span property="familyName">Mackenzie</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:ian.mackenzie@vch.ca" property="email" title="Send email to Ian R.A. Mackenzie">ian.mackenzie@vch.ca</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, BC V6T 2B5, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Mackenzie/Ian+R+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup> <a id="eadd0015" href="mailto:ian.mackenzie@vch.ca" property="email" title="Send email to Ian R.A. Mackenzie">ian.mackenzie@vch.ca</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Anthony W.P.</span> <span property="familyName">Fitzpatrick</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anthony W.P.</span> <span property="familyName">Fitzpatrick</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:anthony.fitzpatrick@columbia.edu" property="email" title="Send email to Anthony W.P. Fitzpatrick">anthony.fitzpatrick@columbia.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Fitzpatrick/Anthony+W+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">15</a></sup> <a id="eadd0020" href="mailto:anthony.fitzpatrick@columbia.edu" property="email" title="Send email to Anthony W.P. Fitzpatrick">anthony.fitzpatrick@columbia.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Taub Institute for Research on Alzheimer’s Disease and the Aging Brain, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Structural Biology, Stanford University School of Medicine, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Microbiology &amp; Immunology, Columbia University, New York, NY 10032, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Proteomics and Metabolomics Core Facility, Weill Cornell Medicine, New York, NY 10021, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Centre for Cognitive Neurology, Department of Neurology, New York University School of Medicine, New York, NY 10016, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Applied and Translational Neurogenomics, VIB Center for Molecular Neurology, VIB, Antwerp, Belgium</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Department of Biomedical Sciences, University of Antwerp, Antwerp, Belgium</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, BC V6T 2B5, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Department of Pathology and Laboratory Medicine, St. Paul’s Hospital, Providence Health Care, Vancouver, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">Department of Molecular, Cellular and Developmental Biology, University of Colorado Boulder, Boulder, CO 80309, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received January 25, 2022</span>; <span>Revised February 14, 2022</span>; <span>Accepted February 23, 2022</span>; <span>Published online March 4, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2022.02.026" property="sameAs" target="_blank">10.1016/j.cell.2022.02.026 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867422002598" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" target="_blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2022.02.026" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0009-8"><img src="/cms/asset/22cfc590-b36f-4b13-8bb4-d77ad75db589/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 8"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2822%2900259-8" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (2)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900259-8" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.02.026">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2822%2900259-8" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2022.02.026&amp;pii=S0092-8674%2822%2900259-8" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900259-8" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.02.026">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2022.02.026" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867422002598&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867422002598&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=8&amp;pii=S0092-8674%2821%29X0009-8" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00265-3" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(22)00260-4" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (2)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Cryo-EM structures of brain-derived TMEM106B fibrils from neurodegenerative diseases</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Endolysosomal membrane protein TMEM106B C-terminal fragment forms amyloid fibrils</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">TMEM106B fibrillization is widespread among diverse neurodegenerative proteinopathies</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Identification of fibrillization pathway potentially implicated in neurodegeneration</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Misfolding and aggregation of disease-specific proteins, resulting in the formation of filamentous cellular inclusions, is a hallmark of neurodegenerative disease with characteristic filament structures, or conformers, defining each proteinopathy. Here we show that a previously unsolved amyloid fibril composed of a 135 amino acid C-terminal fragment of TMEM106B is a common finding in distinct human neurodegenerative diseases, including cases characterized by abnormal aggregation of TDP-43, tau, or α-synuclein protein. A combination of cryoelectron microscopy and mass spectrometry was used to solve the structures of TMEM106B fibrils at a resolution of 2.7&nbsp;Å from postmortem human brain tissue afflicted with frontotemporal lobar degeneration with TDP-43 pathology (FTLD-TDP, n&nbsp;= 8), progressive supranuclear palsy (PSP, n&nbsp;= 2), or dementia with Lewy bodies (DLB, n&nbsp;= 1). The commonality of abundant amyloid fibrils composed of TMEM106B, a lysosomal/endosomal protein, to a broad range of debilitating human disorders indicates a shared fibrillization pathway that may initiate or accelerate neurodegeneration.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/c41f6bcf-3372-427b-9384-a3d0b1cfc9f1/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2022.02.026/asset/d72e4e8c-c3d7-47e9-b554-6a10ed08462b/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/c41f6bcf-3372-427b-9384-a3d0b1cfc9f1/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;cryo-EM&quot;" alt="Search on this keyword" rel="nofollow">cryo-EM</a></li><li><a href="/action/doSearch?AllField=&quot;neurodegeneration&quot;" alt="Search on this keyword" rel="nofollow">neurodegeneration</a></li><li><a href="/action/doSearch?AllField=&quot;TMEM106B&quot;" alt="Search on this keyword" rel="nofollow">TMEM106B</a></li><li><a href="/action/doSearch?AllField=&quot;FTLD-TDP&quot;" alt="Search on this keyword" rel="nofollow">FTLD-TDP</a></li><li><a href="/action/doSearch?AllField=&quot;DLB&quot;" alt="Search on this keyword" rel="nofollow">DLB</a></li><li><a href="/action/doSearch?AllField=&quot;PSP&quot;" alt="Search on this keyword" rel="nofollow">PSP</a></li><li><a href="/action/doSearch?AllField=&quot;amyloid+fibrils&quot;" alt="Search on this keyword" rel="nofollow">amyloid fibrils</a></li><li><a href="/action/doSearch?AllField=&quot;proteolysis&quot;" alt="Search on this keyword" rel="nofollow">proteolysis</a></li><li><a href="/action/doSearch?AllField=&quot;lysosome&quot;" alt="Search on this keyword" rel="nofollow">lysosome</a></li><li><a href="/action/doSearch?AllField=&quot;endosome&quot;" alt="Search on this keyword" rel="nofollow">endosome</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">Neurodegenerative proteinopathies are characterized by the deposition of filamentous protein aggregates in neurons and/or glia (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Jucker and Walker, 2013</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Jucker, M. ∙ Walker, L.C.</div><div><strong>Self-propagation of pathogenic protein aggregates in neurodegenerative diseases</strong></div><div><em>Nature.</em> 2013; <strong>501</strong>:45-51</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.JuckerL.C.WalkerSelf-propagation+of+pathogenic+protein+aggregates+in+neurodegenerative+diseasesNature50120134551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The transactive response DNA-binding protein-43 (TDP-43), the microtubule-associated phosphoprotein tau, and α-synuclein protein can each misfold and accrue intracellularly into tangled filamentous inclusions manifesting in neurodegenerative diseases known collectively as TDP-43 proteinopathies, tauopathies, and synucleinopathies, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Lee et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Lee, V.M. ∙ Goedert, M. ∙ Trojanowski, J.Q.</div><div><strong>Neurodegenerative tauopathies</strong></div><div><em>Annu. Rev. Neurosci.</em> 2001; <strong>24</strong>:1121-1159</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.M.LeeM.GoedertJ.Q.TrojanowskiNeurodegenerative+tauopathiesAnnu.+Rev.+Neurosci.24200111211159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The structures of TDP-43 and tau filaments in&nbsp;frontotemporal lobar degeneration (FTLD) (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Arseni et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Arseni, D. ∙ Hasegawa, M. ∙ Murzin, A.G. ...</div><div><strong>Structure of pathological TDP-43 filaments from ALS with FTLD</strong></div><div><em>Nature.</em> 2022; <strong>601</strong>:139-143</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ArseniM.HasegawaA.G.MurzinF.KametaniM.AraiM.YoshidaB.Ryskeldi-FalconStructure+of+pathological+TDP-43+filaments+from+ALS+with+FTLDNature6012022139143" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Shi et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Shi, Y. ∙ Zhang, W. ∙ Yang, Y. ...</div><div><strong>Structure-based classification of tauopathies</strong></div><div><em>Nature.</em> 2021; <strong>598</strong>:359-363</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ShiW.ZhangY.YangA.G.MurzinB.FalconA.KotechaM.van+BeersA.TarutaniF.KametaniH.J.GarringerStructure-based+classification+of+tauopathiesNature5982021359363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Zhang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Zhang, W. ∙ Tarutani, A. ∙ Newell, K.L. ...</div><div><strong>Novel tau filament fold in corticobasal degeneration</strong></div><div><em>Nature.</em> 2020; <strong>580</strong>:283-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.ZhangA.TarutaniK.L.NewellA.G.MurzinT.MatsubaraB.FalconR.VidalH.J.GarringerY.ShiT.IkeuchiNovel+tau+filament+fold+in+corticobasal+degenerationNature5802020283287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and α-synuclein fibrils in multiple system atrophy (MSA) (<span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Schweighauser et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Schweighauser, M. ∙ Shi, Y. ∙ Tarutani, A. ...</div><div><strong>Structures of α-synuclein filaments from multiple system atrophy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:464-469</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserY.ShiA.TarutaniF.KametaniA.G.MurzinB.GhettiT.MatsubaraT.TomitaT.AndoK.HasegawaStructures+of+%CE%B1-synuclein+filaments+from+multiple+system+atrophyNature5852020464469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) have recently been reported. What has emerged from these studies is that each disease has a homotypic molecular fold, or conformer, characteristic of the underlying neuropathology (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Falcon et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Falcon, B. ∙ Zhang, W. ∙ Murzin, A.G. ...</div><div><strong>Structures of filaments from Pick’s disease reveal a novel tau protein fold</strong></div><div><em>Nature.</em> 2018; <strong>561</strong>:137-140</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.FalconW.ZhangA.G.MurzinG.MurshudovH.J.GarringerR.VidalR.A.CrowtherB.GhettiS.H.W.ScheresM.GoedertStructures+of+filaments+from+Pick%E2%80%99s+disease+reveal+a+novel+tau+protein+foldNature5612018137140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" aria-label="Reference 2019" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">2019</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Falcon, B. ∙ Zivanov, J. ∙ Zhang, W. ...</div><div><strong>Novel tau filament fold in chronic traumatic encephalopathy encloses hydrophobic molecules</strong></div><div><em>Nature.</em> 2019; <strong>568</strong>:420-423</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.FalconJ.ZivanovW.ZhangA.G.MurzinH.J.GarringerR.VidalR.A.CrowtherK.L.NewellB.GhettiM.GoedertS.H.W.ScheresNovel+tau+filament+fold+in+chronic+traumatic+encephalopathy+encloses+hydrophobic+moleculesNature5682019420423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Fitzpatrick et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Fitzpatrick, A.W.P. ∙ Falcon, B. ∙ He, S. ...</div><div><strong>Cryo-EM structures of tau filaments from Alzheimer’s disease</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:185-190</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickB.FalconS.HeA.G.MurzinG.MurshudovH.J.GarringerR.A.CrowtherB.GhettiM.GoedertS.H.W.ScheresCryo-EM+structures+of+tau+filaments+from+Alzheimer%E2%80%99s+diseaseNature5472017185190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Zhang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Zhang, W. ∙ Tarutani, A. ∙ Newell, K.L. ...</div><div><strong>Novel tau filament fold in corticobasal degeneration</strong></div><div><em>Nature.</em> 2020; <strong>580</strong>:283-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.ZhangA.TarutaniK.L.NewellA.G.MurzinT.MatsubaraB.FalconR.VidalH.J.GarringerY.ShiT.IkeuchiNovel+tau+filament+fold+in+corticobasal+degenerationNature5802020283287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The evidence for the “one conformer <i>per</i> disease” paradigm is growing, with the presence of unknown buried cofactors (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Falcon et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Falcon, B. ∙ Zivanov, J. ∙ Zhang, W. ...</div><div><strong>Novel tau filament fold in chronic traumatic encephalopathy encloses hydrophobic molecules</strong></div><div><em>Nature.</em> 2019; <strong>568</strong>:420-423</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.FalconJ.ZivanovW.ZhangA.G.MurzinH.J.GarringerR.VidalR.A.CrowtherK.L.NewellB.GhettiM.GoedertS.H.W.ScheresNovel+tau+filament+fold+in+chronic+traumatic+encephalopathy+encloses+hydrophobic+moleculesNature5682019420423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-3" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-3" aria-controls="sref64-3">Zhang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref64-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Zhang, W. ∙ Tarutani, A. ∙ Newell, K.L. ...</div><div><strong>Novel tau filament fold in corticobasal degeneration</strong></div><div><em>Nature.</em> 2020; <strong>580</strong>:283-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.ZhangA.TarutaniK.L.NewellA.G.MurzinT.MatsubaraB.FalconR.VidalH.J.GarringerY.ShiT.IkeuchiNovel+tau+filament+fold+in+corticobasal+degenerationNature5802020283287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and diverse patterns of posttranslational modification (<span class="dropBlock reference-citations"><a id="body-ref-sref2-4" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-4" aria-controls="sref2-4">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Kametani et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Kametani, F. ∙ Yoshida, M. ∙ Matsubara, T. ...</div><div><strong>Comparison of Common and Disease-Specific Post-translational Modifications of Pathological Tau Associated With a Wide Range of Tauopathies</strong></div><div><em>Front. Neurosci.</em> 2020; <strong>14</strong>:581936</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KametaniM.YoshidaT.MatsubaraS.MurayamaY.SaitoI.KawakamiM.OnayaH.TanakaA.KakitaA.C.RobinsonComparison+of+Common+and+Disease-Specific+Post-translational+Modifications+of+Pathological+Tau+Associated+With+a+Wide+Range+of+TauopathiesFront.+Neurosci.142020581936" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) mediating the structural diversity of fibrillar polymorphs.</div><div id="p0055" role="paragraph">Here, using a combination of cryoelectron microscopy (cryo-EM) and mass spectrometry (MS), we show that a previously unsolved amyloid fibril found in cases representing a variety of neurodegenerative conditions is composed of a 135 amino acid C-terminal fragment of TMEM106B, a known risk gene for FTLD-TDP and aging (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Feng et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Feng, T. ∙ Lacrampe, A. ∙ Hu, F.</div><div><strong>Physiological and pathological functions of TMEM106B: a gene associated with brain aging and multiple brain disorders</strong></div><div><em>Acta Neuropathol.</em> 2021; <strong>141</strong>:327-339</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.FengA.LacrampeF.HuPhysiological+and+pathological+functions+of+TMEM106B%3A+a+gene+associated+with+brain+aging+and+multiple+brain+disordersActa+Neuropathol.1412021327339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Van Deerlin et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Van Deerlin, V.M. ∙ Sleiman, P.M. ∙ Martinez-Lage, M. ...</div><div><strong>Common variants at 7p21 are associated with frontotemporal lobar degeneration with TDP-43 inclusions</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:234-239</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.M.Van+DeerlinP.M.SleimanM.Martinez-LageA.Chen-PlotkinL.S.WangN.R.Graff-RadfordD.W.DicksonR.RademakersB.F.BoeveM.GrossmanCommon+variants+at+7p21+are+associated+with+frontotemporal+lobar+degeneration+with+TDP-43+inclusionsNat.+Genet.422010234239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The fibrillization of TMEM106B into identical fibrillar structures in a wide range of sporadic or genetic TDP-43 proteinopathies (FTLD-TDP), a tauopathy (PSP), and a synucleinopathy (DLB) points toward a commonality between these diverse neurodegenerative diseases. This suggests that the formation of amyloid fibrils composed of TMEM106B, a lysosomal/endosomal protein, may contribute to pathogenicity via a loss or gain of function.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Structures of TMEM106B amyloid fibrils in diverse neurodegenerative diseases</h3><div id="p0060" role="paragraph">TMEM106B is a 274 amino acid type II transmembrane protein (<a id="crosref0375" href="#fig1">Figure&nbsp;1</a>A) that under physiological conditions spans the lysosomal and endosomal membranes of neurons, glia, endothelial cells, and pericytes (<a id="crosref0380" href="#fig1">Figure&nbsp;1</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Busch et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Busch, J.I. ∙ Martinez-Lage, M. ∙ Ashbridge, E. ...</div><div><strong>Expression of TMEM106B, the frontotemporal lobar degeneration-associated protein, in normal and diseased human brain</strong></div><div><em>Acta Neuropathol. Commun.</em> 2013; <strong>1</strong>:36</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.I.BuschM.Martinez-LageE.AshbridgeM.GrossmanV.M.Van+DeerlinF.HuV.M.LeeJ.Q.TrojanowskiA.S.Chen-PlotkinExpression+of+TMEM106B%2C+the+frontotemporal+lobar+degeneration-associated+protein%2C+in+normal+and+diseased+human+brainActa+Neuropathol.+Commun.1201336" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Lang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lang, C.M. ∙ Fellerer, K. ∙ Schwenk, B.M. ...</div><div><strong>Membrane orientation and subcellular localization of transmembrane protein 106B (TMEM106B), a major risk factor for frontotemporal lobar degeneration</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:19355-19365</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.LangK.FellererB.M.SchwenkP.H.KuhnE.KremmerD.EdbauerA.CapellC.HaassMembrane+orientation+and+subcellular+localization+of+transmembrane+protein+106B+%28TMEM106B%29%2C+a+major+risk+factor+for+frontotemporal+lobar+degenerationJ.%C2%A0Biol.+Chem.28720121935519365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Current knowledge points to a physiological role for TMEM106B in the proper sorting and transport of endosomes and lysosomes in these various cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Feng et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Feng, T. ∙ Sheng, R.R. ∙ Solé-Domènech, S. ...</div><div><strong>A role of the frontotemporal lobar degeneration risk factor TMEM106B in myelination</strong></div><div><em>Brain.</em> 2020; <strong>143</strong>:2255-2271</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.FengR.R.ShengS.Sol%C3%A9-Dom%C3%A8nechM.UllahX.ZhouC.S.MendozaL.C.M.EnriquezI.I.KatzD.H.PaushterP.M.SullivanA+role+of+the+frontotemporal+lobar+degeneration+risk+factor+TMEM106B+in+myelinationBrain143202022552271" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Lang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lang, C.M. ∙ Fellerer, K. ∙ Schwenk, B.M. ...</div><div><strong>Membrane orientation and subcellular localization of transmembrane protein 106B (TMEM106B), a major risk factor for frontotemporal lobar degeneration</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:19355-19365</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.LangK.FellererB.M.SchwenkP.H.KuhnE.KremmerD.EdbauerA.CapellC.HaassMembrane+orientation+and+subcellular+localization+of+transmembrane+protein+106B+%28TMEM106B%29%2C+a+major+risk+factor+for+frontotemporal+lobar+degenerationJ.%C2%A0Biol.+Chem.28720121935519365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">Lüningschrör et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Lüningschrör, P. ∙ Werner, G. ∙ Stroobants, S. ...</div><div><strong>The FTLD Risk Factor TMEM106B Regulates the Transport of Lysosomes at the Axon Initial Segment of Motoneurons</strong></div><div><em>Cell Rep.</em> 2020; <strong>30</strong>:3506-3519.e6</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.L%C3%BCningschr%C3%B6rG.WernerS.StroobantsS.KakutaB.DombertD.SinskeR.WannerR.L%C3%BCllmann-RauchB.WefersW.WurstThe+FTLD+Risk+Factor+TMEM106B+Regulates+the+Transport+of+Lysosomes+at+the+Axon+Initial+Segment+of+MotoneuronsCell+Rep.30202035063519.e6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Rademakers et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Rademakers, R. ∙ Nicholson, A.M. ∙ Ren, Y. ...</div><div><strong>Loss of Tmem106b leads to cerebellum Purkinje cell death and motor deficits</strong></div><div><em>Brain Pathol.</em> 2021; <strong>31</strong>:e12945</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.RademakersA.M.NicholsonY.RenS.KogaH.P.NguyenM.BrooksW.QiaoZ.S.QuicksallB.MatchettR.B.PerkersonLoss+of+Tmem106b+leads+to+cerebellum+Purkinje+cell+death+and+motor+deficitsBrain+Pathol.312021e12945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Schwenk et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Schwenk, B.M. ∙ Lang, C.M. ∙ Hogl, S. ...</div><div><strong>The FTLD risk factor TMEM106B and MAP6 control dendritic trafficking of lysosomes</strong></div><div><em>EMBO J.</em> 2014; <strong>33</strong>:450-467</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.M.SchwenkC.M.LangS.HoglS.TahirovicD.OrozcoK.RentzschS.F.LichtenthalerC.C.HoogenraadA.CapellC.HaassD.EdbauerThe+FTLD+risk+factor+TMEM106B+and+MAP6+control+dendritic+trafficking+of+lysosomesEMBO+J.332014450467" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Stroobants et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Stroobants, S. ∙ D’Hooge, R. ∙ Damme, M.</div><div><strong>Aged Tmem106b knockout mice display gait deficits in coincidence with Purkinje cell loss and only limited signs of non-motor dysfunction</strong></div><div><em>Brain Pathol.</em> 2021; <strong>31</strong>:223-238</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.StroobantsR.D%E2%80%99HoogeM.DammeAged+Tmem106b+knockout+mice+display+gait+deficits+in+coincidence+with+Purkinje+cell+loss+and+only+limited+signs+of+non-motor+dysfunctionBrain+Pathol.312021223238" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Zhou et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Zhou, X. ∙ Brooks, M. ∙ Jiang, P. ...</div><div><strong>Loss of Tmem106b exacerbates FTLD pathologies and causes motor deficits in progranulin-deficient mice</strong></div><div><em>EMBO Rep.</em> 2020; <strong>21</strong>:e50197</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=X.ZhouM.BrooksP.JiangS.KogaA.R.ZuberiM.C.BakerT.M.ParsonsM.Castanedes-CaseyV.PhillipsA.L.LibreroLoss+of+Tmem106b+exacerbates+FTLD+pathologies+and+causes+motor+deficits+in+progranulin-deficient+miceEMBO+Rep.212020e50197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). TMEM106B consists of a C-terminal domain that projects in the lysosomal lumen, a single-pass transmembrane domain, and an N-terminal domain that extends into the cytosol (<a id="crosref0430" href="#fig1">Figure&nbsp;1</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref36-3" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-3" aria-controls="sref36-3">Lang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref36-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lang, C.M. ∙ Fellerer, K. ∙ Schwenk, B.M. ...</div><div><strong>Membrane orientation and subcellular localization of transmembrane protein 106B (TMEM106B), a major risk factor for frontotemporal lobar degeneration</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:19355-19365</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.LangK.FellererB.M.SchwenkP.H.KuhnE.KremmerD.EdbauerA.CapellC.HaassMembrane+orientation+and+subcellular+localization+of+transmembrane+protein+106B+%28TMEM106B%29%2C+a+major+risk+factor+for+frontotemporal+lobar+degenerationJ.%C2%A0Biol.+Chem.28720121935519365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/85748ff9-f9bc-438f-bfac-6b99d43dc2f5/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2022.02.026/asset/59f10f77-64f5-44cd-af99-0bc0ff508589/main.assets/gr1.jpg" height="190" width="501" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/85748ff9-f9bc-438f-bfac-6b99d43dc2f5/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Overview of native TMEM106B</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Schematic view of TMEM106B with its N-terminal domain (NTD), transmembrane domain (TM), and C-terminal domain (CTD).</div><div id="fspara0020" role="paragraph">(B) A predicted structure of TMEM106B from ROBETTA (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Kim et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Kim, D.E. ∙ Chivian, D. ∙ Baker, D.</div><div><strong>Protein structure prediction and analysis using the Robetta server</strong></div><div><em>Nucleic Acids Res.</em> 2004; <strong>32</strong>:W526-W531</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.E.KimD.ChivianD.BakerProtein+structure+prediction+and+analysis+using+the+Robetta+serverNucleic+Acids+Res.322004W526-W531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) colored with the same&nbsp;scheme as (A), indicating sites at which the C-terminal domain may get cleaved.</div><div id="fspara0025" role="paragraph">(C) Aggregation propensity mapped onto a predicted structure of TMEM106B(120–254) from AlphaFold (<span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Jumper et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Jumper, J. ∙ Evans, R. ∙ Pritzel, A. ...</div><div><strong>Highly accurate protein structure prediction with AlphaFold</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:583-589</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.JumperR.EvansA.PritzelT.GreenM.FigurnovO.RonnebergerK.TunyasuvunakoolR.BatesA.%C5%BD%C3%ADdekA.PotapenkoHighly+accurate+protein+structure+prediction+with+AlphaFoldNature5962021583589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (positive values in blue indicate soluble regions and negative values in red correspond to aggregation-prone regions).</div></div></div></div></figcaption></figure></div><div id="p0065" role="paragraph">The N terminus, TMEM106B(1–96) (<a id="crosref0440" href="#fig1">Figures 1</a>A and 1B), is involved in lysosomal and endosomal trafficking via interactions with MAP6 and can interact with itself as well as with TMEM106C to form homo- and hetero-multimers at the surface of the lysosome (<span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Schwenk et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Schwenk, B.M. ∙ Lang, C.M. ∙ Hogl, S. ...</div><div><strong>The FTLD risk factor TMEM106B and MAP6 control dendritic trafficking of lysosomes</strong></div><div><em>EMBO J.</em> 2014; <strong>33</strong>:450-467</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.M.SchwenkC.M.LangS.HoglS.TahirovicD.OrozcoK.RentzschS.F.LichtenthalerC.C.HoogenraadA.CapellC.HaassD.EdbauerThe+FTLD+risk+factor+TMEM106B+and+MAP6+control+dendritic+trafficking+of+lysosomesEMBO+J.332014450467" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Stagi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Stagi, M. ∙ Klein, Z.A. ∙ Gould, T.J. ...</div><div><strong>Lysosome size, motility and stress response regulated by fronto-temporal dementia modifier TMEM106B</strong></div><div><em>Mol. Cell. Neurosci.</em> 2014; <strong>61</strong>:226-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.StagiZ.A.KleinT.J.GouldJ.BewersdorfS.M.StrittmatterLysosome+size%2C+motility+and+stress+response+regulated+by+fronto-temporal+dementia+modifier+TMEM106BMol.+Cell.+Neurosci.612014226240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The transmembrane domain, TMEM106B(97–117), is a single-pass α-helix (<a id="crosref0455" href="#fig1">Figures 1</a>A and 1B). Although there is <i>in&nbsp;vitro</i> evidence that the lumenal C terminus, TMEM106B(118–274), may undergo proteolytic cleavage to release a C-terminal fragment (<a id="crosref0460" href="#fig1">Figure&nbsp;1</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Brady et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Brady, O.A. ∙ Zhou, X. ∙ Hu, F.</div><div><strong>Regulated intramembrane proteolysis of the frontotemporal lobar degeneration risk factor, TMEM106B, by signal peptide peptidase-like 2a (SPPL2a)</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2014; <strong>289</strong>:19670-19680</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.A.BradyX.ZhouF.HuRegulated+intramembrane+proteolysis+of+the+frontotemporal+lobar+degeneration+risk+factor%2C+TMEM106B%2C+by+signal+peptide+peptidase-like+2a+%28SPPL2a%29J.%C2%A0Biol.+Chem.28920141967019680" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), this has not been confirmed <i>in&nbsp;vivo</i> under physiological or disease conditions. Importantly, the 135 amino acid C-terminal fragment that we identified in fibrils from FTLD-TDP, PSP, and DLB cases, TMEM106B(120–254), is predicted to have a high β strand content and to be prone to aggregation (<a id="crosref0470" href="#fig1">Figure&nbsp;1</a>C) (<span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Jumper et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Jumper, J. ∙ Evans, R. ∙ Pritzel, A. ...</div><div><strong>Highly accurate protein structure prediction with AlphaFold</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:583-589</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.JumperR.EvansA.PritzelT.GreenM.FigurnovO.RonnebergerK.TunyasuvunakoolR.BatesA.%C5%BD%C3%ADdekA.PotapenkoHighly+accurate+protein+structure+prediction+with+AlphaFoldNature5962021583589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). TMEM106B has many known posttranslational modifications, with a heavily glycosylated C terminus (<span class="dropBlock reference-citations"><a id="body-ref-sref36-4" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-4" aria-controls="sref36-4">Lang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref36-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lang, C.M. ∙ Fellerer, K. ∙ Schwenk, B.M. ...</div><div><strong>Membrane orientation and subcellular localization of transmembrane protein 106B (TMEM106B), a major risk factor for frontotemporal lobar degeneration</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:19355-19365</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.LangK.FellererB.M.SchwenkP.H.KuhnE.KremmerD.EdbauerA.CapellC.HaassMembrane+orientation+and+subcellular+localization+of+transmembrane+protein+106B+%28TMEM106B%29%2C+a+major+risk+factor+for+frontotemporal+lobar+degenerationJ.%C2%A0Biol.+Chem.28720121935519365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we show that a truncated C-terminal fragment, TMEM106B(120–254), aggregates into amyloid fibrils in diverse genetic and sporadic proteinopathies and present experimentally determined atomic models of the fibrils extracted from 11 postmortem human brains.</div><div id="p0070" role="paragraph">Protein filaments were isolated from insoluble protein fractions derived from human postmortem brain tissue representing a number of neurodegenerative disorders. Remarkably, cryo-EM 3D reconstructions of fibrils extracted from five FTLD-TDP type A cases (four with different <i>GRN</i> mutations and one sporadic case), two FTLD-TDP type B cases, one FTLD-TDP type C case, two cases of PSP (a 4R tauopathy), and one case of DLB (<a id="crosref0485" href="#mmc1">Table S1</a>; <a id="crosref0490" href="#figs1">Figures S1</a> and <a id="crosref0495" href="#figs2">S2</a>), found two common fibril subtypes (<a id="crosref0500" href="#fig2">Figures 2</a> and <a id="crosref0505" href="#figs3">S3</a>). In each of these cases, there were wide, 2-fold symmetric, twisted ribbons with a helical pitch of ∼2,100&nbsp;Å, a maximum width of ∼260&nbsp;Å, and a minimum width of ∼120&nbsp;Å, which we designate “doublet” fibrils (<a id="crosref0510" href="#fig2">Figures 2</a> and <a id="crosref0515" href="#figs3">S3</a>A). There were also narrow, twisted rods displaying a regular helical pitch of ∼2,100&nbsp;Å and an approximately uniform width of ∼125&nbsp;Å, which we refer to as “singlet” fibrils (<a id="crosref0520" href="#fig2">Figures 2</a> and <a id="crosref0525" href="#figs3">S3</a>B). A structural variation of the singlet fibril was found in one case (<a id="crosref0530" href="#fig2">Figure&nbsp;2</a>, FTLD-TDP case 7/type B case 2), with a shorter helical pitch of ∼1,250&nbsp;Å (<a id="crosref0535" href="#figs3">Figure&nbsp;S3</a>C). The ratio of singlet-to-doublet fibril populations were found to vary between 1:2 and 2:1 for the majority of cases. All density maps show clear inter-β sheet separation perpendicular to the long axis of the fibrils, with separation of β strands discernible at resolutions better than 4.5&nbsp;Å (<a id="crosref0540" href="#fig2">Figure&nbsp;2</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Fitzpatrick et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Fitzpatrick, A.W.P. ∙ Falcon, B. ∙ He, S. ...</div><div><strong>Cryo-EM structures of tau filaments from Alzheimer’s disease</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:185-190</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickB.FalconS.HeA.G.MurzinG.MurshudovH.J.GarringerR.A.CrowtherB.GhettiM.GoedertS.H.W.ScheresCryo-EM+structures+of+tau+filaments+from+Alzheimer%E2%80%99s+diseaseNature5472017185190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The backbone trace of all cross-sections is clear (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Falcon et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Falcon, B. ∙ Zhang, W. ∙ Murzin, A.G. ...</div><div><strong>Structures of filaments from Pick’s disease reveal a novel tau protein fold</strong></div><div><em>Nature.</em> 2018; <strong>561</strong>:137-140</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.FalconW.ZhangA.G.MurzinG.MurshudovH.J.GarringerR.VidalR.A.CrowtherB.GhettiS.H.W.ScheresM.GoedertStructures+of+filaments+from+Pick%E2%80%99s+disease+reveal+a+novel+tau+protein+foldNature5612018137140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and shows a generic fibrillar fold common to TDP-43 proteinopathies (FTLD-TDP), a tauopathy (PSP), and a synucleinopathy (DLB) (<a id="crosref0555" href="#fig2">Figure&nbsp;2</a>).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/39d025e9-10e9-4005-8fe8-3a448fc30dcd/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2022.02.026/asset/85a89381-932c-4d5c-a9f9-b1d3c7205902/main.assets/gr2.jpg" height="510" width="497" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/39d025e9-10e9-4005-8fe8-3a448fc30dcd/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Cryo-EM reconstructions of TMEM106B fibrils</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0035" role="paragraph">Cross-sections (10 <i>z</i>-slice average) of the TMEM106B(120–254) singlet and doublet fibril unsharpened density maps from eight cases of FTLD-TDP, two cases of PSP, and one case of DLB. See also <a id="crosref1505" href="#figs1">Figures S1</a>–<a id="crosref1510" href="#figs3">S3</a> and <a id="crosref1515" href="#figs6">S6</a> and <a id="crosref1520" href="#mmc1">Tables S2</a> and <a id="crosref1525" href="#mmc1">S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/51fd2fe6-acd0-493d-9e61-12dd169c92d0/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2022.02.026/asset/e007aac5-86b4-4a52-b8d8-1bcfc6701910/main.assets/figs1.jpg" height="535" width="505" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/51fd2fe6-acd0-493d-9e61-12dd169c92d0/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Immunohistochemistry of FTLD-TDP cases used for cryo-EM and mass spectrometry, related to <a id="crosref1565" href="#fig2">Figure&nbsp;2</a> and <a id="crosref1570" href="#sec-4">STAR Methods</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0125" role="paragraph">TDP-43 immunoreactive pathologic findings in frontal cortex of cases used for cryo-EM and mass spectrometry. Cases 1-5 and 9 (Type A case 6) showed changes typical of FTLD-TDP type A, with abundant compact neuronal cytoplasmic inclusions, short dystrophic neurites and occasional lentiform neuronal intranuclear inclusions in the upper cortical laminae. Cases 6 (Type B case 1) and 7 (Type B case 2) had FTLD-TDP type B pathology, characterized by diffuse, granular neuronal cytoplasmic inclusions in all cortical layers. Case 8 (Type C) had long, tortuous dystrophic neurites, diagnostic of FTLD-TDP type C. Scale bar, 20&nbsp;μm.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/d2f514ce-895e-4ee1-8929-24a84a4c2ea3/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2022.02.026/asset/648b9086-8a47-4f08-8011-d04bdd150a40/main.assets/figs2.jpg" height="477" width="659" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/d2f514ce-895e-4ee1-8929-24a84a4c2ea3/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Neuropathological and biochemical analysis of three PSP cases used for cryo-EM, related to <a id="crosref1575" href="#fig2">Figure&nbsp;2</a> and <a id="crosref1580" href="#sec-4">STAR Methods</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0135" role="paragraph">(A) and (B) The pigment-containing neurons of the substantia nigra have globus neurofibrillary tangles (hematoxylin and eosin), and tau immunohistochemistry shows neuronal and glial lesions (pretangles, tufted astrocytes, and oligodendroglial coiled bodies). Scale bar, 20&nbsp;μm. (C) Sarkosyl-insoluble fraction extracted from the caudate/putamen of four different PSP cases as evaluated by immunoblotting for the tau antibodies 12E8 (pS262/356), CP13 (pS202), PHF1 (pS396/404), and E1 (human tau-specific). Tau filaments from (Mayo Clinic, Florida) PSP case #1 were subsequently used for cryo-EM analysis. (D-G) Pathologic findings in frontal cortex of PSP case 2 (D, E) and PSP case 3 (F, G) (both from UBC) had abundant neuronal and glial inclusions, including neurofibrillary tangles and tufted astrocytes, that were immunoreactive for hyperphosphorylated tau. (H, I) show immunohistochemistry of the DLB case used for cryo-EM. Frontal cortex with synuclein and also with thioflavin S fluorescent microscopy. Diffuse cortical LBs and minimal Alzheimer-type plaques are illustrated. (D, F) scale bar, 50&nbsp;μm. (E, G) scale bar, 10&nbsp;μm. (H, I) scale bar, 50&nbsp;μm.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/fcb2312e-9cdd-491a-8b50-a5d11e17f59f/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2022.02.026/asset/b5fbf1d8-a6b4-44b9-b1e9-caa14b238f5b/main.assets/figs3.jpg" height="641" width="659" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/fcb2312e-9cdd-491a-8b50-a5d11e17f59f/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Comparison of the observed fibril types, related to <a id="crosref1585" href="#fig2">Figure&nbsp;2</a> and <a id="crosref1590" href="#sec-4">STAR Methods</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0145" role="paragraph">(A-C) Pitch views of the (A) TMEM106B doublet, (B) singlet, and (C) highly twisted singlet fibrils and their respective cross-sections. (A-C) are on the same scale. (D, E) Pitch views of PSP tau filaments from PSP (D) case 1 and (E) case 3 and their respective cross-sections. (D, E) are on the same scale.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">Density maps of the unknown singlet and doublet fibril subtypes were determined to resolutions of 3.0&nbsp;Å and 2.7&nbsp;Å, respectively (<a id="crosrefs0010" href="#fig2">Figures 2–5</a>, and <a id="crosref0560" href="#figs4">S4</a>; <a id="crosref0565" href="#mmc1">Tables S2</a> and <a id="crosref0570" href="#mmc1">S3</a>). To identify these unknown fibrils, MS-based proteomics of fractionated tissue from FTLD-TDP case 1 (<a id="crosref1715" href="#mmc1">Table S5</a>) was used to identify 600 potential protein candidates. The well-resolved sidechain density of the doublet fibril was mapped to this reduced proteome using cryoID (<span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Ho et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ho, C.M. ∙ Li, X. ∙ Lai, M. ...</div><div><strong>Bottom-up structural proteomics: cryoEM of protein complexes enriched from the cellular milieu</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:79-85</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.HoX.LiM.LaiT.C.TerwilligerJ.R.BeckJ.WohlschlegelD.E.GoldbergA.W.P.FitzpatrickZ.H.ZhouBottom-up+structural+proteomics%3A+cryoEM+of+protein+complexes+enriched+from+the+cellular+milieuNat.+Methods1720207985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and <i>findMySequence</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref13-1" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-1" aria-controls="sref13-1">Chojnowski et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref13-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Chojnowski, G. ∙ Simpkin, A.J. ∙ Leonardo, D.A. ...</div><div><strong><i>findMySequence</i>: a neural-network-based approach for identification of unknown proteins in X-ray crystallography and cryo-EM</strong></div><div><em>IUCrJ.</em> 2021; <strong>9</strong>:86-97</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.ChojnowskiA.J.SimpkinD.A.LeonardoW.Seifert-DavilaD.E.Vivas-RuizR.M.KeeganD.J.RigdenfindMySequence%3A+a+neural-network-based+approach+for+identification+of+unknown+proteins+in+X-ray+crystallography+and+cryo-EMIUCrJ920218697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) software, which unambiguously identified an amino acid sequence corresponding to residues 120–254&nbsp;of TMEM106B (see <a id="crosref0585" href="#sec-4">STAR Methods</a>). Fibrils formed by TMEM106B(120–154) exhibit ultrastructural polymorphism with a common protofilament (<a id="crosref0590" href="#fig4">Figure&nbsp;4</a>A) existing as either a singlet fibril (<a id="crosref0595" href="#fig5">Figure&nbsp;5</a>A and 5C) or in a 2-fold symmetrical juxtaposition as a doublet fibril (<a id="crosref0600" href="#fig3">Figure&nbsp;3</a>A). The inter-protofilament interface of the doublet fibrils is mediated by an apparently non-proteinaceous anionic cofactor that binds to the sidechains of residues K178 and R180 of each protofilament with near-stoichiometric occupancy (<a id="crosref0605" href="#fig3">Figure&nbsp;3</a>B).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/936825cb-6d07-4fa4-ab47-c21da3da49a9/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2022.02.026/asset/dc71aa9f-ece9-4f11-aa0d-15009e77c089/main.assets/gr3.jpg" height="188" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/936825cb-6d07-4fa4-ab47-c21da3da49a9/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Cryo-EM structure of a TMEM106B doublet fibril</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0045" role="paragraph">Cryo-EM density (mesh) and atomic model (sticks) of (A) a TMEM106B doublet fibril and (B) the interface between the two protofilaments in the TMEM106B doublet fibril mediated by a non-proteinaceous, anionic cofactor (purple mesh) that binds to the sidechains of residues K178 and R180 of each protofilament. See also <a id="crosref1530" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref1535" href="#mmc1">Table S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/fa0fc319-5fde-4524-bf71-890892d07910/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2022.02.026/asset/551429da-a659-4e8e-8ece-c99b8d8fa347/main.assets/figs4.jpg" height="546" width="367" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/fa0fc319-5fde-4524-bf71-890892d07910/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Cryo-EM resolution estimates of a combined cryo-EM protofilament map from multiple FTLD-TDP cases, related to <a id="crosref1595" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0155" role="paragraph">Local resolution estimates (top) and Fourier Shell Correlation (FSC) curve (bottom) between two independently refined half-maps for the TMEM106B(120-254) protofilament.</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph">The protofilament core consists of residues 120–254, derived from the lumenal topological domain (<a id="crosref0610" href="#fig4">Figures 4</a>A and <a id="crosref0615" href="#figs5">S5</a>A), which is distinct from the stable N-terminally cleaved fragments from the cytoplasmic topological domain that have previously been reported (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Brady et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Brady, O.A. ∙ Zhou, X. ∙ Hu, F.</div><div><strong>Regulated intramembrane proteolysis of the frontotemporal lobar degeneration risk factor, TMEM106B, by signal peptide peptidase-like 2a (SPPL2a)</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2014; <strong>289</strong>:19670-19680</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.A.BradyX.ZhouF.HuRegulated+intramembrane+proteolysis+of+the+frontotemporal+lobar+degeneration+risk+factor%2C+TMEM106B%2C+by+signal+peptide+peptidase-like+2a+%28SPPL2a%29J.%C2%A0Biol.+Chem.28920141967019680" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The 135 residues adopt a C-shaped protofilament with the N terminus embedded in the center, surrounded by a long connecting strand that meanders back on itself to form a closely packed, three-layered C-terminal fold (<a id="crosref0625" href="#fig4">Figures 4</a>A and <a id="crosref0630" href="#figs5">S5</a>A). In total, 19 β strands were identified (β1–β19, <a id="crosref0635" href="#fig5">Figures 5</a>D, <a id="crosref0640" href="#figs5">S5</a>A, and S5C) and found to be linked with either short loops (Q138-R140, T166-A167, G177-A179, G188-P189, V199-P200, E206-S208, C214-T215, K220-N223, Q229-V232, F237-G238, Q245-V251), a gentle turn (N154), or tight single residue turns (G127, Y132, N145, E161, N183, Q194). Diverse sidechain-sidechain interactions facilitate the packing or positioning of β strands (<a id="crosref0645" href="#fig4">Figures 4</a>A, <a id="crosref0650" href="#figs5">S5</a>A, and S5C).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/0ace9aad-344d-4f3c-9a97-967c231cac21/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2022.02.026/asset/d0b18d65-f49e-46ca-b68e-f85dc7ea63c6/main.assets/gr4.jpg" height="475" width="504" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/0ace9aad-344d-4f3c-9a97-967c231cac21/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Cryo-EM structure of a TMEM106B protofilament highlighting the key structural features</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0055" role="paragraph">(A) Cryo-EM density (mesh) and atomic model (sticks) of a TMEM106B protofilament.</div><div id="fspara0060" role="paragraph">(B) A disulfide bond between C214 and C253.</div><div id="fspara0065" role="paragraph">(C) Polymorphic site T185S and glycosylated asparagine at N183.</div><div id="fspara0070" role="paragraph">(D–F) Glycosylated N164 (D), N151 (E), and N145 (F).</div><div id="fspara0075" role="paragraph">See also <a id="crosref1540" href="#figs4">Figures S4</a> and <a id="crosref1545" href="#figs5">S5</a> and <a id="crosref1550" href="#mmc1">Table S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/5b96af82-3320-441f-bd3f-71a0df77f6ce/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2022.02.026/asset/7af3d11c-c3db-4074-bd43-2b96212d9a81/main.assets/gr5.jpg" height="465" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/5b96af82-3320-441f-bd3f-71a0df77f6ce/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Cryo-EM structure of a highly twisted TMEM106B singlet fibril and comparison of singlet fibril molecular polymorphs</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0085" role="paragraph">(A) Cryo-EM density (mesh) and atomic model (sticks) of a highly twisted TMEM106B singlet fibril.</div><div id="fspara0090" role="paragraph">(B) Magnified view of an unknown density bound to K178 in a highly twisted TMEM106B singlet fibril.</div><div id="fspara0095" role="paragraph">(C) Overlay of the atomic models (Cα chain shown) of the low-twist (green) and high-twist (pink) singlet fibrils.</div><div id="fspara0100" role="paragraph">(D) Comparison of secondary structure motifs formed by the low-twist (green) singlet fibril, high-twist (pink) singlet fibril, and native protein predicted by AlphaFold (blue) with experimentally determined post-translational modifications of the highly twisted TMEM106B singlet fibril.</div><div id="fspara0105" role="paragraph">See also <a id="crosref1555" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref1560" href="#mmc1">Table S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/ecc671a6-d9ba-4b82-a326-22b85e89f15b/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2022.02.026/asset/4743ac11-53f4-49aa-b8e7-2d5afdf57613/main.assets/figs5.jpg" height="642" width="499" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/ecc671a6-d9ba-4b82-a326-22b85e89f15b/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Schematic views of polymorphic TMEM106B(120–254) amyloid fibrils and fibril structure-based protease predictions, related to <a id="crosref1600" href="#fig3">Figures 3</a>–<a id="crosref1605" href="#fig5">5</a> and <a id="crosref1610" href="#mmc1">Table S4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0165" role="paragraph">Singlet fibrils with (A) low inter-strand twist and (B) high inter-strand twist are shown. Bold line denotes β strand secondary structure with experimentally detected glycosylations shown in cyan. There is an unknown density (orange) attached to K178 in the highly twisted singlet fibril. (C) Doublet fibril with the amino acids K178 and R180 binding to an anionic cofactor highlighted in red. (D) Enzymes predicted to cleave TMEM106B at site S120.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Starting from the N terminus, strands β1-β3 forms a Z shape, with the tight&nbsp;junction Y132 forming a putative hydrogen bond with N154 that effectively hold neighboring strands β5 and β6 in place. A closely packed β helix is formed by strands β3, β4, and β5. Following a gentle turn at N154, strand β6 is interrupted by E161, which forms a salt bridge with H239 that contributes to the positioning of strand β18. A short β7 strand leads into a long β8 strand that forms a short antiparallel sheet-sheet interface with strand β17. Strands β8-β10 form a large, enclosed hydrophobic cavity, involving residues I176, A179, L181, I184, and I186. The cavity is likely to contain small cofactors as detached densities are observed frequently in sharpened maps. Strand β11 marks the beginning of the C-terminal three-layer fold forming a tight convergence point with strand β16 and loops V230–V232 through the hydrophobic residues I186, L190, M227, and V230. Hydrophobic clustering also locks strands β12, β13 in place with strand β15 nestled in between through small, apolar residues V199, I217, and I219. A bulky, aromatic β14 strand may be stabilized by π-π interactions between Y209 and Y211, and between Y211 and F213, as the density indicates that two rotamers are accessible to Y211. Strand β15 packs against the extreme C terminus through the formation of a disulfide bond between C214 and C253, connecting to strand β16 via the loop K220-N223 (a&nbsp;potential cation-π interaction is formed by K220 and H222). N223 also plays an important role in facilitating the interdigitation of sidechains between strands β16 and β18 by forming a hydrogen bond with E246. The potential strain is relieved by forming an L-shape pin turn with the short β17 strand. A multi-sheet, cross-β interface between strands β16→β18→β1 cements the center of the fold with a short β19 strand closing the protofilament structure at the potential cleavage site G254. The disulfide bond between C214 and C253 (<a id="crosref0655" href="#fig4">Figure&nbsp;4</a>B) stabilizes the fibril structure. Ultrastructural polymorphism of the singlet and doublet fibrils is mediated by a non-proteinaceous density nestled between K178 and R180, connecting the two protofilaments of the doublet fibril (<a id="crosref0660" href="#fig3">Figures 3</a> and <a id="crosref0665" href="#figs5">S5</a>C). In cross-section, this measures approximately 9&nbsp;Å × 4&nbsp;Å and is presumably anionic, given that it is bound to positively charged sidechains (<a id="crosref0670" href="#fig3">Figures 3</a>B and <a id="crosref0675" href="#figs5">S5</a>C).</div></section><section id="sec-2-2"><h3>Structural role of posttranslational modifications in TMEM106B amyloid fibrils</h3><div id="p0090" role="paragraph">Post-translational modifications were preserved for the filament sample and allowed assignment of glycans in the atomic model that match with previous studies of the native protein. Native TMEM106B has six potential N-glycosylation sites all located on the lumenal domain, with the first three on residues N145, N151, and N164 (N1–N3) being non-complex glycosylation and the last three on N182, N183, and N256 (N4–N6) being complex N-glycosylation (<span class="dropBlock reference-citations"><a id="body-ref-sref36-5" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-5" aria-controls="sref36-5">Lang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref36-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lang, C.M. ∙ Fellerer, K. ∙ Schwenk, B.M. ...</div><div><strong>Membrane orientation and subcellular localization of transmembrane protein 106B (TMEM106B), a major risk factor for frontotemporal lobar degeneration</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:19355-19365</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.LangK.FellererB.M.SchwenkP.H.KuhnE.KremmerD.EdbauerA.CapellC.HaassMembrane+orientation+and+subcellular+localization+of+transmembrane+protein+106B+%28TMEM106B%29%2C+a+major+risk+factor+for+frontotemporal+lobar+degenerationJ.%C2%A0Biol.+Chem.28720121935519365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the protofilament structure, we found that these N-glycosylations were largely retained as densities consistent with NAG were identified at residues N183 (<a id="crosref0685" href="#fig4">Figure&nbsp;4</a>C), N164 (<a id="crosref0690" href="#fig4">Figure&nbsp;4</a>D), N151 (<a id="crosref0695" href="#fig4">Figure&nbsp;4</a>E), and N145 (<a id="crosref0700" href="#fig4">Figure&nbsp;4</a>F). All prominent non-proteinaceous densities are shown schematically in <a id="crosref0705" href="#figs5">Figures S5</a>A–S5C.</div><div id="p0095" role="paragraph">The structure of TMEM106B(182–185) is significant as this region has a disease-risk associated polymorphism (T185S, <a id="crosref0710" href="#mmc1">Table S1</a>). It includes a potentially phosphorylated T185, a heavily glycosylated N183 (<a id="crosref0715" href="#fig4">Figures 4</a>A and 4C), and a N182 that also shows proximal, non-proteinaceous density in some maps. The density near T185 shows a putative phosphate density. The phosphorylation of T185 is thought to determine the glycosylation state of N183 (<span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Nicholson et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Nicholson, A.M. ∙ Finch, N.A. ∙ Wojtas, A. ...</div><div><strong>TMEM106B p.T185S regulates TMEM106B protein levels: implications for frontotemporal dementia</strong></div><div><em>J.&nbsp;Neurochem.</em> 2013; <strong>126</strong>:781-791</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.M.NicholsonN.A.FinchA.WojtasM.C.BakerR.B.Perkerson3rdM.Castanedes-CaseyL.RousseauL.BenussiG.BinettiR.GhidoniTMEM106B+p.T185S+regulates+TMEM106B+protein+levels%3A+implications+for+frontotemporal+dementiaJ.%C2%A0Neurochem.1262013781791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which shows a large, strong density connected to the Nδ2 atom corresponding to multiple glycans (<a id="crosref0725" href="#fig4">Figure&nbsp;4</a>C). Interestingly, TMEM106B(120–254) fibrils were observed irrespective of the presence of the risk (T185) or protective (S185) haplotype (<a id="crosref0730" href="#mmc1">Table S1</a>). All TMEM106B(120–254) fibril structures have been built using a threonine at residue 185, as it is not possible to distinguish between a serine and threonine sidechain (van der Waals volume of 73&nbsp;Å<sup>3</sup> and 93&nbsp;Å<sup>3</sup>, respectively) or an average of both sidechains at a resolution of 2.7&nbsp;Å.</div></section><section id="sec-2-3"><h3>Singlet TMEM106B(120–254) amyloid fibrils display molecular polymorphism</h3><div id="p0100" role="paragraph">The protofilament structure (singlet fibril) formed by TMEM106B(120–154) is identical across many cases with a uniform inter-strand twist angle of −0.4° (<a id="crosref0735" href="#figs3">Figure&nbsp;S3</a>B; <a id="crosref0740" href="#mmc1">Tables S2</a> and <a id="crosref0745" href="#mmc1">S3</a>). However, distinct misfolding minima exist with a similar, but structurally distinct, conformer displaying an inter-strand twist angle of −0.67°. The more twisted singlet fibril is also comprised of residues S120 to G254, adopting a C shape similar to the less twisted singlet fibril and involving an embedded N terminus and a three-layered C terminus (<a id="crosref0750" href="#fig5">Figures 5</a>A and <a id="crosref0755" href="#figs5">S5</a>B). However, the C-terminal fold is less kinked and more compact, showing an ordered rectangular arrangement. Notably, in sharp contrast to the outward-pointing corner turn observed in the TMEM106B protofilament (<a id="crosref0760" href="#fig4">Figures 4</a>A, <a id="crosref0765" href="#figs5">S5</a>A, and S5C), in this instance the chain curves inward and forms an L shape pointing directly toward the extreme C terminus (<a id="crosref0770" href="#fig5">Figures 5</a>A and <a id="crosref0775" href="#figs5">S5</a>B). In total, 19 β strands were identified (β1–β19), matching the number of&nbsp;β&nbsp;strands we found in the less twisted singlet fibril. Although residues S120-G254 form both low- and high-twist singlet fibrils, comparison of the secondary structure motifs adopted by TMEM106B(120–254) in each molecular polymorph (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Fitzpatrick and Saibil, 2019</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Fitzpatrick, A.W. ∙ Saibil, H.R.</div><div><strong>Cryo-EM of amyloid fibrils and cellular aggregates</strong></div><div><em>Curr. Opin. Struct. Biol.</em> 2019; <strong>58</strong>:34-42</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.FitzpatrickH.R.SaibilCryo-EM+of+amyloid+fibrils+and+cellular+aggregatesCurr.+Opin.+Struct.+Biol.5820193442" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Petkova et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Petkova, A.T. ∙ Leapman, R.D. ∙ Guo, Z. ...</div><div><strong>Self-propagating, molecular-level polymorphism in Alzheimer’s beta-amyloid fibrils</strong></div><div><em>Science.</em> 2005; <strong>307</strong>:262-265</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.PetkovaR.D.LeapmanZ.GuoW.M.YauM.P.MattsonR.TyckoSelf-propagating%2C+molecular-level+polymorphism+in+Alzheimer%E2%80%99s+beta-amyloid+fibrilsScience3072005262265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) show differences with 21 in 93 residues having a β strand/loop mismatch (<a id="crosref0790" href="#fig5">Figure&nbsp;5</a>D).</div><div id="p0105" role="paragraph">In general, β strands (β1–β6) at the N terminus are identical in terms of amino acid composition and highly similar in spatial arrangements. However, from β7 onward, the more twisted singlet fibril shows significant variation from its less twisted counterpart (<a id="crosref0795" href="#fig5">Figures 5</a>C, <a id="crosref0800" href="#figs5">S5</a>A, and S5B). Strand β8 is shorter, packed against β17 by forming a short, hydrophobic antiparallel sheet-sheet interface. However, this interface is not tightly packed due to the lack of interaction between F237 and Q170. Strand β8 then leads into β9 with a flexible short loop consisting of S172-I176. There is a strong, large density visibly attached to K178 (<a id="crosref0805" href="#fig5">Figures 5</a>B and <a id="crosref0810" href="#figs5">S5</a>B) that could be an anionic cofactor or posttranslational modification (<span class="dropBlock reference-citations"><a id="body-ref-sref2-5" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-5" aria-controls="sref2-5">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Strands β9–β13 organize into a continuous long strand, with β10 marking the beginning of the C-terminal three-layered fold. These five strands (β9–β13) are arranged horizontally in a straight line, connected with short loops (G188-P189, V199-P200) and tight single residue turns (N183, Q194). Strand β12 is packed closely against β16, possibly by hydrophobic clustering with the small hydrophobic residues I195, I224, and L226. Strand β13 continues into β14 with a near 90-degree turn, with A204 at the corner. We hypothesize that a more complex interaction occurs in this aromatic region as the density suggests two rotamers for Y209, which is able to form either a methionine-aromatic interaction with M207 or π-π interaction with Y211. At the C-terminal&nbsp;end of β14, C214 again forms a disulfide bond with C253, the same as observed in the less twisted singlet fibril. This disulfide bond results in a sharp tip that steers into a conserved region consisting of β15–β17 and positions β19 into forming a flattened and smooth strand following β18.</div></section><section id="sec-2-4"><h3>Comparison of TMEM106B in pathological and non-pathological human tissue</h3><div id="p0110" role="paragraph">As the discovery of TMEM106B fibrils is a recent and surprising finding, antibodies that are specific to TMEM106B fibrillar signatures, such as PHF1 (pS396/404) or pS129 available for tau and α-synuclein filaments, respectively, have not yet been generated and validated. Antibodies that recognize fibrillar phospho-epitopes are necessary to unambiguously image TMEM106B fibrils in patient tissue using immunofluorescence and to quantitatively measure the amount of TMEM106B fibrils in insoluble protein fractions.</div><div id="p0115" role="paragraph">Despite the current lack of reagents that specifically bind to an exclusively fibrillar epitope, we performed western blot analysis on fractionated tissue pellets from five neurologically normal individuals aged between 68 and 73 years of age using a commercially available TMEM106B C-terminal antibody (<a id="crosref0820" href="#figs6">Figure&nbsp;S6</a>). For comparison, we also included a 48-year-old FTLD-TDP type A homogenate (case 5; TMEM106B fibrils solved by cryo-EM, <a id="crosref1720" href="#fig2">Figure 2</a> and <a id="crosref1725" href="#mmc1">Table S2</a>) in an adjacent lane and performed separate western blot analysis on fractionated tissue pellets from FTLD-TDP types A (case 4; TMEM106B fibrils solved by cryo-EM, <a id="crosref1730" href="#fig2">Figure 2</a> and <a id="crosref1735" href="#mmc1">Table&nbsp;S2</a>), B, and C along with five α-synucleinopathies (<a id="crosref0825" href="#figs6">Figure&nbsp;S6</a>). To probe for TMEM106B, we used a polyclonal antibody that binds to residues (204–253). As this antibody is not specific to a fibrillar epitope, all TMEM106B species are potentially detected. We observe bands consistent with monomeric TMEM106B(120–254), molecular weight of 15.49&nbsp;kDa, at ∼16&nbsp;kDa (<a id="crosref0830" href="#figs6">Figure&nbsp;S6</a>) and multiple bands ∼25&nbsp;kDa (<a id="crosref0835" href="#figs6">Figure&nbsp;S6</a>) that may correspond to glycosylated TMEM106B(120–254) monomers. Notably, there are many higher molecular weight bands at 110&nbsp;kDa, 135&nbsp;kDa, and 190&nbsp;kDa that are consistent with TMEM106B oligomerization (<span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Jiang et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Jiang, Y.X. ∙ Cao, Q. ∙ Sawaya, M.R. ...</div><div><strong>Amyloid fibrils in frontotemporal lobar degeneration with TDP-43 inclusions are composed of TMEM106B, rather than TDP-43</strong></div><div><em>Preprint at bioRxiv.</em> 2022;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.X.JiangQ.CaoM.R.SawayaR.AbskharonP.GeM.DeTureD.W.DicksonJ.Y.FuR.R.Ogorzalek+LooJ.A.LooAmyloid+fibrils+in+frontotemporal+lobar+degeneration+with+TDP-43+inclusions+are+composed+of+TMEM106B%2C+rather+than+TDP-43Preprint+at+bioRxiv202210.1101%2F2022.01.31.478523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0845" href="#figs6">Figure&nbsp;S6</a>). The general trend we observe is that these high molecular weight species are more abundant in pathological cases compared to non-pathological controls, although there are exceptions (e.g., FTLD-TDP case 4 is a sporadic case that shows a high concentration of TMEM106B fibrils by cryo-EM but not by western blot).</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/72b2e44f-1a9a-4678-ac43-4964e4501b3f/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2022.02.026/asset/3eb60cbc-bba6-4703-a99a-8a8edffbe77a/main.assets/figs6.jpg" height="615" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/72b2e44f-1a9a-4678-ac43-4964e4501b3f/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Western blots of sarkosyl-insoluble fractions from donor brains probed with a TMEM106B polyclonal antibody that binds to residues (204–253), related to <a id="crosref1615" href="#fig2">Figure&nbsp;2</a> and <a id="crosref1620" href="#sec-4">STAR Methods</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">(A) Equally-exposed western blots of sarkosyl-insoluble fractions from pathological and non-pathological tissues. (B) High contrast images of western blots in (A). Type A, Type B, Type C: FTLD-TDP subtypes. DLB: Dementia with Lewy Bodies. NN: Neurologically Normal.</div></div></div></div></figcaption></figure></div><div id="p0120" role="paragraph">Importantly, we and others (<span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Jiang et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Jiang, Y.X. ∙ Cao, Q. ∙ Sawaya, M.R. ...</div><div><strong>Amyloid fibrils in frontotemporal lobar degeneration with TDP-43 inclusions are composed of TMEM106B, rather than TDP-43</strong></div><div><em>Preprint at bioRxiv.</em> 2022;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.X.JiangQ.CaoM.R.SawayaR.AbskharonP.GeM.DeTureD.W.DicksonJ.Y.FuR.R.Ogorzalek+LooJ.A.LooAmyloid+fibrils+in+frontotemporal+lobar+degeneration+with+TDP-43+inclusions+are+composed+of+TMEM106B%2C+rather+than+TDP-43Preprint+at+bioRxiv202210.1101%2F2022.01.31.478523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) contend that western blots do not reliably detect TMEM106B aggregation. An antibody has recently been generated that detects non-fibrillar TMEM106B by binding to residues (239–250), which are buried in the fibril core (<span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Schweighauser et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Schweighauser, M. ∙ Arseni, D. ∙ Huang, M. ...</div><div><strong>Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserD.ArseniM.HuangS.L%C3%B6vestamY.ShiY.YangW.ZhangA.KotechaH.J.GarringerR.VidalAge-Dependent+Formation+of+TMEM106B+Amyloid+Filaments+in+Human+BrainPreprint+at+bioRxiv202110.1101%2F2021.11.09.467923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Such an antibody can only measure accumulation of monomeric TMEM106B in the soluble or insoluble fractions and is therefore not suitable for detecting fibrils.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0125" role="paragraph">Our study has identified previously undetected amyloid fibrils composed of aggregated TMEM106B(120–254) protein fragments in postmortem brain tissue from multiple TDP-43 proteinopathies, a tauopathy, and a synucleinopathy. These observations were both unexpected and surprising. We further note that TMEM106B(120–254) fibrils were not observed in our previous studies of Alzheimer’s disease (<span class="dropBlock reference-citations"><a id="body-ref-sref2-6" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-6" aria-controls="sref2-6">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25-3" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-3" aria-controls="sref25-3">Fitzpatrick et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref25-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Fitzpatrick, A.W.P. ∙ Falcon, B. ∙ He, S. ...</div><div><strong>Cryo-EM structures of tau filaments from Alzheimer’s disease</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:185-190</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickB.FalconS.HeA.G.MurzinG.MurshudovH.J.GarringerR.A.CrowtherB.GhettiM.GoedertS.H.W.ScheresCryo-EM+structures+of+tau+filaments+from+Alzheimer%E2%80%99s+diseaseNature5472017185190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or CBD tau filaments (<span class="dropBlock reference-citations"><a id="body-ref-sref2-7" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-7" aria-controls="sref2-7">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although this suggests that TMEM106B(120–254) fibrils may not be a consistent feature of all tauopathies, our absence of evidence could simply be due to the limited n-number and varying brain regions in our cryo-EM studies of other tauopathies. It has also recently been suggested that variations in tissue fractionation protocols may significantly affect TMEM106B fibril yield (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Schweighauser et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Schweighauser, M. ∙ Arseni, D. ∙ Huang, M. ...</div><div><strong>Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserD.ArseniM.HuangS.L%C3%B6vestamY.ShiY.YangW.ZhangA.KotechaH.J.GarringerR.VidalAge-Dependent+Formation+of+TMEM106B+Amyloid+Filaments+in+Human+BrainPreprint+at+bioRxiv202110.1101%2F2021.11.09.467923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0130" role="paragraph">Notably, the abundance of TMEM106B(120–254) fibrils in two of our PSP cases is very high. We find that in PSP case 1, which also contains the PSP tau fold (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Shi et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Shi, Y. ∙ Zhang, W. ∙ Yang, Y. ...</div><div><strong>Structure-based classification of tauopathies</strong></div><div><em>Nature.</em> 2021; <strong>598</strong>:359-363</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ShiW.ZhangY.YangA.G.MurzinB.FalconA.KotechaM.van+BeersA.TarutaniF.KametaniH.J.GarringerStructure-based+classification+of+tauopathiesNature5982021359363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0885" href="#figs3">Figure&nbsp;S3</a>D; <a id="crosref0890" href="#mmc1">Table S3</a>), TMEM106B(120–254) singlet fibrils are equally abundant as tau filaments. In PSP case 2, TMEM106B(120–254) singlet and doublet fibrils are approximately equal in number, with no PSP tau filaments observed in this sample (<a id="crosref0895" href="#fig2">Figure&nbsp;2</a>; <a id="crosref0900" href="#mmc1">Table S3</a>). This is perhaps not surprising as PSP tau filaments are extremely sparse in the majority of PSP tissue samples we&nbsp;have screened (See <a id="crosref0905" href="#sec-4">STAR Methods</a>). In contrast, PSP case&nbsp;3&nbsp;only contained PSP tau filaments (<a id="crosref0910" href="#figs3">Figure&nbsp;S3</a>E), and TMEM106B(120–254) fibrils were not observed. Recently, two cases with FTLD-TDP type B were shown to contain narrow, highly twisted filaments formed by TDP-43 (cross section ∼40&nbsp;Å × 70&nbsp;Å, helical pitch ∼600&nbsp;Å), although these fibrils were not found in all cortical regions and were not confirmed in other FTLD-TDP subtypes (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Arseni et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Arseni, D. ∙ Hasegawa, M. ∙ Murzin, A.G. ...</div><div><strong>Structure of pathological TDP-43 filaments from ALS with FTLD</strong></div><div><em>Nature.</em> 2022; <strong>601</strong>:139-143</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ArseniM.HasegawaA.G.MurzinF.KametaniM.AraiM.YoshidaB.Ryskeldi-FalconStructure+of+pathological+TDP-43+filaments+from+ALS+with+FTLDNature6012022139143" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We observe TMEM106B(120–254) fibrils to be the predominant fibrillar species in all FTLD-TDP cases we studied (<a id="crosref0920" href="#fig2">Figure&nbsp;2</a>; <a id="crosref0925" href="#mmc1">Table S2</a>), irrespective of fractionation protocol (See <a id="crosref0930" href="#sec-4">STAR Methods</a>). Finally, DLB is traditionally characterized by untwisted α-synuclein filaments with widths less than 100&nbsp;Å (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Schweighauser et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Schweighauser, M. ∙ Shi, Y. ∙ Tarutani, A. ...</div><div><strong>Structures of α-synuclein filaments from multiple system atrophy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:464-469</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserY.ShiA.TarutaniF.KametaniA.G.MurzinB.GhettiT.MatsubaraT.TomitaT.AndoK.HasegawaStructures+of+%CE%B1-synuclein+filaments+from+multiple+system+atrophyNature5852020464469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the present DLB case, we observe copious amounts of TMEM106B(120–254) singlet and doublet fibrils (<a id="crosref0940" href="#fig2">Figure&nbsp;2</a>; <a id="crosref0945" href="#mmc1">Table S3</a>), but narrow, untwisted α-synuclein filaments were not observed (presumably due to insufficient fibril yield, see <a id="crosref0950" href="#sec-4">STAR Methods</a>).</div><div id="p0135" role="paragraph">Interestingly, TMEM106B fibril structures similar to those identified in our study in patients across a wide range of proteinopathies as well as neurologically normal aged individuals was recently reported (<span class="dropBlock reference-citations"><a id="body-ref-sref51-3" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-3" aria-controls="sref51-3">Schweighauser et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Schweighauser, M. ∙ Arseni, D. ∙ Huang, M. ...</div><div><strong>Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserD.ArseniM.HuangS.L%C3%B6vestamY.ShiY.YangW.ZhangA.KotechaH.J.GarringerR.VidalAge-Dependent+Formation+of+TMEM106B+Amyloid+Filaments+in+Human+BrainPreprint+at+bioRxiv202110.1101%2F2021.11.09.467923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although the findings are similar, our study is distinct in focusing primarily on genetic and sporadic forms of FTLD-TDP (n&nbsp;= 8, <a id="crosref0960" href="#mmc1">Table S1</a>), whereas the work described in the recent report draws mainly from various tauopathies and synucleinopathies. The consensus finding that TMEM106B fibrils occur across diverse proteinopathies makes it difficult to determine whether TMEM106B fibrils are disease specific or are a general feature of neurodegeneration and aging. Importantly, the observation of increased TMEM106B fibrils in neurologically normal aged individuals (<span class="dropBlock reference-citations"><a id="body-ref-sref51-4" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-4" aria-controls="sref51-4">Schweighauser et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Schweighauser, M. ∙ Arseni, D. ∙ Huang, M. ...</div><div><strong>Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserD.ArseniM.HuangS.L%C3%B6vestamY.ShiY.YangW.ZhangA.KotechaH.J.GarringerR.VidalAge-Dependent+Formation+of+TMEM106B+Amyloid+Filaments+in+Human+BrainPreprint+at+bioRxiv202110.1101%2F2021.11.09.467923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) does not preclude a role in disease etiology as a similar observation is seen for tau (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Brier et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Brier, M.R. ∙ Gordon, B. ∙ Friedrichsen, K. ...</div><div><strong>Tau and Aβ imaging, CSF measures, and cognition in Alzheimer’s disease</strong></div><div><em>Sci. Transl. Med.</em> 2016; <strong>8</strong>:338ra66</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.R.BrierB.GordonK.FriedrichsenJ.McCarthyA.SternJ.ChristensenC.OwenP.AldeaY.SuJ.HassenstabTau+and+A%CE%B2+imaging%2C+CSF+measures%2C+and+cognition+in+Alzheimer%E2%80%99s+diseaseSci.+Transl.+Med.82016338ra66" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Chatterjee et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Chatterjee, P. ∙ Pedrini, S. ∙ Stoops, E. ...</div><div><strong>Plasma glial fibrillary acidic protein is elevated in cognitively normal older adults at risk of Alzheimer’s disease</strong></div><div><em>Transl. Psychiatry.</em> 2021; <strong>11</strong>:27</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.ChatterjeeS.PedriniE.StoopsK.GoozeeV.L.VillemagneP.R.AsihI.M.W.VerberkP.DaveK.TaddeiH.R.SohrabiPlasma+glial+fibrillary+acidic+protein+is+elevated+in+cognitively+normal+older+adults+at+risk+of+Alzheimer%E2%80%99s+diseaseTransl.+Psychiatry11202127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and α-synuclein filaments (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Hatton et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Hatton, C. ∙ Ghanem, S.S. ∙ Koss, D.J. ..., International DLB Genetics Consortium</div><div><strong>Prion-like α-synuclein pathology in the brain of infants with Krabbe disease</strong></div><div><em>Brain.</em> 2022;</div><div>Published online January 6, 2022</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.HattonS.S.GhanemD.J.KossI.Y.AbdiE.GibbonsR.GuerreiroJ.BrasL.WalkerE.GelpiW.HeywoodInternational+DLB+Genetics+ConsortiumPrion-like+%CE%B1-synuclein+pathology+in+the+brain+of+infants+with+Krabbe+diseaseBrain202210.1093%2Fbrain%2Fawac002Published+online+January+6%2C+2022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, it is unclear whether the formation of TMEM106B amyloid fibrils represents a primary pathogenic process or a non-specific secondary phenomenon (e.g., general downstream marker of lysosomal stress). The observation of TMEM106B fibrils in some, but not all, cases of PSP (<a id="crosref0985" href="#fig2">Figures 2</a>, <a id="crosref0990" href="#figs3">S3</a>D, and S3E) (<span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">Shi et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Shi, Y. ∙ Zhang, W. ∙ Yang, Y. ...</div><div><strong>Structure-based classification of tauopathies</strong></div><div><em>Nature.</em> 2021; <strong>598</strong>:359-363</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ShiW.ZhangY.YangA.G.MurzinB.FalconA.KotechaM.van+BeersA.TarutaniF.KametaniH.J.GarringerStructure-based+classification+of+tauopathiesNature5982021359363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and DLB (<a id="crosref1000" href="#fig2">Figure&nbsp;2</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref51-5" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-5" aria-controls="sref51-5">Schweighauser et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Schweighauser, M. ∙ Arseni, D. ∙ Huang, M. ...</div><div><strong>Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserD.ArseniM.HuangS.L%C3%B6vestamY.ShiY.YangW.ZhangA.KotechaH.J.GarringerR.VidalAge-Dependent+Formation+of+TMEM106B+Amyloid+Filaments+in+Human+BrainPreprint+at+bioRxiv202110.1101%2F2021.11.09.467923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) suggests that the degree of fibrillization is highly variable in neurodegenerative diseases. Nevertheless, the combined trend across the two studies is that in the majority of cases with known neurodegeneration, fibrillization of TMEM106B is substantially greater when compared to age-matched controls (<a id="crosref1010" href="#figs6">Figure&nbsp;S6</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref51-6" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-6" aria-controls="sref51-6">Schweighauser et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Schweighauser, M. ∙ Arseni, D. ∙ Huang, M. ...</div><div><strong>Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserD.ArseniM.HuangS.L%C3%B6vestamY.ShiY.YangW.ZhangA.KotechaH.J.GarringerR.VidalAge-Dependent+Formation+of+TMEM106B+Amyloid+Filaments+in+Human+BrainPreprint+at+bioRxiv202110.1101%2F2021.11.09.467923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0140" role="paragraph">Despite the small number of cases, this trend is most pronounced in the FTLD-TDP spectrum of neurodegenerative disease. All eight cases of FTLD-TDP in our study, representing all major pathological and genetic subtypes (<a id="crosref1020" href="#mmc1">Table S1</a>; <a id="crosref1025" href="#figs1">Figure&nbsp;S1</a>), contained large amounts of TMEM106B fibrils <i>per</i> gram of frontal cortex (<a id="crosref1030" href="#fig2">Figures 2</a> and <a id="crosref1035" href="#figs6">S6</a>). One relatively young patient (5th decade at death) with FTLD-TDP type A was replete with TMEM106B fibrils (<a id="crosref1040" href="#fig2">Figures 2</a> and <a id="crosref1045" href="#figs6">S6</a>), arguing against TMEM106B fibrillization representing a non-specific feature of aging (<span class="dropBlock reference-citations"><a id="body-ref-sref51-7" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-7" aria-controls="sref51-7">Schweighauser et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Schweighauser, M. ∙ Arseni, D. ∙ Huang, M. ...</div><div><strong>Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserD.ArseniM.HuangS.L%C3%B6vestamY.ShiY.YangW.ZhangA.KotechaH.J.GarringerR.VidalAge-Dependent+Formation+of+TMEM106B+Amyloid+Filaments+in+Human+BrainPreprint+at+bioRxiv202110.1101%2F2021.11.09.467923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, the observation that genetic variation in <i>TMEM106B</i> is a strong modifier of FTLD-TDP disease risk suggests that TMEM106B fibrilization may play a more central role in FTLD-TDP pathogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Finch et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Finch, N. ∙ Carrasquillo, M.M. ∙ Baker, M. ...</div><div><strong>TMEM106B regulates progranulin levels and the penetrance of FTLD in GRN mutation carriers</strong></div><div><em>Neurology.</em> 2011; <strong>76</strong>:467-474</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.FinchM.M.CarrasquilloM.BakerN.J.RutherfordG.CoppolaM.Dejesus-HernandezR.CrookT.HunterR.GhidoniL.BenussiTMEM106B+regulates+progranulin+levels+and+the+penetrance+of+FTLD+in+GRN+mutation+carriersNeurology762011467474" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Van Deerlin et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Van Deerlin, V.M. ∙ Sleiman, P.M. ∙ Martinez-Lage, M. ...</div><div><strong>Common variants at 7p21 are associated with frontotemporal lobar degeneration with TDP-43 inclusions</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:234-239</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.M.Van+DeerlinP.M.SleimanM.Martinez-LageA.Chen-PlotkinL.S.WangN.R.Graff-RadfordD.W.DicksonR.RademakersB.F.BoeveM.GrossmanCommon+variants+at+7p21+are+associated+with+frontotemporal+lobar+degeneration+with+TDP-43+inclusionsNat.+Genet.422010234239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">van der Zee et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>van der Zee, J. ∙ Van Langenhove, T. ∙ Kleinberger, G. ...</div><div><strong>TMEM106B is associated with frontotemporal lobar degeneration in a clinically diagnosed patient cohort</strong></div><div><em>Brain.</em> 2011; <strong>134</strong>:808-815</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.van+der+ZeeT.Van+LangenhoveG.KleinbergerK.SleegersS.EngelborghsR.VandenbergheP.SantensM.Van+den+BroeckG.JorisJ.BrysTMEM106B+is+associated+with+frontotemporal+lobar+degeneration+in+a+clinically+diagnosed+patient+cohortBrain1342011808815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0145" role="paragraph">If TMEM106B(120–254) is a potential driver of neurodegeneration, it will be important to determine what triggers the formation of these amyloid fibrils. Cellular models have revealed that overexpression of TMEM106B is associated with proteolytic cleavage that generates one or more N-terminal fragments and a presumed (though not yet observed) C-terminal fragment (<span class="dropBlock reference-citations"><a id="body-ref-sref6-3" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-3" aria-controls="sref6-3">Brady et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref6-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Brady, O.A. ∙ Zhou, X. ∙ Hu, F.</div><div><strong>Regulated intramembrane proteolysis of the frontotemporal lobar degeneration risk factor, TMEM106B, by signal peptide peptidase-like 2a (SPPL2a)</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2014; <strong>289</strong>:19670-19680</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.A.BradyX.ZhouF.HuRegulated+intramembrane+proteolysis+of+the+frontotemporal+lobar+degeneration+risk+factor%2C+TMEM106B%2C+by+signal+peptide+peptidase-like+2a+%28SPPL2a%29J.%C2%A0Biol.+Chem.28920141967019680" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We speculate that cleavage of the native TMEM106B fold by protease(s) generates an aggregation-prone fragment TMEM106B(120–254). As the isoelectric point of this lumenal C-terminal fragment is pH 5.4, the acidic environment of the lysosome (pH 4.5–5.5) would favor charge-neutralization and possibly fibrillization (<span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Smirnovas et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Smirnovas, V. ∙ Winter, R. ∙ Funck, T. ...</div><div><strong>Thermodynamic properties underlying the α-helix-to-β-sheet transition, aggregation, and amyloidogenesis of polylysine as probed by calorimetry, densimetry, and ultrasound velocimetry</strong></div><div><em>J.&nbsp;Phys. Chem. B.</em> 2005; <strong>109</strong>:19043-19045</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.SmirnovasR.WinterT.FunckW.DzwolakThermodynamic+properties+underlying+the+%CE%B1-helix-to-%CE%B2-sheet+transition%2C+aggregation%2C+and+amyloidogenesis+of+polylysine+as+probed+by+calorimetry%2C+densimetry%2C+and+ultrasound+velocimetryJ.%C2%A0Phys.+Chem.+B10920051904319045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The misfolding and aggregation of TMEM106B(120–254) may be influenced by posttranslational modifications (<span class="dropBlock reference-citations"><a id="body-ref-sref2-8" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-8" aria-controls="sref2-8">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), such as glycosylation (N145, N151, N164, N183), and binding of cofactors (K178, R180) increasing the likelihood of molecular (<span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Petkova et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Petkova, A.T. ∙ Leapman, R.D. ∙ Guo, Z. ...</div><div><strong>Self-propagating, molecular-level polymorphism in Alzheimer’s beta-amyloid fibrils</strong></div><div><em>Science.</em> 2005; <strong>307</strong>:262-265</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.PetkovaR.D.LeapmanZ.GuoW.M.YauM.P.MattsonR.TyckoSelf-propagating%2C+molecular-level+polymorphism+in+Alzheimer%E2%80%99s+beta-amyloid+fibrilsScience3072005262265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Zhang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Zhang, W. ∙ Falcon, B. ∙ Murzin, A.G. ...</div><div><strong>Heparin-induced tau filaments are polymorphic and differ from those in Alzheimer’s and Pick’s diseases</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43584</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.ZhangB.FalconA.G.MurzinJ.FanR.A.CrowtherM.GoedertS.H.ScheresHeparin-induced+tau+filaments+are+polymorphic+and+differ+from+those+in+Alzheimer%E2%80%99s+and+Pick%E2%80%99s+diseaseseLife82019e43584" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and ultrastructural (<span class="dropBlock reference-citations"><a id="body-ref-sref2-9" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-9" aria-controls="sref2-9">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Fitzpatrick et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Fitzpatrick, A.W.P. ∙ Debelouchina, G.T. ∙ Bayro, M.J. ...</div><div><strong>Atomic structure and hierarchical assembly of a cross-β amyloid fibril</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:5468-5473</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickG.T.DebelouchinaM.J.BayroD.K.ClareM.A.CaporiniV.S.BajajC.P.JaroniecL.WangV.LadizhanskyS.A.M%C3%BCllerAtomic+structure+and+hierarchical+assembly+of+a+cross-%CE%B2+amyloid+fibrilProc.+Natl.+Acad.+Sci.+USA110201354685473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref25-4" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" aria-label="Reference 2017" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-4" aria-controls="sref25-4">2017</a><div class="dropBlock__holder" data-db-target-of="sref25-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Fitzpatrick, A.W.P. ∙ Falcon, B. ∙ He, S. ...</div><div><strong>Cryo-EM structures of tau filaments from Alzheimer’s disease</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:185-190</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickB.FalconS.HeA.G.MurzinG.MurshudovH.J.GarringerR.A.CrowtherB.GhettiM.GoedertS.H.W.ScheresCryo-EM+structures+of+tau+filaments+from+Alzheimer%E2%80%99s+diseaseNature5472017185190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), polymorphism which manifest as amyloid fibril subtypes.</div><div id="p0150" role="paragraph">Based on our TMEM106B(120–254) fibril structures, we predict that cleavage at S120 may be due to thrombin, granzyme A, kallikrein-related peptidase 4, or cathepsin P (<a id="crosref1110" href="#mmc1">Table S4</a>; <a id="crosref1115" href="#figs5">Figure&nbsp;S5</a>D). Granzyme A is particularly intriguing due to its role in the inflammatory response and the biogenic link between lytic granules and lysosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">van Daalen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>van Daalen, K.R. ∙ Reijneveld, J.F. ∙ Bovenschen, N.</div><div><strong>Modulation of Inflammation by Extracellular Granzyme A</strong></div><div><em>Front. Immunol.</em> 2020; <strong>11</strong>:931</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.R.van+DaalenJ.F.ReijneveldN.BovenschenModulation+of+Inflammation+by+Extracellular+Granzyme+AFront.+Immunol.112020931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The degree to which proteolytic cleavage of the C-terminal fragment occurs <i>in&nbsp;vivo</i> is uncertain; however, it raises the possibility that factors that increase lysosomal stress, such as aging or reduced levels&nbsp;of progranulin (PGRN), combined with the presence of the higher expressing <i>TMEM106B</i> risk haplotype may drive more TMEM106B into the proteolytic pathway, resulting in the generation of TMEM106B C-terminal fragments in excess of the critical aggregation concentration favoring fibril formation (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Hortschansky et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Hortschansky, P. ∙ Christopeit, T. ∙ Schroeckh, V. ...</div><div><strong>Thermodynamic analysis of the aggregation propensity of oxidized Alzheimer’s beta-amyloid variants</strong></div><div><em>Protein Sci.</em> 2005; <strong>14</strong>:2915-2918</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.HortschanskyT.ChristopeitV.SchroeckhM.F%C3%A4ndrichThermodynamic+analysis+of+the+aggregation+propensity+of+oxidized+Alzheimer%E2%80%99s+beta-amyloid+variantsProtein+Sci.14200529152918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0155" role="paragraph">Given the presence of these previously undetected amyloid fibrils in myriad proteinopathies, it is pertinent to ask what&nbsp;possible effect the cleavage and aggregation of TMEM106B(120–254) could have on the trafficking and degradation of TDP-43, tau, and α-synuclein filaments. For now, we can only speculate, but given our current knowledge, we outline various possibilities based upon our structural observations (<a id="crosref1130" href="#fig6">Figure&nbsp;6</a>). As TMEM106B is a lysosomal/endosomal protein, its loss of function via fibrillization could disrupt lysosomal trafficking on microtubules, impairing cellular clearance mechanisms (<span class="dropBlock reference-citations"><a id="body-ref-sref6-4" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-4" aria-controls="sref6-4">Brady et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref6-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Brady, O.A. ∙ Zhou, X. ∙ Hu, F.</div><div><strong>Regulated intramembrane proteolysis of the frontotemporal lobar degeneration risk factor, TMEM106B, by signal peptide peptidase-like 2a (SPPL2a)</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2014; <strong>289</strong>:19670-19680</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.A.BradyX.ZhouF.HuRegulated+intramembrane+proteolysis+of+the+frontotemporal+lobar+degeneration+risk+factor%2C+TMEM106B%2C+by+signal+peptide+peptidase-like+2a+%28SPPL2a%29J.%C2%A0Biol.+Chem.28920141967019680" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">Lüningschrör et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Lüningschrör, P. ∙ Werner, G. ∙ Stroobants, S. ...</div><div><strong>The FTLD Risk Factor TMEM106B Regulates the Transport of Lysosomes at the Axon Initial Segment of Motoneurons</strong></div><div><em>Cell Rep.</em> 2020; <strong>30</strong>:3506-3519.e6</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.L%C3%BCningschr%C3%B6rG.WernerS.StroobantsS.KakutaB.DombertD.SinskeR.WannerR.L%C3%BCllmann-RauchB.WefersW.WurstThe+FTLD+Risk+Factor+TMEM106B+Regulates+the+Transport+of+Lysosomes+at+the+Axon+Initial+Segment+of+MotoneuronsCell+Rep.30202035063519.e6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52-3" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-3" aria-controls="sref52-3">Schwenk et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref52-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Schwenk, B.M. ∙ Lang, C.M. ∙ Hogl, S. ...</div><div><strong>The FTLD risk factor TMEM106B and MAP6 control dendritic trafficking of lysosomes</strong></div><div><em>EMBO J.</em> 2014; <strong>33</strong>:450-467</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.M.SchwenkC.M.LangS.HoglS.TahirovicD.OrozcoK.RentzschS.F.LichtenthalerC.C.HoogenraadA.CapellC.HaassD.EdbauerThe+FTLD+risk+factor+TMEM106B+and+MAP6+control+dendritic+trafficking+of+lysosomesEMBO+J.332014450467" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). If TMEM106B(120–254) forms fibrils in the lumen,&nbsp;degradation of these protein aggregates places an extra&nbsp;burden on the lysosome. This will impede disposal of aberrantly aggregated filaments delivered to the lysosomes by autophagosomes or endosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Jiang and Bhaskar, 2020</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Jiang, S. ∙ Bhaskar, K.</div><div><strong>Degradation and Transmission of Tau by Autophagic-Endolysosomal Networks and Potential Therapeutic Targets for Tauopathy</strong></div><div><em>Front. Mol. Neurosci.</em> 2020; <strong>13</strong>:586731</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.JiangK.BhaskarDegradation+and+Transmission+of+Tau+by+Autophagic-Endolysosomal+Networks+and+Potential+Therapeutic+Targets+for+TauopathyFront.+Mol.+Neurosci.132020586731" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). If the lysosomal burden is increased, there may also be lysosomal leakage resulting in the release of TMEM106B(120–254) fragments and aggregates into the cytosol where they can propagate. Generally, we hypothesize that the fibrillization of TMEM106B(120–254) inside or outside the lysosome will trigger lysosomal dysfunction promoting the accumulation of TDP-43, tau, and α-synuclein filaments in susceptible individuals (<a id="crosref1155" href="#fig6">Figure&nbsp;6</a>). If TMEM106B(120–254) fibrils were to accumulate intracellularly, the loss/gain of function arising from the fibrillization of TMEM106B(120–254) would lead to markedly less efficient removal of TDP-43, tau, and α-synuclein filaments through neuronal or glial lysosomal degradation. Ultimately, our conjecture is that TMEM106B(120–254) fibrils exacerbate lysosomal stress and amplify or accelerate the accumulation of pathological proteins (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Monaco and Fraldi, 2020</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Monaco, A. ∙ Fraldi, A.</div><div><strong>Protein Aggregation and Dysfunction of Autophagy-Lysosomal Pathway: A Vicious Cycle in Lysosomal Storage Diseases</strong></div><div><em>Front. Mol. Neurosci.</em> 2020; <strong>13</strong>:37</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MonacoA.FraldiProtein+Aggregation+and+Dysfunction+of+Autophagy-Lysosomal+Pathway%3A+A+Vicious+Cycle+in+Lysosomal+Storage+DiseasesFront.+Mol.+Neurosci.13202037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.02.026/asset/9e83a9d9-e5f4-476a-9873-6e9c270666f0/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2022.02.026/asset/6c26ddd9-d35e-4623-98f4-a42c937c3ba9/main.assets/gr6.jpg" height="305" width="373" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/9e83a9d9-e5f4-476a-9873-6e9c270666f0/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Structure-based model of a possible interrelationship between TMEM106B, aggregated filaments, and lysosomes in neurodegenerative diseases</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0115" role="paragraph">Under physiological conditions, TMEM106B spans the lysosomal/endosomal membrane. Upon cleavage of the C-terminal fragment, TMEM106B(120–254) fibrils may form. It is currently unknown whether TMEM106B(120–254) fibrillizes in the lumen or whether lysosomal leakage occurs and TMEM106B(120–254) fragments aggregate into fibrils in the cytosol. Based on our set of TMEM106B fibril structures, we speculate that the aggregation of TMEM106B(120–254) into fibrils leads to lysosomal dysfunction, which promotes the accumulation of aberrantly aggregated amyloid fibrils such as those formed by TDP-43 (PDB:<a id="intref0310" href="http://firstglance.jmol.org/fg.htm?mol7PY2" target="_blank">7PY2</a>, green sticks), tau (PDB:<a id="intref0315" href="http://firstglance.jmol.org/fg.htm?mol7P65" target="_blank">7P65</a>, <a id="intref0320" href="http://firstglance.jmol.org/fg.htm?molblue" target="_blank">blue</a> sticks), or α-synuclein.</div></div></div></div></figcaption></figure></div><div id="p0160" role="paragraph">In conclusion, we have determined the structures of TMEM106B(120–254) fibrils identified in a range of phenotypically distinct proteinopathies using cryo-EM. This unknown fibrillization pathway in all major TDP-43 proteinopathies, a tauopathy, and a synucleinopathy raises many intriguing questions. While future experiments will determine whether TMEM106B(120–254) fibril formation represents a loss or gain of function, our findings identify a common fibrillization pathway and widespread structural target that may be implicated in neurodegeneration.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0165" role="paragraph">At the time of publication, an antibody that binds exclusively to&nbsp;TMEM106B(120–254) fibrils does not exist. This has prevented immunohistochemistry experiments to directly visualize TMEM106B(120–254) fibrils in postmortem human tissue using immunofluorescence or immunogold electron microscopy.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0170" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Rabbit polyclonal anti-TMEM106B (204-253)</td><td>Novus Biologicals</td><td>Cat#NBP1-91311; RRID: <a id="intref0025" href="https://antibodyregistry.org/AB_11019681" target="_blank">AB_11019681</a>; QC18333-42825</td></tr><tr><td>IRDye 800CW Donkey anti-Rabbit IgG Secondary Antibody</td><td>LI-COR Biosciences</td><td>Cat#926-32213; RRID: <a id="intref0030" href="https://antibodyregistry.org/AB_621848" target="_blank">AB_621848</a>; D11005-09</td></tr><tr><td>Mouse monoclonal anti-TMEM106B (1-46)</td><td>Proteintech</td><td>Cat#60333-1-Ig; RRID: <a id="intref0035" href="https://antibodyregistry.org/AB_2881442" target="_blank">AB_2881442</a></td></tr><tr><td>Rabbit polyclonal anti-TMEM106B (101-200)</td><td>Bioss</td><td>Cat#bs-11694R; RRID: <a id="intref0040" href="https://antibodyregistry.org/AB_2905622" target="_blank">AB_2905622</a></td></tr><tr><td>Rabbit polyclonal anti-TMEM106B (111-190)</td><td>Biorbyt</td><td>Cat#orb158617; RRID: <a id="intref0045" href="https://antibodyregistry.org/AB_2905623" target="_blank">AB_2905623</a></td></tr><tr><td>Rabbit polyclonal anti-TMEM106B (150-275)</td><td>Proteintech</td><td>Cat#20995-1-AP; RRID: <a id="intref0050" href="https://antibodyregistry.org/AB_10694293" target="_blank">AB_10694293</a></td></tr><tr><td>Rabbit polyclonal anti-TMEM106B (204-253)</td><td>Sigma Aldrich</td><td>Cat#SAB2106773; RRID <a id="intref0055" href="https://antibodyregistry.org/AB_2905624" target="_blank">AB_2905624</a></td></tr><tr><td>Rabbit polyclonal anti-TMEM106B (218-252)</td><td>LSBio</td><td>Cat#LS-C757550; RRID: <a id="intref0060" href="https://antibodyregistry.org/AB_2905625" target="_blank">AB_2905625</a></td></tr><tr><td>Mouse monoclonal anti-pTau (phosphorylated at Ser262 and Ser356)</td><td>P. Seubert, Elan Pharmaceuticals; San Francisco, CA; USA</td><td>12E8</td></tr><tr><td>Mouse monoclonal anti-pTau (phosphorylated at Ser396 and Ser404)</td><td>P. Davies, Albert Einstein College of Medicine; New York, NY; USA</td><td>PHF1; RRID: <a id="intref0065" href="https://antibodyregistry.org/AB_2315150" target="_blank">AB_2315150</a></td></tr><tr><td>Mouse monoclonal anti-pTau (phosphorylated at Ser202)</td><td>P. Davies, Albert Einstein College of Medicine; New York, NY; USA</td><td>CP13; RRID: <a id="intref0070" href="https://antibodyregistry.org/AB_2314223" target="_blank">AB_2314223</a></td></tr><tr><td>Rabbit polyclonal anti-Tau (Human-specific)</td><td>L. Petrucelli, Mayo Clinic; Jacksonville, FL; USA</td><td>E1; RRID: 2819185</td></tr><tr><td>Rabbit polyclonal anti-TDP-43</td><td>Proteintech</td><td>Cat#10782-2-AP; RRID: <a id="intref0075" href="https://antibodyregistry.org/AB_615042" target="_blank">AB_615042</a></td></tr><tr><td>Mouse monoclonal anti-pTau (phosphorylated at Ser202 and Thr205)</td><td>Thermo Fisher</td><td>Cat#MN1020; RRID: <a id="intref0080" href="https://antibodyregistry.org/AB_223647" target="_blank">AB_223647</a></td></tr><tr><td>Mouse monoclonal anti-α-synuclein</td><td>Invitrogen</td><td>Cat# 180215; RRID: <a id="intref0085" href="https://antibodyregistry.org/AB_86714" target="_blank">AB_86714</a></td></tr><tr><td>Mouse monoclonal anti-beta amyloid</td><td>Dako</td><td>Cat#M0872; RRID: <a id="intref0090" href="https://antibodyregistry.org/AB_2056966" target="_blank">AB_2056966</a></td></tr><tr class="rowsep"><td>Rabbit polyclonal anti-ubiquitin</td><td>Dako</td><td>Cat#Z0458; RRID: <a id="intref0095" href="https://antibodyregistry.org/AB_2315524" target="_blank">AB_2315524</a></td></tr><tr class="rowsep"><td colspan="3"><b>Biological Samples</b></td></tr><tr><td>FTLD-TDP frontal cortex tissue</td><td>University of British Columbia (UBC)</td><td><a id="intref0100" href="https://neurology.med.ubc.ca/programs/alzheimer-disease-and-related-disorders-program/" target="_blank">https://neurology.med.ubc.ca/programs/alzheimer-disease-and-related-disorders-program/</a></td></tr><tr><td>PSP Case 1 caudate tissue</td><td>Mayo Clinic Brain Bank</td><td><a id="intref0105" href="https://www.mayo.edu/research/departments-divisions/department-neuroscience-florida/brain-bank" target="_blank">https://www.mayo.edu/research/departments-divisions/department-neuroscience-florida/brain-bank</a></td></tr><tr><td>PSP Cases 2 and 3 frontal cortex tissue</td><td>University of British Columbia (UBC)</td><td><a id="intref0110" href="https://neurology.med.ubc.ca/programs/alzheimer-disease-and-related-disorders-program/" target="_blank">https://neurology.med.ubc.ca/programs/alzheimer-disease-and-related-disorders-program/</a></td></tr><tr><td>DLB frontal cortex tissue</td><td>Mayo Clinic Brain Bank</td><td><a id="intref0115" href="https://www.mayo.edu/research/departments-divisions/department-neuroscience-florida/brain-bank" target="_blank">https://www.mayo.edu/research/departments-divisions/department-neuroscience-florida/brain-bank</a></td></tr><tr class="rowsep"><td>Neurologically normal frontal cortex tissue</td><td>University of British Columbia (UBC)</td><td><a id="interref0010" href="https://neurology.med.ubc.ca/programs/alzheimer-disease-and-related-disorders-program/" target="_blank">https://neurology.med.ubc.ca/programs/alzheimer-disease-and-related-disorders-program/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>Intercept Blocking Buffer</td><td>LI-COR Biosciencs</td><td>Cat#927-70001; 210816</td></tr><tr><td>iBlot 2 PVDF Mini Stacks</td><td>Invitrogen/Thermo Fisher Scientific</td><td>Cat#IB24002; 2PM131221-01</td></tr><tr><td>Novex WedgeWell 4-20% Tris-Glycine Gel</td><td>Invitrogen/Thermo Fisher Scientific</td><td>Cat#XP-4205BOX; 21030390</td></tr><tr><td>SDS-PAGE Running Buffer [10X]</td><td>GBiosciences</td><td>Cat#786-029G; 181405</td></tr><tr><td>LI-COR Odyssey CLx</td><td>LI-COR Biosciences</td><td>CLX-0707</td></tr><tr><td>iBlot 2 Gel Transfer Device</td><td>Invitrogen/Thermo Fisher Scientific</td><td>Cat#IB21001; 0302180117</td></tr><tr><td>BLUEstain 2 Protein ladder 5-245&nbsp;kDa</td><td>Gold Biotechnology</td><td>Cat#P008-500; 051301P008</td></tr><tr><td>Paraformaldehyde 32% Solution</td><td>Electron Microscopy Sciences</td><td>Cat#15714-S; 190107-10</td></tr><tr><td>XCell Sure Lock Mini-Cell</td><td>Invitrogen/Thermo Fisher Scientific</td><td>Cat#EI0001</td></tr><tr><td>cOmplete Mini, EDTA-free protease inhibitor cocktail tablets</td><td>Roche Diagnostics</td><td>Cat#11836170001; 35440400</td></tr><tr><td>Phenylmethylsulfonyl fluoride (PMSF)</td><td>Gold Biotechnology</td><td>Cat#P-470-10; 0302.091819A</td></tr><tr class="rowsep"><td>Polytron Homogenizer</td><td>Kinematica</td><td>Cat#11030012</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr><td>Raw mass spectrometry data</td><td>This paper</td><td>MassIVE: MSV000088875</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (all cases combined) singlet</td><td>This paper</td><td>EMDB: EMD-26296</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (all cases combined) protofilament</td><td>This paper</td><td>EMDB: EMD-26279</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 1) singlet</td><td>This paper</td><td>EMDB: EMD-26295</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 1) doublet</td><td>This paper</td><td>EMDB: EMD-26294</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 1) protofilament</td><td>This paper</td><td>EMDB: EMD-26274</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 2) singlet</td><td>This paper</td><td>EMDB: EMD-26275</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 2) doublet</td><td>This paper</td><td>EMDB: EMD-26293</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 3) singlet</td><td>This paper</td><td>EMDB: EMD-26292</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 3) doublet</td><td>This paper</td><td>EMDB: EMD-26291</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 4) singlet</td><td>This paper</td><td>EMDB: EMD-26276</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 4) doublet</td><td>This paper</td><td>EMDB: EMD-26290</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type A (case 5) doublet</td><td>This paper</td><td>EMDB: EMD-26289</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type B case 1 (case 6) singlet</td><td>This paper</td><td>EMDB: EMD-26288</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type B case 2 (case 7) high-twist singlet</td><td>This paper</td><td>EMDB: EMD-26278</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type C (case 8) singlet</td><td>This paper</td><td>EMDB: EMD-26277</td></tr><tr><td>Cryo-EM density map of FTLD-TDP Type C (case 8) doublet</td><td>This paper</td><td>EMDB: EMD-26287</td></tr><tr><td>Cryo-EM density map of PSP case 1 singlet</td><td>This paper</td><td>EMDB: EMD-26284</td></tr><tr><td>Cryo-EM density map of PSP case 2 singlet</td><td>This paper</td><td>EMDB: EMD-26285</td></tr><tr><td>Cryo-EM density map of PSP case 2 doublet</td><td>This paper</td><td>EMDB: EMD-26286</td></tr><tr><td>Cryo-EM density map of PSP case 2 protofilament</td><td>This paper</td><td>EMDB: EMD-26273</td></tr><tr><td>Cryo-EM density map of PSP case 1 tau filament</td><td>This paper</td><td>EMDB: EMD-26268</td></tr><tr><td>Cryo-EM density map of PSP case 3 tau filament</td><td>This paper</td><td>EMDB: EMD-26283</td></tr><tr><td>Cryo-EM density map of DLB singlet</td><td>This paper</td><td>EMDB: EMD-26281</td></tr><tr><td>Cryo-EM density map of DLB doublet</td><td>This paper</td><td>EMDB: EMD-26282</td></tr><tr><td>Atomic model of FTLD-TDP Type A (all cases combined) protofilament</td><td>This paper</td><td>PDB: <a id="intref0120" href="http://firstglance.jmol.org/fg.htm?mol7U16" target="_blank">7U16</a></td></tr><tr><td>Atomic model of FTLD-TDP Type A (T185S) (all cases combined) protofilament</td><td>This paper</td><td>PDB: <a id="intref0125" href="http://firstglance.jmol.org/fg.htm?mol7U18" target="_blank">7U18</a></td></tr><tr><td>Atomic model of FTLD-TDP Type A (case 1) protofilament</td><td>This paper</td><td>PDB: <a id="intref0130" href="http://firstglance.jmol.org/fg.htm?mol7U11" target="_blank">7U11</a></td></tr><tr><td>Atomic model of FTLD-TDP Type A (case 2) singlet</td><td>This paper</td><td>PDB: <a id="intref0135" href="http://firstglance.jmol.org/fg.htm?mol7U12" target="_blank">7U12</a></td></tr><tr><td>Atomic model of FTLD-TDP Type A (case 4) singlet</td><td>This paper</td><td>PDB: <a id="intref0140" href="http://firstglance.jmol.org/fg.htm?mol7U13" target="_blank">7U13</a></td></tr><tr><td>Atomic model of FTLD-TDP Type B case 2 (case 7) high-twist singlet</td><td>This paper</td><td>PDB: <a id="intref0145" href="http://firstglance.jmol.org/fg.htm?mol7U15" target="_blank">7U15</a></td></tr><tr><td>Atomic model of FTLD-TDP Type B case 2 (case 7) (T185S) high-twist singlet</td><td>This paper</td><td>PDB: <a id="intref0150" href="http://firstglance.jmol.org/fg.htm?mol7U17" target="_blank">7U17</a></td></tr><tr><td>Atomic model of FTLD-TDP Type C (case 8) singlet</td><td>This paper</td><td>PDB: <a id="intref0155" href="http://firstglance.jmol.org/fg.htm?mol7U14" target="_blank">7U14</a></td></tr><tr><td>Atomic model of PSP case 1 tau filament</td><td>This paper</td><td>PDB: <a id="intref0160" href="http://firstglance.jmol.org/fg.htm?mol7U0Z" target="_blank">7U0Z</a></td></tr><tr class="rowsep"><td>Atomic model of PSP case 2 protofilament</td><td>This paper</td><td>PDB: <a id="intref0165" href="http://firstglance.jmol.org/fg.htm?mol7U10" target="_blank">7U10</a></td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>MotionCor2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.-P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.Q.ZhengE.PalovcakJ.-P.ArmacheK.A.VerbaY.ChengD.A.AgardMotionCor2%3A+anisotropic+correction+of+beam-induced+motion+for+improved+cryo-electron+microscopyNat.+Methods142017331332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0170" href="https://msg.ucsf.edu/software" target="_blank">https://msg.ucsf.edu/software</a></td></tr><tr><td>Gctf v1.06</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Zhang, 2016</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.ZhangGctf%3A+Real-time+CTF+determination+and+correctionJ.%C2%A0Struct.+Biol.1932016112" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0175" href="https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/zhang-software/" target="_blank">https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/zhang-software/</a></td></tr><tr><td>RELION</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">He and Scheres, 2017</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>He, S. ∙ Scheres, S.H.W.</div><div><strong>Helical reconstruction in RELION</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2017; <strong>198</strong>:163-176</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.HeS.H.W.ScheresHelical+reconstruction+in+RELIONJ.%C2%A0Struct.+Biol.1982017163176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0180" href="https://www2.mrc-lmb.cam.ac.uk/relion" target="_blank">https://www2.mrc-lmb.cam.ac.uk/relion</a></td></tr><tr><td>Chimera</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Pettersen et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF Chimera--a visualization system for exploratory research and analysis</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2004; <strong>25</strong>:1605-1612</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.F.PettersenT.D.GoddardC.C.HuangG.S.CouchD.M.GreenblattE.C.MengT.E.FerrinUCSF+Chimera--a+visualization+system+for+exploratory+research+and+analysisJ.%C2%A0Comput.+Chem.25200416051612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0185" href="https://www.cgl.ucsf.edu/chimera/" target="_blank">https://www.cgl.ucsf.edu/chimera/</a></td></tr><tr><td>Coot</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.EmsleyB.LohkampW.G.ScottK.CowtanFeatures+and+development+of+CootActa+Crystallogr.+D+Biol.+Crystallogr.662010486501" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0190" href="https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/" target="_blank">https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/</a></td></tr><tr><td>Phenix.real_space_refine (v1.17.1-3660)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Afonine et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Afonine, P.V. ∙ Poon, B.K. ∙ Read, R.J. ...</div><div><strong>Real-space refinement in PHENIX for cryo-EM and crystallography</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:531-544</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.V.AfonineB.K.PoonR.J.ReadO.V.SobolevT.C.TerwilligerA.UrzhumtsevP.D.AdamsReal-space+refinement+in+PHENIX+for+cryo-EM+and+crystallographyActa+Crystallogr.+D+Struct.+Biol.742018531544" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0195" href="https://phenix-online.org/" target="_blank">https://phenix-online.org/</a></td></tr><tr><td>MolProbity</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Chen et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Chen, V.B. ∙ Arendall, 3rd, W.B. ∙ Headd, J.J. ...</div><div><strong>MolProbity: all-atom structure validation for macromolecular crystallography</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:12-21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.B.ChenW.B.Arendall3rdJ.J.HeaddD.A.KeedyR.M.ImmorminoG.J.KapralL.W.MurrayJ.S.RichardsonD.C.RichardsonMolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallographyActa+Crystallogr.+D+Biol.+Crystallogr.6620101221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0200" href="http://molprobity.biochem.duke.edu/" target="_blank">http://molprobity.biochem.duke.edu/</a></td></tr><tr><td>CamSol method</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref55-1" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-1" aria-controls="sref55-1">Sormanni et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Sormanni, P. ∙ Aprile, F.A. ∙ Vendruscolo, M.</div><div><strong>The CamSol method of rational design of protein mutants with enhanced solubility</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 2015; <strong>427</strong>:478-490</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.SormanniF.A.AprileM.VendruscoloThe+CamSol+method+of+rational+design+of+protein+mutants+with+enhanced+solubilityJ.%C2%A0Mol.+Biol.4272015478490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0205" href="http://www-vendruscolo.ch.cam.ac.uk/camsolmethod.html" target="_blank">http://www-vendruscolo.ch.cam.ac.uk/camsolmethod.html</a></td></tr><tr><td>PyMOL</td><td>&nbsp;</td><td><a id="intref0210" href="https://www.pymol.org" target="_blank">https://www.pymol.org</a></td></tr><tr><td>AlphaFold</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref33-3" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-3" aria-controls="sref33-3">Jumper et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref33-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Jumper, J. ∙ Evans, R. ∙ Pritzel, A. ...</div><div><strong>Highly accurate protein structure prediction with AlphaFold</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:583-589</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.JumperR.EvansA.PritzelT.GreenM.FigurnovO.RonnebergerK.TunyasuvunakoolR.BatesA.%C5%BD%C3%ADdekA.PotapenkoHighly+accurate+protein+structure+prediction+with+AlphaFoldNature5962021583589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0215" href="https://alphafold.ebi.ac.uk/" target="_blank">https://alphafold.ebi.ac.uk/</a></td></tr><tr><td>TrRosetta</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Yang, J. ∙ Anishchenko, I. ∙ Park, H. ...</div><div><strong>Improved protein structure prediction using predicted interresidue orientations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:1496-1503</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.YangI.AnishchenkoH.ParkZ.PengS.OvchinnikovD.BakerImproved+protein+structure+prediction+using+predicted+interresidue+orientationsProc.+Natl.+Acad.+Sci.+USA117202014961503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0220" href="https://yanglab.nankai.edu.cn/trRosetta/" target="_blank">https://yanglab.nankai.edu.cn/trRosetta/</a></td></tr><tr><td>Robetta</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Kim et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Kim, D.E. ∙ Chivian, D. ∙ Baker, D.</div><div><strong>Protein structure prediction and analysis using the Robetta server</strong></div><div><em>Nucleic Acids Res.</em> 2004; <strong>32</strong>:W526-W531</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.E.KimD.ChivianD.BakerProtein+structure+prediction+and+analysis+using+the+Robetta+serverNucleic+Acids+Res.322004W526-W531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0225" href="http://robetta.bakerlab.org/" target="_blank">http://robetta.bakerlab.org/</a></td></tr><tr><td>DeepEMHancer</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Sanchez-Garcia et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Sanchez-Garcia, R. ∙ Gomez-Blanco, J. ∙ Cuervo, A. ...</div><div><strong>DeepEMhancer: a deep learning solution for cryo-EM volume post-processing</strong></div><div><em>Commun Biol.</em> 2021; <strong>4</strong>:874</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.Sanchez-GarciaJ.Gomez-BlancoA.CuervoJ.M.CarazoC.O.S.SorzanoJ.VargasDeepEMhancer%3A+a+deep+learning+solution+for+cryo-EM+volume+post-processingCommun+Biol.42021874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0230" href="https://cosmic-cryoem.org/tools/deepemhancer/" target="_blank">https://cosmic-cryoem.org/tools/deepemhancer/</a></td></tr><tr><td>Deep Tracer</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Pfab et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Pfab, J. ∙ Phan, N.M. ∙ Si, D.</div><div><strong>DeepTracer for fast de novo cryo-EM protein structure modeling and special studies on CoV-related complexes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong></div><div>e2017525118</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.PfabN.M.PhanD.SiDeepTracer+for+fast+de+novo+cryo-EM+protein+structure+modeling+and+special+studies+on+CoV-related+complexesProc.+Natl.+Acad.+Sci.+USA1182021e2017525118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0235" href="https://deeptracer.uw.edu/home" target="_blank">https://deeptracer.uw.edu/home</a></td></tr><tr><td>Cryo-ID</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Ho et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ho, C.M. ∙ Li, X. ∙ Lai, M. ...</div><div><strong>Bottom-up structural proteomics: cryoEM of protein complexes enriched from the cellular milieu</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:79-85</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.HoX.LiM.LaiT.C.TerwilligerJ.R.BeckJ.WohlschlegelD.E.GoldbergA.W.P.FitzpatrickZ.H.ZhouBottom-up+structural+proteomics%3A+cryoEM+of+protein+complexes+enriched+from+the+cellular+milieuNat.+Methods1720207985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0240" href="https://github.com/EICN-UCLA/cryoID" target="_blank">https://github.com/EICN-UCLA/cryoID</a></td></tr><tr><td>findMySequence</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref13-2" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-2" aria-controls="sref13-2">Chojnowski et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref13-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Chojnowski, G. ∙ Simpkin, A.J. ∙ Leonardo, D.A. ...</div><div><strong><i>findMySequence</i>: a neural-network-based approach for identification of unknown proteins in X-ray crystallography and cryo-EM</strong></div><div><em>IUCrJ.</em> 2021; <strong>9</strong>:86-97</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.ChojnowskiA.J.SimpkinD.A.LeonardoW.Seifert-DavilaD.E.Vivas-RuizR.M.KeeganD.J.RigdenfindMySequence%3A+a+neural-network-based+approach+for+identification+of+unknown+proteins+in+X-ray+crystallography+and+cryo-EMIUCrJ920218697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0245" href="https://gitlab.com/gchojnowski/findmysequence" target="_blank">https://gitlab.com/gchojnowski/findmysequence</a></td></tr><tr><td>ProCleave server</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Li et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, F. ∙ Leier, A. ∙ Liu, Q. ...</div><div><strong>Procleave: Predicting Protease-specific Substrate Cleavage Sites by Combining Sequence and Structural Information</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2020; <strong>18</strong>:52-64</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.LiA.LeierQ.LiuY.WangD.XiangT.AkutsuG.I.WebbA.I.SmithT.Marquez-LagoJ.LiJ.SongProcleave%3A+Predicting+Protease-specific+Substrate+Cleavage+Sites+by+Combining+Sequence+and+Structural+InformationGenomics+Proteomics+Bioinformatics1820205264" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0250" href="http://procleave.erc.monash.edu/" target="_blank">http://procleave.erc.monash.edu/</a></td></tr><tr><td>COSMIC</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Cianfrocco et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Cianfrocco, M.A. ∙ Wong-Barnum, M. ∙ Youn, C. ...</div><div><strong>COSMIC<sup>2</sup>: A Science Gateway for Cryo-Electron Microscopy Structure Determination</strong></div><div><strong>Proceedings of the Practice and Experience in Advanced Research Computing 2017 on Sustainability, Success and Impact</strong></div><div>2017</div><div>Published online July 7, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.A.CianfroccoM.Wong-BarnumC.YounR.WagnerA.E.LeschzinerCOSMIC2%3A+A+Science+Gateway+for+Cryo-Electron+Microscopy+Structure+DeterminationProceedings+of+the+Practice+and+Experience+in+Advanced+Research+Computing+2017+on+Sustainability%2C+Success+and+Impact201710.1145%2F3093338.3093390Published+online+July+7%2C+2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0255" href="https://github.com/cianfrocco-lab/COSMIC-CryoEM-Gateway" target="_blank">https://github.com/cianfrocco-lab/COSMIC-CryoEM-Gateway</a></td></tr><tr><td>MEROPS database</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Rawlings et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Rawlings, N.D. ∙ Barrett, A.J. ∙ Thomas, P.D. ...</div><div><strong>The MEROPS database of proteolytic enzymes, their substrates and inhibitors in 2017 and a comparison with peptidases in the PANTHER database</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D624-D632</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.D.RawlingsA.J.BarrettP.D.ThomasX.HuangA.BatemanR.D.FinnThe+MEROPS+database+of+proteolytic+enzymes%2C+their+substrates+and+inhibitors+in+2017+and+a+comparison+with+peptidases+in+the+PANTHER+databaseNucleic+Acids+Res.46D12018D624D632" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0260" href="https://www.ebi.ac.uk/merops/" target="_blank">https://www.ebi.ac.uk/merops/</a></td></tr><tr class="rowsep"><td>MaxQuant</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Cox and Mann, 2008</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cox, J. ∙ Mann, M.</div><div><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong></div><div><em>Nat. Biotechnol.</em> 2008; <strong>26</strong>:1367-1372</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.CoxM.MannMaxQuant+enables+high+peptide+identification+rates%2C+individualized+p.p.b.-range+mass+accuracies+and+proteome-wide+protein+quantificationNat.+Biotechnol.26200813671372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0265" href="https://www.maxquant.org/" target="_blank">https://www.maxquant.org/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Holey carbon grids (Au R1.2/1.3, 300 mesh)</td><td>Quantifoil</td><td>N/A</td></tr><tr><td>BioRender</td><td>&nbsp;</td><td><a id="intref0270" href="https://biorender.com/" target="_blank">https://biorender.com/</a></td></tr><tr><td>atom2svg</td><td>&nbsp;</td><td><a id="intref0275" href="https://gist.github.com/biochem-fan/026ec2f191fee9285424d12fc2b84ce7" target="_blank">https://gist.github.com/biochem-fan/026ec2f191fee9285424d12fc2b84ce7</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2822%2900259-8" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0175" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Anthony W. P. Fitzpatrick (<a id="intref0280" href="mailto:anthony.fitzpatrick@columbia.edu">anthony.fitzpatrick@columbia.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0180" role="paragraph">This study did not generate new unique reagents.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Acquisition of human tissues</h4><div id="p0185" role="paragraph">The tissues used for this study were from the frontal cortices of brains with FTLD-TDP (n&nbsp;= 11, 4 males, 7 females), PSP (n&nbsp;= 3, 2 male, 1 female), DLB (n&nbsp;= 6, 3 males, 3 females), and neurologically normal controls (n&nbsp;= 5, 3 males, 2 females), with the exception&nbsp;of PSP case 1 tissue being taken from the caudate region. Patients had been followed at the University of British Columbia Clinic for Dementia and Related Disorders. Informed consent for autopsy and the use of tissue for research was obtained from the legal next-of-kin in accordance with institutional ethics guidelines. Human brain samples were collected under a Mayo Clinic IRB-approved autopsy protocol. Tissues were de-identified prior to use, with no identifiers to patient names linked to brain samples. Patient confidentiality is strictly kept. As such, the utilization of autopsy derived, de-identified human brain tissue&nbsp;in the current study is exempt from IRB regulations. All tissue was handled with universal precautions. For human postmortem brain tissue requests from the Mayo Clinic Brain Bank, see <a id="intref0285" href="https://www.mayo.edu/research/departments-divisions/department-neuroscience-florida/brain-bank" target="_blank">https://www.mayo.edu/research/departments-divisions/department-neuroscience-florida/brain-bank</a>.</div></section><section id="sec-4-3-2"><h4>Overview of FTLD-TDP cases</h4><div id="p0190" role="paragraph">Filaments for cryo-EM analysis were extracted from fresh-frozen post-mortem brain tissue samples from eight cases with FTLD-TDP, including four with FTLD-TDP type A caused by different <i>GRN</i> mutations (one of which also carried the <i>C9orf72</i> repeat expansion, Case 5) and four sporadic FTLD-TDP (one type A, two type B and one type C), in which <i>GRN</i> and <i>C9orf72</i> mutations were excluded (<a id="crosref1630" href="#mmc1">Table S1</a>). The <i>TMEM106B</i> haplotypes were determined for each case.</div></section><section id="sec-4-3-3"><h4>Clinical history&nbsp;of FTLD-TDP Type A case 1</h4><div id="p01951" role="paragraph">The patient was a 60-year-old man with a five-year history of behavioral variant FTD (bvFTD) and mild parkinsonism. There was a family history of early-onset dementia and genetic sequencing of the patient identified a <i>GRN</i> mutation (c.1072C&gt;T).</div></section><section id="sec-4-3-4"><h4>Clinical history of FTLD-TDP Type A case 2</h4><div id="p0195" role="paragraph">The patient was a 55-year-old woman with a seven-year history of non-fluent variant primary progressive aphasia (nfvPPA) and subsequent behavioral abnormalities. Detailed family history was not available; however, genetic analysis identified a <i>GRN</i> mutation (c.1180-1G&gt;A).</div></section><section id="sec-4-3-5"><h4>Clinical history of FTLD-TDP Type A case 3</h4><div id="p0200" role="paragraph">The patient was a 60-year-old woman with a history of psychosis who developed progressive aphasia and behavioral changes at age 54. There was a family history of dementia and parkinsonism and genetic analysis identified a <i>GRN</i> mutation in the patient (c.102delC).</div></section><section id="sec-4-3-6"><h4>Clinical history of FTLD-TDP Type A case 4</h4><div id="p0205" role="paragraph">The patient was an 89-year-old man with a 10-year history of sporadic nfvPPA.</div></section><section id="sec-4-3-7"><h4>Clinical history of FTLD-TDP Type A case 5</h4><div id="p0210" role="paragraph">The patient was a 48-year-old woman who developed progressive non-fluent aphasia at age 44 and had evidence of mild bulbar weakness the year prior to her death. There was a strong family history of FTD and genetic analysis demonstrated a <i>GRN</i> mutation (c.87_90dup) as well as the abnormal <i>C9orf72</i> repeat expansion in the patient and other affected family members.</div></section><section id="sec-4-3-8"><h4>Pathology of FTLD-TDP Type A cases</h4><div id="p01952" role="paragraph">All five cases with FTLD-TDP type A pathology had abundant compact neuronal cytoplasmic inclusions (NCI), short dystrophic neurites (DN) and occasional lentiform neuronal intranuclear inclusions (NII) concentrated in the upper neocortical laminae (Figure&nbsp;S1). Case 1 and Case 3 also had tau-immunoreactive neurofibrillary tangles (NFT) restricted to the entorhinal cortex and/or hippocampus (Braak stage I and III, respectively). In addition to FTLD-TDP, Case 5 also had mild loss of lower motor neurons in the hypoglossal nucleus with rare TDP-43-positive inclusions and abundant dipeptide repeat immunoreactive neuronal inclusions, characteristic of the C9orf72 repeat expansion.</div></section><section id="sec-4-3-9"><h4>Clinical history and pathology of FTLD-TDP Type B case 1 (case 6)</h4><div id="p0215" role="paragraph">The patient was a 62-year-old man with a four-year history of behavioral and cognitive changes followed by progressive weakness, consistent with amyotrophic lateral sclerosis (ALS). TDP-43 immunohistochemistry demonstrated abundant NCI throughout all layers of the neocortex, with most having a diffuse, granular morphology, typical of FTLD-TDP type B. There was also significant loss of spinal cord lower motor neurons with many containing TDP-43-immunoreactive inclusions. NFT were restricted to the hippocampus (Braak stage III).</div></section><section id="sec-4-3-10"><h4>Clinical history and pathology of FTLD-TDP Type B case 2 (case 7)</h4><div id="p0220" role="paragraph">The patient was a 74-year-old Asian woman who developed progressive limb and bulbar weakness at age 72. Electrophysiological studies revealed denervation in multiple muscle groups, consistent with a diagnosis of ALS. Cognitive assessment at age 73 demonstrated significant problems with short-term memory, attention and abstract thought. Postmortem examination confirmed ALS with degeneration of the corticospinal tracts and loss of lower motor neurons with many of the remaining cells containing TDP-43 immunoreactive cytoplasmic inclusions. TDP-43 pathology in the primary motor cortex and prefrontal cortex consisted of abundant diffuse, granular neuronal cytoplasmic inclusions and small neuropil dots in all cortical laminae, consistent with FTLD-TDP type B. Neurofibrillary tangles were restricted to the hippocampus (Braak stage III/VI).</div></section><section id="sec-4-3-11"><h4>Clinical history and pathology of FTLD-TDP Type C case 1 (case 8)</h4><div id="p0225" role="paragraph">The patient was a 69-year-old man with a 14-year history of progressive language problems with prominent word finding difficulties, characteristic of semantic variant PPA (svPPA), and mild behavioral problems. TDP-43 immunohistochemistry demonstrated numerous DN in the neocortex, many of which were long and tortuous, typical of FTLD-TDP type C. He also had a number of small chronic cerebral infarcts.</div></section><section id="sec-4-3-12"><h4>Clinical history and pathology of FTLD-TDP Type A case 6 (case 9)</h4><div id="p0230" role="paragraph">The patient was a woman who developed bvFTD at age 53 and died at age 60. There was a strong family history of FTD and genetic analysis identified a <i>GRN</i> mutation (c.87_90dup). Post-mortem evaluation demonstrated FTLD-TDP type A as the only significant pathology.</div></section><section id="sec-4-3-13"><h4>Clinical history and pathology of FTLD-TDP Type B case 3 (case 10)</h4><div id="p0235" role="paragraph">The patient was a 58-year-old woman who presented with nfvPPA followed by mild changes in personality and behavior. During the year prior to death, she developed swallowing difficulty and electrophysiological studies demonstrated lower motor dysfunction. There was no family history of neurological disease and no <i>GRN</i> or <i>C9orf72</i> mutations were detected in the patient. Post mortem evaluation demonstrated typical changes of FTLD-TDP type B and ALS with TDP-43-positive inclusions in lower motor neurons. There were rare diffuse senile plaques in the neocortex and neurofibrillary tangles were restricted to the entorhinal/transentorhinal cortex (Braak II/VI).</div></section><section id="sec-4-3-14"><h4>Clinical history and pathology of FTLD-TDP Type C case 2 (case 11)</h4><div id="p0240" role="paragraph">The patient was a 72-year-old woman who developed language difficulty consistent with svPPA, personality and behavioral changes and poor memory in her early sixties. There was no family history of neurological disease and no <i>GRN</i> or <i>C9orf72</i> mutations were detected. Post mortem exam demonstrated FTLD-TDP type C pathology and neurofibrillary tangles restricted to the hippocampus (Braak IV/VI).</div></section><section id="sec-4-3-15"><h4>Clinical history and pathology of PSP case 1</h4><div id="p0245" role="paragraph">The patient was a 68-year-old white man who developed double vision six years prior to death. He subsequently developed slurred speech, bradykinesia, difficulty walking and unexplained falls, clinically diagnosed as Parkinson-plus syndrome. Post-mortem evaluation revealed findings of cortical-predominant PSP with severe neuronal and glial tau pathology in the motor and premotor cortices, and marked white matter oligodendroglial pathology at the gray-white junction. There was no TDP-43 pathology.</div></section><section id="sec-4-3-16"><h4>Clinical history and pathology of PSP case 2</h4><div id="p0250" role="paragraph">The patient was a 75-year-old man with a 10-year history of progressive speech apraxia, cognitive changes and ophthalmoplegia. Gallyas-positive, tau-immunoreactive NFT and glial inclusions, including tufted astrocytes, ramified astrocytes, threads and coil bodies were present throughout the neocortex and numerous subcortical regions, consistent with PSP. There was no TDP-43 or α-synuclein-immunoreactive pathology.</div></section><section id="sec-4-3-17"><h4>Clinical history and pathology of PSP case 3</h4><div id="p0255" role="paragraph">The patient was a 59-year-old woman with eight years of progressive cognitive impairment, behavioral changes, supranuclear gaze palsy, gait problems, dysphagia and akinetic-rigidity, clinically diagnosed as PSP. Typical neuropathological features of PSP were present including numerous tau-positive NFT, tufted astrocytes and other glial inclusions throughout the neocortex and many other brain regions. There was no TDP-43 and α-synuclein pathology.</div></section><section id="sec-4-3-18"><h4>Clinical history and pathology of DLB case 1</h4><div id="p0260" role="paragraph">This 68-year-old man had a 10-year history of parkinsonism, with subsequent cognitive impairment. His parkinsonism was characterized by tremors, stiffness, gait difficulties, falls and handwriting difficulties. His cognitive problems included change in personality, disorientation and visual hallucinations. Post mortem examination demonstrated degeneration of the substantia nigra and locus ceruleus. Histologic studies showed diffuse α-synuclein immunoreactive cortical Lewy bodies and minimal Alzheimer type pathology (Braak stage I, Thal phase 1, CERAD neuritic plaque score 0).</div></section><section id="sec-4-3-19"><h4>Clinical history and pathology of DLB cases used for western blot analysis</h4><div id="p0265" role="paragraph">DLB cases used for western blot analysis had a pathological diagnosis of DLB or variants thereof. Namely, case 2 (male, 57 years old, DLB A53T), case 3 (male, 66 years old, familial DLB), case 4 (female, 62 years old, DLB), case 5 (female, 81 years old, severe DLB), case 6 (female, 71 years old DLB A53T).</div></section><section id="sec-4-3-20"><h4>Clinical history and pathology of neurologically normal cases (NN) used in western blot analysis</h4><div id="p0270" role="paragraph">Neurologically normal cases NN1 (male, 70 years old), NN2 (male, 73 years old), NN3 (female, 73 years old), NN4 (male, 68 years old), and NN5 (female, 70 years old) had no significant neurodegenerative pathology apart from age-related senile plaques (sparse in NN5, moderate in NN3).</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Histology, immunohistochemistry, immunoblotting</h4><div id="p0275" role="paragraph">At the time of publication, an antibody that binds exclusively to TMEM106B(120–254) fibrils does not exist. This has prevented immunohistochemistry experiments to directly visualize TMEM106B(120–254) fibrils in postmortem human tissue using immunofluorescence or immunogold electron microscopy.</div><section id="sec-4-4-1-1"><h5>FTLD-TDP</h5><div id="p0280" role="paragraph">The post-mortem evaluation followed consensus recommendations for the neuropathological assessment of dementia (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Montine et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Montine, T.J. ∙ Phelps, C.H. ∙ Beach, T.G. ..., National Institute on Aging, Alzheimer’s Association</div><div><strong>National Institute on Aging-Alzheimer’s Association guidelines for the neuropathologic assessment of Alzheimer’s disease: a practical approach</strong></div><div><em>Acta Neuropathol.</em> 2012; <strong>123</strong>:1-11</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.J.MontineC.H.PhelpsT.G.BeachE.H.BigioN.J.CairnsD.W.DicksonC.DuyckaertsM.P.FroschE.MasliahS.S.MirraNational+Institute+on+AgingAlzheimer%E2%80%99s+AssociationNational+Institute+on+Aging-Alzheimer%E2%80%99s+Association+guidelines+for+the+neuropathologic+assessment+of+Alzheimer%E2%80%99s+disease%3A+a+practical+approachActa+Neuropathol.1232012111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and FTLD (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Cairns et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Cairns, N.J. ∙ Bigio, E.H. ∙ Mackenzie, I.R. ..., Consortium for Frontotemporal Lobar Degeneration</div><div><strong>Neuropathologic diagnostic and nosologic criteria for frontotemporal lobar degeneration: consensus of the Consortium for Frontotemporal Lobar Degeneration</strong></div><div><em>Acta Neuropathol.</em> 2007; <strong>114</strong>:5-22</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.J.CairnsE.H.BigioI.R.MackenzieM.NeumannV.M.LeeK.J.HatanpaaC.L.White3rdJ.A.SchneiderL.T.GrinbergG.HallidayConsortium+for+Frontotemporal+Lobar+DegenerationNeuropathologic+diagnostic+and+nosologic+criteria+for+frontotemporal+lobar+degeneration%3A+consensus+of+the+Consortium+for+Frontotemporal+Lobar+DegenerationActa+Neuropathol.1142007522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). FTLD-TDP subtypes were assigned in accordance with recent consensus criteria (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Mackenzie et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Mackenzie, I.R. ∙ Neumann, M. ∙ Baborie, A. ...</div><div><strong>A harmonized classification system for FTLD-TDP pathology</strong></div><div><em>Acta Neuropathol.</em> 2011; <strong>122</strong>:111-113</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.R.MackenzieM.NeumannA.BaborieD.M.SampathuD.Du+PlessisE.JarosR.H.PerryJ.Q.TrojanowskiD.M.MannV.M.LeeA+harmonized+classification+system+for+FTLD-TDP+pathologyActa+Neuropathol.1222011111113" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Five μm thick sections of formalin fixed, paraffin embedded tissue were stained using hematoxylin and eosin and Bielschowsky silver methods. Routine diagnostic Immunohistochemistry (IHC) was performed using the Ventana BenchMark XT automated immunostainer with primary antibodies against TDP-43, hyperphosphorylated tau (pS202, pT205), α-synuclein, beta-amyloid, and ubiquitin.</div><div id="p0285" role="paragraph">In an attempt to demonstrate TMEM106B in post-mortem tissue, we tested a number of commercially available antibodies raised against different regions of the protein, using a wide range of IHC protocols. A mouse monoclonal antibody against amino acids 1-46 of the N-terminal region showed a pattern of reactivity consistent with the expected lysosomal localization of physiological TMEM106B, consisting of small cytoplasmic dots in neuronal perikarya and oligodendrocytes. However, this antibody did not label any structures that resembled the pathological inclusions seen with TDP-43 IHC. Despite employing a number of different pretreatment and amplification steps, none of the antibodies against C-terminal epitopes showed any staining that was interpreted as being specific for either physiological or pathological forms of TMEM106B. These included rabbit polyclonal antibodies against TMEM106B epitopes (101-200), (111-190), (150-275), (204-253), and (218-252).</div></section><section id="sec-4-4-1-2"><h5>PSP</h5><div id="p0290" role="paragraph">For Case 1 paraffin sections were immunostained for tau (CP13, 1:1,000) using a DAKO Autostainer. For Cases 2 and 3 sections of formalin fixed, paraffin embedded tissue were stained using hematoxylin and eosin, Bielschowsky and Gallyas silver methods. Immunohistochemistry was performed using the Ventana BenchMark XT automated immunostainer with primary antibodies against TDP-43, hyperphosphorylated tau (pS202, pT205), α-synuclein, beta-amyloid, and ubiquitin.</div></section><section id="sec-4-4-1-3"><h5>DLB</h5><div id="p0295" role="paragraph">Immunohistochemistry for α-synuclein (NACP) was performed after formic acid with the DAKO Autostainer.</div></section></section><section id="sec-4-4-2"><h4>Genetic sequencing</h4><div id="p0300" role="paragraph">DNA was extracted from frozen brain tissue using standard techniques. Sanger sequencing analysis of all exons of the <i>GRN</i> gene and expansion analysis of the GGGGCC repeat in <i>C9orf72</i> using the 2-step protocol was performed in all FTLD-TDP cases as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Baker et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Baker, M. ∙ Mackenzie, I.R. ∙ Pickering-Brown, S.M. ...</div><div><strong>Mutations in progranulin cause tau-negative frontotemporal dementia linked to chromosome 17</strong></div><div><em>Nature.</em> 2006; <strong>442</strong>:916-919</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BakerI.R.MackenzieS.M.Pickering-BrownJ.GassR.RademakersC.LindholmJ.SnowdenJ.AdamsonA.D.SadovnickS.RollinsonMutations+in+progranulin+cause+tau-negative+frontotemporal+dementia+linked+to+chromosome+17Nature4422006916919" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">DeJesus-Hernandez et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>DeJesus-Hernandez, M. ∙ Mackenzie, I.R. ∙ Boeve, B.F. ...</div><div><strong>Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS</strong></div><div><em>Neuron.</em> 2011; <strong>72</strong>:245-256</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.DeJesus-HernandezI.R.MackenzieB.F.BoeveA.L.BoxerM.BakerN.J.RutherfordA.M.NicholsonN.A.FinchH.FlynnJ.AdamsonExpanded+GGGGCC+hexanucleotide+repeat+in+noncoding+region+of+C9ORF72+causes+chromosome+9p-linked+FTD+and+ALSNeuron722011245256" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>TMEM106B</i> SNP rs3173615 (encoding T185S) was genotyped in all cases using an inventoried Taqman SNP genotyping assay C__27465458_10, as described (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Nicholson et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Nicholson, A.M. ∙ Finch, N.A. ∙ Wojtas, A. ...</div><div><strong>TMEM106B p.T185S regulates TMEM106B protein levels: implications for frontotemporal dementia</strong></div><div><em>J.&nbsp;Neurochem.</em> 2013; <strong>126</strong>:781-791</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.M.NicholsonN.A.FinchA.WojtasM.C.BakerR.B.Perkerson3rdM.Castanedes-CaseyL.RousseauL.BenussiG.BinettiR.GhidoniTMEM106B+p.T185S+regulates+TMEM106B+protein+levels%3A+implications+for+frontotemporal+dementiaJ.%C2%A0Neurochem.1262013781791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-3"><h4>Human brain tissue fractionation</h4><section id="sec-4-4-3-1"><h5>All cases except PSP case 1</h5><div id="p0305" role="paragraph">Added 10 volumes (w/v) of cold buffer I (10&nbsp;mM Tris, 800&nbsp;mM NaCl, 1&nbsp;mM EGTA, 5mM EDTA, 10% sucrose, 1mM PMSF, Roche protease and phosphate inhibitor, pH 7.4) to frozen brain tissue from frontal cortex and homogenized by polytron (PT2500E, Kinematica, AG). The homogenized samples were centrifuged at 20,100&nbsp;g for 20&nbsp;min at 4°C and the supernatant was collected. The pellet was resuspended with 1 volume of cold buffer I, homogenized again and centrifuged at 21,100&nbsp;g for 20&nbsp;min at 4°C. The supernatant was then collected and combined with saved supernatant after the 1st centrifugation step. The supernatant was incubated with sarkosyl at a final concentration of 1% with gently shaking for 1&nbsp;h at room temperature. Subsequently, the samples were centrifuged at 100,000&nbsp;g for 1&nbsp;h at 4°C. The final pellet was resuspended in 20&nbsp;μl cold buffer II (5&nbsp;mM Tris pH 7.4) and saved for cryo-EM grid preparation. With regard to α-synuclein fibrils in our DLB case, we note that the tissue fractionation protocol used in the present study is similar to a previous cryo-EM paper (<span class="dropBlock reference-citations"><a id="body-ref-sref50-3" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-3" aria-controls="sref50-3">Schweighauser et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref50-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Schweighauser, M. ∙ Shi, Y. ∙ Tarutani, A. ...</div><div><strong>Structures of α-synuclein filaments from multiple system atrophy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:464-469</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserY.ShiA.TarutaniF.KametaniA.G.MurzinB.GhettiT.MatsubaraT.TomitaT.AndoK.HasegawaStructures+of+%CE%B1-synuclein+filaments+from+multiple+system+atrophyNature5852020464469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, we did not observe thin (&lt; 100&nbsp;Å wide), untwisted fibrils characteristic of α-synuclein filaments in our images. Similarly, despite tissue fractionating type B case 2 (case 7) using our protocol or a recently published protocol for obtaining TDP-43 fibrils (<span class="dropBlock reference-citations"><a id="body-ref-sref3-3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-3" aria-controls="sref3-3">Arseni et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref3-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Arseni, D. ∙ Hasegawa, M. ∙ Murzin, A.G. ...</div><div><strong>Structure of pathological TDP-43 filaments from ALS with FTLD</strong></div><div><em>Nature.</em> 2022; <strong>601</strong>:139-143</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ArseniM.HasegawaA.G.MurzinF.KametaniM.AraiM.YoshidaB.Ryskeldi-FalconStructure+of+pathological+TDP-43+filaments+from+ALS+with+FTLDNature6012022139143" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we do not observe a significant number of TDP-43 fibrils in our images. Although we performed identical purification methods to isolate filaments from neurologically normal tissue samples (n&nbsp;= 5), these cases all had markedly smaller sarkosyl-insoluble pellets than those derived from diseased tissue (2-3 times smaller per gram of frontal cortex).</div></section><section id="sec-4-4-3-2"><h5>PSP case 1</h5><div id="p0310" role="paragraph">To obtain a sufficient yield of tau filaments from PSP tissue, we evaluated the amount of tau present within the sarkosyl-insoluble (P3) fraction from four different brain regions in three different PSP cases, without pronase-treatment to preserve posttranslational modifications (<span class="dropBlock reference-citations"><a id="body-ref-sref2-10" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-10" aria-controls="sref2-10">Arakhamia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Carlomagno et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Carlomagno, Y. ∙ Manne, S. ∙ DeTure, M. ...</div><div><strong>The AD tau core spontaneously self-assembles and recruits full-length tau to filaments</strong></div><div><em>Cell Rep.</em> 2021; <strong>34</strong>:108843</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.CarlomagnoS.ManneM.DeTureM.PrudencioY.J.ZhangR.Hanna+Al-ShaikhJ.A.DunmoreL.M.DaughrityY.SongM.Castanedes-CaseyThe+AD+tau+core+spontaneously+self-assembles+and+recruits+full-length+tau+to+filamentsCell+Rep.342021108843" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As tau levels were consistently highest in the P3 fractions from caudate and frontal cortex, we screened these two regions in an additional 12 PSP cases. Using EM, we observed a tractable filament&nbsp;yield (∼0.006&nbsp;mg/mL) with low background in the P3 fraction from the caudate of case #1 (<a id="crosref1740" href="#figs3">Figure&nbsp;S3</a>D). To further characterize the PSP case used for cryo-EM, we evaluated total and phosphorylated tau levels in the P3 fraction of case #1 and 3 additional PSP cases by western blot (<a id="crosref1745" href="#figs2">Figure&nbsp;S2</a>C). The similarities in positivity for common disease-associated phosphoepitopes indicates case #1 is a typical PSP case (<a id="crosref1750" href="#figs2">Figure&nbsp;S2</a>C), suggesting that the morphology of tau filaments in this case will be representative of PSP.</div></section></section><section id="sec-4-4-4"><h4>Mass spectrometry</h4><div id="p0315" role="paragraph">A sample imaged by cryo-EM, FTLD-TDP type A case 1, was incubated in 4M guanidinium hydrochloride to disaggregate the amyloid fibrils. In solution, digestion was performed using LysC and trypsin, followed by desalting and LC-MS/MS for protein identification and quantification. The data were processed by MaxQuant (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Cox and Mann, 2008</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cox, J. ∙ Mann, M.</div><div><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong></div><div><em>Nat. Biotechnol.</em> 2008; <strong>26</strong>:1367-1372</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.CoxM.MannMaxQuant+enables+high+peptide+identification+rates%2C+individualized+p.p.b.-range+mass+accuracies+and+proteome-wide+protein+quantificationNat.+Biotechnol.26200813671372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mass spectrometry data was searched against Uniprot human protein database. The TMEM106B fragment that was identified by mass spectrometry in FTLD-TDP type A case 1 was TMEM106B(130-139) (<a id="crosref1635" href="#mmc2">Table S5</a>A). Mass spectrometry was performed in an identical manner on a second sample, FTLD-TDP type A case 6 (case 9), to identify proteins and phosphorylation sites (<a id="crosref1640" href="#mmc2">Table S5</a>). To identify phosphorylation sites, the digest was enriched with TiO<sub>2</sub> for phosphopeptides before being subjected to LC-MS/MS. The TMEM106B fragments that were identified by mass spectrometry in FTLD-TDP type A case 6 (case 9) were TMEM106B(130-139) and TMEM106B(248-255) (<a id="crosref1645" href="#mmc2">Table S5</a>B), with no phosphorylation sites detected on these peptides (<a id="crosref1650" href="#mmc2">Table S5</a>C). Raw data was deposited on MassIVE (Project Accession # MSV000088875).</div></section><section id="sec-4-4-5"><h4>Western blots of sarkosyl-insoluble fractions from donor brains</h4><div id="p0320" role="paragraph">Sarkosyl-insoluble pellets from fractionated tissue samples were resolved on Novex WedgeWell 4%–20% Tris-Glycine gels (Invitrogen) and electrophoresis performed at 25 Milliamps, 2 h. Proteins were dry transferred onto 0.2&nbsp;μm PVDF membranes using iBlot 2 Mini Stacks in an iBlot 2 Gel Transfer Device (Invitrogen) at 23 Volts for 6&nbsp;min. Before blocking in Intercept Blocking Buffer (LI-COR), membranes were fixed in 1% paraformaldehyde for 30&nbsp;min at room temperature. Blots were incubated with primary antibody TMEM106B against epitope (204-253) overnight at 4 degrees. Membranes were washed three times (10&nbsp;min each) in PBS-T, then incubated with secondary antibody IRDye 800CW Donkey anti-Rabbit at 1:10,000 for 1&nbsp;h at room temperature. Membranes were washed three times (10&nbsp;min each) in PBS-T and imaged on LI-COR Odyssey CLx.</div></section><section id="sec-4-4-6"><h4>Predicted native fold of TMEM106B and aggregation propensity</h4><div id="p0325" role="paragraph">The native structure of TMEM106B was predicted used AlphaFold (<span class="dropBlock reference-citations"><a id="body-ref-sref33-4" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-4" aria-controls="sref33-4">Jumper et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref33-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Jumper, J. ∙ Evans, R. ∙ Pritzel, A. ...</div><div><strong>Highly accurate protein structure prediction with AlphaFold</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:583-589</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.JumperR.EvansA.PritzelT.GreenM.FigurnovO.RonnebergerK.TunyasuvunakoolR.BatesA.%C5%BD%C3%ADdekA.PotapenkoHighly+accurate+protein+structure+prediction+with+AlphaFoldNature5962021583589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), TrRosetta (<span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Yang, J. ∙ Anishchenko, I. ∙ Park, H. ...</div><div><strong>Improved protein structure prediction using predicted interresidue orientations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:1496-1503</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.YangI.AnishchenkoH.ParkZ.PengS.OvchinnikovD.BakerImproved+protein+structure+prediction+using+predicted+interresidue+orientationsProc.+Natl.+Acad.+Sci.+USA117202014961503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and Robetta (<span class="dropBlock reference-citations"><a id="body-ref-sref35-3" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-3" aria-controls="sref35-3">Kim et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref35-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Kim, D.E. ∙ Chivian, D. ∙ Baker, D.</div><div><strong>Protein structure prediction and analysis using the Robetta server</strong></div><div><em>Nucleic Acids Res.</em> 2004; <strong>32</strong>:W526-W531</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.E.KimD.ChivianD.BakerProtein+structure+prediction+and+analysis+using+the+Robetta+serverNucleic+Acids+Res.322004W526-W531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All three methods converged on highly similar structures of the full length-protein with the most variability predicted to be in the disordered N terminus. Predicted structures of TMEM106B(120-254) using these tools have a backbone root mean square deviation&nbsp;&lt; 2&nbsp;Å. The aggregation propensity of each TMEM106B(120-254) amino acid at pH 5 were predicted using CamSol (<span class="dropBlock reference-citations"><a id="body-ref-sref55-2" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-2" aria-controls="sref55-2">Sormanni et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Sormanni, P. ∙ Aprile, F.A. ∙ Vendruscolo, M.</div><div><strong>The CamSol method of rational design of protein mutants with enhanced solubility</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 2015; <strong>427</strong>:478-490</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.SormanniF.A.AprileM.VendruscoloThe+CamSol+method+of+rational+design+of+protein+mutants+with+enhanced+solubilityJ.%C2%A0Mol.+Biol.4272015478490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with an average insolubility of −1.39. Predictions of TMEM106B(120-254) aggregation propensity at lysosomal pH 5 and cytosolic pH 7 using the CAMSOL method are similar (integer pH values only possible).</div></section><section id="sec-4-4-7"><h4>Cryo-electron microscopy</h4><section id="sec-4-4-7-1"><h5>All cases except PSP case 1</h5><div id="p0330" role="paragraph">For cryoEM, 3&nbsp;μl aliquots of purified TMEM106B filaments from human brain tissue was applied to glow-discharged holey carbon Quantifoil Au R1.2/1.3 grids (Electron Microscopy Science, Hatfield, PA), blotted with filter paper to create a thin film by removing excess sample using a FEI Vitrobot Mark III at 25°C and 100% humidity (blot force setting 0 and blotting time between 3.5–5.5 s) and plunge-frozen into liquid ethane. High-resolution data was collected primarily at the Zuckerman Institute, Columbia University with additional movies collected at the Columbia University Cryo-EM facility and New York Structural Biology Center (NYSBC), on Titan Krios microscopes equipped with BioQuantum-K3 imaging systems (see <a id="crosref1655" href="#mmc1">Tables S2</a> and <a id="crosref1660" href="#mmc1">S3</a> for details). All movies were collected in counting mode with a nominal pixel size of 1.07&nbsp;Å/pixel, dose rate of 1.5 e/Å<sup>2</sup>/frame, over 40 frames. The underfocus range was set between 1.4 and 2.8&nbsp;μm (<a id="crosref1665" href="#mmc1">Tables S2</a> and <a id="crosref1670" href="#mmc1">S3</a>).</div></section><section id="sec-4-4-7-2"><h5>PSP Case 1</h5><div id="p0335" role="paragraph">3&nbsp;μL aliquots of purified tau filaments from PSP human brain tissue (∼0.006&nbsp;mg/mL) were applied to glow-discharged holey carbon grids (Quantifoil Au R1.2/1.3, 300 mesh), blotted with filter paper and plunge-frozen into liquid ethane using an FEI Vitrobot Mark IV. Images of the frozen grids were collected on three FEI Titan Krios microscopes on multiple occasions due to the paucity of the tau filaments. Imaging conditions were always identical (<a id="crosref1675" href="#mmc1">Table S3</a>), but minute differences in Å/pixel had to be corrected for during image processing. All images were acquired at 300 kV and energy filtered (slit width of 20 eV) using a Gatan K2/K3 Summit detector in counting mode. All movies were collected at a nominal pixel size of 1.0605&nbsp;Å, using a dose rate of 1.5 electrons per Å<sup>2</sup> per frame and an underfocus range of 1 to 3&nbsp;μm (<a id="crosref1680" href="#mmc1">Table S3</a>).</div></section></section><section id="sec-4-4-8"><h4>Helical reconstruction</h4><div id="p0340" role="paragraph">Movie frames were gain-corrected, aligned, dose-weighted, and summed using MotionCor (<span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.-P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.Q.ZhengE.PalovcakJ.-P.ArmacheK.A.VerbaY.ChengD.A.AgardMotionCor2%3A+anisotropic+correction+of+beam-induced+motion+for+improved+cryo-electron+microscopyNat.+Methods142017331332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) implemented in RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Zivanov et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Since amyloid fibrils extend over a few microns, spanning entire micrographs, during motion correction no patching was employed. Parameters for the contrast transfer function were estimated from each motion-corrected micrograph using Gctf (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Zhang, 2016</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.ZhangGctf%3A+Real-time+CTF+determination+and+correctionJ.%C2%A0Struct.+Biol.1932016112" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All subsequent image processing was mainly performed using helical reconstruction in RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">He and Scheres, 2017</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>He, S. ∙ Scheres, S.H.W.</div><div><strong>Helical reconstruction in RELION</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2017; <strong>198</strong>:163-176</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.HeS.H.W.ScheresHelical+reconstruction+in+RELIONJ.%C2%A0Struct.+Biol.1982017163176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref25-5" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-5" aria-controls="sref25-5">Fitzpatrick et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref25-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Fitzpatrick, A.W.P. ∙ Falcon, B. ∙ He, S. ...</div><div><strong>Cryo-EM structures of tau filaments from Alzheimer’s disease</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:185-190</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickB.FalconS.HeA.G.MurzinG.MurshudovH.J.GarringerR.A.CrowtherB.GhettiM.GoedertS.H.W.ScheresCryo-EM+structures+of+tau+filaments+from+Alzheimer%E2%80%99s+diseaseNature5472017185190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Singlet and doublet fibrils were manually picked and separated with initial segmentation at 10.7&nbsp;Å/pixel and then 3.21&nbsp;Å/pixel to determine helical parameters (<a id="crosref1685" href="#mmc1">Table S2</a>). Initial models of the singlet and doublet fibrils were generated from 2D class averages of helical pitch-views of the fibrils using a cylinder as an initial model and helically reconstructed in C1 using 3D classification with a T regularization value of 20. The doublet initial model quickly converged using FTLD-TDP particles; the singlet fibril initial model was generated from a PSP dataset in the following manner.</div><div id="p0345" role="paragraph">Helical processing of TMEM106B(120-254) singlet fibrils from PSP case 1 began with segments extracted at 3.1815&nbsp;Å/pixel and 2D classified to determine the helical parameters. The low twist angles hampered initial model generation and so a cylinder was used as a starting model in 3D classification of 8 classes with a T regularization value of 20. One class consistently showed a C-shaped protofilament, irrespective of starting model (which was low-pass filtered to 40&nbsp;Å). Particles from this class were re-extracted at 1.0605&nbsp;Å/pixel and 3D classification with local optimization of the helical twist and rise was performed with T&nbsp;= 100, decreasing the angular and translational sampling as the rotational and translational accuracy of the helical 3D reconstruction improved. A final subset of particles was 3D auto-refined with T&nbsp;= 100 and postprocessed in RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Zivanov et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Per-particle estimates of ctf values were used to improve resolution during a final refinement with an overall resolution of 4.4&nbsp;Å being attained. This map had a fairly even distribution of views and was used as an initial model for refinement of TMEM106B(120-254) singlet fibrils from all other cases.</div><div id="p0350" role="paragraph">To achieve a high-resolution doublet fibril map, particles from selected 3D classes were re-extracted at 1.07&nbsp;Å/pixel and re-aligned and averaged to generate 2D classes. Optimal 2D class averages showing clear β strand separation were selected and used for helical reconstruction in C1 using 3D classification with a T regularization value of 100. A persistent 2-fold symmetry was apparent in the doublet fibril reconstructions and this was imposed in subsequent 3D auto-refinements. Other symmetries, such as a 2-start helix or a helix with opposite hand, were tested but failed to attain high-resolution. Helical twist and rise were optimized during 3D auto-refinements&nbsp;(<a id="crosref1690" href="#mmc1">Table S2</a>) and once the resolution of the 3D reconstruction was better than 4&nbsp;Å, particles were corrected using Bayesian polishing and local ctf parameter refinement in RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref67-3" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-3" aria-controls="sref67-3">Zivanov et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref67-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to improve resolution further. Due to the low average inter-strand twist of approximately −0.4°, many reconstructions appeared to have anisotropic cross-sections. The two-fold symmetry of the doublet fibril was exploited to correct for this anisotropy by invoking relion_particle_symmetry_expand and refining the two masked, subtracted, protofilaments of the doublet cross-section separately, then combining the aligned particles in a final 3D auto-refinement. This improved our reconstructions significantly and a combined, sharpened map using the best particles from each FTLD-TDP case was generated and used for model building. Postprocessing was performed in RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref67-4" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-4" aria-controls="sref67-4">Zivanov et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref67-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and DeepEMHancer (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Sanchez-Garcia et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Sanchez-Garcia, R. ∙ Gomez-Blanco, J. ∙ Cuervo, A. ...</div><div><strong>DeepEMhancer: a deep learning solution for cryo-EM volume post-processing</strong></div><div><em>Commun Biol.</em> 2021; <strong>4</strong>:874</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.Sanchez-GarciaJ.Gomez-BlancoA.CuervoJ.M.CarazoC.O.S.SorzanoJ.VargasDeepEMhancer%3A+a+deep+learning+solution+for+cryo-EM+volume+post-processingCommun+Biol.42021874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) via COSMIC (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Cianfrocco et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Cianfrocco, M.A. ∙ Wong-Barnum, M. ∙ Youn, C. ...</div><div><strong>COSMIC<sup>2</sup>: A Science Gateway for Cryo-Electron Microscopy Structure Determination</strong></div><div><strong>Proceedings of the Practice and Experience in Advanced Research Computing 2017 on Sustainability, Success and Impact</strong></div><div>2017</div><div>Published online July 7, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.A.CianfroccoM.Wong-BarnumC.YounR.WagnerA.E.LeschzinerCOSMIC2%3A+A+Science+Gateway+for+Cryo-Electron+Microscopy+Structure+DeterminationProceedings+of+the+Practice+and+Experience+in+Advanced+Research+Computing+2017+on+Sustainability%2C+Success+and+Impact201710.1145%2F3093338.3093390Published+online+July+7%2C+2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Local resolution estimates of the combined map were determined using RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref67-5" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-5" aria-controls="sref67-5">Zivanov et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref67-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and displayed using Chimera (<span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Pettersen et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF Chimera--a visualization system for exploratory research and analysis</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2004; <strong>25</strong>:1605-1612</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.F.PettersenT.D.GoddardC.C.HuangG.S.CouchD.M.GreenblattE.C.MengT.E.FerrinUCSF+Chimera--a+visualization+system+for+exploratory+research+and+analysisJ.%C2%A0Comput.+Chem.25200416051612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (Figure&nbsp;S4). The final overall resolution estimates were calculated from Fourier shell correlations at 0.143 between the two independently refined half-maps in RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref67-6" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-6" aria-controls="sref67-6">Zivanov et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref67-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (Figure&nbsp;S4).</div><div id="p0355" role="paragraph">Helical processing of tau filaments from PSP case 1 was performed using RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref27-3" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-3" aria-controls="sref27-3">He and Scheres, 2017</a><div class="dropBlock__holder" data-db-target-of="sref27-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>He, S. ∙ Scheres, S.H.W.</div><div><strong>Helical reconstruction in RELION</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2017; <strong>198</strong>:163-176</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.HeS.H.W.ScheresHelical+reconstruction+in+RELIONJ.%C2%A0Struct.+Biol.1982017163176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as previously described&nbsp;(<span class="dropBlock reference-citations"><a id="body-ref-sref25-6" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-6" aria-controls="sref25-6">Fitzpatrick et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref25-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Fitzpatrick, A.W.P. ∙ Falcon, B. ∙ He, S. ...</div><div><strong>Cryo-EM structures of tau filaments from Alzheimer’s disease</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:185-190</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickB.FalconS.HeA.G.MurzinG.MurshudovH.J.GarringerR.A.CrowtherB.GhettiM.GoedertS.H.W.ScheresCryo-EM+structures+of+tau+filaments+from+Alzheimer%E2%80%99s+diseaseNature5472017185190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Twisted tau filaments were manually selected and segments were extracted at various Å/pixel (1&nbsp;×, 2&nbsp;×, 3&nbsp;×, 5&nbsp;× binned) and 2D classified to determine the helical parameters and ascertain whether - or not - the filament had C2 symmetry. Initial models were generated from 2D class averages of helical pitch views and used as a reference for helical reconstruction in C1 using 3D classification with a T regularization value of 100. When a clear chain trace emerged in the helical cross-section at 1.0605&nbsp;Å/pixel, a final subset of particles was 3D auto-refined with T&nbsp;= 20.</div><div id="p0360" role="paragraph">During the preparation of this manuscript, the structure of tau filaments from a PSP case was solved to a resolution of 2.7&nbsp;Å. The public release of this high-resolution map (EMD-13218) and raw micrographs (EMPIAR-10765) helped in resolving continuous main-chain density in 3D reconstructions of tau filaments from PSP case 1 using our final subset of particles. Helical twist and rise were optimized in the range −0.7° to −0.9° and 4.7&nbsp;Å to 4.8&nbsp;Å, respectively. Once the helical parameters were optimized (<a id="crosref1695" href="#mmc1">Table S3</a>), helical symmetry remained fixed in all subsequent refinements. Postprocessing was performed in RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref67-7" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-7" aria-controls="sref67-7">Zivanov et al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref67-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and DeepEMHancer (<span class="dropBlock reference-citations"><a id="body-ref-sref49-3" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-3" aria-controls="sref49-3">Sanchez-Garcia et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref49-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Sanchez-Garcia, R. ∙ Gomez-Blanco, J. ∙ Cuervo, A. ...</div><div><strong>DeepEMhancer: a deep learning solution for cryo-EM volume post-processing</strong></div><div><em>Commun Biol.</em> 2021; <strong>4</strong>:874</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.Sanchez-GarciaJ.Gomez-BlancoA.CuervoJ.M.CarazoC.O.S.SorzanoJ.VargasDeepEMhancer%3A+a+deep+learning+solution+for+cryo-EM+volume+post-processingCommun+Biol.42021874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) via COSMIC (<span class="dropBlock reference-citations"><a id="body-ref-sref14-3" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-3" aria-controls="sref14-3">Cianfrocco et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Cianfrocco, M.A. ∙ Wong-Barnum, M. ∙ Youn, C. ...</div><div><strong>COSMIC<sup>2</sup>: A Science Gateway for Cryo-Electron Microscopy Structure Determination</strong></div><div><strong>Proceedings of the Practice and Experience in Advanced Research Computing 2017 on Sustainability, Success and Impact</strong></div><div>2017</div><div>Published online July 7, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.A.CianfroccoM.Wong-BarnumC.YounR.WagnerA.E.LeschzinerCOSMIC2%3A+A+Science+Gateway+for+Cryo-Electron+Microscopy+Structure+DeterminationProceedings+of+the+Practice+and+Experience+in+Advanced+Research+Computing+2017+on+Sustainability%2C+Success+and+Impact201710.1145%2F3093338.3093390Published+online+July+7%2C+2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) also aiding interpretation of the maps. The final overall resolution of 4.2&nbsp;Å was calculated from Fourier shell correlations at 0.143 between the two independently refined half-maps in RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref67-8" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-8" aria-controls="sref67-8">Zivanov et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref67-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1700" href="#mmc1">Table S3</a>). Tau filaments from PSP case 3 were extremely sparse, with only 3,942 particles at 3.22&nbsp;Å/pixel being used in a final 3D auto-refinement with T&nbsp;= 4 resulting in a map consistent in size and shape with EMD-13218 low-pass filtered to 6.5&nbsp;Å (<a id="crosref1755" href="#figs3">Figure&nbsp;S3</a>E).</div></section><section id="sec-4-4-9"><h4>Model building</h4><div id="p0365" role="paragraph">A continuous Cα chain was generated from the combined map using DeepTracer (<span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Pfab et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Pfab, J. ∙ Phan, N.M. ∙ Si, D.</div><div><strong>DeepTracer for fast de novo cryo-EM protein structure modeling and special studies on CoV-related complexes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong></div><div>e2017525118</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.PfabN.M.PhanD.SiDeepTracer+for+fast+de+novo+cryo-EM+protein+structure+modeling+and+special+studies+on+CoV-related+complexesProc.+Natl.+Acad.+Sci.+USA1182021e2017525118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Tryptic digest-mass spectrometry of a fibril sample extracted from FTLD-TDP type A case 1 identified 600 proteins which we used to help identify the fibril’s constituent protein. By cross-referencing the proteins in the sample identified by mass spectrometry to the 2.7&nbsp;Å density map using cryo-ID (<span class="dropBlock reference-citations"><a id="body-ref-sref28-3" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-3" aria-controls="sref28-3">Ho et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref28-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ho, C.M. ∙ Li, X. ∙ Lai, M. ...</div><div><strong>Bottom-up structural proteomics: cryoEM of protein complexes enriched from the cellular milieu</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:79-85</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.HoX.LiM.LaiT.C.TerwilligerJ.R.BeckJ.WohlschlegelD.E.GoldbergA.W.P.FitzpatrickZ.H.ZhouBottom-up+structural+proteomics%3A+cryoEM+of+protein+complexes+enriched+from+the+cellular+milieuNat.+Methods1720207985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and <i>findMySequence</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref13-3" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-3" aria-controls="sref13-3">Chojnowski et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref13-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Chojnowski, G. ∙ Simpkin, A.J. ∙ Leonardo, D.A. ...</div><div><strong><i>findMySequence</i>: a neural-network-based approach for identification of unknown proteins in X-ray crystallography and cryo-EM</strong></div><div><em>IUCrJ.</em> 2021; <strong>9</strong>:86-97</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.ChojnowskiA.J.SimpkinD.A.LeonardoW.Seifert-DavilaD.E.Vivas-RuizR.M.KeeganD.J.RigdenfindMySequence%3A+a+neural-network-based+approach+for+identification+of+unknown+proteins+in+X-ray+crystallography+and+cryo-EMIUCrJ920218697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we unambiguously determined that the fibril was composed of TMEM106B(120-254). TMEM106B(120-254) sidechains were manually added to the 135 amino acid Cα chain and iteratively improved using real-space refinement in COOT (<span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.EmsleyB.LohkampW.G.ScottK.CowtanFeatures+and+development+of+CootActa+Crystallogr.+D+Biol.+Crystallogr.662010486501" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A 3-chain stack of identical rungs was generated in Chimera and multiple rounds of backbone and sidechain geometry refinement were performed manually followed by local real-space refinement in COOT (<span class="dropBlock reference-citations"><a id="body-ref-sref17-3" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-3" aria-controls="sref17-3">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref17-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.EmsleyB.LohkampW.G.ScottK.CowtanFeatures+and+development+of+CootActa+Crystallogr.+D+Biol.+Crystallogr.662010486501" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For refinement of the atomic model built into the combined map (<a id="crosref1455" href="#fig4">Figures 4</a> and S4), we used phenix.real_space_refine applying secondary structure, non-crystallographic symmetry, and hydrogen bond restraints (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Afonine et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Afonine, P.V. ∙ Poon, B.K. ∙ Read, R.J. ...</div><div><strong>Real-space refinement in PHENIX for cryo-EM and crystallography</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:531-544</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.V.AfonineB.K.PoonR.J.ReadO.V.SobolevT.C.TerwilligerA.UrzhumtsevP.D.AdamsReal-space+refinement+in+PHENIX+for+cryo-EM+and+crystallographyActa+Crystallogr.+D+Struct.+Biol.742018531544" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Once we had a good atomic model of the TMEM106B(120-254) chain, additional glycans were added to match the density map in COOT (<span class="dropBlock reference-citations"><a id="body-ref-sref17-4" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-4" aria-controls="sref17-4">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref17-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.EmsleyB.LohkampW.G.ScottK.CowtanFeatures+and+development+of+CootActa+Crystallogr.+D+Biol.+Crystallogr.662010486501" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A final refinement of the atomic model, including glycans was then performed and validated using MolProbity (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Chen et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Chen, V.B. ∙ Arendall, 3rd, W.B. ∙ Headd, J.J. ...</div><div><strong>MolProbity: all-atom structure validation for macromolecular crystallography</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:12-21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.B.ChenW.B.Arendall3rdJ.J.HeaddD.A.KeedyR.M.ImmorminoG.J.KapralL.W.MurrayJ.S.RichardsonD.C.RichardsonMolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallographyActa+Crystallogr.+D+Biol.+Crystallogr.6620101221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1705" href="#mmc1">Table S2</a>). All other structures were generated using this atomic model as an initial reference and iteratively refined in an identical manner. Figures were made using Pymol (<a id="intref0290" href="https://www.pymol.org" target="_blank">https://www.pymol.org</a>), Chimera (<span class="dropBlock reference-citations"><a id="body-ref-sref45-3" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-3" aria-controls="sref45-3">Pettersen et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref45-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF Chimera--a visualization system for exploratory research and analysis</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2004; <strong>25</strong>:1605-1612</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.F.PettersenT.D.GoddardC.C.HuangG.S.CouchD.M.GreenblattE.C.MengT.E.FerrinUCSF+Chimera--a+visualization+system+for+exploratory+research+and+analysisJ.%C2%A0Comput.+Chem.25200416051612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), atom2svg (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">biochem-fan</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div>biochem-fan atom2svg.py. <a id="intref0305" href="https://gist.github.com/biochem-fan/026ec2f191fee9285424d12fc2b84ce7" target="_blank">https://gist.github.com/biochem-fan/026ec2f191fee9285424d12fc2b84ce7</a>.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=biochem-fan+atom2svg.py.+https%3A%2F%2Fgist.github.com%2Fbiochem-fan%2F026ec2f191fee9285424d12fc2b84ce7." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and BioRender (<a id="intref0295" href="http://biorender.com" target="_blank">biorender.com</a>).</div></section><section id="sec-4-4-10"><h4>Predicted enzymatic cleavage</h4><div id="p0370" role="paragraph">To investigate potential proteases responsible for TMEM106B cleavage at position 120 we utilized the ProCleave server (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Li et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, F. ∙ Leier, A. ∙ Liu, Q. ...</div><div><strong>Procleave: Predicting Protease-specific Substrate Cleavage Sites by Combining Sequence and Structural Information</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2020; <strong>18</strong>:52-64</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.LiA.LeierQ.LiuY.WangD.XiangT.AkutsuG.I.WebbA.I.SmithT.Marquez-LagoJ.LiJ.SongProcleave%3A+Predicting+Protease-specific+Substrate+Cleavage+Sites+by+Combining+Sequence+and+Structural+InformationGenomics+Proteomics+Bioinformatics1820205264" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the MEROPS database (<span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Rawlings et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Rawlings, N.D. ∙ Barrett, A.J. ∙ Thomas, P.D. ...</div><div><strong>The MEROPS database of proteolytic enzymes, their substrates and inhibitors in 2017 and a comparison with peptidases in the PANTHER database</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D624-D632</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.D.RawlingsA.J.BarrettP.D.ThomasX.HuangA.BatemanR.D.FinnThe+MEROPS+database+of+proteolytic+enzymes%2C+their+substrates+and+inhibitors+in+2017+and+a+comparison+with+peptidases+in+the+PANTHER+databaseNucleic+Acids+Res.46D12018D624D632" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The MEROPS database of proteolytic enzymes, their substrates and inhibitors in 2017 and a comparison with peptidases in the PANTHER database. Nucleic Acids Res 46, D624-D632.) ProCleave results with scores above 0.5 at position TMEM106B(120) having the highest score are listed in <a id="crosref1710" href="#mmc1">Table S4</a>: Granzyme A (single site), Thrombin (3 sites) and Kallikrein-related peptidase 4 (3 sites). Cathepsin P is not currently in the ProCleave suite but does have a predicted cleavage motif in MEROPS consistent with selective cleavage at position 120 of TMEM106B.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0375" role="paragraph">The 0.143 Fourier Shell Correlation (FSC) criterion was used to estimate resolution of cryo-EM density maps.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0380" role="paragraph">The authors are grateful to Dr. R.S. Mann for helpful discussions and to the patients and their families for their participation in this work. Human biological samples and associated data were obtained from the Mayo Clinic Brain Bank and UBC. This work was supported by the National Institutes of Health (NIH)/National Institute of Neurological Disorder and Stroke (U01NS110438 to L.P. and A.W.P.F, U54NS110435 to D.W.D. and A.W.P.F.); the Association for Frontotemporal Degeneration; Canadian Institutes of Health Research (74580 to I.R.A.M.); MCDB Neurodegenerative Disease Fund (to M.H.B.S.). The authors are grateful to Simon Cheung for performing IHC, Dr. H. Uryu for electron microscopy, Drs. X. Chen and K. Song for help collecting data on the Titan Krios at the University of Massachusetts Medical School Cryo-EM Core Facility, and R. Grassucci and Dr. Z. Zhang for help collecting data at the Columbia University Cryo-Electron Microscopy Center including the Titan Krios housed at the Zuckerman Institute and the Titan Krios housed at the Simons Electron Microscopy Center and National Resource for Automated Molecular Microscopy (New York Structural Biology Center). This paper is dedicated to the memory of Mr. Patrick Trainor.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0385" role="paragraph">A.W.P.F., I.R.A.M., and M.H.B.S. conceived the experiments. I.R.A.M. identified patients and performed neuropathology on FTLD-TDP cases and PSP cases 2 and 3. D.W.D. and M.D. identified patients and performed neuropathology on PSP case 1. A.W.P.F. and C.L. purified amyloid fibrils from FTLD-TDP cases, PSP cases 2 and 3, and DLB. A.W.P.F. and C.L. performed western blots on sarkosyl-insoluble pellets fractionated from FTLD-TDP, DLB, and non-pathological tissue samples. Y.C. purified amyloid fibrils from PSP case 1. A.W.P.F., J.W., and C.W. performed cryo-EM experiments. A.W.P.F., A.C., X.X., J.W., T.A., and M.S. processed cryo-EM data. A.W.P.F., C.L., and G.Z. performed and analyzed mass spectrometry experiments. Y.C., D.W.D., and L.P. provided a PSP fibril sample. J.P., B.H., L.M.F., M.L.D., and R.R. made intellectual contributions. R.R. performed genetic analyses on all FTLD-TDP cases. A.W.P.F., X.X., J.W., and M.H.B.S. built and refined atomic models. A.W.P.F., A.C., X.X., J.W., T.A., M.S., S.D., M.T., C.N.C., R.R., D.W.D., L.P., M.H.B.S., and I.R.A.M. prepared the manuscript. M.H.B.S., I.R.A.M., and A.W.P.F. supervised the project.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0390" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (2)</h2><div id="p0400" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.02.026/attachment/d4e4f32a-492a-4102-8a27-02bcec3ec65c/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (124.06 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Document S1. Tables S1–S4</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.02.026/attachment/9a1f9262-7932-4c51-97e4-17736e26eba6/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (675.93 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S5. All mass spectrometry raw data on FTLD-TDP type A case 1 and FTLD-TDP type A case 6 (case 9), related to Figures 3–5 and STAR Methods</div><div id="espara0010" role="paragraph">(A) All peptides identified in sarkosyl-insoluble pellet fractionated from FTLD-TDP type A case 1. (B) All peptides identified in sarkosyl-insoluble pellet fractionated from FTLD-TDP type A case 6 (case 9). (C) All phosphorylation sites identified in sarkosyl-insoluble pellet fractionated from FTLD-TDP type A case 6 (case 9).</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">All cryo-EM maps have been deposited to the Electron Microscopy Data Bank (EMDB; <a id="intref0010" href="https://www.ebi.ac.uk/pdbe/emdb/" target="_blank">https://www.ebi.ac.uk/pdbe/emdb/</a>). All refined atomic models have been deposited to the Protein Data Bank (PDB; <a id="intref0015" href="https://www.rcsb.org/" target="_blank">https://www.rcsb.org/</a>). All EMDB and PDB accession numbers are listed in the <a id="crosref1625" href="#sec-4-1">key resources table</a>. The mass spectrometry raw data has been deposited to MassIVE (<a id="intref0020" href="https://massive.ucsd.edu" target="_blank">https://massive.ucsd.edu</a>) under the identifier MSV000088875.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">The paper does not report original code.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Afonine, P.V. ∙ Poon, B.K. ∙ Read, R.J. ...</div><div><strong>Real-space refinement in PHENIX for cryo-EM and crystallography</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:531-544</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.V.AfonineB.K.PoonR.J.ReadO.V.SobolevT.C.TerwilligerA.UrzhumtsevP.D.AdamsReal-space+refinement+in+PHENIX+for+cryo-EM+and+crystallographyActa+Crystallogr.+D+Struct.+Biol.742018531544" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Arakhamia, T. ∙ Lee, C.E. ∙ Carlomagno, Y. ...</div><div><strong>Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:633-644.e12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.ArakhamiaC.E.LeeY.CarlomagnoD.M.DuongS.R.KundingerK.WangD.WilliamsM.DeTureD.W.DicksonC.N.CookPosttranslational+Modifications+Mediate+the+Structural+Diversity+of+Tauopathy+StrainsCell1802020633644.e12" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Arseni, D. ∙ Hasegawa, M. ∙ Murzin, A.G. ...</div><div><strong>Structure of pathological TDP-43 filaments from ALS with FTLD</strong></div><div><em>Nature.</em> 2022; <strong>601</strong>:139-143</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ArseniM.HasegawaA.G.MurzinF.KametaniM.AraiM.YoshidaB.Ryskeldi-FalconStructure+of+pathological+TDP-43+filaments+from+ALS+with+FTLDNature6012022139143" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Baker, M. ∙ Mackenzie, I.R. ∙ Pickering-Brown, S.M. ...</div><div><strong>Mutations in progranulin cause tau-negative frontotemporal dementia linked to chromosome 17</strong></div><div><em>Nature.</em> 2006; <strong>442</strong>:916-919</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BakerI.R.MackenzieS.M.Pickering-BrownJ.GassR.RademakersC.LindholmJ.SnowdenJ.AdamsonA.D.SadovnickS.RollinsonMutations+in+progranulin+cause+tau-negative+frontotemporal+dementia+linked+to+chromosome+17Nature4422006916919" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div>biochem-fan atom2svg.py. <a id="intref0305" href="https://gist.github.com/biochem-fan/026ec2f191fee9285424d12fc2b84ce7" target="_blank">https://gist.github.com/biochem-fan/026ec2f191fee9285424d12fc2b84ce7</a>.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=biochem-fan+atom2svg.py.+https%3A%2F%2Fgist.github.com%2Fbiochem-fan%2F026ec2f191fee9285424d12fc2b84ce7." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Brady, O.A. ∙ Zhou, X. ∙ Hu, F.</div><div><strong>Regulated intramembrane proteolysis of the frontotemporal lobar degeneration risk factor, TMEM106B, by signal peptide peptidase-like 2a (SPPL2a)</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2014; <strong>289</strong>:19670-19680</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.A.BradyX.ZhouF.HuRegulated+intramembrane+proteolysis+of+the+frontotemporal+lobar+degeneration+risk+factor%2C+TMEM106B%2C+by+signal+peptide+peptidase-like+2a+%28SPPL2a%29J.%C2%A0Biol.+Chem.28920141967019680" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Brier, M.R. ∙ Gordon, B. ∙ Friedrichsen, K. ...</div><div><strong>Tau and Aβ imaging, CSF measures, and cognition in Alzheimer’s disease</strong></div><div><em>Sci. Transl. Med.</em> 2016; <strong>8</strong>:338ra66</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.R.BrierB.GordonK.FriedrichsenJ.McCarthyA.SternJ.ChristensenC.OwenP.AldeaY.SuJ.HassenstabTau+and+A%CE%B2+imaging%2C+CSF+measures%2C+and+cognition+in+Alzheimer%E2%80%99s+diseaseSci.+Transl.+Med.82016338ra66" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Busch, J.I. ∙ Martinez-Lage, M. ∙ Ashbridge, E. ...</div><div><strong>Expression of TMEM106B, the frontotemporal lobar degeneration-associated protein, in normal and diseased human brain</strong></div><div><em>Acta Neuropathol. Commun.</em> 2013; <strong>1</strong>:36</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.I.BuschM.Martinez-LageE.AshbridgeM.GrossmanV.M.Van+DeerlinF.HuV.M.LeeJ.Q.TrojanowskiA.S.Chen-PlotkinExpression+of+TMEM106B%2C+the+frontotemporal+lobar+degeneration-associated+protein%2C+in+normal+and+diseased+human+brainActa+Neuropathol.+Commun.1201336" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Cairns, N.J. ∙ Bigio, E.H. ∙ Mackenzie, I.R. ..., Consortium for Frontotemporal Lobar Degeneration</div><div><strong>Neuropathologic diagnostic and nosologic criteria for frontotemporal lobar degeneration: consensus of the Consortium for Frontotemporal Lobar Degeneration</strong></div><div><em>Acta Neuropathol.</em> 2007; <strong>114</strong>:5-22</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.J.CairnsE.H.BigioI.R.MackenzieM.NeumannV.M.LeeK.J.HatanpaaC.L.White3rdJ.A.SchneiderL.T.GrinbergG.HallidayConsortium+for+Frontotemporal+Lobar+DegenerationNeuropathologic+diagnostic+and+nosologic+criteria+for+frontotemporal+lobar+degeneration%3A+consensus+of+the+Consortium+for+Frontotemporal+Lobar+DegenerationActa+Neuropathol.1142007522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Carlomagno, Y. ∙ Manne, S. ∙ DeTure, M. ...</div><div><strong>The AD tau core spontaneously self-assembles and recruits full-length tau to filaments</strong></div><div><em>Cell Rep.</em> 2021; <strong>34</strong>:108843</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.CarlomagnoS.ManneM.DeTureM.PrudencioY.J.ZhangR.Hanna+Al-ShaikhJ.A.DunmoreL.M.DaughrityY.SongM.Castanedes-CaseyThe+AD+tau+core+spontaneously+self-assembles+and+recruits+full-length+tau+to+filamentsCell+Rep.342021108843" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Chatterjee, P. ∙ Pedrini, S. ∙ Stoops, E. ...</div><div><strong>Plasma glial fibrillary acidic protein is elevated in cognitively normal older adults at risk of Alzheimer’s disease</strong></div><div><em>Transl. Psychiatry.</em> 2021; <strong>11</strong>:27</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.ChatterjeeS.PedriniE.StoopsK.GoozeeV.L.VillemagneP.R.AsihI.M.W.VerberkP.DaveK.TaddeiH.R.SohrabiPlasma+glial+fibrillary+acidic+protein+is+elevated+in+cognitively+normal+older+adults+at+risk+of+Alzheimer%E2%80%99s+diseaseTransl.+Psychiatry11202127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Chen, V.B. ∙ Arendall, 3rd, W.B. ∙ Headd, J.J. ...</div><div><strong>MolProbity: all-atom structure validation for macromolecular crystallography</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:12-21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.B.ChenW.B.Arendall3rdJ.J.HeaddD.A.KeedyR.M.ImmorminoG.J.KapralL.W.MurrayJ.S.RichardsonD.C.RichardsonMolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallographyActa+Crystallogr.+D+Biol.+Crystallogr.6620101221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13-1" title="View in article">13.</a></div><div>Chojnowski, G. ∙ Simpkin, A.J. ∙ Leonardo, D.A. ...</div><div><strong><i>findMySequence</i>: a neural-network-based approach for identification of unknown proteins in X-ray crystallography and cryo-EM</strong></div><div><em>IUCrJ.</em> 2021; <strong>9</strong>:86-97</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.ChojnowskiA.J.SimpkinD.A.LeonardoW.Seifert-DavilaD.E.Vivas-RuizR.M.KeeganD.J.RigdenfindMySequence%3A+a+neural-network-based+approach+for+identification+of+unknown+proteins+in+X-ray+crystallography+and+cryo-EMIUCrJ920218697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Cianfrocco, M.A. ∙ Wong-Barnum, M. ∙ Youn, C. ...</div><div><strong>COSMIC<sup>2</sup>: A Science Gateway for Cryo-Electron Microscopy Structure Determination</strong></div><div><strong>Proceedings of the Practice and Experience in Advanced Research Computing 2017 on Sustainability, Success and Impact</strong></div><div>2017</div><div>Published online July 7, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.A.CianfroccoM.Wong-BarnumC.YounR.WagnerA.E.LeschzinerCOSMIC2%3A+A+Science+Gateway+for+Cryo-Electron+Microscopy+Structure+DeterminationProceedings+of+the+Practice+and+Experience+in+Advanced+Research+Computing+2017+on+Sustainability%2C+Success+and+Impact201710.1145%2F3093338.3093390Published+online+July+7%2C+2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Cox, J. ∙ Mann, M.</div><div><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong></div><div><em>Nat. Biotechnol.</em> 2008; <strong>26</strong>:1367-1372</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.CoxM.MannMaxQuant+enables+high+peptide+identification+rates%2C+individualized+p.p.b.-range+mass+accuracies+and+proteome-wide+protein+quantificationNat.+Biotechnol.26200813671372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>DeJesus-Hernandez, M. ∙ Mackenzie, I.R. ∙ Boeve, B.F. ...</div><div><strong>Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS</strong></div><div><em>Neuron.</em> 2011; <strong>72</strong>:245-256</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.DeJesus-HernandezI.R.MackenzieB.F.BoeveA.L.BoxerM.BakerN.J.RutherfordA.M.NicholsonN.A.FinchH.FlynnJ.AdamsonExpanded+GGGGCC+hexanucleotide+repeat+in+noncoding+region+of+C9ORF72+causes+chromosome+9p-linked+FTD+and+ALSNeuron722011245256" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.EmsleyB.LohkampW.G.ScottK.CowtanFeatures+and+development+of+CootActa+Crystallogr.+D+Biol.+Crystallogr.662010486501" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Falcon, B. ∙ Zhang, W. ∙ Murzin, A.G. ...</div><div><strong>Structures of filaments from Pick’s disease reveal a novel tau protein fold</strong></div><div><em>Nature.</em> 2018; <strong>561</strong>:137-140</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.FalconW.ZhangA.G.MurzinG.MurshudovH.J.GarringerR.VidalR.A.CrowtherB.GhettiS.H.W.ScheresM.GoedertStructures+of+filaments+from+Pick%E2%80%99s+disease+reveal+a+novel+tau+protein+foldNature5612018137140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Falcon, B. ∙ Zivanov, J. ∙ Zhang, W. ...</div><div><strong>Novel tau filament fold in chronic traumatic encephalopathy encloses hydrophobic molecules</strong></div><div><em>Nature.</em> 2019; <strong>568</strong>:420-423</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.FalconJ.ZivanovW.ZhangA.G.MurzinH.J.GarringerR.VidalR.A.CrowtherK.L.NewellB.GhettiM.GoedertS.H.W.ScheresNovel+tau+filament+fold+in+chronic+traumatic+encephalopathy+encloses+hydrophobic+moleculesNature5682019420423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Feng, T. ∙ Sheng, R.R. ∙ Solé-Domènech, S. ...</div><div><strong>A role of the frontotemporal lobar degeneration risk factor TMEM106B in myelination</strong></div><div><em>Brain.</em> 2020; <strong>143</strong>:2255-2271</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.FengR.R.ShengS.Sol%C3%A9-Dom%C3%A8nechM.UllahX.ZhouC.S.MendozaL.C.M.EnriquezI.I.KatzD.H.PaushterP.M.SullivanA+role+of+the+frontotemporal+lobar+degeneration+risk+factor+TMEM106B+in+myelinationBrain143202022552271" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Feng, T. ∙ Lacrampe, A. ∙ Hu, F.</div><div><strong>Physiological and pathological functions of TMEM106B: a gene associated with brain aging and multiple brain disorders</strong></div><div><em>Acta Neuropathol.</em> 2021; <strong>141</strong>:327-339</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.FengA.LacrampeF.HuPhysiological+and+pathological+functions+of+TMEM106B%3A+a+gene+associated+with+brain+aging+and+multiple+brain+disordersActa+Neuropathol.1412021327339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Finch, N. ∙ Carrasquillo, M.M. ∙ Baker, M. ...</div><div><strong>TMEM106B regulates progranulin levels and the penetrance of FTLD in GRN mutation carriers</strong></div><div><em>Neurology.</em> 2011; <strong>76</strong>:467-474</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.FinchM.M.CarrasquilloM.BakerN.J.RutherfordG.CoppolaM.Dejesus-HernandezR.CrookT.HunterR.GhidoniL.BenussiTMEM106B+regulates+progranulin+levels+and+the+penetrance+of+FTLD+in+GRN+mutation+carriersNeurology762011467474" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Fitzpatrick, A.W. ∙ Saibil, H.R.</div><div><strong>Cryo-EM of amyloid fibrils and cellular aggregates</strong></div><div><em>Curr. Opin. Struct. Biol.</em> 2019; <strong>58</strong>:34-42</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.FitzpatrickH.R.SaibilCryo-EM+of+amyloid+fibrils+and+cellular+aggregatesCurr.+Opin.+Struct.+Biol.5820193442" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Fitzpatrick, A.W.P. ∙ Debelouchina, G.T. ∙ Bayro, M.J. ...</div><div><strong>Atomic structure and hierarchical assembly of a cross-β amyloid fibril</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:5468-5473</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickG.T.DebelouchinaM.J.BayroD.K.ClareM.A.CaporiniV.S.BajajC.P.JaroniecL.WangV.LadizhanskyS.A.M%C3%BCllerAtomic+structure+and+hierarchical+assembly+of+a+cross-%CE%B2+amyloid+fibrilProc.+Natl.+Acad.+Sci.+USA110201354685473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">25.</a></div><div>Fitzpatrick, A.W.P. ∙ Falcon, B. ∙ He, S. ...</div><div><strong>Cryo-EM structures of tau filaments from Alzheimer’s disease</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:185-190</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.W.P.FitzpatrickB.FalconS.HeA.G.MurzinG.MurshudovH.J.GarringerR.A.CrowtherB.GhettiM.GoedertS.H.W.ScheresCryo-EM+structures+of+tau+filaments+from+Alzheimer%E2%80%99s+diseaseNature5472017185190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Hatton, C. ∙ Ghanem, S.S. ∙ Koss, D.J. ..., International DLB Genetics Consortium</div><div><strong>Prion-like α-synuclein pathology in the brain of infants with Krabbe disease</strong></div><div><em>Brain.</em> 2022;</div><div>Published online January 6, 2022</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.HattonS.S.GhanemD.J.KossI.Y.AbdiE.GibbonsR.GuerreiroJ.BrasL.WalkerE.GelpiW.HeywoodInternational+DLB+Genetics+ConsortiumPrion-like+%CE%B1-synuclein+pathology+in+the+brain+of+infants+with+Krabbe+diseaseBrain202210.1093%2Fbrain%2Fawac002Published+online+January+6%2C+2022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">27.</a></div><div>He, S. ∙ Scheres, S.H.W.</div><div><strong>Helical reconstruction in RELION</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2017; <strong>198</strong>:163-176</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.HeS.H.W.ScheresHelical+reconstruction+in+RELIONJ.%C2%A0Struct.+Biol.1982017163176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Ho, C.M. ∙ Li, X. ∙ Lai, M. ...</div><div><strong>Bottom-up structural proteomics: cryoEM of protein complexes enriched from the cellular milieu</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:79-85</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.HoX.LiM.LaiT.C.TerwilligerJ.R.BeckJ.WohlschlegelD.E.GoldbergA.W.P.FitzpatrickZ.H.ZhouBottom-up+structural+proteomics%3A+cryoEM+of+protein+complexes+enriched+from+the+cellular+milieuNat.+Methods1720207985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Hortschansky, P. ∙ Christopeit, T. ∙ Schroeckh, V. ...</div><div><strong>Thermodynamic analysis of the aggregation propensity of oxidized Alzheimer’s beta-amyloid variants</strong></div><div><em>Protein Sci.</em> 2005; <strong>14</strong>:2915-2918</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.HortschanskyT.ChristopeitV.SchroeckhM.F%C3%A4ndrichThermodynamic+analysis+of+the+aggregation+propensity+of+oxidized+Alzheimer%E2%80%99s+beta-amyloid+variantsProtein+Sci.14200529152918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Jiang, S. ∙ Bhaskar, K.</div><div><strong>Degradation and Transmission of Tau by Autophagic-Endolysosomal Networks and Potential Therapeutic Targets for Tauopathy</strong></div><div><em>Front. Mol. Neurosci.</em> 2020; <strong>13</strong>:586731</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.JiangK.BhaskarDegradation+and+Transmission+of+Tau+by+Autophagic-Endolysosomal+Networks+and+Potential+Therapeutic+Targets+for+TauopathyFront.+Mol.+Neurosci.132020586731" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">31.</a></div><div>Jiang, Y.X. ∙ Cao, Q. ∙ Sawaya, M.R. ...</div><div><strong>Amyloid fibrils in frontotemporal lobar degeneration with TDP-43 inclusions are composed of TMEM106B, rather than TDP-43</strong></div><div><em>Preprint at bioRxiv.</em> 2022;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.X.JiangQ.CaoM.R.SawayaR.AbskharonP.GeM.DeTureD.W.DicksonJ.Y.FuR.R.Ogorzalek+LooJ.A.LooAmyloid+fibrils+in+frontotemporal+lobar+degeneration+with+TDP-43+inclusions+are+composed+of+TMEM106B%2C+rather+than+TDP-43Preprint+at+bioRxiv202210.1101%2F2022.01.31.478523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Jucker, M. ∙ Walker, L.C.</div><div><strong>Self-propagation of pathogenic protein aggregates in neurodegenerative diseases</strong></div><div><em>Nature.</em> 2013; <strong>501</strong>:45-51</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.JuckerL.C.WalkerSelf-propagation+of+pathogenic+protein+aggregates+in+neurodegenerative+diseasesNature50120134551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">33.</a></div><div>Jumper, J. ∙ Evans, R. ∙ Pritzel, A. ...</div><div><strong>Highly accurate protein structure prediction with AlphaFold</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:583-589</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.JumperR.EvansA.PritzelT.GreenM.FigurnovO.RonnebergerK.TunyasuvunakoolR.BatesA.%C5%BD%C3%ADdekA.PotapenkoHighly+accurate+protein+structure+prediction+with+AlphaFoldNature5962021583589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Kametani, F. ∙ Yoshida, M. ∙ Matsubara, T. ...</div><div><strong>Comparison of Common and Disease-Specific Post-translational Modifications of Pathological Tau Associated With a Wide Range of Tauopathies</strong></div><div><em>Front. Neurosci.</em> 2020; <strong>14</strong>:581936</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KametaniM.YoshidaT.MatsubaraS.MurayamaY.SaitoI.KawakamiM.OnayaH.TanakaA.KakitaA.C.RobinsonComparison+of+Common+and+Disease-Specific+Post-translational+Modifications+of+Pathological+Tau+Associated+With+a+Wide+Range+of+TauopathiesFront.+Neurosci.142020581936" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">35.</a></div><div>Kim, D.E. ∙ Chivian, D. ∙ Baker, D.</div><div><strong>Protein structure prediction and analysis using the Robetta server</strong></div><div><em>Nucleic Acids Res.</em> 2004; <strong>32</strong>:W526-W531</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.E.KimD.ChivianD.BakerProtein+structure+prediction+and+analysis+using+the+Robetta+serverNucleic+Acids+Res.322004W526-W531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">36.</a></div><div>Lang, C.M. ∙ Fellerer, K. ∙ Schwenk, B.M. ...</div><div><strong>Membrane orientation and subcellular localization of transmembrane protein 106B (TMEM106B), a major risk factor for frontotemporal lobar degeneration</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:19355-19365</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.LangK.FellererB.M.SchwenkP.H.KuhnE.KremmerD.EdbauerA.CapellC.HaassMembrane+orientation+and+subcellular+localization+of+transmembrane+protein+106B+%28TMEM106B%29%2C+a+major+risk+factor+for+frontotemporal+lobar+degenerationJ.%C2%A0Biol.+Chem.28720121935519365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Lee, V.M. ∙ Goedert, M. ∙ Trojanowski, J.Q.</div><div><strong>Neurodegenerative tauopathies</strong></div><div><em>Annu. Rev. Neurosci.</em> 2001; <strong>24</strong>:1121-1159</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.M.LeeM.GoedertJ.Q.TrojanowskiNeurodegenerative+tauopathiesAnnu.+Rev.+Neurosci.24200111211159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">38.</a></div><div>Li, F. ∙ Leier, A. ∙ Liu, Q. ...</div><div><strong>Procleave: Predicting Protease-specific Substrate Cleavage Sites by Combining Sequence and Structural Information</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2020; <strong>18</strong>:52-64</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.LiA.LeierQ.LiuY.WangD.XiangT.AkutsuG.I.WebbA.I.SmithT.Marquez-LagoJ.LiJ.SongProcleave%3A+Predicting+Protease-specific+Substrate+Cleavage+Sites+by+Combining+Sequence+and+Structural+InformationGenomics+Proteomics+Bioinformatics1820205264" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39-1" title="View in article">39.</a></div><div>Lüningschrör, P. ∙ Werner, G. ∙ Stroobants, S. ...</div><div><strong>The FTLD Risk Factor TMEM106B Regulates the Transport of Lysosomes at the Axon Initial Segment of Motoneurons</strong></div><div><em>Cell Rep.</em> 2020; <strong>30</strong>:3506-3519.e6</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.L%C3%BCningschr%C3%B6rG.WernerS.StroobantsS.KakutaB.DombertD.SinskeR.WannerR.L%C3%BCllmann-RauchB.WefersW.WurstThe+FTLD+Risk+Factor+TMEM106B+Regulates+the+Transport+of+Lysosomes+at+the+Axon+Initial+Segment+of+MotoneuronsCell+Rep.30202035063519.e6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Mackenzie, I.R. ∙ Neumann, M. ∙ Baborie, A. ...</div><div><strong>A harmonized classification system for FTLD-TDP pathology</strong></div><div><em>Acta Neuropathol.</em> 2011; <strong>122</strong>:111-113</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.R.MackenzieM.NeumannA.BaborieD.M.SampathuD.Du+PlessisE.JarosR.H.PerryJ.Q.TrojanowskiD.M.MannV.M.LeeA+harmonized+classification+system+for+FTLD-TDP+pathologyActa+Neuropathol.1222011111113" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">41.</a></div><div>Monaco, A. ∙ Fraldi, A.</div><div><strong>Protein Aggregation and Dysfunction of Autophagy-Lysosomal Pathway: A Vicious Cycle in Lysosomal Storage Diseases</strong></div><div><em>Front. Mol. Neurosci.</em> 2020; <strong>13</strong>:37</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MonacoA.FraldiProtein+Aggregation+and+Dysfunction+of+Autophagy-Lysosomal+Pathway%3A+A+Vicious+Cycle+in+Lysosomal+Storage+DiseasesFront.+Mol.+Neurosci.13202037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Montine, T.J. ∙ Phelps, C.H. ∙ Beach, T.G. ..., National Institute on Aging, Alzheimer’s Association</div><div><strong>National Institute on Aging-Alzheimer’s Association guidelines for the neuropathologic assessment of Alzheimer’s disease: a practical approach</strong></div><div><em>Acta Neuropathol.</em> 2012; <strong>123</strong>:1-11</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.J.MontineC.H.PhelpsT.G.BeachE.H.BigioN.J.CairnsD.W.DicksonC.DuyckaertsM.P.FroschE.MasliahS.S.MirraNational+Institute+on+AgingAlzheimer%E2%80%99s+AssociationNational+Institute+on+Aging-Alzheimer%E2%80%99s+Association+guidelines+for+the+neuropathologic+assessment+of+Alzheimer%E2%80%99s+disease%3A+a+practical+approachActa+Neuropathol.1232012111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">43.</a></div><div>Nicholson, A.M. ∙ Finch, N.A. ∙ Wojtas, A. ...</div><div><strong>TMEM106B p.T185S regulates TMEM106B protein levels: implications for frontotemporal dementia</strong></div><div><em>J.&nbsp;Neurochem.</em> 2013; <strong>126</strong>:781-791</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.M.NicholsonN.A.FinchA.WojtasM.C.BakerR.B.Perkerson3rdM.Castanedes-CaseyL.RousseauL.BenussiG.BinettiR.GhidoniTMEM106B+p.T185S+regulates+TMEM106B+protein+levels%3A+implications+for+frontotemporal+dementiaJ.%C2%A0Neurochem.1262013781791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">44.</a></div><div>Petkova, A.T. ∙ Leapman, R.D. ∙ Guo, Z. ...</div><div><strong>Self-propagating, molecular-level polymorphism in Alzheimer’s beta-amyloid fibrils</strong></div><div><em>Science.</em> 2005; <strong>307</strong>:262-265</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.PetkovaR.D.LeapmanZ.GuoW.M.YauM.P.MattsonR.TyckoSelf-propagating%2C+molecular-level+polymorphism+in+Alzheimer%E2%80%99s+beta-amyloid+fibrilsScience3072005262265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">45.</a></div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF Chimera--a visualization system for exploratory research and analysis</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2004; <strong>25</strong>:1605-1612</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.F.PettersenT.D.GoddardC.C.HuangG.S.CouchD.M.GreenblattE.C.MengT.E.FerrinUCSF+Chimera--a+visualization+system+for+exploratory+research+and+analysisJ.%C2%A0Comput.+Chem.25200416051612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">46.</a></div><div>Pfab, J. ∙ Phan, N.M. ∙ Si, D.</div><div><strong>DeepTracer for fast de novo cryo-EM protein structure modeling and special studies on CoV-related complexes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong></div><div>e2017525118</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.PfabN.M.PhanD.SiDeepTracer+for+fast+de+novo+cryo-EM+protein+structure+modeling+and+special+studies+on+CoV-related+complexesProc.+Natl.+Acad.+Sci.+USA1182021e2017525118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Rademakers, R. ∙ Nicholson, A.M. ∙ Ren, Y. ...</div><div><strong>Loss of Tmem106b leads to cerebellum Purkinje cell death and motor deficits</strong></div><div><em>Brain Pathol.</em> 2021; <strong>31</strong>:e12945</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.RademakersA.M.NicholsonY.RenS.KogaH.P.NguyenM.BrooksW.QiaoZ.S.QuicksallB.MatchettR.B.PerkersonLoss+of+Tmem106b+leads+to+cerebellum+Purkinje+cell+death+and+motor+deficitsBrain+Pathol.312021e12945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">48.</a></div><div>Rawlings, N.D. ∙ Barrett, A.J. ∙ Thomas, P.D. ...</div><div><strong>The MEROPS database of proteolytic enzymes, their substrates and inhibitors in 2017 and a comparison with peptidases in the PANTHER database</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D624-D632</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.D.RawlingsA.J.BarrettP.D.ThomasX.HuangA.BatemanR.D.FinnThe+MEROPS+database+of+proteolytic+enzymes%2C+their+substrates+and+inhibitors+in+2017+and+a+comparison+with+peptidases+in+the+PANTHER+databaseNucleic+Acids+Res.46D12018D624D632" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">49.</a></div><div>Sanchez-Garcia, R. ∙ Gomez-Blanco, J. ∙ Cuervo, A. ...</div><div><strong>DeepEMhancer: a deep learning solution for cryo-EM volume post-processing</strong></div><div><em>Commun Biol.</em> 2021; <strong>4</strong>:874</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.Sanchez-GarciaJ.Gomez-BlancoA.CuervoJ.M.CarazoC.O.S.SorzanoJ.VargasDeepEMhancer%3A+a+deep+learning+solution+for+cryo-EM+volume+post-processingCommun+Biol.42021874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Schweighauser, M. ∙ Shi, Y. ∙ Tarutani, A. ...</div><div><strong>Structures of α-synuclein filaments from multiple system atrophy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:464-469</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserY.ShiA.TarutaniF.KametaniA.G.MurzinB.GhettiT.MatsubaraT.TomitaT.AndoK.HasegawaStructures+of+%CE%B1-synuclein+filaments+from+multiple+system+atrophyNature5852020464469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">51.</a></div><div>Schweighauser, M. ∙ Arseni, D. ∙ Huang, M. ...</div><div><strong>Age-Dependent Formation of TMEM106B Amyloid Filaments in Human Brain</strong></div><div><em>Preprint at bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SchweighauserD.ArseniM.HuangS.L%C3%B6vestamY.ShiY.YangW.ZhangA.KotechaH.J.GarringerR.VidalAge-Dependent+Formation+of+TMEM106B+Amyloid+Filaments+in+Human+BrainPreprint+at+bioRxiv202110.1101%2F2021.11.09.467923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">52.</a></div><div>Schwenk, B.M. ∙ Lang, C.M. ∙ Hogl, S. ...</div><div><strong>The FTLD risk factor TMEM106B and MAP6 control dendritic trafficking of lysosomes</strong></div><div><em>EMBO J.</em> 2014; <strong>33</strong>:450-467</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.M.SchwenkC.M.LangS.HoglS.TahirovicD.OrozcoK.RentzschS.F.LichtenthalerC.C.HoogenraadA.CapellC.HaassD.EdbauerThe+FTLD+risk+factor+TMEM106B+and+MAP6+control+dendritic+trafficking+of+lysosomesEMBO+J.332014450467" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">53.</a></div><div>Shi, Y. ∙ Zhang, W. ∙ Yang, Y. ...</div><div><strong>Structure-based classification of tauopathies</strong></div><div><em>Nature.</em> 2021; <strong>598</strong>:359-363</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ShiW.ZhangY.YangA.G.MurzinB.FalconA.KotechaM.van+BeersA.TarutaniF.KametaniH.J.GarringerStructure-based+classification+of+tauopathiesNature5982021359363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">54.</a></div><div>Smirnovas, V. ∙ Winter, R. ∙ Funck, T. ...</div><div><strong>Thermodynamic properties underlying the α-helix-to-β-sheet transition, aggregation, and amyloidogenesis of polylysine as probed by calorimetry, densimetry, and ultrasound velocimetry</strong></div><div><em>J.&nbsp;Phys. Chem. B.</em> 2005; <strong>109</strong>:19043-19045</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.SmirnovasR.WinterT.FunckW.DzwolakThermodynamic+properties+underlying+the+%CE%B1-helix-to-%CE%B2-sheet+transition%2C+aggregation%2C+and+amyloidogenesis+of+polylysine+as+probed+by+calorimetry%2C+densimetry%2C+and+ultrasound+velocimetryJ.%C2%A0Phys.+Chem.+B10920051904319045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55-1" title="View in article">55.</a></div><div>Sormanni, P. ∙ Aprile, F.A. ∙ Vendruscolo, M.</div><div><strong>The CamSol method of rational design of protein mutants with enhanced solubility</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 2015; <strong>427</strong>:478-490</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.SormanniF.A.AprileM.VendruscoloThe+CamSol+method+of+rational+design+of+protein+mutants+with+enhanced+solubilityJ.%C2%A0Mol.+Biol.4272015478490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Stagi, M. ∙ Klein, Z.A. ∙ Gould, T.J. ...</div><div><strong>Lysosome size, motility and stress response regulated by fronto-temporal dementia modifier TMEM106B</strong></div><div><em>Mol. Cell. Neurosci.</em> 2014; <strong>61</strong>:226-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.StagiZ.A.KleinT.J.GouldJ.BewersdorfS.M.StrittmatterLysosome+size%2C+motility+and+stress+response+regulated+by+fronto-temporal+dementia+modifier+TMEM106BMol.+Cell.+Neurosci.612014226240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>Stroobants, S. ∙ D’Hooge, R. ∙ Damme, M.</div><div><strong>Aged Tmem106b knockout mice display gait deficits in coincidence with Purkinje cell loss and only limited signs of non-motor dysfunction</strong></div><div><em>Brain Pathol.</em> 2021; <strong>31</strong>:223-238</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.StroobantsR.D%E2%80%99HoogeM.DammeAged+Tmem106b+knockout+mice+display+gait+deficits+in+coincidence+with+Purkinje+cell+loss+and+only+limited+signs+of+non-motor+dysfunctionBrain+Pathol.312021223238" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">58.</a></div><div>van Daalen, K.R. ∙ Reijneveld, J.F. ∙ Bovenschen, N.</div><div><strong>Modulation of Inflammation by Extracellular Granzyme A</strong></div><div><em>Front. Immunol.</em> 2020; <strong>11</strong>:931</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.R.van+DaalenJ.F.ReijneveldN.BovenschenModulation+of+Inflammation+by+Extracellular+Granzyme+AFront.+Immunol.112020931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">59.</a></div><div>Van Deerlin, V.M. ∙ Sleiman, P.M. ∙ Martinez-Lage, M. ...</div><div><strong>Common variants at 7p21 are associated with frontotemporal lobar degeneration with TDP-43 inclusions</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:234-239</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.M.Van+DeerlinP.M.SleimanM.Martinez-LageA.Chen-PlotkinL.S.WangN.R.Graff-RadfordD.W.DicksonR.RademakersB.F.BoeveM.GrossmanCommon+variants+at+7p21+are+associated+with+frontotemporal+lobar+degeneration+with+TDP-43+inclusionsNat.+Genet.422010234239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">60.</a></div><div>van der Zee, J. ∙ Van Langenhove, T. ∙ Kleinberger, G. ...</div><div><strong>TMEM106B is associated with frontotemporal lobar degeneration in a clinically diagnosed patient cohort</strong></div><div><em>Brain.</em> 2011; <strong>134</strong>:808-815</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.van+der+ZeeT.Van+LangenhoveG.KleinbergerK.SleegersS.EngelborghsR.VandenbergheP.SantensM.Van+den+BroeckG.JorisJ.BrysTMEM106B+is+associated+with+frontotemporal+lobar+degeneration+in+a+clinically+diagnosed+patient+cohortBrain1342011808815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">61.</a></div><div>Yang, J. ∙ Anishchenko, I. ∙ Park, H. ...</div><div><strong>Improved protein structure prediction using predicted interresidue orientations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:1496-1503</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.YangI.AnishchenkoH.ParkZ.PengS.OvchinnikovD.BakerImproved+protein+structure+prediction+using+predicted+interresidue+orientationsProc.+Natl.+Acad.+Sci.+USA117202014961503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">62.</a></div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.ZhangGctf%3A+Real-time+CTF+determination+and+correctionJ.%C2%A0Struct.+Biol.1932016112" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Zhang, W. ∙ Falcon, B. ∙ Murzin, A.G. ...</div><div><strong>Heparin-induced tau filaments are polymorphic and differ from those in Alzheimer’s and Pick’s diseases</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43584</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.ZhangB.FalconA.G.MurzinJ.FanR.A.CrowtherM.GoedertS.H.ScheresHeparin-induced+tau+filaments+are+polymorphic+and+differ+from+those+in+Alzheimer%E2%80%99s+and+Pick%E2%80%99s+diseaseseLife82019e43584" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">64.</a></div><div>Zhang, W. ∙ Tarutani, A. ∙ Newell, K.L. ...</div><div><strong>Novel tau filament fold in corticobasal degeneration</strong></div><div><em>Nature.</em> 2020; <strong>580</strong>:283-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.ZhangA.TarutaniK.L.NewellA.G.MurzinT.MatsubaraB.FalconR.VidalH.J.GarringerY.ShiT.IkeuchiNovel+tau+filament+fold+in+corticobasal+degenerationNature5802020283287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">65.</a></div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.-P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.Q.ZhengE.PalovcakJ.-P.ArmacheK.A.VerbaY.ChengD.A.AgardMotionCor2%3A+anisotropic+correction+of+beam-induced+motion+for+improved+cryo-electron+microscopyNat.+Methods142017331332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">66.</a></div><div>Zhou, X. ∙ Brooks, M. ∙ Jiang, P. ...</div><div><strong>Loss of Tmem106b exacerbates FTLD pathologies and causes motor deficits in progranulin-deficient mice</strong></div><div><em>EMBO Rep.</em> 2020; <strong>21</strong>:e50197</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=X.ZhouM.BrooksP.JiangS.KogaA.R.ZuberiM.C.BakerT.M.ParsonsM.Castanedes-CaseyV.PhillipsA.L.LibreroLoss+of+Tmem106b+exacerbates+FTLD+pathologies+and+causes+motor+deficits+in+progranulin-deficient+miceEMBO+Rep.212020e50197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">67.</a></div><div>Zivanov, J. ∙ Nakane, T. ∙ Forsberg, B.O. ...</div><div><strong>New tools for automated high-resolution cryo-EM structure determination in RELION-3</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e42166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ZivanovT.NakaneB.O.ForsbergD.KimaniusW.J.HagenE.LindahlS.H.ScheresNew+tools+for+automated+high-resolution+cryo-EM+structure+determination+in+RELION-3eLife72018e42166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (12)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/59f10f77-64f5-44cd-af99-0bc0ff508589/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/85a89381-932c-4d5c-a9f9-b1d3c7205902/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/e007aac5-86b4-4a52-b8d8-1bcfc6701910/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/648b9086-8a47-4f08-8011-d04bdd150a40/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/b5fbf1d8-a6b4-44b9-b1e9-caa14b238f5b/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/dc71aa9f-ece9-4f11-aa0d-15009e77c089/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/551429da-a659-4e8e-8ece-c99b8d8fa347/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/d0b18d65-f49e-46ca-b68e-f85dc7ea63c6/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/7af3d11c-c3db-4074-bd43-2b96212d9a81/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/4743ac11-53f4-49aa-b8e7-2d5afdf57613/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/3eb60cbc-bba6-4703-a99a-8a8edffbe77a/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/asset/6c26ddd9-d35e-4623-98f4-a42c937c3ba9/main.assets/gr6.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867422002598%3Bjournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00098%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (2)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/attachment/d4e4f32a-492a-4102-8a27-02bcec3ec65c/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (124.06 KB)</span></a><div class="core-description">Document S1. Tables S1–S4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.02.026/attachment/9a1f9262-7932-4c51-97e4-17736e26eba6/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (675.93 KB)</span></a><div class="core-description">Table S5. All mass spectrometry raw data on FTLD-TDP type A case 1 and FTLD-TDP type A case 6 (case 9), related to Figures 3–5 and STAR Methods</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867422002598" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(22)02677-7" target="_blank" title="Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases">Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Anthony W. Fitzpatrick</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">February 10, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(22)00342-7" target="_blank" title="Flying under the radar: TMEM106B(120–254) fibrils break out in diverse neurodegenerative disorders">Flying under the radar: TMEM106B(120–254) fibrils break out in diverse neurodegenerative disorders</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Copley et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">April 14, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/molecular-medicine/fulltext/S1471-4914(25)00009-7" target="_blank" title="Development of asparagine endopeptidase inhibitors for treating neurodegenerative diseases">Development of asparagine endopeptidase inhibitors for treating neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Meng et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Molecular Medicine</i></span><span id="item_date">February 25, 2025</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)01458-X" target="_blank" title="Physiological shedding and C-terminal proteolytic processing of TMEM106B">Physiological shedding and C-terminal proteolytic processing of TMEM106B</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Held et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">December 21, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/molecular-medicine/fulltext/S1471-4914(25)00057-7" target="_blank" title="MicroRNAs in the biology and hallmarks of neurodegenerative diseases">MicroRNAs in the biology and hallmarks of neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Dhuppar et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Molecular Medicine</i></span><span id="item_date">April 7, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)01040-5" target="_blank" title="Relationship between endocrine disruptors and neurodegenerative diseases: systematic review and meta-analysis">Relationship between endocrine disruptors and neurodegenerative diseases: systematic review and meta-analysis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">May 29, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00009-3" target="_blank" title="Partial loss of MCU mitigates pathology in&nbsp;vivo across a diverse range of neurodegenerative disease models">Partial loss of MCU mitigates pathology in&nbsp;vivo across a diverse range of neurodegenerative disease models</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Twyning et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">January 17, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/neuron/fulltext/S0896-6273(24)00359-3" target="_blank" title="Cryo-EM structures of pathogenic fibrils and their impact on neurodegenerative disease research">Cryo-EM structures of pathogenic fibrils and their impact on neurodegenerative disease research</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Todd et al.</li></ul></div><div class="linked-article__details"><span><i>Neuron</i></span><span id="item_date">June 3, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/molecular-medicine/fulltext/S1471-4914(24)00219-3" target="_blank" title="Harnessing the immune system: vaccines to fight neurodegenerative diseases">Harnessing the immune system: vaccines to fight neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gonzalez-Artero et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Molecular Medicine</i></span><span id="item_date">September 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(24)00131-6" target="_blank" title="Innate immune activation in neurodegenerative diseases">Innate immune activation in neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Castro-Gomez et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">April 09, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)01272-5" target="_blank" title="TMEM106B-mediated SARS-CoV-2 infection allows for robust ACE2-independent infection in&nbsp;vitro but not in&nbsp;vivo">TMEM106B-mediated SARS-CoV-2 infection allows for robust ACE2-independent infection in&nbsp;vitro but not in&nbsp;vivo</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yan et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">October 30, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)16285-8" target="_blank" title="Perceptual constancy of pareidolias across paper and digital testing formats in neurodegenerative diseases">Perceptual constancy of pareidolias across paper and digital testing formats in neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Revankar et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">November 8, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)11463-6" target="_blank" title="Integrative analysis of causal associations between neurodegenerative diseases and colorectal cancer">Integrative analysis of causal associations between neurodegenerative diseases and colorectal cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 29, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/immunology/fulltext/S1471-4906(24)00064-4" target="_blank" title="Will cellular immunotherapies end neurodegenerative diseases?">Will cellular immunotherapies end neurodegenerative diseases?</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Boskovic et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Immunology</i></span><span id="item_date">April 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)08786-3" target="_blank" title="Computational and spectroscopic insight into the binding of citral with human transferrin: Targeting neurodegenerative diseases">Computational and spectroscopic insight into the binding of citral with human transferrin: Targeting neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Shamsi et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">June 7, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/nucleic-acids/fulltext/S2162-2531(24)00138-0" target="_blank" title="Selection of DNA aptamers that prevent the fibrillization of α-synuclein protein in cellular and mouse models">Selection of DNA aptamers that prevent the fibrillization of α-synuclein protein in cellular and mouse models</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>McConnell et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy - Nucleic Acids</i></span><span id="item_date">June 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)17369-0" target="_blank" title="AI-optimized electrochemical aptasensors for stable, reproducible detection of neurodegenerative diseases, cancer, and coronavirus">AI-optimized electrochemical aptasensors for stable, reproducible detection of neurodegenerative diseases, cancer, and coronavirus</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Tayfour Ahmed et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">December 18, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)00365-7" target="_blank" title="Therapeutic effects of saffron and its components on neurodegenerative diseases">Therapeutic effects of saffron and its components on neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hamedani et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">January 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)05449-5" target="_blank" title="Research trends and hotspots of ferroptosis in neurodegenerative diseases from 2013 to 2023: A bibliometrics study">Research trends and hotspots of ferroptosis in neurodegenerative diseases from 2013 to 2023: A bibliometrics study</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 8, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-stem-cell/fulltext/S1934-5909(23)00088-7" target="_blank" title="Advancing cell therapy for neurodegenerative diseases">Advancing cell therapy for neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sally Temple</li></ul></div><div class="linked-article__details"><span><i>Cell Stem Cell</i></span><span id="item_date">April 20, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)06552-6" target="_blank" title="Applications of magnetic resonance spectroscopy in diagnosis of neurodegenerative diseases: A systematic review">Applications of magnetic resonance spectroscopy in diagnosis of neurodegenerative diseases: A systematic review</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Abbaspour et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 29, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)00084-1" target="_blank" title="Vitamin D and neurodegenerative diseases">Vitamin D and neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">January 11, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/molecular-medicine/fulltext/S1471-4914(23)00190-9" target="_blank" title="Bioelectronic medicine potentiates endogenous NSCs for neurodegenerative diseases">Bioelectronic medicine potentiates endogenous NSCs for neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yu et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Molecular Medicine</i></span><span id="item_date">September 19, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/structure/fulltext/S0969-2126(23)00284-8" target="_blank" title="Molecular rules governing the structural polymorphism of amyloid fibrils in neurodegenerative diseases">Molecular rules governing the structural polymorphism of amyloid fibrils in neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>Structure</i></span><span id="item_date">August 31, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/neurosciences/fulltext/S0166-2236(23)00174-1" target="_blank" title="Inflammasome assembly in neurodegenerative diseases">Inflammasome assembly in neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Singh et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Neurosciences</i></span><span id="item_date">August 24, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/neuron/fulltext/S0896-6273(24)00286-1" target="_blank" title="Friend or foe: The role of stress granule in neurodegenerative disease">Friend or foe: The role of stress granule in neurodegenerative disease</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Cui et al.</li></ul></div><div class="linked-article__details"><span><i>Neuron</i></span><span id="item_date">May 13, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)02419-2" target="_blank" title="Cerebellum in neurodegenerative diseases: Advances, challenges, and prospects">Cerebellum in neurodegenerative diseases: Advances, challenges, and prospects</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">October 17, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(23)00645-1" target="_blank" title="TMEM106B is a receptor mediating ACE2-independent SARS-CoV-2 cell entry">TMEM106B is a receptor mediating ACE2-independent SARS-CoV-2 cell entry</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Baggen et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">July 7, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(22)01575-6" target="_blank" title="Hallmarks of neurodegenerative diseases">Hallmarks of neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wilson et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">February 16, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)15188-2" target="_blank" title="Genetic variability of incretin receptors affects the occurrence of neurodegenerative diseases and their characteristics">Genetic variability of incretin receptors affects the occurrence of neurodegenerative diseases and their characteristics</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Vogrinc et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">October 18, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/immunology/fulltext/S1471-4906(24)00190-X" target="_blank" title="Novel human iPSC models of neuroinflammation in neurodegenerative disease and regenerative medicine">Novel human iPSC models of neuroinflammation in neurodegenerative disease and regenerative medicine</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Summers et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Immunology</i></span><span id="item_date">September 21, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/neurosciences/fulltext/S0166-2236(23)00069-3" target="_blank" title="IGF2 in memory, neurodevelopmental disorders, and neurodegenerative diseases">IGF2 in memory, neurodevelopmental disorders, and neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Cristina M. Alberini</li></ul></div><div class="linked-article__details"><span><i>Trends in Neurosciences</i></span><span id="item_date">April 6, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/endocrinology-metabolism/fulltext/S1043-2760(23)00092-9" target="_blank" title="Apolipoprotein E in lipid metabolism and neurodegenerative disease">Apolipoprotein E in lipid metabolism and neurodegenerative disease</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yang et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Endocrinology &amp; Metabolism</i></span><span id="item_date">June 23, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/neurosciences/fulltext/S0166-2236(24)00024-9" target="_blank" title="Astrocytes in selective vulnerability to neurodegenerative disease">Astrocytes in selective vulnerability to neurodegenerative disease</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zimmer et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Neurosciences</i></span><span id="item_date">March 22, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(24)00738-X" target="_blank" title="Novel insights into the ROCK-JAK-STAT signaling pathway in upper respiratory tract infections and neurodegenerative diseases">Novel insights into the ROCK-JAK-STAT signaling pathway in upper respiratory tract infections and neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">November 6, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/molecular-medicine/fulltext/S1471-4914(23)00220-4" target="_blank" title="Diabetes: a tipping point in neurodegenerative diseases">Diabetes: a tipping point in neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Santiago et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Molecular Medicine</i></span><span id="item_date">October 10, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)01324-X" target="_blank" title="AAV-GRN partially corrects motor deficits and ALS/FTLD-related pathology in Tmem106b−/−Grn−/− mice">AAV-GRN partially corrects motor deficits and ALS/FTLD-related pathology in Tmem106b−/−Grn−/− mice</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Feng et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">June 28, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)02053-4" target="_blank" title="Identification of proteotoxic and proteoprotective bacteria that non-specifically affect proteins associated with neurodegenerative diseases">Identification of proteotoxic and proteoprotective bacteria that non-specifically affect proteins associated with neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Walker et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">August 27, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(23)00738-9" target="_blank" title="Unveiling TMEM106B: SARS-CoV-2’s secret entrance to the cell">Unveiling TMEM106B: SARS-CoV-2’s secret entrance to the cell</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lacrampe et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">August 03, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(23)00194-5" target="_blank" title="Targeting gasdermin E in neurodegenerative diseases">Targeting gasdermin E in neurodegenerative diseases</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhou et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">June 20, 2023</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2022.02.026","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.02.026/asset/59f10f77-64f5-44cd-af99-0bc0ff508589/main.assets/gr1.jpg" height="190" width="501" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/85748ff9-f9bc-438f-bfac-6b99d43dc2f5/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.02.026/asset/85a89381-932c-4d5c-a9f9-b1d3c7205902/main.assets/gr2.jpg" height="510" width="497" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/39d025e9-10e9-4005-8fe8-3a448fc30dcd/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.02.026/asset/e007aac5-86b4-4a52-b8d8-1bcfc6701910/main.assets/figs1.jpg" height="535" width="505" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/51fd2fe6-acd0-493d-9e61-12dd169c92d0/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.02.026/asset/648b9086-8a47-4f08-8011-d04bdd150a40/main.assets/figs2.jpg" height="477" width="659" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/d2f514ce-895e-4ee1-8929-24a84a4c2ea3/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.02.026/asset/b5fbf1d8-a6b4-44b9-b1e9-caa14b238f5b/main.assets/figs3.jpg" height="641" width="659" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/fcb2312e-9cdd-491a-8b50-a5d11e17f59f/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.02.026/asset/dc71aa9f-ece9-4f11-aa0d-15009e77c089/main.assets/gr3.jpg" height="188" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/936825cb-6d07-4fa4-ab47-c21da3da49a9/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.02.026/asset/551429da-a659-4e8e-8ece-c99b8d8fa347/main.assets/figs4.jpg" height="546" width="367" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/fa0fc319-5fde-4524-bf71-890892d07910/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.02.026/asset/d0b18d65-f49e-46ca-b68e-f85dc7ea63c6/main.assets/gr4.jpg" height="475" width="504" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/0ace9aad-344d-4f3c-9a97-967c231cac21/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.02.026/asset/7af3d11c-c3db-4074-bd43-2b96212d9a81/main.assets/gr5.jpg" height="465" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/5b96af82-3320-441f-bd3f-71a0df77f6ce/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.02.026/asset/4743ac11-53f4-49aa-b8e7-2d5afdf57613/main.assets/figs5.jpg" height="642" width="499" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/ecc671a6-d9ba-4b82-a326-22b85e89f15b/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.02.026/asset/3eb60cbc-bba6-4703-a99a-8a8edffbe77a/main.assets/figs6.jpg" height="615" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/72b2e44f-1a9a-4678-ac43-4964e4501b3f/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.02.026/asset/6c26ddd9-d35e-4623-98f4-a42c937c3ba9/main.assets/gr6.jpg" height="305" width="373" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/9e83a9d9-e5f4-476a-9873-6e9c270666f0/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.02.026/asset/59f10f77-64f5-44cd-af99-0bc0ff508589/main.assets/gr1.jpg" height="190" width="501" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/85748ff9-f9bc-438f-bfac-6b99d43dc2f5/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.02.026/asset/85a89381-932c-4d5c-a9f9-b1d3c7205902/main.assets/gr2.jpg" height="510" width="497" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/39d025e9-10e9-4005-8fe8-3a448fc30dcd/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.02.026/asset/e007aac5-86b4-4a52-b8d8-1bcfc6701910/main.assets/figs1.jpg" height="535" width="505" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/51fd2fe6-acd0-493d-9e61-12dd169c92d0/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.02.026/asset/648b9086-8a47-4f08-8011-d04bdd150a40/main.assets/figs2.jpg" height="477" width="659" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/d2f514ce-895e-4ee1-8929-24a84a4c2ea3/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.02.026/asset/b5fbf1d8-a6b4-44b9-b1e9-caa14b238f5b/main.assets/figs3.jpg" height="641" width="659" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/fcb2312e-9cdd-491a-8b50-a5d11e17f59f/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.02.026/asset/dc71aa9f-ece9-4f11-aa0d-15009e77c089/main.assets/gr3.jpg" height="188" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/936825cb-6d07-4fa4-ab47-c21da3da49a9/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.02.026/asset/551429da-a659-4e8e-8ece-c99b8d8fa347/main.assets/figs4.jpg" height="546" width="367" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/fa0fc319-5fde-4524-bf71-890892d07910/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.02.026/asset/d0b18d65-f49e-46ca-b68e-f85dc7ea63c6/main.assets/gr4.jpg" height="475" width="504" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/0ace9aad-344d-4f3c-9a97-967c231cac21/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.02.026/asset/7af3d11c-c3db-4074-bd43-2b96212d9a81/main.assets/gr5.jpg" height="465" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/5b96af82-3320-441f-bd3f-71a0df77f6ce/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.02.026/asset/4743ac11-53f4-49aa-b8e7-2d5afdf57613/main.assets/figs5.jpg" height="642" width="499" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/ecc671a6-d9ba-4b82-a326-22b85e89f15b/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.02.026/asset/3eb60cbc-bba6-4703-a99a-8a8edffbe77a/main.assets/figs6.jpg" height="615" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/72b2e44f-1a9a-4678-ac43-4964e4501b3f/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.02.026/asset/6c26ddd9-d35e-4623-98f4-a42c937c3ba9/main.assets/gr6.jpg" height="305" width="373" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.02.026/asset/9e83a9d9-e5f4-476a-9873-6e9c270666f0/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00259-8';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00259-8';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749593100689","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"185","issueNumber":"8","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867422002598","coverDate":"202204","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Homotypic fibrillization of TMEM106B across diverse neurodegenerative diseases","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc2ae92d504ed1',t:'MTc0OTU5MzEwMC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc2ae92d504ed1&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2022.02.026&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>