<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="6671e1a8-0b6f-4551-b171-d2d193e31d4a" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867420306139;journal:journal:cell;pageGroup:string:Article View;issue:issue:pii\:S0092867419X00148;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="An Autonomous Oscillation Times and Executes Centriole Biogenesis">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)30613-9">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/567f8620-ac3c-4440-8246-ef4db8ba1150/fx1.jpg">
    


        <meta property="og:description" content="Feedback-driven oscillations in centriolar Plk4 kinase levels—normally entrained by
the cell-cycle oscillator but capable of running autonomously—trigger and time centriole
biogenesis to ensure that daughter centrioles grow at the right time and to the right
size.">
    











        <meta name="twitter:title" content="An Autonomous Oscillation Times and Executes Centriole Biogenesis">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/567f8620-ac3c-4440-8246-ef4db8ba1150/fx1.jpg">
    


        <meta name="twitter:description" content="Feedback-driven oscillations in centriolar Plk4 kinase levels—normally entrained by
the cell-cycle oscillator but capable of running autonomously—trigger and time centriole
biogenesis to ensure that daughter centrioles grow at the right time and to the right
size.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>An Autonomous Oscillation Times and Executes Centriole Biogenesis: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="An Autonomous Oscillation Times and Executes Centriole Biogenesis"> <meta name="citation_date" content="2020/06/25"> <meta name="citation_online_date" content="2020/06/11"> <meta name="citation_volume" content="181"> <meta name="citation_issue" content="7"> <meta name="citation_firstpage" content="1566"> <meta name="citation_lastpage" content="1581.e27"> <meta name="citation_pmid" content="32531200"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867420306139"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867420306139"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867420306139"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2020.05.018"> <meta name="citation_pmid" content="32531200"> <meta name="citation_author" content="Mustafa G. Aydogan"><meta name="citation_author_email" content="mustafa.aydogan@path.ox.ac.uk"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Thomas L. Steinacker"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Mohammad Mofatteh"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Zachary M. Wilmott"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author_institution" content="Mathematical Institute, University of Oxford, Oxford OX2 6GG, UK"><meta name="citation_author" content="Felix Y. Zhou"><meta name="citation_author_institution" content="Ludwig Institute for Cancer Research, University of Oxford, Oxford OX3 7DQ, UK"><meta name="citation_author" content="Lisa Gartenmann"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Alan Wainman"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Saroj Saurya"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Zsofia A. Novak"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Siu-Shing Wong"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Alain Goriely"><meta name="citation_author_institution" content="Mathematical Institute, University of Oxford, Oxford OX2 6GG, UK"><meta name="citation_author" content="Michael A. Boemo"><meta name="citation_author_email" content="mb915@cam.ac.uk"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_author" content="Jordan W. Raff"><meta name="citation_author_email" content="jordan.raff@path.ox.ac.uk"><meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK"><meta name="citation_keywords" content="centriole; centriole duplication; centrosome; cell cycle; organelle biogenesis; organelle sizing; biological oscillator; biological timing; FCS"><meta name="citation_reference" content="citation_title=Evidence that a positive feedback loop drives centrosome maturation in fly embryos; citation_author=I. Alvarez-Rodrigo; citation_author=T.L. Steinacker; citation_author=S. Saurya; citation_author=P.T. Conduit; citation_author=J. Baumbach; citation_author=Z.A. Novak; citation_author=M.G. Aydogan; citation_author=A. Wainman; citation_author=J.W. Raff; citation_journal_title=eLife; citation_volume=8; citation_firstpage=D430; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The PLK4-STIL-SAS-6 module at the core of centriole duplication; citation_author=C. Arquint; citation_author=E.A. Nigg; citation_journal_title=Biochem. Soc. Trans.; citation_volume=44; citation_firstpage=1253; citation_lastpage=1263; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A homeostatic clock sets daughter centriole size in flies; citation_author=M.G. Aydogan; citation_author=A. Wainman; citation_author=S. Saurya; citation_author=T.L. Steinacker; citation_author=A. Caballe; citation_author=Z.A. Novak; citation_author=J. Baumbach; citation_author=N. Muschalik; citation_author=J.W. Raff; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=217; citation_firstpage=1233; citation_lastpage=1248; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Dissociation of centrosome replication events from cycles of DNA synthesis and mitotic division in hydroxyurea-arrested Chinese hamster ovary cells; citation_author=R. Balczon; citation_author=L. Bao; citation_author=W.E. Zimmer; citation_author=K. Brown; citation_author=R.P. Zinkowski; citation_author=B.R. Brinkley; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=130; citation_firstpage=105; citation_lastpage=115; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=SIMcheck: a Toolbox for Successful Super-resolution Structured Illumination Microscopy; citation_author=G. Ball; citation_author=J. Demmerle; citation_author=R. Kaufmann; citation_author=I. Davis; citation_author=I.M. Dobbie; citation_author=L. Schermelleh; citation_journal_title=Sci. Rep.; citation_volume=5; citation_firstpage=15915; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Centriole Biogenesis: From Identifying the Characters to Understanding the Plot; citation_author=N. Banterle; citation_author=P. Gönczy; citation_journal_title=Annu. Rev. Cell Dev. Biol.; citation_volume=33; citation_firstpage=23; citation_lastpage=49; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Centrosome amplification can initiate tumorigenesis in flies; citation_author=R. Basto; citation_author=K. Brunk; citation_author=T. Vinadogrova; citation_author=N. Peel; citation_author=A. Franz; citation_author=A. Khodjakov; citation_author=J.W. Raff; citation_journal_title=Cell; citation_volume=133; citation_firstpage=1032; citation_lastpage=1042; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Dissecting the function and assembly of acentriolar microtubule organizing centers in Drosophila cells in&nbsp;vivo; citation_author=J. Baumbach; citation_author=Z.A. Novak; citation_author=J.W. Raff; citation_author=A. Wainman; citation_journal_title=PLoS Genet.; citation_volume=11; citation_firstpage=e1005261; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Centrosomes and cilia in human disease; citation_author=M. Bettencourt-Dias; citation_author=F. Hildebrandt; citation_author=D. Pellman; citation_author=G. Woods; citation_author=S.A. Godinho; citation_journal_title=Trends Genet.; citation_volume=27; citation_firstpage=307; citation_lastpage=315; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Drosophila asterless and vertebrate Cep152 Are orthologs essential for centriole duplication; citation_author=S. Blachon; citation_author=J. Gopalakrishnan; citation_author=Y. Omori; citation_author=A. Polyanovsky; citation_author=A. Church; citation_author=D. Nicastro; citation_author=J. Malicki; citation_author=T. Avidor-Reiss; citation_journal_title=Genetics; citation_volume=180; citation_firstpage=2081; citation_lastpage=2094; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Asterless is a Polo-like kinase 4 substrate that both activates and inhibits kinase activity depending on its phosphorylation state; citation_author=C.J. Boese; citation_author=J. Nye; citation_author=D.W. Buster; citation_author=T.A. McLamarrah; citation_author=A.E. Byrnes; citation_author=K.C. Slep; citation_author=N.M. Rusan; citation_author=G.C. Rogers; citation_journal_title=Mol. Biol. Cell; citation_volume=29; citation_firstpage=2874; citation_lastpage=2886; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Mitotic trigger waves and the spatial coordination of the Xenopus cell cycle; citation_author=J.B. Chang; citation_author=J.E. Ferrell; citation_journal_title=Nature; citation_volume=500; citation_firstpage=603; citation_lastpage=607; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Multiple protein phosphatases are required for mitosis in Drosophila; citation_author=F. Chen; citation_author=V. Archambault; citation_author=A. Kar; citation_author=P. Lio’; citation_author=P.P. D’Avino; citation_author=R. Sinka; citation_author=K. Lilley; citation_author=E.D. Laue; citation_author=P. Deak; citation_author=L. Capalbo; citation_author=D.M. Glover; citation_journal_title=Curr. Biol.; citation_volume=17; citation_firstpage=293; citation_lastpage=303; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Cep152 acts as a scaffold for recruitment of Plk4 and CPAP to the centrosome; citation_author=O. Cizmecioglu; citation_author=M. Arnold; citation_author=R. Bahtz; citation_author=F. Settele; citation_author=L. Ehret; citation_author=U. Haselmann-Weiss; citation_author=C. Antony; citation_author=I. Hoffmann; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=191; citation_firstpage=731; citation_lastpage=739; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The constitution of protoplasm; citation_author=A. Claude; citation_journal_title=Science; citation_volume=97; citation_firstpage=451; citation_lastpage=456; citation_publication_date=1943; "><meta name="citation_reference" content="citation_title=Re-examining the role of Drosophila Sas-4 in centrosome assembly using two-colour-3D-SIM FRAP; citation_author=P.T. Conduit; citation_author=A. Wainman; citation_author=Z.A. Novak; citation_author=T.T. Weil; citation_author=J.W. Raff; citation_journal_title=eLife; citation_volume=4; citation_firstpage=1032; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The SCF/Slimb ubiquitin ligase limits centrosome amplification through degradation of SAK/PLK4; citation_author=I. Cunha-Ferreira; citation_author=A. Rodrigues-Martins; citation_author=I. Bento; citation_author=M. Riparbelli; citation_author=W. Zhang; citation_author=E. Laue; citation_author=G. Callaini; citation_author=D.M. Glover; citation_author=M. Bettencourt-Dias; citation_journal_title=Curr. Biol.; citation_volume=19; citation_firstpage=43; citation_lastpage=49; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Regulation of autophosphorylation controls PLK4 self-destruction and centriole number; citation_author=I. Cunha-Ferreira; citation_author=I. Bento; citation_author=A. Pimenta-Marques; citation_author=S.C. Jana; citation_author=M. Lince-Faria; citation_author=P. Duarte; citation_author=J. Borrego-Pinto; citation_author=S. Gilberto; citation_author=T. Amado; citation_author=D. Brito; citation_author=et al.; citation_journal_title=Curr. Biol.; citation_volume=23; citation_firstpage=2245; citation_lastpage=2254; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Waves of Cdk1 Activity in S Phase Synchronize the Cell Cycle in Drosophila Embryos; citation_author=V.E. Deneke; citation_author=A. Melbinger; citation_author=M. Vergassola; citation_author=S. Di Talia; citation_journal_title=Dev. Cell; citation_volume=38; citation_firstpage=399; citation_lastpage=412; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Asterless is a scaffold for the onset of centriole assembly; citation_author=N.S. Dzhindzhev; citation_author=Q.D. Yu; citation_author=K. Weiskopf; citation_author=G. Tzolovsky; citation_author=I. Cunha-Ferreira; citation_author=M. Riparbelli; citation_author=A. Rodrigues-Martins; citation_author=M. Bettencourt-Dias; citation_author=G. Callaini; citation_author=D.M. Glover; citation_journal_title=Nature; citation_volume=467; citation_firstpage=714; citation_lastpage=718; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A genome-wide RNAi screen to dissect centriole duplication and centrosome maturation in Drosophila; citation_author=J. Dobbelaere; citation_author=F. Josué; citation_author=S. Suijkerbuijk; citation_author=B. Baum; citation_author=N. Tapon; citation_author=J. Raff; citation_journal_title=PloS. Biol.; citation_volume=6; citation_firstpage=e224; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Plk4 phosphorylates Ana2 to trigger Sas6 recruitment and procentriole formation; citation_author=N.S. Dzhindzhev; citation_author=G. Tzolovsky; citation_author=Z. Lipinszki; citation_author=S. Schneider; citation_author=R. Lattao; citation_author=J. Fu; citation_author=J. Debski; citation_author=M. Dadlez; citation_author=D.M. Glover; citation_journal_title=Curr. Biol.; citation_volume=24; citation_firstpage=2526; citation_lastpage=2532; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Baseline Correction with Asymmetric Least Squares Smoothing. Leiden University Medical Centre report, 2005; citation_author=P.H. Eilers; citation_author=H.F. Boelens; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Perfect and Near-Perfect Adaptation in Cell Signaling; citation_author=J.E. Ferrell; citation_journal_title=Cell Syst.; citation_volume=2; citation_firstpage=62; citation_lastpage=67; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The centriole duplication cycle; citation_author=E.N. Fırat-Karalar; citation_author=T. Stearns; citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_volume=369; citation_firstpage=20130460; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Molecular regulation of hepatic fibrosis, an integrated cellular response to tissue injury; citation_author=S.L. Friedman; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=275; citation_firstpage=2247; citation_lastpage=2250; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Centrosome duplication continues in cycloheximide-treated Xenopus blastulae in the absence of a detectable cell cycle; citation_author=D.L. Gard; citation_author=S. Hafezi; citation_author=T. Zhang; citation_author=S.J. Doxsey; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=110; citation_firstpage=2033; citation_lastpage=2042; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=Organelle growth control through limiting pools of cytoplasmic components; citation_author=N.W. Goehring; citation_author=A.A. Hyman; citation_journal_title=Curr. Biol.; citation_volume=22; citation_firstpage=R330; citation_lastpage=R339; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Centriole assembly at a glance; citation_author=P. Gönczy; citation_author=G.N. Hatzopoulos; citation_journal_title=J.&nbsp;Cell Sci.; citation_volume=132; citation_firstpage=jcs228833; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Plk4 trans-autophosphorylation regulates centriole number by controlling betaTrCP-mediated degradation; citation_author=G. Guderian; citation_author=J. Westendorf; citation_author=A. Uldschmid; citation_author=E.A.E. Nigg; citation_journal_title=J.&nbsp;Cell Sci.; citation_volume=123; citation_firstpage=2163; citation_lastpage=2169; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Cep152 interacts with Plk4 and is required for centriole duplication; citation_author=E.M. Hatch; citation_author=A. Kulukian; citation_author=A.J. Holland; citation_author=D.W. Cleveland; citation_author=T. Stearns; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=191; citation_firstpage=721; citation_lastpage=729; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Polo-like kinase 4 kinase activity limits centrosome overduplication by autoregulating its own stability; citation_author=A.J. Holland; citation_author=W. Lan; citation_author=S. Niessen; citation_author=H. Hoover; citation_author=D.W. Cleveland; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=188; citation_firstpage=191; citation_lastpage=198; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Drosophila Cyclin B3 is required for female fertility and is dispensable for mitosis like Cyclin B; citation_author=H.W. Jacobs; citation_author=J.A. Knoblich; citation_author=C.F. Lehner; citation_journal_title=Genes Dev.; citation_volume=12; citation_firstpage=3741; citation_lastpage=3751; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=SciPy: Open Source Scientific Tools for Python; citation_author=E. Jones; citation_author=T. Oliphant; citation_author=P. Peterson; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Structural basis of the 9-fold symmetry of centrioles; citation_author=D. Kitagawa; citation_author=I. Vakonakis; citation_author=N. Olieric; citation_author=M. Hilbert; citation_author=D. Keller; citation_author=V. Olieric; citation_author=M. Bortfeld; citation_author=M.C. Erat; citation_author=I. Flückiger; citation_author=P. Gönczy; citation_author=M.O. Steinmetz; citation_journal_title=Cell; citation_volume=144; citation_firstpage=364; citation_lastpage=375; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Polo-like kinase 4 autodestructs by generating its Slimb-binding phosphodegron; citation_author=J.E. Klebba; citation_author=D.W. Buster; citation_author=A.L. Nguyen; citation_author=S. Swatkoski; citation_author=M. Gucek; citation_author=N.M. Rusan; citation_author=G.C. Rogers; citation_journal_title=Curr. Biol.; citation_volume=23; citation_firstpage=2255; citation_lastpage=2261; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Two Polo-like kinase 4 binding domains in Asterless perform distinct roles in regulating kinase stability; citation_author=J.E. Klebba; citation_author=B.J. Galletta; citation_author=J. Nye; citation_author=K.M. Plevock; citation_author=D.W. Buster; citation_author=N.A. Hollingsworth; citation_author=K.C. Slep; citation_author=N.M. Rusan; citation_author=G.C. Rogers; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=208; citation_firstpage=401; citation_lastpage=414; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Statistical accuracy in fluorescence correlation spectroscopy; citation_author=D.E. Koppel; citation_journal_title=Phys. Rev. A; citation_volume=10; citation_firstpage=1938; citation_lastpage=1945; citation_publication_date=1974; "><meta name="citation_reference" content="citation_title=Plk4-dependent phosphorylation of STIL is required for centriole duplication; citation_author=A.-S. Kratz; citation_author=F. Bärenz; citation_author=K.T. Richter; citation_author=I. Hoffmann; citation_journal_title=Biol. Open; citation_volume=4; citation_firstpage=370; citation_lastpage=377; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Autoamplification and Competition Drive Symmetry Breaking: Initiation of Centriole Duplication by the PLK4-STIL Network; citation_author=M. Leda; citation_author=A.J. Holland; citation_author=A.B. Goryachev; citation_journal_title=iScience; citation_volume=8; citation_firstpage=222; citation_lastpage=235; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Determination of the Plk4/Sak consensus phosphorylation motif using peptide spots arrays; citation_author=G.C. Leung; citation_author=C.S.W. Ho; citation_author=I.M. Blasutig; citation_author=J.M. Murphy; citation_author=F. Sicheri; citation_journal_title=FEBS Lett.; citation_volume=581; citation_firstpage=77; citation_lastpage=83; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Observing the cell in its native state: Imaging subcellular dynamics in multicellular organisms; citation_author=T.-L. Liu; citation_author=S. Upadhyayula; citation_author=D.E. Milkie; citation_author=V. Singh; citation_author=K. Wang; citation_author=I.A. Swinburne; citation_author=K.R. Mosaliganti; citation_author=Z.M. Collins; citation_author=T.W. Hiscock; citation_author=J. Shea; citation_author=et al.; citation_journal_title=Science; citation_volume=360; citation_firstpage=eaaq1392; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Periodic cyclin-Cdk activity entrains an autonomous Cdc14 release oscillator; citation_author=Y. Lu; citation_author=F.R. Cross; citation_journal_title=Cell; citation_volume=141; citation_firstpage=268; citation_lastpage=279; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Egg size, embryonic development time and ovoviviparity in Drosophila species; citation_author=T.A. Markow; citation_author=S. Beall; citation_author=L.M. Matzkin; citation_journal_title=J.&nbsp;Evol. Biol.; citation_volume=22; citation_firstpage=430; citation_lastpage=434; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Organellar relationships in the Golgi region of the pancreatic beta cell line, HIT-T15, visualized by high resolution electron tomography; citation_author=B.J. Marsh; citation_author=D.N. Mastronarde; citation_author=K.F. Buttle; citation_author=K.E. Howell; citation_author=J.R. McIntosh; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=98; citation_firstpage=2399; citation_lastpage=2406; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Cell Geometry: How Cells Count and Measure Size; citation_author=W.F. Marshall; citation_journal_title=Annu. Rev. Biophys.; citation_volume=45; citation_firstpage=49; citation_lastpage=64; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Control mechanism of the circadian clock for timing of cell division in&nbsp;vivo; citation_author=T. Matsuo; citation_author=S. Yamaguchi; citation_author=S. Mitsui; citation_author=A. Emi; citation_author=F. Shimoda; citation_author=H. Okamura; citation_journal_title=Science; citation_volume=302; citation_firstpage=255; citation_lastpage=259; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=RNAi of mitotic cyclins in Drosophila uncouples the nuclear and centrosome cycle; citation_author=M.L. McCleland; citation_author=P.H. O’Farrell; citation_journal_title=Curr. Biol.; citation_volume=18; citation_firstpage=245; citation_lastpage=254; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=An ordered pattern of Ana2 phosphorylation by Plk4 is required for centriole assembly; citation_author=T.A. McLamarrah; citation_author=D.W. Buster; citation_author=B.J. Galletta; citation_author=C.J. Boese; citation_author=J.M. Ryniawec; citation_author=N.A. Hollingsworth; citation_author=A.E. Byrnes; citation_author=C.W. Brownlee; citation_author=K.C. Slep; citation_author=N.M. Rusan; citation_author=G.C. Rogers; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=217; citation_firstpage=1217; citation_lastpage=1231; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The hidden rhythms of the dividing cell; citation_author=D.O. Morgan; citation_journal_title=Cell; citation_volume=141; citation_firstpage=224; citation_lastpage=226; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Mechanisms of organelle biogenesis govern stochastic fluctuations in organelle abundance; citation_author=S. Mukherji; citation_author=E.K. O’Shea; citation_journal_title=eLife; citation_volume=3; citation_firstpage=e02678; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Once and only once: mechanisms of centriole duplication and their deregulation in disease; citation_author=E.A. Nigg; citation_author=A.J. Holland; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=19; citation_firstpage=297; citation_lastpage=312; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Centrioles, centrosomes, and cilia in health and disease; citation_author=E.A. Nigg; citation_author=J.W. Raff; citation_journal_title=Cell; citation_volume=139; citation_firstpage=663; citation_lastpage=678; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Protein phosphatases in the regulation of mitosis; citation_author=J. Nilsson; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=218; citation_firstpage=395; citation_lastpage=409; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Design principles of biochemical oscillators; citation_author=B. Novák; citation_author=J.J. Tyson; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=9; citation_firstpage=981; citation_lastpage=991; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Asterless licenses daughter centrioles to duplicate for the first time in Drosophila embryos; citation_author=Z.A. Novak; citation_author=P.T. Conduit; citation_author=A. Wainman; citation_author=J.W. Raff; citation_journal_title=Curr. Biol.; citation_volume=24; citation_firstpage=1276; citation_lastpage=1282; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Direct interaction of Plk4 with STIL ensures formation of a single procentriole per parental centriole; citation_author=M. Ohta; citation_author=T. Ashikawa; citation_author=Y. Nozaki; citation_author=H. Kozuka-Hata; citation_author=H. Goto; citation_author=M. Inagaki; citation_author=M. Oyama; citation_author=D. Kitagawa; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=5267; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Phase separation of Polo-like kinase 4 by autoactivation and clustering drives centriole biogenesis; citation_author=J.-E. Park; citation_author=L. Zhang; citation_author=J.K. Bang; citation_author=T. Andresson; citation_author=F. DiMaio; citation_author=K.S. Lee; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=4959; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Precise measurement of diffusion coefficients using scanning fluorescence correlation spectroscopy; citation_author=Z. Petrásek; citation_author=P. Schwille; citation_journal_title=Biophys. J.; citation_volume=94; citation_firstpage=1437; citation_lastpage=1448; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Drosophila wee1 Has an Essential Role in the Nuclear Divisions of Early Embryogenesis; citation_author=D. Price; citation_author=S. Rabinovitch; citation_author=P.H. O’Farrell; citation_author=S.D. Campbell; citation_journal_title=Genetics; citation_volume=155; citation_firstpage=159; citation_lastpage=166; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=The second-order analysis of stationary point processes; citation_author=B.D. Ripley; citation_journal_title=J.&nbsp;Appl. Probab.; citation_volume=13; citation_firstpage=255; citation_lastpage=266; citation_publication_date=1976; "><meta name="citation_reference" content="citation_title=A multicomponent assembly pathway contributes to the formation of acentrosomal microtubule arrays in interphase Drosophila cells; citation_author=G.C. Rogers; citation_author=N.M. Rusan; citation_author=M. Peifer; citation_author=S.L. Rogers; citation_journal_title=Mol. Biol. Cell; citation_volume=19; citation_firstpage=3163; citation_lastpage=3178; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=The SCF Slimb ubiquitin ligase regulates Plk4/Sak levels to block centriole reduplication; citation_author=G.C. Rogers; citation_author=N.M. Rusan; citation_author=D.M. Roberts; citation_author=M. Peifer; citation_author=S.L. Rogers; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=184; citation_firstpage=225; citation_lastpage=239; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Comparison and accuracy of methods to determine the confocal volume for quantitative fluorescence correlation spectroscopy; citation_author=S. Rüttinger; citation_author=V. Buschmann; citation_author=B. Krämer; citation_author=R. Erdmann; citation_author=R. Macdonald; citation_author=F. Koberling; citation_journal_title=J.&nbsp;Microsc.; citation_volume=232; citation_firstpage=343; citation_lastpage=352; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Monitoring triplet state dynamics with fluorescence correlation spectroscopy: bias and correction; citation_author=A. Schönle; citation_author=C. Von Middendorff; citation_author=C. Ringemann; citation_author=S.W. Hell; citation_author=C. Eggeling; citation_journal_title=Microsc. Res. Tech.; citation_volume=77; citation_firstpage=528; citation_lastpage=536; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Estimating the Dimension of a Model; citation_author=G. Schwarz; citation_journal_title=Ann. Stat.; citation_volume=6; citation_firstpage=461; citation_lastpage=464; citation_publication_date=1978; "><meta name="citation_reference" content="citation_title=A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum; citation_author=N.C. Shaner; citation_author=G.G. Lambert; citation_author=A. Chammas; citation_author=Y. Ni; citation_author=P.J. Cranfill; citation_author=M.A. Baird; citation_author=B.R. Sell; citation_author=J.R. Allen; citation_author=R.N. Day; citation_author=M. Israelsson; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=407; citation_lastpage=409; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Far-red fluorescent tags for protein imaging in living tissues; citation_author=D. Shcherbo; citation_author=C.S. Murphy; citation_author=G.V. Ermakova; citation_author=E.A. Solovieva; citation_author=T.V. Chepurnykh; citation_author=A.S. Shcheglov; citation_author=V.V. Verkhusha; citation_author=V.Z. Pletnev; citation_author=K.L. Hazelwood; citation_author=P.M. Roche; citation_author=et al.; citation_journal_title=Biochem. J.; citation_volume=418; citation_firstpage=567; citation_lastpage=574; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=DNA-replication checkpoint control at the Drosophila midblastula transition; citation_author=O.C. Sibon; citation_author=V.A. Stevenson; citation_author=W.E. Theurkauf; citation_journal_title=Nature; citation_volume=388; citation_firstpage=93; citation_lastpage=97; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Protein synthesis and the cell cycle: centrosome reproduction in sea urchin eggs is not under translational control; citation_author=G. Sluder; citation_author=F.J. Miller; citation_author=R. Cole; citation_author=C.L. Rieder; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=110; citation_firstpage=2025; citation_lastpage=2032; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=Implementation of integral feedback control in biological systems; citation_author=P.R. Somvanshi; citation_author=A.K. Patel; citation_author=S. Bhartiya; citation_author=K.V. Venkatesh; citation_journal_title=Wiley Interdiscip. Rev. Syst. Biol. Med.; citation_volume=7; citation_firstpage=301; citation_lastpage=316; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A quantitative model for the cdc2 control of S phase and mitosis in fission yeast; citation_author=B. Stern; citation_author=P. Nurse; citation_journal_title=Trends Genet.; citation_volume=12; citation_firstpage=345; citation_lastpage=350; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Drosophila Wee1 kinase regulates Cdk1 and mitotic entry during embryogenesis; citation_author=J. Stumpff; citation_author=T. Duncan; citation_author=E. Homola; citation_author=S.D. Campbell; citation_author=T.T. Su; citation_journal_title=Curr. Biol.; citation_volume=14; citation_firstpage=2143; citation_lastpage=2148; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=CDK Substrate Phosphorylation and Ordering the Cell Cycle; citation_author=M.P. Swaffer; citation_author=A.W. Jones; citation_author=H.R. Flynn; citation_author=A.P. Snijders; citation_author=P. Nurse; citation_journal_title=Cell; citation_volume=167; citation_firstpage=1750; citation_lastpage=1761; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Quantitative Phosphoproteomics Reveals the Signaling Dynamics of Cell-Cycle Kinases in the Fission Yeast Schizosaccharomyces pombe; citation_author=M.P. Swaffer; citation_author=A.W. Jones; citation_author=H.R. Flynn; citation_author=A.P. Snijders; citation_author=P. Nurse; citation_journal_title=Cell Rep.; citation_volume=24; citation_firstpage=503; citation_lastpage=514; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Feedback loops in the Plk4-STIL-HsSAS6 network coordinate site selection for procentriole formation; citation_author=D. Takao; citation_author=K. Watanabe; citation_author=K. Kuroki; citation_author=D. Kitagawa; citation_journal_title=Biol. Open; citation_volume=8; citation_firstpage=bio047175; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=TrackMate: An open and extensible platform for single-particle tracking; citation_author=J.-Y. Tinevez; citation_author=N. Perry; citation_author=J. Schindelin; citation_author=G.M. Hoopes; citation_author=G.D. Reynolds; citation_author=E. Laplantine; citation_author=S.Y. Bednarek; citation_author=S.L. Shorte; citation_author=K.W. Eliceiri; citation_journal_title=Methods; citation_volume=115; citation_firstpage=80; citation_lastpage=90; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Robust, tunable biological oscillations from interlinked positive and negative feedback loops; citation_author=T.Y.-C. Tsai; citation_author=Y.S. Choi; citation_author=W. Ma; citation_author=J.R. Pomerening; citation_author=C. Tang; citation_author=J.E. Ferrell; citation_journal_title=Science; citation_volume=321; citation_firstpage=126; citation_lastpage=129; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Structures of SAS-6 suggest its organization in centrioles; citation_author=M. van Breugel; citation_author=M. Hirono; citation_author=A. Andreeva; citation_author=H.-A. Yanagisawa; citation_author=S. Yamaguchi; citation_author=Y. Nakazawa; citation_author=N. Morgner; citation_author=M. Petrovich; citation_author=I.-O. Ebong; citation_author=C.V. Robinson; citation_author=et al.; citation_journal_title=Science; citation_volume=331; citation_firstpage=1196; citation_lastpage=1199; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Structure of the SAS-6 cartwheel hub from Leishmania major; citation_author=M. van Breugel; citation_author=R. Wilcken; citation_author=S.H. McLaughlin; citation_author=T.J. Rutherford; citation_author=C.M. Johnson; citation_journal_title=eLife; citation_volume=3; citation_firstpage=e01812; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=FoCuS-point: software for STED fluorescence correlation and time-gated single photon counting; citation_author=D. Waithe; citation_author=M.P. Clausen; citation_author=E. Sezgin; citation_author=C. Eggeling; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=958; citation_lastpage=960; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A proteomics landscape of circadian clock in mouse liver; citation_author=Y. Wang; citation_author=L. Song; citation_author=M. Liu; citation_author=R. Ge; citation_author=Q. Zhou; citation_author=W. Liu; citation_author=R. Li; citation_author=J. Qie; citation_author=B. Zhen; citation_author=Y. Wang; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=1553; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Self-organization of Plk4 regulates symmetry breaking in centriole duplication; citation_author=S. Yamamoto; citation_author=D. Kitagawa; citation_journal_title=Nat. Comm.; citation_volume=10; citation_firstpage=1810; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Circadian gating of the cell cycle revealed in single cyanobacterial cells; citation_author=Q. Yang; citation_author=B.F. Pando; citation_author=G. Dong; citation_author=S.S. Golden; citation_author=A. van Oudenaarden; citation_journal_title=Science; citation_volume=327; citation_firstpage=1522; citation_lastpage=1526; citation_publication_date=2010; "><meta name="citation_abstract" content="<h2>Summary</h2><p>The accurate timing and execution of organelle biogenesis is crucial for cell physiology. Centriole biogenesis is regulated by Polo-like kinase 4 (Plk4) and initiates in S-phase when a daughter centriole grows from the side of a pre-existing mother. Here, we show that a Plk4 oscillation at the base of the growing centriole initiates and times centriole biogenesis to ensure that centrioles grow at the right time and to the right size. The Plk4 oscillation is normally entrained to the cell-cycle oscillator but can run autonomously of it—potentially explaining why centrioles can duplicate independently of cell-cycle progression. Mathematical modeling indicates that the Plk4 oscillation can be generated by a time-delayed negative feedback loop in which Plk4 inactivates the interaction with its centriolar receptor through multiple rounds of phosphorylation. We hypothesize that similar organelle-specific oscillations could regulate the timing and execution of organelle biogenesis more generally.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=10845&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9&amp;be=3624&amp;fe=10615&amp;dc=9560&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749586788155,%22n%22:0,%22f%22:3088,%22dn%22:3089,%22dne%22:3089,%22c%22:3089,%22s%22:3089,%22ce%22:3089,%22rq%22:3089,%22rp%22:3600,%22rpe%22:8838,%22dl%22:3606,%22di%22:9554,%22ds%22:9560,%22de%22:9865,%22dc%22:10615,%22l%22:10615,%22le%22:10760%7D,%22navigation%22:%7B%7D%7D&amp;fp=8004&amp;fcp=8004&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style>stream {
  position: relative;
  display: block;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930613-9" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)30613-9"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930613-9" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-203129" aria-expanded="false" aria-controls="login-links-203129" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-203129" id="login-links--203129" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930613-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930613-9" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930613-9" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-203129" data-opens="#links-dialog-203129" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-203129" aria-modal="true"><button data-closes="#links-dialog-203129" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930613-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930613-9" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930613-9" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.05.018">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(20)30613-9" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0014-8"><span>Volume 181</span>, <span>Issue 7</span></a></span><span class="meta-panel__pages"><span>p1566-1581.e27</span></span><span class="meta-panel__onlineDate">June 25, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=7&amp;pii=S0092-8674%2819%29X0014-8" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">An Autonomous Oscillation Times and Executes Centriole Biogenesis</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem" class="corresponding-author"><a href="#au1"><span property="givenName">Mustafa G.</span> <span property="familyName">Aydogan</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#fn2" role="doc-noteref">5</a></sup> <a id="eadd0010" href="mailto:mustafa.aydogan@path.ox.ac.uk" property="email" title="Send email to Mustafa G. Aydogan">mustafa.aydogan@path.ox.ac.uk</a></span> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au2"><span property="givenName">Thomas L.</span> <span property="familyName">Steinacker</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#fn2" role="doc-noteref">5</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au3"><span property="givenName">Mohammad</span> <span property="familyName">Mofatteh</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au4"><span property="givenName">Zachary M.</span> <span property="familyName">Wilmott</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#aff2" property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au5"><span property="givenName">Felix Y.</span> <span property="familyName">Zhou</span></a><sup class="xref"><a href="#aff3" property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au6"><span property="givenName">Lisa</span> <span property="familyName">Gartenmann</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au7"><span property="givenName">Alan</span> <span property="familyName">Wainman</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au8"><span property="givenName">Saroj</span> <span property="familyName">Saurya</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au9"><span property="givenName">Zsofia A.</span> <span property="familyName">Novak</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au10"><span property="givenName">Siu-Shing</span> <span property="familyName">Wong</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au11"><span property="givenName">Alain</span> <span property="familyName">Goriely</span></a><sup class="xref"><a href="#aff2" property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><a href="#au12"><span property="givenName">Michael A.</span> <span property="familyName">Boemo</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#fn1" role="doc-noteref">4</a></sup> <a id="eadd0015" href="mailto:mb915@cam.ac.uk" property="email" title="Send email to Michael A. Boemo">mb915@cam.ac.uk</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><a href="#au13"><span property="givenName">Jordan W.</span> <span property="familyName">Raff</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#fn3" role="doc-noteref">6</a></sup> <a id="eadd0020" href="mailto:jordan.raff@path.ox.ac.uk" property="email" title="Send email to Jordan W. Raff">jordan.raff@path.ox.ac.uk</a></span><span data-displayed-on="all"> … <span><span data-action="reveal">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide">Show less</span></span></span></span></div><div class="core-self-citation"><div id="core-affiliations-notes"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Mathematical Institute, University of Oxford, Oxford OX2 6GG, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Ludwig Institute for Cancer Research, University of Oxford, Oxford OX3 7DQ, UK</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">4</div><div id="fn1"><div id="ntpara0010" role="paragraph">Present address: Department of Pathology, University of Cambridge, Cambridge CB2 1QP, UK</div></div></div><div role="doc-footnote" data-has="label"><div class="label">5</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">6</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info"><div class="history"><div>Publication History:</div><div><span>Received April 5, 2019</span>; <span>Revised December 19, 2019</span>; <span>Accepted May 8, 2020</span>; <span>Published online June 11, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2020.05.018" property="sameAs" target="_blank">10.1016/j.cell.2020.05.018 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867420306139" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 The Author(s). Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution (CC BY 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2020.05.018" data-id="article-info-crossmark" class="crossmark__link"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div><div id="core-linked-content">
    

        

</div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0014-8"><img src="/cms/asset/0fd37f06-ae38-4845-85e5-5baaf425cdc6/cov200h.gif" alt="Cover Image - Cell, Volume 181, Issue 7"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2820%2930613-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article"><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results and Discussion</a></li><li><a href="#sec-3">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (2)</a></li><li><a href="#references">References</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930613-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.05.018">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2930613-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.05.018&amp;pii=S0092-8674%2820%2930613-9" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930613-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.05.018">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.05.018" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867420306139&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867420306139&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=7&amp;pii=S0092-8674%2819%29X0014-8" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)30575-4" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)30574-2" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Centriolar Plk4 levels oscillate and act as a switch for centriole biogenesis</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Oscillations may be generated via an Asl/Plk4 delayed negative feedback loop</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Plk4 oscillations are entrained and phase-locked by the Cdk/Cyclin oscillator (CCO)</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Plk4 oscillations can drive centriole biogenesis even when the CCO is perturbed</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">The accurate timing and execution of organelle biogenesis is crucial for cell physiology. Centriole biogenesis is regulated by Polo-like kinase 4 (Plk4) and initiates in S-phase when a daughter centriole grows from the side of a pre-existing mother. Here, we show that a Plk4 oscillation at the base of the growing centriole initiates and times centriole biogenesis to ensure that centrioles grow at the right time and to the right size. The Plk4 oscillation is normally entrained to the cell-cycle oscillator but can run autonomously of it—potentially explaining why centrioles can duplicate independently of cell-cycle progression. Mathematical modeling indicates that the Plk4 oscillation can be generated by a time-delayed negative feedback loop in which Plk4 inactivates the interaction with its centriolar receptor through multiple rounds of phosphorylation. We hypothesize that similar organelle-specific oscillations could regulate the timing and execution of organelle biogenesis more generally.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/8e2917d9-6e91-4753-8548-ef60ba042bb0/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2020.05.018/asset/567f8620-ac3c-4440-8246-ef4db8ba1150/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/8e2917d9-6e91-4753-8548-ef60ba042bb0/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;centriole&quot;" alt="Search on this keyword" rel="nofollow">centriole</a></li><li><a href="/action/doSearch?AllField=&quot;centriole+duplication&quot;" alt="Search on this keyword" rel="nofollow">centriole duplication</a></li><li><a href="/action/doSearch?AllField=&quot;centrosome&quot;" alt="Search on this keyword" rel="nofollow">centrosome</a></li><li><a href="/action/doSearch?AllField=&quot;cell+cycle&quot;" alt="Search on this keyword" rel="nofollow">cell cycle</a></li><li><a href="/action/doSearch?AllField=&quot;organelle+biogenesis&quot;" alt="Search on this keyword" rel="nofollow">organelle biogenesis</a></li><li><a href="/action/doSearch?AllField=&quot;organelle+sizing&quot;" alt="Search on this keyword" rel="nofollow">organelle sizing</a></li><li><a href="/action/doSearch?AllField=&quot;biological+oscillator&quot;" alt="Search on this keyword" rel="nofollow">biological oscillator</a></li><li><a href="/action/doSearch?AllField=&quot;biological+timing&quot;" alt="Search on this keyword" rel="nofollow">biological timing</a></li><li><a href="/action/doSearch?AllField=&quot;FCS&quot;" alt="Search on this keyword" rel="nofollow">FCS</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">Albert Claude’s landmark paper (<a id="crosref0115" href="#bib15" role="doc-biblioref" data-xml-rid="bib15">Claude, 1943</a>) challenged the idea that cells are a mere bag of enzymes whose contents grow freely in the cytoplasm with no active regulation. We now appreciate the diverse and compact nature of the many organelles in the cytoplasm (<a id="crosref0120" href="#bib44" role="doc-biblioref" data-xml-rid="bib44">Marsh et&nbsp;al., 2001</a>), yet the physical mechanisms that regulate the number and size of these organelles remain largely unknown (<a id="crosref0125" href="#bib45" role="doc-biblioref" data-xml-rid="bib45">Marshall, 2016</a>). For most organelles in the cell, however, this question has been difficult to address, as the variation in their numbers and 3D-shape has made it challenging to monitor their growth—or to even determine which parameter (e.g., their surface area, volume, or perhaps the amount of a limiting component) best defines their size.</div><div id="p0035" role="paragraph">Centrioles are highly structured organelles that form centrosomes and cilia (<a id="crosrefs0010" href="#bib9" role="doc-biblioref" data-xml-rid="bib9 bib51 bib52">Bettencourt-Dias et&nbsp;al., 2011; Nigg and Holland, 2018; Nigg and Raff, 2009</a>). Their linear structure and tightly controlled pattern of duplication makes them an attractive model with which to study organelle biogenesis (<a id="crosrefs0015" href="#bib27" role="doc-biblioref" data-xml-rid="bib27 bib45">Goehring and Hyman, 2012; Marshall, 2016</a>). Most cells are born with a single pair of centrioles that duplicate precisely once during S-phase, when a daughter centriole grows out orthogonally from the base of each mother until it reaches the same size as its mother (<a id="crosrefs0020" href="#bib6" role="doc-biblioref" data-xml-rid="bib6 bib24 bib51">Banterle and Gönczy, 2017; Fırat-Karalar and Stearns, 2014; Nigg and Holland, 2018</a>). To monitor the dynamics of centriole growth, we recently examined living syncytial <i>Drosophila</i> embryos where we could follow the assembly of hundreds of centrioles as they duplicate in near-synchrony in a common cytoplasm (<a id="crosref0130" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>). These studies revealed that centriole growth in these embryos is homeostatic: when centrioles grow slowly, they grow for a longer period; when centrioles grow quickly, they grow for a shorter period. As a result, centrioles grow to a consistent size.</div><div id="p0040" role="paragraph">Polo-like kinase 4 (Plk4) is the master regulator of centriole biogenesis and it is initially recruited to a ring around the mother centriole, but this ring resolves into a single focus on the side of the mother, defining the site of daughter centriole assembly (<a id="crosrefs0025" href="#bib6" role="doc-biblioref" data-xml-rid="bib6 bib24 bib39 bib51 bib74">Banterle and Gönczy, 2017; Fırat-Karalar and Stearns, 2014; Leda et&nbsp;al., 2018; Nigg and Holland, 2018; Takao et&nbsp;al., 2019</a>). Unexpectedly, we found that Plk4 not only determines the position of this site, but also helps to establish the inverse relationship between the rate and period of daughter centriole growth (<a id="crosref0135" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>). Plk4 presumably influences the rate of centriole growth, at least in part, by phosphorylating Ana2/STIL to promote its interaction with Sas-6 and, consequently, the assembly of the central cartwheel (<a id="crosrefs0030" href="#bib21" role="doc-biblioref" data-xml-rid="bib21 bib38 bib56">Dzhindzhev et&nbsp;al., 2014; Kratz et&nbsp;al., 2015; Ohta et&nbsp;al., 2014</a>), the 9-fold symmetric structure that forms the backbone of the growing daughter centriole (<a id="crosrefs0035" href="#bib34" role="doc-biblioref" data-xml-rid="bib34 bib77 bib78">Kitagawa et&nbsp;al., 2011; van Breugel et&nbsp;al., 2011, 2014</a>). It is less clear, however, how Plk4 might influence the period of centriole growth.</div><div id="p0045" role="paragraph">Recent studies have shown that Plk4 localizes to centrioles in a cyclical manner in both fly embryos (<a id="crosref0140" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>) and human cultured cells (<a id="crosref0145" href="#bib74" role="doc-biblioref" data-xml-rid="bib74">Takao et&nbsp;al., 2019</a>), but the functional significance of this localization pattern is unclear. Here, we show that a Plk4 oscillation at the base of the growing centriole initiates and times centriole biogenesis in fly embryos.</div></section><section id="sec-2"><h2>Results and Discussion</h2><section id="sec-2-1"><h3>Plk4 Levels Oscillate at the Base of Growing Daughter Centrioles</h3><div id="p0050" role="paragraph">To investigate the cyclical recruitment of Plk4 to the centrioles, we generated flies transgenically expressing Plk4-mNeonGreen (Plk4-NG) under the control of its own promoter in a <i>Plk4</i> mutant background. We monitored centriolar Plk4-NG levels in living <i>Drosophila</i> syncytial embryos, where the duration of S-phase gradually elongates over nuclear cycles 11–13 (<a id="crosref0150" href="#fig1">Figures 1</a>A, <a id="crosref0155" href="#figs1">S1</a>, and <a id="crosref0160" href="#figs2">S2</a>A; <a id="crosref0165" href="#mmc3">Video S1</a>). Centriolar Plk4-NG levels oscillated during each cycle: levels started to rise in M-phase, peaked in early-mid S-phase, and were minimal by the next M-phase (<a id="crosref0170" href="#fig1">Figures 1</a>A and <a id="crosref0175" href="#figs2">S2</a>A). We fit the S-phase oscillations in individual embryos (<a id="crosref0180" href="#figs1">Figures S1</a>C and S1D) to derive an average S-phase oscillation for each cycle (<a id="crosref0185" href="#fig1">Figure&nbsp;1</a>B).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/785cb90b-5e6f-4a01-ae43-b21d5a345ac3/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2020.05.018/asset/9a0ada8a-fa18-4499-89b4-369f484a5d1d/main.assets/gr1.jpg" height="809" width="762" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/785cb90b-5e6f-4a01-ae43-b21d5a345ac3/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Plk4 Levels Oscillate at the Centriole in a Process Entrained by the CCO</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Top panel: micrograph shows an image from a time-lapse movie of an embryo expressing Plk4-NG. Middle panels: micrographs illustrate the centriolar Plk4-NG oscillation during nuclear cycle 12—obtained by superimposing all the Plk4-NG foci (n&nbsp;= 60) at each time point (see <a id="crosref1375" href="#sec-3">STAR Methods</a>). Bottom panel: quantification of centriolar Plk4-NG levels during nuclear cycles 11–13 in a single embryo (red arrows highlight equivalent time points in the middle panels).</div><div id="fspara0020" role="paragraph">(B) Graphs show the mathematical regression of centriolar Plk4-NG dynamics during S-phase of cycles 11–13 (regression mean ± SEM). R<sup>2</sup> values indicate goodness-of-fit. N ≥ 15 embryos; n&nbsp;= 24, 37, and 53 centrioles (mean) per embryo over cycles 11–13, respectively.</div><div id="fspara0025" role="paragraph">(C) The bar charts quantify the oscillation parameters—derived from the data shown in (B). Data are presented as mean ± SD. Statistical significance was assessed using an ordinary one-way ANOVA test (for Gaussian-distributed data) or a Kruskal-Wallis test (<sup>∗∗∗</sup>p&nbsp;&lt; 0.001; <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001; ns, not significant).</div><div id="fspara0030" role="paragraph">(D) Micrographs show, and pie charts quantify, the distribution of Plk4-NG at centrioles assessed by 3D-SIM at the indicated phases of the nuclear cycle (see <a id="crosref1380" href="#sec-3">STAR Methods</a>). N&nbsp;= 6 embryos per cell-cycle stage; n&nbsp;= 20 centrioles per embryo; all images were scored blindly by 3 assessors and the mean score is shown (scale bar, 0.5&nbsp;μm).</div><div id="fspara0035" role="paragraph">(E) Graph shows the mean regression of Plk4-NG oscillations in nuclear cycle 12 of WT embryos (green), or in embryos where the genetic dose of either cyclin B (<i>CycB</i><sup><i>1/2</i></sup>; blue) or grapes (<i>Drosophila</i> Chk1) (<i>grp</i><sup><i>1/2</i></sup>; red) has been halved to slow or speed-up the nuclear cycles, respectively. Dashed lines mark the center (peak) of the Plk4-NG oscillations (denoted with C), and dotted lines indicate the time of NEB (denoted with N) for each genotype. N ≥ 14 embryos for each condition; n&nbsp;= 55, 43, and 44 centrioles (mean) per embryo in WT, <i>CycB</i><sup><i>1/2</i></sup>, and <i>grp</i><sup><i>1/2</i></sup> embryos, respectively. To clearly illustrate the phase shift in the oscillations, the highest mean fluorescence signal for each group was normalized to 1.</div><div id="fspara0040" role="paragraph">(F) Bar charts quantify the time at which the Plk4-NG oscillations peaked, the length of S-phase, and the ratio between them (C/N)—derived from the data shown in (E). Data are presented as mean ± SD. Statistical significance was assessed using an ordinary one-way ANOVA test (for Gaussian-distributed data) or a Kruskal-Wallis test (<sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001; ns, not significant).</div><div id="fspara0045" role="paragraph">See also <a id="crosref1385" href="#fig6">Figures 6</a>, <a id="crosref1390" href="#figs1">S1</a>, and <a id="crosref1395" href="#figs2">S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/337913a2-787e-44db-81c2-0794b921ef91/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2020.05.018/asset/ccb63240-e19a-4932-858d-3071936bc8a2/main.assets/figs1.jpg" height="915" width="731" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/337913a2-787e-44db-81c2-0794b921ef91/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Summary of the Protocol for Image Acquisition, Processing, and Analysis of the Plk4-NG Oscillations, Related to <a id="crosref1540" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0215" role="paragraph">(A) Diagram illustrates the centrioles in ~2&nbsp;h old embryo expressing Plk4-NG being imaged on a spinning-disk confocal system.</div><div id="fspara0220" role="paragraph">(B) Micrograph shows a typical image of the tracks of the Plk4-NG centrioles in S-phase of cycle 12, tracked using the ImageJ plugin, TrackMate.</div><div id="fspara0225" role="paragraph">(C) Graphs show the Plk4-NG oscillation during cycle 12 in a single embryo quantified from the tracks of either several individual centriole pairs (i), or the Mean ± SD oscillation calculated from the tracks of &gt; 90 centriole pairs (ii). The data for each embryo was then regressed using a Lorentzian equation (<i>red</i> line, iii)—see (D) for an explanation of the rationale for choosing this function. This process was repeated for multiple embryos to calculate a Mean ± SEM regression for nuclear cycle 12 (iv). <i>R</i><sup><i>2</i></sup> values indicate the goodness-of-fit (Mean ± SD) of the regression. CS&nbsp;= time of centrosome separation (set to 0); NEB&nbsp;= time of nuclear envelope breakdown.</div><div id="fspara0230" role="paragraph">(D) Table shows the various models that were tested to fit the Plk4-NG oscillation data. <i>R</i><sup><i>2</i></sup> and <i>SS</i><sub><i>Abs</i></sub> (absolute sum of squares) values indicate the goodness of fit. The Lorentzian function was the best fit for the majority of embryos, so it was used for all further analyses.</div><div id="fspara0235" role="paragraph">Further details of these models are provided in <a id="crosref1545" href="#sec-3">STAR Methods</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/46c5fdf4-b243-4ee0-acb4-83ebdec9c5c4/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2020.05.018/asset/6e8b3c5b-bba6-4f5e-9611-27e59239cdeb/main.assets/figs2.jpg" height="790" width="763" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/46c5fdf4-b243-4ee0-acb4-83ebdec9c5c4/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Plk4-NG Oscillations in Individual Embryos, Related to <a id="crosref1550" href="#fig1">Figures 1</a> and <a id="crosref1555" href="#fig5">5</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0245" role="paragraph">(A) Graphs show the Mean ± SD centriolar fluorescence intensity of Plk4-NG (two copies of a transgene expressed from its own promoter in a <i>Plk4</i> null mutant background) during nuclear cycles 11-13 in 5 different embryos imaged on a spinning-disk confocal system. n&nbsp;= 26 centrioles (mean) tracked starting from cycle 11 per embryo. See <a id="crosref1560" href="#sec-3">STAR Methods</a> for full details of image acquisition and data analysis.</div><div id="fspara0250" role="paragraph">(B) Same as in (A), but showing the Plk4-NG oscillation in 5 embryos arrested in interphase by the injection of dsRNAs against Cyclin A, B and B3.</div><div id="fspara0255" role="paragraph">See <a id="crosref1565" href="#fig5">Figure&nbsp;5</a>A for further details on sample numbers and experimental protocol.</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph"><div id="mmc3" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc3" data-type="mp4"><span id="video_stream_mmc3_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhNjlmMDAxZTM2NWFkYjMxZWM3NGI5ZDc4NjFjMzA2NSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.jpW2kvxkHLpGAwg2jN66ZA8G9Y5FGD9paJNOs9NWIjEjwTw--hN4d_bHR1wIRsoTVb1GC-A_o2oS8aLCDTL7B_mB2Fl4q-KZLmCCdJ9UAAvQwo394yQFeGAgFPCVL9U1GWAlyd6v8geHgWtHka3cX7hUi6YSKR0WGBhfYRjMnhDjiOfvKYlmkzsO4jXZpqwZwxT2kVYo0X7u7C-nY4w204cJTUCgjnuzRrlTAk7b-SurBSM5HpJ4fZrHnryAtriNHpJvG4-US8HOTE7u3qIhVdkd5_yChKcv7DODu1mB07iV0Q6W4ShGmM88uCIikC7BzadYErcPjZ1lm0r75iaX2A</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhNjlmMDAxZTM2NWFkYjMxZWM3NGI5ZDc4NjFjMzA2NSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.jpW2kvxkHLpGAwg2jN66ZA8G9Y5FGD9paJNOs9NWIjEjwTw--hN4d_bHR1wIRsoTVb1GC-A_o2oS8aLCDTL7B_mB2Fl4q-KZLmCCdJ9UAAvQwo394yQFeGAgFPCVL9U1GWAlyd6v8geHgWtHka3cX7hUi6YSKR0WGBhfYRjMnhDjiOfvKYlmkzsO4jXZpqwZwxT2kVYo0X7u7C-nY4w204cJTUCgjnuzRrlTAk7b-SurBSM5HpJ4fZrHnryAtriNHpJvG4-US8HOTE7u3qIhVdkd5_yChKcv7DODu1mB07iV0Q6W4ShGmM88uCIikC7BzadYErcPjZ1lm0r75iaX2A" class="cloudflare-stream-player ready tracked" id="video_stream_mmc3" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.7821%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhNjlmMDAxZTM2NWFkYjMxZWM3NGI5ZDc4NjFjMzA2NSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.jpW2kvxkHLpGAwg2jN66ZA8G9Y5FGD9paJNOs9NWIjEjwTw--hN4d_bHR1wIRsoTVb1GC-A_o2oS8aLCDTL7B_mB2Fl4q-KZLmCCdJ9UAAvQwo394yQFeGAgFPCVL9U1GWAlyd6v8geHgWtHka3cX7hUi6YSKR0WGBhfYRjMnhDjiOfvKYlmkzsO4jXZpqwZwxT2kVYo0X7u7C-nY4w204cJTUCgjnuzRrlTAk7b-SurBSM5HpJ4fZrHnryAtriNHpJvG4-US8HOTE7u3qIhVdkd5_yChKcv7DODu1mB07iV0Q6W4ShGmM88uCIikC7BzadYErcPjZ1lm0r75iaX2A" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.05.018/attachment/872f6c67-d9d6-4a06-9f10-2b54899b2006/mmc3.mp4" download="mmc3.mp4" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Video (22.67 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Video S1. Monitoring Plk4-NG Oscillations in a <i>Drosophila</i> Embryo, Related to Figure&nbsp;1</div><div id="espara0010" role="paragraph">Time-lapse video of an embryo expressing two copies of Plk4-NG (expressed transgenically from the endogenous <i>Plk4</i> promoter) in a <i>Plk4</i> mutant background, observed on a spinning-disk confocal microscope through nuclear cycles 11-13. The movie is a maximum-intensity projection that has been photo-bleach corrected, but not background subtracted for visual clarity. Time (min:sec) is shown at the top left, and stage of the cell cycle is indicated at the bottom left.</div></div></div></div><div id="p0060" role="paragraph">Not surprisingly, the Plk4 oscillations appeared to be entrained by the core Cdk/Cyclin oscillator as their period increased as nuclear cycles slowed during cycles 11–13 (<a id="crosref0190" href="#fig1">Figure&nbsp;1</a>C). Moreover, genetically altering the duration of the nuclear cycles elicited corresponding alterations in the Plk4 oscillation period (<a id="crosref0195" href="#fig1">Figures 1</a>E and 1F). Interestingly, however, the Plk4 oscillation exhibited adaptive behavior: as the period (<i>T</i>) of the oscillation tended to increase at successive cycles, its amplitude (<i>A</i>) tended to decrease, so that the total amount of Plk4 recruited to centrioles—i.e., the area under the S-phase oscillation curve (area under the curve [Ω])—remained relatively constant (<a id="crosref0200" href="#fig1">Figure&nbsp;1</a>C).</div><div id="p0065" role="paragraph">Plk4 is initially recruited to a ring around the mother centriole that resolves into a single hub that defines the site of daughter centriole assembly (<a id="crosrefs0040" href="#bib6" role="doc-biblioref" data-xml-rid="bib6 bib24 bib51">Banterle and Gönczy, 2017; Fırat-Karalar and Stearns, 2014; Nigg and Holland, 2018</a>). To examine how this localization related to the Plk4 oscillations, we used 3D-structured illumination super-resolution microscopy (3D-SIM) to assess the centriolar localization of Plk4 during the nuclear cycles in living embryos. Plk4-NG was only very briefly detectable in a ring during late-mitosis; at all other stages it appeared largely as a single hub (<a id="crosref0205" href="#fig1">Figure&nbsp;1</a>D). Thus, the recruitment and loss of Plk4 from the centriole wall is not responsible for the S-phase oscillation we observe in these embryos; instead, centriolar Plk4-NG levels oscillate at the base of the growing daughter centriole.</div></section><section id="sec-2-2"><h3>Plk4 Oscillations Time and Execute Centriole Biogenesis</h3><div id="p0070" role="paragraph">To test whether the Plk4 oscillations were important for centriole biogenesis, we generated flies co-expressing Plk4-NG (in a <i>Plk4</i> mutant background) and the centriole cartwheel component Sas-6-mCherry, which is irreversibly incorporated into the base of the growing daughter centriole cartwheel and can be used to monitor centriole growth in fly embryos (<a id="crosref0210" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>). These flies laid embryos that often failed to hatch (<a id="crosref0215" href="#figs3">Figure&nbsp;S3</a>C), but we simultaneously measured Plk4 oscillations and centriole growth in those embryos that appeared to be developing normally (<a id="crosref0220" href="#fig2">Figures 2</a>A, <a id="crosref0225" href="#figs3">S3</a>A, and S3B; <a id="crosref0230" href="#mmc4">Video S2</a>). The mother centrioles in these embryos were often slightly delayed in initiating daughter centriole growth (<a id="crosref0235" href="#fig2">Figures 2</a>A, <a id="crosref0240" href="#figs3">S3</a>D, and S3E), allowing us to measure the amount of Plk4 at the centrioles when daughter centrioles either started or stopped growing (<a id="crosref0245" href="#fig2">Figure&nbsp;2</a>A, colored dotted lines).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/e36df0fd-1adf-4327-9dcb-309d64f6b160/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2020.05.018/asset/a08a93e1-cc39-4c4d-9509-bdfbfc49f9e4/main.assets/gr2.jpg" height="520" width="673" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/e36df0fd-1adf-4327-9dcb-309d64f6b160/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Plk4 Oscillations Initiate and Time Centriole Biogenesis</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0055" role="paragraph">(A) Graphs show the mean regression of Plk4-NG oscillations (red, green, and blue lines for cycles 11–13, respectively) and centriole growth (monitored by Sas-6-mCherry incorporation, black lines) measured simultaneously in embryos during S-phase of cycles 11–13. For ease of presentation, the SEM for these data are not shown, but are presented in <a id="crosref1400" href="#figs3">Figures S3</a>A and S3B. Dotted lines indicate the centriolar Plk4 levels at which centrioles “start” or “stop” growing. N&nbsp;= 17 embryos (cycles 11 and 12), and 8 embryos (cycle 13); n&nbsp;= 19, 31, and 45 centrioles (mean) per embryo in cycles 11–13, respectively. See <a id="crosref1405" href="#sec-3">STAR Methods</a> for an explanation of data normalization and scaling.</div><div id="fspara0060" role="paragraph">(B) Bar charts quantify the centriolar Plk4-NG threshold levels at which centrioles start and stop growing during cycles 11–13—derived from the data shown in (A). Data are presented as mean ± SD. Statistical significance was assessed using an unpaired t test with Welch’s correction (for Gaussian-distributed data) or an unpaired Mann-Whitney test (ns, not significant).</div><div id="fspara0065" role="paragraph">(C) Eight embryos in which the centrioles did not grow (<a id="crosref1410" href="#figs3">Figures S3</a>D and S3E) were excluded from the analysis shown in (A) and (B); the scatter graph shown here illustrates how the mean amplitude of the Plk4 oscillations in each of these eight embryos (red dots) tended to be lower than the mean amplitude (±SEM) of the Plk4 oscillations in the embryos where the centrioles did grow.</div><div id="fspara0070" role="paragraph">See also <a id="crosref1415" href="#figs3">Figure&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/e5c330bb-e43b-4625-8b97-2a98e790edc4/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2020.05.018/asset/4f544576-8845-4516-8dfe-bedae930392f/main.assets/figs3.jpg" height="950" width="628" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/e5c330bb-e43b-4625-8b97-2a98e790edc4/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Simultaneously Measuring Centriole Growth and the Plk4 Oscillation in the Same Embryos, Related to <a id="crosref1570" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0265" role="paragraph">(A and B) Graphs show the same data presented in <a id="crosref1575" href="#fig2">Figure&nbsp;2</a>A, but with the SEM included (as these error bars were omitted from <a id="crosref1580" href="#fig2">Figure&nbsp;2</a>A for ease of presentation). CS&nbsp;= centrosome separation and NEB&nbsp;= nuclear envelope breakdown. <i>R</i><sup><i>2</i></sup> values indicate the goodness of fit.</div><div id="fspara0270" role="paragraph">(C) Graph quantifies the embryo hatching frequency in embryos laid by either wild-type (Oregon-R) females or females simultaneously expressing Sas-6-mCherry and Plk4-NG in a <i>Plk4</i> mutant background (all mated with WT males). At least 4 technical repeats were carried out over several days, and a total of at least 400 embryos were analyzed.</div><div id="fspara0275" role="paragraph">(D) Cartoon graphs (i.e., imaginary data) illustrate the three different centriole growth phenotypes we observed in the <i>Plk4</i> mutant embryos that simultaneously express 2 copies of Plk4-NG and one copy of Sas-6-mCherry. In our previous analysis of centriole growth kinetics (<a id="crosref1585" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>) almost all embryos started to incorporate Sas-6-GFP at the very start of S-phase (“Growth on time,” left graph). In the embryos analyzed here (with a more complicated genotype, and expressing Sas-6-mCherry rather than Sas-6-GFP), some of the embryos exhibited a clear delay in initiating the incorporation of Sas-6-mCherry (“Late growth,” middle graph), while others did not appear to incorporate significant amounts of Sas-6-mCherry at all (“No growth,” right graph).</div><div id="fspara0280" role="paragraph">(E) Pie charts quantify the percentage of embryos exhibiting each centriole growth phenotype at each nuclear cycle. Note that embryos exhibiting the “No growth” phenotype were excluded from the analysis shown in (A) and (B) and in <a id="crosref1590" href="#fig2">Figure&nbsp;2</a>A, although the amplitude of the Plk4 oscillations in these embryos was analyzed separately (<a id="crosref1595" href="#fig2">Figure&nbsp;2</a>C): we observed 8 embryos in total that exhibited the “No growth” phenotype (1 in cycle 12, and 7 in cycle 13). Centriolar Plk4-NG levels continued to oscillate in these embryos, and the scatter graph shown in <a id="crosref1600" href="#fig2">Figure&nbsp;2</a>C plots the peak amplitude of the Plk4-NG oscillations in these 8 embryos overlaid on the average “threshold” level of Plk4-NG at which centrioles started to grow in the population of embryos that did exhibit Sas-6-mCherry incorporation. This threshold was very similar at cycle 12 and 13, so the threshold shown in <a id="crosref1605" href="#fig2">Figure&nbsp;2</a>C is taken from cycle 13 embryos (as 7 of the 8 embryos shown here were at cycle 13). The Plk4-NG oscillation in all but one of the 8 embryos failed to reach the average “threshold” level that would normally initiate centriole growth in these embryos.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI4NWY3YjMwYTdmOGExZjVmMTJiNGM5NTE3MWY0ZmEzZiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.SKQ2dHVAJgXHBpgy21If_Jblwy6YQYST2l47SiKDJM0W_4cACvXZVvMVgl4hl9ua-KTR-KteEZUuFY-l61RZhclEvz0TPAY8HaP-jpmNG3ZZV3FU5B55Ny4wduFEIFHGgDtUj9GgCq0h72QG_qhC2GpRADUDor8Z_Ch2SNraPM51RFUesPl37Y9ysaFIVjwYBkLYAIpA3XwMCvzywwyfP1-VqedjmvpQfTvIRUYPza-9KuF2y1LxAC0FznOAZUtQ-jOeYpGA3UwtkL1C0vQQkG4aH7l8jzRlrBJ12hYcx_3uiYjupjf5idt27Xp9D7b2xzpFyI2mS_dTdMEG_f69_Q</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI4NWY3YjMwYTdmOGExZjVmMTJiNGM5NTE3MWY0ZmEzZiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.SKQ2dHVAJgXHBpgy21If_Jblwy6YQYST2l47SiKDJM0W_4cACvXZVvMVgl4hl9ua-KTR-KteEZUuFY-l61RZhclEvz0TPAY8HaP-jpmNG3ZZV3FU5B55Ny4wduFEIFHGgDtUj9GgCq0h72QG_qhC2GpRADUDor8Z_Ch2SNraPM51RFUesPl37Y9ysaFIVjwYBkLYAIpA3XwMCvzywwyfP1-VqedjmvpQfTvIRUYPza-9KuF2y1LxAC0FznOAZUtQ-jOeYpGA3UwtkL1C0vQQkG4aH7l8jzRlrBJ12hYcx_3uiYjupjf5idt27Xp9D7b2xzpFyI2mS_dTdMEG_f69_Q" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 78.9469%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI4NWY3YjMwYTdmOGExZjVmMTJiNGM5NTE3MWY0ZmEzZiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.SKQ2dHVAJgXHBpgy21If_Jblwy6YQYST2l47SiKDJM0W_4cACvXZVvMVgl4hl9ua-KTR-KteEZUuFY-l61RZhclEvz0TPAY8HaP-jpmNG3ZZV3FU5B55Ny4wduFEIFHGgDtUj9GgCq0h72QG_qhC2GpRADUDor8Z_Ch2SNraPM51RFUesPl37Y9ysaFIVjwYBkLYAIpA3XwMCvzywwyfP1-VqedjmvpQfTvIRUYPza-9KuF2y1LxAC0FznOAZUtQ-jOeYpGA3UwtkL1C0vQQkG4aH7l8jzRlrBJ12hYcx_3uiYjupjf5idt27Xp9D7b2xzpFyI2mS_dTdMEG_f69_Q" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.05.018/attachment/4d3eea98-f649-4f26-ac49-5d355d910139/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (6.24 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S2. Monitoring Sas-6-mCherry Incorporation and Plk4-NG Oscillations Simultaneously in the Same Embryo, Related to Figure&nbsp;2</div><div id="espara0015" role="paragraph">Time-lapse movie of an embryo expressing one copy of Sas-6-mCherry (expressed transgenically from the endogenous <i>Sas-6</i> promotor) and two copies of Plk4-NG (expressed transgenically from the endogenous <i>Plk4</i> promoter) in a <i>Plk4</i> mutant background, observed on a spinning-disk confocal microscope during S-phase of nuclear cycle 12. The movie is a maximum-intensity projection that has been photo-bleach corrected, but not background subtracted for visual clarity. Time (Min:Sec) is shown at the top left, and stage of the cell cycle is indicated at the bottom left.</div></div></div></div><div id="p0080" role="paragraph">Strikingly, the centriolar levels of Plk4 at which centriole growth initiated at each cycle (“Start”; <a id="crosref0250" href="#fig2">Figures 2</a>A and 2B) were not significantly different than the levels at which centriole growth stopped <i>(</i>“Stop”; <a id="crosref0255" href="#fig2">Figures 2</a>A and 2B). This suggests that at each cycle there is a threshold level of centriolar Plk4 that is required to support centriole growth: above this threshold the centrioles can grow, below this threshold they cannot. If the threshold concept is correct, then mother centrioles that failed to recruit sufficient Plk4 should not grow a daughter. We observed that the centrioles in a fraction of the embryos expressing both Plk4-NG and Sas-6-mCherry (mostly at nuclear cycle 13) separated at the start of S-phase but did not detectably incorporate Sas-6-mCherry, indicating that daughter centrioles did not grow (<a id="crosref0260" href="#figs3">Figures S3</a>D and S3E)—a defect that may explain why many of these embryos failed to hatch (<a id="crosref0265" href="#figs3">Figure&nbsp;S3</a>C). Intriguingly, centriolar Plk4 levels continued to oscillate in these embryos, but the average amplitude of these oscillations was lower than in the embryos in which centrioles continued to duplicate—and it was almost always below the average threshold at which centriole growth was normally initiated (<a id="crosref0270" href="#fig2">Figure&nbsp;2</a>C). Together, these results suggest that the Plk4 oscillations initiate, and determine the duration of, centriole growth.</div></section><section id="sec-2-3"><h3>Mathematical Modeling of the Plk4 Oscillation</h3><div id="p0085" role="paragraph">Oscillations in biology are often generated by delayed feedback circuits (<a id="crosref0275" href="#bib76" role="doc-biblioref" data-xml-rid="bib76">Tsai et&nbsp;al., 2008</a>). In <i>Drosophila</i>, Plk4 is recruited to centrioles by Asterless (Asl), which also activates Plk4, allowing it to phosphorylate both itself and Asl at multiple sites (<a id="crosrefs0045" href="#bib11" role="doc-biblioref" data-xml-rid="bib11 bib20 bib36">Boese et&nbsp;al., 2018; Dzhindzhev et&nbsp;al., 2010; Klebba et&nbsp;al., 2015</a>). Human Asl (Cep152) also binds, and is phosphorylated by, Plk4 <i>in&nbsp;vitro</i> (<a id="crosrefs0050" href="#bib14" role="doc-biblioref" data-xml-rid="bib14 bib30">Cizmecioglu et&nbsp;al., 2010; Hatch et&nbsp;al., 2010</a>). We realized that this system could form a time-delayed negative feedback network capable of generating Plk4 oscillations if the activation of Plk4 by Asl eventually led to the inhibition of their interaction.</div><div id="p0090" role="paragraph">A simple version of such a scenario is illustrated in <a id="crosref0280" href="#fig3">Figures 3</a>A and 3B. At the start of each oscillation cycle, we envisage that unphosphorylated Asl receptors on the mother centriole recruit Plk4 to the site of daughter centriole assembly with high affinity (<a id="crosref0285" href="#fig3">Figure&nbsp;3</a>A, (i)). Binding activates Plk4, allowing it to phosphorylate itself (<a id="crosrefs0055" href="#bib18" role="doc-biblioref" data-xml-rid="bib18 bib31 bib35">Cunha-Ferreira et&nbsp;al., 2013; Holland et&nbsp;al., 2010; Klebba et&nbsp;al., 2013</a>), Ana2/STIL (<a id="crosrefs0060" href="#bib21" role="doc-biblioref" data-xml-rid="bib21 bib38 bib48 bib56">Dzhindzhev et&nbsp;al., 2014; Kratz et&nbsp;al., 2015; McLamarrah et&nbsp;al., 2018; Ohta et&nbsp;al., 2014</a>) and Asl/Cep152 (<a id="crosrefs0065" href="#bib11" role="doc-biblioref" data-xml-rid="bib11 bib30">Boese et&nbsp;al., 2018; Hatch et&nbsp;al., 2010</a>) at multiple sites (<a id="crosref0290" href="#fig3">Figure&nbsp;3</a>A, (ii)). The phosphorylated Ana2 promotes cartwheel assembly, potentially explaining why a threshold level of Plk4 is required to promote centriole growth—but in our model this reaction is not important for the Plk4 oscillation per se, so we do not consider it further. We speculate that the phosphorylation of Asl at multiple sites reduces its affinity for Plk4, so that the bound Plk4 molecules are released, leaving behind the phosphorylated Asl-receptor that can no longer recruit Plk4 (<a id="crosref0295" href="#fig3">Figure&nbsp;3</a>A, (iii)) (see the end of this section for how this network can be reset to trigger subsequent rounds of oscillations).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/f554fcf9-b714-4be0-bba2-7c1d16a50035/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2020.05.018/asset/7a39c96a-c872-486e-8b5c-65b9c92e9644/main.assets/gr3.jpg" height="740" width="680" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/f554fcf9-b714-4be0-bba2-7c1d16a50035/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">A Simple Mathematical Model of the Plk4 Oscillation and Experimental Investigations to Test Its Predictions</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0080" role="paragraph">(A) Diagram of the model. (i) During mitosis, Asl receptors (red) on the surface of the mother centriole start to bind Plk4 (green) with high affinity (<i>k</i><sub><i>1</i></sub>). (ii) Once bound, Plk4 is activated, and it starts to phosphorylate itself, Ana2 (black) and Asl (<i>k</i><sub><i>2</i></sub>) at multiple sites (indicated by dotted black arrows and black dots). (iii) We speculate that, after several rounds of phosphorylation, Asl is converted to a state with low affinity for Plk4, so phosphorylated Plk4 is released (<i>k</i><sub><i>3</i></sub>)—and likely degraded. These Asl receptors are now inactivated and can no longer bind Plk4 to promote centriole growth.</div><div id="fspara0085" role="paragraph">(B) Schematic depicts the topology of the mathematical model (see <a id="crosref1420" href="#sec-3">STAR Methods</a> for full details of the model). Asl-p and Plk4-p indicate phosphorylated proteins. Bold arrows indicate the dominant direction of the reactions. This model discretely examines centriolar Plk4-NG levels only during S-phase of each cycle. We speculate that a phosphatase normally removes the phosphate groups from Asl during mitosis to reset the system for the next oscillation (red arrow; <i>k</i><sub><i>4</i></sub>), and we extend the model to include this step elsewhere (<a id="crosref1425" href="#figs4">Figures S4</a>A and S4B).</div><div id="fspara0090" role="paragraph">(C) Graphs show the Lorentzian fit of the Plk4-NG oscillation data during S-phase of cycles 11–13 (solid lines) overlaid with analytical solutions to the model (dotted lines). R<sup>2</sup> values indicate goodness-of-fit.</div><div id="fspara0095" role="paragraph">(D) Bar charts quantify the average cytosolic concentration and centriolar fluorescence of Asl-GFP at the start of S-phase of each cycle. Cytosolic Asl-GFP was measured using FCS; each data point represents the average of 4–6 10-s recordings from a single embryo (<a id="crosref1430" href="#figs5">Figure&nbsp;S5</a>). Centriolar Asl-GFP was measured using confocal microscopy, as described in <a id="crosref1435" href="#sec-3">STAR Methods</a>. N ≥ 14 embryos for each cell cycle; n&nbsp;= 48, 70, and 130 centrioles (mean) per embryo in cycles 11–13, respectively. Data are presented as mean ± SEM. Statistical significance was assessed using an ordinary one-way ANOVA test (ns, not significant).</div><div id="fspara0100" role="paragraph">(E) Bar chart shows the relative abundance of Plk4-NG at the start of each nuclear cycle measured by PeCoS (see <a id="crosref1440" href="#sec-3">STAR Methods</a>; <a id="crosref1445" href="#figs6">Figure&nbsp;S6</a>). Each data point represents a single 180-s recording from a single embryo. Statistical significance was assessed using a Kruskal-Wallis test (<sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001). Data are presented as mean ± SD.</div><div id="fspara0105" role="paragraph">See also <a id="crosrefs0135" href="#figs4">Figures S4, S5, and S6</a> and <a id="crosref1450" href="#mmc2">Data S1</a> (first and fourth charts as well as the Monte Carlo analysis).</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">This network (<a id="crosref0300" href="#fig3">Figure&nbsp;3</a>B; see mathematical model 1 in <a id="crosref0305" href="#sec-3">STAR Methods</a>) maps onto a set of coupled linear ordinary differential equations, which we solved analytically. Solutions to this first model (model 1 in <a id="crosref0310" href="#sec-3">STAR Methods</a>) fit the discrete Plk4 oscillation data from each S-phase of nuclear cycles 11–13 very well (<a id="crosref0315" href="#fig3">Figure&nbsp;3</a>C; R<sup>2</sup> &gt; 0.99). Although the model may overfit the data, these solutions were within a reasonable and generally narrow parameter space (<a id="crosref0320" href="#fig3">Figures 3</a>C and <a id="crosref0325" href="#figs5">S5</a>; <a id="crosref0330" href="#mmc2">Data S1</a>, first and fourth charts). Nevertheless, we believe this model is likely to be oversimplified. Plk4’s ability to phosphorylate itself, for example, could help to generate the oscillation by promoting Plk4 degradation (<a id="crosrefs0070" href="#bib17" role="doc-biblioref" data-xml-rid="bib17 bib29 bib31 bib61">Cunha-Ferreira et&nbsp;al., 2009; Guderian et&nbsp;al., 2010; Holland et&nbsp;al., 2010; Rogers et&nbsp;al., 2009</a>) or lowering the affinity of the Asl::Plk4 interaction—as has recently been demonstrated (<a id="crosref0335" href="#bib57" role="doc-biblioref" data-xml-rid="bib57">Park et&nbsp;al., 2019</a>). Moreover, the model considers the behavior of only Asl and Plk4, when other factors, such as Ana2/STIL, are likely to modulate the systems behavior (<a id="crosrefs0075" href="#bib2" role="doc-biblioref" data-xml-rid="bib2 bib28">Arquint and Nigg, 2016; Gönczy and Hatzopoulos, 2019</a>). Finally, the model does not consider the possibility that Plk4 bound to one receptor could phosphorylate nearby receptors, or the Plk4 bound to nearby receptors, to influence their behavior—a concept that may be important when considering how Plk4 ultimately localizes to only a single site on the side of the mother centriole (<a id="crosrefs0080" href="#bib39" role="doc-biblioref" data-xml-rid="bib39 bib74">Leda et&nbsp;al., 2018; Takao et&nbsp;al., 2019</a>).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/889afad9-9dfc-4f51-9a65-70b77cf45f58/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2020.05.018/asset/184e0100-406c-4eb3-bbd2-75c16576071e/main.assets/figs5.jpg" height="578" width="676" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/889afad9-9dfc-4f51-9a65-70b77cf45f58/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">FCS Analysis of Cytosolic Asl Levels, Related to <a id="crosref1650" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0325" role="paragraph">(A) Schematic workflow describes the acquisition and analysis of point Fluorescence Correlation Spectroscopy (FCS) measurements (see <a id="crosref1655" href="#sec-3">STAR Methods</a> for further details). The 488nm laser beam is positioned at the centriolar plane in embryos expressing 2 copies of Asl-GFP (under the control of its own promoter in an <i>asl</i> mutant background). (i) At the beginning of every cycle, when the old and new mother centrioles have just separated (<i>white</i> arrows), 6x 10 s FCS measurements were taken at a point in the cytosol maximally distant from the centrioles (center of <i>red</i> crosshairs). (ii) This generated 6 autocorrelation functions (ACFs) (a typical example is shown here). (iii) In the FoCuS-point software, 8 different models were fitted to each ACF<i>.</i> (iv) The model that best fitted the majority of the data (#4 in this case) was chosen based on the Bayesian information criterion, and all ACFs were then fitted to this model. (v) The fitted ACFs were corrected for background noise which was determined by measurements in WT embryos. (vi) The ACFs used for further analysis were then restricted by excluding individual outlier measurements based on a ROUT-outlier test (Q&nbsp;= 1%) (these outlier measurements usually had a poor signal-to-noise ratio and gave concentrations that were often biologically unrealistic, and were presumably generated when a centriole or non-specific fluorescent structure passed through the analyzed volume).</div><div id="fspara0330" role="paragraph">(B) Graph shows the average ACFs (represented as Mean ± SEM) for nuclear cycles 11-14 before background corrections. All individual ACFs were used to calculate the cytosolic concentration data shown in <a id="crosref1660" href="#fig3">Figure&nbsp;3</a>D.</div><div id="fspara0335" role="paragraph">(C) Western blot shows the protein levels of the Asl-GFP in either the early or late cell cycles from embryos of the same genotype used in (A) and (B). This supports the results obtained from the FCS measurements, and suggests that total Asl levels do not change significantly during the development of the syncytial embryo. Early and late embryos were separated based on their distinct morphology (judged by eye using a dissection microscope). Actin is shown as a loading control. A representative blot is shown from two technical repeats.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">In order to demonstrate how this network could be reset for the next oscillation, we extended our model (model 2 in <a id="crosref0340" href="#sec-3">STAR Methods</a>) to allow a protein phosphatase (PPTase) to be activated during M-phase to dephosphorylate Asl (<a id="crosref0345" href="#fig3">Figures 3</a>B, red arrow, and <a id="crosref0350" href="#figs4">S4</a>A). This resetting is biologically plausible, because the activities of several PPTases are regulated during the cell cycle (<a id="crosref0355" href="#bib53" role="doc-biblioref" data-xml-rid="bib53">Nilsson, 2019</a>). This model can be solved exactly, and its solutions generate robust centriolar Plk4 oscillations within the context of a system that, like the early <i>Drosophila</i> embryo, alternates between periods of S- and M-phases (<a id="crosref0360" href="#figs4">Figure&nbsp;S4</a>B). Thus, our minimal model illustrates that a classical “time delayed negative-feedback” network (<a id="crosref0365" href="#bib54" role="doc-biblioref" data-xml-rid="bib54">Novák and Tyson, 2008</a>) can generate Plk4 oscillations, although the precise molecular details of this system remain to be fully elucidated.</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/816d1ccb-7803-4a82-aa0b-5f54f4edb9e9/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2020.05.018/asset/af227a4e-d7ae-4506-af2f-01d84d77738d/main.assets/figs4.jpg" height="950" width="688" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/816d1ccb-7803-4a82-aa0b-5f54f4edb9e9/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Theoretical and Experimental Assessment of Several Assumptions Made in the Mathematical Model, Related to <a id="crosref1610" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0290" role="paragraph">(A) Our mathematical model depicted in <a id="crosref1615" href="#fig3">Figures 3</a>A and 3B only discretely examines the Plk4-NG oscillation during S-phase of each nuclear cycle. The schematic here shows our speculation that a phosphatase normally removes the phosphate groups (<i>dotted</i> circles) from Asl (<i>red</i>) during mitosis to reset the system for the next oscillation at rate <i>k</i><sub><i>4</i></sub> (<i>dotted black</i> arrow).</div><div id="fspara0295" role="paragraph">(B) We implemented this step to extend the original model and plotted the mathematical solution for the percentage of Asl-bound Plk4 molecules (<i>black</i> curve) for a total of 14 nuclear cycles. For simplicity we kept the length of S-phase and mitosis constant through all 14 cycles (see <a id="crosref1620" href="#sec-3">STAR Methods</a> for further details of this extended model).</div><div id="fspara0300" role="paragraph">(C) Schematic shows the Serine (S) and Threonine (T) residues (in bold) that were mutated to Alanine in the Asl-13A construct. <i>Dark gray</i> boxes show the relative positions of the previously mapped Plk4-interacting regions within the N-terminal (<a id="crosref1625" href="#bib20" role="doc-biblioref" data-xml-rid="bib20">Dzhindzhev et&nbsp;al., 2010</a>) and C-terminal (<a id="crosref1630" href="#bib36" role="doc-biblioref" data-xml-rid="bib36">Klebba et&nbsp;al., 2015</a>) regions of Asl.</div><div id="fspara0305" role="paragraph">(D) Micrographs show images from time-lapse movies of embryos expressing Asl-WT-mKate2 and Asl-13A-mKate2 (under the control of their own promoters in an <i>asl</i> mutant background), respectively.</div><div id="fspara0310" role="paragraph">(E) Graphs show the regression data (<i>solid</i> lines) for Plk4-NG oscillations in cycle 12 in embryos expressing either Asl-WT (<i>green</i>) or Asl-13A (<i>dark gray</i>) (both without any fluorescent tag) simultaneously with Plk4-NG. N ≥ 25 embryos for each condition; n&nbsp;= 71 and 68 centrioles (mean) per embryo in <i>Asl-WT</i> or <i>Asl-13A</i>, respectively (collection of two trials performed by two independent researchers, blinded for each other’s data). Data are presented as Mean ± SEM <i>R</i><sup><i>2</i></sup> values indicate goodness-of-fit for the regressions. CS&nbsp;= Centrosome separation; NEB&nbsp;= Nuclear envelope breakdown.</div><div id="fspara0315" role="paragraph">(F) In (B) it is assumed that the cytosolic concentration of Plk4 is kept constant over all cycles. The graph here plots an alternative model where the total number of Plk4 molecules in the embryo is kept constant at all cycles. The number of centrioles doubles each cycle, and the mathematical solution for the percentage of Asl-bound Plk4 molecules (<i>black</i> curve), and the percentage of Plk4 molecules that remain in the cytoplasm (<i>red</i> curve), is depicted over 14 nuclear cycles (see <a id="crosref1635" href="#sec-3">STAR Methods</a> for further details and implications of this model). For the first few nuclear cycles, almost all of the Plk4 remains in the cytoplasm since there are only a few centrioles. In the later cycles, however, the amount of Plk4 sequestered by the Asl receptors increases exponentially, as the number of centrioles increase by a factor of 2 in each cycle. Therefore, the rate at which the Asl receptors are able to recruit Plk4 from the cytoplasm decreases, resulting in a reduction in the amplitude of the Plk4 oscillation. This aspect of the model is consistent with our experimental observations that the amplitude of the Plk4 oscillation decreases at later cycles (<a id="crosref1640" href="#fig1">Figure&nbsp;1</a>), as does the cytosolic concentration of Plk4 (<a id="crosref1645" href="#fig3">Figure&nbsp;3</a>E). An alternative, or additional, mechanism that might explain these observations is that the Plk4 molecules activated by binding to Asl may be more likely to autophosphorylate to stimulate their degradation, so ensuring that more Plk4 is degraded at each cycle as the number of centrioles increase. Interestingly, in either of these scenarios, increasing centriole numbers leads to increasing Plk4 depletion from the cytosol, potentially allowing embryos to effectively “count” their centrioles.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-4"><h3>Testing Predictions of the Mathematical Models</h3><div id="p0105" role="paragraph">A key feature of our models is that the phosphorylation of Asl by Plk4 reduces their affinity (although, as discussed above, Plk4’s ability to phosphorylate itself, and other factors, could also help to generate the oscillation). To test the plausibility of this idea, we mutated 13 potential Plk4 phosphorylation sites in Asl to Ala (Asl-13A) (<a id="crosref0370" href="#figs4">Figure&nbsp;S4</a>C). These sites were selected based on their conservation, their similarity to known Plk-family consensus sites (<a id="crosref0375" href="#bib40" role="doc-biblioref" data-xml-rid="bib40">Leung et&nbsp;al., 2007</a>), their proximity to the N- and C-terminal regions of Asl that are thought to interact with Plk4 (<a id="crosref0380" href="#bib11" role="doc-biblioref" data-xml-rid="bib11">Boese et&nbsp;al., 2018</a>), and a previous analysis of sites in the Asl N-terminal region that are either phosphorylated by Plk4 kinase domain <i>in&nbsp;vitro</i> or have been shown to be phosphorylated in cultured <i>Drosophila</i> cells (<a id="crosref0385" href="#bib11" role="doc-biblioref" data-xml-rid="bib11">Boese et&nbsp;al., 2018</a>). If some of these sites are normally phosphorylated by Plk4 to reduce the affinity of the Asl::Plk4 interaction, we would predict that expressing Asl-13A in the presence of endogenous, unlabeled Asl would lead to an increase in centriolar Plk4-NG levels—because the Plk4 should unbind from the mutant Asl receptors less efficiently. Although Asl-13A-mKate2 localized to centrioles less efficiently than Asl-WT-mKate2 (<a id="crosref0390" href="#figs4">Figure&nbsp;S4</a>D), expressing untagged Asl-13A increased the amplitude of the Plk4-NG oscillation (<a id="crosref0395" href="#figs4">Figure&nbsp;S4</a>E), consistent with our idea that phosphorylating Asl can reduce its affinity for Plk4 (<a id="crosref0400" href="#fig3">Figures 3</a>A and 3B).</div><div id="p0110" role="paragraph">An inspection of the parameters generated by our model revealed that the reduction in the amplitude of the Plk4 oscillation at successive nuclear cycles was driven primarily by a reduction in the cytosolic concentration of Plk4 (that determines <i>k</i><sub>1</sub>, the rate at which Plk4 binds to Asl), while total levels of the Asl receptor (<i>A</i><sub><i>tot</i></sub>) remain relatively constant (<a id="crosref0405" href="#mmc2">Data S1</a>, first chart). To test if this was the case, we first used fluorescence correlation spectroscopy (FCS) (<a id="crosref0410" href="#figs5">Figure&nbsp;S5</a>) to examine the cytosolic concentration of Asl-GFP. Although the number of centrioles assembled doubles at each successive cycle, the average cytosolic concentration of Asl-GFP, and the average centriolar levels of Asl-GFP, remained relatively constant at the start of each successive cycle (<a id="crosref0415" href="#fig3">Figure&nbsp;3</a>D), as predicted by our model. Unfortunately, the cytosolic concentration of Plk4-NG was too low to be measured by conventional FCS, so we developed a new method, peak counting spectroscopy (PeCoS), to measure relative protein abundance at lower concentrations (see <a id="crosref0420" href="#sec-3">STAR Methods</a>) (<a id="crosref0425" href="#figs6">Figure&nbsp;S6</a>). This revealed that, in contrast to Asl-GFP, the cytosolic levels of Plk4-NG tended to decrease at successive nuclear cycles (<a id="crosref0430" href="#fig3">Figure&nbsp;3</a>E), as predicted by the model.</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/64260a1e-2f6b-4592-bc32-f2ff5284b3f5/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2020.05.018/asset/5f3d5e83-6ff9-464e-8fd7-b2de48dc532a/main.assets/figs6.jpg" height="621" width="762" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/64260a1e-2f6b-4592-bc32-f2ff5284b3f5/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Peak Counting Spectroscopy Analysis of Cytosolic Plk4 Levels, Related to <a id="crosref1665" href="#fig3">Figures 3</a> and <a id="crosref1670" href="#fig6">6</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0345" role="paragraph">(A) Schematic workflow describes the acquisition and analysis of <i>Peak Counting Spectroscopy</i> (PeCoS) measurements. (i) In addition to embryos expressing Plk4-NG under its own endogenous promoter, embryos of two other genotypes were placed on the same imaging dish. One expressing a green-fluorescent centriole marker to allow correction of the spherical aberration caused by coverslip thickness variation, the other expressing Asl-mKate2 to determine the autofluorescence background threshold for the Plk4-NG expressing embryos—Asl-mKate2 allows one to determine the correct plane (containing the centrioles; <i>white</i> arrows) for background measurement, while the mKate2 fluorophore does not interfere with the PeCoS measurements. (ii) As for FCS (see <a id="crosref1675" href="#figs5">Figure&nbsp;S5</a>), a 488nm laser beam is positioned near the cortex of embryos, and the measurements are taken at a single point in the cytosol (<i>red</i> crosshairs) at the beginning of S-phase, but for 1x 180 s, in both control and Plk4-NG expressing embryos (iii). Afterward, (iv) an appropriate threshold is calculated from the control embryos, so that the background contributes less than 5 peaks on average during each recording. Following background subtraction, (v and vi) the number of peaks is quantified.</div><div id="fspara0350" role="paragraph">(B) To compare the effective linear concentration range of FCS and PeCoS we assessed a two-fold dilution series of the Alexa488 dye. At high dye concentrations, FCS (<i>black</i> symbols) exhibits a near-linear response, while PeCoS (<i>gray</i> symbols) is saturated—presumably because there are too many fluorophores in the effective volume (<i>V</i><sub><i>eff</i></sub>) for them to be measured as individual peaks. At intermediate dye concentrations, both methods exhibit a near linear response. At low concentrations (~&nbsp;&lt; 0.2nM), however, FCS becomes unreliable while PeCoS continues to have a near-linear response.</div><div id="fspara0355" role="paragraph">(C) The bar chart shows the <i>in&nbsp;vivo</i> validation of PeCoS. A significant difference in the number of peaks per minute was observed between embryos expressing either 1x or 2x copies of Plk4-NG (under the control of its endogenous promoter), which were measured at the beginning of S-phase in nuclear cycle 12. Each data point represents a 180&nbsp;s recording from a single embryo. Statistical significance was assessed using Mann-Whitney test (<sup>∗∗∗</sup>p&nbsp;&lt; 0.001). Data are presented as Mean ± SD</div><div id="fspara0360" role="paragraph">(D) Western blot analysis of Plk4-GFP (arrow) levels in early and late embryos supports the conclusion from the PeCoS analysis (<a id="crosref1680" href="#fig3">Figure&nbsp;3</a>E) that cytosolic Plk4 levels are lower in late embryos than in early embryos. Prominent non-specific bands are indicated (<sup>∗</sup>). A representative blot is shown from two technical repeats.</div><div id="fspara0365" role="paragraph">(E) The bar chart compares the cytosolic levels of Plk4-NG (under the control of its endogenous promoter; at the beginning of S-phase in Cycle 13) between WT and <i>Wee1</i><sup><i>−/−</i></sup> embryos (the same genotypes as in <a id="crosref1685" href="#fig6">Figure&nbsp;6</a>). Statistical significance was assessed using an ordinary unpaired t test (ns, not significant). Data are presented as Mean ± SD.</div></div></div></div></figcaption></figure></div><div id="p0115" role="paragraph">Why do cytosolic Plk4 levels decrease at successive nuclear cycles? Our modeling suggests that if total Plk4 levels in the developing embryo remain constant (i.e., the rate of Plk4 degradation and synthesis are balanced), then the doubling of centriole numbers at each cycle can lead to the depletion of cytosolic Plk4—particularly during later nuclear cycles—as an increasing fraction of the protein is sequestered by the increasing number of centrioles (<a id="crosref0435" href="#figs4">Figure&nbsp;S4</a>F). Alternatively (or additionally), Plk4 molecules that are activated by binding to Asl may be more likely to phosphorylate themselves to stimulate their degradation, ensuring that more Plk4 is degraded at each cycle as the number of centrioles increase. Interestingly, in either of these scenarios, increasing centriole numbers lead to more Plk4 depletion from the cytosol, potentially allowing embryos to effectively “count” their centrioles.</div></section><section id="sec-2-5"><h3>The Plk4 Oscillation Can Adapt to Changes in Plk4 Levels to Maintain a Constant Centriole Size</h3><div id="p0120" role="paragraph">Our finding that cytosolic levels of Asl remain constant at successive cycles while cytosolic Plk4 levels decrease suggests a rationale for why centriole biogenesis may be regulated by an oscillatory system. In our models, Asl effectively functions as an integrator (<a id="crosrefs0085" href="#bib23" role="doc-biblioref" data-xml-rid="bib23 bib69">Ferrell, 2016; Somvanshi et&nbsp;al., 2015</a>) whose levels are kept constant so that it can measure changes in the input (cytosolic Plk4 levels) and adapt the oscillation to maintain a constant output (centriole size). If this interpretation is correct, then the Plk4 oscillation should adapt to maintain a constant centriole size when Plk4 levels change, but not when Asl levels change. To test this, we monitored Plk4-NG oscillations in embryos laid by mothers where we genetically halved the dose of either Plk4-NG (hereafter <i>Plk4-NG</i><sup><i>1/2</i></sup> embryos) or <i>asl</i> (hereafter <i>asl</i><sup><i>1/2</i></sup> embryos). Centrioles appeared to duplicate normally in both sets of embryos, but the Plk4 oscillation parameters were altered: in <i>Plk4-NG</i><sup><i>1/2</i></sup> embryos, <i>A</i> decreased but there was a compensatory increase in <i>T</i>, so Ω remained relatively constant (<a id="crosref0440" href="#fig4">Figures 4</a>A and <a id="crosref0445" href="#figs7">S7</a>A); in <i>asl</i><sup><i>1/2</i></sup> embryos, <i>A</i> decreased, but there was no compensatory change in <i>T</i>, so Ω decreased (<a id="crosref0450" href="#fig4">Figures 4</a>B and <a id="crosref0455" href="#figs7">S7</a>B–S7D).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/6a22ce5c-03c1-475e-8702-8004ba1ff64e/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2020.05.018/asset/434e1856-1b8b-47c9-826a-e2457ae19d52/main.assets/gr4.jpg" height="877" width="758" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/6a22ce5c-03c1-475e-8702-8004ba1ff64e/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">The Plk4 Oscillator Can Adapt to Changes in Plk4 Concentration but Not to Changes in Asl Concentration</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0115" role="paragraph">(A and B) Graphs show the regression data (solid lines) and mathematical solutions (dotted lines) for Plk4-NG oscillations in cycle 12 for experiments where either (A) the genetic dose of <i>Plk4-NG</i> was halved (<i>Plk4-NG</i><sup><i>1/2</i></sup>), or (B) the genetic dose of <i>asl</i> was halved (<i>asl</i><sup><i>1/2</i></sup>) (gray lines) compared to controls (green lines). (A) N ≥ 11 embryos for each condition; n&nbsp;= 47 and 42 centrioles (mean) per embryo in control or <i>Plk4-NG</i><sup><i>1/2</i></sup> groups, respectively. (B) N&nbsp;= 18 embryos for each condition; n&nbsp;= 44 and 43 centrioles (mean) per embryo in control or <i>asl</i><sup><i>1/2</i></sup> groups, respectively. Data are presented as mean ± SEM. Bar charts quantify oscillation parameters, as indicated; data are presented as mean ± SD.</div><div id="fspara0120" role="paragraph">(C) Graph quantifies the parameters of cartwheel growth—as measured by Sas-6-GFP fluorescence incorporation (<a id="crosref1455" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>)—in WT and <i>asl</i><sup><i>1/2</i></sup> embryos; data are presented as mean ± SEM. Bar charts quantify growth parameters presented as mean ± SD. N&nbsp;= 17 embryos for each condition; n&nbsp;= 77 and 72 centrioles (mean) per embryo in WT or <i>asl</i><sup><i>1/2</i></sup> groups, respectively. Statistical significance was assessed using an unpaired t test with Welch’s correction (for Gaussian-distributed data) or an unpaired Mann-Whitney test (<sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001; <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001; ns, not significant). R<sup>2</sup> values indicate goodness-of-fit for the mathematical solutions.</div><div id="fspara0125" role="paragraph">See also <a id="crosref1460" href="#figs7">Figure&nbsp;S7</a> and <a id="crosref1465" href="#mmc2">Data S1</a> (second and third charts).</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/282e6a62-556d-45a0-928b-435af41b89ec/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2020.05.018/asset/1763d082-da69-4796-9b6d-23bdedf069c7/main.assets/figs7.jpg" height="714" width="499" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/282e6a62-556d-45a0-928b-435af41b89ec/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Quantification of Centriolar Asl and Cytosolic Plk4 Levels When the Genetic Dose of <i>asl</i> or <i>Plk4</i> Is Halved, Related to <a id="crosref1690" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0380" role="paragraph">(A) Micrograph shows an image of Asl-mCherry at centrioles in an embryo in early S-phase (just after centrosome separation). Bar charts quantify the average centriolar Asl-mCherry levels in early S-phase in either WT embryos (WT) or in embryos where the genetic dose of <i>Plk4</i> has been halved (<i>Plk4</i><sup><i>1/2</i></sup>). N&nbsp;= 17 embryos for each condition; n&nbsp;= 67 and 58 centrioles (mean) per embryo in WT or <i>Plk4</i><sup><i>1/2</i></sup> groups, respectively. Average centriolar Asl levels do not change significantly when the genetic dosage of <i>Plk4</i> is halved, in agreement with the prediction of our model (see Data S1; first, second and third charts).</div><div id="fspara0385" role="paragraph">(B) Same schema as (A), but showing the localization of Asl-GFP, and quantifying the centriolar levels of Asl-GFP in <i>asl</i> mutant embryos expressing either 1 (Asl-GFP<sup>1x</sup>) or 2 (Asl-GFP<sup>2x</sup>) copies of Asl-GFP. N&nbsp;= 10 embryos for each condition; n&nbsp;= 59 and 54 centrioles (mean) per embryo in Asl-GFP<sup>1x</sup> or Asl-GFP<sup>2x</sup> groups, respectively. This analysis reveals that centriolar Asl-GFP levels drop by ~30% when the genetic dosage of <i>Asl-GFP</i> is halved, in good agreement with the prediction of our model (see Data S1; first, second and third charts). Data are represented as Mean ± SEM. Statistical significance was assessed using an unpaired t test with Welch’s correction (for Gaussian-distributed data) or an unpaired Mann-Whitney test (<sup>∗∗</sup>p&nbsp;&lt; 0.01; ns, not significant).</div><div id="fspara0390" role="paragraph">(C) Western blot compares the protein levels of Plk4-GFP (<i>arrow</i>) (expressed under the control of its own promoter in a <i>Plk4</i> mutant background) in otherwise WT embryos or in embryos in which the genetic dosage of <i>asl</i> has been halved. This analysis reveals that Plk4-GFP levels in the embryo do not change dramatically when the genetic dosage of <i>asl</i> is halved, in agreement with the prediction of our model. WT embryos (Lane 1) are shown as a negative control to demonstrate that the Plk4-GFP band is only detected in embryos expressing Plk4-GFP. Prominent non-specific bands are indicated (<sup>∗</sup>). Actin is shown as a loading control. A representative blot is shown from two technical repeats.</div><div id="fspara0395" role="paragraph">(D) The bar chart compares the number of Plk4-NG peaks per minute that was observed between normal embryos (<i>WT</i>) or embryos where the genetic dose of Asl was halved (<i>asl</i><sup><i>1/2</i></sup>). Measurements were performed at the beginning of S-phase in nuclear cycle 12. Each data point represents a 180&nbsp;s recording from a single embryo. Statistical significance was assessed using an ordinary unpaired t test (for Gaussian-distributed data) or a Mann-Whitney test (ns, p &gt; 0.05). Data are presented as Mean ± SD.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">Our mathematical model (model 1) could fit both sets of data well (<a id="crosref0460" href="#fig4">Figures 4</a>A and 4B; R<sup>2</sup> &gt; 0.99), generating a reasonable range of parameters (<a id="crosref0465" href="#mmc2">Data S1</a>, second and third charts), several of which we again validated experimentally (<a id="crosref0470" href="#figs7">Figure&nbsp;S7</a>; see mathematical modeling section in <a id="crosref0475" href="#sec-3">STAR Methods</a>). Interestingly, if we took the normal parameters derived from our model and simply adjusted the amount of Asl or Plk4 in the model to the levels we experimentally measured in the half-dose embryos, the model fit the data less well (not shown). This suggests that changing the concentration of one component is likely to influence the concentration and/or behavior of other components so that several parameters of the Plk4 oscillation are altered. This seems plausible, as the core centriole duplication proteins are known to interact with and influence each other in multiple ways (<a id="crosrefs0090" href="#bib2" role="doc-biblioref" data-xml-rid="bib2 bib28 bib51">Arquint and Nigg, 2016; Gönczy and Hatzopoulos, 2019; Nigg and Holland, 2018</a>).</div><div id="p0130" role="paragraph">Consistent with our observation that the Plk4-NG oscillations adapt in <i>Plk4-NG</i><sup><i>1/2</i></sup> embryos by reducing <i>A</i> and increasing <i>T</i> to maintain a relatively constant Ω, we previously showed that halving the genetic dose of Plk4 led to the centrioles growing slowly, but for a longer period of time, to maintain a constant size (<a id="crosref0480" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>). In contrast, we would predict that daughter centrioles in <i>asl</i><sup><i>1/2</i></sup> embryos should grow more slowly (as <i>A</i> is decreased), but for a normal period (as <i>T</i> is unchanged), and so centrioles would be too short (as Ω decreases). We measured the parameters of daughter centriole growth in <i>asl</i><sup><i>1/2</i></sup> embryos and confirmed that this was the case (<a id="crosref0485" href="#fig4">Figure&nbsp;4</a>C). Together, these experiments suggest that the Plk4 oscillatory network functions to maintain a constant centriole size even when Plk4 levels vary.</div></section><section id="sec-2-6"><h3>Plk4 Oscillations Can Execute Centriole Duplication Independently of a Robust Cdk/Cyclin Cell-Cycle Oscillator</h3><div id="p0135" role="paragraph">Although the Plk4 oscillations in fly embryos are normally entrained by the cell-cycle oscillator (CCO) (<a id="crosref0490" href="#fig1">Figures 1</a>E and 1F), it has long been known that centrioles can continue to duplicate in many systems even when several other aspects of cell-cycle progression are blocked (<a id="crosrefs0095" href="#bib4" role="doc-biblioref" data-xml-rid="bib4 bib26 bib68">Balczon et&nbsp;al., 1995; Gard et&nbsp;al., 1990; Sluder et&nbsp;al., 1990</a>). We wondered whether this might be because Plk4 oscillations can continue to drive centriole biogenesis even in the absence of a robust CCO. To test this possibility, we injected embryos with double-stranded RNAs (dsRNAs) targeting the three embryonic mitotic cyclins: A, B, and B3. These embryos arrest in an interphase-like state with intact nuclei that do not duplicate their DNA, but where centrosomes can continue to duplicate (<a id="crosref0495" href="#bib47" role="doc-biblioref" data-xml-rid="bib47">McCleland and O’Farrell, 2008</a>). We initially injected embryos in nuclear cycles 7–8 and monitored Plk4-NG behavior ∼30&nbsp;min later. In all such embryos, we observed an initial synchronous round of centriole duplication without NEB (indicating that the CCO was perturbed), followed by one or more rounds of less synchronous centriole duplication (<a id="crosref0500" href="#fig5">Figures 5</a>A and <a id="crosref0505" href="#figs2">S2</a>B; <a id="crosref0510" href="#mmc5">Video S3</a>). Strikingly, a normal Plk4-NG oscillation was associated with the first, synchronous, round of centriole duplication, but subsequent oscillations were more variable (<a id="crosref0515" href="#fig5">Figures 5</a>A and <a id="crosref0520" href="#figs2">S2</a>B).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/46fbae2b-5e1e-4bfd-bfbb-d5be637d4e19/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2020.05.018/asset/d0be32aa-2b44-4cbc-921f-3ff8acd6062d/main.assets/gr5.jpg" height="535" width="750" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/46fbae2b-5e1e-4bfd-bfbb-d5be637d4e19/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Plk4 Levels Can Continue to Oscillate and Promote Centriole Duplication Even When the CCO is Perturbed</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0135" role="paragraph">(A) Graph shows the Plk4-NG oscillations in an embryo injected with dsRNA against cyclin A-B-B3; the schema above the graph illustrates the experimental protocol. The nuclei in this embryo arrest in interphase, but centrioles go through an additional round of division—centriole separation (CS)—accompanied by a Plk4 oscillation. See <a id="crosref1470" href="#figs2">Figure&nbsp;S2</a>B for additional examples; n&nbsp;= 30 centrioles (mean) per embryo.</div><div id="fspara0140" role="paragraph">(B) Graphs show the raw (black lines) and filtered (red lines) fluorescence intensity data of 3 individual “fertile” centrioles and 3 individual “sterile” centrioles within the same cyclin-depleted embryo. The fertile centrioles duplicate (black dotted lines), and these events were often closely associated with computed Plk4 oscillation peaks (red dotted lines) (see <a id="crosref1475" href="#sec-3">STAR Methods</a> for further details of the peak calling methodology).</div><div id="fspara0145" role="paragraph">(C) An unbiased computational analysis of all 45 fertile centrioles in 3 embryos reveals that &gt;80% of the computationally detected Plk4 oscillation peaks occur within 5&nbsp;min of an experimentally observed duplication event. A simulation with randomly distributed centriole duplication events and Plk4 oscillation peaks showed a mean time separation of 10.5&nbsp;min (data not shown).</div><div id="fspara0150" role="paragraph">(D) Venn diagram shows how, using a 5-min window, the oscillation peaks can be used to predict duplication events with both high precision and high recall (40/49 Plk4 oscillation peaks are associated with a duplication event, and 40/52 duplication events are associated with a Plk4 oscillation peak).</div><div id="fspara0155" role="paragraph">(E) Graph shows the ability of Plk4 oscillation peaks to “retrieve” centriole duplication events across all peak prominences. All detected oscillation peaks were ranked in order of their peak prominence from high to low (black dots) and assigned uniquely to a duplication event if within a 5-min time window. The graph then plots the precision and recall values if the threshold for calling a peak were set as the peak prominence value of each peak (in descending order). Below the detected peak that is associated with a peak prominence threshold of 0.12, the precision dramatically drops, suggesting the existence of a minimum peak amplitude for centriole duplication. At this threshold, precision and recall are jointly optimized. Note, if there were no overall correlation between Plk4 peaks and a duplication event, the integrated area under the curve across all peak prominences or average precision (AP) for the 5-min time window (AP<sub>5min</sub>) would be ~50% (given by # duplications/(# duplications&nbsp;+ # peaks)); so the score of ~75% indicates a meaningful correlation.</div><div id="fspara0160" role="paragraph">(F) Graph shows the correlation between the time of the computationally determined Plk4 peaks and their respective experimentally observed duplication events. Correlation strength was examined using Pearson’s correlation coefficient (r&nbsp;&lt; 0.40 weak; 0.40&nbsp;&lt; r&nbsp;&lt; 0.60 moderate; r &gt; 0.60 strong); significance of correlation was determined by the p value (p&nbsp;&lt; 0.05) (see <a id="crosref1480" href="#sec-3">STAR Methods</a> for a full description of this analysis).</div><div id="fspara0165" role="paragraph">See also <a id="crosref1485" href="#figs2">Figures S2</a> and <a id="crosref1490" href="#figs8">S8</a>.</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph"><div id="mmc5" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc5" data-type="mp4"><span id="video_stream_mmc5_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5YmY2NjI3NWI4NzYyMjYyZGRiMDhiMTBhNzc0ZjM0OSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.SiQqSoHtLH-bpbw56UjRXk0W09iwNzv3CNIbGVDtzM1CMVGyU4gjSfMdV3BuLd8lDWEQioin2TVy3uPiv2FgsxsGaCVhqtSEbIGprqbWXG48hf_2b6AjKQHsrPGcHHcz2m8NZGcxyX3FYQvD8Qazv0yh_omv5HdKiAw2ZFbyhDlcknsDxp_xEOZ6_2XF_iUzYKPY9juDqQRFn3Wa09LOwg6znkdq1Y1JnygjtYMlq7JRloXG9vo9FPfIeHwkInvWs2ycNE4GNLDxTFEAi4_YGytcJ5KFuFkcuoM0sUc0xEz8GKRW6GHHcpswtBHO5ptlQiOdZ93ye_Zz7oYD4J498A</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5YmY2NjI3NWI4NzYyMjYyZGRiMDhiMTBhNzc0ZjM0OSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.SiQqSoHtLH-bpbw56UjRXk0W09iwNzv3CNIbGVDtzM1CMVGyU4gjSfMdV3BuLd8lDWEQioin2TVy3uPiv2FgsxsGaCVhqtSEbIGprqbWXG48hf_2b6AjKQHsrPGcHHcz2m8NZGcxyX3FYQvD8Qazv0yh_omv5HdKiAw2ZFbyhDlcknsDxp_xEOZ6_2XF_iUzYKPY9juDqQRFn3Wa09LOwg6znkdq1Y1JnygjtYMlq7JRloXG9vo9FPfIeHwkInvWs2ycNE4GNLDxTFEAi4_YGytcJ5KFuFkcuoM0sUc0xEz8GKRW6GHHcpswtBHO5ptlQiOdZ93ye_Zz7oYD4J498A" class="cloudflare-stream-player ready tracked" id="video_stream_mmc5" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 53.412%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5YmY2NjI3NWI4NzYyMjYyZGRiMDhiMTBhNzc0ZjM0OSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.SiQqSoHtLH-bpbw56UjRXk0W09iwNzv3CNIbGVDtzM1CMVGyU4gjSfMdV3BuLd8lDWEQioin2TVy3uPiv2FgsxsGaCVhqtSEbIGprqbWXG48hf_2b6AjKQHsrPGcHHcz2m8NZGcxyX3FYQvD8Qazv0yh_omv5HdKiAw2ZFbyhDlcknsDxp_xEOZ6_2XF_iUzYKPY9juDqQRFn3Wa09LOwg6znkdq1Y1JnygjtYMlq7JRloXG9vo9FPfIeHwkInvWs2ycNE4GNLDxTFEAi4_YGytcJ5KFuFkcuoM0sUc0xEz8GKRW6GHHcpswtBHO5ptlQiOdZ93ye_Zz7oYD4J498A" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.05.018/attachment/151856d5-0900-41de-8343-115faa97419c/mmc5.mp4" download="mmc5.mp4" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Video (39.00 MB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Video S3. Centrioles Can Continue to Duplicate in Embryos Arrested for a Short Period in Interphase by Mitotic Cyclin Depletion, Related to Figure&nbsp;5</div><div id="espara0020" role="paragraph">Time-lapse movie of an embryo expressing two copies of Plk4-NG (expressed transgenically from the endogenous <i>Plk4</i> promoter) in a <i>Plk4</i> mutant background, observed on a spinning-disk confocal microscope through nuclear cycle 12. The embryo was injected with cyclin A-B-B3 dsRNA in ∼cycle 8, approximately 30-40&nbsp;min prior to the start of the movie. The movie on the left is the maximum intensity projection of the slices where centrioles are in focus. The movie on the right is the maximum intensity projection of the slices where nuclei are in focus. Note how the centrioles undergo at least two rounds of duplication, the second of which is more asynchronous and occurs without nuclear envelope breakdown (indicating that the nuclei are arrested in an interphase-like state). These videos have been photo-bleach corrected, but not background subtracted for visual clarity. Time (Min:Sec) is shown at the top left.</div></div></div></div><div id="p0145" role="paragraph">We reasoned that any residual Plk4-NG oscillations in these embryos might be triggered by residual CCO oscillations that could trigger centriole duplication, but not DNA synthesis or NEB. While one can never rule out the possibility of residual CCO activity, we tried to overcome this potential problem by examining centriole behavior in embryos in which the CCO was likely to be more fully suppressed by injecting the embryos earlier (nuclear cycles 2–4) and monitoring them later (after ∼90&nbsp;min). The centrioles in these embryos were now completely dissociated from the non-dividing nuclei and they appeared to divide stochastically, with some centrioles duplicating one or more times, and others not duplicating at all (<a id="crosref0525" href="#figs8">Figure&nbsp;S8</a>; <a id="crosref0530" href="#mmc6">Video S4</a>). The CCO coordinates cell-cycle events in normal early embryos by spreading as a chemical trigger wave (<a id="crosrefs0100" href="#bib12" role="doc-biblioref" data-xml-rid="bib12 bib19">Chang and Ferrell, 2013; Deneke et&nbsp;al., 2016</a>), but duplicating centrioles did not detectably trigger the duplication of nearby centrioles (<a id="crosref0535" href="#figs8">Figure&nbsp;S8</a>F). Thus, the “decision” to duplicate in these CCO-suppressed embryos appears to be largely intrinsic to each individual centriole.</div><div class="figure-wrap"><figure id="figs8" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/b2159a80-f29d-4c21-a7a6-dd3cda95e38b/main.assets/figs8_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs8" src="/cms/10.1016/j.cell.2020.05.018/asset/edc42c4f-3ae3-469d-8801-2fc797919875/main.assets/figs8.jpg" height="864" width="762" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/b2159a80-f29d-4c21-a7a6-dd3cda95e38b/main.assets/figs8_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs8-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S8</span> <span class="figure__title__text">The Average Centriolar Plk4-NG Level on Individual Centrioles Can Be Used to Predict Stochastic Centriole Duplications in Embryos Arrested in Interphase by Mitotic Cyclin Depletion, Related to <a id="crosref1695" href="#fig5">Figures 5</a> and <a id="crosref1700" href="#figs2">S2</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs8-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0410" role="paragraph">(A) The pie chart quantifies the percentage of centrioles that continued to duplicate in embryos where cyclin A-B-B3 dsRNA was injected into embryos at nuclear cycle 2-4, and centriole behavior assessed ~90&nbsp;min later. <i>Ambiguous</i> (gray) indicates the fraction of centrioles whose duplication state could not be unambiguously determined due to their drifting out of focus during imaging.</div><div id="fspara0415" role="paragraph">(B) Bar chart shows the mean signal-to-noise ratio (SNR) of Plk4-NG fluorescence signals from sterile and fertile centrioles (<i>red</i> and <i>green</i>, respectively) through the entire period of observation. Data are presented as Mean ± SD. Statistical significance of SNR was tested using a t test assuming equal variance (<sup>∗∗∗</sup>p&nbsp;&lt; 0.001).</div><div id="fspara0420" role="paragraph">(C) Heatmap histogram of all SNR values from sterile and fertile centrioles. <i>Red dashed</i> line shows the unbiased threshold, determined automatically from Otsu thresholding for distinguishing sterile and fertile centrioles. Heatmap (<i>Red</i>: Sterile and <i>Green</i>: Fertile) indicates the fraction of fertile/sterile centrioles in each column. Note that, the higher the SNR, the more fertile the centrioles are.</div><div id="fspara0425" role="paragraph">(D) Confusion matrix shows the classification performance of sterile versus fertile centriole Plk4-NG signals using the Otsu threshold in (C) as a proportion of the total number of signals, n&nbsp;= 81 centrioles from 3 embryos.</div><div id="fspara0430" role="paragraph">(E) Heatmap plots demonstrate the spatial (x,y) coordinates of all centriole duplication events in a representative cycling embryo (left; at the beginning of cycle 13 when centrioles are separating over the course of ~3.5&nbsp;min) and non-cycling embryo (right; captured over ~60&nbsp;min), as each duplication event colored <i>light blue</i> to <i>dark red</i> to represent early and late time points, respectively. The black points plot the observed spatial (x,y) positions of all centrioles, duplicating and non-duplicating, at all time points. Note that the duplications in the cycling embryo are spatially and temporally coordinated (tending to divide first at the top of the embryo and later at the bottom of the embryo), while the duplications in the non-cycling embryo occur over a longer time-scale and do not appear to be coordinated in space or time.</div><div id="fspara0435" role="paragraph">(F) To test more rigorously whether the centriole duplication events in non-cycling embryos are largely stochastic, we calculated Ripley <i>K</i> statistics for all the non-cycling embryos used in (A–D). This statistic provides a measure of whether the temporal duplication events have spatial preference by measuring the average number of events that occur as a function of distance from individual centrioles. Curves were computed from the (x,y) coordinates of only the duplicating centrioles (denoted <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>K</mi><mrow><mi>d</mi><mi>i</mi><mi>v</mi><mi>p</mi><mi>o</mi><mi>i</mi><mi>n</mi><mi>t</mi></mrow></msub></mrow></math>, <i>red</i> line) and of all centrioles at all times (denoted <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>K</mi><mrow><mi>a</mi><mi>l</mi><mi>l</mi><mi>p</mi><mi>o</mi><mi>i</mi><mi>n</mi><mi>t</mi><mi>s</mi></mrow></msub></mrow></math>, <i>black</i> line). The sigmoidal increase in the statistic as a function of distance in both cases suggests that duplication events do not cluster spatially at short distances (&lt; 50-100 pixels). The trend and amplitudes of <i>red</i> and <i>black</i> lines (mean ± SD) are very similar and fall in each other’s statistical confidence range, indicating that duplicating centrioles do not exhibit additional spatial clustering above the natural spatial distribution of centrioles.</div><div id="fspara0440" role="paragraph">(G) Schematic depicts the topology of the mathematical model that illustrates how Plk4 oscillations at individual centrioles could be generated to trigger stochastic duplications in non-cycling embryos. Briefly, we no longer assume that a PPTase acts in discrete bursts during mitosis and instead assume a continuous low-level PPTase activity (10% of the activity in cycling embryos). We allow individual Asl receptors to bind Plk4 and be phosphorylated until they release Plk4 (as in our original model), and to be continuously be slowly dephosphorylated by the PPTase. Asl-p and Plk4-p indicate phosphorylated proteins. Bold arrows indicate the dominant direction of the reactions.</div><div id="fspara0445" role="paragraph">(H) Graph shows how the percentage of Asl receptors that are bound to Plk4 changes over time at 10 individual centrioles, two of which have been colored <i>red</i> or <i>green</i>. The centrioles are initially synchronized, as their Asl receptors all start in a dephosphorylated, unbound, state and so exhibit a coordinated pulse of Plk4 binding. As time progresses, however, the Asl receptors lose synchrony (as their dephosphorylation is no longer entrained by the CCO), and so each centriole exhibits low amplitude stochastic oscillations. These oscillations may be sufficient to trigger centriole duplication under these conditions of interphase arrest with low CCO activity, as evident from our experimental observations (<a id="crosref1705" href="#fig5">Figures 5</a>B–5F and <a id="crosref1710" href="#mmc6">Video S4</a>; see <a id="crosref1715" href="#sec-3">STAR Methods</a> for full details of the model).</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph"><div id="mmc6" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc6" data-type="mp4"><span id="video_stream_mmc6_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlOTRkNzM5YmEwZjA4MmI3MDZmODUwYTQ0NjFkYTFkNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.DASj12sPFzBhqQXF4TrzamJe98ULGxyMKJZaRl89ZNkfwnZhA4_d3fXeWf2syvbdR_9h9JrtgvprioxGWIV4MruVxzxWEBuKFjrFCWe-fUa-K_WzPT71V77h6u4IYJhrH7KQeXIzLagLC-W25Gn9A3fOXBvEcWFGshGUu0U-iTI1iAozqjeL3hFiWJelmbwA4n1SEJ_IcBkj6SJtlx1oxfncEJJb5tmRXOEH3uR-j_ZNc09LOcXv9aKPMCucKgO8YUfMl0qUZEl2IH5m1xDSzOyV1dhD4ROjS50e7dejNnWUoMYmQSYkwAzvitYt_eZ03wM2HGN09vYa01htsHoaMQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlOTRkNzM5YmEwZjA4MmI3MDZmODUwYTQ0NjFkYTFkNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.DASj12sPFzBhqQXF4TrzamJe98ULGxyMKJZaRl89ZNkfwnZhA4_d3fXeWf2syvbdR_9h9JrtgvprioxGWIV4MruVxzxWEBuKFjrFCWe-fUa-K_WzPT71V77h6u4IYJhrH7KQeXIzLagLC-W25Gn9A3fOXBvEcWFGshGUu0U-iTI1iAozqjeL3hFiWJelmbwA4n1SEJ_IcBkj6SJtlx1oxfncEJJb5tmRXOEH3uR-j_ZNc09LOcXv9aKPMCucKgO8YUfMl0qUZEl2IH5m1xDSzOyV1dhD4ROjS50e7dejNnWUoMYmQSYkwAzvitYt_eZ03wM2HGN09vYa01htsHoaMQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc6" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 57.3237%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlOTRkNzM5YmEwZjA4MmI3MDZmODUwYTQ0NjFkYTFkNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA4Mzk2fQ.DASj12sPFzBhqQXF4TrzamJe98ULGxyMKJZaRl89ZNkfwnZhA4_d3fXeWf2syvbdR_9h9JrtgvprioxGWIV4MruVxzxWEBuKFjrFCWe-fUa-K_WzPT71V77h6u4IYJhrH7KQeXIzLagLC-W25Gn9A3fOXBvEcWFGshGUu0U-iTI1iAozqjeL3hFiWJelmbwA4n1SEJ_IcBkj6SJtlx1oxfncEJJb5tmRXOEH3uR-j_ZNc09LOcXv9aKPMCucKgO8YUfMl0qUZEl2IH5m1xDSzOyV1dhD4ROjS50e7dejNnWUoMYmQSYkwAzvitYt_eZ03wM2HGN09vYa01htsHoaMQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.05.018/attachment/92700f4f-9a64-4111-b668-4ae95937867e/mmc6.mp4" download="mmc6.mp4" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Video (30.08 MB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Video S4. Centrioles Duplicate Stochastically in Embryos Arrested for a Long Period in Interphase by Mitotic Cyclin Depletion, Related to Figure&nbsp;5</div><div id="espara0025" role="paragraph">Time-lapse movie of an embryo expressing two copies of Plk4-NG (expressed transgenically from the endogenous <i>Plk4</i> promoter) in a <i>Plk4</i> mutant background, observed on a spinning-disk confocal microscope. The embryo was injected with cyclin A-B-B3 dsRNA in ∼cycle 2-4, approximately 90&nbsp;min prior to the start of the movie. The movie on the left is the maximum intensity projection of the slices where centrioles are in focus. The movie on the right is the maximum intensity projection of the slices where nuclei are in focus. Note that a small number of large nuclei are present throughout the time-course of the movie (indicating that they are arrested in an interphase-like state), but some centrioles duplicate one or more times in an apparently stochastic manner. The Plk4-NG oscillations on individual centrioles are less obvious than in normally cycling embryos, but an unbiased computational analysis of these movies indicates that individual centriole duplication events are correlated with individual centriolar Plk4-NG oscillations (Figures 5B–5F and S8A–S8D). These videos have been photo-bleach corrected, but not background subtracted for visual clarity. Time (Min:Sec) is shown at the top left.</div></div></div></div><div id="p0155" role="paragraph">To test whether these stochastic centriole duplications were triggered by Plk4 oscillations, we measured Plk4-NG fluorescence levels at individual centrioles. The raw intensity data were noisy, but duplicating “fertile” centrioles appeared to exhibit more prominent Plk4-NG oscillations than non-duplicating “sterile” centrioles (<a id="crosref0540" href="#fig5">Figure&nbsp;5</a>B). Moreover, the average centriolar Plk4-NG fluorescence level (expressed as signal-to-noise ratio [SNR]) was significantly higher at fertile centrioles (<a id="crosref0545" href="#figs8">Figure&nbsp;S8</a>B), and Plk4-NG SNR values could distinguish fertile and sterile centrioles, correctly predicting centriole fertility or sterility ∼74% and ∼71% of the time, respectively (<a id="crosref0550" href="#figs8">Figures S8</a>C and S8D).</div><div id="p0160" role="paragraph">Upon filtering the raw oscillation data, we found that the peaks of the Plk4-NG oscillations (see <a id="crosref0555" href="#sec-3">STAR Methods</a> for a description of peak-calling methodology) were often associated with centriole duplication events (<a id="crosref0560" href="#fig5">Figure&nbsp;5</a>B). An unbiased computational analysis of all the 45 fertile centrioles that we observed in 3 different embryos revealed that the predicted Plk4-NG oscillation peaks predicted centriole duplication events with high precision (40/49 Plk4-NG peaks were associated with a duplication event that occurred within ±5&nbsp;min of the peak) and recall (40/52 duplication events occurred within ±5&nbsp;min of a Plk4-NG oscillation peak) (<a id="crosref0565" href="#fig5">Figures 5</a>C and 5D). Computer simulations revealed that a random distribution of the duplication events lead to an average time of &gt;10&nbsp;min between the peaks and duplication events, indicating that the observed association was not random. Moreover, a rank ordering of the Plk4-NG oscillations based on amplitude revealed that the higher the amplitude of the oscillation, the more likely it was to be associated with a centriole duplication event (<a id="crosref0570" href="#fig5">Figure&nbsp;5</a>E), while plotting the relative timing of the Plk4-NG oscillations and the centriole duplication events revealed a strong positive correlation (<a id="crosref0575" href="#fig5">Figure&nbsp;5</a>F; Pearson <i>r</i>&nbsp;= 0.9580, p&nbsp;&lt; 0.0001). We conclude that individual centrioles can organize autonomous Plk4 oscillations that can drive centriole duplication even in the absence of a robust CCO. This potentially explains how centrioles can continue to duplicate independently of many other cell-cycle events.</div></section><section id="sec-2-7"><h3>The CCO Can Phase-Lock the Plk4 Oscillation to Coordinate Centriole Duplication with Other Cell-Cycle Events</h3><div id="p0165" role="paragraph">It is widely believed that the CCO acts primarily as a “ratchet” whose activity increases over the cell cycle to trigger the sequential execution of cell-cycle events such as DNA replication, centriole duplication, nuclear envelope breakdown (NEB), and spindle assembly (<a id="crosrefs0105" href="#bib70" role="doc-biblioref" data-xml-rid="bib70 bib72 bib73">Stern and Nurse, 1996; Swaffer et&nbsp;al., 2016, 2018</a>). An interesting alternative possibility is that the CCO could act as a “phase-locker” whose function is simply to entrain the phase of a network of autonomous oscillations, each of which is responsible for the execution of a specific cell-cycle event (<a id="crosref0580" href="#bib42" role="doc-biblioref" data-xml-rid="bib42">Lu and Cross, 2010</a>). The Plk4 oscillation appears to time and execute centriole biogenesis, and it can trigger centriole duplication independently of a robust CCO, so it is an excellent candidate for such an autonomous oscillation.</div><div id="p0170" role="paragraph">To better understand how the CCO might entrain the Plk4 oscillation, we measured the average period of the stochastic Plk4 oscillations in cyclin-depleted embryos (20.5 ± 4.6&nbsp;min) and compared this to the average period of the Plk4 oscillations in cycles 11–12 (11.7 ± 0.7&nbsp;min) and 12–13 (14.9 ± 1.7&nbsp;min). The natural period of the autonomous Plk4 oscillation in these early embryos is therefore similar to, but slightly slower than, the period of the Plk4 oscillations normally enforced by the CCO, indicating that the CCO could entrain the Plk4 oscillation by speeding up a phase of its natural cycle.</div><div id="p0175" role="paragraph">To examine which phase this might be, we tested for correlations between various parameters of the Plk4 oscillation and the length of S- or M-phase. During cycles 11–13, we observed a significant correlation between the timing of the Plk4-NG oscillation trough in M-phase and the duration of M-phase (<a id="crosref0585" href="#fig6">Figure&nbsp;6</a>, lower scatterplots in the light yellow panel), suggesting that the CCO entrains the Plk4 oscillation by speeding it up during M-phase. This is consistent with our minimal model, in which the CCO entrains the Plk4 oscillation by ensuring the rapid and coordinated dephosphorylation of Asl during M-phase (<a id="crosref0590" href="#figs4">Figures S4</a>A and S4B).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/59d38c6b-1456-44ca-aa92-c6eeff86fa56/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2020.05.018/asset/35f6e93e-b41b-4699-8176-b392e8a1f825/main.assets/gr6.jpg" height="566" width="762" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/59d38c6b-1456-44ca-aa92-c6eeff86fa56/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">The CCO Phase-Locks the Plk4 Oscillations in Mitosis of Cycles 11–13 Independently of Wee1 and in Interphase of Cycle 13 in a Wee1-Dependent Manner</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">Scatterplots illustrate correlations between various parameters of the Plk4 oscillation and the length of S- or M-phase in nuclear cycles 11–13 in WT and <i>Wee1</i><sup>−/−</sup> embryos (see Plk4-NG smooth curve fitting and parameter extraction in <a id="crosref1495" href="#sec-3">STAR Methods</a> for details of how these parameters [along with their descriptions] were obtained in an unbiased way). During cycles 11–13, there is a significant correlation between the timing of the Plk4 oscillation trough in M-phase and the duration of M-phase (lower scatterplots in the light yellow panel), suggesting that the CCO entrains the Plk4 oscillation during M-phase. This entrainment is not altered in the <i>Wee1</i><sup>−/−</sup> embryos. During nuclear cycle 13, there is an additional correlation between the peak of the Plk4 oscillation and S-phase length that is lost in the <i>Wee1</i><sup>−/−</sup> embryos (upper rightmost scatterplot in the light yellow panel). The plots for the WT group were generated with the data obtained from <a id="crosref1500" href="#fig1">Figures 1</a>A and <a id="crosref1505" href="#figs2">S2</a>A, as well as 5 additional embryos of the same genotype. N&nbsp;= 10 embryos; n&nbsp;= 23 centrioles (mean; starting from cycle 11) per embryo in <i>Wee1</i><sup>−/−</sup> group. Correlation strength was examined using Pearson’s correlation coefficient (r&nbsp;&lt; 0.40 weak; 0.40&nbsp;&lt; r&nbsp;&lt; 0.60 moderate; r &gt; 0.60 strong); significance of correlation was determined by the p value (p&nbsp;&lt; 0.05).</div><div id="fspara0180" role="paragraph">See also <a id="crosref1510" href="#figs4">Figures S4</a> and <a id="crosref1515" href="#figs6">S6</a>E.</div></div></div></div></figcaption></figure></div><div id="p0180" role="paragraph">We also noticed an additional correlation between the peak of the Plk4-NG oscillation and S-phase length in cycle 13 (<a id="crosref0595" href="#fig6">Figure&nbsp;6</a>, upper rightmost scatterplot in the light yellow panel). This is not surprising, as a Wee1-dependent checkpoint dramatically slows the CCO—and many other aspects of S-phase progression—particularly during nuclear cycle 13 (<a id="crosrefs0110" href="#bib19" role="doc-biblioref" data-xml-rid="bib19 bib71">Deneke et&nbsp;al., 2016; Stumpff et&nbsp;al., 2004</a>). Moreover, in <i>Wee1</i><sup>−/−</sup> embryos, the correlation between the Plk4-NG oscillation trough and M-phase length was maintained (<a id="crosref0600" href="#fig6">Figure&nbsp;6</a>, lower rightmost scatterplot in the light yellow panel), while the correlation between the Plk4-NG oscillation peak and S-phase length was lost (<a id="crosref0605" href="#fig6">Figure&nbsp;6</a>; upper rightmost scatterplot in the light yellow panel), demonstrating that Wee1 can influence the Plk4 oscillation in S-phase. Interestingly, the cytosolic levels of Plk4-NG were essentially the same in wild-type (WT) and <i>Wee1</i><sup>−/−</sup> embryos (<a id="crosref0610" href="#figs6">Figure&nbsp;S6</a>E), indicating that cell-cycle regulators can influence the Plk4 oscillation without changing Plk4’s cytosolic concentration. This supports the model prediction that the drop in cytosolic Plk4 levels at successive nuclear cycles (<a id="crosref0615" href="#fig3">Figure&nbsp;3</a>E) is not, on its own, sufficient to account for the change in Plk4 oscillation parameters we observe from cycles 11–13. This presumably explains why the model requires several parameters to change slightly at each successive cycle to best fit the data (<a id="crosref0620" href="#mmc2">Data S1</a>, first chart).</div><div id="p0185" role="paragraph">Taken together, our observations are consistent with the phase-locker model of cell-cycle regulation (<a id="crosref0625" href="#bib42" role="doc-biblioref" data-xml-rid="bib42">Lu and Cross, 2010</a>). We propose that the Plk4 oscillation may be an exemplar of an autonomously oscillating system that can independently drive a cellular event (centriole duplication), but that is normally phase-locked by the CCO to ensure its proper coordination with other biological events and with cell division.</div></section><section id="sec-2-8"><h3>A Model to Generate Autonomous Plk4 Oscillations in the Absence of a CCO</h3><div id="p0190" role="paragraph">How can a Plk4 oscillation be generated independently of the CCO? Our mathematical model (model 2 in <a id="crosref0630" href="#sec-3">STAR Methods</a>) cannot explain this, as it requires a PPTase to reset the system specifically during M-phase (<a id="crosref0635" href="#figs4">Figures S4</a>A and S4B). Interestingly, if we extend the model to allow the PPTase to have a constant low-level of activity (∼10% of the level normally required to reset the system in M-phase) (<a id="crosref0640" href="#figs8">Figure&nbsp;S8</a>G) this new model (model 3 in <a id="crosref0645" href="#sec-3">STAR Methods</a>) recapitulates several features of centriole duplication in the cyclin-depleted embryos (<a id="crosref0650" href="#figs8">Figure&nbsp;S8</a>H). This model predicts that after a last round of mitosis the centrioles in the cyclin-depleted embryos will undergo a single synchronous Plk4 oscillation (as all of the Asl receptors start this first cyclin-depleted cycle in a dephosphorylated state), but subsequent Plk4 oscillations rapidly dampen as the individual Asl receptors lose synchrony, and the system tends toward a steady state—where some of the centriolar Asl receptors are Plk4-bound and being phosphorylated, while others are not Plk4-bound and are being dephosphorylated (<a id="crosref0655" href="#figs8">Figure&nbsp;S8</a>H). Intriguingly, the inherent noise in the system generated stochastic Plk4 oscillations that could plausibly drive centriole duplication (<a id="crosref0660" href="#figs8">Figure&nbsp;S8</a>H)—potentially mimicking the stochastic Plk4 oscillations and centriole duplication events that we observe in the cyclin-depleted embryos (<a id="crosref0665" href="#fig5">Figure&nbsp;5</a>B).</div><div id="p0195" role="paragraph">In this model, each Asl receptor effectively behaves as an independent oscillator—alternating between a Plk4-bound form that is being phosphorylated and a non-Plk4-bound form that is being dephosphorylated. In the presence of the CCO, the Asl receptors generate coordinated Plk4 oscillations because the CCO synchronizes them every nuclear cycle by providing a coordinated burst of PPTase activity during mitosis.</div></section><section id="sec-2-9"><h3>Plk4 Oscillations Are Detectable in Non-dividing Mouse Liver Cells and Can Be Entrained by the Circadian Clock</h3><div id="p0200" role="paragraph">In species as distant as cyanobacteria and mammals, the CCO can be entrained to the circadian clock (<a id="crosrefs0115" href="#bib46" role="doc-biblioref" data-xml-rid="bib46 bib81">Matsuo et&nbsp;al., 2003; Yang et&nbsp;al., 2010</a>). We wondered, therefore, whether the autonomous Plk4 oscillation could also be entrained by the circadian clock. We examined a recently published diurnal proteome from non-regenerating mouse liver (<a id="crosref0670" href="#bib80" role="doc-biblioref" data-xml-rid="bib80">Wang et&nbsp;al., 2018</a>), where hepatocytes, the major building blocks of the liver, are largely quiescent (<a id="crosref0675" href="#bib25" role="doc-biblioref" data-xml-rid="bib25">Friedman, 2000</a>). Several key cell-cycle regulators (such as Cdk1, cyclin E, cyclin B1, and Plk1) were not detectable at any stage of the diurnal cycle, confirming that these cells were largely quiescent. In contrast, Plk4 protein (but not transcript) levels exhibited a striking oscillation that was entrained to the light/dark cycles (<a id="crosref0680" href="#fig7">Figures 7</a>A–7C). We presume that this oscillation is sub-threshold for centriole biogenesis—because centrioles should not be duplicating in these non-dividing cells—and simply reflects the ability of the Plk4 system to oscillate in a way that can be entrained by the circadian clock.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.018/asset/ab6b3ec0-4aa2-48fc-8f02-78e31fabecd4/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2020.05.018/asset/1fe3e791-a9c6-4385-9734-fd3b61db87fb/main.assets/gr7.jpg" height="953" width="715" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.018/asset/ab6b3ec0-4aa2-48fc-8f02-78e31fabecd4/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Plk4 Levels May Autonomously Oscillate in Mouse Liver Cells Entrained by the Circadian Clock Where Levels of PP2A Catalytic Subunit Oscillates Precisely out of Phase</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0190" role="paragraph">(A) Diagram shows the workflow used by <a id="crosref1520" href="#bib80" role="doc-biblioref" data-xml-rid="bib80">Wang et&nbsp;al. (2018)</a> to obtain a diurnal proteome of the whole liver of light/dark-entrained mice.</div><div id="fspara0195" role="paragraph">(B) Graphs reproduced from <a id="crosref1525" href="#bib80" role="doc-biblioref" data-xml-rid="bib80">Wang et&nbsp;al. (2018)</a> show the relative diurnal expression of the circadian clock transcripts Bmal1 and Per1 as internal controls. We re-analyzed the diurnal proteome produced in this study—comprising a matrix of <i>Z</i> scores for 6,780 proteins identified during 2 circadian cycles (supplemental dataset 9 from <a id="crosref1530" href="#bib80" role="doc-biblioref" data-xml-rid="bib80">Wang et&nbsp;al. [2018</a>]).</div><div id="fspara0200" role="paragraph">(C) Graphs we derived show the relative protein levels of Plk4 and the cartwheel component STIL (Ana2 in flies). Plk4 levels strongly spike in a periodic manner every circadian cycle, whereas STIL levels appear to randomly fluctuate and show neither a discernible pattern of oscillation nor any entrainment to the circadian clock. Because these cells are generally not proliferating, centrioles should not be duplicating, so the Plk4 oscillations are presumably sub-threshold for centriole biogenesis. Thus, Plk4 oscillations are detectable in non-dividing mammalian cells, where they are entrained by the circadian clock.</div><div id="fspara0205" role="paragraph">(D) Graphs examine in the non-dividing liver cells the behavior of mouse homologs of all the mitotic PPTase subunits that function in flies (<a id="crosref1535" href="#bib13" role="doc-biblioref" data-xml-rid="bib13">Chen et&nbsp;al., 2007</a>). Among the 27 PPTase subunits examined, only PPP2CB (highlighted with a red dotted frame) exhibited a clear oscillatory behavior that is similar to Plk4, and the period of these oscillations was precisely out of phase with the Plk4 oscillation.</div></div></div></div></figcaption></figure></div><div id="p0205" role="paragraph">In our model, a mitotic PPTase that dephosphorylates Asl-receptors out of phase with Plk4 is required to generate Plk4 oscillations (<a id="crosref0685" href="#figs4">Figures S4</a>A and S4B). We therefore used the mouse dataset to examine the behavior of the mouse homologs of all the mitotic PPTase subunits that function in flies (<a id="crosref0690" href="#bib13" role="doc-biblioref" data-xml-rid="bib13">Chen et&nbsp;al., 2007</a>). Among the 27 PPTase subunits examined, only PPP2CB exhibited a clear oscillatory behavior that is similar to Plk4, and the period of these oscillations was precisely out of phase with the Plk4 oscillation (<a id="crosref0695" href="#fig7">Figure&nbsp;7</a>D, highlighted with a red dotted frame). Intriguingly, PPP2CB is the homolog of Mts, the catalytic subunit of PP2A in <i>Drosophila</i> that localizes to centrosomes specifically during mitosis in fly cells, and its knockdown leads to centrosome duplication defects (<a id="optu4YZkzoPLh" href="#bib82" role="doc-biblioref" data-xml-rid="bib82">Dobbelaere et al., 2008</a>). Thus, PP2A is an excellent candidate for the PPTase that may normally dephosphorylate centriolar Asl during mitosis.</div><div id="p0210" role="paragraph">Remarkably, ∼8% of the ∼6,800 proteins in the mouse dataset exhibited a 24 h-entrained oscillatory behavior. It is unclear why so many proteins oscillate in this way, or whether any of these oscillations are of functional significance. Nevertheless, these observations indicate that there are many other proteins, and so perhaps many different biological processes, that have a largely under-appreciated ability to oscillate.</div></section><section id="sec-2-10"><h3>Concluding Remarks</h3><div id="p0215" role="paragraph">There is great interest in determining the physical and molecular principles that cells use to regulate the biogenesis of their organelles (<a id="crosrefs0120" href="#bib41" role="doc-biblioref" data-xml-rid="bib41 bib50">Liu et&nbsp;al., 2018; Mukherji and O’Shea, 2014</a>). The idea that an organelle-specific oscillation could time and execute organelle biogenesis has, to our knowledge, not been proposed previously. We suggest that the Plk4 centriole oscillation could be a paradigm for a general mechanism describing the regulation of organelle biogenesis: oscillations in the levels/activity of key regulatory factors essential for organelle biogenesis could precisely time the initiation and duration of the growth process, ensuring that organelles grow at the right time and to the appropriate size. In such a model, the CCO and circadian clocks could act simply as “phase-lockers” (<a id="crosrefs0125" href="#bib42" role="doc-biblioref" data-xml-rid="bib42 bib49">Lu and Cross, 2010; Morgan, 2010</a>), whose function is to entrain the phase of a network of autonomous oscillators to ensure that biological processes occur in a coordinated manner.</div></section></section><section id="sec-3"><h2>STAR★Methods</h2><section id="sec-3-1"><h3>Key Resources Table</h3><div id="p0220" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Mouse anti-GFP</td><td>Roche</td><td>RRID: <a id="intref0010" href="https://antibodyregistry.org/AB_390913" target="_blank">AB_390913</a></td></tr><tr><td>Mouse anti-Actin</td><td>Sigma</td><td>RRID: <a id="intref0015" href="https://antibodyregistry.org/AB_476730" target="_blank">AB_476730</a></td></tr><tr class="rowsep"><td>HRPO-linked anti-mouse IgG</td><td>Sigma / GE Healthcare</td><td>Cat# GENA931</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>QuikChange II XL mutagenesis kit</td><td>Agilent Technologies</td><td>Cat# 200521</td></tr><tr><td>Q5 Site Directed Mutagenesis kit</td><td>New England Biolabs</td><td>Cat# E0554S</td></tr><tr><td>Voltalef grade H10S oil</td><td>Arkema</td><td>N/A</td></tr><tr class="rowsep"><td>Alexa Fluor 488 NHS Ester</td><td>Thermo Fisher Scientific</td><td>Cat# A20000</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Organisms/Strains</b></td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4-mNeonGreen</td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4<sup>Aa74</sup> (Plk4 null mutant)</td><td><a id="crosref0700" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a></td><td>FlyBase ID: FBab0049012</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-mKate2</td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Sas-6-mCherry</td><td><a id="crosref0705" href="#bib60" role="doc-biblioref" data-xml-rid="bib60">Rogers et&nbsp;al., 2008</a></td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: CycB<sup>2</sup></td><td><a id="crosref0710" href="#bib32" role="doc-biblioref" data-xml-rid="bib32">Jacobs et&nbsp;al., 1998</a></td><td>FlyBase ID: FBal0094855</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: grp<sup>fsA4</sup></td><td><a id="crosref0715" href="#bib67" role="doc-biblioref" data-xml-rid="bib67">Sibon et&nbsp;al., 1997</a></td><td>FlyBase ID: FBal0062815</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-GFP</td><td><a id="crosref0720" href="#bib10" role="doc-biblioref" data-xml-rid="bib10">Blachon et&nbsp;al., 2008</a></td><td>FlyBase ID:FBtp0040947</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: asl<sup>B46</sup></td><td><a id="crosref0725" href="#bib8" role="doc-biblioref" data-xml-rid="bib8">Baumbach et&nbsp;al., 2015</a></td><td>FlyBase ID: FBal0343439</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4-GFP</td><td><a id="crosref0730" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a></td><td>FlyBase ID: FBal0343977</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-mCherry</td><td><a id="crosref0735" href="#bib16" role="doc-biblioref" data-xml-rid="bib16">Conduit et&nbsp;al., 2015</a></td><td>FlyBase ID: FBal0343645</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Sas-6-GFP</td><td><a id="crosref0740" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a></td><td>FlyBase ID: FBtp0131375</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-13A-mKate2</td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-13A</td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl</td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: wee1<sup>∗</sup></td><td>(Homozygous viable mutant derived from<a id="opt5SbWoU8UOw" href="#bib83" role="doc-biblioref" data-xml-rid="bib83">Price et al., 2000</a>; courtesy of Prof. Shelagh Campbell)</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4-mNeonGreen, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-mKate2 / Cyo; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4-mNeonGreen, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Sas-6-mCherry /&nbsp;+; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4-mNeonGreen, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: CycB<sup>2</sup> /&nbsp;+; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4-mNeonGreen, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: grp<sup>fsA4</sup> /&nbsp;+; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4-mNeonGreen, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-GFP / Asl-GFP; asl<sup>B46</sup> / asl<sup>B46</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Oregon-R (Wild-type strain)</td><td>Kyoto Stock Center</td><td>FlyBase ID: <a id="intref0020" href="https://flybase.org/reports/FBst0324696.html" target="_blank">FBst0324696</a></td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-mKate2, asl<sup>B46</sup> /&nbsp;+</td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4-GFP / Cyo; Plk4<sup>Aa74</sup> / Plk4<sup>Aa74</sup></td><td><a id="crosref0745" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a></td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-mCherry /&nbsp;+; Plk4<sup>Aa74</sup> /&nbsp;+</td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4-mNeonGreen /&nbsp;+; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4-mNeonGreen /&nbsp;+; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / asl<sup>B46</sup>, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-GFP /&nbsp;+; asl<sup>B46</sup> / asl<sup>B46</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Plk4-GFP / Cyo; asl<sup>B46</sup>, Plk4<sup>Aa74</sup> / Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Sas-6-GFP /&nbsp;+; asl<sup>B46</sup> /&nbsp;+</td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-13A-mKate2 / Asl-13A-mKate2; asl<sup>B46</sup> / asl<sup>B46</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-mKate2 / Asl-mKate2; asl<sup>B46</sup> / asl<sup>B46</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl-13A /&nbsp;+; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4-mNeonGreen, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr><td><i>D.&nbsp;melanogaster</i>: Asl /&nbsp;+; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4-mNeonGreen, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td><i>D.&nbsp;melanogaster</i>: wee1<sup>∗</sup> / wee1<sup>∗</sup>; Plk4-mNeonGreen, Plk4<sup>Aa74</sup> / Plk4-mNeonGreen, Plk4<sup>Aa74</sup></td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>Primers to introduce the NheI restriction enzyme sites into the mCherry C-terminal Gateway vector, see <a id="crosref0750" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td>Primers to replace the mCherry tag with mNeonGreen by homologous recombination on the destination vector, see <a id="crosref0755" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td>Primers to replace the mCherry tag with mKate2 by homologous recombination on the destination vector, see <a id="crosref0760" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td>Primers to remove the NheI restriction enzyme sites from the destination vector via site-directed mutagenesis (mNeonGreen vector), see <a id="crosref0765" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td>Primers to remove the NheI restriction enzyme sites from the destination vector via site-directed mutagenesis (mKate2 vector), see <a id="crosref0770" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td>Primers to amplify Cyclin A, B or B3, see <a id="crosref0775" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td>Primers to introduce various site directed mutations for Asl-13A construct, see <a id="crosref0780" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td>Primers to delete mKate2 to generate endogenous Asl-13A construct without a fluorescent tag, see <a id="crosref0785" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr class="rowsep"><td>Primers to generate endogenous Asl construct without a fluorescent tag, see <a id="crosref0790" href="#mmc1">Table S1</a>.</td><td>Invitrogen, Thermo Fisher Scientific</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>mCherry C-terminal Gateway vector</td><td><a id="crosref0795" href="#bib7" role="doc-biblioref" data-xml-rid="bib7">Basto et&nbsp;al., 2008</a></td><td>N/A</td></tr><tr><td>pDONR-Zeo vector</td><td>Thermo Fisher Scientific</td><td>Cat# 12535035</td></tr><tr><td>mNeonGreen vector</td><td><a id="crosref0800" href="#bib65" role="doc-biblioref" data-xml-rid="bib65">Shaner et&nbsp;al., 2013</a></td><td>N/A</td></tr><tr><td>mKate2 vector</td><td><a id="crosref0805" href="#bib66" role="doc-biblioref" data-xml-rid="bib66">Shcherbo et&nbsp;al., 2009</a></td><td>N/A</td></tr><tr class="rowsep"><td>Asl-mKate2 P-element transformation vector</td><td>This study</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>Fiji (ImageJ)</td><td>National Institutes of Health</td><td><a id="intref0025" href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a></td></tr><tr><td>TrackMate</td><td><a id="crosref0810" href="#bib75" role="doc-biblioref" data-xml-rid="bib75">Tinevez et&nbsp;al., 2017</a></td><td><a id="intref0030" href="https://imagej.net/TrackMate" target="_blank">https://imagej.net/TrackMate</a></td></tr><tr><td>Prism 7 and 8</td><td>GraphPad</td><td><a id="intref0035" href="https://www.graphpad.com/scientific-software/prism/" target="_blank">https://www.graphpad.com/scientific-software/prism/</a></td></tr><tr><td>Scipy’s <i>find_peaks</i> function</td><td><a id="crosref0815" href="#bib33" role="doc-biblioref" data-xml-rid="bib33">Jones et&nbsp;al., 2001</a></td><td><a id="intref0040" href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.find_peaks.html" target="_blank">https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.find_peaks.html</a></td></tr><tr><td>Asymmetric baseline smoothing</td><td><a id="crosref0820" href="#bib22" role="doc-biblioref" data-xml-rid="bib22">Eilers and Boelens, 2005</a></td><td>N/A</td></tr><tr><td>Zen Black Software</td><td>Zeiss</td><td><a id="intref0045" href="https://www.zeiss.com/microscopy/us/products/microscope-software/zen.html" target="_blank">https://www.zeiss.com/microscopy/us/products/microscope-software/zen.html</a></td></tr><tr><td>FoCuS-Point Software</td><td><a id="crosref0825" href="#bib79" role="doc-biblioref" data-xml-rid="bib79">Waithe et&nbsp;al., 2016</a></td><td>N/A</td></tr><tr><td>The equations used for mathematical modeling and regressions</td><td>This paper</td><td><a id="intref0050" href="https://github.com/RaffLab/centriole_oscillator_model" target="_blank">https://github.com/RaffLab/centriole_oscillator_model</a></td></tr><tr><td>Python script to automate PeCoS analysis</td><td>This paper</td><td><a id="intref0055" href="https://github.com/RaffLab/centriole_oscillator_model" target="_blank">https://github.com/RaffLab/centriole_oscillator_model</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2930613-9" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-3-2"><h3>Resource Availability</h3><section id="sec-3-2-1"><h4>Lead Contact</h4><div id="p0225" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Jordan W. Raff (<a id="intref0060" href="mailto:jordan.raff@path.ox.ac.uk">jordan.raff@path.ox.ac.uk</a>).</div></section><section id="sec-3-2-2"><h4>Materials Availability</h4><div id="p0230" role="paragraph">All unique/stable reagents generated in this study are available from the Lead Contact without restriction, unless for commercial application, in which case a completed Materials Transfer Agreement will be requested. There is restriction to the availability of dsRNA cocktails produced in this study, as they only last for ∼6&nbsp;months without degradation (if preserved at conditions indicated in the <a id="crosref0830" href="#sec-3">STAR Methods</a>), and therefore these cocktails are recommended to be made fresh using the protocol described in the <a id="crosref0835" href="#sec-3">STAR Methods</a>. Fly alleles and plasmids (with original source species) generated in this study will be requested by FlyBase administration to deposit onto FlyBase public archives within 6&nbsp;months following the publication of this study. Compound and recombinant flies are deposited to the Lead Contact’s laboratory stocks (without direct public access), but are available without restriction upon request.</div></section><section id="sec-3-2-3"><h4>Data and Code Availability</h4><div id="p0235" role="paragraph">The codes generated to perform mathematical modeling and regressions are available in the following web link:&nbsp;&lt; <a id="intref0065" href="https://github.com/RaffLab/centriole_oscillator_model" target="_blank">https://github.com/RaffLab/centriole_oscillator_model</a> &gt;. The code generated to automate PeCoS analysis procedure is available in the following web link:&nbsp;&lt; <a id="intref0070" href="https://github.com/RaffLab/PeCoS" target="_blank">https://github.com/RaffLab/PeCoS</a> &gt;. Source 3D time-lapse spinning-disk confocal micrographs and SIM reconstruction datasets supporting the current study are of sizes between 10 and 20GB for each experiment (exceeding the current upload limits of public repositories) and therefore have been deposited in Open Microscopy Environment (OMERO) repository. These are available without restriction, via file transfer systems, when requested from the Lead Contact – unless for commercial application, in which case a completed Materials Transfer Agreement will be requested.</div></section></section><section id="sec-3-3"><h3>Experimental Model and Subject Details</h3><section id="sec-3-3-1"><h4><i>D.&nbsp;melanogaster</i> stocks and husbandry</h4><div id="p0240" role="paragraph">The specific <i>D.&nbsp;melanogaster</i> stocks used, generated and/or tested in this study are listed in <a id="crosref0840" href="#sec-3-1">Key Resources Table</a>. To generate Plk4-mNeonGreen and Asl-mKate2 constructs: 1) NheI restriction enzyme sites were introduced into an mCherry C-terminal Gateway vector (<a id="crosref0845" href="#bib7" role="doc-biblioref" data-xml-rid="bib7">Basto et&nbsp;al., 2008</a>), using the Quikchange II XL mutagenesis kit (Agilent Technologies). 2) The mCherry tag was replaced with either mNeonGreen (<a id="crosref0850" href="#bib65" role="doc-biblioref" data-xml-rid="bib65">Shaner et&nbsp;al., 2013</a>) (Allele Biotechnology) or mKate2 (<a id="crosref0855" href="#bib66" role="doc-biblioref" data-xml-rid="bib66">Shcherbo et&nbsp;al., 2009</a>) tags by homologous recombination via In-fusion Cloning (TaKaRa). 3) NheI restriction enzyme sites were removed via site-directed mutagenesis, using the Quikchange II XL mutagenesis kit (Agilent Technologies). These vectors were recombined via Gateway technology to pDONR-Zeo vectors (Thermo Fisher Scientific) where the genetic regions of either <i>Plk4</i> (<a id="crosref0860" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>) or <i>asl</i> (<a id="crosref0865" href="#bib55" role="doc-biblioref" data-xml-rid="bib55">Novak et&nbsp;al., 2014</a>) were previously cloned from 2 kb upstream of the start codon up to (but excluding) the stop codon. Similary, to generate an Asl construct without any fluorescent tag, the endogeneous Asl stop codon was introduced to the Asl pDONR by site-directed mutagenesis, using the Quikchange II XL mutagenesis kit.</div><div id="p0245" role="paragraph">To generate the Asl-13A-mKate2 construct, 13 point mutations (<a id="crosref0870" href="#figs4">Figure&nbsp;S4</a>C) were introduced into the endogenous Asl-mKate2 P-element transformation vector (this study) through NEB Q5 Site Directed Mutagenesis (NEB #E0554S) in four sequential mutagenesis steps. To generate the endogenous Asl-13A construct without any fluorescent tag, the mKate2 coding sequence was removed, and a stop codon was introduced immediately following the Asl coding sequence in the eAsl-13A-mKate2 vector (described above) using NEB Q5 Site Directed Mutagenesis.</div><div id="p0250" role="paragraph">Primer sequences used to generate these constructs are listed in <a id="crosref0875" href="#mmc1">Table S1</a>. Transgenic lines were generated using standard P-element mediated transformation by the Fly Facility in the Department of Genetics, University of Cambridge (Cambridge, England, UK) or BestGene Inc. (USA). Flies were maintained at 18°C or 25°C on <i>Drosophila</i> culture medium (0.77% agar, 6.9% maize, 0.8% soya, 1.4% yeast, 6.9% malt, 1.9% molasses, 0.5% propionic acid, 0.03% ortho-phosphoric acid, and 0.3% nipagin) in vials or bottles.</div></section></section><section id="sec-3-4"><h3>Method Details</h3><section id="sec-3-4-1"><h4>Hatching experiments</h4><div id="p0255" role="paragraph">To measure embryo hatching rates, 0-3&nbsp;h embryos were collected and aged for 24 h, and the % of embryos that hatched out of their chorion was calculated.</div></section><section id="sec-3-4-2"><h4>Synthesis of double-stranded RNA</h4><div id="p0260" role="paragraph">Double-stranded RNAs (dsRNAs) against cyclins A, B and B3 were synthesized essentially as described previously (<a id="crosref0880" href="#bib47" role="doc-biblioref" data-xml-rid="bib47">McCleland and O’Farrell, 2008</a>). Primer sequences used for gene amplification are listed in <a id="crosref0885" href="#mmc1">Table S1</a>. The resulting RNA was precipitated with 8&nbsp;μL of 3M Na-Acetate and 220&nbsp;μL of 100% ethanol before washing with 70% cold ethanol. The RNA pellets were air-dried and resuspended in 30&nbsp;μL of RNase-free diethylpyrocarbonate-treated water (Thermo Fisher Scientific). To generate double-stranded molecules, RNAs were placed in a 67.5°C water bath for 30&nbsp;min, and allowed to cool to room temperature over 90&nbsp;min. Unincorporated UTPs were removed using CHROMA SPIN-100-DEPC-H<sub>2</sub>O columns (Clontech) according to the manufacturer’s instructions. To confirm the synthesis of the correct RNA product, 3&nbsp;μL of the final reaction was subjected to electrophoresis on a 1.5% agarose gel using 2xRNA loading buffer (Thermo Fisher Scientific). A 1:1 mix of RNA and loading buffer was heated to 65°C for 5&nbsp;min and then placed on ice to denature any secondary structure of RNA.</div></section><section id="sec-3-4-3"><h4>Embryo collections and dsRNA injections</h4><div id="p0265" role="paragraph">For embryo collections, 25% cranberry-raspberry juice plates (2% sucrose and 1.8% agar with a drop of yeast suspension) were used. Embryos for imaging experiments were collected for 1h at 25°C, and aged at 25°C for ∼45–60&nbsp;min. Embryos were dechorionated by hand, mounted on a strip of glue on a 35-mm glass-bottom Petri dish with 14&nbsp;mm micro-well (MatTek), and were left to desiccate for 1&nbsp;min at 25°C. After desiccation, the embryos were covered with Voltalef grade H10S oil (Arkema). Embryos for dsRNA injection experiments were treated in the same way except that the desiccation period was increased to 5-6&nbsp;min. Embryos were injected with dsRNA at a needle concentration of 0.6–0.8&nbsp;mg/ml.</div></section><section id="sec-3-4-4"><h4>Immunoblotting</h4><div id="p0270" role="paragraph">Immunoblotting was performed as described previously (<a id="crosref0890" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>). Primary antibodies used in this study are as follows: mouse anti-GFP (Roche; RRID: <a id="intref0075" href="https://antibodyregistry.org/AB_390913" target="_blank">AB_390913</a>) and mouse anti-Actin (Sigma; RRID: <a id="intref0080" href="https://antibodyregistry.org/AB_476730" target="_blank">AB_476730</a>). Both the antibodies were used at 1:500 dilution in blocking solution (<a id="crosref0895" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>). For all blots, 10, 20 or 30 staged early embryos were boiled in sample buffer and loaded in each lane. The incubation period for primary antibodies was 1&nbsp;h (or overnight at 4°C). Membranes were quickly washed 3x in&nbsp;TBST (TBS and 0.1% Tween 20) and then incubated with HRPO-linked anti-mouse IgG (both GE Healthcare) diluted 1:3,000 in blocking solution for 45&nbsp;min. Membranes were washed 3x15min in TBST and then incubated in SuperSignal West Femto Maximum Sensitivity Substrate (Thermo Fisher Scientific). Membranes were exposed to film using exposure times that ranged from&nbsp;&lt; 1 to 60s.</div></section><section id="sec-3-4-5"><h4>Image acquisition, processing, and analysis</h4><section id="sec-3-4-5-1"><h5>Spinning disk confocal microscopy</h5><div id="p0275" role="paragraph">Living embryos were imaged at room temperature using a system equipped with an EM-CCD Andor iXon+ camera on a Nikon Eclipse TE200-E microscope using a Plan-Apochromat 60x/1.42-NA oil DIC lens, controlled with Andor IQ2 software. Confocal sections of 17 slices at 0.5μm intervals were collected every 30 s. A 488nm laser was used to excite mNeonGreen and GFP, and a 568nm laser was used to excite mCherry and mKate2. Emission discrimination filters were applied when mNeonGreen and mCherry were imaged together.</div><div id="p0280" role="paragraph">Post-acquisition image processing was performed using Fiji (National Institutes of Health). Maximum-intensity projections of the images were first bleach-corrected with Fiji’s <i>exponential fit</i> algorithm, and background was subtracted using the <i>subtract background</i> tool with a rolling ball radius of 10 pixels. Plk4-NG, Sas-6-mCherry or -GFP, and Asl-mCherry or -GFP were tracked using the Fiji plug-in TrackMate (<a id="crosref0900" href="#bib75" role="doc-biblioref" data-xml-rid="bib75">Tinevez et&nbsp;al., 2017</a>) with a track spot diameter size of 1.1&nbsp;μm. When Plk4-NG was continuously monitored over cycles 11-13, the maximum intensity projections were limited to ± 5 slices from the central plane of the nuclei, as the nuclei and centrosomes progressively get closer to the embryo cortex at successive cycles. This processing more accurately compares the dynamics of centriolar Plk4-NG at successive cycles by avoiding fluctuations due to the varying depths of the centrioles in the embryo. The regressions for the centriole growth curves (Sas-6-GFP or -mCherry) were calculated in Prism 7 (GraphPad Software), as described previously (<a id="crosref0905" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>). The regressions for the Plk4 oscillation curves (Plk4-NG) were calculated using the <i>nonlinear regression</i> (curve fit) function in Prism 7. Discrete Plk4 oscillation curves in S-phase were initially fitted against four different functions to assess the most suitable regression model: 1) Lorentzian, 2) Gaussian, 3) Increase – Constant – Decrease, and 4) Increase – Decrease. Among these models, Lorentzian best fit the data (<a id="crosref0910" href="#figs1">Figure&nbsp;S1</a>D), so all the discrete Plk4 oscillation curves in S-phase were regressed using this function. The Lorentzian and Gaussian functions are described in Prism 7, while the latter two functions are in-house algorithms (<a id="crosref0915" href="#bib1" role="doc-biblioref" data-xml-rid="bib1">Alvarez-Rodrigo et&nbsp;al., 2019</a>).</div><div id="p0285" role="paragraph">In order to plot the dynamics of Plk4-NG and Sas-6-mCherry together (<a id="crosref0920" href="#fig2">Figures 2</a>, <a id="crosref0925" href="#figs3">S3</a>A, and S3B), the highest mean fluorescence signal for each tag was normalized to 1 and was accordingly scaled across cycles 11-13 (the scaling factor for Plk4-NG was calculated from the data shown in <a id="crosref0930" href="#fig1">Figures 1</a>B and 1C). Note that the amplitude of the Plk4 oscillation does not appear to decrease significantly between nuclear cycles 11-13 in the data shown in <a id="crosref0935" href="#fig2">Figure&nbsp;2</a>A—in contrast to the Plk4 oscillations shown in <a id="crosref0940" href="#fig1">Figure&nbsp;1</a>B. This is not because of the scaling procedure applied to the data shown in <a id="crosref0945" href="#fig2">Figure&nbsp;2</a>A (described above), but rather because embryos that failed to grow their centrioles were excluded from the analysis shown in <a id="crosref0950" href="#fig2">Figure&nbsp;2</a>A. The amplitude of the Plk4 oscillations was lower in these embryos (<a id="crosref0955" href="#fig2">Figure&nbsp;2</a>C), so embryos with low amplitude Plk4 oscillations were effectively excluded from the analysis shown in <a id="crosref0960" href="#fig2">Figure&nbsp;2</a>A. Almost all of these excluded embryos were at nuclear cycle 13, so the “average” oscillation at nuclear cycle 13 is in reality an average of only those embryos that had a relatively high amplitude Plk4 oscillation.</div><div id="p0290" role="paragraph">In all the imaging experiments, the beginning of S-phase was taken as the time at which the old and new mother centrioles were first detected to separate from each other (termed “centrosome separation” or “CS”). Entry into mitosis was taken as the time of nuclear envelope breakdown (NEB), which could be determined in our movies by adjusting the contrast to visualize when the cytosolic pool of the fluorescent protein was first observed to enter into the nucleus.</div></section></section><section id="sec-3-4-6"><h4>Analysis of centriole “fertility” in embryos injected with dsRNA against cyclin A-B-B3</h4><div id="p0295" role="paragraph">In experiments where we depleted embryos of mitotic cyclins during early rounds of nuclear division, we observed qualitatively that “fertile” centrioles exhibited distinct Plk4-NG fluorescence peaks that often appeared to correlate with centriole duplication events, while “sterile” centrioles exhibited no obvious peaks (<a id="crosref0965" href="#fig5">Figure&nbsp;5</a>B). To test if we could more quantitatively distinguish between fertile and sterile centrioles, we computationally analyzed all 81 centrioles that we could track throughout the observation period in 3 different embryos. We first assessed the average signal-to-noise ratio (SNR) of Plk4-NG fluorescence of each centriole over the entire observation period and found that fertile centrioles exhibited a significantly higher SNR than sterile centrioles—assessed using a t test assuming equal variance (<a id="crosref0970" href="#figs8">Figure&nbsp;S8</a>B). The distribution of SNR within sterile and fertile centriole signals was unimodal and symmetrically distributed (<a id="crosref0975" href="#figs8">Figure&nbsp;S8</a>C), so we attempted to classify centrioles in an unbiased way by thresholding the SNR. Based on the bimodality of the SNR, an automatic threshold was determined from the data using Otsu thresholding (<i>red dashed</i> line <a id="crosref0980" href="#figs8">Figure&nbsp;S8</a>C); the classification performance was summarized in a visual confusion matrix, which shows the proportion of correctly and falsely classified signals (<a id="crosref0985" href="#figs8">Figure&nbsp;S8</a>D). This unbiased computational method successfully classified ∼74% of the fertile centrioles and ∼71% of the sterile centrioles.</div><section id="sec-3-4-6-1"><h5>Peak Calling</h5><div id="p0300" role="paragraph">We next tested whether computationally identified peaks in the Plk4-NG signal were correlated with centriole duplication events. Plk4-NG peaks were only called on signals whose fluctuation (as measured by signal-to-noise ratio, SNR) was greater than a certain defined threshold (0.1, see below). A peak was defined only as a local maximum in intensity. To call a peak, the Plk4-NG signal intensity was compared to the signal intensity at neighboring times. Here an unbiased distance&nbsp;= 1 was set, that is, an intensity at time <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>t</mi></math> is a peak only if the intensity is higher than those at both <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>t</mi><mo linebreak="goodbreak" linebreakstyle="after">−</mo><mn>1</mn></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>t</mi><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mn>1</mn></mrow></math>. To filter noise detections, a threshold of 0.1 was placed on the peak prominence. Peak prominence measures the extent to which a detected peak stands out from its surrounding – it is defined as the vertical distance between the peak and its lowest contour line (Scipy’s <i>find_peaks</i> function) (<a id="crosref0990" href="#bib33" role="doc-biblioref" data-xml-rid="bib33">Jones et&nbsp;al., 2001</a>). The choice of 0.1 as a threshold was guided by comparing a peaks predictive power given no cut-off with the (ground truth) duplication time. This analysis indicated that the optimal peak prominence cut-off—i.e., the point at which the power of the peaks to predict duplication events (see below) sharply drops off—was 0.12 (<i>green dot</i>, <a id="crosref0995" href="#fig5">Figure&nbsp;5</a>E). The observed steep drop-off in predictive power below this threshold supports the view that that there is likely to be a minimal amount of centriolar Plk4-NG that is necessary to trigger duplication under these conditions. Moreover, this unbiased computational approach identified ∼4x as many peaks in the fertile centrioles when compared to the sterile centrioles.</div><div id="p0305" role="paragraph">To determine whether the filtered Plk4-NG peaks were predictive of centriole duplication, we determined all the peaks above the 0.1 threshold for the fertile centriole signals and assessed whether these peaks could be used to “retrieve” the real or relevant time points for centriole duplication. The performance of such retrieval can be evaluated using “precision” (the number of relevant retrievals among all retrieved instances—in this case the number of Plk4 peaks associated with a centriole duplication event divided by the total number of Plk4 peaks) and “recall” (the number of relevant instances retrieved of the total relevant instances—in this case the number of Plk4 peaks associated with a centriole duplication event divided by the total number of centriole duplication events), as defined below.<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>P</mi><mi>r</mi><mi>e</mi><mi>c</mi><mi>i</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi><mspace width="0.25em"></mspace><mo linebreak="badbreak">=</mo><mspace width="0.25em"></mspace><mfrac><mrow><mi>N</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.25em"></mspace><mi>o</mi><mi>f</mi><mrow><mo>(</mo><mrow><mi>R</mi><mi>e</mi><mi>l</mi><mi>e</mi><mi>v</mi><mi>a</mi><mi>n</mi><mi>t</mi><mspace width="0.25em"></mspace><mo>&amp;</mo><mspace width="0.25em"></mspace><mi>R</mi><mi>e</mi><mi>t</mi><mi>r</mi><mi>i</mi><mi>e</mi><mi>v</mi><mi>e</mi><mi>d</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>N</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.25em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.25em"></mspace><mrow><mo>(</mo><mrow><mi>R</mi><mi>e</mi><mi>t</mi><mi>r</mi><mi>i</mi><mi>e</mi><mi>v</mi><mi>e</mi><mi>d</mi></mrow><mo>)</mo></mrow></mrow></mfrac><mo>,</mo><mspace width="0.25em"></mspace></mrow></math></div></div></div><div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>R</mi><mi>e</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>l</mi><mspace width="0.25em"></mspace><mo linebreak="badbreak">=</mo><mfrac><mrow><mi>N</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.25em"></mspace><mi>o</mi><mi>f</mi><mrow><mo>(</mo><mrow><mi>R</mi><mi>e</mi><mi>l</mi><mi>e</mi><mi>v</mi><mi>a</mi><mi>n</mi><mi>t</mi><mspace width="0.25em"></mspace><mo>&amp;</mo><mspace width="0.25em"></mspace><mi>R</mi><mi>e</mi><mi>t</mi><mi>r</mi><mi>i</mi><mi>e</mi><mi>v</mi><mi>e</mi><mi>d</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>N</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.25em"></mspace><mi>o</mi><mi>f</mi><mrow><mo>(</mo><mrow><mi>R</mi><mi>e</mi><mi>l</mi><mi>e</mi><mi>v</mi><mi>a</mi><mi>n</mi><mi>t</mi></mrow><mo>)</mo></mrow></mrow></mfrac></mrow></math></div></div></div>The evaluation of such a system naturally depends on the cut-off to call a positive match between the Plk4-NG signal peak and its corresponding centriole duplication time. Too small a cut-off (e.g., 0&nbsp;minutes) is unrealistic: no system can predict time perfectly; while too large a cut-off (e.g., 15&nbsp;min) is too lenient and non-specific. <a id="crosref1000" href="#fig6">Figure&nbsp;6</a>C plots the precision evaluated over all centrioles for different temporal cut-offs attempting to uniquely match Plk4-NG peaks to the nearest duplication time within a given time window. The elbow point (<i>red dashed</i> line) at 5&nbsp;min was selected as an appropriate cut-off with a precision of ∼80%. (Note that the recall was not plotted in this graph, but it exhibits a similar behavior to the precision: the <i>Number of (Relevant)</i>&nbsp;= 52, while the <i>Number of (Retrieved)</i>&nbsp;= 49). This temporal cut-off can also be interpreted as an estimate of the temporal accuracy to which Plk4-NG peak time associates with centriole duplication time. For comparison, we also derived the mean temporal separation distance of peaks and duplication events, if the same number of experimental centriole duplication times were randomly distributed over the same time interval for each embryo. 1000 simulations were run per embryo to produce a distribution. Across all embryos, an average temporal separation distance (for randomly distributed duplication times) was 10.5&nbsp;minutes (data not shown), twice as long as the chosen 5&nbsp;min cut-off, thus the association is not coincidental.</div><div id="p0310" role="paragraph">In addition, we assessed the precision and recall performance over different possible threshold values (based on peak prominence) used to call a Plk4 ‘peak’. To do this, we computed the precision-recall curve. All detected Plk4 peaks (<i>Black</i> dots; <a id="crosref1005" href="#fig5">Figure&nbsp;5</a>E) were ranked according to their peak prominence from high to low and were assigned uniquely to a duplication event according to a 5&nbsp;min time window for determining a positive match. Peaks that could not be uniquely assigned in such a manner were regarded as ‘negatives’. The graph then plots the precision, recall values if the threshold for calling a peak were set as the peak prominence value of each peak in descending order. Beyond the detected peak associated with a peak prominence of 0.12 (i.e., points right of this point), the precision drops sharply. At this threshold, precision and recall are jointly optimized. This suggests that a minimum level of Plk4-NG peak fluorescence intensity is required to predict duplication. The ability of Plk4 peaks to predict duplication across all peak prominences (over the selected time window of 5&nbsp;min) is quantified by the integrated area under the curve or average precision (AP<sub>5min</sub>). If there were no overall correlation between a Plk4 peak and a duplication event, AP<sub>5min</sub> would be 51.5% (given by # duplications / (# duplications&nbsp;+ # peaks)); the score of ∼75% indicates a strong overall correlation (<a id="crosref1010" href="#fig5">Figure&nbsp;5</a>E).</div><div id="p0315" role="paragraph">Finally, the correlation between the Plk4-NG peaks and times of centriole duplication was examined (<a id="crosref1015" href="#fig5">Figure&nbsp;5</a>F), which provided an alternative accuracy test. Plk4-NG peaks were uniquely matched to the nearest centriole division times without using a temporal cut-off over individual centrioles from three independent embryos. Pearson correlation r, <i>R</i><sup><i>2</i></sup> and <i>P value</i>s are reported as goodness of fit. The fitted regression line, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>y</mi><mo linebreak="goodbreak" linebreakstyle="after">=</mo><mn>0.87</mn><mi>x</mi><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mn>3.69</mn></mrow></math>. Together, these unbiased computational analyses indicate that the Plk4 oscillations at individual centrioles are highly correlated with the time at which these centrioles duplicate.</div></section></section><section id="sec-3-4-7"><h4>Spatiotemporal heatmap of centriole duplications</h4><div id="p0320" role="paragraph">To visually assess whether there is bias as to where and when centriole duplications happen, the (x,y) position of all duplicating centrioles were overlaid on the (x,y) positions of all centrioles (duplicating and non-duplicating at all times; <i>black</i> dots) (Figure&nbsp;S8E). To grade the temporal sequence of centriole separations, duplication points were colored blue to red. To enable comparison across embryos relative to the same geometric reference, the embryonic width and anterior-posterior were set as <i>x</i>- and <i>y</i>-axes, respectively, by applying a principal component analysis on the extracted (x,y) positions of all centrioles.</div></section><section id="sec-3-4-8"><h4>Spatial clustering assessment of centriole duplications</h4><div id="p0325" role="paragraph">To statistically measure whether centriole duplications are enriched at particular spatial regions over time, Ripley <i>K</i> statistics was used (<a id="crosref1020" href="#bib59" role="doc-biblioref" data-xml-rid="bib59">Ripley, 1976</a>). In the field of spatial statistics, given a set of (x,y) points, the Ripley <i>K</i> statistic detects deviations from spatial homogeneity at different distances between points or spatial scales (e.g., such analyses are heavily used in geophysics to map out the spatial distribution of natural disasters and in crime statistics for detecting high incidence areas). For a dataset of <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>n</mi></math> points, the Ripley <i>K</i> statistic, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msup><mi>K</mi><mrow><mi>R</mi><mi>i</mi><mi>p</mi><mi>l</mi><mi>e</mi><mi>y</mi></mrow></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mspace width="0.25em"></mspace></mrow></math>is the mean spatial occurrence of two points, point <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>i</mi></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>j</mi></math> having a separation distance, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>d</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math> less than the search distance threshold of <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>d</mi></math>:<div id="ufd3" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msup><mi>K</mi><mrow><mi>R</mi><mi>i</mi><mi>p</mi><mi>l</mi><mi>e</mi><mi>y</mi></mrow></msup><mrow><mo>(</mo><mi>d</mi><mo>)</mo></mrow><mo linebreak="badbreak">=</mo><mfrac><mn>1</mn><mi>λ</mi></mfrac><munder><mo>∑</mo><mrow><mi>i</mi><mo linebreak="badbreak">≠</mo><mi>j</mi></mrow></munder><mrow><mfrac><mrow><mi mathvariant="double-struck">I</mi><mrow><mo>(</mo><mrow><msub><mi>d</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">&lt;</mo><mi>d</mi></mrow><mo>)</mo></mrow></mrow><mi>n</mi></mfrac></mrow></mrow></math></div></div></div><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>λ</mi></math> is the average density of points (estimated as <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mi>n</mi><mo>/</mo><mi>A</mi></mrow></mrow></math> where the number of total points, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>n</mi></math> is divided by the area of the region containing all points, <i>A</i>), and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi mathvariant="double-struck">I</mi></math> is the indicator counting function (<math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi mathvariant="double-struck">I</mi></math>&nbsp;= 1 if its operand is true, 0 otherwise). Thus, if points are homogeneously spread in 2D, the Ripley <i>K</i> statistic should vary quadratically as <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>π</mi><msup><mi>d</mi><mn>2</mn></msup></mrow></math>. The basic test assumes (x,y) points occur at any spatial position continuously in the image. However, centrioles only duplicate in certain discrete positions within fly embryos. Thus, to examine evidence of spatial clustering from the natural distribution of centrioles, we assessed difference in the Ripley <i>K</i> statistics computed from the (x,y) positions of all duplicating centrioles and the (x,y) positions of all centrioles accumulated over time.</div></section><section id="sec-3-4-9"><h4>Plk4-NG smooth curve fitting and parameter extraction</h4><div id="p0330" role="paragraph">To enable accurate extraction of signal parameters from the Plk4-NG oscillations, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>y</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow></math> the signals were robustly fit to a smooth 1D function. Here a mixture of <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>N</mi></math> Gaussian and linear trend was used with the following functional form to enable local modeling of the peak and troughs of signals:<div id="ufd4" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>y</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo linebreak="badbreak">=</mo><mrow><mo>(</mo><mrow><mi>A</mi><mo linebreak="badbreak">+</mo><mi>B</mi><mi>t</mi></mrow><mo>)</mo></mrow><mo linebreak="goodbreak">+</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo linebreak="badbreak">=</mo><mn>1</mn></mrow><mi>N</mi></munderover><mrow><msub><mi>C</mi><mi>i</mi></msub><msup><mi>e</mi><mrow><mo>−</mo><mfrac><mrow><msup><mrow><mrow><mo>(</mo><mrow><mi>t</mi><mo linebreak="badbreak">−</mo><msub><mi>μ</mi><mi>i</mi></msub></mrow><mo>)</mo></mrow></mrow><mn>2</mn></msup></mrow><mrow><msubsup><mi>σ</mi><mi>i</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></msup></mrow></mrow></math></div></div></div>where <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>t</mi></math> is time, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>A</mi><mo>,</mo><mi>B</mi></mrow></math> are the constant and slope of a linear trend line, and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>C</mi><mi>i</mi></msub><mo>,</mo><mspace width="0.25em"></mspace><msub><mi>μ</mi><mi>i</mi></msub><mo>,</mo><mspace width="0.25em"></mspace><msub><mi>σ</mi><mi>i</mi></msub></mrow></math> are the amplitude, time and temporal duration of the <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>i</mi></math><sup>th</sup> Gaussian, respectively. This function was fit in two steps. In the first step <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>A</mi><mo>,</mo><mi>B</mi></mrow></math> was fit by applying least-squares linear regression on the baseline trend line that is extracted from asymmetric baseline smoothing (<a id="crosref1025" href="#bib22" role="doc-biblioref" data-xml-rid="bib22">Eilers and Boelens, 2005</a>). In the second step, peak and trough positions were first detected on the de-trended signal, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>y</mi><mo>'</mo><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mspace width="0.25em"></mspace></mrow></math> after subtraction of the fitted trend line in the first step from the original signal, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>y</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow></math>, so as to determine the number <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>N</mi></math> of Gaussians to fit. The mixture of Gaussians was then fit by iterative non-linear regression using a robust Cauchy loss function. From the fit signal, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>y</mi><msub><mrow><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow><mrow><mi>f</mi><mi>i</mi><mi>t</mi><mi>t</mi><mi>e</mi><mi>d</mi></mrow></msub></mrow></math>, peak and trough positions were re-detected, and the following signal parameters were extracted (<a id="crosref1030" href="#fig6">Figure&nbsp;6</a>):<div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0335" role="paragraph"><b>Acceleration rate</b> <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>(</mo><mrow><mrow><mrow><mi mathvariant="bold">Δ</mi><mi mathvariant="bold">F</mi><mi mathvariant="bold">l</mi><mi mathvariant="bold">u</mi><mi mathvariant="bold">o</mi><mo>.</mo><mspace width="0.25em"></mspace><mi mathvariant="bold">A</mi><mo>.</mo><mi mathvariant="bold">U</mi><mo>.</mo></mrow><mo linebreak="badbreak">/</mo><mrow><mi mathvariant="bold">s</mi><mi mathvariant="bold">e</mi><mi mathvariant="bold">c</mi><mi mathvariant="bold">s</mi></mrow></mrow></mrow><mo>)</mo></mrow></mrow></math><b>:</b> The maximum rate of increase in fluorescence between successive time points during a trough to peak oscillation phase.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0340" role="paragraph"><b>Deceleration rate</b> <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>(</mo><mrow><mrow><mrow><mi mathvariant="bold">Δ</mi><mi mathvariant="bold">F</mi><mi mathvariant="bold">l</mi><mi mathvariant="bold">u</mi><mi mathvariant="bold">o</mi><mo>.</mo><mspace width="0.25em"></mspace><mi mathvariant="bold">A</mi><mo>.</mo><mi mathvariant="bold">U</mi><mo>.</mo></mrow><mo linebreak="badbreak">/</mo><mrow><mi mathvariant="bold">s</mi><mi mathvariant="bold">e</mi><mi mathvariant="bold">c</mi><mi mathvariant="bold">s</mi></mrow></mrow></mrow><mo>)</mo></mrow></mrow></math><b>:</b> The maximum rate of decrease in fluorescence between successive time points during a peak to trough oscillation phase.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0345" role="paragraph"><b>Oscillation peak time</b> <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>(</mo><mi mathvariant="bold-italic">s</mi><mo>)</mo></mrow></mrow></math><b>:</b> The time point corresponding to the maximum fluorescence.</div></div></div><div id="u0045" role="listitem"><div class="label">•</div><div class="content"><div id="p0350" role="paragraph"><b>Oscillation trough time</b> <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>(</mo><mi mathvariant="bold-italic">s</mi><mo>)</mo></mrow></mrow></math> <b>into mitosis:</b> The end point of a trough after which the fluorescence begins to accelerate upward.</div></div></div></div></div><div id="p0355" role="paragraph">The method described here was also used to determine the period of Plk4 oscillations (measuring peak-to-peak time; see <a id="crosref1035" href="#sec-2">Results and Discussion</a>) both in normal embryos and in embryos where dsRNA was injected against cyclins A, B and B3 to halt the progression of cell cycle.</div></section><section id="sec-3-4-10"><h4>3D-Structured Illumination Microscopy (3D-SIM)</h4><div id="p0360" role="paragraph">Living embryos were imaged at room temperature using a DeltaVision OMX V3 Blaze microscope (GE Healthcare). The system was equipped with a 60x/1.42-NA oil UPlanSApo objective (Olympus Corp.), 488nm and 593nm diode lasers, and Edge 5.5 sCMOS cameras (PCO). Spherical aberration was reduced by matching the refractive index of the immersion oil (1.514) to that of the embryos. 3D-SIM image stacks consisting of six slices at 0.125μm intervals were acquired in five phases and from three angles per slice. The raw acquisition was reconstructed using softWoRx 6.1 (GE Healthcare) with a Wiener filter setting of 0.006 and channel-specific optical transfer functions (OTFs). Filters used for the green and red channels were a 540/80 center band pass filter and a 605-long pass filter, respectively. For two-color 3D-SIM, images from green and red channels were registered with the alignment coordination information obtained from the calibrations using 0.2μm-diameter TetraSpeck beads (Thermo Fisher Scientific) in the OMX Editor software. The SIMCheck plug-in in ImageJ (National Institutes of Health) was used to assess the quality of the SIM reconstructions (<a id="crosref1040" href="#bib5" role="doc-biblioref" data-xml-rid="bib5">Ball et&nbsp;al., 2015</a>); only images that passed this test were used.</div></section><section id="sec-3-4-11"><h4>Mathematical modeling and its experimental validation</h4><section id="sec-3-4-11-1"><h5>Model 1: A simple mathematical model of discrete S-phase Plk4 oscillations</h5><div id="p0365" role="paragraph"><a id="crosref1045" href="#fig3">Figures 3</a>A and 3B, specify a regulatory network where Plk4 binds to an Asl receptor with high affinity; this activates Plk4, allowing it to phosphorylate itself and Asl multiple times. After a certain number of phosphorylations, Asl switches to a new state that binds Plk4 with low affinity. As a result, Plk4 unbinds, leaving Asl in a phosphorylated, low affinity state. We suspect that Asl is normally dephosphorylated by a phosphatase in M-phase, which “resets” it to a high-affinity state in preparation for the next oscillation in S-phase (this additional step is considered in Model 2 below). In this first model, the gradual conversion of centriolar Asl to a low affinity binding state forms a time-delayed negative feedback loop wherein Asl effectively activates Plk4 to gradually promote its own inhibition. After making assumptions about the chemical kinetics of the system and imposing suitable initial conditions, the behavior of this regulatory network can be simulated by mapping it onto a set of coupled ordinary differential equations.</div><div id="p0370" role="paragraph">In the model, it is assumed that the diffusion of Plk4 is sufficiently fast such that it remains well-mixed in the cytoplasm and that centrioles are large macromolecular structures; this implies that Plk4 and Asl receptors on the centriole follow mass-action kinetics. Let <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mi mathvariant="bold-italic">P</mi><mo>]</mo></mrow></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msub><mi mathvariant="bold-italic">A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow></mrow></math> denote the concentrations of unbound cytosolic Plk4 and unbound centriolar Asl, (per unit volume and area, respectively). Plk4 binds to Asl with the fixed rate constant <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi mathvariant="bold-italic">k</mi></math>, and the rate constant of the reverse reaction is sufficiently small that any unbinding is ignored. Once Plk4 is bound to Asl, it can only unbind after it has phosphorylated Asl a certain number of times (<i>N)</i>. We denote by <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msubsup><mi mathvariant="bold-italic">A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mspace width="0.25em"></mspace></mrow></math> the concentration of Asl receptor that has bound Plk4, but has not yet been phosphorylated, and by <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msubsup><mi mathvariant="bold-italic">A</mi><mi mathvariant="bold-italic">i</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math> the concentration of Asl receptors that have been phosphorylated <i>i</i> times. Throughout this paper, we use an asterisk superscript to denote an Asl receptor that has bound Plk4 and a numerical subscript to denote the phosphorylation state of the receptor. Each phosphorylation of Asl by Asl-bound Plk4 has rate constant <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi mathvariant="bold-italic">k</mi><mn>2</mn></msub></mrow></math> and, following <i>N</i> phosphorylations, the Asl is switched to a state that binds Plk4 with very low affinity <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msubsup><mi mathvariant="bold-italic">A</mi><mi mathvariant="bold-italic">N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math>. Once Asl has been converted to this low affinity state, Plk4 unbinds at rate <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi mathvariant="bold-italic">k</mi><mn>3</mn></msub></mrow></math>. The rate constant of the reaction where Plk4 binds to Asl receptor in this low affinity state is assumed to be sufficiently small that this reaction is ignored in the model.</div><div id="p0375" role="paragraph">Intuitively, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>k</mi></math> scales the affinity with which Plk4 binds to an Asl receptor. By mass-action kinetics, the rate of this reaction is given by <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>k</mi><mrow><mo>[</mo><mi>P</mi><mo>]</mo></mrow><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow></mrow></math>. It is assumed in the model that Plk4 is abundant enough in the cytoplasm that its concentration does not decrease over the few minutes of a single S-phase cycle; this assumption means that <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mi>P</mi><mo>]</mo></mrow></mrow></math> remains constant over that time (see below for further discussion of this assumption). Therefore, the number of parameters in the model is reduced by introducing a new rate constant <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi mathvariant="bold-italic">k</mi><mn>1</mn></msub><mo linebreak="goodbreak" linebreakstyle="after">=</mo><mi mathvariant="bold-italic">k</mi><mrow><mo>[</mo><mi mathvariant="bold-italic">P</mi><mo>]</mo></mrow></mrow></math>.</div><div id="p0380" role="paragraph">Using the assumptions above, the regulatory network in <a id="crosref1050" href="#fig3">Figures 3</a>A and 3B is simulated using the following set of ordinary differential equations which are solved over the time domain 0&nbsp;≤ <i>t</i> ≤ <i>S</i>, where <i>S</i> is the length of S-phase:<div id="fd1" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>1</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(1)</div></div><div id="fd2" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(2)</div></div><div id="fd3" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mrow><mi>N</mi><mo linebreak="badbreak">−</mo><mn>1</mn></mrow><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>3</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(3)</div></div><div id="fd4" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>1</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(4)</div></div>Appropriate initial conditions at t&nbsp;= 0 are,<div id="fd5" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="badbreak">=</mo><msubsup><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup><mo>;</mo><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">=</mo><msub><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn></msub><mo>;</mo><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">=</mo><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>2</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">=</mo><mo>⋯</mo><mo linebreak="goodbreak">=</mo><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">=</mo><mn>0</mn><mo>.</mo></mrow></math></div></div><div class="label">(5)</div></div>In <a id="crosref1055" href="#fd5">Equation 5</a>, the positive constant <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msubsup><mrow><mover accent="true"><mi mathvariant="bold-italic">A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn><mo>∗</mo></msubsup></mrow></math> is the initial amount of Plk4-bound Asl at the centriole at the start of each S-phase, which is determined experimentally for each cell cycle using the techniques described in the <i>Image acquisition, processing and analysis</i> section of <a id="crosref1060" href="#sec-3">STAR Methods</a>. The constant <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mover accent="true"><mi mathvariant="bold-italic">A</mi><mo>ˆ</mo></mover></mrow></math> is the initial amount of unbound Asl, so the total amount of Asl in the system is given by <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>A</mi><mrow><mi>t</mi><mi>o</mi><mi>t</mi><mspace width="0.25em"></mspace></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">=</mo><msubsup><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn><mo>∗</mo></msubsup><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mspace width="0.25em"></mspace><msub><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn></msub></mrow></math>.</div><div id="p0385" role="paragraph">Since the model specified by <a id="crosrefs0130" href="#fd1">Equations 1–5</a> is a system of linear differential equations with constant coefficients, it has an analytical solution that can be expressed as a sum of exponentials. Values for the parameters <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn></msub></mrow></math>, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>1</mn></msub></mrow></math>, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow></math>, and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>3</mn></msub></mrow></math> can then be determined by fitting the curve <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mo>⋯</mo><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow></math> to the experimentally measured data for the amount of Asl-bound Plk4 (i.e., the Plk4 that is recruited to the centriole) over time. Fitting was done using a <i>trust-region</i> algorithm to optimize a nonlinear least-squares penalty function.</div></section><section id="sec-3-4-11-2"><h5>Parameter fitting</h5><div id="p0390" role="paragraph">The fitting was constrained to enforce that all parameters were positive, and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>1</mn></msub></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow></math> were taken to be less than 1. Each cycle was fitted individually using the discrete Plk4-NG oscillation data from S-phase of cycles 11, 12 and 13 (<a id="crosref1065" href="#fig3">Figure&nbsp;3</a>C). Parameter values are shown in <a id="crosref1070" href="#mmc2">Data S1</a> (First, second and third charts). As explained below, the solutions to this model are very insensitive to variations in <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>3</mn></msub></mrow></math> (see <a id="crosref1075" href="#mmc2">Data S1</a>, the <i>Monte Carlo</i> analysis), so in the solutions presented here <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>3</mn></msub></mrow></math> was kept at a constant value of 0.06906, which was the best-fit parameter value for cycle 12 (<a id="crosref1080" href="#mmc2">Data S1</a>; first, second and third charts).</div></section><section id="sec-3-4-11-3"><h5>Picking the value of N, the number of phosphorylation sites:</h5><div id="p0395" role="paragraph">In the model, we assumed that Asl had to be phosphorylated by Plk4 <i>N</i> times before it switched to a low-affinity state—indicated by variables <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo>,</mo><mo>⋯</mo><mo>,</mo><mspace width="0.25em"></mspace><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math>. We tested the effect of the number of phosphorylation sites on the model solution by using N&nbsp;= 1, 4, 9, 14, or 16. The best fit curves for <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mo>⋯</mo><mo linebreak="goodbreak" linebreakstyle="after">+</mo><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow></math> suggested that the model is a good fit for the data for any value of N &gt; 4 (N&nbsp;= 1 (<i>R</i><sup><i>2</i></sup>&nbsp;= 0.9152), N&nbsp;= 4 (<i>R</i><sup><i>2</i></sup>&nbsp;= 0.9886), N&nbsp;= 9 (<i>R</i><sup><i>2</i></sup>&nbsp;= 0.9996), N&nbsp;= 14 (<i>R</i><sup><i>2</i></sup>&nbsp;= 0.9962) or N&nbsp;= 16 (<i>R</i><sup><i>2</i></sup>&nbsp;= 0.9931)). So, we use N&nbsp;= 9 corresponding to 10 phosphorylation sites of Asl (1 unbound, 9 bound with various stages of phosphorylation) in all subsequent modeling, although we note that any value above 4 works essentially equally well.</div><div id="p0400" role="paragraph"><a id="crosref1085" href="#mmc2">Data S1</a>, first chart, shows that the <i>trust-region</i> algorithm finds a very good fit (<i>R</i><sup><i>2</i></sup> &gt; 0.99) for the model to the experimental data (<a id="crosref1090" href="#fig3">Figure&nbsp;3</a>C), but this provides little information about uniqueness of the fit as there may be other subsets of the parameter space that also provide a good fit to the data. To see if any such regions could be detected, the parameter space was further explored by using a <i>Metropolis-Hastings Markov chain Monte Carlo</i> algorithm. Four Markov chains were started at the positions in the parameter space specified in Data S1 (fourth chart). The <i>Monte Carlo</i> analysis in <a id="crosref1095" href="#mmc2">Data S1</a> shows the six two-dimensional traces of the four-dimensional parameter space. For clarity, only points that provided a good fit to the cycle 12 data (<i>R</i><sup><i>2</i></sup> &gt; 0.95) are shown.</div><div id="p0405" role="paragraph">The results in <a id="crosref1100" href="#mmc2">Data S1</a> (the <i>Monte Carlo</i> analysis) reveal how sensitive the model is to changes in each parameter value. The model only fit the data well for a relatively narrow range of values for <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>1</mn></msub></mrow></math>, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow></math>, and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn></msub></mrow></math>. In contrast, the fits are mostly insensitive to <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>3</mn></msub></mrow></math>. This is likely, because the rate of phosphorylating Asl at multiple sites is relatively slow compared to the rate at which Plk4 is subsequently released from the multiply phosphorylated Asl—so the rate of release is not limiting. The Monte Carlo simulations also reveal correlations between <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>1</mn></msub></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow></math>, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>1</mn></msub></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn></msub></mrow></math>, and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn></msub></mrow></math>. For example, these results show that if <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mrow><mover accent="true"><mi>A</mi><mo>ˆ</mo></mover></mrow><mn>0</mn></msub></mrow></math> (the initial amount of unbound Asl receptor at the start of S-phase) is reduced, the model can still fit the data well if <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow></math> is decreased and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>1</mn></msub></mrow></math> is increased. While these results suggest that there is a single, continuous region of the parameter space that provides a good fit to the data, it is still possible that there are other such regions that the Markov chains in <a id="crosref1105" href="#mmc2">Data S1</a> (the <i>Monte Carlo</i> analysis) did not explore. However, the results in <a id="crosref1110" href="#mmc2">Data S1</a> (the <i>Monte Carlo</i> analysis, panel B) show that the points, which are identified at the center of the parameter region, provide the best fit to the data. This suggests the nonlinear least-squares minima found by the <i>trust-region</i> fitting is insensitive to the initial seed.</div><div id="p0410" role="paragraph">Interestingly, the best-fit parameters for cycles 11–13 showed that the biggest difference between the parameters of the Plk4 oscillations at each cycle is in <i>k</i><sub>1</sub>—the rate at which Plk4 binds to Asl (which is dependent on the cytosolic concentration of Plk4). Although our model assumes that the cytosolic concentration of Plk4 remains constant during the S-phase period within each cycle, if the phosphorylated Plk4 molecules that are released from the Asl receptor are ultimately degraded—and there is good evidence that Asl activates Plk4 to promote Plk4 degradation (<a id="crosref1115" href="#bib36" role="doc-biblioref" data-xml-rid="bib36">Klebba et&nbsp;al., 2015</a>)—there could be a wave of phosphorylated-Plk4 degradation in the cytoplasm toward the end of S-phase. If so, the cytosolic levels of Plk4 would get successively lower at the start of each successive cycle, as our PeCoS analysis indicates is the case (<a id="crosref1120" href="#fig3">Figure&nbsp;3</a>E).</div><div id="p0415" role="paragraph">The effects of reducing the genetic dose of Plk4 by half (<i>Plk4-NG</i><sup><i>1/2</i></sup> embryos—see <a id="crosref1125" href="#sec-2">Results and Discussion</a> for details) were analyzed. Our PeCoS analysis indicated that there was a ∼45% drop in the cytosolic levels of the Plk4-NG protein in the <i>Plk4-NG</i><sup><i>1/2</i></sup> embryos (<a id="crosref1130" href="#figs6">Figure&nbsp;S6</a>C). When the model was fit to <i>Plk4-NG</i><sup><i>1/2</i></sup> oscillation, the best-fit (<i>R</i><sup><i>2</i></sup>&nbsp;= 0.996) parameter had a <i>k</i><sub>1</sub> value that was ∼39% of the control value (<a id="crosref1135" href="#mmc2">Data S1</a>, second chart), so in reasonable agreement with the 45% drop in cytosolic Plk4 levels we measured experimentally. These parameter values also suggested that the total amount of centriolar Asl <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>(</mo><mrow><msub><mi>A</mi><mrow><mi>t</mi><mi>o</mi><mi>t</mi><mspace width="0.25em"></mspace></mrow></msub></mrow><mo>)</mo></mrow></mrow></math>should remain relatively unchanged between the <i>Plk4-NG</i> and <i>Plk4-NG</i><sup><i>1/2</i></sup> conditions (<a id="crosref1140" href="#mmc2">Data S1</a>, second chart). Centriolar Asl levels were analyzed in embryos expressing Asl-mCherry in either WT versus <i>Plk4</i><sup><i>1/2</i></sup> conditions, and our findings showed that this was indeed the case (<a id="crosref1145" href="#figs7">Figure&nbsp;S7</a>A).</div><div id="p0420" role="paragraph">Next, the effects of reducing the genetic dose of <i>asl</i> by half (<i>asl</i><sup><i>1/2</i></sup> embryos—see <a id="crosref1150" href="#sec-2">Results and Discussion</a> for details) were analyzed. Interestingly, the best-fit parameter values (<i>R</i><sup><i>2</i></sup>&nbsp;= 0.999) predicted that the total centriolar Asl levels (A<sub>tot</sub>) would be reduced by only ∼28% in <i>asl</i><sup><i>1/2</i></sup> embryos (<a id="crosref1155" href="#mmc2">Data S1</a>, third chart). This value was therefore directly measured in embryos expressing either one or two copies of Asl-GFP (under the control of its own promoter in an <i>asl</i> mutant background). Encouragingly our findings showed that reducing the genetic dose of Asl-GFP by half led to a reduction of only ∼30% in centriolar Asl-GFP levels (<a id="crosref1160" href="#figs7">Figure&nbsp;S7</a>B). Moreover, the parameter values suggested that the concentration of Plk4 (incorporated in the <i>k</i><sub>1</sub> term) should not vary significantly between WT and <i>asl</i><sup><i>1/2</i></sup> conditions (<a id="crosref1165" href="#mmc2">Data S1</a>, third chart). We confirmed this prediction using western blotting for Plk4-GFP and PeCoS for Plk4-NG (both transgenically expressed from their own promoters in a <i>Plk4</i> mutant background) in control and <i>asl</i><sup><i>1/2</i></sup> embryos (<a id="crosref1170" href="#figs7">Figures S7</a>C and S7D).</div><div id="p0425" role="paragraph">Taken together, these analyses indicate that our model can robustly describe the Plk4-NG oscillations under normal conditions (<a id="crosref1175" href="#fig3">Figure&nbsp;3</a>C) and when the levels of either Plk4 or Asl are perturbed experimentally (<a id="crosref1180" href="#fig4">Figures 4</a>A and 4B). Moreover, the model makes several plausible predictions about the relative levels of these proteins in the perturbed conditions that are close to the levels that we measured experimentally.</div><div id="p0430" role="paragraph">Finally, the best-fit value of <i>k</i><sub>2</sub> (reflecting the kinase activity of individual Plk4 molecules) decreased slightly between cycles 11 to 12 and decreased more significantly between cycles 12 to 13 (by ∼9% and ∼37%, respectively); <i>k</i><sub>2</sub> also decreased when levels of Plk4 were genetically reduced in <i>Plk4-NG</i><sup><i>1/2</i></sup> embryos (by ∼25%)—but not when Asl levels were genetically reduced in <i>asl</i><sup><i>1/2</i></sup> embryos (<a id="crosref1185" href="#mmc2">Data S1</a>; first, second and third charts). The molecular basis for this inferred decrease in kinase activity remains unknown, but we believe it is biologically plausible. We previously suggested that centriolar Plk4 was likely to integrate several inputs at the start of each cycle (from, for example, cell cycle regulators, or its activator Ana2/STIL) and adjust its kinase activity in response to the lengthening of S-phase during successive nuclear cycles (<a id="crosref1190" href="#bib3" role="doc-biblioref" data-xml-rid="bib3">Aydogan et&nbsp;al., 2018</a>). Moreover, our finding that Wee1 kinase, an important cell cycle regulator, can influence the Plk4 oscillation parameters in S-phase strongly supports this hypothesis (<a id="crosref1195" href="#fig6">Figure&nbsp;6</a>).</div></section></section><section id="sec-3-4-12"><h4>Model 2: Generating robust Plk4 oscillations entrained by the CCO</h4><div id="p0435" role="paragraph">The network described above was further extended to test the possibility of generating robust oscillations in centriolar Plk4 levels. To do so, we allow the Asl receptors to be dephosphorylated by a phosphatase at rate <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi mathvariant="bold-italic">k</mi><mn>4</mn></msub></mrow></math> (<a id="crosref1200" href="#figs4">Figure&nbsp;S4</a>A). Subject to the constraint that there is no Plk4 bound to the Asl receptors initially (i.e., at the start of cycle 1), we model multiple cycles by allowing the phosphatase to be active only during mitosis, so that <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi mathvariant="bold-italic">k</mi><mn>4</mn></msub></mrow></math> is nonvanishing only in this period. Therefore, the system reads:<div id="fd6" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>1</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(6)</div></div><div id="fd7" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(7)</div></div><div id="fd8" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mrow><mi>N</mi><mo linebreak="badbreak">−</mo><mn>1</mn></mrow><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>3</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(8)</div></div><div id="fd9" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>1</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>1</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(9)</div></div><div id="fd10" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>1</mn></msub></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>2</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>1</mn></msub></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(10)</div></div><div id="fd11" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msub><mi>A</mi><mi>N</mi></msub></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>3</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mi>N</mi></msub></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(11)</div></div>subject to the initial conditions,<div id="fd12" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="badbreak">=</mo><mn>1</mn><mo>;</mo><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>1</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">=</mo><mo>⋯</mo><mo linebreak="goodbreak">=</mo><mrow><mo>[</mo><mrow><msub><mi>A</mi><mi>N</mi></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">=</mo><mn>0</mn><mo>;</mo><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">=</mo><mo>⋯</mo><mo linebreak="goodbreak">=</mo><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">=</mo><mn>0</mn><mo>.</mo></mrow></math></div></div><div class="label">(12)</div></div>It is further assumed that the embryo is in mitosis for 30% of the total time in each nuclear cycle and all cycle times are kept constant. Hence, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>4</mn></msub><mo linebreak="goodbreak" linebreakstyle="after">=</mo><mn>0</mn><mspace width="0.25em"></mspace></mrow></math> for <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mn>0</mn><mo linebreak="goodbreak" linebreakstyle="after">&lt;</mo><mi>t</mi><mspace width="0.25em"></mspace><mtext>mod&nbsp;</mtext><mi>T</mi><mo linebreak="goodbreak" linebreakstyle="after">&lt;</mo><mn>0.7</mn><mi>T</mi></mrow></math> and a positive constant for <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mn>0.7</mn><mi>T</mi><mo linebreak="goodbreak" linebreakstyle="after">&lt;</mo><mi>t</mi><mspace width="0.25em"></mspace><mtext>mod&nbsp;</mtext><mi>T</mi><mo linebreak="goodbreak" linebreakstyle="after">&lt;</mo><mi>T</mi></mrow></math>, where <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>T</mi></math> is the period of the cell cycle. Values for the rate constants are determined by fitting the exact analytical solution in the S-phase of cycle 12 to the Lorentzian regression of the experimental data (<i>R</i><sup><i>2</i></sup>&nbsp;= 0.9870). In the first instance, we assumed that the cytosolic concentration of Plk4 remains constant over the nuclear cycles (see below). We plot the exact solution for the percentage of Asl-bound Plk4 molecules for a total of 14 nuclear cycles, as is the case in fly embryos. This minimal model was sufficient to generate sustained oscillations in centriolar Plk4 levels (<a id="crosref1205" href="#figs4">Figure&nbsp;S4</a>B; <i>k</i><sub><i>4</i></sub>&nbsp;= 0.0708).</div><div id="p0440" role="paragraph">As an alternative to the assumption that the cytosolic concentration of Plk4 is constant over the cycles, we also considered the case where the total number of Plk4 molecules in the embryo is kept constant (so any Plk4 degradation is balanced by new synthesis). In this model, as the number of centrioles, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>N</mi><mi>C</mi></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow></math>, increases at successive cycles so the number of available Plk4 molecules in the cytosol initially decreases during S-phase (as Plk4 binds to centriolar Asl receptors), and then increases (as Plk4 unbinds from Asl receptors). We estimate that there are <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>N</mi><mi>P</mi></msub></mrow></math>&nbsp;= 10<sup>5</sup> molecules of Plk4 in an embryo of 0.01mm<sup>3</sup> volume (<a id="crosref1210" href="#bib43" role="doc-biblioref" data-xml-rid="bib43">Markow et&nbsp;al., 2009</a>); a concentration of ∼10nM, in agreement with that measured in human cells (<a id="optmwDNFZj1az" href="#bib84" role="doc-biblioref" data-xml-rid="bib84">Yamamoto and Kitagawa, 2019</a>), but potentially higher than we infer from our observation that Plk4 levels are too low to be measured by FCS (as we cannot infer absolute protein concentration from our PeCoS experiments). To simulate the effect of centriole duplications, we double the number of centrioles each cycle and assume that, at each centriole duplication, the bound Plk4 (attached to Asl receptors) is equally split between the mother and separating daughter. To consider the Cdk/Cyclin trigger wave that sweeps through embryos (<a id="crosref1215" href="#bib19" role="doc-biblioref" data-xml-rid="bib19">Deneke et&nbsp;al., 2016</a>), it is assumed that the duplicated centrioles separate nearly synchronously over the last 10% of the time-window in each cycle. Based on our 3D-SIM microscopy data, we assume that each centriole has ∼30 Asl receptors, as we essentially only need to consider Plk4 binding to Asl at the site of centriole assembly (the model works well for between ∼20-80 receptors), and that there is a single centriole in cycle 1. With these modifications to the model, the system reads<div id="fd13" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>1</mn></msub><mrow><mo>[</mo><mi>P</mi><mo>]</mo></mrow><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><mfrac><mn>1</mn><mrow><msub><mi>N</mi><mi>C</mi></msub></mrow></mfrac><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup><mfrac><mrow><mi>d</mi><msub><mi>N</mi><mi>C</mi></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mrow></math></div></div><div class="label">(13)</div></div><div id="fd14" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><mfrac><mn>1</mn><mrow><msub><mi>N</mi><mi>C</mi></msub></mrow></mfrac><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup><mfrac><mrow><mi>d</mi><msub><mi>N</mi><mi>C</mi></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mrow></math></div></div><div class="label">(14)</div></div><div id="fd15" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>2</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mrow><mi>N</mi><mo linebreak="badbreak">−</mo><mn>1</mn></mrow><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>3</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><mfrac><mn>1</mn><mrow><msub><mi>N</mi><mi>C</mi></msub></mrow></mfrac><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup><mfrac><mrow><mi>d</mi><msub><mi>N</mi><mi>C</mi></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mrow></math></div></div><div class="label">(15)</div></div><div id="fd16" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>1</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>1</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>0</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">+</mo><mspace width="0.25em"></mspace><mfrac><mn>1</mn><mrow><msub><mi>N</mi><mi>C</mi></msub></mrow></mfrac><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup><mfrac><mrow><mi>d</mi><msub><mi>N</mi><mi>C</mi></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mrow></math></div></div><div class="label">(16)</div></div><div id="fd17" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>1</mn></msub></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>2</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mn>1</mn></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">+</mo><mspace width="0.25em"></mspace><mfrac><mn>1</mn><mrow><msub><mi>N</mi><mi>C</mi></msub></mrow></mfrac><msubsup><mi>A</mi><mn>1</mn><mo linebreak="badbreak">∗</mo></msubsup><mfrac><mrow><mi>d</mi><msub><mi>N</mi><mi>C</mi></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mrow></math></div></div><div class="label">(17)</div></div><div id="fd18" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mfrac><mrow><mi>d</mi><mrow><mo>[</mo><mrow><msub><mi>A</mi><mi>N</mi></msub></mrow><mo>]</mo></mrow></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo linebreak="badbreak">=</mo><msub><mi>k</mi><mn>3</mn></msub><mrow><mo>[</mo><mrow><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">−</mo><msub><mi>k</mi><mn>4</mn></msub><mrow><mo>[</mo><mrow><msub><mi>A</mi><mi>N</mi></msub></mrow><mo>]</mo></mrow><mo linebreak="goodbreak">+</mo><mspace width="0.25em"></mspace><mfrac><mn>1</mn><mrow><msub><mi>N</mi><mi>C</mi></msub></mrow></mfrac><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup><mfrac><mrow><mi>d</mi><msub><mi>N</mi><mi>C</mi></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mrow></math></div></div><div class="label">(18)</div></div><div id="fd19" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mi>P</mi><mo>]</mo></mrow><mo linebreak="badbreak">=</mo><mn>1</mn><mo linebreak="goodbreak">−</mo><mspace width="0.25em"></mspace><mfrac><mrow><msub><mi>N</mi><mi>R</mi></msub><msub><mi>N</mi><mi>C</mi></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow><mrow><msub><mi>N</mi><mi>P</mi></msub></mrow></mfrac><munderover><mo>∑</mo><mrow><mi>n</mi><mo linebreak="badbreak">=</mo><mn>0</mn></mrow><mi>N</mi></munderover><mrow><msubsup><mi>A</mi><mi>n</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow></mrow></math></div></div><div class="label">(19)</div></div>subject to the initial conditions (12).</div><div id="p0445" role="paragraph">We plot the solution of the model for the percentage of Asl-bound Plk4 molecules, as well as the percentage of Plk4 molecules that remain in the cytoplasm, over 14 nuclear cycles (<a id="crosref1220" href="#figs4">Figure&nbsp;S4</a>F; <i>R</i><sup><i>2</i></sup>&nbsp;= 0.9871 and <i>k</i><sub><i>4</i></sub>&nbsp;= 0.0612).</div><div id="p0450" role="paragraph">We observe a small spike as the centrioles begin to separate at the end of each mitosis (<a id="crosref1225" href="#figs4">Figure&nbsp;S4</a>F). The spike is small, due to the slight asynchrony in centriole separations; if the centrioles <i>were</i> to separate all simultaneously, the concentration would instantly halve at this point, since the number of receptors would all double. Interestingly, these small spikes are consistent with our experimental observations (<a id="crosref1230" href="#fig1">Figures 1</a>A and <a id="crosref1235" href="#figs2">S2</a>A). Moreover, we emphasize that, for the first few nuclear cycles, almost all of the Plk4 remains in the cytoplasm since there are only a few centrioles. In the later cycles, however, the amount of Plk4 sequestered by the Asl receptors increases exponentially, as the number of centrioles increase by a factor of 2 in each cycle. Therefore, the rate at which the Asl receptors are able to recruit Plk4 from the cytoplasm decreases, resulting in a reduction in the amplitude of the Plk4 oscillation (<a id="crosref1240" href="#figs4">Figure&nbsp;S4</a>F). This feature of the model is also consistent with our experimental observations (<a id="crosref1245" href="#fig1">Figures 1</a>B and 1C).</div></section><section id="sec-3-4-13"><h4>Model 3: Stochastic duplications</h4><div id="p0455" role="paragraph">Finally, we have also developed a discrete mathematical model, which is analogous to Model 2 described previously, in order to consider the possibility of stochastic centriole duplication in non-cycling embryos (as in those injected with dsRNA against cyclins A, B and B3; <a id="crosref1250" href="#figs8">Figure&nbsp;S8</a>G). As before, we assume that unphosphorylated Asl receptors bind Plk4 with high affinity until they become fully phosphorylated and Plk4 unbinds, and we initially assume that the Asl receptors can be dephosphorylated during mitosis. We model this system stochastically by defining the state vector for each receptor to be<div id="fd20" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>V</mi><mo linebreak="badbreak">=</mo><mrow><mo>(</mo><mrow><msub><mi>A</mi><mn>0</mn></msub><mo>,</mo><mspace width="0.25em"></mspace><mo>⋯</mo><mo>,</mo><mspace width="0.25em"></mspace><msub><mi>A</mi><mi>N</mi></msub><mo>,</mo><mspace width="0.25em"></mspace><msubsup><mi>A</mi><mn>0</mn><mo linebreak="badbreak">∗</mo></msubsup><mo>,</mo><mo>⋯</mo><mo>,</mo><mspace width="0.25em"></mspace><msubsup><mi>A</mi><mi>N</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>)</mo></mrow></mrow></math></div></div><div class="label">(20)</div></div>At any given time, precisely one entry of <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>V</mi></math> is equal to unity, corresponding to the state which the receptor is in at that moment, and all other entries are equal to zero. We allow the receptor to change state over time according to the transition matrix:<div id="fd21" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>M</mi><mo linebreak="badbreak">=</mo><mspace width="0.25em"></mspace><mrow><mo>[</mo><mrow><mtable><mtr><mtd><mrow><msub><mi>Q</mi><mn>1</mn></msub></mrow></mtd><mtd><mn>0</mn></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋯</mo></mtd><mtd><mn>0</mn></mtd><mtd><mrow><msub><mi>P</mi><mn>1</mn></msub></mrow></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋯</mo></mtd><mtd><mo>⋯</mo></mtd><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mrow><msub><mi>P</mi><mn>4</mn></msub></mrow></mtd><mtd><mrow><msub><mi>Q</mi><mn>4</mn></msub></mrow></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋱</mo></mtd><mtd><mn>0</mn></mtd><mtd><mn>0</mn></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋮</mo></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><mrow><msub><mi>P</mi><mn>4</mn></msub></mrow></mtd><mtd><mrow><msub><mi>Q</mi><mn>4</mn></msub></mrow></mtd><mtd><mo>⋱</mo></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋮</mo></mtd><mtd><mo>⋮</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋮</mo></mtd></mtr><mtr><mtd><mo>⋮</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋮</mo></mtd><mtd><mo>⋮</mo></mtd><mtd><mo>⋮</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋮</mo></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><mo>⋯</mo></mtd><mtd><mn>0</mn></mtd><mtd><mrow><msub><mi>P</mi><mn>4</mn></msub></mrow></mtd><mtd><mrow><msub><mi>Q</mi><mn>4</mn></msub></mrow></mtd><mtd><mn>0</mn></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋮</mo></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><mo>⋯</mo></mtd><mtd><mo>⋯</mo></mtd><mtd><mn>0</mn></mtd><mtd><mn>0</mn></mtd><mtd><mrow><msub><mi>Q</mi><mn>2</mn></msub></mrow></mtd><mtd><mrow><msub><mi>P</mi><mn>2</mn></msub></mrow></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋮</mo></mtd></mtr><mtr><mtd><mo>⋮</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋮</mo></mtd><mtd><mo>⋮</mo></mtd><mtd><mrow><msub><mi>P</mi><mrow><mn>4,2</mn></mrow></msub></mrow></mtd><mtd><mrow><msub><mi>Q</mi><mrow><mn>2,4</mn></mrow></msub></mrow></mtd><mtd><mrow><msub><mi>P</mi><mrow><mn>2,4</mn></mrow></msub></mrow></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋮</mo></mtd></mtr><mtr><mtd><mo>⋮</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋮</mo></mtd><mtd><mo>⋮</mo></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mo>⋮</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mn>0</mn></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋮</mo></mtd><mtd><mo>⋱</mo></mtd><mtd><mrow><msub><mi>P</mi><mrow><mn>4,2</mn></mrow></msub></mrow></mtd><mtd><mrow><msub><mi>Q</mi><mrow><mn>2,4</mn></mrow></msub></mrow></mtd><mtd><mrow><msub><mi>P</mi><mrow><mn>2,4</mn></mrow></msub></mrow></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><mo>⋯</mo></mtd><mtd><mo>⋯</mo></mtd><mtd><mn>0</mn></mtd><mtd><mrow><msub><mi>P</mi><mrow><mn>3,4</mn></mrow></msub></mrow></mtd><mtd><mn>0</mn></mtd><mtd><mo>⋯</mo></mtd><mtd><mrow><mspace width="0.25em"></mspace><mn>0</mn></mrow></mtd><mtd><mrow><msub><mi>P</mi><mrow><mn>4,3</mn></mrow></msub></mrow></mtd><mtd><mrow><msub><mi>Q</mi><mrow><mn>3,4</mn></mrow></msub></mrow></mtd></mtr></mtable></mrow><mo>]</mo></mrow></mrow></math></div></div><div class="label">(21)</div></div>where<div id="fd22" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>P</mi><mi>i</mi></msub><mo linebreak="badbreak">=</mo><mn>1</mn><mo linebreak="goodbreak">−</mo><msup><mi>e</mi><mrow><mo>−</mo><msub><mi>k</mi><mi>i</mi></msub></mrow></msup><mo>,</mo><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><msub><mi>P</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak">=</mo><mfrac><mrow><msub><mi>k</mi><mi>i</mi></msub></mrow><mrow><msub><mi>k</mi><mi>i</mi></msub><mo linebreak="badbreak">+</mo><msub><mi>k</mi><mi>j</mi></msub></mrow></mfrac><mrow><mo>(</mo><mrow><mn>1</mn><mo linebreak="badbreak">−</mo><msup><mi>e</mi><mrow><mo>−</mo><mrow><mo>(</mo><mrow><msub><mi>k</mi><mi>i</mi></msub><mo linebreak="badbreak">+</mo><msub><mi>k</mi><mi>j</mi></msub></mrow><mo>)</mo></mrow></mrow></msup></mrow><mo>)</mo></mrow><mo>,</mo><mspace width="0.25em"></mspace></mrow></math></div></div><div class="label">(22)</div></div><div id="fd23" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>Q</mi><mi>i</mi></msub><mo linebreak="badbreak">=</mo><msup><mi>e</mi><mrow><mo>−</mo><msub><mi>k</mi><mi>i</mi></msub></mrow></msup><mo>,</mo><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><msub><mi>Q</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak">=</mo><msup><mi>e</mi><mrow><mo>−</mo><mrow><mo>(</mo><mrow><msub><mi>k</mi><mi>i</mi></msub><mo linebreak="badbreak">+</mo><msub><mi>k</mi><mi>j</mi></msub></mrow><mo>)</mo></mrow></mrow></msup><mo>,</mo></mrow></math></div></div><div class="label">(23)</div></div>describe the probabilities of a receptor changing state and remaining in the same state which arise in our model. We may allow the cytosolic Plk4 concentration to vary in this model by making the substitution <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>k</mi><mn>1</mn></msub><mo linebreak="goodbreak" linebreakstyle="after">→</mo><mspace width="0.25em"></mspace><msub><mi>k</mi><mn>1</mn></msub><mrow><mo>[</mo><mi>P</mi><mo>]</mo></mrow></mrow></math> and using (19) to compute <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>[</mo><mi>P</mi><mo>]</mo></mrow></mrow></math> (evaluating the sum over all receptors being simulated). We also assume that, if a receptor is in a Plk4-bound state <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>(</mo><mrow><msubsup><mi>A</mi><mi>n</mi><mo linebreak="badbreak">∗</mo></msubsup></mrow><mo>)</mo></mrow></mrow></math> during the last 10% of a cycle, the Plk4 will unbind <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mrow><mo>(</mo><mrow><msubsup><mi>A</mi><mi>n</mi><mo linebreak="badbreak">∗</mo></msubsup><mo linebreak="badbreak">→</mo><mspace width="0.25em"></mspace><msub><mi>A</mi><mi>n</mi></msub></mrow><mo>)</mo></mrow></mrow></math> with 50% probability during that time period in order to simulate mother-daughter separation.</div><div id="p0460" role="paragraph">In this model, each Asl receptor behaves as an independent oscillator—alternating between a Plk4-bound form that is being phosphorylated, and an unbound form that is being dephosphorylated. In the presence of the CCO, the individual Asl receptors generate coordinated oscillations because the CCO effectively synchronizes them every cycle by ensuring a coordinated burst of PPTase activity during mitosis. This activity is lost in the absence of the CCO, but instead we allow the PPTase to be active at a low, but constant, level (10% of the mitotic activity in cycling embryos). We plot the Asl-bound Plk4 levels for a total of 10 centrioles (each with 30 receptors as assumed above; <a id="crosref1255" href="#figs8">Figure&nbsp;S8</a>H). We observe that the centrioles are initially synchronized, since they all start in an unbound state, and display a single round of Plk4 binding. However, as time progresses, the Asl receptors lose synchrony, and each centriole exhibits stochastic, low-amplitude oscillations. Such oscillations may be sufficient to trigger duplications at individual centrioles, as evident from our experimental observations (<a id="crosref1260" href="#fig5">Figures 5</a>B–5F and <a id="crosref1265" href="#figs8">S8</a>B–S8F; <a id="crosref1270" href="#mmc6">Video S4</a>).</div><div id="p0465" role="paragraph">All the equations used for mathematical modeling and regressions are available in the following web link:&nbsp;&lt; <a id="intref0085" href="https://github.com/RaffLab/centriole_oscillator_model" target="_blank">https://github.com/RaffLab/centriole_oscillator_model</a> &gt;.</div></section><section id="sec-3-4-14"><h4>Fluorescence Correlation Spectroscopy (FCS)</h4><section id="sec-3-4-14-1"><h5>FCS setup and measurements</h5><div id="p0470" role="paragraph">Point FCS measurements were performed on a confocal Zeiss LSM 880 (Argon laser excitation at 488&nbsp;nm and GaASP detector) with the Zen Black Software. A C-Apochromat 40x/1.2&nbsp;W objective and a pinhole setting of 1AU were used. A laser power of 10&nbsp;μW was used, and no photobleaching was observed during the measurements. The microscope was kept at 25°C using the Zeiss inbuilt heating insert P and the heating unit XL. A schematic overview of the methodology used is shown in <a id="crosref1275" href="#figs5">Figure&nbsp;S5</a>A, and a comparison of the average autocorrelation curves generated at the start of S-phase of nuclear cycles 11-14 is shown in <a id="crosref1280" href="#figs5">Figure&nbsp;S5</a>B.</div><div id="p0475" role="paragraph">The effective volume of the imaging setup was estimated to be 0.28 fL by averaging the estimate obtained by three independent methods, as described previously (<a id="crosref1285" href="#bib62" role="doc-biblioref" data-xml-rid="bib62">Rüttinger et&nbsp;al., 2008</a>): <b>1)</b> Measuring the concentration of a soluble Alexa Fluor 488 NHS Ester dilution series (100&nbsp;nM, 10&nbsp;nM, 1&nbsp;nM and 0.1&nbsp;nM); <b>2)</b> Measuring the diffusion time for Alexa Fluor 488 NHS Ester (same concentrations) in water at 25°C. The measured diffusion time was then compared to a previously reported diffusion coefficient for the Alexa Fluor 488 NHS Ester (<a id="crosref1290" href="#bib58" role="doc-biblioref" data-xml-rid="bib58">Petrásek and Schwille, 2008</a>); <b>3)</b> Imaging subresolution beads (FluoSpheres Carboxylate-Modified Microspheres, 0.1&nbsp;μm) and determining the effective volume via Gaussian fitting with the <i>line tool</i> and <i>Z-axis profile</i> in ImageJ (Bethasda, USA).</div><div id="p0480" role="paragraph">Embryo collections (from mother flies expressing Asl-GFP under the control of its own promoter in an <i>asl</i> mutant background) were as described above, with the exception of using high precision 35&nbsp;mm, high Glass Bottom μ-dishes (ibidi). Before every measurement, spherical aberrations were adjusted on the correction collar of the objective by maximizing the count-rate per molecule (CPM). At the beginning of S-phase in each cell cycle (when the old and new mother centrioles were separating), consecutive cytosolic measurements were made 6x for 10&nbsp;s each at the centriolar plane of the embryo. Individual recordings where centrioles moved through the measurement spot, based on the highly erratic shape of the correlation curve (&lt; 2% of all recordings), were discarded.</div></section><section id="sec-3-4-14-2"><h5>Autocorrelation analysis and post-acquisition curve fitting</h5><div id="p0485" role="paragraph">The autocorrelation function, G(τ), was calculated during each measurement in the Zen Black software using the following equation:<div id="ufd5" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mi>G</mi><mfenced><mi>τ</mi></mfenced><mo>=</mo><mfrac><mfenced close=">" open="<"><mrow><mi>δ</mi><mi>I</mi><mfenced><mi>t</mi></mfenced><mo>·</mo><mi>δ</mi><mi>I</mi><mfenced><mrow><mi>t</mi><mo>+</mo><mi>τ</mi></mrow></mfenced></mrow></mfenced><mfenced close=">" open="<"><mrow><mi>δ</mi><mi>I</mi><msup><mfenced><mi>t</mi></mfenced><mn>2</mn></msup></mrow></mfenced></mfrac></math></div></div></div>where <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mo>⟨</mo><mo>⟩</mo></math> denotes a time average, <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>δ</mi><mi>I</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow></math> describes the intensity fluctuation at the time point <i>t</i>, and τ states the lag time of the autocorrelation.</div><div id="p0490" role="paragraph">All 10 s-recordings were then fitted with 8 different 3D diffusion models using the software FoCuS-Point (<a id="crosref1295" href="#bib79" role="doc-biblioref" data-xml-rid="bib79">Waithe et&nbsp;al., 2016</a>) with the following equation:<div id="ufd6" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><msub><mi>G</mi><mrow><mn>3</mn><mi>D</mi></mrow></msub><mfenced><mi>τ</mi></mfenced><mo>=</mo><mstyle displaystyle="false"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><msub><mi>D</mi><mi>s</mi></msub></munderover></mstyle><msub><mi>A</mi><mi>k</mi></msub><msup><mfenced><mrow><mn>1</mn><mo>+</mo><msup><mfenced><mfrac><mi>τ</mi><msub><mi>τ</mi><mrow><mi>x</mi><mi>y</mi><mi>k</mi></mrow></msub></mfrac></mfenced><msup><mi>α</mi><mi>k</mi></msup></msup></mrow></mfenced><mrow><mo>−</mo><mn>1</mn></mrow></msup><msup><mfenced><mrow><mn>1</mn><mo>+</mo><mfenced><mfrac><mi>τ</mi><mrow><mi>A</mi><msup><mi>R</mi><mn>2</mn></msup><msub><mi>τ</mi><mrow><mi>x</mi><mi>y</mi><mi>k</mi></mrow></msub></mrow></mfrac></mfenced></mrow></mfenced><mrow><mo>−</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup></math></div></div></div>where <i>A</i><sub><i>k</i></sub> defines the fraction of a diffusing species for which the sum of all diffusing species equals 1, τ<sub><i>xy</i></sub> describes the average residence time of the diffusing species in <i>V</i><sub><i>eff</i></sub>, α accounts for anomalous subdiffusion within the cytoplasm, and <i>AR</i> is a structural parameter that describes the relationship among the x, y and z-axes of the excitation volume.</div><div id="p0495" role="paragraph">Dark states of the fluorophore were fitted with the following formula:<div id="ufd7" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>G</mi><mi>T</mi></msub><mrow><mo>(</mo><mi>τ</mi><mo>)</mo></mrow><mo linebreak="badbreak">=</mo><mn>1</mn><mo linebreak="goodbreak">+</mo><msubsup><mo>∑</mo><mrow><mi>j</mi><mo linebreak="badbreak">=</mo><mn>1</mn></mrow><mrow><msub><mi>T</mi><mi>s</mi></msub></mrow></msubsup><mrow><mfrac><mrow><msub><mi>T</mi><mi>j</mi></msub></mrow><mrow><mn>1</mn><mo linebreak="badbreak">−</mo><msub><mi>T</mi><mi>j</mi></msub></mrow></mfrac><mo>·</mo><msup><mi>e</mi><mrow><mrow><mrow><mo>−</mo><mi>τ</mi></mrow><mo linebreak="badbreak">/</mo><mrow><msub><mi>τ</mi><mrow><mi>T</mi><mi>j</mi></mrow></msub></mrow></mrow></mrow></msup></mrow></mrow></math></div></div></div>where <i>T</i> depicts the triplet population, and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msub><mi>τ</mi><mi>T</mi></msub></mrow></math> states the triplet correlation time during which the fluorophore stays in the dark state (<a id="crosref1300" href="#bib63" role="doc-biblioref" data-xml-rid="bib63">Schönle et&nbsp;al., 2014</a>).</div><div id="p0500" role="paragraph">The data was fitted within the boundaries of 4x10<sup>−4</sup> ms and 1.5x10<sup>3</sup> ms, and the dark states were restricted to 10-300&nbsp;μs for the blinking state, and 1-10&nbsp;μs for the triplet state. The models (<b>Ms</b>) were defined as the following: <b>M1)</b> 1 diffusing species (ds) 0 blinking states (bs) 0 triplet states (ts); <b>M2)</b> 1 ds 1bs 0ts; <b>M3)</b> 1 ds 0bs 1ts; <b>M4)</b> 1 ds 1bs 1ts; <b>M5)</b> 2 ds 0bs 0ts; <b>M6)</b> 2 ds 1bs 0ts; <b>M7)</b> 2 ds 0bs 1ts; <b>M8)</b> 2 ds 1bs 1ts. In all models, the structural parameter <i>AR</i> and the anomalous subdiffusion parameter α were kept constant at 5 and 0.7, respectively.</div><div id="p0505" role="paragraph">In order to avoid over-fitting the data, the most plausible model to describe the autocorrelation functions was selected using the Bayesian Information Criterion (BIC), which is based on the likelihood function, but introduces a penalty term for the complexity (number of variables) for the models (<a id="crosref1305" href="#bib64" role="doc-biblioref" data-xml-rid="bib64">Schwarz, 1978</a>). In this study, <b>M4</b> was the preferred model to describe Asl-GFP diffusion (<a id="crosref1310" href="#figs5">Figure&nbsp;S5</a>A<sub>(iv)</sub>). The concentration was calculated from the FoCuS-point fit data of the preferred model:<div id="ufd8" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mfenced close=">" open="<"><mi>N</mi></mfenced><mo>=</mo><mfrac><mn>1</mn><msub><mi>G</mi><mn>0</mn></msub></mfrac><mspace width="0.25em"></mspace><mo>,</mo><mspace width="0.25em"></mspace><mi>c</mi><mi>o</mi><mi>n</mi><mi>c</mi><mo>.</mo><mo>=</mo><mfrac><mfenced close=">" open="<"><mi>N</mi></mfenced><msub><mi>V</mi><mrow><mi>e</mi><mi>f</mi><mi>f</mi></mrow></msub></mfrac></math></div></div></div>where <i>N</i> states the average number of particles within the effective volume <i>V</i><sub><i>eff</i></sub>, and <i>G</i><sub><i>0</i></sub> represents the height of the autocorrelation function at τ&nbsp;= 0.</div></section></section><section id="sec-3-4-15"><h4>FCS background corrections</h4><div id="p0510" role="paragraph">In order to estimate the contribution of the background noise, 22 wild-type embryos were measured with the same laser intensity (10&nbsp;μW) and in roughly the same plane and developmental stage as the Asl-GFP embryos. Despite no observable correlated background, the uncorrelated background contributed ∼30% of the total photon count rate, presumably due to the low concentration of cytosolic Asl-GFP and the high autofluorescence of the embryo itself (<a id="crosref1315" href="#figs5">Figure&nbsp;S5</a><sub>(v</sub><sub>)</sub>). Background corrections were performed after the autocorrelation analysis by calculating the correction factor <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><msup><mi>χ</mi><mn>2</mn></msup><mspace width="0.25em"></mspace></mrow></math>using the following formula (<a id="crosref1320" href="#bib37" role="doc-biblioref" data-xml-rid="bib37">Koppel, 1974</a>):<div id="ufd9" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mfrac><mn>1</mn><msup><mi>χ</mi><mn>2</mn></msup></mfrac><mo>=</mo><mfrac><mn>1</mn><msup><mfenced><mrow><mn>1</mn><mo>+</mo><mfenced close=">" open="<"><mi>b</mi></mfenced><mo>/</mo><mfenced close=">" open="<"><mi>f</mi></mfenced></mrow></mfenced><mn>2</mn></msup></mfrac></math></div></div></div><div id="ufd10" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mfenced close=">" open="<"><mi>N</mi></mfenced><mo>=</mo><mfrac><mn>1</mn><mrow><msup><mi>χ</mi><mn>2</mn></msup><msub><mi>G</mi><mn>0</mn></msub></mrow></mfrac></math></div></div></div>where <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mfenced close=">" open="<"><mi>b</mi></mfenced></math> denotes the average background and <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mfenced close=">" open="<"><mi>f</mi></mfenced><mspace width="0.25em"></mspace></math>states the average count rate of the sample.</div></section><section id="sec-3-4-16"><h4>Data restriction</h4><div id="p0515" role="paragraph">In some FCS measurements a sudden drop in CPM was observed, possibly due to movements within the embryo or the embryo drifting away from the measurement plane. When this happened, a strong, often unreasonable increase in concentration was observed. These outliers were therefore discarded based on a ROUT outlier test (with the aggression factor Q&nbsp;= 1%), which was performed on all 10 s-long concentration measurements (the <i>red</i> data points in <a id="crosref1325" href="#figs5">Figure&nbsp;S5</a><sub>(vi)</sub>). Only the embryos with at least 4x10 s recordings (after discarding outliers and erratically shaped ACFs) were included in the final analysis.</div></section><section id="sec-3-4-17"><h4>Peak Counting Spectroscopy (PeCoS)</h4><div id="p0520" role="paragraph">FCS was not sensitive enough to investigate the cytosolic concentration of Plk4-NG, presumably because its concentration was too low. We therefore developed a new method that we term <i>Peak Counting Spectroscopy</i> (PeCoS) that allows the relative concentration of low abundance proteins to be measured accurately (<a id="crosref1330" href="#figs6">Figure&nbsp;S6</a>). PeCoS uses the same set up as the point FCS protocol described above, but it differs in terms of its data acquisition and analysis. In PeCoS, the intensity peaks, which are generated by a fluorophore moving through the effective volume, are counted as a proxy for concentration. Due to the low cytosolic concentration of the fluorescently-tagged protein of interest (e.g., Plk4-NG in this study), spherical aberrations could not be corrected within the same embryo where the measurements were taken. Therefore, embryos that express a bright fluorescent centriolar marker were positioned next to the experimental embryos on the same imaging dish and these were used for correction collar adjustment (<a id="crosref1335" href="#figs6">Figure&nbsp;S6</a>A<sub>(i)</sub>). Experimental recordings were then captured for 180&nbsp;s (instead of 6x10 s), as the number of particles that pass through the field of view was usually very low. Before every measurement, the observation region was pre-bleached with the same laser intensity (10&nbsp;μW) for 3&nbsp;s to bleach away any potential immobile fraction.</div><div id="p0525" role="paragraph">Instead of autocorrelation analysis, the resulting intensity traces (<a id="crosref1340" href="#figs6">Figure&nbsp;S6</a>A<sub>(iii)</sub>) were quantified for their number of peaks, which originate from a fluorophore moving through the excitation volume and causing a detectable burst of photons. In order to determine the cut-off threshold, which was used to subtract the background noise, 40 control embryos were measured (<a id="crosref1345" href="#figs6">Figure&nbsp;S6</a>A<sub>(iii)</sub>). These control embryos were from mothers expressing Asl-mKate2 to allow measurements at the centriolar plane and at the right nuclear cycle stage (beginning of S-phase). “Mean&nbsp;+ n<sup>∗</sup>SD” (where n&nbsp;= 1,2,3,…) of all control recordings was subtracted from each control recording, and the threshold that resulted in an average of less than five peaks (per 180&nbsp;s control measurement) was subtracted from all intensity traces (<a id="crosref1350" href="#figs6">Figure&nbsp;S6</a>A<sub>(iv-vi)</sub>). This threshold was found to be a good compromise for minimizing the background noise without discarding too much information. A Python script was written in order to automate this procedure, which is available via&nbsp;&lt; <a id="intref0090" href="https://github.com/RaffLab/PeCoS" target="_blank">https://github.com/RaffLab/PeCoS</a> &gt;. The subtraction of “Mean&nbsp;+ 8<sup>∗</sup>SD” resulted in an average peak count of 3.25 for the control recordings, and this was used for the background subtraction in all <i>in&nbsp;vivo</i> measurements. In the peak detection algorithm above, a peak was defined as any consecutive value (photon count) that surpasses the subtracted threshold (<a id="crosref1355" href="#figs6">Figure&nbsp;S6</a>A<sub>(vi)</sub>).</div><div id="p0530" role="paragraph">In order to assess the effective concentration range of the PeCoS methodology, two-fold dilution series of Alexa488 NHS Ester were measured, and for every sample, both the ACF and the number of peaks were calculated using FCS and PeCoS (where Background&nbsp;= Mean ± 23<sup>∗</sup>SD (water)), respectively. As expected, PeCoS did not perform well at high concentrations, where, presumably, too many particles move simultaneously through the excitation volume; at lower particle concentrations, however, (where FCS was no longer accurate) the number of peaks decreased in a nearly linear fashion (<a id="crosref1360" href="#figs6">Figure&nbsp;S6</a>B). To test the sensitivity of PeCoS under <i>in&nbsp;vivo</i> conditions, the cytosolic concentration of Plk4-NG was measured at the beginning of nuclear cycle 12 in embryos expressing either one (1x) or two (2x) copies of <i>Plk4-NG</i> in the <i>Plk4</i> mutant background. PeCoS analysis indicated a 90% increase in the number of peaks (per minute) in the 2x embryos compared to the 1x embryos, indicating the effectiveness of PeCoS in measuring the relative cytosolic concentration of low abundance proteins (<a id="crosref1365" href="#figs6">Figure&nbsp;S6</a>C).</div></section></section><section id="sec-3-5"><h3>Quantification and Statistical Analysis</h3><div id="p0535" role="paragraph">The details for quantification, statistical tests, sample numbers, definitions of center, and the measures for dispersion and precision are described in the main text, relevant figure legends, or relevant sections of <a id="crosref1370" href="#sec-3">STAR Methods</a>. Significance in statistical tests was defined by p&nbsp;&lt; 0.05. To determine whether the data values were normally distributed, a D’Agostino–Pearson omnibus normality test was applied. Prism 7 and 8 were used for all the modeling and statistical analyses.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0540" role="paragraph">We are grateful to Laura Hankins, Fabio Echegaray Iturra, Marjorie Fournier, Christoffer Lagerholm, and Bela Novak for advice and discussion and Alissa M. Kleinnijenhuis and members of the Raff laboratory for critically reading the manuscript. Microscopy was performed at the Micron Oxford Advanced Bioimaging Unit, funded by a Strategic Award from the Wellcome Trust (107457). The research was funded by a Wellcome Trust Senior Investigator Award (104575 to T.L.S., M.M., Z.M.W., A.W., S.S., and Z.A.N.), Edward Penley Abraham Scholarships (to M.G.A. and L.G.), a Cancer Research UK Oxford Centre Prize DPhil Studentship (C5255/A23225 to S.-S.W.), a Balliol Jason Hu Scholarship, (to S.-S.W.), a Clarendon Scholarship (to S.-S.W.), and Ludwig Institute for Cancer Research funding (to F.Y.Z.). M.A.B. was supported by a Biotechnology and Biological Sciences Research Council grant (BB/N016858/1) and the St. Cross Emanoel Lee Junior Research Fellowship.</div><section id="sec-4"><h3>Author Contributions</h3><div id="p0545" role="paragraph">This study was conceptualized by M.G.A., T.L.S., M.A.B., and J.W.R. Investigation was done by M.G.A., T.L.S., M.M., Z.M.W., L.G., A.W., S.S., and M.A.B. Data were analyzed by M.G.A., T.L.S., Z.M.W., L.G., F.Y.Z., and M.B.A. Methodology was developed by M.G.A., T.L.S., M.M., Z.M.W., S.-S.W., F.Y.Z., M.A.B., and J.W.R. Project was administered by M.G.A., M.A.B., and J.W.R. Resources were shared/made by M.G.A., M.M., L.G., A.W., S.S., S.-S.W., Z.A.N., and M.A.B. Software development was carried out by M.G.A., T.L.S., Z.M.W., S.-S.W., F.Y.Z., and M.A.B. Overall supervision was done by M.G.A., A.G., and J.W.R. Validation experiments/analyses were carried out by M.G.A., A.W., S.S., and J.W.R. M.G.A., T.L.S., A.G., M.A.B., and J.W.R. wrote and edited the draft with significant input from all authors.</div></section><section id="sec-5"><h3>Declaration of Interests</h3><div id="p0550" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (2)</h2><div id="p0560" role="paragraph"><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.05.018/attachment/c43bc122-eda1-4fed-a1fc-f420caa4baf8/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (10.77 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Oligonucleotides Used in This Study, Related to Figures 1, 2, 3, 4, 5, and 6 and STAR Methods</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.05.018/attachment/883d55f9-02d8-4970-8e73-a86d925571c7/mmc2.pdf" download="mmc2.pdf" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">PDF (482.41 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Data S1. Parameter Values and Monte Carlo Analysis Used in the Mathematical Modeling, Related to Figures 3, 4A, and 4B and STAR Methods</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label">1.</div><div>Alvarez-Rodrigo, I. ∙ Steinacker, T.L. ∙ Saurya, S. ...</div><div><strong>Evidence that a positive feedback loop drives centrosome maturation in fly embryos</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:D430</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.50130" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85071968973" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.50130" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label">2.</div><div>Arquint, C. ∙ Nigg, E.A.</div><div><strong>The PLK4-STIL-SAS-6 module at the core of centriole duplication</strong></div><div><em>Biochem. Soc. Trans.</em> 2016; <strong>44</strong>:1253-1263</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/BST20160116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85009759086" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27911707/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2FBST20160116&amp;pmid=27911707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label">3.</div><div>Aydogan, M.G. ∙ Wainman, A. ∙ Saurya, S. ...</div><div><strong>A homeostatic clock sets daughter centriole size in flies</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2018; <strong>217</strong>:1233-1248</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201801014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85044785786" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29500190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201801014&amp;pmid=29500190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label">4.</div><div>Balczon, R. ∙ Bao, L. ∙ Zimmer, W.E. ...</div><div><strong>Dissociation of centrosome replication events from cycles of DNA synthesis and mitotic division in hydroxyurea-arrested Chinese hamster ovary cells</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1995; <strong>130</strong>:105-115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.130.1.105" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0029027586" target="_blank" aria-label="Open Scopus in new tab">Scopus (267)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7790366/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.130.1.105&amp;pmid=7790366" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label">5.</div><div>Ball, G. ∙ Demmerle, J. ∙ Kaufmann, R. ...</div><div><strong>SIMcheck: a Toolbox for Successful Super-resolution Structured Illumination Microscopy</strong></div><div><em>Sci. Rep.</em> 2015; <strong>5</strong>:15915</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep15915" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84946607023" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26525406/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep15915&amp;pmid=26525406" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label">6.</div><div>Banterle, N. ∙ Gönczy, P.</div><div><strong>Centriole Biogenesis: From Identifying the Characters to Understanding the Plot</strong></div><div><em>Annu. Rev. Cell Dev. Biol.</em> 2017; <strong>33</strong>:23-49</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-cellbio-100616-060454" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85030988640" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28813178/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-cellbio-100616-060454&amp;pmid=28813178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label">7.</div><div>Basto, R. ∙ Brunk, K. ∙ Vinadogrova, T. ...</div><div><strong>Centrosome amplification can initiate tumorigenesis in flies</strong></div><div><em>Cell.</em> 2008; <strong>133</strong>:1032-1042</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2008.05.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2008.05.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-44649117902" target="_blank" aria-label="Open Scopus in new tab">Scopus (435)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18555779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2008.05.039&amp;pmid=18555779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label">8.</div><div>Baumbach, J. ∙ Novak, Z.A. ∙ Raff, J.W. ...</div><div><strong>Dissecting the function and assembly of acentriolar microtubule organizing centers in Drosophila cells in&nbsp;vivo</strong></div><div><em>PLoS Genet.</em> 2015; <strong>11</strong>:e1005261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1005261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84930795923" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26020779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1005261&amp;pmid=26020779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label">9.</div><div>Bettencourt-Dias, M. ∙ Hildebrandt, F. ∙ Pellman, D. ...</div><div><strong>Centrosomes and cilia in human disease</strong></div><div><em>Trends Genet.</em> 2011; <strong>27</strong>:307-315</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.tig.2011.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.tig.2011.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-79960629889" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21680046/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tig.2011.05.004&amp;pmid=21680046" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label">10.</div><div>Blachon, S. ∙ Gopalakrishnan, J. ∙ Omori, Y. ...</div><div><strong>Drosophila asterless and vertebrate Cep152 Are orthologs essential for centriole duplication</strong></div><div><em>Genetics.</em> 2008; <strong>180</strong>:2081-2094</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.108.095141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-61849092578" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18854586/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.108.095141&amp;pmid=18854586" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label">11.</div><div>Boese, C.J. ∙ Nye, J. ∙ Buster, D.W. ...</div><div><strong>Asterless is a Polo-like kinase 4 substrate that both activates and inhibits kinase activity depending on its phosphorylation state</strong></div><div><em>Mol. Biol. Cell.</em> 2018; <strong>29</strong>:2874-2886</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E18-07-0445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85056330129" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30256714/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E18-07-0445&amp;pmid=30256714" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label">12.</div><div>Chang, J.B. ∙ Ferrell, Jr., J.E.</div><div><strong>Mitotic trigger waves and the spatial coordination of the Xenopus cell cycle</strong></div><div><em>Nature.</em> 2013; <strong>500</strong>:603-607</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12321" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84883135332" target="_blank" aria-label="Open Scopus in new tab">Scopus (131)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23863935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12321&amp;pmid=23863935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label">13.</div><div>Chen, F. ∙ Archambault, V. ∙ Kar, A. ...</div><div><strong>Multiple protein phosphatases are required for mitosis in Drosophila</strong></div><div><em>Curr. Biol.</em> 2007; <strong>17</strong>:293-303</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2007.01.068&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2007.01.068&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-33846943656" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17306545/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2007.01.068&amp;pmid=17306545" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label">14.</div><div>Cizmecioglu, O. ∙ Arnold, M. ∙ Bahtz, R. ...</div><div><strong>Cep152 acts as a scaffold for recruitment of Plk4 and CPAP to the centrosome</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2010; <strong>191</strong>:731-739</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201007107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-78349263512" target="_blank" aria-label="Open Scopus in new tab">Scopus (219)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21059844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201007107&amp;pmid=21059844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label">15.</div><div>Claude, A.</div><div><strong>The constitution of protoplasm</strong></div><div><em>Science.</em> 1943; <strong>97</strong>:451-456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.97.2525.451" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-12644292519" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17789864/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.97.2525.451&amp;pmid=17789864" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label">16.</div><div>Conduit, P.T. ∙ Wainman, A. ∙ Novak, Z.A. ...</div><div><strong>Re-examining the role of Drosophila Sas-4 in centrosome assembly using two-colour-3D-SIM FRAP</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:1032</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.08483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84949934080" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.08483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label">17.</div><div>Cunha-Ferreira, I. ∙ Rodrigues-Martins, A. ∙ Bento, I. ...</div><div><strong>The SCF/Slimb ubiquitin ligase limits centrosome amplification through degradation of SAK/PLK4</strong></div><div><em>Curr. Biol.</em> 2009; <strong>19</strong>:43-49</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2008.11.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2008.11.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-58149159563" target="_blank" aria-label="Open Scopus in new tab">Scopus (195)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19084407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2008.11.037&amp;pmid=19084407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label">18.</div><div>Cunha-Ferreira, I. ∙ Bento, I. ∙ Pimenta-Marques, A. ...</div><div><strong>Regulation of autophosphorylation controls PLK4 self-destruction and centriole number</strong></div><div><em>Curr. Biol.</em> 2013; <strong>23</strong>:2245-2254</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2013.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2013.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84889103130" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24184099/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2013.09.037&amp;pmid=24184099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label">19.</div><div>Deneke, V.E. ∙ Melbinger, A. ∙ Vergassola, M. ...</div><div><strong>Waves of Cdk1 Activity in S Phase Synchronize the Cell Cycle in Drosophila Embryos</strong></div><div><em>Dev. Cell.</em> 2016; <strong>38</strong>:399-412</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.devcel.2016.07.023&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.devcel.2016.07.023&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84995477433" target="_blank" aria-label="Open Scopus in new tab">Scopus (96)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27554859/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2016.07.023&amp;pmid=27554859" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label">20.</div><div>Dzhindzhev, N.S. ∙ Yu, Q.D. ∙ Weiskopf, K. ...</div><div><strong>Asterless is a scaffold for the onset of centriole assembly</strong></div><div><em>Nature.</em> 2010; <strong>467</strong>:714-718</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-77957982182" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20852615/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09445&amp;pmid=20852615" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="optdSTitH6neN" class="citation"><div class="citation-content"><div class="label">21.</div><div>Dobbelaere, J. ∙ Josué, F. ∙ Suijkerbuijk, S. ...</div><div><strong>A genome-wide RNAi screen to dissect centriole duplication and centrosome maturation in Drosophila</strong></div><div><em>PloS. Biol.</em> 2008; <strong>6</strong>:e224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0060224" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-54749133489" target="_blank" aria-label="Open Scopus in new tab">Scopus (189)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798690/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0060224&amp;pmid=18798690" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label">22.</div><div>Dzhindzhev, N.S. ∙ Tzolovsky, G. ∙ Lipinszki, Z. ...</div><div><strong>Plk4 phosphorylates Ana2 to trigger Sas6 recruitment and procentriole formation</strong></div><div><em>Curr. Biol.</em> 2014; <strong>24</strong>:2526-2532</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2014.08.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2014.08.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84913582254" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25264260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2014.08.061&amp;pmid=25264260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label">23.</div><div>Eilers, P.H. ∙ Boelens, H.F.</div><div><strong>Baseline Correction with Asymmetric Least Squares Smoothing. Leiden University Medical Centre report, 2005</strong></div><div>2005</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.H.EilersH.F.BoelensBaseline+Correction+with+Asymmetric+Least+Squares+Smoothing.+Leiden+University+Medical+Centre+report%2C+20052005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label">24.</div><div>Ferrell, Jr., J.E.</div><div><strong>Perfect and Near-Perfect Adaptation in Cell Signaling</strong></div><div><em>Cell Syst.</em> 2016; <strong>2</strong>:62-67</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cels.2016.02.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cels.2016.02.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84959495218" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2016.02.006&amp;pmid=27135159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label">25.</div><div>Fırat-Karalar, E.N. ∙ Stearns, T.</div><div><strong>The centriole duplication cycle</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2014; <strong>369</strong>:20130460</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2013.0460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84904562135" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25047614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2013.0460&amp;pmid=25047614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label">26.</div><div>Friedman, S.L.</div><div><strong>Molecular regulation of hepatic fibrosis, an integrated cellular response to tissue injury</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2000; <strong>275</strong>:2247-2250</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1074%2Fjbc.275.4.2247&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1074%2Fjbc.275.4.2247&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0034723290" target="_blank" aria-label="Open Scopus in new tab">Scopus (1937)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10644669/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.275.4.2247&amp;pmid=10644669" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label">27.</div><div>Gard, D.L. ∙ Hafezi, S. ∙ Zhang, T. ...</div><div><strong>Centrosome duplication continues in cycloheximide-treated Xenopus blastulae in the absence of a detectable cell cycle</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1990; <strong>110</strong>:2033-2042</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.110.6.2033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0025371941" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2190990/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.110.6.2033&amp;pmid=2190990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label">28.</div><div>Goehring, N.W. ∙ Hyman, A.A.</div><div><strong>Organelle growth control through limiting pools of cytoplasmic components</strong></div><div><em>Curr. Biol.</em> 2012; <strong>22</strong>:R330-R339</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2012.03.046&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2012.03.046&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84860777508" target="_blank" aria-label="Open Scopus in new tab">Scopus (155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22575475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2012.03.046&amp;pmid=22575475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label">29.</div><div>Gönczy, P. ∙ Hatzopoulos, G.N.</div><div><strong>Centriole assembly at a glance</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2019; <strong>132</strong>:jcs228833</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.228833" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85061997983" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787112/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.228833&amp;pmid=30787112" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label">30.</div><div>Guderian, G. ∙ Westendorf, J. ∙ Uldschmid, A. ...</div><div><strong>Plk4 trans-autophosphorylation regulates centriole number by controlling betaTrCP-mediated degradation</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2010; <strong>123</strong>:2163-2169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.068502" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-77954354411" target="_blank" aria-label="Open Scopus in new tab">Scopus (163)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20516151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.068502&amp;pmid=20516151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label">31.</div><div>Hatch, E.M. ∙ Kulukian, A. ∙ Holland, A.J. ...</div><div><strong>Cep152 interacts with Plk4 and is required for centriole duplication</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2010; <strong>191</strong>:721-729</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201006049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-78349243322" target="_blank" aria-label="Open Scopus in new tab">Scopus (219)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21059850/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201006049&amp;pmid=21059850" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label">32.</div><div>Holland, A.J. ∙ Lan, W. ∙ Niessen, S. ...</div><div><strong>Polo-like kinase 4 kinase activity limits centrosome overduplication by autoregulating its own stability</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2010; <strong>188</strong>:191-198</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.200911102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-76149140089" target="_blank" aria-label="Open Scopus in new tab">Scopus (220)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20100909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.200911102&amp;pmid=20100909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label">33.</div><div>Jacobs, H.W. ∙ Knoblich, J.A. ∙ Lehner, C.F.</div><div><strong>Drosophila Cyclin B3 is required for female fertility and is dispensable for mitosis like Cyclin B</strong></div><div><em>Genes Dev.</em> 1998; <strong>12</strong>:3741-3751</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.12.23.3741" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0005203182" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9851980/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.12.23.3741&amp;pmid=9851980" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label">34.</div><div>Jones, E. ∙ Oliphant, T. ∙ Peterson, P.</div><div><strong>SciPy: Open Source Scientific Tools for Python</strong></div><div>2001</div><div><a id="intref0100" href="http://www.scipy.org/" target="_blank">http://www.scipy.org/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.JonesT.OliphantP.PetersonSciPy%3A+Open+Source+Scientific+Tools+for+Python2001http%3A%2F%2Fwww.scipy.org%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label">35.</div><div>Kitagawa, D. ∙ Vakonakis, I. ∙ Olieric, N. ...</div><div><strong>Structural basis of the 9-fold symmetry of centrioles</strong></div><div><em>Cell.</em> 2011; <strong>144</strong>:364-375</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2011.01.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2011.01.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-79651473154" target="_blank" aria-label="Open Scopus in new tab">Scopus (274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21277013/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2011.01.008&amp;pmid=21277013" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label">36.</div><div>Klebba, J.E. ∙ Buster, D.W. ∙ Nguyen, A.L. ...</div><div><strong>Polo-like kinase 4 autodestructs by generating its Slimb-binding phosphodegron</strong></div><div><em>Curr. Biol.</em> 2013; <strong>23</strong>:2255-2261</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2013.09.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2013.09.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84889084229" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24184097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2013.09.019&amp;pmid=24184097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label">37.</div><div>Klebba, J.E. ∙ Galletta, B.J. ∙ Nye, J. ...</div><div><strong>Two Polo-like kinase 4 binding domains in Asterless perform distinct roles in regulating kinase stability</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2015; <strong>208</strong>:401-414</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201410105" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84924664113" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25688134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201410105&amp;pmid=25688134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label">38.</div><div>Koppel, D.E.</div><div><strong>Statistical accuracy in fluorescence correlation spectroscopy</strong></div><div><em>Phys. Rev. A.</em> 1974; <strong>10</strong>:1938-1945</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevA.10.1938" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0000397420" target="_blank" aria-label="Open Scopus in new tab">Scopus (390)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevA.10.1938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label">39.</div><div>Kratz, A.-S. ∙ Bärenz, F. ∙ Richter, K.T. ...</div><div><strong>Plk4-dependent phosphorylation of STIL is required for centriole duplication</strong></div><div><em>Biol. Open.</em> 2015; <strong>4</strong>:370-377</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/bio.201411023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84979691996" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25701666/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fbio.201411023&amp;pmid=25701666" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label">40.</div><div>Leda, M. ∙ Holland, A.J. ∙ Goryachev, A.B.</div><div><strong>Autoamplification and Competition Drive Symmetry Breaking: Initiation of Centriole Duplication by the PLK4-STIL Network</strong></div><div><em>iScience.</em> 2018; <strong>8</strong>:222-235</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.isci.2018.10.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.isci.2018.10.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85056818022" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30340068/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2018.10.003&amp;pmid=30340068" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label">41.</div><div>Leung, G.C. ∙ Ho, C.S.W. ∙ Blasutig, I.M. ...</div><div><strong>Determination of the Plk4/Sak consensus phosphorylation motif using peptide spots arrays</strong></div><div><em>FEBS Lett.</em> 2007; <strong>581</strong>:77-83</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.febslet.2006.11.080" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-33845660449" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17174311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.febslet.2006.11.080&amp;pmid=17174311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label">42.</div><div>Liu, T.-L. ∙ Upadhyayula, S. ∙ Milkie, D.E. ...</div><div><strong>Observing the cell in its native state: Imaging subcellular dynamics in multicellular organisms</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaaq1392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaq1392" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85045622293" target="_blank" aria-label="Open Scopus in new tab">Scopus (402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29674564/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaq1392&amp;pmid=29674564" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label">43.</div><div>Lu, Y. ∙ Cross, F.R.</div><div><strong>Periodic cyclin-Cdk activity entrains an autonomous Cdc14 release oscillator</strong></div><div><em>Cell.</em> 2010; <strong>141</strong>:268-279</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2010.03.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2010.03.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-77951763153" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20403323/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2010.03.021&amp;pmid=20403323" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label">44.</div><div>Markow, T.A. ∙ Beall, S. ∙ Matzkin, L.M.</div><div><strong>Egg size, embryonic development time and ovoviviparity in Drosophila species</strong></div><div><em>J.&nbsp;Evol. Biol.</em> 2009; <strong>22</strong>:430-434</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1420-9101.2008.01649.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-58549083588" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19032497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1420-9101.2008.01649.x&amp;pmid=19032497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label">45.</div><div>Marsh, B.J. ∙ Mastronarde, D.N. ∙ Buttle, K.F. ...</div><div><strong>Organellar relationships in the Golgi region of the pancreatic beta cell line, HIT-T15, visualized by high resolution electron tomography</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2001; <strong>98</strong>:2399-2406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.051631998" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0035956989" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11226251/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.051631998&amp;pmid=11226251" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label">46.</div><div>Marshall, W.F.</div><div><strong>Cell Geometry: How Cells Count and Measure Size</strong></div><div><em>Annu. Rev. Biophys.</em> 2016; <strong>45</strong>:49-64</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-biophys-062215-010905" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84978389126" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27145879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-biophys-062215-010905&amp;pmid=27145879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label">47.</div><div>Matsuo, T. ∙ Yamaguchi, S. ∙ Mitsui, S. ...</div><div><strong>Control mechanism of the circadian clock for timing of cell division in&nbsp;vivo</strong></div><div><em>Science.</em> 2003; <strong>302</strong>:255-259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1086271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0141889955" target="_blank" aria-label="Open Scopus in new tab">Scopus (955)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12934012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1086271&amp;pmid=12934012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label">48.</div><div>McCleland, M.L. ∙ O’Farrell, P.H.</div><div><strong>RNAi of mitotic cyclins in Drosophila uncouples the nuclear and centrosome cycle</strong></div><div><em>Curr. Biol.</em> 2008; <strong>18</strong>:245-254</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2008.01.041&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2008.01.041&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-39249084889" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18291653/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2008.01.041&amp;pmid=18291653" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label">49.</div><div>McLamarrah, T.A. ∙ Buster, D.W. ∙ Galletta, B.J. ...</div><div><strong>An ordered pattern of Ana2 phosphorylation by Plk4 is required for centriole assembly</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2018; <strong>217</strong>:1217-1231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201605106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85044771967" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29496738/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201605106&amp;pmid=29496738" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label">50.</div><div>Morgan, D.O.</div><div><strong>The hidden rhythms of the dividing cell</strong></div><div><em>Cell.</em> 2010; <strong>141</strong>:224-226</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2010.03.042&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2010.03.042&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-77951744543" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20403319/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2010.03.042&amp;pmid=20403319" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label">51.</div><div>Mukherji, S. ∙ O’Shea, E.K.</div><div><strong>Mechanisms of organelle biogenesis govern stochastic fluctuations in organelle abundance</strong></div><div><em>eLife.</em> 2014; <strong>3</strong>:e02678</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.02678" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24916159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.02678&amp;pmid=24916159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label">52.</div><div>Nigg, E.A. ∙ Holland, A.J.</div><div><strong>Once and only once: mechanisms of centriole duplication and their deregulation in disease</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2018; <strong>19</strong>:297-312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm.2017.127" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85044769626" target="_blank" aria-label="Open Scopus in new tab">Scopus (294)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29363672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm.2017.127&amp;pmid=29363672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label">53.</div><div>Nigg, E.A. ∙ Raff, J.W.</div><div><strong>Centrioles, centrosomes, and cilia in health and disease</strong></div><div><em>Cell.</em> 2009; <strong>139</strong>:663-678</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2009.10.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2009.10.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-70350771277" target="_blank" aria-label="Open Scopus in new tab">Scopus (729)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19914163/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2009.10.036&amp;pmid=19914163" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label">54.</div><div>Nilsson, J.</div><div><strong>Protein phosphatases in the regulation of mitosis</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2019; <strong>218</strong>:395-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201809138" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85061040617" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30446607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201809138&amp;pmid=30446607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label">55.</div><div>Novák, B. ∙ Tyson, J.J.</div><div><strong>Design principles of biochemical oscillators</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2008; <strong>9</strong>:981-991</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm2530" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-56749160113" target="_blank" aria-label="Open Scopus in new tab">Scopus (868)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18971947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm2530&amp;pmid=18971947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label">56.</div><div>Novak, Z.A. ∙ Conduit, P.T. ∙ Wainman, A. ...</div><div><strong>Asterless licenses daughter centrioles to duplicate for the first time in Drosophila embryos</strong></div><div><em>Curr. Biol.</em> 2014; <strong>24</strong>:1276-1282</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2014.04.023&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2014.04.023&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84902169026" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24835456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2014.04.023&amp;pmid=24835456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label">57.</div><div>Ohta, M. ∙ Ashikawa, T. ∙ Nozaki, Y. ...</div><div><strong>Direct interaction of Plk4 with STIL ensures formation of a single procentriole per parental centriole</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:5267</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms6267" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84929463214" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25342035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms6267&amp;pmid=25342035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label">58.</div><div>Park, J.-E. ∙ Zhang, L. ∙ Bang, J.K. ...</div><div><strong>Phase separation of Polo-like kinase 4 by autoactivation and clustering drives centriole biogenesis</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4959</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12619-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85074281895" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31672968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12619-2&amp;pmid=31672968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label">59.</div><div>Petrásek, Z. ∙ Schwille, P.</div><div><strong>Precise measurement of diffusion coefficients using scanning fluorescence correlation spectroscopy</strong></div><div><em>Biophys. J.</em> 2008; <strong>94</strong>:1437-1448</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1529%2Fbiophysj.107.108811&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1529%2Fbiophysj.107.108811&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-38949147878" target="_blank" aria-label="Open Scopus in new tab">Scopus (387)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17933881/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1529%2Fbiophysj.107.108811&amp;pmid=17933881" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="optOwgpSd6Xd3" class="citation"><div class="citation-content"><div class="label">60.</div><div>Price, D. ∙ Rabinovitch, S. ∙ O’Farrell, P.H. ...</div><div><strong>Drosophila wee1 Has an Essential Role in the Nuclear Divisions of Early Embryogenesis</strong></div><div><em>Genetics.</em> 2000; <strong>155</strong>:159-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/genetics/155.1.159" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10790391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgenetics%2F155.1.159&amp;pmid=10790391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label">61.</div><div>Ripley, B.D.</div><div><strong>The second-order analysis of stationary point processes</strong></div><div><em>J.&nbsp;Appl. Probab.</em> 1976; <strong>13</strong>:255-266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2307/3212829" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2307%2F3212829" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label">62.</div><div>Rogers, G.C. ∙ Rusan, N.M. ∙ Peifer, M. ...</div><div><strong>A multicomponent assembly pathway contributes to the formation of acentrosomal microtubule arrays in interphase Drosophila cells</strong></div><div><em>Mol. Biol. Cell.</em> 2008; <strong>19</strong>:3163-3178</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E07-10-1069" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-51349104215" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18463166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E07-10-1069&amp;pmid=18463166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label">63.</div><div>Rogers, G.C. ∙ Rusan, N.M. ∙ Roberts, D.M. ...</div><div><strong>The SCF Slimb ubiquitin ligase regulates Plk4/Sak levels to block centriole reduplication</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2009; <strong>184</strong>:225-239</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.200808049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-60849113138" target="_blank" aria-label="Open Scopus in new tab">Scopus (195)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19171756/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.200808049&amp;pmid=19171756" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label">64.</div><div>Rüttinger, S. ∙ Buschmann, V. ∙ Krämer, B. ...</div><div><strong>Comparison and accuracy of methods to determine the confocal volume for quantitative fluorescence correlation spectroscopy</strong></div><div><em>J.&nbsp;Microsc.</em> 2008; <strong>232</strong>:343-352</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2818.2008.02105.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-55349101296" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19017233/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2818.2008.02105.x&amp;pmid=19017233" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label">65.</div><div>Schönle, A. ∙ Von Middendorff, C. ∙ Ringemann, C. ...</div><div><strong>Monitoring triplet state dynamics with fluorescence correlation spectroscopy: bias and correction</strong></div><div><em>Microsc. Res. Tech.</em> 2014; <strong>77</strong>:528-536</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jemt.22368" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84903158930" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24729575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjemt.22368&amp;pmid=24729575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label">66.</div><div>Schwarz, G.</div><div><strong>Estimating the Dimension of a Model</strong></div><div><em>Ann. Stat.</em> 1978; <strong>6</strong>:461-464</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1214/aos/1176344136" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1214%2Faos%2F1176344136" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label">67.</div><div>Shaner, N.C. ∙ Lambert, G.G. ∙ Chammas, A. ...</div><div><strong>A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:407-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2413" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84877578867" target="_blank" aria-label="Open Scopus in new tab">Scopus (834)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23524392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2413&amp;pmid=23524392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label">68.</div><div>Shcherbo, D. ∙ Murphy, C.S. ∙ Ermakova, G.V. ...</div><div><strong>Far-red fluorescent tags for protein imaging in living tissues</strong></div><div><em>Biochem. J.</em> 2009; <strong>418</strong>:567-574</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/BJ20081949" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-62149150302" target="_blank" aria-label="Open Scopus in new tab">Scopus (422)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19143658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2FBJ20081949&amp;pmid=19143658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label">69.</div><div>Sibon, O.C. ∙ Stevenson, V.A. ∙ Theurkauf, W.E.</div><div><strong>DNA-replication checkpoint control at the Drosophila midblastula transition</strong></div><div><em>Nature.</em> 1997; <strong>388</strong>:93-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/40439" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0030744385" target="_blank" aria-label="Open Scopus in new tab">Scopus (231)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9214509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F40439&amp;pmid=9214509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label">70.</div><div>Sluder, G. ∙ Miller, F.J. ∙ Cole, R. ...</div><div><strong>Protein synthesis and the cell cycle: centrosome reproduction in sea urchin eggs is not under translational control</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1990; <strong>110</strong>:2025-2032</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.110.6.2025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0025373108" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2351692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.110.6.2025&amp;pmid=2351692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label">71.</div><div>Somvanshi, P.R. ∙ Patel, A.K. ∙ Bhartiya, S. ...</div><div><strong>Implementation of integral feedback control in biological systems</strong></div><div><em>Wiley Interdiscip. Rev. Syst. Biol. Med.</em> 2015; <strong>7</strong>:301-316</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/wsbm.1307" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84942296319" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26033862/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fwsbm.1307&amp;pmid=26033862" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label">72.</div><div>Stern, B. ∙ Nurse, P.</div><div><strong>A quantitative model for the cdc2 control of S phase and mitosis in fission yeast</strong></div><div><em>Trends Genet.</em> 1996; <strong>12</strong>:345-350</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2FS0168-9525%2896%2980016-3&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2FS0168-9525%2896%2980016-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-0030249292" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8855663/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0168-9525%2896%2980016-3&amp;pmid=8855663" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label">73.</div><div>Stumpff, J. ∙ Duncan, T. ∙ Homola, E. ...</div><div><strong>Drosophila Wee1 kinase regulates Cdk1 and mitotic entry during embryogenesis</strong></div><div><em>Curr. Biol.</em> 2004; <strong>14</strong>:2143-2148</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2004.11.050&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cub.2004.11.050&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-10344246106" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15589158/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2004.11.050&amp;pmid=15589158" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label">74.</div><div>Swaffer, M.P. ∙ Jones, A.W. ∙ Flynn, H.R. ...</div><div><strong>CDK Substrate Phosphorylation and Ordering the Cell Cycle</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:1750-1761</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2016.11.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.cell.2016.11.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85006276125" target="_blank" aria-label="Open Scopus in new tab">Scopus (231)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27984725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.11.034&amp;pmid=27984725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label">75.</div><div>Swaffer, M.P. ∙ Jones, A.W. ∙ Flynn, H.R. ...</div><div><strong>Quantitative Phosphoproteomics Reveals the Signaling Dynamics of Cell-Cycle Kinases in the Fission Yeast Schizosaccharomyces pombe</strong></div><div><em>Cell Rep.</em> 2018; <strong>24</strong>:503-514</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.celrep.2018.06.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=10.1016%2Fj.celrep.2018.06.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85049348262" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29996109/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.06.036&amp;pmid=29996109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label">76.</div><div>Takao, D. ∙ Watanabe, K. ∙ Kuroki, K. ...</div><div><strong>Feedback loops in the Plk4-STIL-HsSAS6 network coordinate site selection for procentriole formation</strong></div><div><em>Biol. Open.</em> 2019; <strong>8</strong>:bio047175</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/bio.047175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85072946366" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31533936/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fbio.047175&amp;pmid=31533936" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label">77.</div><div>Tinevez, J.-Y. ∙ Perry, N. ∙ Schindelin, J. ...</div><div><strong>TrackMate: An open and extensible platform for single-particle tracking</strong></div><div><em>Methods.</em> 2017; <strong>115</strong>:80-90</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ymeth.2016.09.016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85002002648" target="_blank" aria-label="Open Scopus in new tab">Scopus (1886)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27713081/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymeth.2016.09.016&amp;pmid=27713081" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label">78.</div><div>Tsai, T.Y.-C. ∙ Choi, Y.S. ∙ Ma, W. ...</div><div><strong>Robust, tunable biological oscillations from interlinked positive and negative feedback loops</strong></div><div><em>Science.</em> 2008; <strong>321</strong>:126-129</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1156951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-46849090495" target="_blank" aria-label="Open Scopus in new tab">Scopus (569)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18599789/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1156951&amp;pmid=18599789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label">79.</div><div>van Breugel, M. ∙ Hirono, M. ∙ Andreeva, A. ...</div><div><strong>Structures of SAS-6 suggest its organization in centrioles</strong></div><div><em>Science.</em> 2011; <strong>331</strong>:1196-1199</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1199325" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-79952280152" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21273447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1199325&amp;pmid=21273447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label">80.</div><div>van Breugel, M. ∙ Wilcken, R. ∙ McLaughlin, S.H. ...</div><div><strong>Structure of the SAS-6 cartwheel hub from Leishmania major</strong></div><div><em>eLife.</em> 2014; <strong>3</strong>:e01812</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.01812" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84898737594" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24596152/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.01812&amp;pmid=24596152" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label">81.</div><div>Waithe, D. ∙ Clausen, M.P. ∙ Sezgin, E. ...</div><div><strong>FoCuS-point: software for STED fluorescence correlation and time-gated single photon counting</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:958-960</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv687" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-84962177645" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26589275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv687&amp;pmid=26589275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label">82.</div><div>Wang, Y. ∙ Song, L. ∙ Liu, M. ...</div><div><strong>A proteomics landscape of circadian clock in mouse liver</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:1553</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-03898-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85045888533" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29674717/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-03898-2&amp;pmid=29674717" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="opt1Ntbzqkxei" class="citation"><div class="citation-content"><div class="label">83.</div><div>Yamamoto, S. ∙ Kitagawa, D.</div><div><strong>Self-organization of Plk4 regulates symmetry breaking in centriole duplication</strong></div><div><em>Nat. Comm.</em> 2019; <strong>10</strong>:1810</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09847-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-85064545095" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31000710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09847-x&amp;pmid=31000710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label">84.</div><div>Yang, Q. ∙ Pando, B.F. ∙ Dong, G. ...</div><div><strong>Circadian gating of the cell cycle revealed in single cyanobacterial cells</strong></div><div><em>Science.</em> 2010; <strong>327</strong>:1522-1526</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1181759" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.018&amp;key=2-s2.0-77949826561" target="_blank" aria-label="Open Scopus in new tab">Scopus (131)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20299597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1181759&amp;pmid=20299597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loading" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867420306139%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00148%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867420306139" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"><div id="au1" property="author" typeof="Person"><div class="heading"><span property="givenName">Mustafa G.</span> <span property="familyName">Aydogan</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:mustafa.aydogan@path.ox.ac.uk" property="email" title="Send email to Mustafa G. Aydogan">mustafa.aydogan@path.ox.ac.uk</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">5</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Aydogan/Mustafa+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au2" property="author" typeof="Person"><div class="heading"><span property="givenName">Thomas L.</span> <span property="familyName">Steinacker</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">5</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Steinacker/Thomas+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au3" property="author" typeof="Person"><div class="heading"><span property="givenName">Mohammad</span> <span property="familyName">Mofatteh</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Mofatteh/Mohammad" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au4" property="author" typeof="Person"><div class="heading"><span property="givenName">Zachary M.</span> <span property="familyName">Wilmott</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Mathematical Institute, University of Oxford, Oxford OX2 6GG, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Wilmott/Zachary+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au5" property="author" typeof="Person"><div class="heading"><span property="givenName">Felix Y.</span> <span property="familyName">Zhou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Ludwig Institute for Cancer Research, University of Oxford, Oxford OX3 7DQ, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Zhou/Felix+Y" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au6" property="author" typeof="Person"><div class="heading"><span property="givenName">Lisa</span> <span property="familyName">Gartenmann</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Gartenmann/Lisa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au7" property="author" typeof="Person"><div class="heading"><span property="givenName">Alan</span> <span property="familyName">Wainman</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Wainman/Alan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au8" property="author" typeof="Person"><div class="heading"><span property="givenName">Saroj</span> <span property="familyName">Saurya</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Saurya/Saroj" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au9" property="author" typeof="Person"><div class="heading"><span property="givenName">Zsofia A.</span> <span property="familyName">Novak</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Novak/Zsofia+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au10" property="author" typeof="Person"><div class="heading"><span property="givenName">Siu-Shing</span> <span property="familyName">Wong</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Wong/Siu-Shing" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au11" property="author" typeof="Person"><div class="heading"><span property="givenName">Alain</span> <span property="familyName">Goriely</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Mathematical Institute, University of Oxford, Oxford OX2 6GG, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Goriely/Alain" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au12" property="author" typeof="Person"><div class="heading"><span property="givenName">Michael A.</span> <span property="familyName">Boemo</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:mb915@cam.ac.uk" property="email" title="Send email to Michael A. Boemo">mb915@cam.ac.uk</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">4</div><div id="fn1"><div id="ntpara0010" role="paragraph">Present address: Department of Pathology, University of Cambridge, Cambridge CB2 1QP, UK</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Boemo/Michael+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au13" property="author" typeof="Person"><div class="heading"><span property="givenName">Jordan W.</span> <span property="familyName">Raff</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:jordan.raff@path.ox.ac.uk" property="email" title="Send email to Jordan W. Raff">jordan.raff@path.ox.ac.uk</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">6</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sir William Dunn School of Pathology, University of Oxford, Oxford OX1 3RE, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Raff/Jordan+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2020.05.018","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30613-9';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749586796813","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"181","issueNumber":"7","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867420306139","coverDate":"202006","type":"XOCS-JOURNAL:SCOPE-FULL","title":"An Autonomous Oscillation Times and Executes Centriole Biogenesis","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94db90e55aa04b32',t:'MTc0OTU4Njc5MS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94db90e55aa04b32&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>