<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="fa3bb894-98b0-4657-b3b7-4a519e9e7324" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;article:article:pii\:S0092867421009946;website:website:cell-site;ctype:string:Journal Content;issue:issue:pii\:S0092867420X00201;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Antigen dominance hierarchies shape TCF1+ progenitor CD8 T&nbsp;cell phenotypes in tumors">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00994-6">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/a7495479-56e7-4f99-bc59-aa5b93d51517/fx1.jpg">
    


        <meta property="og:description" content="Analysis of antigen dominance hierarchy and dynamics in a lung cancer model highlights
the opportunity of optimizing T&nbsp;cell responses against subdominant neoantigens for
better checkpoint blockade response through vaccination that adjusts T&nbsp;cell progenitor
subpopulations">
    











        <meta name="twitter:title" content="Antigen dominance hierarchies shape TCF1+ progenitor CD8 T&nbsp;cell phenotypes in tumors">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/a7495479-56e7-4f99-bc59-aa5b93d51517/fx1.jpg">
    


        <meta name="twitter:description" content="Analysis of antigen dominance hierarchy and dynamics in a lung cancer model highlights
the opportunity of optimizing T&nbsp;cell responses against subdominant neoantigens for
better checkpoint blockade response through vaccination that adjusts T&nbsp;cell progenitor
subpopulations">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00994-6">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Antigen dominance hierarchies shape TCF1+ progenitor CD8 T&nbsp;cell phenotypes in tumors: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Antigen dominance hierarchies shape TCF1+ progenitor CD8 T&nbsp;cell phenotypes in tumors"> <meta name="citation_date" content="2021/09/16"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="19"> <meta name="citation_firstpage" content="4996"> <meta name="citation_lastpage" content="5014.e26"> <meta name="citation_pmid" content="34534464"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421009946"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421009946"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421009946"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.08.020"> <meta name="citation_pmid" content="34534464"> <meta name="citation_author" content="Megan L. Burger"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Amanda M. Cruz"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Grace E. Crossland"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Giorgio Gaglia"><meta name="citation_author_institution" content="Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Ludwig Center at Harvard, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Laboratory of Systems Pharmacology, Department of Systems Biology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Cecily C. Ritch"><meta name="citation_author_institution" content="Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Ludwig Center at Harvard, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Laboratory of Systems Pharmacology, Department of Systems Biology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Sarah E. Blatt"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Arjun Bhutkar"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="David Canner"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Tamina Kienka"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Sara Z. Tavana"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Alexia L. Barandiaran"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Andrea Garmilla"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Jason M. Schenkel"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Michelle Hillman"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Izumi de los Rios Kobara"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Amy Li"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Alex M. Jaeger"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="William L. Hwang"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Department of Radiation Oncology, Massachusetts General Hospital, Boston, MA 02114, USA"><meta name="citation_author" content="Peter M.K. Westcott"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Michael P. Manos"><meta name="citation_author_institution" content="Melanoma Disease Center, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Center for Immuno-oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA"><meta name="citation_author" content="Marta M. Holovatska"><meta name="citation_author_institution" content="Melanoma Disease Center, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Center for Immuno-oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA"><meta name="citation_author" content="F. Stephen Hodi"><meta name="citation_author_institution" content="Melanoma Disease Center, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Center for Immuno-oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA"><meta name="citation_author" content="Aviv Regev"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Genentech, 1 DNA Way, South San Francisco, CA 94080, USA"><meta name="citation_author" content="Sandro Santagata"><meta name="citation_author_institution" content="Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Ludwig Center at Harvard, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Laboratory of Systems Pharmacology, Department of Systems Biology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Oncologic Pathology, Dana Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Tyler Jacks"><meta name="citation_author_email" content="tjacks@mit.edu"><meta name="citation_author_institution" content="David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_keywords" content="TCF1; CD8 T cell; neoantigen; immunodominance; CCR6; Tc17; vaccine; checkpoint blockade; lung cancer"><meta name="citation_reference" content="citation_title=Evolution of Neoantigen Landscape during Immune Checkpoint Blockade in Non-Small Cell Lung Cancer; citation_author=V. Anagnostou; citation_author=K.N. Smith; citation_author=P.M. Forde; citation_author=N. Niknafs; citation_author=R. Bhattacharya; citation_author=J. White; citation_author=T. Zhang; citation_author=V. Adleff; citation_author=J. Phallen; citation_author=N. Wali; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=7; citation_firstpage=264; citation_lastpage=276; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Intravascular staining for discrimination of vascular and tissue leukocytes; citation_author=K.G. Anderson; citation_author=K. Mayer-Barber; citation_author=H. Sung; citation_author=L. Beura; citation_author=B.R. James; citation_author=J.J. Taylor; citation_author=L. Qunaj; citation_author=T.S. Griffith; citation_author=V. Vezys; citation_author=D.L. Barber; citation_author=D. Masopust; citation_journal_title=Nat. Protoc.; citation_volume=9; citation_firstpage=209; citation_lastpage=222; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases; citation_author=M. Andreatta; citation_author=J. Corria-Osorio; citation_author=S. Müller; citation_author=R. Cubas; citation_author=G. Coukos; citation_author=S.J. Carmona; citation_journal_title=Nat. Commun.; citation_volume=12; citation_firstpage=2965; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The mouse Gene Expression Database (GXD): 2021 update; citation_author=R.M. Baldarelli; citation_author=C.M. Smith; citation_author=J.H. Finger; citation_author=T.F. Hayamizu; citation_author=I.J. McCright; citation_author=J. Xu; citation_author=D.R. Shaw; citation_author=J.S. Beal; citation_author=O. Blodgett; citation_author=J. Campbell; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=49; citation_number=D1; citation_firstpage=D924; citation_lastpage=D931; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=T cell receptor fingerprinting enables in-depth characterization of the interactions governing recognition of peptide-MHC complexes; citation_author=A.K. Bentzen; citation_author=L. Such; citation_author=K.K. Jensen; citation_author=A.M. Marquard; citation_author=L.E. Jessen; citation_author=N.J. Miller; citation_author=C.D. Church; citation_author=R. Lyngaa; citation_author=D.M. Koelle; citation_author=J.C. Becker; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_doi=10.1038/nbt.4303; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=ilastik: interactive machine learning for (bio)image analysis; citation_author=S. Berg; citation_author=D. Kutra; citation_author=T. Kroeger; citation_author=C.N. Straehle; citation_author=B.X. Kausler; citation_author=C. Haubold; citation_author=M. Schiegg; citation_author=J. Ales; citation_author=T. Beier; citation_author=M. Rudy; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=16; citation_firstpage=1226; citation_lastpage=1232; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Antigen receptor repertoire profiling from RNA-seq data; citation_author=D.A. Bolotin; citation_author=S. Poslavsky; citation_author=A.N. Davydov; citation_author=F.E. Frenkel; citation_author=L. Fanchi; citation_author=O.I. Zolotareva; citation_author=S. Hemmers; citation_author=E.V. Putintseva; citation_author=A.S. Obraztsova; citation_author=M. Shugay; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=35; citation_firstpage=908; citation_lastpage=911; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Integrating single-cell transcriptomic data across different conditions, technologies, and species; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=P. Smibert; citation_author=E. Papalexi; citation_author=R. Satija; citation_journal_title=Nat. Biotechnol.; citation_volume=36; citation_firstpage=411; citation_lastpage=420; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The single-cell transcriptional landscape of mammalian organogenesis; citation_author=J. Cao; citation_author=M. Spielmann; citation_author=X. Qiu; citation_author=X. Huang; citation_author=D.M. Ibrahim; citation_author=A.J. Hill; citation_author=F. Zhang; citation_author=S. Mundlos; citation_author=L. Christiansen; citation_author=F.J. Steemers; citation_author=et al.; citation_journal_title=Nature; citation_volume=566; citation_firstpage=496; citation_lastpage=502; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Cancer immunotherapy. A dendritic cell vaccine increases the breadth and diversity of melanoma neoantigen-specific T&nbsp;cells; citation_author=B.M. Carreno; citation_author=V. Magrini; citation_author=M. Becker-Hapak; citation_author=S. Kaabinejadian; citation_author=J. Hundal; citation_author=A.A. Petti; citation_author=A. Ly; citation_author=W.R. Lie; citation_author=W.H. Hildebrand; citation_author=E.R. Mardis; citation_author=G.P. Linette; citation_journal_title=Science; citation_volume=348; citation_firstpage=803; citation_lastpage=808; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=T helper 17 cells play a critical pathogenic role in lung cancer; citation_author=S.H. Chang; citation_author=S.G. Mirabolfathinejad; citation_author=H. Katta; citation_author=A.M. Cumpian; citation_author=L. Gong; citation_author=M.S. Caetano; citation_author=S.J. Moghaddam; citation_author=C. Dong; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=111; citation_firstpage=5664; citation_lastpage=5669; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Functional interpretation of single cell similarity maps; citation_author=D. DeTomaso; citation_author=M.G. Jones; citation_author=M. Subramaniam; citation_author=T. Ashuach; citation_author=C.J. Ye; citation_author=N. Yosef; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=4376; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Qualifying antibodies for image-based immune profiling and multiplexed tissue imaging; citation_author=Z. Du; citation_author=J.R. Lin; citation_author=R. Rashid; citation_author=Z. Maliga; citation_author=S. Wang; citation_author=J.C. Aster; citation_author=B. Izar; citation_author=P.K. Sorger; citation_author=S. Santagata; citation_journal_title=Nat. Protoc.; citation_volume=14; citation_firstpage=2900; citation_lastpage=2930; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase; citation_author=M. DuPage; citation_author=A.L. Dooley; citation_author=T. Jacks; citation_journal_title=Nat. Protoc.; citation_volume=4; citation_firstpage=1064; citation_lastpage=1072; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression; citation_author=M. DuPage; citation_author=A.F. Cheung; citation_author=C. Mazumdar; citation_author=M.M. Winslow; citation_author=R. Bronson; citation_author=L.M. Schmidt; citation_author=D. Crowley; citation_author=J. Chen; citation_author=T. Jacks; citation_journal_title=Cancer Cell; citation_volume=19; citation_firstpage=72; citation_lastpage=85; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Expression of tumour-specific antigens underlies cancer immunoediting; citation_author=M. DuPage; citation_author=C. Mazumdar; citation_author=L.M. Schmidt; citation_author=A.F. Cheung; citation_author=T. Jacks; citation_journal_title=Nature; citation_volume=482; citation_firstpage=405; citation_lastpage=409; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Promiscuous binding of extracellular peptides to cell surface class I MHC protein; citation_author=H.N. Eisen; citation_author=X.H. Hou; citation_author=C. Shen; citation_author=K. Wang; citation_author=V.K. Tanguturi; citation_author=C. Smith; citation_author=K. Kozyrytska; citation_author=L. Nambiar; citation_author=C.A. McKinley; citation_author=J. Chen; citation_author=R.J. Cohen; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=4580; citation_lastpage=4585; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Relapse or eradication of cancer is predicted by peptide-major histocompatibility complex affinity; citation_author=B. Engels; citation_author=V.H. Engelhard; citation_author=J. Sidney; citation_author=A. Sette; citation_author=D.C. Binder; citation_author=R.B. Liu; citation_author=D.M. Kranz; citation_author=S.C. Meredith; citation_author=D.A. Rowley; citation_author=H. Schreiber; citation_journal_title=Cancer Cell; citation_volume=23; citation_firstpage=516; citation_lastpage=526; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=RobustGaSP: Robust Gaussian Stochastic Process Emulation, 2016. R package version 0.5.7; citation_author=M. Gu; citation_author=J. Paloma; citation_author=J.O. Berger; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Checkpoint blockade cancer immunotherapy targets tumour-specific mutant antigens; citation_author=M.M. Gubin; citation_author=X. Zhang; citation_author=H. Schuster; citation_author=E. Caron; citation_author=J.P. Ward; citation_author=T. Noguchi; citation_author=Y. Ivanova; citation_author=J. Hundal; citation_author=C.D. Arthur; citation_author=W.J. Krebber; citation_author=et al.; citation_journal_title=Nature; citation_volume=515; citation_firstpage=577; citation_lastpage=581; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing; citation_author=X. Guo; citation_author=Y. Zhang; citation_author=L. Zheng; citation_author=C. Zheng; citation_author=J. Song; citation_author=Q. Zhang; citation_author=B. Kang; citation_author=Z. Liu; citation_author=L. Jin; citation_author=R. Xing; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=978; citation_lastpage=985; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression; citation_author=C. Hafemeister; citation_author=R. Satija; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=296; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Array programming with NumPy; citation_author=C.R. Harris; citation_author=K.J. Millman; citation_author=S.J. van der Walt; citation_author=R. Gommers; citation_author=P. Virtanen; citation_author=D. Cournapeau; citation_author=E. Wieser; citation_author=J. Taylor; citation_author=S. Berg; citation_author=N.J. Smith; citation_author=et al.; citation_journal_title=Nature; citation_volume=585; citation_firstpage=357; citation_lastpage=362; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Fate mapping of IL-17-producing T&nbsp;cells in inflammatory responses; citation_author=K. Hirota; citation_author=J.H. Duarte; citation_author=M. Veldhoen; citation_author=E. Hornsby; citation_author=Y. Li; citation_author=D.J. Cua; citation_author=H. Ahlfors; citation_author=C. Wilhelm; citation_author=M. Tolaini; citation_author=U. Menzel; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=12; citation_firstpage=255; citation_lastpage=263; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Turning the corner on therapeutic cancer vaccines; citation_author=R.E. Hollingsworth; citation_author=K. Jansen; citation_journal_title=NPJ Vaccines; citation_volume=4; citation_firstpage=7; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Tapasin enhances MHC class I peptide presentation according to peptide half-life; citation_author=M. Howarth; citation_author=A. Williams; citation_author=A.B. Tolstrup; citation_author=T. Elliott; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=11737; citation_lastpage=11742; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Proliferating Transitory T Cells with an Effector-like Transcriptional Signature Emerge from PD-1+ Stem-like CD8+ T Cells during Chronic Infection; citation_author=W.H. Hudson; citation_author=J. Gensheimer; citation_author=M. Hashimoto; citation_author=A. Wieland; citation_author=R.M. Valanparambil; citation_author=P. Li; citation_author=J.X. Lin; citation_author=B.T. Konieczny; citation_author=S.J. Im; citation_author=G.J. Freeman; citation_author=et al.; citation_journal_title=Immunity; citation_volume=51; citation_firstpage=1043; citation_lastpage=1058.e4; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Comparing Measures of Sparsity; citation_author=N.R.S. Hurley; citation_journal_title=IEEE Trans. Inf. Theory; citation_volume=55; citation_firstpage=4723; citation_lastpage=4741; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Commensal Microbiota Promote Lung Cancer Development via γδ T Cells; citation_author=C. Jin; citation_author=G.K. Lagoudas; citation_author=C. Zhao; citation_author=S. Bullman; citation_author=A. Bhutkar; citation_author=B. Hu; citation_author=S. Ameh; citation_author=D. Sandel; citation_author=X.S. Liang; citation_author=S. Mazzilli; citation_author=et al.; citation_journal_title=Cell; citation_volume=176; citation_firstpage=998; citation_lastpage=1013.e16; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The CCL20-CCR6 Axis in Cancer Progression; citation_author=S. Kadomoto; citation_author=K. Izumi; citation_author=A. Mizokami; citation_journal_title=Int. J. Mol. Sci.; citation_volume=21; citation_firstpage=5186; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Genomic correlates of response to immune checkpoint blockade; citation_author=T.E. Keenan; citation_author=K.P. Burke; citation_author=E.M. Van Allen; citation_journal_title=Nat. Med.; citation_volume=25; citation_firstpage=389; citation_lastpage=402; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Checkpoint Blockade Immunotherapy Induces Dynamic Changes in PD-1-CD8+ Tumor-Infiltrating T Cells; citation_author=S. Kurtulus; citation_author=A. Madi; citation_author=G. Escobar; citation_author=M. Klapholz; citation_author=J. Nyman; citation_author=E. Christian; citation_author=M. Pawlak; citation_author=D. Dionne; citation_author=J. Xia; citation_author=O. Rozenblatt-Rosen; citation_author=et al.; citation_journal_title=Immunity; citation_volume=50; citation_firstpage=181; citation_lastpage=194.e6; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_author=B. Langmead; citation_author=C. Trapnell; citation_author=M. Pop; citation_author=S.L. Salzberg; citation_journal_title=Genome Biol.; citation_volume=10; citation_firstpage=R25; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome; citation_author=B. Li; citation_author=C.N. Dewey; citation_journal_title=BMC Bioinformatics; citation_volume=12; citation_firstpage=323; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Molecular signatures database (MSigDB) 3.0; citation_author=A. Liberzon; citation_author=A. Subramanian; citation_author=R. Pinchback; citation_author=H. Thorvaldsdóttir; citation_author=P. Tamayo; citation_author=J.P. Mesirov; citation_journal_title=Bioinformatics; citation_volume=27; citation_firstpage=1739; citation_lastpage=1740; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Highly multiplexed immunofluorescence imaging of human tissues and tumors using t-CyCIF and conventional optical microscopes; citation_author=J.R. Lin; citation_author=B. Izar; citation_author=S. Wang; citation_author=C. Yapp; citation_author=S. Mei; citation_author=P.M. Shah; citation_author=S. Santagata; citation_author=P.K. Sorger; citation_journal_title=eLife; citation_volume=7; citation_firstpage=e31657; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Non-classical Immunity Controls Microbiota Impact on Skin Immunity and Tissue Repair; citation_author=J.L. Linehan; citation_author=O.J. Harrison; citation_author=S.J. Han; citation_author=A.L. Byrd; citation_author=I. Vujkovic-Cvijin; citation_author=A.V. Villarino; citation_author=S.K. Sen; citation_author=J. Shaik; citation_author=M. Smelkinson; citation_author=S. Tamoutounour; citation_author=et al.; citation_journal_title=Cell; citation_volume=172; citation_firstpage=784; citation_lastpage=796.e18; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Immunological ignorance is an enabling feature of the oligo-clonal T&nbsp;cell response to melanoma neoantigens; citation_author=G.P. Linette; citation_author=M. Becker-Hapak; citation_author=Z.L. Skidmore; citation_author=M.L. Baroja; citation_author=C. Xu; citation_author=J. Hundal; citation_author=D.H. Spencer; citation_author=W. Fu; citation_author=C. Cummins; citation_author=M. Robnett; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=23662; citation_lastpage=23670; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Moderated estimation of fold&nbsp;change and dispersion for RNA-seq data with DESeq2; citation_author=M.I. Love; citation_author=W. Huber; citation_author=S. Anders; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=550; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A robust and high-throughput Cre reporting and characterization system for the whole mouse brain; citation_author=L. Madisen; citation_author=T.A. Zwingman; citation_author=S.M. Sunkin; citation_author=S.W. Oh; citation_author=H.A. Zariwala; citation_author=H. Gu; citation_author=L.L. Ng; citation_author=R.D. Palmiter; citation_author=M.J. Hawrylycz; citation_author=A.R. Jones; citation_author=et al.; citation_journal_title=Nat. Neurosci.; citation_volume=13; citation_firstpage=133; citation_lastpage=140; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Cancer exome analysis reveals a T-cell-dependent mechanism of cancer immunoediting; citation_author=H. Matsushita; citation_author=M.D. Vesely; citation_author=D.C. Koboldt; citation_author=C.G. Rickert; citation_author=R. Uppaluri; citation_author=V.J. Magrini; citation_author=C.D. Arthur; citation_author=J.M. White; citation_author=Y.S. Chen; citation_author=L.K. Shea; citation_author=et al.; citation_journal_title=Nature; citation_volume=482; citation_firstpage=400; citation_lastpage=404; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Clonal neoantigens elicit T&nbsp;cell immunoreactivity and sensitivity to immune checkpoint blockade; citation_author=N. McGranahan; citation_author=A.J. Furness; citation_author=R. Rosenthal; citation_author=S. Ramskov; citation_author=R. Lyngaa; citation_author=S.K. Saini; citation_author=M. Jamal-Hanjani; citation_author=G.A. Wilson; citation_author=N.J. Birkbak; citation_author=C.T. Hiley; citation_author=et al.; citation_journal_title=Science; citation_volume=351; citation_firstpage=1463; citation_lastpage=1469; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Umap: Uniform manifold approximation and projection for dimension reduction; citation_author=L. McInnes; citation_author=J. Healy; citation_author=J. Melville; citation_journal_title=arXiv; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Data Structures for Statistical Computing in Python. Proceedings of the 9th Python in Science Conference (SCIPY 2010); citation_author=W. McKinney; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Generalized Louvain Method for Community Detection in Large Networks; citation_inbook_title=Proceedings of the 2011 11th International Conference on Intelligent Systems Design and Applications; citation_author=P.D. Meo; citation_author=P. De Meo; citation_author=E. Ferrara; citation_author=G. Fiumara; citation_author=A. Provetti; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Subsets of exhausted CD8+ T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade; citation_author=B.C. Miller; citation_author=D.R. Sen; citation_author=R. Al Abosy; citation_author=K. Bi; citation_author=Y.V. Virkud; citation_author=M.W. LaFleur; citation_author=K.B. Yates; citation_author=A. Lako; citation_author=K. Felt; citation_author=G.S. Naik; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=20; citation_firstpage=326; citation_lastpage=336; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The immune system profoundly restricts intratumor genetic heterogeneity; citation_author=I. Milo; citation_author=M. Bedora-Faure; citation_author=Z. Garcia; citation_author=R. Thibaut; citation_author=L. Périé; citation_author=G. Shakhar; citation_author=L. Deriano; citation_author=P. Bousso; citation_journal_title=Sci. Immunol.; citation_volume=3; citation_firstpage=eaat1435; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=RColorBrewer: ColorBrewer Palettes. R package version 1.1-2; citation_author=E. Neuwirth; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=An immunogenic personal neoantigen vaccine for patients with melanoma; citation_author=P.A. Ott; citation_author=Z. Hu; citation_author=D.B. Keskin; citation_author=S.A. Shukla; citation_author=J. Sun; citation_author=D.J. Bozym; citation_author=W. Zhang; citation_author=A. Luoma; citation_author=A. Giobbie-Hurder; citation_author=L. Peter; citation_author=et al.; citation_journal_title=Nature; citation_volume=547; citation_firstpage=217; citation_lastpage=221; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A Phase Ib Trial of Personalized Neoantigen Therapy Plus Anti-PD-1 in Patients with Advanced Melanoma, Non-small Cell Lung Cancer, or Bladder Cancer; citation_author=P.A. Ott; citation_author=S. Hu-Lieskovan; citation_author=B. Chmielowski; citation_author=R. Govindan; citation_author=A. Naing; citation_author=N. Bhardwaj; citation_author=K. Margolin; citation_author=M.M. Awad; citation_author=M.D. Hellmann; citation_author=J.J. Lin; citation_author=et al.; citation_journal_title=Cell; citation_volume=183; citation_firstpage=347; citation_lastpage=362.e24; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The molecular signature of CD8+ T&nbsp;cells undergoing deletional tolerance; citation_author=I.A. Parish; citation_author=S. Rao; citation_author=G.K. Smyth; citation_author=T. Juelich; citation_author=G.S. Denyer; citation_author=G.M. Davey; citation_author=A. Strasser; citation_author=W.R. Heath; citation_journal_title=Blood; citation_volume=113; citation_firstpage=4575; citation_lastpage=4585; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=A BaSiC tool for background and shading correction of optical microscopy images; citation_author=T. Peng; citation_author=K. Thorn; citation_author=T. Schroeder; citation_author=L. Wang; citation_author=F.J. Theis; citation_author=C. Marr; citation_author=N. Navab; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=14836; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=T Cell Epitope Predictions; citation_author=B. Peters; citation_author=M. Nielsen; citation_author=A. Sette; citation_journal_title=Annu. Rev. Immunol.; citation_volume=38; citation_firstpage=123; citation_lastpage=145; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=CD8(+) T&nbsp;cell differentiation and dysfunction in cancer; citation_author=M. Philip; citation_author=A. Schietinger; citation_journal_title=Nat. Rev. Immunol.; citation_doi=10.1038/s41577-021-00574-3; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Reversed graph embedding resolves complex single-cell trajectories; citation_author=X. Qiu; citation_author=Q. Mao; citation_author=Y. Tang; citation_author=L. Wang; citation_author=R. Chawla; citation_author=H.A. Pliner; citation_author=C. Trapnell; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=979; citation_lastpage=982; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Highly multiplexed immunofluorescence images and single-cell data of immune markers in tonsil and lung cancer; citation_author=R. Rashid; citation_author=G. Gaglia; citation_author=Y.A. Chen; citation_author=J.R. Lin; citation_author=Z. Du; citation_author=Z. Maliga; citation_author=D. Schapiro; citation_author=C. Yapp; citation_author=J. Muhlich; citation_author=A. Sokolov; citation_author=et al.; citation_journal_title=Sci. Data; citation_volume=6; citation_firstpage=323; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Neoantigen-directed immune escape in lung cancer evolution; citation_author=R. Rosenthal; citation_author=E.L. Cadieux; citation_author=R. Salgado; citation_author=M.A. Bakir; citation_author=D.A. Moore; citation_author=C.T. Hiley; citation_author=T. Lund; citation_author=M. Tanić; citation_author=J.L. Reading; citation_author=K. Joshi; citation_author=TRACERx Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=567; citation_firstpage=479; citation_lastpage=485; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma; citation_author=M. Sade-Feldman; citation_author=K. Yizhak; citation_author=S.L. Bjorgaard; citation_author=J.P. Ray; citation_author=C.G. de Boer; citation_author=R.W. Jenkins; citation_author=D.J. Lieb; citation_author=J.H. Chen; citation_author=D.T. Frederick; citation_author=M. Barzily-Rokni; citation_author=et al.; citation_journal_title=Cell; citation_volume=176; citation_firstpage=404; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Egr-2 and Egr-3 are negative regulators of T&nbsp;cell activation; citation_author=M. Safford; citation_author=S. Collins; citation_author=M.A. Lutz; citation_author=A. Allen; citation_author=C.T. Huang; citation_author=J. Kowalski; citation_author=A. Blackford; citation_author=M.R. Horton; citation_author=C. Drake; citation_author=R.H. Schwartz; citation_author=J.D. Powell; citation_journal_title=Nat. Immunol.; citation_volume=6; citation_firstpage=472; citation_lastpage=480; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Personalized RNA mutanome vaccines mobilize poly-specific therapeutic immunity against cancer; citation_author=U. Sahin; citation_author=E. Derhovanessian; citation_author=M. Miller; citation_author=B.P. Kloke; citation_author=P. Simon; citation_author=M. Löwer; citation_author=V. Bukur; citation_author=A.D. Tadmor; citation_author=U. Luxemburger; citation_author=B. Schrörs; citation_author=et al.; citation_journal_title=Nature; citation_volume=547; citation_firstpage=222; citation_lastpage=226; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Empty peptide-receptive MHC class I molecules for efficient detection of antigen-specific T&nbsp;cells; citation_author=S.K. Saini; citation_author=T. Tamhane; citation_author=R. Anjanappa; citation_author=A. Saikia; citation_author=S. Ramskov; citation_author=M. Donia; citation_author=I.M. Svane; citation_author=S.N. Jakobsen; citation_author=M. Garcia-Alai; citation_author=M. Zacharias; citation_author=et al.; citation_journal_title=Sci. Immunol.; citation_volume=4; citation_firstpage=eaau9039; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Rapid modelling of cooperating genetic events in cancer through somatic genome editing; citation_author=F.J. Sánchez-Rivera; citation_author=T. Papagiannakopoulos; citation_author=R. Romero; citation_author=T. Tammela; citation_author=M.R. Bauer; citation_author=A. Bhutkar; citation_author=N.S. Joshi; citation_author=L. Subbaraj; citation_author=R.T. Bronson; citation_author=W. Xue; citation_author=T. Jacks; citation_journal_title=Nature; citation_volume=516; citation_firstpage=428; citation_lastpage=431; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Conventional type I dendric cells maintain areservoir of proliferative tumor-antigen specific TCF-1+ CD8+ T cells in tumor-draining lymph nodes; citation_author=J.M. Schenkel; citation_author=R.H. Herbst; citation_author=D. Canner; citation_author=A. Li; citation_author=M. Hillman; citation_author=S.L. Shanahan; citation_author=G. Gibbons; citation_author=O.C. Smith; citation_author=J.Y. Kim; citation_author=P. Westcott; citation_author=et al.; citation_journal_title=Immunity; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Low and variable tumor reactivity of the intratumoral TCR repertoire in human cancers; citation_author=W. Scheper; citation_author=S. Kelderman; citation_author=L.F. Fanchi; citation_author=C. Linnemann; citation_author=G. Bendle; citation_author=M.A.J. de Rooij; citation_author=C. Hirt; citation_author=R. Mezzadra; citation_author=M. Slagter; citation_author=K. Dijkstra; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=25; citation_firstpage=89; citation_lastpage=94; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Immunodominance and tumor escape; citation_author=H. Schreiber; citation_author=T.H. Wu; citation_author=J. Nachman; citation_author=W.M. Kast; citation_journal_title=Semin. Cancer Biol.; citation_volume=12; citation_firstpage=25; citation_lastpage=31; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Direct binding of peptide to empty MHC class I molecules on intact cells and in&nbsp;vitro; citation_author=T.N. Schumacher; citation_author=M.T. Heemels; citation_author=J.J. Neefjes; citation_author=W.M. Kast; citation_author=C.J. Melief; citation_author=H.L. Ploegh; citation_journal_title=Cell; citation_volume=62; citation_firstpage=563; citation_lastpage=567; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=Cancer Neoantigens; citation_author=T.N. Schumacher; citation_author=W. Scheper; citation_author=P. Kvistborg; citation_journal_title=Annu. Rev. Immunol.; citation_volume=37; citation_firstpage=173; citation_lastpage=200; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=TOX is a&nbsp;critical regulator of tumour-specific T&nbsp;cell differentiation; citation_author=A.C. Scott; citation_author=F. Dündar; citation_author=P. Zumbo; citation_author=S.S. Chandran; citation_author=C.A. Klebanoff; citation_author=M. Shakiba; citation_author=P. Trivedi; citation_author=L. Menocal; citation_author=H. Appleby; citation_author=S. Camara; citation_author=et al.; citation_journal_title=Nature; citation_volume=571; citation_firstpage=270; citation_lastpage=274; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Intratumoral Tcf1+PD-1+CD8+ T Cells with Stem-like Properties Promote Tumor Control in Response to Vaccination and Checkpoint Blockade Immunotherapy; citation_author=I. Siddiqui; citation_author=K. Schaeuble; citation_author=V. Chennupati; citation_author=S.A. Fuertes Marraco; citation_author=S. Calderon-Copete; citation_author=D. Pais Ferreira; citation_author=S.J. Carmona; citation_author=L. Scarpellino; citation_author=D. Gfeller; citation_author=S. Pradervand; citation_author=et al.; citation_journal_title=Immunity; citation_volume=50; citation_firstpage=195; citation_lastpage=211.e10; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Bystander CD8+ T&nbsp;cells are abundant and phenotypically distinct in human tumour infiltrates; citation_author=Y. Simoni; citation_author=E. Becht; citation_author=M. Fehlings; citation_author=C.Y. Loh; citation_author=S.L. Koo; citation_author=K.W.W. Teng; citation_author=J.P.S. Yeong; citation_author=R. Nahar; citation_author=T. Zhang; citation_author=H. Kared; citation_author=et al.; citation_journal_title=Nature; citation_volume=557; citation_firstpage=575; citation_lastpage=579; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Mechanism of tumor rejection with doublets of CTLA-4, PD-1/PD-L1, or IDO blockade involves restored IL-2 production and proliferation of CD8(+) T&nbsp;cells directly within the tumor microenvironment; citation_author=S. Spranger; citation_author=H.K. Koblish; citation_author=B. Horton; citation_author=P.A. Scherle; citation_author=R. Newton; citation_author=T.F. Gajewski; citation_journal_title=J.&nbsp;Immunother. Cancer; citation_volume=2; citation_firstpage=3; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=IL-17+ CD8+ T&nbsp;cells: Differentiation, phenotype and role in inflammatory disease; citation_author=U. Srenathan; citation_author=K. Steel; citation_author=L.S. Taams; citation_journal_title=Immunol. Lett.; citation_volume=178; citation_firstpage=20; citation_lastpage=26; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics; citation_author=M. Stoeckius; citation_author=S. Zheng; citation_author=B. Houck-Loomis; citation_author=S. Hao; citation_author=B.Z. Yeung; citation_author=W.M. Mauck; citation_author=P. Smibert; citation_author=R. Satija; citation_journal_title=Genome Biol.; citation_volume=19; citation_firstpage=224; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Targeting of cancer neoantigens with donor-derived T&nbsp;cell receptor repertoires; citation_author=E. Strønen; citation_author=M. Toebes; citation_author=S. Kelderman; citation_author=M.M. van Buuren; citation_author=W. Yang; citation_author=N. van Rooij; citation_author=M. Donia; citation_author=M.L. Böschen; citation_author=F. Lund-Johansen; citation_author=J. Olweus; citation_author=T.N. Schumacher; citation_journal_title=Science; citation_volume=352; citation_firstpage=1337; citation_lastpage=1341; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Comprehensive Integration of Single-Cell Data; citation_author=T. Stuart; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=C. Hafemeister; citation_author=E. Papalexi; citation_author=W.M. Mauck; citation_author=Y. Hao; citation_author=M. Stoeckius; citation_author=P. Smibert; citation_author=R. Satija; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1888; citation_lastpage=1902; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles; citation_author=A. Subramanian; citation_author=P. Tamayo; citation_author=V.K. Mootha; citation_author=S. Mukherjee; citation_author=B.L. Ebert; citation_author=M.A. Gillette; citation_author=A. Paulovich; citation_author=S.L. Pomeroy; citation_author=T.R. Golub; citation_author=E.S. Lander; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_firstpage=15545; citation_lastpage=15550; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq; citation_author=I. Tirosh; citation_author=B. Izar; citation_author=S.M. Prakadan; citation_author=M.H. Wadsworth; citation_author=D. Treacy; citation_author=J.J. Trombetta; citation_author=A. Rotem; citation_author=C. Rodman; citation_author=C. Lian; citation_author=G. Murphy; citation_author=et al.; citation_journal_title=Science; citation_volume=352; citation_firstpage=189; citation_lastpage=196; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Mucosal-associated invariant T&nbsp;cells and disease; citation_author=A. Toubal; citation_author=I. Nel; citation_author=S. Lotersztajn; citation_author=A. Lehuen; citation_journal_title=Nat. Rev. Immunol.; citation_volume=19; citation_firstpage=643; citation_lastpage=657; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=From Louvain to Leiden: guaranteeing well-connected communities; citation_author=V.A. Traag; citation_author=L. Waltman; citation_author=N.J. van Eck; citation_journal_title=Sci. Rep.; citation_volume=9; citation_firstpage=5233; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=‘Final common pathway’ of human cancer immunotherapy: targeting random somatic mutations; citation_author=E. Tran; citation_author=P.F. Robbins; citation_author=S.A. Rosenberg; citation_journal_title=Nat. Immunol.; citation_volume=18; citation_firstpage=255; citation_lastpage=262; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells; citation_author=C. Trapnell; citation_author=D. Cacchiarelli; citation_author=J. Grimsby; citation_author=P. Pokharel; citation_author=S. Li; citation_author=M. Morse; citation_author=N.J. Lennon; citation_author=K.J. Livak; citation_author=T.S. Mikkelsen; citation_author=J.L. Rinn; citation_journal_title=Nat. Biotechnol.; citation_volume=32; citation_firstpage=381; citation_lastpage=386; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=T Cell Factor 1-Expressing Memory-like CD8(+) T Cells Sustain the Immune Response to Chronic Viral Infections; citation_author=D.T. Utzschneider; citation_author=M. Charmoy; citation_author=V. Chennupati; citation_author=L. Pousse; citation_author=D.P. Ferreira; citation_author=S. Calderon-Copete; citation_author=M. Danilo; citation_author=F. Alfei; citation_author=M. Hofmann; citation_author=D. Wieland; citation_author=et al.; citation_journal_title=Immunity; citation_volume=45; citation_firstpage=415; citation_lastpage=427; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Neoantigen landscape dynamics during human melanoma-T cell interactions; citation_author=E.M. Verdegaal; citation_author=N.F. de Miranda; citation_author=M. Visser; citation_author=T. Harryvan; citation_author=M.M. van Buuren; citation_author=R.S. Andersen; citation_author=S.R. Hadrup; citation_author=C.E. van der Minne; citation_author=R. Schotte; citation_author=H. Spits; citation_author=et al.; citation_journal_title=Nature; citation_volume=536; citation_firstpage=91; citation_lastpage=95; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The RAS-dependent ERF control of cell proliferation and differentiation is mediated by c-Myc repression; citation_author=M. Verykokakis; citation_author=C. Papadaki; citation_author=E. Vorgia; citation_author=L. Le Gallic; citation_author=G. Mavrothalassitis; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=282; citation_firstpage=30285; citation_lastpage=30294; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=gplots: Various R programming tools for plotting data. R Package; citation_author=G. Warnes; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Key Parameters of Tumor Epitope Immunogenicity Revealed Through a Consortium Approach Improve Neoantigen Prediction; citation_author=D.K. Wells; citation_author=M.M. van Buuren; citation_author=K.K. Dang; citation_author=V.M. Hubbard-Lucey; citation_author=K.C.F. Sheehan; citation_author=K.M. Campbell; citation_author=A. Lamb; citation_author=J.P. Ward; citation_author=J. Sidney; citation_author=A.B. Blazquez; citation_author=Tumor Neoantigen Selection Alliance; citation_author=et al.; citation_journal_title=Cell; citation_volume=183; citation_firstpage=818; citation_lastpage=834.e13; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=γδT17 cells promote the accumulation and expansion of myeloid-derived suppressor cells in human colorectal cancer; citation_author=P. Wu; citation_author=D. Wu; citation_author=C. Ni; citation_author=J. Ye; citation_author=W. Chen; citation_author=G. Hu; citation_author=Z. Wang; citation_author=C. Wang; citation_author=Z. Zhang; citation_author=W. Xia; citation_author=et al.; citation_journal_title=Immunity; citation_volume=40; citation_firstpage=785; citation_lastpage=800; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response; citation_author=T.D. Wu; citation_author=S. Madireddi; citation_author=P.E. de Almeida; citation_author=R. Banchereau; citation_author=Y.J. Chen; citation_author=A.S. Chitre; citation_author=E.Y. Chiang; citation_author=H. Iftikhar; citation_author=W.E. O’Gorman; citation_author=A. Au-Yeung; citation_author=et al.; citation_journal_title=Nature; citation_volume=579; citation_firstpage=274; citation_lastpage=278; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Confronting complexity: real-world immunodominance in antiviral CD8+ T&nbsp;cell responses; citation_author=J.W. Yewdell; citation_journal_title=Immunity; citation_volume=25; citation_firstpage=533; citation_lastpage=543; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Clonal replacement of tumor-specific T&nbsp;cells following PD-1 blockade; citation_author=K.E. Yost; citation_author=A.T. Satpathy; citation_author=D.K. Wells; citation_author=Y. Qi; citation_author=C. Wang; citation_author=R. Kageyama; citation_author=K.L. McNamara; citation_author=J.M. Granja; citation_author=K.Y. Sarin; citation_author=R.A. Brown; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=25; citation_firstpage=1251; citation_lastpage=1259; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Immune recognition of somatic mutations leading to complete durable regression in metastatic breast cancer; citation_author=N. Zacharakis; citation_author=H. Chinnasamy; citation_author=M. Black; citation_author=H. Xu; citation_author=Y.C. Lu; citation_author=Z. Zheng; citation_author=A. Pasetto; citation_author=M. Langhan; citation_author=T. Shelton; citation_author=T. Prickett; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=724; citation_lastpage=730; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=CD4+ T Cell Help Is Required for the Formation of a Cytolytic CD8+ T Cell Subset that Protects against Chronic Infection and Cancer; citation_author=R. Zander; citation_author=D. Schauder; citation_author=G. Xin; citation_author=C. Nguyen; citation_author=X. Wu; citation_author=A. Zajac; citation_author=W. Cui; citation_journal_title=Immunity; citation_volume=51; citation_firstpage=1028; citation_lastpage=1042.e4; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The role of interleukin-17 in tumor development and progression; citation_author=J. Zhao; citation_author=X. Chen; citation_author=T. Herjan; citation_author=X. Li; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=217; citation_firstpage=e20190297; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Massively parallel digital transcriptional profiling of single cells; citation_author=G.X. Zheng; citation_author=J.M. Terry; citation_author=P. Belgrader; citation_author=P. Ryvkin; citation_author=Z.W. Bent; citation_author=R. Wilson; citation_author=S.B. Ziraldo; citation_author=T.D. Wheeler; citation_author=G.P. McDermott; citation_author=J. Zhu; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=14049; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=CD8(+) T&nbsp;cells that produce interleukin-17 regulate myeloid-derived suppressor cells and are associated with survival time of patients with gastric cancer; citation_author=Y. Zhuang; citation_author=L.S. Peng; citation_author=Y.L. Zhao; citation_author=Y. Shi; citation_author=X.H. Mao; citation_author=W. Chen; citation_author=K.C. Pang; citation_author=X.F. Liu; citation_author=T. Liu; citation_author=J.Y. Zhang; citation_author=et al.; citation_journal_title=Gastroenterology; citation_volume=143; citation_firstpage=951; citation_publication_date=2012; "><meta name="citation_abstract" content="<h2>Summary</h2><p>CD8 T&nbsp;cell responses against different tumor neoantigens occur simultaneously, yet little is known about the interplay between responses and its impact on T&nbsp;cell function and tumor control. In mouse lung adenocarcinoma, we found that immunodominance is established in tumors, wherein CD8 T&nbsp;cell expansion is predominantly driven by the antigen that most stably binds MHC. T&nbsp;cells responding to subdominant antigens were enriched for a TCF1<sup>+</sup> progenitor phenotype correlated with response to immune checkpoint blockade (ICB) therapy. However, the subdominant T&nbsp;cell response did not preferentially benefit from ICB due to a dysfunctional subset of TCF1<sup>+</sup> cells marked by CCR6 and Tc17 differentiation. Analysis of human samples and sequencing datasets revealed that CCR6<sup>+</sup> TCF1<sup>+</sup> cells exist across human cancers and are not correlated with ICB response. Vaccination eliminated CCR6<sup>+</sup> TCF1<sup>+</sup> cells and dramatically improved the subdominant response, highlighting a strategy to optimally engage concurrent neoantigen responses against tumors.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=6291&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)00994-6&amp;be=2850&amp;fe=6253&amp;dc=5871&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749591471305,%22n%22:0,%22f%22:2220,%22dn%22:2221,%22dne%22:2221,%22c%22:2221,%22s%22:2221,%22ce%22:2221,%22rq%22:2221,%22rp%22:2834,%22rpe%22:3525,%22dl%22:2840,%22di%22:3764,%22ds%22:5871,%22de%22:5883,%22dc%22:6253,%22l%22:6253,%22le%22:6283%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900994-6" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00994-6"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00994-6"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900994-6" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-259175" aria-expanded="false" aria-controls="login-links-259175" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-259175" id="login-links--259175" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900994-6" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900994-6" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900994-6" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-259175" data-opens="#links-dialog-259175" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-259175" aria-modal="true"><button data-closes="#links-dialog-259175" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900994-6" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900994-6" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900994-6" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.08.020">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)00994-6" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0020-1"><span>Volume 184</span>, <span>Issue 19</span></a></span><span class="meta-panel__pages"><span>p4996-5014.e26</span></span><span class="meta-panel__onlineDate">September 16, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=19&amp;pii=S0092-8674%2820%29X0020-1" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Antigen dominance hierarchies shape TCF1<sup>+</sup> progenitor CD8 T&nbsp;cell phenotypes in tumors</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Megan L.</span> <span property="familyName">Burger</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Megan L.</span> <span property="familyName">Burger</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Burger/Megan+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Amanda M.</span> <span property="familyName">Cruz</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Amanda M.</span> <span property="familyName">Cruz</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Cruz/Amanda+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Grace E.</span> <span property="familyName">Crossland</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Grace E.</span> <span property="familyName">Crossland</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Crossland/Grace+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Giorgio</span> <span property="familyName">Gaglia</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Giorgio</span> <span property="familyName">Gaglia</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Ludwig Center at Harvard, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Systems Pharmacology, Department of Systems Biology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gaglia/Giorgio" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Cecily C.</span> <span property="familyName">Ritch</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Cecily C.</span> <span property="familyName">Ritch</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Ludwig Center at Harvard, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Systems Pharmacology, Department of Systems Biology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ritch/Cecily+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Sarah E.</span> <span property="familyName">Blatt</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sarah E.</span> <span property="familyName">Blatt</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Blatt/Sarah+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Arjun</span> <span property="familyName">Bhutkar</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Arjun</span> <span property="familyName">Bhutkar</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bhutkar/Arjun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">David</span> <span property="familyName">Canner</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David</span> <span property="familyName">Canner</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Canner/David" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Tamina</span> <span property="familyName">Kienka</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tamina</span> <span property="familyName">Kienka</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kienka/Tamina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Sara Z.</span> <span property="familyName">Tavana</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sara Z.</span> <span property="familyName">Tavana</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Tavana/Sara+Z" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Alexia L.</span> <span property="familyName">Barandiaran</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alexia L.</span> <span property="familyName">Barandiaran</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Barandiaran/Alexia+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Andrea</span> <span property="familyName">Garmilla</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrea</span> <span property="familyName">Garmilla</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Garmilla/Andrea" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Jason M.</span> <span property="familyName">Schenkel</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jason M.</span> <span property="familyName">Schenkel</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Schenkel/Jason+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Michelle</span> <span property="familyName">Hillman</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michelle</span> <span property="familyName">Hillman</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hillman/Michelle" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Izumi</span> <span property="familyName">de los Rios Kobara</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Izumi</span> <span property="familyName">de los Rios Kobara</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/de+los+Rios+Kobara/Izumi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Amy</span> <span property="familyName">Li</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Amy</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Amy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Alex M.</span> <span property="familyName">Jaeger</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alex M.</span> <span property="familyName">Jaeger</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jaeger/Alex+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">William L.</span> <span property="familyName">Hwang</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">William L.</span> <span property="familyName">Hwang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Radiation Oncology, Massachusetts General Hospital, Boston, MA 02114, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hwang/William+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Peter M.K.</span> <span property="familyName">Westcott</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Peter M.K.</span> <span property="familyName">Westcott</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Westcott/Peter+M+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Michael P.</span> <span property="familyName">Manos</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael P.</span> <span property="familyName">Manos</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Melanoma Disease Center, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Immuno-oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Manos/Michael+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Marta M.</span> <span property="familyName">Holovatska</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marta M.</span> <span property="familyName">Holovatska</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Melanoma Disease Center, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Immuno-oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Holovatska/Marta+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">F. Stephen</span> <span property="familyName">Hodi</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">F. Stephen</span> <span property="familyName">Hodi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Melanoma Disease Center, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Immuno-oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hodi/F+Stephen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Aviv</span> <span property="familyName">Regev</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Aviv</span> <span property="familyName">Regev</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Genentech, 1 DNA Way, South San Francisco, CA 94080, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Regev/Aviv" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Sandro</span> <span property="familyName">Santagata</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sandro</span> <span property="familyName">Santagata</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Ludwig Center at Harvard, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Systems Pharmacology, Department of Systems Biology, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Oncologic Pathology, Dana Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Santagata/Sandro" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Tyler</span> <span property="familyName">Jacks</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tyler</span> <span property="familyName">Jacks</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:tjacks@mit.edu" property="email" title="Send email to Tyler Jacks">tjacks@mit.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jacks/Tyler" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">13</a></sup> <a id="eadd0010" href="mailto:tjacks@mit.edu" property="email" title="Send email to Tyler Jacks">tjacks@mit.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-linked-content" aria-expanded="false" tabindex="0" role="button"><span>Linked Articles (1)</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Ludwig Center at Harvard, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Laboratory of Systems Pharmacology, Department of Systems Biology, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Department of Radiation Oncology, Massachusetts General Hospital, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Melanoma Disease Center, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Center for Immuno-oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Department of Medical Oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Genentech, 1 DNA Way, South San Francisco, CA 94080, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Department of Oncologic Pathology, Dana Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received January 11, 2021</span>; <span>Revised June 25, 2021</span>; <span>Accepted August 16, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.08.020" property="sameAs" target="_blank">10.1016/j.cell.2021.08.020 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421009946" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.08.020" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div><div id="core-linked-content" role="region" data-expandable="item" style="display: none;">
    

        


    

            

            

            <section data-widget-id="e587233e-22aa-4221-8ae1-22df489d46f7" data-widget-name="UX3linkedContentList" class="axel-linked-articles"><div class="section-paragraph linked-article"><ul class="rlist"><li class="linked-article__item"><div class="linked-article__article-type">Article</div><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(21)00360-5" target="_blank" title="Conventional type I dendritic cells maintain a reservoir of proliferative tumor-antigen specific TCF-1+ CD8+ T&nbsp;cells in tumor-draining lymph nodes">Conventional type I dendritic cells maintain a reservoir of proliferative tumor-antigen specific TCF-1+ CD8+ T&nbsp;cells in tumor-draining lymph nodes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Schenkel et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">September 16, 2021</span></div></li></ul></div></section>

            

        
</div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0020-1"><img src="/cms/asset/2e88ad06-06f9-46c6-945a-449e0a716103/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 19"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2900994-6" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900994-6" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.08.020">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900994-6" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.08.020&amp;pii=S0092-8674%2821%2900994-6" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900994-6" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.08.020">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.08.020" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421009946&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421009946&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=19&amp;pii=S0092-8674%2820%29X0020-1" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)00946-6" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)00888-6" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Peptide-MHC binding properties underlie antigen dominance hierarchies in a lung cancer model</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">CD8 T cells responding to subdominant antigens are enriched in TCF1<sup>+</sup> progenitors</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">CCR6<sup>+</sup> TCF1<sup>+</sup> progenitor cells contribute poorly to checkpoint blockade response</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Vaccination can eliminate CCR6<sup>+</sup> TCF1<sup>+</sup> cells and improve the subdominant response</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">CD8 T&nbsp;cell responses against different tumor neoantigens occur simultaneously, yet little is known about the interplay between responses and its impact on T&nbsp;cell function and tumor control. In mouse lung adenocarcinoma, we found that immunodominance is established in tumors, wherein CD8 T&nbsp;cell expansion is predominantly driven by the antigen that most stably binds MHC. T&nbsp;cells responding to subdominant antigens were enriched for a TCF1<sup>+</sup> progenitor phenotype correlated with response to immune checkpoint blockade (ICB) therapy. However, the subdominant T&nbsp;cell response did not preferentially benefit from ICB due to a dysfunctional subset of TCF1<sup>+</sup> cells marked by CCR6 and Tc17 differentiation. Analysis of human samples and sequencing datasets revealed that CCR6<sup>+</sup> TCF1<sup>+</sup> cells exist across human cancers and are not correlated with ICB response. Vaccination eliminated CCR6<sup>+</sup> TCF1<sup>+</sup> cells and dramatically improved the subdominant response, highlighting a strategy to optimally engage concurrent neoantigen responses against tumors.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/30fa7a14-538d-40d8-a92d-8b122ddc2142/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.08.020/asset/a7495479-56e7-4f99-bc59-aa5b93d51517/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/30fa7a14-538d-40d8-a92d-8b122ddc2142/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;TCF1&quot;" alt="Search on this keyword" rel="nofollow">TCF1</a></li><li><a href="/action/doSearch?AllField=&quot;CD8+T+cell&quot;" alt="Search on this keyword" rel="nofollow">CD8 T cell</a></li><li><a href="/action/doSearch?AllField=&quot;neoantigen&quot;" alt="Search on this keyword" rel="nofollow">neoantigen</a></li><li><a href="/action/doSearch?AllField=&quot;immunodominance&quot;" alt="Search on this keyword" rel="nofollow">immunodominance</a></li><li><a href="/action/doSearch?AllField=&quot;CCR6&quot;" alt="Search on this keyword" rel="nofollow">CCR6</a></li><li><a href="/action/doSearch?AllField=&quot;Tc17&quot;" alt="Search on this keyword" rel="nofollow">Tc17</a></li><li><a href="/action/doSearch?AllField=&quot;vaccine&quot;" alt="Search on this keyword" rel="nofollow">vaccine</a></li><li><a href="/action/doSearch?AllField=&quot;checkpoint+blockade&quot;" alt="Search on this keyword" rel="nofollow">checkpoint blockade</a></li><li><a href="/action/doSearch?AllField=&quot;lung+cancer&quot;" alt="Search on this keyword" rel="nofollow">lung cancer</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">Accumulating evidence indicates that peptides derived from mutated proteins presented on major histocompatibility (MHC) molecules, termed neoantigens, drive productive T&nbsp;cell responses to tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref68-1" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-1" aria-controls="sref68-1">Schumacher et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref68-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Schumacher, T.N. ∙ Scheper, W. ∙ Kvistborg, P.</div><div><strong>Cancer Neoantigens</strong></div><div><em>Annu. Rev. Immunol.</em> 2019; <strong>37</strong>:173-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-042617-053402" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85065015974" target="_blank" aria-label="Open Scopus in new tab">Scopus (374)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30550719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-042617-053402&amp;pmid=30550719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Neoantigen-specific CD8 T&nbsp;cells expand in response to immune checkpoint blockade (ICB) therapies (e.g., anti-PD1/anti-PDL1), and patient responses correlate with high neoantigen and/or mutational burden (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Keenan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Keenan, T.E. ∙ Burke, K.P. ∙ Van Allen, E.M.</div><div><strong>Genomic correlates of response to immune checkpoint blockade</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:389-402</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0382-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85062627637" target="_blank" aria-label="Open Scopus in new tab">Scopus (324)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30842677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0382-x&amp;pmid=30842677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, adoptive T&nbsp;cell therapies targeting neoantigens can promote meaningful tumor regression (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Tran et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Tran, E. ∙ Robbins, P.F. ∙ Rosenberg, S.A.</div><div><strong>‘Final common pathway’ of human cancer immunotherapy: targeting random somatic mutations</strong></div><div><em>Nat. Immunol.</em> 2017; <strong>18</strong>:255-262</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3682" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85012974126" target="_blank" aria-label="Open Scopus in new tab">Scopus (354)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28198830/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3682&amp;pmid=28198830" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Zacharakis et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Zacharakis, N. ∙ Chinnasamy, H. ∙ Black, M. ...</div><div><strong>Immune recognition of somatic mutations leading to complete durable regression in metastatic breast cancer</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:724-730</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0040-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85048021996" target="_blank" aria-label="Open Scopus in new tab">Scopus (589)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29867227/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0040-8&amp;pmid=29867227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These observations provide strong rationale for development of immunotherapies directly targeting neoantigens and, in particular, have fueled recent large-scale efforts to develop neoantigen-targeted cancer vaccines (<span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Hollingsworth and Jansen, 2019</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hollingsworth, R.E. ∙ Jansen, K.</div><div><strong>Turning the corner on therapeutic cancer vaccines</strong></div><div><em>NPJ Vaccines.</em> 2019; <strong>4</strong>:7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41541-019-0103-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061253209" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30774998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41541-019-0103-y&amp;pmid=30774998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0055" role="paragraph">To identify therapeutically actionable neoantigens for immunotherapies, computational pipelines have been developed to predict immunogenic epitopes from exome sequencing of patient tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Peters et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Peters, B. ∙ Nielsen, M. ∙ Sette, A.</div><div><strong>T Cell Epitope Predictions</strong></div><div><em>Annu. Rev. Immunol.</em> 2020; <strong>38</strong>:123-145</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-082119-124838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85084167551" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32045313/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-082119-124838&amp;pmid=32045313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Wells et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Wells, D.K. ∙ van Buuren, M.M. ∙ Dang, K.K. ..., Tumor Neoantigen Selection Alliance</div><div><strong>Key Parameters of Tumor Epitope Immunogenicity Revealed Through a Consortium Approach Improve Neoantigen Prediction</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:818-834.e13</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2020.09.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2020.09.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85094120480" target="_blank" aria-label="Open Scopus in new tab">Scopus (274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33038342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.015&amp;pmid=33038342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although these algorithms identify tens to hundreds of potential neoantigens, the vast majority of these epitopes do not elicit a detectable T&nbsp;cell response. Studies profiling T&nbsp;cell reactivity against predicted neoantigens have generally identified only a few, and sometimes zero, bona fide T&nbsp;cell-reactive epitopes across most cancer types (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Linette et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Linette, G.P. ∙ Becker-Hapak, M. ∙ Skidmore, Z.L. ...</div><div><strong>Immunological ignorance is an enabling feature of the oligo-clonal T&nbsp;cell response to melanoma neoantigens</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:23662-23670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1906026116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85075233562" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31685621/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1906026116&amp;pmid=31685621" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">McGranahan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>McGranahan, N. ∙ Furness, A.J. ∙ Rosenthal, R. ...</div><div><strong>Clonal neoantigens elicit T&nbsp;cell immunoreactivity and sensitivity to immune checkpoint blockade</strong></div><div><em>Science.</em> 2016; <strong>351</strong>:1463-1469</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf1490" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84962301577" target="_blank" aria-label="Open Scopus in new tab">Scopus (2306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26940869/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf1490&amp;pmid=26940869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Scheper et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Scheper, W. ∙ Kelderman, S. ∙ Fanchi, L.F. ...</div><div><strong>Low and variable tumor reactivity of the intratumoral TCR repertoire in human cancers</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:89-94</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0266-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85058049841" target="_blank" aria-label="Open Scopus in new tab">Scopus (391)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30510250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0266-5&amp;pmid=30510250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Simoni et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Simoni, Y. ∙ Becht, E. ∙ Fehlings, M. ...</div><div><strong>Bystander CD8<sup>+</sup> T&nbsp;cells are abundant and phenotypically distinct in human tumour infiltrates</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:575-579</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0130-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85047608686" target="_blank" aria-label="Open Scopus in new tab">Scopus (868)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29769722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0130-2&amp;pmid=29769722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). If this low frequency of T&nbsp;cell reactivity reflects the number of neoantigens capable of eliciting an anti-tumor response, this could pose a significant challenge in the selection of relevant neoantigens for immunotherapies.</div><div id="p0060" role="paragraph">Poor T&nbsp;cell reactivity against computationally predicted neoantigens may be, in part, attributable to shortcomings of prediction algorithms or detection limits of T&nbsp;cell assays. There are also plausible biological explanations, including selection against tumor cells expressing immunogenic neoantigens, resulting in loss of T&nbsp;cell-reactive antigens over time (i.e., immunoediting). Immunoediting has been observed in mouse models (<span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">DuPage et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>DuPage, M. ∙ Mazumdar, C. ∙ Schmidt, L.M. ...</div><div><strong>Expression of tumour-specific antigens underlies cancer immunoediting</strong></div><div><em>Nature.</em> 2012; <strong>482</strong>:405-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84857118418" target="_blank" aria-label="Open Scopus in new tab">Scopus (423)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22318517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10803&amp;pmid=22318517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Matsushita et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Matsushita, H. ∙ Vesely, M.D. ∙ Koboldt, D.C. ...</div><div><strong>Cancer exome analysis reveals a T-cell-dependent mechanism of cancer immunoediting</strong></div><div><em>Nature.</em> 2012; <strong>482</strong>:400-404</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10755" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84862776855" target="_blank" aria-label="Open Scopus in new tab">Scopus (1008)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22318521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10755&amp;pmid=22318521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Milo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Milo, I. ∙ Bedora-Faure, M. ∙ Garcia, Z. ...</div><div><strong>The immune system profoundly restricts intratumor genetic heterogeneity</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaat1435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aat1435" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85057108717" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30470696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aat1435&amp;pmid=30470696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and in conjunction with clinical response to ICB and adoptive T&nbsp;cell therapies (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Anagnostou et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Anagnostou, V. ∙ Smith, K.N. ∙ Forde, P.M. ...</div><div><strong>Evolution of Neoantigen Landscape during Immune Checkpoint Blockade in Non-Small Cell Lung Cancer</strong></div><div><em>Cancer Discov.</em> 2017; <strong>7</strong>:264-276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-16-0828" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85014747024" target="_blank" aria-label="Open Scopus in new tab">Scopus (669)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28031159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-16-0828&amp;pmid=28031159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Verdegaal et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Verdegaal, E.M. ∙ de Miranda, N.F. ∙ Visser, M. ...</div><div><strong>Neoantigen landscape dynamics during human melanoma-T cell interactions</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:91-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18945" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84976406873" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27350335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18945&amp;pmid=27350335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although it is difficult to quantify the extent of immunoediting during tumor progression in humans (<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Rosenthal et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Rosenthal, R. ∙ Cadieux, E.L. ∙ Salgado, R. ..., TRACERx Consortium</div><div><strong>Neoantigen-directed immune escape in lung cancer evolution</strong></div><div><em>Nature.</em> 2019; <strong>567</strong>:479-485</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1032-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85063265508" target="_blank" aria-label="Open Scopus in new tab">Scopus (600)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30894752/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1032-7&amp;pmid=30894752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), evidence in mice suggests it may depend on the tumor type or tissue context (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">DuPage et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>DuPage, M. ∙ Cheung, A.F. ∙ Mazumdar, C. ...</div><div><strong>Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression</strong></div><div><em>Cancer Cell.</em> 2011; <strong>19</strong>:72-85</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-78651451436" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21251614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2010.11.011&amp;pmid=21251614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">DuPage et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>DuPage, M. ∙ Mazumdar, C. ∙ Schmidt, L.M. ...</div><div><strong>Expression of tumour-specific antigens underlies cancer immunoediting</strong></div><div><em>Nature.</em> 2012; <strong>482</strong>:405-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84857118418" target="_blank" aria-label="Open Scopus in new tab">Scopus (423)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22318517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10803&amp;pmid=22318517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Matsushita et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Matsushita, H. ∙ Vesely, M.D. ∙ Koboldt, D.C. ...</div><div><strong>Cancer exome analysis reveals a T-cell-dependent mechanism of cancer immunoediting</strong></div><div><em>Nature.</em> 2012; <strong>482</strong>:400-404</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10755" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84862776855" target="_blank" aria-label="Open Scopus in new tab">Scopus (1008)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22318521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10755&amp;pmid=22318521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, antigen immunodominance may be established during anti-tumor immune responses (<span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Schreiber et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Schreiber, H. ∙ Wu, T.H. ∙ Nachman, J. ...</div><div><strong>Immunodominance and tumor escape</strong></div><div><em>Semin. Cancer Biol.</em> 2002; <strong>12</strong>:25-31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/scbi.2001.0401" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-0036150946" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11926408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fscbi.2001.0401&amp;pmid=11926408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As observed in acute viral infection, one or two epitopes may saturate the T&nbsp;cell response and suppress responses to additional antigens (<span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">Yewdell, 2006</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Yewdell, J.W.</div><div><strong>Confronting complexity: real-world immunodominance in antiviral CD8+ T&nbsp;cell responses</strong></div><div><em>Immunity.</em> 2006; <strong>25</strong>:533-543</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2006.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2006.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-33749528390" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17046682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2006.09.005&amp;pmid=17046682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Supporting this idea, healthy donor T&nbsp;cells can respond to tumor antigens that did not elicit a response from endogenous tumor infiltrating T&nbsp;cells, suggesting that responses against some antigens are depleted or suppressed during tumor progression (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Strønen et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Strønen, E. ∙ Toebes, M. ∙ Kelderman, S. ...</div><div><strong>Targeting of cancer neoantigens with donor-derived T&nbsp;cell receptor repertoires</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:1337-1341</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2288" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84969764539" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27198675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2288&amp;pmid=27198675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Further, vaccines against melanoma neoantigens generate <i>de novo</i> responses, highlighting a degree of unrealized neoantigen immunogenicity in tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Carreno et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Carreno, B.M. ∙ Magrini, V. ∙ Becker-Hapak, M. ...</div><div><strong>Cancer immunotherapy. A dendritic cell vaccine increases the breadth and diversity of melanoma neoantigen-specific T&nbsp;cells</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:803-808</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa3828" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84928195112" target="_blank" aria-label="Open Scopus in new tab">Scopus (996)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25837513/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa3828&amp;pmid=25837513" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Ott et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Ott, P.A. ∙ Hu, Z. ∙ Keskin, D.B. ...</div><div><strong>An immunogenic personal neoantigen vaccine for patients with melanoma</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:217-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22991" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85024377220" target="_blank" aria-label="Open Scopus in new tab">Scopus (1967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22991&amp;pmid=28678778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Sahin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Sahin, U. ∙ Derhovanessian, E. ∙ Miller, M. ...</div><div><strong>Personalized RNA mutanome vaccines mobilize poly-specific therapeutic immunity against cancer</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:222-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85024396858" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678784/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23003&amp;pmid=28678784" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, T&nbsp;cell responses against some neoantigens might outcompete others and narrow the T&nbsp;cell response to fewer epitopes.</div><div id="p0065" role="paragraph">To study the dynamics of anti-tumor T&nbsp;cell responses, our group previously engineered a Kras/p53-driven (KP) autochthonous mouse model of lung adenocarcinoma to express tumor-specific neoantigens (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">DuPage et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>DuPage, M. ∙ Cheung, A.F. ∙ Mazumdar, C. ...</div><div><strong>Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression</strong></div><div><em>Cancer Cell.</em> 2011; <strong>19</strong>:72-85</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-78651451436" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21251614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2010.11.011&amp;pmid=21251614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">DuPage et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>DuPage, M. ∙ Dooley, A.L. ∙ Jacks, T.</div><div><strong>Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:1064-1072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2009.95" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-68149157175" target="_blank" aria-label="Open Scopus in new tab">Scopus (641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19561589/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2009.95&amp;pmid=19561589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Tumors are initiated from single transformed cells <i>in situ</i> and develop over ∼5&nbsp;months, allowing ample time for tumor-immune crosstalk and longitudinal sampling of the T&nbsp;cell response at different tumor stages. Two model CD8 T&nbsp;cell neoantigens are expressed in tumor cells as a fusion to luciferase: SIINFEKL (SIIN) from chicken ovalbumin and a synthetic peptide, SIYRYYGL (SIY) (termed LucOS). In this KP LucOS model, the CD8 T&nbsp;cell response initially slows tumor progression, but exhausts over time. Importantly, neoantigen and MHC expression is sustained in advanced tumors, indicating mechanisms other than immunoediting hinder the T&nbsp;cell response.</div><div id="p0070" role="paragraph">In the LucOS model, SIIN and SIY are concurrently expressed, providing an experimental model to study the interaction of distinct neoantigen-specific CD8 T&nbsp;cell compartments in the context of an anti-tumor response. Here, we find an antigen dominance hierarchy is established between SIIN and SIY in lung tumors that limits expansion of the subdominant CD8 T&nbsp;cell response and represses differentiation from a progenitor to an exhausted cell state. Progenitor CD8 T&nbsp;cells are correlated with response to ICB in mice and humans (<span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Philip and Schietinger, 2021</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Philip, M. ∙ Schietinger, A.</div><div><strong>CD8(+) T&nbsp;cell differentiation and dysfunction in cancer</strong></div><div><em>Nat. Rev. Immunol.</em> 2021;</div><div>Published online July 12, 2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-021-00574-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85109907412" target="_blank" aria-label="Open Scopus in new tab">Scopus (399)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34253904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-021-00574-3&amp;pmid=34253904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), yet their biology remains poorly understood. We uncover a previously undescribed subpopulation of progenitor cells that can arise from antigen subdominance and correlates with impaired functionality and poor response to ICB. Therapeutic vaccination eliminates this subpopulation and greatly expands the subdominant response, highlighting vaccination as a strategy to optimally engage simultaneous CD8 T&nbsp;cell responses against multiple neoantigens in tumors.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Longitudinal analysis uncovers heterogeneity between CD8 T&nbsp;cell responses to different neoantigens in lung adenocarcinoma</h3><div id="p0075" role="paragraph">To investigate how CD8 T&nbsp;cell responses to tumor neoantigens may interact, we longitudinally characterized SIIN- versus SIY-specific CD8 T&nbsp;cells in tumors initiated by intratracheal instillation of lentivirus containing LucOS and <i>Cre</i> recombinase in <i>Kras</i><sup>LSL-G12D/+</sup>;<i>p53</i><sup>fl/fl</sup> mice (<a id="crosref0410" href="#fig1">Figure&nbsp;1</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref15-3" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-3" aria-controls="sref15-3">DuPage et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref15-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>DuPage, M. ∙ Cheung, A.F. ∙ Mazumdar, C. ...</div><div><strong>Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression</strong></div><div><em>Cancer Cell.</em> 2011; <strong>19</strong>:72-85</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-78651451436" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21251614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2010.11.011&amp;pmid=21251614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At the peak of the response (5&nbsp;weeks), we observed that SIIN drove substantially greater CD8 T&nbsp;cell expansion compared to SIY in the tumor-bearing lung (<a id="crosref0420" href="#fig1">Figures 1</a>B, 1C, <a id="crosref0425" href="#figs1">S1</a>A, and S1B). The SIIN response contracted sharply thereafter, whereas the SIY response persisted better over time, resulting in similar numbers of SIIN- and SIY-specific cells by 12&nbsp;weeks. Both populations increasingly displayed hallmarks of dysfunction over time, exhibiting progressive decline in proliferation (Ki67) and upregulation of multiple inhibitory receptors (PD1, LAG3, and TIM3) (<a id="crosref0430" href="#fig1">Figures 1</a>D, 1E, <a id="crosref0435" href="#figs1">S1</a>C, and S1D). Notably, inhibitory receptor expression was lower on SIY-specific cells at 5 and 8&nbsp;weeks compared to SIIN-specific cells, suggesting SIY-specific cells exhibit slower kinetics of dysfunction. A greater proportion of SIY-specific cells also expressed a marker of long-lived T&nbsp;cells, interleukin (IL)-7R, which may contribute to their persistence (<a id="crosref0440" href="#fig1">Figure&nbsp;1</a>F). 30% of SIIN-specific cells produced interferon gamma (IFNγ) compared to 15% of SIY-specific cells at 5&nbsp;weeks, indicating that SIIN-specific cells have greater effector function early in the response (<a id="crosref0445" href="#fig1">Figure&nbsp;1</a>G). However, few cells co-expressed IFNγ and tumor necrosis factor alpha (TNF-α), or were “polyfunctional”, a phenotype associated with better tumor control (<span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Spranger et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Spranger, S. ∙ Koblish, H.K. ∙ Horton, B. ...</div><div><strong>Mechanism of tumor rejection with doublets of CTLA-4, PD-1/PD-L1, or IDO blockade involves restored IL-2 production and proliferation of CD8(+) T&nbsp;cells directly within the tumor microenvironment</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2014; <strong>2</strong>:3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/2051-1426-2-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84977161108" target="_blank" aria-label="Open Scopus in new tab">Scopus (472)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24829760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F2051-1426-2-3&amp;pmid=24829760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, only a small proportion of cells expressed the cytotoxicity-associated molecule granzyme B (GZMB; SIIN 7% versus SIY 4%) (<a id="crosref0455" href="#fig1">Figure&nbsp;1</a>H). These data are consistent with both SIIN- and SIY-specific CD8 T&nbsp;cells having poor anti-tumor activity overall; however, SIIN-specific cells have greater effector function compared to SIY-specific cells.</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/43e8cbf1-a514-4c0a-b0dd-5bca1a070c48/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.08.020/asset/f9f5c693-ce08-429e-accd-c63c625ce422/main.assets/gr1.jpg" height="631" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/43e8cbf1-a514-4c0a-b0dd-5bca1a070c48/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Longitudinal analysis uncovers heterogeneity between CD8 T&nbsp;cell responses to different neoantigens in lung adenocarcinoma</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) KP LucOS genetically engineered mouse model of lung adenocarcinoma expressing tumor-specific neoantigens SIINFEKL (SIIN) and SIYRYYGL (SIY).</div><div id="fspara0020" role="paragraph">(B and C) Percentage (B) and absolute number (C) of SIIN- and SIY-specific CD8<sup>+</sup> CD44<sup>+</sup> T&nbsp;cells in tumor-bearing lung tissue by H-2K<sup>b</sup> peptide-MHC tetramer staining and flow cytometry.</div><div id="fspara0025" role="paragraph">(D–H) Percentage of SIIN- and SIY-specific CD8 T&nbsp;cells expressing Ki67 (D), PD1, LAG3 and TIM3 (E), IL7R (F), IFNγ and TNF-α (G), and GZMB (H) by flow cytometry.</div><div id="fspara0030" role="paragraph">(B)–(H) are representative of ≥3 independent experiments per time point, n ≥ 5 mice per group. Results are expressed as mean&nbsp;+ SD. Statistics were calculated by two-tailed Student’s t test: ns, not significant, <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01, <sup>∗∗∗</sup>p&nbsp;&lt; 0.001.</div><div id="fspara0035" role="paragraph">See also <a id="crosref1915" href="#figs1">Figure&nbsp;S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/1940e08e-eb95-4630-bfed-7bbef6d4f87f/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.08.020/asset/1a56d60e-3afb-42ce-9a33-b0969747cd70/main.assets/figs1.jpg" height="432" width="506" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/1940e08e-eb95-4630-bfed-7bbef6d4f87f/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">SIINFEKL- and SIYRYYGL-specific CD8 T&nbsp;cells in KP LucOS lung tumors display hallmarks of dysfunction over time, related to <a id="crosref2040" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0325" role="paragraph">(A) Flow cytometry gating strategy for tumor SIIN- and SIY-specific CD8 T&nbsp;cells. A single cell suspension from tumor-bearing lung was gated on singlets and live (Zombie dye-negative) cells, then lymphocytes based on FSC/SSC. The lymphocyte gate was then analyzed for the proportion of tissue-resident CD8<sup>+</sup> T&nbsp;cells based on the absence of staining by an intravascularly circulating anti-CD45 antibody (see <a id="crosref2045" href="#sec-4">STAR Methods</a>) and positive staining for CD8α. Tissue-resident CD8<sup>+</sup> cells were further gated on positive CD44 staining prior to analysis of SIINFEKL (SIIN) and SIY (SIYRYYGL) peptide-MHC (H-2K<sup>b</sup>) tetramer staining.</div><div id="fspara0330" role="paragraph">(B) Percentage of SIIN- versus SIY-specific cells of CD8 T&nbsp;cells assayed over time by flow cytometry.</div><div id="fspara0335" role="paragraph">(C) Representative flow cytometry plots of the percentage of SIIN- and SIY-specific cells expressing Ki67 over time.</div><div id="fspara0340" role="paragraph">(D) Representative flow cytometry plots comparing PD1, LAG3 and TIM3 expression between SIIN- and SIY-specific cells at 5&nbsp;weeks. Naive population (gray) represents CD44- tetramer- CD8α+ T&nbsp;cells. Geometric mean fluorescence intensity (gMFI) for PD1, LAG3 and TIM3 is plotted over time for SIIN- versus SIY-specific cells. All data points were compiled from a single experiment for gMFI comparison.</div><div id="fspara0345" role="paragraph">(B-D) are representative of ≥ 3 independent experiments per time point, n ≥ 5 mice per group.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Single-cell RNA-sequencing reveals enrichment of a TCF1<sup>+</sup> progenitor cell phenotype among SIY-specific CD8 T&nbsp;cells</h3><div id="p0080" role="paragraph">To further investigate phenotypic differences between SIIN- and SIY-specific CD8 T&nbsp;cells, we performed single-cell RNA-sequencing (scRNA-seq) with paired T&nbsp;cell receptor (TCR) sequencing at 5&nbsp;weeks post-tumor initiation. Comparison of transcriptional profiles by uniform manifold approximation and projection (UMAP) and unsupervised clustering revealed similar distribution of SIIN- and SIY-specific cells; however, some clusters were significantly enriched for cells directed against one antigen versus the other (<a id="crosref0460" href="#fig2">Figures 2</a>A–2C; <a id="crosref0465" href="#mmc3">Table S3</a>). To explore this transcriptional heterogeneity, we classified single cells using “ProjecTILs” T&nbsp;cell atlases (<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Andreatta et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) derived from mouse lymphocytic choriomeningitis virus (LCMV) infection and B16 melanoma and MC38 colorectal carcinoma tumor infiltrating lymphocytes (TILS) (<a id="crosref0475" href="#fig2">Figures 2</a>D and <a id="crosref0480" href="#figs2">S2</a>A; <a id="crosref0485" href="#mmc1">Table S1</a>). Consistent with <a id="crosref0490" href="#fig1">Figure&nbsp;1</a>, SIIN-specific cells were enriched for “effector” and “exhausted” T&nbsp;cell signatures, whereas SIY-specific cells were enriched for signatures of less differentiated “naive” and “memory precursor” states. Further, cluster 7, which was almost exclusively composed of SIY-specific cells, displayed hallmarks of suppressed activation, including poor clonal expansion and high expression of <i>AY036118.1</i> (ETS-related transcription factor 1 [ERF1]), a transcriptional repressor of <i>c-myc</i> and cell proliferation (<a id="crosref0495" href="#figs2">Figures S2</a>C, S2E, and <a id="crosref0500" href="#figs4">S4</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Verykokakis et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Verykokakis, M. ∙ Papadaki, C. ∙ Vorgia, E. ...</div><div><strong>The RAS-dependent ERF control of cell proliferation and differentiation is mediated by c-Myc repression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:30285-30294</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1074%2Fjbc.M704428200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1074%2Fjbc.M704428200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-35748955822" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17699159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M704428200&amp;pmid=17699159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Intriguingly, SIY-specific cells were also enriched for signatures of “progenitor” cells, a population marked by stem-like ability to replenish more cytotoxic, but terminally exhausted cells (<a id="crosref0510" href="#fig2">Figures&nbsp;2</a>D and <a id="crosref2290" href="#mmc1">S2</a>B; <a id="crosref0515" href="#mmc3">Table S3</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Utzschneider et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Utzschneider, D.T. ∙ Charmoy, M. ∙ Chennupati, V. ...</div><div><strong>T Cell Factor 1-Expressing Memory-like CD8(+) T Cells Sustain the Immune Response to Chronic Viral Infections</strong></div><div><em>Immunity.</em> 2016; <strong>45</strong>:415-427</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2016.07.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2016.07.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85002609650" target="_blank" aria-label="Open Scopus in new tab">Scopus (656)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27533016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2016.07.021&amp;pmid=27533016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Miller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Miller, B.C. ∙ Sen, D.R. ∙ Al Abosy, R. ...</div><div><strong>Subsets of exhausted CD8<sup>+</sup> T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:326-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0312-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061846466" target="_blank" aria-label="Open Scopus in new tab">Scopus (1090)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30778252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0312-6&amp;pmid=30778252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Recent studies have shown ICB acts to promote differentiation of progenitor cells, rather than to reverse dysfunction of terminally exhausted cells (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Kurtulus et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Kurtulus, S. ∙ Madi, A. ∙ Escobar, G. ...</div><div><strong>Checkpoint Blockade Immunotherapy Induces Dynamic Changes in PD-1<sup>-</sup>CD8<sup>+</sup> Tumor-Infiltrating T Cells</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:181-194.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.11.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.11.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059794223" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30635236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.014&amp;pmid=30635236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Miller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Miller, B.C. ∙ Sen, D.R. ∙ Al Abosy, R. ...</div><div><strong>Subsets of exhausted CD8<sup>+</sup> T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:326-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0312-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061846466" target="_blank" aria-label="Open Scopus in new tab">Scopus (1090)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30778252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0312-6&amp;pmid=30778252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-1" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-1" aria-controls="sref70-1">Siddiqui et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref70-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Siddiqui, I. ∙ Schaeuble, K. ∙ Chennupati, V. ...</div><div><strong>Intratumoral Tcf1<sup>+</sup>PD-1<sup>+</sup>CD8<sup>+</sup> T Cells with Stem-like Properties Promote Tumor Control in Response to Vaccination and Checkpoint Blockade Immunotherapy</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:195-211.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.12.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.12.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059786958" target="_blank" aria-label="Open Scopus in new tab">Scopus (873)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30635237/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.12.021&amp;pmid=30635237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); hence, progenitor cells appear to be the therapeutically relevant target of ICB therapy. Progenitor cells are commonly defined by expression of the transcription factor T&nbsp;cell factor 1 (TCF1) and the absence of terminal exhaustion markers, such as <i>Havcr2</i> (TIM3) (<span class="dropBlock reference-citations"><a id="body-ref-sref46-3" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-3" aria-controls="sref46-3">Miller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref46-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Miller, B.C. ∙ Sen, D.R. ∙ Al Abosy, R. ...</div><div><strong>Subsets of exhausted CD8<sup>+</sup> T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:326-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0312-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061846466" target="_blank" aria-label="Open Scopus in new tab">Scopus (1090)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30778252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0312-6&amp;pmid=30778252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SIY-specific cells were preferentially assigned to clusters 4 and 8 (C4 and C8) that were enriched for progenitor gene signatures and expressed <i>Tcf7</i> (encoding TCF1) and other progenitor markers, including <i>Il7r</i> (<a id="crosref0550" href="#fig2">Figures 2</a>C–2E, <a id="crosref0555" href="#figs2">S2</a>C and S2D; <a id="crosref0560" href="#mmc1">Tables S1</a> and <a id="crosref0565" href="#mmc3">S3</a>). Conversely, SIIN-specific cells were preferentially assigned to clusters C2 and C3, which expressed markers of cytotoxicity and exhaustion, including <i>Gzmb</i> and <i>Havcr2.</i></div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/e3b37f14-ebd0-45a5-b28e-68be642339d1/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.08.020/asset/cdc7e560-4d19-4082-a375-9d7a289d7122/main.assets/gr2.jpg" height="768" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/e3b37f14-ebd0-45a5-b28e-68be642339d1/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Single-cell RNA-sequencing reveals enrichment of a TCF1<sup>+</sup> progenitor cell phenotype among SIY-specific CD8 T&nbsp;cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0045" role="paragraph">(A) UMAP embedding of scRNA-seq 5′ gene expression data comparing SIIN- and SIY-specific CD8 T&nbsp;cells at 5&nbsp;weeks post-tumor initiation. n&nbsp;= 10 mice, 4,023 SIIN and 1,861 SIY cells.</div><div id="fspara0050" role="paragraph">(B) Unsupervised clustering of the scRNA-seq data in (A).</div><div id="fspara0055" role="paragraph">(C) Assignment of SIIN- versus SIY-specific cells to the clusters in (B). Brackets indicate significant enrichment (p&nbsp;&lt; 0.05) for SIIN- or SIY-specific cells.</div><div id="fspara0060" role="paragraph">(D) Classification of individual cells from (A) using the ProjecTILs pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Andreatta et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SIY-specific cells are enriched for a progenitor phenotype (TIL, p&nbsp;= 0.002; LCMV, p&nbsp;= 2.51E−15); SIIN-specific cells are enriched for an exhausted phenotype (TIL, p&nbsp;= 4.01E−17; LCMV, p&nbsp;= 1.52E−10).</div><div id="fspara0065" role="paragraph">(E) Log-normalized expression of genes associated with progenitor (<i>Tcf7</i> and <i>Il7r</i>) or exhausted <i>Gzmb</i> and <i>Havcr2</i>) cell states.</div><div id="fspara0070" role="paragraph">(F) Heatmap depicting proportions of TCR clonotypes (rows) assigned to each cluster (columns), for clonotypes ≥5 cells with ≥1 cell assigned to TCF1<sup>+</sup> progenitor clusters C4 or C8.</div><div id="fspara0075" role="paragraph">(G and H) Percentage of SIIN- versus SIY-specific cells expressing TCF1, TIM3, and GZMB (G) and the proportion of SIY-specific TCF1<sup>+</sup> TIM3<sup>−</sup> and TCF1<sup>−</sup> cells expressing IFNγ and TNF-α (H) at 5&nbsp;weeks by flow cytometry.</div><div id="fspara0080" role="paragraph">(I) Percentage and absolute number of SIIN- and SIY-specific TCF1<sup>+</sup> TIM3<sup>−</sup> cells over time by flow cytometry.</div><div id="fspara0085" role="paragraph">(G)–(I) are representative of 3 independent experiments, n ≥ 5 mice per group.</div><div id="fspara0090" role="paragraph">Results are expressed as mean&nbsp;+ SD. Statistics were calculated by two-tailed Student’s t test: ns, not significant,&nbsp;<sup>∗</sup>p&nbsp;&lt; 0.05,&nbsp;<sup>∗∗</sup>p&nbsp;&lt; 0.01,&nbsp;<sup>∗∗∗</sup>p&nbsp;&lt; 0.001. See also <a id="crosref1925" href="#figs2">Figures S2</a> and <a id="crosref1930" href="#figs4">S4</a> and <a id="crosref1935" href="#mmc1">Tables S1</a> and <a id="crosref1940" href="#mmc3">S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/fb8b7bd2-d50c-4615-b040-4ffcd4fb7726/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.08.020/asset/12d36262-77b2-4645-b8ae-d336cf1756f9/main.assets/figs2.jpg" height="939" width="751" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/fb8b7bd2-d50c-4615-b040-4ffcd4fb7726/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">A population of TCF1<sup>+</sup> progenitor cells is identified by scRNA-sequencing of SIIN- and SIY-specific CD8 T&nbsp;cells, related to <a id="crosref2050" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0355" role="paragraph">(A) Classification of individual cells from <a id="crosref2055" href="#fig2">Figure&nbsp;2</a>A using mouse ProjecTILs atlases (<span class="dropBlock reference-citations"><a id="body-ref-sref3-3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-3" aria-controls="sref3-3">Andreatta et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref3-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) from lymphocytic choriomeningitis virus (LCMV) infection and tumor infiltrating lymphocytes (TIL; B16 and MC38 transplant tumor models).</div><div id="fspara0360" role="paragraph">(B) Scoring of individual cells from <a id="crosref2065" href="#fig2">Figure&nbsp;2</a>A for enrichment of progenitor versus terminally-exhausted gene signatures from chronic LCMV (clone 13) infection and B16 melanoma TIL mouse datasets (<span class="dropBlock reference-citations"><a id="body-ref-sref46-4" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-4" aria-controls="sref46-4">Miller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref46-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Miller, B.C. ∙ Sen, D.R. ∙ Al Abosy, R. ...</div><div><strong>Subsets of exhausted CD8<sup>+</sup> T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:326-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0312-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061846466" target="_blank" aria-label="Open Scopus in new tab">Scopus (1090)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30778252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0312-6&amp;pmid=30778252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SIIN-specific cells are enriched for the terminally-exhausted signatures (tumor p&nbsp;= 2.90E-14; LCMV p&nbsp;= 2.24E-26), while SIY-specific cells are enriched for the tumor progenitor signature (tumor p&nbsp;= 0.004; LCMV p&nbsp;= 0.15). Statistics were calculated by hypergeometric test with a z-score cutoff of 0.5.</div><div id="fspara0365" role="paragraph">(C) Heatmap of the top 10 differentially expressed genes per cluster from <a id="crosref2075" href="#fig2">Figure&nbsp;2</a>B. Select genes used to define the phenotypic labels assigned to each cluster are highlighted on the righthand side.</div><div id="fspara0370" role="paragraph">(D) Expression of the indicated progenitor-associated genes on cells from <a id="crosref2080" href="#fig2">Figure&nbsp;2</a>A.</div><div id="fspara0375" role="paragraph">(E) Expression of <i>AY036118.1</i> (ERF1), the top differentially expressed gene in cluster 7, and activation markers <i>Cd69</i> and <i>Cd44</i> on cells from <a id="crosref2085" href="#fig2">Figure&nbsp;2</a>A.</div><div id="fspara0380" role="paragraph">(F-G) Flow cytometric analysis of PD1 (E) and Ki67 (F) expression by TCF1<sup>+</sup> TIM3<sup>−</sup> (colored) versus TCF1<sup>−</sup> (black line) SIIN- and SIY-specific CD8 T&nbsp;cells at 5&nbsp;weeks. The naive population (gray) represents CD44<sup>−</sup> tetramer<sup>−</sup> CD8<sup>+</sup> T&nbsp;cells. Representative of 3 independent experiments, n ≥ 5 mice per group.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">The proximity of clusters C4 and C8 to C2 and C3 on the UMAP suggests transcriptional similarity between these populations, consistent with the previously proposed lineage relationship between progenitor and exhausted cells (<a id="crosref0570" href="#fig2">Figure&nbsp;2</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref70-2" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-2" aria-controls="sref70-2">Siddiqui et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref70-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Siddiqui, I. ∙ Schaeuble, K. ∙ Chennupati, V. ...</div><div><strong>Intratumoral Tcf1<sup>+</sup>PD-1<sup>+</sup>CD8<sup>+</sup> T Cells with Stem-like Properties Promote Tumor Control in Response to Vaccination and Checkpoint Blockade Immunotherapy</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:195-211.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.12.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.12.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059786958" target="_blank" aria-label="Open Scopus in new tab">Scopus (873)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30635237/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.12.021&amp;pmid=30635237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We further examined this relationship by analyzing the distribution of TCR clonotypes (≥5 cells) with at least one cell assigned to progenitor clusters C4 or C8 (<a id="crosref0580" href="#fig2">Figure&nbsp;2</a>F). Unsupervised hierarchical clustering largely segregated SIIN and SIY progenitor cell-containing clonotypes, revealing antigen-specific distribution patterns across clusters of cell states (<a id="crosref0585" href="#fig2">Figure&nbsp;2</a>F). SIIN clonotypes were distributed across most clusters but were enriched for cells assigned to exhausted cluster C2 (<a id="crosref0590" href="#fig2">Figures 2</a>F and <a id="crosref0595" href="#figs4">S4</a>C). This supports a lineage relationship between progenitor and exhausted cell states and suggests SIIN clonotypes containing progenitor cells are well-progressed on the path to exhaustion. In contrast, both progenitor cell-containing and total SIY clonotypes were biased in cell distribution to progenitor clusters C4 and C8 (<a id="crosref0600" href="#fig2">Figures 2</a>F, <a id="crosref0605" href="#figs4">S4</a>A, and S4C), suggesting SIY-specific cells are skewed toward a less differentiated state. Further, while SIIN and SIY clonotype numbers were comparable (≥2 cells: SIIN 153, SIY 149), we observed that SIY clonotypes were smaller in size, indicating that clonal expansion may be repressed among the SIY response (<a id="crosref0610" href="#figs4">Figure&nbsp;S4</a>B).</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/17ad1dc4-9532-48b8-a9d7-ef96c84e2a1c/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.08.020/asset/2f84430c-9cda-42f7-988c-54067b657aab/main.assets/figs4.jpg" height="931" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/17ad1dc4-9532-48b8-a9d7-ef96c84e2a1c/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Analysis of TCR clonotypes from SIIN versus SIY scRNA-seq data delineates relationships between CD8 T&nbsp;cell states, related to <a id="crosref2115" href="#fig2">Figures 2</a> and <a id="crosref2120" href="#fig5">5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0445" role="paragraph">(A) Distribution of TCR clonotypes (≥5 cells) across all clusters. Proportion of cells scale: 1&nbsp;= contains 100% of cells assigned to the clonotype, 0&nbsp;= contains 0% of cells assigned to the clonotype. Hierarchical clustering was performed on the clonotypes (rows) and the clusters (columns). “Clonotype clusters,” antigen and mouse assignments, and the number of cells in each clonotype (size) are indicated on side-bars. A measure of the spread in distribution of each clonotype across clusters, or “sparsity” (Gini index) (<span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Hurley, 2009</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Hurley, N.R.S.</div><div><strong>Comparing Measures of Sparsity</strong></div><div><em>IEEE Trans. Inf. Theory.</em> 2009; <strong>55</strong>:4723-4741</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/TIT.2009.2027527" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-70349622761" target="_blank" aria-label="Open Scopus in new tab">Scopus (599)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FTIT.2009.2027527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), is also included. Additional side-bars show the proportion of cells in each cluster expressing key phenotypic genes: <i>Gzmb, Havcr2, Cx3cr1, Tcf7, Ccr6</i> and <i>Il17a</i>.</div><div id="fspara0450" role="paragraph">(B) UMAP depicting clonotype size for individual cells belonging to SIIN and SIY clonotypes. Scale bar indicates the number of cells in a clonotype. Cells that belong to very small clonotypes or do not have a clonotype assignment are shown in gray. The violin plot shows a comparison of the size of SIIN versus SIY clonotypes containing ≥ 2 cells.</div><div id="fspara0455" role="paragraph">(C) ECDF plots depicting the distribution of SIIN and SIY clonotypes to the indicated clusters. Enrichment is expressed as p value by KS test.</div><div id="fspara0460" role="paragraph">(D) UMAP plots highlighting cells assigned to each of the clonotype clusters depicted on the clonotype cluster side-bar in (A). Clonotype cluster 7 is enriched for a Tc17 phenotype.</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph">Flow cytometric analysis confirmed that SIY-specific cells were enriched for a progenitor cell phenotype, referred to hereafter as “TCF1<sup>+</sup> progenitor” and defined as CD8α<sup>+</sup> CD44<sup>+</sup> TCF1<sup>+</sup> TIM3<sup>−</sup> cells (<a id="crosref0615" href="#fig2">Figure&nbsp;2</a>G). Consistent with previous reports (<span class="dropBlock reference-citations"><a id="body-ref-sref46-5" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-5" aria-controls="sref46-5">Miller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref46-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Miller, B.C. ∙ Sen, D.R. ∙ Al Abosy, R. ...</div><div><strong>Subsets of exhausted CD8<sup>+</sup> T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:326-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0312-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061846466" target="_blank" aria-label="Open Scopus in new tab">Scopus (1090)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30778252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0312-6&amp;pmid=30778252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-3" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-3" aria-controls="sref70-3">Siddiqui et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref70-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Siddiqui, I. ∙ Schaeuble, K. ∙ Chennupati, V. ...</div><div><strong>Intratumoral Tcf1<sup>+</sup>PD-1<sup>+</sup>CD8<sup>+</sup> T Cells with Stem-like Properties Promote Tumor Control in Response to Vaccination and Checkpoint Blockade Immunotherapy</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:195-211.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.12.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.12.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059786958" target="_blank" aria-label="Open Scopus in new tab">Scopus (873)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30635237/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.12.021&amp;pmid=30635237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the majority of these cells expressed the inhibitory receptor PD1 (<a id="crosref0630" href="#figs2">Figure&nbsp;S2</a>F). Few expressed GZMB, indicating they are generally not cytotoxic, but they were more proliferative than TCF1<sup>−</sup> cells and produced IFNγ and TNF-α (<a id="crosref0635" href="#fig2">Figures 2</a>G, 2H, and <a id="crosref0640" href="#figs2">S2</a>G). SIY-specific cells were enriched for TCF1<sup>+</sup> progenitor cells compared to SIIN-specific cells at 5 and 8&nbsp;weeks; however, their proportion became similar by 12&nbsp;weeks (<a id="crosref0645" href="#fig2">Figure&nbsp;2</a>I). The percentage of TCF1<sup>+</sup> cells increased over time, but the absolute number decreased, mirroring the overall contraction of the CD8 T&nbsp;cell response (<a id="crosref0650" href="#fig2">Figure&nbsp;2</a>I). Altogether, these data indicate that SIY-specific cells are enriched for a less differentiated, TCF1<sup>+</sup> progenitor state early in the response to KP LucOS lung tumors, whereas SIIN-specific cells differentiate more rapidly to an exhausted cell state.</div></section><section id="sec-2-3"><h3>Enrichment of TCF1<sup>+</sup> progenitor cells in the SIY response is driven by an antigen dominance hierarchy</h3><div id="p0095" role="paragraph">Having observed that SIIN drove a larger CD8 T&nbsp;cell response than SIY, we hypothesized that competition between the T&nbsp;cell responses might underlie enrichment of TCF1<sup>+</sup> progenitor cells among the SIY response. To test this, we engineered lentiviruses to express SIIN-only or SIY-only as fusions to luciferase (LucSIIN and LucSIY) (<a id="crosref0655" href="#fig3">Figure&nbsp;3</a>A). Strikingly, in LucSIY mice, SIY-specific cells expanded similarly to SIIN-specific cells in LucOS and LucSIIN mice (<a id="crosref0660" href="#fig3">Figures 3</a>A, 3B, and <a id="crosref0665" href="#figs3">S3</a>A). Furthermore, enrichment for TCF1<sup>+</sup> progenitor cells in the SIY response was no longer observed (<a id="crosref0670" href="#fig3">Figures 3</a>C and <a id="crosref0675" href="#figs3">S3</a>B). A higher proportion of SIY-specific cells in LucSIY mice expressed markers of effector memory and exhausted cells (CX3CR1, GZMB, PD1, LAG3, and TIM3) (<a id="crosref0680" href="#figs3">Figures S3</a>C–S3F). Thus, the SIY response in LucSIY mice more closely resembled the response against SIIN in LucOS and LucSIIN mice. These data demonstrate that an antigen dominance hierarchy can form in tumors and this significantly influences the phenotype of cells responding to the subdominant antigen. Competition with SIIN promotes a TCF1<sup>+</sup> progenitor phenotype among SIY-specific cells and hinders differentiation to effector and exhausted cell states.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/0d3a188a-01d6-4e2a-bfef-62439c659e0d/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.08.020/asset/e4bcfd47-cc7e-4d14-8040-bd09a617b5e4/main.assets/gr3.jpg" height="671" width="753" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/0d3a188a-01d6-4e2a-bfef-62439c659e0d/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Enrichment of TCF1<sup>+</sup> progenitor cells in the SIY response is driven by an antigen dominance hierarchy</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0100" role="paragraph">(A) Lentiviral constructs expressing SIIN and SIY (LucOS), SIIN-only (LucSIIN), and SIY-only (LucSIY) and representative flow cytometry plots of antigen-specific cells in the lung 5–6&nbsp;weeks post-tumor initiation.</div><div id="fspara0105" role="paragraph">(B and C) Percentage of SIIN- and SIY-specific cells of CD8 T&nbsp;cells (B) and the proportion that are TCF1<sup>+</sup> TIM3<sup>−</sup> (C) in LucOS, LucSIIN, and LucSIY mice at 5–6&nbsp;weeks by flow cytometry.</div><div id="fspara0110" role="paragraph">(D) Lentiviral constructs expressing SIINYEKL (Y5) in combination with SIY (LucY5S) or alone (LucY5) and representative flow cytometry plots of antigen-specific cells in the lung 5–6&nbsp;weeks post-tumor initiation.</div><div id="fspara0115" role="paragraph">(E) pMHC affinity for each antigen assessed by flow cytometric quantification of H-2K<sup>b</sup> stabilization on TAP-deficient RMA-S cells. Dotted lines indicate 50% binding concentration, which approximates the affinity (K<sub>d</sub>) of each peptide for H-2K<sup>b</sup>.</div><div id="fspara0120" role="paragraph">(F) pMHC complex stability (K<sub>off</sub>) for each antigen measured by differential scanning fluorimetry of pMHC thermal melting. Dotted lines indicate the melting temperature (Tm) of each pMHC complex. Displayed Tm values are averaged from 3 experiments.</div><div id="fspara0125" role="paragraph">(G) TCR cross-reactivity measured by SIIN and Y5 tetramer co-staining of CD8 T&nbsp;cells from spleens of SIIN-vaccinated mice. Percentage of tetramer&nbsp;positive cells stained by both tetramers is graphed.</div><div id="fspara0130" role="paragraph">(H and I) Percentage of Y5-, SIIN-, and SIY-specific cells of CD8 T&nbsp;cells (H) and the proportion that are TCF1<sup>+</sup> TIM3<sup>−</sup> (I) at 5–6&nbsp;weeks in LucOS, LucY5S, LucY5, and LucSIIN mice.</div><div id="fspara0135" role="paragraph">(A–F), (H), and (I) are representative of ≥3 independent experiments, n ≥ 5 mice per group. (E) and (G) are representative of 2 independent experiments.</div><div id="fspara0140" role="paragraph">Results are expressed as mean&nbsp;+ SD. Statistics were calculated by two-tailed Student’s t test: ns, not significant,&nbsp;<sup>∗</sup>p&nbsp;&lt; 0.05,&nbsp;<sup>∗∗</sup>p&nbsp;&lt; 0.01,&nbsp;<sup>∗∗∗</sup>p&nbsp;&lt; 0.001. See also <a id="crosref1945" href="#figs3">Figure&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/9a1c5285-7abf-4dee-88d0-404ccb745fd3/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.08.020/asset/d7dd3ad0-69c3-4a84-b4f8-5420c9e3104d/main.assets/figs3.jpg" height="837" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/9a1c5285-7abf-4dee-88d0-404ccb745fd3/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Expression of SIY alone results in less enrichment for TCF1<sup>+</sup> progenitor cells and greater expression of effector/exhaustion markers, related to <a id="crosref2090" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0390" role="paragraph">(A-B) Quantification of the absolute number of total (A) and TCF1<sup>+</sup> TIM3<sup>−</sup> (B) SIIN- and SIY-specific CD8 T&nbsp;cells in LucOS versus LucSIIN and LucSIY mice at 5-6&nbsp;weeks.</div><div id="fspara0395" role="paragraph">(C) Representative flow cytometry plots of PD1, LAG3 and TIM3 expression between SIIN- and SIY-specific cells in LucOS, LucSIIN and LucSIY mice at 5-6&nbsp;weeks. The niave population (gray) represents CD44- tetramer- CD8+ T cells.</div><div id="fspara0400" role="paragraph">(D-F) Flow cytometric analysis of the proportion of SIIN- and SIY-specific cells expressing CX3CR1 (D), GZMB (E) and Ki67 (F) at 5-6&nbsp;weeks.</div><div id="fspara0405" role="paragraph">(G) Melting curves associated with the Differential Scanning Fluorimetry (DSF) assay in <a id="crosref2095" href="#fig3">Figure&nbsp;3</a>F. The inverse derivative (-dF/dT) was calculated from these curves to determine the pMHC melting temperature (Tm) as a measure of complex stability (K<sub>off</sub>).</div><div id="fspara0410" role="paragraph">(H) Related to <a id="crosref2100" href="#fig3">Figure&nbsp;3</a>G. Flow cytometric analysis of CD8 T&nbsp;cells from the spleens of SIIN-vaccinated wild-type mice co-stained with SIIN- and SIY-specific H-2K<sup>b</sup> tetramers or Y5- and SIY-specific H-2K<sup>b</sup> tetramers. The SIY-specific tetramer serves as a negative control for tetramer binding. Representative of 2 independent experiments, n ≥ 5 mice per experiment.</div><div id="fspara0415" role="paragraph">(I) Flow cytometric quantification of H-2K<sup>b</sup> stabilization on the surface of TAP-deficient RMA-S cells across a peptide concentration gradient. Dotted lines&nbsp;indicate 50% binding concentration which approximates the affinity (K<sub>d</sub>) of each peptide on H-2K<sup>b</sup>. mLAMA4 and mALG8 are compared to SIIN, SIY and Y5, which are also shown in <a id="crosref2105" href="#fig3">Figure&nbsp;3</a>E. Representative of 2 independent experiments.</div><div id="fspara0420" role="paragraph">(J) Differential Scanning Fluorimetry assay for pMHC thermal melting as a measure of complex stability (K<sub>off</sub>). The first derivative (-dF/dT) of the melting curve was calculated to determine the melting temperatures (Tm). Dotted lines&nbsp;indicate the Tm (minimum -dF/dT value) of each pMHC complex in a representative experiment. mLAMA4 and mALG8 are compared to SIIN, SIY and Y5, which are also shown in <a id="crosref2110" href="#fig3">Figure&nbsp;3</a>F. Displayed Tm values are averaged from 3 independent experiments.</div><div id="fspara0425" role="paragraph">(K) Diagram of lentiviral constructs expressing mALG8 and mLAMA4 (LucAL), mALG8-only (LucA) and mLAMA4-only (LucL). Representative flow cytometry plots showing the proportion of mALG8- versus mLAMA4-specific CD8 T&nbsp;cells in the lungs of KP tumor-bearing mice expressing each construct at 5-6&nbsp;weeks.</div><div id="fspara0430" role="paragraph">(L-M) Percentage of mALG8- and mLAMA4-specific cells of CD8 T&nbsp;cells (L) and the proportion that are TCF1<sup>+</sup> TIM3<sup>−</sup> (M) at 5-6&nbsp;weeks in LucAL, LucA and LucL mice.</div><div id="fspara0435" role="paragraph">(A-D) and (K-M) are representative of ≥ 3 independent experiments, n ≥ 5 mice per group.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">In acute viral infection, several factors underlie antigen immunodominance, including relative stability of peptide-MHC (pMHC) complexes, number of pMHC complexes on the surface of antigen-presenting cells, and functional avidity of the T&nbsp;cell repertoire for each antigen (<span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Yewdell, 2006</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Yewdell, J.W.</div><div><strong>Confronting complexity: real-world immunodominance in antiviral CD8+ T&nbsp;cell responses</strong></div><div><em>Immunity.</em> 2006; <strong>25</strong>:533-543</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2006.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2006.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-33749528390" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17046682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2006.09.005&amp;pmid=17046682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SIINFEKL is known to bind H-2K<sup>b</sup> MHC molecules with greater affinity and stability than SIYRYYGL (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Eisen et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Eisen, H.N. ∙ Hou, X.H. ∙ Shen, C. ...</div><div><strong>Promiscuous binding of extracellular peptides to cell surface class I MHC protein</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:4580-4585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1201586109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84863350387" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22403068/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1201586109&amp;pmid=22403068" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); thus, we hypothesized that differential MHC binding might contribute to the observed dominance hierarchy. To test this, we created lentiviruses expressing SIINYEKL (Y5), a point mutant of SIIN that binds H-2K<sup>b</sup> molecules with reduced stability (K<sub>off</sub>) and affinity (K<sub>d</sub>) (<a id="crosref0695" href="#fig3">Figures 3</a>D–3F and <a id="crosref0700" href="#figs3">S3</a>G) (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Howarth et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Howarth, M. ∙ Williams, A. ∙ Tolstrup, A.B. ...</div><div><strong>Tapasin enhances MHC class I peptide presentation according to peptide half-life</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:11737-11742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0306294101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-4143051423" target="_blank" aria-label="Open Scopus in new tab">Scopus (152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15286279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0306294101&amp;pmid=15286279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The Y5 mutation conserves key TCR contact residues and is largely cross reactive with the SIIN T&nbsp;cell precursor pool (<a id="crosref0710" href="#fig3">Figures 3</a>G and <a id="crosref0715" href="#figs3">S3</a>H) (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Bentzen et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bentzen, A.K. ∙ Such, L. ∙ Jensen, K.K. ...</div><div><strong>T cell receptor fingerprinting enables in-depth characterization of the interactions governing recognition of peptide-MHC complexes</strong></div><div><em>Nat. Biotechnol.</em> 2018;</div><div>Published online November 19, 2018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85058748162" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30451992/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4303&amp;pmid=30451992" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). By comparing SIIN and Y5, we can decouple the contribution of pMHC binding and TCR repertoire/avidity to antigen dominance. Remarkably, when Y5 and SIY were expressed together (LucY5S), we found the antigen dominance hierarchy was reversed compared to LucOS, favoring SIY as the immunodominant antigen (<a id="crosref0725" href="#fig3">Figures 3</a>D and 3H). SIY-specific T&nbsp;cells expanded substantially more than Y5-specific cells and the Y5 response was enriched for TCF1<sup>+</sup> progenitor cells (<a id="crosref0730" href="#fig3">Figures 3</a>H and 3I). When Y5 was expressed alone (LucY5), expansion of Y5-specific cells rebounded, and enrichment for TCF1<sup>+</sup> progenitor cells was no longer observed. These results demonstrate that differential pMHC binding can establish an antigen dominance hierarchy in cancer, and subdominant responses, irrespective of the antigen, are enriched for a TCF1<sup>+</sup> progenitor cell phenotype. Y5 bound H-2K<sup>b</sup> with higher affinity, but lower stability than SIY (<a id="crosref0735" href="#fig3">Figures 3</a>E and 3F), suggesting pMHC stability plays a larger role than affinity in establishing Y5 subdominance.</div><div id="p0105" role="paragraph">To evaluate the dynamics between antigens with similar MHC binding properties, we expressed two neoantigens derived from point mutations in murine methylcholanthrene (MCA)-induced sarcoma, mALG8 and mLAMA4 (<a id="crosref0740" href="#figs3">Figure&nbsp;S3</a>K) (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Gubin et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gubin, M.M. ∙ Zhang, X. ∙ Schuster, H. ...</div><div><strong>Checkpoint blockade cancer immunotherapy targets tumour-specific mutant antigens</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:577-581</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13988" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84920921528" target="_blank" aria-label="Open Scopus in new tab">Scopus (1639)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428507/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13988&amp;pmid=25428507" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Compared to mLAMA4, mALG8 binds H-2K<sup>b</sup> with similar, but slightly lower affinity and stability (<a id="crosref0750" href="#figs3">Figures S3</a>I and S3J). Expression of mALG8 or mLAMA4 alone (LucA and LucL, respectively) in tumors resulted in responses of equivalent magnitude (<a id="crosref0755" href="#figs3">Figure&nbsp;S3</a>L). However, when co-expressed (LucAL), neither response became dominant in terms of T&nbsp;cell expansion but both decreased in magnitude compared to the single-antigen setting (<a id="crosref0760" href="#figs3">Figure&nbsp;S3</a>L). The proportion of TCF1<sup>+</sup> progenitor cells was also increased for both responses (<a id="crosref0765" href="#figs3">Figure&nbsp;S3</a>M), trending toward greater enrichment for the mALG8 response, the slightly weaker MHC binder. Thus, antigens with similar MHC binding properties are largely co-dominant, but their competition can drive enrichment for a TCF1<sup>+</sup> progenitor phenotype, similar to subdominant antigens.</div></section><section id="sec-2-4"><h3>Subdominant antigen-specific T&nbsp;cells do not preferentially benefit from anti-PD1/CTLA4 therapy</h3><div id="p0110" role="paragraph">Given that TCF1<sup>+</sup> progenitor cells play a key role in CD8 T&nbsp;cell responses to ICB therapy, we investigated whether SIY-specific cells in LucOS mice might preferentially benefit from ICB. In response to 1&nbsp;week of anti-PD1/CTLA4 treatment, SIIN- and SIY-specific T&nbsp;cell expansion was greatest at 5&nbsp;weeks, diminished at 8&nbsp;weeks, and absent by 12&nbsp;weeks, mirroring the kinetics of T&nbsp;cell dysfunction and contraction of TCF1<sup>+</sup> progenitor cells described above (<a id="crosref0770" href="#fig4">Figures 4</a>A and 4B). ICB treatment shifted cells away from a TCF1<sup>+</sup> TIM3<sup>−</sup> phenotype and toward a TCF1<sup>−</sup> TIM3<sup>+</sup> phenotype, consistent with increased differentiation to an exhausted state (<a id="crosref0775" href="#fig4">Figure&nbsp;4</a>C). The absolute number of TCF1<sup>+</sup> TIM3<sup>−</sup> cells increased, despite their decreased proportion, indicating that TCF1<sup>+</sup> progenitor cells also expand with ICB (<a id="crosref0780" href="#fig4">Figure&nbsp;4</a>D). ICB treatment also increased cell proliferation (Ki67) and cytotoxicity (GZMB) (<a id="crosref0785" href="#fig4">Figures 4</a>E and 4F). Surprisingly, despite being enriched for TCF1<sup>+</sup> progenitor cells, SIY-specific cells did not respond better to ICB compared to SIIN-specific&nbsp;cells. A somewhat greater shift in TCF1<sup>+</sup> TIM3<sup>−</sup> to TCF<sup>−</sup> TIM3<sup>+</sup> cells was observed, but did not translate into greater T&nbsp;cell expansion or effector function (<a id="crosref0790" href="#fig4">Figures 4</a>B–4F).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/45c10afd-6196-4793-8db2-7f8487f4ffbc/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.08.020/asset/383d0732-efc3-4698-93e6-f67753a4a822/main.assets/gr4.jpg" height="656" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/45c10afd-6196-4793-8db2-7f8487f4ffbc/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Subdominant antigen-specific T&nbsp;cells do not preferentially benefit from anti-PD1/CTLA4 therapy</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0150" role="paragraph">(A) Time course of anti-PD1/CTLA4 therapy initiated at 5, 8, or 12&nbsp;weeks post-tumor initiation, comprised of 3 doses (arrows) of isotype control (Iso) or anti-PD1/CTLA4 (PC) antibodies in LucOS mice.</div><div id="fspara0155" role="paragraph">(B–G) Absolute number of SIIN- versus SIY-specific CD8 T&nbsp;cells (B), the proportion that are TCF1<sup>+</sup> TIM3<sup>−</sup> versus TCF1<sup>−</sup> TIM3<sup>+</sup> (C), the absolute number of TCF1<sup>+</sup> TIM3<sup>−</sup> cells (D), and the proportion expressing Ki67 (E), GZMB (F), and CX3CR1 (TCF1<sup>+</sup> and total) (G) assessed by flow cytometry in LucOS mice following treatment with Iso or PC starting at 8&nbsp;weeks unless otherwise indicated. Representative of ≥3 independent experiments, n ≥ 5 mice per group.</div><div id="fspara0855" role="paragraph">Results are expressed as mean&nbsp;+ SD. Statistics were calculated by two-tailed Student’s t test: ns, not significant,&nbsp;<sup>∗</sup>p&nbsp;&lt; 0.05,&nbsp;<sup>∗∗</sup>p&nbsp;&lt; 0.01,&nbsp;<sup>∗∗∗</sup>p&nbsp;&lt; 0.001.</div></div></div></div></figcaption></figure></div><div id="p0115" role="paragraph">Recent studies in chronic LCMV infection and cancer have identified the chemokine receptor CX3CR1 as a marker of cells recently differentiated from TCF1<sup>+</sup> progenitor cells (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Hudson et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Hudson, W.H. ∙ Gensheimer, J. ∙ Hashimoto, M. ...</div><div><strong>Proliferating Transitory T Cells with an Effector-like Transcriptional Signature Emerge from PD-1<sup>+</sup> Stem-like CD8<sup>+</sup> T Cells during Chronic Infection</strong></div><div><em>Immunity.</em> 2019; <strong>51</strong>:1043-1058.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2019.11.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2019.11.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076168642" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31810882/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2019.11.002&amp;pmid=31810882" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">Zander et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Zander, R. ∙ Schauder, D. ∙ Xin, G. ...</div><div><strong>CD4<sup>+</sup> T Cell Help Is Required for the Formation of a Cytolytic CD8<sup>+</sup> T Cell Subset that Protects against Chronic Infection and Cancer</strong></div><div><em>Immunity.</em> 2019; <strong>51</strong>:1028-1042.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2019.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2019.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076189014" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31810883/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2019.10.009&amp;pmid=31810883" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CX3CR1 expression correlates with cytotoxic function and depletion of CX3CR1<sup>+</sup> cells results in a loss of disease control. CX3CR1 was expressed on a subset of SIIN- and SIY-specific TCF1<sup>+</sup> cells and likely marks cells that are actively differentiating (<a id="crosref0805" href="#fig4">Figure&nbsp;4</a>G). CX3CR1 expression was lower on SIY-specific TCF1<sup>+</sup> cells and SIY-specific cells overall, suggesting SIY cells are repressed in their differentiation from a progenitor to an exhausted cell state (<a id="crosref0810" href="#fig4">Figure&nbsp;4</a>G). ICB treatment was able to rescue much of this deficit, consistent with ICB promoting differentiation of progenitor cells (<a id="crosref0815" href="#fig4">Figure&nbsp;4</a>G). Similarly, expression of SIY alone in LucSIY mice rescued CX3CR1<sup>+</sup> cells to SIIN levels, which suggests that competition with the SIIN response represses SIY cell differentiation (<a id="crosref0820" href="#figs3">Figure&nbsp;S3</a>D). Notably, however, ICB was insufficient to fully rescue differentiation of SIY-specific cells, as the SIY response matched, but did not exceed the SIIN response, disproportionate with the enrichment of TCF1<sup>+</sup> cells among the SIY response.</div></section><section id="sec-2-5"><h3>The subdominant CD8 T&nbsp;cell response is enriched for a CCR6<sup>+</sup> TCF1<sup>+</sup> progenitor cell subset with a Tc17 differentiation trajectory</h3><div id="p0120" role="paragraph">One possible explanation for a worse than expected SIY response to ICB is that SIY-specific TCF1<sup>+</sup> progenitor cells are intrinsically less functional than their SIIN counterparts. To explore potential heterogeneity in progenitor cell phenotype or differentiation, we performed a separate analysis of progenitor clusters C4 and C8 and exhausted clusters C2 and C3 from the scRNA-seq data (<a id="crosref0825" href="#fig5">Figure&nbsp;5</a>A). Both C4 and C8 were enriched for a number of genes previously associated with TCF1<sup>+</sup> progenitor cells (e.g., <i>Tcf7</i>, <i>Xcl1</i>, <i>Slamf6</i>, and <i>Ccr7</i>), but were localized distally on the UMAP and were characterized by unique gene signatures (<a id="crosref0830" href="#fig5">Figure&nbsp;5</a>B; <a id="crosref0835" href="#mmc3">Table S3</a>). C8 was marked by genes associated with memory T&nbsp;cells, including <i>Klf2</i>, <i>S1pr1</i>, and <i>Il7r</i>, whereas C4 expressed markers of T&nbsp;cell dysfunction, tolerance, and anergy, and most notably, showed strong enrichment for a gene signature of Tc17 cells (<a id="crosref0840" href="#fig5">Figures 5</a>B, 5C, <a id="crosref0845" href="#figs5">S5</a>A, and S5B; <a id="crosref0850" href="#mmc2">Tables S2</a> and <a id="crosref0855" href="#mmc3">S3</a>).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/04750a5c-6941-4610-b724-d634550950e6/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.08.020/asset/551ceadb-6fa1-4f99-985c-432d27d0e115/main.assets/gr5.jpg" height="941" width="730" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/04750a5c-6941-4610-b724-d634550950e6/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">The subdominant CD8 T&nbsp;cell response is enriched for a CCR6<sup>+</sup> TCF1<sup>+</sup> progenitor cell subset with a Tc17 differentiation trajectory</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0165" role="paragraph">(A) UMAP embedding of clusters C2, C3, C4, and C8 from the scRNA-seq data in <a id="crosref1950" href="#fig2">Figure&nbsp;2</a>B and log-normalized expression of <i>Tcf7</i> and <i>Havcr2</i>.</div><div id="fspara0170" role="paragraph">(B) Expression (mean log(expression&nbsp;+1)) of genes associated with the indicated classifications across clusters C2, C3, C4, and C8.</div><div id="fspara0175" role="paragraph">(C) Scoring of cells from (A) for enrichment of a gene signature differentially upregulated in Tc17 versus Tc1 CD8 T&nbsp;cells in the skin of mice infected with <i>S.&nbsp;epidermidis</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Linehan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Linehan, J.L. ∙ Harrison, O.J. ∙ Han, S.J. ...</div><div><strong>Non-classical Immunity Controls Microbiota Impact on Skin Immunity and Tissue Repair</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:784-796.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2017.12.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2017.12.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85041116653" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358051/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.033&amp;pmid=29358051" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1960" href="#mmc2">Table S2</a>).</div><div id="fspara0180" role="paragraph">(D) Monocle3 lineage trajectories overlaid with UMAP visualizations of <i>Ccr6</i>, <i>Rorc</i>, and <i>Il17a</i> expression.</div><div id="fspara0185" role="paragraph">(E) TCR clonotype cluster 7 (see <a id="crosref1965" href="#figs4">Figures S4</a>A and S4D) highlighted on the UMAP plots from (A).</div><div id="fspara0190" role="paragraph">(F and G) Percentage of SIIN- and SIY-specific cells expressing CCR6 (F) and the proportion of these cells expressing TCF1, SLAMF6, and RORγT (G) at 5&nbsp;weeks by flow cytometry. CCR6<sup>+</sup> RORγT<sup>+</sup> cells are broken down by TCF1 expression.</div><div id="fspara0195" role="paragraph">(H) Percentage of SIIN- and SIY-specific cells expressing IL17A at 5&nbsp;weeks and flow cytometry plots showing co-expression of IL17A with CCR6, RORγT, and TCF1 on SIY-specific cells.</div><div id="fspara0200" role="paragraph">(I and J) Percentage of SIIN- and SIY-specific TCF1<sup>+</sup> TIM3<sup>−</sup> cells expressing CCR6 over time and following 1&nbsp;week of treatment with isotype (Iso) or anti-PD1/CTLA4 (PC) antibodies starting at 5&nbsp;weeks.</div><div id="fspara0205" role="paragraph">(K) Percentage of SIIN- and SIY-specific TCF1<sup>+</sup> cells that express CX3CR1 broken down by CCR6 expression following Iso or PC treatment at 5&nbsp;weeks.</div><div id="fspara0210" role="paragraph">(L) Irradiated KP mice were reconstituted with IL17a<sup>Cre</sup> x Rosa26<sup>LSL-tdTomato</sup> bone marrow and LucOS tumors were initiated. Flow cytometric analysis of the percentage of SIY-specific cells expressing Tomato (marking current/prior IL17A expression) following 1&nbsp;week of Iso or PC treatment starting at 5&nbsp;weeks is shown. Data are compiled from 2 independent experiments.</div><div id="fspara0215" role="paragraph">Data in (F)–(K) are representative of ≥3 independent experiments, n ≥ 5 mice per group.</div><div id="fspara0220" role="paragraph">Results are expressed as mean&nbsp;+ SD. Statistics were calculated by two-tailed Student’s t test: ns, not significant,&nbsp;<sup>∗</sup>p&nbsp;&lt; 0.05,&nbsp;<sup>∗∗</sup>p&nbsp;&lt; 0.01,&nbsp;<sup>∗∗∗</sup>p&nbsp;&lt; 0.001. See also <a id="crosref1970" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref1975" href="#mmc3">Table S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/edbba6dc-0c38-480a-bd9e-49ef2eb4a786/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.08.020/asset/8385286e-9c1b-408d-8dee-d0c1ab88052b/main.assets/figs5.jpg" height="911" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/edbba6dc-0c38-480a-bd9e-49ef2eb4a786/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">scRNA-seq analysis reveals phenotypic heterogeneity among TCF1<sup>+</sup> progenitor CD8 T&nbsp;cells in KP LucOS tumors, related to <a id="crosref2130" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0470" role="paragraph">(A) Expression of select genes from the heatmap in <a id="crosref2135" href="#fig5">Figure&nbsp;5</a>B projected onto the UMAP from <a id="crosref2140" href="#fig5">Figure&nbsp;5</a>A.</div><div id="fspara0475" role="paragraph">(B) Scoring of cells from <a id="crosref2145" href="#fig5">Figure&nbsp;5</a>A for enrichment of a gene signature differentially upregulated in Tc17 versus Tc1 CD8 T&nbsp;cells in the skin of mice treated with Imiquimod (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Linehan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Linehan, J.L. ∙ Harrison, O.J. ∙ Han, S.J. ...</div><div><strong>Non-classical Immunity Controls Microbiota Impact on Skin Immunity and Tissue Repair</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:784-796.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2017.12.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2017.12.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85041116653" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358051/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.033&amp;pmid=29358051" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and signatures of T&nbsp;cell deletional tolerance (<span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Parish et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Parish, I.A. ∙ Rao, S. ∙ Smyth, G.K. ...</div><div><strong>The molecular signature of CD8+ T&nbsp;cells undergoing deletional tolerance</strong></div><div><em>Blood.</em> 2009; <strong>113</strong>:4575-4585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-10-185223" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-66549100257" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19204323/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-10-185223&amp;pmid=19204323" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and T&nbsp;cell anergy (<span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Safford et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Safford, M. ∙ Collins, S. ∙ Lutz, M.A. ...</div><div><strong>Egr-2 and Egr-3 are negative regulators of T&nbsp;cell activation</strong></div><div><em>Nat. Immunol.</em> 2005; <strong>6</strong>:472-480</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni1193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-18244375249" target="_blank" aria-label="Open Scopus in new tab">Scopus (351)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15834410/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni1193&amp;pmid=15834410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). See also <a id="crosref2165" href="#mmc2">Table S2</a>.</div><div id="fspara0480" role="paragraph">(C) Flow cytometric analysis of CCR6+ versus CCR6- subsets of SIIN- and SIY-specific TCF1<sup>+</sup> cells for expression of surface markers of dysfunction, tolerance and anergy at 5&nbsp;weeks. Representative of ≥ 2 independent experiments per marker, n ≥ 5 mice per experiment.</div><div id="fspara0485" role="paragraph">(D) Percentage of SIIN- and SIY-specific CCR6<sup>+</sup> cells expressing the TCF1<sup>+</sup> progenitor cell-associated marker SLAMF6 at 5&nbsp;weeks.</div><div id="fspara0490" role="paragraph">(E-F) Percentage of SIIN- and SIY-specific TCF1<sup>+</sup> cells expressing Ki67 (E) and GZMB (F), separated into CCR6<sup>-</sup><sup>−</sup> versus CCR6<sup>+</sup> subsets.</div><div id="fspara0495" role="paragraph">(G) Related to <a id="crosref2170" href="#fig5">Figure&nbsp;5</a>I. Proportion of SIIN- and SIY-specific TCF1<sup>+</sup> TIM3<sup>−</sup> cells that are CCR6<sup>+</sup> versus CCR6<sup>−</sup> over time.</div><div id="fspara0500" role="paragraph">(H) Related to <a id="crosref2175" href="#fig5">Figure&nbsp;5</a>K. Representative flow cytometric analysis of the proportion of SIIN- and SIY-specific TCF1<sup>+</sup> cells that express CX3CR1 and CCR6 following one week of treatment with isotype control (Iso) or anti-PD1/CTLA4 (PC) antibodies starting at 5&nbsp;weeks.</div><div id="fspara0505" role="paragraph">(I-J) Related to <a id="crosref2180" href="#fig5">Figure&nbsp;5</a>L. Flow cytometric profiling of KP mice reconstituted with IL17a<sup>Cre</sup> x Rosa26<sup>LSL-tdTomato</sup> bone marrow and infected with Lenti-LucOS lentivirus. Cells that express or have previously expressed IL17A are marked by Tomato fluorescence. Representative flow cytometric analyses of CCR6, TCF1 and RORɣT expression on Tomato<sup>+</sup> versus Tomato- SIY-specific CD8 T&nbsp;cells are shown (I). Graph depicts the proportion of Tomato<sup>+</sup> SIY-specific cells following one week of treatment with isotype control (Iso) versus anti-PD1/CTLA4 (PC) antibodies at 5&nbsp;weeks (J). Data are from 2 independent experiments.</div><div id="fspara0510" role="paragraph">(D-H) are representative of ≥ 3 independent experiments, n ≥ 5 mice per group.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">Tc17 cells, marked by expression of the chemokine receptor CCR6, are an IL-17A-producing CD8 T&nbsp;cell subset commonly associated with autoimmune inflammation (<span class="dropBlock reference-citations"><a id="body-ref-sref73-1" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-1" aria-controls="sref73-1">Srenathan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref73-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Srenathan, U. ∙ Steel, K. ∙ Taams, L.S.</div><div><strong>IL-17+ CD8+ T&nbsp;cells: Differentiation, phenotype and role in inflammatory disease</strong></div><div><em>Immunol. Lett.</em> 2016; <strong>178</strong>:20-26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.imlet.2016.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84971596825" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27173097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.imlet.2016.05.001&amp;pmid=27173097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>Ccr6</i> was highly expressed in progenitor cluster C4 and overlapped with <i>Tcf7</i> expression; however, cells exhibiting other hallmarks of Tc17 cells, including <i>Rorc</i> (RORγT) and <i>Il17a</i> expression, were predominantly found adjacent to <i>Tcf7</i>-expressing cells within C4 (<a id="crosref0865" href="#fig5">Figures&nbsp;5</a>A and 5D). Monocle3 analysis of putative lineage trajectories (<span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Cao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref82-1" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-1" aria-controls="sref82-1">Trapnell et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref82-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Trapnell, C. ∙ Cacchiarelli, D. ∙ Grimsby, J. ...</div><div><strong>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:381-386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84900873950" target="_blank" aria-label="Open Scopus in new tab">Scopus (3557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2859&amp;pmid=24658644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) predicted a trajectory connecting <i>Tcf7</i>/<i>Ccr6</i>-expressing cells with the Tc17 population (<a id="crosref0880" href="#fig5">Figure&nbsp;5</a>D). Further, unsupervised clustering of TCR clonotypes revealed a group of five clonotypes, clonotype cluster 7, that predominantly contained cells spanning both the <i>Tcf7</i>/<i>Ccr6</i> and <i>Rorc</i>/<i>Il17a</i> regions of C4 (<a id="crosref0885" href="#fig5">Figures 5</a>E, <a id="crosref0890" href="#figs4">S4</a>A, and S4D). Only a small number of TCR clonotypes expressed <i>Il17a</i> and might suggest a subset of TCRs promote Tc17 differentiation (<a id="crosref0895" href="#figs4">Figure&nbsp;S4</a>A). However, the majority of these clonotypes expressed Il17a in a small proportion of cells and were distributed across multiple clusters, indicating Tc17-associated clonotypes also give rise to other cell states.</div><div id="p0130" role="paragraph">Flow cytometry analyses confirmed the presence of CCR6<sup>+</sup> SIIN- and SIY-specific cells and revealed enrichment among the SIY response (<a id="crosref0900" href="#fig5">Figure&nbsp;5</a>F). The majority of CCR6<sup>+</sup> cells expressed TCF1 as well as SLAMF6, another marker used to define TCF1<sup>+</sup> progenitor cells (<a id="crosref0905" href="#fig5">Figures 5</a>G and <a id="crosref0910" href="#figs5">S5</a>D) (<span class="dropBlock reference-citations"><a id="body-ref-sref46-6" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-6" aria-controls="sref46-6">Miller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref46-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Miller, B.C. ∙ Sen, D.R. ∙ Al Abosy, R. ...</div><div><strong>Subsets of exhausted CD8<sup>+</sup> T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:326-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0312-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061846466" target="_blank" aria-label="Open Scopus in new tab">Scopus (1090)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30778252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0312-6&amp;pmid=30778252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Most CCR6<sup>+</sup> cells also expressed RORγT and approximately half of these expressed TCF1 (<a id="crosref0920" href="#fig5">Figure&nbsp;5</a>G). SIY-specific cells produced more IL17A compared to SIIN-specific cells, especially among the RORγT<sup>+</sup> and CCR6<sup>+</sup> populations, indicating increased Tc17 differentiation of SIY-specific cells (<a id="crosref0925" href="#fig5">Figure&nbsp;5</a>H). Comparing functionality of CCR6<sup>+</sup> TCF1<sup>+</sup> versus CCR6<sup>−</sup> TCF1<sup>+</sup> cells, we found both populations proliferated similarly (Ki67), but GZMB expression was largely restricted to a small proportion of CCR6<sup>−</sup> TCF1<sup>+</sup> cells (<a id="crosref0930" href="#figs5">Figures S5</a>E and S5F). CCR6<sup>+</sup> TCF1<sup>+</sup> cells had higher expression of tolerance/anergy markers (CD200, EGR2, and CD83), inhibitory receptors (PD1, LAG3, and TIGIT), and costimulatory receptors often upregulated on exhausted cells (ICOS and OX40) compared to CCR6<sup>−</sup> TCF1<sup>+</sup> cells (<a id="crosref0935" href="#figs5">Figure&nbsp;S5</a>C). CCR6<sup>+</sup> TCF1<sup>+</sup> cells also expressed higher levels of TOX, a transcription factor closely tied with T&nbsp;cell dysfunction (<span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Scott et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Scott, A.C. ∙ Dündar, F. ∙ Zumbo, P. ...</div><div><strong>TOX is a&nbsp;critical regulator of tumour-specific T&nbsp;cell differentiation</strong></div><div><em>Nature.</em> 2019; <strong>571</strong>:270-274</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1324-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85068312747" target="_blank" aria-label="Open Scopus in new tab">Scopus (665)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31207604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1324-y&amp;pmid=31207604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Altogether, these results uncover previously undescribed heterogeneity among TCF1<sup>+</sup> progenitor cells. High expression of markers of dysfunction/tolerance, low GZMB expression, and Tc17 differentiation suggest CCR6<sup>+</sup> TCF1<sup>+</sup> cells represent an unconventional TCF1<sup>+</sup> population with reduced functionality.</div><div id="p0135" role="paragraph">Remarkably, the CCR6<sup>+</sup> subset made up nearly 40% of SIY-specific TCF1<sup>+</sup> progenitor cells, accounting for much of the enrichment of TCF1<sup>+</sup> progenitor cells among the SIY response at 5 and 8&nbsp;weeks (<a id="crosref0945" href="#fig5">Figures 5</a>I and <a id="crosref0950" href="#figs5">S5</a>G). The proportion of CCR6<sup>+</sup> TCF1<sup>+</sup> progenitor cells increased over time for the SIIN response, correlating with the kinetics of dysfunction and loss of ICB response (<a id="crosref0955" href="#fig5">Figures 5</a>I and <a id="crosref0960" href="#figs5">S5</a>G). Thus, a higher proportion of CCR6<sup>+</sup> TCF1<sup>+</sup> versus CCR6<sup>−</sup> TCF1<sup>+</sup> progenitor cells correlates with worse ICB response and suggests the CCR6<sup>+</sup> subset has reduced capacity to respond to ICB. This may explain why SIY-specific cells do not respond better than SIIN-specific cells to ICB despite enrichment for TCF1<sup>+</sup> progenitor cells. Consistent with this, the proportion of CCR6<sup>+</sup> TCF1<sup>+</sup> cells was reduced after 1&nbsp;week of anti-PD1/CTLA4 ICB treatment (<a id="crosref0965" href="#fig5">Figure&nbsp;5</a>J). Because total TCF1<sup>+</sup> cell numbers increase with ICB response (<a id="crosref0970" href="#fig4">Figure&nbsp;4</a>D), this indicates that ICB preferentially expands or recruits CCR6<sup>−</sup> TCF1<sup>+</sup> cells. Additionally, little to no expression of CX3CR1 was observed on CCR6<sup>+</sup> TCF1<sup>+</sup> cells, whereas up to 20% of CCR6<sup>−</sup> TCF1<sup>+</sup> cells expressed CX3CR1 (<a id="crosref0975" href="#fig5">Figures 5</a>K and <a id="crosref0980" href="#figs5">S5</a>H). This suggests CCR6<sup>+</sup> TCF1<sup>+</sup> cells are impaired in conventional differentiation to a cytotoxic state.</div><div id="p0140" role="paragraph">To assess the functionality of CCR6<sup>+</sup> TCF1<sup>+</sup>-derived Tc17 cells, we utilized an IL17A reporter allele to permanently mark cells that have expressed IL17A with Tomato fluorescence (<a id="crosref0985" href="#fig5">Figure&nbsp;5</a>L) (<span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Hirota et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Hirota, K. ∙ Duarte, J.H. ∙ Veldhoen, M. ...</div><div><strong>Fate mapping of IL-17-producing T&nbsp;cells in inflammatory responses</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:255-263</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-79951677394" target="_blank" aria-label="Open Scopus in new tab">Scopus (929)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21278737/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1993&amp;pmid=21278737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with the flow cytometry data, the majority of SIY-specific CCR6<sup>+</sup> and RORγT<sup>+</sup> cells expressed Tomato (<a id="crosref0995" href="#figs5">Figure&nbsp;S5</a>I). Around 30% of TCF1<sup>+</sup> cells also expressed Tomato and harbored most of the CCR6<sup>+</sup> population (<a id="crosref1000" href="#figs5">Figure&nbsp;S5</a>I). Strikingly, Tomato<sup>+</sup> cells expressed no GZMB, even following 1&nbsp;week of anti-PD-1/CTLA4 ICB therapy (<a id="crosref1005" href="#fig5">Figure&nbsp;5</a>L). This indicates cells that have expressed IL17A do not differentiate into cytotoxic cells. Further, the proportion of Tomato<sup>+</sup> cells was reduced following ICB treatment and suggests ICB reduces differentiation down the Tc17 pathway (<a id="crosref1010" href="#figs5">Figure&nbsp;S5</a>J). Altogether, these data suggest that CCR6<sup>+</sup> TCF1<sup>+</sup> and Tc17 cells constitute functionally inferior cell states that contribute poorly to therapeutic response to ICB.</div></section><section id="sec-2-6"><h3>Therapeutic vaccination breaks antigen dominance and eliminates the CCR6<sup>+</sup> subset of TCF1<sup>+</sup> progenitor cells</h3><div id="p0145" role="paragraph">CCR6 upregulation and Tc17 differentiation in autoimmune diseases is driven by reactivity to self-antigens (<span class="dropBlock reference-citations"><a id="body-ref-sref73-2" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-2" aria-controls="sref73-2">Srenathan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref73-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Srenathan, U. ∙ Steel, K. ∙ Taams, L.S.</div><div><strong>IL-17+ CD8+ T&nbsp;cells: Differentiation, phenotype and role in inflammatory disease</strong></div><div><em>Immunol. Lett.</em> 2016; <strong>178</strong>:20-26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.imlet.2016.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84971596825" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27173097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.imlet.2016.05.001&amp;pmid=27173097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, we hypothesized that suboptimal T&nbsp;cell priming conditions might induce this phenotype in tumors and sought to improve T&nbsp;cell priming conditions by therapeutic vaccination of LucOS mice with SIIN and SIY long peptides (<a id="crosref1020" href="#fig6">Figure&nbsp;6</a>A). Vaccination vastly improved the SIIN- and SIY-specific CD8 T&nbsp;cell responses in the tumor-bearing lung, increasing T&nbsp;cell expansion, proliferation (Ki67), and cytotoxicity (GZMB) (<a id="crosref1025" href="#fig6">Figures 6</a>B–6D). Greater fold expansion was observed for the SIY versus SIIN response, suggesting SIY-specific cells preferentially benefit from vaccination. Spatial profiling by tissue-based cyclic immunofluorescence (t-CyCIF) showed increased infiltration of CD8 T&nbsp;cells into tumors post-vaccination, with greater proportions expressing Ki67 and GZMB (<a id="crosref1030" href="#fig6">Figures 6</a>E–6G, <a id="crosref1035" href="#figs6">S6</a>B, and S6C). This correlated with decreased tumor size, particularly when immune cell infiltrate was excluded (<a id="crosref1040" href="#fig6">Figures 6</a>F and <a id="crosref1045" href="#figs6">S6</a>A). The effects of vaccination were antigen-specific, with no contribution of adjuvant alone (<a id="crosref1050" href="#figs6">Figure&nbsp;S6</a>D). Interestingly, little response was observed in the lung 7&nbsp;days after the initial vaccine dose (<a id="crosref1055" href="#fig6">Figures 6</a>H and <a id="crosref1060" href="#figs6">S6</a>E). However, antigen-specific T&nbsp;cells were expanded in the blood, inguinal lymph nodes, and spleen, especially for the SIY response (<a id="crosref1065" href="#fig6">Figures 6</a>I and <a id="crosref1070" href="#figs6">S6</a>F). These data indicate that vaccination acts predominantly by increasing priming and/or expanding activated cells in the periphery rather than boosting the existing response in the lung.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/8d9ceff0-dd4c-44d9-b0e4-8579bd03e413/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.08.020/asset/4bef12f6-8a06-4c83-9cde-5261625e76a7/main.assets/gr6.jpg" height="941" width="637" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/8d9ceff0-dd4c-44d9-b0e4-8579bd03e413/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Therapeutic vaccination breaks antigen dominance and eliminates the CCR6<sup>+</sup> subset of TCF1<sup>+</sup> progenitor cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0230" role="paragraph">(A) SIIN and SIY long-peptide and cyclic di-GMP vaccine. Mice were vaccinated at 6&nbsp;weeks (subcutaneous [s.c.] tail base), boosted at 8&nbsp;weeks, and analyzed 9&nbsp;weeks post-tumor initiation.</div><div id="fspara0235" role="paragraph">(B–D) Absolute number of SIIN- and SIY-specific cells (B) and the percentage expressing Ki67 (C) and GZMB (D) in PBS control (Ctrl) versus vaccine (Vax) treated mice by flow cytometry.</div><div id="fspara0240" role="paragraph">(E–G) Tissue-based cyclic immunofluorescence (t-CyCIF) imaging of lung lobes of Ctrl versus Vax-treated mice (E) and quantification of CD8 T&nbsp;cell infiltration and tumor size (F) and the percentage of Ki67- and GZMB-positive CD8 T&nbsp;cells in tumors (G). Results are averaged from all tumors from two lung lobes per mouse.</div><div id="fspara0245" role="paragraph">(H and I) Absolute number of SIIN- and SIY-specific cells in the lung (H) and the percentage in the blood (I) of Ctrl versus Vax treated mice 7&nbsp;days after the initial vaccine dose.</div><div id="fspara0250" role="paragraph">(J) Absolute number of SIIN- and SIY-specific cells in mice treated with Ctrl, Vax, SIIN-only vaccine, or SIY-only vaccine as in (A).</div><div id="fspara0255" role="paragraph">(K and L) Percentage of SIIN- and SIY-specific cells that are TCF1<sup>+</sup> TIM3<sup>−</sup> (K) and the proportion of these cells that express CCR6 (L) in Ctrl versus Vax treated mice by flow cytometry.</div><div id="fspara0260" role="paragraph">(M) Proportion of CCR6<sup>+</sup> SIIN- and SIY-specific cells expressing RORγT in Ctrl versus Vax treated mice by flow cytometry.</div><div id="fspara0265" role="paragraph">(B)–(D) and (J)–(M) are representative of ≥3 independent experiments, n ≥ 5 mice per group. (H) and (I) are compiled from 2 independent experiments.</div><div id="fspara0270" role="paragraph">Results are expressed as mean&nbsp;+ SD. Statistics were calculated by two-tailed Student’s t test: ns, not significant,&nbsp;<sup>∗</sup>p&nbsp;&lt; 0.05,&nbsp;<sup>∗∗</sup>p&nbsp;&lt; 0.01,&nbsp;<sup>∗∗∗</sup>p&nbsp;&lt; 0.001. See also <a id="crosref1980" href="#figs6">Figure&nbsp;S6</a> and <a id="crosref1985" href="#mmc5">Table S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/859b5d11-4346-4fca-ac86-b61262096fb6/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.08.020/asset/119b0925-e1e9-47cd-976d-4b55d3ec62d7/main.assets/figs6.jpg" height="941" width="746" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/859b5d11-4346-4fca-ac86-b61262096fb6/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Vaccination with SIIN and SIY long peptides increases CD8 T&nbsp;cell proliferation and cytotoxicity correlating with decreased tumor burden, related to <a id="crosref2185" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0520" role="paragraph">(A) Representative t-CyCIF images of Nkx2.1 (epithelial/tumor cells), B220 (B cells), CD4 (T helper cells), FOXP3 (Treg cells) and CD8 (T cytotoxic cells) staining in lung tumors from control (Ctrl) versus SIIN/SIY vaccine (Vax) treated KP LucOS mice.</div><div id="fspara0525" role="paragraph">(B) Quantification of CD4<sup>+</sup> T&nbsp;cells, Foxp3<sup>+</sup> CD4<sup>+</sup> Treg cells and CD20<sup>+</sup> B cells in tumors from Ctrl versus Vax treated KP LucOS mice. Results represent averages of all individual tumor data from two lung lobes per mouse.</div><div id="fspara0530" role="paragraph">(C) Tumor size measurements based on cell counts within annotated tumor area and absolute annotated tumor area as determined by analysis of H&amp;E stained slides and t-CyCIF cell counts.</div><div id="fspara0535" role="paragraph">(D) Absolute number of SIIN- and SIY-specific cells and the percentage expressing Ki67 and GZMB in mice treated with PBS versus cyclic-di-GMP adjuvant according to the vaccination regimen outlined in <a id="crosref2190" href="#fig6">Figure&nbsp;6</a>A.</div><div id="fspara0540" role="paragraph">(E) Percentage of SIIN- and SIY-specific cells that are Ki67<sup>+</sup>, GZMB<sup>+</sup> and TCF1<sup>+</sup> TIM3<sup>−</sup> in Ctrl and Vax treated mice 7&nbsp;days post-vaccination by flow cytometry. The proportion of TCF1<sup>+</sup> TIM3<sup>−</sup> cells that express CCR6 is also shown. Data are compiled from 2 independent experiments.</div><div id="fspara0545" role="paragraph">(F) Flow cytometric analysis of the proportion of SIIN- and SIY-specific cells of CD8 T&nbsp;cells in the blood, spleen, inguinal lymph node (iLN) and mediastinal lymph node (mLN) in Ctrl and Vax treated mice 7&nbsp;days post-vaccination. Fold change between Ctrl and Vax conditions in each compartment is graphed. Representative of 1-2 independent experiments.</div><div id="fspara0550" role="paragraph">(G) Flow cytometric analysis of the percentage of GZMB<sup>+</sup> cells and the percentage of TCF1<sup>+</sup> TIM3<sup>−</sup> cells expressing CCR6 among SIIN- and SIY-specific cells from mice treated with Ctrl, SIIN/SIY vaccine (Vax), SIIN-only vaccine or SIY-only vaccine, 3&nbsp;weeks after the initial vaccination (9&nbsp;weeks post-tumor initiation). Representative of 3 independent experiments, n ≥ 5 mice per group.</div><div id="fspara0555" role="paragraph">(H) Absolute number of Y5- and SIY-specific cells in mice treated with a Y5 and SIY long-peptide vaccine according to the regimen outlined in <a id="crosref2195" href="#fig6">Figure&nbsp;6</a>A. The percentage of Y5- and SIY-specific cells expressing Ki67, GZMB, and TCF1<sup>+</sup> TIM3<sup>−</sup> cells and the proportion of TCF1<sup>+</sup> TIM3<sup>−</sup> cells expressing CCR6 is shown.</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph">Comparison of the SIIN/SIY pooled vaccine to SIIN-only and SIY-only vaccines demonstrated that the benefits of vaccination were restricted to the targeted antigen (<a id="crosref1075" href="#fig6">Figures 6</a>J and <a id="crosref1080" href="#figs6">S6</a>G). However, the SIY-only vaccine was able to increase the magnitude and functionality of the SIY response to a similar extent as the SIIN/SIY vaccine (<a id="crosref1085" href="#fig6">Figures 6</a>J and <a id="crosref1090" href="#figs6">S6</a>G). This indicates that targeting a subdominant antigen alone with vaccination can be sufficient to rescue the subdominant response to the level of the dominant response. The SIY response remained enriched for TCF1<sup>+</sup> progenitor cells compared to the SIIN response after vaccination against SIIN/SIY or SIY-only (<a id="crosref1095" href="#fig6">Figure&nbsp;6</a>K and data not shown); however, CCR6<sup>+</sup> TCF1<sup>+</sup> progenitor cells were all but eliminated (<a id="crosref1100" href="#fig6">Figures 6</a>L and <a id="crosref1105" href="#figs6">S6</a>G). Among the remaining CCR6<sup>+</sup> cells, fewer expressed RORγT, suggesting reduced propensity for Tc17 differentiation (<a id="crosref1110" href="#fig6">Figure&nbsp;6</a>M). Altogether, these results provide strong evidence that CCR6 expression among TCF1<sup>+</sup> cells and Tc17 differentiation is driven by suboptimal T&nbsp;cell priming. Moreover, they demonstrate that vaccination can “break” antigen dominance, improving the overall contribution and quality of the subdominant T&nbsp;cell response.</div><div id="p0155" role="paragraph">We further assessed whether vaccination could break the antigen dominance hierarchy between SIY and Y5 in LucY5S mice. Surprisingly, vaccination expanded only the dominant SIY response whereas subdominant Y5-specific cells remained low in number (<a id="crosref1115" href="#figs6">Figure&nbsp;S6</a>H). Still, Y5-specific cells benefitted functionally, with similar increases in GZMB and Ki67 expression and depletion of CCR6<sup>+</sup> TCF1<sup>+</sup> cells compared to SIY-specific cells (<a id="crosref1120" href="#figs6">Figure&nbsp;S6</a>H). Because Y5 is much less stable on MHC compared to SIY and SIIN (see <a id="crosref1125" href="#fig3">Figure&nbsp;3</a>F), this suggests a minimum threshold of pMHC stability may be required to overcome antigen dominance in response to vaccination. Still, the functional enhancement of Y5-specific cells indicates that inclusion of unstable MHC binders in vaccines may still be beneficial overall.</div></section><section id="sec-2-7"><h3>CCR6<sup>+</sup> TCF1<sup>+</sup> progenitor and Tc17 cells are found across human cancers</h3><div id="p0160" role="paragraph">To extend our findings to human cancer, we scored clusters from the mouse scRNA-seq dataset for enrichment of gene signatures (<a id="crosref1130" href="#mmc4">Table S4</a>) derived from three published human CD8 T&nbsp;cell scRNA-seq datasets spanning lung, melanoma, colon, endometrial, and renal cancers (<span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Guo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-1" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-1" aria-controls="sref89-1">Wu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref89-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The mouse TCF1<sup>+</sup> progenitor cluster, C8, was strongly enriched for gene signatures from clusters associated with progenitor and memory cell phenotypes, most notably two clusters described as containing TCF1<sup>+</sup> progenitor cells in melanoma (Sade-Feldman CD8_4- and CD8_6-Memory/Effector) but also Guo CD8_C2-CD28 and C3-CX3CR1 and Wu 8.6.KLRB1 (<a id="crosref1150" href="#fig7">Figures 7</a>A and <a id="crosref1155" href="#figs7">S7</a>A). In contrast, mouse cluster C4, containing CCR6<sup>+</sup> TCF1<sup>+</sup> cells, showed weak to no enrichment for these signatures and was instead enriched for signatures of T&nbsp;cell exhaustion, including Sade-Feldman CD8_1-Exhaustion/CellCycle and Guo CD8_C6_LAYN (<a id="crosref1160" href="#fig7">Figures 7</a>A and <a id="crosref1165" href="#figs7">S7</a>A; <a id="crosref1170" href="#mmc4">Table S4</a>). In a reciprocal analysis, we scored the tumor CD8 T&nbsp;cell subset of the <span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Wu et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> pan-cancer dataset with gene signatures of mouse clusters C8 and C4 and found strong enrichment of C8 in the “progenitor region” of the UMAP, marked by <i>TCF7</i> expression and cells assigned to the published progenitor-like cluster, 8.6.KLRB1 (<a id="crosref1180" href="#fig7">Figures 7</a>B and <a id="crosref1185" href="#figs7">S7</a>B). The C4 signature was weakly, but more broadly enriched across the UMAP, including in regions with <i>HAVCR2</i> expression and cells assigned to published clusters with exhausted phenotypes, 8.3.Trm and 8.5.Mitosis (<a id="crosref1190" href="#fig7">Figures 7</a>B and <a id="crosref1195" href="#figs7">S7</a>B). These results suggest that both C8 and C4 cell populations resemble populations in human cancer; however, the C8 population is better aligned with progenitor cell states described in humans.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/c08187bf-0e9e-4a71-8444-a314fef839e1/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.08.020/asset/0843a143-7377-433d-8607-2b63ebb42301/main.assets/gr7.jpg" height="942" width="717" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/c08187bf-0e9e-4a71-8444-a314fef839e1/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">CCR6<sup>+</sup> TCF1<sup>+</sup> and Tc17 cells are found across human cancers</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0280" role="paragraph">(A) Enrichment of CD8 gene signatures from three human cancer scRNA-seq datasets (<a id="crosref1990" href="#mmc4">Table S4</a>) in the mouse scRNA-seq clusters from <a id="crosref1995" href="#fig2">Figure&nbsp;2</a>B. Human clusters, right; mouse clusters, top. Colored boxes indicate significant enrichment, p ≤ 0.05.</div><div id="fspara0285" role="paragraph">(B and C) UMAP of tumor CD8 T&nbsp;cells from a human pan-cancer scRNA-seq dataset (<span class="dropBlock reference-citations"><a id="body-ref-sref89-3" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-3" aria-controls="sref89-3">Wu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref89-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) showing enrichment of gene signatures derived from mouse progenitor clusters C4 and C8 (B) and expression of <i>TCF7</i>, <i>HAVCR2</i>, co-expression of <i>TCF7</i>, <i>CCR6</i>, and <i>RORC</i> and co-expression of <i>IL17A</i> with <i>TCF7</i>, <i>CCR6</i>, and/or <i>RORC</i> (C).</div><div id="fspara0290" role="paragraph">(D–F) t-CYCIF images of human lung adenocarcinoma and metastatic melanoma samples (D and E) and quantification of CD8 T&nbsp;cells expressing TCF1 and CCR6 (F).</div><div id="fspara0295" role="paragraph">(G) UMAP embedding of CD8 T&nbsp;cells from an anti-PD1 and/or anti-CTLA4 melanoma scRNA-seq dataset (<span class="dropBlock reference-citations"><a id="body-ref-sref59-3" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-3" aria-controls="sref59-3">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) showing cell assignment to published clusters CD8_1 thru CD8_6 and responder versus non-responder patients.</div><div id="fspara0300" role="paragraph">(H) Scoring of cells in (G) for enrichment of gene signatures (<a id="crosref2010" href="#mmc3">Table S3</a>) derived from mouse <i>Ccr6</i><sup>-</sup><i>Tcf7</i><sup>+</sup> and <i>Ccr6</i><sup>+</sup><i>Tcf7</i><sup>+</sup> cells.</div><div id="fspara0305" role="paragraph">(I and J) Enrichment of the gene signatures from (H) in clusters CD8_1 thru CD8_6 (I) and responders versus non-responders (J).</div><div id="fspara0310" role="paragraph">In (C) and (I)–(J), mucosal invariant T (MAIT) cells that phenotypically resemble Tc17 cells were excluded from the analysis based on TCR sequence (<a id="crosref2015" href="#mmc6">Table S6</a>; <a id="crosref2020" href="#sec-4">STAR Methods</a>).</div><div id="fspara0315" role="paragraph">Results are expressed as mean&nbsp;+ SD. Statistics were calculated by two-tailed Student’s t test: ns, not significant,&nbsp;<sup>∗</sup>p&nbsp;&lt; 0.05,&nbsp;<sup>∗∗</sup>p&nbsp;&lt; 0.01,&nbsp;<sup>∗∗∗</sup>p&nbsp;&lt; 0.001. See also <a id="crosref2025" href="#figs7">Figure&nbsp;S7</a> and <a id="crosref2030" href="#mmc5">Tables S5</a> and <a id="crosref2035" href="#mmc7">S7</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.08.020/asset/a4f2fcc6-fccf-4d44-9b0a-dc3d7130dd12/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.08.020/asset/2ea710a0-9bbc-47f0-b838-e0d0643d2893/main.assets/figs7.jpg" height="939" width="730" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/a4f2fcc6-fccf-4d44-9b0a-dc3d7130dd12/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Analysis of tumor CD8 T&nbsp;cells from published human scRNA-seq datasets and tissue samples highlights human relevance of identified mouse TCF1<sup>+</sup> progenitor cell subsets, related to <a id="crosref2200" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0565" role="paragraph">(A) Scoring of cells from the mouse SIIN versus SIY scRNA-seq data in <a id="crosref2205" href="#fig2">Figure&nbsp;2</a>A for expression of gene signatures derived from the human scRNA-seq clusters described in <a id="crosref2210" href="#fig7">Figure&nbsp;7</a>A (<span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Guo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-4" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-4" aria-controls="sref59-4">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-4" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-4" aria-controls="sref89-4">Wu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref89-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (see also <a id="crosref2230" href="#mmc4">Table S4</a>). Signatures associated with progenitor and exhausted cell phenotypes are included.</div><div id="fspara0570" role="paragraph">(B) UMAP plots of tumor-derived CD8 T&nbsp;cells from the reanalysis of the Wu et&nbsp;al. scRNA-seq dataset in <a id="crosref2235" href="#fig7">Figures 7</a>B and 7C highlighting cells assigned to select clusters from the published <span class="dropBlock reference-citations"><a id="body-ref-sref89-5" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-5" aria-controls="sref89-5">Wu et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref89-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> analysis.</div><div id="fspara0575" role="paragraph">(C) Reanalysis of the CD8 T&nbsp;cell subset from four scRNA-seq datasets derived from multiple human cancer types: non-small cell lung adenocarcinoma (NSCLC), <span class="dropBlock reference-citations"><a id="body-ref-sref21-3" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-3" aria-controls="sref21-3">Guo et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref21-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; melanoma, <span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Tirosh et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad0501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84963614956" target="_blank" aria-label="Open Scopus in new tab">Scopus (2762)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27124452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad0501&amp;pmid=27124452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and <span class="dropBlock reference-citations"><a id="body-ref-sref59-5" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-5" aria-controls="sref59-5">Sade-Feldman et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref59-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; basal cell carcinoma, <span class="dropBlock reference-citations"><a id="body-ref-sref91-1" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-1" aria-controls="sref91-1">Yost et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref91-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Yost, K.E. ∙ Satpathy, A.T. ∙ Wells, D.K. ...</div><div><strong>Clonal replacement of tumor-specific T&nbsp;cells following PD-1 blockade</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:1251-1259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0522-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069859998" target="_blank" aria-label="Open Scopus in new tab">Scopus (881)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31359002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0522-3&amp;pmid=31359002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Cells expressing <i>TCF7</i>, <i>HAVCR2, CCR6, RORC</i> and/or <i>IL17A</i> above a 0.0-0.5 log-normalized cutoff are highlighted on the respective UMAP plots. Mucosal invariant T (MAIT) cells that phenotypically resemble Tc17 cells (<span class="dropBlock reference-citations"><a id="body-ref-sref79-1" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-1" aria-controls="sref79-1">Toubal et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Toubal, A. ∙ Nel, I. ∙ Lotersztajn, S. ...</div><div><strong>Mucosal-associated invariant T&nbsp;cells and disease</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:643-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0191-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069455629" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31308521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0191-y&amp;pmid=31308521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were excluded from the analysis based on TCR sequence (<a id="crosref2270" href="#mmc6">Table S6</a>; <a id="crosref2275" href="#sec-4">STAR Methods</a>).</div><div id="fspara0580" role="paragraph">(D) Representative single channel images for the tissue cyclic immunofluorescence (t-CyCIF) imaging analysis of human lung adenocarcinoma samples in <a id="crosref2280" href="#fig7">Figure&nbsp;7</a>D.</div><div id="fspara0585" role="paragraph">(E-F) Quantification of CD8 T&nbsp;cells expressing TCF1 by t-CyCIF imaging in <a id="crosref2285" href="#fig7">Figure&nbsp;7</a>D.</div><div id="fspara0590" role="paragraph">(G) Representative flow cytometry plots of the proportion of CCR6<sup>+</sup> and TCF1<sup>+</sup> TIM3<sup>−</sup> cells staining positive for SIIN- and SIY-specific H-2K<sup>b</sup> tetramers. Representative of ≥ 3 independent experiments, n ≥ 5 mice per experiment.</div></div></div></div></figcaption></figure></div><div id="p0165" role="paragraph">In the reanalysis of the <span class="dropBlock reference-citations"><a id="body-ref-sref89-6" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-6" aria-controls="sref89-6">Wu et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref89-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> dataset, we further identified cells co-expressing <i>TCF7</i> and <i>CCR6</i>, some of which also expressed <i>RORC</i>, indicating that CCR6<sup>+</sup> TCF1<sup>+</sup> cells are indeed found in human cancer (<a id="crosref1205" href="#fig7">Figure&nbsp;7</a>C; <a id="crosref1210" href="#mmc6">Table S6</a>). <i>IL17A</i>-expressing cells, some co-expressing <i>TCF7</i>, <i>CCR6</i>, and/or <i>RORC</i>, were also found (<a id="crosref1215" href="#fig7">Figure&nbsp;7</a>C; <a id="crosref1220" href="#mmc6">Table S6</a>). Although <i>CCR6</i><sup><i>+</i></sup> <i>TCF7</i><sup><i>+</i></sup> cells were predominantly localized to the progenitor region of the UMAP, <i>IL17A</i>-expressing cells were distally localized in the region expressing <i>HAVCR2</i> and exhibiting a more differentiated phenotype. We identified similar cell populations with analogous localization patterns in four additional scRNA-seq datasets from individual cancer types, including lung cancer, melanoma, and basal cell carcinoma (<a id="crosref1225" href="#figs7">Figure&nbsp;S7</a>C) (<span class="dropBlock reference-citations"><a id="body-ref-sref21-4" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-4" aria-controls="sref21-4">Guo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref21-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref59-6" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-6" aria-controls="sref59-6">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Tirosh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad0501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84963614956" target="_blank" aria-label="Open Scopus in new tab">Scopus (2762)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27124452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad0501&amp;pmid=27124452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref91-2" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-2" aria-controls="sref91-2">Yost et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref91-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Yost, K.E. ∙ Satpathy, A.T. ∙ Wells, D.K. ...</div><div><strong>Clonal replacement of tumor-specific T&nbsp;cells following PD-1 blockade</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:1251-1259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0522-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069859998" target="_blank" aria-label="Open Scopus in new tab">Scopus (881)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31359002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0522-3&amp;pmid=31359002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We further validated the presence of CCR6<sup>+</sup> TCF1<sup>+</sup> cells in human tumor tissue by t-CyCIF imaging and found these cells made up 2% of CD8 T&nbsp;cells on average across early-stage lung adenocarcinoma and metastatic melanoma samples (<a id="crosref1250" href="#fig7">Figures 7</a>D and <a id="crosref1255" href="#figs7">S7</a>D; <a id="crosref1260" href="#mmc5">Tables S5</a> and <a id="crosref1265" href="#mmc7">S7</a>). Our observations in mouse lung adenocarcinoma indicate that although only a small percentage of TCF1<sup>+</sup> cells are antigen-experienced, the majority of CCR6<sup>+</sup> cells are contained within the activated, tumor antigen-specific response (<a id="crosref1270" href="#figs7">Figure&nbsp;S7</a>G). Hence, despite comprising a small proportion of total CD8 T&nbsp;cells, CCR6<sup>+</sup> TCF1<sup>+</sup> cells likely constitute a larger fraction of tumor-reactive CD8 T&nbsp;cells compared to CCR6<sup>−</sup> TCF1<sup>+</sup> cells in humans.</div><div id="p0170" role="paragraph">Finally, we directly investigated a correlation between CCR6<sup>+</sup> TCF1<sup>+</sup> cells and response to ICB by reanalyzing the CD8 T&nbsp;cell subset of the <span class="dropBlock reference-citations"><a id="body-ref-sref59-7" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-7" aria-controls="sref59-7">Sade-Feldman et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref59-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> melanoma dataset from patients treated with ICB. Consistent with published observations, cells derived from responder versus non-responder patients segregated spatially on the UMAP, with responder cells colocalizing with <i>TCF7</i> expression and progenitor-associated clusters CD8_4 thru CD8_6 (<a id="crosref1280" href="#fig7">Figures 7</a>G and <a id="crosref1285" href="#figs7">S7</a>C). Scoring the dataset with signatures derived from <i>CCR6</i><sup>+</sup> <i>TCF7</i><sup>+</sup> versus <i>CCR6</i><sup>−</sup> <i>TCF7</i><sup>+</sup> cells from the mouse scRNA-seq data revealed significant enrichment of the <i>CCR6</i><sup>−</sup> <i>TCF7</i><sup>+</sup> signature in clusters CD8_4 and CD8_6 and cells from responder patients (<a id="crosref1290" href="#fig7">Figures 7</a>H–7J). In contrast, the <i>CCR6</i><sup>+</sup> <i>TCF7</i><sup>+</sup> signature was enriched in exhaustion-associated cluster CD8_1, and no significant enrichment was observed in responder cells. These data indicate that CCR6<sup>−</sup> TCF1<sup>+</sup>, and not CCR6<sup>+</sup> TCF1<sup>+</sup> cells, are positively correlated with patient response to ICB. This is consistent with our findings in mouse lung adenocarcinoma and suggests CCR6<sup>+</sup> TCF1<sup>+</sup> cells contribute poorly to ICB response in human patients.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0175" role="paragraph">Neoantigens can drive potent anti-tumor T&nbsp;cell responses, yet T&nbsp;cell reactivity against predicted neoantigens is starkly limited across most cancers (<span class="dropBlock reference-citations"><a id="body-ref-sref68-2" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-2" aria-controls="sref68-2">Schumacher et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref68-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Schumacher, T.N. ∙ Scheper, W. ∙ Kvistborg, P.</div><div><strong>Cancer Neoantigens</strong></div><div><em>Annu. Rev. Immunol.</em> 2019; <strong>37</strong>:173-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-042617-053402" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85065015974" target="_blank" aria-label="Open Scopus in new tab">Scopus (374)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30550719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-042617-053402&amp;pmid=30550719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our study demonstrates that antigen dominance hierarchies in tumors may contribute to this narrowly focused CD8 T&nbsp;cell response. In KP mouse lung adenocarcinomas expressing pairs of immunogenic neoantigens, we found that CD8 T&nbsp;cell responses against different neoantigens compete and can result in the establishment of an immunodominant antigen. This profoundly impacted the response against the subdominant antigen, suppressing T&nbsp;cell expansion, differentiation, and effector function. These findings demonstrate that CD8 T&nbsp;cell responses against concurrently expressed tumor neoantigens are interdependent, and their competition can limit the overall diversity and effectiveness of the anti-tumor response.</div><div id="p0180" role="paragraph">Multiple factors may contribute to the establishment of antigen dominance hierarchies in tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Schreiber et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Schreiber, H. ∙ Wu, T.H. ∙ Nachman, J. ...</div><div><strong>Immunodominance and tumor escape</strong></div><div><em>Semin. Cancer Biol.</em> 2002; <strong>12</strong>:25-31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/scbi.2001.0401" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-0036150946" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11926408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fscbi.2001.0401&amp;pmid=11926408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90-3" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-3" aria-controls="sref90-3">Yewdell, 2006</a><div class="dropBlock__holder" data-db-target-of="sref90-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Yewdell, J.W.</div><div><strong>Confronting complexity: real-world immunodominance in antiviral CD8+ T&nbsp;cell responses</strong></div><div><em>Immunity.</em> 2006; <strong>25</strong>:533-543</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2006.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2006.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-33749528390" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17046682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2006.09.005&amp;pmid=17046682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, using the Y5 mutant of SIIN that poorly binds MHC, we demonstrated that pMHC binding plays a central role. It has previously been shown that only tumor peptides with high affinity for MHC are efficiently cross-presented (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Engels et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Engels, B. ∙ Engelhard, V.H. ∙ Sidney, J. ...</div><div><strong>Relapse or eradication of cancer is predicted by peptide-major histocompatibility complex affinity</strong></div><div><em>Cancer Cell.</em> 2013; <strong>23</strong>:516-526</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2013.03.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2013.03.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84876381248" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23597565/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2013.03.018&amp;pmid=23597565" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); hence, weak pMHC binding may limit opportunities for good interactions with antigen presenting cells (APCs). The subdominant SIY and Y5 responses in KP LucOS and LucY5S mice, respectively, were rescued when these antigens were expressed alone, despite pMHC binding being unchanged. Thus, it may be that competition between CD8 T&nbsp;cell responses for interactions with APCs amplifies a deficit in MHC binding and results in formation of antigen dominance hierarchies.</div><div id="p0185" role="paragraph">Subdominant T&nbsp;cell responses in our model were associated with enrichment of TCF1<sup>+</sup> progenitor cells that drive CD8 T&nbsp;cell responses to ICB therapy in mice and humans (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Philip and Schietinger, 2021</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Philip, M. ∙ Schietinger, A.</div><div><strong>CD8(+) T&nbsp;cell differentiation and dysfunction in cancer</strong></div><div><em>Nat. Rev. Immunol.</em> 2021;</div><div>Published online July 12, 2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-021-00574-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85109907412" target="_blank" aria-label="Open Scopus in new tab">Scopus (399)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34253904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-021-00574-3&amp;pmid=34253904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, we found that the subdominant SIY response in KP LucOS mice did not preferentially benefit from ICB therapy, possibly due to predominance of a dysfunctional subset of TCF1<sup>+</sup> cells marked by CCR6 expression. Another recent study from our lab corroborates there being heterogeneity among TCF1<sup>+</sup> cells and identifies a SLAMF6<sup>+</sup> subset, that contains CCR6<sup>+</sup> TCF1<sup>+</sup> cells, as a population recently arrived from the tumor draining lymph node (<span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Schenkel et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Schenkel, J.M. ∙ Herbst, R.H. ∙ Canner, D. ...</div><div><strong>Conventional type I dendric cells maintain areservoir of proliferative tumor-antigen specific TCF-1<sup>+</sup> CD8<sup>+</sup> T cells in tumor-draining lymph nodes</strong></div><div><em>Immunity.</em> 2021;</div><div>–</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2021.08.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2021.08.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85115016953" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34534439/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2021.08.026&amp;pmid=34534439" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Analysis of human scRNA-seq datasets spanning multiple cancer types and imaging of human lung adenocarcinoma and melanoma samples indicates this heterogeneity is reflected across human&nbsp;cancers (this study and <span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Schenkel et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Schenkel, J.M. ∙ Herbst, R.H. ∙ Canner, D. ...</div><div><strong>Conventional type I dendric cells maintain areservoir of proliferative tumor-antigen specific TCF-1<sup>+</sup> CD8<sup>+</sup> T cells in tumor-draining lymph nodes</strong></div><div><em>Immunity.</em> 2021;</div><div>–</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2021.08.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2021.08.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85115016953" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34534439/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2021.08.026&amp;pmid=34534439" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, although CCR6<sup>−</sup> TCF1<sup>+</sup> cells were correlated with melanoma patient response to ICB therapy, no correlation was observed for CCR6<sup>+</sup> TCF1<sup>+</sup> cells. Therefore, CCR6<sup>+</sup> TCF1<sup>+</sup> cells appear to contribute poorly to ICB response in both mice and humans.</div><div id="p0190" role="paragraph">In KP LucOS mice, we identified a lineage trajectory from CCR6<sup>+</sup> TCF1<sup>+</sup> cells to a Tc17 population. Tc17 cells have previously been reported in cancer, but their role remains unclear. Although Tc17 cells have been reported to mediate tumor control in some transplant tumor settings, their production of IL17A may also contribute to tumor-promoting inflammation (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Chang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Chang, S.H. ∙ Mirabolfathinejad, S.G. ∙ Katta, H. ...</div><div><strong>T helper 17 cells play a critical pathogenic role in lung cancer</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:5664-5669</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1319051111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84898777944" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24706787/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1319051111&amp;pmid=24706787" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-3" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-3" aria-controls="sref73-3">Srenathan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref73-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Srenathan, U. ∙ Steel, K. ∙ Taams, L.S.</div><div><strong>IL-17+ CD8+ T&nbsp;cells: Differentiation, phenotype and role in inflammatory disease</strong></div><div><em>Immunol. Lett.</em> 2016; <strong>178</strong>:20-26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.imlet.2016.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84971596825" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27173097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.imlet.2016.05.001&amp;pmid=27173097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Zhao et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Zhao, J. ∙ Chen, X. ∙ Herjan, T. ...</div><div><strong>The role of interleukin-17 in tumor development and progression</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2020; <strong>217</strong>:e20190297</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20190297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31727782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20190297&amp;pmid=31727782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we found that CD8 T&nbsp;cells marked by an&nbsp;IL17A reporter did not express GZMB, even in response to ICB therapy, suggesting Tc17 cells do not contribute to cytotoxic responses against tumors. Furthermore, our lab previously showed that treatment of KP lung adenocarcinoma with a neutralizing antibody to IL17A significantly reduces tumor burden by inhibiting recruitment of tumor-promoting neutrophils (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Jin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Jin, C. ∙ Lagoudas, G.K. ∙ Zhao, C. ...</div><div><strong>Commensal Microbiota Promote Lung Cancer Development via γδ T Cells</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:998-1013.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.040&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.040&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061787031" target="_blank" aria-label="Open Scopus in new tab">Scopus (623)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30712876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.040&amp;pmid=30712876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly, in humans, tumor infiltration by myeloid-derived suppressor cells correlates with accumulation of IL17A-producing cells (<span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Wu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Wu, P. ∙ Wu, D. ∙ Ni, C. ...</div><div><strong>γδT17 cells promote the accumulation and expansion of myeloid-derived suppressor cells in human colorectal cancer</strong></div><div><em>Immunity.</em> 2014; <strong>40</strong>:785-800</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2014.03.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2014.03.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84900482064" target="_blank" aria-label="Open Scopus in new tab">Scopus (468)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24816404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2014.03.013&amp;pmid=24816404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Zhuang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Zhuang, Y. ∙ Peng, L.S. ∙ Zhao, Y.L. ...</div><div><strong>CD8(+) T&nbsp;cells that produce interleukin-17 regulate myeloid-derived suppressor cells and are associated with survival time of patients with gastric cancer</strong></div><div><em>Gastroenterology.</em> 2012; <strong>143</strong>:951</div><div>62.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1053%2Fj.gastro.2012.06.010&amp;cf=fulltext&amp;site=ygast-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1053%2Fj.gastro.2012.06.010&amp;cf=pdf&amp;site=ygast-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84866733757" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22710190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1053%2Fj.gastro.2012.06.010&amp;pmid=22710190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, in addition to contributing poorly to ICB&nbsp;response, CCR6<sup>+</sup> TCF1<sup>+</sup> cell differentiation to a Tc17 phenotype may also promote tumorigenesis through IL17A production.</div><div id="p0195" role="paragraph">Tc17 differentiation is predominantly observed in autoimmune inflammation and is driven by autoreactive TCR signals (<span class="dropBlock reference-citations"><a id="body-ref-sref73-4" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-4" aria-controls="sref73-4">Srenathan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref73-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Srenathan, U. ∙ Steel, K. ∙ Taams, L.S.</div><div><strong>IL-17+ CD8+ T&nbsp;cells: Differentiation, phenotype and role in inflammatory disease</strong></div><div><em>Immunol. Lett.</em> 2016; <strong>178</strong>:20-26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.imlet.2016.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84971596825" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27173097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.imlet.2016.05.001&amp;pmid=27173097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, we hypothesized that CCR6<sup>+</sup> TCF1<sup>+</sup> cells might be derived from suboptimal TCR stimulation during priming and differentiation. Consistent with this, vaccination of KP LucOS mice initially expanded cells in the blood and peripheral lymphoid organs, especially for the subdominant SIY response. TCF1<sup>+</sup> cells were also significantly expanded in&nbsp;the lung following the vaccine boost; however, the CCR6<sup>+</sup> subset was lost. Increased antigen presentation with vaccination likely alleviates the constraints of low-affinity/-stability pMHC binding, improving TCR stimulation and reducing differentiation of CCR6<sup>+</sup> TCF1<sup>+</sup> cells. An increasingly immunosuppressive tumor microenvironment may further contribute to poor TCR stimulation as tumors progress and could explain why SIIN-specific cells develop a CCR6<sup>+</sup> TCF1<sup>+</sup> phenotype over time.</div><div id="p0200" role="paragraph">Vaccination of KP LucOS mice dramatically improved the magnitude and functionality of tumor-reactive CD8 T&nbsp;cells, and the response to vaccination was correlated with a reduction in&nbsp;tumor burden. Likewise, early clinical studies have demonstrated the therapeutic potential of neoantigen vaccines in melanoma, where they improve T&nbsp;cell priming and tumor control (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Hollingsworth and Jansen, 2019</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hollingsworth, R.E. ∙ Jansen, K.</div><div><strong>Turning the corner on therapeutic cancer vaccines</strong></div><div><em>NPJ Vaccines.</em> 2019; <strong>4</strong>:7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41541-019-0103-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061253209" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30774998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41541-019-0103-y&amp;pmid=30774998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Vaccination has shown a remarkable ability to elicit <i>de novo</i> T&nbsp;cell responses (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Ott et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Ott, P.A. ∙ Hu, Z. ∙ Keskin, D.B. ...</div><div><strong>An immunogenic personal neoantigen vaccine for patients with melanoma</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:217-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22991" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85024377220" target="_blank" aria-label="Open Scopus in new tab">Scopus (1967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22991&amp;pmid=28678778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Sahin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Sahin, U. ∙ Derhovanessian, E. ∙ Miller, M. ...</div><div><strong>Personalized RNA mutanome vaccines mobilize poly-specific therapeutic immunity against cancer</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:222-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85024396858" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678784/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23003&amp;pmid=28678784" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and, in combination with anti-PD1 therapy, also promotes epitope spreading to neoantigens not included in the vaccine (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Ott et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Ott, P.A. ∙ Hu-Lieskovan, S. ∙ Chmielowski, B. ...</div><div><strong>A Phase Ib Trial of Personalized Neoantigen Therapy Plus Anti-PD-1 in Patients with Advanced Melanoma, Non-small Cell Lung Cancer, or Bladder Cancer</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:347-362.e24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2020.08.053&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2020.08.053&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85092455165" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33064988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.053&amp;pmid=33064988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore, vaccination against a range of predicted immunogenic neoantigens may be effective at priming new T&nbsp;cell responses even in cancers with few pre-treatment T&nbsp;cell-reactive epitopes. Our data suggest the presence of CCR6<sup>+</sup> TCF1<sup>+</sup> cells is indicative of subdominant and/or poorly primed T&nbsp;cell responses in tumors. Consequently, CCR6<sup>+</sup> TCF1<sup>+</sup> cells might mark tumors that would preferentially benefit from vaccination and should be further explored as a biomarker of patient response to vaccine therapies.</div><div id="p0205" role="paragraph">pMHC affinity is used for identification of neoepitopes for cancer vaccines. High-affinity neoepitopes are typically chosen (<span class="dropBlock reference-citations"><a id="body-ref-sref49-3" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-3" aria-controls="sref49-3">Ott et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref49-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Ott, P.A. ∙ Hu, Z. ∙ Keskin, D.B. ...</div><div><strong>An immunogenic personal neoantigen vaccine for patients with melanoma</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:217-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22991" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85024377220" target="_blank" aria-label="Open Scopus in new tab">Scopus (1967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22991&amp;pmid=28678778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and may selectively target immunodominant T&nbsp;cell responses. Our results indicate subdominant T&nbsp;cell responses against some lower pMHC affinity neoantigens may experience greater benefit from vaccination and remain enriched for TCF1<sup>+</sup> progenitor cells. Because TCF1<sup>+</sup> cells repopulate the cytotoxic T&nbsp;cell pool and are ICB-responsive, engaging subdominant T&nbsp;cell responses through vaccination might result in more durable tumor control and better response to ICB. Overall, these findings provide rationale for evaluating the relative response to high versus low pMHC affinity/stability antigens in clinical trials of pooled neoantigen cancer vaccines, where subdominant antigens may contribute more to tumor control than previously realized.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0210" role="paragraph">In this study, neoantigen pairs were expressed clonally in tumors, and the expression level of each antigen was held constant. Human tumors are generally thought to be more heterogenous, containing variable numbers of neoantigens expressed at different levels and with varying clonality. These factors may influence hierarchical ordering of antigens and could be explored in future studies by modifying antigen number, expression level, and clonal fraction.</div><div id="p0215" role="paragraph">In the mouse scRNA-seq analysis, we mapped a differentiation trajectory of CCR6<sup>+</sup> TCF1<sup>+</sup> cells to a Tc17 population that lacks cytotoxic capacity, even after ICB. However, because lineage tracing by adoptive transfer is technically intractable in the KP lung cancer model, we were unable to directly delineate differentiation of CCR6<sup>+</sup> TCF1<sup>+</sup> cells. Use of novel genetic alleles, such as a fate-mapping CCR6 reporter, or investigation in other models may shed further light on the relative functionality and plasticity of CCR6<sup>+</sup> TCF1<sup>+</sup> cells in response to ICB.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0225" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td><i>InVivo</i>mAb PD1</td><td>BioXCell</td><td>Clone 29F.1A12; Cat# BE0273; RRID: <a id="interref275" href="https://antibodyregistry.org/AB_2687796" target="_blank">AB_2687796</a></td></tr><tr><td><i>InVivo</i>mAb CTLA4</td><td>BioXCell</td><td>Clone 9H10; Cat# BE0131; RRID: <a id="interref280" href="https://antibodyregistry.org/AB_10950184" target="_blank">AB_10950184</a></td></tr><tr><td><i>InVivo</i>mAb rat IgG2a</td><td>BioXCell</td><td>Clone 2A3; Cat# BE0089; RRID: <a id="interref285" href="https://antibodyregistry.org/AB_1107769" target="_blank">AB_1107769</a></td></tr><tr><td><i>InVivo</i>mAb polyclonal Syrian Hamster IgG</td><td>BioXCell</td><td>Cat# BE0087; RRID: <a id="interref290" href="https://antibodyregistry.org/AB_1107782" target="_blank">AB_1107782</a></td></tr><tr><td>FC: CD8ɑ</td><td>BD Biosciences</td><td>Clone 53-6.7; Cat# 563786, 612759; RRID: <a id="interref295" href="https://antibodyregistry.org/AB_2732919" target="_blank">AB_2732919</a>, <a id="interref300" href="https://antibodyregistry.org/AB_2870090" target="_blank">AB_2870090</a></td></tr><tr><td>FC: CCR6 (CD196)</td><td>BioLegend</td><td>Clone 29-2L17; Cat# 129822; RRID: <a id="interref305" href="https://antibodyregistry.org/AB_2687019" target="_blank">AB_2687019</a></td></tr><tr><td>FC: CX3CR1</td><td>BioLegend</td><td>Clone SA011F11; Cat# 149031; RRID: <a id="interref310" href="https://antibodyregistry.org/AB_2565939" target="_blank">AB_2565939</a></td></tr><tr><td>FC: CXCR3 (CD183)</td><td>BD Biosciences</td><td>Clone CXCR3-173; Cat# 562937, 741032; RRID: <a id="interref315" href="https://antibodyregistry.org/AB_2687551" target="_blank">AB_2687551</a>, <a id="interref320" href="https://antibodyregistry.org/AB_2740650" target="_blank">AB_2740650</a></td></tr><tr><td>FC: CD44</td><td>BD Biosciences; BioLegend</td><td>Clone IM7; Cat# 563736, 612799, 103057, 103059; RRID: <a id="interref325" href="https://antibodyregistry.org/AB_2738395" target="_blank">AB_2738395</a>, <a id="interref330" href="https://antibodyregistry.org/AB_2870126" target="_blank">AB_2870126</a>, <a id="interref335" href="https://antibodyregistry.org/AB_2564214" target="_blank">AB_2564214</a>, <a id="interref340" href="https://antibodyregistry.org/AB_2571953" target="_blank">AB_2571953</a></td></tr><tr><td>FC: CD45</td><td>ThermoFisher Scientific; BD Biosciences</td><td>Clone 30-F11; Cat# 47-0451-80, 562420; RRID: <a id="interref345" href="https://antibodyregistry.org/AB_1548790" target="_blank">AB_1548790</a>, <a id="interref350" href="https://antibodyregistry.org/AB_11154401" target="_blank">AB_11154401</a></td></tr><tr><td>FC: CD83</td><td>BioLegend</td><td>Clone Michel-19; Cat# 121506; RRID: <a id="interref355" href="https://antibodyregistry.org/AB_572009" target="_blank">AB_572009</a></td></tr><tr><td>FC: CD200</td><td>BD Biosciences</td><td>Clone OX-90; Cat# 565546; RRID: <a id="interref360" href="https://antibodyregistry.org/AB_2739288" target="_blank">AB_2739288</a></td></tr><tr><td>FC: EGR2</td><td>ThermoFisher Scientific</td><td>Clone erongr2; Cat# 17-6691-80; RRID: <a id="interref365" href="https://antibodyregistry.org/AB_11150966" target="_blank">AB_11150966</a></td></tr><tr><td>FC: Granzyme B</td><td>BioLegend, BD Biosciences</td><td>Clone GB11; Cat# 515408, 560213; RRID: <a id="interref370" href="https://antibodyregistry.org/AB_2562196" target="_blank">AB_2562196</a>, <a id="interref375" href="https://antibodyregistry.org/AB_1645453" target="_blank">AB_1645453</a></td></tr><tr><td>FC: H-2K<sup>b</sup></td><td>ThermoFisher Scientific</td><td>Clone AF6-88.5.5.3; Cat# 17-5958-82; RRID: <a id="interref380" href="https://antibodyregistry.org/AB_1311280" target="_blank">AB_1311280</a></td></tr><tr><td>FC: ICOS (CD278)</td><td>ThermoFisher Scientific</td><td>Clone 7E.17G9; Cat# 46-9942-80; RRID: <a id="interref385" href="https://antibodyregistry.org/AB_2744727" target="_blank">AB_2744727</a></td></tr><tr><td>FC: IFN-γ</td><td>BioLegend</td><td>Clone XMG1.2; Cat# 505822; RRID: <a id="interref390" href="https://antibodyregistry.org/AB_961359" target="_blank">AB_961359</a></td></tr><tr><td>FC: IL17A</td><td>ThermoFisher Scientific</td><td>Clone 17B7; Cat# 11-7177-81; RRID: <a id="interref395" href="https://antibodyregistry.org/AB_763581" target="_blank">AB_763581</a></td></tr><tr><td>FC: IL7R (CD127)</td><td>ThermoFisher Scientific</td><td>Clone A7R34; Cat# 25-1271-82; RRID: <a id="interref400" href="https://antibodyregistry.org/AB_469649" target="_blank">AB_469649</a></td></tr><tr><td>FC: Ki67</td><td>BD Biosciences</td><td>Clone B56; Cat# 561277, 563756; RRID: <a id="interref410" href="https://antibodyregistry.org/AB_10611571" target="_blank">AB_10611571</a>, <a id="interref405" href="https://antibodyregistry.org/AB_2732007" target="_blank">AB_2732007</a></td></tr><tr><td>FC: LAG3 (CD233)</td><td>BD Biosciences, ThermoFisher Scientific; BioLegend</td><td>Clone C9B7W; Cat# 741820, 46-2231-82, 11-2231-80, 125219; RRID: <a id="interref425" href="https://antibodyregistry.org/AB_11151334" target="_blank">AB_11151334</a>, <a id="interref420" href="https://antibodyregistry.org/AB_2572483" target="_blank">AB_2572483</a>, <a id="interref415" href="https://antibodyregistry.org/AB_2566571" target="_blank">AB_2566571</a></td></tr><tr><td>FC: OX40 (CD134)</td><td>BioLegend</td><td>Clone OX-86; Cat# 119419; RRID: <a id="interref430" href="https://antibodyregistry.org/AB_2687175" target="_blank">AB_2687175</a></td></tr><tr><td>FC: PD1 (CD279)</td><td>BioLegend</td><td>Clone RMP1-30; Cat# 109110, 109121; RRID: <a id="interref440" href="https://antibodyregistry.org/AB_572017" target="_blank">AB_572017</a>, <a id="interref435" href="https://antibodyregistry.org/AB_2687080" target="_blank">AB_2687080</a></td></tr><tr><td>FC: RORƔT</td><td>ThermoFisher Scientific</td><td>Clone B2D; Cat# 25-6981-82, 17-6981-82; RRID: <a id="interref445" href="https://antibodyregistry.org/AB_2784671" target="_blank">AB_2784671</a>, <a id="interref450" href="https://antibodyregistry.org/AB_2573254" target="_blank">AB_2573254</a></td></tr><tr><td>FC: SLAMF6 (LY-108)</td><td>BD Biosciences</td><td>Clone 13G3; Cat# 742272; RRID: <a id="interref455" href="https://antibodyregistry.org/AB_2871448" target="_blank">AB_2871448</a></td></tr><tr><td>FC: TCF1/TCF7</td><td>Cell Signaling Technology</td><td>Clone C63D9; Cat# 6444, 6709, 9066; RRID: <a id="interref470" href="https://antibodyregistry.org/AB_2797627" target="_blank">AB_2797627</a>, <a id="interref465" href="https://antibodyregistry.org/AB_2797631" target="_blank">AB_2797631</a>, <a id="interref460" href="https://antibodyregistry.org/AB_2797696" target="_blank">AB_2797696</a></td></tr><tr><td>FC: TIGIT</td><td>BD Biosciences</td><td>Clone 1G9; Cat# 565474; RRID: <a id="interref475" href="https://antibodyregistry.org/AB_2739254" target="_blank">AB_2739254</a></td></tr><tr><td>FC: TIM3</td><td>BioLegend</td><td>Clone RMT3-23; Cat# 119721, 119725; RRID: <a id="interref485" href="https://antibodyregistry.org/AB_2616907" target="_blank">AB_2616907</a>, <a id="interref480" href="https://antibodyregistry.org/AB_2716066" target="_blank">AB_2716066</a></td></tr><tr><td>FC: TNFα</td><td>BioLegend</td><td>Clone MP6-XT22; Cat# 506306; RRID: <a id="interref490" href="https://antibodyregistry.org/AB_315427" target="_blank">AB_315427</a></td></tr><tr><td>FC: TOX</td><td>Miltenyi Biotec</td><td>Clone REA473; Cat# 130-118-335; RRID: <a id="interref495" href="https://antibodyregistry.org/AB_2751485" target="_blank">AB_2751485</a></td></tr><tr><td>FC: H-2Kb SIINFEKL monomer</td><td>NIH Tetramer Core Facility</td><td>Custom</td></tr><tr><td>FC: H-2Kb SIYRYYGL monomer</td><td>NIH Tetramer Core Facility</td><td>Custom</td></tr><tr><td>FC: H-2Kb SIINYEKL monomer</td><td>NIH Tetramer Core Facility</td><td>Custom</td></tr><tr><td>FC: H-2Kb mALG8 monomer</td><td>NIH Tetramer Core Facility</td><td>Custom</td></tr><tr><td>FC: H-2Kb mLAMA4 monomer</td><td>NIH Tetramer Core Facility</td><td>Custom</td></tr><tr><td>Hashtag: TotalSeq-C anti-mouse</td><td>BioLegend</td><td>Clones M1/42, 30-F11; Cat# 155861, 155863, 155865, 155867, 155869, 155871, 155873, 155875, 155877, 155879; RRID: <a id="interref545" href="https://antibodyregistry.org/AB_2800693" target="_blank">AB_2800693</a>, <a id="interref540" href="https://antibodyregistry.org/AB_2800694" target="_blank">AB_2800694</a>, <a id="interref535" href="https://antibodyregistry.org/AB_2800695" target="_blank">AB_2800695</a>, <a id="interref530" href="https://antibodyregistry.org/AB_2800696" target="_blank">AB_2800696</a>, <a id="interref525" href="https://antibodyregistry.org/AB_2800697" target="_blank">AB_2800697</a>, <a id="interref520" href="https://antibodyregistry.org/AB_2819910" target="_blank">AB_2819910</a>, <a id="interref515" href="https://antibodyregistry.org/AB_2819911" target="_blank">AB_2819911</a>, <a id="interref510" href="https://antibodyregistry.org/AB_2819912" target="_blank">AB_2819912</a>, <a id="interref505" href="https://antibodyregistry.org/AB_2819913" target="_blank">AB_2819913</a>, <a id="interref500" href="https://antibodyregistry.org/AB_2819914" target="_blank">AB_2819914</a></td></tr><tr><td>tCycif: anti-mouse B220 (CD45R)</td><td>ThermoFisher Scientific</td><td>Clone RA3-6B2; Cat# 41-0452-80; RRID: <a id="interref550" href="https://antibodyregistry.org/AB_2573598" target="_blank">AB_2573598</a></td></tr><tr><td>tCycif: anti-mouse CD4</td><td>ThermoFisher Scientific</td><td>Clone 4SM95; Cat# 41-9766-82; RRID: <a id="interref555" href="https://antibodyregistry.org/AB_2573637" target="_blank">AB_2573637</a></td></tr><tr><td>tCycif: anti-mouse CD8α</td><td>Cell Signaling Technology</td><td>Clone D4W2Z; Cat# 98941S; RRID: <a id="interref560" href="https://antibodyregistry.org/AB_2756376" target="_blank">AB_2756376</a></td></tr><tr><td>tCycif: anti-mouse FOXP3</td><td>ThermoFisher Scientific</td><td>Clone FJK-16 s; Cat# 11-5773-82; RRID: <a id="interref565" href="https://antibodyregistry.org/AB_465243" target="_blank">AB_465243</a></td></tr><tr><td>tCycif: anti-human αSMA</td><td>Abcam</td><td>Clone EPR5368; Cat# ab202509; RRID: <a id="interref570" href="https://antibodyregistry.org/AB_2868435" target="_blank">AB_2868435</a></td></tr><tr><td>tCycif: anti-human CD4</td><td>R&amp;D Systems</td><td>Cat# FAB8165G; RRID: <a id="interref575" href="https://antibodyregistry.org/AB_2728839" target="_blank">AB_2728839</a></td></tr><tr><td>tCycif: anti-human CD8α</td><td>ThermoFisher Scientific</td><td>Clone AMC908; Cat# 50-0008-80; RRID: <a id="interref580" href="https://antibodyregistry.org/AB_2574148" target="_blank">AB_2574148</a></td></tr><tr><td>tCycif: anti-human CD20</td><td>ThermoFisher Scientific</td><td>Clone L26; Cat# 50-0202-80; RRID: <a id="interref585" href="https://antibodyregistry.org/AB_11151691" target="_blank">AB_11151691</a></td></tr><tr><td>tCycif: anti-human CD45</td><td>BioLegend</td><td>Clone HI30; Cat# 304008; RRID: <a id="interref590" href="https://antibodyregistry.org/AB_314396" target="_blank">AB_314396</a></td></tr><tr><td>tCycif: anti-human CCR6</td><td>Abcam</td><td>Clone EPR22259; Cat# ab243852; RRID: <a id="interref595" href="https://antibodyregistry.org/AB_2860033" target="_blank">AB_2860033</a></td></tr><tr><td>tCycif: anti-human FOXP3</td><td>eBioscience</td><td>Clone 236A/E7; Cat# 41-4777-80; RRID: <a id="interref600" href="https://antibodyregistry.org/AB_2573608" target="_blank">AB_2573608</a></td></tr><tr><td>tCycif: anti-human TCF1</td><td>Cell Signaling Technology</td><td>Clone C63D9; Cat# 6444S; RRID: <a id="interref605" href="https://antibodyregistry.org/AB_2797627" target="_blank">AB_2797627</a></td></tr><tr><td>tCycif: anti-human TTF1 (also reacts with mouse)</td><td>Abcam</td><td>Clone EPR595(2); Cat# ab206726; RRID: <a id="interref610" href="https://antibodyregistry.org/AB_2857980" target="_blank">AB_2857980</a></td></tr><tr><td>Purification: Ly6G Rat Monoclonal</td><td>BioLegend</td><td>Clone 1A8; Cat# 127602; RRID: <a id="interref615" href="https://antibodyregistry.org/AB_1089180" target="_blank">AB_1089180</a></td></tr><tr><td>Purification: EpCam (CD326) Rat Monoclonal</td><td>BioLegend</td><td>Clone G8.8; Cat# 118202; RRID: <a id="interref620" href="https://antibodyregistry.org/AB_1089027" target="_blank">AB_1089027</a></td></tr><tr class="rowsep"><td>Purification: F4/80 Rat Monoclonal</td><td>BioLegend</td><td>Clone BM8; Cat# 123102; RRID: <a id="interref625" href="https://antibodyregistry.org/AB_893506" target="_blank">AB_893506</a></td></tr><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr><td>Human formalin fixed paraffin embedded tissue samples from lung adenocarcinoma cases</td><td>Partners Healthcare Institutional Review Board at Brigham Health, Boston, MA, USA</td><td>Excess tissue, discarded tissue protocol number 2018P001627</td></tr><tr class="rowsep"><td>Human formalin fixed paraffin embedded tissue samples from metastatic melanoma cases</td><td>Specialized Histopathology Services - Longwood Core Brigham and Womens Hospital, Boston, MA, USA</td><td>Project: 05-042 - T Jacks Lab slides</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Collagenase IV</td><td>Worthington Biochemical</td><td>Cat# LS004189</td></tr><tr><td>DNase I</td><td>Sigma-Aldrich</td><td>Cat# 10104159001</td></tr><tr><td>Monensin</td><td>BioLegend</td><td>Cat# 420701</td></tr><tr><td>Golgi Plug</td><td>BD Biosciences</td><td>Cat# 555028</td></tr><tr><td>Phorbol-12-myristate-13-acetate (PMA)</td><td>EMD Millipore</td><td>Cat# 524400</td></tr><tr><td>Ionomycin Calcium Salt</td><td>Sigma</td><td>Cat# 13909</td></tr><tr><td>Streptavidin, allophycocyanin conjugate</td><td>Invitrogen</td><td>Cat# S32362</td></tr><tr><td>Streptavidin, R-phycoerythrin conjugate</td><td>Invitrogen</td><td>Cat# S21388</td></tr><tr><td>Mirus TransIT LT1</td><td>Mirus Bio</td><td>Cat# MIR 2300</td></tr><tr><td>SYPRO Orange Protein Gel Stain</td><td>Thermo Fisher Scientific</td><td>Cat# S6651</td></tr><tr><td>Cyclic-di-GMP</td><td>Invitrogen</td><td>Cat# tlrl-nacdg</td></tr><tr><td>SIINFEKL peptide</td><td>New England Peptide</td><td>Custom</td></tr><tr><td>SIYRYYGL peptide</td><td>New England Peptide</td><td>Custom</td></tr><tr><td>SIINYEKL peptide</td><td>New England Peptide</td><td>Custom</td></tr><tr><td>ITYTWTRL peptide (mALG8)</td><td>Genscript</td><td>Custom</td></tr><tr><td>VGFNFRTL peptide (mLAMA4)</td><td>Genscript</td><td>Custom</td></tr><tr><td>SMLVLLPDEVSGLEQLE<span data-style="underline">SIINYEKL</span>TEWTS</td><td>New England Peptide</td><td>Custom</td></tr><tr><td>SMLVLLPDEVSGLEQLE<span data-style="underline">SIINFEKL</span>TEWTS peptide</td><td>New England Peptide</td><td>Custom</td></tr><tr><td>GRCVGSEQLE<span data-style="underline">SIYRYYGL</span>LLKERSEQKLIS peptide</td><td>New England Peptide</td><td>Custom</td></tr><tr><td>Empty peptide-receptive H-2K<sup>b</sup> molecules</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Saini et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Saini, S.K. ∙ Tamhane, T. ∙ Anjanappa, R. ...</div><div><strong>Empty peptide-receptive MHC class I molecules for efficient detection of antigen-specific T&nbsp;cells</strong></div><div><em>Sci. Immunol.</em> 2019; <strong>4</strong>:eaau9039</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aau9039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069751787" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31324690/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aau9039&amp;pmid=31324690" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Generated in-house</td></tr><tr><td>Zombie Fixable Viability Kit</td><td>BioLegend</td><td>Cat# 423102</td></tr><tr class="rowsep"><td>Ghost Dye</td><td>Tonbo Biosciences</td><td>Cat# 13-0865-T500</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Intracellular Fixation &amp; Permeabilization Buffer Set Kit</td><td>ThermoFisher Scientific</td><td>Cat# 88-8824-00</td></tr><tr><td>Dynabead Sheep Anti-Rat IgG</td><td>ThermoFisher Scientific</td><td>Cat#11035</td></tr><tr><td>Chromium Next GEM Single Cell 5′ Library and Gel Bead Kit</td><td>10X Genomics</td><td>Cat# 000167</td></tr><tr><td>Chromium Single Cell V(D)J Enrichment Kit, Mouse T Cell</td><td>10X Genomics</td><td>Cat#1000071</td></tr><tr class="rowsep"><td>Easysep Dead Cell Removal (Annexin V) Kit</td><td>StemCell</td><td>Cat#17899</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Mouse KP LucOS SIIN versus SIY scRNA-seq</td><td>GEO</td><td>GEO: <a id="intref0010" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164177" target="_blank">GSE164177</a></td></tr><tr><td>Sade-Feldman et&nbsp;al. scRNA-seq dataset</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59-8" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-8" aria-controls="sref59-8">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="intref0015" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120575" target="_blank">GSE120575</a></td></tr><tr><td>Wu et&nbsp;al. scRNA-seq dataset</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref89-7" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-7" aria-controls="sref89-7">Wu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref89-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>GEO: <a id="intref0020" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139555" target="_blank">GSE139555</a></td></tr><tr><td>Guo et&nbsp;al. scRNA-seq dataset</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref21-5" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-5" aria-controls="sref21-5">Guo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref21-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="intref0025" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99254" target="_blank">GSE99254</a></td></tr><tr><td>Yost et&nbsp;al. scRNA-seq dataset</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref91-3" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-3" aria-controls="sref91-3">Yost et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref91-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Yost, K.E. ∙ Satpathy, A.T. ∙ Wells, D.K. ...</div><div><strong>Clonal replacement of tumor-specific T&nbsp;cells following PD-1 blockade</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:1251-1259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0522-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069859998" target="_blank" aria-label="Open Scopus in new tab">Scopus (881)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31359002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0522-3&amp;pmid=31359002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>GEO: <a id="intref0030" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123813" target="_blank">GSE123813</a></td></tr><tr><td>Tirosh et&nbsp;al. scRNA-seq dataset</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref78-3" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-3" aria-controls="sref78-3">Tirosh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref78-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad0501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84963614956" target="_blank" aria-label="Open Scopus in new tab">Scopus (2762)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27124452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad0501&amp;pmid=27124452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="intref0035" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056" target="_blank">GSE72056</a>; Broad Data Use Oversight System: DUOS-000002</td></tr><tr><td>Mouse and human single cell RNA-sequencing analysis code</td><td>This Paper</td><td>GitHub: <a id="intref0040" href="https://doi.org/10.5281/zenodo.5110770">https://doi.org/10.5281/zenodo.5110770</a></td></tr><tr class="rowsep"><td>t-CYCIF analysis code</td><td>This Paper</td><td>GitHub: <a id="intref0045" href="https://doi.org/10.5281/zenodo.5104869">https://doi.org/10.5281/zenodo.5104869</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>GreenGo 3TZ for lentiviral titering</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">Sánchez-Rivera et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Sánchez-Rivera, F.J. ∙ Papagiannakopoulos, T. ∙ Romero, R. ...</div><div><strong>Rapid modelling of cooperating genetic events in cancer through somatic genome editing</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:428-431</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13906" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84923118778" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25337879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13906&amp;pmid=25337879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Available upon request</td></tr><tr><td>293FS<sup>∗</sup> viral packaging cell line</td><td>This paper</td><td>Available upon request</td></tr><tr class="rowsep"><td>RMA-S TAP-deficient cell line</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Schumacher et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Schumacher, T.N. ∙ Heemels, M.T. ∙ Neefjes, J.J. ...</div><div><strong>Direct binding of peptide to empty MHC class I molecules on intact cells and in&nbsp;vitro</strong></div><div><em>Cell.</em> 1990; <strong>62</strong>:563-567</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2F0092-8674%2890%2990020-F&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2F0092-8674%2890%2990020-F&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-0025006862" target="_blank" aria-label="Open Scopus in new tab">Scopus (418)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2199065/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2890%2990020-F&amp;pmid=2199065" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Available upon request</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr><td>Mouse: B6.129S4-<i>Kras<sup>tm4Tyj</sup></i>/J</td><td>Jackson Laboratories</td><td>Stock No: 008179</td></tr><tr><td>Mouse: B6.129P2-<i>Trp53<sup>tm1Brn</sup></i>/J</td><td>Jackson Laboratories</td><td>Stock No: 008462</td></tr><tr><td>Mouse: <i>Il17a<sup>tm1.1(icre)Stck</sup></i>/J</td><td>Jackson Laboratories</td><td>Stock No: 016879</td></tr><tr class="rowsep"><td>Mouse: B6.Cg-<i>Gt(ROSA)26Sor<sup>tm14(CAG-tdTomato)Hze</sup></i>/J</td><td>Jackson Laboratories</td><td>Stock No: 007914</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>Plasmid: Lenti-LucOS</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref15-4" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-4" aria-controls="sref15-4">DuPage et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref15-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>DuPage, M. ∙ Cheung, A.F. ∙ Mazumdar, C. ...</div><div><strong>Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression</strong></div><div><em>Cancer Cell.</em> 2011; <strong>19</strong>:72-85</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-78651451436" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21251614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2010.11.011&amp;pmid=21251614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene Cat# 22777</td></tr><tr><td>Plasmid: Lenti-LucSIY</td><td>This Paper</td><td>Addgene Cat# 174044</td></tr><tr><td>Plasmid: Lenti-LucSIIN</td><td>This Paper</td><td>Addgene Cat# 174043</td></tr><tr><td>Plasmid: Lenti-LucY5S</td><td>This Paper</td><td>Addgene Cat# 174045</td></tr><tr><td>Plasmid: Lenti-LucY5</td><td>This Paper</td><td>Addgene Cat# 174046</td></tr><tr><td>Plasmid: Lenti-LucA</td><td>This paper</td><td>Addgene Cat# 174047</td></tr><tr><td>Plasmid: Lenti-LucL</td><td>This paper</td><td>Addgene Cat# 174048</td></tr><tr class="rowsep"><td>Plasmid: Lenti-LucAL</td><td>This paper</td><td>Addgene Cat# 174049</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>FlowJo</td><td>FlowJo</td><td>Version 10.7.1 <a id="intref0050" href="https://www.flowjo.com/solutions/flowjo/downloads" target="_blank">https://www.flowjo.com/solutions/flowjo/downloads</a></td></tr><tr><td>Aperio ImageScope</td><td>Leica Biosystems</td><td>Version 12 <a id="intref0055" href="https://www.leicabiosystems.com" target="_blank">https://www.leicabiosystems.com</a></td></tr><tr><td>ImageJ</td><td>NIH</td><td><a id="intref0060" href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a></td></tr><tr><td>ImageJ BaSiC Plugin</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Peng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Peng, T. ∙ Thorn, K. ∙ Schroeder, T. ...</div><div><strong>A BaSiC tool for background and shading correction of optical microscopy images</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14836" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85020638067" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28594001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14836&amp;pmid=28594001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0065" href="https://www.helmholtz-muenchen.de/icb/research/groups/marr-lab/software/basic/index.html" target="_blank">https://www.helmholtz-muenchen.de/icb/research/groups/marr-lab/software/basic/index.html</a></td></tr><tr><td>ASHLAR</td><td>The Python Package Index</td><td><a id="intref0070" href="https://pypi.org/project/ashlar/" target="_blank">https://pypi.org/project/ashlar/</a></td></tr><tr><td>ilastik</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Berg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Berg, S. ∙ Kutra, D. ∙ Kroeger, T. ...</div><div><strong>ilastik: interactive machine learning for (bio)image analysis</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1226-1232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0582-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85075429634" target="_blank" aria-label="Open Scopus in new tab">Scopus (1467)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0582-9&amp;pmid=31570887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="https://www.ilastik.org/download.html" target="_blank">https://www.ilastik.org/download.html</a></td></tr><tr><td>MATLAB</td><td>MathWorks</td><td>version 2018a <a id="intref0080" href="https://www.mathworks.com/products/matlab.html" target="_blank">https://www.mathworks.com/products/matlab.html</a></td></tr><tr><td>R/Rstudio</td><td>Rstudio</td><td>version 1.2.5042 <a id="intref0085" href="https://www.rstudio.com/" target="_blank">https://www.rstudio.com/</a></td></tr><tr><td>Seurat</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>version 4.0.0 <a id="intref0090" href="https://cran.r-project.org/web/packages/Seurat/index.html" target="_blank">https://cran.r-project.org/web/packages/Seurat/index.html</a></td></tr><tr><td>CellRanger</td><td>10X Genomics</td><td>version 3.1.0 <a id="intref0095" href="https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome" target="_blank">https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome</a></td></tr><tr><td>ComplexHeatmap</td><td>CRAN <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Gu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Gu, M. ∙ Paloma, J. ∙ Berger, J.O.</div><div><strong>RobustGaSP: Robust Gaussian Stochastic Process Emulation, 2016. R package version 0.5.7</strong></div><div>2016</div><div><a id="intref0255" href="https://CRANR-projectorg/package=RobustGaSP" target="_blank">https://CRANR-projectorg/package=RobustGaSP</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.GuJ.PalomaJ.O.BergerRobustGaSP%3A+Robust+Gaussian+Stochastic+Process+Emulation%2C+2016.+R+package+version+0.5.72016https%3A%2F%2FCRANR-projectorg%2Fpackage%3DRobustGaSP" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>version 2.4.3 <a id="intref0100" href="https://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html</a></td></tr><tr><td>Tidyr</td><td>CRAN</td><td>version 1.1.2 <a id="intref0105" href="https://tidyr.tidyverse.org" target="_blank">https://tidyr.tidyverse.org</a>, <a id="intref0110" href="https://github.com/tidyverse/tidyr" target="_blank">https://github.com/tidyverse/tidyr</a></td></tr><tr><td>Cowplot</td><td>CRAN</td><td>version 1.1.0 <a id="intref0115" href="https://wilkelab.org/cowplot/" target="_blank">https://wilkelab.org/cowplot/</a></td></tr><tr><td>GGplot2</td><td>CRAN</td><td>version 3.2.2 <a id="intref0120" href="https://ggplot2.tidyverse.org" target="_blank">https://ggplot2.tidyverse.org</a>, <a id="intref0125" href="https://github.com/tidyverse/ggplot2" target="_blank">https://github.com/tidyverse/ggplot2</a></td></tr><tr><td>GridExtra</td><td>CRAN</td><td>version 2.3 <a id="intref0130" href="https://cran.r-project.org/web/packages/gridExtra/index.html" target="_blank">https://cran.r-project.org/web/packages/gridExtra/index.html</a></td></tr><tr><td>Plyr</td><td>CRAN</td><td>version 1.8.6 <a id="intref0135" href="https://cran.r-project.org/web/packages/plyr/index.html" target="_blank">https://cran.r-project.org/web/packages/plyr/index.html</a></td></tr><tr><td>Dplyr</td><td>CRAN</td><td>version 1.0.2 <a id="intref0140" href="https://cran.r-project.org/web/packages/dplyr/index.html" target="_blank">https://cran.r-project.org/web/packages/dplyr/index.html</a></td></tr><tr><td>Monocle3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref82-2" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-2" aria-controls="sref82-2">Trapnell et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref82-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Trapnell, C. ∙ Cacchiarelli, D. ∙ Grimsby, J. ...</div><div><strong>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:381-386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84900873950" target="_blank" aria-label="Open Scopus in new tab">Scopus (3557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2859&amp;pmid=24658644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>version 0.2.3.0 <a id="intref0145" href="https://cole-trapnell-lab.github.io/monocle3/" target="_blank">https://cole-trapnell-lab.github.io/monocle3/</a></td></tr><tr><td>Patchwork</td><td>CRAN</td><td>version 1.1.1 <a id="intref0150" href="https://github.com/thomasp85/patchwork" target="_blank">https://github.com/thomasp85/patchwork</a></td></tr><tr><td>Viridis</td><td>CRAN</td><td>version 0.5.1 <a id="intref0155" href="https://github.com/sjmgarnier/viridis" target="_blank">https://github.com/sjmgarnier/viridis</a></td></tr><tr><td>Circlize</td><td>CRAN</td><td>Version: 0.4.11 <a id="intref0160" href="https://github.com/jokergoo/circlize" target="_blank">https://github.com/jokergoo/circlize</a></td></tr><tr><td>Stringr</td><td>CRAN</td><td>Version: 1.4.0 <a id="intref0165" href="https://stringr.tidyverse.org" target="_blank">https://stringr.tidyverse.org</a>, <a id="intref0170" href="https://github.com/tidyverse/stringr" target="_blank">https://github.com/tidyverse/stringr</a></td></tr><tr><td>Philentropy</td><td>CRAN</td><td>Version: 0.4.0 <a id="intref0175" href="https://github.com/drostlab/philentropy" target="_blank">https://github.com/drostlab/philentropy</a></td></tr><tr><td>Dendextend</td><td>CRAN</td><td>Version: 1.14.0 <a id="intref0180" href="https://cran.r-project.org/web/packages/dendextend/index.html" target="_blank">https://cran.r-project.org/web/packages/dendextend/index.html</a></td></tr><tr><td>FactoEXTRA</td><td>CRAN</td><td>Version: 1.0.7 <a id="intref0185" href="http://www.sthda.com/english/rpkgs/factoextra" target="_blank">http://www.sthda.com/english/rpkgs/factoextra</a></td></tr><tr><td>ProjecTILs</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref3-4" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-4" aria-controls="sref3-4">Andreatta et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref3-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>v.0.5.1 <a id="intref0190" href="https://github.com/carmonalab/ProjecTILs" target="_blank">https://github.com/carmonalab/ProjecTILs</a></td></tr><tr><td>gplots</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Warnes, 2015</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Warnes, G.</div><div><strong>gplots: Various R programming tools for plotting data. R Package</strong></div><div>2015</div><div><a id="intref0270" href="https://rdrr.io/cran/gplots/" target="_blank">https://rdrr.io/cran/gplots/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.Warnesgplots%3A+Various+R+programming+tools+for+plotting+data.+R+Package2015https%3A%2F%2Frdrr.io%2Fcran%2Fgplots%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>v.3.1.0 <a id="intref0195" href="https://cran.r-project.org/web/packages/gplots/index.html" target="_blank">https://cran.r-project.org/web/packages/gplots/index.html</a></td></tr><tr><td>RColorBrewer</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Neuwirth, 2014</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Neuwirth, E.</div><div><strong>RColorBrewer: ColorBrewer Palettes. R package version 1.1-2</strong></div><div>2014</div><div><a id="intref0265" href="https://rdrr.io/cran/RColorBrewer/man/ColorBrewer.html" target="_blank">https://rdrr.io/cran/RColorBrewer/man/ColorBrewer.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.NeuwirthRColorBrewer%3A+ColorBrewer+Palettes.+R+package+version+1.1-22014https%3A%2F%2Frdrr.io%2Fcran%2FRColorBrewer%2Fman%2FColorBrewer.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>v.1.1-2 <a id="intref0200" href="https://cran.r-project.org/web/packages/RColorBrewer/index.html" target="_blank">https://cran.r-project.org/web/packages/RColorBrewer/index.html</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900994-6" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0230" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Tyler Jacks (<a id="intref0205" href="mailto:tjacks@mit.edu">tjacks@mit.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0235" role="paragraph">Plasmids generated in this study have been deposited to Addgene: Lenti-LucOS (Cat. No. 22777), Lenti-LucSIIN (Cat. No. 174043), Lenti-LucSIY (Cat. No. 174044), Lenti-LucY5 (Cat. No. 174046), Lenti-LucY5S (Cat. No. 174045), Lenti-LucAL (Cat. No. 174049), Lenti-LucA (Cat. No. 174047), Lenti-LucL (Cat. No. 174048).</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Mice</h4><div id="p0240" role="paragraph"><i>Kras</i><sup>LSL-G12D/+</sup>;<i>p53</i><sup>fl/fl</sup> (KP) and IL17a<sup>Cre</sup>;Rosa26<sup>LSL-tdTomato</sup> mice have been previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref15-5" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-5" aria-controls="sref15-5">DuPage et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref15-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>DuPage, M. ∙ Cheung, A.F. ∙ Mazumdar, C. ...</div><div><strong>Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression</strong></div><div><em>Cancer Cell.</em> 2011; <strong>19</strong>:72-85</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-78651451436" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21251614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2010.11.011&amp;pmid=21251614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Hirota et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Hirota, K. ∙ Duarte, J.H. ∙ Veldhoen, M. ...</div><div><strong>Fate mapping of IL-17-producing T&nbsp;cells in inflammatory responses</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:255-263</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-79951677394" target="_blank" aria-label="Open Scopus in new tab">Scopus (929)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21278737/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1993&amp;pmid=21278737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Madisen et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Madisen, L. ∙ Zwingman, T.A. ∙ Sunkin, S.M. ...</div><div><strong>A robust and high-throughput Cre reporting and characterization system for the whole mouse brain</strong></div><div><em>Nat. Neurosci.</em> 2010; <strong>13</strong>:133-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-73949140059" target="_blank" aria-label="Open Scopus in new tab">Scopus (4610)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20023653/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2467&amp;pmid=20023653" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and were on a C57BL/6 genetic background. Male and female KP mice were used in all experiments and experimental arms were gender and age-matched within 3&nbsp;weeks. 8&nbsp;week old, female IL17a<sup>Cre</sup>;Rosa26<sup>LSL-tdTomato</sup> were used for the collection of bone marrow to reconstitute lethally irradiated KP mice. All studies were performed under an animal protocol approved by the Massachusetts Institute of Technology (MIT) Committee on Animal Care. Mice were assessed for morbidity according to guidelines set by the MIT Division of Comparative Medicine and were humanely sacrificed prior to natural expiration.</div></section><section id="sec-4-3-2"><h4>Lentiviral Tumor Induction</h4><div id="p0245" role="paragraph">2.5&nbsp;× 10<sup>4</sup> PFU of lentivirus containing <i>Cre</i> recombinase and model neoantigens were injected intratracheally into KP mice as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">DuPage et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>DuPage, M. ∙ Dooley, A.L. ∙ Jacks, T.</div><div><strong>Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:1064-1072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2009.95" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-68149157175" target="_blank" aria-label="Open Scopus in new tab">Scopus (641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19561589/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2009.95&amp;pmid=19561589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref15-6" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" aria-label="Reference 2011" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-6" aria-controls="sref15-6">2011</a><div class="dropBlock__holder" data-db-target-of="sref15-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>DuPage, M. ∙ Cheung, A.F. ∙ Mazumdar, C. ...</div><div><strong>Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression</strong></div><div><em>Cancer Cell.</em> 2011; <strong>19</strong>:72-85</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-78651451436" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21251614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2010.11.011&amp;pmid=21251614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mice were at least 8&nbsp;weeks old at the time of injection. Details of the lentiviral constructs and lentivirus production are included below. Mice were randomized post-infection for therapy trials.</div></section><section id="sec-4-3-3"><h4>Bone Marrow Chimeras</h4><div id="p0250" role="paragraph">Bone marrow was harvested from the femur and tibia of 8&nbsp;week-old female IL17a<sup>Cre</sup>;Rosa26<sup>LSL-tdTomato</sup> mice into sterile RPMI 1640 media. Total bone marrow was pelleted by centrifugation at 1200&nbsp;rpm for 5&nbsp;minutes, resuspended in PBS and filtered through a 70&nbsp;μm cell strainer. 1x10<sup>7</sup> cells in 200 ul PBS were injected retroorbitally into lethally irradiated KP mice (two doses of 550 rad, three hours apart). 6&nbsp;weeks after bone marrow transfer, mice were injected intratracheally with Lenti-LucOS lentivirus as described above.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Lentiviral Constructs and Lentivirus Production</h4><div id="p0255" role="paragraph">The LucOS lentiviral construct was previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref15-7" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-7" aria-controls="sref15-7">DuPage et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref15-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>DuPage, M. ∙ Cheung, A.F. ∙ Mazumdar, C. ...</div><div><strong>Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression</strong></div><div><em>Cancer Cell.</em> 2011; <strong>19</strong>:72-85</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-78651451436" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21251614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2010.11.011&amp;pmid=21251614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The LucSIIN and LucSIY vectors were generated by deleting the 24 nucleotides of SIINFEKL or SIYRYYGL coding sequence, respectively, from the LucOS vector. The LucY5S vector was generated by creating a point mutation at amino acid position 5 in SIINFEKL in the LucOS construct. To make the LucY5 vector, the 24 nucleotides of SIYRYYGL coding sequence were deleted from LucY5S. The LucA and LucL vectors were generated by fusing the mALG8 or mLAMA4 neoepitope sequence plus approximately 6 flanking amino acids to luciferase. These sequences were cloned in tandom to create LucAL, with care to avoid creation of additional potentially immunogenic epitopes.</div><div id="p0260" role="paragraph">Lentiviruses were produced in 293FS<sup>∗</sup> cells by transfection with the lentiviral constructs above, psPAX2 and VSV-G packaging plasmids at a 4:3:1 ratio using Mirus TransIT LT1 (MirusBio). At 48- and 72-hours post-transfection, viral supernatant was collected, passed through 0.45&nbsp;μm filters and concentrated by ultracentrifugation (25,000&nbsp;rpm for 2 hours at 4°C). Concentrated lentivirus was resuspended in a 1:1 solution of Opti-MEM and HBSS and stored at −80°C. Viral titers were determined using the GreenGo 3TZ cell line as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">Sánchez-Rivera et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Sánchez-Rivera, F.J. ∙ Papagiannakopoulos, T. ∙ Romero, R. ...</div><div><strong>Rapid modelling of cooperating genetic events in cancer through somatic genome editing</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:428-431</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13906" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84923118778" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25337879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13906&amp;pmid=25337879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-2"><h4>Anti-PD1/anti-CTLA4 Therapy</h4><div id="p0265" role="paragraph">KP LucOS mice were treated for one week with <i>Invivo</i>mAb anti-PD1 (29F.1A12; BioXCell) and <i>Invivo</i>mAb anti-CTLA4 (9H10; BioXCell) monoclonal antibodies or <i>Invivo</i> isotype controls (Rag IgG2a, 2A3; Syrian Hamster, polyclonal; BioXCell). An initial dose of 200&nbsp;μg each of anti-PD1 and anti-CTLA4 (or isotype control) was injected i.p. at day 0, followed by 200&nbsp;μg anti-PD1 and 100&nbsp;μg anti-CTLA (or isotype control) at day 3 and 6. All doses were delivered i.p. in 200&nbsp;μL of PBS. Mice were analyzed one day following the last dose.</div></section><section id="sec-4-4-3"><h4>Therapeutic Vaccination</h4><div id="p0270" role="paragraph">KP LucOS mice were vaccinated via subcutaneous tail-based injection with 30 amino acid peptides containing SIINFEKL, SIYRYYGL and/or SIINYEKL (10&nbsp;nmol; New England Peptide) and cyclic di-GMP adjuvant (0.25&nbsp;mg/ml; Invitrogen) 6&nbsp;weeks post-tumor initiation. A booster injection was given 14&nbsp;days later and mice were euthanized 9&nbsp;weeks post-tumor initiation. All doses were delivered in two 50&nbsp;μL boluses and control mice were injected with PBS. The long peptide sequences used were: SMLVLLPDEVSGLEQLE<span data-style="underline">SIINFEKL</span>TEWTS, GRCVGSEQLE<span data-style="underline">SIYRYYGL</span>LLKERSEQKLIS and SMLVLLPDEVSGLEQLE<span data-style="underline">SIINYEKL</span>TEWTS (New England Peptide).</div><div id="p0275" role="paragraph">For assessment of SIIN-specific TCR cross-reactivity with Y5, wild-type mice were vaccinated against SIINFEKL as described above.</div></section><section id="sec-4-4-4"><h4>Tissue Collection and Flow Cytometry</h4><div id="p0280" role="paragraph">To distinguish lung tissue-resident versus circulating immune cells, mice were injected retroorbitally with a fluorescently-conjugated anti-CD45 antibody (PE-CF594 or AlexaFluor780; 30-F11; BD Bioscience) 2-3&nbsp;minutes prior to euthanasia (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Anderson et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Anderson, K.G. ∙ Mayer-Barber, K. ∙ Sung, H. ...</div><div><strong>Intravascular staining for discrimination of vascular and tissue leukocytes</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:209-222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84891707105" target="_blank" aria-label="Open Scopus in new tab">Scopus (541)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.005&amp;pmid=24385150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Circulating CD8 T&nbsp;cells staining positive for the CD45 antibody were excluded from our analyses. Lung tissue was collected and minced by hand with spring scissors and incubated in 125&nbsp;U/mL collagenase IV (Worthington Biochemical) and 40&nbsp;U/mL DNase I (Sigma-Aldrich) for 30&nbsp;minutes at 37°C. Following incubation, the tissue was dissociated using the m_lung_2.0.1 protocol on a gentleMACS Dissociator using gentleMACS C tubes (Miltenyi Biotec) and passed through a 70&nbsp;μm cell strainer. Spleen and lymph nodes were dissociated through a 70&nbsp;μm cell strainer into RPMI 1640 media containing 1% heat-inactivated fetal bovine serum. Cell suspensions were pelleted by centrifugation (1200&nbsp;rpm for 5&nbsp;minutes), resuspended in 1X RBC Lysis Buffer (eBioscience) and incubated on ice for 10&nbsp;minutes to lyse red blood cells. Pellets were resuspended in PBS and single cell suspensions were transferred to a 96-well U-bottom plate. Cells were then stained with a fixable viability dye to exclude dead cells (20&nbsp;minutes on ice; Zombie Dye; Invitrogen; Tonbo Ghost Dye; Tonbo Biosciences) and resuspended in FACS buffer (1% heat-inactivated FBS in PBS) and stained with the following surface antibodies for 15-30&nbsp;minutes on ice: CD8ɑ (53-6.7), CCR6 (29-2L17), CX3CR1 (SA011F11), CXCR3 (CXCR3-173), CD44 (IM7), IL7R (A7R34), LAG3 (C9B7W), PD1 (RMP1-30), SLAMF6 (13G3), TIGIT (1G9), TIM3 (RMT3-23), OX40 (OX-86), ICOS (7E.17G9), CD200 (OX-90), CD83 (Michel-19) purchased from ThermoFisher Scientific, BD Biosciences or Biolegend (see <a id="crosref1510" href="#sec-4-1">Key resources table</a>). Cells were simultaneously stained with H-2K<sup>b</sup> peptide-MHC tetramers specific to SIINFEKL, SIYRYYGL, SIINYEKL, mALG8 or mLAMA4 (monomer, NIH Tetramer Core Facility; PE and APC streptavidin, Invitrogen). For intracellular staining, cells were fixed for 1 hour at room temperature using the eBioscience Fixation/Permeabilization Kit (ThermoFisher Scientific). Cells were then stained overnight at at 4°C with the following antibodies: TCF1/TCF7 (C63D9), RORɣT (B2D), TBET (eBio4B10), Granzyme B (GB11), Ki67 (B56), TOX (REA473), EGR2 (erongr2) purchased from Cell Signaling Technology, ThermoFisher Scientific, BD Biosciences, Biolegend or Miltenyi Biotec (see <a id="crosref1515" href="#sec-4-1">Key resources table</a>). Samples were analyzed on a BD Biosciences LSR Fortessa or LSR II Flow Cytometry Analyzer.</div><div id="p0285" role="paragraph">For assaying cytokine production, T&nbsp;cells were enriched in lung samples by depletion of tumor cells and myeloid cells. Cells were incubated with 2&nbsp;μg each of purified Ly-6G, EpCAM and F4/80 antibodies (Biolegend; 4°C for 20&nbsp;minutes), then 125&nbsp;μL of sheep anti-rat Dynabeads (Invitrogen; 4°C for 30&nbsp;minutes with rotation). Note, to avoid depleting intravascularly labeled CD45+ cells, a mouse anti-mouse CD45.2 antibody (Brilliant Violet 510, Biolegend) was used for the intravascular stain. A Dynabeads magnet (Invitrogen) was applied to remove the beads and the T&nbsp;cell-containing supernatant was transferred to a clean tube, washed with 10&nbsp;mL FACS buffer and pelleted by centrifugation (1200&nbsp;rpm for 5&nbsp;minutes). Pellets were resuspended in T&nbsp;cell media (RPMI 1640 containing 10% FBS, 10&nbsp;mM HEPES, 1&nbsp;mM sodium pyruvate, 1X MEM Non-essential amino acids, 2&nbsp;mM L-glutamine, 0.275&nbsp;mM beta-mercaptoethanol, and 50&nbsp;U/mL penicillin-streptomycin) and transferred to a 96-well U-bottom plate. To assay IFNƔ and TNFɑ production, samples were pelleted by centrifugation (1200&nbsp;rpm for 5&nbsp;minutes) and were resuspended in T&nbsp;cell media plus 1X Monensin (Biolegend), 1X Golgi Plug (BD Bioscience) and SIINFEKL or SIYRYYGL peptide (167&nbsp;nM; New England Peptide). For IL17A staining, cells were resuspended in T&nbsp;cell media containing PMA (2.5&nbsp;ng/ml; EMD Millipore) and Ionomycin (1&nbsp;μM; Sigma-Aldrich), 1X Monensin and 1X Golgi Plug. Samples were then incubated at 37°C for 4-5 hours. Ten percent of each sample was aliquoted and left unstimulated as a control. Cells were subjected to viability staining, surface staining and fixation as described above and stained with the following antibodies overnight at 4°C: IFN-ɣ (XMG1.2), TNFɑ (MP6-XT22) and IL17A (17B7) from ThermoFisher Scientific, BD Biosciences or Biolegend (see <a id="crosref1520" href="#sec-4-1">Key resources table</a>).</div></section><section id="sec-4-4-5"><h4>scRNA-seq on Mouse CD8&nbsp;T Cells</h4><div id="p0290" role="paragraph">Lung single cell suspensions were prepared from 10 mice as described above. Dead cells were removed with an EasySep Dead Cell Removal Kit (Annexin V; StemCell). Cells were then stained with 1&nbsp;μg TotalSeq-C hashtag antibodies 1-10 (M1/42, 30-F11; Biolegend) and surface antibodies against CD44 (IM7), CD8ɑ (53-6.7) and H-2K<sup>b</sup>-SIINFEKL and H-2K<sup>b</sup>-SIYRYYGL tetramers for 30&nbsp;minutes on ice. FACS sorting was performed on a FACS Aria III (BD Biosciences) and CD44+ CD8α+ SIINFEKL+ and CD44+ CD8α+ SIYRYYGL+ T&nbsp;cells were sorted into separate tubes, counted and resuspended in PBS (no calcium or magnesium) with 0.04% BSA. Approximately 65,000 SIINFEKL+ cells and 14,000 SIYRYYGL+ cells were loaded across two channels of a Chromium single-cell 5′ chip (10X Genomics) according to manufacturer’s instructions. Single cells were partitioned into droplets with gel beads to form emulsions, after which cellular lysis and barcoded reverse transcription of mRNA was performed. Paired 5′ gene expression, hashtag barcode and V(D)J libraries were prepared using the following kits from 10X Genomics and protocols provided by the manufacturer: Chromium Next GEM Single Cell 5′ Library and Gel Bead Kit v1.1; Chromium Single Cell V(D)J Enrichment Kit, Mouse T Cell. RNA expression libraries were sequenced individually using HiSeq X (Illumina). VDJ and hashtag barcode libraries were sequenced with HiSeq 2500 (Illumina).</div></section><section id="sec-4-4-6"><h4>Peptide Stabilization of H-2K<sup>b</sup> on RMA-S Cells</h4><div id="p0295" role="paragraph">TAP-deficient RMA-S cells were cultured in RPMI 1640 media (10% fetal bovine serum, 2&nbsp;mM L-glutamine, 10&nbsp;mM HEPES and&nbsp;50&nbsp;U/μl Penicillin-Streptomycin) at 26°C overnight (16 hours) to increase expression of empty MHC molecules on the cell surface.&nbsp;Cells were then plated at 100,000 cells per well in a 96-well U-bottom plate (150&nbsp;μL per well). 10-fold serial dilutions of&nbsp;peptides (10&nbsp;mM in DMSO stock) were generated in RPMI media at 4X final concentration in a 96-well U-bottom plate. 50&nbsp;μL of peptide was added to the RMA-S cells, bringing the final volume to 200&nbsp;μl. Cells were incubated with peptide for 2 hours at 26°C to allow for peptide binding followed by 1 hour at 37°C to degrade empty MHC molecules. Each condition was plated in triplicate. Cells&nbsp;were washed in FACS buffer (1% heat-inactivated FBS in PBS) and stained with an APC-conjugated H-2K<sup>b</sup> (AF6-88.5.5.3;&nbsp;ThermoFisher Scientific) antibody for 15&nbsp;minutes on ice. Plates were assayed on a BD Biosciences LSR II flow cytometry analyzer.</div></section><section id="sec-4-4-7"><h4>Differential Scanning Fluorimetry Assay for pMHC Thermal Melting</h4><div id="p0300" role="paragraph">Empty peptide-receptive H-2K<sup>b</sup> molecules (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Saini et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Saini, S.K. ∙ Tamhane, T. ∙ Anjanappa, R. ...</div><div><strong>Empty peptide-receptive MHC class I molecules for efficient detection of antigen-specific T&nbsp;cells</strong></div><div><em>Sci. Immunol.</em> 2019; <strong>4</strong>:eaau9039</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aau9039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069751787" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31324690/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aau9039&amp;pmid=31324690" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were diluted to 200&nbsp;μg/ml in 50&nbsp;mM HEPES pH 7.5 containing 1X SYPRO Orange dye. 18&nbsp;μL of this solution was combined with 2&nbsp;μL of 100&nbsp;μM peptide (diluted in PBS) in triplicate in a 384 well quantitative PCR plate. The plate was incubated on ice for 10&nbsp;minutes to allow for peptide-MHC interaction. The plate was then assayed on a Roche LightCycler 480 instrument. Fluorescence was acquired with excitation at 465&nbsp;nM and emission at 580&nbsp;nM (SYPRO Orange filter), Samples were equilibrated at 25°C for 1&nbsp;min and then heated with a gradient of 0.1°C/s from 25°C to 95°C with 6 fluorescence readings for every 1°C. Melting curves were normalized to the minimum and maximum fluorescence value and the inverse of the first derivative (-dF/dT) was then calculated. The minimum value of -dF/dT was taken at the T<sub>m</sub>.</div></section><section id="sec-4-4-8"><h4>Histology and H&amp;E Staining</h4><div id="p0305" role="paragraph">Tumor-bearing lung lobes and a portion of the spleen were fixed in 4% paraformaldehyde overnight and embedded in paraffin. Hematoxylin and eosin stain (H&amp;E stain) was performed with a standard method by the Hope Babette Tang Histology Facility at Koch Institute. Digitally scanned images of H&amp;E slides were obtained with an Aperio ScanScope AT2 at 20X magnification and tumor area was annotated using Aperio ImageScope software.</div></section><section id="sec-4-4-9"><h4>t-CyCIF Staining and Imaging</h4><div id="p0310" role="paragraph">The t-CyCIF experimental protocol was conducted as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Du et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Du, Z. ∙ Lin, J.R. ∙ Rashid, R. ...</div><div><strong>Qualifying antibodies for image-based immune profiling and multiplexed tissue imaging</strong></div><div><em>Nat. Protoc.</em> 2019; <strong>14</strong>:2900-2930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0206-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85072716410" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534232/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0206-y&amp;pmid=31534232" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Lin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lin, J.R. ∙ Izar, B. ∙ Wang, S. ...</div><div><strong>Highly multiplexed immunofluorescence imaging of human tissues and tumors using t-CyCIF and conventional optical microscopes</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e31657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.31657" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85052091363" target="_blank" aria-label="Open Scopus in new tab">Scopus (369)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29993362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.31657&amp;pmid=29993362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, the mouse and human lung adenocarcinoma and human melanoma FFPE slides were baked at 60°C for 30&nbsp;min, dewaxed using Bond Dewax Solution (Leica Biosystems) at 72°C, and antigen retrieval was performed with Epitope Retrieval 1 Solution (Leica Biosystems) at 100°C for 20&nbsp;minutes using the BOND RX Automated IHC/ISH Stainer (Leica Biosystems). All antibodies were diluted in Odyssey Intercept Buffer (plus Hoechst 33342 0.25&nbsp;μg/mL; LI-COR Biosciences) and incubated overnight at 4°C in the dark. See the <a id="crosref1540" href="#sec-4-1">Key resources table</a> for the complete list of antibodies. Slides were coverslipped using 20%–50% glycerol solution (Sigma-Aldrich) in PBS. Images were taken using DAPI, FITC, Cy3, and Cy5 channels on the RareCyte CyteFinder Instrument (20x/0.75NA objective lens). After imaging, the fluorophores were inactivated with photobleaching solution (4.5% H2O2 and 20&nbsp;mM NaOH in PBS) for 45&nbsp;minutes under LED ligfFigurehts.</div></section><section id="sec-4-4-10"><h4>t-CyCIF Image Processing</h4><div id="p0315" role="paragraph">The image processing of tissue cyclic immunofluorescence is organized in the following steps, each of which is described in detail below:<div id="olist0010" role="list"><div id="o0010" role="listitem"><div class="label">i)</div><div class="content"><div id="p0320" role="paragraph">the software ASHLAR is used to stitch, register, and correct for image acquisition artifacts (using the BaSiC algorithm). The output of ASHLAR is a single pyramid ome.tiff file for each region imaged;</div></div></div><div id="o0015" role="listitem"><div class="label">ii)</div><div class="content"><div id="p0325" role="paragraph">the ome.tiff file is re-cut into tiles (typically 5000x5000 pixels) containing only the highest resolution image for all channels. One random cropped image (250x250 pixels) per tile is outputted for segmentation training (using ImageJ/Fiji);</div></div></div><div id="o0020" role="listitem"><div class="label">iii)</div><div class="content"><div id="p0330" role="paragraph">using the ilastik software the labeling of nuclear, cytoplasmic and background areas are trained on the cropped images. Based on the user training the Ilastik software outputs a 3-color RGB image with label probabilities;</div></div></div><div id="o0025" role="listitem"><div class="label">iv)</div><div class="content"><div id="p0335" role="paragraph">the RBG probability images are thresholded and watershed in MATLAB to segment the nuclear area. The cytoplasmic measurements are derived by dilating the nuclear mask;</div></div></div><div id="o0030" role="listitem"><div class="label">v)</div><div class="content"><div id="p0340" role="paragraph">single-cell measurements are extracted for each channel (cell pixel median and mean for both nuclear and cytoplasmic area) as well as morphological measurements of area, solidity, and cell coordinates location.</div></div></div></div></div></section><section id="sec-4-4-11"><h4>BaSiC</h4><div id="p0345" role="paragraph">The BaSiC ImageJ plugin tool was used to perform background and shading correction of the original images (<span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Peng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Peng, T. ∙ Thorn, K. ∙ Schroeder, T. ...</div><div><strong>A BaSiC tool for background and shading correction of optical microscopy images</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14836" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85020638067" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28594001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14836&amp;pmid=28594001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The&nbsp;BaSiC algorithm calculates the flatfield, the change in effective illumination across an image, and the darkfield, which captures&nbsp;the camera offset and thermal noise. The dark field correction image is subtracted from the original image, and the result is divided by&nbsp;the flatfield image correction to obtain the final image.</div></section><section id="sec-4-4-12"><h4>ASHLAR</h4><div id="p0350" role="paragraph">Alignment by Simultaneous Harmonization of Layer/Adjacency Registration (ASHLAR) is used to stitch together image tiles and register image tiles in subsequent layers to those in the first layer (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Lin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lin, J.R. ∙ Izar, B. ∙ Wang, S. ...</div><div><strong>Highly multiplexed immunofluorescence imaging of human tissues and tumors using t-CyCIF and conventional optical microscopes</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e31657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.31657" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85052091363" target="_blank" aria-label="Open Scopus in new tab">Scopus (369)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29993362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.31657&amp;pmid=29993362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Rashid et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Rashid, R. ∙ Gaglia, G. ∙ Chen, Y.A. ...</div><div><strong>Highly multiplexed immunofluorescence images and single-cell data of immune markers in tonsil and lung cancer</strong></div><div><em>Sci. Data.</em> 2019; <strong>6</strong>:323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41597-019-0332-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076825816" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31848351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41597-019-0332-y&amp;pmid=31848351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the first image layer, neighboring image tiles are aligned to one another via a phase correlation algorithm that corrects for local state positioning error. A similar method is applied for subsequent layers to align tiles to their corresponding tile in the first layer. ASHLAR outputs an OME-TIFF file containing a multi-channel mosaic of the full image across all imaging cycles. Full codes available at: <a id="intref0210" href="https://github.com/labsyspharm/ashlar" target="_blank">https://github.com/labsyspharm/ashlar</a>.</div></section><section id="sec-4-4-13"><h4>Ilastik</h4><div id="p0355" role="paragraph">Ilastik is a machine learning based bioimage analysis tool that is used to obtain nuclear and cytoplasmic segmentation masks from OME-TIFF files (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Berg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Berg, S. ∙ Kutra, D. ∙ Kroeger, T. ...</div><div><strong>ilastik: interactive machine learning for (bio)image analysis</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1226-1232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0582-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85075429634" target="_blank" aria-label="Open Scopus in new tab">Scopus (1467)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0582-9&amp;pmid=31570887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For increased processing speed, randomly selected 250&nbsp;× 250 pixel regions from the original OME-TIFF are used as training data. ilastik’s interactive user interface allows the user to provide training annotations on the cropped regions. Users are presented with a subset of the channels stacked images and label pixels as either nuclear area, cytoplasmic area, or background area. The annotations are used to train non-linear classifiers that are applied to the entire image to obtain probability masks describing the probabilities of each pixel belonging to the nuclear, cytoplasmic, or background area. A MATLAB (version 2018a) script uses these masks to construct binary masks for nuclear and cytoplasmic area.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><section id="sec-4-5-1"><h4>Single-cell RNA Sequencing Analysis</h4><section id="sec-4-5-1-1"><h5>Mouse CD8 scRNA-seq Data Processing, Cell Clustering, and Differential Expression Analysis</h5><div id="p0360" role="paragraph">Cell Ranger, version 3.1.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref95-1" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-1" aria-controls="sref95-1">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref95-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Zheng, G.X. ∙ Terry, J.M. ∙ Belgrader, P. ...</div><div><strong>Massively parallel digital transcriptional profiling of single cells</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14049</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85009446777" target="_blank" aria-label="Open Scopus in new tab">Scopus (3501)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14049&amp;pmid=28091601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), was used to process raw sequencing data for RNA expression, VDJ, and Cell Hashing libraries from mouse CD8 scRNA sequencing experiments. RNA expression data was aligned to the GRCm38/mm10 reference mouse transcriptome (version 3.0.0) and VDJ sequencing data was aligned to the prebuilt mouse (GRCm38/mm10) VDJ reference supplied by 10X Genomics (version 3.1.0) (<span class="dropBlock reference-citations"><a id="body-ref-sref95-2" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-2" aria-controls="sref95-2">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref95-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Zheng, G.X. ∙ Terry, J.M. ∙ Belgrader, P. ...</div><div><strong>Massively parallel digital transcriptional profiling of single cells</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14049</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85009446777" target="_blank" aria-label="Open Scopus in new tab">Scopus (3501)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14049&amp;pmid=28091601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cell Ranger estimated 6,472 and 3,646 cells with 15,939 and 14,834 genes detected for RNA expression libraries of SIIN- and SIY- specific CD8 T&nbsp;cells at a sequencing depth of 80.4% and 76.9%, respectively. VDJ libraries captured an estimated 4,713 and 2,705 cells with 3,857 and 2,196 of those cells containing productive V-J Spanning Pair for SIIN- and SIY- specific CD8 T&nbsp;cell libraries, respectively. Cell Hashing libraries were estimated to capture 6,073 and 3,319 cells at a sequencing saturation of 10.3% and 13.2% for SIIN- and SIY- specific CD8&nbsp;T Cell libraries, respectively. Cell Hashing data were demultiplexed with Seurat, version 4.0.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Stoeckius et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Stoeckius, M. ∙ Zheng, S. ∙ Houck-Loomis, B. ...</div><div><strong>Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-018-1603-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85058891736" target="_blank" aria-label="Open Scopus in new tab">Scopus (518)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30567574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-018-1603-1&amp;pmid=30567574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), using counts normalized by centered-log ratio (CLR) transformation at a positive quantile threshold of 0.98 to infer which mouse each cell was harvested from.</div><div id="p0365" role="paragraph">Seurat (version 4.0.0) was used to normalize, scale, select variable features, and perform differential gene expression analysis. To&nbsp;filter out dying cells with poor data quality and remove probable doublets, individual cells with&nbsp;&lt; 100 or &gt; 4000 detected genes, with &gt; 20,000 reads, or with &gt; 5% of reads aligned to mitochondrial genome in RNA expression libraries were removed from downstream analyses. Thereafter, any remaining cells, called as doublets, were also removed from downstream analyses. RNA expression data were normalized by total expression per cell and natural log transformed with a scale factor of 10,000. Expression for all genes in SIIN- and SIY- libraries were first centered by subtracting average expression of each gene and subsequently scaled by dividing gene expression levels by their standard deviations. SIIN- and SIY- libraries were then merged in Seurat (“merge.data&nbsp;= TRUE”). For each cell passing quality control thresholds, metadata assignments for V(D)J clonotypes and cell hashing (mouse of origin) were made using <i>Python</i> with <i>Pandas</i> and <i>Numpy</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Harris et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Harris, C.R. ∙ Millman, K.J. ∙ van der Walt, S.J. ...</div><div><strong>Array programming with NumPy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:357-362</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2649-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85091129687" target="_blank" aria-label="Open Scopus in new tab">Scopus (10809)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32939066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2649-2&amp;pmid=32939066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">McKinney, 2010</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>McKinney, W.</div><div><strong>Data Structures for Statistical Computing in Python. Proceedings of the 9th Python in Science Conference (SCIPY 2010)</strong></div><div>2010</div><div><a id="intref0260" href="https://conference.scipy.org/proceedings/scipy2010/pdfs/mckinney.pdf" target="_blank">https://conference.scipy.org/proceedings/scipy2010/pdfs/mckinney.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.McKinneyData+Structures+for+Statistical+Computing+in+Python.+Proceedings+of+the+9th+Python+in+Science+Conference+%28SCIPY+2010%292010https%3A%2F%2Fconference.scipy.org%2Fproceedings%2Fscipy2010%2Fpdfs%2Fmckinney.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0370" role="paragraph">The top 2,000 variable genes used for downstream dimensionality reductions were selected in Seurat using the <i>vst</i> method (<span class="dropBlock reference-citations"><a id="body-ref-sref8-3" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-3" aria-controls="sref8-3">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Principal component analysis (PCA) was performed for the first 50 principal components (PCs). An estimation of PCs to include for downstream analyses was performed with the elbow method and JackStraw analysis. Based on this estimate, and in manual consideration of gene features of PCs, the first 30 PCs were selected for downstream analyses. A shared nearest neighbor graph (SNN, k&nbsp;= 20) was constructed with 30 PCs and used to perform Louvain clustering (default parameters, resolution) (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Meo et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Meo, P.D. ∙ De Meo, P. ∙ Ferrara, E. ...</div><div><strong>Generalized Louvain Method for Community Detection in Large Networks</strong></div><div><strong>Proceedings of the 2011 11th International Conference on Intelligent Systems Design and Applications</strong></div><div>2011</div><div>https://www.doi.org/10.1109/isda.2011.6121636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/ISDA.2011.6121636" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84857613816" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FISDA.2011.6121636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The data was then visually projected in 2-dimensional space by using the Uniform Manifold Approximation and Projection (UMAP) algorithm (<span class="dropBlock reference-citations"><a id="body-ref-sref8-4" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-4" aria-controls="sref8-4">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">McInnes et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>McInnes, L. ∙ Healy, J. ∙ Melville, J.</div><div><strong>Umap: Uniform manifold approximation and projection for dimension reduction</strong></div><div><em>arXiv.</em> 2018;</div><div>arXiv:1802.03426</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.McInnesJ.HealyJ.MelvilleUmap%3A+Uniform+manifold+approximation+and+projection+for+dimension+reductionarXiv2018arXiv%3A1802.03426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in Seurat (default parameters). Differential gene expression was performed between all clusters using the default Wilcoxon Ranked Sum test (FindAllMarkers, min.pct&nbsp;= 0.25) (<a id="crosref1615" href="#mmc3">Table S3</a>). The top 10 differentially expressed genes for each cluster are represented in a heatmap produced with ComplexHeatmap (cluster_columns&nbsp;= FALSE, cluster_rows&nbsp;= FALSE) (<a id="crosref1620" href="#figs2">Figure&nbsp;S2</a>D). Enrichment for SIIN or SIY was calculated by hypergeometric test (phyper, Stats R package; alpha&nbsp;= 0.05).</div><div id="p0375" role="paragraph">Gene expression scores were projected on UMAPs with Seurat using the FeaturePlot function (order&nbsp;= TRUE). UMAP embeddings that also depict gene expression scores were generated using the plot_cells function from Monocle3.</div></section></section><section id="sec-4-5-2"><h4>Subsetted C2, C3, C4, C8 Analysis and Trajectory Inference</h4><div id="p0380" role="paragraph">A separate analysis of cells in the mouse CD8 scRNA-seq dataset assigned to C2, C3, C4 and C8 was performed with Monocle3, version 0.2.3.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Cao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Qiu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Qiu, X. ∙ Mao, Q. ∙ Tang, Y. ...</div><div><strong>Reversed graph embedding resolves complex single-cell trajectories</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:979-982</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4402" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85031017685" target="_blank" aria-label="Open Scopus in new tab">Scopus (2116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28825705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4402&amp;pmid=28825705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref82-3" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-3" aria-controls="sref82-3">Trapnell et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref82-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Trapnell, C. ∙ Cacchiarelli, D. ∙ Grimsby, J. ...</div><div><strong>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:381-386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84900873950" target="_blank" aria-label="Open Scopus in new tab">Scopus (3557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2859&amp;pmid=24658644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The raw RNA expression data for these cells were subsetted in Seurat and exported to Monocle3 for dimensionality reduction, clustering, and trajectory analyses. Preprocessing of raw expression data (normalization and PCA) was performed for 20 PCs, using all other default parameters. Further dimensionality reduction was performed with the UMAP algorithm (default parameters, except: umap.n_neighbors&nbsp;= 30L, umap.fast_sgd&nbsp;= FALSE, preprocess_method&nbsp;= “PCA”) (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">McInnes et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>McInnes, L. ∙ Healy, J. ∙ Melville, J.</div><div><strong>Umap: Uniform manifold approximation and projection for dimension reduction</strong></div><div><em>arXiv.</em> 2018;</div><div>arXiv:1802.03426</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.McInnesJ.HealyJ.MelvilleUmap%3A+Uniform+manifold+approximation+and+projection+for+dimension+reductionarXiv2018arXiv%3A1802.03426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Leiden clustering (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Traag et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Traag, V.A. ∙ Waltman, L. ∙ van Eck, N.J.</div><div><strong>From Louvain to Leiden: guaranteeing well-connected communities</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:5233</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-41695-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85063527139" target="_blank" aria-label="Open Scopus in new tab">Scopus (1921)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30914743/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-41695-z&amp;pmid=30914743" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was performed to group cells into clusters for trajectory analyses (resolution&nbsp;= 0.001) with otherwise default parameters. For trajectory analyses, a principal graph was learned and plotted in UMAP space (learn_graph, use_partition&nbsp;= FALSE, rann.k&nbsp;= 20). Monocle3&nbsp;UMAP coordinates were exported to Seurat for gene expression and signature analysis. Differentially expressed genes were calculated in Seurat, as described above.</div><div id="p0385" role="paragraph">Gene lists reflecting CD8 T&nbsp;cell states and functionality were curated from a combination of the differentially expressed genes between Seurat clusters and literature review. Clusters C2, C3, C4 and C8 were scored for expression of these genes by the same methodology of Monocle3′s plot_genes_by_group function. Gene expression scores were then scaled for each cluster and visualized in heatmaps produced with ComplexHeatmap in R (cluster_columns&nbsp;= FALSE, cluster_rows&nbsp;= FALSE) (<a id="crosref1650" href="#fig5">Figure&nbsp;5</a>B).</div></section><section id="sec-4-5-3"><h4>ProjecTILs T Cell Subtype Classification and Annotation</h4><div id="p0390" role="paragraph">The mouse CD8 scRNA-seq data was aligned to the reference tumor-infiltrating lymphocytes (TIL) atlas and reference lymphocytic choriomeningitis virus (LCMV)-specific CD8 T&nbsp;cell atlas using the ProjecTILs R package (version 0.5.1) (<span class="dropBlock reference-citations"><a id="body-ref-sref3-5" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-5" aria-controls="sref3-5">Andreatta et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref3-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A normalized expression matrix (Seurat logNorm data) was provided as input and non-T cells were filtered using TILPRED-1.0 before alignment and batch-effect correction. Additional parameters for “make.projection” were kept as the default. Cell states were predicted for each cell using a nearest-neighbor algorithm (ProjecTILs “cellstate.predict”) (<a id="crosref1660" href="#mmc1">Table S1</a>) and overlaid onto the original mouse UMAP with TILPRED filtered cells set as “NA.” Cell states of interest were highlighted on UMAP plots using Seurat (<span class="dropBlock reference-citations"><a id="body-ref-sref8-5" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-5" aria-controls="sref8-5">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with cells of interest plotted last. Cell state annotations were used “as-is” (without a confidence threshold). We further validated these assignments by applying a confidence score threshold of 0.5 for calling a cell state and confirmed that cell state clusters remain the same.</div><div id="p0395" role="paragraph">Enrichment of SIY and SIIN cells labeled as TIL (or LCMV) progenitor or exhausted states was evaluated using a hypergeometric test (phyper, Stats R package; alpha&nbsp;= 0.05). Cells annotated as progenitor or exhausted were tested similarly for enrichment in the mouse clusters.</div></section><section id="sec-4-5-4"><h4>Gene Signature Scoring and UMAP Projection</h4><div id="p0400" role="paragraph">Scoring of published mouse gene signatures was performed with Seurat (version 4.0.0) (<span class="dropBlock reference-citations"><a id="body-ref-sref8-6" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-6" aria-controls="sref8-6">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using the AddModuleScore function, which calculates mean expression level for each signature subtracted by aggregated expression of control signatures for each cell. Signatures were filtered to exclude genes not detected in the query dataset.</div><div id="p0405" role="paragraph">For projection of the TIL/LCMV progenitor exhausted and terminally exhausted signatures (<span class="dropBlock reference-citations"><a id="body-ref-sref46-7" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-7" aria-controls="sref46-7">Miller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref46-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Miller, B.C. ∙ Sen, D.R. ∙ Al Abosy, R. ...</div><div><strong>Subsets of exhausted CD8<sup>+</sup> T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:326-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0312-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061846466" target="_blank" aria-label="Open Scopus in new tab">Scopus (1090)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30778252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0312-6&amp;pmid=30778252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), deletional tolerance signature (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Parish et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Parish, I.A. ∙ Rao, S. ∙ Smyth, G.K. ...</div><div><strong>The molecular signature of CD8+ T&nbsp;cells undergoing deletional tolerance</strong></div><div><em>Blood.</em> 2009; <strong>113</strong>:4575-4585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-10-185223" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-66549100257" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19204323/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-10-185223&amp;pmid=19204323" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and anergy signature (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Safford et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Safford, M. ∙ Collins, S. ∙ Lutz, M.A. ...</div><div><strong>Egr-2 and Egr-3 are negative regulators of T&nbsp;cell activation</strong></div><div><em>Nat. Immunol.</em> 2005; <strong>6</strong>:472-480</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni1193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-18244375249" target="_blank" aria-label="Open Scopus in new tab">Scopus (351)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15834410/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni1193&amp;pmid=15834410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), published gene signatures were filtered for genes captured in the mouse scRNA seq dataset (<a id="crosref1690" href="#mmc2">Table S2</a>). Cells were then scored for each resultant signature using the AddModuleScore function in Seurat and highlighted on the Monocle3-based mouse UMAP of the mouse cluster C2, C3, C4 and C8 analysis.</div><div id="p0410" role="paragraph">For projection of the Tc17 versus Tc1 gene signatures from <span class="dropBlock reference-citations"><a id="body-ref-sref37-3" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-3" aria-controls="sref37-3">Linehan et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref37-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Linehan, J.L. ∙ Harrison, O.J. ∙ Han, S.J. ...</div><div><strong>Non-classical Immunity Controls Microbiota Impact on Skin Immunity and Tissue Repair</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:784-796.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2017.12.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2017.12.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85041116653" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358051/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.033&amp;pmid=29358051" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, raw RNA-seq sequence FASTQ files for Tc17 and Tc1 cells from the skin of Imiquimod-treated or <i>S.&nbsp;epidermidis-</i>infected mice were obtained from the Sequence Read Archive (SRA accession PRJNA419368; Imiquimod Tc17 runs: SRX3425166, SRX3425165, SRX3425145; Imiquimod Tc1 runs: SRX3425154, SRX3425153, SRX3425146; <i>S.&nbsp;epidermidis</i> Tc17 runs: SRX3425152, SRX3425151, SRX3425149; <i>S.&nbsp;epidermidis</i> Tc1 runs: SRX3425150, SRX3425148, SRX3425147). Sequences were aligned to the mouse genome (NCBI37/mm9 assembly) using Bowtie, v. 1.2.3 (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Langmead et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Langmead, B. ∙ Trapnell, C. ∙ Pop, M. ...</div><div><strong>Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</strong></div><div><em>Genome Biol.</em> 2009; <strong>10</strong>:R25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2009-10-3-r25" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-62349130698" target="_blank" aria-label="Open Scopus in new tab">Scopus (16509)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19261174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2009-10-3-r25&amp;pmid=19261174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and feature counts were quantified using rsem, v. 1.3.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Li and Dewey, 2011</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Li, B. ∙ Dewey, C.N.</div><div><strong>RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</strong></div><div><em>BMC Bioinformatics.</em> 2011; <strong>12</strong>:323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-12-323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-79961123152" target="_blank" aria-label="Open Scopus in new tab">Scopus (13979)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21816040/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-12-323&amp;pmid=21816040" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and UCSC mm9 genome annotation (<a id="intref0215" href="http://genome.ucsc.edu" target="_blank">genome.ucsc.edu</a>). Pairwise gene expression differential analyses were conducted with the DESeq2 package, v. 1.16.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold&nbsp;change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (48497)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), in R (v. 3.6.0; <a id="intref0220" href="http://r-project.org" target="_blank">r-project.org</a>). The Tc17 versus Tc1 (<i>S.&nbsp;epidermidis</i> or Imiquimod; <a id="crosref1715" href="#mmc2">Table S2</a>) signatures were scored using Seurat (AddModuleScore with default parameters) and highlighted on the Monocle3-based mouse UMAP of the mouse cluster C2, C3, C4 and C8 analysis.</div><div id="p0415" role="paragraph"><i>CCR6+ TCF7+</i> and <i>CCR6- TCF7+</i> signatures were derived from the mouse scRNA-seq data by identifying differentially expressed genes using Seurat (FindMarkers with default parameters) with a fold change &gt; 2 and adjusted p value&nbsp;&lt; 0.05 (<a id="crosref1720" href="#mmc3">Table S3</a>). <i>Ccr6+ Tcf7+</i> marker genes were defined by comparing cells from cluster C4 expressing both <i>Ccr6</i> and <i>Tcf7</i> (log-normalized expression count &gt; 0) to that of all other cells in the dataset except those in cluster C4. To define <i>Ccr6- Tcf7+</i> marker genes, cells from cluster C8 expressing <i>Tcf7</i> but not <i>Ccr6</i> were compared to the expression of all other cells in the dataset except those in cluster C8. Mouse symbols were translated to human symbols using the Broad GSEA chip file “Mouse Gene Symbol Remapping to Human Orthologs MSigDB.v7.2.chip” (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Liberzon et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Liberzon, A. ∙ Subramanian, A. ∙ Pinchback, R. ...</div><div><strong>Molecular signatures database (MSigDB) 3.0</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:1739-1740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-79958135926" target="_blank" aria-label="Open Scopus in new tab">Scopus (3806)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21546393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr260&amp;pmid=21546393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Subramanian et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Subramanian, A. ∙ Tamayo, P. ∙ Mootha, V.K. ...</div><div><strong>Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:15545-15550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0506580102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-27344435774" target="_blank" aria-label="Open Scopus in new tab">Scopus (33620)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0506580102&amp;pmid=16199517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), subsequently scored in the Sade-Feldman dataset and visualized with UMAPs using Seurat.</div></section><section id="sec-4-5-5"><h4>TCR Clonotype Analysis</h4><div id="p0420" role="paragraph">Cells with productive TCR sequences in the mouse CD8 scRNA-seq dataset were assigned to clonotypes by Cell Ranger. While the vast majority (581 of 652) of clonotypes appear to originate exclusively from a single mouse, a minority (22 of 652) of clonotypes appear to have one or more cells that originate from multiple mice. These clonotypes were identified in Python with <i>Pandas</i> and excluded from downstream analyses. Most of the clonotypes detected by Cell Ranger contain less than 5 cells (548 of 652), which may reflect their underlying biological abundance (i.e., degree of clonal expansion). However, it is impossible to decouple this underlying biology from artifacts of technical dropout in these lowly-abundant clonotypes. As such, clonotypes containing less than 5 cells were removed in most analyses, resulting in a total of 103 clonotypes.</div><div id="p0425" role="paragraph">In order to understand how clonotypes were distributed across the Seurat clusters (C0-C10), a matrix was generated that quantifies the number of cells in each clonotype assigned to each Seurat cluster. To permit comparison of clonotypes of different sizes, this matrix was normalized to reflect the fraction of cells in each clonotype. To cluster clonotypes with similar distributions across Seurat clusters, Euclidean distance was first calculated between clonotypes in the normalized clonotype-cluster matrix. Visualizations of the clonotype-cluster matrix were made with ComplexHeatmap. Rows (clonotypes) in this heatmap were ordered by hierarchical clustering (method&nbsp;= “ward.D”) of default pairwise euclidean distances. Columns (clusters) were ordered by hierarchical clustering (method&nbsp;= “ward.D”) of pairwise Jaccard distance. Subsequently, complete linkage hierarchical clustering (hclust, <i>R</i>, method&nbsp;= “ward.D”) was performed on these distance calculations. A total of 12 clonotype clusters were generated by cutting the resulting dendrogram (h&nbsp;= 0.6, cutree, <i>R</i>).</div><div id="p0430" role="paragraph">Cells assigned to each clonotype were scored for expression of <i>Gzmb, Havcr2, Cx3cr1, Tcf7, Ccr6,</i> and <i>Il17a</i> by counting the number of cells with gene expression values &gt; 0.5. To test enrichment of these genes, we performed a hypergeometric test (phyper, Stats R package; alpha&nbsp;= 0.05). To test enrichment of clonotypes specific for SIIN and SIY in C4+C8 and C2, we performed a 2-dimensional 2-sample KS test and visualized differences in proportion of cells with an empirical cumulative distribution plot. Additionally, to provide a metric for how equally distributed each clonotype is across clusters, we calculated the Gini index (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Hurley, 2009</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Hurley, N.R.S.</div><div><strong>Comparing Measures of Sparsity</strong></div><div><em>IEEE Trans. Inf. Theory.</em> 2009; <strong>55</strong>:4723-4741</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/TIT.2009.2027527" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-70349622761" target="_blank" aria-label="Open Scopus in new tab">Scopus (599)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FTIT.2009.2027527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for each clonotype.</div><div id="p0435" role="paragraph">To visualize the location of cells from large and small clonotypes in UMAP space, we assigned each cell a value equivalent to the size of the clonotype assigned to each cell and plotted this feature in Seurat.</div></section><section id="sec-4-5-6"><h4>Enrichment of Human Signatures in Mouse Clusters</h4><div id="p0440" role="paragraph">Marker genes for published CD8 T&nbsp;cell cluster signatures from <span class="dropBlock reference-citations"><a id="body-ref-sref89-8" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-8" aria-controls="sref89-8">Wu et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref89-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> <a id="crosref1745" href="#mmc4">Table S4</a>, <span class="dropBlock reference-citations"><a id="body-ref-sref21-6" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-6" aria-controls="sref21-6">Guo et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref21-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> <a id="crosref1755" href="#mmc3">Table S3</a>, and <span class="dropBlock reference-citations"><a id="body-ref-sref59-9" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-9" aria-controls="sref59-9">Sade-Feldman et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref59-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> <a id="crosref1765" href="#mmc4">Table S4</a> were translated from human symbols to mouse symbols using the Mouse Genome Informatics (MGI) mouse/human orthology with phenotype annotations obtained on November 16, 2020 (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Baldarelli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Baldarelli, R.M. ∙ Smith, C.M. ∙ Finger, J.H. ...</div><div><strong>The mouse Gene Expression Database (GXD): 2021 update</strong></div><div><em>Nucleic Acids Res.</em> 2020; <strong>49</strong>:D924-D931</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85099429019" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) [<a id="intref0225" href="http://www.informatics.jax.org/downloads/reports/HMD_HumanPhenotype.rpt" target="_blank">http://www.informatics.jax.org/downloads/reports/HMD_HumanPhenotype.rpt</a>]. For ambiguous mappings (i.e., human genes mapping to more than one mouse gene), we first prioritized the lowercase equivalent of the human gene, if available. For cases where lowercase equivalents were not an option, we checked to see if one of the possible translations was in the mouse dataset and if so, then we arbitrarily selected the first translation (but avoided symbols with the Gm<sup>∗</sup> prefix if another option was available). For cases where neither the lowercase equivalent nor a match in the mouse dataset was found, the first translation was kept as a placeholder. In addition, genes without a mapping were dropped and not included in the final lists. Signature gene lists used for downstream analysis are provided in <a id="crosref1775" href="#mmc4">Table S4</a>. Human signatures were scored in the mouse dataset using Seurat (AddModuleScore with default parameters) and visualized on the mouse Seurat-based UMAP (<a id="crosref1780" href="#figs7">Figure&nbsp;S7</a>A).</div><div id="p0445" role="paragraph">Heatmaps illustrating enrichment of human signatures in mouse clusters (<a id="crosref1785" href="#fig7">Figure&nbsp;7</a>A) were generated in R using heatmap.2 and data was preprocessed using a custom script. For each human signature and mouse cluster, a Wilcoxon rank-sum test was performed to test for differential enrichment of a signature across clusters (i.e., a set of signature scores for a given cluster compared to signature scores across all other clusters) similar to <span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">DeTomaso et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>DeTomaso, D. ∙ Jones, M.G. ∙ Subramaniam, M. ...</div><div><strong>Functional interpretation of single cell similarity maps</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4376</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12235-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85072701938" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31558714/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12235-0&amp;pmid=31558714" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. To reduce overall noise and highlight enrichment for clusters that included “sufficient” high-scoring cells, significance testing was conditioned according to distributional (i.e., quantile) cutoffs. Differential enrichment between sets of signature scores was only tested if the following conditions were met: the upper 2<sup>nd</sup> percentile of scores of cells within the cluster exceeded (i) the upper 2<sup>nd</sup> percentile of scores of cells outside the cluster, (ii) the upper 2<sup>nd</sup> percentile of scores of cells across the signature, and (iii) the upper 40<sup>th</sup> percentile of scores of cells across all signatures (i.e., a full matrix of signature scores per cell per cluster).</div></section><section id="sec-4-5-7"><h4>Analysis of External Human scRNA-seq Datasets</h4><section id="sec-4-5-7-1"><h5>GSE139555 Wu et&nbsp;al</h5><div id="p0450" role="paragraph">Original cluster assignments for cells in the <span class="dropBlock reference-citations"><a id="body-ref-sref89-9" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-9" aria-controls="sref89-9">Wu et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref89-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> dataset were retrieved from GEO (accession GSE139555; GSE139555_tcell_metadata.txt). Additionally, utility functions within the software library released in the GEO record (GSE139555_software.tar.gz) were used for extracting raw count tables with code updates to subset the dataset to contain cells assigned to CD8 T&nbsp;cell clusters in the original study (8.1.Teff, 8.2.Tem, 8.3a.Trm, 8.3b.Trm, 8.3c.Trm, 8.4.Chrom, 8.5.Mitosis, 8.6.KLRB1). Raw count tables for CD8 T&nbsp;cells per tumor sample, ignoring cells from normal adjacent tissue and peripheral blood samples across all patients, were then used in subsequent analyses. Counts per tumor sample were normalized using Seurat v3.2 <i>SCTransform</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref8-7" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-7" aria-controls="sref8-7">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while regressing out percentage mitochondrial content per cell. Samples were integrated in a hierarchical fashion (per patient, then across patients) using the Seurat reference-based workflow (<span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (7328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Choice of reference sample did not affect result interpretation. PCA (Principal Component Analysis with top 30 components used for downstream analyses) dimensionality reduction and UMAP (Uniform Manifold Approximation and Projection) embeddings were generated using the integrated dataset.</div><div id="p0455" role="paragraph">For projection of mouse scRNA-seq cluster gene signatures onto the Wu analysis, the top 20 positive marker genes for each mouse&nbsp;cluster (C4 and C8, clustering resolution 0.7) were identified using Seurat (FindMarkers) with a fold change cutoff of 1.2 and adjusted p value less than 0.05. Mouse symbols were translated to human symbols using the Broad GSEA chip file (MSigDB v7.2; <a id="intref0230" href="https://data.broadinstitute.org/gsea-msigdb/msigdb/annotations_versioned/Mouse_Gene_Symbol_Remapping_Human_Orthologs_MSigDB.v7.2.chip" target="_blank">https://data.broadinstitute.org/gsea-msigdb/msigdb/annotations_versioned/Mouse_Gene_Symbol_Remapping_Human_Orthologs_MSigDB.v7.2.chip</a>)(<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Liberzon et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Liberzon, A. ∙ Subramanian, A. ∙ Pinchback, R. ...</div><div><strong>Molecular signatures database (MSigDB) 3.0</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:1739-1740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-79958135926" target="_blank" aria-label="Open Scopus in new tab">Scopus (3806)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21546393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr260&amp;pmid=21546393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Subramanian et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Subramanian, A. ∙ Tamayo, P. ∙ Mootha, V.K. ...</div><div><strong>Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:15545-15550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0506580102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-27344435774" target="_blank" aria-label="Open Scopus in new tab">Scopus (33620)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0506580102&amp;pmid=16199517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), subsequently scored in the Wu dataset and visualized on the Wu UMAP using Seurat. Signature gene lists used for downstream analysis (<a id="crosref1825" href="#mmc3">Table S3</a>) reflect the collection of unique genes for which an ortholog has been mapped in human.</div></section></section><section id="sec-4-5-8"><h4>GSE120575 Sade-Feldman et&nbsp;al</h4><div id="p0460" role="paragraph">Log2-TPM counts were retrieved from GEO along with patient characteristics, including response to immune checkpoint blockade, and processed with Seurat v3.0.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A Seurat object was created such that the counts slot and data slot were populated with TPM counts and log-TPM counts, respectively. Only cells assigned to published CD8 T&nbsp;cell clusters (<a id="crosref1835" href="#mmc2">Table S2</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref59-10" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-10" aria-controls="sref59-10">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were selected and used for downstream analyses. Highly variable genes were identified and the data was scaled (while regressing out percentage mitochondrial content per cell and the number of detected reads per cell) followed by dimensionality reduction with PCA and UMAP (using 24 principal components) using Seurat’s standard preprocessing procedure.</div></section><section id="sec-4-5-9"><h4>GSE99254 Guo et&nbsp;al</h4><div id="p0465" role="paragraph">Raw counts were retrieved from GEO (accession GSE99254). Cells from seven CD8 T&nbsp;cell clusters in the published analysis were retained for downstream analyses. This subset was processed using the SCT normalization flow of Seurat v3.0.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref22-3" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-3" aria-controls="sref22-3">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref22-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Following dimensionality reduction using PCA, 12 principal components were used for downstream analyses including UMAP embeddings.</div></section><section id="sec-4-5-10"><h4>GSE72056 Tirosh et&nbsp;al</h4><div id="p0470" role="paragraph">Log2(TPM/10&nbsp;+ 1) counts were retrieved from GEO along with patient metadata and processed with Seurat v3.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref22-4" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-4" aria-controls="sref22-4">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref22-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A Seurat object was created such that the counts slot and data slot were populated with TPM counts and log-TPM counts, respectively. The dataset was filtered to retain T&nbsp;cells based on metadata annotation and further filtered to retain only CD8 T&nbsp;cells by filtering on expression of CD8A (&gt; 0) and CD4 (&nbsp;= 0). Only six patient samples with 50 or more CD8 T&nbsp;cells were selected and used for downstream analyses and integrated using Seurat’s reciprocal PCA (RPCA) integration flow. 30 principal components were used for downstream UMAP dimensionality reduction and data embedding.</div></section><section id="sec-4-5-11"><h4>GSE123813 Yost et&nbsp;al</h4><div id="p0475" role="paragraph">Raw counts were retrieved from GEO along with patient information and processed with Seurat v3.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref22-5" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-5" aria-controls="sref22-5">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref22-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The dataset was filtered to retain basal cell carcinoma (BCC) T&nbsp;cells based on metadata annotation and further filtered to retain only CD8 T&nbsp;cells by filtering on expression of CD8A (&gt; 0) and CD4 (&nbsp;= 0). A hierarchical integration flow based on SCT normalization was used, similar to the one described for Wu et&nbsp;al. above (GSE139555). Pre- and post-treatment samples per patient were integrated in the first round and cross-patient integration was performed using Seurat’s reference-based integration workflow. The code was largely based on that used for the GSE139555 analysis (described above). 30 principal components were used for downstream analyses.</div><div id="p0480" role="paragraph">For the analyses of GSE139555, GSE120575, GSE99254, GSE72056, GSE123813 above, the Seurat default “RNA” assay with log-normalized counts was used for feature plots, marker gene identification, and all expression-based analyses. For single gene and co-expression analyses, log-normalized expression count thresholds of 0.0 (raw count datasets GSE139555, GSE99254, GSE123813) and 0.5 (TPM datasets GSE72056, GSE120575) were used for considering a gene as expressed in a given cell.</div></section><section id="sec-4-5-12"><h4>Identification of mucosal associated invariant T (MAIT) cells</h4><div id="p0485" role="paragraph">In the human scRNA-seq datasets we identified a population of mucosal associated invariant T (MAIT) cells that phenotypically resemble Tc17 cells (<span class="dropBlock reference-citations"><a id="body-ref-sref79-2" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-2" aria-controls="sref79-2">Toubal et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Toubal, A. ∙ Nel, I. ∙ Lotersztajn, S. ...</div><div><strong>Mucosal-associated invariant T&nbsp;cells and disease</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:643-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0191-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069455629" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31308521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0191-y&amp;pmid=31308521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). MAIT cells express invariant TCRs that recognize antigen presented on the MHC-related molecule MR1. Cells with these TCRs were assigned to background (i.e., not highlighted) in the <i>TCF7/CCR6/RORC/IL17A</i> expression plots and excluded from statistical analyses of CCR6+/− TCF7+ signature enrichment in <a id="crosref1865" href="#fig7">Figures 7</a>C, 7I, 7J, and <a id="crosref1870" href="#figs7">S7</a>C. Importantly, MAIT cells are also found in mice, but do not express CD8 and are not found among SIIN- or SIY-specific T&nbsp;cells that are MHC I K<sup>b</sup>-restricted in KP LucOS lung tumors.</div><div id="p0490" role="paragraph">For Wu et&nbsp;al., clonotype assignments were retrieved from GEO record GSE139555. MAIT cells were identified based on their invariant TCR using the following logic: (alpha_v&nbsp;= “TRAV1-2”) AND one of ($alpha_j&nbsp;= “TRAJ33”) OR ($alpha_j&nbsp;= “TRAJ12”) OR ($alpha_j&nbsp;= “TRAJ20”) and assigned to the background set (not highlighted) in co-expression analysis (<span class="dropBlock reference-citations"><a id="body-ref-sref79-3" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-3" aria-controls="sref79-3">Toubal et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Toubal, A. ∙ Nel, I. ∙ Lotersztajn, S. ...</div><div><strong>Mucosal-associated invariant T&nbsp;cells and disease</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:643-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0191-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069455629" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31308521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0191-y&amp;pmid=31308521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For Sade-Feldman et&nbsp;al., clonotype assignments were requested and obtained from the authors (private communication). MAIT cells were identified using the logic described above. For <span class="dropBlock reference-citations"><a id="body-ref-sref21-7" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-7" aria-controls="sref21-7">Guo et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref21-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, cells identified as belonging to a MAIT cluster (CD8-C7-SLC4A10) in the published analysis were tagged as MAIT cells. For Tirosh et&nbsp;al., we obtained raw scRNaseq FASTQ files through the Broad Institute Data Used Oversight System (<a id="intref0235" href="https://duos.broadinstitute.org" target="_blank">https://duos.broadinstitute.org</a>). This data was processed using MiXCR’s v3.0.13 “analyze shotgun” workflow (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Bolotin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bolotin, D.A. ∙ Poslavsky, S. ∙ Davydov, A.N. ...</div><div><strong>Antigen receptor repertoire profiling from RNA-seq data</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:908-911</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3979" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85031088639" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29020005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3979&amp;pmid=29020005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to catalog the T&nbsp;cell receptor repertoire. No cells satisfying the MAIT clonotype assignments described above were detected in this set. For Yost et&nbsp;al., clonotype assignments were requested and obtained from the authors (private communication). MAIT cells were identified using the logic described above.</div><div id="p0495" role="paragraph">Lists of MAIT cells per dataset are included in <a id="crosref1890" href="#mmc6">Table S6</a>. Using a relaxed approach, cells without clonotype information were assumed to be non-MAIT. Overall, MAIT cells were not highlighted in co-expression plots and were dropped from CCR6+/− TCF7+ signature enrichment analyses.</div></section><section id="sec-4-5-13"><h4>Enrichment of CCR6+/− TCF7+ signatures in GSE120575 (<span class="dropBlock reference-citations"><a id="body-ref-sref59-11" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-11" aria-controls="sref59-11">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</h4><div id="p0500" role="paragraph">A hypergeometric test (phyper, Stats R package; alpha&nbsp;= 0.05) was used to test for enrichment of cells from published clusters in the set of cells with high <i>CCR6+ TCF7+</i> or <i>CCR6- TCF7+</i> signature scores (standardized signature score &gt; 2). Similarly, cells annotated as responders/non-responders were tested for enrichment in the set of cells with high <i>CCR6+/− TCF7+</i> signature scores (standardized signature score &gt; 2). Response categories were determined by RESIST criteria: responders, complete or partial response; non-responders, stable or progressive disease (<span class="dropBlock reference-citations"><a id="body-ref-sref59-12" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-12" aria-controls="sref59-12">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). MAIT cells were removed from the dataset prior to calculating proportions.</div></section><section id="sec-4-5-14"><h4>t-CyCIF Data Analysis Workflow</h4><div id="p0505" role="paragraph">The data analysis is divided in a set of pre-processing steps in which data from different tissues is i) log2-transformed and aggregated together, ii) filtered for image analysis errors, and iii) normalized on a channel-by-channel basis across the entire data from a single experiment. All the steps are performed in MATLAB.</div></section><section id="sec-4-5-15"><h4>Data Aggregation</h4><div id="p0510" role="paragraph">The image processing workflow outputs one ome.tiff image and one data file (.mat) for each tissue area imaged. The data matrices from each .mat file are concatenated into a single matrix for each metric measured (median/mean, nuclear/cytoplasmic) into a single structure (“AggrResults”). The morphological data (i.e., area, solidity, and centroid coordinates) is concatenated into a single structure (“MorpResults”), which also contains the indexing vector to keep track of the tissue of origin within the dataset.</div></section><section id="sec-4-5-16"><h4>Data Filtering</h4><div id="p0515" role="paragraph">Single cells are filtered to identify and potentially exclude from subsequent analysis errors in segmentation and cells lost through the rounds of imaging. Two types of criteria are used to filter cells: morphological criteria based on cell object segmented area, which are applied to all the rounds for the cell object, and DAPI-based criteria which are applied to the DAPI measurement for each imaging round. The latter corrects for cell loss during cycling and computational misalignment, which are both round specific.</div><div id="p0520" role="paragraph">Morphological filtering criteria are: 1) nuclear area within a user-input range; 2) cytoplasmic area within a user-input range; 3) nuclear object solidity above a user-input threshold. DAPI-based criteria are: 1) nuclear DAPI measurement above a user-input threshold; 2) ratio between nuclear and cytoplasmic DAPI measurement above a user-input threshold. The filter information for the criteria is allocated to a logical (0-1) structure ‘Filter’, which is used to select the cells to analyze in the further analysis by indexing. The threshold selection is dataset dependent and is performed by data inspection. The values used in each dataset are available with the codes used for data analysis in the github repository</div><div id="p0525" role="paragraph"><a id="intref0240" href="https://github.com/santagatalab/2021_Burger_et_al_CyCIF_codes" target="_blank">https://github.com/santagatalab/2021_Burger_et_al_CyCIF_codes</a>.</div></section><section id="sec-4-5-17"><h4>Data Normalization</h4><div id="p0530" role="paragraph">Each channel distribution is normalized by probability density function (pdf) centering and rescaling. The aim is to center the distribution of the log2 fluorescent signal at 0 and rescale the width of the distribution to be able to compare across channels. The data is first log-transformed (base 2). The standard normalization is performed using a 2-component Gaussian mixture model, each model capturing the negative and the positive cell population. If the 2-component model fails to approximate the channel distribution, two other strategies are attempted: i) a 3-component model is used assuming the components with the two highest means are the negative and positive distribution (i.e., discarding the lowest component) or ii) the user selects a percentage ‘x’ of assumed positive cells and a single Gaussian distribution fit is performed on the remainder of the data to capture the negative distribution. The single Gaussian fit is then used as the lower component in a 2-component model to estimate the distribution of the positive population. The strategy chosen for each channel in each dataset is available in the github repository <a id="intref0245" href="https://github.com/santagatalab/2021_Burger_et_al_CyCIF_codes" target="_blank">https://github.com/santagatalab/2021_Burger_et_al_CyCIF_codes</a>. The “add_coeff” is defined as the intersection of the negative and positive distributions. The “mult_coeff” is defined as the difference between the mean of the negative and positive distributions. The full distribution is normalized by subtracting the add_coeff and dividing by the mult_coeff. The normalization is performed on the nuclear and cytoplasmic single-cell, single-channel distributions individually. The data preprocessing workflow is performed on all datasets. The individual analyses used in the paper are performed only in select datasets as follows.</div></section><section id="sec-4-5-18"><h4>Cell Type Calling</h4><div id="p0535" role="paragraph">Cells from tissue-based experiments are classified into lineage compartments by cell type markers, by gating on the sign of the normalized values of cell type markers. Cells are scored by increasingly deeper lineage layers, starting with separating them into epithelial, immune and stromal compartments, and subdividing the immune population into lymphoid lineages. The cell types identified and the markers used to do so are included in <a id="crosref1905" href="#mmc7">Table S7</a>. If a cell was called as more than one single cell-type, this is defined as a conflict. The conflicts are resolved by comparing the markers that triggered each of the cell type calls and assigning the cell type with the highest marker level. If the markers are within 10% of each other, the cell is deemed “not classifiable” for the specific layer.</div></section><section id="sec-4-5-19"><h4>Identification of CCR6+ TCF1+ CD8+ T Cells</h4><div id="p0540" role="paragraph">The quantification of positive cells was performed on a single channel basis for CD8, CCR6 and TCF1. For CD8 and TCF1 signal the cells were called positive based on the normalization described above. However, in the lung cancer tissues imaged, CD20 positive B cells expressed high levels of CCR6, consistent with the current literature (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Kadomoto et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Kadomoto, S. ∙ Izumi, K. ∙ Mizokami, A.</div><div><strong>The CCL20-CCR6 Axis in Cancer Progression</strong></div><div><em>Int. J. Mol. Sci.</em> 2020; <strong>21</strong>:5186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/ijms21155186" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85088246429" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32707869/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fijms21155186&amp;pmid=32707869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The variability of B cells between individual tissues and the spatial proximity between T&nbsp;cells and B cells can potentially lead to overestimation of CCR6 positivity. In order to avoid potential biases we modified the protocol in three ways: 1) the CCR6 protein was normalized at a single tissue basis to match the negative signal distribution, 2) when calculating the CCR6+ CD8+ cells (independent of TCF1 status) we discarded all cells closer than 32.5&nbsp;μm (100 pixels), and 3) we imaged the CCR6 protein twice in the first and last round of the experiment.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0545" role="paragraph">We would like to acknowledge R. Herbst, T. Delorey, D. Phillips, the Broad Genomics Platform, and the Whitehead Institute Genomics Technology Core for assistance with scRNA-seq; C. Backlund for input on vaccination techniques; Koch Institute’s Robert A. Swanson Biotechnology Center for assistance with flow cytometry and histology; K. Yee, J. Texeira, K. Anderson, K. Mercer, and M. Magendantz for administrative and logistical support; and S. Spranger, M. Birnbaum, C. Jin, and B. Eskiocak for their insightful intellectual input. This work was supported by Howard Hughes Medical Institute (to T.J.), Ludwig Center at Harvard (to S.S.), NIH U54 (CA225088 to S.S.), NIH (T32HL007627 to G.G.), NCI Cancer Center (Support Grant P30-CA1405), Bridge Project, a partnership between the Koch Institute for Integrative Cancer Research at MIT and the Dana-Farber/Harvard Cancer Center (to T.J. and S.S.), and fellowship awards from Jane Coffin Childs Memorial Fund for Medical Research and Ludwig Center for Molecular Oncology at MIT (to M.L.B.). T.J. is a David H. Koch Professor of Biology and Daniel K. Ludwig Scholar.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0550" role="paragraph">M.L.B., A.M.C., G.E.C., G.G., S.E.B., A.B., D.C., J.M.S., A.L., A.M.J., W.L.H., A.R., S.S., and T.J. designed the study. M.L.B., G.E.C., T.K., S.Z.T., A.L.B., A.G., and I.d.l.R.K. performed the mouse experiments in the lab of T.J. M.L.B., G.E.C., M.H., W.L.H., and P.M.K.W. performed the mouse scRNA-seq in the labs of T.J. and A.R. A.M.C., S.E.B., and A.B. performed the scRNA-seq computational analyses. G.G. and C.C.R. performed the t-CyCIF in the lab of S.S. F.S.H., M.J.M., and M.H. provided melanoma samples. M.L.B. wrote the manuscript with input from other authors.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0555" role="paragraph">T.J. is on the board of directors of Amgen and ThermoFisher Scientific, a co-founder of Dragonfly Therapeutics and T2 Biosystems, an SAB member of Dragonfly Therapeutics, SQZ Biotech, and Skyhawk Therapeutics and is president of Break Through Cancer. The T.J. lab receives funding from the Johnson &amp; Johnson Lung Cancer Initiative and The Lustgarten Foundation for Pancreatic Cancer Research, but this funding did not support this research. A.R. is a founder and equity holder of Celsius Therapeutics, an equity holder in Immunitas Therapeutics and until August 31, 2020, was an SAB member of Syros Pharmaceuticals, Neogene Therapeutics, Asimov, and ThermoFisher Scientific. A.R. is an employee of Genentech (Roche Group) from August 1, 2020. S.S. is a consultant for RareCyte, Inc. A.M.C. interned at Skyhawk Therapeutics. None of these affiliations represent a conflict of interest with respect to the design or execution of this study or interpretation of data presented in this manuscript.</div></section><section id="sec-7"><h3>Inclusion and diversity</h3><div id="p0560" role="paragraph">We worked to ensure sex balance in the selection of non-human subjects. One or more of the authors of this paper self-identifies as an underrepresented ethnic minority in science.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (7)</h2><div id="p0570" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.08.020/attachment/2f66928c-83ff-45ed-bee7-19aaa0f264e2/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (208.29 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. ProjecTILs cell state assignments for the mouse scRNA-seq data and enrichment statistics for SIIN- and SIY-specific cells, related to Figures 2 and S2</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.08.020/attachment/2da968df-a656-4072-8a39-c3bc04caac3a/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (2.70 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Published mouse gene signatures projected onto the mouse scRNA-seq data, related to Figures 5, S2, and S5</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.08.020/attachment/4d0f636c-4cbd-4755-ae0c-270b401553a6/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (356.53 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Top differentially expressed genes and gene signatures for the mouse scRNA-seq clusters and CCR6<sup>+</sup>/CCR6<sup>−</sup> TCF1<sup>+</sup> cell subsets, related to Figures 2, 5, and 7</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.08.020/attachment/89739073-b202-46d2-a67b-b30efc469088/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (58.21 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Published human gene signatures projected onto the mouse scRNA-seq data, related to Figures 7 and S7</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.08.020/attachment/b3df9c7a-28f0-42a8-aeff-64e4b6cc592e/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (10.30 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Cell type calling parameters for analysis of mouse and human t-CyCIF images, related to Figures 6 and 7</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.08.020/attachment/5bd22f2c-d7ca-42b0-9e5c-78e46c8d445b/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (3.51 MB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table S6. Gene co-expression cell assignments for the tumor CD8 T&nbsp;cell analysis of human scRNA-seq datasets, related to Figures 7 and S7</div></div></div><div id="mmc7" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.08.020/attachment/61a70147-de97-4d6d-9543-eeb283891ea7/mmc7.xlsx" download="mmc7.xlsx" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Spreadsheet (13.00 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Table S7. Clinical information for lung adenocarcinoma and melanoma samples profiled by t-CyCIF, related to Figure&nbsp;7</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">The mouse scRNA-seq data have been deposited to GEO and is publicly available as of the date of publication. The accession number is listed in the <a id="crosref0250" href="#sec-4-1">Key resources table</a>. This paper additionally included analyses of publicly available human scRNA-seq datasets. The accession numbers for these datasets are listed in the <a id="crosref0255" href="#sec-4-1">Key resources table</a>.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">All original code for the mouse and human scRNA-seq analyses and the t-CYCIF imaging analyses has been deposited on GitHub and is publicly available as of the date of publication. The DOIs are listed in the <a id="crosref0260" href="#sec-4-1">Key resources table</a>.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Anagnostou, V. ∙ Smith, K.N. ∙ Forde, P.M. ...</div><div><strong>Evolution of Neoantigen Landscape during Immune Checkpoint Blockade in Non-Small Cell Lung Cancer</strong></div><div><em>Cancer Discov.</em> 2017; <strong>7</strong>:264-276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-16-0828" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85014747024" target="_blank" aria-label="Open Scopus in new tab">Scopus (669)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28031159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-16-0828&amp;pmid=28031159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Anderson, K.G. ∙ Mayer-Barber, K. ∙ Sung, H. ...</div><div><strong>Intravascular staining for discrimination of vascular and tissue leukocytes</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:209-222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84891707105" target="_blank" aria-label="Open Scopus in new tab">Scopus (541)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.005&amp;pmid=24385150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Andreatta, M. ∙ Corria-Osorio, J. ∙ Müller, S. ...</div><div><strong>Interpretation of T&nbsp;cell states from single-cell transcriptomics data using reference atlases</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-23324-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85106287056" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34017005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-23324-4&amp;pmid=34017005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Baldarelli, R.M. ∙ Smith, C.M. ∙ Finger, J.H. ...</div><div><strong>The mouse Gene Expression Database (GXD): 2021 update</strong></div><div><em>Nucleic Acids Res.</em> 2020; <strong>49</strong>:D924-D931</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85099429019" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Bentzen, A.K. ∙ Such, L. ∙ Jensen, K.K. ...</div><div><strong>T cell receptor fingerprinting enables in-depth characterization of the interactions governing recognition of peptide-MHC complexes</strong></div><div><em>Nat. Biotechnol.</em> 2018;</div><div>Published online November 19, 2018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85058748162" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30451992/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4303&amp;pmid=30451992" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Berg, S. ∙ Kutra, D. ∙ Kroeger, T. ...</div><div><strong>ilastik: interactive machine learning for (bio)image analysis</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1226-1232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0582-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85075429634" target="_blank" aria-label="Open Scopus in new tab">Scopus (1467)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0582-9&amp;pmid=31570887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Bolotin, D.A. ∙ Poslavsky, S. ∙ Davydov, A.N. ...</div><div><strong>Antigen receptor repertoire profiling from RNA-seq data</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:908-911</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3979" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85031088639" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29020005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3979&amp;pmid=29020005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">9.</a></div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Carreno, B.M. ∙ Magrini, V. ∙ Becker-Hapak, M. ...</div><div><strong>Cancer immunotherapy. A dendritic cell vaccine increases the breadth and diversity of melanoma neoantigen-specific T&nbsp;cells</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:803-808</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa3828" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84928195112" target="_blank" aria-label="Open Scopus in new tab">Scopus (996)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25837513/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa3828&amp;pmid=25837513" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Chang, S.H. ∙ Mirabolfathinejad, S.G. ∙ Katta, H. ...</div><div><strong>T helper 17 cells play a critical pathogenic role in lung cancer</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:5664-5669</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1319051111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84898777944" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24706787/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1319051111&amp;pmid=24706787" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>DeTomaso, D. ∙ Jones, M.G. ∙ Subramaniam, M. ...</div><div><strong>Functional interpretation of single cell similarity maps</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4376</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12235-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85072701938" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31558714/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12235-0&amp;pmid=31558714" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Du, Z. ∙ Lin, J.R. ∙ Rashid, R. ...</div><div><strong>Qualifying antibodies for image-based immune profiling and multiplexed tissue imaging</strong></div><div><em>Nat. Protoc.</em> 2019; <strong>14</strong>:2900-2930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0206-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85072716410" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534232/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0206-y&amp;pmid=31534232" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>DuPage, M. ∙ Dooley, A.L. ∙ Jacks, T.</div><div><strong>Conditional mouse lung cancer models using adenoviral or lentiviral delivery of Cre recombinase</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:1064-1072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2009.95" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-68149157175" target="_blank" aria-label="Open Scopus in new tab">Scopus (641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19561589/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2009.95&amp;pmid=19561589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>DuPage, M. ∙ Cheung, A.F. ∙ Mazumdar, C. ...</div><div><strong>Endogenous T&nbsp;cell responses to antigens expressed in lung adenocarcinomas delay malignant tumor progression</strong></div><div><em>Cancer Cell.</em> 2011; <strong>19</strong>:72-85</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-78651451436" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21251614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2010.11.011&amp;pmid=21251614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>DuPage, M. ∙ Mazumdar, C. ∙ Schmidt, L.M. ...</div><div><strong>Expression of tumour-specific antigens underlies cancer immunoediting</strong></div><div><em>Nature.</em> 2012; <strong>482</strong>:405-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84857118418" target="_blank" aria-label="Open Scopus in new tab">Scopus (423)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22318517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10803&amp;pmid=22318517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Eisen, H.N. ∙ Hou, X.H. ∙ Shen, C. ...</div><div><strong>Promiscuous binding of extracellular peptides to cell surface class I MHC protein</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:4580-4585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1201586109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84863350387" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22403068/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1201586109&amp;pmid=22403068" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Engels, B. ∙ Engelhard, V.H. ∙ Sidney, J. ...</div><div><strong>Relapse or eradication of cancer is predicted by peptide-major histocompatibility complex affinity</strong></div><div><em>Cancer Cell.</em> 2013; <strong>23</strong>:516-526</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2013.03.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.ccr.2013.03.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84876381248" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23597565/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2013.03.018&amp;pmid=23597565" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Gu, M. ∙ Paloma, J. ∙ Berger, J.O.</div><div><strong>RobustGaSP: Robust Gaussian Stochastic Process Emulation, 2016. R package version 0.5.7</strong></div><div>2016</div><div><a id="intref0255" href="https://CRANR-projectorg/package=RobustGaSP" target="_blank">https://CRANR-projectorg/package=RobustGaSP</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.GuJ.PalomaJ.O.BergerRobustGaSP%3A+Robust+Gaussian+Stochastic+Process+Emulation%2C+2016.+R+package+version+0.5.72016https%3A%2F%2FCRANR-projectorg%2Fpackage%3DRobustGaSP" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Gubin, M.M. ∙ Zhang, X. ∙ Schuster, H. ...</div><div><strong>Checkpoint blockade cancer immunotherapy targets tumour-specific mutant antigens</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:577-581</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13988" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84920921528" target="_blank" aria-label="Open Scopus in new tab">Scopus (1639)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428507/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13988&amp;pmid=25428507" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">21.</a></div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Harris, C.R. ∙ Millman, K.J. ∙ van der Walt, S.J. ...</div><div><strong>Array programming with NumPy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:357-362</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2649-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85091129687" target="_blank" aria-label="Open Scopus in new tab">Scopus (10809)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32939066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2649-2&amp;pmid=32939066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">24.</a></div><div>Hirota, K. ∙ Duarte, J.H. ∙ Veldhoen, M. ...</div><div><strong>Fate mapping of IL-17-producing T&nbsp;cells in inflammatory responses</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:255-263</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-79951677394" target="_blank" aria-label="Open Scopus in new tab">Scopus (929)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21278737/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1993&amp;pmid=21278737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">25.</a></div><div>Hollingsworth, R.E. ∙ Jansen, K.</div><div><strong>Turning the corner on therapeutic cancer vaccines</strong></div><div><em>NPJ Vaccines.</em> 2019; <strong>4</strong>:7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41541-019-0103-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061253209" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30774998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41541-019-0103-y&amp;pmid=30774998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Howarth, M. ∙ Williams, A. ∙ Tolstrup, A.B. ...</div><div><strong>Tapasin enhances MHC class I peptide presentation according to peptide half-life</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:11737-11742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0306294101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-4143051423" target="_blank" aria-label="Open Scopus in new tab">Scopus (152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15286279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0306294101&amp;pmid=15286279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Hudson, W.H. ∙ Gensheimer, J. ∙ Hashimoto, M. ...</div><div><strong>Proliferating Transitory T Cells with an Effector-like Transcriptional Signature Emerge from PD-1<sup>+</sup> Stem-like CD8<sup>+</sup> T Cells during Chronic Infection</strong></div><div><em>Immunity.</em> 2019; <strong>51</strong>:1043-1058.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2019.11.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2019.11.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076168642" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31810882/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2019.11.002&amp;pmid=31810882" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#figs4" title="View in article">28.</a></div><div>Hurley, N.R.S.</div><div><strong>Comparing Measures of Sparsity</strong></div><div><em>IEEE Trans. Inf. Theory.</em> 2009; <strong>55</strong>:4723-4741</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/TIT.2009.2027527" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-70349622761" target="_blank" aria-label="Open Scopus in new tab">Scopus (599)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FTIT.2009.2027527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Jin, C. ∙ Lagoudas, G.K. ∙ Zhao, C. ...</div><div><strong>Commensal Microbiota Promote Lung Cancer Development via γδ T Cells</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:998-1013.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.040&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.040&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061787031" target="_blank" aria-label="Open Scopus in new tab">Scopus (623)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30712876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.040&amp;pmid=30712876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Kadomoto, S. ∙ Izumi, K. ∙ Mizokami, A.</div><div><strong>The CCL20-CCR6 Axis in Cancer Progression</strong></div><div><em>Int. J. Mol. Sci.</em> 2020; <strong>21</strong>:5186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/ijms21155186" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85088246429" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32707869/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fijms21155186&amp;pmid=32707869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Keenan, T.E. ∙ Burke, K.P. ∙ Van Allen, E.M.</div><div><strong>Genomic correlates of response to immune checkpoint blockade</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:389-402</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0382-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85062627637" target="_blank" aria-label="Open Scopus in new tab">Scopus (324)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30842677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0382-x&amp;pmid=30842677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Kurtulus, S. ∙ Madi, A. ∙ Escobar, G. ...</div><div><strong>Checkpoint Blockade Immunotherapy Induces Dynamic Changes in PD-1<sup>-</sup>CD8<sup>+</sup> Tumor-Infiltrating T Cells</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:181-194.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.11.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.11.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059794223" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30635236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.014&amp;pmid=30635236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Langmead, B. ∙ Trapnell, C. ∙ Pop, M. ...</div><div><strong>Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</strong></div><div><em>Genome Biol.</em> 2009; <strong>10</strong>:R25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2009-10-3-r25" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-62349130698" target="_blank" aria-label="Open Scopus in new tab">Scopus (16509)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19261174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2009-10-3-r25&amp;pmid=19261174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Li, B. ∙ Dewey, C.N.</div><div><strong>RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</strong></div><div><em>BMC Bioinformatics.</em> 2011; <strong>12</strong>:323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-12-323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-79961123152" target="_blank" aria-label="Open Scopus in new tab">Scopus (13979)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21816040/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-12-323&amp;pmid=21816040" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">35.</a></div><div>Liberzon, A. ∙ Subramanian, A. ∙ Pinchback, R. ...</div><div><strong>Molecular signatures database (MSigDB) 3.0</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:1739-1740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-79958135926" target="_blank" aria-label="Open Scopus in new tab">Scopus (3806)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21546393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr260&amp;pmid=21546393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">36.</a></div><div>Lin, J.R. ∙ Izar, B. ∙ Wang, S. ...</div><div><strong>Highly multiplexed immunofluorescence imaging of human tissues and tumors using t-CyCIF and conventional optical microscopes</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e31657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.31657" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85052091363" target="_blank" aria-label="Open Scopus in new tab">Scopus (369)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29993362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.31657&amp;pmid=29993362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#fig5" title="View in article">37.</a></div><div>Linehan, J.L. ∙ Harrison, O.J. ∙ Han, S.J. ...</div><div><strong>Non-classical Immunity Controls Microbiota Impact on Skin Immunity and Tissue Repair</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:784-796.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2017.12.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2017.12.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85041116653" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358051/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.033&amp;pmid=29358051" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Linette, G.P. ∙ Becker-Hapak, M. ∙ Skidmore, Z.L. ...</div><div><strong>Immunological ignorance is an enabling feature of the oligo-clonal T&nbsp;cell response to melanoma neoantigens</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:23662-23670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1906026116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85075233562" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31685621/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1906026116&amp;pmid=31685621" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold&nbsp;change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (48497)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Madisen, L. ∙ Zwingman, T.A. ∙ Sunkin, S.M. ...</div><div><strong>A robust and high-throughput Cre reporting and characterization system for the whole mouse brain</strong></div><div><em>Nat. Neurosci.</em> 2010; <strong>13</strong>:133-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-73949140059" target="_blank" aria-label="Open Scopus in new tab">Scopus (4610)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20023653/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2467&amp;pmid=20023653" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Matsushita, H. ∙ Vesely, M.D. ∙ Koboldt, D.C. ...</div><div><strong>Cancer exome analysis reveals a T-cell-dependent mechanism of cancer immunoediting</strong></div><div><em>Nature.</em> 2012; <strong>482</strong>:400-404</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10755" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84862776855" target="_blank" aria-label="Open Scopus in new tab">Scopus (1008)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22318521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10755&amp;pmid=22318521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>McGranahan, N. ∙ Furness, A.J. ∙ Rosenthal, R. ...</div><div><strong>Clonal neoantigens elicit T&nbsp;cell immunoreactivity and sensitivity to immune checkpoint blockade</strong></div><div><em>Science.</em> 2016; <strong>351</strong>:1463-1469</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf1490" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84962301577" target="_blank" aria-label="Open Scopus in new tab">Scopus (2306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26940869/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf1490&amp;pmid=26940869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">43.</a></div><div>McInnes, L. ∙ Healy, J. ∙ Melville, J.</div><div><strong>Umap: Uniform manifold approximation and projection for dimension reduction</strong></div><div><em>arXiv.</em> 2018;</div><div>arXiv:1802.03426</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.McInnesJ.HealyJ.MelvilleUmap%3A+Uniform+manifold+approximation+and+projection+for+dimension+reductionarXiv2018arXiv%3A1802.03426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>McKinney, W.</div><div><strong>Data Structures for Statistical Computing in Python. Proceedings of the 9th Python in Science Conference (SCIPY 2010)</strong></div><div>2010</div><div><a id="intref0260" href="https://conference.scipy.org/proceedings/scipy2010/pdfs/mckinney.pdf" target="_blank">https://conference.scipy.org/proceedings/scipy2010/pdfs/mckinney.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.McKinneyData+Structures+for+Statistical+Computing+in+Python.+Proceedings+of+the+9th+Python+in+Science+Conference+%28SCIPY+2010%292010https%3A%2F%2Fconference.scipy.org%2Fproceedings%2Fscipy2010%2Fpdfs%2Fmckinney.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Meo, P.D. ∙ De Meo, P. ∙ Ferrara, E. ...</div><div><strong>Generalized Louvain Method for Community Detection in Large Networks</strong></div><div><strong>Proceedings of the 2011 11th International Conference on Intelligent Systems Design and Applications</strong></div><div>2011</div><div>https://www.doi.org/10.1109/isda.2011.6121636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/ISDA.2011.6121636" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84857613816" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FISDA.2011.6121636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">46.</a></div><div>Miller, B.C. ∙ Sen, D.R. ∙ Al Abosy, R. ...</div><div><strong>Subsets of exhausted CD8<sup>+</sup> T&nbsp;cells differentially mediate tumor control and respond to checkpoint blockade</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:326-336</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0312-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85061846466" target="_blank" aria-label="Open Scopus in new tab">Scopus (1090)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30778252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0312-6&amp;pmid=30778252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Milo, I. ∙ Bedora-Faure, M. ∙ Garcia, Z. ...</div><div><strong>The immune system profoundly restricts intratumor genetic heterogeneity</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaat1435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aat1435" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85057108717" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30470696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aat1435&amp;pmid=30470696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Neuwirth, E.</div><div><strong>RColorBrewer: ColorBrewer Palettes. R package version 1.1-2</strong></div><div>2014</div><div><a id="intref0265" href="https://rdrr.io/cran/RColorBrewer/man/ColorBrewer.html" target="_blank">https://rdrr.io/cran/RColorBrewer/man/ColorBrewer.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.NeuwirthRColorBrewer%3A+ColorBrewer+Palettes.+R+package+version+1.1-22014https%3A%2F%2Frdrr.io%2Fcran%2FRColorBrewer%2Fman%2FColorBrewer.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">49.</a></div><div>Ott, P.A. ∙ Hu, Z. ∙ Keskin, D.B. ...</div><div><strong>An immunogenic personal neoantigen vaccine for patients with melanoma</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:217-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22991" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85024377220" target="_blank" aria-label="Open Scopus in new tab">Scopus (1967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22991&amp;pmid=28678778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">50.</a></div><div>Ott, P.A. ∙ Hu-Lieskovan, S. ∙ Chmielowski, B. ...</div><div><strong>A Phase Ib Trial of Personalized Neoantigen Therapy Plus Anti-PD-1 in Patients with Advanced Melanoma, Non-small Cell Lung Cancer, or Bladder Cancer</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:347-362.e24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2020.08.053&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2020.08.053&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85092455165" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33064988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.053&amp;pmid=33064988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">51.</a></div><div>Parish, I.A. ∙ Rao, S. ∙ Smyth, G.K. ...</div><div><strong>The molecular signature of CD8+ T&nbsp;cells undergoing deletional tolerance</strong></div><div><em>Blood.</em> 2009; <strong>113</strong>:4575-4585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-10-185223" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-66549100257" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19204323/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-10-185223&amp;pmid=19204323" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">52.</a></div><div>Peng, T. ∙ Thorn, K. ∙ Schroeder, T. ...</div><div><strong>A BaSiC tool for background and shading correction of optical microscopy images</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14836" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85020638067" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28594001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14836&amp;pmid=28594001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">53.</a></div><div>Peters, B. ∙ Nielsen, M. ∙ Sette, A.</div><div><strong>T Cell Epitope Predictions</strong></div><div><em>Annu. Rev. Immunol.</em> 2020; <strong>38</strong>:123-145</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-082119-124838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85084167551" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32045313/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-082119-124838&amp;pmid=32045313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">54.</a></div><div>Philip, M. ∙ Schietinger, A.</div><div><strong>CD8(+) T&nbsp;cell differentiation and dysfunction in cancer</strong></div><div><em>Nat. Rev. Immunol.</em> 2021;</div><div>Published online July 12, 2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-021-00574-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85109907412" target="_blank" aria-label="Open Scopus in new tab">Scopus (399)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34253904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-021-00574-3&amp;pmid=34253904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">55.</a></div><div>Qiu, X. ∙ Mao, Q. ∙ Tang, Y. ...</div><div><strong>Reversed graph embedding resolves complex single-cell trajectories</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:979-982</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4402" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85031017685" target="_blank" aria-label="Open Scopus in new tab">Scopus (2116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28825705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4402&amp;pmid=28825705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">56.</a></div><div>Rashid, R. ∙ Gaglia, G. ∙ Chen, Y.A. ...</div><div><strong>Highly multiplexed immunofluorescence images and single-cell data of immune markers in tonsil and lung cancer</strong></div><div><em>Sci. Data.</em> 2019; <strong>6</strong>:323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41597-019-0332-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076825816" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31848351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41597-019-0332-y&amp;pmid=31848351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">57.</a></div><div>Rosenthal, R. ∙ Cadieux, E.L. ∙ Salgado, R. ..., TRACERx Consortium</div><div><strong>Neoantigen-directed immune escape in lung cancer evolution</strong></div><div><em>Nature.</em> 2019; <strong>567</strong>:479-485</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1032-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85063265508" target="_blank" aria-label="Open Scopus in new tab">Scopus (600)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30894752/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1032-7&amp;pmid=30894752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">58.</a></div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2018.12.034&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059305897" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30633907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.12.034&amp;pmid=30633907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">59.</a></div><div>Safford, M. ∙ Collins, S. ∙ Lutz, M.A. ...</div><div><strong>Egr-2 and Egr-3 are negative regulators of T&nbsp;cell activation</strong></div><div><em>Nat. Immunol.</em> 2005; <strong>6</strong>:472-480</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni1193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-18244375249" target="_blank" aria-label="Open Scopus in new tab">Scopus (351)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15834410/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni1193&amp;pmid=15834410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">60.</a></div><div>Sahin, U. ∙ Derhovanessian, E. ∙ Miller, M. ...</div><div><strong>Personalized RNA mutanome vaccines mobilize poly-specific therapeutic immunity against cancer</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:222-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85024396858" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28678784/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23003&amp;pmid=28678784" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">61.</a></div><div>Saini, S.K. ∙ Tamhane, T. ∙ Anjanappa, R. ...</div><div><strong>Empty peptide-receptive MHC class I molecules for efficient detection of antigen-specific T&nbsp;cells</strong></div><div><em>Sci. Immunol.</em> 2019; <strong>4</strong>:eaau9039</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aau9039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069751787" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31324690/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aau9039&amp;pmid=31324690" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63-1" title="View in article">62.</a></div><div>Sánchez-Rivera, F.J. ∙ Papagiannakopoulos, T. ∙ Romero, R. ...</div><div><strong>Rapid modelling of cooperating genetic events in cancer through somatic genome editing</strong></div><div><em>Nature.</em> 2014; <strong>516</strong>:428-431</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13906" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84923118778" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25337879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13906&amp;pmid=25337879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">63.</a></div><div>Schenkel, J.M. ∙ Herbst, R.H. ∙ Canner, D. ...</div><div><strong>Conventional type I dendric cells maintain areservoir of proliferative tumor-antigen specific TCF-1<sup>+</sup> CD8<sup>+</sup> T cells in tumor-draining lymph nodes</strong></div><div><em>Immunity.</em> 2021;</div><div>–</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2021.08.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2021.08.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85115016953" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34534439/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2021.08.026&amp;pmid=34534439" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">64.</a></div><div>Scheper, W. ∙ Kelderman, S. ∙ Fanchi, L.F. ...</div><div><strong>Low and variable tumor reactivity of the intratumoral TCR repertoire in human cancers</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:89-94</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0266-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85058049841" target="_blank" aria-label="Open Scopus in new tab">Scopus (391)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30510250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0266-5&amp;pmid=30510250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">65.</a></div><div>Schreiber, H. ∙ Wu, T.H. ∙ Nachman, J. ...</div><div><strong>Immunodominance and tumor escape</strong></div><div><em>Semin. Cancer Biol.</em> 2002; <strong>12</strong>:25-31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/scbi.2001.0401" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-0036150946" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11926408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fscbi.2001.0401&amp;pmid=11926408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">66.</a></div><div>Schumacher, T.N. ∙ Heemels, M.T. ∙ Neefjes, J.J. ...</div><div><strong>Direct binding of peptide to empty MHC class I molecules on intact cells and in&nbsp;vitro</strong></div><div><em>Cell.</em> 1990; <strong>62</strong>:563-567</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2F0092-8674%2890%2990020-F&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2F0092-8674%2890%2990020-F&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-0025006862" target="_blank" aria-label="Open Scopus in new tab">Scopus (418)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2199065/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2890%2990020-F&amp;pmid=2199065" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68-1" title="View in article">67.</a></div><div>Schumacher, T.N. ∙ Scheper, W. ∙ Kvistborg, P.</div><div><strong>Cancer Neoantigens</strong></div><div><em>Annu. Rev. Immunol.</em> 2019; <strong>37</strong>:173-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-042617-053402" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85065015974" target="_blank" aria-label="Open Scopus in new tab">Scopus (374)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30550719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-042617-053402&amp;pmid=30550719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">68.</a></div><div>Scott, A.C. ∙ Dündar, F. ∙ Zumbo, P. ...</div><div><strong>TOX is a&nbsp;critical regulator of tumour-specific T&nbsp;cell differentiation</strong></div><div><em>Nature.</em> 2019; <strong>571</strong>:270-274</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1324-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85068312747" target="_blank" aria-label="Open Scopus in new tab">Scopus (665)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31207604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1324-y&amp;pmid=31207604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70-1" title="View in article">69.</a></div><div>Siddiqui, I. ∙ Schaeuble, K. ∙ Chennupati, V. ...</div><div><strong>Intratumoral Tcf1<sup>+</sup>PD-1<sup>+</sup>CD8<sup>+</sup> T Cells with Stem-like Properties Promote Tumor Control in Response to Vaccination and Checkpoint Blockade Immunotherapy</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:195-211.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.12.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2018.12.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85059786958" target="_blank" aria-label="Open Scopus in new tab">Scopus (873)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30635237/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.12.021&amp;pmid=30635237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">70.</a></div><div>Simoni, Y. ∙ Becht, E. ∙ Fehlings, M. ...</div><div><strong>Bystander CD8<sup>+</sup> T&nbsp;cells are abundant and phenotypically distinct in human tumour infiltrates</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:575-579</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0130-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85047608686" target="_blank" aria-label="Open Scopus in new tab">Scopus (868)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29769722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0130-2&amp;pmid=29769722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">71.</a></div><div>Spranger, S. ∙ Koblish, H.K. ∙ Horton, B. ...</div><div><strong>Mechanism of tumor rejection with doublets of CTLA-4, PD-1/PD-L1, or IDO blockade involves restored IL-2 production and proliferation of CD8(+) T&nbsp;cells directly within the tumor microenvironment</strong></div><div><em>J.&nbsp;Immunother. Cancer.</em> 2014; <strong>2</strong>:3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/2051-1426-2-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84977161108" target="_blank" aria-label="Open Scopus in new tab">Scopus (472)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24829760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F2051-1426-2-3&amp;pmid=24829760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73-1" title="View in article">72.</a></div><div>Srenathan, U. ∙ Steel, K. ∙ Taams, L.S.</div><div><strong>IL-17+ CD8+ T&nbsp;cells: Differentiation, phenotype and role in inflammatory disease</strong></div><div><em>Immunol. Lett.</em> 2016; <strong>178</strong>:20-26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.imlet.2016.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84971596825" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27173097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.imlet.2016.05.001&amp;pmid=27173097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">73.</a></div><div>Stoeckius, M. ∙ Zheng, S. ∙ Houck-Loomis, B. ...</div><div><strong>Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-018-1603-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85058891736" target="_blank" aria-label="Open Scopus in new tab">Scopus (518)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30567574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-018-1603-1&amp;pmid=30567574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">74.</a></div><div>Strønen, E. ∙ Toebes, M. ∙ Kelderman, S. ...</div><div><strong>Targeting of cancer neoantigens with donor-derived T&nbsp;cell receptor repertoires</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:1337-1341</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2288" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84969764539" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27198675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2288&amp;pmid=27198675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">75.</a></div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (7328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">76.</a></div><div>Subramanian, A. ∙ Tamayo, P. ∙ Mootha, V.K. ...</div><div><strong>Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:15545-15550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0506580102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-27344435774" target="_blank" aria-label="Open Scopus in new tab">Scopus (33620)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0506580102&amp;pmid=16199517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#figs7" title="View in article">77.</a></div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad0501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84963614956" target="_blank" aria-label="Open Scopus in new tab">Scopus (2762)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27124452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad0501&amp;pmid=27124452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#figs7" title="View in article">78.</a></div><div>Toubal, A. ∙ Nel, I. ∙ Lotersztajn, S. ...</div><div><strong>Mucosal-associated invariant T&nbsp;cells and disease</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:643-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0191-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069455629" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31308521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0191-y&amp;pmid=31308521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">79.</a></div><div>Traag, V.A. ∙ Waltman, L. ∙ van Eck, N.J.</div><div><strong>From Louvain to Leiden: guaranteeing well-connected communities</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:5233</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-41695-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85063527139" target="_blank" aria-label="Open Scopus in new tab">Scopus (1921)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30914743/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-41695-z&amp;pmid=30914743" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">80.</a></div><div>Tran, E. ∙ Robbins, P.F. ∙ Rosenberg, S.A.</div><div><strong>‘Final common pathway’ of human cancer immunotherapy: targeting random somatic mutations</strong></div><div><em>Nat. Immunol.</em> 2017; <strong>18</strong>:255-262</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3682" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85012974126" target="_blank" aria-label="Open Scopus in new tab">Scopus (354)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28198830/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3682&amp;pmid=28198830" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82-1" title="View in article">81.</a></div><div>Trapnell, C. ∙ Cacchiarelli, D. ∙ Grimsby, J. ...</div><div><strong>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:381-386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84900873950" target="_blank" aria-label="Open Scopus in new tab">Scopus (3557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2859&amp;pmid=24658644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">82.</a></div><div>Utzschneider, D.T. ∙ Charmoy, M. ∙ Chennupati, V. ...</div><div><strong>T Cell Factor 1-Expressing Memory-like CD8(+) T Cells Sustain the Immune Response to Chronic Viral Infections</strong></div><div><em>Immunity.</em> 2016; <strong>45</strong>:415-427</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2016.07.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2016.07.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85002609650" target="_blank" aria-label="Open Scopus in new tab">Scopus (656)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27533016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2016.07.021&amp;pmid=27533016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">83.</a></div><div>Verdegaal, E.M. ∙ de Miranda, N.F. ∙ Visser, M. ...</div><div><strong>Neoantigen landscape dynamics during human melanoma-T cell interactions</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:91-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18945" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84976406873" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27350335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18945&amp;pmid=27350335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">84.</a></div><div>Verykokakis, M. ∙ Papadaki, C. ∙ Vorgia, E. ...</div><div><strong>The RAS-dependent ERF control of cell proliferation and differentiation is mediated by c-Myc repression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:30285-30294</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1074%2Fjbc.M704428200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1074%2Fjbc.M704428200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-35748955822" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17699159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M704428200&amp;pmid=17699159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">85.</a></div><div>Warnes, G.</div><div><strong>gplots: Various R programming tools for plotting data. R Package</strong></div><div>2015</div><div><a id="intref0270" href="https://rdrr.io/cran/gplots/" target="_blank">https://rdrr.io/cran/gplots/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.Warnesgplots%3A+Various+R+programming+tools+for+plotting+data.+R+Package2015https%3A%2F%2Frdrr.io%2Fcran%2Fgplots%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">86.</a></div><div>Wells, D.K. ∙ van Buuren, M.M. ∙ Dang, K.K. ..., Tumor Neoantigen Selection Alliance</div><div><strong>Key Parameters of Tumor Epitope Immunogenicity Revealed Through a Consortium Approach Improve Neoantigen Prediction</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:818-834.e13</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2020.09.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.cell.2020.09.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85094120480" target="_blank" aria-label="Open Scopus in new tab">Scopus (274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33038342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.015&amp;pmid=33038342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">87.</a></div><div>Wu, P. ∙ Wu, D. ∙ Ni, C. ...</div><div><strong>γδT17 cells promote the accumulation and expansion of myeloid-derived suppressor cells in human colorectal cancer</strong></div><div><em>Immunity.</em> 2014; <strong>40</strong>:785-800</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2014.03.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2014.03.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84900482064" target="_blank" aria-label="Open Scopus in new tab">Scopus (468)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24816404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2014.03.013&amp;pmid=24816404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89-1" title="View in article">88.</a></div><div>Wu, T.D. ∙ Madireddi, S. ∙ de Almeida, P.E. ...</div><div><strong>Peripheral T&nbsp;cell expansion predicts tumour infiltration and clinical response</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:274-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2056-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85080085152" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32103181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2056-8&amp;pmid=32103181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90-1" title="View in article">89.</a></div><div>Yewdell, J.W.</div><div><strong>Confronting complexity: real-world immunodominance in antiviral CD8+ T&nbsp;cell responses</strong></div><div><em>Immunity.</em> 2006; <strong>25</strong>:533-543</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2006.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2006.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-33749528390" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17046682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2006.09.005&amp;pmid=17046682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#figs7" title="View in article">90.</a></div><div>Yost, K.E. ∙ Satpathy, A.T. ∙ Wells, D.K. ...</div><div><strong>Clonal replacement of tumor-specific T&nbsp;cells following PD-1 blockade</strong></div><div><em>Nat. Med.</em> 2019; <strong>25</strong>:1251-1259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-019-0522-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85069859998" target="_blank" aria-label="Open Scopus in new tab">Scopus (881)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31359002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-019-0522-3&amp;pmid=31359002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">91.</a></div><div>Zacharakis, N. ∙ Chinnasamy, H. ∙ Black, M. ...</div><div><strong>Immune recognition of somatic mutations leading to complete durable regression in metastatic breast cancer</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:724-730</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0040-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85048021996" target="_blank" aria-label="Open Scopus in new tab">Scopus (589)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29867227/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0040-8&amp;pmid=29867227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">92.</a></div><div>Zander, R. ∙ Schauder, D. ∙ Xin, G. ...</div><div><strong>CD4<sup>+</sup> T Cell Help Is Required for the Formation of a Cytolytic CD8<sup>+</sup> T Cell Subset that Protects against Chronic Infection and Cancer</strong></div><div><em>Immunity.</em> 2019; <strong>51</strong>:1028-1042.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2019.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1016%2Fj.immuni.2019.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85076189014" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31810883/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2019.10.009&amp;pmid=31810883" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">93.</a></div><div>Zhao, J. ∙ Chen, X. ∙ Herjan, T. ...</div><div><strong>The role of interleukin-17 in tumor development and progression</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2020; <strong>217</strong>:e20190297</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20190297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31727782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20190297&amp;pmid=31727782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95-1" title="View in article">94.</a></div><div>Zheng, G.X. ∙ Terry, J.M. ∙ Belgrader, P. ...</div><div><strong>Massively parallel digital transcriptional profiling of single cells</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14049</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-85009446777" target="_blank" aria-label="Open Scopus in new tab">Scopus (3501)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14049&amp;pmid=28091601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">95.</a></div><div>Zhuang, Y. ∙ Peng, L.S. ∙ Zhao, Y.L. ...</div><div><strong>CD8(+) T&nbsp;cells that produce interleukin-17 regulate myeloid-derived suppressor cells and are associated with survival time of patients with gastric cancer</strong></div><div><em>Gastroenterology.</em> 2012; <strong>143</strong>:951</div><div>62.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1053%2Fj.gastro.2012.06.010&amp;cf=fulltext&amp;site=ygast-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=10.1053%2Fj.gastro.2012.06.010&amp;cf=pdf&amp;site=ygast-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.08.020&amp;key=2-s2.0-84866733757" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22710190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1053%2Fj.gastro.2012.06.010&amp;pmid=22710190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/f9f5c693-ce08-429e-accd-c63c625ce422/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/1a56d60e-3afb-42ce-9a33-b0969747cd70/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/cdc7e560-4d19-4082-a375-9d7a289d7122/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/12d36262-77b2-4645-b8ae-d336cf1756f9/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/2f84430c-9cda-42f7-988c-54067b657aab/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/e4bcfd47-cc7e-4d14-8040-bd09a617b5e4/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/d7dd3ad0-69c3-4a84-b4f8-5420c9e3104d/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/383d0732-efc3-4698-93e6-f67753a4a822/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/551ceadb-6fa1-4f99-985c-432d27d0e115/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/8385286e-9c1b-408d-8dee-d0c1ab88052b/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/4bef12f6-8a06-4c83-9cde-5261625e76a7/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/119b0925-e1e9-47cd-976d-4b55d3ec62d7/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/0843a143-7377-433d-8607-2b63ebb42301/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/asset/2ea710a0-9bbc-47f0-b838-e0d0643d2893/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Barticle%3Aarticle%3Apii%5C%3AS0092867421009946%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00201%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (7)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/attachment/2f66928c-83ff-45ed-bee7-19aaa0f264e2/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (208.29 KB)</span></a><div class="core-description">Table S1. ProjecTILs cell state assignments for the mouse scRNA-seq data and enrichment statistics for SIIN- and SIY-specific cells, related to Figures 2 and S2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/attachment/2da968df-a656-4072-8a39-c3bc04caac3a/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (2.70 MB)</span></a><div class="core-description">Table S2. Published mouse gene signatures projected onto the mouse scRNA-seq data, related to Figures 5, S2, and S5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/attachment/4d0f636c-4cbd-4755-ae0c-270b401553a6/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (356.53 KB)</span></a><div class="core-description">Table S3. Top differentially expressed genes and gene signatures for the mouse scRNA-seq clusters and CCR6<sup>+</sup>/CCR6<sup>−</sup> TCF1<sup>+</sup> cell subsets, related to Figures 2, 5, and 7</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/attachment/89739073-b202-46d2-a67b-b30efc469088/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (58.21 KB)</span></a><div class="core-description">Table S4. Published human gene signatures projected onto the mouse scRNA-seq data, related to Figures 7 and S7</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/attachment/b3df9c7a-28f0-42a8-aeff-64e4b6cc592e/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (10.30 KB)</span></a><div class="core-description">Table S5. Cell type calling parameters for analysis of mouse and human t-CyCIF images, related to Figures 6 and 7</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/attachment/5bd22f2c-d7ca-42b0-9e5c-78e46c8d445b/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (3.51 MB)</span></a><div class="core-description">Table S6. Gene co-expression cell assignments for the tumor CD8 T&nbsp;cell analysis of human scRNA-seq datasets, related to Figures 7 and S7</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.08.020/attachment/61a70147-de97-4d6d-9543-eeb283891ea7/mmc7.xlsx" download="mmc7.xlsx"><span id="mmc7-link">Spreadsheet (13.00 KB)</span></a><div class="core-description">Table S7. Clinical information for lung adenocarcinoma and melanoma samples profiled by t-CyCIF, related to Figure&nbsp;7</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421009946" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.08.020","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Antigen dominance hierarchies shape TCF1+ progenitor CD8 T&nbsp;cell phenotypes in tumors</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.08.020/asset/f9f5c693-ce08-429e-accd-c63c625ce422/main.assets/gr1.jpg" height="631" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/43e8cbf1-a514-4c0a-b0dd-5bca1a070c48/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.08.020/asset/1a56d60e-3afb-42ce-9a33-b0969747cd70/main.assets/figs1.jpg" height="432" width="506" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/1940e08e-eb95-4630-bfed-7bbef6d4f87f/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.08.020/asset/cdc7e560-4d19-4082-a375-9d7a289d7122/main.assets/gr2.jpg" height="768" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/e3b37f14-ebd0-45a5-b28e-68be642339d1/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.08.020/asset/12d36262-77b2-4645-b8ae-d336cf1756f9/main.assets/figs2.jpg" height="939" width="751" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/fb8b7bd2-d50c-4615-b040-4ffcd4fb7726/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.08.020/asset/2f84430c-9cda-42f7-988c-54067b657aab/main.assets/figs4.jpg" height="931" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/17ad1dc4-9532-48b8-a9d7-ef96c84e2a1c/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.08.020/asset/e4bcfd47-cc7e-4d14-8040-bd09a617b5e4/main.assets/gr3.jpg" height="671" width="753" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/0d3a188a-01d6-4e2a-bfef-62439c659e0d/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.08.020/asset/d7dd3ad0-69c3-4a84-b4f8-5420c9e3104d/main.assets/figs3.jpg" height="837" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/9a1c5285-7abf-4dee-88d0-404ccb745fd3/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.08.020/asset/383d0732-efc3-4698-93e6-f67753a4a822/main.assets/gr4.jpg" height="656" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/45c10afd-6196-4793-8db2-7f8487f4ffbc/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.08.020/asset/551ceadb-6fa1-4f99-985c-432d27d0e115/main.assets/gr5.jpg" height="941" width="730" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/04750a5c-6941-4610-b724-d634550950e6/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.08.020/asset/8385286e-9c1b-408d-8dee-d0c1ab88052b/main.assets/figs5.jpg" height="911" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/edbba6dc-0c38-480a-bd9e-49ef2eb4a786/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.08.020/asset/4bef12f6-8a06-4c83-9cde-5261625e76a7/main.assets/gr6.jpg" height="941" width="637" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/8d9ceff0-dd4c-44d9-b0e4-8579bd03e413/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.08.020/asset/119b0925-e1e9-47cd-976d-4b55d3ec62d7/main.assets/figs6.jpg" height="941" width="746" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/859b5d11-4346-4fca-ac86-b61262096fb6/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.08.020/asset/0843a143-7377-433d-8607-2b63ebb42301/main.assets/gr7.jpg" height="942" width="717" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/c08187bf-0e9e-4a71-8444-a314fef839e1/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.08.020/asset/2ea710a0-9bbc-47f0-b838-e0d0643d2893/main.assets/figs7.jpg" height="939" width="730" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/a4f2fcc6-fccf-4d44-9b0a-dc3d7130dd12/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.08.020/asset/f9f5c693-ce08-429e-accd-c63c625ce422/main.assets/gr1.jpg" height="631" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/43e8cbf1-a514-4c0a-b0dd-5bca1a070c48/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.08.020/asset/1a56d60e-3afb-42ce-9a33-b0969747cd70/main.assets/figs1.jpg" height="432" width="506" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/1940e08e-eb95-4630-bfed-7bbef6d4f87f/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.08.020/asset/cdc7e560-4d19-4082-a375-9d7a289d7122/main.assets/gr2.jpg" height="768" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/e3b37f14-ebd0-45a5-b28e-68be642339d1/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.08.020/asset/12d36262-77b2-4645-b8ae-d336cf1756f9/main.assets/figs2.jpg" height="939" width="751" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/fb8b7bd2-d50c-4615-b040-4ffcd4fb7726/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.08.020/asset/2f84430c-9cda-42f7-988c-54067b657aab/main.assets/figs4.jpg" height="931" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/17ad1dc4-9532-48b8-a9d7-ef96c84e2a1c/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.08.020/asset/e4bcfd47-cc7e-4d14-8040-bd09a617b5e4/main.assets/gr3.jpg" height="671" width="753" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/0d3a188a-01d6-4e2a-bfef-62439c659e0d/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.08.020/asset/d7dd3ad0-69c3-4a84-b4f8-5420c9e3104d/main.assets/figs3.jpg" height="837" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/9a1c5285-7abf-4dee-88d0-404ccb745fd3/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.08.020/asset/383d0732-efc3-4698-93e6-f67753a4a822/main.assets/gr4.jpg" height="656" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/45c10afd-6196-4793-8db2-7f8487f4ffbc/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.08.020/asset/551ceadb-6fa1-4f99-985c-432d27d0e115/main.assets/gr5.jpg" height="941" width="730" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/04750a5c-6941-4610-b724-d634550950e6/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.08.020/asset/8385286e-9c1b-408d-8dee-d0c1ab88052b/main.assets/figs5.jpg" height="911" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/edbba6dc-0c38-480a-bd9e-49ef2eb4a786/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.08.020/asset/4bef12f6-8a06-4c83-9cde-5261625e76a7/main.assets/gr6.jpg" height="941" width="637" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/8d9ceff0-dd4c-44d9-b0e4-8579bd03e413/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.08.020/asset/119b0925-e1e9-47cd-976d-4b55d3ec62d7/main.assets/figs6.jpg" height="941" width="746" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/859b5d11-4346-4fca-ac86-b61262096fb6/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.08.020/asset/0843a143-7377-433d-8607-2b63ebb42301/main.assets/gr7.jpg" height="942" width="717" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/c08187bf-0e9e-4a71-8444-a314fef839e1/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.08.020/asset/2ea710a0-9bbc-47f0-b838-e0d0643d2893/main.assets/figs7.jpg" height="939" width="730" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.08.020/asset/a4f2fcc6-fccf-4d44-9b0a-dc3d7130dd12/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00994-6';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00994-6';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749591474891","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"19","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421009946","coverDate":"202109","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Antigen dominance hierarchies shape TCF1supsup progenitor CD8 Txa0cell phenotypes in tumors","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc03369ecd4ee3',t:'MTc0OTU5MTQ3NC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc03369ecd4ee3&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.6.2&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.08.020&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>