<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="e33318cc-9130-45b2-962b-81b599d47738" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;issue:issue:pii\:S0092867421X0013X;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867422004585;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Zn-regulated GTPase metalloprotein activator 1 modulates vertebrate zinc homeostasis">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(22)00458-5">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/37d5261a-fcc2-40a7-99f4-da74ff659b70/fx1.jpg">
    


        <meta property="og:description" content="ZNG1 is a putative Zn metallochaperone that supports the activity of Zn-dependent
proteins in vertebrates, thereby maintaining Zn homeostasis and cellular integrity.">
    











        <meta name="twitter:title" content="Zn-regulated GTPase metalloprotein activator 1 modulates vertebrate zinc homeostasis">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/37d5261a-fcc2-40a7-99f4-da74ff659b70/fx1.jpg">
    


        <meta name="twitter:description" content="ZNG1 is a putative Zn metallochaperone that supports the activity of Zn-dependent
proteins in vertebrates, thereby maintaining Zn homeostasis and cellular integrity.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(22)00458-5">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Zn-regulated GTPase metalloprotein activator 1 modulates vertebrate zinc homeostasis: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Zn-regulated GTPase metalloprotein activator 1 modulates vertebrate zinc homeostasis"> <meta name="citation_date" content="2022/06/09"> <meta name="citation_online_date" content="2022/05/17"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="12"> <meta name="citation_firstpage" content="2148"> <meta name="citation_lastpage" content="2163.e27"> <meta name="citation_pmid" content="35584702"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867422004585"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867422004585"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867422004585"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2022.04.011"> <meta name="citation_pmid" content="35584702"> <meta name="citation_author" content="Andy Weiss"><meta name="citation_author_orcid" content="0000-0002-5221-1032"><meta name="citation_author_institution" content="Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA"><meta name="citation_author" content="Caitlin C. Murdoch"><meta name="citation_author_institution" content="Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA"><meta name="citation_author" content="Katherine A. Edmonds"><meta name="citation_author_orcid" content="0000-0002-1282-9858"><meta name="citation_author_institution" content="Department of Chemistry, Indiana University, Bloomington, IN 47405, USA"><meta name="citation_author" content="Matthew R. Jordan"><meta name="citation_author_orcid" content="0000-0003-0095-8078"><meta name="citation_author_institution" content="Department of Chemistry, Indiana University, Bloomington, IN 47405, USA"><meta name="citation_author_institution" content="Department of Molecular and Cellular Biochemistry, Indiana University, Bloomington, IN 47405, USA"><meta name="citation_author" content="Andrew J. Monteith"><meta name="citation_author_institution" content="Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA"><meta name="citation_author" content="Yasiru R. Perera"><meta name="citation_author_institution" content="Departments of Biochemistry and Chemistry, Center for Structural Biology, Vanderbilt University, Nashville, TN 37240, USA"><meta name="citation_author" content="Aslin M. Rodríguez Nassif"><meta name="citation_author_institution" content="Departments of Biochemistry and Chemistry, Center for Structural Biology, Vanderbilt University, Nashville, TN 37240, USA"><meta name="citation_author" content="Amber M. Petoletti"><meta name="citation_author_orcid" content="0000-0002-8287-0274"><meta name="citation_author_institution" content="Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA"><meta name="citation_author" content="William N. Beavers"><meta name="citation_author_institution" content="Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA"><meta name="citation_author" content="Matthew J. Munneke"><meta name="citation_author_institution" content="Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA"><meta name="citation_author" content="Sydney L. Drury"><meta name="citation_author_institution" content="Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA"><meta name="citation_author" content="Evan S. Krystofiak"><meta name="citation_author_institution" content="Cell Imaging Shared Resource, Vanderbilt University, Nashville, TN 37232, USA"><meta name="citation_author" content="Kishore Thalluri"><meta name="citation_author_orcid" content="0000-0001-5862-4631"><meta name="citation_author_institution" content="Department of Chemistry, Indiana University, Bloomington, IN 47405, USA"><meta name="citation_author" content="Hongwei Wu"><meta name="citation_author_institution" content="Department of Chemistry, Indiana University, Bloomington, IN 47405, USA"><meta name="citation_author" content="Angela R.S. Kruse"><meta name="citation_author_institution" content="Departments of Chemistry and Biochemistry, Mass Spectrometry Research Center, Vanderbilt University, Nashville, TN 37235, USA"><meta name="citation_author" content="Richard D. DiMarchi"><meta name="citation_author_institution" content="Department of Chemistry, Indiana University, Bloomington, IN 47405, USA"><meta name="citation_author" content="Richard M. Caprioli"><meta name="citation_author_institution" content="Departments of Chemistry and Biochemistry, Mass Spectrometry Research Center, Vanderbilt University, Nashville, TN 37235, USA"><meta name="citation_author" content="Jeffrey M. Spraggins"><meta name="citation_author_institution" content="Departments of Chemistry and Biochemistry, Mass Spectrometry Research Center, Vanderbilt University, Nashville, TN 37235, USA"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN 37232, USA"><meta name="citation_author" content="Walter J. Chazin"><meta name="citation_author_orcid" content="0000-0002-2180-0790"><meta name="citation_author_institution" content="Departments of Biochemistry and Chemistry, Center for Structural Biology, Vanderbilt University, Nashville, TN 37240, USA"><meta name="citation_author" content="David P. Giedroc"><meta name="citation_author_email" content="giedroc@indiana.edu"><meta name="citation_author_institution" content="Department of Chemistry, Indiana University, Bloomington, IN 47405, USA"><meta name="citation_author_institution" content="Department of Molecular and Cellular Biochemistry, Indiana University, Bloomington, IN 47405, USA"><meta name="citation_author" content="Eric P. Skaar"><meta name="citation_author_email" content="eric.skaar@vumc.org"><meta name="citation_author_institution" content="Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA"><meta name="citation_keywords" content="zinc; zinc finger; metalloprotein; GTPase; COG0523; ZNG1; CBWD; METAP1; zf-C6H2; metallochaperone"><meta name="citation_reference" content="citation_title=Structural basis for the functional differences between type I and type II human methionine aminopeptidases; citation_author=A. Addlagatta; citation_author=X. Hu; citation_author=J.O. Liu; citation_author=B.W. Matthews; citation_journal_title=Biochemistry; citation_volume=44; citation_firstpage=14741; citation_lastpage=14749; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Counting the zinc-proteins encoded in the human genome; citation_author=C. Andreini; citation_author=L. Banci; citation_author=I. Bertini; citation_author=A. Rosato; citation_journal_title=J.&nbsp;Proteome Res.; citation_volume=5; citation_firstpage=196; citation_lastpage=201; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Cotranslational processing and protein turnover in eukaryotic cells; citation_author=S.M. Arfin; citation_author=R.A. Bradshaw; citation_journal_title=Biochemistry; citation_volume=27; citation_firstpage=7979; citation_lastpage=7984; citation_publication_date=1988; "><meta name="citation_reference" content="citation_title=Metal trafficking: from maintaining the metal homeostasis to future drug design; citation_author=L.A. Ba; citation_author=M. Doering; citation_author=T. Burkholz; citation_author=C. Jacob; citation_journal_title=Metallomics; citation_volume=1; citation_firstpage=292; citation_lastpage=311; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=DNA damage, mutagenesis and cancer; citation_author=A.K. Basu; citation_journal_title=Int. J. Mol. Sci.; citation_volume=19; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The protein-binding potential of C2H2 zinc finger domains; citation_author=K.J. Brayer; citation_author=S. Kulshreshtha; citation_author=D.J. Segal; citation_journal_title=Cell Biochem. Biophys.; citation_volume=51; citation_firstpage=9; citation_lastpage=19; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Inhibition of ATPIF1 ameliorates severe mitochondrial respiratory chain dysfunction in mammalian cells; citation_author=W.W. Chen; citation_author=K. Birsoy; citation_author=M.M. Mihaylova; citation_author=H. Snitkin; citation_author=I. Stasinski; citation_author=B. Yucel; citation_author=E.C. Bayraktar; citation_author=J.E. Carette; citation_author=C.B. Clish; citation_author=T.R. Brummelkamp; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=7; citation_firstpage=27; citation_lastpage=34; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Identification of the human zinc transcriptional regulatory element (ZTRE): a palindromic protein-binding DNA sequence responsible for zinc-induced transcriptional repression; citation_author=L.J. Coneyworth; citation_author=K.A. Jackson; citation_author=J. Tyson; citation_author=H.J. Bosomworth; citation_author=E. van der Hagen; citation_author=G.M. Hann; citation_author=O.A. Ogo; citation_author=D.C. Swann; citation_author=J.C. Mathers; citation_author=R.A. Valentine; citation_author=D. Ford; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=287; citation_firstpage=36567; citation_lastpage=36581; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ; citation_author=J. Cox; citation_author=M.Y. Hein; citation_author=C.A. Luber; citation_author=I. Paron; citation_author=N. Nagaraj; citation_author=M. Mann; citation_journal_title=Mol. Cell. Proteomics; citation_volume=13; citation_firstpage=2513; citation_lastpage=2526; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=WebLogo: a sequence logo generator; citation_author=G.E. Crooks; citation_author=G. Hon; citation_author=J.M. Chandonia; citation_author=S.E. Brenner; citation_journal_title=Genome Res.; citation_volume=14; citation_firstpage=1188; citation_lastpage=1190; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=NMRPipe: a multidimensional spectral processing system based on UNIX pipes; citation_author=F. Delaglio; citation_author=S. Grzesiek; citation_author=G.W. Vuister; citation_author=G. Zhu; citation_author=J. Pfeifer; citation_author=A. Bax; citation_journal_title=J.&nbsp;Biomol. NMR; citation_volume=6; citation_firstpage=277; citation_lastpage=293; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Zinc fluxes and zinc transporter genes in chronic diseases; citation_author=C. Devirgiliis; citation_author=P.D. Zalewski; citation_author=G. Perozzi; citation_author=C. Murgia; citation_journal_title=Mutat. Res.; citation_volume=622; citation_firstpage=84; citation_lastpage=93; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=A new generation of homology search tools based on probabilistic inference; citation_author=S.R. Eddy; citation_journal_title=Genome Inform.; citation_volume=23; citation_firstpage=205; citation_lastpage=211; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=COG0523 proteins: a functionally diverse family of transition metal-regulated G3E P-loop GTP hydrolases from bacteria to man; citation_author=K.A. Edmonds; citation_author=M.R. Jordan; citation_author=D.P. Giedroc; citation_journal_title=Metallomics; citation_volume=13; citation_firstpage=mfab046; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Zinc transporters and the cellular trafficking of zinc; citation_author=D.J. Eide; citation_journal_title=Biochim. Biophys. Acta; citation_volume=1763; citation_firstpage=711; citation_lastpage=722; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Zebrafish Embryo Microinjection: Ribonucleoprotein Delivery Using the Alt-R CRISPR-Cas9 System; citation_author=J. Essner; citation_publication_date=2016; citation_publisher=Integrated DNA Technologies; "><meta name="citation_reference" content="citation_title=DcGO: database of domain-centric ontologies on functions, phenotypes, diseases and more; citation_author=H. Fang; citation_author=J. Gough; citation_journal_title=Nucleic Acids Res.; citation_volume=41; citation_firstpage=D536; citation_lastpage=D544; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=C2H2 zinc finger proteins: the largest but poorly explored family of higher eukaryotic transcription factors; citation_author=A.A. Fedotova; citation_author=A.N. Bonchuk; citation_author=V.A. Mogila; citation_author=P.G. Georgiev; citation_journal_title=Acta Nat.; citation_volume=9; citation_firstpage=47; citation_lastpage=58; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Zinc and the risk for infectious disease; citation_author=C. Fischer Walker; citation_author=R.E. Black; citation_journal_title=Annu. Rev. Nutr.; citation_volume=24; citation_firstpage=255; citation_lastpage=275; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Role of superoxide radical in mitochondrial dehydrogenase reactions; citation_author=H.J. Forman; citation_author=J.A. Kennedy; citation_journal_title=Biochem. Biophys. Res. Commun.; citation_volume=60; citation_firstpage=1044; citation_lastpage=1050; citation_publication_date=1974; "><meta name="citation_reference" content="citation_title=The proteomics of N-terminal methionine cleavage; citation_author=F. Frottin; citation_author=A. Martinez; citation_author=P. Peynot; citation_author=S. Mitra; citation_author=R.C. Holz; citation_author=C. Giglione; citation_author=T. Meinnel; citation_journal_title=Mol. Cell. Proteomics; citation_volume=5; citation_firstpage=2336; citation_lastpage=2349; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Early scanning of nascent polypeptides inside the ribosomal tunnel by NAC; citation_author=M. Gamerdinger; citation_author=K. Kobayashi; citation_author=A. Wallisch; citation_author=S.G. Kreft; citation_author=C. Sailer; citation_author=R. Schlömer; citation_author=N. Sachs; citation_author=A. Jomaa; citation_author=F. Stengel; citation_author=N. Ban; citation_author=E. Deuerling; citation_journal_title=Mol. Cell; citation_volume=75; citation_firstpage=996; citation_lastpage=1006.e8; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The ATPase inhibitory Factor 1 (IF1): a master regulator of energy metabolism and of cell survival; citation_author=J. García-Bermúdez; citation_author=J.M. Cuezva; citation_journal_title=Biochim. Biophys. Acta; citation_volume=1857; citation_firstpage=1167; citation_lastpage=1182; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Positional proteomics reveals differences in N-terminal proteoform stability; citation_author=D. Gawron; citation_author=E. Ndah; citation_author=K. Gevaert; citation_author=P. Van Damme; citation_journal_title=Mol. Syst. Biol.; citation_volume=12; citation_firstpage=858; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The eukaryotic N-end rule pathway: conserved mechanisms and diverse functions; citation_author=D.J. Gibbs; citation_author=J. Bacardit; citation_author=A. Bachmair; citation_author=M.J. Holdsworth; citation_journal_title=Trends Cell Biol.; citation_volume=24; citation_firstpage=603; citation_lastpage=611; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Enzymatic assembly of overlapping DNA fragments; citation_author=D.G. Gibson; citation_journal_title=Methods Enzymol.; citation_volume=498; citation_firstpage=349; citation_lastpage=361; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Enzymatic assembly of DNA molecules up to several hundred kilobases; citation_author=D.G. Gibson; citation_author=L. Young; citation_author=R.Y. Chuang; citation_author=J.C. Venter; citation_author=C.A. Hutchison; citation_author=H.O. Smith; citation_journal_title=Nat. Methods; citation_volume=6; citation_firstpage=343; citation_lastpage=345; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Protein N-terminal methionine excision; citation_author=C. Giglione; citation_author=A. Boularot; citation_author=T. Meinnel; citation_journal_title=Cell. Mol. Life Sci.; citation_volume=61; citation_firstpage=1455; citation_lastpage=1474; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=DEAF-1, a novel protein that binds an essential region in a deformed response element; citation_author=C.T. Gross; citation_author=W. McGinnis; citation_journal_title=EMBO J.; citation_volume=15; citation_firstpage=1961; citation_lastpage=1970; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=The world health report 2002—reducing risks, promoting healthy life; citation_author=J.J. Guilbert; citation_journal_title=Educ. Health (Abingdon); citation_volume=16; citation_firstpage=230; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Automated NMR structure calculation with CYANA; citation_author=P. Güntert; citation_journal_title=Methods Mol. Biol.; citation_volume=278; citation_firstpage=353; citation_lastpage=378; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A subset of the diverse COG0523 family of putative metal chaperones is linked to zinc homeostasis in all kingdoms of life; citation_author=C.E. Haas; citation_author=D.A. Rodionov; citation_author=J. Kropat; citation_author=D. Malasarn; citation_author=S.S. Merchant; citation_author=V. de Crécy-Lagard; citation_journal_title=BMC Genomics; citation_volume=10; citation_firstpage=470; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Oxidative phosphorylation and ultrastructural transformation in mitochondria in the intact ascites tumor cell; citation_author=C.R. Hackenbrock; citation_author=T.G. Rehn; citation_author=E.C. Weinbach; citation_author=J.J. Lemasters; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=51; citation_firstpage=123; citation_lastpage=137; citation_publication_date=1971; "><meta name="citation_reference" content="citation_title=S-trap, an ultrafast sample-preparation approach for shotgun proteomics; citation_author=M. Hailemariam; citation_author=R.V. Eguez; citation_author=H. Singh; citation_author=S. Bekele; citation_author=G. Ameni; citation_author=R. Pieper; citation_author=Y. Yu; citation_journal_title=J.&nbsp;Proteome Res.; citation_volume=17; citation_firstpage=2917; citation_lastpage=2924; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=BS69/ZMYND11 C-terminal domains bind and inhibit EBNA2; citation_author=M.R. Harter; citation_author=C.D. Liu; citation_author=C.L. Shen; citation_author=E. Gonzalez-Hurtado; citation_author=Z.M. Zhang; citation_author=M. Xu; citation_author=E. Martinez; citation_author=C.W. Peng; citation_author=J. Song; citation_journal_title=PLoS Pathog.; citation_volume=12; citation_firstpage=e1005414; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Stereospecific assignments of the isopropyl methyl groups of the membrane protein OmpX in DHPC micelles; citation_author=C. Hilty; citation_author=G. Wider; citation_author=C. Fernández; citation_author=K. Wüthrich; citation_journal_title=J.&nbsp;Biomol. NMR; citation_volume=27; citation_firstpage=377; citation_lastpage=382; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Ensembl 2021; citation_author=K.L. Howe; citation_author=P. Achuthan; citation_author=J. Allen; citation_author=J. Allen; citation_author=J. Alvarez-Jarreta; citation_author=M.R. Amode; citation_author=I.M. Armean; citation_author=A.G. Azov; citation_author=R. Bennett; citation_author=J. Bhai; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=49; citation_firstpage=D884; citation_lastpage=D891; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Elucidation of the function of type 1 human methionine aminopeptidase during cell cycle progression; citation_author=X. Hu; citation_author=A. Addlagatta; citation_author=J. Lu; citation_author=B.W. Matthews; citation_author=J.O. Liu; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=103; citation_firstpage=18148; citation_lastpage=18153; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Tris(2-pyridylmethyl)amine (TPA) as a membrane-permeable chelator for interception of biological mobile zinc; citation_author=Z. Huang; citation_author=X.A. Zhang; citation_author=M. Bosch; citation_author=S.J. Smith; citation_author=S.J. Lippard; citation_journal_title=Metallomics; citation_volume=5; citation_firstpage=648; citation_lastpage=655; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Identification of cryptic subunits from an apicomplexan ATP synthase; citation_author=D. Huet; citation_author=E. Rajendran; citation_author=G.G. van Dooren; citation_author=S. Lourido; citation_journal_title=eLife; citation_volume=7; citation_firstpage=e38097; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The BioPlex network: a systematic exploration of the human interactome; citation_author=E.L. Huttlin; citation_author=L. Ting; citation_author=R.J. Bruckner; citation_author=F. Gebreab; citation_author=M.P. Gygi; citation_author=J. Szpyt; citation_author=S. Tam; citation_author=G. Zarraga; citation_author=G. Colby; citation_author=K. Baltier; citation_author=et al.; citation_journal_title=Cell; citation_volume=162; citation_firstpage=425; citation_lastpage=440; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Application of iterative soft thresholding for fast reconstruction of NMR data non-uniformly sampled with multidimensional poisson gap scheduling; citation_author=S.G. Hyberts; citation_author=A.G. Milbradt; citation_author=A.B. Wagner; citation_author=H. Arthanari; citation_author=G. Wagner; citation_journal_title=J.&nbsp;Biomol. NMR; citation_volume=52; citation_firstpage=315; citation_lastpage=327; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Poisson-gap sampling and forward maximum entropy reconstruction for enhancing the resolution and sensitivity of protein NMR data; citation_author=S.G. Hyberts; citation_author=K. Takeuchi; citation_author=G. Wagner; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=132; citation_firstpage=2145; citation_lastpage=2147; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A continuous, regenerative coupled GTPase assay for dynamin-related proteins; citation_author=E. Ingerman; citation_author=J. Nunnari; citation_journal_title=Methods Enzymol.; citation_volume=404; citation_firstpage=611; citation_lastpage=619; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Efficient multiplex biallelic zebrafish genome editing using a CRISPR nuclease system; citation_author=L.E. Jao; citation_author=S.R. Wente; citation_author=W. Chen; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=13904; citation_lastpage=13909; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Omics assisted N-terminal Proteoform and protein expression profiling on methionine aminopeptidase 1 (MetAP1) deletion; citation_author=V. Jonckheere; citation_author=D. Fijałkowska; citation_author=P. Van Damme; citation_journal_title=Mol. Cell. Proteomics; citation_volume=17; citation_firstpage=694; citation_lastpage=708; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Mechanistic insights into the metal-dependent activation of Zn(II)-dependent metallochaperones; citation_author=M.R. Jordan; citation_author=J. Wang; citation_author=A. Weiss; citation_author=E.P. Skaar; citation_author=D.A. Capdevila; citation_author=D.P. Giedroc; citation_journal_title=Inorg. Chem.; citation_volume=58; citation_firstpage=13661; citation_lastpage=13672; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Highly accurate protein structure prediction with AlphaFold; citation_author=J. Jumper; citation_author=R. Evans; citation_author=A. Pritzel; citation_author=T. Green; citation_author=M. Figurnov; citation_author=O. Ronneberger; citation_author=K. Tunyasuvunakool; citation_author=R. Bates; citation_author=A. Žídek; citation_author=A. Potapenko; citation_author=et al.; citation_journal_title=Nature; citation_volume=596; citation_firstpage=583; citation_lastpage=589; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Kinetics and mechanism of the ninhydrin reaction with DL-methionine in the absence and the presence of organic solvents; citation_author=Kabir-ud-Din; citation_author=M. Bano; citation_author=I.A. Khan; citation_journal_title=Indian J. Chem.; citation_volume=42B; citation_firstpage=1132; citation_lastpage=1136; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Structural and functional analysis of the DEAF-1 and BS69 MYND domains; citation_author=F. Kateb; citation_author=H. Perrin; citation_author=K. Tripsianes; citation_author=P. Zou; citation_author=R. Spadaccini; citation_author=M. Bottomley; citation_author=T.M. Franzmann; citation_author=J. Buchner; citation_author=S. Ansieau; citation_author=M. Sattler; citation_journal_title=PLoS One; citation_volume=8; citation_firstpage=e54715; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The N-terminal methionine of cellular proteins as a degradation signal; citation_author=H.K. Kim; citation_author=R.R. Kim; citation_author=J.H. Oh; citation_author=H. Cho; citation_author=A. Varshavsky; citation_author=C.S. Hwang; citation_journal_title=Cell; citation_volume=156; citation_firstpage=158; citation_lastpage=169; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Causal analysis approaches in ingenuity pathway analysis; citation_author=A. Krämer; citation_author=J. Green; citation_author=J. Pollard; citation_author=S. Tugendreich; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=523; citation_lastpage=530; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The functions of metamorphic metallothioneins in zinc and copper metabolism; citation_author=A. Krężel; citation_author=W. Maret; citation_journal_title=Int. J. Mol. Sci.; citation_volume=18; citation_firstpage=1237; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=TimeTree: a resource for timelines, Timetrees, and divergence times; citation_author=S. Kumar; citation_author=G. Stecher; citation_author=M. Suleski; citation_author=S.B. Hedges; citation_journal_title=Time. Mol. Biol. Evol.; citation_volume=34; citation_firstpage=1812; citation_lastpage=1819; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Program DYNAFIT for the analysis of enzyme kinetic data: application to HIV proteinase; citation_author=P. Kuzmic; citation_journal_title=Anal. Biochem.; citation_volume=237; citation_firstpage=260; citation_lastpage=273; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy; citation_author=W. Lee; citation_author=M. Tonelli; citation_author=J.L. Markley; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=1325; citation_lastpage=1327; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Which one among Zn(II), Co(II), Mn(II), and Fe(II) is the most efficient ion for the methionine aminopeptidase catalyzed reaction?; citation_author=M. Leopoldini; citation_author=N. Russo; citation_author=M. Toscano; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=129; citation_firstpage=7776; citation_lastpage=7784; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Amino-terminal protein processing in Saccharomyces cerevisiae is an essential function that requires two distinct methionine aminopeptidases; citation_author=X. Li; citation_author=Y.H. Chang; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=92; citation_firstpage=12357; citation_lastpage=12361; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=TTC5 mediates autoregulation of tubulin via mRNA degradation; citation_author=Z. Lin; citation_author=I. Gasic; citation_author=V. Chandrasekaran; citation_author=N. Peters; citation_author=S. Shao; citation_author=T.J. Mitchison; citation_author=R.S. Hegde; citation_journal_title=Science; citation_volume=367; citation_firstpage=100; citation_lastpage=104; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Structural basis for recognition of SMRT/N-CoR by the MYND domain and its contribution to AML1/ETO’s activity; citation_author=Y. Liu; citation_author=W. Chen; citation_author=J. Gaudet; citation_author=M.D. Cheney; citation_author=L. Roudaia; citation_author=T. Cierpicki; citation_author=R.C. Klet; citation_author=K. Hartman; citation_author=T.M. Laue; citation_author=N.A. Speck; citation_author=J.H. Bushweller; citation_journal_title=Cancer Cell; citation_volume=11; citation_firstpage=483; citation_lastpage=497; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta C(T)) method; citation_author=K.J. Livak; citation_author=T.D. Schmittgen; citation_journal_title=Methods; citation_volume=25; citation_firstpage=402; citation_lastpage=408; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Single cell analysis reveals multiple requirements for zinc in the mammalian cell cycle; citation_author=M.N. Lo; citation_author=L.J. Damon; citation_author=J. Wei Tay; citation_author=S. Jia; citation_author=A.E. Palmer; citation_journal_title=eLife; citation_volume=9; citation_firstpage=e51107; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Improved pulse sequences for sequence specific assignment of aromatic proton resonances in proteins; citation_author=F. Löhr; citation_author=R. Hänsel; citation_author=V.V. Rogov; citation_author=V. Dötsch; citation_journal_title=J.&nbsp;Biomol. NMR; citation_volume=37; citation_firstpage=205; citation_lastpage=224; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Superoxide radicals as precursors of mitochondrial hydrogen peroxide; citation_author=G. Loschen; citation_author=A. Azzi; citation_author=C. Richter; citation_author=L. Flohé; citation_journal_title=FEBS Lett.; citation_volume=42; citation_firstpage=68; citation_lastpage=72; citation_publication_date=1974; "><meta name="citation_reference" content="citation_title=NMRbox: a resource for biomolecular NMR computation; citation_author=M.W. Maciejewski; citation_author=A.D. Schuyler; citation_author=M.R. Gryk; citation_author=I.I. Moraru; citation_author=P.R. Romero; citation_author=E.L. Ulrich; citation_author=H.R. Eghbalnia; citation_author=M. Livny; citation_author=F. Delaglio; citation_author=J.C. Hoch; citation_journal_title=Biophys. J.; citation_volume=112; citation_firstpage=1529; citation_lastpage=1534; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Skyline: an open source document editor for creating and analyzing targeted proteomics experiments; citation_author=B. MacLean; citation_author=D.M. Tomazela; citation_author=N. Shulman; citation_author=M. Chambers; citation_author=G.L. Finney; citation_author=B. Frewen; citation_author=R. Kern; citation_author=D.L. Tabb; citation_author=D.C. Liebler; citation_author=M.J. MacCoss; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=966; citation_lastpage=968; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Tools for analyzing and predicting N-terminal protein modifications; citation_author=T. Meinnel; citation_author=C. Giglione; citation_journal_title=Proteomics; citation_volume=8; citation_firstpage=626; citation_lastpage=649; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=ColabFold—making protein folding accessible to all; citation_author=M. Mirdita; citation_author=S. Ovchinnikov; citation_author=M. Steinegger; citation_journal_title=bioRxiv; citation_doi=10.1101/2021.08.15.456425; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Imaging of mitochondrial and non-mitochondrial responses in cultured rat hippocampal neurons exposed to micromolar concentrations of TMRM; citation_author=A. Monteith; citation_author=W. Marszalec; citation_author=P. Chan; citation_author=J. Logan; citation_author=W. Yu; citation_author=N. Schwarz; citation_author=D. Wokosin; citation_author=P. Hockberger; citation_journal_title=PLoS One; citation_volume=8; citation_firstpage=e58059; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo; citation_author=M. Moreno-Mateos; citation_author=C. Vejnar; citation_author=J.-D. Beaudoin; citation_author=J.P. Fernandez; citation_author=E.K. Mis; citation_author=M.K. Khokha; citation_author=A.J. Giraldez; citation_author=et al.; citation_journal_title=Nat Methods; citation_volume=12; citation_doi=10.1038/nmeth.3543; citation_firstpage=982; citation_lastpage=988; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Acinetobacter baumannii response to host-mediated zinc limitation requires the transcriptional regulator Zur; citation_author=B.L. Mortensen; citation_author=S. Rathi; citation_author=W.J. Chazin; citation_author=E.P. Skaar; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=196; citation_firstpage=2616; citation_lastpage=2626; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=SDT: a virus classification tool based on pairwise sequence alignment and identity calculation; citation_author=B.M. Muhire; citation_author=A. Varsani; citation_author=D.P. Martin; citation_journal_title=PLoS One; citation_volume=9; citation_firstpage=e108277; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Analysis of the interplay of protein biogenesis factors at the ribosome exit site reveals new role for NAC; citation_author=Y. Nyathi; citation_author=M.R. Pool; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=210; citation_firstpage=287; citation_lastpage=301; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The zinc finger protein ZNF658 regulates the transcription of genes involved in zinc homeostasis and affects ribosome biogenesis through the zinc transcriptional regulatory element; citation_author=O.A. Ogo; citation_author=J. Tyson; citation_author=S.J. Cockell; citation_author=A. Howard; citation_author=R.A. Valentine; citation_author=D. Ford; citation_journal_title=Mol. Cell. Biol.; citation_volume=35; citation_firstpage=977; citation_lastpage=987; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Metallochaperones, an intracellular shuttle service for metal ions; citation_author=T.V. O'Halloran; citation_author=V.C. Culotta; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=275; citation_firstpage=25057; citation_lastpage=25060; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Normal table of postembryonic zebrafish development: staging by externally visible anatomy of the living fish; citation_author=D.M. Parichy; citation_author=M.R. Elizondo; citation_author=M.G. Mills; citation_author=T.N. Gordon; citation_author=R.E. Engeszer; citation_journal_title=Dev. Dyn.; citation_volume=238; citation_firstpage=2975; citation_lastpage=3015; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Zng1 is a GTP-dependent zinc transferase needed for activation of methionine aminopeptidase; citation_author=M. Pasquini; citation_author=N. Grosjean; citation_author=K.K. Hixson; citation_author=C.D. Nicora; citation_author=E.F. Yee; citation_author=M. Lipton; citation_author=I.K. Blaby; citation_author=J.D. Haley; citation_author=C.E. Blaby-Haas; citation_journal_title=Cell Rep; citation_volume=39; citation_doi=10.1016/j.celrep.2022.110834; citation_firstpage=110834; citation_publication_date=2022; "><meta name="citation_reference" content="citation_title=Tautomeric states of the active-site histidines of phosphorylated and unphosphorylated IIIGlc, a signal-transducing protein from Escherichia coli, using two-dimensional heteronuclear NMR techniques; citation_author=J.G. Pelton; citation_author=D.A. Torchia; citation_author=N.D. Meadow; citation_author=S. Roseman; citation_journal_title=Protein Sci.; citation_volume=2; citation_firstpage=543; citation_lastpage=558; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=SUMO fusion technology for enhanced protein expression and purification in prokaryotes and eukaryotes; citation_author=R.J. Peroutka Iii; citation_author=S.J. Orcutt; citation_author=J.E. Strickler; citation_author=T.R. Butt; citation_journal_title=Methods Mol. Biol.; citation_volume=705; citation_firstpage=15; citation_lastpage=30; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Mitochondrial morphology transitions and functions: implications for retrograde signaling?; citation_author=M. Picard; citation_author=O.S. Shirihai; citation_author=B.J. Gentil; citation_author=Y. Burelle; citation_journal_title=Am. J. Physiol. Regul. Integr. Comp. Physiol.; citation_volume=304; citation_firstpage=R393; citation_lastpage=R406; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=N-terminal acetyltransferases and sequence requirements for N-terminal acetylation of eukaryotic proteins; citation_author=B. Polevoda; citation_author=F. Sherman; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=325; citation_firstpage=595; citation_lastpage=622; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Discovery of human zinc deficiency: its impact on human health and disease; citation_author=A.S. Prasad; citation_journal_title=Adv. Nutr.; citation_volume=4; citation_firstpage=176; citation_lastpage=190; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Bengamides, heterocyclic anthelmintics from a Jaspidae marine sponge; citation_author=E. Quinoa; citation_author=M. Adamczeski; citation_author=P. Crews; citation_author=G.J. Bakus; citation_journal_title=J.&nbsp;Org. Chem.; citation_volume=51; citation_firstpage=4494; citation_lastpage=4497; citation_publication_date=1986; "><meta name="citation_reference" content="citation_title=Chemical shift assignments of zinc finger domain of methionine aminopeptidase 1 (MetAP1) from Homo sapiens; citation_author=K. Rachineni; citation_author=T. Arya; citation_author=K.K. Singarapu; citation_author=A. Addlagatta; citation_author=J. Bharatam; citation_journal_title=Biomol. NMR Assign.; citation_volume=9; citation_firstpage=351; citation_lastpage=353; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Analysis of lethality and malformations during zebrafish (Danio rerio) development; citation_author=A. Raghunath; citation_author=E. Perumal; citation_journal_title=Methods Mol. Biol.; citation_volume=1797; citation_firstpage=337; citation_lastpage=363; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Deciphering key features in protein structures with the new ENDscript server; citation_author=X. Robert; citation_author=P. Gouet; citation_journal_title=Nucleic Acids Res.; citation_volume=42; citation_firstpage=W320; citation_lastpage=W324; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Metallochaperones: bind and deliver; citation_author=A.C. Rosenzweig; citation_journal_title=Chem. Biol.; citation_volume=9; citation_firstpage=673; citation_lastpage=677; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Heteronuclear multidimensional NMR experiments for the structure determination of proteins in solution employing pulsed field gradients; citation_author=M. Sattler; citation_author=J. Schleucher; citation_author=C. Griesinger; citation_journal_title=Prog. Nucl. Magn. Reson. Spectrosc.; citation_volume=34; citation_firstpage=93; citation_lastpage=158; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Measurement of mitochondrial membrane potential using fluorescent rhodamine derivatives; citation_author=R.C. Scaduto; citation_author=L.W. Grotyohann; citation_journal_title=Biophys. J.; citation_volume=76; citation_firstpage=469; citation_lastpage=477; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=E. Frise; citation_author=V. Kaynig; citation_author=M. Longair; citation_author=T. Pietzsch; citation_author=S. Preibisch; citation_author=C. Rueden; citation_author=S. Saalfeld; citation_author=B. Schmid; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=676; citation_lastpage=682; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Overcoming expression and purification problems of RhoGDI using a family of “parallel” expression vectors; citation_author=P. Sheffield; citation_author=S. Garrard; citation_author=Z. Derewenda; citation_journal_title=Protein Expr. Purif.; citation_volume=15; citation_firstpage=34; citation_lastpage=39; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Protein structural information derived from NMR chemical shift with the neural network program TALOS-N; citation_author=Y. Shen; citation_author=A. Bax; citation_journal_title=Methods Mol. Biol.; citation_volume=1260; citation_firstpage=17; citation_lastpage=32; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Interrelations between Essential Metal Ions and Human Diseases; citation_author=A. Sigel; citation_author=H. Sigel; citation_author=R.K.O. Sigel; citation_publication_date=2013; citation_publisher=Springer; "><meta name="citation_reference" content="citation_title=Dynamic light scattering: a practical guide and applications in biomedical sciences; citation_author=J. Stetefeld; citation_author=S.A. McKenna; citation_author=T.R. Patel; citation_journal_title=Biophys. Rev.; citation_volume=8; citation_firstpage=409; citation_lastpage=427; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Metal binding properties of Escherichia coli YjiA, a member of the metal homeostasis-associated COG0523 family of GTPases; citation_author=A.M. Sydor; citation_author=M. Jost; citation_author=K.S. Ryan; citation_author=K.E. Turo; citation_author=C.D. Douglas; citation_author=C.L. Drennan; citation_author=D.B. Zamble; citation_journal_title=Biochemistry; citation_volume=52; citation_firstpage=1788; citation_lastpage=1801; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Proteomics-based target identification: bengamides as a new class of methionine aminopeptidase inhibitors; citation_author=H. Towbin; citation_author=K.W. Bair; citation_author=J.A. DeCaprio; citation_author=M.J. Eck; citation_author=S. Kim; citation_author=F.R. Kinder; citation_author=A. Morollo; citation_author=D.R. Mueller; citation_author=P. Schindler; citation_author=H.K. Song; citation_author=et al.; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=278; citation_firstpage=52964; citation_lastpage=52971; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=The Perseus computational platform for comprehensive analysis of (prote)omics data; citation_author=S. Tyanova; citation_author=T. Temu; citation_author=P. Sinitcyn; citation_author=A. Carlson; citation_author=M.Y. Hein; citation_author=T. Geiger; citation_author=M. Mann; citation_author=J. Cox; citation_journal_title=Nat. Methods; citation_volume=13; citation_firstpage=731; citation_lastpage=740; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Proteomics. Tissue-based map of the human proteome; citation_author=M. Uhlén; citation_author=L. Fagerberg; citation_author=B.M. Hallström; citation_author=C. Lindskog; citation_author=P. Oksvold; citation_author=A. Mardinoglu; citation_author=Å. Sivertsson; citation_author=C. Kampf; citation_author=E. Sjöstedt; citation_author=A. Asplund; citation_author=et al.; citation_journal_title=Science; citation_volume=347; citation_firstpage=1260419; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=BioMagResBank; citation_author=E.L. Ulrich; citation_author=H. Akutsu; citation_author=J.F. Doreleijers; citation_author=Y. Harano; citation_author=Y.E. Ioannidis; citation_author=J. Lin; citation_author=M. Livny; citation_author=S. Mading; citation_author=D. Maziuk; citation_author=Z. Miller; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=36; citation_firstpage=D402; citation_lastpage=D408; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=UniProt: the universal protein knowledgebase in 2021; citation_author=UniProt Consortium; citation_journal_title=Nucleic Acids Res.; citation_volume=49; citation_firstpage=D480; citation_lastpage=D489; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The N-end rule pathway and regulation by proteolysis; citation_author=A. Varshavsky; citation_journal_title=Protein Sci.; citation_volume=20; citation_firstpage=1298; citation_lastpage=1345; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Yeast methionine aminopeptidase type 1 is ribosome-associated and requires its N-terminal zinc finger domain for normal function in&nbsp;vivo; citation_author=J.A. Vetro; citation_author=Y.H. Chang; citation_journal_title=J.&nbsp;Cell. Biochem.; citation_volume=85; citation_firstpage=678; citation_lastpage=688; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Methionine-aminopeptidase: emerging role in angiogenesis; citation_inbook_title=Aminopeptidases in Biology and Disease; citation_author=J.A. Vetro; citation_author=B. Dummitt; citation_author=Y. Chang; citation_publication_date=2004; citation_publisher=Kluwer Academic/Plenum Publishers; "><meta name="citation_reference" content="citation_title=Inhibition of the mitochondrial ATPase function by IF1 changes the spatiotemporal organization of ATP synthase; citation_author=V. Weissert; citation_author=B. Rieger; citation_author=S. Morris; citation_author=T. Arroum; citation_author=O.E. Psathaki; citation_author=T. Zobel; citation_author=G. Perkins; citation_author=K.B. Busch; citation_journal_title=Biochim. Biophys. Acta Bioenerg.; citation_volume=1862; citation_firstpage=148322; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The ligand environment of zinc stored in vesicles; citation_author=G. Wellenreuther; citation_author=M. Cianci; citation_author=R. Tucoulou; citation_author=W. Meyer-Klaucke; citation_author=H. Haase; citation_journal_title=Biochem. Biophys. Res. Commun.; citation_volume=380; citation_firstpage=198; citation_lastpage=203; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The Bengamides: a mini-review of natural sources, analogues, biological properties, biosynthetic origins, and future prospects; citation_author=K.N. White; citation_author=K. Tenney; citation_author=P. Crews; citation_journal_title=J.&nbsp;Nat. Prod.; citation_volume=80; citation_firstpage=740; citation_lastpage=755; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Diverse fates of paralogs following segmental duplication of telomeric genes; citation_author=A. Wong; citation_author=E.J. Vallender; citation_author=K. Heretis; citation_author=Y. Ilkin; citation_author=B.T. Lahn; citation_author=C.L. Martin; citation_author=D.H. Ledbetter; citation_journal_title=Genomics; citation_volume=84; citation_firstpage=239; citation_lastpage=247; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Protein N-terminal processing: substrate specificity of Escherichia coli and human methionine aminopeptidases; citation_author=Q. Xiao; citation_author=F. Zhang; citation_author=B.A. Nacev; citation_author=J.O. Liu; citation_author=D. Pei; citation_journal_title=Biochemistry; citation_volume=49; citation_firstpage=5588; citation_lastpage=5599; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Two-dimensional NMR experiments for correlating carbon-13.beta. and proton.delta./.epsilon. chemical shifts of aromatic residues in 13C-labeled proteins via scalar couplings; citation_author=T. Yamazaki; citation_author=J.D. Forman-Kay; citation_author=L.E. Kay; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=115; citation_firstpage=11054; citation_lastpage=11055; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Generation of Targeted Mutations in Zebrafish Using the CRISPR/Cas System; citation_author=L. Yin; citation_author=L.-E. Jao; citation_author=W. Chen; citation_author=et al.; citation_journal_title=Methods Mol Biol; citation_volume=1332; citation_doi=10.1007/978-1-4939-2917-7_16; citation_firstpage=205; citation_lastpage=217; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21WAF1/CIP1; citation_author=Y. Zhang; citation_author=E.C. Griffith; citation_author=J. Sage; citation_author=T. Jacks; citation_author=J.O. Liu; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=97; citation_firstpage=6427; citation_lastpage=6432; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Suspension trapping (STrap) sample preparation method for bottom-up proteomics analysis; citation_author=A. Zougman; citation_author=P.J. Selby; citation_author=R.E. Banks; citation_journal_title=Proteomics; citation_volume=14; citation_publication_date=2014; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Zinc (Zn) is an essential micronutrient and cofactor for up to 10% of proteins in living organisms. During Zn limitation, specialized enzymes called metallochaperones are predicted to allocate Zn to specific metalloproteins. This function has been putatively assigned to G3E GTPase COG0523 proteins, yet no Zn metallochaperone has been experimentally identified in any organism. Here, we functionally characterize a family of COG0523 proteins that is conserved across vertebrates. We identify Zn metalloprotease methionine aminopeptidase 1 (METAP1) as a COG0523 client, leading to the redesignation of this group of COG0523 proteins as the Zn-regulated GTPase metalloprotein activator (ZNG1) family. Using biochemical, structural, genetic, and pharmacological approaches across evolutionarily divergent models, including zebrafish and mice, we demonstrate a critical role for ZNG1 proteins in regulating cellular Zn homeostasis. Collectively, these data reveal the existence of a family of Zn metallochaperones and assign ZNG1 an important role for intracellular Zn trafficking.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=6903&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(22)00458-5&amp;be=3838&amp;fe=6772&amp;dc=6518&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749593657367,%22n%22:0,%22f%22:3208,%22dn%22:3209,%22dne%22:3209,%22c%22:3209,%22s%22:3209,%22ce%22:3209,%22rq%22:3209,%22rp%22:3822,%22rpe%22:4514,%22dl%22:3827,%22di%22:4797,%22ds%22:6518,%22de%22:6531,%22dc%22:6771,%22l%22:6772,%22le%22:6810%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900458-5" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00458-5"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(22)00458-5"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900458-5" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-389253" aria-expanded="false" aria-controls="login-links-389253" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-389253" id="login-links--389253" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900458-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900458-5" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900458-5" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-389253" data-opens="#links-dialog-389253" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-389253" aria-modal="true"><button data-closes="#links-dialog-389253" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900458-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900458-5" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900458-5" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2022.04.011">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(22)00458-5" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0013-X"><span>Volume 185</span>, <span>Issue 12</span></a></span><span class="meta-panel__pages"><span>p2148-2163.e27</span></span><span class="meta-panel__onlineDate">June 09, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=12&amp;pii=S0092-8674%2821%29X0013-X" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Zn-regulated GTPase metalloprotein activator 1 modulates vertebrate zinc homeostasis</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Andy</span> <span property="familyName">Weiss</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andy</span> <span property="familyName">Weiss</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5221-1032" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5221-1032</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Weiss/Andy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">8</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Caitlin C.</span> <span property="familyName">Murdoch</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Caitlin C.</span> <span property="familyName">Murdoch</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Murdoch/Caitlin+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">8</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Katherine A.</span> <span property="familyName">Edmonds</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Katherine A.</span> <span property="familyName">Edmonds</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1282-9858" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1282-9858</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Indiana University, Bloomington, IN 47405, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Edmonds/Katherine+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Matthew R.</span> <span property="familyName">Jordan</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matthew R.</span> <span property="familyName">Jordan</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0095-8078" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0095-8078</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Indiana University, Bloomington, IN 47405, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular and Cellular Biochemistry, Indiana University, Bloomington, IN 47405, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jordan/Matthew+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Andrew J.</span> <span property="familyName">Monteith</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew J.</span> <span property="familyName">Monteith</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Monteith/Andrew+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Yasiru R.</span> <span property="familyName">Perera</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yasiru R.</span> <span property="familyName">Perera</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Departments of Biochemistry and Chemistry, Center for Structural Biology, Vanderbilt University, Nashville, TN 37240, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Perera/Yasiru+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Aslin M.</span> <span property="familyName">Rodríguez Nassif</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Aslin M.</span> <span property="familyName">Rodríguez Nassif</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Departments of Biochemistry and Chemistry, Center for Structural Biology, Vanderbilt University, Nashville, TN 37240, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rodr%C3%ADguez+Nassif/Aslin+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Amber M.</span> <span property="familyName">Petoletti</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Amber M.</span> <span property="familyName">Petoletti</span> <a class="orcid-id" href="https://orcid.org/0000-0002-8287-0274" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-8287-0274</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Petoletti/Amber+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">William N.</span> <span property="familyName">Beavers</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">William N.</span> <span property="familyName">Beavers</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Beavers/William+N" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Matthew J.</span> <span property="familyName">Munneke</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matthew J.</span> <span property="familyName">Munneke</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Munneke/Matthew+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Sydney L.</span> <span property="familyName">Drury</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sydney L.</span> <span property="familyName">Drury</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Drury/Sydney+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Evan S.</span> <span property="familyName">Krystofiak</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Evan S.</span> <span property="familyName">Krystofiak</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Cell Imaging Shared Resource, Vanderbilt University, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Krystofiak/Evan+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Kishore</span> <span property="familyName">Thalluri</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kishore</span> <span property="familyName">Thalluri</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5862-4631" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5862-4631</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Indiana University, Bloomington, IN 47405, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Thalluri/Kishore" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Hongwei</span> <span property="familyName">Wu</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Hongwei</span> <span property="familyName">Wu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Indiana University, Bloomington, IN 47405, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wu/Hongwei" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Angela R.S.</span> <span property="familyName">Kruse</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Angela R.S.</span> <span property="familyName">Kruse</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Departments of Chemistry and Biochemistry, Mass Spectrometry Research Center, Vanderbilt University, Nashville, TN 37235, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kruse/Angela+R+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Richard D.</span> <span property="familyName">DiMarchi</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Richard D.</span> <span property="familyName">DiMarchi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Indiana University, Bloomington, IN 47405, USA</span></div></div><div class="core-author-link"><a href="/authored-by/DiMarchi/Richard+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Richard M.</span> <span property="familyName">Caprioli</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Richard M.</span> <span property="familyName">Caprioli</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Departments of Chemistry and Biochemistry, Mass Spectrometry Research Center, Vanderbilt University, Nashville, TN 37235, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Caprioli/Richard+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Jeffrey M.</span> <span property="familyName">Spraggins</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jeffrey M.</span> <span property="familyName">Spraggins</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Departments of Chemistry and Biochemistry, Mass Spectrometry Research Center, Vanderbilt University, Nashville, TN 37235, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Spraggins/Jeffrey+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Walter J.</span> <span property="familyName">Chazin</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Walter J.</span> <span property="familyName">Chazin</span> <a class="orcid-id" href="https://orcid.org/0000-0002-2180-0790" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-2180-0790</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Departments of Biochemistry and Chemistry, Center for Structural Biology, Vanderbilt University, Nashville, TN 37240, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chazin/Walter+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">David P.</span> <span property="familyName">Giedroc</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David P.</span> <span property="familyName">Giedroc</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:giedroc@indiana.edu" property="email" title="Send email to David P. Giedroc">giedroc@indiana.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Indiana University, Bloomington, IN 47405, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular and Cellular Biochemistry, Indiana University, Bloomington, IN 47405, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Giedroc/David+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup> <a id="eadd0010" href="mailto:giedroc@indiana.edu" property="email" title="Send email to David P. Giedroc">giedroc@indiana.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Eric P.</span> <span property="familyName">Skaar</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Eric P.</span> <span property="familyName">Skaar</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:eric.skaar@vumc.org" property="email" title="Send email to Eric P. Skaar">eric.skaar@vumc.org</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">9</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Skaar/Eric+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">9</a></sup> <a id="eadd0015" href="mailto:eric.skaar@vumc.org" property="email" title="Send email to Eric P. Skaar">eric.skaar@vumc.org</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-linked-content" aria-expanded="false" tabindex="0" role="button"><span>Linked Articles (1)</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Pathology, Microbiology, and Immunology, Vanderbilt Institute for Infection, Immunology and Inflammation, Vanderbilt University Medical Center, Nashville, TN 37232, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Chemistry, Indiana University, Bloomington, IN 47405, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Molecular and Cellular Biochemistry, Indiana University, Bloomington, IN 47405, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Departments of Biochemistry and Chemistry, Center for Structural Biology, Vanderbilt University, Nashville, TN 37240, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Cell Imaging Shared Resource, Vanderbilt University, Nashville, TN 37232, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Departments of Chemistry and Biochemistry, Mass Spectrometry Research Center, Vanderbilt University, Nashville, TN 37235, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN 37232, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">9</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received October 5, 2021</span>; <span>Revised February 7, 2022</span>; <span>Accepted April 7, 2022</span>; <span>Published online May 17, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2022.04.011" property="sameAs" target="_blank">10.1016/j.cell.2022.04.011 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867422004585" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2022 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2022.04.011" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div><div id="core-linked-content" role="region" data-expandable="item" style="display: none;">
    

        


    

            

            

            <section data-widget-id="e587233e-22aa-4221-8ae1-22df489d46f7" data-widget-name="UX3linkedContentList" class="axel-linked-articles"><div class="section-paragraph linked-article"><ul class="rlist"><li class="linked-article__item"><div class="linked-article__article-type">Article</div><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(22)00607-6" target="_blank" title="Zng1 is a GTP-dependent zinc transferase needed for activation of methionine aminopeptidase">Zng1 is a GTP-dependent zinc transferase needed for activation of methionine aminopeptidase</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Pasquini et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">May 17, 2022</span></div></li></ul></div></section>

            

        
</div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0013-X"><img src="/cms/asset/4589f75f-8ef9-4728-9561-607fa4e2c7d9/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 12"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2822%2900458-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (6)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900458-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.04.011">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2822%2900458-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2022.04.011&amp;pii=S0092-8674%2822%2900458-5" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900458-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.04.011">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2022.04.011" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867422004585&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867422004585&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=12&amp;pii=S0092-8674%2821%29X0013-X" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00463-9" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(22)00532-3" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (6)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">The N terminus of ZNG1 interacts with the zinc finger of METAP1</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">ZNG1 promotes METAP1 activity in a GTP-hydrolysis- and Zn-dependent manner</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Loss of ZNG1 impairs mitochondrial function and cellular proliferation</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph"><i>Zng1</i> mutant animals are sensitive to Zn starvation and inhibition of METAP activity</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Zinc (Zn) is an essential micronutrient and cofactor for up to 10% of proteins in living organisms. During Zn limitation, specialized enzymes called metallochaperones are predicted to allocate Zn to specific metalloproteins. This function has been putatively assigned to G3E GTPase COG0523 proteins, yet no Zn metallochaperone has been experimentally identified in any organism. Here, we functionally characterize a family of COG0523 proteins that is conserved across vertebrates. We identify Zn metalloprotease methionine aminopeptidase 1 (METAP1) as a COG0523 client, leading to the redesignation of this group of COG0523 proteins as the Zn-regulated GTPase metalloprotein activator (ZNG1) family. Using biochemical, structural, genetic, and pharmacological approaches across evolutionarily divergent models, including zebrafish and mice, we demonstrate a critical role for ZNG1 proteins in regulating cellular Zn homeostasis. Collectively, these data reveal the existence of a family of Zn metallochaperones and assign ZNG1 an important role for intracellular Zn trafficking.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/6114fcb2-6a3c-4be6-96ec-08d9cef1fe64/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2022.04.011/asset/37d5261a-fcc2-40a7-99f4-da74ff659b70/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/6114fcb2-6a3c-4be6-96ec-08d9cef1fe64/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;zinc&quot;" alt="Search on this keyword" rel="nofollow">zinc</a></li><li><a href="/action/doSearch?AllField=&quot;zinc+finger&quot;" alt="Search on this keyword" rel="nofollow">zinc finger</a></li><li><a href="/action/doSearch?AllField=&quot;metalloprotein&quot;" alt="Search on this keyword" rel="nofollow">metalloprotein</a></li><li><a href="/action/doSearch?AllField=&quot;GTPase&quot;" alt="Search on this keyword" rel="nofollow">GTPase</a></li><li><a href="/action/doSearch?AllField=&quot;COG0523&quot;" alt="Search on this keyword" rel="nofollow">COG0523</a></li><li><a href="/action/doSearch?AllField=&quot;ZNG1&quot;" alt="Search on this keyword" rel="nofollow">ZNG1</a></li><li><a href="/action/doSearch?AllField=&quot;CBWD&quot;" alt="Search on this keyword" rel="nofollow">CBWD</a></li><li><a href="/action/doSearch?AllField=&quot;METAP1&quot;" alt="Search on this keyword" rel="nofollow">METAP1</a></li><li><a href="/action/doSearch?AllField=&quot;zf-C6H2&quot;" alt="Search on this keyword" rel="nofollow">zf-C6H2</a></li><li><a href="/action/doSearch?AllField=&quot;metallochaperone&quot;" alt="Search on this keyword" rel="nofollow">metallochaperone</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">Metals are essential micronutrients that are indispensable for cellular processes in all kingdoms of life. Zinc (Zn) is the second most abundant transition metal in humans, serving as a structural or enzymatic cofactor for approximately 10% of the proteome (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Andreini et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Andreini, C. ∙ Banci, L. ∙ Bertini, I. ...</div><div><strong>Counting the zinc-proteins encoded in the human genome</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2006; <strong>5</strong>:196-201</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/pr050361j" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-30744471169" target="_blank" aria-label="Open Scopus in new tab">Scopus (740)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16396512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fpr050361j&amp;pmid=16396512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consequently, perturbations in Zn homeostasis are linked to human disease, including growth deficiencies, immune defects, neurological disorders, and cancers (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Basu, 2018</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Basu, A.K.</div><div><strong>DNA damage, mutagenesis and cancer</strong></div><div><em>Int. J. Mol. Sci.</em> 2018; <strong>19</strong></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.K.BasuDNA+damage%2C+mutagenesis+and+cancerInt.+J.+Mol.+Sci.192018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Devirgiliis et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Devirgiliis, C. ∙ Zalewski, P.D. ∙ Perozzi, G. ...</div><div><strong>Zinc fluxes and zinc transporter genes in chronic diseases</strong></div><div><em>Mutat. Res.</em> 2007; <strong>622</strong>:84-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mrfmmm.2007.01.013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-34547681207" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17374385/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mrfmmm.2007.01.013&amp;pmid=17374385" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Fischer Walker and Black, 2004</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Fischer Walker, C. ∙ Black, R.E.</div><div><strong>Zinc and the risk for infectious disease</strong></div><div><em>Annu. Rev. Nutr.</em> 2004; <strong>24</strong>:255-275</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.nutr.23.011702.073054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-3242736546" target="_blank" aria-label="Open Scopus in new tab">Scopus (386)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15189121/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.nutr.23.011702.073054&amp;pmid=15189121" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Prasad, 2013</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Prasad, A.S.</div><div><strong>Discovery of human zinc deficiency: its impact on human health and disease</strong></div><div><em>Adv. Nutr.</em> 2013; <strong>4</strong>:176-190</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.3945%2Fan.112.003210&amp;cf=fulltext&amp;site=advnut-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.3945%2Fan.112.003210&amp;cf=pdf&amp;site=advnut-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84882978950" target="_blank" aria-label="Open Scopus in new tab">Scopus (477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23493534/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3945%2Fan.112.003210&amp;pmid=23493534" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This is particularly alarming as Zn deficiency is the fifth most important risk factor for mortality in developing countries, affecting close to half of the world’s population (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Guilbert, 2003</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Guilbert, J.J.</div><div><strong>The world health report 2002—reducing risks, promoting healthy life</strong></div><div><em>Educ. Health (Abingdon).</em> 2003; <strong>16</strong>:230</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14741909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=14741909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0055" role="paragraph">Cellular Zn is present at levels similar to major metabolites such as ATP. However, the majority of Zn within the cell is associated with Zn-requiring metalloproteins, Zn-storage proteins, or maintained in vesicular storage (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Krężel and Maret, 2017</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Krężel, A. ∙ Maret, W.</div><div><strong>The functions of metamorphic metallothioneins in zinc and copper metabolism</strong></div><div><em>Int. J. Mol. Sci.</em> 2017; <strong>18</strong>:1237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/ijms18061237" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85020827370" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28598392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fijms18061237&amp;pmid=28598392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Sigel et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Sigel, A. ∙ Sigel, H. ∙ Sigel, R.K.O.</div><div><strong>Interrelations between Essential Metal Ions and Human Diseases</strong></div><div>Springer, 2013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-94-007-7500-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-94-007-7500-8" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Wellenreuther et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Wellenreuther, G. ∙ Cianci, M. ∙ Tucoulou, R. ...</div><div><strong>The ligand environment of zinc stored in vesicles</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2009; <strong>380</strong>:198-203</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbrc.2009.01.074" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-59849127893" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19171119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbrc.2009.01.074&amp;pmid=19171119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), resulting in extremely low levels of freely available Zn (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Ba et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Ba, L.A. ∙ Doering, M. ∙ Burkholz, T. ...</div><div><strong>Metal trafficking: from maintaining the metal homeostasis to future drug design</strong></div><div><em>Metallomics.</em> 2009; <strong>1</strong>:292-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/b904533c" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77149122885" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21305127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fb904533c&amp;pmid=21305127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). During Zn limitation, the metalation of critical metalloproteins is thought to require the hierarchical distribution of Zn to ensure their function. Akin to other metals such as copper (<span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">O'Halloran and Culotta, 2000</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>O'Halloran, T.V. ∙ Culotta, V.C.</div><div><strong>Metallochaperones, an intracellular shuttle service for metal ions</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2000; <strong>275</strong>:25057-25060</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.R000006200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.R000006200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0034682776" target="_blank" aria-label="Open Scopus in new tab">Scopus (669)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10816601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.R000006200&amp;pmid=10816601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), targeted transfer of Zn to metalloproteins is hypothesized to be mediated by specialized proteins referred to as metallochaperones (<span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Rosenzweig, 2002</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Rosenzweig, A.C.</div><div><strong>Metallochaperones: bind and deliver</strong></div><div><em>Chem. Biol.</em> 2002; <strong>9</strong>:673-677</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS1074-5521%2802%2900156-4&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS1074-5521%2802%2900156-4&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0035986679" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12079778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1074-5521%2802%2900156-4&amp;pmid=12079778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), yet no such protein has been identified to date.</div><div id="p0060" role="paragraph">Members of the ubiquitous G3E family of P-loop GTPases deliver/insert different metal cofactors to client metalloproteins (<span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Haas et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Haas, C.E. ∙ Rodionov, D.A. ∙ Kropat, J. ...</div><div><strong>A subset of the diverse COG0523 family of putative metal chaperones is linked to zinc homeostasis in all kingdoms of life</strong></div><div><em>BMC Genomics.</em> 2009; <strong>10</strong>:470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-10-470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-70449732603" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19822009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-10-470&amp;pmid=19822009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This family includes COG0523 proteins, a poorly understood subgroup found across all branches of life (<span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Edmonds et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Edmonds, K.A. ∙ Jordan, M.R. ∙ Giedroc, D.P.</div><div><strong>COG0523 proteins: a functionally diverse family of transition metal-regulated G3E P-loop GTP hydrolases from bacteria to man</strong></div><div><em>Metallomics.</em> 2021; <strong>13</strong>:mfab046</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/mtomcs/mfab046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85113538279" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34302342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmtomcs%2Fmfab046&amp;pmid=34302342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; Haas et&nbsp;al., 2009). COG0523 proteins are characterized by a conserved N-terminal GTPase domain, and GTP hydrolysis is thought to provide energy for the transfer&nbsp;of metals to client proteins (<span class="dropBlock reference-citations"><a id="body-ref-sref47-1" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-1" aria-controls="sref47-1">Jordan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Jordan, M.R. ∙ Wang, J. ∙ Weiss, A. ...</div><div><strong>Mechanistic insights into the metal-dependent activation of Zn(II)-dependent metallochaperones</strong></div><div><em>Inorg. Chem.</em> 2019; <strong>58</strong>:13661-13672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.inorgchem.9b01173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85073651560" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31247880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.inorgchem.9b01173&amp;pmid=31247880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, COG0523 proteins have been linked to cellular Zn homeostasis, as the expression of several bacterial and eukaryotic COG0523 members is induced during conditions of Zn starvation (<span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Coneyworth et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Coneyworth, L.J. ∙ Jackson, K.A. ∙ Tyson, J. ...</div><div><strong>Identification of the human zinc transcriptional regulatory element (ZTRE): a palindromic protein-binding DNA sequence responsible for zinc-induced transcriptional repression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:36567-36581</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M112.397000&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M112.397000&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84867793466" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22902622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M112.397000&amp;pmid=22902622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Haas et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Haas, C.E. ∙ Rodionov, D.A. ∙ Kropat, J. ...</div><div><strong>A subset of the diverse COG0523 family of putative metal chaperones is linked to zinc homeostasis in all kingdoms of life</strong></div><div><em>BMC Genomics.</em> 2009; <strong>10</strong>:470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-10-470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-70449732603" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19822009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-10-470&amp;pmid=19822009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-2" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-2" aria-controls="sref47-2">Jordan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Jordan, M.R. ∙ Wang, J. ∙ Weiss, A. ...</div><div><strong>Mechanistic insights into the metal-dependent activation of Zn(II)-dependent metallochaperones</strong></div><div><em>Inorg. Chem.</em> 2019; <strong>58</strong>:13661-13672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.inorgchem.9b01173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85073651560" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31247880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.inorgchem.9b01173&amp;pmid=31247880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Mortensen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Mortensen, B.L. ∙ Rathi, S. ∙ Chazin, W.J. ...</div><div><strong>Acinetobacter baumannii response to host-mediated zinc limitation requires the transcriptional regulator Zur</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2014; <strong>196</strong>:2616-2626</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.01650-14" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84903388692" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24816603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.01650-14&amp;pmid=24816603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-1" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-1" aria-controls="sref73-1">Ogo et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref73-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Ogo, O.A. ∙ Tyson, J. ∙ Cockell, S.J. ...</div><div><strong>The zinc finger protein ZNF658 regulates the transcription of genes involved in zinc homeostasis and affects ribosome biogenesis through the zinc transcriptional regulatory element</strong></div><div><em>Mol. Cell. Biol.</em> 2015; <strong>35</strong>:977-987</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01298-14" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25582195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01298-14&amp;pmid=25582195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Despite the assignment of COG0523 proteins as putative nucleotide hydrolysis-powered Zn metallochaperones, conclusive experimental evidence in support of this proposed function has not been established.</div><div id="p0065" role="paragraph">In this manuscript, we investigate a conserved family of vertebrate COG0523 proteins formerly referred to as CobW-domain containing proteins (CBWDs). Based on the shared function of CBWD in vertebrates and yeast (YNR029c, <span class="dropBlock reference-citations"><a id="body-ref-optkIjaU1XKKm-1" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optkIjaU1XKKm-1" aria-controls="optkIjaU1XKKm-1">Pasquini et al., 2022</a><div class="dropBlock__holder" data-db-target-of="optkIjaU1XKKm-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Pasquini, M. ∙ Grosjean, N. ∙ Hixson, K.K. ...</div><div><strong>Zng1 is a GTP-dependent zinc transferase needed for activation of methionine aminopeptidase</strong></div><div><em>Cell Rep.</em> 2022; <strong>39</strong>:110834</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2022.110834&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2022.110834&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85130211918" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35584675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2022.110834&amp;pmid=35584675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we have named this protein family ZNG1 for Zn-regulated GTPase metalloprotein activator 1. Yeast-two-hybrid screens using orthologs in zebrafish (Zng1), mouse (ZNG1), and human (ZNG1E) identified methionine aminopeptidase 1 (METAP1) as a shared ZNG1 client across vertebrates. METAPs perform N-terminal methionine excision (NME), a process that co-translationally removes N-terminal initiator methionine (iMet) for approximately half of newly synthesized peptides, contributing to protein stability, maturation, modification, and subcellular localization (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Arfin and Bradshaw, 1988</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Arfin, S.M. ∙ Bradshaw, R.A.</div><div><strong>Cotranslational processing and protein turnover in eukaryotic cells</strong></div><div><em>Biochemistry.</em> 1988; <strong>27</strong>:7979-7984</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00421a001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3069123/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00421a001&amp;pmid=3069123" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Frottin et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Frottin, F. ∙ Martinez, A. ∙ Peynot, P. ...</div><div><strong>The proteomics of N-terminal methionine cleavage</strong></div><div><em>Mol. Cell. Proteomics.</em> 2006; <strong>5</strong>:2336-2349</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M600225-MCP200&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M600225-MCP200&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33845993905" target="_blank" aria-label="Open Scopus in new tab">Scopus (289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16963780/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M600225-MCP200&amp;pmid=16963780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Gawron et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Gawron, D. ∙ Ndah, E. ∙ Gevaert, K. ...</div><div><strong>Positional proteomics reveals differences in N-terminal proteoform stability</strong></div><div><em>Mol. Syst. Biol.</em> 2016; <strong>12</strong>:858</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20156662" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84959423232" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26893308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20156662&amp;pmid=26893308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Gibbs et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Gibbs, D.J. ∙ Bacardit, J. ∙ Bachmair, A. ...</div><div><strong>The eukaryotic N-end rule pathway: conserved mechanisms and diverse functions</strong></div><div><em>Trends Cell Biol.</em> 2014; <strong>24</strong>:603-611</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.tcb.2014.05.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.tcb.2014.05.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24874449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tcb.2014.05.001&amp;pmid=24874449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Giglione et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Giglione, C. ∙ Boularot, A. ∙ Meinnel, T.</div><div><strong>Protein N-terminal methionine excision</strong></div><div><em>Cell. Mol. Life Sci.</em> 2004; <strong>61</strong>:1455-1474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00018-004-3466-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15197470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00018-004-3466-8&amp;pmid=15197470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Kim et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Kim, H.K. ∙ Kim, R.R. ∙ Oh, J.H. ...</div><div><strong>The N-terminal methionine of cellular proteins as a degradation signal</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:158-169</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.cell.2013.11.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.cell.2013.11.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84892802083" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.031&amp;pmid=24361105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Meinnel and Giglione, 2008</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Meinnel, T. ∙ Giglione, C.</div><div><strong>Tools for analyzing and predicting N-terminal protein modifications</strong></div><div><em>Proteomics.</em> 2008; <strong>8</strong>:626-649</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pmic.200700592" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-40549132858" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18203265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpmic.200700592&amp;pmid=18203265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Polevoda and Sherman, 2003</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Polevoda, B. ∙ Sherman, F.</div><div><strong>N-terminal acetyltransferases and sequence requirements for N-terminal acetylation of eukaryotic proteins</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 2003; <strong>325</strong>:595-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-2836(02)01269-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0037462954" target="_blank" aria-label="Open Scopus in new tab">Scopus (354)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12507466/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-2836%2802%2901269-X&amp;pmid=12507466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Varshavsky, 2011</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Varshavsky, A.</div><div><strong>The N-end rule pathway and regulation by proteolysis</strong></div><div><em>Protein Sci.</em> 2011; <strong>20</strong>:1298-1345</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.666" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-79960683356" target="_blank" aria-label="Open Scopus in new tab">Scopus (480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21633985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.666&amp;pmid=21633985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). METAPs require a metal cofactor for enzymatic activity, a role often attributed to Zn (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Leopoldini et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Leopoldini, M. ∙ Russo, N. ∙ Toscano, M.</div><div><strong>Which one among Zn(II), Co(II), Mn(II), and Fe(II) is the most efficient ion for the methionine aminopeptidase catalyzed reaction?</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2007; <strong>129</strong>:7776-7784</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja068168t" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84961982737" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17523636/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja068168t&amp;pmid=17523636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Vetro, 2004</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Vetro, J.A. ∙ Dummitt, B. ∙ Chang, Y.</div><div><strong>Methionine-aminopeptidase: emerging role in angiogenesis</strong></div><div><div>Hooper, N.M. ∙ Lendeckel, U. (Editors)</div></div><div><strong>Aminopeptidases in Biology and Disease</strong></div><div>Kluwer Academic/Plenum Publishers, 2004</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4419-8869-0_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4419-8869-0_2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our biochemical and functional characterization of the ZNG1-METAP1 interaction and the solution structure of the interacting domains establish a model of METAP1 activation by ZNG1 in a GTP-hydrolysis- and Zn-dependent&nbsp;manner, consistent with ZNG1 acting as a metallochaperone. Zebrafish and mouse models of <i>Zng1</i> deficiency suggest that ZNG1 supports METAP1 activity <i>in&nbsp;vivo</i> and the&nbsp;mutation of <i>Zng1</i> results in increased sensitivity to dietary Zn starvation. Furthermore, <i>Zng1</i> mutation leads to diminished cellular proliferation and mitochondrial dysfunction, resulting in&nbsp;impaired respiration. Collectively these data support the assignment of ZNG1 as a Zn metallochaperone and illustrate the importance of ZNG1 proteins for organismal health and development.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>ZNG1s are ancient COG0523 proteins with conserved metalloprotein interactions</h3><div id="p0070" role="paragraph">To interrogate the conservation of vertebrate ZNG1 proteins, amino acid sequences spanning an evolutionary distance of over 400 million years were compared (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Howe et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Howe, K.L. ∙ Achuthan, P. ∙ Allen, J. ...</div><div><strong>Ensembl 2021</strong></div><div><em>Nucleic Acids Res.</em> 2021; <strong>49</strong>:D884-D891</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa942" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85099428320" target="_blank" aria-label="Open Scopus in new tab">Scopus (659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33137190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa942&amp;pmid=33137190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Kumar et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Kumar, S. ∙ Stecher, G. ∙ Suleski, M. ...</div><div><strong>TimeTree: a resource for timelines, Timetrees, and divergence times</strong></div><div><em>Time. Mol. Biol. Evol.</em> 2017; <strong>34</strong>:1812-1819</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msx116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28387841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsx116&amp;pmid=28387841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although many vertebrates, including mice and zebrafish, encode for a single ZNG1 protein, five intact ZNG1 paralogs are found in humans (ZNG1A-F, with <i>ZNG1DP</i> being a pseudogene) (<span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Wong et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Wong, A. ∙ Vallender, E.J. ∙ Heretis, K. ...</div><div><strong>Diverse fates of paralogs following segmental duplication of telomeric genes</strong></div><div><em>Genomics.</em> 2004; <strong>84</strong>:239-247</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ygeno.2004.03.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-3042517001" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15233989/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ygeno.2004.03.001&amp;pmid=15233989" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). With over 60% sequence conservation across all analyzed ZNG1 proteins (<a id="crosref0360" href="#fig1">Figures&nbsp;1</a>A and <a id="crosref0365" href="#figs1">S1</a>A), the known catalytic motifs and metal-binding residues of the GTPase domain are uniformly present and conserved (<a id="crosref0370" href="#fig1">Figure&nbsp;1</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Edmonds et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Edmonds, K.A. ∙ Jordan, M.R. ∙ Giedroc, D.P.</div><div><strong>COG0523 proteins: a functionally diverse family of transition metal-regulated G3E P-loop GTP hydrolases from bacteria to man</strong></div><div><em>Metallomics.</em> 2021; <strong>13</strong>:mfab046</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/mtomcs/mfab046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85113538279" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34302342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmtomcs%2Fmfab046&amp;pmid=34302342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-3" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-3" aria-controls="sref32-3">Haas et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref32-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Haas, C.E. ∙ Rodionov, D.A. ∙ Kropat, J. ...</div><div><strong>A subset of the diverse COG0523 family of putative metal chaperones is linked to zinc homeostasis in all kingdoms of life</strong></div><div><em>BMC Genomics.</em> 2009; <strong>10</strong>:470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-10-470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-70449732603" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19822009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-10-470&amp;pmid=19822009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Notably, conserved regions outside of the GTPase domain also are present in the N terminus (<a id="crosref0385" href="#fig1">Figure&nbsp;1</a>B).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/208f765c-15f2-46af-a641-dc0e1b3a8cd0/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2022.04.011/asset/153693ba-8b18-4497-b7a5-d5b8a4010a39/main.assets/gr1.jpg" height="669" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/208f765c-15f2-46af-a641-dc0e1b3a8cd0/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Identification of metalloprotein targets of Zn-regulated GTPase metalloprotein activator 1 (ZNG1) in vertebrates</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Cladogram and analysis of ZNG1 protein sequence conservation in vertebrates. Numbers on branches denote nucleotide substitutions per site.</div><div id="fspara0020" role="paragraph">(B) Amino acid conservation along the length of vertebrate ZNG1s indicates high sequence conservation, particularly in the GTPase domains and at the N terminus. Lighter shading reflects higher conservation.</div><div id="fspara0025" role="paragraph">(C) Yeast-two-hybrid screens using full-length human, mouse, and zebrafish ZNG1s identify unique and shared interacting proteins.</div><div id="fspara0030" role="paragraph">(D) Molecular function enrichment analysis of ZNG1 interaction protein PFAM domains.</div><div id="fspara0035" role="paragraph">(E) Number and cellular activity of ZNG1 client Zn metalloproteins detected in yeast-two-hybrid screens.</div><div id="fspara0040" role="paragraph">(F) ZNG1 zf interaction domains in Zn metalloproteins shown in (E).</div><div id="fspara0045" role="paragraph">(G) Conserved ZNG1 interaction domain of METAP1 across species overlaps with a C<sub>6</sub>H<sub>2</sub> domain. Yellow shading indicates minimal conserved region.</div><div id="fspara0050" role="paragraph">See also <a id="crosref1685" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref1690" href="#mmc1">Table&nbsp;S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/ed3ab668-0e71-4b4b-9348-ed5c38016a03/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2022.04.011/asset/7601887c-9c23-4646-bdcd-3dd5cdc5ca31/main.assets/figs1.jpg" height="560" width="753" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/ed3ab668-0e71-4b4b-9348-ed5c38016a03/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Conservation and tissue-specific expression of ZNG1s, related to <a id="crosref1740" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0345" role="paragraph">(A) Multiple sequence alignment of vertebrate ZNG1 proteins depicted in cladogram of <a id="crosref1745" href="#fig1">Figure&nbsp;1</a>A. White shading in the bottom bar indicates high conservation.</div><div id="fspara0350" role="paragraph">(B) Expression levels of human ZNG1A, ZNG1B, ZNG1C, ZNG1E, and ZNG1F across different tissue and cell types. Expression data obtained from the human protein atlas.</div><div id="fspara0355" role="paragraph">(C) Multiple sequence alignment of human ZNG1 paralogs.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">Despite broad evolutionary conservation, the functions of COG0523 proteins, including ZNG1, remain unknown—in part due to a lack of knowledge of interaction partners. To identify potential client proteins, three independent yeast-two-hybrid screens were performed using full-length zebrafish Zng1, mouse ZNG1, and human ZNG1E proteins as bait. ZNG1E was chosen as the representative human protein due to its consistently high expression across different tissue types and high similarity to the consensus sequence of the human ZNG1 proteins (<a id="crosref0390" href="#figs1">Figures&nbsp;S1</a>B and S1C) (<span class="dropBlock reference-citations"><a id="body-ref-sref96-1" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-1" aria-controls="sref96-1">Uhlén et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref96-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Uhlén, M. ∙ Fagerberg, L. ∙ Hallström, B.M. ...</div><div><strong>Proteomics. Tissue-based map of the human proteome</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1260419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260419&amp;pmid=25613900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Of note, species-specific nomenclature for genes and proteins from the representative backgrounds will be maintained throughout the manuscript (e.g., human: <i>ZNG1E</i>, ZNG1E; mouse: <i>Zng1</i>, ZNG1; zebrafish: <i>zng1</i>, Zng1). Each screen identified at least nine putative ZNG1 interacting partners (<a id="crosref0400" href="#fig1">Figure&nbsp;1</a>C; <a id="crosref0405" href="#mmc1">Table&nbsp;S1</a>). Although each dataset revealed species-specific interactions, METAP1 was identified in all three screens.</div><div id="p0080" role="paragraph">In addition to identifying interaction partners, the yeast-two-hybrid approach also discerned predicted Selected Interacting Domains (SID) of target proteins. SIDs were analyzed to identify the presence of specific protein domains that are partially or completely contained within each SID (<a id="crosref0410" href="#mmc1">Table&nbsp;S1</a>). Enrichment analysis using the dcGO platform (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Fang and Gough, 2013</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Fang, H. ∙ Gough, J.</div><div><strong>DcGO: database of domain-centric ontologies on functions, phenotypes, diseases and more</strong></div><div><em>Nucleic Acids Res.</em> 2013; <strong>41</strong>:D536-D544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gks1080" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84876528480" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23161684/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgks1080&amp;pmid=23161684" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) revealed a high prevalence of domains with functions in protein and chromatin binding, as well as factors with regulatory roles (<a id="crosref0420" href="#fig1">Figure&nbsp;1</a>D). Since <i>ZNG1</i> expression is associated with conditions of Zn limitation (<span class="dropBlock reference-citations"><a id="body-ref-sref73-2" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-2" aria-controls="sref73-2">Ogo et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref73-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Ogo, O.A. ∙ Tyson, J. ∙ Cockell, S.J. ...</div><div><strong>The zinc finger protein ZNF658 regulates the transcription of genes involved in zinc homeostasis and affects ribosome biogenesis through the zinc transcriptional regulatory element</strong></div><div><em>Mol. Cell. Biol.</em> 2015; <strong>35</strong>:977-987</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01298-14" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25582195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01298-14&amp;pmid=25582195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we hypothesized that ZNG1 aids in the mobilization of cellular Zn during limitation and that proteins with known Zn-binding domains would be interaction partners enriched in these datasets. Indeed, 12 Zn metalloproteins were identified across all three screens, including proteins involved in transcription, protein processing, and the cell cycle (<a id="crosref0430" href="#fig1">Figure&nbsp;1</a>E). For most of these proteins, the predicted site of interaction with ZNG1 harbors a Zn finger (zf) motif, including Cys<sub>2</sub>His<sub>2</sub> (C<sub>2</sub>H<sub>2</sub>), Cys<sub>6</sub>His<sub>2</sub> (C<sub>6</sub>H<sub>2</sub>), or myeloid, Nervy, and DEAF-1 (MYND) (<a id="crosref0435" href="#fig1">Figure&nbsp;1</a>F), consistent with known roles of zf domains in facilitating protein-protein interactions (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Brayer et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Brayer, K.J. ∙ Kulshreshtha, S. ∙ Segal, D.J.</div><div><strong>The protein-binding potential of C2H2 zinc finger domains</strong></div><div><em>Cell Biochem. Biophys.</em> 2008; <strong>51</strong>:9-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12013-008-9007-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-50649114260" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18286240/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12013-008-9007-6&amp;pmid=18286240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Fedotova et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Fedotova, A.A. ∙ Bonchuk, A.N. ∙ Mogila, V.A. ...</div><div><strong>C<sub>2</sub>H<sub>2</sub> zinc finger proteins: the largest but poorly explored family of higher eukaryotic transcription factors</strong></div><div><em>Acta Nat.</em> 2017; <strong>9</strong>:47-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.32607/20758251-2017-9-2-47-58" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85021326833" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28740726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.32607%2F20758251-2017-9-2-47-58&amp;pmid=28740726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Gross and McGinnis, 1996</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gross, C.T. ∙ McGinnis, W.</div><div><strong>DEAF-1, a novel protein that binds an essential region in a deformed response element</strong></div><div><em>EMBO J.</em> 1996; <strong>15</strong>:1961-1970</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1996.tb00547.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8617243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1996.tb00547.x&amp;pmid=8617243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because METAP1 was the single conserved ZNG1 interaction identified in all yeast-two-hybrid screens, the METAP1-ZNG1 interaction was selected for additional investigation. To further define the region of METAP1 that interacts with ZNG1, the results from the three screens were compared to predict a minimal ZNG1-interacting region (<a id="crosref0455" href="#fig1">Figure&nbsp;1</a>G), revealing an N-terminal segment of 108 residues containing the C6H2 domain.</div></section><section id="sec-2-2"><h3>The METAP1 C6H2 interacts with a conserved motif in ZNG1</h3><div id="p0085" role="paragraph">The interaction of full-length recombinant ZNG1 and METAP1 from mouse and zebrafish was confirmed by affinity chromatography (<a id="crosref0460" href="#fig2">Figures&nbsp;2</a>A and <a id="crosref0465" href="#figs2">S2</a>A–S2C) and by dynamic light scattering (DLS) (<a id="crosref0470" href="#fig2">Figure&nbsp;2</a>B). Masses obtained from DLS for murine ZNG1 and METAP1 individually, as well as for the equimolar complex, were consistent with theoretical masses (<a id="crosref0475" href="#fig2">Figure&nbsp;2</a>B). Size exclusion chromatography was performed using murine ZNG1 and N-terminally truncated METAP1<sub>1–83</sub> containing the complete C6H2 domain (<a id="crosref0480" href="#figs2">Figure&nbsp;S2</a>D). The observed shift in elution volume of the 1:1 mixture, compared with individual proteins, indicates that the N terminus of METAP1 is sufficient to mediate the interaction of ZNG1 with METAP1 (<a id="crosref0485" href="#fig2">Figures&nbsp;2</a>C and <a id="crosref0490" href="#figs2">S2</a>D).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/69614d67-1762-478a-a0c3-15b8086d46e4/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2022.04.011/asset/4a93b725-7bdd-41b5-ac29-d47e783bbb8b/main.assets/gr2.jpg" height="565" width="506" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/69614d67-1762-478a-a0c3-15b8086d46e4/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">ZNG1 interacts with METAP1 via a unique C6H2 Zn finger domain</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0060" role="paragraph">(A) Affinity chromatography of mouse (<i>Mm</i>) His-MBP-ZNG1 bound to amylose resin with purified <i>Mm</i> METAP1. Detection by SDS PAGE (top two panels) and immunoblot (bottom). W, wash; F, flow through; E, elution.</div><div id="fspara0065" role="paragraph">(B) DLS analysis of full-length <i>Mm</i> ZNG1 and METAP1 shows complex formation <i>in&nbsp;vitro</i>.</div><div id="fspara0070" role="paragraph">(C) Size exclusion chromatography of full-length <i>Mm</i> ZNG1 and METAP1<sub>1–83</sub>.</div><div id="fspara0075" role="paragraph">(D) Identification of a conserved N-terminal ‘CPELVPI’ motif in <i>Mm</i>, human (<i>Hs</i>), and zebrafish (<i>Dr</i>) ZNG1s and synthetic peptides.</div><div id="fspara0080" role="paragraph">(E) Spectra of W45 fluorescence quenching of 3&nbsp;μM METAP1<sub>1–59</sub> upon <i>Mm</i> ZNG1&nbsp;N terminus binding.</div><div id="fspara0085" role="paragraph">(F) Peptide-binding curve from (E)&nbsp;fit to a 1:1 binding model.</div><div id="fspara0090" role="paragraph">(G) Measured affinities of METAP1<sub>1–59</sub> and METAP1<sub>1–79</sub> binding to <i>Mm, Hs</i>, and <i>Dr</i> ZNG1&nbsp;N termini.</div><div id="fspara0095" role="paragraph">(H) <sup>1</sup>H <sup>15</sup>N HSQC NMR spectra of METAP1<sub>1–59</sub> alone (left), in complex with <i>Mm</i> ZNG1 N-terminal peptide (middle), and as N-terminal <i>Mm</i> ZNG1 peptide fusion (right).</div><div id="fspara0100" role="paragraph">(I) Chemical shift perturbations in <sup>15</sup>N METAP1<sub>1–59</sub> upon binding to <i>Mm</i> ZNG1 peptide.</div><div id="fspara0105" role="paragraph">See also <a id="crosref1695" href="#figs2">Figure&nbsp;S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/83815183-32e6-47fe-9663-d2f75b27c7ef/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2022.04.011/asset/6118a3a0-b223-424c-ae08-41335827f906/main.assets/figs2.jpg" height="808" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/83815183-32e6-47fe-9663-d2f75b27c7ef/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Biochemical interaction of ZNG1 and METAP1, related to <a id="crosref1750" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0365" role="paragraph">(A–C) Affinity chromatography of full-length immobilized zebrafish MBP-Metap1 bound amylose (A)&nbsp;or empty control resin (B)&nbsp;with purified zebrafish Zng1 passed over (FT, flow through). Columns were serially washed (W). Elution of Metap1 (denoted E) resulted in detection of Zng1 shown on SDS PAGE (top two panels) and by immunoblot (bottom). Immunoblot signal quantified by densitometry (C).</div><div id="fspara0370" role="paragraph">(D) Size exclusion chromatography of mouse METAP<sub>1–83</sub> interaction with full-length ZNG1, visualized by SDS PAGE.</div><div id="fspara0375" role="paragraph">(E) Alignment of METAP1 C2H6 zf domains from zebrafish (<i>Dr</i>), mouse (<i>Mm</i>), human (<i>Hs</i>), and <i>Saccharomyces cerevisiae</i> (<i>Sc</i>) with MYND zf domains from indicated <i>Hs</i> proteins AML-1, BS69, and DEAF-1.</div><div id="fspara0380" role="paragraph">(F) Ribbon diagrams of previously determined structures of MYND:peptide complexes (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Harter et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Harter, M.R. ∙ Liu, C.D. ∙ Shen, C.L. ...</div><div><strong>BS69/ZMYND11 C-terminal domains bind and inhibit EBNA2</strong></div><div><em>PLoS Pathog.</em> 2016; <strong>12</strong>:e1005414</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1005414" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84959451149" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26845565/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1005414&amp;pmid=26845565" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Liu et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Liu, Y. ∙ Chen, W. ∙ Gaudet, J. ...</div><div><strong>Structural basis for recognition of SMRT/N-CoR by the MYND domain and its contribution to AML1/ETO’s activity</strong></div><div><em>Cancer Cell.</em> 2007; <strong>11</strong>:483-497</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.ccr.2007.04.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.ccr.2007.04.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-34249812945" target="_blank" aria-label="Open Scopus in new tab">Scopus (96)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17560331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2007.04.010&amp;pmid=17560331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0385" role="paragraph">(G) Consensus sequence logo of MYND and C<sub>6</sub>H<sub>2</sub> domains.</div><div id="fspara0390" role="paragraph">(H) Metal stoichiometries of <i>Mm</i> METAP1<sub>1–59</sub> and METAP1<sub>1–79</sub> as purified and after loading with Zn, as measured by ICP-MS. “n.d.” (not detected) indicates measured metal concentrations below 0.01 equiv.</div><div id="fspara0395" role="paragraph">(I) Representative spectra of fluorescence quenching of W45 in <i>Mm</i> METAP1<sub>1–79</sub> by titration with <i>Mm</i> ZNG1 peptide.</div><div id="fspara0400" role="paragraph">(J) <sup>1</sup>H <sup>15</sup>N HSQC NMR spectrum of <i>Mm</i> METAP1<sub>1–59</sub> in 2:1 complex with <i>Mm</i> ZNG1 N-terminal peptide, showing equal populations in bound and free states. Data in (C)&nbsp;are represented as mean ± SEM.</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph">Determination of a minimal binding site on ZNG1 for the interaction with METAP1 was informed by the observation that C6H2 domains resemble MYND domains, with four pairs of conserved cysteine and histidine residues and conserved aromatic residues (<a id="crosref0495" href="#figs2">Figures&nbsp;S2</a>E–S2G). No structure has been determined for a C6H2 domain, but structures of MYND domains in AML1/ETO, DEAF-1, and BS69 reveal a ββα architecture, two Zn ions coordinated in an interdigitated pattern, and a binding interface for peptides containing a PxLxP motif (<a id="crosref0500" href="#figs2">Figures&nbsp;S2</a>E and S2G) (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Harter et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Harter, M.R. ∙ Liu, C.D. ∙ Shen, C.L. ...</div><div><strong>BS69/ZMYND11 C-terminal domains bind and inhibit EBNA2</strong></div><div><em>PLoS Pathog.</em> 2016; <strong>12</strong>:e1005414</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1005414" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84959451149" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26845565/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1005414&amp;pmid=26845565" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Kateb et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kateb, F. ∙ Perrin, H. ∙ Tripsianes, K. ...</div><div><strong>Structural and functional analysis of the DEAF-1 and BS69 MYND domains</strong></div><div><em>PLoS One.</em> 2013; <strong>8</strong>:e54715</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0054715" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84872854119" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23372760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0054715&amp;pmid=23372760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Liu et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Liu, Y. ∙ Chen, W. ∙ Gaudet, J. ...</div><div><strong>Structural basis for recognition of SMRT/N-CoR by the MYND domain and its contribution to AML1/ETO’s activity</strong></div><div><em>Cancer Cell.</em> 2007; <strong>11</strong>:483-497</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.ccr.2007.04.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.ccr.2007.04.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-34249812945" target="_blank" aria-label="Open Scopus in new tab">Scopus (96)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17560331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2007.04.010&amp;pmid=17560331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The METAP1 C6H2 domain also binds two Zn ions, even when truncated to&nbsp;METAP1<sub>1–59</sub> or METAP1<sub>1–79</sub>, as measured by inductively coupled plasma-mass spectrometry (ICP-MS) (<a id="crosref0520" href="#figs2">Figure&nbsp;S2</a>H). Moreover, a conserved CPELVPI sequence at the N terminus of ZNG1 orthologs matches the PxLxP motif that MYND binds (<a id="crosref0525" href="#fig1">Figures&nbsp;1</a>B and <a id="crosref0530" href="#fig2">2</a>D).</div><div id="p0095" role="paragraph">METAP1 binding of ZNG1 peptides was confirmed by monitoring quenching of W45 fluorescence in murine METAP1<sub>1–59</sub> and METAP1<sub>1–79</sub> upon addition of chemically synthesized N-terminal ZNG1 peptides from human, mouse, and zebrafish (<a id="crosref0535" href="#fig2">Figures&nbsp;2</a>D, 2E, and <a id="crosref0540" href="#figs2">S2</a>I). These data reveal a 1:1 binding stoichiometry and a <i>K</i><sub>D</sub>&nbsp;≈ 0.1&nbsp;μM (<a id="crosref0545" href="#fig2">Figures&nbsp;2</a>F and 2G), which is ≥100-fold tighter than the MYND domains with their cognate peptides (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Kateb et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kateb, F. ∙ Perrin, H. ∙ Tripsianes, K. ...</div><div><strong>Structural and functional analysis of the DEAF-1 and BS69 MYND domains</strong></div><div><em>PLoS One.</em> 2013; <strong>8</strong>:e54715</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0054715" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84872854119" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23372760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0054715&amp;pmid=23372760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; Liu et&nbsp;al., 2007). Mouse, human, and zebrafish ZNG1 peptides bound with similar affinities to both METAP1<sub>1–59</sub> and METAP1<sub>1–79</sub> (<a id="crosref0555" href="#fig2">Figure&nbsp;2</a>G), indicating that the N-terminal 59 residues are sufficient for binding. Combined, these findings identify the specific regions of interaction between ZNG1 and METAP1.</div><div id="p0100" role="paragraph">NMR spectroscopy was used to explore the interaction in&nbsp;more detail. <sup>1</sup>H–<sup>15</sup>N HSQC spectra of <sup>15</sup>N-enriched METAP1<sub>1–59</sub> show excellent signal dispersion and uniform intensities, indicating a stable, well-folded domain (<a id="crosref0560" href="#fig2">Figure&nbsp;2</a>H, left). Addition of a half-molar equivalent of ZNG1 peptide produces a second set of peaks of equal intensity (<a id="crosref0565" href="#figs2">Figure&nbsp;S2</a>J), indicating a complex in the slow chemical exchange regime on the NMR time scale; addition of a full molar equivalent of ZNG1 peptide reveals a saturated complex (<a id="crosref0570" href="#fig2">Figure&nbsp;2</a>H, middle). These results confirm tighter binding than previously observed for MYND domains (<span class="dropBlock reference-citations"><a id="body-ref-sref50-3" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-3" aria-controls="sref50-3">Kateb et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref50-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kateb, F. ∙ Perrin, H. ∙ Tripsianes, K. ...</div><div><strong>Structural and functional analysis of the DEAF-1 and BS69 MYND domains</strong></div><div><em>PLoS One.</em> 2013; <strong>8</strong>:e54715</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0054715" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84872854119" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23372760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0054715&amp;pmid=23372760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; Liu et&nbsp;al., 2007) and motivate a more detailed analysis of the peptide-binding interface.</div></section><section id="sec-2-3"><h3>Solution structure of the METAP1 C6H2:ZNG1 peptide complex</h3><div id="p0105" role="paragraph">An NMR chemical shift perturbation assay was used to identify the METAP1 residues most directly involved in peptide binding. Complete backbone chemical shift assignments for both the peptide-free and peptide-bound states of METAP1<sub>1–59</sub> (<a id="crosref0580" href="#figs3">Figure&nbsp;S3</a>A) reveal that residues 21–24 are strongly affected by peptide binding, suggesting direct contact (<a id="crosref0585" href="#fig2">Figure&nbsp;2</a>I), while residues 1–18 and 27–32 are relatively unaffected and likely distant from the binding site. Further, chemical shift-based secondary structure predictions reveal that the METAP1 domain has a ββαβα architecture in the free and bound forms (<a id="crosref0590" href="#figs3">Figure&nbsp;S3</a>B, top, middle), similar to but distinct from the ββα architecture of MYND domains (<a id="crosref0595" href="#figs2">Figure&nbsp;S2</a>E) (<span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">Shen and Bax, 2015</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Shen, Y. ∙ Bax, A.</div><div><strong>Protein structural information derived from NMR chemical shift with the neural network program TALOS-N</strong></div><div><em>Methods Mol. Biol.</em> 2015; <strong>1260</strong>:17-32</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-2239-0_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84917706654" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25502373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-2239-0_2&amp;pmid=25502373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The secondary structure is minimally perturbed by peptide binding.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/3999aa1a-40a9-4f72-a41e-056b13c6f2db/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2022.04.011/asset/fd90e8c9-b5be-4ed4-b04a-ec6e730f8d47/main.assets/figs3.jpg" height="812" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/3999aa1a-40a9-4f72-a41e-056b13c6f2db/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">NMR structural characterization of ZNG1 and METAP1, related to <a id="crosref1765" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0410" role="paragraph">(A) Assigned <sup>1</sup>H <sup>15</sup>N HSQC NMR spectra of <i>Mm</i> METAP1<sub>1–59</sub> alone (left), in 1:1 complex with <i>Mm</i> ZNG1 N-terminal peptide (middle), and with N-terminally fused <i>Mm</i> ZNG1 N-terminal peptide (right).</div><div id="fspara0415" role="paragraph">(B) Chemical shift-based secondary structure predictions for <i>Mm</i> METAP1<sub>1–59</sub> alone (top), the 1:1 complex with <i>Mm</i> ZNG1 N-terminal peptide (middle), and with N-terminally fused <i>Mm</i> ZNG1 N-terminal peptide (bottom).</div><div id="fspara0420" role="paragraph">(C) Sequence diagram of fusion construct of <i>Mm</i> ZNG1 N-terminal peptide with <i>Mm</i> METAP1<sub>1–59</sub>.</div><div id="fspara0425" role="paragraph">(D) Overlay of <sup>1</sup>H <sup>15</sup>N HSQC NMR spectra of <i>Mm</i> METAP1<sub>1–59</sub> in 1:1 complex with <i>Mm</i> ZNG1 N-terminal peptide (blue contours) and with N-terminally fused <i>Mm</i> ZNG1 N-terminal peptide (red contours), showing that the fusion strategy eliminates peak splitting artifacts.</div><div id="fspara0430" role="paragraph">(E) Chemical shift perturbations caused by the fusion construct, relative to the complex.</div><div id="fspara0435" role="paragraph">(F) <sup>15</sup>N R<sub>1</sub> relaxation rates for the fusion (black) and free domain (red).</div><div id="fspara0440" role="paragraph">(G) <sup>15</sup>N R<sub>2</sub> relaxation rates for the fusion (black) and free domain (red).</div><div id="fspara0445" role="paragraph">(H) Surface representation of the fusion structure, colored by <sup>1</sup>H,<sup>15</sup>N chemical shift perturbations upon peptide binding. The peptide is shown as blue sticks. P23 is colored black.</div><div id="fspara0450" role="paragraph">(I) Ribbon diagram and electrostatic surface representation showing the predicted peptide-binding interface of the <i>S.&nbsp;cerevisiae</i> Map1p C6H2 domain with a Zng1p peptide.</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">Further NMR studies of the bound form of METAP1<sub>1–59</sub> were not possible due to peak splitting for many residues, a limitation circumvented by studying a fusion construct inspired by the structure determination of the AML1/ETO MYND bound to the SMRT peptide (<span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">Liu et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Liu, Y. ∙ Chen, W. ∙ Gaudet, J. ...</div><div><strong>Structural basis for recognition of SMRT/N-CoR by the MYND domain and its contribution to AML1/ETO’s activity</strong></div><div><em>Cancer Cell.</em> 2007; <strong>11</strong>:483-497</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.ccr.2007.04.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.ccr.2007.04.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-34249812945" target="_blank" aria-label="Open Scopus in new tab">Scopus (96)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17560331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2007.04.010&amp;pmid=17560331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The ZNG1 peptide was expressed at the N terminus of METAP1<sub>1–59</sub> (<a id="crosref0610" href="#figs3">Figure&nbsp;S3</a>C). The <sup>15</sup>N-labeled fusion reproduces the spectra of the bound form with additional crosspeaks for the ZNG1 peptide but without the multiplets observed previously (<a id="crosref0615" href="#fig2">Figures&nbsp;2</a>H and <a id="crosref0620" href="#figs3">S3</a>D). Minimal perturbations to chemical shifts, localized to the N terminus of METAP1, validate the utility of the fusion construct (<a id="crosref0625" href="#figs3">Figures&nbsp;S3</a>A, right; S3B, bottom; and S3E). Moreover, backbone dynamics show that the linker region is flexible and that the dynamics of the domain are consistent in the free and fusion forms (<a id="crosref0630" href="#fig3">Figures&nbsp;3</a>A, 3B, <a id="crosref0635" href="#figs3">S3</a>F, and S3G). The central region of the peptide encompassing the CPELVPI sequence has elevated heteronuclear nuclear Overhauser effect (NOE) values indicative of tight binding. An initial, unrefined NMR structure of the ZNG1<sub>10–30</sub> METAP1<sub>1–59</sub> fusion, relying exclusively on automated NOE assignments with no additional constraints, clearly shows the same cross-brace Zn-binding topology as in an MYND domain. Zn1 is coordinated by C9, C14, C36, and C40, and Zn2 by C22, C25, H48, and H52 (<a id="crosref0640" href="#fig3">Figure&nbsp;3</a>C).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/9d4b1127-30cc-44b4-ab57-35e2629ce270/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2022.04.011/asset/27cad41c-42b3-48d1-8033-c683a724a1ec/main.assets/gr3.jpg" height="433" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/9d4b1127-30cc-44b4-ab57-35e2629ce270/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">NMR structure of METAP1-ZNG1 complex</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0115" role="paragraph">(A) <sup>1</sup>H,<sup>15</sup>N NOE for the fusion (black) and free domain (red).</div><div id="fspara0120" role="paragraph">(B) Structure of the 20 lowest-energy conformers of the <i>Mm</i> ZNG1<sub>10–30</sub> METAP1<sub>1–59</sub> fusion, colored by <sup>1</sup>H,<sup>15</sup>N NOE.</div><div id="fspara0125" role="paragraph">(C) Ribbon diagram of <i>Mm</i> ZNG1<sub>10–30</sub> METAP1<sub>1–59</sub> showing Zn coordination.</div><div id="fspara0130" role="paragraph">(D) Stereo view of 20 lowest-energy conformers of <i>Mm</i> ZNG1<sub>10–30</sub> METAP1<sub>1–59</sub>, colored by secondary structure, with helices in green, beta strands in purple, and peptide in blue.</div><div id="fspara0135" role="paragraph">(E) Electrostatic surface map of the METAP1<sub>1–59</sub> peptide interface.</div><div id="fspara0140" role="paragraph">(F) N-terminal ZNG1 motif among eukaryotic cluster 3 COG0523 proteins.</div><div id="fspara0145" role="paragraph">See also <a id="crosref1700" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref1705" href="#mmc2">Table&nbsp;S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0115" role="paragraph">The refined solution structure, the first of a C<sub>6</sub>H<sub>2</sub> domain, shows a ββαβα cross-brace fold with a 1,3,2 antiparallel β-sheet in which strand-2 hydrogen bonds with the peptide (<a id="crosref0645" href="#fig3">Figure&nbsp;3</a>D; <a id="crosref0650" href="#mmc2">Table&nbsp;S2</a>), consistent with the pattern of <sup>1</sup>H, <sup>15</sup>N chemical shift perturbations (<a id="crosref0655" href="#figs3">Figure&nbsp;S3</a>H). The negatively charged peptide binds a hydrophobic groove surrounded by a positively charged surface of conserved lysine residues (<a id="crosref0660" href="#fig3">Figures&nbsp;3</a>E, <a id="crosref0665" href="#figs2">S2</a>E, and S2F). In the hydrophobic groove, C18′ and P19′ of ZNG1 make hydrophobic contacts with W45 and F41 in METAP1, L21′ inserts into the pocket lined by F35 and F41, and V22′ and I24′ pack against P23 and Y34, respectively. Sidechains of Q21 and Q38 in METAP1 form hydrogen bonds with the ZNG1 peptide backbone. These data reveal the molecular features of the ZNG1-METAP1 interaction, showing that the high affinity comes from a combination of electrostatic and hydrophobic interactions, as well as hydrogen bonds, consistent with the conservation of these residues in METAP1 and ZNG1 (<a id="crosref0670" href="#fig3">Figure&nbsp;3</a>F).</div></section><section id="sec-2-4"><h3>ZNG1 supports METAP1 function in a Zn-dependent manner</h3><div id="p0120" role="paragraph">Based on our previous work with bacterial COG0523 proteins, we hypothesize that ZNG1 uses GTP hydrolysis to transfer Zn from the high-affinity CxCC binding site into the METAP1 active site, thereby restoring METAP1 activity under conditions of low cellular Zn (<span class="dropBlock reference-citations"><a id="body-ref-sref47-3" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-3" aria-controls="sref47-3">Jordan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Jordan, M.R. ∙ Wang, J. ∙ Weiss, A. ...</div><div><strong>Mechanistic insights into the metal-dependent activation of Zn(II)-dependent metallochaperones</strong></div><div><em>Inorg. Chem.</em> 2019; <strong>58</strong>:13661-13672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.inorgchem.9b01173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85073651560" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31247880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.inorgchem.9b01173&amp;pmid=31247880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition to the two Zn ions coordinated by the C6H2 domain, METAP1 binds two metal ions in the active site (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Addlagatta et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Addlagatta, A. ∙ Hu, X. ∙ Liu, J.O. ...</div><div><strong>Structural basis for the functional differences between type I and type II human methionine aminopeptidases</strong></div><div><em>Biochemistry.</em> 2005; <strong>44</strong>:14741-14749</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi051691k" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-27744565323" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16274222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi051691k&amp;pmid=16274222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Recombinant murine METAP1 can therefore be loaded with Zn to four molar equivalents (termed Zn<sub>4</sub>METAP1). Removal of Zn by extended exposure to EDTA disrupts the zf fold (<a id="crosref0685" href="#figs4">Figures&nbsp;S4</a>A and S4B); however, gentler treatment with chelators, including nitrilotriacetic acid (NTA; <i>K</i><sub>Zn</sub>&nbsp;≈ 10&nbsp;nM), selectively removes metal from the active site while leaving the zf domain intact, generating Zn<sub>2</sub>METAP1 (<a id="crosref0690" href="#figs4">Figure&nbsp;S4</a>C). Mag-fura-2 (mf2) and quin-2 Zn competition experiments with Zn<sub>2</sub>METAP1 show two high affinity (<i>K</i><sub>Zn1</sub>&nbsp;≈ 8 pM, <i>K</i><sub>Zn2</sub>&nbsp;≈ 20 pM) and two low affinity Zn-binding sites (<a id="crosref0695" href="#fig4">Figures&nbsp;4</a>A and <a id="crosref0700" href="#figs4">S4</a>D). Murine ZNG1 binds three molar equivalents of Zn (<a id="crosref0705" href="#figs4">Figures&nbsp;S4</a>E and S4F), with one high-affinity site (<i>K</i><sub>Zn1</sub>&nbsp;≈ 5 pM), likely coordinated by the CxCC motif in the core GTPase domain (<a id="crosref0710" href="#fig4">Figure&nbsp;4</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref47-4" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-4" aria-controls="sref47-4">Jordan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Jordan, M.R. ∙ Wang, J. ∙ Weiss, A. ...</div><div><strong>Mechanistic insights into the metal-dependent activation of Zn(II)-dependent metallochaperones</strong></div><div><em>Inorg. Chem.</em> 2019; <strong>58</strong>:13661-13672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.inorgchem.9b01173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85073651560" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31247880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.inorgchem.9b01173&amp;pmid=31247880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and two low affinity sites. Like other COG0523 proteins of the same phylogenetic grouping (<span class="dropBlock reference-citations"><a id="body-ref-sref14-3" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-3" aria-controls="sref14-3">Edmonds et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref14-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Edmonds, K.A. ∙ Jordan, M.R. ∙ Giedroc, D.P.</div><div><strong>COG0523 proteins: a functionally diverse family of transition metal-regulated G3E P-loop GTP hydrolases from bacteria to man</strong></div><div><em>Metallomics.</em> 2021; <strong>13</strong>:mfab046</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/mtomcs/mfab046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85113538279" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34302342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmtomcs%2Fmfab046&amp;pmid=34302342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), GTP hydrolysis is slow and slightly inhibited when Zn is bound solely at the high-affinity site (ZnZNG1) (<a id="crosref0725" href="#fig4">Figures&nbsp;4</a>C and <a id="crosref0730" href="#figs4">S4</a>G) (<span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">Sydor et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Sydor, A.M. ∙ Jost, M. ∙ Ryan, K.S. ...</div><div><strong>Metal binding properties of <i>Escherichia coli</i> YjiA, a member of the metal homeostasis-associated COG0523 family of GTPases</strong></div><div><em>Biochemistry.</em> 2013; <strong>52</strong>:1788-1801</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi301600z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84874997087" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24449932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi301600z&amp;pmid=24449932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, ZnZNG1 GTPase activity is stimulated by Zn<sub>2</sub>METAP1, while ApoZNG1 is not (<a id="crosref0740" href="#fig4">Figure&nbsp;4</a>C).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/68f767d7-d60f-4d77-af8f-a5be830e68c4/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2022.04.011/asset/89cdb485-f309-4d79-963a-bbc61c901aa7/main.assets/gr4.jpg" height="555" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/68f767d7-d60f-4d77-af8f-a5be830e68c4/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">ZNG1 enhances METAP1 aminopeptidase activity <i>in&nbsp;vitro</i></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0155" role="paragraph">(A) Representative titration of 1.0&nbsp;μM METAP1, 1.0&nbsp;μM quin-2 with Zn.</div><div id="fspara0160" role="paragraph">(B) Representative titration of 1.3&nbsp;μM ZNG1, 1.3&nbsp;μM quin-2 with Zn. Solid lines are global fits of independent replicates, dashed lines are simulations of 10-fold greater and weaker affinities.</div><div id="fspara0165" role="paragraph">(C) GTPase activity of ZNG1 with or without Zn and/or Zn<sub>2</sub>METAP1.</div><div id="fspara0170" role="paragraph">(D) Initial rate of Met release from 750&nbsp;μM MAHAIHY peptide by 20&nbsp;nM METAP1 in different metalation states.</div><div id="fspara0175" role="paragraph">(E) Activation of 10&nbsp;μM Zn<sub>2</sub>METAP1 by 25&nbsp;μM ZNG1, with or without Zn and 500&nbsp;μM guanine nucleotides.</div><div id="fspara0180" role="paragraph">(F) Activation of 10&nbsp;μM Zn<sub>2</sub>METAP1 by 22.5&nbsp;μM Zn or 25&nbsp;μM ZnZNG1, with or without 500&nbsp;μM guanine nucleotides, all with 250&nbsp;μM NTA. In (E)&nbsp;and (F), activation reactions were diluted to 20&nbsp;nM METAP1 prior to monitoring cleavage of 750&nbsp;μM MAHAIHY.</div><div id="fspara0185" role="paragraph">(G) Mechanistic model of GTP hydrolysis-coupled Zn transfer from ZNG1 to the METAP1 active site.</div><div id="fspara0190" role="paragraph">See also <a id="crosref1710" href="#figs4">Figure&nbsp;S4</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/5a859675-82f6-4465-9834-d0bcaf64ad5f/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2022.04.011/asset/dd0ca4d3-5389-4386-b080-878e40168391/main.assets/figs4.jpg" height="737" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/5a859675-82f6-4465-9834-d0bcaf64ad5f/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text"><i>In&nbsp;vitro</i> METAP1 activity assay, related to <a id="crosref1770" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0460" role="paragraph">(A) Unfolding of METAP1<sub>1–59</sub> as measured by a decrease in W45 fluorescence as a function of EDTA concentration and time.</div><div id="fspara0465" role="paragraph">(B) Unfolding of METAP1<sub>1–59</sub> as monitored by <sup>15</sup>N-HSQC spectra recorded at increasing concentrations of EDTA. METAP1<sub>1–59</sub> in the presence of 50&nbsp;mM EDTA gives spectra indicative of protein aggregation.</div><div id="fspara0470" role="paragraph">(C) Metal stoichiometries of <i>Mm</i> METAP1 as purified, supplemented with excess Zn, or when treated with various metal chelators as measured by ICP-MS. “n.d.” (not detected) indicates measured metal concentrations below 0.01 equiv.</div><div id="fspara0475" role="paragraph">(D) Zn<sub>2</sub>METAP1 Zn binding. Representative titration of Zn into 1&nbsp;μM Zn<sub>2</sub>METAP1 and 0.8&nbsp;μM mf2.</div><div id="fspara0480" role="paragraph">(E) ZNG1 Zn binding. Representative titration of Zn into 7.5&nbsp;μM ZNG1 and 16&nbsp;μM mf2.</div><div id="fspara0485" role="paragraph">For (D)&nbsp;and (E), solid lines represent the global fit from three independent replicates and dashed lines represent simulations of 10-fold greater and weaker affinities.</div><div id="fspara0490" role="paragraph">(F) Zn-binding affinities of <i>Mm</i> METAP1 and <i>Mm</i> ZNG1 as measured by competition with quin2 and mf2.</div><div id="fspara0495" role="paragraph">(G) ZNG1 GTPase activity as measured by enzyme-coupled assay in Apo and Zn states.</div><div id="fspara0500" role="paragraph">(H) Representative time-course analysis of the cleavage of 125&nbsp;μM MAHAIHY peptide by Zn<sub>4</sub>METAP1, by monitoring the release of Met. Indicated timepoint was analyzed by LC-ESI-MS (see I).</div><div id="fspara0505" role="paragraph">(I) Extracted ion chromatograms of product AHAIHY (top) and substrate MAHAIHY (bottom), peptides taken from the indicated 10-min reaction from (H).</div><div id="fspara0510" role="paragraph">(J) Initial rates of MAHAIHY peptide cleavage at varying substrate concentrations by Zn<sub>4</sub>METAP1.</div><div id="fspara0515" role="paragraph">(K) Michaelis-Menten kinetics of the Zn<sub>4</sub>METAP1 cleavage of the MAHAIHY peptide.</div><div id="fspara0520" role="paragraph">(L) Initial rate of cleavage of 750&nbsp;μM MAHAIHY by 20&nbsp;nM <i>Mm</i> Zn<sub>2</sub>METAP1 or Zn<sub>4</sub>METAP1 in the presence or absence of Apo or Zn-bound murine ZNG1, all in the absence of GTP.</div><div id="fspara0525" role="paragraph">(M) Cleavage of 100&nbsp;μM MAHAIHY peptide after incubation of 1&nbsp;μM <i>Dr</i> Zng1 and 1&nbsp;μM Metap1. Data are represented as mean ± SEM. Data in (M)&nbsp;analyzed by one-way ANOVA with Tukey’s multiple comparison test. Significance denoted as: <sup>∗</sup> p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup> p&nbsp;&lt;&nbsp;0.01, <sup>∗∗∗</sup> p&nbsp;&lt;&nbsp;0.001, <sup>∗∗∗∗</sup> p&nbsp;&lt;&nbsp;0.0001.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">To assess the effect of ZNG1 on METAP1 activity <i>in&nbsp;vitro</i>, we first determined initial rates of aminopeptidase activity for the various Zn-bound forms of METAP1 by quantifying released methionine (Met) from the heptapeptide MAHAIHY, a known METAP1 target (<span class="dropBlock reference-citations"><a id="body-ref-sref106-1" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-1" aria-controls="sref106-1">Xiao et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref106-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Xiao, Q. ∙ Zhang, F. ∙ Nacev, B.A. ...</div><div><strong>Protein N-terminal processing: substrate specificity of <i>Escherichia coli</i> and human methionine aminopeptidases</strong></div><div><em>Biochemistry.</em> 2010; <strong>49</strong>:5588-5599</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi1005464" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77954194681" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20521764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi1005464&amp;pmid=20521764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0750" href="#figs4">Figures&nbsp;S4</a>H–S4K). Active site-bound Zn is required for activity as Zn<sub>2</sub>METAP1 exhibits no turnover, while Zn<sub>3</sub>METAP1 partially recovers the activity of fully active Zn<sub>4</sub>METAP1 (<a id="crosref0755" href="#fig4">Figure&nbsp;4</a>D). Incubation of ApoZNG1 and Zn<sub>2</sub>METAP1 shows no activation of Zn<sub>2</sub>METAP1, while the addition of ZnZNG1 recovers only small amounts of activity (<a id="crosref0760" href="#fig4">Figure&nbsp;4</a>E). ZnZNG1 shows no activation of fully metalated Zn<sub>4</sub>METAP1 (<a id="crosref0765" href="#figs4">Figure&nbsp;S4</a>L), consistent with a regulatory impact only on undermetalated METAP1. We next assessed the nucleotide dependence of ZnZNG1 activation of Zn<sub>2</sub>METAP1 by incubation with GTP, GDP, or nonhydrolyzable GTP analog, GDPNP. In the presence of GTP, ZnZNG1 nearly fully activates METAP1, and GTP hydrolysis is required for this activation as ZnZNG1 in the presence of GDPNP exhibited no activation (<a id="crosref0770" href="#fig4">Figure&nbsp;4</a>E). Although GDP shows some ZnZNG1-dependent activation of METAP1 above that of ZnZNG1 alone, it is far less than that of GTP (<a id="crosref0775" href="#fig4">Figure&nbsp;4</a>E). This could result from a conformational change in ZNG1 after GTP hydrolysis that may facilitate the release of Zn into the METAP1 active site (<span class="dropBlock reference-citations"><a id="body-ref-sref47-5" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-5" aria-controls="sref47-5">Jordan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Jordan, M.R. ∙ Wang, J. ∙ Weiss, A. ...</div><div><strong>Mechanistic insights into the metal-dependent activation of Zn(II)-dependent metallochaperones</strong></div><div><em>Inorg. Chem.</em> 2019; <strong>58</strong>:13661-13672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.inorgchem.9b01173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85073651560" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31247880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.inorgchem.9b01173&amp;pmid=31247880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We performed the same activation experiments in the presence of NTA to ensure Zn-dependent activation of Zn<sub>2</sub>METAP1 is due to Zn transfer from ZNG1 directly to METAP1, rather than from Zn in solution. The GTP-bound form of ZnZNG1 again exhibits significant activation of Zn<sub>2</sub>METAP1 above that of NTA-bound Zn, while GDPNP has no effect (<a id="crosref0785" href="#fig4">Figure&nbsp;4</a>F). Similarly, ZnZNG1 in the presence and absence of GDP shows only minor activation of Zn<sub>2</sub>METAP1 activity (<a id="crosref0790" href="#fig4">Figure&nbsp;4</a>F). Zebrafish Zng1 also stimulates apo-Metap1 activity (<a id="crosref0795" href="#figs4">Figure&nbsp;S4</a>M). Together, these data establish that ZNG1 activates Zn<sub>2</sub>METAP1 in a GTP-hydrolysis and Zn-dependent manner (<a id="crosref0800" href="#fig4">Figure&nbsp;4</a>G).</div></section><section id="sec-2-5"><h3>Zng1 promotes <i>in&nbsp;vivo</i> Metap function in zebrafish</h3><div id="p0130" role="paragraph">To study the <i>in&nbsp;vivo</i> role of Zng1, we turned to a zebrafish model which encodes a single copy of <i>zng1</i> (<a id="crosref0805" href="#fig1">Figure&nbsp;1</a>A). Based on the reported Zn-dependent repression of <i>ZNG1</i> in human cell lines (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Coneyworth et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Coneyworth, L.J. ∙ Jackson, K.A. ∙ Tyson, J. ...</div><div><strong>Identification of the human zinc transcriptional regulatory element (ZTRE): a palindromic protein-binding DNA sequence responsible for zinc-induced transcriptional repression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:36567-36581</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M112.397000&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M112.397000&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84867793466" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22902622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M112.397000&amp;pmid=22902622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-3" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-3" aria-controls="sref73-3">Ogo et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref73-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Ogo, O.A. ∙ Tyson, J. ∙ Cockell, S.J. ...</div><div><strong>The zinc finger protein ZNF658 regulates the transcription of genes involved in zinc homeostasis and affects ribosome biogenesis through the zinc transcriptional regulatory element</strong></div><div><em>Mol. Cell. Biol.</em> 2015; <strong>35</strong>:977-987</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01298-14" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25582195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01298-14&amp;pmid=25582195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we first sought to assess <i>zng1</i> expression in a zebrafish. To modulate organismal Zn levels, chemical treatment of zebrafish larvae was performed via immersion in system water containing the Zn chelator N,N,N′,N′-Tetrakis(2-pyridylmethyl)ethylenediamine (TPEN) resulting in a significant induction of <i>zng1</i> transcript in wild-type (WT) larvae (<a id="crosref0820" href="#fig5">Figure&nbsp;5</a>A). To investigate Zng1-Metap1 interaction <i>in&nbsp;vivo</i>, we generated <i>zng1</i> mutant zebrafish (referred to as <i>zng1</i><sup><i>−/−</i></sup>) using CRISPR-Cas9 gene editing (<a id="crosref0825" href="#fig5">Figures&nbsp;5</a>A and <a id="crosref0830" href="#figs5">S5</a>A). <i>zng1</i> mutant fish survived to adulthood under Zn-replete conditions, yet morphometric analysis revealed that <i>zng1</i> mutant larvae exhibited moderate developmental delay with reduced height at anterior anal fin (HAA), standard length, and snout to vent length at 6&nbsp;days post fertilization (dpf) (<a id="crosref0835" href="#fig5">Figures&nbsp;5</a>B–5D and <a id="crosref0840" href="#figs5">S5</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Parichy et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Parichy, D.M. ∙ Elizondo, M.R. ∙ Mills, M.G. ...</div><div><strong>Normal table of postembryonic zebrafish development: staging by externally visible anatomy of the living fish</strong></div><div><em>Dev. Dyn.</em> 2009; <strong>238</strong>:2975-3015</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-70949095044" target="_blank" aria-label="Open Scopus in new tab">Scopus (514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19891001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22113&amp;pmid=19891001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/337f3234-ff52-4d6c-b47b-82f6ab98628e/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2022.04.011/asset/fea10efc-7d8f-4cd7-95d2-9a04b5ab9f65/main.assets/gr5.jpg" height="760" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/337f3234-ff52-4d6c-b47b-82f6ab98628e/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text"><i>zng1</i> mutant zebrafish display Zn-dependent sensitivity to Metap inhibition and phenocopy <i>metap1</i> mutant larvae</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0200" role="paragraph">(A) <i>zng1</i> expression levels measured by RT-qPCR in TPEN-treated WT and <i>zng1</i><sup><i>−/−</i></sup> whole zebrafish larvae, 6 dpf (n&nbsp;= 4 pools of 30 larvae per genotype/treatment group).</div><div id="fspara0205" role="paragraph">(B–D) Height at anterior anal fin (HAA) (B), standard length (C), and snout to vent length (D)&nbsp;of <i>zng1</i><sup><i>−/−</i></sup> and WT larvae, 6 dpf. Each data point represents an individual larva.</div><div id="fspara0210" role="paragraph">(E) Survival of WT and <i>zng1</i><sup><i>−/−</i></sup> mutant larvae treated with Bengamide B and TPEN (n&nbsp;= 3 groups of five larvae/genotype/treatment) over 3&nbsp;days of treatment and terminating at 6 dpf.</div><div id="fspara0215" role="paragraph">(F) Brightfield images of <i>metap1</i> mutant larvae treated with 1&nbsp;μM Bengamide B or vehicle control, 6 dpf. Yellow arrows denote abnormal pathologies. PCE, pericardial edema; CD, craniofacial defects; YSE, yolk sack edema; USB, uninflated swim bladder. Scale bars, 1&nbsp;mm.</div><div id="fspara0220" role="paragraph">(G) Quantification of gross pathology score of Bengamide B-treated <i>metap1</i> mutant and WT larvae, 6 dpf. Pathology scores calculated as sum of each pathology indicated by the yellow arrows in (F). Each data point represents an individual larva.</div><div id="fspara0225" role="paragraph">(H) Brightfield images of <i>zng1</i> mutant larvae treated with 500&nbsp;nM Bengamide B or vehicle control, 6 dpf. Scale bars, 1&nbsp;mm.</div><div id="fspara0230" role="paragraph">(I) Quantification of gross pathology score of Bengamide B-treated <i>zng1</i> mutant larvae as depicted above, 6 dpf.</div><div id="fspara0235" role="paragraph">(J) Brightfield images of WT larvae treated with 1&nbsp;μM Bengamide B or vehicle control, 6 dpf. Scale bars, 1&nbsp;mm.</div><div id="fspara0240" role="paragraph">(K) Survival of WT and <i>zng1</i><sup><i>−/−</i></sup> larvae treated with TNP-470 and TPEN (n&nbsp;= 3 groups of five larvae/genotype/treatment) over 3&nbsp;days of treatment and terminating at 6 dpf. Data in (A), (E), and (K)&nbsp;analyzed by two-way ANOVA with Tukey’s multiple comparison test. Data in (B), (C), (D), (G), and (I)&nbsp;analyzed by Student’s t test.</div><div id="fspara0245" role="paragraph">Data are represented as mean ± SEM. Significance denoted as: ∗p&nbsp;&lt; 0.05, ∗∗p &lt; 0.01, ∗∗∗p&nbsp;&lt; 0.001, ∗∗∗∗p&nbsp;&lt; 0.0001. See also <a id="crosref1715" href="#figs5">Figure&nbsp;S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/8860597d-dd1e-4946-b059-7febbf39fce6/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2022.04.011/asset/c2a7bfd4-ea95-4c20-b8e1-b25ee29982d4/main.assets/figs5.jpg" height="784" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/8860597d-dd1e-4946-b059-7febbf39fce6/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Generation of <i>zng1</i> and <i>metap1</i> mutant zebrafish, related to <a id="crosref1775" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0535" role="paragraph">(A) Schematic of zebrafish <i>zng1</i> locus with gRNA shown by red bar in second exon. Mutant alleles and <i>in silico</i> translations indicated. Arrows indicate screening primer binding sites.</div><div id="fspara0540" role="paragraph">(B) Genotype distributions of WT, <i>zng1</i><sup><i>−/+</i></sup>, and homozygous <i>zng1</i><sup><i>−/−</i></sup> 6 dpf larvae. Number of animals indicated in white text.</div><div id="fspara0545" role="paragraph">(C) Survival of WT and <i>zng1</i><sup><i>−/−</i></sup> mutant larvae treated with Bengamide B and TPEN (n&nbsp;= 3 groups of five larvae/genotype/treatment) over the course of 3&nbsp;days of treatment.</div><div id="fspara0550" role="paragraph">(D) Measurement of height at anal fin (HAA) of WT larvae at 6 dpf following 3&nbsp;days of exposure to 500&nbsp;nM Bengamide B or vehicle control. Each data point represents an individual larva.</div><div id="fspara0555" role="paragraph">(E) Schematic of zebrafish <i>metap1</i> locus with gRNA shown by red bar in first exon. Mutant allele and <i>in silico</i> translations indicated. Arrows indicate screening primer binding sites.</div><div id="fspara0560" role="paragraph">(F) Genotype distributions of WT, <i>metap1</i><sup><i>−/+</i></sup>, and homozygous <i>metap1</i><sup><i>−/−</i></sup> 6 dpf larvae. Number of animals indicated in white text.</div><div id="fspara0565" role="paragraph">(G) Percent of WT and <i>metap1</i><sup><i>−/−</i></sup> larvae with pericardial edema, uninflated swim bladder, craniofacial abnormalities, or yolk sac edema at 6 dpf following treatment with 1&nbsp;μM Bengamide B or vehicle control for 3&nbsp;days (n&nbsp;= 6–8 larvae/genotype).</div><div id="fspara0570" role="paragraph">(H) Percent of WT and <i>zng1</i><sup><i>−/−</i></sup> larvae with pericardial edema, uninflated swim bladder, craniofacial abnormalities, or yolk sac edema at 6 dpf following treatment with 500&nbsp;nM Bengamide B of vehicle control for 3&nbsp;days (n&nbsp;= 18–20 larvae/genotype).</div><div id="fspara0575" role="paragraph">(I) Survival of WT and <i>zng1</i><sup><i>−/−</i></sup> larvae treated with TNP-470 and TPEN (n&nbsp;= 3 groups of five larvae/genotype/treatment) over the course of 3&nbsp;days of treatment. Data are represented as mean ± SEM. Data in (B)&nbsp;and (F)&nbsp;analyzed by chi-square goodness-of-fit test. Data in (D)&nbsp;analyzed by Student’s t test. Significance denoted as: <sup>∗</sup> p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup> p&nbsp;&lt;&nbsp;0.01, <sup>∗∗∗</sup> p&nbsp;&lt;&nbsp;0.001, <sup>∗∗∗∗</sup> p&nbsp;&lt;&nbsp;0.0001.</div></div></div></div></figcaption></figure></div><div id="p0135" role="paragraph">We hypothesized a role of Zng1 in maintaining Metap1 function and Zn homeostasis in zebrafish. Zebrafish encode for a single copy of Metap1 with high sequence similarity to mammalian METAP1 and two paralogs of Metap2 (Metap2a and Metap2b). Previous studies indicate functional redundancy between METAP1 and METAP2, suggesting that zebrafish Metap2 paralogs could compensate for decreased Metap1 activity in <i>zng1</i><sup><i>−/−</i></sup> animals (<span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Frottin et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Frottin, F. ∙ Martinez, A. ∙ Peynot, P. ...</div><div><strong>The proteomics of N-terminal methionine cleavage</strong></div><div><em>Mol. Cell. Proteomics.</em> 2006; <strong>5</strong>:2336-2349</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M600225-MCP200&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M600225-MCP200&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33845993905" target="_blank" aria-label="Open Scopus in new tab">Scopus (289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16963780/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M600225-MCP200&amp;pmid=16963780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Li and Chang, 1995</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Li, X. ∙ Chang, Y.H.</div><div><strong>Amino-terminal protein processing in Saccharomyces cerevisiae is an essential function that requires two distinct methionine aminopeptidases</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1995; <strong>92</strong>:12357-12361</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.92.26.12357" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029585125" target="_blank" aria-label="Open Scopus in new tab">Scopus (256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8618900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.92.26.12357&amp;pmid=8618900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106-2" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-2" aria-controls="sref106-2">Xiao et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref106-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Xiao, Q. ∙ Zhang, F. ∙ Nacev, B.A. ...</div><div><strong>Protein N-terminal processing: substrate specificity of <i>Escherichia coli</i> and human methionine aminopeptidases</strong></div><div><em>Biochemistry.</em> 2010; <strong>49</strong>:5588-5599</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi1005464" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77954194681" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20521764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi1005464&amp;pmid=20521764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To investigate the effects of impaired Metap1 function on organismal health, Metap1 activity <i>in&nbsp;vivo</i> was modulated using pharmacologic inhibition of Metap enzymes in larval zebrafish. WT and <i>zng1</i><sup><i>−/−</i></sup> larvae were exposed to the established METAP1/2 inhibitor, Bengamide B, in Zn-deplete or replete conditions, and survival was monitored over three days (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Quinoa et&nbsp;al., 1986</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Quinoa, E. ∙ Adamczeski, M. ∙ Crews, P. ...</div><div><strong>Bengamides, heterocyclic anthelmintics from a Jaspidae marine sponge</strong></div><div><em>J.&nbsp;Org. Chem.</em> 1986; <strong>51</strong>:4494-4497</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jo00373a036" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjo00373a036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">White et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>White, K.N. ∙ Tenney, K. ∙ Crews, P.</div><div><strong>The Bengamides: a mini-review of natural sources, analogues, biological properties, biosynthetic origins, and future prospects</strong></div><div><em>J.&nbsp;Nat. Prod.</em> 2017; <strong>80</strong>:740-755</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jnatprod.6b00970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85016142079" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28185457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jnatprod.6b00970&amp;pmid=28185457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>zng1</i> mutant larvae displayed increased sensitivity to Bengamide B with 100% lethality when treated with 1&nbsp;μM of inhibitor as compared with WT controls, which remained viable (<a id="crosref0875" href="#fig5">Figures&nbsp;5</a>E and <a id="crosref0880" href="#figs5">S5</a>C). We hypothesized that inhibition of Metap activity in <i>zng1</i> mutant animals would be exacerbated in conditions of low Zn. Indeed, co-treatment of <i>zng1</i> mutant larvae with a sublethal dose of Bengamide B (500&nbsp;nM) and TPEN resulted in decreased viability as compared with treatment with 500&nbsp;nM Bengamide B alone (<a id="crosref0885" href="#fig5">Figures&nbsp;5</a>E and <a id="crosref0890" href="#figs5">S5</a>C). Notably, WT animals treated with Bengamide B also displayed significantly reduced HAA (<a id="crosref0895" href="#figs5">Figure&nbsp;S5</a>D), mirroring the defects observed in <i>zng1</i><sup><i>−/−</i></sup> larvae (<a id="crosref0900" href="#fig5">Figure&nbsp;5</a>B) and indicating a functional link between Zng1 and Metap1 <i>in&nbsp;vivo</i>. Combined, these data suggest Metap activity is impaired in <i>zng1</i> mutant larvae, especially in Zn-deplete conditions.</div><div id="p0140" role="paragraph">To further evaluate Zng1 modulation of <i>in&nbsp;vivo</i> Metap1 activity, <i>metap1</i> mutant zebrafish were generated by CRISPR-Cas9 (referred to as <i>metap1</i><sup><i>−/−</i></sup>) (<a id="crosref0905" href="#figs5">Figures&nbsp;S5</a>E and S5F). As expected, pharmacologic inhibition of total Metap activity in <i>metap1</i><sup><i>−/−</i></sup> larvae resulted in significantly higher morphologic abnormalities as compared with WT controls at 6 dpf (<a id="crosref0910" href="#fig5">Figures&nbsp;5</a>F and 5G). These abnormalities included pericardial edema (PCE), yolk sack edema (YSE), uninflated swim bladder (USB), and craniofacial defects (CD) (<a id="crosref0915" href="#fig5">Figures&nbsp;5</a>F and <a id="crosref0920" href="#figs5">S5</a>G) (<span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Raghunath and Perumal, 2018</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Raghunath, A. ∙ Perumal, E.</div><div><strong>Analysis of lethality and malformations during zebrafish (<i>Danio rerio</i>) development</strong></div><div><em>Methods Mol. Biol.</em> 2018; <strong>1797</strong>:337-363</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-7883-0_18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85048624330" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29896702/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-7883-0_18&amp;pmid=29896702" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Importantly, <i>zng1</i> mutant larvae exhibited similar defects when exposed to the maximum tolerable dose of Bengamide B (<a id="crosref0930" href="#fig5">Figures&nbsp;5</a>H–5J and <a id="crosref0935" href="#figs5">S5</a>H), consistent with a role for Zng1 in the promotion of Metap1 activity <i>in&nbsp;vivo</i>.</div><div id="p0145" role="paragraph">Finally, to test whether Metap1 can compensate for Metap2-inhibition in WT animals but not in <i>zng1</i>-deficient animals (<span class="dropBlock reference-citations"><a id="body-ref-sref21-3" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-3" aria-controls="sref21-3">Frottin et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref21-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Frottin, F. ∙ Martinez, A. ∙ Peynot, P. ...</div><div><strong>The proteomics of N-terminal methionine cleavage</strong></div><div><em>Mol. Cell. Proteomics.</em> 2006; <strong>5</strong>:2336-2349</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M600225-MCP200&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M600225-MCP200&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33845993905" target="_blank" aria-label="Open Scopus in new tab">Scopus (289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16963780/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M600225-MCP200&amp;pmid=16963780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106-3" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-3" aria-controls="sref106-3">Xiao et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref106-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Xiao, Q. ∙ Zhang, F. ∙ Nacev, B.A. ...</div><div><strong>Protein N-terminal processing: substrate specificity of <i>Escherichia coli</i> and human methionine aminopeptidases</strong></div><div><em>Biochemistry.</em> 2010; <strong>49</strong>:5588-5599</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi1005464" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77954194681" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20521764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi1005464&amp;pmid=20521764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the METAP2-specific inhibitor TNP-470 was applied by immersion and larval survival was assessed (<span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Zhang et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Zhang, Y. ∙ Griffith, E.C. ∙ Sage, J. ...</div><div><strong>Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21WAF1/CIP1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2000; <strong>97</strong>:6427-6432</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.97.12.6427" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0034612236" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10841547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.97.12.6427&amp;pmid=10841547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similar to what was observed following treatment with Bengamide B, <i>zng1</i><sup><i>−/−</i></sup> larvae displayed decreased survival compared with WT animals following TNP-470 exposure, which became more pronounced in Zn-deplete conditions (<a id="crosref0955" href="#fig5">Figures&nbsp;5</a>K and <a id="crosref0960" href="#figs5">S5</a>I). Cumulatively, these data suggest that Metap1 activity is perturbed in <i>zng1</i> mutant larvae and, consequently, the pharmacologic potency of both Metap inhibitors is enhanced in the absence of Zng1, particularly during Zn deprivation.</div></section><section id="sec-2-6"><h3><i>Zng1</i><sup><i>−/−</i></sup> mice are sensitive to dietary Zn starvation and display signs of mitochondrial dysfunction</h3><div id="p0150" role="paragraph">To investigate the impact of ZNG1 in a mammalian model, <i>Zng1</i> mutant mice (referred to as <i>Zng1</i><sup><i>−/−</i></sup>) were generated using CRISPR–Cas9. The resulting deletion spanning the annotated start codon, as well as 50 amino acids within exon 1 (including the CPELVPI motif), was confirmed at the transcriptional and protein level (<a id="crosref0965" href="#figs6">Figures&nbsp;S6</a>A–S6C). <i>Zng1</i><sup><i>−/−</i></sup> animals were viable and did not display anomalies in gross morphology. Nevertheless, breeding patterns showed non-Mendelian inheritance (<a id="crosref0970" href="#fig6">Figures&nbsp;6</a>A, <a id="crosref0975" href="#figs6">S6</a>D, and S6E), suggesting that ZNG1 functions in early development. We next assessed whether ZNG1 affects Zn homeostasis during conditions of dietary Zn deprivation by administering a low-Zn diet to WT and <i>Zng1</i><sup><i>−/−</i></sup> mice. Mice deficient in <i>Zng1</i> displayed decreased weight gain when fed a low Zn diet (<a id="crosref0980" href="#fig6">Figure&nbsp;6</a>B), yet no difference in food uptake was observed. Notably, both WT and <i>Zng1</i><sup><i>−/−</i></sup> animals gained weight at similar rates when fed a control diet (<a id="crosref0985" href="#figs6">Figure&nbsp;S6</a>F). Although a low Zn diet was shown to effectively decrease Zn levels in various organ systems, differences in organ Zn levels between WT and mutant animals were not observed (<a id="crosref0990" href="#figs6">Figure&nbsp;S6</a>G). These data suggest that loss of <i>Zng1</i> leads to failure to access existing Zn pools rather than affecting organ-specific total Zn levels.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/fd629031-3eb6-4ae4-8fe7-ec886a09737a/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2022.04.011/asset/2a7b30e0-073a-4de3-ad4c-3e4e4d8e109d/main.assets/gr6.jpg" height="464" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/fd629031-3eb6-4ae4-8fe7-ec886a09737a/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text"><i>Zng1</i> mutant mice exhibit signatures of mitochondrial dysfunction on a Zn-deficient diet</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0255" role="paragraph">(A) Genotype distribution of WT, <i>Zng1</i><sup><i>+/−</i></sup>, and <i>Zng1</i><sup><i>−/−</i></sup> mice. Number of animals per genotype indicated in white text.</div><div id="fspara0260" role="paragraph">(B) Percent weight gain of 5- to 7-week-old WT and <i>Zng1</i> mutant mice that were placed on a Zn-deficient diet for 6&nbsp;weeks (n&nbsp;= 4–5 mice/genotype/sex).</div><div id="fspara0265" role="paragraph">(C) Differential protein abundances from dissected kidneys of 11- to 13-week-old female WT and <i>Zng1</i> mutant mice that were maintained on a low Zn diet for 5&nbsp;weeks. Proteins that putatively localize to the mitochondria are highlighted in red (n&nbsp;= 5 mice/genotype).</div><div id="fspara0270" role="paragraph">(D) IPA analysis of differentially abundant proteins (<i>Zng1</i><sup><i>−/−</i></sup><i>/</i> WT) with significance of gene enrichment and number of differentially abundant proteins for each pathway depicted. <i>Z</i> scores are denoted within teal bars and indicate the predicted effects on each pathway (positive values: activation; negative values: inhibition, n.a.: no prediction through IPA available due to insufficient evidence in the Knowledge Base). Red bars indicate enriched proteins in <i>Zng1</i> mutant animals and blue bars represent proteins with reduced abundance. Data in (A)&nbsp;analyzed by chi-squared goodness-of-fit test. Data in (B)&nbsp;analyzed by two-way ANOVA with Sidak’s multiple comparison test within each sex.</div><div id="fspara0275" role="paragraph">Data are represented as mean ± SEM. Significance denoted as: ∗p&nbsp;&lt; 0.05, ∗∗p &lt; 0.01, ∗∗∗p&nbsp;&lt; 0.001, ∗∗∗∗p&nbsp;&lt; 0.0001. See also <a id="crosref1720" href="#figs6">Figure&nbsp;S6</a> and <a id="crosref1725" href="#mmc3">Table&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/5fe1d88f-8716-46db-b896-bc785e89c005/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2022.04.011/asset/9067d552-f77b-43a1-b294-d8206d0f6acf/main.assets/figs6.jpg" height="911" width="753" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/5fe1d88f-8716-46db-b896-bc785e89c005/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Generation of <i>Zng1</i> mutant mice, related to <a id="crosref1780" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0585" role="paragraph">(A) Schematic of mouse <i>Zng1</i> locus with gRNA shown by red bar in first exon.</div><div id="fspara0590" role="paragraph">(B) RT-qPCR of <i>Zng1</i> transcript in WT and <i>Zng1</i> mutant kidneys (n&nbsp;= 4 mice/genotype). Normalized to <i>Actb</i> expression.</div><div id="fspara0595" role="paragraph">(C) Normalized peak area of ZNG1 peptides in WT and <i>Zng1</i><sup><i>−/−</i></sup> tissue (brain) measured using data-independent acquisition mass spectrometry proteomics.</div><div id="fspara0600" role="paragraph">(D and E) Breeding patterns of male and female mice resulting from <i>Zng1</i> heterozygous crosses. Number of animals indicated in white text.</div><div id="fspara0605" role="paragraph">(F) Five- to seven-week-old male or female mice were placed on a control diet for the course of 6&nbsp;weeks and percentage of weight gain measured (n&nbsp;= 5–8 mice/genotype/sex).</div><div id="fspara0610" role="paragraph">(G) Five- to seven-week-old female mice were placed on a control or low Zn diet for the course of 6&nbsp;weeks before assessment of tissue Zn levels (n&nbsp;= 6–8 mice/genotype/sex).</div><div id="fspara0615" role="paragraph">(H) Summary network of ATPIF1 upstream regulatory connections identified by IPA. Analysis was based on all differentially abundant proteins (p value ≤ 0.05) from proteomics data comparing renal protein content from WT and <i>Zng1</i> mutant mice (<a id="crosref1785" href="#fig6">Figure&nbsp;6</a>C).</div><div id="fspara0620" role="paragraph">(I) RT-qPCR analysis of transcripts encoding for most differentially abundant proteins in <i>Zng1</i> mutant mice (<a id="crosref1790" href="#fig6">Figure&nbsp;6</a>C; <a id="crosref1795" href="#mmc3">Table&nbsp;S3</a>). Normalized to expression of <i>Ppia</i>, <i>Pgam</i>, and <i>18S</i> genes. For (B), (C), and (I), RNA or protein was isolated from the tissue of 11- to 13-week-old female mice that were maintained on a low Zn diet for 5&nbsp;weeks. Data are represented as mean ± SEM. Data in (B), (C), and (I)&nbsp;analyzed by Student’s t test. Data in (D)&nbsp;and (E)&nbsp;analyzed by chi-square goodness-of-fit test. Data in (F)&nbsp;and (G)&nbsp;analyzed by two-way ANOVA with Sidak’s post-test. In (F)&nbsp;two-way ANOVA was run for each sex independently. Significance denoted as: <sup>∗</sup> p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup> p&nbsp;&lt;&nbsp;0.01, <sup>∗∗∗</sup> p&nbsp;&lt;&nbsp;0.001, <sup>∗∗∗∗</sup> p&nbsp;&lt;&nbsp;0.0001.</div></div></div></div></figcaption></figure></div><div id="p0155" role="paragraph">Due to the importance of METAP1 for maintaining cellular proteostasis (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Hu et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Hu, X. ∙ Addlagatta, A. ∙ Lu, J. ...</div><div><strong>Elucidation of the function of type 1 human methionine aminopeptidase during cell cycle progression</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:18148-18153</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0608389103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33845310513" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17114291/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0608389103&amp;pmid=17114291" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; Jonckheere et&nbsp;al., 2018), the impact of <i>Zng1</i> deficiency on the global proteome in Zn-deplete conditions was evaluated (<a id="crosref1000" href="#fig6">Figures&nbsp;6</a>C and 6D). Kidneys were selected for this analysis because ZNG1 proteins are highly expressed in human renal tissue (<a id="crosref1005" href="#figs1">Figure&nbsp;S1</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref96-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-2" aria-controls="sref96-2">Uhlén et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref96-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Uhlén, M. ∙ Fagerberg, L. ∙ Hallström, B.M. ...</div><div><strong>Proteomics. Tissue-based map of the human proteome</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1260419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260419&amp;pmid=25613900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sixty-three differentially abundant proteins were identified in the <i>Zng1</i> mutant compared with the WT control (±2-fold cutoff; <a id="crosref1015" href="#fig6">Figure&nbsp;6</a>C; <a id="crosref1020" href="#mmc3">Table&nbsp;S3</a>). Analysis of predicted localizations of these proteins using the UniProt database (<span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">UniProt Consortium, 2021</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>UniProt Consortium</div><div><strong>UniProt: the universal protein knowledgebase in 2021</strong></div><div><em>Nucleic Acids Res.</em> 2021; <strong>49</strong>:D480-D489</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa1100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85099417502" target="_blank" aria-label="Open Scopus in new tab">Scopus (2385)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33237286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa1100&amp;pmid=33237286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) suggested that a large number putatively localize to the mitochondria, particularly those with decreased abundance (<a id="crosref1030" href="#fig6">Figures&nbsp;6</a>C and 6D). Ingenuity pathway analysis (IPA) was performed to predict cellular processes affected by the loss of <i>Zng1</i> and revealed that numerous proteins play roles in canonical mitochondrial pathways (<a id="crosref1035" href="#fig6">Figure&nbsp;6</a>D) (<span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Krämer et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Krämer, A. ∙ Green, J. ∙ Pollard, Jr., J. ...</div><div><strong>Causal analysis approaches in ingenuity pathway analysis</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:523-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt703" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84894571860" target="_blank" aria-label="Open Scopus in new tab">Scopus (2850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24336805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt703&amp;pmid=24336805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Notably, IPA also indicated that Zn-starved <i>Zng1</i> animals experience mitochondrial dysfunction, further suggesting negative effects on mitochondrial integrity. Although most of the mitochondrial proteins were decreased in abundance in mutant animals (<a id="crosref1045" href="#fig6">Figures&nbsp;6</a>C and 6D), some proteins in this group were significantly elevated in <i>Zng1</i><sup><i>−/−</i></sup> mice, including the mitochondrial ATPase inhibitory factor 1 (IF1, ATPIF1,&nbsp;+7.2-fold) (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">García-Bermúdez and Cuezva, 2016</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>García-Bermúdez, J. ∙ Cuezva, J.M.</div><div><strong>The ATPase inhibitory Factor 1 (IF1): a master regulator of energy metabolism and of cell survival</strong></div><div><em>Biochim. Biophys. Acta.</em> 2016; <strong>1857</strong>:1167-1182</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbabio.2016.02.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84961219079" target="_blank" aria-label="Open Scopus in new tab">Scopus (86)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26876430/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbabio.2016.02.004&amp;pmid=26876430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ATPIF1 is a main regulator of ATP synthase, and blocks both ATP synthesis and hydrolysis activities (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Chen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Chen, W.W. ∙ Birsoy, K. ∙ Mihaylova, M.M. ...</div><div><strong>Inhibition of ATPIF1 ameliorates severe mitochondrial respiratory chain dysfunction in mammalian cells</strong></div><div><em>Cell Rep.</em> 2014; <strong>7</strong>:27-34</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2014.02.046&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2014.02.046&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24685140/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2014.02.046&amp;pmid=24685140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Subsequent IPA upstream regulator analysis (URA) (<span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Krämer et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Krämer, A. ∙ Green, J. ∙ Pollard, Jr., J. ...</div><div><strong>Causal analysis approaches in ingenuity pathway analysis</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:523-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt703" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84894571860" target="_blank" aria-label="Open Scopus in new tab">Scopus (2850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24336805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt703&amp;pmid=24336805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) identified ATPIF as a main regulatory node (p value of overlap&nbsp;= 2.4E-33), with experimental linkage to a subset of (mitochondrial) proteins identified as being negatively affected by loss of ZNG1 (<a id="crosref1065" href="#figs6">Figure&nbsp;S6</a>H). To evaluate whether changes in protein abundance were solely a result of altered protein processing, transcriptomic analysis of a subset of the most differentially abundant proteins was performed. These data revealed modest alterations in transcript levels, with some changes reaching statistical significance, indicating effects of <i>Zng1</i> deficiency at both the transcript and protein level (<a id="crosref1070" href="#figs6">Figure&nbsp;S6</a>I). Together, these results suggest that ZNG1 is important for maintaining mitochondrial function and organismal homeostasis during Zn starvation.</div></section><section id="sec-2-7"><h3>ZNG1 promotes cellular proliferation and respiration in Zn-deplete conditions</h3><div id="p0160" role="paragraph">Given that <i>Zng1</i> mutant mice displayed signs of impaired Zn homeostasis as well as disruption of mitochondrial function, a cell culture model was generated to further characterize the role of ZNG1. Murine proximal tubular epithelial cells (TKPTS) were selected as a model, since mitochondrial dysfunction was detected in murine <i>Zng1</i><sup><i>−/−</i></sup> renal tissue and TKPTS cell lines express both ZNG1 and METAP1 (<a id="crosref1075" href="#fig7">Figures&nbsp;7</a>A, 7B, and <a id="crosref1080" href="#figs7">S7</a>A–S7C). <i>Zng1</i> mutant cells (<a id="crosref1085" href="#figs7">Figure&nbsp;S7</a>A) were generated using CRISPR-Cas9, and decreased mRNA expression (<a id="crosref1090" href="#fig7">Figures&nbsp;7</a>B and <a id="crosref1095" href="#figs7">S7</a>D) and protein abundance (<a id="crosref1100" href="#figs7">Figure&nbsp;S7</a>C) of ZNG1 were confirmed. The expression of <i>Zng1</i> in TKPTS cells was dependent on Zn availability (<a id="crosref1105" href="#fig7">Figure&nbsp;7</a>A). Although <i>Zng1</i><sup><i>−/−</i></sup> and WT cells grew at similar rates in untreated media, growth in the presence of Zn chelator tris(2-pyridylmethyl)amine (TPA) significantly slowed proliferation of the <i>Zng1</i> mutant cells compared with WT cells, as measured by increased retention of CellTrace dye (<a id="crosref1110" href="#fig7">Figures&nbsp;7</a>C, <a id="crosref1115" href="#figs7">S7</a>E, and S7F). Of note, exposure of WT TKPTS cells to the METAP1/2 inhibitor Bengamide B similarly reduced cellular proliferation, thereby phenocopying the growth impairment observed in <i>Zng1</i> mutant cells in Zn-deplete conditions (<a id="crosref1120" href="#figs7">Figure&nbsp;S7</a>G). Because ZNG1 supports the proper function of METAP1 <i>in&nbsp;vitro</i> (<a id="crosref1125" href="#fig4">Figure&nbsp;4</a>), we postulated that <i>Zng1</i>-deficient cells would display increased sensitivity to METAP inhibition. Due to the overlapping substrate specificity of METAP1 and METAP2, TKPTS cells were exposed to the METAP2-specific inhibitor TNP-470. This strategy increases cellular reliance on METAP1, thereby permitting the evaluation of ZNG1-dependent METAP1 activity. Indeed, <i>Zng1</i><sup><i>−/−</i></sup> cells exhibited slowed proliferation relative to WT cells when exposed to TNP-470 (<a id="crosref1130" href="#fig7">Figures&nbsp;7</a>D, <a id="crosref1135" href="#figs7">S7</a>H, and S7I), and complementation with full-length <i>Zng1</i> significantly enhanced the proliferation of <i>Zng1</i> mutant cells (<a id="crosref1140" href="#figs7">Figure&nbsp;S7</a>J). Notably, decreased proliferation and increased sensitivity to METAP inhibition have been reported in METAP1-deficient HAP1 cells (<span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Jonckheere et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Jonckheere, V. ∙ Fijałkowska, D. ∙ Van Damme, P.</div><div><strong>Omics assisted N-terminal Proteoform and protein expression profiling on methionine aminopeptidase 1 (MetAP1) deletion</strong></div><div><em>Mol. Cell. Proteomics.</em> 2018; <strong>17</strong>:694-708</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.RA117.000360&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.RA117.000360&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85045027596" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29317475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.RA117.000360&amp;pmid=29317475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which is consistent with the possibility that defects observed in <i>Zng1</i><sup><i>−/−</i></sup> cells may in part be mediated by dysregulated METAP1 activity.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/488964cf-621c-45de-99e5-aa647e316428/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2022.04.011/asset/bda76e41-88a6-4732-abd8-abcebd1a9f98/main.assets/gr7.jpg" height="843" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/488964cf-621c-45de-99e5-aa647e316428/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">ZNG1 regulates cellular respiration through stimulation of METAP activity</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0285" role="paragraph">(A) Expression of <i>Zng1</i> in TKPTS cells treated with TPA.</div><div id="fspara0290" role="paragraph">(B) <i>Zng1</i> transcript levels in untreated WT and <i>Zng1</i> mutant cells.</div><div id="fspara0295" role="paragraph">(C and D) Cell proliferation quantified by CellTrace median fluorescence intensity (MFI) in Zn-deplete conditions (C)&nbsp;and following treatment with METAP2 inhibitor TNP-470 (D)&nbsp;using flow cytometry.</div><div id="fspara0300" role="paragraph">(E) Ratio of processed 14-3-3γ protein levels in vehicle or TPA-treated <i>Zng1</i><sup><i>−/−</i></sup><i>/</i> WT cells (n&nbsp;= 4 genotype/treatment).</div><div id="fspara0305" role="paragraph">(F) Representative TEM images of WT and <i>Zng1</i> mutant cells grown in Zn-deplete conditions (Scale bars, 500&nbsp;nm). Red box (top) outlines region of higher magnification (bottom).</div><div id="fspara0310" role="paragraph">(G and H) Non-linear regression analysis of the size distribution of mitochondria (G)&nbsp;and mean signal intensity (H)&nbsp;in WT and <i>Zng1</i> mutant cells treated with TPA (WT, n&nbsp;= 112 mitochondria; <i>Zng1</i><sup><i>−/−</i></sup> n&nbsp;= 101 mitochondria).</div><div id="fspara0315" role="paragraph">(I) ATP levels in cell lysates from untreated WT and <i>Zng1</i> mutant cells.</div><div id="fspara0320" role="paragraph">(J) Mitochondrial membrane potential (ΔΨ<sub>m</sub>) in untreated and TPA-treated cells.</div><div id="fspara0325" role="paragraph">(K) Production of mitochondrial superoxide in cells treated with TNP-470 was quantified by flow cytometry.</div><div id="fspara0330" role="paragraph">(L and M) Oxygen consumption rates (OCR) by cells treated with vehicle or TPA. OCR was normalized by cell number. Data in (A), (B), (H), and (I)&nbsp;analyzed by Student’s t test. Data in (E)&nbsp;analyzed by Welch’s t test. Data in (G)&nbsp;analyzed by non-linear regression goodness-of-fit test. Data in (C), (D), (J), (K), and (M)&nbsp;analyzed by two-way ANOVA with Tukey’s multiple comparison test.</div><div id="fspara0335" role="paragraph">Data are represented as mean ± SEM. Significance denoted as: ∗p&nbsp;&lt; 0.05, ∗∗p &lt; 0.01, ∗∗∗p&nbsp;&lt; 0.001, ∗∗∗∗p&nbsp;&lt; 0.0001. See also <a id="crosref1730" href="#figs7">Figure&nbsp;S7</a> and <a id="crosref1735" href="#mmc3">Table&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.04.011/asset/f6de66a7-4e04-4ec0-96c2-91c0130370a6/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2022.04.011/asset/5537e573-363c-4ea3-abec-f3108489c889/main.assets/figs7.jpg" height="941" width="715" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/f6de66a7-4e04-4ec0-96c2-91c0130370a6/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Generation and validation of <i>Zng1</i> mutant alleles in TKPTS cells, related to <a id="crosref1800" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0630" role="paragraph">(A) Sequence alignment of WT and <i>Zng1</i> mutant alleles generated by CRISPR-Cas9. Two independent mutant clones were isolated, clone 1 (Δ2/Δ2) and clone 2 (Δ2/Δ7), with deletion alleles that result in frameshift mutations.</div><div id="fspara0635" role="paragraph">(B) Immunoblot for METAP1 in WT and <i>Zng1</i><sup><i>−/−</i></sup> TKPTS cells.</div><div id="fspara0640" role="paragraph">(C) Immunoblot for ZNG1 and tubulin control in WT, <i>Zng1</i><sup><i>−/−</i></sup>, and <i>Zng1</i><sup><i>−/−</i></sup>&nbsp;+ <i>Zng1</i> TKPTS cells.</div><div id="fspara0645" role="paragraph">(D) <i>Zng1</i> transcript levels in WT, <i>Zng1</i> clone 1, and <i>Zng1</i> clone 2 TKPTS cells.</div><div id="fspara0650" role="paragraph">(E and F) CellTrace fluorescence levels in WT, <i>Zng1</i> clone 1, and <i>Zng1</i> clone 2 TKPTS cells treated with TPA (E). FACS histogram associated with quantification of indicated panel in <a id="crosref1805" href="#fig7">Figure&nbsp;7</a>C (F).</div><div id="fspara0655" role="paragraph">(G) Proliferation measured by CellTrace fluorescence of untreated and Bengamide B-treated WT TKPTS cells.</div><div id="fspara0660" role="paragraph">(H and I) CellTrace fluorescence levels in WT, <i>Zng1</i> clone 1, and <i>Zng1</i> clone 2 TKPTS cells treated with TNP-470 (H). Histogram associated with quantification of indicated panel in <a id="crosref1810" href="#fig7">Figure&nbsp;7</a>D (I).</div><div id="fspara0665" role="paragraph">(J) Proliferation measured by CellTrace fluorescence of untreated and TNP-470-treated WT, <i>Zng1</i><sup><i>−/−</i></sup>, and <i>Zng1</i><sup><i>−/−</i></sup><i>+ Zng1</i> TKPTS cells.</div><div id="fspara0670" role="paragraph">(K and L) Proteomics was performed on WT and <i>Zng1</i><sup><i>−/−</i></sup> TKPTS cells grown in the presence of TPA or Bengamide B. The abundance of iMet-containing and processed N termini of 14-3-3γ (MVDREQLVQK and VDREQLVQK, respectively) in WT and mutant cells upon TPA treatment was determined to assess the percentage of unprocessed protein (K). Changes to canonical pathways (as determined by IPA) comparing protein signatures from TKPTS cells with impaired <i>Zng1</i> and WT cells upon inhibition of METAP with Bengamide B (L). Depicted are pathways that meet the following criteria for one of the two conditions: significance cutoff of p value(−log<sub>10</sub>) ≥1.3, |<i>Z</i> score| ≥ 2. Pathways with the highest total score (combined p value and <i>Z</i> score) are on top, and color-coding reflects <i>Z</i> score (positive values: activation; negative values: inhibition). Gray boxes denote pathways for which no <i>Z</i> score was determined due to a limited number (&lt;4) of proteins affected in a specific pathway.</div><div id="fspara0675" role="paragraph">(M) Mitochondrial roundness of WT and <i>Zng1</i> mutant TKPTS cells grown in Zn-deplete conditions, as quantified from TEM.</div><div id="fspara0680" role="paragraph">(N) TMRM fluorescence in WT, <i>Zng1</i><sup><i>−/−</i></sup> clone 1, and <i>Zng1</i><sup><i>−/−</i></sup> clone 2 TKPTS cells.</div><div id="fspara0685" role="paragraph">(O) MitoTracker levels in WT, <i>Zng1</i> clone 1, and <i>Zng1</i> clone 2 TKPTS cells.</div><div id="fspara0690" role="paragraph">(P and Q) MitoSOX levels in WT, <i>Zng1</i><sup><i>−/−</i></sup> clone 1, and <i>Zng1</i><sup><i>−/−</i></sup> clone 2 TKPTS cells treated with TNP-470. (N) FACS histogram associated with quantification of indicated panel in <a id="crosref1815" href="#fig7">Figure&nbsp;7</a>.</div><div id="fspara0695" role="paragraph">(R) Mitochondrial superoxide measured by MitoSOX levels in untreated and TNP-470-treated WT, <i>Zng1</i><sup><i>−/−</i></sup>, and <i>Zng1</i><sup><i>−/−</i></sup><i>+ Zng1</i> TKPTS cells.</div><div id="fspara0700" role="paragraph">(S) Confocal composite micrographs of WT and <i>Zng1</i><sup><i>−/−</i></sup> cells illustrating ATPIF1 distribution (magenta), mitochondria (cyan), and DAPI (gray). Scale bars, 20&nbsp;μm. Data are represented as mean ± SEM. Data in (D)&nbsp;and (N)&nbsp;were analyzed by one-way ANOVA. Data in (K)&nbsp;and (M)&nbsp;analyzed by Student’s t test. Data in (E), (G), (H), (J), (O), (Q), and (R)&nbsp;were analyzed by two-way ANOVA with Tukey’s multiple comparison test. Significance denoted as: <sup>∗</sup> p&nbsp;&lt;&nbsp;0.05, <sup>∗∗</sup> p&nbsp;&lt;&nbsp;0.01, <sup>∗∗∗</sup> p&nbsp;&lt;&nbsp;0.001, <sup>∗∗∗∗</sup>&nbsp;p&nbsp;&lt;&nbsp;0.0001.</div></div></div></div></figcaption></figure></div><div id="p0165" role="paragraph">To further assess a potential impairment of METAP1 function in the <i>Zng1</i><sup><i>−/−</i></sup> background, the abundance and iMet-status of the known METAP-substrate 14-3-3γ were probed though proteomic profiling (<span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Towbin et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Towbin, H. ∙ Bair, K.W. ∙ DeCaprio, J.A. ...</div><div><strong>Proteomics-based target identification: bengamides as a new class of methionine aminopeptidase inhibitors</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2003; <strong>278</strong>:52964-52971</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M309039200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M309039200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-9144222006" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14534293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M309039200&amp;pmid=14534293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Analysis of WT and <i>Zng1</i><sup><i>−/−</i></sup> cells grown in Zn-replete and Zn-limiting (TPA-treated) conditions revealed reduced levels of 14-3-3γ in <i>Zng1</i> mutant cells (<a id="crosref1155" href="#fig7">Figure&nbsp;7</a>E). We hypothesized that this reduction was a result of impaired METAP1-dependent iMet removal, and accordingly detected a trend toward higher iMet retention at the 14-3-3γ N terminus in TPA-treated <i>Zng1</i><sup><i>−/−</i></sup> mutant cells compared with WT cells (<a id="crosref1160" href="#figs7">Figure&nbsp;S7</a>K). Taken together, these data are indicative of a reduction of METAP1 activity in <i>Zng1</i><sup><i>−/−</i></sup> cells. To further test whether the proteomic changes in the <i>Zng1</i> mutant resemble those of METAP-deficient cells, proteomic analysis was performed on WT cells treated with Bengamide B and the results compared with those of the TPA-treated <i>Zng1</i> mutant cells (<a id="crosref1165" href="#mmc4">Table&nbsp;S4</a>). IPA was performed and significantly affected pathways in both datasets determined. Subsequent comparison of the two analyses showed remarkable overlap of the pathways affected by a loss of <i>Zng1</i> and pharmacological inhibition of METAP (<a id="crosref1170" href="#figs7">Figure&nbsp;S7</a>L). Notably, pathways affected in both datasets included oxidative phosphorylation and TCA cycle (<a id="crosref1175" href="#figs7">Figure&nbsp;S7</a>L), as well as mitochondrial dysfunction (p value[−log<sub>10</sub>] of 3 and 10.5 for <i>Zng1</i><sup><i>−/−</i></sup> and Bengamide B treated, respectively). Collectively, these findings demonstrate that mutation of <i>Zng1</i> phenocopies a METAP-deficient system, and that mitochondrial processes are affected by loss of ZNG1.</div><div id="p0170" role="paragraph">The reduced proliferation of <i>Zng1</i> mutant cells (<a id="crosref1180" href="#fig7">Figures&nbsp;7</a>C and 7D), in combination with proteomic signatures of mitochondrial distress in <i>Zng1</i><sup><i>−/−</i></sup> cell lines and mice (<a id="crosref1185" href="#fig6">Figures&nbsp;6</a>C and <a id="crosref1190" href="#figs7">S7</a>L), are consistent with a restriction in mitochondrial energy production. Since mitochondrial morphology reflects function (<span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Picard et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Picard, M. ∙ Shirihai, O.S. ∙ Gentil, B.J. ...</div><div><strong>Mitochondrial morphology transitions and functions: implications for retrograde signaling?</strong></div><div><em>Am. J. Physiol. Regul. Integr. Comp. Physiol.</em> 2013; <strong>304</strong>:R393-R406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/ajpregu.00584.2012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84877105774" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23364527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fajpregu.00584.2012&amp;pmid=23364527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Zn-starved WT and <i>Zng1</i> mutant cells were imaged using transmission electron microscopy (TEM). <i>Zng1</i> mutant cells were morphologically distinct (<a id="crosref1200" href="#fig7">Figure&nbsp;7</a>F) and displayed a significant change in mitochondrial cross-sectional area (<a id="crosref1205" href="#fig7">Figure&nbsp;7</a>G) in the absence of alterations of mitochondrial roundness, suggesting that mitochondria from <i>Zng1</i><sup><i>−/−</i></sup> cells are swollen (<a id="crosref1210" href="#figs7">Figure&nbsp;S7</a>M). Additionally, the mitochondria of <i>Zng1</i><sup><i>−/−</i></sup> cells displayed increased electron density (<a id="crosref1215" href="#fig7">Figure&nbsp;7</a>H). Similar alterations in mitochondrial morphology have previously been associated with the uncoupling of the mitochondrial membrane potential from ATP synthesis (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Hackenbrock et&nbsp;al., 1971</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hackenbrock, C.R. ∙ Rehn, T.G. ∙ Weinbach, E.C. ...</div><div><strong>Oxidative phosphorylation and ultrastructural transformation in mitochondria in the intact ascites tumor cell</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1971; <strong>51</strong>:123-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.51.1.123" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0015144531" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5111873/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.51.1.123&amp;pmid=5111873" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Huet et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Huet, D. ∙ Rajendran, E. ∙ van Dooren, G.G. ...</div><div><strong>Identification of cryptic subunits from an apicomplexan ATP synthase</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e38097</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.38097" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85058422356" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30204085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.38097&amp;pmid=30204085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Weissert et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Weissert, V. ∙ Rieger, B. ∙ Morris, S. ...</div><div><strong>Inhibition of the mitochondrial ATPase function by IF1 changes the spatiotemporal organization of ATP synthase</strong></div><div><em>Biochim. Biophys. Acta Bioenerg.</em> 2021; <strong>1862</strong>:148322</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbabio.2020.148322" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85096199594" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33065099/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbabio.2020.148322&amp;pmid=33065099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with this, cellular ATP levels quantified from the lysates of <i>Zng1</i><sup><i>−/−</i></sup> TKPTS cells were decreased compared with WT cells (<a id="crosref1235" href="#fig7">Figure&nbsp;7</a>I). Together, these data suggest that ZNG1 contributes to maintaining mitochondrial metabolic homeostasis, and that a loss of ZNG1 renders cells less capable of sustaining the energy necessary for growth during conditions of stress.</div><div id="p0175" role="paragraph">To assess broad changes in oxidative phosphorylation, mitochondrial membrane potential was quantified using tetramethylrhodamine methyl ester (TMRM) at a quenching concentration, where a decrease in fluorescence intensity correlates with an increase in membrane potential (<span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Monteith et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Monteith, A. ∙ Marszalec, W. ∙ Chan, P. ...</div><div><strong>Imaging of mitochondrial and non-mitochondrial responses in cultured rat hippocampal neurons exposed to micromolar concentrations of TMRM</strong></div><div><em>PLoS One.</em> 2013; <strong>8</strong>:e58059</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0058059" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84874583009" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23483968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0058059&amp;pmid=23483968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Scaduto and Grotyohann, 1999</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Scaduto, Jr., R.C. ∙ Grotyohann, L.W.</div><div><strong>Measurement of mitochondrial membrane potential using fluorescent rhodamine derivatives</strong></div><div><em>Biophys. J.</em> 1999; <strong>76</strong>:469-477</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0006-3495%2899%2977214-0&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0006-3495%2899%2977214-0&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0345120940" target="_blank" aria-label="Open Scopus in new tab">Scopus (965)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9876159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0006-3495%2899%2977214-0&amp;pmid=9876159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). TMRM fluorescence was decreased in <i>Zng1</i><sup><i>−/−</i></sup> cells compared with WT cells in conditions of Zn starvation, indicating that mitochondria within <i>Zng1</i> mutant cells are hyperpolarized (<a id="crosref1250" href="#fig7">Figures&nbsp;7</a>J and <a id="crosref1255" href="#figs7">S7</a>N), consistent with reduced ATP synthase activity. Mitochondrial superoxide (O<sub>2</sub><sup>−⋅</sup>) production, a byproduct of electron leakage during electron shuttling across the inner mitochondrial membrane (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Forman and Kennedy, 1974</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Forman, H.J. ∙ Kennedy, J.A.</div><div><strong>Role of superoxide radical in mitochondrial dehydrogenase reactions</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 1974; <strong>60</strong>:1044-1050</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0006-291X(74)90418-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4372996/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0006-291X%2874%2990418-5&amp;pmid=4372996" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Loschen et&nbsp;al., 1974</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Loschen, G. ∙ Azzi, A. ∙ Richter, C. ...</div><div><strong>Superoxide radicals as precursors of mitochondrial hydrogen peroxide</strong></div><div><em>FEBS Lett.</em> 1974; <strong>42</strong>:68-72</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0014-5793(74)80281-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0016148483" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4859511/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0014-5793%2874%2980281-4&amp;pmid=4859511" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), was quantified using the fluorescent dye, MitoSOX. Mitochondrial O<sub>2</sub><sup>−⋅</sup> levels in <i>Zng1</i> mutant cells were lower than in WT cells, and this phenotype is&nbsp;exacerbated upon METAP2 inhibition (<a id="crosref1270" href="#fig7">Figures&nbsp;7</a>K and <a id="crosref1275" href="#figs7">S7</a>O–S7R). These results suggest that electron shuttling is decreased in <i>Zng1</i> mutant cells despite hyperpolarization of the mitochondria.</div><div id="p0180" role="paragraph">To further interrogate the implications of loss of <i>Zng1</i> on oxidative phosphorylation, the oxygen consumption rate (OCR) of WT and <i>Zng1</i><sup><i>−/−</i></sup> cells grown in media alone or in the presence of TPA was quantified by extracellular flux analysis. Although <i>Zng1</i><sup><i>−/−</i></sup> cells exhibited a trend toward decreased basal and maximal respiration compared with WT cells, pre-treatment of mutant cells with TPA significantly impaired basal and maximal respiration, ATP production, and proton leak (<a id="crosref1280" href="#fig7">Figures&nbsp;7</a>L and 7M). Notably, <i>Zng1</i><sup><i>−/−</i></sup> cells showed accumulated ATPIF1 puncta colocalizing with mitochondrial structures (<a id="crosref1285" href="#figs7">Figure&nbsp;S7</a>S), consistent with proteomic analysis of <i>Zng1</i> mutant animals (<a id="crosref1290" href="#fig6">Figures&nbsp;6</a>C and 6D). These results support a model whereby loss of <i>Zng1</i> negatively effects METAP1 activity, particularly during Zn starvation. In turn, impaired MEATP1-dependent processing of the proteome likely disrupts mitochondrial homeostasis. Specifically, <i>Zng1</i> mutant cells display hyperpolarization of the mitochondrial membrane with decreased oxidative phosphorylation, consistent with ATPIF1-dependent inhibition of ATP synthase. Taken together, these findings demonstrate that ZNG1 activates METAP1 activity and plays a critical role in regulating mitochondrial function and energy metabolism during Zn starvation.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0185" role="paragraph">Zn is an essential structural and enzymatic cofactor for many proteins in all branches of life. Consequently, vertebrates have developed sophisticated systems for Zn uptake, intracellular distribution into organelles, and efflux to maintain organismal and cellular Zn homeostasis (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Eide, 2006</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Eide, D.J.</div><div><strong>Zinc transporters and the cellular trafficking of zinc</strong></div><div><em>Biochim. Biophys. Acta.</em> 2006; <strong>1763</strong>:711-722</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbamcr.2006.03.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33746929895" target="_blank" aria-label="Open Scopus in new tab">Scopus (666)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16675045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbamcr.2006.03.005&amp;pmid=16675045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although numerous aspects of Zn transport and trafficking have been well characterized, mechanisms for allocating Zn within the cell and to individual proteins remain largely unexplored. Distribution of this essential nutrient to “high-priority” (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Eide, 2006</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Eide, D.J.</div><div><strong>Zinc transporters and the cellular trafficking of zinc</strong></div><div><em>Biochim. Biophys. Acta.</em> 2006; <strong>1763</strong>:711-722</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbamcr.2006.03.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33746929895" target="_blank" aria-label="Open Scopus in new tab">Scopus (666)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16675045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbamcr.2006.03.005&amp;pmid=16675045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) proteins has been attributed to hypothetical Zn metallochaperones. However, functionally validated interactions with target proteins and <i>bona fide</i> Zn metallochaperone function have not been reported. In this study, evolutionarily divergent systems were used to establish a role for ZNG1 in regulating cellular Zn homeostasis, identifying METAP1 as a conserved client of ZNG1 and providing mechanistic insights into ZNG1-dependent METAP1 activation. We propose that GTP hydrolysis drives Zn transfer from ZNG1 into the METAP1 catalytic site, either through the energy provided by phosphodiester bond cleavage or a conformational change upon hydrolysis, thereby metalating and activating METAP1. The Zn affinities of ZNG1 and METAP1 are similar, suggesting that Zn transfer may have some element of kinetic control; alternatively, GDP modestly lowers the affinity of ZNG1 for Zn, facilitating capture by METAP1. In any case, the Zn- and GTP-dependent activation of METAP1 by ZNG1 is consistent with the assignment of ZNG1 as a metallochaperone.</div><div id="p0190" role="paragraph">The ZNG1-METAP1 interactions described in this study are likely to be more broadly conserved across eukaryotes. Consistent with our findings, this interaction was also suggested by a large-scale screen of the human interactome (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Huttlin et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Huttlin, E.L. ∙ Ting, L. ∙ Bruckner, R.J. ...</div><div><strong>The BioPlex network: a systematic exploration of the human interactome</strong></div><div><em>Cell.</em> 2015; <strong>162</strong>:425-440</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.cell.2015.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.cell.2015.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84937213168" target="_blank" aria-label="Open Scopus in new tab">Scopus (871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26186194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.06.043&amp;pmid=26186194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and identified by a recent study that investigated the ZNG1 ortholog, Zng1p, in <i>Saccharomyces cerevisiae</i> (<span class="dropBlock reference-citations"><a id="body-ref-optkIjaU1XKKm-2" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optkIjaU1XKKm-2" aria-controls="optkIjaU1XKKm-2">Pasquini et al., 2022</a><div class="dropBlock__holder" data-db-target-of="optkIjaU1XKKm-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Pasquini, M. ∙ Grosjean, N. ∙ Hixson, K.K. ...</div><div><strong>Zng1 is a GTP-dependent zinc transferase needed for activation of methionine aminopeptidase</strong></div><div><em>Cell Rep.</em> 2022; <strong>39</strong>:110834</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2022.110834&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2022.110834&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85130211918" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35584675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2022.110834&amp;pmid=35584675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The ZNG1-METAP1 interaction in vertebrates is mediated by a conserved N-terminal CPELVPI motif in ZNG1 and the C6H2 domain of METAP1. Our solution structure of the interaction domain reveals that the conserved features, including the hydrophobic binding pockets and the electrostatic surface of METAP1 required for ZNG1 binding, may well be shared among other ZNG1 clients or binding partners. Sequence analysis reveals that approximately half of all eukaryotic ZNG1 orthologs (<span class="dropBlock reference-citations"><a id="body-ref-sref14-4" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-4" aria-controls="sref14-4">Edmonds et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref14-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Edmonds, K.A. ∙ Jordan, M.R. ∙ Giedroc, D.P.</div><div><strong>COG0523 proteins: a functionally diverse family of transition metal-regulated G3E P-loop GTP hydrolases from bacteria to man</strong></div><div><em>Metallomics.</em> 2021; <strong>13</strong>:mfab046</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/mtomcs/mfab046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85113538279" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34302342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmtomcs%2Fmfab046&amp;pmid=34302342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) share an N-terminal “[D/E]<sub>n</sub>ψPxLVp” motif in which a series of negatively charged residues [E/D]<sub>n</sub> are followed by a non-aromatic hydrophobic residue (ψ) and a conserved proline, leucine, and valine (<a id="crosref1315" href="#fig3">Figure&nbsp;3</a>F). Our structure is consistent with this revised motif; it likely captures key features that describe an evolutionarily conserved ZNG1-METAP1 interaction. Indeed, the N terminus of <i>S.&nbsp;cerevisiae</i> Zng1p lacks the exact vertebrate CPELVPI motif but matches the more general “[D/E]<sub>n</sub>ψPxLVp” pattern. Although the sequence identity of the yeast and mouse Map1p/METAP1 N-terminal domains is less than 40% (<a id="crosref1320" href="#figs2">Figure&nbsp;S2</a>E), modeling of the Zng1p-Map1p interaction predicts a nearly identical fold and similar peptide interface (<a id="crosref1325" href="#figs3">Figure&nbsp;S3</a>I), where positive charges surround a hydrophobic groove and engage negatively charged ZNG1 peptides in both complexes. These findings suggest that our structure of the ZNG1-METAP1 interaction domain may be representative of how evolutionarily distant ZNG1 orthologs interact with their targets.</div><div id="p0195" role="paragraph">The structure and revised ZNG1 motif also provide insight into what proteins may compete with ZNG1 to perform its metallochaperone function. Although many proteins contain a more general PxLxP sequence, very few match the “[D/E]<sub>n</sub>ψPxLVp” pattern identified here. One notable candidate is in the C terminus of the β-subunit of the nascent polypeptide-associated complex (NAC), which protrudes from the ribosome exit tunnel (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Gamerdinger et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Gamerdinger, M. ∙ Kobayashi, K. ∙ Wallisch, A. ...</div><div><strong>Early scanning of nascent polypeptides inside the ribosomal tunnel by NAC</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:996-1006.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.molcel.2019.06.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.molcel.2019.06.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85071433346" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31377116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.06.030&amp;pmid=31377116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Lin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Lin, Z. ∙ Gasic, I. ∙ Chandrasekaran, V. ...</div><div><strong>TTC5 mediates autoregulation of tubulin via mRNA degradation</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:100-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaz4352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85077476517" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31727855/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaz4352&amp;pmid=31727855" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This motif likely explains how the C6H2 domain localizes METAP1 to the ribosome in the presence of NAC (<span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Nyathi and Pool, 2015</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Nyathi, Y. ∙ Pool, M.R.</div><div><strong>Analysis of the interplay of protein biogenesis factors at the ribosome exit site reveals new role for NAC</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2015; <strong>210</strong>:287-301</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201410086" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84940445222" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26195668/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201410086&amp;pmid=26195668" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Vetro and Chang, 2002</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Vetro, J.A. ∙ Chang, Y.H.</div><div><strong>Yeast methionine aminopeptidase type 1 is ribosome-associated and requires its N-terminal zinc finger domain for normal function in&nbsp;vivo</strong></div><div><em>J.&nbsp;Cell. Biochem.</em> 2002; <strong>85</strong>:678-688</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcb.10161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0036235240" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11968008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcb.10161&amp;pmid=11968008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The high affinity of ZNG1 for METAP1 may enable it to outcompete NAC under conditions of low cellular Zn.</div><div id="p0200" role="paragraph">Our zebrafish and mouse models of <i>Zng1</i> deficiency link ZNG1 to mitochondrial function, organismal development, and Zn homeostasis. The connection between METAP1 and mitochondrial integrity is not without precedent. A previous study that used transcriptomic and proteomic approaches to characterize the effects of METAP1 loss in human HAP1 cells found changes to mitochondrial transcript and protein levels as a result of <i>METAP1</i> mutation (<span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Jonckheere et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Jonckheere, V. ∙ Fijałkowska, D. ∙ Van Damme, P.</div><div><strong>Omics assisted N-terminal Proteoform and protein expression profiling on methionine aminopeptidase 1 (MetAP1) deletion</strong></div><div><em>Mol. Cell. Proteomics.</em> 2018; <strong>17</strong>:694-708</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.RA117.000360&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.RA117.000360&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85045027596" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29317475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.RA117.000360&amp;pmid=29317475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Such data offer a potential explanation as to how the mutation of <i>Zng1</i> in TKPTS cells results in mitochondrial dysfunction and attenuated cellular respiration, and ultimately explain the detrimental effects observed in <i>Zng1</i>-deficient animals. Collectively, these findings highlight the importance of ZNG1 for survival during Zn limitation, likely by ensuring functional METAP1-dependent NME and maintaining animal energy balance during such conditions.</div><div id="p0205" role="paragraph">In conclusion, we established the vertebrate COG0523 ZNG1 protein family as a major feature of the vertebrate adaptive response to severe Zn deprivation. Although these data show a clear physical interaction between ZNG1 and the metalloprotein METAP1, we also provide structural insights into how ZNG1 could interact with other targets. These findings reveal an evolutionary conserved strategy to ensure targeted Zn distribution to a network of high-priority client proteins during conditions of severe Zn starvation.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0210" role="paragraph">The current study establishes biochemical and biological details of ZNG1 activation of METAP1. Nevertheless, additional evidence of direct metal transfer from ZNG1 to METAP1 is needed; further, tools are not yet available to probe the METAP1 metalation status <i>in&nbsp;vivo</i> as a function of bioavailable Zn. Therefore, this study relies heavily on the functional implications of <i>Zng1</i> deficiency in different model systems, all of which strongly suggest a Zn-dependent impairment of METAP1-function in <i>Zng1</i> mutants. Overall, these limitations highlight a future demand for more detailed exploration of both the biological and biophysical underpinnings of ZNG1 activity.</div><div id="p0215" role="paragraph">The catalytic GTPase domain is conserved in all COG0523 proteins; thus, the contribution of ZNG1 GTPase activity to its cellular function must be further characterized. Although METAP1 activates ZNG1 GTPase activity, hydrolysis is surprisingly low <i>in&nbsp;vitro</i> and may suggest additional unidentified accessory proteins. Additionally, the exact roles of nucleotide binding/hydrolysis remain to be explored, as well as the atomic details of the interaction between the catalytic domains of ZNG1 and METAP1, and whether the low affinity Zn-binding sites in both proteins play any role in Zn transfer. Moreover, due to the likelihood that ZNG1 proteins support a variety of critical Zn-dependent proteins and essential cellular functions, future studies are required to determine the entirety of the ZNG1 interactome while elucidating what we anticipate will be the common features of a ZNG1-client protein interaction.</div><div id="p0220" role="paragraph">Deconvoluting potential pleiotropic effects resulting from loss of ZNG1 presents an obstacle for the <i>in&nbsp;vivo</i> characterization of the ZNG1-METAP1 interaction. Data generated from the yeast-two-hybrid screen in this study suggest that ZNG1 could mobilize Zn to support the function of multiple protein clients in the cell. Consequently, it is possible that the effects observed in our cell culture and animal models upon mutation of <i>Zng1</i> are not solely driven by impairment of METAP1. Rather, failure to metalate additional Zn-dependent ZNG1-targets could compound the effects resulting from decreased METAP1 activity after loss of ZNG1. Consequently, although this study describes the downstream effects of <i>Zng1</i> mutation (and METAP1 inhibition) on mitochondrial function in detail, the specific mechanisms by which ZNG1 mediates these effects are not yet known.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0225" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>anti-CBWD1</td><td>Sigma</td><td>CAT#HPA042813, RRID:AB_2678181</td></tr><tr><td>anti-OxPhos Complex V inhibitor mouse IgG1 clone 5E2</td><td>Invitrogen</td><td>CAT# A21355, RRID:AB_1501857</td></tr><tr><td>Polyclonal METAP1 antibody</td><td>Invitrogen</td><td>CAT#PAS-58202</td></tr><tr><td>Goat anti-Mouse IgG alexa 555</td><td>Invitrogen</td><td>CAT# A21424</td></tr><tr><td>IRDye 800CW Donkey anti-Rabbit</td><td>Li-Cor</td><td>CAT#926-32213, RRID:AB_621848</td></tr><tr><td>IRDye 680LT Goat anti-Mouse</td><td>Li-Cor</td><td>CAT#926-68020</td></tr><tr class="rowsep"><td>anti-Tubulin</td><td>Cell signaling</td><td>CAT#3873S</td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td>E.&nbsp;coli DH5a</td><td>Gift from Dr. Olaf Schneewind</td><td>N/A</td></tr><tr class="rowsep"><td>E.&nbsp;coli BL21</td><td>Gift from Dr. Olaf Schneewind</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Fmoc-Amino acids</td><td>Vivitide</td><td>CAT#<br>CT01-001<br>CT04-001<br>CT03-001<br>CT4-001<br>CT07-001<br>CT11-001<br>CT10-001<br>CT17-001<br>CT15-001<br>CT20-001<br>CT19-001</td></tr><tr><td><i>N</i>,<i>N</i>'-Diisopropylcarbodiimide (DIC)</td><td>Chem-Impex International</td><td>CAT#00110</td></tr><tr><td>Ethyl (hydroxyimino)cyanoacetate<br>Oxyma</td><td>Aapptec</td><td>CAT#CXZ021</td></tr><tr><td>Trifluoroacetic acid</td><td>Chem-Impex International</td><td>CAT#00289</td></tr><tr><td>n-Methyl, 2-Pyrrolidone (NMP)</td><td>Pharmco</td><td>CAT#253000ACS</td></tr><tr><td>Acetonitrile</td><td>Fisher Chemical</td><td>CAT#UN1648</td></tr><tr><td>Methylene Chloride (DCM)</td><td>Fisher Chemical</td><td>CAT#UN1593</td></tr><tr><td>Cell Trace Violet</td><td>Invitrogen</td><td>CAT#C34557</td></tr><tr><td>MitoSOX</td><td>Thermofisher</td><td>CAT#M36008</td></tr><tr><td>MitoTracker™ Deep Red FM</td><td>Invitrogen</td><td>CAT#M22426</td></tr><tr><td>TNP-470</td><td>Sigma</td><td>CAT#T1455-5MG</td></tr><tr><td>Bengamide B</td><td>Santa cruz</td><td>CAT#sc-397521A</td></tr><tr><td>N,N,N′,N′-Tetrakis(2-pyridylmethyl)ethylenediamine (TPEN)</td><td>Sigma</td><td>CAT#P4413</td></tr><tr><td>Tris(2-pyridylmethyl)amine</td><td>Sigma</td><td>CAT#723134</td></tr><tr><td>Tetramethylrhodamine methyl ester (TMRM)</td><td>Invitrogen</td><td>CAT#T668</td></tr><tr><td>DpnII</td><td>New England Biolabs</td><td>CAT#R0543S</td></tr><tr><td>Trifluoroacetic acid</td><td>Sigma</td><td>CAT#299537</td></tr><tr><td>IPTG</td><td>Roche</td><td>CAT#11411446001</td></tr><tr><td>Acetonitrile</td><td>Fisher</td><td>CAT#A998-4</td></tr><tr><td>Tri Reagent</td><td>Sigma</td><td>CAT#T9424</td></tr><tr><td>Amylose resin</td><td>New England Biolabs</td><td>CAT#E8021L</td></tr><tr><td>OPTIMA Nitric Acid</td><td>Fisher</td><td>CAT#A467-500</td></tr><tr><td>Ultratrace Hydrogen Peroxide</td><td>Sigma</td><td>CAT#95321</td></tr><tr><td>UltraPure Water</td><td>Invitrogen</td><td>CAT#10977-023</td></tr><tr><td>ARISTAR ICP Standard Mix</td><td>VWR</td><td>CAT#BDH82026-108</td></tr><tr><td>Sodium Cacodylate Trihydrate</td><td>Electron Microscopy Sciences</td><td>CAT#12310</td></tr><tr><td>Glutaraldehyde</td><td>Electron Microscopy Sciences</td><td>CAT#16310</td></tr><tr><td>Tannic Acid</td><td>Electron Microscopy Sciences</td><td>CAT#21700</td></tr><tr><td>Osmium Tetroxide</td><td>Electron Microscopy Sciences</td><td>CAT#19110</td></tr><tr><td>Uranyl Acetate</td><td>Electron Microscopy Sciences</td><td>CAT#22400</td></tr><tr><td>Propylene Oxide</td><td>Electron Microscopy Sciences</td><td>CAT#20411</td></tr><tr><td>Quetol 651</td><td>Electron Microscopy Sciences</td><td>CAT#20440</td></tr><tr><td>ERL4221</td><td>Electron Microscopy Sciences</td><td>CAT#15004</td></tr><tr><td>Nonenyl Succinic Anhydride</td><td>Electron Microscopy Sciences</td><td>CAT#19050</td></tr><tr><td>DER 736</td><td>Electron Microscopy Sciences</td><td>CAT#13000</td></tr><tr><td>Benzyldimethylamine</td><td>Electron Microscopy Sciences</td><td>CAT#11400</td></tr><tr><td>Lead Nitrate</td><td>Electron Microscopy Sciences</td><td>CAT#17900</td></tr><tr><td>Sodium Citrate</td><td>Electron Microscopy Sciences</td><td>CAT#21140</td></tr><tr><td>Ethanol</td><td>Decon Labs</td><td>CAT#2716</td></tr><tr><td>GTP</td><td>Sigma</td><td>CAT#10106399001</td></tr><tr><td>NADH</td><td>Sigma</td><td>CAT#N8129</td></tr><tr><td>phosphoenolpyruvate</td><td>Sigma</td><td>CAT#86007</td></tr><tr><td>Pyruvate kinase / lactate dehydrogenase</td><td>Sigma</td><td>CAT#P0294</td></tr><tr><td>TCEP</td><td>Chem-Impex</td><td>CAT#23004</td></tr><tr><td>Quin-2</td><td>Sigma</td><td>CAT#08520</td></tr><tr><td>Mag-fura-2</td><td>Invitrogen</td><td>CAT#M1290</td></tr><tr><td>EDTA</td><td>Alfa Aesar</td><td>CAT#A173850B</td></tr><tr><td>Chelex</td><td>Bio-Rad</td><td>CAT#141253</td></tr><tr><td>ZnCl2</td><td>Sigma</td><td>CAT#208086-100G</td></tr><tr><td>Magnesium chloride</td><td>Fisher</td><td>CAT#AA1079714</td></tr><tr><td>Ninhydrin</td><td>Sigma</td><td>CAT#151173-10G</td></tr><tr><td>Malachite green oxalate</td><td>Fisher</td><td>CAT#229780250</td></tr><tr><td>Ammonium molybdate</td><td>Sigma</td><td>CAT#277908-5G</td></tr><tr><td>Phosphate</td><td>Fluka</td><td>CAT#HONCE3039-100ML</td></tr><tr><td>Geneticin sulfate G418</td><td>Fisher</td><td>CAT#10131035</td></tr><tr><td>Lipofectamine 3000</td><td>Invitrogen</td><td>CAT#L3000-008</td></tr><tr><td>Insulin</td><td>Sigma</td><td>CAT#SLCF5002</td></tr><tr><td>Fetal Bovine Serum (FBS)</td><td>R and D Systems</td><td>CAT#S11150</td></tr><tr><td>Hepes</td><td>Fisher</td><td>CAT#BP310-500</td></tr><tr><td>Lysozyme</td><td>ThermoFisher</td><td>CAT#89833</td></tr><tr><td>HisPur Cobalt Resin</td><td>ThermoFisher</td><td>CAT#89964</td></tr><tr><td>Maltose</td><td>Sigma</td><td>CAT#63423-100mL</td></tr><tr><td>HRV 3C Protease</td><td>Pierce</td><td>CAT#88947</td></tr><tr><td>Nitrilotriacetic acid (NTA)</td><td>Sigma</td><td>CAT#9877</td></tr><tr><td>RIPA</td><td>ThermoFisher</td><td>CAT#89900</td></tr><tr><td>Halt Protease Inhibitor</td><td>ThermoFisher</td><td>CAT#78442</td></tr><tr><td>Tris/Glycine/SDS Buffer</td><td>Bio-Rad</td><td>CAT#1610732</td></tr><tr><td>Ponceau S Solution</td><td>Sigma</td><td>CAT#P7170</td></tr><tr><td>Odyssey Blocking Buffer</td><td>Li-Cor</td><td>CAT#927-40000</td></tr><tr><td>Glacial Acetic acid</td><td>Millipore</td><td>CAT#AX0074-2</td></tr><tr><td>Citric Acid</td><td>Sigma</td><td>CAT#Z1909</td></tr><tr><td>DMSO</td><td>Fisher</td><td>CAT#D136</td></tr><tr><td>Phenol:chloroform:IAA</td><td>Fisher</td><td>CAT#BP17521-100</td></tr><tr><td>4% Paraformaldehyde</td><td>Alfa Aesar</td><td>CAT#J61899</td></tr><tr class="rowsep"><td>2X iQ SYBR Green SuperMix</td><td>Bio-Rad</td><td>CAT#170882</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Seahorse XF Cell Mito Stress Test Kit</td><td>Agilent</td><td>CAT#103015-100</td></tr><tr class="rowsep"><td>Pierce™ HRV 3C Protease Solution Kit (2 units/μL)</td><td>ThermoFisher</td><td>CAT#88946</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>NMR chemical shifts (free zinc finger) &amp; backbone relaxation rates</td><td>BioMagResBank</td><td>51117</td></tr><tr><td>NMR chemical shifts (bound zinc finger)</td><td>BioMagResBank</td><td>51118</td></tr><tr><td>NMR chemical shifts (fusion)</td><td>BioMagResBank</td><td>30956</td></tr><tr><td>NMR backbone relaxation rates (fusion)</td><td>BioMagResBank</td><td>51119</td></tr><tr><td>ZNG1-METAP1-zf fusion structure</td><td>Protein Data Bank</td><td>7SEK</td></tr><tr class="rowsep"><td>Murine kidney and TKPTS cell line proteomics</td><td>ProteomeXchange</td><td>562287</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>Mouse: TKPTS WT parent</td><td>ATCC</td><td>CRL-3361; RRID: CVCL_UJ13</td></tr><tr><td>Mouse: TKPTS <i>Zng1</i> (-2/-2)</td><td>This manuscript</td><td>N/A</td></tr><tr><td>Mouse: TKPTS <i>Zng1</i> (-7/-2)</td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td>Mouse: TKPTS <i>Zng1</i> (-2/-2); <i>cmv:Zng1</i></td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr><td>Zebrafish: <i>zng1</i> (-5)</td><td>This manuscript</td><td>N/A</td></tr><tr><td>Zebrafish: <i>metap1</i> (-17)</td><td>This manuscript</td><td>N/A</td></tr><tr><td>Zebrafish: <i>zng1</i>(-11)</td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td>Mouse: <i>Zng1</i> (-180)</td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr class="rowsep"><td>See <a id="crosref1355" href="#mmc5">Table&nbsp;S5</a> for oligonucleotides</td><td>N/A</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>Mouse: pcDNA3.1-C-(k)-DYK-Cbwd1</td><td>GenScript</td><td>OMu12215</td></tr><tr><td>Mouse: pcDNA3.1-C-(k)-DYK-Metap1</td><td>GenScript</td><td>OMu05035</td></tr><tr><td>Zebrafish: pCRII - <i>zng1</i></td><td>This manuscript</td><td>N/A</td></tr><tr><td>pLM302 zebrafish <i>zng1</i></td><td>This manuscript</td><td>N/A</td></tr><tr><td>pLM302 zebrafish <i>metap1</i></td><td>This manuscript</td><td>N/A</td></tr><tr><td>pLM302 mouse <i>Zng1</i></td><td>This manuscript</td><td>N/A</td></tr><tr><td>pLM302 mouse <i>Metap1</i></td><td>This manuscript</td><td>N/A</td></tr><tr><td>pBG102 mouse <i>Metap1</i> 1-83</td><td>This manuscript</td><td>N/A</td></tr><tr><td>pSUMO mouse <i>Metap1</i></td><td>This manuscript</td><td>N/A</td></tr><tr><td>pSUMO mouse <i>Metap1</i> 1-59</td><td>This manuscript</td><td>N/A</td></tr><tr><td>pSUMO mouse <i>Metap1</i> 1-79</td><td>This manuscript</td><td>N/A</td></tr><tr><td>pSUMO mouse <i>Zng1</i> 10-30 <i>Metap1</i> 1-59 fusion</td><td>This manuscript</td><td>N/A</td></tr><tr><td>Tagless pHis mouse Zng1 (NdeI-BamHI)</td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td>pT3TS-nls-zCas9-nls</td><td>Addgene</td><td>CAT#46757</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Dynamics 7.5</td><td>Wyatt Technologies</td><td>DYNAMICS Software - Wyatt Technology</td></tr><tr><td>Fiji</td><td>Open Source (ImageJ)</td><td>Fiji (imagej.net)</td></tr><tr><td>ChromLab Software</td><td>Bio-Rad</td><td>ChromLab Software | Bio-Rad Laboratories</td></tr><tr><td>NMRBox</td><td>University of Connecticut Health Center</td><td><a id="intref0015" href="https://nmrbox.org/" target="_blank">https://nmrbox.org/</a></td></tr><tr><td>HMMER</td><td>HMMER</td><td><a id="intref0020" href="http://hmmer.org/" target="_blank">http://hmmer.org/</a></td></tr><tr><td>Topspin 4</td><td>Bruker</td><td><a id="intref0025" href="https://www.bruker.com/en/products-and-solutions/mr/nmr-software/topspin.html" target="_blank">https://www.bruker.com/en/products-and-solutions/mr/nmr-software/topspin.html</a></td></tr><tr><td>hmsIST</td><td>Harvard Medical School</td><td><a id="intref0030" href="https://bionmr.cores.ucla.edu/?page_id=440" target="_blank">https://bionmr.cores.ucla.edu/?page_id=440</a></td></tr><tr><td>NMRPipe</td><td>National Institute of Standards and Technology</td><td><a id="intref0035" href="https://www.ibbr.umd.edu/nmrpipe/index.html" target="_blank">https://www.ibbr.umd.edu/nmrpipe/index.html</a></td></tr><tr><td>CARA</td><td><a id="intref0040" href="https://wuthrich-group.ethz.ch/" target="_blank">https://wuthrich-group.ethz.ch/</a></td><td><a id="intref0045" href="http://cara.nmr.ch/" target="_blank">http://cara.nmr.ch/</a></td></tr><tr><td>NMRFAM-Sparky</td><td>University of Wisconsin</td><td><a id="intref0050" href="https://nmrfam.wisc.edu/nmrfam-sparky-distribution/" target="_blank">https://nmrfam.wisc.edu/nmrfam-sparky-distribution/</a></td></tr><tr><td>CYANA</td><td>LA Systems</td><td><a id="intref0055" href="http://www.cyana.org" target="_blank">http://www.cyana.org</a></td></tr><tr><td>NUS Poisson Gap Sampling Schedule generator for Topspin</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Hyberts et&nbsp;al. (2010)</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Hyberts, S.G. ∙ Takeuchi, K. ∙ Wagner, G.</div><div><strong>Poisson-gap sampling and forward maximum entropy reconstruction for enhancing the resolution and sensitivity of protein NMR data</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2010; <strong>132</strong>:2145-2147</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja908004w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77950835733" target="_blank" aria-label="Open Scopus in new tab">Scopus (256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20121194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja908004w&amp;pmid=20121194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://github.com/nomadiq/nusPGS_TS3_TS4_Distro" target="_blank">https://github.com/nomadiq/nusPGS_TS3_TS4_Distro</a></td></tr><tr><td>IMOD</td><td>University of Colorado</td><td><a id="intref0065" href="https://bio3d.colorado.edu/imod/" target="_blank">https://bio3d.colorado.edu/imod/</a></td></tr><tr><td>SerialEM</td><td>University of Colorado</td><td><a id="intref0070" href="https://bio3d.colorado.edu/SerialEM/" target="_blank">https://bio3d.colorado.edu/SerialEM/</a></td></tr><tr><td>Wave Desktop Software</td><td>Agilent Technologies</td><td><a id="intref0075" href="https://www.agilent.com/en/product/cell-analysis/real-time-cell-metabolic-analysis/xf-software/seahorse-wave-desktop-software-740897" target="_blank">https://www.agilent.com/en/product/cell-analysis/real-time-cell-metabolic-analysis/xf-software/seahorse-wave-desktop-software-740897</a></td></tr><tr><td>CLC Genomics Workbench</td><td>Qiagen</td><td><a id="intref0080" href="https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-clc-genomics-workbench/" target="_blank">https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-clc-genomics-workbench/</a></td></tr><tr><td>Ingenuity Pathway Analysis (IPA)</td><td>Qiagen</td><td><a id="intref0085" href="https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-ipa/" target="_blank">https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-ipa/</a></td></tr><tr><td>Agilent Mass Hunter Workstation Software version A.01.02</td><td>Agilent</td><td><a id="intref0090" href="https://www.agilent.com/en/product/atomic-spectroscopy/inductively-coupled-plasma-mass-spectrometry-icp-ms/icp-ms-software/icp-ms-masshunter-software" target="_blank">https://www.agilent.com/en/product/atomic-spectroscopy/inductively-coupled-plasma-mass-spectrometry-icp-ms/icp-ms-software/icp-ms-masshunter-software</a></td></tr><tr class="rowsep"><td>BioRender</td><td>BioRender</td><td>BioRender.com</td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>HisTrap™ High Performance</td><td>Cytiva: Research &amp; Biopharmaceutical Manufacturing</td><td>N/A</td></tr><tr><td>HisPur™ Cobalt Resin</td><td>Cytiva: Research &amp; Biopharmaceutical Manufacturing</td><td>HisPur™ Cobalt Resin (thermofisher.com)</td></tr><tr><td>Superdex 200 Increase 10/300 GL</td><td>Cytiva: Research &amp; Biopharmaceutical Manufacturing</td><td>Superdex 200 Increase 10/300 GL | Cytiva (cytivalifesciences.com)</td></tr><tr><td>NGC Quest 10</td><td>Bio-Rad</td><td>NGC Quest 10 Chromatography System | Bio-Rad Laboratories</td></tr><tr><td>DynaPro NanoStar</td><td>Wyatt Technologies</td><td>DynaPro NanoStar Dynamic Light Scattering Detector for Nanoparticles (wyatt.com)</td></tr><tr><td>Supelco Ascentis Express C18 Column (50&nbsp;mm x 2.1&nbsp;mm, 5 micro particles)</td><td>Sigma</td><td>50509-U</td></tr><tr><td>SecurityGuard C18 Cartridge (3.2&nbsp;mm x 8&nbsp;mm)</td><td>Phenomenex</td><td>AJ0-4287</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2822%2900458-5" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0230" role="paragraph">Further information and request for resources and reagents should be directed to and will be fulfilled by the lead contact, Eric P. Skaar (<a id="interref0010" href="mailto:eric.skaar@vumc.org">eric.skaar@vumc.org</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0235" role="paragraph">All materials generated in this study are available from the <a id="crosref1370" href="#sec-4-2-1">lead contact</a> upon request.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Ethics statement</h4><div id="p0240" role="paragraph">Mouse and zebrafish studies were approved by the Institutional Animal Care and Use Committees of Vanderbilt University Medical Center (protocol numbers M1900043-00 and M1900076-00 respectively) in accordance with the Public Health Service Policy on the Human Care and Use of Laboratory Animals under the Unites States of America National Institutes of Health (NIH) Office of Laboratory Animal Welfare (OLAW).</div></section><section id="sec-4-3-2"><h4>Zebrafish husbandry</h4><div id="p0245" role="paragraph">All zebrafish lines were maintained on a mixed Tübingen (Tü) / AB background on a 14:10&nbsp;h light:dark cycle in a recirculating aquaculture system. Embryos were collected from natural matings and maintained in embryo medium (0.03% Instant Ocean Sea Salt in RO water) at a density of ≤ 1 larva / mL at 28°C 14:10&nbsp;h light:dark cycle. At 3 dpf, larvae were split randomly into treatment groups. All larvae used in experiments are of indeterminate sex and between ages 0 – 6 dpf as reported in the figure legends.</div></section><section id="sec-4-3-3"><h4>Mouse husbandry and diet</h4><div id="p0250" role="paragraph">Experiments were performed using adult age-matched C57BL/6J (Jackson Laboratories) or C57BL/6J <i>Zng1</i><sup><i>-/-</i></sup> (breeding colony) mice. Animals were maintained at the Vanderbilt University Medical Center Animal Facilities and housed in groups of five. For each experiment, <i>Zng1</i><sup><i>-/-</i></sup> mice from multiple litters were included. Mice of both sexes were between 5-13&nbsp;weeks of age. The exact age as well as the sex of animals used for each individual experiment are denoted in the corresponding figure legends. For routine colony maintenance, mice were fed a standard chow diet (LabDiets; Rodent Chow Diet 5001). For manipulation of organismal Zn levels, mice were fed a defined Zn-free or control diet (Dyets Inc., AIN-93M Purified Rodent Diet with or without Zn supplementation at 29 parts per million). For experimental endpoints, animals were humanely euthanized. All animal experiments were approved and performed in compliance with the Institutional Animal Care and Use Committee (IACUC) of Vanderbilt University.</div></section><section id="sec-4-3-4"><h4>Cell culture conditions</h4><div id="p0255" role="paragraph">Mycoplasma-negative WT parental TKPTS and pooled <i>Zng1</i> CRISPR/Cas9 mutant cells were ordered from Synthego (<a id="intref0095" href="https://www.synthego.com/" target="_blank">https://www.synthego.com/</a>). Briefly, cells were propagated in DMEM/F-12 50/50 1X (Corning 10-092-CV) supplemented with 7% heat-inactivated fetal bovine serum (R and D Systems, S11150) with insulin (Sigma SLCF5002 – 0.3&nbsp;mL of 10mg/mL per 500&nbsp;mL media) under 5% CO<sub>2</sub>. Cells were trypsinized and passaged using TrypLE express (Thermo Fisher Scientific, 12605010).</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Yeast-Two-Hybrid screening and analysis</h4><div id="p0260" role="paragraph">All yeast-two-hybrid screens were performed by Hybrigenics Services using full-length human ZNG1E, mouse ZNG1, and zebrafish Zng1. Murine <i>Zng1</i> and human <i>ZNG1E</i> sequences were obtained from Genscript (Clone ID OMu12215 and OHu42907 in the pcDNA3.1-C-(k)DYK backbone, respectively). Full length zebrafish <i>Zng1</i> was amplified from 6&nbsp;days post fertilization (dpf) whole larval cDNA with P1 and P2 (<a id="crosref1375" href="#mmc5">Table&nbsp;S5</a>) and cloned into pCRII TOPO (Invitrogen, K465001) and sequence verified by Sanger sequencing. The sequences described above were used as template for the generation of all subsequent <i>ZNG1</i> constructs. Yeast-two-hybrid screens to identify interaction partners of ZNG1 proteins from different species were performed on a mixed (A549, H1703, H460) lung tumor cell library (human), an adult kidney library (mouse), or a whole embryo 20&nbsp;h post fertilization library (zebrafish). Any interacting partners that were scored as experimental artifact were excluded from further analysis.</div></section><section id="sec-4-4-2"><h4>Recombinant protein cloning</h4><div id="p0265" role="paragraph">Full-length murine <i>Zng1</i> was amplified from pcDNA3.1-C-(k)DYK-<i>Zng1</i> (GenScript, Clone ID OMu12215) using primers P3 and P4 (<a id="crosref1380" href="#mmc5">Table&nbsp;S5</a>). PCR products were inserted into the expression vector pLM302 (Center for Structural Biology, Vanderbilt University) containing a 3C protease cleavable N-terminal 6xHis and Maltose Binding Protein (MBP) tag using <i>Bam</i>HI and <i>Eco</i>RI restriction sites.</div><div id="p0270" role="paragraph">Full-length murine <i>Metap1</i> was amplified from pcDNA3.1-C-(k)DYK-<i>Metap1</i> (GenScript, Clone ID OMu05035) using primer pair P5/P6. Vector pLM302 was linearized by PCR (primers P7/P8; <a id="crosref1385" href="#mmc5">Table&nbsp;S5</a>). Both amplicons were joined using the NEBuilder HiFi DNA Assembly Master Mix (New England Biolabs, E2621S).</div><div id="p0275" role="paragraph">A SUMO-tagged, truncated murine METAP1<sub>1-83</sub> was prepared by amplifying the N-terminal portion of the murine <i>Metap1</i> construct using primers P9 and P10 (<a id="crosref1390" href="#mmc5">Table&nbsp;S5</a>). The PCR product was cloned into the expression vector pBG102 carrying a 3C protease cleavable N-terminal 6xHis and SUMO tag (Center for Structural Biology, Vanderbilt University) using the <i>Bam</i>HI and <i>Eco</i>RI restriction sites.</div><div id="p0280" role="paragraph">For peptide, metal-binding, and METAP1 activation studies, murine <i>Zng1</i> was amplified using primers P11 and P12 before being cloned into the <i>Nde</i>I and <i>Bam</i>HI sites in the pHis vector (<span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Sheffield et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Sheffield, P. ∙ Garrard, S. ∙ Derewenda, Z.</div><div><strong>Overcoming expression and purification problems of RhoGDI using a family of “parallel” expression vectors</strong></div><div><em>Protein Expr. Purif.</em> 1999; <strong>15</strong>:34-39</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/prep.1998.1003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0033082686" target="_blank" aria-label="Open Scopus in new tab">Scopus (530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10024467/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fprep.1998.1003&amp;pmid=10024467" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mouse <i>Metap1</i>, METAP1<sub>1-59</sub>, and METAP1<sub>1-79</sub> were cloned into the <i>Nde</i>I and <i>Hin</i>dIII sites in the pSUMO vector (<span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Peroutka Iii et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Peroutka Iii, R.J. ∙ Orcutt, S.J. ∙ Strickler, J.E. ...</div><div><strong>SUMO fusion technology for enhanced protein expression and purification in prokaryotes and eukaryotes</strong></div><div><em>Methods Mol. Biol.</em> 2011; <strong>705</strong>:15-30</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-61737-967-3_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-79952459386" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21125378/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-61737-967-3_2&amp;pmid=21125378" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) containing N-terminal 6xHis and SUMO tags using primers P13-P16 (<a id="crosref1405" href="#mmc5">Table&nbsp;S5</a>) via isothermal assembly (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Gibson et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Gibson, D.G. ∙ Young, L. ∙ Chuang, R.Y. ...</div><div><strong>Enzymatic assembly of DNA molecules up to several hundred kilobases</strong></div><div><em>Nat. Methods.</em> 2009; <strong>6</strong>:343-345</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1318" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-67349270900" target="_blank" aria-label="Open Scopus in new tab">Scopus (5532)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19363495/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1318&amp;pmid=19363495" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The fusion of a 21-residue peptide from murine ZNG1 (ZNG1<sub>10-30</sub>) onto the N-terminus of the truncated METAP1<sub>1-59</sub>, used for the NMR structure, was generated by isothermal assembly (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Gibson, 2011</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Gibson, D.G.</div><div><strong>Enzymatic assembly of overlapping DNA fragments</strong></div><div><em>Methods Enzymol.</em> 2011; <strong>498</strong>:349-361</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/B978-0-12-385120-8.00015-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-79957471219" target="_blank" aria-label="Open Scopus in new tab">Scopus (491)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21601685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FB978-0-12-385120-8.00015-2&amp;pmid=21601685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) of oligonucleotides P17 and P18 (<a id="crosref1420" href="#mmc5">Table&nbsp;S5</a>) into the <i>Bam</i>HI site of the METAP1<sub>1-59</sub> construct.</div><div id="p0285" role="paragraph">The full-length zebrafish <i>zng1</i> coding sequence was cloned from 6 dpf whole larval cDNA using primers P1 and P2 into pCRII (Invitrogen) as described above (<a id="crosref1425" href="#mmc5">Table&nbsp;S5</a>). Full-length zebrafish <i>zng1</i> was subcloned from TOPO pCRII into pLM302 using primer pair P19/P20 (<a id="crosref1430" href="#mmc2">Table&nbsp;S2</a>). Full-length zebrafish <i>metap1</i> was amplified from 6 dpf whole larval cDNA and cloned into pLM302 using primer pair P21/P22 (<a id="crosref1435" href="#mmc5">Table&nbsp;S5</a>).</div><div id="p0290" role="paragraph">All constructs were confirmed by Sanger sequencing. Murine 6xHis-SUMO-tagged METAP1<sub>1-83</sub> was transformed into <i>E.&nbsp;coli</i> C41(DE3). All other constructs were transformed into <i>E.&nbsp;coli</i> BL21(DE3) for expression.</div></section><section id="sec-4-4-3"><h4>Recombinant protein expression and purification</h4><div id="p0295" role="paragraph">All bacterial expression strains were grown aerobically in Lysogeny Broth (LB) while shaking. Depending on the specific expression vector, media were supplemented with antibiotic to ensure plasmid maintenance, including 50&nbsp;μg/mL kanamycin, 100&nbsp;μg/mL ampicillin, or 35&nbsp;μg/mL chloramphenicol. Bacterial cultures were grown at 37 °C to OD<sub>600</sub>&nbsp;= 0.6. Protein expression was initiated with the addition of 0.2-1&nbsp;mM isopropyl β-D-1-thiogalactopyranoside (IPTG), the temperature was lowered to 18 °C, and the cultures allowed to grow for 14-16 h. Cells were harvested by centrifugation at 4,500x g for 15&nbsp;min at 4 °C and stored at -20 °C prior to protein isolation.</div><div id="p0300" role="paragraph">For affinity chromatography, size exclusion chromatography, and dynamic light scattering experiments, the protein was isolated as follows. Bacterial pellets were resuspended in 10&nbsp;mL lysis/wash buffer (20&nbsp;mM HEPES (pH 7.5), 150&nbsp;mM NaCl). Lysozyme (Thermo Fisher Scientific, 89833) was added at a final concentration of 1&nbsp;mg/mL before incubating on ice for 10&nbsp;min. Cells were lysed by sonication and cellular debris removed by centrifugation (4 °C, 10,000x g, 30&nbsp;min). The supernatant was loaded on a pre-washed column containing amylose resin (New England Biolabs, E8021). Columns were washed with 10 volumes of lysis/wash buffer before elution of bound protein with lysis/wash buffer containing 10&nbsp;mM maltose. Protein purity and quantity was assessed by SDS-PAGE and BCA assay (Pierce, 23225 and 23209). To remove the 6xHis-MBP tag, protein was incubated with HRV 3C protease (Pierce, 88947) for 12-16h (4 °C with agitation). Cleaved tag was removed by loading the mixture on a pre-washed column containing HisPur Cobalt Resin (Thermo Fisher Scientific, 89964). The flow through was collected and protein purity and quantity assessed by SDS-PAGE and BCA assay (Pierce, 23225 and 23209). Purified protein was used immediately or flash frozen and stored at -80 °C.</div><div id="p0305" role="paragraph">For peptide-binding, metal-binding, and METAP1 activation assays, ZNG1 and METAP1 were prepared as follows. Cells expressing untagged ZNG1 were resuspended in 25&nbsp;mM Tris (pH 8.0), 500&nbsp;mM NaCl, 2.5&nbsp;mM EDTA, and 5&nbsp;mM tris(2- carboxyethyl)phosphine (TCEP). Cells were lysed by sonication for 15&nbsp;min, and the resulting lysate was centrifuged at 13,000x g for 20&nbsp;min at 4 °C to remove cellular debris. The resulting supernatant was treated with 0.015% polyethylenimine on ice to precipitate nucleic acids, which were removed by centrifugation at 13,000x g for 20&nbsp;min at 4 °C. The supernatant was treated with 40% ammonium sulfate on ice for 30&nbsp;min to precipitate ZNG1. The mixture was separated by centrifugation at 13,000x g for 20&nbsp;min at 4 °C, and the resulting pellet containing ZNG1 was resuspended in 25&nbsp;mM Tris (pH 8.0), 150&nbsp;mM NaCl, 2.5&nbsp;mM EDTA, and 5&nbsp;mM TCEP. This solution was dialyzed into 25&nbsp;mM Tris (pH 8.0), 50&nbsp;mM NaCl, 2.5&nbsp;mM EDTA, and 5&nbsp;mM TCEP overnight, injected onto a preequilibrated HiTrap Q FF (Cytiva Life Sciences, 17515601), and eluted with a gradient from 50&nbsp;mM to 1&nbsp;M NaCl. Fractions of &gt;90% purity were combined for additional purification by size-exclusion chromatography (HiLoad 16/600 Superdex 200 pg, Cytiva Life Sciences 28989335) in 25&nbsp;mM Tris (pH 8.0), 150&nbsp;mM NaCl, 2.5&nbsp;mM EDTA, and 5&nbsp;mM TCEP. All fractions of &gt;95% purity (by inspection of SDS PAGE gels) were collected and buffer- exchanged 10<sup>6</sup>-fold into chelex-treated (Biorad 141253) 25&nbsp;mM 4-(2-hydroxyethyl)-1- piperazineethanesulfonic acid (HEPES (pH 7.4)), 150&nbsp;mM NaCl, and 5&nbsp;mM TCEP to create the final prep of ZNG1. ZNG1 prepared in this way was devoid of bound metal (&lt;0.01&nbsp;mol equiv of Zn(II) measured by ICP-MS).</div><div id="p0310" role="paragraph">For SUMO-tagged METAP1 constructs, cells were resuspended in 25&nbsp;mM Tris (pH 8.0), 500&nbsp;mM NaCl, and 5&nbsp;mM TCEP. Cells were lysed by sonication for 15&nbsp;min, and the resulting lysate was centrifuged at 13,000x g for 20&nbsp;min at 4 °C to remove cellular debris. The resulting supernatant was injected onto a HisTrap FF (Cytiva Life Sciences, 17525501) column and eluted with a gradient from 0 to 500&nbsp;mM imidazole. Fractions of &gt;90% purity were combined and treated with SUMO protease for 2&nbsp;h at room temperature to remove the N-terminal His-SUMO tag and dialyzed into 25&nbsp;mM Tris (pH 8.0), 500&nbsp;mM NaCl, and 2&nbsp;mM TCEP. The cleaved protein product was injected onto a HisTrap FF column and the flow through was collected and concentrated for additional purification by size-exclusion chromatography (HiLoad 16/600 Superdex 200 pg for full-length METAP1 or HiLoad 16/600 Superdex 75 pg for truncated METAP1) in 25&nbsp;mM Tris (pH 8.0), 150&nbsp;mM NaCl, 2&nbsp;mM TCEP. All fractions of &gt;95% purity (by inspection of SDS PAGE gels) were collected and full-length protein was treated with chelator as indicated below. Purified protein was buffer-exchanged 10<sup>6</sup>-fold into chelexed 25&nbsp;mM HEPES (pH 7.4), 150&nbsp;mM NaCl, and 2&nbsp;mM TCEP to create the final prep of METAP1, METAP1<sub>1-59</sub>, or METAP1<sub>1-79</sub>. Notably, we experienced significant decreases in METAP1 enzymatic activity from frozen protein stocks frozen over extended amounts of time (&gt;1&nbsp;month) and aliquots that had gone through multiple freeze thaw cycles. Therefore, final preparations of full-length METAP1 utilized in <i>in&nbsp;vitro</i> experiments in this study were purified, stripped of metal, and aliquoted for -80 °C storage all within one day. Moreover, frozen aliquots were thawed no more than twice to ensure maximal METAP1 enzymatic function. Proteins were analyzed by ICP-MS to identify metal stoichiometries (<a id="crosref1440" href="#figs2">Figure&nbsp;S2</a>F). To remove excess metal, METAP1 was treated with either 2&nbsp;mM EDTA, 10&nbsp;mM NTA, or 4 molar equivalents N,N,N',N-terakis-(2-pyridylmethyl)ethylenediamine (TPEN) for 1&nbsp;h at 4°C and then buffer-exchanged 10<sup>6</sup>-fold into chelexed 25&nbsp;mM HEPES (pH 7.4), 150&nbsp;mM NaCl, and 2&nbsp;mM TCEP. To load Zn(II) into METAP1<sub>1-59</sub> and METAP1<sub>1-79</sub>, 50&nbsp;μM Zn(II) and 1&nbsp;mM NTA were added to protein solutions for 1&nbsp;h at 4 °C and then buffer-exchanged 10<sup>6</sup>-fold into chelexed 25&nbsp;mM HEPES (pH 7.4), 150&nbsp;mM NaCl, and 2&nbsp;mM TCEP. Molar extinction coefficients at 280&nbsp;nm (35410 M<sup>-1</sup> cm<sup>-1</sup> for ZNG1, 55350 M<sup>-1</sup> cm<sup>-1</sup> for METAP1, 6990 M<sup>-1</sup> cm<sup>-1</sup> for METAP<sub>1-59</sub>, and 17990 M<sup>-1</sup> cm<sup>-1</sup> for METAP1<sub>1-79</sub>) were used to calculate protein concentration.</div><div id="p0315" role="paragraph">Uniformly <sup>15</sup>N- and <sup>15</sup>N<sup>13</sup>C-labeled METAP1<sup>1-59</sup> and the ZNG1<sub>10-30</sub>-METAP1<sub>1-59</sub> fusion were prepared for NMR experiments using the SUMO tagged constructs as described above, except for the following details: The expression strains were grown in M9&nbsp;minimal media with 1 g/L <sup>15</sup>N ammonium chloride (Cambridge Isotope Laboratories) as the sole nitrogen source and either 4 g/L unlabeled glucose or 2 g/L <sup>13</sup>C-glucose (Cambridge Isotope Laboratories) as the sole carbon source. 10 % <sup>13</sup>C-labeled protein was prepared using 1&nbsp;g <sup>15</sup>NH<sub>4</sub>Cl, 0.4&nbsp;g <sup>13</sup>C<sub>6</sub> -glucose, and 3.6&nbsp;g unlabeled glucose. The lysis buffer was supplemented with EDTA-free protease inhibitor cocktail (Roche). Nickel-loaded HisTrap FF columns were used to isolate the protein from clarified lysate and after cleavage of the SUMO tag. To ensure that the metal binding sites were homogeneously occupied by Zn(II), the protein was exchanged 10<sup>3</sup>-fold into Zn-loading buffer (25&nbsp;mM HEPES, 150&nbsp;mM NaCl, pH 7.4, treated with chelex and supplemented with 1&nbsp;mM nitrilotriacetic acid [NTA], 50&nbsp;μM Zn(II), and 2&nbsp;mM TCEP) using an Amicon ultra centrifugal filter unit (3&nbsp;kDa cut-off), prior to 10<sup>3</sup>-fold exchange into NMR buffer (chelex-treated 10&nbsp;mM sodium phosphate, 150&nbsp;mM NaCl, pH 7.0, supplemented with 2&nbsp;mM TCEP). These samples were confirmed by ICP-MS to contain 2.0 equivalents of Zn(II) and less than 0.01 equivalent of Ni(II).</div></section><section id="sec-4-4-4"><h4>Affinity chromatography to determine ZNG1-METAP1 interaction</h4><div id="p0320" role="paragraph">To probe the interaction of the full-length proteins, His-tagged-MBP-ZNG1 (Mouse) was isolated as described above. Protein was loaded onto a previously equilibrated Ni-NTA column (Cytiva Lifesciences, 17-5248-05). The column was washed three times with five column volumes of lysis/wash buffer (10&nbsp;mM Tris (pH 8) 150&nbsp;mM NaCl). Following the washes, purified METAP1 with the tag cleaved and removed was loaded onto the column and washed three additional times. After washing, the complex was eluted using wash buffer containing 500&nbsp;mM imidazole. Fractions were visualized on a 4-20% Bis-Tris SDS-PAGE gel. Western Blot (methods described below) using a polyclonal METAP1 antibody (PA5-58202, 1:1,000) was performed to confirm identity of METAP1.</div><div id="p0325" role="paragraph">To test the interaction of zebrafish Zng1 and Metap1, His-tagged-MBP-Metap1 was isolated as described above, cleaved with HRV 3C protease (Pierce, 88947) passed through a HisPur Cobalt Resin (Thermo Scientific, 89964) column, and dialyzed in lysis/wash buffer overnight at 4 °C. Subsequently, His-tagged-MBP-Zng1 was purified as described above with the following exception. Following flow through and washing of the amylose resin (New England Biolabs, E8021), purified Metap1 was passed over the column containing immobilized MBP-Zng1 or empty resin as control. Columns were subsequently washed with 100mL of lysis/wash buffer prior to elution and fractions collected. Western blot detection of Zng1 (anti-CBWD1, HPA042813) of wash and elution fractions was performed using methods described below. Densitometry of Zng1 signal in each fraction was measured using Fiji software (<span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31040)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-5"><h4>Dynamic light scattering</h4><div id="p0330" role="paragraph">Purified and tag-free murine ZNG1 and METAP1 were incubated for 1&nbsp;h at 4 <sup>o</sup>C before the experiments. The individual proteins and the mixture were filtered through 0.1&nbsp;μm syringe filters to remove dust and trace residual lipid vesicles. Experiments were performed at 25 °C and data were analyzed using Dynamics 7.5 software (Wyatt Technologies). The DLS software calculates an apparent hydrodynamic radius (Rh) via the Stokes-Einstein equation (<span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Stetefeld et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Stetefeld, J. ∙ McKenna, S.A. ∙ Patel, T.R.</div><div><strong>Dynamic light scattering: a practical guide and applications in biomedical sciences</strong></div><div><em>Biophys. Rev.</em> 2016; <strong>8</strong>:409-427</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12551-016-0218-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85000399690" target="_blank" aria-label="Open Scopus in new tab">Scopus (824)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28510011/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12551-016-0218-6&amp;pmid=28510011" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-6"><h4>Size exclusion chromatography</h4><div id="p0335" role="paragraph">The interaction of full-length murine ZNG1 with METAP1<sub>1-83</sub> was performed by size exclusion chromatography. Recombinant proteins were purified using methods described above and incubated for 1&nbsp;h at 4 °C followed by passing over a Superdex 200 increase 10/300 GL column (Cytiva Lifesciences, 28990944). The buffer contained 50&nbsp;mM Tris (pH 8) and 300&nbsp;mM NaCl and the experiment was performed with a flow rate of 0.5&nbsp;mL/min. The resulting fractions were separated and visualized by SDS-PAGE.</div></section><section id="sec-4-4-7"><h4>Immunoblot detection of ZNG1 and METAP1</h4><div id="p0340" role="paragraph">For protein isolation, WT and <i>Zng1</i> mutant TKPTS cells were washed once with warm PBS, trypsinized as described above, and transferred into fresh conical tubes. Samples were centrifuged at 200x g for 10&nbsp;min. Cells were subsequently washed once with PBS and resuspended in 500&nbsp;μL of cold RIPA buffer (Thermo Fisher Scientific, 89900) supplemented with Halt protease inhibitor (Thermo Fisher Scientific, 78442). Cells were subsequently vortexed for 10 s and incubated on ice for 20&nbsp;min prior to centrifugation for 15&nbsp;min at 15,000x g. Supernatant was transferred into fresh Eppendorf tubes and protein concentration was measured by BCA assay (Thermo Fisher Scientific, 23225). Samples normalized for protein input were separated on SDS PAGE gels in 1X Tris/Glycine/SDS Buffer (Bio-Rad, 1610732) and transferred using a semi-dry method using Bio-Rad trans blot turbo (25V, 1A, 18&nbsp;min) onto a nitrocellulose membrane (Li-COR, 926-31092). Transfer efficiency was evaluated using staining with Ponceau S solution (Sigma P7170) following manufacturer’s protocol. Membranes were washed three times in PBS supplemented with 0.2% tween (PBS-T), rocking for&nbsp;5&nbsp;min at room temperature. Blocking was performed using Odyssey (PBS) blocking buffer (Li-COR 927-40000) for 1&nbsp;h at room temperature and subsequently washed three times with PBS-T. Primary antibodies were applied (anti-CBWD1, Sigma, HPA042813,1:500 dilution; anti-Tubulin, Cell signaling, 3873S, 1:1,000 dilution; anti-METAP1, Thermo Fisher Scientific PA5-58202, 1:1,000) in Odyssey blocking buffer and incubated overnight at 4 °C rocking. Membranes were washed three times with PBS-T for 5&nbsp;min at room temperature. Secondary antibody solutions (Li-COR), species specific IR800CW or IR680LT, 1:2,000) were applied in Odyssey (PBS) blocking buffer, and membranes were incubated in the dark for 2&nbsp;h rocking at room temperature. Following secondary antibody staining, membranes were washed three times with PBS-T and visualized using a ChemiDoc MP Imaging System with appropriate channels for detection.</div></section><section id="sec-4-4-8"><h4>Murine METAP1 methionine aminopeptidase assay</h4><div id="p0345" role="paragraph">Methionine aminopeptidase activity was evaluated by quantifying the release of Met from the heptapeptide MAHAIHY (GenScript) as adapted from (<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Kabir-ud-Din, 2003</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Kabir-ud-Din ∙ Bano, M. ∙ Khan, I.A.</div><div><strong>Kinetics and mechanism of the ninhydrin reaction with DL-methionine in the absence and the presence of organic solvents</strong></div><div><em>Indian J. Chem.</em> 2003; <strong>42B</strong>:1132-1136</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Kabir-ud-DinM.BanoI.A.KhanKinetics+and+mechanism+of+the+ninhydrin+reaction+with+DL-methionine+in+the+absence+and+the+presence+of+organic+solventsIndian+J.+Chem.42B200311321136" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Reactions were conducted at 25°C in 25&nbsp;mM HEPES pH 7.4, 150&nbsp;mM NaCl, 2&nbsp;mM TCEP and consisted of 20&nbsp;nM murine METAP1 with 0, 1, or 2 molar equivalents of Zn. Reactions were initiated with the addition of varying concentrations of MAHAIHY peptide. 50&nbsp;μL aliquots were taken every 30 s from 30-150 s and the reaction was quenched with the addition of 10&nbsp;μL glacial acetic acid (Supelco, Sigma-Aldrich). 60&nbsp;μL of 4% w/v ninhydrin (Sigma-Aldrich) in 100% ACN was added to each quenched reaction and incubated at 37 °C for 1&nbsp;h to detect free Met. The absorbance at 570&nbsp;nm was measured and the cleaved Met concentration was determined from a standard curve of 10-500&nbsp;μM Met. All data are reported as the average of triplicate measurements from each evaluated experimental condition. Midpoint and completed reactions were additionally analyzed by LC-ESI-MS to confirm the generation of the expected AHAIHY cleaved peptide product (<a id="crosref1460" href="#figs4">Figure&nbsp;S4</a>I).</div></section><section id="sec-4-4-9"><h4>Mag-fura-2 Zn(II) competition assays</h4><div id="p0350" role="paragraph">Experiments were performed using an HP8453 UV−vis spectrophotometer or an ISS PC1 spectrofluorometer at 25.0 °C. Apo-mf2 (Invitrogen) was monitored by measuring the absorbance at 366&nbsp;nm or by fluorescence emission at 505&nbsp;nm upon excitation at 366&nbsp;nm. Zn-mf2 was monitored by measuring the absorbance at 324&nbsp;nm or the emission at 505&nbsp;nm upon excitation at 324&nbsp;nm. Several [protein]/[mf2] values were used for replicate experiments and analyzed globally to better calculate the stoichiometry. In a typical fluorescence experiment, 1&nbsp;μM protein and 1&nbsp;μM mf2 in 2.5&nbsp;mL were incubated for 10&nbsp;min in chelexed buffer [25&nbsp;mM HEPES (pH&nbsp;7.4), 150&nbsp;mM NaCl, and 2.5&nbsp;mM TCEP], and aliquots of Zn(II) were added. The equilibrium time was 2&nbsp;min between subsequent additions of metal. The peak intensities at 324 and 366&nbsp;nm from triplicate experiments were globally fit to 1:1, 2:1, 3:1, and 4:1 binding models, with the <i>K</i><sub>Zn</sub> value of mf2 fixed to 5 x 10<sup>8</sup> M<sup>−1</sup> using Dynafit (<span class="dropBlock reference-citations"><a id="body-ref-sref55-1" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-1" aria-controls="sref55-1">Kuzmic, 1996</a><div class="dropBlock__holder" data-db-target-of="sref55-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Kuzmic, P.</div><div><strong>Program DYNAFIT for the analysis of enzyme kinetic data: application to HIV proteinase</strong></div><div><em>Anal. Biochem.</em> 1996; <strong>237</strong>:260-273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/abio.1996.0238" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029902679" target="_blank" aria-label="Open Scopus in new tab">Scopus (1358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8660575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fabio.1996.0238&amp;pmid=8660575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For ZNG1, the 3:1 binding model provided the best fit of the mf2 data alone, whereas the 4:1 binding model provided the best fit of the METAP1 mf2 data. The selected 4:1 binding model was subsequently used for a simultaneous global fit of triplicate mf2 data sets along with triplicate quin2 datasets for METAP1 (Note S1).</div></section><section id="sec-4-4-10"><h4>Quin-2 Zn(II) competition assays</h4><div id="p0355" role="paragraph">Experiments were performed using an ISS PC1 spectrofluorometer at 25.0 °C by monitoring the fluorescence emission of quin2 at 490&nbsp;nm upon excitation at 353&nbsp;nm, with quenching of the 490&nbsp;nm emission maximum as aliquots of Zn(II) were added. Experiments were performed in triplicate at multiple protein:quin2 ratios. In a typical experiment, 1-2&nbsp;μM protein and 1-2&nbsp;μM quin-2 (Sigma 08520) were incubated for 10&nbsp;min in chelexed buffer [25&nbsp;mM HEPES (pH 7.4), 150&nbsp;mM NaCl, and 2.5&nbsp;mM TCEP] at a total volume of 3&nbsp;mL, and 0.3&nbsp;μM aliquots of Zn(II) were added. The equilibrium time was 12&nbsp;min between subsequent additions of metal. The peak emission intensities at 490&nbsp;nm from triplicate experiments were globally fit along with triplicate mf2 datasets using Dynafit (<span class="dropBlock reference-citations"><a id="body-ref-sref55-2" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-2" aria-controls="sref55-2">Kuzmic, 1996</a><div class="dropBlock__holder" data-db-target-of="sref55-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Kuzmic, P.</div><div><strong>Program DYNAFIT for the analysis of enzyme kinetic data: application to HIV proteinase</strong></div><div><em>Anal. Biochem.</em> 1996; <strong>237</strong>:260-273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/abio.1996.0238" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029902679" target="_blank" aria-label="Open Scopus in new tab">Scopus (1358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8660575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fabio.1996.0238&amp;pmid=8660575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with the <i>K</i><sub>Zn</sub> value of quin2 fixed to 2&nbsp;×&nbsp;10<sup>11</sup> M<sup>−1</sup>.</div></section><section id="sec-4-4-11"><h4>Intrinsic trp fluorescence quenching assays</h4><div id="p0360" role="paragraph">Peptide binding experiments were performed using an ISS PC1 spectrofluorometer at 25.0 °C. Intrinsic Trp fluorescence was monitored using λ<sub>ex</sub>&nbsp;= 292&nbsp;nm and λ<sub>em</sub>&nbsp;= 350&nbsp;nm, with quenching of the 350&nbsp;nm emission maxima as aliquots of ZNG1 peptide were added. Different [METAP1<sub>1-59/1-79</sub>] values were used for replicate experiments to better calculate the binding stoichiometry and analyzed globally. In a typical experiment, 5&nbsp;μM METAP1<sub>1-59</sub>/<sub>1-79</sub> in 2.5&nbsp;mL were incubated for 10&nbsp;min in chelexed buffer [25&nbsp;mM HEPES (pH 7.4), 150&nbsp;mM NaCl, and 2.5&nbsp;mM TCEP], and aliquots of ZNG1 peptide were added. The equilibrium time was 2&nbsp;min between subsequent additions of ZNG1 peptide. The peak emission intensities at 350&nbsp;nm from triplicate experiments were globally fit to a 1:1 binding model using Dynafit (<span class="dropBlock reference-citations"><a id="body-ref-sref55-3" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-3" aria-controls="sref55-3">Kuzmic, 1996</a><div class="dropBlock__holder" data-db-target-of="sref55-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Kuzmic, P.</div><div><strong>Program DYNAFIT for the analysis of enzyme kinetic data: application to HIV proteinase</strong></div><div><em>Anal. Biochem.</em> 1996; <strong>237</strong>:260-273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/abio.1996.0238" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029902679" target="_blank" aria-label="Open Scopus in new tab">Scopus (1358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8660575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fabio.1996.0238&amp;pmid=8660575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0365" role="paragraph">EDTA titrations of METAP1<sub>1-56</sub> were performed using in a Synergy Neo2 plate reader to monitor Trp fluorescence using λ<sub>ex</sub>&nbsp;= 292&nbsp;nm and λ<sub>em</sub>&nbsp;= 350&nbsp;nm for a series of EDTA (Alfa Aesar A107130B) concentrations ranging from 0.3&nbsp;μM to 5&nbsp;mM, for incubation times ranging from 10&nbsp;min to 6&nbsp;h at 25 °C. Intensities from triplicate experiments were globally fit to a 2:1 Zn:METAP1<sub>1-56</sub> binding model using DynaFit to give an upper bound estimate of the affinity (<span class="dropBlock reference-citations"><a id="body-ref-sref55-4" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-4" aria-controls="sref55-4">Kuzmic, 1996</a><div class="dropBlock__holder" data-db-target-of="sref55-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Kuzmic, P.</div><div><strong>Program DYNAFIT for the analysis of enzyme kinetic data: application to HIV proteinase</strong></div><div><em>Anal. Biochem.</em> 1996; <strong>237</strong>:260-273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/abio.1996.0238" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029902679" target="_blank" aria-label="Open Scopus in new tab">Scopus (1358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8660575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fabio.1996.0238&amp;pmid=8660575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with the <i>K</i><sub>Zn</sub> value of EDTA fixed to 7.6&nbsp;×&nbsp;10<sup>12</sup> M<sup>−1</sup>.</div></section><section id="sec-4-4-12"><h4>ZNG1 peptide synthesis</h4><div id="p0370" role="paragraph">The peptides were prepared as C-terminal amides and acetylated at the N-terminus by Fmoc solid-phase synthesis on Chem matrix Rink-Amide resin and purified by reverse-phase HPLC. Peptides were dried and resuspended in water. Calculated molar extinction coefficients at 214&nbsp;nm were used to determine peptide concentration.</div></section><section id="sec-4-4-13"><h4>NMR</h4><section id="sec-4-4-13-1"><h5>Assignments</h5><div id="p0375" role="paragraph">NMR samples contained 0.5-2&nbsp;mM uniformly <sup>15</sup>N- and <sup>15</sup>N<sup>13</sup>C-labeled protein, with 10 % v/v D2O and 0.3&nbsp;mM 2,2-dimethyl-2-silapentanesulfonic acid (DSS) as an internal reference. An equivalent 2&nbsp;mM, <sup>15</sup>N<sup>13</sup>C-labeled 100% D<sub>2</sub>O sample was prepared by lyophilization of a 100% H<sub>2</sub>O sample, followed by resuspension in D<sub>2</sub>O (Cambridge Isotope Laboratories). NMR spectra were recorded at 25 °C on a Bruker Avance Neo 600 MHz spectrometer equipped with a cryogenic probe or on a Varian VNMRS 800 MHz spectrometer with a room temperature 5&nbsp;mm PFG HCN probe in the METACyt Biomolecular NMR Laboratory at Indiana University, Bloomington. Data were collected using Topspin 4.0.9 (Bruker) or VNMRJ 4.2 (Varian), processed using NMRPipe (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Delaglio et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Delaglio, F. ∙ Grzesiek, S. ∙ Vuister, G.W. ...</div><div><strong>NMRPipe: a multidimensional spectral processing system based on UNIX pipes</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 1995; <strong>6</strong>:277-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00197809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029400480" target="_blank" aria-label="Open Scopus in new tab">Scopus (11573)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8520220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00197809&amp;pmid=8520220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and&nbsp;analyzed using CARA (<a id="intref0100" href="http://cara.nmr.ch/doku.php" target="_blank">http://cara.nmr.ch/doku.php</a>) and NMRFAM-Sparky (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Lee et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Lee, W. ∙ Tonelli, M. ∙ Markley, J.L.</div><div><strong>NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:1325-1327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84927748573" target="_blank" aria-label="Open Scopus in new tab">Scopus (1039)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25505092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu830&amp;pmid=25505092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), all on NMRbox (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Maciejewski et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Maciejewski, M.W. ∙ Schuyler, A.D. ∙ Gryk, M.R. ...</div><div><strong>NMRbox: a resource for biomolecular NMR computation</strong></div><div><em>Biophys. J.</em> 2017; <strong>112</strong>:1529-1534</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.bpj.2017.03.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.bpj.2017.03.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85018654124" target="_blank" aria-label="Open Scopus in new tab">Scopus (193)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28445744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2017.03.011&amp;pmid=28445744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0380" role="paragraph">Backbone chemical shifts were assigned for each state using the following standard triple-resonance experiments: HNcaCB, CBCAcoNH, and HNCO (<span class="dropBlock reference-citations"><a id="body-ref-sref86-1" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-1" aria-controls="sref86-1">Sattler, 1999</a><div class="dropBlock__holder" data-db-target-of="sref86-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sattler, M. ∙ Schleucher, J. ∙ Griesinger, C.</div><div><strong>Heteronuclear multidimensional NMR experiments for the structure determination of proteins in solution employing pulsed field gradients</strong></div><div><em>Prog. Nucl. Magn. Reson. Spectrosc.</em> 1999; <strong>34</strong>:93-158</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0079-6565%2898%2900025-9&amp;cf=fulltext&amp;site=jpnmrs-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0079-6565%2898%2900025-9&amp;cf=pdf&amp;site=jpnmrs-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0079-6565%2898%2900025-9" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Chemical shift assignments were not obtained for the C’ of K59 or C22 in the free or bound states because an HNCO does not include resonances for C’ in terminal residues or in residues preceding proline. An HNcaCO of the fusion construct provided these chemical shifts as well as those of C18 and V22 in the ZNG1 peptide, which also precede proline residues. With one proline in the C6H2 domain and two proline residues in the peptide, 100% of all possible amide <sup>1</sup>H, <sup>15</sup>N atoms are assigned in all three states. 97% of all possible C’ atoms are assigned in the free and bound forms, and 100% in the fusion. 100% of all possible Ca and Cb atoms are assigned in the free domain and fusion, but 98% of all Ca and Cb atoms in the bound form, without assignments for C22 due to chemical exchange broadening. The backbone assignments of METAP1<sub>1-59</sub> in the absence of peptide are consistent with and expand upon those previously published for human METAP1<sub>1-83</sub> (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Rachineni et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Rachineni, K. ∙ Arya, T. ∙ Singarapu, K.K. ...</div><div><strong>Chemical shift assignments of zinc finger domain of methionine aminopeptidase 1 (MetAP1) from Homo sapiens</strong></div><div><em>Biomol. NMR Assign.</em> 2015; <strong>9</strong>:351-353</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12104-015-9608-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84941935610" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25921012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12104-015-9608-0&amp;pmid=25921012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0385" role="paragraph">Aliphatic side-chain assignments were obtained using HBHAcoNH, HcccoNH-TOCSY, hCccoNH-TOCSY, HcCH-TOCSY, hCCH-TOCSY experiments (<span class="dropBlock reference-citations"><a id="body-ref-sref86-2" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-2" aria-controls="sref86-2">Sattler, 1999</a><div class="dropBlock__holder" data-db-target-of="sref86-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sattler, M. ∙ Schleucher, J. ∙ Griesinger, C.</div><div><strong>Heteronuclear multidimensional NMR experiments for the structure determination of proteins in solution employing pulsed field gradients</strong></div><div><em>Prog. Nucl. Magn. Reson. Spectrosc.</em> 1999; <strong>34</strong>:93-158</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0079-6565%2898%2900025-9&amp;cf=fulltext&amp;site=jpnmrs-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0079-6565%2898%2900025-9&amp;cf=pdf&amp;site=jpnmrs-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0079-6565%2898%2900025-9" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while hbCBcgcdHD, hbCBcgcdceHE, and H<sup>ar</sup>(CC-TOCSY-CGCBCACO)NH for were used for aromatic side-chains (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Löhr et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Löhr, F. ∙ Hänsel, R. ∙ Rogov, V.V. ...</div><div><strong>Improved pulse sequences for sequence specific assignment of aromatic proton resonances in proteins</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2007; <strong>37</strong>:205-224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-006-9128-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33947608952" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17237975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-006-9128-4&amp;pmid=17237975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">Yamazaki, 1993</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Yamazaki, T. ∙ Forman-Kay, J.D. ∙ Kay, L.E.</div><div><strong>Two-dimensional NMR experiments for correlating carbon-13.beta. and proton.delta./.epsilon. chemical shifts of aromatic residues in 13C-labeled proteins via scalar couplings</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 1993; <strong>115</strong>:11054-11055</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja00076a099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja00076a099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The 3D experiments were collected using non-uniform sampling with Poisson gap scheduling (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Hyberts et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Hyberts, S.G. ∙ Takeuchi, K. ∙ Wagner, G.</div><div><strong>Poisson-gap sampling and forward maximum entropy reconstruction for enhancing the resolution and sensitivity of protein NMR data</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2010; <strong>132</strong>:2145-2147</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja908004w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77950835733" target="_blank" aria-label="Open Scopus in new tab">Scopus (256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20121194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja908004w&amp;pmid=20121194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and reconstructed with istHMS (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Hyberts et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Hyberts, S.G. ∙ Milbradt, A.G. ∙ Wagner, A.B. ...</div><div><strong>Application of iterative soft thresholding for fast reconstruction of NMR data non-uniformly sampled with multidimensional poisson gap scheduling</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2012; <strong>52</strong>:315-327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-012-9611-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84863114255" target="_blank" aria-label="Open Scopus in new tab">Scopus (308)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22331404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-012-9611-z&amp;pmid=22331404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sidechain assignments of the peptide-C6H2 fusion construct are essentially complete, with the assignment of the single methionine methyl group confirmed by a 3D <sup>13</sup>C-NOESY-HSQC. Stereospecific assignments of methyl groups for all 3 valine residues and all 5 leucine residues in the fusion construct were obtained using a combination of high-resolution <sup>1</sup>H<sup>13</sup>C-HSQC and constant-time <sup>1</sup>H<sup>13</sup>C-HSQC of a 10% <sup>13</sup>C-labeled sample (<span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Hilty et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Hilty, C. ∙ Wider, G. ∙ Fernández, C. ...</div><div><strong>Stereospecific assignments of the isopropyl methyl groups of the membrane protein OmpX in DHPC micelles</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2003; <strong>27</strong>:377-382</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1025877326533" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0142149113" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14512734/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1025877326533&amp;pmid=14512734" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The chemical shifts and protonation states of the Nd and Ne atoms of the two histidine sidechains were assigned using a <sup>1</sup>H<sup>15</sup>N-HSQC experiment optimized for <sup>2</sup>J<sub>NH</sub> transfer (<span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Pelton et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Pelton, J.G. ∙ Torchia, D.A. ∙ Meadow, N.D. ...</div><div><strong>Tautomeric states of the active-site histidines of phosphorylated and unphosphorylated IIIGlc, a signal-transducing protein from Escherichia coli, using two-dimensional heteronuclear NMR techniques</strong></div><div><em>Protein Sci.</em> 1993; <strong>2</strong>:543-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.5560020406" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8518729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.5560020406&amp;pmid=8518729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), demonstrating that the Nd atoms are protonated and the Ne atoms coordinate Zn<sup>II</sup>. A small number of non-amide exchangeable protons were observed in 3D <sup>15</sup>N-NOESY-HSQC spectra of the fusion construct, and were attributed to the O<sup>g</sup>H hydroxyl groups of Ser58, Ser65, and Thr68. Chemical shifts were analyzed for indications of secondary structure using TALOSN (<span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Shen and Bax, 2015</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Shen, Y. ∙ Bax, A.</div><div><strong>Protein structural information derived from NMR chemical shift with the neural network program TALOS-N</strong></div><div><em>Methods Mol. Biol.</em> 2015; <strong>1260</strong>:17-32</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-2239-0_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84917706654" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25502373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-2239-0_2&amp;pmid=25502373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-13-2"><h5>Dynamics</h5><div id="p0390" role="paragraph">NMR relaxation measurements including <sup>15</sup>N spin relaxation rates, R<sub>1</sub> and R<sub>2</sub>, and <sup>1</sup>H-<sup>15</sup>N heteronuclear NOE (hNOE) values were recorded in an interleaved manner using standard experiments. The relaxation delays used were 0.02, 0.06, 0.10, 0.20, 0.40, 0.60, 0.80, and 1.2&nbsp;s for R<sub>1</sub> and 0.017, 0.034, 0.051, 0.068, 0.085, 0.102, 0.119, 0.136, 0.153, 0.170, 0.204, and 0.238&nbsp;s for R<sub>2</sub>. Residue-specific R<sub>1</sub> and R<sub>2</sub> values were obtained from fits of peak intensities vs. relaxation time to a single exponential decay function, while hNOE ratios were ascertained directly from intensities in experiments recorded with (2&nbsp;s relaxation delay followed by 3&nbsp;s saturation) and without saturation (relaxation delay of 5 s). Errors in hNOE values were calculated by propagating the error from the signal to noise.</div></section><section id="sec-4-4-13-3"><h5>Structure determination</h5><div id="p0395" role="paragraph">Initial structure calculations were performed with CYANA using automatically assigned NOESY peak lists with no further restraints, revealing the expected cross-brace zinc chelation topology (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Güntert, 2004</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Güntert, P.</div><div><strong>Automated NMR structure calculation with CYANA</strong></div><div><em>Methods Mol. Biol.</em> 2004; <strong>278</strong>:353-378</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1385/1-59259-809-9:353" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15318003/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1385%2F1-59259-809-9%3A353&amp;pmid=15318003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Zn coordination geometry was defined and maintained for subsequent calculations using distance restraints (Zn-Sg&nbsp;= 2.3&nbsp;Å and Zn-Ne2&nbsp;= 1.95&nbsp;Å). Hydrogen bond restraints were added where supported by NOEs, chemical shift-based secondary structure predictions, and slow rates of hydrogen-deuterium exchange. The structure has been deposited into the Protein Data Bank with accession ID 7SEK.</div></section></section><section id="sec-4-4-14"><h4>GTPase activity measurements (malachite green assay)</h4><div id="p0400" role="paragraph">Reactions were run in 25&nbsp;mM HEPES pH 7.4, 150&nbsp;mM KCl, 2&nbsp;mM MgCl<sub>2</sub>, 2&nbsp;mM TCEP. 20&nbsp;μL reactions of 5&nbsp;μM ZNG1 in the presence or absence of 5&nbsp;μM Zn and 5&nbsp;μM Zn<sub>2</sub>METAP1 were initiated with the addition of 500&nbsp;μM GTP (Sigma 10106399001) and incubated at 37 °C for 90&nbsp;min. GTPase reactions were quenched and inorganic phosphate was detected with the addition of 120&nbsp;μL of 1&nbsp;mM malachite green oxalate (Acros Organics), 10&nbsp;mM ammonium molybdate (Sigma-Aldrich) in 1&nbsp;M HCl. This reaction was incubated in the dark at ambient temperature for 10&nbsp;min and then quenched with the addition of 60&nbsp;μL 35% citric acid. Mixtures were incubated for 10&nbsp;min and inorganic phosphate concentration was calculated based on the absorbance at 680&nbsp;nm relative to a standard curve (5-500&nbsp;μM phosphate, Fluka Analytical).</div></section><section id="sec-4-4-15"><h4>GTPase activity measurements (coupled assay)</h4><div id="p0405" role="paragraph">GTPase activity was measured following a previously described protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Ingerman and Nunnari, 2005</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Ingerman, E. ∙ Nunnari, J.</div><div><strong>A continuous, regenerative coupled GTPase assay for dynamin-related proteins</strong></div><div><em>Methods Enzymol.</em> 2005; <strong>404</strong>:611-619</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0076-6879(05)04053-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-30544450009" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16413304/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0076-6879%2805%2904053-X&amp;pmid=16413304" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Reactions were run in 25&nbsp;mM HEPES pH 7.4, 150&nbsp;mM KCl, 2&nbsp;mM MgCl<sub>2</sub>, 2&nbsp;mM TCEP in a total volume of 100&nbsp;μL and included 4&nbsp;μM ZNG1, 500&nbsp;μM GTP (Sigma 10106399001), 260&nbsp;μM NADH (Sigma N8129), 2&nbsp;mM phosphoenolpyruvate (Sigma 860077), and 1&nbsp;μL pre-mixed Pyruvate Kinase / Lactate Dehydrogenase (Sigma P0294) at 37 °C. Reactions were monitored in real time by a decrease in absorbance at 340&nbsp;nm in a Synergy Neo2 plate reader.</div></section><section id="sec-4-4-16"><h4>Murine ZNG1 activation of METAP1 assay</h4><div id="p0410" role="paragraph">Activation assays were performed in 25&nbsp;mM HEPES pH 7.4, 150&nbsp;mM KCl, 2&nbsp;mM MgCl2, 2&nbsp;mM TCEP. 250&nbsp;μM NTA was included where indicated. 10&nbsp;μM Zn<sub>2</sub>METAP1 was incubated with 25&nbsp;μM apo or ZnZNG1 (ZNG1 loaded with 0.9 eq, 22.5&nbsp;μM, Zn) at 37 °C for 1 h. Reactions were diluted 500-fold in 25&nbsp;mM HEPES pH 7.4, 150&nbsp;mM NaCl, 2&nbsp;mM TCEP to a final concentration of 20&nbsp;nM METAP1 and methionine aminopeptidase activity was assayed as described above by initiating the enzymatic reaction with addition of 750&nbsp;μM MAHAIHY. When using GTP, GDP, or GDPNP, the activation reaction contained 500&nbsp;μM of the indicated nucleotide. A control activation reaction of Zn-NTA contained 22.5&nbsp;μM and 250&nbsp;μM NTA in reaction buffer.</div></section><section id="sec-4-4-17"><h4>Zebrafish Metap1 methionine aminopeptidase activity</h4><div id="p0415" role="paragraph">A single endpoint assay was used for these experiments. Reaction buffer (20&nbsp;mM HEPES (pH 7.5), 150&nbsp;mM NaCl) was chelex treated overnight at 4 °C. 1&nbsp;μM solutions of zebrafish Metap1 and Zng1 were made in reaction buffer. To strip Zn from the METAP1 active site, 3&nbsp;μM of TPEN was incubated with 1μM METAP1 at 4 °C for 1 h. To prepare holo-Zng1 (Zn bound), Zng1 was incubated with 10&nbsp;μM of ZnCl<sub>2</sub> for 1&nbsp;h at 4 °C. As a control, Metap1 was incubated with reaction buffer alone. Following the 1&nbsp;h incubation, Zng1 and Metap1 were washed four times with chelex-treated reaction buffer using protein concentration columns (Amicon Ultra, UFC500396). After washes, proteins were mixed and incubated at 37 °C for 30&nbsp;min. Following incubation, 100&nbsp;μM of the synthetic peptide MAHAIHY (Genscript) was added to reactions and incubated for 90&nbsp;min at 37 °C. Reactions were stopped by the addition of a final concentration of 2% trifluoroacetic acid and samples were centrifuged at max speed for 5&nbsp;min. Clear supernatant was transferred into fresh glass vials for HPLC analysis.</div><div id="p0420" role="paragraph">Twenty μL of each sample and standard was analyzed on an Agilent 1260 Infinity II system. Analytes were separated by gradient HPLC on a Supelco Ascentis Express C<sub>18</sub> column (50 x 2.1&nbsp;mm, 5&nbsp;μm) with a Phenomenex SecurityGuard C<sub>18</sub> cartridge (3.2 x 8&nbsp;mm) at a flow rate of 0.4&nbsp;mL/min using 0.1 % trifluoroacetic acid in water and 0.1% trifluoroacetic acid in acetonitrile as the A and B mobile phases, respectively. The gradient was held at 0% B for 2&nbsp;min, then ramped to 100% B over the next 18&nbsp;min. The column was washed at 100% B for 5&nbsp;min, then equilibrated to 0% B for 5&nbsp;min. MAHAIHY and AHAIHY were detected using absorbance at 280&nbsp;nm, and retention times were confirmed using each standard. Methionine aminopeptidase relative activity was determined by calculating the percent of MAHAIHY converted to AHAIHY using the AUC values at 280&nbsp;nm for each peptide.</div></section><section id="sec-4-4-18"><h4>Zebrafish mutagenesis</h4><div id="p0425" role="paragraph">Targeted deletion of the <i>zng1</i> and <i>metap1</i> genes were performed using CRISPR/Cas9 gene editing targeting the second exon of <i>zng1</i> and first exon of <i>metap1</i> as described elsewhere (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Essner, 2016</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Essner, J.</div><div><strong>Zebrafish Embryo Microinjection: Ribonucleoprotein Delivery Using the Alt-R CRISPR-Cas9 System</strong></div><div>Integrated DNA Technologies, 2016</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.EssnerZebrafish+Embryo+Microinjection%3A+Ribonucleoprotein+Delivery+Using+the+Alt-R+CRISPR-Cas9+System2016Integrated+DNA+Technologies" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, the guide RNA sequences 5’ GACCCACAGCTCAGATCC 3’, targeting zebrafish <i>zng1</i> and 5’ AGGTGGGACACTGGAGCT 3’ targeting <i>metap1</i> were designed using CRISPRscan (<span class="dropBlock reference-citations"><a id="body-ref-optUDRwk4IsEX" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optUDRwk4IsEX" aria-controls="optUDRwk4IsEX">Moreno-Mateos et al., 2015</a><div class="dropBlock__holder" data-db-target-of="optUDRwk4IsEX"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Moreno-Mateos, M. ∙ Vejnar, C. ∙ Beaudoin, J.-D. ...</div><div><strong>CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo</strong></div><div><em>Nat Methods.</em> 2015; <strong>12</strong>:982-988</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84959123021" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322839/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3543&amp;pmid=26322839" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and were synthesized using the oligo-based method (<span class="dropBlock reference-citations"><a id="body-ref-optYOx9AXlXJG" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optYOx9AXlXJG" aria-controls="optYOx9AXlXJG">Yin et al., 2015</a><div class="dropBlock__holder" data-db-target-of="optYOx9AXlXJG"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Yin, L. ∙ Jao, L.-E. ∙ Chen, W. ...</div><div><strong>Generation of Targeted Mutations in Zebrafish Using the CRISPR/Cas System</strong></div><div><em>Methods Mol Biol.</em> 2015; <strong>1332</strong>:205-217</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-2917-7_16" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84939507321" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26285757/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-2917-7_16&amp;pmid=26285757" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (P23-P25; <a id="crosref1565" href="#mmc5">Table&nbsp;S5</a>). Cas9 mRNA was generated from <i>Xba</i>I (New England Biolabs) digested pT3TS-nls-zCas9-nls plasmid (Addgene, 46757), and <i>in&nbsp;vitro</i> transcribed using mMESSAGE mMACHINE T3 kit (Thermo Fisher Scientific, AM1348) (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Jao et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Jao, L.E. ∙ Wente, S.R. ∙ Chen, W.</div><div><strong>Efficient multiplex biallelic zebrafish genome editing using a CRISPR nuclease system</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:13904-13909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1308335110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84882788354" target="_blank" aria-label="Open Scopus in new tab">Scopus (880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23918387/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1308335110&amp;pmid=23918387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A cocktail consisting of 150&nbsp;ng/μL of nls-zCas9-nls and 120&nbsp;ng/μL of gRNA, 0.05% phenol red, 120&nbsp;mM KCl, and 20&nbsp;mM HEPES (pH 7.0) was prepared, and approximately 1–2 nL was injected directly into the cell of one cell stage Tübingen zebrafish embryos. Mutagenesis was initially screened using Melt Doctor High Resolution Melting Assay (Thermo Fisher Scientific, 4409535), and identified two independent <i>zng1</i> alleles which were confirmed as deletions by Sanger sequencing of TOPO-cloned PCR products. Subsequent screening of the Δ11 (allele designation <i>vu2</i>) was performed by PCR (primers P26/P27; <a id="crosref1575" href="#mmc5">Table&nbsp;S5</a>) and products were resolved on 2% agarose Tris-Borate-EDTA (TBE) gels. Screening of the Δ5 allele (allele designation <i>vu1</i>) was performed by PCR amplification using primers P26 and P27 (<a id="crosref1580" href="#mmc5">Table&nbsp;S5</a>), followed by purification and <i>Dpn</i>II digest (New England Biolabs). All <i>zng1</i> mutant alleles result in the loss of a single <i>Dpn</i>II restriction site present in the WT sequence. Sanger sequencing identified a 17 base pair deletion in the <i>metap1</i> first exon (allele designation <i>vu3</i>) which was genotyped by PCR using P28/P29 (<a id="crosref1585" href="#mmc5">Table&nbsp;S5</a>) and resolved on 2% agarose Tris-Borate-EDTA (TBE) gels.</div></section><section id="sec-4-4-19"><h4>Zebrafish pharmacological manipulations</h4><div id="p0430" role="paragraph">For all experiments larval zebrafish were bred naturally and collected into embryo media and were maintained at 28 °C on a 14/10&nbsp;h light/dark cycle for duration of experiments. For survival experiments, zebrafish larvae were split into even densities into 6 or 12 multi-well plates in 5&nbsp;mL or 1&nbsp;mL embryo media respectively at 3 dpf. Stock solutions of METAP2 inhibitor TNP-470 (Sigma, T1455) and METAP1/2 inhibitor Bengamide B (Santa cruz, sc-397521A) were prepared in ACS-grade DMSO (Fisher Scientific, D136) and TPEN (Sigma, P4413) was prepared in ethanol and stored at -20 °C. TNP-470, Bengamide B, and/or TPEN were applied into embryo media as well as a DMSO and/or ethanol vehicle controls at 3 dpf. Zebrafish survival was monitored daily until 6 dpf and all moribund or dead animals were removed. For RT-qPCR experiments, larvae were split at 3 dpf into groups of 20-30 larvae in 20&nbsp;mL of embryo media supplemented either 20&nbsp;μM of TPEN or ethanol vehicle control. For imaging experiments, 6 dpf zebrafish larvae treated with 500&nbsp;nM to 1&nbsp;μM of Bengamide B or vehicle control were imaged by a Leica Thunder M205FCA stereomicroscope with a Leica DFC9000 GT camera.</div></section><section id="sec-4-4-20"><h4>Mouse mutagenesis</h4><div id="p0435" role="paragraph">Generation of <i>Zng1</i><sup><i>-/-</i></sup> mice. Zng1<sup>-/-</sup> mice were produced using CRISPR-Cas9 genome editing in a C57Bl/6J strain (The Jackson Laboratory) through the Vanderbilt Genome Editing Resource (Vanderbilt University). Ribonucleoprotein complexes containing crRNA&nbsp;+ tracrRNA (ctRNA) and SpCas9 protein (MilliporeSigma) targeting DNA sequences within <i>Zng1</i> exon 1 and a single stranded DNA donor containing an <i>Eco</i>RI restriction site were delivered by pronuclear injection into 1-cell C57BL/6J embryos. crRNA sequences were crRNA: 5’TGGAGTCACTGGTGTCCTGC crRNA: 5’CCGGGTACTTAGGTAATTAA<sup>∗</sup> and 180 nucleotide ssODN donor sequence: ctagaagagaggccccagcccctacctcgaagattcgtcctcggccaggaggcccgggtgcctgaggagacacggcagccccccttaGAATTCggacaccagtgactccagaactaaagagctgccgccagtcgcgtttccgcgacaggattcagtcggcgcggggctgatgatgtcacc. F0 founder animals were screened for precise <i>Zng1</i> deletions by standard PCR and restriction fragment length polymorphism assays. Deletions were confirmed by Sanger sequencing and verified by a second round of Sanger sequencing in the heterozygous N1 generation. N1 generation <i>Zng1</i><sup><i>-/-</i></sup> heterozygous mice containing the desired mutation were backcrossed at least three generations into the C57Bl/6J strain to allow for segregation of potential off-target editing events prior to familial breeding to produce homozygous offspring. Genotyping of mice from the <i>Zng1</i> mutant breeding colony was performed by Transnetyx using ear punches of individual animals and primer pair P30/P31 (<a id="crosref1590" href="#mmc5">Table&nbsp;S5</a>).</div></section><section id="sec-4-4-21"><h4>Cell line mutagenesis</h4><div id="p0445" role="paragraph">Targeted deletion of the first exon of murine <i>Zng1</i> was performed using the guide RNA ‘ATTCCAGTCACAATTGTCAC’. Clonal selection of mutant cells was performed by dilution plating. Individual clonal populations of cells were subsequently screened using primers P32 and P33 (<a id="crosref1595" href="#mmc5">Table&nbsp;S5</a>) followed by Sanger sequencing of TOPO-cloned PCR products. Two independent mutant clones were identified and harbored a homozygous Δ2/Δ2 base pair deletion (clone 1) and a Δ7/Δ2 base pair deletion (clone 2).</div></section><section id="sec-4-4-22"><h4>Transfection of <i>Zng1</i> mutant cells</h4><div id="p0450" role="paragraph">To generate linearized DNA for transfection, pcDNA3.1-C-(k)DYK-<i>Zng1</i><sub><i>murine</i></sub> was digested with <i>Sca</i>I. Cells were grown to 80-90% confluency in 6-well TC treated plates and transfected with lipofectamine reagent using manufacturer’s protocol (Lipofectamine 3000, Invitrogen L3000-008). Briefly, 2.5&nbsp;μg of linear DNA was mixed with P3000 reagent and opti-MEM media (Gibco, 11058-021) to prepare the DNA solution. DNA solution and lipofectamine reagent solutions were mixed at a 1:1 ratio and incubated for 15&nbsp;min at room temperature. DNA-lipid complexes were added to cells dropwise and cells were incubated overnight at 37 °C at 5% CO<sub>2</sub>. G418 (RPI G64500-10) was added at a concentration of 300&nbsp;μg/ml in complete DMEM/F12 50/50 complete cell culture media described above. G418 resistant cells were diluted and clonally expanded to identify clonal populations of transfected cells. Expression of full-length ZNG1 was validated using Western blot against ZNG1.</div></section><section id="sec-4-4-23"><h4>Mass spectrometry proteomics of murine tissue and cell lines</h4><div id="p0455" role="paragraph">Kidneys were isolated from 11-13-week old female C57BL/6 or C57BL/6 <i>Zng1</i><sup><i>-/-</i></sup> mice that were fed a low Zn diet for 5&nbsp;weeks. Animals were humanely euthanized, kidneys isolated, and flash frozen in liquid nitrogen. For protein isolation, frozen organs were gently thawed on ice and transferred through a 70&nbsp;μM sterile cell strainer (Thermo Fisher Scientific, 22-363-548) into 5&nbsp;mL of ice-cold PBS supplemented with Halt protease inhibitor (Thermo Fisher Scientific, 78430). Filters were flushed an additional two times with PBS with protease inhibitor. Cells were pelleted (4 °C, 400x g, 10&nbsp;min) and pellets resuspended in 500&nbsp;μL of RIPA buffer with Halt protease inhibitor (Thermo Fisher Scientific, 78430). Following rigorous vortexing, incubation on ice (10&nbsp;min), and a second round of vortexing, cellular debris were pelleted by centrifugation (4 °C, 10,000x g, 10&nbsp;min) and supernatant transferred into fresh tubes for further analysis.</div><div id="p0460" role="paragraph">Protein concentrations were determined using the 660 Protein Assay (Pierce, 22660). Equal amounts of protein (200&nbsp;μg) were processed for LC-MS/MS using s-traps (Protifi) (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Hailemariam et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hailemariam, M. ∙ Eguez, R.V. ∙ Singh, H. ...</div><div><strong>S-trap, an ultrafast sample-preparation approach for shotgun proteomics</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2018; <strong>17</strong>:2917-2924</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jproteome.8b00505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85052299545" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30114372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jproteome.8b00505&amp;pmid=30114372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">Zougman et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Zougman, A. ∙ Selby, P.J. ∙ Banks, R.E.</div><div><strong>Suspension trapping (STrap) sample preparation method for bottom-up proteomics analysis</strong></div><div><em>Proteomics.</em> 2014; <strong>14</strong></div><div>1006–1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pmic.201300553" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84898664545" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24678027/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpmic.201300553&amp;pmid=24678027" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, proteins were reduced with dithiothreitol (DTT), alkylated with iodoacetamide (IAA), acidified using phosphoric acid, and combined with s-trap loading buffer (90% MeOH, 100mM TEAB). Proteins were loaded onto s-traps, washed, and finally digested with Trypsin/Lys-C (1:100, w:w; enzyme:protein) overnight at 37 °C. Peptides were eluted and dried with a vacuum concentrator. Peptides were resuspended in H2O/1% acetonitrile/0.1% formic acid for LC-MS/MS analysis.</div><div id="p0465" role="paragraph">Peptides were separated using a 75&nbsp;μm x 50&nbsp;cm C18 reversed-phase-HPLC column (Thermo Fisher Scientific, 164570) on an Ultimate 3000&nbsp;UHPLC (Thermo Fisher Scientific) with a 120&nbsp;min gradient (2-32% ACN with 0.1% formic acid) and analyzed on a hybrid quadrupole-Orbitrap instrument (Q Exactive Plus, Thermo Fisher Scientific). Full MS survey scans were acquired at 70,000 resolution. The top 10 most abundant ions were selected for MS/MS analysis.</div><div id="p0470" role="paragraph">Raw data files were processed in MaxQuant (v 1.6.14.0, <a id="intref0105" href="http://www.maxquant.org" target="_blank">www.maxquant.org</a>) and searched against the current UniProt <i>Mus musculus</i> protein sequences database. Search parameters included constant modification of cysteine by carbamidomethylation and the variable modifications, methionine oxidation and protein N-term acetylation. Proteins were identified using the filtering criteria of 1% protein and peptide false discovery rate. Protein intensity values were normalized using the MaxQuant LFQ function (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Cox et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Cox, J. ∙ Hein, M.Y. ∙ Luber, C.A. ...</div><div><strong>Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ</strong></div><div><em>Mol. Cell. Proteomics.</em> 2014; <strong>13</strong>:2513-2526</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M113.031591&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M113.031591&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84907197082" target="_blank" aria-label="Open Scopus in new tab">Scopus (2744)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24942700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M113.031591&amp;pmid=24942700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0475" role="paragraph">Label free quantitation analysis was performed using Perseus (v 1.6.14.0), software developed for the analysis of -omics data (<span class="dropBlock reference-citations"><a id="body-ref-sref95-1" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-1" aria-controls="sref95-1">Tyanova et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref95-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Tyanova, S. ∙ Temu, T. ∙ Sinitcyn, P. ...</div><div><strong>The Perseus computational platform for comprehensive analysis of (prote)omics data</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:731-740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84976274986" target="_blank" aria-label="Open Scopus in new tab">Scopus (3641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27348712/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3901&amp;pmid=27348712" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). LFQ Intensity values were Log<sub>2</sub>-transformed, and then filtered to include proteins containing at least 60% valid values (reported LFQ intensities) in at least one experimental group. Finally, the missing values in the filtered dataset were replaced using the imputation function in Perseus with default parameters (<span class="dropBlock reference-citations"><a id="body-ref-sref95-2" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-2" aria-controls="sref95-2">Tyanova et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref95-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Tyanova, S. ∙ Temu, T. ∙ Sinitcyn, P. ...</div><div><strong>The Perseus computational platform for comprehensive analysis of (prote)omics data</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:731-740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84976274986" target="_blank" aria-label="Open Scopus in new tab">Scopus (3641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27348712/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3901&amp;pmid=27348712" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Statistical analyses were carried out using the filtered and imputed protein groups file. Statistically significant changes in protein abundance are determined using Welch’s t-test <i>P</i>-values and <i>Z</i> scores.</div></section><section id="sec-4-4-24"><h4>Mass spectrometry proteomic quantification of ZNG1 peptides</h4><div id="p0480" role="paragraph">Murine brain tissue was homogenized in NP40 buffer (50&nbsp;mM trizma base, 150&nbsp;mM sodium chloride, 1&nbsp;mM EDTA, and 1% nonidet 40) using cycles of sonication. Protein concentration was quantified using a Bicinchoninic Acid assay (Sigma Aldrich, QPBCA). For each sample, 10&nbsp;μg of protein was suspended in 8&nbsp;M urea containing 50mM ammonium bicarbonate to a final volume of 90&nbsp;μL. Proteins were reduced using a final concentration of 20&nbsp;mM dithiothreitol (DTT, Thermo Scientific A39255) at 37 °C for 45&nbsp;min, followed by alkylation using 20&nbsp;mM iodoacetamide (IAA, Thermo Scientific A39271) at room temperature for 15&nbsp;min. Samples were diluted to reduce the concentration of urea to &lt;1&nbsp;M using 50&nbsp;mM ammonium bicarbonate and proteins were digested using 1&nbsp;μg of LysN (Thermo Scientific 90300) at 50 °C for 2 h. Oasis mixed-mode cation exchange (MCX) solid phase extraction cartridges (Waters Corporation, Milford, MA 186000252) were used to remove lipids and other contaminants from the digested peptides. Peptide samples were dried using a centrifugal evaporator.</div><div id="p0485" role="paragraph">Dried digested samples were reconstituted in HPLC-grade water containing 0.1% formic acid by vortexing and centrifuged briefly to remove any particulates. A total of 250&nbsp;ng of each sample was desalted and concentrated using C18 EvoTips (EvoSep EV2001). Liquid chromatography mass spectrometry was performed using an EvoSepOne (EvoSep) coupled to an Orbitrap Fusion (ThermoFisher Scientific) equipped with a Nanospray Flex Ion source. Chromatography was performed using a 15&nbsp;cm long column containing ReproSil-Pur C18 1.9&nbsp;μm beads by Dr Maisch C18 and a fused silica emitter tip (20&nbsp;μm inner diameter) (EvoSep EV1112, EV1111). Peptides were eluted from the column using an EvoSep 20 samples/day Whisper method. Mass spectrometry was performed on an Orbitrap Fusion (ThermoFisher Scientific) equipped with a Nanospray Flex Ion source. MS1 was performed using Orbitrap detection at a resolution of 60,000 with a standard AGC target. Data independent acquisition was performed using quadropole isolation with nonoverlapping windows of 16 m/z from precursor masses 400-1008 with HCD fragmentation (33% collision energy) and Orbitrap detection with a resolution of 30,000. The normalized peak areas for ZNG1 peptides were calculated using Skyline (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">MacLean et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>MacLean, B. ∙ Tomazela, D.M. ∙ Shulman, N. ...</div><div><strong>Skyline: an open source document editor for creating and analyzing targeted proteomics experiments</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:966-968</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77951965920" target="_blank" aria-label="Open Scopus in new tab">Scopus (3010)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20147306/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq054&amp;pmid=20147306" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-25"><h4>Gene expression analysis</h4><div id="p0490" role="paragraph">For RNA isolation from dissected mouse tissue, mice were humanely euthanized, tissues harvested and immediately flash frozen in liquid nitrogen and stored at -80 °C. Mouse tissues were transferred into Lysing matrix B tubes (MP, 6911100) containing 700&nbsp;μL of RLT buffer (Qiagen, 74104) with 1% (v/v) Β-mercaptoethanol. Samples were homogenized by bead beating 6 times for 40&nbsp;s at 6&nbsp;m/s. Tubes were kept on ice for 1&nbsp;min in between steps. Samples were centrifuged and homogenate was added to a fresh tube containing 600&nbsp;μL of phenol:chloroform:iAA (Fisher Scientific, BP17521-100) then vortexed for 30 s. Samples were spun at max speed for 15&nbsp;min and the aqueous phase was transferred to a tube containing 700&nbsp;μL of 100% ethanol. Subsequent isolation steps were performed using the Qiagen RNeasy Kit following manufacturer's protocol (Qiagen). gDNA was depleted from final RNA samples by treatment with TURBO DNA-free Kit (Invitrogen). For RNA isolation from zebrafish, pools of 20–30 whole larvae were collected into 1&nbsp;mL of TRIzol (Thermo Fisher Scientific, 15596026) and stored at -80 °C. For TKPTS cell lines, cells were washed with PBS and collected with 500&nbsp;μL TRIzol from multi-well plates and stored at -80 °C. Zebrafish larvae and cell lines were homogenized by passing samples 10–15 times through a 27-gauge needle. RNA was isolated following the manufacturer’s protocol with the following modification: a second wash with 70% ethanol (prepared in DEPC-treated H<sub>2</sub>O) was performed. RNA was treated with DNaseI following manufacturer’s instructions (New England Biolabs, M0303L). cDNA from RNA isolated by any of the methods described above was synthesized using the iScript kit (Bio-Rad, 1708891). RT-qPCR was performed in duplicate-quadruplicate 25&nbsp;μL reactions using 2X iQ SYBR Green SuperMix (BioRad, 1708882) run on an BioRad CFX96 Real-Time System instrument using gene-specific primers (zebrafish: P34-P39, mice: P40-P43, cell lines: 44-69; <a id="crosref1630" href="#mmc5">Table&nbsp;S5</a>). Data were analyzed with the ΔΔCt method (<span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Livak and Schmittgen, 2001</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Livak, K.J. ∙ Schmittgen, T.D.</div><div><strong>Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta C(T)) method</strong></div><div><em>Methods.</em> 2001; <strong>25</strong>:402-408</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/meth.2001.1262" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0035710746" target="_blank" aria-label="Open Scopus in new tab">Scopus (125166)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fmeth.2001.1262&amp;pmid=11846609" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or presented as relative expression values.</div></section><section id="sec-4-4-26"><h4>Immunofluorescence staining of cells</h4><div id="p0495" role="paragraph">WT and <i>Zng1</i> mutant TKPTS cells were seeded onto sterile coverslips in 6-well tissue culture treated multi-well plates. At 60-80% confluency, cells were washed three times with warm phenol red free DMEM (Thermo Fisher Scientific, 21041025) prior to fixation with 4% paraformaldehyde solution (PFA, Alfa aesar, J61899) for 15&nbsp;min at 37 °C then washed three times for 5&nbsp;min with PBS. Cells were subsequently permeabilized by incubation in 0.2% Triton X-100 in PBS for 10&nbsp;min at room temperature. Following permeabilization, cells were washed three times for 5&nbsp;min with PBS at room temperature. Cells were blocked using Odyssey (PBS) blocking buffer (LI-COR) for 1&nbsp;h at room temperature or overnight at 4 °C. Primary antibodies were diluted in Odyssey blocking solution and incubated overnight at 4 °C rocking (anti-OxPhos Complex V inhibitor Clone 5E2, Invitrogen A21355, 1:200). Cells were washed three times with PBS for 5&nbsp;min. Secondary antibodies (Thermo Fisher Scientific) and Hoechst (Invitrogen, H3570) were added to Odyssey blocking buffer (Hoechst, 1:1,000, Alexa -conjugated secondaries 1:2000) and cells were incubated for 2&nbsp;h rocking at room temperature. Following secondary staining, cells were washed three times with PBS and mounted onto slides with 20 uL of ProLong Gold antifade reagent (Invitrogen, 2273617). Slides cured in the dark overnight at room temperature prior to imaging. Slides were imaged on the Zeiss LSM 880 using the 63X oil objective (1.40 Plan-APOCHROMAT OIL, WD=0.19mm).</div></section><section id="sec-4-4-27"><h4>Determination of metal levels by inductively coupled plasma mass spectrometry (ICP-MS)</h4><div id="p0500" role="paragraph">Murine tissue was homogenized in PBS supplemented with 1% IGEPAL using a Bullet Blender (Next Advance) at 4 °C. Mouse tissue homogenate were acid digested in 400&nbsp;μL of Optima grade nitric acid (Thermo Fisher Scientific, A467-500) and 100 μL of Ultratrace hydrogen peroxide (Sigma, 95321) at 65 °C for 48 h. After digestion, the acid content was diluted to below 10% with 3.5&nbsp;mL of UltraPure water (Invitrogen, 10977-023). Elemental quantification on acid-digested samples was performed using an Agilent 7700 inductively coupled plasma mass spectrometer (Agilent) attached to a Teledyne CETAC Technologies ASX-560 autosampler (Teledyne CETAC Technologies). The following settings were fixed for the analysis Cell Entrance&nbsp;= -40 V, Cell Exit&nbsp;= -60 V, Plate Bias&nbsp;= -60 V, OctP Bias&nbsp;= -18 V, and collision cell Helium Flow&nbsp;= 4.5&nbsp;mL/min. Optimal voltages for Extract 2, Omega Bias, Omega Lens, OctP RF, and Deflect were determined empirically before each sample set was analyzed. Element calibration curves were generated using ARISTAR ICP Standard Mix (VWR, BDH82026-108). Samples were introduced by peristaltic pump with 0.5&nbsp;mm internal diameter tubing through a MicroMist borosilicate glass nebulizer (Agilent). Samples were initially up taken at 0.5 rps for 30&nbsp;s followed by 30&nbsp;s at 0.1 rps to stabilize the signal. Samples were analyzed in Spectrum mode at 0.1 rps collecting three points across each peak and performing three replicates of 100 sweeps for each element analyzed. Sampling probe and tubing were rinsed for 20&nbsp;s at 0.5 rps with 2% nitric acid between every sample. Data were acquired and analyzed using the Agilent Mass Hunter Workstation Software version A.01.02.</div></section><section id="sec-4-4-28"><h4>Transmission electron microscopy</h4><div id="p0505" role="paragraph">WT and <i>Zng1</i> mutant TKPTS cells treated with 3&nbsp;μM TPA (Sigma, 723134) were fixed in 2.5% glutaraldehyde in 0.1&nbsp;M cacodylate for 1&nbsp;h at room temperature followed by 24&nbsp;h at 4 °C. The cells were sequentially postfixed in 1% tannic acid, followed by 1% OsO4, then enblock stained with 1% uranyl acetate. Samples were dehydrated in a graded ethanol series and gradually infiltrated with a Quetol 651 formulation Spurrs resin using propylene oxide as the transition solvent. The resin was polymerized at 60 °C for 48 h. Thin sections were cut using a Leica UC7 ultramicrotome with a nominal thickness of 70&nbsp;nm and collected onto 300 mesh copper grids. All samples were post-stained with 2% uranyl acetate and lead citrate.</div><div id="p0510" role="paragraph">TEM imaging was performed on a Tecnai T12 operating at 100 keV using an AMT nanosprint CMOS camera. Single images were acquired using the AMT Capture Engine software. Images for quantification were acquired using serialEM to acquire montages of entire gridsquares, using identical lens currents between samples. These montages were reconstructed using the IMOD/etomo software suite. Mitochondria size, shape, and electron density quantification was performed using the FIJI ROI manager to manually segmentation all mitochondrial cross-sections within randomly selected cells until at least 100 mitochondria were measured for each treatment.</div></section><section id="sec-4-4-29"><h4>Flow cytometry measurement of cell proliferation and mitochondrial superoxide</h4><div id="p0515" role="paragraph">TKPTS cells were seeding at 2,000 to 4,000 cells per well of 24-or 12-well tissue culture treated multi-well plates and incubated overnight at 37 °C. Cells were staining with CellTRACE violet (Invitrogen, C34557) as described by manufacturer's protocol. Briefly, CellTRACE solution was prepared in PBS at a 1:1,000 dilution, added to the cells, and cells were incubated for 20&nbsp;min at 37 °C. Cells were then washed two times in cell culture media supplemented with 7% FBS, and replaced with fresh cell culture media supplemented with desired chemical treatments (3&nbsp;μM TPA, Sigma, 723134; 5&nbsp;μg/ml TNP470, Sigma, T1455; or 50&nbsp;nM Bengamide B, Santa cruz, sc-397521A). Stock solutions of drugs were prepared in ACS-grade DMSO (Fisher Scientific, D136). For growth in Zn deplete conditions, TPA was chosen because it exhibits lower cytotoxicity towards cell lines compared to other chelators (e.g. TPEN) (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Huang et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Huang, Z. ∙ Zhang, X.A. ∙ Bosch, M. ...</div><div><strong>Tris(2-pyridylmethyl)amine (TPA) as a membrane-permeable chelator for interception of biological mobile zinc</strong></div><div><em>Metallomics.</em> 2013; <strong>5</strong>:648-655</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c3mt00103b" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84878785188" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23715510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc3mt00103b&amp;pmid=23715510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Lo et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Lo, M.N. ∙ Damon, L.J. ∙ Wei Tay, J. ...</div><div><strong>Single cell analysis reveals multiple requirements for zinc in the mammalian cell cycle</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:e51107</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.51107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85078876811" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32014109/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.51107&amp;pmid=32014109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After three days of growth, cells were stained with MitoTracker far-red (Invitrogen, M22426; 1:2500 dilution for 20&nbsp;min at 37 °C) and MitoSOX red (Molecular probes, M36008 : 1,1000 for 20&nbsp;min at 37 °C) following manufacture’s protocols. Cells were washed with PBS, trypsinized, and transferred to Eppendorf tubes prior to fixation in 4% PFA (Alfa aesar, J61899). Cells were fixed for 20&nbsp;min at room temperature, centrifuged at 200x g for 10&nbsp;min and washed with PBS prior to resuspension in 300&nbsp;μL FACS buffer (2% FBS, 0.02% sodium azide in PBS). All flow cytometry data were collected using a BD LSRII flow cytometer with FACSDIVA software and analyzed using FlowJo (FlowJo LLC). Samples were gated forward scatter height (FSC-H) by forward scatter area (FSC-A) and side scatter height (SSC-H) by side scatter area (SSC-A) to remove doublet populations. The singlet population was gated SSC-A by FSC-A to isolate live cells. The resulting cell population was then assessed for assay-specific fluorescent markers and the median fluorescence intensity (MFI) quantified.</div></section><section id="sec-4-4-30"><h4>Seahorse XF96 cell mitochondrial stress test</h4><div id="p0520" role="paragraph">8,000 WT or <i>Zng1</i> mutant TKPTS cells (vehicle treated or serially passaged in 3&nbsp;μM TPA [Sigma, 723134]) were seeded in individual wells of seahorse plates in complete DMEM-F12 50/50 media described above. Assay was performed following manufacturer's instructions. Briefly, cells were resuspended in mitochondria assay media (Seahorse Agilent pH 7.4 media&nbsp;+ 1&nbsp;mM pyruvate&nbsp;+ 10&nbsp;mM glucose&nbsp;+ 2&nbsp;mM glutamine). No TKPTS cells were added to the four wells in the corners of the plate as a background control. For the mitochondrial stress test, the cartridge was sequentially loaded with oligomycin (15μM), FCCP (15&nbsp;μM), and rotenone/antimycin A (5&nbsp;μM) (diluted in mitochondria assay media) for injection into the wells. To normalize across wells by cell density, the Seahorse plate was imaged using a BioTek Cytation 5. Oxygen consumption and extracellular acidification rates were collected using a Seahorse XFe96 Analyzer and analyzed using Wave Desktop software (Agilent Technologies).</div></section><section id="sec-4-4-31"><h4>Measuring cellular ATP production</h4><div id="p0525" role="paragraph">Cellular ATP levels were measured from WT and <i>Zng1</i> mutant TKPTS cell lysates following manufacturer’s protocols (Sigma, MAK113). Briefly, 20,000 cells/well WT and <i>Zng1</i> mutant cells were seeded in clear-bottom black 96-well plates (Costar, 3603) and incubated for 36&nbsp;h at 37 °C under 5% CO<sub>2</sub>. ATP levels were measured using Cell Titer Glo 2.0 (Promega, G9241). To normalize to cell number, cells were incubated with Hoechst (1:1,000 in PBS) for 15&nbsp;min at 37 °C in cell culture incubator then washed with PBS prior to the addition of 100&nbsp;μL of fresh PBS. Hoechst signal was measured in the BioTek Cytation 5 (ex/em 350/461). Subsequently, 100&nbsp;μL of Cell Titer Glo reagent was applied directly to the wells and incubated in the dark for 15&nbsp;min at room temperature. Luminescence was measured in the BioTeck Cytation 5 and expressed as relative light units (RLU).</div></section><section id="sec-4-4-32"><h4>Tetramethylrhodamine Methyl Ester (TMRM) Assay</h4><div id="p0530" role="paragraph">WT and <i>Zng1</i> mutant TKPTS cells untreated or treated with 3&nbsp;μM TPA (Sigma, 723134) were seeded at a density of 4,000 cells / well in clear-bottom black 96-well plates (Costar, 3603) and incubated overnight at 37 °C under 5% CO<sub>2</sub>. Adherent cells were stained with MitoTracker deep red (Invitrogen, M22426) as described above. TMRM (Invitrogen, T668) staining solution was prepared following manufacturer's instructions. Briefly 100&nbsp;mM TMRM was prepared in phenol red-free cell growth medium and was added to adherent cells at a quenching concentration of 200&nbsp;nM as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Monteith et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Monteith, A. ∙ Marszalec, W. ∙ Chan, P. ...</div><div><strong>Imaging of mitochondrial and non-mitochondrial responses in cultured rat hippocampal neurons exposed to micromolar concentrations of TMRM</strong></div><div><em>PLoS One.</em> 2013; <strong>8</strong>:e58059</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0058059" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84874583009" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23483968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0058059&amp;pmid=23483968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells were incubated for 1-2&nbsp;h at 37 °C at 5% CO<sub>2</sub>. TMRM and MitoTracker fluorescence was measured using a BioTek Cytation 5 (TMRM- ex/em 548&nbsp;nm/574&nbsp;nm and MitoTracker- ex/em 644&nbsp;nm/665&nbsp;nm). For each genotype and treatment condition, 4-8 independent wells were analyzed and TMRM signal was normalized to MitoTracker signal.</div></section><section id="sec-4-4-33"><h4>Computational tools</h4><div id="p0535" role="paragraph">Sequence analysis, alignments and cladograms were created using CLC Genomics Workbench (version 20.0.1.) (Qiagen). Visualization of pairwise alignments was generated using Sequence Demarcation Tool (SDTv1.2) (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Muhire et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Muhire, B.M. ∙ Varsani, A. ∙ Martin, D.P.</div><div><strong>SDT: a virus classification tool based on pairwise sequence alignment and identity calculation</strong></div><div><em>PLoS One.</em> 2014; <strong>9</strong>:e108277</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0108277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84907426295" target="_blank" aria-label="Open Scopus in new tab">Scopus (945)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25259891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0108277&amp;pmid=25259891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Structural multiple sequence alignments were generated using ESPript (<span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Robert and Gouet, 2014</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Robert, X. ∙ Gouet, P.</div><div><strong>Deciphering key features in protein structures with the new ENDscript server</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:W320-W324</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku316" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84904790793" target="_blank" aria-label="Open Scopus in new tab">Scopus (3682)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24753421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku316&amp;pmid=24753421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An iterative HMMsearch was used to identify variations in the ZNG1 N-terminal motif (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Eddy, 2009</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Eddy, S.R.</div><div><strong>A new generation of homology search tools based on probabilistic inference</strong></div><div><em>Genome Inform.</em> 2009; <strong>23</strong>:205-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1142/9781848165632_0019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20180275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1142%2F9781848165632_0019&amp;pmid=20180275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). HMMsearch was also used to identify matches to the refined motif in yeast, human, mouse, and zebrafish genomes. Sequence logo motifs were generated using WebLogo 3.7.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Crooks et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Crooks, G.E. ∙ Hon, G. ∙ Chandonia, J.M. ...</div><div><strong>WebLogo: a sequence logo generator</strong></div><div><em>Genome Res.</em> 2004; <strong>14</strong>:1188-1190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.849004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-2142738304" target="_blank" aria-label="Open Scopus in new tab">Scopus (8336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15173120/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.849004&amp;pmid=15173120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). AlphaFold2 modeling of the yeast Map1p-Zng1p interaction was performed using the ColabFold AlphaFold2_advanced jupyter notebook with max_recycles set to 48 (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Jumper et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Jumper, J. ∙ Evans, R. ∙ Pritzel, A. ...</div><div><strong>Highly accurate protein structure prediction with AlphaFold</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:583-589</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03819-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85110248105" target="_blank" aria-label="Open Scopus in new tab">Scopus (5905)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03819-2&amp;pmid=34265844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Mirdita et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Mirdita, M. ∙ Ovchinnikov, S. ∙ Steinegger, M.</div><div><strong>ColabFold—making protein folding accessible to all</strong></div><div><em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.08.15.456425" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85119536212" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.08.15.456425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Pathway analysis of proteins with significantly altered abundance (as determined by Welch’s t-test) was performed using the Ingenuity Pathway Analysis tool (IPA, Qiagen).</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0540" role="paragraph">All experiments were repeated at least two times and statistical analyses were performed with GraphPad Prism v.9 or OriginPro. Data are presented as mean ± SEM unless otherwise noted in figure legend. For comparisons between 2 groups a two tailed student’s t-test or Mann-Whitney test was applied. For comparisons between 3 or more groups, a one-way ANOVA with Tukey’s multiple comparisons test was used. For experiments with 2 independent variables, a two-way ANOVA was performed with Tukey’s or Sidak’s multiple comparisons test. Significance was set <i>as P</i>&nbsp;&lt;&nbsp;0.05, and denoted as: <sup>∗</sup> <i>P</i>&nbsp;&lt;&nbsp;0.05, <i><sup>∗∗</sup> P</i>&nbsp;&lt;&nbsp;0.01, <i><sup>∗∗∗</sup> P</i>&nbsp;&lt;&nbsp;0.001, <i><sup>∗∗∗∗</sup>&nbsp;P</i>&nbsp;&lt;&nbsp;0.0001. Sample sizes are indicated in the figure legends.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0545" role="paragraph">This research was supported by the NIH grants R01AI150701 and R01AI101171 (E.P.S., W.J.C., and D.P.G.), as well as R35GM118157 (D.P.G.), T32ES007028 (A.W. and C.C.M.), F32AI157215 (A.W.), F32GM142246 (C.C.M.), F32HL144081 (A.J.M.), T32GM109825 (M.R.J.), T32GM131994 (M.R.J.), T32DK101003 (A.R.S.K.), R01AI138581, and R01AI145992 (J.M.S. and R.M.C.). E.P.S. was supported by the Ernest W. Goodpasture Chair in Pathology and the National Cancer Institute (NCI) Vanderbilt-Ingram Cancer Center Support Grant (P30CA068485). A.W. (18POST33990262) and W.N.B. (18POST34030426) were supported by the American Heart Association. Generation of <i>Zng1</i><sup><i>−/−</i></sup> mice by the Vanderbilt Genome Editing Resource was supported by the Cancer Center Support Grant (CA68485), the Vanderbilt Diabetes Research and Training Center (DK020593), and the Center for Stem Cell Biology. Confocal microscopy and TEM imaging was performed in collaboration with the Vanderbilt Cell Imaging Shared Resource (supported by NIH grants CA68485, DK20593, DK58404, DK59637, and EY08126). Flow Cytometry at the VUMC Flow Cytometry Shared Resource was supported by the Vanderbilt Ingram Cancer Center (P30 CA68485) and the Vanderbilt Digestive Disease Research Center (DK058404). The funders had no role in the study design, data collections and analysis, decision to publish, or preparation of the manuscript. We are grateful for Rachel Hart’s assistance in TEM image collection and for technical assistance by Sophia A. Kvaal. The authors thank the University of South Florida CDDI Proteomics Facility for their support. We thank the members of the Skaar lab, as well as Dr. Borden Lacy, Dr. James Cassat, and Dr. Maulik Patel for critical review of the manuscript.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0550" role="paragraph">Conceptualization, A.W., C.C.M., K.A.E., M.R.J., D.P.G., and E.P.S.; investigation, A.W., C.C.M., K.A.E., M.R.J., A.J.M., Y.R.P., A.M.R.N., A.M.P., W.N.B., S.L.D., M.J.M., E.S.K., A.R.S.K., and H.W.; formal analysis, A.W., C.C.M., K.A.E., M.R.J., and E.S.K.; resources, K.T. and R.D.D; supervision, R.D.D., J.M.S., R.M.C., W.J.C., D.P.G., and E.P.S.; writing – original draft, A.W., C.C.M., K.A.E., M.R.J., A.J.M., W.J.C., D.P.G., and E.P.S.; writing – review &amp; editing, all authors.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0555" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (6)</h2><div id="p0565" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.04.011/attachment/c907db87-73ec-44eb-918c-40afc0df9de2/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (107.08 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table&nbsp;S1. Overview of interaction partners of human, murine and zebrafish ZNG1 as identified by yeast-two-hybrid screen, related to Figure&nbsp;1</div><div id="espara0010" role="paragraph">SID, Selected Interacting domain; NA, No predicted Pfam/SMART domain overlapping with the SID</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.04.011/attachment/4c443444-812a-4dac-8cd0-06a03aa7721b/mmc2.pdf" download="mmc2.pdf" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">PDF (102.31 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table&nbsp;S2. Structural statistics for NMR solution structure of the fusion of ZNG<sub>110–30</sub> with METAP1<sub>1–59</sub>, related to Figures&nbsp;3 and S3</div><div id="espara0015" role="paragraph"><sup>a</sup>Calculated over the core of the structure, excluding the flexible N and C termini and the linker region, including ZNG1 residues 17–25 and METAP1 residues 7–53.</div><div id="espara0020" role="paragraph"><sup>b</sup>As computed by Procheck.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.04.011/attachment/6e76340c-de41-48a1-af20-e295da93f3b5/mmc3.pdf" download="mmc3.pdf" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">PDF (58.96 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table&nbsp;S3. Proteomics hits from low zinc diet, related to Figures&nbsp;6 and S6</div><div id="espara0025" role="paragraph"><sup>∗</sup> denotes proteins with known or predicted mitochondrial localization.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.04.011/attachment/fc173f84-c37f-48a8-b72e-207a9266f0ed/mmc4.zip" download="mmc4.zip" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Archive (3.79 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table&nbsp;S4. Proteomics analysis of WT and <i>Zng1</i><sup><i>−/−</i></sup> TKPTS cells grown in the presence of absence of TPA or Bengamide B, related to Figures&nbsp;7 and S7</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.04.011/attachment/a238fb9d-a6e0-4e98-ad5f-57c2ad9ffcb5/mmc5.pdf" download="mmc5.pdf" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">PDF (55.14 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table&nbsp;S5. Primers used in this study, related to STAR Methods</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.04.011/attachment/2d599bf2-585d-41af-a906-c06ccd8b820a/mmc6.pdf" download="mmc6.pdf" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">PDF (45.87 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Methods S1. Dynafit script for global fit of mf2 and quin2 zinc titrations to 4:1 binding model, related to Figures&nbsp;4 and S4</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">Yeast-two-hybrid data are provided within <a id="crosref0155" href="#mmc1">Table&nbsp;S1</a>. Proteomics data are provided within <a id="crosref0160" href="#mmc3">Tables&nbsp;S3</a> and <a id="crosref0165" href="#mmc4">S4</a>. Additionally, all mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier ProteomeXchange: PXD031516. NMR structure information is provided within <a id="crosref0170" href="#mmc2">Table&nbsp;S2</a> and the structure is deposited on PDB:7SEK. All backbone and side chain chemical shifts have been deposited in the BioMagResBank under accession number BMRB: 51117 (free), BMRB:&nbsp;51118 (bound), and BMRB:&nbsp;30956 (fusion) (<span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Ulrich et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Ulrich, E.L. ∙ Akutsu, H. ∙ Doreleijers, J.F. ...</div><div><strong>BioMagResBank</strong></div><div><em>Nucleic Acids Res.</em> 2008; <strong>36</strong>:D402-D408</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkm957" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-38549138986" target="_blank" aria-label="Open Scopus in new tab">Scopus (1297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17984079/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkm957&amp;pmid=17984079" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Backbone relaxation rates have been deposited under accession numbers BMRB:&nbsp;51117 (free) and BMRB:&nbsp;51119 (fusion). The AlphaFold2 structural model of the yeast complex is available in ModelArchive at <a id="intref0010" href="https://modelarchive.org/doi/10.5452/ma-1hnkh" target="_blank">https://modelarchive.org/doi/10.5452/ma-1hnkh</a>.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">Code used in this study is in supplemental methods S1 (<a id="crosref0180" href="#mmc6">Methods S1</a>).</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the <a id="crosref0185" href="#sec-4-2-1">Lead Contact</a> upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Addlagatta, A. ∙ Hu, X. ∙ Liu, J.O. ...</div><div><strong>Structural basis for the functional differences between type I and type II human methionine aminopeptidases</strong></div><div><em>Biochemistry.</em> 2005; <strong>44</strong>:14741-14749</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi051691k" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-27744565323" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16274222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi051691k&amp;pmid=16274222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Andreini, C. ∙ Banci, L. ∙ Bertini, I. ...</div><div><strong>Counting the zinc-proteins encoded in the human genome</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2006; <strong>5</strong>:196-201</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/pr050361j" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-30744471169" target="_blank" aria-label="Open Scopus in new tab">Scopus (740)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16396512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fpr050361j&amp;pmid=16396512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Arfin, S.M. ∙ Bradshaw, R.A.</div><div><strong>Cotranslational processing and protein turnover in eukaryotic cells</strong></div><div><em>Biochemistry.</em> 1988; <strong>27</strong>:7979-7984</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00421a001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3069123/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00421a001&amp;pmid=3069123" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Ba, L.A. ∙ Doering, M. ∙ Burkholz, T. ...</div><div><strong>Metal trafficking: from maintaining the metal homeostasis to future drug design</strong></div><div><em>Metallomics.</em> 2009; <strong>1</strong>:292-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/b904533c" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77149122885" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21305127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fb904533c&amp;pmid=21305127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Basu, A.K.</div><div><strong>DNA damage, mutagenesis and cancer</strong></div><div><em>Int. J. Mol. Sci.</em> 2018; <strong>19</strong></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.K.BasuDNA+damage%2C+mutagenesis+and+cancerInt.+J.+Mol.+Sci.192018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Brayer, K.J. ∙ Kulshreshtha, S. ∙ Segal, D.J.</div><div><strong>The protein-binding potential of C2H2 zinc finger domains</strong></div><div><em>Cell Biochem. Biophys.</em> 2008; <strong>51</strong>:9-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12013-008-9007-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-50649114260" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18286240/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12013-008-9007-6&amp;pmid=18286240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Chen, W.W. ∙ Birsoy, K. ∙ Mihaylova, M.M. ...</div><div><strong>Inhibition of ATPIF1 ameliorates severe mitochondrial respiratory chain dysfunction in mammalian cells</strong></div><div><em>Cell Rep.</em> 2014; <strong>7</strong>:27-34</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2014.02.046&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2014.02.046&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24685140/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2014.02.046&amp;pmid=24685140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Coneyworth, L.J. ∙ Jackson, K.A. ∙ Tyson, J. ...</div><div><strong>Identification of the human zinc transcriptional regulatory element (ZTRE): a palindromic protein-binding DNA sequence responsible for zinc-induced transcriptional repression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2012; <strong>287</strong>:36567-36581</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M112.397000&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M112.397000&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84867793466" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22902622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M112.397000&amp;pmid=22902622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Cox, J. ∙ Hein, M.Y. ∙ Luber, C.A. ...</div><div><strong>Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ</strong></div><div><em>Mol. Cell. Proteomics.</em> 2014; <strong>13</strong>:2513-2526</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M113.031591&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M113.031591&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84907197082" target="_blank" aria-label="Open Scopus in new tab">Scopus (2744)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24942700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M113.031591&amp;pmid=24942700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Crooks, G.E. ∙ Hon, G. ∙ Chandonia, J.M. ...</div><div><strong>WebLogo: a sequence logo generator</strong></div><div><em>Genome Res.</em> 2004; <strong>14</strong>:1188-1190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.849004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-2142738304" target="_blank" aria-label="Open Scopus in new tab">Scopus (8336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15173120/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.849004&amp;pmid=15173120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Delaglio, F. ∙ Grzesiek, S. ∙ Vuister, G.W. ...</div><div><strong>NMRPipe: a multidimensional spectral processing system based on UNIX pipes</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 1995; <strong>6</strong>:277-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00197809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029400480" target="_blank" aria-label="Open Scopus in new tab">Scopus (11573)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8520220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00197809&amp;pmid=8520220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Devirgiliis, C. ∙ Zalewski, P.D. ∙ Perozzi, G. ...</div><div><strong>Zinc fluxes and zinc transporter genes in chronic diseases</strong></div><div><em>Mutat. Res.</em> 2007; <strong>622</strong>:84-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mrfmmm.2007.01.013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-34547681207" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17374385/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mrfmmm.2007.01.013&amp;pmid=17374385" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Eddy, S.R.</div><div><strong>A new generation of homology search tools based on probabilistic inference</strong></div><div><em>Genome Inform.</em> 2009; <strong>23</strong>:205-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1142/9781848165632_0019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20180275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1142%2F9781848165632_0019&amp;pmid=20180275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Edmonds, K.A. ∙ Jordan, M.R. ∙ Giedroc, D.P.</div><div><strong>COG0523 proteins: a functionally diverse family of transition metal-regulated G3E P-loop GTP hydrolases from bacteria to man</strong></div><div><em>Metallomics.</em> 2021; <strong>13</strong>:mfab046</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/mtomcs/mfab046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85113538279" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34302342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmtomcs%2Fmfab046&amp;pmid=34302342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Eide, D.J.</div><div><strong>Zinc transporters and the cellular trafficking of zinc</strong></div><div><em>Biochim. Biophys. Acta.</em> 2006; <strong>1763</strong>:711-722</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbamcr.2006.03.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33746929895" target="_blank" aria-label="Open Scopus in new tab">Scopus (666)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16675045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbamcr.2006.03.005&amp;pmid=16675045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Essner, J.</div><div><strong>Zebrafish Embryo Microinjection: Ribonucleoprotein Delivery Using the Alt-R CRISPR-Cas9 System</strong></div><div>Integrated DNA Technologies, 2016</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.EssnerZebrafish+Embryo+Microinjection%3A+Ribonucleoprotein+Delivery+Using+the+Alt-R+CRISPR-Cas9+System2016Integrated+DNA+Technologies" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Fang, H. ∙ Gough, J.</div><div><strong>DcGO: database of domain-centric ontologies on functions, phenotypes, diseases and more</strong></div><div><em>Nucleic Acids Res.</em> 2013; <strong>41</strong>:D536-D544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gks1080" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84876528480" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23161684/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgks1080&amp;pmid=23161684" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Fedotova, A.A. ∙ Bonchuk, A.N. ∙ Mogila, V.A. ...</div><div><strong>C<sub>2</sub>H<sub>2</sub> zinc finger proteins: the largest but poorly explored family of higher eukaryotic transcription factors</strong></div><div><em>Acta Nat.</em> 2017; <strong>9</strong>:47-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.32607/20758251-2017-9-2-47-58" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85021326833" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28740726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.32607%2F20758251-2017-9-2-47-58&amp;pmid=28740726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Fischer Walker, C. ∙ Black, R.E.</div><div><strong>Zinc and the risk for infectious disease</strong></div><div><em>Annu. Rev. Nutr.</em> 2004; <strong>24</strong>:255-275</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.nutr.23.011702.073054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-3242736546" target="_blank" aria-label="Open Scopus in new tab">Scopus (386)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15189121/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.nutr.23.011702.073054&amp;pmid=15189121" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Forman, H.J. ∙ Kennedy, J.A.</div><div><strong>Role of superoxide radical in mitochondrial dehydrogenase reactions</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 1974; <strong>60</strong>:1044-1050</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0006-291X(74)90418-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4372996/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0006-291X%2874%2990418-5&amp;pmid=4372996" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">21.</a></div><div>Frottin, F. ∙ Martinez, A. ∙ Peynot, P. ...</div><div><strong>The proteomics of N-terminal methionine cleavage</strong></div><div><em>Mol. Cell. Proteomics.</em> 2006; <strong>5</strong>:2336-2349</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M600225-MCP200&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.M600225-MCP200&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33845993905" target="_blank" aria-label="Open Scopus in new tab">Scopus (289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16963780/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M600225-MCP200&amp;pmid=16963780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Gamerdinger, M. ∙ Kobayashi, K. ∙ Wallisch, A. ...</div><div><strong>Early scanning of nascent polypeptides inside the ribosomal tunnel by NAC</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:996-1006.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.molcel.2019.06.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.molcel.2019.06.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85071433346" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31377116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.06.030&amp;pmid=31377116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>García-Bermúdez, J. ∙ Cuezva, J.M.</div><div><strong>The ATPase inhibitory Factor 1 (IF1): a master regulator of energy metabolism and of cell survival</strong></div><div><em>Biochim. Biophys. Acta.</em> 2016; <strong>1857</strong>:1167-1182</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbabio.2016.02.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84961219079" target="_blank" aria-label="Open Scopus in new tab">Scopus (86)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26876430/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbabio.2016.02.004&amp;pmid=26876430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Gawron, D. ∙ Ndah, E. ∙ Gevaert, K. ...</div><div><strong>Positional proteomics reveals differences in N-terminal proteoform stability</strong></div><div><em>Mol. Syst. Biol.</em> 2016; <strong>12</strong>:858</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20156662" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84959423232" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26893308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20156662&amp;pmid=26893308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Gibbs, D.J. ∙ Bacardit, J. ∙ Bachmair, A. ...</div><div><strong>The eukaryotic N-end rule pathway: conserved mechanisms and diverse functions</strong></div><div><em>Trends Cell Biol.</em> 2014; <strong>24</strong>:603-611</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.tcb.2014.05.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.tcb.2014.05.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24874449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tcb.2014.05.001&amp;pmid=24874449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Gibson, D.G.</div><div><strong>Enzymatic assembly of overlapping DNA fragments</strong></div><div><em>Methods Enzymol.</em> 2011; <strong>498</strong>:349-361</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/B978-0-12-385120-8.00015-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-79957471219" target="_blank" aria-label="Open Scopus in new tab">Scopus (491)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21601685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FB978-0-12-385120-8.00015-2&amp;pmid=21601685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Gibson, D.G. ∙ Young, L. ∙ Chuang, R.Y. ...</div><div><strong>Enzymatic assembly of DNA molecules up to several hundred kilobases</strong></div><div><em>Nat. Methods.</em> 2009; <strong>6</strong>:343-345</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1318" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-67349270900" target="_blank" aria-label="Open Scopus in new tab">Scopus (5532)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19363495/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1318&amp;pmid=19363495" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>Giglione, C. ∙ Boularot, A. ∙ Meinnel, T.</div><div><strong>Protein N-terminal methionine excision</strong></div><div><em>Cell. Mol. Life Sci.</em> 2004; <strong>61</strong>:1455-1474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00018-004-3466-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15197470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00018-004-3466-8&amp;pmid=15197470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Gross, C.T. ∙ McGinnis, W.</div><div><strong>DEAF-1, a novel protein that binds an essential region in a deformed response element</strong></div><div><em>EMBO J.</em> 1996; <strong>15</strong>:1961-1970</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1996.tb00547.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8617243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1996.tb00547.x&amp;pmid=8617243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Guilbert, J.J.</div><div><strong>The world health report 2002—reducing risks, promoting healthy life</strong></div><div><em>Educ. Health (Abingdon).</em> 2003; <strong>16</strong>:230</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14741909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=14741909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Güntert, P.</div><div><strong>Automated NMR structure calculation with CYANA</strong></div><div><em>Methods Mol. Biol.</em> 2004; <strong>278</strong>:353-378</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1385/1-59259-809-9:353" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15318003/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1385%2F1-59259-809-9%3A353&amp;pmid=15318003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>Haas, C.E. ∙ Rodionov, D.A. ∙ Kropat, J. ...</div><div><strong>A subset of the diverse COG0523 family of putative metal chaperones is linked to zinc homeostasis in all kingdoms of life</strong></div><div><em>BMC Genomics.</em> 2009; <strong>10</strong>:470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-10-470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-70449732603" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19822009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-10-470&amp;pmid=19822009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Hackenbrock, C.R. ∙ Rehn, T.G. ∙ Weinbach, E.C. ...</div><div><strong>Oxidative phosphorylation and ultrastructural transformation in mitochondria in the intact ascites tumor cell</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1971; <strong>51</strong>:123-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.51.1.123" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0015144531" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5111873/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.51.1.123&amp;pmid=5111873" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Hailemariam, M. ∙ Eguez, R.V. ∙ Singh, H. ...</div><div><strong>S-trap, an ultrafast sample-preparation approach for shotgun proteomics</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2018; <strong>17</strong>:2917-2924</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jproteome.8b00505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85052299545" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30114372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jproteome.8b00505&amp;pmid=30114372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#figs2" title="View in article">35.</a></div><div>Harter, M.R. ∙ Liu, C.D. ∙ Shen, C.L. ...</div><div><strong>BS69/ZMYND11 C-terminal domains bind and inhibit EBNA2</strong></div><div><em>PLoS Pathog.</em> 2016; <strong>12</strong>:e1005414</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1005414" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84959451149" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26845565/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1005414&amp;pmid=26845565" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Hilty, C. ∙ Wider, G. ∙ Fernández, C. ...</div><div><strong>Stereospecific assignments of the isopropyl methyl groups of the membrane protein OmpX in DHPC micelles</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2003; <strong>27</strong>:377-382</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1025877326533" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0142149113" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14512734/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1025877326533&amp;pmid=14512734" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Howe, K.L. ∙ Achuthan, P. ∙ Allen, J. ...</div><div><strong>Ensembl 2021</strong></div><div><em>Nucleic Acids Res.</em> 2021; <strong>49</strong>:D884-D891</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa942" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85099428320" target="_blank" aria-label="Open Scopus in new tab">Scopus (659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33137190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa942&amp;pmid=33137190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Hu, X. ∙ Addlagatta, A. ∙ Lu, J. ...</div><div><strong>Elucidation of the function of type 1 human methionine aminopeptidase during cell cycle progression</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:18148-18153</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0608389103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33845310513" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17114291/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0608389103&amp;pmid=17114291" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Huang, Z. ∙ Zhang, X.A. ∙ Bosch, M. ...</div><div><strong>Tris(2-pyridylmethyl)amine (TPA) as a membrane-permeable chelator for interception of biological mobile zinc</strong></div><div><em>Metallomics.</em> 2013; <strong>5</strong>:648-655</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c3mt00103b" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84878785188" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23715510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc3mt00103b&amp;pmid=23715510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Huet, D. ∙ Rajendran, E. ∙ van Dooren, G.G. ...</div><div><strong>Identification of cryptic subunits from an apicomplexan ATP synthase</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e38097</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.38097" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85058422356" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30204085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.38097&amp;pmid=30204085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">41.</a></div><div>Huttlin, E.L. ∙ Ting, L. ∙ Bruckner, R.J. ...</div><div><strong>The BioPlex network: a systematic exploration of the human interactome</strong></div><div><em>Cell.</em> 2015; <strong>162</strong>:425-440</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.cell.2015.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.cell.2015.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84937213168" target="_blank" aria-label="Open Scopus in new tab">Scopus (871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26186194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.06.043&amp;pmid=26186194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Hyberts, S.G. ∙ Milbradt, A.G. ∙ Wagner, A.B. ...</div><div><strong>Application of iterative soft thresholding for fast reconstruction of NMR data non-uniformly sampled with multidimensional poisson gap scheduling</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2012; <strong>52</strong>:315-327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-012-9611-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84863114255" target="_blank" aria-label="Open Scopus in new tab">Scopus (308)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22331404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-012-9611-z&amp;pmid=22331404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">43.</a></div><div>Hyberts, S.G. ∙ Takeuchi, K. ∙ Wagner, G.</div><div><strong>Poisson-gap sampling and forward maximum entropy reconstruction for enhancing the resolution and sensitivity of protein NMR data</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2010; <strong>132</strong>:2145-2147</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja908004w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77950835733" target="_blank" aria-label="Open Scopus in new tab">Scopus (256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20121194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja908004w&amp;pmid=20121194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Ingerman, E. ∙ Nunnari, J.</div><div><strong>A continuous, regenerative coupled GTPase assay for dynamin-related proteins</strong></div><div><em>Methods Enzymol.</em> 2005; <strong>404</strong>:611-619</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0076-6879(05)04053-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-30544450009" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16413304/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0076-6879%2805%2904053-X&amp;pmid=16413304" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Jao, L.E. ∙ Wente, S.R. ∙ Chen, W.</div><div><strong>Efficient multiplex biallelic zebrafish genome editing using a CRISPR nuclease system</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:13904-13909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1308335110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84882788354" target="_blank" aria-label="Open Scopus in new tab">Scopus (880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23918387/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1308335110&amp;pmid=23918387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">46.</a></div><div>Jonckheere, V. ∙ Fijałkowska, D. ∙ Van Damme, P.</div><div><strong>Omics assisted N-terminal Proteoform and protein expression profiling on methionine aminopeptidase 1 (MetAP1) deletion</strong></div><div><em>Mol. Cell. Proteomics.</em> 2018; <strong>17</strong>:694-708</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.RA117.000360&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fmcp.RA117.000360&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85045027596" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29317475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.RA117.000360&amp;pmid=29317475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47-1" title="View in article">47.</a></div><div>Jordan, M.R. ∙ Wang, J. ∙ Weiss, A. ...</div><div><strong>Mechanistic insights into the metal-dependent activation of Zn(II)-dependent metallochaperones</strong></div><div><em>Inorg. Chem.</em> 2019; <strong>58</strong>:13661-13672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.inorgchem.9b01173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85073651560" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31247880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.inorgchem.9b01173&amp;pmid=31247880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Jumper, J. ∙ Evans, R. ∙ Pritzel, A. ...</div><div><strong>Highly accurate protein structure prediction with AlphaFold</strong></div><div><em>Nature.</em> 2021; <strong>596</strong>:583-589</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03819-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85110248105" target="_blank" aria-label="Open Scopus in new tab">Scopus (5905)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03819-2&amp;pmid=34265844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>Kabir-ud-Din ∙ Bano, M. ∙ Khan, I.A.</div><div><strong>Kinetics and mechanism of the ninhydrin reaction with DL-methionine in the absence and the presence of organic solvents</strong></div><div><em>Indian J. Chem.</em> 2003; <strong>42B</strong>:1132-1136</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Kabir-ud-DinM.BanoI.A.KhanKinetics+and+mechanism+of+the+ninhydrin+reaction+with+DL-methionine+in+the+absence+and+the+presence+of+organic+solventsIndian+J.+Chem.42B200311321136" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Kateb, F. ∙ Perrin, H. ∙ Tripsianes, K. ...</div><div><strong>Structural and functional analysis of the DEAF-1 and BS69 MYND domains</strong></div><div><em>PLoS One.</em> 2013; <strong>8</strong>:e54715</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0054715" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84872854119" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23372760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0054715&amp;pmid=23372760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">51.</a></div><div>Kim, H.K. ∙ Kim, R.R. ∙ Oh, J.H. ...</div><div><strong>The N-terminal methionine of cellular proteins as a degradation signal</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:158-169</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.cell.2013.11.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.cell.2013.11.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84892802083" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.031&amp;pmid=24361105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">52.</a></div><div>Krämer, A. ∙ Green, J. ∙ Pollard, Jr., J. ...</div><div><strong>Causal analysis approaches in ingenuity pathway analysis</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:523-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt703" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84894571860" target="_blank" aria-label="Open Scopus in new tab">Scopus (2850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24336805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt703&amp;pmid=24336805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">53.</a></div><div>Krężel, A. ∙ Maret, W.</div><div><strong>The functions of metamorphic metallothioneins in zinc and copper metabolism</strong></div><div><em>Int. J. Mol. Sci.</em> 2017; <strong>18</strong>:1237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/ijms18061237" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85020827370" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28598392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fijms18061237&amp;pmid=28598392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">54.</a></div><div>Kumar, S. ∙ Stecher, G. ∙ Suleski, M. ...</div><div><strong>TimeTree: a resource for timelines, Timetrees, and divergence times</strong></div><div><em>Time. Mol. Biol. Evol.</em> 2017; <strong>34</strong>:1812-1819</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msx116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28387841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsx116&amp;pmid=28387841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55-1" title="View in article">55.</a></div><div>Kuzmic, P.</div><div><strong>Program DYNAFIT for the analysis of enzyme kinetic data: application to HIV proteinase</strong></div><div><em>Anal. Biochem.</em> 1996; <strong>237</strong>:260-273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/abio.1996.0238" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029902679" target="_blank" aria-label="Open Scopus in new tab">Scopus (1358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8660575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fabio.1996.0238&amp;pmid=8660575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Lee, W. ∙ Tonelli, M. ∙ Markley, J.L.</div><div><strong>NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:1325-1327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84927748573" target="_blank" aria-label="Open Scopus in new tab">Scopus (1039)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25505092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu830&amp;pmid=25505092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>Leopoldini, M. ∙ Russo, N. ∙ Toscano, M.</div><div><strong>Which one among Zn(II), Co(II), Mn(II), and Fe(II) is the most efficient ion for the methionine aminopeptidase catalyzed reaction?</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2007; <strong>129</strong>:7776-7784</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja068168t" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84961982737" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17523636/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja068168t&amp;pmid=17523636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">58.</a></div><div>Li, X. ∙ Chang, Y.H.</div><div><strong>Amino-terminal protein processing in Saccharomyces cerevisiae is an essential function that requires two distinct methionine aminopeptidases</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1995; <strong>92</strong>:12357-12361</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.92.26.12357" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0029585125" target="_blank" aria-label="Open Scopus in new tab">Scopus (256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8618900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.92.26.12357&amp;pmid=8618900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">59.</a></div><div>Lin, Z. ∙ Gasic, I. ∙ Chandrasekaran, V. ...</div><div><strong>TTC5 mediates autoregulation of tubulin via mRNA degradation</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:100-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaz4352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85077476517" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31727855/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaz4352&amp;pmid=31727855" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#figs2" title="View in article">60.</a></div><div>Liu, Y. ∙ Chen, W. ∙ Gaudet, J. ...</div><div><strong>Structural basis for recognition of SMRT/N-CoR by the MYND domain and its contribution to AML1/ETO’s activity</strong></div><div><em>Cancer Cell.</em> 2007; <strong>11</strong>:483-497</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.ccr.2007.04.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.ccr.2007.04.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-34249812945" target="_blank" aria-label="Open Scopus in new tab">Scopus (96)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17560331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2007.04.010&amp;pmid=17560331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">61.</a></div><div>Livak, K.J. ∙ Schmittgen, T.D.</div><div><strong>Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta C(T)) method</strong></div><div><em>Methods.</em> 2001; <strong>25</strong>:402-408</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/meth.2001.1262" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0035710746" target="_blank" aria-label="Open Scopus in new tab">Scopus (125166)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fmeth.2001.1262&amp;pmid=11846609" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">62.</a></div><div>Lo, M.N. ∙ Damon, L.J. ∙ Wei Tay, J. ...</div><div><strong>Single cell analysis reveals multiple requirements for zinc in the mammalian cell cycle</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:e51107</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.51107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85078876811" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32014109/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.51107&amp;pmid=32014109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Löhr, F. ∙ Hänsel, R. ∙ Rogov, V.V. ...</div><div><strong>Improved pulse sequences for sequence specific assignment of aromatic proton resonances in proteins</strong></div><div><em>J.&nbsp;Biomol. NMR.</em> 2007; <strong>37</strong>:205-224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10858-006-9128-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-33947608952" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17237975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10858-006-9128-4&amp;pmid=17237975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">64.</a></div><div>Loschen, G. ∙ Azzi, A. ∙ Richter, C. ...</div><div><strong>Superoxide radicals as precursors of mitochondrial hydrogen peroxide</strong></div><div><em>FEBS Lett.</em> 1974; <strong>42</strong>:68-72</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0014-5793(74)80281-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0016148483" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4859511/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0014-5793%2874%2980281-4&amp;pmid=4859511" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>Maciejewski, M.W. ∙ Schuyler, A.D. ∙ Gryk, M.R. ...</div><div><strong>NMRbox: a resource for biomolecular NMR computation</strong></div><div><em>Biophys. J.</em> 2017; <strong>112</strong>:1529-1534</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.bpj.2017.03.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.bpj.2017.03.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85018654124" target="_blank" aria-label="Open Scopus in new tab">Scopus (193)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28445744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2017.03.011&amp;pmid=28445744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">66.</a></div><div>MacLean, B. ∙ Tomazela, D.M. ∙ Shulman, N. ...</div><div><strong>Skyline: an open source document editor for creating and analyzing targeted proteomics experiments</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:966-968</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77951965920" target="_blank" aria-label="Open Scopus in new tab">Scopus (3010)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20147306/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq054&amp;pmid=20147306" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">67.</a></div><div>Meinnel, T. ∙ Giglione, C.</div><div><strong>Tools for analyzing and predicting N-terminal protein modifications</strong></div><div><em>Proteomics.</em> 2008; <strong>8</strong>:626-649</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pmic.200700592" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-40549132858" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18203265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpmic.200700592&amp;pmid=18203265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">68.</a></div><div>Mirdita, M. ∙ Ovchinnikov, S. ∙ Steinegger, M.</div><div><strong>ColabFold—making protein folding accessible to all</strong></div><div><em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.08.15.456425" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85119536212" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.08.15.456425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">69.</a></div><div>Monteith, A. ∙ Marszalec, W. ∙ Chan, P. ...</div><div><strong>Imaging of mitochondrial and non-mitochondrial responses in cultured rat hippocampal neurons exposed to micromolar concentrations of TMRM</strong></div><div><em>PLoS One.</em> 2013; <strong>8</strong>:e58059</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0058059" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84874583009" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23483968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0058059&amp;pmid=23483968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="optUDRwk4IsEX" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optUDRwk4IsEX" title="View in article">70.</a></div><div>Moreno-Mateos, M. ∙ Vejnar, C. ∙ Beaudoin, J.-D. ...</div><div><strong>CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo</strong></div><div><em>Nat Methods.</em> 2015; <strong>12</strong>:982-988</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84959123021" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322839/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3543&amp;pmid=26322839" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">71.</a></div><div>Mortensen, B.L. ∙ Rathi, S. ∙ Chazin, W.J. ...</div><div><strong>Acinetobacter baumannii response to host-mediated zinc limitation requires the transcriptional regulator Zur</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2014; <strong>196</strong>:2616-2626</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.01650-14" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84903388692" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24816603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.01650-14&amp;pmid=24816603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">72.</a></div><div>Muhire, B.M. ∙ Varsani, A. ∙ Martin, D.P.</div><div><strong>SDT: a virus classification tool based on pairwise sequence alignment and identity calculation</strong></div><div><em>PLoS One.</em> 2014; <strong>9</strong>:e108277</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0108277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84907426295" target="_blank" aria-label="Open Scopus in new tab">Scopus (945)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25259891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0108277&amp;pmid=25259891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">73.</a></div><div>Nyathi, Y. ∙ Pool, M.R.</div><div><strong>Analysis of the interplay of protein biogenesis factors at the ribosome exit site reveals new role for NAC</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2015; <strong>210</strong>:287-301</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201410086" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84940445222" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26195668/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201410086&amp;pmid=26195668" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73-1" title="View in article">74.</a></div><div>Ogo, O.A. ∙ Tyson, J. ∙ Cockell, S.J. ...</div><div><strong>The zinc finger protein ZNF658 regulates the transcription of genes involved in zinc homeostasis and affects ribosome biogenesis through the zinc transcriptional regulatory element</strong></div><div><em>Mol. Cell. Biol.</em> 2015; <strong>35</strong>:977-987</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01298-14" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25582195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01298-14&amp;pmid=25582195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">75.</a></div><div>O'Halloran, T.V. ∙ Culotta, V.C.</div><div><strong>Metallochaperones, an intracellular shuttle service for metal ions</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2000; <strong>275</strong>:25057-25060</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.R000006200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.R000006200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0034682776" target="_blank" aria-label="Open Scopus in new tab">Scopus (669)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10816601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.R000006200&amp;pmid=10816601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">76.</a></div><div>Parichy, D.M. ∙ Elizondo, M.R. ∙ Mills, M.G. ...</div><div><strong>Normal table of postembryonic zebrafish development: staging by externally visible anatomy of the living fish</strong></div><div><em>Dev. Dyn.</em> 2009; <strong>238</strong>:2975-3015</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-70949095044" target="_blank" aria-label="Open Scopus in new tab">Scopus (514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19891001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22113&amp;pmid=19891001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="optkIjaU1XKKm" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optkIjaU1XKKm-1" title="View in article">77.</a></div><div>Pasquini, M. ∙ Grosjean, N. ∙ Hixson, K.K. ...</div><div><strong>Zng1 is a GTP-dependent zinc transferase needed for activation of methionine aminopeptidase</strong></div><div><em>Cell Rep.</em> 2022; <strong>39</strong>:110834</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2022.110834&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2Fj.celrep.2022.110834&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85130211918" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35584675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2022.110834&amp;pmid=35584675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">78.</a></div><div>Pelton, J.G. ∙ Torchia, D.A. ∙ Meadow, N.D. ...</div><div><strong>Tautomeric states of the active-site histidines of phosphorylated and unphosphorylated IIIGlc, a signal-transducing protein from Escherichia coli, using two-dimensional heteronuclear NMR techniques</strong></div><div><em>Protein Sci.</em> 1993; <strong>2</strong>:543-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.5560020406" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8518729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.5560020406&amp;pmid=8518729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">79.</a></div><div>Peroutka Iii, R.J. ∙ Orcutt, S.J. ∙ Strickler, J.E. ...</div><div><strong>SUMO fusion technology for enhanced protein expression and purification in prokaryotes and eukaryotes</strong></div><div><em>Methods Mol. Biol.</em> 2011; <strong>705</strong>:15-30</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-61737-967-3_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-79952459386" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21125378/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-61737-967-3_2&amp;pmid=21125378" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">80.</a></div><div>Picard, M. ∙ Shirihai, O.S. ∙ Gentil, B.J. ...</div><div><strong>Mitochondrial morphology transitions and functions: implications for retrograde signaling?</strong></div><div><em>Am. J. Physiol. Regul. Integr. Comp. Physiol.</em> 2013; <strong>304</strong>:R393-R406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/ajpregu.00584.2012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84877105774" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23364527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fajpregu.00584.2012&amp;pmid=23364527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">81.</a></div><div>Polevoda, B. ∙ Sherman, F.</div><div><strong>N-terminal acetyltransferases and sequence requirements for N-terminal acetylation of eukaryotic proteins</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 2003; <strong>325</strong>:595-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-2836(02)01269-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0037462954" target="_blank" aria-label="Open Scopus in new tab">Scopus (354)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12507466/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-2836%2802%2901269-X&amp;pmid=12507466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">82.</a></div><div>Prasad, A.S.</div><div><strong>Discovery of human zinc deficiency: its impact on human health and disease</strong></div><div><em>Adv. Nutr.</em> 2013; <strong>4</strong>:176-190</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.3945%2Fan.112.003210&amp;cf=fulltext&amp;site=advnut-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.3945%2Fan.112.003210&amp;cf=pdf&amp;site=advnut-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84882978950" target="_blank" aria-label="Open Scopus in new tab">Scopus (477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23493534/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3945%2Fan.112.003210&amp;pmid=23493534" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">83.</a></div><div>Quinoa, E. ∙ Adamczeski, M. ∙ Crews, P. ...</div><div><strong>Bengamides, heterocyclic anthelmintics from a Jaspidae marine sponge</strong></div><div><em>J.&nbsp;Org. Chem.</em> 1986; <strong>51</strong>:4494-4497</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jo00373a036" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjo00373a036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">84.</a></div><div>Rachineni, K. ∙ Arya, T. ∙ Singarapu, K.K. ...</div><div><strong>Chemical shift assignments of zinc finger domain of methionine aminopeptidase 1 (MetAP1) from Homo sapiens</strong></div><div><em>Biomol. NMR Assign.</em> 2015; <strong>9</strong>:351-353</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12104-015-9608-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84941935610" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25921012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12104-015-9608-0&amp;pmid=25921012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">85.</a></div><div>Raghunath, A. ∙ Perumal, E.</div><div><strong>Analysis of lethality and malformations during zebrafish (<i>Danio rerio</i>) development</strong></div><div><em>Methods Mol. Biol.</em> 2018; <strong>1797</strong>:337-363</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-7883-0_18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85048624330" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29896702/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-7883-0_18&amp;pmid=29896702" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">86.</a></div><div>Robert, X. ∙ Gouet, P.</div><div><strong>Deciphering key features in protein structures with the new ENDscript server</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:W320-W324</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku316" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84904790793" target="_blank" aria-label="Open Scopus in new tab">Scopus (3682)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24753421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku316&amp;pmid=24753421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">87.</a></div><div>Rosenzweig, A.C.</div><div><strong>Metallochaperones: bind and deliver</strong></div><div><em>Chem. Biol.</em> 2002; <strong>9</strong>:673-677</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS1074-5521%2802%2900156-4&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS1074-5521%2802%2900156-4&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0035986679" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12079778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1074-5521%2802%2900156-4&amp;pmid=12079778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86-1" title="View in article">88.</a></div><div>Sattler, M. ∙ Schleucher, J. ∙ Griesinger, C.</div><div><strong>Heteronuclear multidimensional NMR experiments for the structure determination of proteins in solution employing pulsed field gradients</strong></div><div><em>Prog. Nucl. Magn. Reson. Spectrosc.</em> 1999; <strong>34</strong>:93-158</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0079-6565%2898%2900025-9&amp;cf=fulltext&amp;site=jpnmrs-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0079-6565%2898%2900025-9&amp;cf=pdf&amp;site=jpnmrs-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0079-6565%2898%2900025-9" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">89.</a></div><div>Scaduto, Jr., R.C. ∙ Grotyohann, L.W.</div><div><strong>Measurement of mitochondrial membrane potential using fluorescent rhodamine derivatives</strong></div><div><em>Biophys. J.</em> 1999; <strong>76</strong>:469-477</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0006-3495%2899%2977214-0&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1016%2FS0006-3495%2899%2977214-0&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0345120940" target="_blank" aria-label="Open Scopus in new tab">Scopus (965)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9876159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0006-3495%2899%2977214-0&amp;pmid=9876159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">90.</a></div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31040)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">91.</a></div><div>Sheffield, P. ∙ Garrard, S. ∙ Derewenda, Z.</div><div><strong>Overcoming expression and purification problems of RhoGDI using a family of “parallel” expression vectors</strong></div><div><em>Protein Expr. Purif.</em> 1999; <strong>15</strong>:34-39</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/prep.1998.1003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0033082686" target="_blank" aria-label="Open Scopus in new tab">Scopus (530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10024467/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fprep.1998.1003&amp;pmid=10024467" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90-1" title="View in article">92.</a></div><div>Shen, Y. ∙ Bax, A.</div><div><strong>Protein structural information derived from NMR chemical shift with the neural network program TALOS-N</strong></div><div><em>Methods Mol. Biol.</em> 2015; <strong>1260</strong>:17-32</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-2239-0_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84917706654" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25502373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-2239-0_2&amp;pmid=25502373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">93.</a></div><div>Sigel, A. ∙ Sigel, H. ∙ Sigel, R.K.O.</div><div><strong>Interrelations between Essential Metal Ions and Human Diseases</strong></div><div>Springer, 2013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-94-007-7500-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-94-007-7500-8" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">94.</a></div><div>Stetefeld, J. ∙ McKenna, S.A. ∙ Patel, T.R.</div><div><strong>Dynamic light scattering: a practical guide and applications in biomedical sciences</strong></div><div><em>Biophys. Rev.</em> 2016; <strong>8</strong>:409-427</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12551-016-0218-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85000399690" target="_blank" aria-label="Open Scopus in new tab">Scopus (824)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28510011/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12551-016-0218-6&amp;pmid=28510011" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">95.</a></div><div>Sydor, A.M. ∙ Jost, M. ∙ Ryan, K.S. ...</div><div><strong>Metal binding properties of <i>Escherichia coli</i> YjiA, a member of the metal homeostasis-associated COG0523 family of GTPases</strong></div><div><em>Biochemistry.</em> 2013; <strong>52</strong>:1788-1801</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi301600z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84874997087" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24449932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi301600z&amp;pmid=24449932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">96.</a></div><div>Towbin, H. ∙ Bair, K.W. ∙ DeCaprio, J.A. ...</div><div><strong>Proteomics-based target identification: bengamides as a new class of methionine aminopeptidase inhibitors</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2003; <strong>278</strong>:52964-52971</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M309039200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=10.1074%2Fjbc.M309039200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-9144222006" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14534293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M309039200&amp;pmid=14534293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95-1" title="View in article">97.</a></div><div>Tyanova, S. ∙ Temu, T. ∙ Sinitcyn, P. ...</div><div><strong>The Perseus computational platform for comprehensive analysis of (prote)omics data</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:731-740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84976274986" target="_blank" aria-label="Open Scopus in new tab">Scopus (3641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27348712/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3901&amp;pmid=27348712" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96-1" title="View in article">98.</a></div><div>Uhlén, M. ∙ Fagerberg, L. ∙ Hallström, B.M. ...</div><div><strong>Proteomics. Tissue-based map of the human proteome</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1260419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260419&amp;pmid=25613900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">99.</a></div><div>Ulrich, E.L. ∙ Akutsu, H. ∙ Doreleijers, J.F. ...</div><div><strong>BioMagResBank</strong></div><div><em>Nucleic Acids Res.</em> 2008; <strong>36</strong>:D402-D408</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkm957" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-38549138986" target="_blank" aria-label="Open Scopus in new tab">Scopus (1297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17984079/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkm957&amp;pmid=17984079" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">100.</a></div><div>UniProt Consortium</div><div><strong>UniProt: the universal protein knowledgebase in 2021</strong></div><div><em>Nucleic Acids Res.</em> 2021; <strong>49</strong>:D480-D489</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa1100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85099417502" target="_blank" aria-label="Open Scopus in new tab">Scopus (2385)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33237286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa1100&amp;pmid=33237286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">101.</a></div><div>Varshavsky, A.</div><div><strong>The N-end rule pathway and regulation by proteolysis</strong></div><div><em>Protein Sci.</em> 2011; <strong>20</strong>:1298-1345</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.666" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-79960683356" target="_blank" aria-label="Open Scopus in new tab">Scopus (480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21633985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.666&amp;pmid=21633985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">102.</a></div><div>Vetro, J.A. ∙ Chang, Y.H.</div><div><strong>Yeast methionine aminopeptidase type 1 is ribosome-associated and requires its N-terminal zinc finger domain for normal function in&nbsp;vivo</strong></div><div><em>J.&nbsp;Cell. Biochem.</em> 2002; <strong>85</strong>:678-688</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcb.10161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0036235240" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11968008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcb.10161&amp;pmid=11968008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">103.</a></div><div>Vetro, J.A. ∙ Dummitt, B. ∙ Chang, Y.</div><div><strong>Methionine-aminopeptidase: emerging role in angiogenesis</strong></div><div><div>Hooper, N.M. ∙ Lendeckel, U. (Editors)</div></div><div><strong>Aminopeptidases in Biology and Disease</strong></div><div>Kluwer Academic/Plenum Publishers, 2004</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4419-8869-0_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4419-8869-0_2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">104.</a></div><div>Weissert, V. ∙ Rieger, B. ∙ Morris, S. ...</div><div><strong>Inhibition of the mitochondrial ATPase function by IF1 changes the spatiotemporal organization of ATP synthase</strong></div><div><em>Biochim. Biophys. Acta Bioenerg.</em> 2021; <strong>1862</strong>:148322</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbabio.2020.148322" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85096199594" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33065099/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbabio.2020.148322&amp;pmid=33065099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">105.</a></div><div>Wellenreuther, G. ∙ Cianci, M. ∙ Tucoulou, R. ...</div><div><strong>The ligand environment of zinc stored in vesicles</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2009; <strong>380</strong>:198-203</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbrc.2009.01.074" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-59849127893" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19171119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbrc.2009.01.074&amp;pmid=19171119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">106.</a></div><div>White, K.N. ∙ Tenney, K. ∙ Crews, P.</div><div><strong>The Bengamides: a mini-review of natural sources, analogues, biological properties, biosynthetic origins, and future prospects</strong></div><div><em>J.&nbsp;Nat. Prod.</em> 2017; <strong>80</strong>:740-755</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jnatprod.6b00970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-85016142079" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28185457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jnatprod.6b00970&amp;pmid=28185457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">107.</a></div><div>Wong, A. ∙ Vallender, E.J. ∙ Heretis, K. ...</div><div><strong>Diverse fates of paralogs following segmental duplication of telomeric genes</strong></div><div><em>Genomics.</em> 2004; <strong>84</strong>:239-247</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ygeno.2004.03.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-3042517001" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15233989/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ygeno.2004.03.001&amp;pmid=15233989" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106-1" title="View in article">108.</a></div><div>Xiao, Q. ∙ Zhang, F. ∙ Nacev, B.A. ...</div><div><strong>Protein N-terminal processing: substrate specificity of <i>Escherichia coli</i> and human methionine aminopeptidases</strong></div><div><em>Biochemistry.</em> 2010; <strong>49</strong>:5588-5599</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi1005464" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-77954194681" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20521764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi1005464&amp;pmid=20521764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">109.</a></div><div>Yamazaki, T. ∙ Forman-Kay, J.D. ∙ Kay, L.E.</div><div><strong>Two-dimensional NMR experiments for correlating carbon-13.beta. and proton.delta./.epsilon. chemical shifts of aromatic residues in 13C-labeled proteins via scalar couplings</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 1993; <strong>115</strong>:11054-11055</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja00076a099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja00076a099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="optYOx9AXlXJG" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optYOx9AXlXJG" title="View in article">110.</a></div><div>Yin, L. ∙ Jao, L.-E. ∙ Chen, W. ...</div><div><strong>Generation of Targeted Mutations in Zebrafish Using the CRISPR/Cas System</strong></div><div><em>Methods Mol Biol.</em> 2015; <strong>1332</strong>:205-217</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-2917-7_16" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84939507321" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26285757/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-2917-7_16&amp;pmid=26285757" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">111.</a></div><div>Zhang, Y. ∙ Griffith, E.C. ∙ Sage, J. ...</div><div><strong>Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21WAF1/CIP1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2000; <strong>97</strong>:6427-6432</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.97.12.6427" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-0034612236" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10841547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.97.12.6427&amp;pmid=10841547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">112.</a></div><div>Zougman, A. ∙ Selby, P.J. ∙ Banks, R.E.</div><div><strong>Suspension trapping (STrap) sample preparation method for bottom-up proteomics analysis</strong></div><div><em>Proteomics.</em> 2014; <strong>14</strong></div><div>1006–1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pmic.201300553" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.04.011&amp;key=2-s2.0-84898664545" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24678027/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpmic.201300553&amp;pmid=24678027" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/153693ba-8b18-4497-b7a5-d5b8a4010a39/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/7601887c-9c23-4646-bdcd-3dd5cdc5ca31/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/4a93b725-7bdd-41b5-ac29-d47e783bbb8b/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/6118a3a0-b223-424c-ae08-41335827f906/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/fd90e8c9-b5be-4ed4-b04a-ec6e730f8d47/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/27cad41c-42b3-48d1-8033-c683a724a1ec/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/89cdb485-f309-4d79-963a-bbc61c901aa7/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/dd0ca4d3-5389-4386-b080-878e40168391/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/fea10efc-7d8f-4cd7-95d2-9a04b5ab9f65/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/c2a7bfd4-ea95-4c20-b8e1-b25ee29982d4/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/2a7b30e0-073a-4de3-ad4c-3e4e4d8e109d/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/9067d552-f77b-43a1-b294-d8206d0f6acf/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/bda76e41-88a6-4732-abd8-abcebd1a9f98/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/asset/5537e573-363c-4ea3-abec-f3108489c889/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867421X0013X%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867422004585%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (6)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/attachment/c907db87-73ec-44eb-918c-40afc0df9de2/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (107.08 KB)</span></a><div class="core-description">Table&nbsp;S1. Overview of interaction partners of human, murine and zebrafish ZNG1 as identified by yeast-two-hybrid screen, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/attachment/4c443444-812a-4dac-8cd0-06a03aa7721b/mmc2.pdf" download="mmc2.pdf"><span id="mmc2-link">PDF (102.31 KB)</span></a><div class="core-description">Table&nbsp;S2. Structural statistics for NMR solution structure of the fusion of ZNG<sub>110–30</sub> with METAP1<sub>1–59</sub>, related to Figures&nbsp;3 and S3</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/attachment/6e76340c-de41-48a1-af20-e295da93f3b5/mmc3.pdf" download="mmc3.pdf"><span id="mmc3-link">PDF (58.96 KB)</span></a><div class="core-description">Table&nbsp;S3. Proteomics hits from low zinc diet, related to Figures&nbsp;6 and S6</div></div></div><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/attachment/fc173f84-c37f-48a8-b72e-207a9266f0ed/mmc4.zip" download="mmc4.zip"><span id="mmc4-link">Archive (3.79 MB)</span></a><div class="core-description">Table&nbsp;S4. Proteomics analysis of WT and <i>Zng1</i><sup><i>−/−</i></sup> TKPTS cells grown in the presence of absence of TPA or Bengamide B, related to Figures&nbsp;7 and S7</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/attachment/a238fb9d-a6e0-4e98-ad5f-57c2ad9ffcb5/mmc5.pdf" download="mmc5.pdf"><span id="mmc5-link">PDF (55.14 KB)</span></a><div class="core-description">Table&nbsp;S5. Primers used in this study, related to STAR Methods</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.04.011/attachment/2d599bf2-585d-41af-a906-c06ccd8b820a/mmc6.pdf" download="mmc6.pdf"><span id="mmc6-link">PDF (45.87 KB)</span></a><div class="core-description">Methods S1. Dynafit script for global fit of mf2 and quin2 zinc titrations to 4:1 binding model, related to Figures&nbsp;4 and S4</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867422004585" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2022.04.011","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Zn-regulated GTPase metalloprotein activator 1 modulates vertebrate zinc homeostasis</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.04.011/asset/153693ba-8b18-4497-b7a5-d5b8a4010a39/main.assets/gr1.jpg" height="669" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/208f765c-15f2-46af-a641-dc0e1b3a8cd0/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.04.011/asset/7601887c-9c23-4646-bdcd-3dd5cdc5ca31/main.assets/figs1.jpg" height="560" width="753" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/ed3ab668-0e71-4b4b-9348-ed5c38016a03/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.04.011/asset/4a93b725-7bdd-41b5-ac29-d47e783bbb8b/main.assets/gr2.jpg" height="565" width="506" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/69614d67-1762-478a-a0c3-15b8086d46e4/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.04.011/asset/6118a3a0-b223-424c-ae08-41335827f906/main.assets/figs2.jpg" height="808" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/83815183-32e6-47fe-9663-d2f75b27c7ef/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.04.011/asset/fd90e8c9-b5be-4ed4-b04a-ec6e730f8d47/main.assets/figs3.jpg" height="812" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/3999aa1a-40a9-4f72-a41e-056b13c6f2db/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.04.011/asset/27cad41c-42b3-48d1-8033-c683a724a1ec/main.assets/gr3.jpg" height="433" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/9d4b1127-30cc-44b4-ab57-35e2629ce270/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.04.011/asset/89cdb485-f309-4d79-963a-bbc61c901aa7/main.assets/gr4.jpg" height="555" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/68f767d7-d60f-4d77-af8f-a5be830e68c4/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.04.011/asset/dd0ca4d3-5389-4386-b080-878e40168391/main.assets/figs4.jpg" height="737" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/5a859675-82f6-4465-9834-d0bcaf64ad5f/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.04.011/asset/fea10efc-7d8f-4cd7-95d2-9a04b5ab9f65/main.assets/gr5.jpg" height="760" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/337f3234-ff52-4d6c-b47b-82f6ab98628e/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.04.011/asset/c2a7bfd4-ea95-4c20-b8e1-b25ee29982d4/main.assets/figs5.jpg" height="784" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/8860597d-dd1e-4946-b059-7febbf39fce6/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.04.011/asset/2a7b30e0-073a-4de3-ad4c-3e4e4d8e109d/main.assets/gr6.jpg" height="464" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/fd629031-3eb6-4ae4-8fe7-ec886a09737a/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.04.011/asset/9067d552-f77b-43a1-b294-d8206d0f6acf/main.assets/figs6.jpg" height="911" width="753" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/5fe1d88f-8716-46db-b896-bc785e89c005/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.04.011/asset/bda76e41-88a6-4732-abd8-abcebd1a9f98/main.assets/gr7.jpg" height="843" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/488964cf-621c-45de-99e5-aa647e316428/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.04.011/asset/5537e573-363c-4ea3-abec-f3108489c889/main.assets/figs7.jpg" height="941" width="715" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/f6de66a7-4e04-4ec0-96c2-91c0130370a6/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.04.011/asset/153693ba-8b18-4497-b7a5-d5b8a4010a39/main.assets/gr1.jpg" height="669" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/208f765c-15f2-46af-a641-dc0e1b3a8cd0/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.04.011/asset/7601887c-9c23-4646-bdcd-3dd5cdc5ca31/main.assets/figs1.jpg" height="560" width="753" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/ed3ab668-0e71-4b4b-9348-ed5c38016a03/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.04.011/asset/4a93b725-7bdd-41b5-ac29-d47e783bbb8b/main.assets/gr2.jpg" height="565" width="506" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/69614d67-1762-478a-a0c3-15b8086d46e4/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.04.011/asset/6118a3a0-b223-424c-ae08-41335827f906/main.assets/figs2.jpg" height="808" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/83815183-32e6-47fe-9663-d2f75b27c7ef/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.04.011/asset/fd90e8c9-b5be-4ed4-b04a-ec6e730f8d47/main.assets/figs3.jpg" height="812" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/3999aa1a-40a9-4f72-a41e-056b13c6f2db/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.04.011/asset/27cad41c-42b3-48d1-8033-c683a724a1ec/main.assets/gr3.jpg" height="433" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/9d4b1127-30cc-44b4-ab57-35e2629ce270/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.04.011/asset/89cdb485-f309-4d79-963a-bbc61c901aa7/main.assets/gr4.jpg" height="555" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/68f767d7-d60f-4d77-af8f-a5be830e68c4/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.04.011/asset/dd0ca4d3-5389-4386-b080-878e40168391/main.assets/figs4.jpg" height="737" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/5a859675-82f6-4465-9834-d0bcaf64ad5f/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.04.011/asset/fea10efc-7d8f-4cd7-95d2-9a04b5ab9f65/main.assets/gr5.jpg" height="760" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/337f3234-ff52-4d6c-b47b-82f6ab98628e/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.04.011/asset/c2a7bfd4-ea95-4c20-b8e1-b25ee29982d4/main.assets/figs5.jpg" height="784" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/8860597d-dd1e-4946-b059-7febbf39fce6/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.04.011/asset/2a7b30e0-073a-4de3-ad4c-3e4e4d8e109d/main.assets/gr6.jpg" height="464" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/fd629031-3eb6-4ae4-8fe7-ec886a09737a/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.04.011/asset/9067d552-f77b-43a1-b294-d8206d0f6acf/main.assets/figs6.jpg" height="911" width="753" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/5fe1d88f-8716-46db-b896-bc785e89c005/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.04.011/asset/bda76e41-88a6-4732-abd8-abcebd1a9f98/main.assets/gr7.jpg" height="843" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/488964cf-621c-45de-99e5-aa647e316428/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.04.011/asset/5537e573-363c-4ea3-abec-f3108489c889/main.assets/figs7.jpg" height="941" width="715" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.04.011/asset/f6de66a7-4e04-4ec0-96c2-91c0130370a6/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00458-5';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00458-5';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749593661779","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"185","issueNumber":"12","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867422004585","coverDate":"202206","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Znregulated GTPase metalloprotein activator 1 modulates vertebrate zinc homeostasis","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc389b1ab44293',t:'MTc0OTU5MzY2MS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc389b1ab44293&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2022.04.011&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>